<body>
            <h1>Affinity Purification of GO-Matryoshka Biosensors from <em>E. coli</em> for Quantitative Ratiometric Fluorescence Analyses</h1>
            <p><strong>Authors:</strong> Mayuri Sadoine, Vanessa Castro-RodrÃ­guez, Tobias Poloczek, Helene Javot, Erdem Sunal, Michael M. Wudick, Wolf B. Frommer</p>
            <h2>Abstract</h2>
            <p>Genetically encoded biosensors are powerful tools for quantitative visualization of ions and metabolites <em>in vivo</em>. Design and optimization of such biosensors typically require analyses of large numbers of variants. Sensor properties determined <em>in vitro</em> such as substrate specificity, affinity, response range, dynamic range, and signal-to-noise ratio are important for evaluating <em>in vivo</em> data. This protocol provides a robust methodology for <em>in vitro</em> binding assays of newly designed sensors. Here we present a detailed protocol for purification and <em>in vitro</em> characterization of genetically encoded sensors, exemplified for the His affinity-tagged GO-(Green-Orange) MatryoshCaMP6s calcium sensor. GO-Matryoshka sensors are based on single-step insertion of a cassette containing two nested fluorescent proteins, circularly permutated fluorescent green FP (cpGFP) and Large Stoke Shift LSSmOrange, within the binding protein of interest, producing ratiometric sensors that exploit the analyte-triggered change in fluorescence of a cpGFP.</p>
            <p><strong>Keywords:</strong> Biosensors, GO-Matryoshka, <em>In vitro</em> binding assay, Protein isolation, His-tag, Calcium, Ratiometric, Fluorescence analysis</p>
            <h2>Background</h2><p>The green fluorescent protein (GFP) was identified in 1962 in the jellyfish <em>Aequorea Victoria</em> (Shimomura <em>et al.</em>, 1962). Thirty years later, its first use as a reporter gene was described (Chalfie <em>et al.</em>, 1994). Since their discovery, GFP variants and other fluorescent proteins have contributed greatly to the principal advancements in the biological sciences, and are now common tools in biomedical research (Frommer <em>et al.</em>, 2009).<br/> <br/>A large number of fluorescent proteins (FP) and FP variants have been used as reporters or fused to proteins in organisms of all kingdoms of life (Chudakov <em>et al.</em>, 2010; Valeur and Berberan-Santos, 2012). Continuously, new fluorescent proteins with enhanced properties have been identified or are being engineered, further improving and extending the toolkit for visualization of <em>in vivo</em> processes. These novel fluorophores allow us to monitor a wide range of real-time processes, from structural organization of cells to dynamic processes in living organisms. Photoactivatable FPs have been successfully used to track molecules and cells in space and time (Misteli and Spector, 1997; March <em>et al.</em>, 2003). Recently, FPs enabled the design of biosensors. Different categories of genetically encoded biosensors are commonly used, including single fluorophore intensity-based sensors, and two-fluorophore sensors based on FÃ¶rster resonance energy transfer (FRET) (Frommer <em>et al.</em>, 2009). Various approaches are being used to carry out multiplex sensor analyses in the same cells (Mehta <em>et al.</em>, 2018).<br/> <br/>The mechanisms by which sensors operate can rely on modifications of the FPs themselves, in response to their interaction with a ligand (as in pHluorin or Clomeleon for protons and chloride, respectively) (Miesenbock <em>et al.</em>, 1998; Kuner and Augustine, 2000). Alternatively, FPs can be grafted onto a ligand-binding domain. Ligand binding triggers conformational rearrangements that affect the fluorescence properties or relative positioning of the fluorophores (Deuschle <em>et al.</em>, 2006; Kaper <em>et al.</em>, 2008). Nowadays, genetically encoded biosensors are widely used <em>in vivo</em> to monitor levels and dynamics of ions and metabolites, the activity of transporters or tension (Frommer <em>et al.</em>, 2009). While biosensors that rely on changes in the fluorescence ratio caused by changes in the relative positioning of a pair of donor and acceptor FPs were successfully used, many of the first generation sensors are limited by their comparatively low dynamic range and signal-to-noise ratio [SNR; for definitions of dynamic range and SNR, see Perez Koldenkova and Nagai (2013)]. Some of the advanced single fluorophore intensiometric biosensors, which involve the use of conformational sensitive FPs (csFPs), can have impressively high dynamic ranges and high SNR. However, changes in the expression level of the biosensor will affect the readout, raising concerns about reliability and artifacts for <em>in vivo</em> use. Actual suitability for <em>in vivo</em> measurements requires comparison of different sensor variants in the target tissues (Perez Koldenkova and Nagai, 2013). Nevertheless, the <em>in vitro</em> properties of such sensors are important pieces of information, including quantitative information on affinity and also on kinetics in the case of rapid processes such as action potentials. To avoid artifacts, intensiometric sensors can be converted into ratiometric sensors by coupling csFPs with a reference FP.<br/> <br/>Recently, a new technology, termed Matryoshka, provided a universal platform to create dual-FP biosensors with a large dynamic range, good stability across a wide range of pH and buffer conditions, and single wavelength excitation thanks to the use of a Large Stokes Shift (LSS) FP (Ast <em>et al.</em>, 2017). Nesting such a reference FP (<em>e.g.</em>, LSSmOrange) within a reporter FP (<em>e.g.</em>, circularly permuted green FP), permits excitation of both FPs at a single excitation wavelength. The technology was successfully applied to generate cytosolic calcium and ammonium transport sensors, GO-MatryoshCaMP and AmTryoshka variants, respectively (Ast <em>et al.</em>, 2017).<br/> <br/>Here, we present a detailed protocol for the purification and characterization by <em>in vitro</em> binding assays of genetically encoded His-tagged Matryoshka biosensors from <em>Escherichia coli</em> K12. The protocol presented here was specifically developed for purification and characterization of GO-MatryoshCaMP6s, but can be used more generally for other fluorescence biosensors with minor modifications.</p>
        </body>

<p class="pptt" id="biaoti37275">## Materials and Reagents</p><ol class="npwtwok"><li>Micropipette tips (Gilson<sup>TM</sup> PIPETMAN Classic<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> Gilson F12360x </a> and, Starlab, TipOne<sup>Â®</sup>, catalog numbers: <a class="blackNoUnder" target="_blank"> 1112-1840 </a> , <a class="blackNoUnder" target="_blank"> 1110-1840 </a> , <a class="blackNoUnder" target="_blank"> 1110-3800 </a> )<br/><em>Note: Pipetting accuracy is essential for the acquisition of reliable raw data. It may help to use of an electrical multichannel pipette (for 100 Âµl) (e.g., Sartorius AG)</em> </li><li>200 Âµl TipOne<sup>Â®</sup> pipette tips (sterile), yellow, conical, rack (STARLAB, catalog number: <a class="blackNoUnder" target="_blank"> S1111-6701-C </a> )</li><li>0.22-Î¼m filter</li><li>Single-use plastic serological pipettes (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 86.1254.001 </a> )</li><li>1 and 2 liter glass flasks for cell culture (SciLabware, catalog numbers: <a class="blackNoUnder" target="_blank"> 1135/26D and 1135/30D </a> )</li><li>50 ml tubes (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 62.547.004 </a> )</li><li>15 ml tubes (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 62.554.002 </a> )</li><li>1.5 ml microtubes (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 72.690.001 </a> )</li><li>Stellar Scientific Centrifuge Tubes, High Speed (20,000 <em>x g</em>), 50 ml, Red Screw Cap (Stellar Scientific, catalog number: <a class="blackNoUnder" target="_blank"> T15-701 </a> )</li><li>Stellar Scientific Centrifuge Tubes, High Speed (20,000 <em>x g</em>), 15 ml, Red Screw Cap (Stellar Scientific, catalog number: <a class="blackNoUnder" target="_blank"> T15-701 </a> )</li><li>Round 92 x 16 mm Petri dishes (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 82.1473 </a> )</li><li>Disposable plastic columns (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 29922 </a> )</li><li>Non-sterile 96-well flat bottom transparent plates (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 9017 </a> ) </li><li>Non-sterile 96-well flat bottom black plates with transparent bottoms (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 3631 </a> )</li><li>Zeba<sup>TM</sup> Spin desalting column (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 89882 </a> )</li><li>Purified plasmids containing His-tagged biosensors for lactose/IPTG inducible expression (<em>e.g.</em>, the calcium sensor MatryoshCaMP6 (Ast <em>et al.</em>, 2017) in the vector pRSET-B) for nickel-nitrilotriacetic acid (Ni-NTA) affinity chromatography, as well as an empty vector, or the biosensor backbone protein lacking FPs (negative control) for expression in BL21 GOLD (DE3) cells. Plasmids and strains are described in more detail below</li><li>Strains<br/><em>Escherichia coli</em> BL21 GOLD (DE3) [<em>fhuA2 [lon] ompT gal (Î» DE3) [dcm] âhsdS; Î» DE3 = Î» sBamHIo âEcoRI-B int::(lacI::PlacUV5::T7 gene1) i21 ânin5</em>] (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> C2527I </a> )</li><li>cOmplete<sup>TM</sup> ULTRA Tablets, Mini, EDTA-free protease inhibitor cocktail (Merck, catalog number: <a class="blackNoUnder" target="_blank"> 11836170001 </a> )</li><li>Ni-NTA Agarose Beads (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 30210 </a> )</li><li>Bradford assay kit (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 500-0006 </a> )</li><li>MOPS (3-(N-morpholino)propanesulfonic acid) buffer (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 6979.3 </a> )</li><li>Imidazole (AppliChem, catalog number: <a class="blackNoUnder" target="_blank"> A1073 </a> ,0500; UN3263)</li><li>D-(+) glucose monohydrate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 14431-43-7 </a> )</li><li>Lactose (Merk, catalog number: <a class="blackNoUnder" target="_blank"> 7660.025 </a> 0)</li><li>BSA (Bovine Serum Albumin; Carl Roth, catalog number: 0<a class="blackNoUnder" target="_blank"> 163.3 </a> )</li><li>Bacto<sup>TM</sup> Tryptone (BD BioSciences, catalog number: <a class="blackNoUnder" target="_blank"> 211705 </a> )</li><li>Bacto<sup>TM</sup> Yeast Extract (BD BioSciences, catalog number: <a class="blackNoUnder" target="_blank"> 212750 </a> )</li><li>Sodium chloride (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 3957.1 </a> )</li><li>Bacto<sup>TM</sup> Agar (BD Biosciences, catalog number: <a class="blackNoUnder" target="_blank"> 214030 </a> )</li><li>Antibiotic(s) required for plasmid selection. For example: ampicillin for vector pRSET-B containing MatryoshCaMP6 (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 69-53-4 </a> )</li><li>Sodium hydroxide (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 1310-73-2 </a> )</li><li>Calcium chloride (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 10043-52-4 </a> )</li><li>4-20% Mini-PROTEAN<sup>Â®</sup> TGX<sup>TM</sup> Precast Protein Gels, 10-well, 50 Âµl (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 4561094 </a> )</li><li>4x SDS Sample buffer (MERK, catalog number: <a class="blackNoUnder" target="_blank"> 70607-3 </a> ) </li><li>PageRuler Prestained Protein Ladder (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 26616 </a> )</li><li>Pierce<sup>TM</sup> 6xHis Protein Tag Stain Reagent Set (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 24570 </a> )</li><li>Lysogeny broth (LB) liquid medium with antibiotics (see Recipes)</li><li>LB solid medium (see Recipes)</li><li>Auto-induction medium (for composition and explanations, see Recipes)</li><li>Lysis buffer (see Recipes)</li><li>Wash buffer (see Recipes)</li><li>Elution buffer (see Recipes)</li><li>Final buffer (see Recipes)</li><li>Ligand binding assay buffer (see Recipes)</li><li>Coomassie stain buffer (see Recipes)</li><li>Stock solution of carbenicillin (see Recipes)</li><li>Running Buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti37276">### Equipment</p><ol class="npwtwok"><li>Standard micropipette or multichannel (8 or 12 channel) pipette (for 100 Âµl) (<em>e.g.</em>, Sartorius AG, catalog number: <a class="blackNoUnder" target="_blank"> 725240 </a> )</li><li>pH meter (InoLab<sup>Â®</sup> pH Level1, catalog number: <a class="blackNoUnder" target="_blank"> 72.690.001 </a> )</li><li>Sonicator (Branson Sonifier cell disruptor B15)</li><li>Centrifuge (Hettich Rotanta <a class="blackNoUnder" target="_blank"> 460/460R </a> )</li><li>Microplate reader with adjustable bandwidths (Spark<sup>Â®</sup>, Tecan)</li><li>Fluorescence stereomicroscope for large fields observations (ZEISS, model: <a class="blackNoUnder" target="_blank"> Axio Zoom.V16 </a> )</li><li>Eppendorf ThermoMixer<sup>Â®</sup> C (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 5382000015 </a> )</li><li>Autoclave (Systec GmbH, model: <a class="blackNoUnder" target="_blank"> Systec V-150 </a> )</li><li>Mini-PROTEAN<sup>Â®</sup> Tetra Vertical Electrophoresis Cell (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1658004 </a> )</li><li>NanoDrop 2000 (LabX, catalog number: <a class="blackNoUnder" target="_blank"> LV40609601 </a> )</li><li>Eppendorf ThermoMixer<sup>Â®</sup> C (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 5382000015 </a> )</li><li>Eppendorf Thermoblock SmartBlock<sup>TM</sup> 1.5 ml (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 5360000038 </a> )</li><li>Gel Doc XR+ Gel Documentation System (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1708195 </a> )<br/></li></ol><p class="pptt" id="biaoti37277">### Software</p><ol class="npwtwok"><li>ecan-related software (Spark<sup>Â®</sup>, Tecan Life Sciences)</li><li>Excel (Microsoft, Microsoft Office Professional 2016)</li><li>MyCurveFit 2019 (MyCurveFit, MyAssays Limited)<br/></li></ol><p class="pptt" id="biaoti37278">### Procedure</p><ol class="npwone"><li>Preparation of purified ratiometric fluorescent biosensors<ol class="npwtwo"><li>Transformation of <em>E. coli</em><br/><ol class="npwthree"><li>Thaw the DNA (minipreps of plasmids carrying the biosensors and negative controls. <em>e.g.</em>, MatryoshCaMP6 (Ast <em>et al.</em>, 2017) and the vector pRSET-B) and aliquots of BL21 GOLD (DE3) competent cells on ice for 10 min.<br/></li><li>Add 10-50 ng of plasmid to the cells and keep on ice for 30 min.<br/><em>Note: DNA quantification can be performed by using a NanoDrop device.</em><br/></li><li>Perform heat-shock by using a heat block at 42 Â°C for 45-60 s.<br/></li><li>Keep on ice for 2 min.<br/></li><li>Recover cells for 1 h with 800 Âµl of LB medium by shaking at 200 rpm at 37 Â°C.<br/><em>Note: SOC (Super optimal broth) medium, a nutrient-rich bacterial growth medium, can be used instead of LB medium for higher recovery after transformation.</em><br/></li><li>Centrifuge at 11,000 <em>x g</em> for 5-10 s.<br/></li><li>Remove most of the supernatant and keep about 100-200 Âµl.<br/></li><li>Resuspend sedimented cells in remaining LB medium and plate dilutions onto solid LB plates (dilutions depend on transformation efficacy) containing the respective antibiotic, here carbenicillin at 100 Î¼g/ml (see Recipes).<br/><em>Note: To optimize colony density, plate different volumes (i.e., 30 and 170 Âµl) or dilute.</em><br/></li><li>Allow bacteria to grow on solid agar overnight at 37 Â°C.<br/></li><li>Store plates at 4 Â°C overnight to achieve fluorophore maturation.</li></ol></li><li>Clone selection and cell culture<br/><ol class="npwthree"><li>Visualize cpGFP fluorescence under a fluorescence stereomicroscope and pick bright colonies (Figure 1).<br/><em>Note: Leaky expression is observed on LB solid plates. Brightness of colonies can be used as a criterion to evaluate the sensor expression levels.</em><br/><br/><img alt="" class="layerphoto" height="236" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200927/20200927102303_4292.png" width="300"/><br/><strong>Figure 1. <em>E. coli</em> cells transformed with empty vector and GO-MatryoshCaMP6s on solid LB plates.</strong> A-B. Colonies obtained after transformation with empty vector. C-D. Cells transformed with GO-MatryoshCaMP6s sensor. Fluorescence of the <em>E. coli</em> cells was recorded using a ZEISS <a class="blackNoUnder" target="_blank"> Axio Zoom.V16 </a> fluorescence stereo zoom microscope and GFP filter settings (Î»<sub>ex</sub> 470/40 nm and Î»<sub>em</sub> 525/50 nm, with a beam splitter at Î»<sub>em</sub> 495 nm). Colonies were visualized using bright-field illumination (A, C) and GFP excitation for fluorescence detection (B, D). Scale bars = 1 mm.<br/><br/></li><li>For each of the 3 biological replicate, inoculate 5 ml of LB liquid medium (with antibiotics for plasmid selection) with 1 independent bright colony and grow overnight in LB plus antibiotic at 37 Â°C and at 200 rpm shaking (Figure 2).<br/></li><li>For each biological replicate, inoculate 200 ml of Auto-induction medium containing antibiotics (see Recipes) with the 5 ml preculture in a large flask and grow at 37 Â°C for 2 h in the dark on a shaker at 220 rpm (Figure 2). Use sample from the remaining preculture for generating glycerol stocks.<br/><em>Notes:</em><br/><ol class="npwfour"><li><em>To get optimal protein yield and avoid toxicity, Auto-induction media (details available <a href="http://www.nzytech.com/products-services/molecular-biology/competent-cells-media/cell-preparation-growth/auto-induction-media/mb179/" target="_blank">here</a>) should be used instead of IPTG, and cells expressing biosensors should be grown at 20 Â°C rather than 37 Â°C. The Auto-induction medium is used for obtaining high-levels of recombinant protein expression of lactose-inducible expression-systems. It does not require the addition of isopropyl Î²-D-1-thiogalactopyranoside (IPTG) and it avoids the constraints of monitoring optical density (O.D.) for the determination of the IPTG supplementation time. The medium contains glucose and lactose that are metabolized at different rates. Glucose is favored initially and represses promoter activity. Once glucose is consumed, repression will be relieved and lactose leads to induction of transcription.</em><br/></li><li><em>Auto-induction media are used for lactose/IPTG inducible expression systems otherwise this part of the protocol has to be customized.</em><br/></li><li><em>Glycerol stocks are generated by mixing cell culture 1:1 with LB medium containing 50% w/v glycerol.</em><br/></li><li><em>When growing 200 ml cultures, flasks should have a volume of 1 or 2 L. Do not fill flasks beyond 20% of total volume to ensure optimal oxygenation.</em><br/></li><li><em>The initial incubation at 37 Â°C is important to initiate faster the growth of large cultures from a single colony. Do not extend the incubation time, as it may later lead to a lower amount of soluble biosensors and an increased occurrence of inclusion bodies.</em> </li></ol></li><li>Transfer the culture to 20 Â°C and grow for 48 h at 220 rpm in the dark to avoid any photo-bleaching of FPs (Figure 2).<br/></li><li>Centrifuge the culture by using high-speed 50 ml centrifuge tubes for 30 min at 20,000 <em>x g</em> at 4 Â°C, discard the supernatant and freeze the sedimented cells at -20 Â°C overnight (Figure 2). Do not skip this freezing step, as it helps the purification process. A yellowish sediment of cells expressing the sensor should be visible.<br/><em>Notes:</em><br/><ol class="npwfour"><li><em>The yield of purified proteins may be increased by freezing and thawing the pellet, which helps to disrupt the cells.</em><br/></li><li><em>High-speed centrifuge tubes for 20,000 x g should be used to avoid breaking of the tubes during centrifugation. Alternatively: Normal falcon tubes can be used by reducing the speed to 17,000 x g in a way to prevent any leakage.</em><br/></li></ol></li></ol><br/><img alt="" class="layerphoto" height="298" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200927/20200927102320_2549.png" width="300"/><br/><strong>Figure 2. Expression of GO-Matryoshka biosensors in </strong><em><strong>E. coli.</strong></em> Workflow of the experiment starting from the screening of the colonies until the harvesting of the expressed biosensor as a bright yellow sediment.<br/><br/></li><li>Biosensor purification<br/><ol class="npwthree"><li>Thaw the pellet on ice.<br/></li><li>Immediately resuspend the cells by pipetting up and down in 5 ml of lysis buffer containing protease inhibitor cocktail.<br/></li><li>Sonicate the cell suspension into the tubes on ice (10-15 cycles, with each cycle consisting of 10 s of sonication and 10-15 s of rest). Be careful with this step since it is important to optimally break the cells while at the same time preventing overheating samples, which may lead to protein denaturation. This step is critical for the quality of the protein sample.<br/><em>Note: Do not heat the sample (e.g., during sonication), which could lead to undesired protein denaturation and aggregation.</em><br/></li><li>Centrifuge the lysate by using high-speed 15 ml centrifuge tubes (4 Â°C; 20,000 <em>x g</em>) for at least 30 min.<br/></li><li>During centrifugation, prepare the column by adding 1-2 ml final volume of Ni-NTA agarose bead material into a column (placed on its holder).<br/></li><li>At room temperature, wash the Ni-NTA beads three times by adding 5-10 ml wash buffer to the column, then let it flow through by gravity.<br/></li><li>Add the lysate to the column. Allow the lysate to flow through by gravity. The His-tagged biosensors will bind to the Ni-NTA beads.<br/></li><li>Wash three times by gravity flow with 5-10 ml wash buffer.<br/></li><li>Elute the sensors by gravity flow with up to 5 successive fractions of 300 Âµl of elution buffer and store in the dark at 4 Â°C for at least 24 h before analysis to allow the maturation of all purified biosensors. Alternatively: incubate the elution fraction for 3 h at 37 Â°C.<br/><em>Note: Due to the longer maturation time of the LSSmOrange compared to derivatives (Shcherbakova et al., 2012), incubation at 4 degrees is crucial to obtain fully matured FPs.</em><br/></li><li>To remove imidazole, salt contaminants and bound ligands, the eluate can be further purified with a Zeba<sup>TM</sup> Spin desalting column in a final volume of 130 Âµl according to the manufacturerâs instructions.</li></ol></li><li>Quality control of purified biosensor <br/>Perform a Bradford assay to determine protein concentration according to manufacturerâs protocol. It is recommended that the concentration is at least about 1 mg/ml otherwise it is recommended to repeat the purification with a reduced elution volume.<br/><em>Note: Biosensors can be stored at 4 Â°C in the dark for a couple of weeks.</em><br/><br/></li></ol></li><li>Load 10-20 Âµg of protein samples into the SDS-Polyacrylamide gel (SDS-PAGE) (Figure 3).<ol class="npwtwo"><li>Mix the samples with 1x SDS Sample buffer and add water to a final volume of 20-40 Âµl.</li><li>Boil the sample at 90 Â°C during 5 min to speed up the process of denaturation. <br/><em>Note: Once denatured, the samples can be kept at room temperature until loading.</em> </li><li>Load the sample into the wells of SDS-PAGE gel.</li><li>After electrophoresis, proceed to the staining of 6xHis affinity-tagged protein (Pierce 6xHis Stain Reagent Set) according to the manufacturerâs instructions and to Coomassie staining for total protein visualization.</li><li>Perform protein detection by using visible light (Coomassie staining) and UV-light excitation at a wavelength 280-310 nm (6xHis affinity-tagged staining).<br/><em>Note: The staining of 6xHis affinity-tagged proteins requires 5.6 pmol of His-tagged proteins per band for detection with a CCD camera or 57 pmol of His-tagged proteins per band for detection with a UV transilluminator.</em><br/><br/><img alt="" class="layerphoto" height="305" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200927/20200927102339_4417.png" width="300"/><br/><strong>Figure 3. Isolation of GO-MatryoshCaMP6s sensors expressed in </strong><em><strong>E. coli.</strong></em> Timeline of the experiment starting from the sedimented cells until the elution of the biosensor by affinity chromatography. Samples loaded into the SDS-PAGE gel are: MW (prestained molecular mass marker), CE (crude extract), L (lysate), FT (flow-through), W (wash), and E1-E5 (eluates from fractions 1 to 5). Upper gel panel was stained for total protein visualization using Coomassie stain buffer and lower gel panel was stained with 6xHis Protein Tag Stain Reagent Set (Thermo Fisher Scientific). Detection was performed by using UV-light excitation at a wavelength 280-310 nm.<br/><br/></li></ol></li><li><em>In vitro</em> binding assay with purified biosensors<ol class="npwtwo"><li>Optimization of the sample dilution in final buffer (<em>i.e.</em>, 20 mM MOPS, pH 7.0, see Recipes) and fluorescence reading parameters<br/>In this section, protocol users will identify the proper dilutions required for each purified biosensor solution based on individual excitation of cpGFP and LSSmOrange using their respective maximum excitation and emission wavelengths. Users will also identify a single wavelength that provides the greatest simultaneous excitation of both cpGFP and LSSmOrange.<br/><ol class="npwthree"><li>Select the fraction with the highest protein content and make a predilution of about 40 times to obtain a final concentration of about 5-10 ÂµM.<br/></li><li>From the pre-diluted sample, prepare a dilution series (1; 1/2; 1/3; 1/4; 1/5; 1/10; 1/100) in a final volume of 200 Âµl and load into a 96-well plate suitable for reading fluorescence intensities and spectra in a microplate reader (see note).<br/><em>Note: Although best results are obtained with black transparent bottom 96-well plates, their cost is comparatively high. The use of this black plastic helps preventing signal contamination between wells and is therefore favored when precise measurements are needed. However, for preliminary sensor characterization, we use regular non-sterile 96-well plates to minimize costs. Alternatively, to avoid light emission contamination between wells, empty wells can be placed systematically in-between samples.</em><br/></li><li>Measure cpGFP and LSSmOrange fluorescence intensities of dilution series in the plate reader using the SPARKControl software and the following parameters: (1) cpGFP: Excitation at 480 nm with a band width of 20 nm, emission at 515 nm with a band width of 15 nm and (2) LSSmOrange: Excitation at 440 nm with a band width of 10 nm, emission at 570 nm with a band width of 15 nm. Adjust the gain and z-position of the reading so that the measurements fall roughly within the middle of the detector range. The gain is expected between 50-90 and should never exceed 130 while the Z position, when using 200 Âµl, is expected at about 20,000 Î¼m.<br/><em>Note: While it is possible to find a compromise by using a single intermediate wavelength to excite both FPs, their excitation maxima are distinct, thus for in vitro characterization in a fast fluorimeter, it is better to excite the two FPs separately.</em><br/></li><li>Perform an excitation scan with 2 nm intervals between 440 to 480 nm to verify which excitation wavelength provides the highest amplitude for both GFP and the LSSmOrange emission in your conditions (Figure 4). This is typically around 455 nm for GO-Matryoshka biosensors.<br/><br/><img alt="" class="layerphoto" height="273" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200927/20200927102405_3509.png" width="300"/><br/> <strong>Figure 4. Excitation parameters for the detection of GO-Matryoshka sensors.</strong> Emission spectra of MatryoshCaMP6 reveal that increasing the excitation wavelength from 440 nm to 480 nm results in a rise in fluorescence intensity of cpGFP emission but a decrease in the emission of the stable reference LSSmOrange. A.u.: arbitrary units.<br/><br/></li><li>Select the optimal dilution factor based on emission intensities obtained in Steps C1c and C1d. The dilution factor depends on each instrument, here the targeted range of intensity is of about 20,000 arbitrary units (a.u.) to stay within the optimal range of the detectors without leading to saturation. Lower gain factors of the instrument will yield higher quality data (less noise).</li></ol></li><li>Binding assays<br/><ol class="npwthree"><li>Design the sample loading pattern of your plate by including technical triplicates and purifications of lysate from 3 biological replicates for each biosensor. Depending on the expected <em>K</em><sub>d</sub>, include suitable concentrations of the ligand (<em>e.g.</em>, dilution series from 0 mM up to 5 times the <em>K</em><sub>d</sub>; use sufficient data points to obtain reliable apo and saturated states as well as linear range of binding isotherm). Make sure that the previously selected dilution factor is compatible with this design, and include control wells that only contain buffer to measure background.<br/><em>Note: The order of components added does not matter. However, we recommend to add the component with the highest volume first, the buffer and end with the lowest volume, the biosensors. Load the 96-well plate according to your loading pattern. Technical triplicates for each of the three colonies will be loaded from a single Master Mix (see notes). Make sure to scale Master Mix for 4 reactions for technical triplicates, to maintain a margin of 1 reaction when pipetting triplicates. In addition, make sure that the final volume in each well remains identical, even with different ligand concentrations. Bubbles in the wells should be avoided, as they might affect the reading.</em><br/><em>Notes:</em><br/><ol class="npwfour"><li><em>Since the analysis is quantitative, accurate pipetting is crucial. Use high precision, use best practices when pipetting and handling pipettes, ensure that pipettes are in good state, ensure proper calibration of pipettes, prepare master mix (i.e., to avoid pipetting low volumes) and pipet slowly in order to reduce pipetting errors. Do not snap plunger, never allow any liquid to enter the shaft.</em><br/></li><li><em>Incubation time is recommended to be minimum 5 min. It is necessary to ensure to have reached binding equilibrium before performing the reading. The preparation of the 96-well plate is ususally taking a couple of minutes which is sufficient.</em><br/></li><li><em>It is recommended to include buffer-only wells as blanks.</em> </li></ol></li><li>Read fluorescence using the plate reader as described in Step C1 to obtain three sets of data for each well: Emission spectrum and point measurements for cpGFP and LSSmOrange of maximum intensities of the GO-Matryoshka sensor (Figure 5).</li></ol><br/><img alt="" class="layerphoto" height="416" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200927/20200927102422_3212.png" width="300"/><br/><strong>Figure 5. Screenshot of an excel worksheet displaying an example of data and parameters from one binding assay experiments.</strong> Three set of data are obtained after performing point measurements and emission spectrum scan of GO-MatryoshCaMP6s in biosensor binding assay.<br/><br/></li><li>Analysis of the data for each biological replicate<br/><ol class="npwthree"><li>Set a default Excel worksheet, which can be used as template for automated analysis of data obtained in the previous section (Figure 5). This should be done by entering the required formula for processing data and generating graphs from any experiment by simply pasting its raw values. This template has to include:<br/><ol class="npwfour"><li>Calculation of the average value for the technical triplicates.<br/></li><li>Subtraction of the background fluorescence of the buffer for each condition and each data type (cpGFP and LSSmOrange intensities as well as the emission spectrum).<br/></li><li>Plotting the data by graphically representing the fluorescence intensity as a function of wavelength and generate emission spectra recorded at different ligand concentrations to quantify intensity changes responding to analyte concentration (Bermejo <em>et al.</em>, 2011) (Figure 6).</li></ol><br/><img alt="" class="layerphoto" height="134" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200927/20200927102441_2684.png" width="300"/><br/><strong>Figure 6. Binding assay using purified Matryoshka (</strong><em><strong>here</strong></em><strong> GO-MatryoshCaMP6s) biosensors.</strong> A. 96-well plate with GO-MatryoshCaMP6s plus increasing calcium concentrations, color code from green (without calcium) to red (10 ÂµM of calcium), and in triplicate. B. The spectrum of MatryoshCaMP6, with excitation at 455 nm. Increasing concentrations of calcium result in a rise in the fluorescence intensity of cpGFP emission while the reference LSSmOrange emission stay stable (slight increase due to bleedthrough). [a.u.]: arbitrary units. Data are presented as the average of three technical replicates.<br/><em>Note: It is observed that bleed-through occurs and leads to a slight but significant apparent increase in fluorescence intensity of the reference FP LSSmOrange. However, separate point measurements by using two excitation wavelengths should get rid of this effect.</em><br/><br/></li><li>Measure the affinity by using three biological replicates:<br/><ol class="npwfour"><li>Calculate the ratio (R) of intensities at 520 nm/575 nm emission for each ligand concentration by using the following equation:<br/>R = FI<sub>520nm</sub>/FI<sub>575nm</sub><br/></li><li>Calculate R<sub>0</sub> as the value of R without substrate.<br/></li><li>Calculate âR the value of R at a specific calcium concentration subtracted by R<sub>0</sub> as follows:<br/>âR = R - R<sub>0</sub> </li><li>Calculate âR/R<sub>0</sub>.<br/></li><li>Plot the data against the concentration by using <em>e.g.</em>, MyCurveFit.<br/></li><li>Use a logarithmic scale for the X axis.<br/></li><li>Apply a sigmoidal fit by using, <em>e.g.</em>, MyCurveFit, and calculate the <em>K</em><sub>d</sub> (Figure 7).</li></ol></li></ol><br/><img alt="" class="layerphoto" height="149" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200927/20200927102502_1543.png" width="300"/><br/><strong>Figure 7. </strong><em><strong>In vitro</strong></em><strong> saturation curves of purified GO-Matryoshka.</strong> Calcium binding isotherms of MatryoshCaMP6s. Analyses of sensor output at different calcium concentrations was performed using purified sensor. Calcium binding was determined as previously described (Ast <em>et al.</em>, 2017). Data are presented as the average of three biological replicates (mean Â± S.E.M. from biological replicates with n = 3).<br/></li></ol></li></ol><p class="pptt" id="biaoti37279">### Recipes</p><ol class="npwtwok"><li>LB liquid medium with antibiotics<br/><ol class="npwthree"><li>Dissolve 10 g Bacto<sup>TM</sup> tryptone, 10 g sodium chloride and 5 g Bacto<sup>TM</sup> yeast extract in 800 ml MilliQ water<br/></li><li>Adjust to pH 7.0 with NaOH<br/></li><li>Adjust final volume to 1 L with MilliQ water<br/></li><li>Autoclave at 121 Â°C for 20 min<br/></li><li>Before use, add the antibiotics corresponding to the required antibiotic resistance used for selection</li></ol></li><li>LB solid medium (pH 7.0)<br/><ol class="npwthree"><li>Dissolve 10 g Bacto<sup>TM</sup> tryptone, 10 g sodium chloride and 5 g Bacto<sup>TM</sup> yeast extract and 15 g of agar in 800 ml MilliQ water<br/></li><li>Adjust to pH 7.0 with NaOH<br/></li><li>Adjust final volume to 1 L with MilliQ water<br/></li><li>Autoclave at 121 Â°C for 20 min<br/></li><li>Allow to cool down to about 50-60 Â°C, then add the antibiotics corresponding to the required antibiotic resistance used for selection</li></ol></li><li>Auto-induction medium (pH 7.0)<br/>LB liquid medium<br/>0.05% w/v D-glucose<br/>0.2% w/v lactose</li><li>Lysis buffer (pH 7.0)<br/>20 mM MOPS<br/>1 cOmplete<sup>TM</sup> ULTRA Tablet Mini protease inhibitor cocktail per 50 ml of solution<br/><em>Note: The buffer can be stored at 4 degrees for months. 1 tablet of cOmplete<sup>TM</sup> ULTRA Tablet Mini protease inhibitor cocktail is dissolved by extensive vortexing.</em> </li><li>Wash buffer (pH 7.0)<br/>20 mM MOPS<br/>1 cOmplete<sup>TM</sup> ULTRA Tablet Mini protease inhibitor cocktail per 50 ml of solution<br/><em>Note: The buffer can be stored at 4 degrees for months. 1 tablet of cOmplete<sup>TM</sup> ULTRA Tablet Mini protease inhibitor cocktail is dissolved by extensive vortexing.</em> </li><li>Elution buffer (pH 7.0):<br/>20 mM MOPS buffer<br/>250 mM imidazole<br/>1 cOmplete<sup>TM</sup> ULTRA Tablet Mini protease inhibitor cocktail per 50 ml of solution<br/><em>Note: The buffer can be stored at 4 degrees. for months 1 tablet of cOmplete<sup>TM</sup> ULTRA Tablet Mini protease inhibitor cocktail is dissolved by extensive vortexing.</em> </li><li>Final buffer (pH 7.0)<br/>20 mM MOPS<br/><em>Note: The buffer can be stored at 4 degrees for months. 1 tablet of cOmplete<sup>TM</sup> ULTRA Tablet Mini protease inhibitor cocktail is dissolved by extensive vortexing.</em> </li><li>Ligand binding assay buffer (pH 7.0)<br/>20 mM MOPS with 0 to 10-20 mM calcium chloride<br/><em>Note: The buffer can be stored at 4 degrees for months. 1 tablet of cOmplete<sup>TM</sup> ULTRA Tablet Mini protease inhibitor cocktail is dissolved by extensive vortexing.</em> </li><li>Coomassie staining buffer<br/>Coomassie Blue R250 staining reagent<br/>10% v/v acetic acid<br/>50% v/v methanol</li><li>Stock solution of carbenicillin:<br/><ol class="npwthree"><li>Dissolve 1 g of disodium carbenicillin in 9.5 ml of MilliQ water<br/></li><li>Adjust final volume to 10 ml<br/></li><li>Filter-sterilize by using a 0.22-Î¼m filter</li></ol><em>Note: Stock solution of carbenicillin can be stored at -20 Â°C.</em> </li><li>Running Buffer<br/>25 mM Tris<br/>192 mM glycine<br/>0.1% (w/v) SDS<br/></li></ol><p class="pptt" id="biaoti37280">### Acknowledgments</p><p class="ppzz">The authors gratefully acknowledge grant support from Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under Germanyâs Excellence StrategyâEXC-2048/1âproject ID 390686111, and the Alexander von Humboldt Professorship to WBF. Support from the CEA-Enhanced Eurotalents program and ANR-19-CE13-0007 âPHLOWZâ is gratefully acknowledged (HJ).</p><p class="pptt" id="biaoti37281">### Competing interests</p><p class="ppzz">W.B.F. is author of the patent filed for the Matryoshka technology (U.S. Appl. No. 15/438,078). The remaining authors have no conï¬ict of interest or competing interest to declare.</p><p class="pptt" id="biaoti37282">### References</p><ol class="npwtwok"><li>Ast, C., Foret, J., Oltrogge, L. M., De Michele, R., Kleist, T. J., Ho, C. H. and Frommer, W. B. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28874729" target="_blank">Ratiometric Matryoshka biosensors from a nested cassette of green- and orange-emitting fluorescent proteins.</a> <em>Nat Commun</em> 8(1): 431.</li><li>Bermejo, C., Haerizadeh, F., Takanaga, H., Chermak, D. and Frommer, W. B. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22036883" target="_blank">Optical sensors for measuring dynamic changes of cytosolic metabolite levels in yeast.</a> <em>Nat Protoc</em> 6(11): 1806-1817.</li><li>Chalfie, M., Tu, Y., Euskirchen, G., Ward, W. W. and Prasher, D. C. (1994). <a href="http://www.ncbi.nlm.nih.gov/pubmed/8303295" target="_blank">Green fluorescent protein as a marker for gene expression.</a> <em>Science</em> 263(5148): 802-805.</li><li>Chudakov, D. M., Matz, M. V., Lukyanov, S. and Lukyanov, K. A. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20664080" target="_blank">Fluorescent proteins and their applications in imaging living cells and tissues.</a> <em>Physiol Rev</em> 90(3): 1103-1163.</li><li>Deuschle, K., Chaudhuri, B., Okumoto, S., Lager, I., Lalonde, S. and Frommer, W. B. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16935985" target="_blank">Rapid metabolism of glucose detected with FRET glucose nanosensors in epidermal cells and intact roots of Arabidopsis RNA-silencing mutants.</a> <em>Plant Cell</em> 18(9): 2314-2325.</li><li>Frommer, W. B., Davidson, M. W. and Campbell, R. E. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19771330" target="_blank">Genetically encoded biosensors based on engineered fluorescent proteins.</a> <em>Chem Soc Rev</em> 38(10): 2833-2841.</li><li>Kaper, T., Lager, I., Looger, L. L., Chermak, D. and Frommer, W. B. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18522753" target="_blank">Fluorescence resonance energy transfer sensors for quantitative monitoring of pentose and disaccharide accumulation in bacteria.</a> <em>Biotechnol Biofuels</em> 1(1): 11.</li><li>Kuner, T. and Augustine, G. J. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11055428" target="_blank">A genetically encoded ratiometric indicator for chloride: capturing chloride transients in cultured hippocampal neurons.</a> <em>Neuron</em> 27(3): 447-459.</li><li>March, J. C., Rao, G. and Bentley, W. E. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12768245" target="_blank">Biotechnological applications of green fluorescent protein.</a> <em>Appl Microbiol Biotechnol</em> 62(4): 303-315.</li><li>Mehta, S., Zhang, Y., Roth, R. H., Zhang, J. F., Mo, A., Tenner, B., Huganir, R. L. and Zhang, J. (2018). <a href="https://doi.org/10.1038/s41556-018-0200-6" target="_blank">Single-fluorophore biosensors for sensitive and multiplexed detection of signalling activities.</a> <em>Nat Cell Biol </em>20(10): 1215-1225.</li><li>Miesenbock, G., De Angelis, D. A. and Rothman, J. E. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9671304" target="_blank">Visualizing secretion and synaptic transmission with pH-sensitive green fluorescent proteins.</a> <em>Nature</em> 394(6689): 192-195.</li><li>Misteli, T. and Spector, D. L. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9335045" target="_blank">Applications of the green fluorescent protein in cell biology and biotechnology.</a> <em>Nat Biotechnol</em> 15(10): 961-964.</li><li>Perez Koldenkova, V. and Nagai, T. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23352808" target="_blank">Genetically encoded Ca(2+) indicators: properties and evaluation.</a> <em>Biochim Biophys Acta</em> 1833(7): 1787-1797.</li><li>Shimomura, O., Johnson, F. H. and Saiga, Y. (1962). <a href="http://www.ncbi.nlm.nih.gov/pubmed/13911999" target="_blank">Extraction, purification and properties of aequorin, a bioluminescent protein from the luminous hydromedusan, Aequorea.</a> <em>J Cell Comp Physiol</em> 59: 223-239.</li><li>Valeur, B. and Berberan-Santos, M. N. (2012). <em><a href="https://onlinelibrary.wiley.com/doi/book/10.1002/9783527650002" target="_blank">Molecular fluorescence: principles and applications.</a></em> John Wiley &amp; Sons.</li><li>Shcherbakova, D. M., Hink, M. A., Joosen, L., Gadella, T. W. and Verkhusha, V. V. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22486524" target="_blank">An orange fluorescent protein with a large Stokes shift for single-excitation multicolor FCCS and FRET imaging.</a> <em>J Am Chem Soc</em> 134(18): 7913-7923.<br/></li></ol>