<body>
            <h1>TetR Regulated <em>in vivo</em> Repression Technology to Identify Conditional Gene Silencing in Genetically Engineerable Bacteria Using <em>Vibrio cholerae</em> Murine Infections as Model System</h1>
            <p><strong>Authors:</strong> Franz G. Zingl, Fabian Mitterer, Himadri B. Thapa, Stefan Schild</p>
            <h2>Abstract</h2>
            <p>Investigation of bacterial gene regulation upon environmental changes is still a challenging task. For example, <em>Vibrio cholerae</em>, a pathogen of the human gastrointestinal tract, faces diverse transient conditions in different compartments upon oral ingestion. Genetic reporter systems have been demonstrated to be extremely powerful tools to unravel gene regulation events in complex conditions, but so far focused mainly on gene induction. Herein, we describe the TetR-controlled recombination-based <em>in vivo</em> expression technology TRIVET, which allows detection of gene silencing events. TRIVET resembles a modified variant of the <em>in vivo</em> expression technology (IVET) as well as recombination-based <em>in vivo</em> expression technology (RIVET), which were used to identify conditional gene induction in several bacteria during host colonization. Like its predecessors, TRIVET is a single cell based reporter system, which allows the analysis of bacterial gene repression in a spatiotemporal manner via phenotypical changes in the resistance profile. Briefly, a promoterless <em>tetR</em> (encoding the transcriptional repressor TetR) can be integrated randomly into the bacterial genome via transposon mutagenesis or site-specific downstream of a promoter of interest via homologous recombination. Reduction of transcriptional expression of TetR results in a de-repression of the TetR-controlled resolvase TnpR, which in turn leads to excision of an antibiotic resistance cassette (also known as res-cassette) and altered resistance profile observable via streaking on ampicillin and kanamycin plates. This alteration can then be quantified as the ratio between resistant and non-resistant isolates. Furthermore, the newly introduced second reporter gene, a promoterless <em>phoA</em> (encoding the alkaline phosphatase PhoA) offers an additional validation step of the results via an independent colorimetric assay to measure enzyme activity. The protocol presented herein also offers an approach to identify the gene locus in case of the random screen for gene repression as well as a quantification of the conditional repression of a gene of interest. Although the current protocol is established for gene repression during host colonization, it can likely be adapted to study gene silencing under various conditions faced by a bacterium.</p>
            <p><strong>Keywords:</strong> Gene expression, Resolution, Mouse, Repression, Screen, Regulation</p>
            <h2>Background</h2><p>Facultative bacterial pathogens constantly need to adapt to varying conditions during environmental passages and host colonization. Optimal survival fitness is assured by transient activation and repression of numerous genes. Unravelling these adaptation processes is key to understand bacterial physiology and identify potential targets for therapeutic intervention strategies. Diverse techniques have been established to study gene expression profiles in bacteria, including microarrays, RNA-Seq and qRT-PCR (Eisen and Brown, 1999; Bookout and Mangelsdorf, 2003; Wang <em>et al.</em>, 2009). Such methods have to deal with the consequences of averaging heterogeneity in the bacterial population and only reflect transcription level at the time-point of harvest. Any unique patterns of gene expression related to specific regions in a subpopulation or transient regulation might be lost. Moreover, the requirement of relatively high quantity of RNA and purity restricts the above-mentioned technologies. In contrast, the TRIVET reporter system allows the detection of transient gene silencing events even in a subset of a complex, heterogenous bacterial population. TRIVET is a robust technique due to an irreversible change in the antibiotic resistance profile on a single cell level. On the contrary, TRIVET requires substantial, labor-intensive genetic engineering of the model organism and is therefore limited to bacteria with available genome sequence and tools for genetic modification.<br/> <br/>Using the facultative human pathogen <em>Vibrio cholerae</em> as a representative example, we describe two potential applications of TRIVET: (i) a random approach to identify <em>in vivo</em> repressed genes during intestinal colonization in the murine model and (ii) a specific approach to study conditional transcriptional silencing of a gene of interest (Cakar <em>et al.</em>, 2018; Zingl <em>et al.</em>, 2020). The TRIVET system consists of three chromosomal elements, <em>i.e.</em>, a <em>tetR</em>-<em>phoA</em>-<em>cat</em> (<em>tpc</em>) reporter cassette, a suicide vector system pTRIVET providing the TetR-controlled resolvase TnpR and the resolution (res)-cassette as target for TnpR.<br/> <br/>The res-cassette of TRIVET is identical to the res-cassette of RIVET (Osorio <em>et al.</em>, 2005) and is integrated in the <em>lacZ</em>-locus of the <em>V. cholerae</em> chromosome. The two selection markers [<em>neo</em>(Km<sup>R</sup>) and <em>sacB</em> (Suc<sup>S</sup>)] are flanked by target sites for the TnpR resolvase also known as the <em>res</em>-sites.<br/> <br/>The second part is the <em>tpc</em>-cassette. For the random approach, the <em>tpc</em>-cassette is subcloned between the IS10 sites of the mini Tn10-system using the pLOF vector delivery system (Herrero <em>et al.</em>, 1990). Noteworthy, <em>tetR</em> and <em>phoA</em> encoding the TetR repressor and the alkaline phosphatase are promoterless, while the <em>cat</em> cassette was subcloned with its own constitutive promoter to allow selection of transposon mutants. Thus, transposon mutagenesis of res-cassette containing strains following chloramphenicol selection, generates random insertions of the <em>tpc</em> reporter cassette. A sub-population of these transposon mutants harbor transcriptional fusions of chromosomal genes to <em>tetR</em> and <em>phoA</em> and will express these genes upon activation of the respective promoter.<br/> <br/>The third component of TRIVET is the pTRIVET suicide plasmid (Ap<sup>R</sup>) containing an 800 bp intergenic region downstream of the <em>lacZ</em> locus and the <em>tnpR</em> (encoding the resolvase). Importantly, the promoter responsible for <em>tnpR</em> expression originates from the tetracycline-resistance gene <em>tetA</em> and is therefore tightly controlled by <em>TetR</em>. Mobilization of this pTRIVET into a res- and <em>tpc</em>-cassette containing <em>V. cholerae</em> strain results in homologous recombination downstream of <em>lacZ</em> in the genome. As the integration site is located downstream of the intrinsic transcriptional terminator of <em>lacZ</em>, undesired read-throughs of the RNA-Polymerase towards <em>tnpR</em> are prevented. Hence, the expression of <em>tnpR</em> solely relies on the <em>TetR</em>-controlled promoter. A comprehensive description of the system including a schematic overview of the genetic elements can be found in a recent publication by Cakar <em>et al.</em> (2019). In strains with <em>tetR</em> expression via the <em>tpc</em>-cassette, repression of <em>tnpR</em> will result in stable res-cassette containing strains, which can be isolated via their unique resistance profile (Ap<sup>R</sup>, Km<sup>R</sup> and Suc<sup>S</sup>). In contrast, no expression of <em>tetR</em> leads to induction of <em>tnpR</em> resulting in excision and irreversible loss of the res-cassette (Ap<sup>R</sup>, Km<sup>S</sup> and Suc<sup>R</sup>). This event is called resolution and results in a resolved strain. Thus, a relatively simple Km<sup>R</sup>-selection will identify strains with sufficient <em>TetR</em> expression with stabilized res-cassette. These strains can be used to infect infant mice. During <em>in vivo</em> colonization, silencing of the chromosomal promoter driving the expression of <em>tetR</em> may occur, which results in an induction of <em>tnpR</em> leading to excision and irreversible loss of the res-cassette. Hence, the loss of the res-cassette during <em>in vivo</em> colonization can be monitored by a phenotypic change to Km<sup>S</sup>/Suc<sup>R</sup>, allowing the subsequent identification of these resolved strains similar to the original RIVET screens. Besides <em>tetR</em>, <em>phoA</em> acts as an independent transcriptional reporter and provides a refinement of the <em>in vivo</em> resolved strains via an alkaline phosphatase activity assay.<br/> <br/>In case of gene specific approach, TRIVET uses the identical res-cassette and pTRIVET elements, while the <em>tpc</em>-cassette needs to be subcloned into the pCVD442 suicide vector flanked by up- and downstream regions of the desired integration site to ensure homologous recombination (Donnenberg and Kaper, 1991). Once the <em>tpc</em>-cassette has been fused to the chromosomal promotor of interest, the res-cassette and the pTRIVET can be subsequently mobilized and integrated into the genome. Such strains harboring a <em>tpc</em>-cassette downstream of a promotor of interest can be used to assay the repression of each fusion under specific conditions (<em>e.g.</em>, <em>in vitro</em> and <em>in vivo</em> as described for the <em>yrb</em>-fusion strain). For example, the amount of resolution upon <em>in vitro</em> and <em>in vivo</em> cultivation can be determined by plating appropriate dilutions on LB-Kn (unresolved CFUs) and LB-Ap (total CFUs) plates. The resolved CFUs can be calculated by subtracting total unresolved CFUs from the total CFUs [(Ap<sup>R</sup> CFU) minus (Km<sup>R</sup> CFU)]. The extent of resolution is given by the resolution frequency (% resolution), calculated as the amount of resolved CFUs divided by total CFUs multiplied by 100. If applicable, <em>phoA</em> acts as an independent transcriptional reporter and allows an additional assessment of the transcriptional activity of the <em>tpc</em>-fusion strain via an alkaline phosphatase assay, which will be described herein for the <em>irgA</em>-fusion strain under iron-replete and iron-deplete conditions.<br/> <br/>Although originally designed to identify gene repression during intestinal colonization, TRIVET can be likely adapted to diverse cultivation conditions (Seper <em>et al.</em>, 2014; Cakar <em>et al.</em>, 2018; List <em>et al.</em>, 2018). This technology can thus help to expand our knowledge of bacterial gene regulation, adaptation processes and the regulatory cascades involved.</p>
        </body>

<p class="pptt" id="biaoti36858">## Materials and Reagents</p><ol class="npwone"><li>Construction of the TRIVET System<br/><ol class="npwtwo"><li>Bacterial strains and plasmids<br/><ol class="npwthree"><li><em>Escherichia coli</em> DH5Î±Î»<em>pir</em> F<sup>â</sup> <em>endA</em>1 <em>glnV</em>44 <em>thi</em>-1 <em>recA</em>1 <em>relA</em>1 <em>gyrA</em>96 <em>deoR nupG</em> Î¦80d<em>lac</em><em>Z</em>ÎM15 Î(<em>lacZYA</em>-<em>argF</em>)U169 <em>hsdR</em>17(r<sub>K</sub><sup>â</sup> m<sub>K</sub><sup>+</sup>) Î»pirRK6 (54)<br/></li><li><em>E. coli</em> XL-1 Fâ²::Tn10 <em>proA</em> + B + <em>lac<sup>q</sup></em> Î(<em>lacZ</em>)M15/<em>recA</em>1 <em>endA</em>1 <em>gyrA</em>46 (Nal<sup>r</sup>) <em>thi hsd</em>R17 (r<sub>K</sub><sup>â</sup> m<sub>K</sub><sup>+</sup>) <em>supE</em>44 <em>relA</em>1 <em>lac</em> (New England Biolabs)<br/></li><li><em>E. coli</em> SM10Î»<em>pirthi</em>-1 <em>thr leu tonA lacY supE recA</em>::RP4-2-Tc::Mu Î»<em>pir</em>RK6 (Miller and Mekalanos, 1988)<br/></li><li><em>V. cholerae</em> WT spontaneous streptomycin resistant (Sm<sup>R</sup>) mutant of E7946 (O1 El Tor Inaba), Sm<sup>R</sup> (Miller <em>et al.</em>, 1989)<br/></li><li>pCVD442 Suicide vector, OriR6K, sacB, Ap<sup>R</sup> (Donnenberg and Kaper, 1991)<br/></li><li>pA<a class="blackNoUnder" target="_blank"> C1000 </a> Cm<sup>R</sup> (Hava <em>et al.</em>, 2003)<br/></li><li>pGOA1193 pIVET5n tnpR, oriR6K mobRP4 lacZ tnpR, Ap<sup>R</sup> (Osorio <em>et al.</em>, 2005)<br/></li><li>pGP704 oriR6K mobRP4, Ap<sup>R</sup> (Miller and Mekalanos, 1988)<br/></li><li>pTrc99A-Km pBR322 origin, Km<sup>R</sup> (Amann <em>et al.</em>, 1988)<br/></li><li>pRR51 (Reed and Grindley, 1981)<br/></li><li>pSL134 pPCR-Script::(<em>res1</em>-<em>res1</em>) (Osorio <em>et al.</em>, 2005)<br/></li><li>pPCR-Script Amp SK(+) (Stratagene)<br/></li><li>pGAO2 [pPCR-Script::(<em>res</em>) (Osorio <em>et al.</em>, 2005)]<br/></li><li>pGAO3 [pPCR-Script::(<em>res1</em>) (Osorio <em>et al.</em>, 2005)]<br/></li><li>pGAO4 [pPCR-Script::(<em>res</em>-<em>res</em>) (Osorio <em>et al.</em>, 2005)]<br/></li><li>pGAO5 [pPCR-Script::(<em>res1</em>-<em>res1</em>) (Osorio <em>et al.</em>, 2005)]<br/></li><li>pGAO6 [pGOA4::<em>neo</em>-<em>sacB</em> (Osorio <em>et al.</em>, 2005)]<br/></li><li>pGAO7 [pGOA5::<em>neo</em>-<em>sacB</em> (Osorio <em>et al.</em>, 2005)]<br/></li><li>pRes pSL111 with res-neo-sacB-res cassette, Km<sup>R</sup> (Osorio <em>et al.</em>, 2005)<br/></li><li>pRes1 pSL111 with res1-neo-sacB-res1 cassette Km<sup>R</sup> (Osorio <em>et al.</em>, 2005)<br/></li><li>pLOFKm Tn10 based delivery plasmid, Km<sup>R</sup>, Ap<sup>R</sup> (Herrero <em>et al.</em>, 1990)</li></ol></li><li>Oligonucleotides<br/><ol class="npwthree"><li>tetR-5â²-BamHIAATGGATCCTAGAGTGTCAACAAAAATTAGGAATTA<br/></li><li>tetR-3â²-KpnlTTAGGTACCATCACGGAAAAAGGTTATGCT<br/></li><li>phoA-5â²-KpnITTTGGTACCTTTTTAATGTATTTGTACATGGAGAA<br/></li><li>phoA-3â²-XbaIAAATCTAGACATTAAGTCTGGTTGCTAACAGCA<br/></li><li>cat-5â²-XbaIAAATCTAGATAAGCTTGATGAAAATTTGTTTGA<br/></li><li>cat-3â²-BamHITTTGGATCCTTCTTCAACTAACGGGGCA<br/></li><li>tetRphoAcat-5â²-NotI TATGCGGCCGCCCTAGGTAATTAGGATCCTAGAGT<br/></li><li>tetRphoAcat-3â²-NotI TTTGCGGCCGCCCTAGGTCTCATCCGCCAAAACAGCCAA<br/></li><li>lacZ-5â²-SacITTTGAGCTCTGATTTACCGCCGCTGCCAA<br/></li><li>lacZ-3â²-NheITTTGCTAGCTTATTGTGGGTGATGACGCTTT<br/></li><li>tnpR-5â²-BgIIICGACCCGGGAGATCTCAATTGTTCGAATTTAGGATACATTTTTAT<br/></li><li>tnpR-3â²-XbaITTTTCTAGATTAAGTTGGGTAACGCCAGGGT<br/></li><li>PtetA-5â²-3â²CTAGCCAGAGAGCCTTAAGGCTCTCTTTTTTCTAATTTTTGTTGACACC<br/>CTATCAGTGATAGAGTTATTTTACCACTCCCTATCAGTGATAGA<br/></li><li>PtetA-3â²-5â²GATCTCTATCACTGATAGGGAGTGGTAAAATAACTCTATCACTGATAGG<br/>GTGTCAACAAAAATTAGAAAAAAGAGAGCCTTAAGGCTCTCTGG<br/></li><li>Ptet1-5â²-3â²CTAGCAGAGAGCCTTAAGGCTCTCTTTTTTCTAATTTTTGTCCCTATCA<br/>GTGATAGAGATTGACATCCCTATCAGTGATAGAGATACTGAGCACATCA</li><li>Ptet1-3â²-5â²GATCTGATGTGCTCAGTATCTCTATCACTGATAGGGATGTCAATCTCTAT<br/>CACTGATAGGGACAAAAATTAGAAAAAAGAGAGCCTTAAGGCTCTCTG</li><li>1RES-FTCTATTGAATTCCGTCCGAAATATTATAAATTATCGCAC</li><li>1RES-RTCTAATCTCGAGTGTATCCTAAATCAAATATCGGACAAG</li><li>1RES1-FTCTAATGAATTCCGTCCGAAATATTACAAATTATCGCAC</li><li>2RES-FTCTAATCTCGAGTCTAGACGTCCGAAATATTATAAATTATCGCAC</li><li>2RES1-FTCTAATCTCGAGTCTAGACGTCCGAAATATTACAAATTATCGCAC</li><li>2RES-RTCTAATGGTACCTGTATCCTAAATCAAATATCGGACAAG</li><li>T3AATTAACCCTCACTAAAGGG</li><li>T7AATACGACTCACTATAGGGC<br/></li></ol></li><li>Materials<br/><ol class="npwthree"><li>Pipette tip (Greiner Bio-One, catalog numbers: <a class="blackNoUnder" target="_blank"> 740290 </a> , 739290 and 771291)<br/></li><li>PCR tubes (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AB0266 </a> )<br/></li><li>Double distilled H<sub>2</sub>O (ddH<sub>2</sub>O)<br/></li><li>dNTPs, 10 mM (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> R0192 </a> )<br/></li><li>DNA Loading Dye (6x) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> R0611 </a> )<br/></li><li>Q5 High-Fidelity DNA Polymerase (NEB, catalog number: <a class="blackNoUnder" target="_blank"> M0491S </a> )<br/></li><li>Taq DNA polymerase (NEB, catalog number: <a class="blackNoUnder" target="_blank"> M0273S </a> )<br/></li><li>GeneRuler 1 kb Plus DNA Ladder (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> SM1331 </a> )<br/></li><li>QIAquickGel Extraction Kit (QIAGEN, catalog number: <a class="blackNoUnder" target="_blank"> 28704 </a> )<br/></li><li>QIAquickPCR Purification Kit (QIAGEN, catalog number: <a class="blackNoUnder" target="_blank"> 28104 </a> )<br/></li><li>QIAprep Spin Miniprep Kit (QIAGEN, catalog number: <a class="blackNoUnder" target="_blank"> 27104 </a> )</li><li>Cut smart buffer (NEB, catalog number: <a class="blackNoUnder" target="_blank"> B7204S </a> )</li><li>Appropriate Restriction Enzymes (NEB)</li><li>Antarctic phosphatase (NEB, catalog number: <a class="blackNoUnder" target="_blank"> M0289S </a> )</li><li>T4 DNA ligase (NEB, catalog number: <a class="blackNoUnder" target="_blank"> M0202S </a> )</li><li>T4 DNA ligase buffer (NEB, catalog number: <a class="blackNoUnder" target="_blank"> B0202S </a> )</li><li>Glycerol (AppliChem, catalog number: <a class="blackNoUnder" target="_blank"> A0970 </a> ,1000)</li><li>Yeast extract (Applichem, catalog number: <a class="blackNoUnder" target="_blank"> A1552 </a> )</li><li>Bacto Tryptone (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 211701 </a> )</li><li>NaCl (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 27810.364 </a> )</li><li>Agar (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> A5054 </a> )</li><li>Appropriate antibiotic for the plasmid vector (<em>e.g.</em>, Ampicillin, Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> K029.2 </a> )</li><li>LB agar plates containing the appropriate antibiotic for the plasmid vector (see Recipes)</li><li>LB medium (see Recipes)<br/></li></ol><br/></li></ol></li><li>Construction of the TRIVET library<ol class="npwtwo"><li>Bacterial strains and plasmids<br/><ol class="npwthree"><li><em>E. coli</em> DH5Î±Î»<em>pir</em> F<sup>â</sup> <em>endA</em>1 <em>glnV</em>44 <em>thi</em>-1 <em>recA</em>1 <em>relA</em>1 <em>gyrA</em>96 <em>deoR nup</em>G Î¦80d<em>lacZ</em>ÎM15 Î(<em>lacZ</em><em>YA</em>-<em>argF</em>)U169 <em>hsdR</em>17(r<sub>K</sub><sup>â</sup> m<sub>K</sub><sup>+</sup>) Î»<em>pir</em>RK6<br/></li><li><em>E. coli</em> SM10Î»<em>pir</em> <em>thi</em>-1 <em>thr leu ton</em><em>A lacY supE recA</em>::RP4-2-Tc::Mu Î»<em>pir</em>RK6<br/></li><li><em>V. cholerae</em> WT spontaneous streptomycin resistant (Sm<sup>R</sup>) mutant of E7946 (O1 El Tor Inaba), Sm<sup>R</sup> (Miller <em>et al.</em>, 1989)<br/></li><li>pCVD442 Suicide vector, OriR6K, <em>sacB</em>, Ap<sup>R</sup> (Donnenberg and Kaper, 1991)<br/></li><li>pRes pSL111 with <em>res</em>-<em>neo</em>-<em>sacB</em>-<em>res</em> cassette, Km<sup>R</sup> (Osorio <em>et al.</em>, 2005)<br/></li><li>pRes1 pSL111 with <em>res1</em>-<em>neo</em>-<em>sacB</em>-<em>res1</em> cassette, Km<sup>R</sup> (Osorio <em>et al.</em>, 2005)<br/></li><li>pLOF::tpc (Cakar <em>et al.</em>, 2018)<br/></li><li>pTRIVET (Cakar <em>et al.</em>, 2018)</li><li>pTRIVET1 (Cakar <em>et al.</em>, 2018)<br/></li><li>pRES [pSL111::(<em>res</em>-<em>neo</em>-<em>sacB</em>-<em>res</em>) (Osorio <em>et al.</em>, 2005)]<br/></li><li>pRES1 [pSL111::(<em>res1</em>-<em>neo</em>-<em>sacB</em>-<em>res1</em>) (Osorio <em>et al.</em>, 2005)]<br/></li></ol></li><li>Oligonucleotides<br/><ol class="npwthree"><li>tetRphoAcat-5â²-SacI ATAGAGCTCAGAGTGTCAACAAAAATTAGGAATTAA<br/></li><li>tetRphoAcat-3â²-SalI TTTGTCGACTTCTTCAACTAACGGGGCA<br/></li><li>irgA-SphI-1 AATGCATGCTCCGAGTAAACCGCAAACACTT<br/></li><li>irgA-SacI-2 AATGAGCTCGTATCCCGCCGCAGTGACCA<br/></li><li>irgA-SalI-3 AAAAGTCGACCGAACTATTCCATGTGT<br/></li><li>irgA-XbaI-4 AATTCTAGACGTGAGGTTTGGCGCTTA</li><li>yrb_SacI_1AAAGAGCTCGCGATATTGGCAATGTTTGAAC<br/></li><li>yrb_SphI_2AAAGCATGCGATAAGGATAATTAATTGGAATC<br/></li><li>yrb_SalI_3AAAGAATTCTCAGGATGTCGACCTAACAG<br/></li><li>yrb_XbaI_4TTTTCTAGAGATTAAGGTTACGGATCAGTTC<br/></li><li>seqPrimer-catGAATTGTCAGATAGGCCTAATG<br/></li></ol></li><li>Materials<br/><ol class="npwthree"><li>PCR tubes (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AB1771 </a> )<br/></li><li>Needle 26G (B. Braun, catalog number: <a class="blackNoUnder" target="_blank"> 4657683 </a> )<br/></li><li>Gavage tubing (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10793527 </a> )<br/></li><li>Gavage syringe 1 ml (Henry Schein, catalog number: <a class="blackNoUnder" target="_blank"> 9003016 </a> )<br/></li><li>5-d-old CD-1 mice<br/></li><li>Double distilled H<sub>2</sub>O (ddH<sub>2</sub>O)<br/></li><li>dNTPs, 10 mM (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> R0192 </a> )<br/></li><li>DNA Loading Dye (6x) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> R0611 </a> )<br/></li><li>Q5 High-Fidelity DNA Polymerase (NEB, catalog number: <a class="blackNoUnder" target="_blank"> M0491S </a> )<br/></li><li>Taq DNA polymerase (NEB, catalog number: <a class="blackNoUnder" target="_blank"> M0273S </a> )<br/></li><li>GeneRuler 1 kb Plus DNA Ladder (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> SM1331 </a> )<br/></li><li>QIAquickGel Extraction Kit (QIAGEN, catalog number: <a class="blackNoUnder" target="_blank"> 28704 </a> )<br/></li><li>QIAquickPCR Purification Kit (QIAGEN, catalog number: <a class="blackNoUnder" target="_blank"> 28104 </a> )<br/></li><li>QIAprep Spin Miniprep Kit (QIAGEN, catalog number: <a class="blackNoUnder" target="_blank"> 27104 </a> )<br/></li><li>QIAprepMidiprep Kit (QIAGEN, catalog number: <a class="blackNoUnder" target="_blank"> 12143 </a> )<br/></li><li>Cut smart buffer (NEB, catalog number: <a class="blackNoUnder" target="_blank"> B7204S </a> )<br/></li><li>Appropriate Restriction Enzymes (NEB)<br/></li><li>Antarctic phosphatase (NEB, catalog number:<a class="blackNoUnder" target="_blank"> M0289S </a> )<br/></li><li>T4 DNA ligase (NEB, catalog number: <a class="blackNoUnder" target="_blank"> M0202S </a> )<br/></li><li>T4 DNA ligasebuffer (NEB, catalog number: <a class="blackNoUnder" target="_blank"> B0202S </a> )<br/></li><li>Yeast extract (Applichem, catalog number: <a class="blackNoUnder" target="_blank"> A1552 </a> )<br/></li><li>NaCl (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 27810.364 </a> )<br/></li><li>Bacto Tryptone (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 211701 </a> )<br/></li><li>Agar (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> A5054 </a> )<br/></li><li>Appropriate antibiotic for the plasmid vector (<em>e.g.</em>, Ampicillin, Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> K029.2 </a> )<br/></li><li>Tris (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 10708976001 </a> )<br/></li><li>MgSO<sub>4</sub> (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M7506 </a> )<br/></li><li>ZnCl<sub>2</sub> (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 746355 </a> )<br/></li><li>SDS (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> L3771 </a> )<br/></li><li>Chloroform (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 372978 </a> )<br/></li><li>KH<sub>2</sub>PO<sub>4</sub> (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P0662 </a> )<br/></li><li><em>p</em>-nitrophenyl phosphate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 4876 </a> )<br/></li><li>EDTA (Sigma-Aldrich, catalog number: 0<a class="blackNoUnder" target="_blank"> 3620 </a> )<br/></li><li>Isoflurane IsoFlo (Zoetis, catalog number: <a class="blackNoUnder" target="_blank"> 50019100 </a> )<br/></li><li>Agarose peqGOLD (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 732-2789 </a> )<br/></li><li>50x TAE buffer (Thermo scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10399519 </a> )<br/></li><li>2,2'-bipyridyl (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> D216305 </a> )<br/></li><li>FeSO<sub>4</sub> (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 450278 </a> )<br/></li><li>X-Gal (5-Bromo-4-chloro-3-indolyl beta-D-galactopyranoside; Thermo scientific, catalog number: <a class="blackNoUnder" target="_blank"> R0404 </a> )<br/></li><li>DMSO (Roth, catalog number: <a class="blackNoUnder" target="_blank"> 4720 </a> )<br/></li><li>LB agar plates containing the appropriate antibiotic for the plasmid vector (see Recipes)<br/></li><li>LB medium (see Recipes)<br/></li><li>Sucrose agar plates (see Recipes)<br/></li></ol></li></ol></li></ol><p class="pptt" id="biaoti36859">### Equipment</p><ol class="npwtwok"><li>Pipettes (Eppendorf)</li><li>Combi-vet isoflurane administration instrument (Rothacher, catalog number: <a class="blackNoUnder" target="_blank"> CV 30301 </a> )</li><li>Scissors (Thermo scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10694962 </a> )</li><li>Tweezers (Thermo scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15809651 </a> )</li><li>Tissue homogenizer (Biospec, catalog number: <a class="blackNoUnder" target="_blank"> 985370 </a> )</li><li>Vortexer (Thermo Fisher Scientific)</li><li>Incubator (Thermo Fisher Scientific, ThermoScientific<sup>TM</sup>, model: Heraeus B12 Function Line, catalog number: 50042307)</li><li>Microcentrifuge (Eppendorf, model: CentrifugeMiniSpin<sup>Â®</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 5452000018 </a> )</li><li>Thermocycler (Bio-Rad <a class="blackNoUnder" target="_blank"> C1000 </a> )</li><li>NanoDrop<sup>TM</sup> (<a class="blackNoUnder" target="_blank"> ND 2000 </a> )</li><li>Photometer (Beckman <a class="blackNoUnder" target="_blank"> DU730 </a> )<br/></li></ol><p class="pptt" id="biaoti36860">### Software</p><ol class="npwtwok"><li>blastN (blast.jcvi.org/cmr-blast/)</li><li>T<sub>m</sub> Calculator Software (<em>e.g.</em>, NEB T<sub>m</sub> Calculator)<br/></li></ol><p class="pptt" id="biaoti36861">### Procedure</p><ol class="npwone"><li>General Protocols<ol class="npwtwo"><li>PCR to amplify fragments for plasmid construction<br/>For PCR reactions follow the individual manufacturer's protocols. Briefly, the temperature was calculated as follows: 2 Â°C for every hybridized AT-pair and 4 Â°C for every GC-pair were added to the melting temperature. The elongation time was calculated after the length of the amplified fragment and the cycle amount was 30. A standard recipe for a PCR reaction to amplify fragments for plasmid construction based on the agents used herein would be:<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td><strong>Reagents</strong><br/></td><td><strong>Volume (Âµl)</strong><br/></td></tr><tr><td>chromosomal DNA<br/></td><td>8</td></tr><tr><td>dNTPâs <br/></td><td>5</td></tr><tr><td>Primer I (1 Âµg/ml) <br/></td><td>5</td></tr><tr><td>Primer II (1 Âµg/ml) <br/></td><td>5</td></tr><tr><td>GC enhancer<br/></td><td>20</td></tr><tr><td>Q5 Polymerase<br/></td><td>3</td></tr><tr><td>Q5 Polymerase buffer<br/></td><td>25</td></tr><tr><td>ddH<sub>2</sub>O<br/></td><td>32</td></tr></tbody></table><ol class="npwthree"><li>Perform PCR reaction.<br/></li><li>Add appropriate volumes of loading dye to the PCR reaction and separate bands by horizontal gel electrophoresis using a 0.8 to 1% agarose gel and apply approx. 10 V per cm gel.<br/></li><li>Excise band from the agarose gel and purify respective PCR fragments via the QIAGEN gel extraction kit following the manufacturer's instructions.<br/></li></ol></li><li>Colony PCR<br/>For PCR reactions follow the individual manufacturer's protocols. The template for the colony PCR originates from a colony picked form an agar plate using a pipette tip. The selected colony is resuspended in 50 Âµl ddH<sub>2</sub>O and boiled at 100 Â°C for 10 min. A standard recipe for a PCR reaction to amplify fragments for plasmid construction based on the agents used herein would be:<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td><strong>Reagents</strong><br/></td><td><strong>Volume (Âµl)</strong><br/></td></tr><tr><td>Template (boiled colony resuspension)<br/></td><td>3</td></tr><tr><td>dNTPâs<br/></td><td>0.5<br/></td></tr><tr><td>Primer I (1 Âµg/ml) <br/></td><td>0.5<br/></td></tr><tr><td>Primer II (1 Âµg/ml) <br/></td><td>0.5<br/></td></tr><tr><td>Thermo Polymerase Buffer <br/></td><td>2.5<br/></td></tr><tr><td>Taq Polymerase <br/></td><td>0.25<br/></td></tr><tr><td>ddH<sub>2</sub>O <br/></td><td>17.75<br/></td></tr></tbody></table><ol class="npwthree"><li>Perform PCR reaction.<br/></li><li>Add appropriate volumes of loading dye to the PCR reaction and separate bands by horizontal gel electrophoresis using a 0.8 to 1% agarose gel and apply approx. 10 V per cm gel.</li></ol></li><li>Restriction digestion<br/>For restriction digests follow the individual manufacturer's protocols. Standard recipes for restriction digests based on the agents used herein would be:<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:700px;"><tbody><tr><td><strong>Reagents</strong><br/></td><td><strong>Volume (Âµl)</strong><br/></td></tr><tr><td>Plasmid (isolated via a QIAGEN Plasmid Midi Kit; 100 ng/Âµl) <br/></td><td>5</td></tr><tr><td>Restriction Enzyme I <br/></td><td>1.5<br/></td></tr><tr><td>Restriction Enzyme II <br/></td><td>1.5<br/></td></tr><tr><td>10x buffer (appropriate to Enzyme) <br/></td><td>10<br/></td></tr><tr><td>ddH<sub>2</sub>O<br/></td><td>81<br/></td></tr></tbody></table><strong>or</strong><br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:700px;"><tbody><tr><td><strong>Reagents</strong><br/></td><td><strong>Volume (Âµl)</strong><br/></td></tr><tr><td>PCR Fragment (gel-purified via a QIAGEN gel extraction kit; 100 ng/Âµl) <br/></td><td>15<br/></td></tr><tr><td>Restriction Enzyme I <br/></td><td>1.5<br/></td></tr><tr><td>Restriction Enzyme II <br/></td><td>1.5<br/></td></tr><tr><td>10x buffer (appropriate to enzyme) <br/></td><td>10<br/></td></tr><tr><td>ddH<sub>2</sub>O<br/></td><td>71<br/></td></tr></tbody></table><ol class="npwthree"><li>Perform overnight (ON) digestion using the appropriate restriction enzyme combinations and temperatures according to the manufacturer's instructions. In case of the PCR fragments appropriate restriction enzymes herein are indicated by the name of the oligonucleotide (see oligonucleotide lists above).<br/></li><li>In case of restriction digestion using plasmids, dephosphorylate 5â²-ends by adding 1 Âµl antarctic phosphatase to the mix and incubate for 1 h at 37 Â°C.<br/></li><li>Add appropriate volumes of loading dye to the samples and separate the bands by horizontal gel electrophoresis using a 0.8 to 1% agarose gel and apply approx. 10 V per cm gel.<br/></li><li>Excise band from the agarose gel and purify the linearized, dephosphorylated plasmid from the gel via the QIAGEN gel extraction kit following the manufacturer's instructions.<br/></li><li>In case of restriction digestion using the PCR fragments, inactivate the enzymes at 65 Â°C for 10 min.<br/></li><li>Subsequently purify the digested PCR fragments using the QIAGEN PCR purification kit following the manufacturer's instructions.</li></ol></li><li>Ligation<br/>For ligations follow the individual manufacturer's protocols. Standard recipes for restriction digestion based on the agents used herein would be:<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:500px;"><tbody><tr><td><strong>Reagents</strong><br/></td><td><strong>Volume (Âµl)</strong><br/></td></tr><tr><td>Plasmid (digested + dephos.; 30 ng/Âµl)<br/></td><td>5</td></tr><tr><td>PCR fragment I (digested; 30 ng/Âµl)<br/></td><td>5.75<br/></td></tr><tr><td>PCR fragment II (digested; 30 ng/Âµl)<br/></td><td>5.75<br/></td></tr><tr><td>10x DNA ligase buffer <br/></td><td>1.5<br/></td></tr><tr><td>T4 DNA ligase<br/></td><td>1</td></tr></tbody></table><ol class="npwthree"><li>Perform ligation at room temperature for 1 h.<br/></li><li>Inactivate the ligase at 65 Â°C for 10 min.</li></ol></li><li>PhoA Assay<br/><ol class="npwthree"><li>Cultivate strains to late logarithmic growth phase under <em>in vitro</em> test conditions [<em>e.g.</em>, herein LB at 37 Â°C and 180 rpm (supplemented with 2,2'-bipyridyl or FeSO<sub>4</sub> in case of the <em>irgA</em>-fusion strains)].<br/></li><li>Pellet 1 ml of each culture using a microcentrifuge (5 min, 6,000 <em>x g</em>, RT).<br/></li><li>Wash cells once in P1 buffer (10 mM Tris-HCl, pH 8.0, 10 mM MgSO<sub>4</sub>) and resuspend the pellet in 1 ml P1 buffer.<br/></li><li>Dilute 0.1 ml of resuspended cells from Step A5c in 0.9 ml P1 buffer and measure OD<sub>600</sub>.<br/></li><li>Add 0.1 ml of resuspended cells from Step A5c into 0.9 ml P2 buffer (1 M Tris-HCl pH 8.0, 0.1 mM ZnCl<sub>2</sub>). In parallel, prepare a negative control without cells.<br/></li><li>To each sample add 50 Âµl of 0.1% SDS and 50 Âµl of chloroform, vortex and incubate at RT for 10 min to permeabilize cells.<br/></li><li>To start the enzymatic reaction, add 0.1 ml of 0.4% p-nitrophenyl phosphate (in 1 M Tris-HCl, pH 8.0) to each tube, vortex, and note time.<br/></li><li>Incubate each tube until a light-yellow color appears, then add 120 Âµl of 0.1 M EDTA, pH 8.0, 1 M KH<sub>2</sub>PO<sub>4</sub> to stop the reaction and note time.<br/></li><li>Measure OD<sub>420</sub> for each sample, using the negative control sample as a blank.<br/></li><li>Calculate the duration between start and stop of the reaction.<br/></li><li>Calculate PhoA activity using the formula stated in the chapter data analysis.<br/></li></ol><br/></li></ol></li><li>Random approach to identify <em>in vivo </em>repressed (ivr) genes of <em>V. cholerae</em> <ol class="npwtwo"><li>Construction of the TRIVET components<br/>The TRIVET system requires construction of several elements partially based on components of the RIVET (Osorio <em>et al.</em>, 2005).<br/><ol class="npwthree"><li>Construction of the <em>tetR-phoA-cat</em> (<em>tpc</em>) cassette for random integration in the bacterial chromosome via transposon mutagenesis.<br/><ol class="npwfour"><li>For optimal PCR results, calculate optimal T<sub>m</sub> for all given oligonucleotides.<br/></li><li>PCR amplify the promoterless <em>tetR</em>-fragment using chromosomal DNA from <em>E. coli</em> XL-1 and oligonucleotide pairs tetR-5â²-BamHI and tetR-3â²-Kpnl (see Step A1 for details).<br/></li><li>PCR amplify the promoterless <em>phoA</em>-fragment using chromosomal DNA from SM10Î»<em>pir </em>oligonucleotide pairs phoA-5â²-KpnI and phoA-3â²-XbaI (see Step A1 for details).<br/></li><li>PCR amplify <em>cat</em> from pA<a class="blackNoUnder" target="_blank"> C1000 </a> . The oligonucleotide pairs cat-5â² to XbaI and cat-3â² to BamHI (see Step A1 for details).<br/></li><li>Gel-purify PCR fragments followed by restriction digestion and second purification (see Steps A1 and A3 for details).<br/></li><li>Digest (using BamHI), dephosphorylate and gel-purify pTrc99A-Km (see Step A3 for details)<br/></li><li>Ligate the digested <em>tetR</em>-, <em>phoA</em>- and <em>cat</em>-fragments into a BamHI-digested and dephosphorylated pTrc99A-Km to obtain the pTrc-tpc plasmid (see Step A4 for details).<br/></li><li>Transform into competent <em>E. coli</em> DH5Î±Î»<em>pir</em> cells and select by plating on LB agar supplemented with Kanamycin (Km) and Chloramphenicol (Cm).<br/></li><li>Incubate plates overnight at 37 Â°C.<br/></li><li>Confirm correct pTrc-tpc construct via colony PCR using the oligonucleotide pairs tetR-5â²-BamHI and cat-3â²-BamHI.<br/></li><li>Grow verified colonies with correct pTrc-tpc construct in LB-Km/Cm + 2% glucose (Glc), overnight at 37 Â°C and 180 rpm.<br/></li><li>Isolate pTrc-tpc using a QIAprep MidiprepKit.<br/></li><li>PCR amplify the entire <em>tetR</em>-<em>phoA</em>-<em>cat</em> (<em>tpc</em>)-cassette using oligonucleotide pairs tetRphoAcat-5â²-NotI and tetRphoAcat-3â²-NotI (see Step A1 for details).<br/></li><li>Gel-purify PCR fragments followed by restriction digestion with NotI and second purification (see Steps A1 and A3 for details).<br/></li><li>Ligate the purified, NotI-digested <em>tpc</em>-cassette into a NotI-digested, dephosphorylated pLOFKm to obtain pLOF::tpc (see Steps A1, A3 and A4 for details).<br/></li><li>Transform ligation products into competent <em>E. coli</em> DH5Î±Î»<em>pir</em> cells and select by plating on LB agar supplemented with ampicillin (Ap) and Cm.<br/></li><li>Incubate plates overnight at 37 Â°C.<br/></li><li>Confirm correct pLOF::tpc construct via colony PCR using the oligonucleotide pairs tetR-5â²-BamHI and cat-3â²-BamHI (see Step A2 for details).<br/></li><li>Grow verified colonies with correct pLOF::tpc construct in LB-Ap/Cm + 2%Glc, overnight at 37 Â°C and180 rpm.<br/></li><li>Isolate pLOF::tpc using the QIAprep Spin Miniprep Kit following the manufacturer's instructions.<br/></li><li>Transform pRES and pRES1 into competent SM10Î»<em>pir</em> cells and select by plating on LB agar supplemented with Ap.</li><li>Incubate plates overnight at 37 Â°C.<br/></li></ol></li><li>Construction of the pTRIVET and pTRIVET1 plasmids<br/><ol class="npwfour" style="font-size:13.3333px;color:#666666;white-space:normal;"><li>PCR amplify the <em>lacZ</em>-fragment from <em>V. cholerae </em>WT chromosomal DNA using oligonucleotide pairs lacZ-5â²-SacI and lacZ-3â²-NheI.<br/></li><li>PCR amplify <em>tnpR</em>-fragment from pGOA1193 using oligonucleotide pairs tnpR-5â²-BgIII and tnpR-3â²-XbaI.<br/></li><li>Gel-purify PCR fragments followed by restriction digestion and second purification (see Steps A1 and A3 for details).<br/></li><li>Mix oligonucleotide pairs PtetA-5â²-3â² and PtetA-3â²-5â² or Ptet1-5â²-3â² and Ptet1-3â²-5â², boil oligonucleotide mix for 5 min to ensure denaturation and cool at RT. This will allow hybridization of the oligonucleotide pairs with a compatible NheI- and BglII-site on each end. The generated dsDNA sequence comprises the original tetA promoter sequence p<sup>tet</sup> (in case of PtetA-5â²-3â² and PtetA-3â²-5â²) or for a less stringent transcriptional regulation a slightly altered tetA promoter sequence p<sup>tet1</sup> with mutated TetR-binding sites (in case of Ptet1-5â²-3â² and Ptet1-3â²-5â²).<br/></li><li>Ligate the digested <em>lacZ</em>-fragment, the digested <em>tnpR</em>-fragment and one of the two tetA promoter sequence versions (p<sup>tet</sup> or p<sup>tet1</sup>, obtained via Step B1bviii) into a SacI/XbaI-digested, dephosphorylated pGP704 resulting in either pTRIVET containing the promoter p<sup>tet</sup> or pTRIVET1 containing the promoter ptet1, respectively (see Step A4 for details).<br/></li><li>Transform ligation products into competent DH5Î±Î»<em>pir</em> cells and select by plating on LB agar supplemented with Ap.<br/></li><li>Incubate plates overnight at 37 Â°C.<br/></li><li>Confirm correct pTRIVET or pTRIVET1 constructs via colony PCR using the oligonucleotide pairs lacZ-3â²-SacI and tnpR-3â²-XbaI as well as restriction analyses.<br/></li><li>Grow verified colonies with correct pTRIVET or PTRIVET1 in LB-Ap + 2% Glc overnight at 37 Â°C and 180 rpm.<br/></li><li>Isolate the pTRIVET and pTRIVET1 plasmids via the QIAprep Spin Miniprep Kit by following the manufacturerâs protocol.<br/></li><li>Transform pTRIVET and pTRIVET1 into competent SM10Î»<em>pir</em> cells and select by plating on LB agar supplemented with Ap.</li><li>Incubate plates overnight at 37 Â°C.</li></ol></li><li>Construction of the pRes and pRes1 according to Osorio <em>et al.</em> (2005)<br/><ol class="npwfour"><li>PCR amplify the <em>res</em> sequence via PCR using pRR51 as a template and oligonucleotide pairs1RES-F and 1RES-R (see Step A1 for details). Alternatively, PCR amplify the <em>res1</em> sequences via PCR using pSL134 and oligonucleotide pairs1RES1-F and 1RES1-R (see Step A1 for details).<br/></li><li>Gel-purify PCR fragments followed by restriction digestion (using EcoRI and XhoI) and second purification (see Steps A1 and A3 for details).<br/></li><li>Ligate the digested PCR fragments into an EcoRI/XhoI-digested, dephosphorylated pPCR-Script Amp SK(+) (Stratagene) to generate plasmids pGOA2 and pGOA3 (see Step A4 for details).<br/></li><li>Use a 2<sup>nd</sup> set of oligonucleotide pairs (2RESF and 2RESR or 2RES1F and 2RESR) to PCR amplify a second copy of <em>res</em> or <em>res1</em>. Subclone them immediately next to their counterparts in pGOA2 and pGOA3 to generate plasmids pGOA4 and pGOA5, which harbor two <em>res</em> or <em>res1</em> sequences, respectively.<br/></li><li>Generate pGOA6 and pGOA7 via subcloning of the <em>sacB</em><em>-neo</em> genes from pGOA1 into XbaI-digested pGOA4 and pGOA5.<br/></li><li>Amplify the res- and res1-cassettes via PCR from pGOA6 and pGOA7 using oligonucleotide pairs T3 and T7 and ligate them into KpnI-digested pSL111 treated with T4 DNA polymerase to generate pRES and pRES1.<br/></li><li>Transform ligation products into competent DH5Î±Î»<em>pir </em>cells and select by plating on LB agar supplemented with Ap.<br/></li><li>Incubate plates overnight at 37 Â°C.<br/></li><li>Confirm correct pRES and pRES1 constructs via restriction analyses and sequencing.<br/></li><li>Isolate the pRES and pRES1 plasmids via the QIAprep Spin Miniprep Kit by following the manufacturerâs protocol.<br/></li><li>Transform pRES and pRES1 into competent SM10Î»<em>pir</em> cells and select by plating on LB agar supplemented with Ap.</li><li>Incubate plates overnight at 37 Â°C.</li></ol></li></ol></li><li>Construction of the TRIVET library<br/><ol class="npwthree"><li>Insertion of the <em>res</em>-cassette<br/><ol class="npwfour"><li>Mobilize suicide plasmids pRES and pRES1 into <em>V. cholerae</em> WT by conjugation and allelic exchange to place each cassette into the <em>lacZ</em>-locus to obtain Vc_res and Vc_res1 as described below.</li><li>Mobilize suicide plasmids pRES or pRES1 into <em>V. cholerae</em> by conjugation of <em>E. coli</em> SM10Î»<em>pir</em> pRES or pRES1 obtained by Step B1c with <em>V. cholerae</em> WT [(streptomycin-resistant (Sm<sup>R</sup>)]. This was performed by cross-streaking sufficient cell material of both strains on the same LB agar plate and incubating plate for 6 h at 37 Â°C. Via homologous regions present on pRES or pRES1 an allelic exchange of the suicide plasmids and the <em>lacZ</em>-locus on the <em>V. cholerae</em> chromosome is possible at low frequency via homologous regions present on pRES or pRES1.</li><li><em>V. cholerae</em> with integrated pRES and pRES1 are then selected by streaking the single colonies form the conjugation mixtures on LB-Sm/Ap/Km agar plates.</li><li>Incubate plates overnight at 37 Â°C.</li><li>Streak-purify colonies on LB-Sm/Ap/Km agar plates at least twice.</li><li>Grow purified colonies in LB-Sm/Km overnight at 37 Â°C, 180 rpm and plate appropriate dilutions on LB-Sm/Km plates supplemented with 5-Bromo-4-chloro-3-indolyl beta-D-galactopyranoside (40 Âµg/ml dissolved in DMSO, X-Gal).</li><li>Incubate plates overnight at 37 Â°C.</li><li>White colonies are potential candidates for double recombination events and allelic exchange of the <em>lacZ</em>-locus with the res- or res1-cassette.</li><li>Confirm loss of suicide vector backbone by streaking on LB-Sm/Km (growth) and LB-Ap (no growth).</li><li>Incubate plates overnight at 37 Â°C and proceed with Sm<sup>R</sup>/Km<sup>R</sup>, but Ap<sup>S</sup> colonies.</li><li>Validate correct insertion of the res or res1-cassette in the <em>lacZ</em>-locus via colony PCR and/or sequencing resulting in strains Vc_res and Vc_res1.</li></ol></li><li>Transposon mutagenesis in Vc_res and Vc_res1.<br/><ol class="npwfour"><li>Transposon mutagenesis is achieved by mobilization of the pLOF::tpc transposon plasmid into <em>V. cholerae</em> via conjugation as described below. Along the incubation period, transfer of plasmid results in spontaneous transposition events in <em>V. cholerae</em>.</li><li>Mobilize suicide vector pLOF::tpc into <em>V. cholerae</em> by conjugation of <em>E. coli</em> SM10Î»<em>pir</em> pLOF::tpc obtained from Step B1a with either Vc_res and Vc_res1 obtained from Step B2a via filter mating on LB agar plates. To do so, grow strains to an OD<sub>600</sub> of approx. 1 and mix recipient and donor in a 2:1 ratio with a final volume of 2 ml.</li><li>Pellet mixture by centrifugation (5,000 <em>x g</em>, 5 min, RT) and gently resuspend cells in 100 Âµl LB broth.</li><li>Place bacterial suspension in the center of a LB plate and incubate plate for 1.5 h at 37 Â°C to allow conjugation.</li><li>Pick up the bacteria of the plate using a pipet tip and resuspend them in 1 ml LB broth.</li><li>Select for transposon mutants by plating appropriate dilutions of the resuspended conjugation mixture to reach approximately 50-200 colonies per LB-Sm/Km/Cm plate.</li><li>Incubate plates overnight at 37 Â°C.</li><li>Purify colonies by streaking on LB-Sm/Km/Cm agar plates and in parallel on LB-Ap agar plates. After overnight incubation at 37 Â°C, pursue only with LB-Sm/Km/Cm-resistant, but Ap-sensitive colonies representing bona fide transposon mutants (loss of pLOF backbone).</li><li>Pool ~500 bona fide colonies to generate heterogeneity by adding 1-2 ml of LB on top of the plate and rinsing multiple times over the colonies.</li><li>Repeat Steps B2bi to B2bvi about 20 times for Vc_res as well as Vc_res1 to obtain several independent transposon pools for Vc_res or Vc_res1, respectively.</li></ol></li><li>Integration of pTRIVET and pTRIVET1<br/><ol class="npwfour"><li>Mobilize suicide plasmids pTRIVET or pTRIVET1 into <em>V. cholerae</em> transposon pools obtained by Step B2b to insert the TRIVET or pTRIVET1 via homologous recombination downstream of the <em>lacZ</em>-locus as described below.</li><li>To do so, cross-streak sufficient cell material of <em>E. coli</em> SM10Î»<em>pir</em> pTRIVET or <em>E. coli</em> SM10Î»<em>pir</em> pTRIVET1 obtained from Step B1b with <em>V. cholerae</em> transposon pools obtained by Step B2b (see Step B2aii for details).</li><li>Resuspend conjugation mixture in 1 ml LB broth and select for insertion of the pTRIVET or pTRIVET by plating appropriate dilutions of the resuspended conjugation mixture on LB-Sm/Km/Ap plates.</li><li>Incubate plates overnight at 37 Â°C.</li><li>Pool ~2,000 colonies to maintain heterogeneity by adding 1-2 ml of LB on top of the plate and rinsing multiple times over the colonies. These pools represent the final TRIVET-pools.</li></ol></li></ol></li><li>Identification of <em>ivr</em> genes<br/><ol class="npwthree"><li>Prescreen for elimination of fusion strains with high resolution frequency due to low TetR expression.<br/><ol class="npwfour"><li>Let each TRIVET-pool grow to late log phase in LB-Sm/Ap broth without Km as a selection marker for maintenance of the res or res1-cassette. This allows resolution of the fusion strains with insufficient TetR-expression to silence <em>tnpR</em>.</li><li>Plate serial dilutions on LB-Sm/Km/Ap plates and incubate overnight at 37 Â°C to select for non-resolved strains in the TRIVET pool (res- or res1 cassette is stably maintained under <em>in</em> <em>vitro</em> conditions).</li><li>Pool ~1,000-2,000 colonies to maintain heterogeneity by adding 1-2 ml of LB on top of the plate and rinsing multiple times over the colonies. (Optional: To confirm heterogeneity, detect restriction fragment length polymorphism of the <em>tpc</em>-fusion loci from 50 randomly picked colonies using Southern blot analysis.)</li></ol></li><li>Screening for <em>ivr</em> genes of <em>V. cholerae</em> using the infant mouse model (<em>in vivo</em>).<br/><ol class="npwfour"><li>Spread an aliquot of each pool of the library in triplicate on LB-Sm/Km/Ap plates and incubate overnight at 37 Â°C.</li><li>Collect ~5,000 colonies from each plate and dilute in LB to a concentration of ~10<sup>6</sup> cfu/ml.</li><li>Use 50 Âµl of the bacterial suspension to intragastrically inoculate 5-d-old CD-1 mice (anesthetized by isoflurane).</li><li>Euthanize mice 22 h post infection by cervical dislocation, remove small intestine and homogenize in 1 ml LB media containing 20% Glycerol using a tissue homogenizer at medium power for 30 s while moving up and down.</li><li>Plate serial dilutions of the homogenate on sucrose agar to select for loss of <em>sacB</em> (selection for <em>in vivo</em> resolved strains now lacking the res or res1 cassette).</li><li>Incubate up to 48 h at RT.</li><li>Confirm resolution (loss of res or res1 cassette by streaking in parallel on LB-Sm/Ap (growth) and LB-Km (no growth).</li><li>Incubate plates overnight at 37 Â°C and proceed to identify the <em>tpc</em>-cassette insertion site according to Step B3d with Sm/Km-resistant, but Km-sensitive colonies.</li></ol></li><li>PhoA assays as refinement step of <em>in vivo</em> resolved strains<br/><ol class="npwfour"><li>Based on previous studies using the resolution technology, the <em>in vivo</em> screen has a substantial false positive rate of spontaneous resolution due to low <em>in vitro</em> expression levels of TetR in some strains of the TRIVET library. The implementation of <em>phoA</em> as a second reporter enables confirmation of substantial expression levels during <em>in vitro</em> growth for each of the <em>in vivo</em> resolved strains identified in Step B3b. We highly recommend this refinement step to eliminate false positives.</li><li>To reduce the relatively high false positive rate of resolvase based screens of approximately 15% a spontaneous <em>in vitr</em><em>o</em> resolution frequency has to be defined as a cut-off. Reanalysis of the combined data acquired by previous resolvase-based screens (Osorio <em>et al.</em>, 2005; Schild <em>et al.</em>, 2007; Seper <em>et al.</em>, 2014) reveal that strains with <em>in vitro</em> resolution frequencies below 30% have only a 5% chance to be false positive. As the tpc cassette remains stably integrated in the chromosome PhoA activity of <em>in vivo</em> resolved strains can be measured under <em>in vitro</em> conditions and correlate the obtained activity to a resolution frequency. According to the results obtained for the irgA-fusion test strain, an <em>in vitro</em> resolution frequency of 30% or lower correlates with PhoA activities of 10 Miller units or higher. Thus, we used this cut-off in the original study describing TRIVET (Cakar <em>et al.</em>, 2018).</li><li>Identify a promoter, which is differentially regulated under controllable <em>in vitro</em> cultivation conditions. Herein, we use the iron-regulated <em>irgA</em> promoter of the <em>V. cholerae</em>, which is highly expressed at low iron levels and repressed at high concentrations of iron. Differential iron availability <em>in vitro</em> can be mimicked in LB by adding varying amounts of the iron chelator 2,2'-bipyridyl or FeSO<sub>4</sub>.</li><li>Construct a <em>V. cholerae</em> TRIVET reporter strain Vc_res_TRIVET<em>irgA::tpc</em> with a specific transcriptional fusion of the <em>tpc</em>-cassette to the iron-regulated <em>irgA</em> promoter (see Step C1 for details).</li><li>Assess in parallel the resolution frequency and PhoA activity of the Vc_res_TRIVETi<em>r</em><em>gA::tpc</em> reporter after growth for 8 h in LB supplemented with varying amounts of iron chelators such as 2,2'-bipyridyl or FeSO<sub>4</sub> (see Steps C2 and C3 for details). We recommend to test a range of 0 to 150 ÂµM final concentrations in 50 ÂµM increments for both agents.</li><li>Correlate PhoA activity and resolution frequency of the Vc_res_TRIVET<em>irgA::tpc</em> strain grown in LB with different iron-availability. Choose the growth condition closest to the cut-off (less than 30% resolution frequency) and use the associated PhoA activity as refinement criterion (in our case 10 Miller Units).</li><li>Cultivate <em>in vivo</em> resolved strains in LB for 8 h at 37 Â°C and 180 rpm.</li><li>Perform PhoA assay as described above (see Step A5 for details).</li><li>Proceed only with <em>in vivo</em> resolved strains showing PhoA Activities equal or higher than the refinement criterion (determined above).</li></ol></li><li>Identification of <em>tpc</em> fusion by subcloning and sequencing<br/><ol class="npwfour"><li>Isolate chromosomal DNA from overnight cultures of the <em>in vivo</em> resolved strains of interest.<br/></li><li>Digest chromosomal DNA with EcoRI and use the digested chromosomal DNA for ligation with an EcoRI-digested, dephosphorylated pBR322 plasmid (see Steps A3 and A4 for details).</li><li>Transform ligation product into <em>E. coli</em> DH5Î±Î»<em>pir</em> and plate on LB-Cm plates to select for the <em>cat</em>-gene of the <em>tpc</em>-cassette including downstream chromosomal DNA of the tpc-fusion site.</li><li>Isolate plasmids of Cm<sup>R</sup>-colonies using a QIAprep Spin Miniprep Kit.</li><li>Sequence with oligonucleotide seqPrimer-cat to obtain downstream sequence <em>tpc</em>-fusion site.</li><li>BLAST sequence against the <em>V. cholerae</em> N16961 genome database using blastN to identify insertion site of the tpc-cassette.</li></ol><br/></li></ol></li></ol></li><li>Resolution assays to measure gene silencing<ol class="npwtwo"><li>Construction of suicide plasmids for transcriptional fusions of the tpc cassette to specific promoters, exemplified by an <em>irgA-tpc</em> or <em>yrb-tpc</em> fusion.<br/><ol class="npwthree"><li>Construction of suicide plasmids.<br/><ol class="npwfour"><li>In case of the <em>irgA-tpc</em> fusion, PCR amplify the <em>tpc</em>-cassette using the oligonucleotide pair tetRphoAcat-5â²-SacI and tetRphoAcat-3â²-SalI as well as pTRc-tpc as template (see Step A1 for details).</li><li>PCR amplify 800 bp fragments located upstream and downstream of the <em>irgA</em>-insertion site (<em>irgA</em> promoter) using the oligonucleotide pairs irgA-SphI-1 and irgA-SacI-2 as well as irgA-SalI-3 and irgA-XbaI-4 (see Step A1 for details).</li><li>In case of the <em>yrb-tpc</em> fusion, PCR amplify the <em>tpc</em>-cassette using the oligonucleotide pair tetRphoAcat-5â²-SacI and tetRphoAcat-3â²-SalI as well as pTRc-tpc as template (see Step A1 for details).</li><li>PCR amplify 800 bp fragments located upstream and downstream of the <em>yrb</em>-insertion site (<em>yrb</em> promoter) of the <em>tpc</em>-cassette using the oligonucleotide pairs, yrb_SacI_1 and yrb_SphI_2 as well as yrb_SalI_3 and yrb_XbaI_4.</li><li>Gel-purify PCR fragments followed by restriction digestion and second purification (see Steps A1 and A3 for details).</li><li>In case of the <em>irgA-tpc</em> fusion, ligate the digested tpc-cassette as well as the up- and downstream fragments (see Steps C1ai and ii) into an SphI/XbaI-digested, dephosphorylated pCVD442.</li><li>In case of the <em>yrb-tpc</em> fusion, ligate the digested tpc-cassette as well as the up- and downstream fragments (see Steps C1aii and iv) into an SphI/XbaI-digested, dephosphorylated pCVD442.</li><li>Transform ligation products into competent DH5Î±Î»<em>pir</em> cells and select by plating on LB agar supplemented with Ap.</li><li>Incubate plates overnight at 37 Â°C</li><li>Confirm correct pCVD442irgA::tpc or pCVD442yrb::tpc constructs via colony PCR using the oligonucleotide pairs irgA-SphI-1 and irgA-XbaI-4 or yrb_SphI_2 and yrb_XbaI_4 , respectively.</li><li>Grow verified colonies with correct pCVD442irgA::tpc or pCVD442yrb::tpc in LB-Ap overnight at 37 Â°C and 180 rpm.</li><li>Isolate the pCVD442irgA::tpc or pCVD442yrb::tpc plasmid via the QIAprep Spin Miniprep Kit by following the manufacturerâs protocol.</li><li>Transform pCVD442irgA::tpc or pCVD442yrb::tpc into competent SM10Î»<em>pir</em> cells and select by plating on LB agar supplemented with Ap.</li><li>Incubate plates overnight at 37 Â°C.</li></ol></li><li>Construction of TRIVET strains harboring a specific <em>tpc</em>-fusion to a gene of interest.<br/><ol class="npwfour"><li>To obtain Vc <em>irgA::tpc</em> and Vc <em>yrb::tpc</em>, mobilize suicide plasmids pCVD442irgA::tpc or pCVD442yrb::tpc into <em>V. cholerae</em> by conjugation and allelic exchange to place the <em>tpc</em>-cassette downstream of the respective promoter.</li><li>Mobilize suicide plasmids pCVD442irgA::tpc or pCVD442yrb::tpc into <em>V. cholerae</em> by conjugation of <em>E. coli</em> SM10Î»<em>pir</em> pCVD442irgA::tpc or pCVD442yrb::tpc obtained by Step C1a with <em>V. cholerae</em> WT [(streptomycin-resistant (Sm<sup>R</sup>)]. To do so, cross-streak sufficient cell material of both strains on the same LB agar plate and incubate plate for 6 h at 37 Â°C. Via homologous regions present on pCVD442irgA::tpc or pCVD442yrb::tpc an allelic exchange of the suicide plasmids and the respective locus on the <em>V. cholerae</em> chromosome at low frequency is possible.</li><li>Streaking of single colonies form the conjugation mixtures on LB-Sm/Ap agar plates selects for <em>V. cholerae</em> with integrated pCVD442irgA::tpc or pCVD442yrb::tpc. </li><li>Incubate plates overnight at 37 Â°C.</li><li>Streak-purify colonies on LB-Sm/Ap agar plates at least twice.</li><li>Grow purified colonies in LB-Sm overnight at 37 Â°C, 180 rpm and plate appropriate dilutions on sucrose agar.</li><li>Incubate plates for up to 48 h at RT.</li><li>Confirm loss of suicide vector backbone by streaking on LB-Sm (growth) and LB-Ap (no growth)</li><li>Incubate plates overnight at 37 Â°C, proceed with Sm<sup>R</sup>, but Ap<sup>S</sup> colonies.</li><li>Validate correct insertion of the <em>tpc</em>-cassette via colony PCR and/or sequencing resulting in Vc <em>irgA::tpc</em> and Vc <em>yrb::tpc</em>.</li><li>Insert res- or res1-cassette into Vc <em>irgA::tpc</em> or Vc <em>yrb::tpc</em> to obtain Vc_res <em>irgA::tpc</em>, Vc_res1 <em>irgA::tpc</em>, Vc_res <em>yrb::tpc</em> or Vc_res1 <em>yrb::tpc</em> as described above (see Step B2a for details).</li><li>Integrate pTRIVET or pTRIVET1 into Vc_res <em>i</em><em>rgA::tpc</em>, Vc_res1 <em>irgA::tpc</em>, Vc_res <em>yrb::tpc</em> or Vc_res1 <em>yrb::tpc</em> to obtain Vc_res_TRIVET <em>irgA::tpc</em>, Vc_res1_TRIVET <em>irgA::tpc</em>, Vc_res_TRIVET <em>yrb::tpc</em> or Vc_res1_TRIVET <em>yrb::tpc</em> as well as Vc_res_TRIVET1 <em>irgA::tpc</em>, Vc_res1_TRIVET1 <em>irgA::tpc</em>, Vc_res_TRIVET1 <em>yrb::tpc</em> or Vc_res1_TRIVET1 <em>yrb::tpc</em> as described above (see Step B2c for details).<br/></li></ol></li></ol></li><li>Quantification of gene silencing<br/><ol class="npwthree"><li><em>In vivo</em> and <em>in vitro</em> resolution assay using<br/><ol class="npwfour"><li>Grow TRIVET strain with a transcriptional fusion of the tpc-cassette to the promoter of interest [<em>e.g.</em>, Vc_res1_TRIVET <em>yrb::tpc</em> as used in Cakar <em>et al.</em> (2018)] on LB- Sm/Km/Ap plates; overnight at 37 Â°C.</li><li>Harvest bacterial cells from plate and resuspend them in LB-Sm/Km/Ap.</li><li>Measure OD<sub>600</sub> and adjust OD<sub>600</sub> to 1 using appropriate dilutions to generate the inoculum.</li><li>To assess the <em>in vivo</em> resolution, infect 5-d-old CD-1 mice (anesthetized by isoflurane) with 50 Âµl of the inoculum (â¼10<sup>6</sup> cfu). At a post infection time point of interest [<em>e.g.</em>, 6 and 22 h as used in Zingl <em>et al.</em> (2020)] euthanize mice by cervical dislocation, remove small intestine and homogenize in 1 ml LB media containing 20% Glycerol. Plate appropriate serial dilutions of the homogenate on LB-Sm/Km and LB-Sm/Ap plates and incubate overnight at 37 Â°C.</li><li>To assess the <em>in vitro</em> resolution, inoculate 5 ml of LB-Sm/Ap with 50 Âµl of the inoculum and incubate parallel to the mouse infection at 37 Â°C and 180 rpm. Plate appropriate serial dilutions of the homogenate on LB-Sm/Km and LB-Sm/Ap plates and incubate overnight at 37 Â°C.</li><li>Assess the amount of single colonies on plates and calculate the Sm/Km-resistant and Sm/Ap-resistant cfu in the original sample.</li><li>Calculate resolution (%) for <em>in vivo</em> and <em>in vitro</em> formulas described in the chapter data analysis.</li></ol></li><li><em>In vitro</em> PhoA Assays as alternative quantification of transcriptional activity.<br/><ol class="npwfour"><li>Grow the strains using cultivation conditions of interest.</li><li>Perform PhoA assay as described above (see Step A5 for details).</li></ol></li></ol></li></ol></li></ol><p class="pptt" id="biaoti36862">### Data analysis</p><p class="ppzz">Results of the resolution assay will be expressed as the percent resolution, which is calculated using the determined cfu of the Sm/Km-resistant and Sm/Ap-resistant population within a given culture sample. The Sm/Ap-resistant colonies reflect the entire population and the Sm/Km-resistant colonies reflects the unresolved population, Subtraction of the unresolved population from the entire population gives the resolved population (loss of res-/res1-cassette).<br/>Percent resolution is calculated as:<br/><br/><strong>    Resolution (%) = [cfu(</strong>Sm/Ap-resistant<strong>) â cfu(</strong>Sm/Km-resistant)<strong>]/cfu</strong><strong>(</strong>Sm/Ap-resistant<strong>)</strong><br/><br/>For calculation of alkaline phosphatase activity use the formula:<br/><br/>    Activity (Miller Units) = (1,000 x OD<sub>420</sub>)/[reaction time (min) x OD<sub>600</sub>]</p><p class="pptt" id="biaoti36863">### Recipes</p><ol class="npwtwok"><li>LB broth/agar (g/L)<br/>Bacto<sup>TM</sup> Tryptone 10 g<br/>Yeast extract 5 g<br/>NaCl 10 g<br/>Agar 15 g</li><li>Sucrose agar (g/L)<br/>Bacto<sup>TM</sup> Tryptone 10 g<br/>Yeast extract 5 g<br/>Agar 15 g<br/>Sucrose 100 g<br/></li></ol><p class="ppzzs">If appropriate add antibiotics after autoclaving in the following concentrations: Antibiotics and other supplements were used in the following final concentrations: streptomycin (Sm, 100 Âµg/ml), ampicillin (Ap, 50 Âµg/ml in combination with other antibiotics, otherwise 100 Âµg/ml), kanamycin (Km, 50 Âµg/ml) and chloramphenicol (Cm, 2 Âµg/ml for <em>V. cholerae</em>; 10 Âµg/ml for <em>E. coli</em>).</p><p class="pptt" id="biaoti36864">### Acknowledgments</p><p class="ppzz">We thank Andrew Camilli (Tufts University, Boston) for providing the original RIVET components and helpful discussions to establish the TRIVET system. This work was supported by Austrian Science Fund (FWF) Grants W901 (DK Molecular Enzymology) (to F.G.Z. and S.S.), 27654 and P25691 (to S.S.).</p><p class="pptt" id="biaoti36865">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti36866">### Ethics</p><p class="ppzz">Animals were used in all experiments in accordance with the rules of the ethics committee at the University of Graz and the corresponding animal protocol, which has been approved by Austrian Federal Ministry of Science and Research Ref. II/10b. Mice were housed with food and water ad libitum and monitored under the care of full-time staff.</p><p class="pptt" id="biaoti36867">### References</p><ol class="npwtwok"><li>Amann, E., Ochs, B. and Abel, K. J. (1988). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3069586" target="_blank">Tightly regulated <em>tac</em> promoter vectors useful for the expression of unfused and fused proteins in <em>Escherichia coli.</em></a> <em>Gene</em> 69: 301-315. </li><li>Bookout, A. L. and Mangelsdorf, D. J. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16604184" target="_blank">Quantitative real-time PCR protocol for analysis of nuclear receptor signaling pathways.</a> <em>Nucl Recept Signal</em> 1: e012.</li><li>Cakar, F., Zingl, F. G., Moisi, M., Reidl, J. and Schild, S. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29463743" target="_blank"><em>In vivo</em> repressed genes of <em>Vibrio cholerae</em> reveal inverse requirements of an H<sup>+</sup>/Cl<sup>-</sup> transporter along the gastrointestinal passage.</a> <em>Proc Natl Acad Sci U S </em><em>A</em> 115(10): E2376-E2385.</li><li>Cakar, F., Zingl, F.G. and Schild, S. (2019). <a href="https://pubmed.ncbi.nlm.nih.gov/30110191/" target="_blank">Silence is golden: gene silencing of <em>V. cholerae</em> during intestinal colonization delivers new aspects to the acid tolerance response.</a> <em>Gut Microbes</em> 10(2): 228-234.</li><li>Donnenberg, M. S. and Kaper, J. B. (1991). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC259042/" target="_blank">Construction of an <em>eae</em> deletion mutant of enteropathogenic <em>Escherichia coli</em> by using a positive-selection suicide vector.</a> <em>Infect Immun</em> 59: 4310-4317. </li><li>Eisen, M. B. and Brown, P. O. (1999). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10349646" target="_blank">DNA arrays for analysis of gene expression.</a> <em>Methods Enzymol</em> 303: 179-205.</li><li>Hava, D. L., Hemsley, C. J. and Camilli, A. (2003). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12511486" target="_blank">Transcriptional regulation in the <em>Streptococcus pneumoniae</em> <em>rlrA </em>pathogenicity islet by RlrA.</a> <em>J Bacteriol</em> 185(2): 413-421. </li><li>Herrero, M., de Lorenzo, V. and Timmis, K. N. (1990). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2172216" target="_blank">Transposon vectors containing non-antibiotic resistance selection markers for cloning and stable chromosomal insertion of foreign genes in gram-negative bacteria.</a> <em>J Bacteriol</em> 172(11): 6557-6567. </li><li>List, C., Grutsch, A., Radler, C., Cakar, F., Zingl, F. G., Schild-Prufert, K. and Schild, S. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29794057" target="_blank">Genes activated by <em>Vibrio cholerae</em> upon exposure to <em>Caenorhabditis elegans</em> reveal the mannose-sensitive hemagglutinin to be essential for colonization.</a> <em>mSphere</em> 3(3).</li><li>Miller, V. L., DiRita, V. J. and Mekalanos, J. J. (1989). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2646275" target="_blank">Identification of toxS, a regulatory gene whose product enhances toxR-mediated activation of the cholera toxin promoter.</a> <em>J Bacteriol</em> 171(3): 1288-1293. </li><li>Miller, V. L. and Mekalanos, J. J. (1988). <a href="https://www.ncbi.nlm.nih.gov/pubmed/2836362" target="_blank">A novel suicide vector and its use in construction of insertion mutations: osmoregulation of outer membrane proteins and virulence determinants in <em>Vibrio cholerae</em> requires <em>toxR</em>.</a> <em>J. Bacteriol</em> 170(6): 2575-2583. </li><li>Osorio, C. G., Crawford, J. A., Michalski, J., Martinez-Wilson, H., Kaper, J. B. and Camilli, A. (2005). <a href="http://www.ncbi.nlm.nih.gov/PubMed/15664940" target="_blank">Second-generation recombination-based <em>in vivo</em> expression technology for large-scale screening for <em>Vibrio cholerae</em> genes induced during infection of the mouse small intestine.</a> <em>Infect Immun</em> 73(2): 972-980.</li><li>Reed, R. R. and Grindley, N. D. (1981). <a href="https://www.ncbi.nlm.nih.gov/pubmed/6269756" target="_blank">Transposon-mediated site-specific recombination <em>in vitro</em>: DNA cleavage and protein-DNA linkage at the recombination site.</a> <em>Cell</em> 25(3): 721-728.</li><li>Schild, S., Tamayo, R., Nelson, E. J., Qadri, F., Calderwood, S. B. and Camilli, A. (2007). <a href="http://www.ncbi.nlm.nih.gov/PubMed/18005744" target="_blank">Genes induced late in infection increase fitness of <em>Vibrio cholerae</em> after release into the environment.</a> <em>Cell Host Microbe</em> 2(4): 264-277. </li><li>Seper, A., Pressler, K., Kariisa, A., Haid, A. G., Roier, S., Leitner, D. R., Reidl, J., Tamayo, R. and Schild, S. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24962154" target="_blank">Identification of genes induced in <em>Vibrio cholerae</em> in a dynamic biofilm system.</a> <em>Int</em> <em>J Med Microbiol</em> 304(5-6): 749-763. </li><li>Wang, Z., Gerstein, M. and Snyder, M. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19015660" target="_blank">RNA-Seq: a revolutionary tool for transcriptomics.</a> <em>Nat Rev Genet </em>10(1): 57-63. </li><li>Zingl, F. G., Kohl, P., Cakar, F., Leitner, D. R., Mitterer, F., Bonnington, K. E., Rechberger, G. N., Kuehn, M. J., Guan, Z., Reidl, J. and Schild, S. (2020). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31901519" target="_blank">Outer membrane vesiculation facilitates surface exchange and <em>in vivo</em> adaptation of <em>Vibrio cholerae</em>.</a> <em>Cell Host Microbe</em> 27(2): 225-237 e228.<br/></li></ol>