<body>
            <h1>A Î²-glucuronidase (GUS) Based Bacterial Competition Assay to Assess Fine Differences in Fitness during Plant Infection</h1>
            <p><strong>Authors:</strong> Julien S. Luneau, Laurent D. NoÃ«l, Emmanuelle Lauber, Alice Boulanger</p>
            <h2>Abstract</h2>
            <p><p>Competition assays are a simple phenotyping strategy that confront two bacterial strains to evaluate their relative fitness. Because they are more accurate than single-strain growth assays, competition assays can be used to highlight slight differences that would not otherwise be detectable. In the frame of host-pathogens interactions, they can be very useful to study the contribution of individual bacterial genes to bacterial fitness and lead to the identification of new adaptive traits. Here, we describe how to perform such competition assays by taking the example of the model phytopathogenic bacterium <em>Xanthomonas campestris</em> pv. <em>campestris</em> during infection of the mesophyll of its cauliflower host. This phenotypic assay is based on the use of a Competitive Index (CI) that compares the relative abundance of co-inoculated strains before and after inoculation. Since multiplication is a direct proxy for bacterial fitness, the evolution of the ratio between both strains in the mixed population is a direct way to assess differences in fitness in a given environment. In this protocol, we exploit the blue staining of GUS-expressing bacteria to count blue <em>vs</em>. white colonies on plates and estimate the competitiveness of the strains of interest in plant mesophyll.</p></p>
            <p><strong>Keywords:</strong> Competitive Index, Competition assay, Fitness, Mesophyll, <em>Xanthomonas</em>, Cauliflower, GUS, Brassica oleracea</p>
            <h2>Background</h2><p><p>Over the years, topics of ecology and evolution have been increasingly important when studying host-microbe interactions. In addition to simply assessing the virulence of bacterial pathogens, matters of fitness and adaptation to the environment are now being addressed. Recently, large-scale high-throughput screening tools, such as RB-TnSeq (Randomly-Barcoded Transposon insertion site Sequencing), have been developed to efficiently identify the genetic determinants of bacterial fitness. RB-TnSeq relies on the co-inoculation of a library of mutagenized bacteria, all carrying a unique barcode, to simultaneously evaluate the contribution of each gene to bacterial survival and growth (Wetmore <em>et al.</em>, 2015). Such studies result in the identification of genes putatively implicated in bacterial adaptation to the host that then need to be confirmed by reverse genetics using simpler fitness assays.<br/></p><p><br/>Early on, it was proposed that competition assays could be used as an answer to this need. Typically, such competition assays are performed by co-inoculating two strains in a 1:1 ratio into a given environment and following the evolution of this ratio over time. If one of the strains grows more than the other, it is considered as more fit (in this specific environment) than its counterpart. The main limiting factor is then to be able to count separately the two mixed populations. A quite straightforward solution here is to use discriminating markers, such as antibiotic resistance, to count the abundance of each strain after plating on selective medium. Differential antibiotic resistances have been widely used in co-inoculation assays (Macho <em>et al.</em>, 2007 and 2010; Feng <em>et al.</em>, 2012), but it is known that such resistances often carry a strong cost of fitness that could interfere in studies that involve fine fitness mechanisms. Also, they require plating the samples twice, once on each selective medium, which can introduce a bias in the evaluation of the strains relative frequencies. Fluorescent markers provide an alternative to antibiotics but require costly imaging equipment (Perrier <em>et al.</em>, 2019). Here, we describe a visual blue/white screening using the enzymatic properties of the beta-glucuronidase (GUS) enzyme. The assay is then performed by co-inoculating a GUS-expressing strain in competition with a non-tagged strain: after plating on medium containing X-gluc (the substrate of the GUS enzyme), colonies of tagged bacteria will turn blue while the others will remain white. A similar Bio-protocol had been proposed before using <em>LacZ</em> as a marker in the <em>Vibrio cholerae</em> animal pathogen (Fu and Mekalanos, 2014). However, even though the concept is similar, we feel that a detailed protocol for plant inoculations is very much needed.<br/></p><p><br/>Here, we chose the bacterial phytopathogen <em>Xanthomonas campestris</em> pv. <em>campestris (Xcc</em>) as a model to illustrate this protocol. <em>Xcc</em> is particularly interesting because it has the specificity to infect multiple organs and tissues of its cauliflower host (<em>Brassica oleracea</em> pv. <em>botrytis</em> cv. Clovis). After entry through the hydathodes, <em>Xcc</em> spreads through the xylem sap-conducting vessels and colonizes the leaf mesophyll of its host, where it causes the typical symptoms of the Black rot disease (Vicente and Holub, 2013; An <em>et al.</em>, 2019). We recently performed an RB-TnSeq screening on <em>Xcc</em> during cauliflower infection (Luneau <em>et al.</em>, 2022), which ultimately led us to set up a competition assay. In this protocol, we describe how this competition assay can be performed in the plant leaf mesophyll. We take the example of the well-described <em>hrpX</em> (transcriptional activator of the Type III Secretion System), <em>hrcV</em> (Type III Secretion System component), and <em>xopAC</em> (Type III Effector) mutants that display contrasted fitness phenotypes when confronted with the <em>Xcc</em> 8004 wild-type strain <em>in planta</em>. This protocol provides a basis that can be adapted for other plant compartments, such as hydathodes or xylem, as well as <em>in vitro</em> studies. Availability of such protocol is particularly relevant given the number of plant pathogens that enter through and thrive in their host leaves.</p></p>
        </body>

<p class="pptt" id="biaoti46632">## Materials and Reagents</p><ol class="npwtwok"><li><p>2-mL microcentrifuge tubes (Sarstedt, catalog number: 72.695.400)</p></li><li><p>5-mL tubes (Sarstedt, catalog number: 72.701)</p></li><li><p>Petri dishes 92 Ã 16 mm (Sarstedt, catalog number: 82.1473)</p></li><li><p>1-mL Blunt syringe (SOFT-JECT HSW, catalog number: 5010.200V0)</p></li><li><p>Sterile 4-mm diameter glass beads (Dutscher, catalog number: 068502)</p></li><li><p>Sterile 96-well V-bottom clear polystyrene microplates (Greiner Bio-One, catalog number: 651101)</p></li><li><p>Sterile pipette tips</p></li><li><p>Compost (Proveen, catalog number: 14926)</p></li><li><p>Cauliflower: <em>Brassica oleracea</em> var. <em>botrytis</em> cv. Clovis F1 (Vilmorin)</p><p><em>Note: Plants are grown under greenhouse conditions and used at 4 weeks after sowing. All inoculations were performed using the second true leaf (second leaf above the cotyledons). After inoculation, plants are kept in classical growth chamber conditions: 9 h light, 22Â°C, and 70% relative humidity.</em> </p></li><li><p><em>Xanthomonas campestris</em> pv. <em>campestris (Xcc</em>) strain 8004::GUS-GFP (WT hereafter; Cerutti <em>et al.</em>, 2017) and variants: <em>Xcc</em> 8004::GUS*-GFP* (WT* hereafter, carrying the point mutations Y468A and Y66A inactivating the catalytic sites of the Î²-glucuronidase and GFP proteins respectively); <em>Xcc</em> 8004::GUS-GFP Î<em>hrpX; Xcc</em> 8004::GUS-GFP Î<em>hrcV; Xcc</em> 8004::GUS-GFP Î<em>xopAC</em> </p><p><em>Note: With concerns to the cost of fitness related to the constitutive expression of the GUS-GFP cassette, we decided to create the non-functional version of the cassette for our reference strain. With this system, the two challenged strains carry and express the GUS-GFP cassette, but only one of them will actually appear as tagged. We thus remove the possibility of a fitness cost associated with the cassette while still being able to differentiate the two mixed strains.</em> </p></li><li><p>Spectrophotometer cells (Standard Polystyrene Semi-micro Cuvettes, Sarstedt, catalog number: 67.742)</p></li><li><p>Sterile distilled water</p></li><li><p>Magnesium chloride (MgCl<sub>2</sub>) (Merck, catalog number: 105832)</p></li><li><p>Yeast extract (Sigma-Aldrich, catalog number: Y1625)</p></li><li><p>Casamino acids (BD, BD Biosciences, catalog number: 223050)</p></li><li><p>Potassium phosphate dibasic (K<sub>2</sub>HPO<sub>4</sub>) (Merck, catalog number: 105101)</p></li><li><p>Agar (BD, BD Biosciences, catalog number: 214010)</p></li><li><p>Magnesium sulfate heptahydrate (MgSO<sub>4</sub>Â·7H<sub>2</sub>O) (Merck, catalog number: 105886)</p></li><li><p>X-Gluc (5-Bromo-4-chloro-3-indoxyl-beta-D-glucuronic acid, cyclohexylammonium salt monohydrate) (BIOSYNTH, catalog number: B-7300)</p></li><li><p>DMF (N,N Dimethylformamide) (Sigma-Aldrich, catalog number: D4551)</p></li><li><p>Rifampicin (Euromedex, catalog number: 1059)</p></li><li><p>Methanol (VWR, catalog number: 20847.320)</p></li><li><p>Pimaricin (DSM, DEVLOCID Instant, catalog number: 00226)</p></li><li><p>MOKA medium (see Recipes)</p></li><li><p>1 mM MgCl<sub>2</sub> solution (see Recipes)</p></li><li><p>X-gluc (see Recipes)</p></li><li><p>Rifampicin (see Recipes)</p></li><li><p>Pimaricin (see Recipes)</p></li></ol><p class="pptt" id="biaoti46633">### Equipment</p><ol class="npwtwok"><li><p>6-mm diameter Hollow punch (Harris, Uni-Core, Electron Microscopy Sciences, catalog number: 69039-60)</p></li><li><p>Hand-held colony counter (Heathrow Scientific, catalog number: HS120000)</p></li><li><p>28Â°C bacteria incubator</p></li><li><p>Centrifuge (Eppendorf, model: 5415D)</p></li><li><p>Beads-assisted grinder (Retsch, catalog number: MM400)</p></li><li><p>Spectrophotometer (Amersham Biosciences, model: Ultrospec 2100 pro)</p></li><li><p>Sterile environment (bunsen burner or microbiology hood)</p></li><li><p>Multi-channel pipette (10â100 ÂµL)</p></li><li><p>Mono-channel pipettes</p></li></ol><p class="pptt" id="biaoti46634">### Procedure</p><ol class="npwone"><li><p>Preparation of bacterial cells for the inoculation</p><ol class="npwtwok"><li><p>Bacterial cells grown on MOKA plates with appropriate antibiotics (<em>i.e.</em>, Rifampicin 50 Âµg/mL here) are used to inoculate 5 mL of liquid MOKA medium (with same antibiotics) and grown overnight at 28Â°C under agitation at 200 rpm.</p></li><li><p>Centrifuge 2 mL of bacterial preculture for 5 min at 6,000 rpm (<em>i.e</em>., 3.3 <em>Ã g</em>) and wash twice in 1 mM MgCl<sub>2</sub>. Resuspend in 1 mM MgCl<sub>2</sub> and measure OD<sub>600nm</sub>.</p></li></ol><ol class="npwtwok" start="3"><li><p>Prepare 1 mL of bacterial suspension in a 2-mL tube so that, for each strain, final OD<sub>600nm</sub> = 0.1 (<em>i.e.</em>, 1 Ã 10<sup>8</sup> cfu/mL; note: this estimation has to be verified when working with other species by plating on solid medium) in 1 mM MgCl<sub>2</sub>.</p></li><li><p>Dilute each suspension to 1/100 in a 5-mL tube to obtain a final inoculum at OD<sub>600nm</sub> = 0.001 (<em>i.e.</em>, 1 Ã 10<sup>6</sup> cfu/mL) in 5 mL of 1 mM MgCl<sub>2</sub>. For the WT* reference strain, prepare 10 mL of suspension at OD<sub>600nm</sub> = 0.001 as it will be used in all mixes.</p></li><li><p>Mix 2 mL of WT* suspension with 2 mL of each of the strains to test in another 5-mL tube (so that each mixed strain is at 5 Ã 10<sup>5</sup> cfu/mL, but the final concentration of the inoculum is 1 Ã 10<sup>6</sup> cfu/mL).</p><br/></li></ol></li><li><p>Inoculation</p><ol class="npwtwok"><li><p>Using a blunt syringe filled with one milliliter of bacterial suspension, infiltrate the mesophyll of the 2<sup>nd</sup> true leaf of cauliflower (for pictures of the procedure, see Cohn <em>et al.</em>, 2015). A large enough area should be infiltrated in order to later be able to sample a 6-mm leaf disc without taking any of the tissues damaged by the syringe. Typically, up to seven separate spots can be infiltrated on the same cauliflower leaf, thus allowing the inoculation of multiple strains or mixes on the same leaf. In this case, be careful to well separate infiltration areas to exclude cross-contamination.</p></li><li><p>Circle the infiltrated area on the lower side of the leaf to mark the infected tissue for later sampling.</p></li><li><p>Place the plants in the growth chamber and water them appropriately.</p><br/></li></ol></li><li><p>Plating of the inoculation suspensions and counting of the initial relative frequencies in mixed populations</p><ol class="npwtwok"><li><p>Transfer 100 ÂµL of inoculation suspension into a 96-well plate with conical bottom.</p></li><li><p>Perform serial 1/10 dilutions (10 ÂµL in 90 ÂµL of 1 mM MgCl<sub>2</sub>) using a multichannel pipette.</p></li><li><p>Take 100 ÂµL of dilution 10<sup>-3</sup> and spread out them on round 92-mm MOKA plates containing the appropriate antibiotics (<em>i.e.</em>, Rifampicin 50 Âµg/mL here) and fungicide (<em>i.e.</em>, Pimaricin 30 Âµg/mL here) as well as X-Gluc 200 ÂµM. In our case, spreading is performed by shaking using 4-mm glass beads until homogeneous repartition of the suspension.</p></li><li><p>Incubate 4â5 days at 28Â°C to allow bacterial growth and blue staining of GUS-expressing colonies.</p></li><li><p>Count blue and white colonies, <em>e.g.</em>, using a hand-held colony counter, to determine the ratio of mixed strains at T<sub>0</sub>. This is necessary for proper normalization during later calculation of the CI (see Data analysis section).</p><br/></li></ol></li><li><p>Sampling, plating, and counting of relative frequencies in mixed populations after incubation</p><ol class="npwtwok"><li><p>Three days after inoculation, sample mesophyll tissues by punching out a 6-mm leaf disc in the infiltrated area defined by the marker label and placing it in 200 ÂµL of 1 mM MgCl<sub>2</sub>.</p></li><li><p>Grind leaf samples using 4-mm glass beads in 2-mL tubes (2 beads per tube) using the beads-assisted grinder for 2 min at 30 Hz.</p></li><li><p>Centrifuge shortly (5 s) the tubes containing the ground material to gather it at the bottom.</p></li><li><p>Transfer 100 ÂµL of ground tissue to a 96-well plate with conical bottom.</p></li><li><p>Proceed as described in Steps C2 to C5. However, because each strain displays a different phenotype, and to ensure the presence of colonies from both strains, plate 50 ÂµL of 10<sup>-2</sup> and 10<sup>-3</sup> dilutions.</p></li></ol><p><em>Note: Classically, we advise collecting bacterial titers by counting the plates displaying between 30 and 300 colonies. However, because competition assays require to finely determine the number of CFU for each of the mixed strains, we recommend counting plates where there are at least 100 colonies for mix-inoculated samples.</em> </p></li></ol><p class="pptt" id="biaoti46635">### Data analysis</p><p>For reproducibility purposes, experiments should be performed in at least four biological replicates. A biological replicate is defined as a replication of a complete experimental procedure with fresh buffers, fresh bacterial cells suspension prepared from cells grown on new solid medium, and new batches of plants. Results of competition assays are displayed as a Competitive Index (CI), which is defined as the ratio of the relative abundance of mutant over WT* strain after incubation normalized by the relative abundance of mutant over WT* strain in the inoculum (Taylor <em>et al.</em>, 1987; Macho <em>et al.</em>, 2007):</p><p><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20220630/20220630224826_7653.jpg" width="300"/> </p><p><em>N</em>: number of colonies</p><p><em>dpi</em>: days post inoculation</p><p>Here, we used the <em>Xcc</em> 8004::GUS-GFP Î<em>hrpX</em>, Î<em>hrcV</em>, and Î<em>xopAC</em> mutants as an example for this protocol. These three strains are Type III Secretion System-related mutants that exhibit contrasted fitness <em>in planta</em> (Feng <em>et al.</em>, 2012; Guy <em>et al.</em>, 2013; Wang <em>et al.</em>, 2015). Competitive indices of these three strains, when co-inoculated with the WT* strain, are shown in Figure 1. <em>Xcc</em> 8004::GUS-GFP is a negative control whose fitness should be the same as the WT* reference strain (<em>i.e.</em>, CI ~1). Strains with CI &lt; 1 are considered less fit than the WT strain in the mesophyll environment.</p><p>We used four independent biological replicates of one plant sample each. The plot was generated using the ggplot2 package of the R software (Wickham, 2016; <a href="https://github.com/tidyverse/ggplot2">https://github.com/tidyverse/ggplot2</a>). We purposely set a log10-based scale to stretch out low CI values that would be hardly differentiable otherwise. Statistical treatment of the data was also achieved with R using the Kruskal-Wallis test. Groups identified as significantly different (<em>P</em> &lt; 0.05) are displayed using the letters âaâ, âbâ, and âcâ in Figure 1A.</p><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3776/bioprotoc-12-13-3776-g001.jpg"/><br/><strong>Figure 1. Competition assay of <em>Xcc</em> 8004::GUS-GFP strain and related mutants in cauliflower mesophyll.</strong> <p>All competition assays were performed by co-inoculating the <em>Xcc</em> 8004::GUS*-GFP* reference strain with <em>Xcc</em> 8004::GUS-GFP derivatives in a 1:1 ratio. Mesophyll discs were sampled at 3 days post-inoculation. A. Graphical representation of competition assays results. Strains showing CI ~1 are considered as fit as the WT strain, while strains with CI &lt; 1 are considered less fit than the WT strain in the mesophyll environment. Horizontal bars show the median for four independent biological replicates of one plant sample each. Statistical significance of fitness differences was evaluated by a Kruskal-Wallis test. Groups identified as significantly different (<em>P</em> &lt; 0.05) are displayed using the letters âaâ, âbâ, and âcâ. Results showing the same letter belong to the same statistical group. B. Photo of a plate displaying colonies from the WT vs. WT* sample at 3 dpi in the mesophyll. Visually, the two populations are roughly present in a 1:1 ratio, illustrating the expected equivalent fitness of both strains. The same 1:1 ratio is obtained after plating any of the inoculation suspensions. C. Photo of a plate displaying colonies from the Î<em>hrpX</em> vs. WT* sample at 3 dpi in the mesophyll. Blue-stained colonies of the Î<em>hrpX</em> strain are strongly under-represented, clearly exposing the large deficit of fitness of this strain.</p><p class="pptt" id="biaoti46636">### Recipes</p><ol class="npwtwok"><li><p>MOKA medium</p><ol class="npwthree"><li><p>Mix 4 g of yeast extract, 8 g of casamino acids, 2 g of K<sub>2</sub>HPO<sub>4</sub>, and 0.3 g of MgSO<sub>4</sub>Â·7H<sub>2</sub>O.</p></li><li><p>Add distilled water to 1 L.</p></li><li><p>Mix until complete dissolution of the reagents.</p></li><li><p>Sterilize 250-mL aliquots of the mixture by autoclaving at 120Â°C for 20 min.</p></li><li><p>Store at room temperature.</p></li><li><p>Solid MOKA medium for plates is prepared identically and complemented with 3.75 g of agar added to each individual 250-mL aliquot before autoclaving.</p></li></ol></li><li><p>1 mM MgCl<sub>2</sub> solution</p><ol class="npwthree"><li><p>Prepare a stock solution of 1 M MgCl<sub>2</sub> by dissolving 2.033 g of MgCl<sub>2</sub>Â·6H<sub>2</sub>O in 10 mL of dH<sub>2</sub>O.</p></li><li><p>Sterilize by autoclaving at 120Â°C for 20 min.</p></li><li><p>Prepare 1 mM MgCl<sub>2</sub> by diluting the stock solution 1/1,000 in sterile dH<sub>2</sub>O.</p></li><li><p>Both solutions are stored at room temperature.</p></li></ol></li><li><p>X-gluc</p><ol class="npwthree"><li><p>Prepare X-gluc 500x stock solution by dissolving 0.54 g of X-gluc in 10 mL of DMF.</p></li><li><p>Store at -20Â°C.</p></li><li><p>Add 500 ÂµL of X-gluc stock solution to 250 mL of MOKA + Agar solution to get a final concentration of 200 ÂµM of X-gluc in plates used for CI measurement.</p></li></ol></li><li><p>Rifampicin</p><ol class="npwthree"><li><p>Dissolve rifampicin powder in methanol to obtain a stock solution at 10 mg/mL.</p></li><li><p>Store at -20Â°C.</p></li><li><p>Dilute rifampicin stock solution to 1/200 in MOKA media to obtain a final concentration of 50 Âµg/mL.</p></li></ol></li><li><p>Pimaricin</p><ol class="npwthree"><li><p>Resuspend pimaricin powder in water to obtain a stock solution at 30 mg/mL.</p></li><li><p>Store at 4Â°C.</p></li><li><p>Dilute pimaricin stock solution to 1/1,000 in MOKA media to obtain a final concentration of 30 Âµg/mL.</p></li></ol></li></ol><p class="pptt" id="biaoti46637">### Acknowledgments</p><p>This work was supported by a Ph.D. grant from the French Ministry of National Education and Research to JSL. JSL, LDN, EL, and AB are supported by the ANR (ANR-19-CE20-0014-01). LIPM is part of the French Laboratory of Excellence project (TULIP ANR-10-LABX-41; ANR-11-IDEX-0002-02).</p><p class="pptt" id="biaoti46638">### Competing interests</p><p>There are no competing interests.</p><p class="pptt" id="biaoti46639">### References</p><ol class="npwtwok"><li>An, S. Q., Potnis, N., Dow, M., VorhÃ¶lter, F. J., He, Y. Q., Becker, A., Teper, D., Li, Y., Wang, N., Bleris, L. and Tang, J. L. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31578554">Mechanistic insights into host adaptation, virulence and epidemiology of the phytopathogen <i>Xanthomonas</i>.</a> <i>FEMS Microbiol Rev</i>.</li><li style="text-align:justify;">Cerutti, A., Jauneau, A., Auriac, M. C., Lauber, E., Martinez, Y., Chiarenza, S., Leonhardt, N., Berthome, R. and Noel, L. D. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28184011">Immunity at cauliflower hydathodes controls systemic Infection by <i>Xanthomonas campestris</i> pv <i>campestris</i>.</a> <i>Plant Physiol</i> 174(2): 700-716.</li><li style="text-align:justify;">Cohn, M., Shybut, M., Dahlbeck, D., and Staskawicz, B. (2015). <a href="https://en.bio-protocol.org/e1522">Assays to Assess Virulence of <i>Xanthoonas axonopodis</i> pv. <i>manihotis</i> on Cassava.</a> <i>Bio-protocol</i> 5(13): 1522.</li><li style="text-align:justify;">Feng, F., Yang, F., Rong, W., Wu, X., Zhang, J., Chen, S., He, C. and Zhou, J. M. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22504181">A <i>Xanthomonas</i> uridine 5'-monophosphate transferase inhibits plant immune kinases.</a> <i>Nature</i> 485(7396): 114-118.</li><li style="text-align:justify;">Fu, Y. and Mekalanos, J. J. (2014). <a href="https://bio-protocol.org/e1147">Infant rabbit colonization competition assays.</a> <i>Bio-protocol</i> 4(11): e1147.</li><li style="text-align:justify;">Guy, E., Lautier, M., Chabannes, M., Roux, B., Lauber, E., Arlat, M. and Noel, L. D. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23951354">xopAC-triggered immunity against <i>Xanthomonas</i> depends on <i>Arabidopsis</i> receptor-like cytoplasmic kinase genes PBL2 and RIPK.</a> <i>PLoS One</i> 8(8): e73469.</li><li style="text-align:justify;">Luneau, J. S., Baudin, M., Quiroz Monnens, T., CarrÃ¨re, S., Bouchez, O., Jardinaud, M. F., Gris, C., FranÃ§ois, J., Ray, J., Torralba, B., Arlat, M., Lewis, J. D., Lauber, E., Deutschbauer, A. M., NoÃ«l, L. D., Boulanger, A. <a href="https://pubmed.ncbi.nlm.nih.gov/35706385/">Genome-wide identification of fitness determinants in the Xanthomonas campestris bacterial pathogen during early stages of plant infection.</a> <i>New Phytol</i>. 2022 Jun 15. </li><li style="text-align:justify;">Macho, A. P., Guidot, A., Barberis, P., BeuzÃ³n, C. R. and Genin, S. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20687809">A competitive index assay identifies several <i>Ralstonia solanacearum</i> type III effector mutant strains with reduced fitness in host plants.</a> <i>Mol Plant Microbe Interact</i> 23(9): 1197-1205.</li><li style="text-align:justify;">Macho, A. P., Zumaquero, A., Ortiz-Martin, I. and BeuzÃ³n, C. R. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20507512">Competitive index in mixed infections: a sensitive and accurate assay for the genetic analysis of <i>Pseudomonas syringae</i>-plant interactions.</a> <i>Mol Plant Pathol</i> 8(4): 437-450.</li><li style="text-align:justify;">Perrier, A., Barlet, X., Rengel, D., Prior, P., Poussier, S., Genin, S. and Guidot, A. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31209989">Spontaneous mutations in a regulatory gene induce phenotypic heterogeneity and adaptation of <i>Ralstonia solanacearum</i> to changing environments.</a> <i>Environ Microbiol</i> 21(8): 3140-3152.</li><li style="text-align:justify;">Taylor, R. K., Miller, V. L., Furlong, D. B. and Mekalanos, J. J. (1987). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2883655">Use of phoA gene fusions to identify a pilus colonization factor coordinately regulated with cholera toxin.</a> <i>Proc Natl Acad Sci U S A</i> 84(9): 2833-2837.</li><li style="text-align:justify;">Vicente, J. G. and Holub, E. B. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23051837"><i>Xanthomonas campestris</i> pv. <i>campestris</i> (cause of black rot of crucifers) in the genomic era is still a worldwide threat to brassica crops.</a> <i>Mol Plant Pathol</i> 14(1): 2-18.</li><li style="text-align:justify;">Wang, G., Roux, B., Feng, F., Guy, E., Li, L., Li, N., Zhang, X., Lautier, M., Jardinaud, M. F., Chabannes, M., <i>et al</i>. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26355215">The decoy Substrate of a pathogen effector and a pseudokinase specify pathogen-induced modified-Self recognition and immunity in plants.</a> <i>Cell Host Microbe</i> 18(3): 285-295.</li><li style="text-align:justify;">Wetmore, K. M., Price, M. N., Waters, R. J., Lamson, J. S., He, J., Hoover, C. A., Blow, M. J., Bristow, J., Butland, G., Arkin, A. P., <i>et al</i>. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25968644">Rapid quantification of mutant fitness in diverse bacteria by sequencing randomly bar-coded transposons.</a> <i>MBio</i> 6(3): e00306-00315.</li><li style="text-align:justify;">Wickham, H. (2016). <a href="https://ggplot2.tidyverse.org/">ggplot2: elegant graphics for data analysis.</a> In: Wickham, H. (Ed.). <i>ggplot2</i>. Springer-Verlag New York. ISBN: 978-3-319-24277-4.</li></ol>