<body>
            <h1>Live-cell Imaging and Quantitative Analysis of Meiotic Divisions in <em>Caenorhabditis elegans</em> Males</h1>
            <p><strong>Authors:</strong> Gunar Fabig, Falko LÃ¶ffler, Christian GÃ¶tze, Thomas MÃ¼ller-Reichert</p>
            <h2>Abstract</h2>
            <p>Live-imaging of meiotic cell division has been performed in extracted spermatocytes of a number of species using phase-contrast microscopy. For the nematode <em>Caenorhabditis elegans</em>, removal of spermatocytes from gonads has damaging effects, as most of the extracted spermatocytes show a high variability in the timing of meiotic divisions or simply arrest during the experiment. Therefore, we developed a live-cell imaging approach for <em>in situ</em> filming of spermatocyte meiosis in whole immobilized <em>C. elegans</em> males, thus allowing an observation of male germ cells within an unperturbed environment. For this, we make use of strains with fluorescently labeled chromosomes and centrosomes. Here we describe how to immobilize male worms for live-imaging. Further, we describe the workflow for the acquisition and processing of data to obtain quantitative information about the dynamics of chromosome segregation in spermatocyte meiosis I and II. In addition, our newly developed approach allows us to re-orient filmed spindles <em>in silico</em>, regardless of the initial 3D orientation in the worm, and analyze spindle dynamics in living worms in a statistically robust manner. Our live-imaging approach is also applicable to <em>C. elegans</em> hermaphrodites and should be expandable to other fluorescently labelled nematodes or other fully transparent small model organisms.</p>
            <p><strong>Keywords:</strong> Male meiosis, Spermatocytes, Spindle, Metaphase, Anaphase, Live-imaging, Dynamics, Light microscopy</p>
            <h2>Background</h2><p>In the past, studies on male meiosis have been performed on isolated and stained specimens to image meiotic spindles, with obvious limitations of using fixed specimens (Boring, 1909; Boveri, 1909; Goldstein, 1977). Live-imaging was applied to analyze the dynamics of meiosis in extracted spermatocytes of a number of species, including insects, such as grasshoppers (Nicklas and Staehly, 1967; Nicklas and Koch, 1969; Ault, 1986), the crane-fly <em>Nephrotoma suturalis</em> (Janicke and LaFountain, 1982; LaFountain <em>et al.</em>, 2002 and 2012; Forer <em>et al.</em>, 2018), the flatworm <em>Mesostoma ehrenbergii</em> (Fegaras and Forer, 2018), the fruit fly <em>Drosophila melanogaster</em> (Bonaccorsi and Gatti, 2017) and spiders (Golding and Paliulis, 2011; Felt <em>et al.</em>, 2017). Extracted insect spermatocytes were observed with phase-contrast and polarized light microscopy and very elegant experiments were performed on spindles or single chromosomes using micromanipulators and laser microsurgery (King and Nicklas, 2000; Nicklas <em>et al.</em>, 2001; Forer <em>et al.</em>, 2013; Sheykhani <em>et al.</em>, 2017). In general, a limitation of this approach is that cells have to be removed from the gonad and elaborate protocols have to be developed for each experiment using a specific organism to account for the physiological requirements for the culture media (<em>i.e.</em>, pH, osmolarity and temperature).<br/><br/>In the nematode <em>Caenorhabditis elegans</em>, microscopic investigation of spermatocyte meiosis was previously achieved by labelling of isolated and fixed gonads with antibodies (Howe <em>et al.</em>, 2001; Shakes <em>et al.</em>, 2009; Peters <em>et al.</em>, 2010; Schvarzstein <em>et al.</em>, 2013). This method, while powerful for analyzing the molecular composition of male meiotic spindles, cannot be used to study spindle dynamics. Alternatively, male gonads can be dissected, and the extracted germ cells can be filmed using Normarski optics. Applying this approach (Figures 1A-1D), it was possible to film cell divisions but the components of the meiotic spindle such as chromosomes and spindle poles were impossible to detect reliably without appropriate fluorescent labels (Shakes <em>et al.</em>, 2009; Gleason <em>et al.</em>, 2012; Vielle <em>et al.</em>, 2016; Hu <em>et al.</em>, 2019). Moreover, the imaging time was limited to only a few minutes as the cells regularly arrested before completion of the meiotic divisions. In our hands, about 90% of the spermatocytes arrested prior to the end of the meiotic divisions due to unknown reasons.<br/><br/>We developed a live-imaging approach that enabled us to study spindle dynamics in 3D in living immobilized males (Figures 1E-1G). With this newly developed approach it is now possible to achieve a parallel imaging of multiple spindles over about 90 min without any damaging effects on spindle dynamics (Fang-Yen <em>et al.</em>, 2009; Kim <em>et al.</em>, 2013). Another method applies patterned agarose pads created by using Vinyl Records as a mold (Rivera Gomez and Schvarzstein, 2018). However, this method relies on the use of anesthetics to completely immobilize the worm for imaging. Here we intended to avoid the use of anesthetics for animal immobilization, thus excluding any possible side effects of drugs on the dynamics of cell division. A technical problem of this approach that had to be solved was the bleaching of the fluorescent markers during the course of imaging, <em>i.e.</em>, a new sample had to be prepared about 1.5 h after the recording started. However, the high number of dividing cells within a single animal enables to collect sufficient data to determine the characteristics of dividing spindles in spermatocyte meiosis.<br/><br/>The challenge was to calculate the properties of dividing spindles over time. Due to the 3D live imaging spindles were not perfectly aligned in the imaging plane but rather randomly oriented in 3D with varying spindle axis angles in each frame. We solved this problem by computationally rotating each spindle in 3D to have a standard orientation for data analysis and then calculating kymographs for each fluorescent channel. By analyzing the kymographs, we could determine the dynamics of the chromosomes and the spindle poles over time with high accuracy. Further, this workflow can be used to image dividing mitotic germ cells in the distal gonad of males and hermaphrodites, which is also a cell type very difficult to image <em>in situ</em>. Our pipeline of imaging acquisition and data processing of cell division in <em>C. elegans</em> males can also be applied for imaging dividing embryos in hermaphrodites of this species and should also be applicable to other small model organisms with fluorescently labeled chromosomes and centrosomes.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201012/20201012185306_2128.jpg" width="300"/><br/><strong>Figure 1. Imaging of <em>C. elegans</em> male germ cells.</strong> A. DIC imaging of a primary spermatocyte dissected from a wild-type (Bristol N2) male. The cell is in prometaphase with chromosomes visible in the center of the cell. Black arrow heads indicating the approx. position of the centrosomes, white arrow indicates the chromosomes. B. Cell in metaphase I. C. Cell in anaphase I. D. Cell finishing first meiotic division. Scale bar for (A-D), 2 Âµm. E. Fluorescence image of an immobilized male (strain TMR17) after bleach correction. This image is a projection of 21 z-slices taken one minute after starting the data acquisition. Chromosomes labelled with histone H2B::mCherry are shown in magenta, spindle poles are labelled with Î³-tubulin::GFP and shown in cyan. The inset (red frame) shows a metaphase I spindle. F. Projection of 24 z-slices, 27 min after starting the image acquisition. In this field-of-view, different regions of a gonad are shown. These regions include meiotic prophase, various meiotic divisions of in meiosis I and II, and mature sperm. The inset (red frame) shows an anaphase I spindle. G. Projection of 21 z-slices, 61 min after starting the image acquisition. The inset (red frame) shows two spindles in metaphase II, Scale bar for (E-G), 5 Âµm.</p>
        </body>

<p class="pptt" id="biaoti38355">## Materials and Reagents</p><ol class="npwtwok"><li>Worm pick (flattened platinum wire mounted on a glass handle). Platinum wire with a diameter of 0.2 mm (Agar Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AGE404-2 </a> )</li><li>Parafilm</li><li>Falcon tubes (50 ml, Greiner Bio-One, Germany)</li><li>Microscope slides (76 x 26 mm x 1 mm, Engelbrecht, catalog number: <a class="blackNoUnder" target="_blank"> 11102 </a> )</li><li>Empty plate or small container</li><li>Filter paper Whatman Grade 589/2 (diameter: 90 mm, Merck, catalog number: <a class="blackNoUnder" target="_blank"> WHA10300109 </a> ). The filter paper was cut radially into 12 pieces</li><li>Polystyrene beads (Polybead<sup>Â®</sup> Microspheres 0.10 Î¼m; Polysciences, catalog number: <a class="blackNoUnder" target="_blank"> 00876-15 </a> )</li><li>High-precision cover slips (18 mm x 18 mm x 0.17 Âµm, Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> LH22.1 </a> )</li><li>Candle wax</li><li>Glass Pasteur pipettes</li><li><em>C. elegans</em> strain N2 (reference strain; Brenner, 1974)</li><li><em>C. elegans</em> strain TMR17 (genotype: unc-119(ed3) III; ddIs6[tbg-1::GFP + unc-119(+)]; ltIs37[pAA64; pie-1::mCherry::HIS-58 + unc-119(+)] IV; strain maintained in the MÃ¼ller-Reichert lab)</li><li><em>E. coli</em> strain OP50 (Brenner, 1974)</li><li>Agarose (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 3810.2 </a> )</li><li>Peptone (BD Biosciences, catalog number: <a class="blackNoUnder" target="_blank"> 4197928 </a> )</li><li>NaCl (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 9265.1 </a> )</li><li>Cholesterol (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 8866.1 </a> )</li><li>CaCl<sub>2</sub> (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> CN93.1 </a> )</li><li>MgCl<sub>2</sub> (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 2189.1 </a> )</li><li>NaOCl (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 9062.3 </a> )</li><li>KH<sub>2</sub>PO<sub>4</sub> (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> P749.2 </a> )</li><li>Na<sub>2</sub>HPO<sub>4</sub> (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> P030.1 </a> )</li><li>MgSO<sub>4</sub> (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> P027.1 </a> )</li><li>NGM-agar plates (see Recipes)</li><li>KPO<sub>4</sub> buffer (see Recipes)</li><li>Bleaching solution (see Recipes)</li><li>M9-buffer (see Recipes)</li></ol><p class="pptt" id="biaoti38356">### Equipment</p><ol class="npwtwok"><li>Fine tweezers</li><li>Incubator set to 20 Â°C (Thermo Fisher, model: <a class="blackNoUnder" target="_blank"> Heraeus BK 6160 </a> )</li><li>Incubator set to 30 Â°C to perform a heat-shock to animals (IKA, model: <a class="blackNoUnder" target="_blank"> KS 4000 i control </a> )</li><li>Stereo microscope with transmitted light (Motic, model: <a class="blackNoUnder" target="_blank"> SMZ-168 BL </a> )</li><li>Microwave</li><li>Spoon/spatula</li><li>Spinning disc confocal microscope (Olympus, model: <a class="blackNoUnder" target="_blank"> IX83 </a> )</li><li>EMCCD camera (Andor, model: <a class="blackNoUnder" target="_blank"> iXon Ultra 897 </a> )</li><li>Spinning disk unit (Yokogawa, model: <a class="blackNoUnder" target="_blank"> CSU-X1 </a> )</li><li>60x water immersion objective (Olympus, model: <a class="blackNoUnder" target="_blank"> UPlanSApo 60x W </a> , N.A. 1.2)<br/></li></ol><p class="pptt" id="biaoti38357">### Software</p><ol class="npwtwok"><li>Fiji software package (<a href="http://www.fiji.sc/" target="_blank">www.fiji.sc</a>, Schindelin <em>et al.</em>, 2012)</li><li>Arivis Vision4D (arivis AG, Munich)</li><li>Microsoft Excel</li><li>Python (<a href="https://www.python.org/" target="_blank">https://www.python.org/</a>)<br/></li></ol><p class="pptt" id="biaoti38358">### Procedure</p><ol class="npwone"><li>Culture of <em>C. elegans</em> males<ol class="npwtwo"><li>Obtain male worms by applying the heat-shock method (Sulston and Hodgkin, 1988).</li><li>Place 10 young L4 hermaphrodites on a fresh NGM-agar plate containing <em>E. coli</em> OP50.</li><li>Seal the plate with Parafilm.</li><li>Expose the plate to 30 Â°C for 6 h.</li><li>Remove the parafilm and place the plate in an incubator at 20 Â°C for 2-3 days.</li><li>Check then regularly for the presence of males on the plate. There should be 5-30 male worms on the plate.</li><li>Transfer as many males as possible to a fresh NGM-agar plate and place 5 L4 hermaphrodites on this plate. Aim to have for 4-5 times more male individuals than hermaphrodites per plate. The progeny of the hermaphrodites should produce ~50% males.</li><li>Maintain a culture of <em>C. elegans</em> males by transferring ~30 males and 5 L4 hermaphrodites to a fresh plate every three days.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Synchronization of male worms<ol class="npwtwo"><li>Pipette 50 Âµl of bleaching solution on a fresh NGM-agar plate containing <em>E. coli</em> OP50.</li><li>Add 10-20 adult hermaphrodites from a male culture plate into the bleaching solution.</li><li>Incubate the plate for three days at 20 Â°C.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparation of agarose pads<ol class="npwtwo"><li>Prepare about 30 clean glass slides and place them next to each other on the bench (Figure 2A).</li><li>Prepare 10% (w/v) agarose powder in M9 buffer in a 50 ml Falcon tube (<em>e.g.</em>, 0.5 g in 5 ml).</li><li>Melt the agarose in a microwave with short heating intervals. Open the lid shortly in between the intervals.<br/><em>Caution: Melted agarose is hot; protective gloves should be worn.</em> </li><li>Use a small spoon or spatula to place a drop of the melted 10% agarose on a glass slide (Figure 2B).</li><li>Immediately add a second glass slide on top and gently apply pressure for 5 s (Figure 2C).</li><li>Repeat this procedure until all glass slides have agarose in between.</li><li>Remove the top slide.</li><li>Use a round object (<em>e.g.</em>, cap of a permanent marker) to cut out agarose pads of a defined diameter of 10 mm. For this, select areas that do not contain any or only a few trapped air bubbles. Such bubbles might impair worm immobilization or the image quality later. Multiple agarose pads can be cut out of a single agar preparation (Figure 2D).</li><li>Grab each agarose pad with fine tweezers and put it to a small container (<em>e.g.</em>, empty plate) filled with M9 buffer (Figure 2E). <br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201012/20201012194053_9947.jpg" width="300"/><br/><strong>Figure 2. Immobilization of worms for live-imaging using agarose pads and Polystyrene beads.</strong> A. Arrangement of glass slides on the laboratory bench. This way, agarose pads can be produced quickly before the agarose starts to solidify. B. Placing of melted agarose on one glass slide using a small spoon. C. Placing of a second glass slide on top of the agarose droplet and simultaneously application of pressure until the agarose is solid (~5 s). D. Excision of an agarose pad (diameter of about 10 mm) using a cap of a permanent marker. E. Transfer of the agarose pad with a fine tweezer to M9 buffer until further use. F. Placing of the agarose pad on a glass slide to start a live-cell experiment and removal of excess of liquid from the side with a piece of filter paper. G. Application of a Polystyrene-bead solution (bead diameter 0.1 Âµm) to the top of the agarose pad. H. After addition of worms, a cover slip was placed on top of the agarose pad. Afterwards, a few microliters of M9 buffer were added to avoid dehydration of the animals. I. Sealing of the cover slip with candle wax and imaging of the slide using fluorescence microscopy.<br/><br/></li><li>Store agarose pads for several days in M9 buffer at 4 Â°C until further use. However, best results for worm immobilization are achieved with freshly prepared agarose pads.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Live-imaging of meiotic spindles in male worms using fluorescent markers<ol class="npwtwo"><li>Select an agarose pad, place it with tweezers in the middle of a new glass slide. Be careful not to damage the surface of the agarose pad as this will reduce the immobilization effect on individual worms.</li><li>Aspirate excess of liquid from the side of the agarose pad with a piece of filter paper (Figure 2F). Be careful not to touch the surface of the agarose pad as also cellulose fibers on the surface of the pad will reduce the immobilization of worms.</li><li>Add 1 Âµl of a polystyrene bead solution on the agarose pad (Figure 2G).</li><li>Place 5-10 three-day old male worms within the bead solution.</li><li>Gently add a cover slip (Figure 2H). It is important to lower the covers slip gently as a dropping of the glass might damage the worms or activate spermatids within the males. A sliding of the mounted cover glass should also be avoided as this might impair the worms as well. If worms are still moving after the assembly procedure, discard the sample and start again with point no. 1.</li><li>Add a few microliters of M9 buffer from the side under the cover slip. Be careful not to add too much as excess liquid will make the cover slip float and decrease the amount of immobilization.</li><li>Seal the edges of the cover slip with melted candle wax (Figure 2I).</li><li>Check again under a dissection microscope if the worms are immobile. If worms are perfectly immobilized at this point, they will not become mobile again, thus long-term imaging will be possible by using the settings described below.</li><li>Place the prepared sample on a fluorescence microscope.</li><li>Select a fully immobilized worm for imaging. Start image acquisition. Samples can be imaged for about 90 min without any detectable effects on cell division. Typically, z-stacks are recorded on a confocal spinning disk microscope in two fluorescence channels with an image stack every 30 s containing about 60 planes (0.33 Âµm z-spacing). For the analysis of spindle dynamics, use a worm strain expressing Î³-tubulin::GFP and histone H2B::mCherry in the germ line. With a 60x water immersion objective (N.A.: 1.2), we generated images with a voxel size of 0.222 x 0.222 x 0.33 Âµm. As the specifications among microscopes and lasers can vary, it is not possible here to give absolute values for the laser power to be used. It is advisable to assess the degree of photobleaching after one hour by comparing the signal intensities in the recorded images at the beginning and at the end of data acquisition. If the signal is bleached very fast or cells stop dividing, reduce the laser power or the exposure time per frame. Then check, if the signal/noise ratio in a new recording allows to detect specific signals. This process needs to be repeated iteratively until the optimal imaging settings for the specific microscope have been determined.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Image pre-processing<ol class="npwtwo"><li>Import the Z-stacks to ImageJ/Fiji. A user manual for ImageJ/Fiji can be found <a href="https://imagej.nih.gov/ij/docs/guide/user-guide.pdf" target="_blank">here</a>.</li><li>Convert them to a hyperstack.</li><li>Correct for bleaching using the method âExponential fitâ.</li><li>Save images.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Image processing<ol class="npwtwo"><li>Import images into arivis Vision4D (Figure 3A).</li><li>Crop out and export individual meiotic cells from the meiotic region of the male gonad (Figure 3B).</li><li>Make a note in which frame anaphase starts.</li><li>Use a 3D median filter with a kernel size of 1 pixel (Figure 3C) to reduce the noise.</li><li>Segment individual centrosomes in each meiotic cell with the tool âBlob finderâ (Figure 3D). The settings have to be chosen in accordance to the used imaging setup. As the image quality varied among the live recordings the segmentation settings had to be adjusted within a narrow range. Typically, we used the following settings for the âBlob Finderâ tool: diameter: 1 Âµm, threshold: 10-20, split sensitivity: 85-95.</li><li>Track the segments over time using the tool âTrack segmentsâ. Typically, we used the following settings: type: âBrownian motionâ, max. distance: 2-4 Âµm.</li><li>Edit and combine tracks in order to have one track connecting the individual centrosomes over time using the tool âTrack editorâ.</li><li>Export a Microsoft Excel file containing the x, y and z coordinates of the geometric center of mass of each of the segmented centrosomes.</li><li>Create a semicolon-delimited CSV-file for the first centrosome track, containing in each line the x, y, and z coordinate of the center of mass each time point.</li><li>Create analogously a second CSV-file for the second centrosome track.</li><li>Use a custom-made Python script to reorient (Watt, 1999) and resample the image data in order to create a kymograph along the spindle axis with a defined radius for each fluorescence channel (Figure 3F, <a href="https://cdn.elifesciences.org/articles/50988/elife-50988-code1-v4.py" target="_blank">https://cdn.elifesciences.org/articles/50988/elife-50988-code1-v4.py</a>).<ol class="npwthree"><li>Give the path to both CSV-files containing the coordinates of the center of the centrosomes (PATH).</li><li>Indicate which fluorescence channel should be processed (CHANNEL).</li><li>Indicate the frame of the data set that should be used as a start (START_FRAME; typically, this was ten time points before the frame of anaphase onset).</li><li>Indicate the pixel size in the resampled output data set (LINE_PIXEL_SIZE; typically 0.1 Âµm was used).</li><li>Indicate the radius around the spindle axis that is used to calculate the fluorescence pixel values within the kymograph (LINE_RADIUS; typically a â9â was used, which means a radius of 0.9 Âµm around the spindle axis, given that the pixel size was set to 0.1 Âµm).</li><li>Indicate the length of extrapolation along the spindle axis after the start and end coordinate (LINE_EXTRAPOLATE, typically 1.0 Âµm before and after the centrosome centers was added to each kymograph).</li><li>Define an output filename for the kymograph CSV-file (OUTPUT_FILENAME).</li></ol></li><li>Calculate the Gaussian weighted sum of fluorescence (Burger and Burge, 2013) in each plane in 0.1 Âµm steps along the spindle axis with a radius of 0.9 Âµm (Figure 3G). This was repeated for all time points (Figure 3H) and the kymograph was saved as an image (Figure 4A) and a csv-file.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201012/20201012194308_4210.jpg" width="300"/><br/><strong>Figure 3. Workflow for image processing using arivis Vision4D.</strong> A. Data after pre-processing showing a meiosis I spindle in dual color fluorescence within one image plane. A selected area (indicated by a yellow box) is exported to a new file. Scale bar, 5 Âµm. B. Exported spindle as indicated in (A). C. Exported data after application of a 3D median filter (kernel size: 1 pixel). This algorithm removes the noise in the image while preserving the edges of structures. D. Segmentation of spindle poles (GFP-channel) using the âBlob finderâ tool. The segmented volume is shown in yellow. Scale bar for B-D, 2 Âµm. E. 3D visualization of the exported image volume. The individual segmented spindle poles were traced over time as represented by lines on each pole. For 16 time points of the recording, white lines show the 3D position of the poles in the âpastâ and gray lines the âfutureâ positions. F. Image showing a rotated and resampled spindle. Yellow dots indicate the center of mass of the segmented spindle poles. G. The volume that is used to assemble a kymograph is indicated by a box (green). Numbers in the coordinate system in (E-G) show distances in Âµm. H. Projections of the exported volumes at different time points that were used for the calculation of kymographs. Scale bar, 1 Âµm.<br/><br/></li><li>Analyze the kymograph for each fluorescence channel to determine the position of the peak intensity from both sides of the spindle over time (Figures 4B-4C).</li><li>Calculate the distance of the peak intensities for each channel over time. This way, the autosome-to-autosome distance (Figure 4D) and the pole-to-pole distance (Figure 4E) can be calculated. In addition, the pole-to-autosome distance (Figure 4F) can also be determined by calculating the distance of the peak intensities from both fluorescence channels on each side of the spindle.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201012/20201012194326_6519.jpg" width="300"/><br/><strong>Figure 4. Kymographic analysis of spindle dynamics.</strong> A. Dual-color kymograph for the spindle as presented and processed in Figure 3. Autosomes are shown in magenta and spindle poles in cyan. Scale bar (white), 1 Âµm, time bar (orange), 2 min. B. Analysis of autosome dynamics. The kymograph of the histone::mCherry channel with the individual time points indicated by blue dashed lines is shown on the left, the fluorescence intensity profiles with highlighted the two peak intensities (red) arrows on the right. C. Analysis of spindle pole movements. The kymograph of the Î³-tubulin::GFP channel (left) and the fluorescence intensity profiles are shown (right). D. Analysis of the autosome-to-autosome distance over time for 31 data sets aligned at anaphase onset (t<sub>0</sub>). Circles represent the mean, shaded area the standard deviation. All mean values are connected by a solid line. E. Analysis of the pole-to-pole distance over time for 31 data sets. F. Analysis of the pole-to-autosome distance over time for 31 data sets.</li></ol></li></ol><p class="pptt" id="biaoti38359">### Data analysis</p><p class="ppzz">For each spindle, the pole-to-pole, the autosome-to-autosome and the pole-to-autosome distance was calculated for each time point from the kymograph data. In addition, the time of anaphase onset was determined as time zero (t<sub>0</sub>) by the first frame showing a clear separation of the autosomes. Accordingly, spindle dynamics in all data sets were then aligned to t<sub>0</sub>. By calculating the average distance (Â± standard deviation) for each time point the dynamic properties of both meiotic spindles were calculated, <em>i.e.</em>, the initial and final spindle length, the initial rate of elongation and the duration of spindle elongation or segregation.</p><p class="pptt" id="biaoti38360">### Recipes</p><ol class="npwtwok"><li>NGM-agar plates (Stiernagle, 2006)<br/>0.25% (w/v) peptone<br/>51 mM NaCl<br/>25 mM KPO<sub>4</sub> buffer<br/>0.005 g/L cholesterol<br/>1 mM CaCl<sub>2</sub><br/>1 mM MgCl<sub>2</sub><br/>2% (w/v) agar<br/>The solution was autoclaved for 15 min at 120 Â°C, afterwards plates were poured</li><li>KPO<sub>4</sub> buffer (1 M)<br/>0.8 M KH<sub>2</sub>PO<sub>4</sub> and 0.8 M K<sub>2</sub>HPO<sub>4</sub> in double-distilled water<br/>Adjust the pH to 6.0<br/>Filter sterilize the solution by using a vacuum filtration system (0.22 Âµm filter mesh size)</li><li>Bleaching solution (Stiernagle, 2006)<br/>500 mM NaOH<br/>2.6% (v/v) NaOCl in double-distilled water<br/>Protect solution from light and renew solution every 3-6 month as NaOCl gets less reactive over time</li><li>M9-buffer (Brenner, 1974)<br/>22 mM KH<sub>2</sub>PO<sub>4</sub><br/>42.3 mM Na<sub>2</sub>HPO<sub>4</sub><br/>85.6 mM NaCl<br/>1 mM MgSO<sub>4</sub><br/>The solution was filtered afterwards (filter pore size 0.2 Î¼m)<br/></li></ol><p class="pptt" id="biaoti38361">### Acknowledgments</p><p class="ppzz">The authors would like to thank the members of the Core Facility Cellular Imaging of the Faculty of Medicine âCarl Gustav Carusâ (TU Dresden, Germany), especially Silke Tulok and Dr. Anja Nobst, and the light microscopy facility at the MPI-CBG (Dresden, Germany) for technical assistance. We also would like to thank Drs. Leocadia Paliulis and Robert Haase for the critical reading of this manuscript. Some strains were provided by the CGC, which is funded by NIH Office of Research Infrastructure Programs (P40 OD010440). Research in the MÃ¼ller-Reichert laboratory is supported by the Deutsche Forschungsgemeinschaft (MU 1423/10â1 to TMR).<br/>  This protocol was part of and derived from Fabig <em>et al.</em> (2020).</p><p class="pptt" id="biaoti38362">### Competing interests</p><p class="ppzz">FL and CG are employed by arivis AG. This company develops commercial image processing software (Vision4D). This software was utilized to process fluorescence microscopy data used in this protocol.</p><p class="pptt" id="biaoti38363">### References</p><ol class="npwtwok"><li>Ault, J. G. (1986). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3698743" target="_blank">Stable versus unstable orientations of sex chromosomes in two grasshopper species.</a> <em>Chromosoma</em> 93(4): 298-304. </li><li>Bonaccorsi, S. and Gatti, M. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28349403" target="_blank"><em>Drosophila</em> Male Meiosis.</a> <em>Methods Mol Biol</em> 1471: 277-288.</li><li>Boring, A. (1909). A small chromosome in <em>Ascaris megalocephala</em>. <em>Arch f Zellf</em> 4: 120-131.</li><li>Boveri, T. (1909). <a href="https://link.springer.com/article/10.1007/BF02161755" target="_blank">Ãber Geschlechtschromosomen bei Nematoden.</a> <em>Arch f Zellf</em> 4: 132-141.</li><li>Brenner, S. (1974). <a href="http://www.ncbi.nlm.nih.gov/pubmed/4366476" target="_blank">The genetics of <em>Caenorhabditis elegans</em>.</a> <em>Genetics</em> 77(1): 71-94. </li><li>Burger, W., and Burge, M. J. (2013). <a href="https://link.springer.com/book/10.1007/978-1-84882-919-0" target="_blank">Principles of Digital Image Processing-Advanced Methods.</a> In: Burger, W., and Burge, M. J. (Eds). <em>Undergraduate Topics in Computer Science</em> Springer-Verlag.</li><li>Fabig, G., Kiewisz, R., Lindow, N., Powers, J. A., Cota, V., Quintanilla, L. J., BruguÃ©s, J., Prohaska, S., Chu, D. S. and MÃ¼ller-Reichert, T. (2020). <a href="https://elifesciences.org/articles/50988" target="_blank">Male meiotic spindle features that efficiently segregate paired and lagging chromosomes.</a> <em>ELife</em> 9: e50988.<br/></li><li>Fang-Yen, C., Wasserman, S., Sengupte, P., and Samuel, A. D. T. (2009). <a href="http://wbg.wormbook.org/2009/12/01/agarose-immobilization-of-c-elegans/" target="_blank">Agarose immobilization of <em>C. elegans</em>.</a> <em>Worm Breederâs Gazette</em>.</li><li>Fegaras, E. and Forer, A. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29468300" target="_blank">Chromosomes selectively detach at one pole and quickly move towards the opposite pole when kinetochore microtubules are depolymerized in <em>Mesostoma ehrenbergii</em> spermatocytes.</a> <em>Protoplasma</em> 255(4): 1205-1224.</li><li>Felt, K. D., Lagerman, M. B., Ravida, N. A., Qian, L., Powers, S. R. and Paliulis, L. V. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28478487" target="_blank">Segregation of the amphitelically attached univalent X chromosome in the spittlebug <em>Philaenus spumarius</em>.</a> <em>Protoplasma</em> 254(6): 2263-2271. </li><li>Forer, A., Ferraro-Gideon, J. and Berns, M. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23315093" target="_blank">Distance segregation of sex chromosomes in crane-fly spermatocytes studied using laser microbeam irradiations.</a> <em>Protoplasma</em> 250(5): 1045-1055.</li><li>Forer, A., Sheykhani, R. and Berns, M. W. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30087895" target="_blank">Anaphase Chromosomes in Crane-Fly Spermatocytes Treated With Taxol (Paclitaxel) Accelerate When Their Kinetochore Microtubules Are Cut: Evidence for Spindle Matrix Involvement With Spindle Forces.</a> <em>Front Cell Dev Biol</em> 6: 77. </li><li>Gleason, E. J., Hartley, P. D., Henderson, M., Hill-Harfe, K. L., Price, P. W., Weimer, R. M., Kroft, T. L., Zhu, G. D., Cordovado, S. and L'Hernault, S. W. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22446317" target="_blank">Developmental genetics of secretory vesicle acidification during <em>Caenorhabditis elegans</em> spermatogenesis.</a> <em>Genetics</em> 191(2): 477-491. </li><li>Golding, A. E. and Paliulis, L. V. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21931842" target="_blank">Karyotype, sex determination, and meiotic chromosome behavior in two pholcid (Araneomorphae, Pholcidae) spiders: implications for karyotype evolution.</a> <em>PLoS One</em> 6(9): e24748. </li><li>Goldstein, P. (1977). <a href="http://www.ncbi.nlm.nih.gov/pubmed/563473" target="_blank">Spermatogenesis and spermiogenesis in <em>Ascaris lumbricoides Var. suum</em>.</a><em> J Morphol </em>154(3): 317-337.</li><li>Howe, M., McDonald, K. L., Albertson, D. G. and Meyer, B. J. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11402066" target="_blank">HIM-10 is required for kinetochore structure and function on <em>Caenorhabditis elegans</em> holocentric chromosomes.</a> <em>J Cell Biol </em>153(6): 1227-1238.</li><li>Hu, J., Cheng, S., Wang, H., Li, X., Liu, S., Wu, M., Liu, Y. and Wang, X. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30990821" target="_blank">Distinct roles of two myosins in <em>C. elegans</em> spermatid differentiation.</a> <em>PLoS Biol</em> 17(4): e3000211.</li><li>Janicke, M. A. and LaFountain, J. R., Jr. (1982). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7128280" target="_blank">Chromosome segregation in crane-fly spermatocytes: cold treatment and cold recovery induce anaphase lag.</a> <em>Chromosoma</em> 85(5): 619-631.</li><li>Kim, E., Sun, L., Gabel, C. V. and Fang-Yen, C. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23301069" target="_blank">Long-term imaging of <em>Caenorhabditis elegans</em> using nanoparticle-mediated immobilization.</a> <em>PLoS One</em> 8(1): e53419.</li><li>King, J. M. and Nicklas, R. B. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11034909" target="_blank">Tension on chromosomes increases the number of kinetochore microtubules but only within limits.</a> <em>J Cell Sci</em> 113 Pt 21: 3815-3823. </li><li>LaFountain, J. R., Jr., Cohan, C. S. and Oldenbourg, R. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22740625" target="_blank">Pac-man motility of kinetochores unleashed by laser microsurgery.</a> <em>Mol Biol Cell</em> 23(16): 3133-3142. </li><li>LaFountain, J. R., Jr., Cole, R. W. and Rieder, C. L. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11896200" target="_blank">Partner telomeres during anaphase in crane-fly spermatocytes are connected by an elastic tether that exerts a backward force and resists poleward motion.</a> <em>J Cell Sci</em> 115(Pt 7): 1541-1549. </li><li>Nicklas, R. B. and Koch, C. A. (1969). <a href="http://www.ncbi.nlm.nih.gov/pubmed/5824068" target="_blank">Chromosome micromanipulation. 3. Spindle fiber tension and the reorientation of mal-oriented chromosomes.</a> <em>J Cell Biol</em> 43(1): 40-50.</li><li>Nicklas, R. B. and Staehly, C. A. (1967). <a href="http://www.ncbi.nlm.nih.gov/pubmed/5339860" target="_blank">Chromosome micromanipulation. I. The mechanics of chromosome attachment to the spindle.</a> <em>Chromosoma</em> 21(1): 1-16.</li><li>Peters, N., Perez, D. E., Song, M. H., Liu, Y., Muller-Reichert, T., Caron, C., Kemphues, K. J. and O'Connell, K. F. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20144993" target="_blank">Control of mitotic and meiotic centriole duplication by the Plk4-related kinase ZYG-1.</a> <em>J Cell Sci</em> 123(Pt 5): 795-805.</li><li>Nicklas, R. B., Waters, J. C., Salmon, E. D. and Ward, S. C. (2001). <a href="https://jcs.biologists.org/content/114/23/4173" target="_blank">Checkpoint signals in grasshopper meiosis are sensitive to microtubule attachment, but tension is still essential.</a> <em>J Cell Sci</em> 114(23): 4173-4183.</li><li>Rivera Gomez, K. and Schvarzstein, M. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/32550397" target="_blank">Immobilization of nematodes for live imaging using an agarose pad produced with a Vinyl Record.</a> <em>MicroPubl Biol</em> 2018.</li><li>Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., Tinevez, J. Y., White, D. J., Hartenstein, V., Eliceiri, K., Tomancak, P. and Cardona, A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22743772" target="_blank">Fiji: an open-source platform for biological-image analysis.</a> <em>Nat Methods</em> 9(7): 676-682. </li><li>Schvarzstein, M., Pattabiraman, D., Bembenek, J. N. and Villeneuve, A. M. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23401519" target="_blank">Meiotic HORMA domain proteins prevent untimely centriole disengagement during <em>Caenorhabditis elegans</em> spermatocyte meiosis.</a> <em>Proc Natl Acad Sci U S A</em> 110(10): E898-907.</li><li>Shakes, D. C., Wu, J. C., Sadler, P. L., Laprade, K., Moore, L. L., Noritake, A. and Chu, D. S. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19696886" target="_blank">Spermatogenesis-specific features of the meiotic program in <em>Caenorhabditis elegans</em>.</a> <em>PLoS Genet</em> 5(8): e1000611.</li><li>Sheykhani, R., Berns, M. and Forer, A. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27935262" target="_blank">Elastic tethers between separating anaphase chromosomes in crane-fly spermatocytes coordinate chromosome movements to the two poles.</a> <em>Cytoskeleton (Hoboken)</em> 74(2): 91-103.</li><li>Stiernagle, T. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18050451" target="_blank">Maintenance of <em>C. elegans.</em></a> <em>WormBook</em>: 1-11.</li><li>Sulston, J. and Hodgkin, J. (1988). <a href="https://www.cshlpress.com/default.tpl?cart=1590484100119941978&amp;fromlink=T&amp;linkaction=full&amp;linksortby=oop_title&amp;--eqSKUdatarq=39" target="_blank">Methods.</a> In: Wood W. B. (Ed.). <em>The Nematode Caenorhabditis elegans.</em> (pp. 587-606). Cold Spring Harbor Laboratory Press.</li><li>Vielle, A., Callemeyn-Torre, N., Gimond, C., Poullet, N., Gray, J. C., Cutter, A. D. and Braendle, C. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27565121" target="_blank">Convergent evolution of sperm gigantism and the developmental origins of sperm size variability in <em>Caenorhabditis</em> nematodes.</a> <em>Evolution</em> 70(11): 2485-2503. </li><li>Watt, H. A. (1999). 3D Computer Graphics, Addison Wesley.<br/></li></ol>