<body>
            <h1>Attachment of a <sup>32</sup>P-phosphate to the 3â² Terminus of a DNA Oligonucleotide</h1>
            <p><strong>Authors:</strong> Joshua C. Cofsky, Jennifer A. Doudna</p>
            <h2>Abstract</h2>
            <p>Biochemical investigations into DNA-binding and DNA-cutting proteins often benefit from the specific attachment of a radioactive label to one of the two DNA termini. In many cases, it is essential to perform two versions of the same experiment: one with the 5â² DNA end labeled and one with the 3â² DNA end labeled. While homogeneous 5â²-radiolabeling can be accomplished using a single kinase-catalyzed phosphorylation step, existing procedures for 3â²-radiolabeling often result in probe heterogeneity, prohibiting precise DNA fragment identification in downstream experiments. We present here a new protocol to efficiently attach a <sup>32</sup>P-phosphate to the 3â² end of a DNA oligonucleotide of arbitrary sequence, relying on inexpensive DNA oligonucleotide modifications (2â²-O-methylribonucleotide and ribonucleotide sugar substitutions), two enzymes (T4 polynucleotide kinase and T4 RNA ligase 2), and the differential susceptibility of DNA and RNA to hydroxide treatment. Radioactive probe molecules produced by this protocol are homogeneous and oxidant-compatible, and they can be used for precise cleavage-site mapping in the context of both DNase enzyme characterization and DNA footprinting assays.<br/><br/><strong>Graphic abstract</strong><br/><br/> <img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201012/20201012235726_7327.jpg" width="300"/><br/></p>
            <p><strong>Keywords:</strong> Radiolabel, <sup>32</sup>P-phosphate, 3â² DNA end, T4 polynucleotide kinase, T4 RNA ligase 2, DNA oligonucleotide, 2â²-O-methylribonucleotide, 2â²,3â²-cyclic phosphate</p>
            <h2>Background</h2><p>A common goal in the study of DNA and DNA-binding proteins is to identify the position and frequency of DNA backbone cleavage events, an essential set of experiments if the investigator aims to determine the mechanism of a novel DNase enzyme or discern protection parameters from DNA footprinting assays. In the classic version of this experiment, the investigator first uses T4 polynucleotide kinase (PNK) to transfer a <sup>32</sup>P-phosphate from [Î³-<sup>32</sup>P]-adenosine triphosphate (ATP) to the 5â²-OH of a DNA oligonucleotide of interest, yielding a 5â²-radiolabeled DNA âprobeâ. After subjecting the probe to a cleavage process, the investigator then resolves the cleavage fragments by polyacrylamide gel electrophoresis (PAGE), visualizes them by autoradiography, and determines cleavage position and frequency from band location and intensity, respectively. However, if a DNA probe molecule is cleaved more than once, the described experiment only allows detection of the cleavage event closest to the 5â² end. To detect additional cleavage events with equivalent positional resolution, an analogous experiment must be performed with the 3â² DNA end radiolabeled.<br/><br/>Because there is currently no known kinase activity that can specifically transfer a <sup>32</sup>P-phosphate to the 3â²-OH of a DNA oligonucleotide, standard 3â²-radiolabeling procedures instead employ either a polymerase or a terminal transferase to catalyze primer extension at the 3â² DNA end, using [Î±-<sup>32</sup>P]-nucleotide triphosphates (NTPs) as a reagent (in different strategies, the sugar moiety of the NTPs could be 2â²-deoxyribose, ribose, or 3â²-deoxyribose) (Wu <em>et al.</em>, 1976). While the resulting probes are suitable for certain experiments, they cannot always be used for unambiguous DNA cleavage-site mapping (<em>i.e.</em>, determining at exactly which phosphodiester a DNA molecule was cut, as measured from the 3â² DNA end). For example, the polymerase- or transferase-based methods suffer from heterogeneity in probe length and/or incorporation of radionuclides at internal backbone positions, both of which can destroy the unique band-to-fragment correspondence that is required for unambiguous cleavage-site mapping. When using probes generated by one of these methods, a single band on a PAGE autoradiograph could be attributed to any of several distinct radioactive cleavage fragments that migrate with indistinguishable electrophoretic mobility. Note that these problems arise even when the <sup>32</sup>P-phosphate lies between the final two nucleotides, as certain nuclease enzymes and chemical processes (such as those associated with permanganate footprinting) can remove the final nucleoside without its phosphate. In one protocol that cleverly addresses this problem, the authors expose an internal radionuclide by sequentially treating a ribonucleotide-tailed radioprobe with alkali, phosphatase, and periodate. However, while this procedure does yield the desired outcomeâa DNA probe homogeneously labeled with a terminal 3â²-<sup>32</sup>P-phosphateâthe initial tailing step requires high concentrations of expensive radioactive ribonucleotides to promote efficient labeling (Jay <em>et al.</em>, 1974).<br/><br/>More recently, many synthesis companies have begun to offer custom DNA oligonucleotides with a fluorophore chemically conjugated to the 3â² end. If a laboratory has the optical instrumentation to directly image fluorophore-tagged DNA in a polyacrylamide gel, these reagents offer a viable alternative to 3â²-radiolabeled DNA probes. Still, this detection technique is far less sensitive than autoradiography and will miss minor products in complex cleavage patterns. Additionally, common fluorophores may be degraded under the harsh chemical treatments associated with certain experiments, such as permanganate footprinting.<br/><br/>Considering the previously mentioned shortcomings, the ideal 3â²-labeling strategy would: (1) take advantage of the high sensitivity and chemical compatibility of <sup>32</sup>P-phosphate-based detection; (2) produce a homogeneous population of DNA probe with a single 3â²-terminal radionuclide; and (3) employ catalytically efficient combinations of common enzymes and substrates to make the procedure inexpensive and fast. We developed a protocol to meet these needs in the context of our studies of DNA binding and cleavage by the RNA-guided DNase CRISPR-Cas12a (Cofsky <em>et al.</em>, 2020). In our protocol, we begin with a DNA oligonucleotide in which the penultimate and final deoxyribonucleotides have been changed to a 2â²-O-methylribonucleotide and a ribonucleotide, respectively, which are standard and cheap modifications at oligonucleotide synthesis companies (Figure 1A). To introduce the <sup>32</sup>P-phosphate to the system, we first perform a standard 5â²-radiolabeling procedure on a âphosphate shuttleâ RNA oligonucleotide (Figure 1B). We then form a double-stranded junction by annealing the phosphate shuttle RNA oligonucleotide, the DNA oligonucleotide, and a bridging splint RNA oligonucleotide (Figure 1C). With the sugar modifications at the DNA oligonucleotideâs 3â² end, this bridge structure can be efficiently ligated by T4 RNA ligase 2, attaching the <sup>32</sup>P-phosphate to the 3â²-OH of the DNA oligonucleotide (Nandakumar and Shuman, 2004; Figure 1D). After ligation, we treat the probe precursor with hot hydroxide, leaving the DNA intact and dissociating the RNA into mononucleotides that can be removed with a spin column (Figures 1E, 1F). The final result is a DNA oligonucleotide with a penultimate 2â²-O-methylribonucleotide and a 3â²-terminal ribonucleotide bearing a 2â²,3â²-cyclic-<sup>32</sup>P-phosphate (Figure 2).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201012/20201012234112_9101.jpg" width="300"/><br/><strong>Figure 1. Protocol for 3â²-radiolabeling, in brief</strong><br/> <br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201012/20201012235354_7172.jpg" width="300"/><br/><strong>Figure 2. Chemical structure of final 3â²-radiolabeled product</strong><br/><br/>This method produces homogeneous 3â²-terminally radiolabeled DNA probe with minimal reagent consumption and sample manipulation. Additionally, if both 5â²-and 3â²-radiolabeling experiments are being performed, the investigator only needs to purchase a single radionucleotide ([Î³-<sup>32</sup>P]-ATP) to produce the required probes, as compared to polymerase-/transferase-based strategies that would also require expensive [Î±-<sup>32</sup>P]-NTPs. In our experiments, we used these probes to detect Cas12a-generated DNA cleavage products (including minor and transient products), leading to the unforeseen finding that Cas12a cleaves its DNA substrate multiple times to form a five-nucleotide gap. We also used these probes in permanganate footprinting experiments that yielded highly position-specific information about the conformation of Cas12a-bound DNA (Cofsky <em>et al.</em>, 2020). Additionally, by using a <sup>32</sup>P-phosphate label in these experiments, we avoided the problems associated with the use of fluorophores in the presence of strong oxidants. Finally, when using this protocol to generate many variant DNA probes, reagent costs were ~90% the cost of the cheapest commercially synthesized fluorophore-tagged DNA oligonucleotides, according to list prices at the time of writing. Therefore, this method may be used to prepare inexpensive DNA probes for any cleavage-site mapping experiment that requires high precision, sensitivity, and chemical compatibility.</p>
        </body>

<p class="pptt" id="biaoti38377">## Materials and Reagents</p><ol class="npwtwok"><li>Illustra MicroSpin G-25 columns (Cytiva, catalog number: <a class="blackNoUnder" target="_blank"> 27532501 </a> ), stored at room temperature</li><li>3â²-modified DNA oligonucleotide (Integrated DNA Technologies) (see Procedure for details), stored at -20 Â°C</li><li>Phosphate shuttle RNA oligonucleotide (<em>in vitro</em> transcribed or commercially synthesized, Integrated DNA Technologies), stored at -80 Â°C</li><li>Splint RNA oligonucleotide (<em>in vitro</em> transcribed or commercially synthesized, Integrated DNA Technologies), stored at -80 Â°C</li><li>T4 polynucleotide kinase, 10,000 U/ml (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M0201S </a> ), stored at -20 Â°C</li><li>T4 polynucleotide kinase reaction buffer, 10x (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> B0201S </a> ; included with product M0201), stored at -20 Â°C</li><li>[Î³-<sup>32</sup>P]-ATP EasyTide, 3000 Ci/mmol, 10 mCi/ml, 1 mCi (PerkinElmer, catalog number: <a class="blackNoUnder" target="_blank"> BLU502A001MC </a> ), stored at -20 Â°C</li><li>T4 RNA ligase 2, 10,000 U/ml (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M0239S </a> ), stored at -20 Â°C</li><li>T4 RNA ligase 2 reaction buffer, 10x (New England Biolabs; included with product <a class="blackNoUnder" target="_blank"> M0239S </a> ), stored at -20 Â°C</li><li>Common reagents:<br/>NaOH (1.35 M stock)<br/>HCl (1.35 M stock)<br/>KCl<br/>Tris-Cl<br/>EDTA<br/>Sodium citrate<br/>MilliQ water (all instances of the word âwaterâ in this protocol refer to MilliQ water)</li><li>5x annealing buffer (see Recipes), stored at 4 Â°C</li><li>1x RNA storage buffer (see Recipes), stored at 4 Â°C<br/></li></ol><p class="pptt" id="biaoti38378">### Equipment</p><ol class="npwtwok"><li>Standard shielding and other protective equipment for work with <sup>32</sup>P</li><li>Temperature-controlled block and/or thermocycler</li><li>Vortexer and microcentrifuge (for use with MicroSpin G-25 columns)</li><li>Optional: benchtop radiation counter (Bioscan QC-4000)<br/></li></ol><p class="pptt" id="biaoti38379">### Procedure</p><ol class="npwone"><li>Order the DNA oligonucleotide with the appropriate modifications<ol class="npwtwo"><li>Order your DNA oligonucleotide with sequence of interest (Note 1). The 3â²-most nucleotide should be modified to be a ribonucleotide. The penultimate nucleotide (immediately 5â² to the ribonucleotide) should be modified to be a 2â²-O-methylribonucleotide. To specify these modifications in the IDT ordering interface, begin with an order for an RNA oligonucleotide at 100 nmol scale. If the sequence of your DNA oligonucleotide is NNNNNN-3â², your sequence input should be NNNNmNrN.<br/>In the example presented in this protocol, the DNA oligonucleotide sequence is:<br/>5â²-GTCATAATGATTTTATCTTCTGGATTGTTGTAAGCAGCATTTGAGCAAAAATCTGTTmGrC-3â², where all letters indicate deoxynucleotides except âmGâ (which indicates 2â²-O-methylguanosine) and ârCâ (which indicates cytidine).</li><li>Resuspend the DNA oligonucleotide to 10 ÂµM in water (Note 2).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Order or transcribe the phosphate shuttle RNA oligonucleotide and the splint RNA oligonucleotide<ol class="npwtwo"><li>The phosphate shuttle RNA oligonucleotide can be ordered from an oligonucleotide synthesis company or produced by <em>in vitro</em> transcription. If producing by <em>in vitro</em> transcription (Notes 3 and 4), use a method whose final product contains a 5â²-OH (either by treating a T7 RNA polymerase product with phosphatase or including a self-cleaving hammerhead ribozyme in the transcript). It is essential that the phosphate shuttle RNA contain a 5â²-OH for the 5â²-radiolabeling step.<ol class="npwthree"><li>Phosphate shuttle RNA oligonucleotide sequence (Note 5):<br/>5â²-GGGUCGGCAUGGCAUCUC-3â²</li><li>We include below the sequence of an <em>in vitro</em> transcription DNA template for T7 RNA polymerase to generate the phosphate shuttle RNA oligonucleotide (Note 6). Transcripts from this template contain a 5â² hammerhead ribozyme that cleaves to yield a 5â²-OH on the final phosphate shuttle oligonucleotide.<br/>5â²-GTCGAAATTAATACGACTCACTATAGGCGACCCCTGATGAGGCCTTCGGGCCGAAACGGTGAAAGCCGTAGGGTCGGCATGGCATCTC-3â²</li></ol></li><li>The 5â² half of the splint RNA oligonucleotide should be complementary to the phosphate shuttle RNA oligonucleotide. The 3â² half of the splint RNA oligonucleotide should be complementary to the 3â² end of the DNA oligonucleotide to be labeled. The splint RNA oligonucleotide can be ordered from an oligonucleotide synthesis company or produced by <em>in vitro</em> transcription. The substituents on either end of this oligonucleotide are unimportant, as the ligase enzyme will bind to its center.<ol class="npwthree"><li>Sequence of splint RNA oligonucleotide in this example:<br/>5â²-GGAGAUGCCAUGCCGACCCGCAACAGAUUUUUGCUCA-3â²</li><li>We include below the sequence of an <em>in vitro</em> transcription DNA template for T7 RNA polymerase to generate the splint RNA oligonucleotide:<br/>5â²-GTCGAAATTAATACGACTCACTATAGGAGATGCCATGCCGACCCGCAACAGATTTTTGCTCA-3â²</li></ol></li><li>Resuspend each RNA oligonucleotide to 10 ÂµM in RNA storage buffer.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>5â²-radiolabel the phosphate shuttle RNA oligonucleotide<ol class="npwtwo"><li>Combine the following reagents (50 Âµl total reaction volume) (Note 7):<br/>5 Âµl 10x T4 PNK buffer<br/>30 Âµl water<br/>5 Âµl phosphate shuttle RNA oligonucleotide (10 ÂµM stock)<br/>2.5 Âµl T4 PNK enzyme (10,000 U/ml stock)<br/>7.5 Âµl [Î³-<sup>32</sup>P]-ATP (3000 Ci/mmol, 10 mCi/ml stock)</li><li>Allow phosphorylation to progress at 37 Â°C for 2 h.</li><li>Heat-inactivate the T4 PNK by incubating at 65 Â°C for 20 min.</li><li>Add 50 Âµl water to the reaction.</li><li>If interested in determining yield, use the benchtop radiation counter to measure the radioactivity in the reaction at this step.</li><li>Spin the reaction through a MicroSpin G-25 column (no resin pre-equilibration necessary), following the manufacturerâs instructions. This step largely removes any free [Î³-<sup>32</sup>P]-ATP that may remain after the reaction.</li><li>Assuming complete recovery of the RNA oligonucleotide, you should have 100 Âµl of eluate containing 500 nM RNA oligonucleotide, some fraction of which has a 5â²-<sup>32</sup>P-phosphate. Use the benchtop radiation counter to measure the radioactivity in the eluate, and determine the efficiency of 5â²-radiolabeling by comparing to the radioactivity measurement recorded before free ATP removal.<br/><br/></li></ol></li><li>Anneal ligase substrates<ol class="npwtwo"><li>For each DNA oligonucleotide to be 3â²-radiolabeled, combine the following (33 Âµl total reaction volume):<br/>6.6 Âµl 5x annealing buffer<br/>0.3 Âµl water<br/>24 Âµl 5â²-radiolabeled phosphate shuttle RNA oligonucleotide (500 nM stock, directly from eluate in previous step)<br/>1 Âµl modified DNA oligonucleotide (10 ÂµM stock)<br/>1.1 Âµl splint RNA oligonucleotide (10 ÂµM stock)</li><li>Place the annealing reaction on a thermocycler, and run the following temperature program (Note 8):<ol class="npwthree"><li>95 Â°C, 2 min.</li><li>Cool to 25 Â°C over the course of 40 min.</li><li>Hold at 25 Â°C.</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Ligate the DNA oligonucleotide to the phosphate shuttle RNA oligonucleotid<ol class="npwtwo"><li>To the 33-Âµl annealing reaction, add:<br/>4 Âµl 10x T4 RNA ligase 2 reaction buffer<br/>1 Âµl 40 mM MgCl<sub>2</sub><br/>2 Âµl T4 RNA ligase 2 (10,000 U/ml stock)</li><li>Incubate at 37 Â°C overnight (~16 h). If performing quality control, keep a sample from immediately after this incubation to run on a gel and determine ligation efficiency (Figure 3).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201013/20201013003752_2532.jpg" width="300"/><br/><strong>Figure 3. Example autoradiograph of denaturing PAGE analysis.</strong> Lane 1: products of ligation reaction, standard protocol. Lane 2: products of ligation reaction using a DNA oligonucleotide that omits the 2â²-O-methyl modification at the penultimate position (but retains the ribonucleotide modification at the final position), demonstrating the importance of the 2â²-O-methyl modification for ligation efficiency. Lanes 3-7: products after RNA degradation of the ligation products in lane 1 with [NaOH] at 20 mM (lane 3), 50 mM (lane 4), 100 mM (lane 5), 150 mM (lane 6), and 200 mM (lane 7). The band in lane 6 represents the product generated by the conditions recommended in this protocol.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Degrade RNA<ol class="npwtwo"><li>To the 40-Âµl ligation reaction, add 5 Âµl 1.35 M NaOH. Mix thoroughly.</li><li>Incubate at 95 Â°C for 10 min.</li><li>Place on ice, and immediately add 5 Âµl 1.35 M HCl. Mix thoroughly.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Remove salts and mononucleotides by buffer exchange<ol class="npwtwo"><li>Pre-equilibrate a MicroSpin G-25 column with 20 mM Tris-Cl, pH 7.9 (room temperature) according to the manufacturerâs instructions (5 x 400-Âµl equilibration spins).</li><li>Apply the 50-Âµl degradation reaction to the column, and spin according to the manufacturerâs instructions. Add 50 Âµl water to the eluate.</li><li>At this point, the sample should have 100 Âµl total volume, containing 10 mM Tris-Cl, pH 7.9 and 100 nM 3â²-radiolabeled DNA oligonucleotide (as well as the heat-inactivated kinase and ligase enzymes).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Yield and quality check<ol class="npwtwo"><li>Use the benchtop radiation counter to check the yield of transfer of radioactivity from the phosphate shuttle RNA oligonucleotide to the final DNA oligonucleotide. We commonly see recovery of ~75% of transferable phosphates (Note 9).</li><li>Run a sample of the labeled DNA oligonucleotide on denaturing PAGE to verify that the sample contains a pure radiolabeled DNA fragment of the correct size (Notes 10 and 11, Figure 3).</li><li>Store the radiolabeled DNA at -20 Â°C. The limiting factor for the expiration of the radiolabeled oligonucleotide is the radioactivity, as the chemical stability of the product (&gt; years) long outlasts the radioactive decay lifetime of <sup>32</sup>P (half-life 14 days). The product can continue to be used as long as it retains enough radioactivity for the researcherâs particular experimental purposes.</li></ol></li></ol><p class="pptt" id="biaoti38380">### Notes</p><ol class="npwtwok"><li>For the purposes of mapping a DNA cleavage site with single-nucleotide resolution, the baseline purification option from oligonucleotide synthesis companies is often insufficient (the product contains significant concentrations of truncated synthesis products that may confound experimentally important cleavage products). To achieve adequate DNA oligonucleotide purity before 3â²-radiolabeling, either perform a PAGE-purification in house or opt for more extensive purification in your commercial order.</li><li>Standard precautions for working with RNA should be followed. Keep gloves and surfaces clean, and when possible, use filter-containing pipette tips. We do not detect any RNase activity when using MilliQ water in reactions containing RNA, but the experimentalist can purchase commercially certified RNase-free water if there are suspicions of RNase contamination. RNA samples should be stored at -80 Â°C (ideal) or -20 Â°C when not in use, but they can be briefly handled on a room-temperature bench between the steps described in this protocol.</li><li>Consult any standard protocol for the optional steps involving <em>in vitro</em> transcription, PAGE purification, analytical PAGE, and autoradiography.</li><li>If RNA oligonucleotides are produced by <em>in vitro</em> transcription, the resulting transcript should then be PAGE-purified for optimal results.</li><li>The phosphate shuttle RNA oligonucleotide sequence is arbitrary, and we simply included here a sequence that worked well in our experiments. This sequence could presumably be changed provided that compensatory substitutions are also made in the splint RNA oligonucleotide.</li><li>For the <em>in vitro</em> transcription template sequences included in this protocol, only the top strand is shown. Double-stranded DNA templates were assembled and amplified by polymerase chain reaction.</li><li>This protocol yields enough 5â²-radiolabeled phosphate shuttle RNA oligonucleotide to 3â²-radiolabel 4 different DNA oligonucleotides (with a final yield of 100 Âµl at 100 nM for each 3â²-radiolabeled DNA oligonucleotide). Scale the volumes up or down to suit the needs of your particular application, taking care to spread the sample across multiple G-25 MicroSpin columns if the volume limitation of an individual column is exceeded.</li><li>In the absence of a thermocycler, an alternative annealing procedure is as follows: incubate samples at 95 Â°C for 2 min on a heat block, turn off the heat block, and allow the system to gradually equilibrate to room temperature.</li><li>The ~75% yield is after accounting for the stoichiometric excess of phosphate shuttle RNA oligonucleotide in the annealing reaction, which results in a 17% subpopulation whose 5â²-<sup>32</sup>P-phosphate is unavailable for transfer. The absolute fraction of recovered radioactivity is ~60%.</li><li>The most common contaminants are depurination products and n+1 products (some fraction of the DNA molecules contain two ribonucleotides on the 3â² end instead of one, as the n+1 phosphodiester was not cleaved in the RNA degradation reaction). In our preparations, these products were minor or not detected at all (there was variation between different probe preparations), and they did not interfere with our experiments. If the contaminant concentration is too high for your particular oligonucleotide preparation, try quenching the RNA degradation reaction with HCl at a variety of temperatures or time points. Identify a combination of temperature and quenching time that yields an acceptable balance of depurination products (which accumulate over time) and n+1 products (which disappear over time due to cleavage and release of the terminal ribonucleotide). Alternatively, PAGE-purify the 3â²-radiolabeled DNA oligonucleotide for maximal purity.</li><li>The identity of the final species can also be confirmed by treating with T4 PNK and checking by PAGE/autoradiography that all radioactivity has moved from the DNA oligonucleotide to an inorganic phosphate (due to the phosphatase activity of the T4 PNK enzyme that removes 2â²,3â²-cyclic phosphates).<br/></li></ol><p class="pptt" id="biaoti38381">### Recipes</p><ol class="npwtwok"><li>5x annealing buffer<br/>50 mM Tris-Cl, pH 7.9 at 25 Â°C<br/>250 mM KCl<br/>5 mM EDTA</li><li>1x RNA storage buffer<br/>2 mM sodium citrate, pH 6.4 at 25 Â°C<br/>0.1 mM EDTA, pH 8.0<br/></li></ol><p class="pptt" id="biaoti38382">### Acknowledgments</p><p class="ppzz">This protocol was developed with support from a National Science Foundation Graduate Research Fellowship (J.C.C.) and National Science Foundation Award # MCB-1817593 (J.A.D.). The protocol was originally developed for use in Cofsky <em>et al.</em>, 2020. We thank David Colognori for a critical reading of the protocol draft.</p><p class="pptt" id="biaoti38383">### Competing interests</p><p class="ppzz">The Regents of the University of California have patents issued and pending for CRISPR technologies on which J.A.D. is an inventor. J.A.D. is a cofounder of Caribou Biosciences, Editas Medicine, Scribe Therapeutics, and Mammoth Biosciences. J.A.D. is a scientific advisory board member of Caribou Biosciences, Intellia Therapeutics, eFFECTOR Therapeutics, Scribe Therapeutics, Mammoth Biosciences, Synthego, Felix Biosciences, and Inari. J.A.D. is a Director at Johnson &amp; Johnson and has research projects sponsored by Biogen, Pfizer, AppleTree Partners, and Roche. J.C.C. has no competing interests to report.</p><p class="pptt" id="biaoti38384">### Ethics</p><p class="ppzz">This protocol does not include the use of human or animal subjects.</p><p class="pptt" id="biaoti38385">### References</p><ol class="npwtwok"><li>Cofsky, J. C., Karandur, D., Huang, C. J., Witte, I. P., Kuriyan, J. and Doudna, J. A. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/32519675" target="_blank">CRISPR-Cas12a exploits R-loop asymmetry to form double-strand breaks.</a> <em>Elife</em> 9: e55143.</li><li>Jay, E., Bambara, R., Padmanabhan, R. and Wu, R. (1974). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10793670" target="_blank">DNA sequence analysis: a general, simple and rapid method for sequencing large oligodeoxyribonucleotide fragments by mapping.</a> <em>Nucleic Acids Res</em> 1(3): 331-353.</li><li>Nandakumar, J. and Shuman, S. (2004). <a href="https://pubmed.ncbi.nlm.nih.gov/15494308/" target="_blank">How an RNA ligase discriminates RNA versus DNA damage.</a> <em>Mol Cell</em> 16(2): 211-221.</li><li>Wu, R., Jay, E., and Roychoudhury, R. (1976). <a href="https://catalog.hathitrust.org/Record/008514073" target="_blank">Nucleotide Sequence Analysis of DNA. In: Methods in Cancer Research.</a> Vol. 12. Busch, H. (Ed.). <em>Academic Press</em>, 87-176.</li></ol>