<body>
            <h1>Analyzing (Re)Capping of mRNA Using Transcript Specific 5' End Sequencing</h1>
            <p><strong>Authors:</strong> Daniel del Valle Morales, Daniel R. Schoenberg</p>
            <h2>Abstract</h2>
            <p>The 5â² cap is a ubiquitous feature of eukaryotic mRNAs. It is added in the nucleus onto newly synthesized pre-mRNA, and in the cytoplasm onto mRNAs after decapping or endonuclease cleavage. Cytoplasmic recapping can occur after loss of the cap at the native 5â² end, or downstream within the body of the mRNA. The identification and location of recapping events is key to understanding the functional consequences of this process. Here we present an approach that addresses this problem, using the Lexogen TeloPrime<sup>Â® </sup>cDNA synthesis kit to tag recapped 5â² ends. TeloPrime uses a proprietary DNA ligase to add a double stranded DNA oligonucleotide onto the 3â² end of cDNA while it is base paired with mRNA. Specificity for capped ends is obtained by the oligonucleotide having an unpaired C residue that base pairs weakly with <sup>m7</sup>G on the mRNA 5â² end. This is followed by PCR amplification of double-stranded cDNA using primers to the appended oligonucleotide and the mRNA of interest. The resulting products are gel purified and sequenced directly (if a single band) or cloned and sequenced. The sequence at the junction between the ligated oligonucleotide and the target mRNA provides the location of the cap on the corresponding transcript. This assay is applicable to all capped transcripts. It can be used with Sanger sequencing for small numbers of transcripts or adapted for use with Illumina library sequencing.</p>
            <p><strong>Keywords:</strong> 5â² Cap, Cap end mapping, mRNA, Primer ligation, PCR</p>
            <h2>Background</h2><p>The N7-methylguanosine cap is a distinguishing feature of all eukaryotic mRNAs. Proteins binding to the cap function all stages of the mRNA lifecycle, including nuclear processing, export, translation and mRNA decay. The 5â² cap is added co-transcriptionally to all mRNAs, and a number of genome wide techniques (<em>e.g.</em>, Capped Analysis of Gene Expression, or CAGE) (Morioka<em> et al</em>., 2020) have been developed that use the identification of capped ends as a way to mark transcription initiation sites. In addition to marking transcription start sites approximately 25% of CAGE tags map downstream within spliced introns (Djebali<em> et al.</em>, 2012). Evidence of the biochemical basis for this came from our labâs identification in 2009 of a cytoplasmic complex capable of restoring the N7-methylguanosine cap onto transcripts with a 5â²-monophosphate end, but not onto transcripts with a 5â²-hydroxyl end (Otsuka <em>et al.</em>, 2009). Cytoplasmic capping is catalyzed by a complex of enzymes including capping enzyme (RNGTT), cap methyltransferase (RNMT) and its activating subunit (RAM), and a kinase that converts the 5â²-monophosphate ends of decapped transcripts to a 5â²-diphosphate substrate for GMP addition (Trotman and Schoenberg, 2019). Our early work was based on use of a dominant negative form of capping enzyme that blocks cytoplasmic capping at the GMP addition step. Expression of this protein resulted in the appearance of a number of uncapped transcripts whose ends were mapped using 5â-RACE to the vicinity of downstream CAGE tags (Kiss <em>et al.</em>, 2015; Berger <em>et al.</em>, 2019). While encouraging, this approach has three major drawbacks; a) it requires separation of capped and uncapped RNAs, b) it assumes uncapped transcripts remain sufficiently stable to be detected, and c) it assumes uncapped ends detected in this manner have undergone limited additional exonucleolytic trimming.<br/><br/>These issues were resolved by the development of a new approach to inhibiting cytoplasmic capping that instead targets cap methylation (Del Valle Morales <em>et al.</em>, 2020). Inhibition of cytoplasmic cap methylation results in the creation of recapped mRNAs with unmethylated G caps which are degraded by cap surveillance enzymes (Grudzien-Nogalska and Kiledjian, 2017). This makes it possible to identify cytoplasmic capping targets by quantitative changes in RNA-Seq data, thus eliminating assumptions about the stability and detectability of uncapped RNAs, the need to separate uncapped from capped RNAs, and concerns for exonucleolytic trimming. By simplifying the process of identifying cytoplasmic capping targets this approach makes it possible to study this process in any cells or tissues expressing the cytoplasmic cap methylation inhibitor. Finally, because these transcripts retain their caps, we paired this approach with a facile method for identifying capped ends that can be scaled from a few mRNAs to larger sequencing libraries.</p>
        </body>

<p class="pptt" id="biaoti38417">## Materials and Reagents</p><ol class="npwtwok"><li>0.2 ml PCR tubes (<em>e.g.</em>, BioExpress, GeneMate, catalog number: <a class="blackNoUnder" target="_blank"> T-3225-1 </a> )</li><li>1.7 ml plastic, sterile, RNase-free microcentrifuge tubes (<em>e.g.</em>, BioExpress, GeneMate, catalog number: <a class="blackNoUnder" target="_blank"> C-3262-1 </a> )</li><li>10 cm or 15 cm culture dishes (<em>e.g</em>., Alkali Scientific, catalog numbers: <a class="blackNoUnder" target="_blank"> TD0100 </a> , TD0150)</li><li>Sterile nitrile gloves</li><li>18 G needle</li><li>Cell lifters (<em>e.g.</em>, BioExpress, GeneMate, catalog number: <a class="blackNoUnder" target="_blank"> T-2443-4 </a> )</li><li>Sterile, RNase-free tips for P10, P20, P100, and P1000 pipets</li><li>CleanCap<sup>Â®</sup> mCherry mRNA (Trilink, catalog number: <a class="blackNoUnder" target="_blank"> L-7203 </a> )</li><li>mCherry forward primer: ATGGTGAGCAAGGGCGAGGAG</li><li>mCherry reverse primer: GCCACCCTTGGTCACCTTCAGC</li><li>Gene specific primer</li><li>U2OS cells (ATCC, catalog number: <a class="blackNoUnder" target="_blank"> HTB-96 </a> ) or HEK293 cells (ATCC, catalog number: <a class="blackNoUnder" target="_blank"> CRL-1573 </a> )</li><li>RNase-free water (<em>e.g.</em>, from a Millipore Synergy water purification system, 18.2 MÎ© cm)</li><li>TeloPrime Full-Length cDNA Amplification Kit V2 (Lexogen, catalog number: 013.24)</li><li>RNA buffer (part of TeloPrime kit)</li><li>DNA buffer (part of TeloPrime kit)</li><li>Reverse transcription mix (RT) (part of TeloPrime kit)</li><li>Reverse transcription primer (RTP) (part of TeloPrime kit)</li><li>Enzyme mix 1 (E1) (part of TeloPrime kit)</li><li>Ligation mix (LM) (part of TeloPrime kit)</li><li>Enzyme mix 2 (E2) (part of TeloPrime kit)</li><li>Second strand synthesis mix (SS) (part of TeloPrime kit)</li><li>Enzyme mix 3 (E3) (part of TeloPrime kit)</li><li>PCR forward primer (FP) (part of TeloPrime kit, 5â-TGGATTGATATGTAATACGACTCACTATAG)</li><li>PCR reverse primer (RP) (part of TeloPrime kit)</li><li>TeloPrime PCR mix (Telo PCR) (part of TeloPrime kit)</li><li>Column binding buffer 1 (CB1) (part of TeloPrime kit)</li><li>Column binding buffer 2 (CB2) (part of TeloPrime kit)</li><li>Column wash buffer (CW) (part of TeloPrime kit)</li><li>Purifications columns (part of TeloPrime kit)</li><li>Collection tubes (part of TeloPrime kit)</li><li>SensiFAST<sup>TM</sup> SYBR<sup>Â®</sup> No-ROX (Bioline, catalog number: <a class="blackNoUnder" target="_blank"> BIO-98050 </a> )</li><li>SYBR Gold Nucleic Acid Gel Stain (Thermo Fisher Scientific, Invitrogen<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> S11494 </a> )</li><li>DNA size marker (<em>e.g.</em>, Thermo Fisher Scientific, Invitrogen<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> AM7778 </a> )</li><li>2x DNA loading dye (<em>e.g.</em>, Thermo Fisher Scientific, Thermo Scientific<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> R0641 </a> )</li><li>40 U/Î¼l RNaseOUT Recombinant Ribonuclease Inhibitor (Thermo Fisher Scientific, Invitrogen<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 10777019 </a> )</li><li>RNA Clean and Concentrator-25 (Zymo Research, catalog number: <a class="blackNoUnder" target="_blank"> R1017 </a> )</li><li>DNA Clean and Concentrator-5 (Zymo Research, catalog number: <a class="blackNoUnder" target="_blank"> D4013 </a> )</li><li>pGEM<sup>Â®</sup>-T Easy Vector Systems (Promega, catalog number: <a class="blackNoUnder" target="_blank"> A1360 </a> )</li><li>pGEM<sup>Â®</sup>-T Easy Vector (part of pGEM<sup>Â®</sup>-T Easy Vector Systems kit)</li><li>T4 DNA Ligase (part of pGEM<sup>Â®</sup>-T Easy Vector Systems kit)</li><li>2X Rapid Ligation Buffer ( part of pGEM<sup>Â®</sup>-T Easy Vector Systems kit)</li><li>Stellar<sup>TM</sup> Competent Cells (Clontech, catalog number: <a class="blackNoUnder" target="_blank"> 636766 </a> )</li><li>Bacterial culture tube (<em>e.g.</em>, VWR, catalog number: <a class="blackNoUnder" target="_blank"> 20170-579 </a> )</li><li>LB Agar Ampicillin-100 (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> L5667 </a> )</li><li>NucleoSpin<sup>Â®</sup> Plasmid DNA Purification Kit (Clontech, catalog number: 740588.50)</li><li>McCoyâs 5A medium (for U2OS cells; Thermo Fisher Scientific, Gibco<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 16600082 </a> )</li><li>Dulbeccoâs modified Eagle medium (DMEM; for HEK293 cells; Thermo Fisher Scientific, Gibco<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 21013024 </a> )</li><li>Fetal bovine serum (FBS; Atlanta Biologicals, catalog number: <a class="blackNoUnder" target="_blank"> S10350 </a> )</li><li>Phosphate-buffered saline (PBS; <em>e.g.</em>, Thermo Fisher Scientific, Gibco<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 10010049 </a> )</li><li>RNase-free 10x Tris/borate/EDTA (TBE) buffer (<em>e.g.</em>, Thermo Fisher Scientific, Invitrogen<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 15581 </a> )</li><li>Ethanol</li><li>Isopropanol</li><li>Sodium chloride (NaCl; <em>e.g</em>., Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP358 </a> )</li><li>5 M NaCl (<em>e.g.</em>, Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9759 </a> )</li><li>3 M sodium acetate, pH 5.2 (<em>e.g.</em>, Thermo Fisher Scientific, Thermo Scientific<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> R1181 </a> )</li><li>Ammonium persulfate (<em>e.g.</em>, Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> A3678 </a> )</li><li>1 M Tris-HCl, pH 7.5 (<em>e.g.</em>, AMRESCO, catalog number: <a class="blackNoUnder" target="_blank"> E691 </a> )</li><li>1 M magnesium chloride (MgCl<sub>2</sub>,<em> e.g.</em>, Thermo Fisher Scientific, Invitrogen<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> AM9530G </a> )</li><li>2 M potassium chloride (KCl; <em>e.g</em>., Alfa Aesar, catalog number: <a class="blackNoUnder" target="_blank"> J75896 </a> )</li><li>0.1 M DTT (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 707265ML </a> )</li><li>MyTaq<sup>TM</sup> Mix (Bioline catalog number: <a class="blackNoUnder" target="_blank"> BIO-25041 </a> )</li><li>Tetramethylethylenediamine (TEMED) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 17919 </a> )</li><li>40% Acrylamide/Bis Solution (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1610144 </a> )</li><li>GlycoBlue<sup>TM</sup> Coprecipitant (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9515 </a> )</li><li>1x TBE (running buffer for urea-PAGE) (see Recipes)</li><li>10% (w/v) ammonium persulfate (see Recipes)</li><li>Cytoplasmic lysis buffer (see Recipes)</li><li>8% native PAGE gel (see Recipes)<br/></li></ol><p class="pptt" id="biaoti38418">### Equipment</p><ol class="npwtwok"><li>Eye protection</li><li>NanoDrop spectrophotometer (Thermo Fisher Scientific, model: <a class="blackNoUnder" target="_blank"> NanoDrop<sup>TM</sup> 1000 </a> , catalog number: <a class="blackNoUnder" target="_blank"> ND-1000 </a> ) or other instrument suitable for quantifying nucleic acids</li><li>P10, P20, P100, and P1000 pipets</li><li>Thermal cycler (<em>e.g.</em>, MJ Research, model: <a class="blackNoUnder" target="_blank"> PTC-200 </a> )</li><li>Quantitative PCR instrument (<em>e.g.</em>, Bio-Rad CFX Connect Real-Time System)</li><li>Heating block (<em>e.g.</em>, Bioer, model: <a class="blackNoUnder" target="_blank"> MB 101 </a> )</li><li>-80 Â°C freezer</li><li>Water-jacketed humidified incubator (<em>e.g.</em>, Thermo Fisher Scientific, Thermo Scientific<sup>TM</sup>, model: <a class="blackNoUnder" target="_blank"> Forma<sup>Â® </sup>Series II </a> ) set to 37 Â°C with 5% CO<sub>2</sub> </li><li>Refrigerated centrifuge (Eppendorf, model: <a class="blackNoUnder" target="_blank"> 5415 R </a> )</li><li>Programmable rotator-mixer (Grant Instruments, model: <a class="blackNoUnder" target="_blank"> PTR-30 </a> ) at 4 Â°C and set to 10 rpm orbital rotation</li><li>Electrophoresis system for PAGE (<em>e.g.</em>, Bio-Rad Laboratories, catalog number: <a class="blackNoUnder" target="_blank"> 1660828EDU </a> )</li><li>Azure Sapphire Biomolecular Imager (or similar imaging system)</li><li>UV transilluminator (BT Lab Systems, model: <a class="blackNoUnder" target="_blank"> BT503 </a> )<br/></li></ol><p class="pptt" id="biaoti38419">### Software</p><ol class="npwtwok"><li>Microsoft Excel software</li><li>ImageJ software</li><li>Bio-Rad CFX Maestro software</li><li>Adobe Photoshop</li><li>Azure Sapphire Capture Software <br/></li></ol><p class="pptt" id="biaoti38420">### Procedure</p><p class="ppzz">A schematic overview of TeloPrime mapping of capped ends is presented in Figure 1.</p><p class="ppzz" style="margin-left:50px;"><img alt="" class="layerphoto" height="350" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201018/20201018182052_1610.jpg" width="480"/><br/><strong>Figure 1. Schematic overview of cap methyltransferase activity assay.</strong> The population of cytoplasmic RNA (large black arrow) is reverse transcribed using the primer (small grey arrow) and reverse transcriptase (large grey arrow) included in the TeloPrime kit. The resulting mRNA:cDNA hybrid is ligated to a double-stranded DNA adapter that has a 5â² overhanging C residue on the antisense strand (red arrow) that weakly base pairs with <sup>m7</sup>G of the cap. Note that the G of the cap is not reverse transcribed. Ligation is performed using a proprietary double stranded DNA ligase included in the TeloPrime kit. RNA is then removed by second strand synthesis (large blue arrow) and specific transcripts are PCR amplified, using primers to the gene of interest (small purple arrow) and to the ligated adapter (small black arrow). Products are separated on a non-denaturing gel that is subsequently stained with SYBR Gold. DNA recovered from individual bands can be sequenced directly, cloned and sequenced, or secondarily processed to add Illumina sequencing adapters.</p><br/><ol class="npwone"><li>Isolation of cytoplasmic RNA<br/><ol class="npwtwo"><li>Nuclear RNA must be removed to assay cytoplasmic recapping. The following steps are used to isolate cytoplasmic RNA.</li><li>Two to three days in advance, seed 10-cm or 15-cm dishes with U2OS, HEK293, or other adherent mammalian cells, at a density that will reach ~80-90% confluence at the time of harvesting. U2OS cells should be grown in McCoyâs 5A medium containing 10% (v/v) FBS; HEK293 cells should be grown in DMEM containing 10% (v/v) FBS. Cells should be grown in a humidified incubator at 37 Â°C with 5% CO<sub>2</sub>.</li><li>On the day of harvesting the cells, chill Cytoplasmic lysis buffer (see Recipes) on ice. Leave out dithiothreitol (DTT) and RNAseOUT until immediately before use.</li><li>Remove culture medium from adherent cells and rinse with phosphate buffered saline (PBS). Harvest the cells by scraping into 1 ml of PBS using a cell lifter and transferring to 1.7 ml centrifuge tubes. Gently pellet the cells (70 <em>x g</em> for 10 min at 4Â°C), and then carefully remove the supernatant. Make note of the approximate volume of the cell pellets and place the tubes on ice.</li><li>Add a volume of Cytoplasmic lysis buffer equivalent to 4-5 times the volume of each cell pellet. Gently resuspend the cells with 10 up-and-down strokes of a P1000 pipet. Incubate the cells on ice for 10 min, and then mix with 5 additional up-and-down strokes of the P1000 pipet.</li><li>Remove nuclei by centrifuging at 16,100<em> x g</em> for 10 min at 4 Â°C, then transfer the supernatants to new, pre-chilled 1.7 ml tubes on ice.</li><li>Recover cytoplasmic RNA from the supernatant using Zymo RNA Clean and Concentrator-25 according to manufacturerâs instructions. Measure the concentration of RNA with a NanoDrop spectrophotometer.<br/><br/></li></ol></li><li>TeloPrime protocol<br/><ol class="npwtwo"><li>TeloPrime is a reverse transcription protocol that will specifically enrich for capped mRNAs. The steps follows the protocol provided in the TeloPrime kit.</li><li>Thaw Reverse transcription mix (RT), Reverse transcription primer (RTP), Ligation mix (LM), and RNA buffer from the TeloPrime kit at room temperature. Thaw Enzyme mix 1 and 2 (E1 and E2 from the TeloPrime Kit), CleanCap<sup>Â®</sup> mCherry mRNA, and RNA sample on ice.</li><li>Prepare a 12 Î¼l mixture in a 0.2 ml PCR tube containing the following: 1 ng of CleanCap<sup>Â®</sup> mCherry mRNA, 2 Î¼g of RNA sample, and fill the remaining volume with RNA buffer. Keep the mixture at room temperature. Add 2 Î¼l of RTP to the mixture.</li><li>Denature the mixture for 30 s at 70 Â°C then cool to 37 Â°C in a thermocycler and maintain this at 37 Â°C while preparing a mixture of 4 Î¼l of Reverse transcription mix (RT) and 2 Î¼l of Enzyme mix 1 (E1). Add this to the sample tube while it is in the thermocycler. Mix the reaction by pipetting slowly, then incubate the reaction for 2 min at 37 Â°C and 50 min at 46 Â°C.</li><li>Equilibrate Column binding buffer 1 (CB1) and Column wash buffer (CW) to room temperature. Place the purification column on a collection tube. Add 20 ml of 100% ethanol to the CW bottle.</li><li>Add 160 Î¼l of CB1 to the reaction and mix well by vortexing. Transfer the reaction to the purification column and centrifuge for 1 min at 12,000 <em>x g</em> at room temperature. Discard the flowthrough. Add 200 Î¼l of CW to the column and centrifuge for 1 min at 12,000<em> x g</em> at room temperature. Repeat for a total of two washes.</li><li>Discard the flowthrough and centrifuge the column for 2 min at 12,000<em> x g</em> at room temperature to remove excess liquid. Transfer the column to a new 1.5 ml tube and add 19 Î¼l of RNA buffer. Centrifuge the tube for 2 min at 12,000 <em>x g</em> at room temperature.</li><li>Transfer 18 Î¼l of the eluate to a new 0.2 ml PCR tube. Prepare a mixture containing 20 Î¼l of Ligation mix (LM) and 2 Î¼l of Enzyme mix 2 (E2) for double stranded adapter ligation. Add this to the eluted cDNA and incubate overnight at 25Â°C (see Note 1).</li><li>The next day add 320 Î¼l of CB1 to the reaction and mix well by vortexing. Transfer the reaction to a new purification column and collection tube, and centrifuge for 1 min at 12,000<em> x g</em> at room temperature. Discard the flowthrough. Add 400 Î¼l of CW to the column and centrifuge for 1 min at 12,000<em> x g</em> at room temperature. Repeat for a total of two washes.</li><li>Discard the flowthrough and centrifuge the column for 2 min at 12,000 <em>x g</em> at room temperature. Transfer the column to a new 1.5 ml tube and add 12 Î¼l of RNA buffer. Centrifuge the tube for 2 min at 12,000 <em>x g</em> at room temperature.</li><li>Thaw second strand mix (SS), forward primer (FP), reverse primer (RP), TeloPrime PCR mix (Telo PCR) and DNA buffer at room temperature. Thaw Enzyme mix 3 (E3) on ice. Equilibrate Column binding buffer 2 (CB2) at room temperature.</li><li>Transfer 11 Î¼l of the eluted cDNA to a new 0.2 ml PCR tube. Mix 8 Î¼l of second strand mix (SS) with 1 Î¼l of E3, add this to the eluted cDNA and mix well. Place the tube in a thermocycler and process for 90 s at 95 Â°C, 60 s at 62 Â°C, 5 min at 72 Â°C.</li><li>Add 160 Î¼l of CB2 to the reaction and mix well by vortexing. Transfer the reaction to a new purification column and collection tube, and centrifuge for 1 min at 12,000 <em>x g</em> at room temperature. Discard the flowthrough. Add 200 Î¼l of CW to the column and centrifuge for 1 min at 12,000 <em>x g</em> at room temperature. Repeat the wash for a total of two washes</li><li>Discard the flowthrough and centrifuge the column for 2 min at 12,000<em> x g</em> at room temperature to remove excess liquid. Transfer the column to a new 1.5 ml tube and add 20 Î¼l of DNA buffer. Centrifuge the tube for 2 min at 12,000<em> x g</em> at room temperature to recover the double stranded cDNA.<br/></li></ol><br/></li><li>Optimizing cDNA amplification<br/><ol class="npwtwo"><li>qPCR amplification of the double stranded cDNA is performed to determine the ideal number of cycles to use for each sample. This also serves as a quality control step.</li><li>Prepare a qPCR reaction containing the following: 1 Î¼l of cDNA, 0.5 Î¼l of forward primer (FP), 0.5 Î¼l of reverse primer (RP), 5 Î¼l of SensiFAST<sup>TM</sup> SYBR<sup>Â®</sup> No-ROX, and 3 Î¼l of RNase free water. Perform the following 40 cycle program on a qPCR thermocycler: 1 cycle of 95 Â°C for 30 s, 50 Â°C for 45 s and 72 Â°C for 5 min; 39 cycles of 95 Â°C for 30 s, 62 Â°C for 30 s and 72 Â°C for 5 min.</li><li>Determine the maximum value in which fluorescence reaches a plateau on the qPCR amplification curve. Calculate where the fluorescence is at 50% (y-axis) and extrapolate the C<sub>T</sub> value (x-axis). See Figure 2A for an example. The C<sub>T</sub> value is the number of cycles used for cDNA amplification (see Note 2).<br/></li></ol><br/></li><li>Endpoint PCR amplification<br/><ol class="npwtwo"><li>The samples are PCR amplified according to the cycles determined in Procedure C to generate enough cDNA for downstream analysis.</li><li>Prepare a PCR reaction in a 0.2 ml PCR tube containing the following: 8 Î¼l of TeloPrime PCR mix, 1 Î¼l of reverse primer (RP), 1 Î¼l of forward primer (FP), 9 Î¼l of cDNA and 1 Î¼l E3. Perform the following PCR reaction: 95 Â°C for 30 s, 50 Â°C for 45 s, 72 Â°C for 5 min; x the number of cycles determined in Step C2.</li><li>Add 160 Î¼l of CB2 to the reaction and mix well by vortexing. Transfer the reaction to the purification column and centrifuge for 1 min at 12,000 <em>x g</em> at room temperature. Discard the flowthrough. Add 200 Î¼l of CW to the column and centrifuge for 1 min at 12,000 <em>x g</em> at room temperature. Repeat the wash for a total of two washes.</li><li>Discard the flowthrough and centrifuge the column for 2 min at 12,000 <em>x g </em>at room temperature. Transfer the column to a new 1.5 ml tube and add 20 Î¼l of DNA buffer. Centrifuge the tube for 2 min at 12,000 <em>x g</em> at room temperature. qPCR amplification of the double stranded cDNA is performed to determine the ideal number of cycles to use for each sample. This also serves as a quality control step.<br/><br/></li></ol></li><li>mCherry normalization for gene specific PCR<br/><ol class="npwtwo"><li>Capped mCherry RNA is included in each sample as a normalization control. Prior to analyzing individual transcripts PCR is performed on the recovered cDNA using mCherry primers. The cDNA preparations are then adjusted prior to gene-specific amplification.</li><li>Prepare a PCR reaction in a 0.2 ml PCR tube for each sample containing the following: 10 Î¼l of myTAQ x2 mix, 1 Î¼l of 10 Î¼M mCherry forward primer, 1 Î¼l of 10 Î¼M mCherry reverse primer, 1 Î¼l of cDNA sample and 7 Î¼l of RNase-free water. Perform the following PCR reaction: 95 Â°C for 1 min, 20 cycles of 95 Â°C for 15 s, 60 Â°C for 15 s, 72 Â°C for 10 s.</li><li>Transfer the PCR reaction to a 1.5 ml tube. Add 1 Î¼l of GlycoBlue<sup>TM</sup>, 2 Î¼l of 3 M sodium acetate, pH 5.2, and 46 Î¼l of ice cold 100% ethanol. Vortex and place at -80 Â°C for 30 min. The product is recovered by centrifuging at 12,000<em> x g</em> for 15 min, at 4 Â°C. Carefully remove the supernatant without disturbing the pellet. Add 200 Î¼l of ice cold 70% ethanol to the pellet and centrifuge for 15 min at 12,000 <em>x g</em> at 4 Â°C. Carefully remove the supernatant without disturbing the pellet. Air dry the pellet at room temperature for 10-15 min. Resuspend the pellet with 10 Î¼l of RNase-free water.</li><li>Add 10 Î¼l of 2x DNA loading dye to the reaction. Load 2 Î¼l of DNA size marker on the first well and each sample to the remaining wells of an 8% native page gel. Electrophorese at 120 V for 30 min in 1x Tris-Borate-EDTA (TBE) buffer. Place the gel in a plastic container slightly larger than itself and stain with a mixture of 20 ml of 1x TBE containing 2 Î¼l of SYBR<sup>TM</sup> Gold nucleic acid gel stain. Incubate the gel at room temperature for 15 min. Image the stained gel on an Azure Sapphire or other suitable imaging instrument. Normalize samples using the intensity of the mCherry bands as determined with ImageJ.<br/></li></ol><br/></li><li>Gene specific PCR<br/><ol class="npwtwo"><li>Gene specific PCR is performed on the normalized samples to determine the 5â cap site of the gene of interest.</li><li>Prepare a PCR reaction in a 0.2 ml PCR tube for each sample containing the following: 10 Î¼l of myTAQ 2x mix, 1 Î¼l of forward primer (FP), 1 Î¼l of 10 Î¼M gene specific reverse primer (see Notes 2 and 3), 1 Î¼l of cDNA sample adjusted to mCherry in Procdure E and 7 Î¼l of RNase-free water. Perform the following PCR reaction: 95 Â°C for 1 min, 20 cycles of 95 Â°C for 15 s, 50 Â°C (see Note 3) for 15 s, 72 Â°C for 10 s.</li><li>Transfer the PCR reaction to a 1.5 ml tube. Add 1 Î¼l of GlycoBlue<sup>TM</sup>, 2 Î¼l of 3 M sodium acetate, pH 5.2, and 46 Î¼l of ice cold 100% ethanol, vortex and place at -80 Â°C for 30 min. Recover the product by centrifuging at 12,000<em> x g</em> for 15 min at 4 Â°C. Carefully remove the supernatant without disturbing the pellet. Add 200 Î¼l of ice cold 70 % ethanol to the pellet and centrifuge for 15 min at 12,000<em> x g</em> at 4 Â°C. Carefully remove the supernatant without disturbing the pellet. Air dry the pellet at room temperature for 10-15 min. Resuspend the pellet with 10 Î¼l of RNase-free water.</li><li>Add 10 Î¼l of 2x DNA loading dye to the reaction. Load DNA size markers in the first 1 or 2 wells, mCherry in the next well, and experimental samples in the remaining wells of an 8% native page gel. Electrophorese in 1x TBE at 120 V for 30 min and stain with SYBR<sup>TM</sup> Gold nucleic acid gel stain as in Step E4, and visualize this on a gel imager (see Note 5). An example of this is shown in Figure 2. This figure shows results for the mCherry control, one mRNA with a single capped end (RPS3) and two mRNAs that have multiple capped end species (EIF3K and EIF3D). The solid dots indicate canonical RefSeq cap sites as determined by sequencing. Save the gel for band purification.<br/><br/><img alt="" class="layerphoto" height="350" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201018/20201018185026_4420.jpg" width="622"/><br/><strong>Figure 2. Examples of an amplification curve, a gel of TeloPrime generated 5â² end PCR products, and sequencing results for EIF3K.</strong> A. An example of a gene-specific amplification curve is shown with the plateau, 50% point and optimal number of cycle for this sample indicated. B. Shown is a gel for cap end PCR products for mCherry, one mRNA that has only a single capped species (RPS3), and two mRNAs that have capped ends located both at the canonical RefSeq cap site (indicated with a black dot) and further downstream. C. PCR amplification of EIF3D and EIF3K generated closely spaced bands. These were excised in a single gel piece and cloned, and multiple colonies were sequence to identify capped ends. The sequences of 24 colonies from EIF3K are shown beneath the figure, with the RefSeq sequence in bold and the number of colonies of a particular sequence indicated in parentheses (data are from Del Valle Morales <em>et al.</em>, 2020).</li></ol></li><br/><li>Purification of gel bands<br/><ol class="npwtwo"><li>The bands of interest are excised and purified from the gel. The purified PCR products can be directly sequenced or in the case of multiple bands, cloned into plasmids for sequencing.</li><li>Place the gel on a UV transilluminator set to 300 nm wavelength. While wearing gloves, lab coat, and a face shield, use a clean razor blade to excise the band of interest of the gel (see Note 6).</li><li>Place the gel slice in a 0.6 ml microfuge tube having a hole at the bottom made with an 18 G needle. Note that if there are multiple closely spaced (as in Figure 2 with EIF3K and EIF3D) these should be recovered in a single gel slice and processed for cloning as described in Procedure H.</li><li>Place the punctured microfuge tube with the gel slice in a 1.5 ml microfuge tube and centrifuge for 2 min at 12,000<em> x g</em> at room temperature to shred the gel. Add 5x volume of RNase-free water to the shredded gel slice. Incubate overnight at room temperature with rotation.</li><li>The next day, centrifuge the tube for 2 min at 12,000 <em>x g</em> at room temperature to collect the gel pieces at the bottom of the tube. Using a 200 Î¼l pipette tip, carefully transfer the supernatant without disturbing the gel pieces to a new 1.5 ml tube.</li><li>To the recovered supernatant add 1 Î¼l of GlycoBlue<sup>TM</sup>, 0.1 volume of 3 M sodium acetate, pH 5.2, and 1 volume of ice-cold 100% isopropanol. Place this in a -80 Â°C freezer for 30 min, then centrifuge for 15 min at 12,000 <em>x g</em> at 4 Â°C. Carefully remove the supernatant without disturbing the pellet. Add 200 Î¼l of ice cold 70% ethanol to the pellet and centrifuge for 15 min at 12,000 <em>x g</em> at 4 Â°C. Carefully remove the supernatant without disturbing the pellet. Air dry the pellet at room temperature for 10-15 min. Resuspend the pellet with 10 Î¼l of RNase-free water. The recovered product of a single PCR band can be directly sequenced by Sanger sequencing. Recovered products from multiple bands need to be cloned and sequenced as described below.<br/></li></ol><br/></li><li>Cloning products from multiple closely spaced bands<br/><ol class="npwtwo"><li>For multiple closely spaced bands, the recovered PCR product are cloned using TA cloning. The transformed bacterial colonies are then selected and sequenced to determine the capping site (see the legend to Figure 2).</li><li>Prepare an A-tailing reaction in a 0.2 ml PCR tube containing 10 Î¼l of recovered PCR products and 10 Î¼l of myTAQ x2 mix. Incubate the reaction in a thermocycler for 20 min at 70 Â°C. Purify the reaction using Zymo DNA Clean and Concentrator-5 according to the manufacturer's instructions. Quantify the concentration of the recovered cDNA with a nanodrop spectrophotometer.</li><li>Prepare a TA-cloning ligation using the pGEM<sup>Â®</sup>-T Easy Vector Systems. Prepare the ligation reaction containing the following: 5 Î¼l of 2x Rapid Ligation Buffer, 1 Î¼l of T4 DNA ligase, an insert to vector (pGEM<sup>Â®</sup>-T Easy Vector) ratio of 1:3 and fill to 10 Î¼l with RNAse-free water. Incubate the reaction at room temperature for 1 h.</li><li>Transform 3 Î¼l of the ligated vector to Stellar competent cells following the manufacturer's instructions. Plate the transformed cells on LB/Ampicillin plates and incubate overnight at 37 Â°C. The next day pick individual colonies from the LB/ampicillin plate with a sterile 10 Î¼l pipette tip and transfer the colonies to a 5 ml bacterial culture tube containing liquid medium (LB/ampicillin). Incubate the bacterial culture tube in an incubated shaker overnight at 37 Â°C. To control for the recovery of plasmids without inserts at least 50 colonies should be sequenced.</li><li>The following day purify the plasmid DNA from the bacterial culture tube using NucleoSpin Plasmid kit according to the manufacturer's instructions. The purified plasmid can be sequenced by Sanger sequencing to identify the insert of the vector.<br/></li></ol></li></ol><ol class="npwone"></ol><p class="pptt" id="biaoti38421">### Notes</p><ol class="npwtwok"><li>The double stranded ligation can be done on a thermocycler overnight.</li><li>The ideal C<sub>T</sub> value for cDNA amplification ranges from 10-25 and will differ depending on the mRNA of interest and its relative representation in the sample.</li><li>The number of cycles used for gene specific PCR will depend on the gene of interest. 20 cycles are used as a baseline.</li><li>Ensure that the Tm of the gene specific primer is within 5 Â°C of the FP.</li><li>The polyacrylamide gel concentration can be altered to improve resolution of closely spaced bands.</li><li>When excising the band from the gel, try to cut out the smallest gel piece possible that contains the band. Excess gel can affect the recovery of the band.</li><li>This protocol can be modified for use with Illumina sequencing to identify capped ends on multiple targets. This is done by performing PCR on the final cDNA product using gene specific primers fused to the Illumina PE1 adapter and TeloPrime Forward Primer (FP) fused to barcoded Illumina PE2 adapter.<br/></li></ol><p class="pptt" id="biaoti38422">### Recipes</p><ol class="npwtwok"><li>1x TBE (running buffer for urea-PAGE)<br/>Bring 100 ml of RNase-free 10x TBE to 1 L with RNase-free water<br/>Store at room temperature</li><li>10% (w/v) ammonium sulfate<br/>Dissolve 1 g of ammonium persulfate in RNase-free water to 10 ml<br/>Store at 4 Â°C</li><li>Cytoplasmic lysis buffer<br/>20 mM Tris-HCl, pH 7.5<br/>150 mM NaCl<br/>5 mM MgCl<sub>2</sub><br/>1 mM DTT<sup>*</sup><br/>0.2% NP-40 <br/>80 U/ml RNaseOUT<sup>*</sup><br/><sup><em>*</em></sup><em>Note: Add immediately before using the buffer.</em> </li><li>8% Native PAGE gel<br/>2.4 ml of 40% Acrylamide/Bis Solution<br/>9.6 ml of 1x TBE<br/>200 Î¼l of 10% (w/v) ammonium persulfate<sup>*</sup><br/>10 Î¼l of TEMED<sup>*</sup><br/><sup><em>*</em></sup><em>Note: Add immediately before pouring the gel.</em><br/></li></ol><p class="pptt" id="biaoti38423">### Acknowledgments</p><p class="ppzz">This work was supported by R01 grant GM084177 from the National Institutes of Health (to D.R.S). The protocol presented here was used for experiments in Del Valle Morales<em> et al.</em> (2020).</p><p class="pptt" id="biaoti38424">### Competing interests</p><p class="ppzz">The authors declare no conflicts of interest or competing interests.</p><p class="pptt" id="biaoti38425">### References</p><ol class="npwtwok"><li>Berger, M. R., Alvarado, R. and Kiss, D. L. (2019). <a href="https://pubmed.ncbi.nlm.nih.gov/30810230/" target="_blank">mRNA 5â ends targeted by cytoplasmic recapping cluster at CAGE tags and select transcripts are alternatively spliced.</a> <em>FEBS Lett</em> 593(7): 670-679.</li><li>Del Valle Morales, D., Trotman, J. B., Bundschuh, R. and Schoenberg, D. R. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31996904" target="_blank">Inhibition of cytoplasmic cap methylation identifies 5â TOP mRNAs as recapping targets and reveals recapping sites downstream of native 5â ends.</a> <em>Nucleic Acids Res</em> 48(7): 3806-3815.</li><li>Djebali, S., Davis, C. A., Merkel, A., Dobin, A., Lassmann, T., Mortazavi, A., Tanzer, A., Lagarde, J.Lin, W. and Schlesinger, F.,<em> et al.</em> (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22955620" target="_blank">Landscape of transcription in human cells.</a><em> Nature</em> 489(7414): 101-108.</li><li>Grudzien-Nogalska, E. and Kiledjian, M. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27425147" target="_blank">New insights into decapping enzymes and selective mRNA decay.</a> <em>Wiley Interdiscip Rev RNA</em> 8(1): e1379. <br/></li><li>Kiss, D. L., Oman, K., Bundschuh, R. and Schoenberg, D. R. (2015). <a href="https://pubmed.ncbi.nlm.nih.gov/25541487/" target="_blank">Uncapped 5â ends of mRNAs targeted by cytoplasmic capping map to the vicinity of downstream CAGE tags.</a><em> Febs Lett </em>589(3): 279-284.</li><li>Morioka, M. S., Kawaji, H., Nishiyori-Sueki, H., Murata, M., Kojima-Ishiyama, M., Carninci, P. and Itoh, M. (2020). <a href="https://pubmed.ncbi.nlm.nih.gov/32124327/" target="_blank">Cap Analysis of Gene Expression (CAGE): A Quantitative and Genome-Wide Assay of Transcription Start Sites.</a> <em>Methods Mol Biol</em> 2120: 277-301.</li><li>Otsuka, Y., Kedersha, N. L. and Schoenberg, D. R. (2009). <a href="https://mcb.asm.org/content/29/8/2155" target="_blank">Identification of a cytoplasmic complex that adds a cap onto 5â-monophosphate RNA.</a> <em>Mol. Cell. Biol. </em>29(8), 2155-2167.</li><li>Trotman, J., and Schoenberg, D. R. (2019). <a href="https://pubmed.ncbi.nlm.nih.gov/30252202/" target="_blank">A recap of RNA recapping.</a> <em>Wiley Interdiscip Rev RNA</em> 10(1): e1504.<br/></li></ol>