<body>
            <h1>Understanding Docking Complexes of Macromolecules Using HADDOCK: The Synergy between Experimental Data and Computations</h1>
            <p><strong>Authors:</strong> Andrea Saponaro, Vincenzo Maione, Alexandre M. J. J.  Bonvin, Francesca Cantini</p>
            <h2>Abstract</h2>
            <p>This protocol illustrates the modelling of a protein-peptide complex using the synergic combination of <em>in silico</em> analysis and experimental results. To this end, we use the integrative modelling software HADDOCK, which possesses the powerful ability to incorporate experimental data, such as NMR Chemical Shift Perturbations and biochemical protein-peptide interaction data, as restraints to guide the docking process. Based on the modelling results, a rational mutagenesis approach is used to validate the generated models. The experimental results allow to select a final structural model best representing the <em>bona fide</em> protein-peptide complex. The described protocol can also be applied to model protein-protein complexes. There is no size limit for the macromolecular complexes that can be characterized by HADDOCK as long as the 3D structures of the individual components are available.</p>
            <p><strong>Keywords:</strong> HADDOCK, Molecular docking, Integrative Modelling, Biomolecular interactions, Macromolecular complex, HCN channels, TRIP8b, ITC, NMR, Rational mutagenesis</p>
            <h2>Background</h2><p>Proteins function in most cases not as single molecules, but rather as part of networks by interacting with a variety of other biomolecules. Therefore, the description at atomic level of the macromolecular complexes they form is an essential milestone in the understanding of molecular mechanisms at the basis of physiological and/or pathophysiological processes, and, most importantly, for the rational design of pharmacologically relevant drugs.<br/> <br/>Here, we present a computational protocol to predict the three dimensional (3D) structure of a protein-peptide complex, whose quality is comparable to that of structures derived from conventional experimental methods such as X-ray, NMR and recently cryo-EM.<br/> <br/>The described protocol may overcome several limitations of the experimental methods, such as, for instance, low binding affinities of the protein partners. Indeed, many macromolecular interactions are often weak at physiological conditions, as they mediate transient/highly dynamic processes. There may be also intrinsic limits of the protein sample, which preclude the use of one or more experimental methods.<br/> <br/>In this context, computational approaches offer valuable alternatives, opening the possibility to describe, at atomic level, physiologically relevant macromolecular complexes (Saponaro <em>et al.</em>, 2018) and concomitant conformational rearrangements induced by the interaction (WeiÃgraeber <em>et al.</em>, 2017, GroÃ <em>et al.</em>, 2018; Porro <em>et al.</em>, 2019), otherwise elusive to the conventional structural biology methods.<br/> <br/>Among the docking software used for modelling macromolecular complexes, HADDOCK, an integrative modelling platform, possesses the powerful advantage of being able to implement experimental data directly into the docking process to guide the computations (Dominguez <em>et al.</em>, 2003). Indeed, HADDOCK is specifically designed as a data-driven docking program, although it has two ab-initio docking modes for cases when no information is available.<br/> <br/>Docking calculations in HADDOCK are driven by ambiguous interaction restraints (AIRs). Those are derived from experimental and/or bioinformatics data and are integrated into the docking calculations in order to restrain the conformational search space (Dominguez <em>et al.</em>, 2003) and satisfy the a priori information about the protein-protein interaction. AIRs are defined through a list of residues that fall under two categories: Active and passive. Active residues are defined as the solvent exposed residues directly involved in the interaction between the two proteins (typically defined based on the available experimental data). Passive residues correspond to the solvent exposed ones, which are close to the active residues. Passive residues are introduced in order to account for the fact that often the experimental data are scarce and not fully covering the true binding interface. The main difference between those two types of residues is that active residues are âforcedâ to be at the interface, while âpassiveâ residues can be at the interface, but are not penalized if not.<br/> <br/>A flowchart highlighting the main steps of HADDOCK docking program is shown in Figure 1. The process consists of three successive stages, each of them possessing a specific aim:<br/> 1) <em>Initial docking by rigid body energy minimization (it0)</em><br/>In this initial docking stage, the interacting molecules are treated as rigid bodies. They are separated in space and randomly rotated for each docking trial. The docking step is a rigid body energy minimization during which AIRs, based on the experimental data, are included in the energy function in order to restrict the sampling of the conformational space and guide the docking.<br/> 2) <em>Semi-flexible simulated annealing in torsion angle space (it1)</em><br/>This is a refinement stage during which flexibility is introduced stepwise, first along the side chains and then along both side chains and backbone of the interfacial residues. Those are automatically defined for each model based on their proximity to the partner molecule. The flexible refinement protocol is based on simulated annealing using torsion angles molecular dynamics.<br/> 3) <em>Final refinement in explicit solvent (water)</em><br/>In this final refinement stage the complexes are typically solvated in a layer of water and subjected to a short restrained molecular dynamic in Cartesian space. The final solutions are clustered and the resulting clusters are ranked based on the average score of their top 4 members (4 is the minimum number of models required to define a cluster).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201014/20201014104811_9185.jpg" width="300"/><br/><strong>Figure 1. Flowchart of the HADDOCK protocol</strong><br/><br/>HADDOCK, by default, randomly discards a percentage (50%) of the AIRs for each docking trial to account for possible false positives in the data. The percentage of discarded AIRs can be turned off, or even increased. The latter option is typically performed in the case AIRs derived from bioinformatics interface prediction data.<br/> <br/>This protocol aims at providing a detailed description of the procedure for generating a 3D model structure of a protein-peptide complex using HADDOCK guided by experimental data. We illustrate the protocol by modelling the structure of the complex formed by the Cyclic Nucleotide Binding Domain (CNBD) of the hyperpolarization-activated cyclic nucleotide-gated (HCN) channels bound to the TRIP8b<sub>nano</sub> peptide. This peptide represents the minimal portion of the brain protein TRIP8b, a regulative subunit of HCN channels, that binds to HCN CNBD (Saponaro <em>et al.</em>, 2018). Solution NMR data are used to guide the modelling and the resulting models are then validated following rational mutagenesis/biochemical assays.</p>
        </body>

<p class="pptt" id="biaoti38440">### Equipment</p><ol class="npwtwok"><li>A computer with internet access and a web browser (preferably Chrome, Safari or FireFox)</li><li>A visualization program such as <a href="https://www.cgl.ucsf.edu/chimera/" target="_blank">UCSF</a> Chimera, or <a href="https://pymol.org/2/" target="_blank">PyMOL</a><br/></li></ol><p class="pptt" id="biaoti38441">### Software</p><ol class="npwtwok"><li>HADDOCK 2.2<br/>HADDOCK 2.2 (High Ambiguity Driven protein-protein DOCKing) is freely available to non-profit users via its web interface <a href="https://haddock.science.uu.nl/services/HADDOCK2.2" target="_blank">here</a> (van Zundert <em>et al.</em>, 2016).<br/>Access does however require registration, which is free for non-profit users.</li><li>Program for protein solvent accessibility calculation<br/>The NACCESS program is freely available <a href="http://wolf.bms.umist.ac.uk/naccess/" target="_blank">here</a> for researchers at academic and non-profit-making institutions. NACCESS is used to identify solvent accessible residues. It is worth noting that Naccess program requires Linux/Unix systems. A free alternative to NACCESS is freeSASA, freely available <a href="https://ssbio.readthedocs.io/en/latest/instructions/freesasa.html" target="_blank">here</a>.<br/><br/></li></ol><p class="ppzzs"><strong>Data requirements</strong> </p><ol class="npwtwok"><li>3D coordinates of the two components of the protein complex, preferably in the bound conformation, provided in PDB or mmCIF format.<br/><em>Note: The 3D coordinates in PDB format of HCN2 CNBD and TRIP8b<sub>nano</sub> are available from the following link: (LINK TO THE PDB). Those structures were solved by solution NMR as described in Saponaro </em><em>et al.</em><em>, 2018. For docking purposes, the unfolded regions of both CNBD and TRIP8b<sub>nano</sub> structures, which are not involved in the binding, were removed. This allows to simplify the docking calculation and avoids spurious interactions between the disordered regions of CNBD and TRIP8b<sub>nano</sub>. In the case of TRIP8b<sub>nano</sub> the unfolded C-terminal tail was removed (I272-N275), whereas in the case of CNBD the unfolded N-terminal region was removed (E521-F533).</em> </li><li>Definition of Active and Passive residues<br/>Active residues can be identified by using a large variety of experimental data, including solution state NMR and biochemical protein-protein interaction data, as described in this protocol. Passive residues can be manually assigned by the operator, as performed in this protocol, or automatically defined by HADDOCK. Indeed, the program uses a 6.5 Ã distance cutoff from the heavy atoms of the active residues to define the passive ones. Active and Passive residues are incorporated in the input page as comma-separated list of residue numbers (see Procedure).<br/><br/><em>Note: In the representative case described in this protocol, we define the following active and passive residues for CNBD and TRIP8b<sub>nano</sub> respectively:</em><br/><br/><em><a href="https://os.bio-protocol.org/attached/file/20200830/CNBD.rar" target="_blank">CNBD</a>:</em><br/><em>Active residues: A546, N547, R650, E653, T654, I657, D658, R659, R662, I663, K665.</em><br/><em>Passive residues: P543, D549, P646, R649, G664 and K666. Manually defined by the operator.</em><br/><br/><em>The above residues are defined based on the NMR Chemical Shift Perturbations (CSP) analysis. CSP highlights the residues of the interacting protein whose HN NMR signal features a significant chemical shift perturbation upon interaction with the peptide. A necessary prerequisite to conduct CSP is the availability of the assignment of the NMR backbone signals of the protein of interest.</em><br/><br/><em>   The CSP (ÎHN) is given by the following equation:</em><br/><br/><div style="text-align:center;"><span style="font-size:16px;">ÎHN={((HNfree-HNbound)<sup>2</sup>+(((Nfree-Nbound))â5)<sup>2</sup>)â2)}<sup>1â2</sup></span> </div><div style="text-align:center;"><span style="font-size:16px;"><br/></span> </div><em>where HNfree, Nfree and HNbound, Nbound, are the chemical shift of amide protons (HN) and amide nitrogen (N) in the peptide-free and bound state respectively. In order to select a meaningful set of residues involved in the binding to the peptide, a cutoff which corresponds to the average CSP value plus one standard deviation should be selected.</em><br/><em>  While more residues might be affected by the binding, we assume that only solvent-expose residues will form contacts. Accordingly, the residues identified by CSP need to be filtered based on their solvent accessibility (either main chain or side chain relative accessibility should be typically &gt; 40-50%). Solvent accessibility can be calculated with the program NACCESS. Passive residues are defined as the residues close in space to active residues and with at least 50% of solvent accessibility. The same procedure here reported is also valid for protein-protein docking calculations and can be applied for both interacting proteins. </em><br/><br/><em><a href="https://os.bio-protocol.org/attached/file/20200830/TRIP8bnano.rar" target="_blank">TRIP8b<sub>nano</sub></a>:</em><br/><em>Active residues: we initially define as active residues a solvent exposed (â¥ 50% of solvent accessibility) stretch of four residues (E239, E240, E241, F242, E243) previously identified as crucial for the binding to the CNBD (Santoro et al., 2011).</em><br/><em>  Passive residues: Since TRIP8b<sub>nano</sub> is a small peptide (40 amino acids), all other solvent accessible (â¥ 50% of relative solvent accessibility) residues of the peptide are defined as passive.</em> </li><li>Experimental data (rational mutagenesis and protein-protein interaction assay). Experimental data can be used to validate the HADDOCK output.<br/></li></ol><p class="pptt" id="biaoti38442">### Procedure</p><p class="ppzz">Submitting a HADDOCK calculation:</p><ol class="npwtwok"><li>HADDOCK calculation can be launched from the web-sever available at: <a href="https://haddock.science.uu.nl/services/HADDOCK2.2" target="_blank">https://haddock.science.uu.nl/services/HADDOCK2.2</a>.<br/><em>Note: In the representative case described in this protocol, the âGuru interfaceâ is used to run HADDOCK since it allows to specify/modify more parameters to fine tune the docking settings. Access to this interface does require âguruâ level, which can be requested by users in their own registration page.</em> </li><li>Fill in the HADDOCK input page (Figure 2):<br/><ol class="npwthree"><li>Provide a name for the job.<br/></li><li>Provide the input data, <em>i.e.</em>, the 3D coordinates and the lists of the active and passive residues as a comma-separated list of residue numbers for each components of the complex.<br/></li><li>It is possible also to personalize the docking run by modifying several default parameters, if needed:<br/><ol class="npwfour"><li>Histidine protonation state for each component.<br/></li><li>Semi-Flexible and Fully Flexible segments.<br/></li><li>Additional restraints such as, for example, distance restraints, dihedral and hydrogen bonds restraints, noncrystallographic symmetry restraints, residual dipolar coupling.</li><li>Sampling parameters.<br/></li><li>Parameters for clustering.<br/></li><li>Energy and interaction parameters.<br/></li><li>Scoring parameters.<br/></li><li>Advanced sampling parameters.<br/></li><li>Solvated docking parameters.<br/></li><li>Analysis parameters.<br/></li></ol><br/><img alt="" class="layerphoto" height="245" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201014/20201014111748_4753.jpg" width="300"/><br/><strong>Figure 2. Example view of an input page of the HADDOCK2.2 web-server</strong><br/><br/><em>Note: A detailed description of the various parameters of the HADDOCK scoring function and of the details about the HADDOCK molecular docking process can be found in the online manual </em><a href="http://www.bonvinlab.org/software/haddock2.2/manual" target="_blank"><em>here</em></a><em>.</em> </li><li>By default, HADDOCK generates 1,000 models in the initial rigid body docking. This parameter can be changed by unfolding the tab named âSampling parametersâ (Figure 3).<br/><em>Note: In the specific case of CNBD-TRIP8b<sub>nano</sub> complex the number of structures is increased by a factor of five.</em> </li><li>The default number of structures further subjected to the semi-flexible simulated annealing (it1) and final refinement in water (itw) is 200.<br/><em>Note: In the specific case of CNBD-TRIP8b<sub>nano</sub> complex this parameter is increased by a factor of two (Figure 3, âSampling parametersâ tab).</em> </li><li>The final water-refined models are clustered and ranked based on HADDOCK score value. Clustering is performed based on pairwise Root Mean Square Deviations (RMSD, the default value is set to a 7.5Ã cutoff) and a minimum number of members to define a cluster (default value is set to four) (Figure 3, âParameter for clusteringâ tab). The clusters are then ranked based on the average HADDOCK score of their top four members. The RMSD cut-off value is crucial for a proper clustering of the structures and different clustering cutoffs might be tried.<br/><em>Note: In the specific case of the CNBD-TRIP8b<sub>nano</sub> complex the RMSD cut-off was kept to the default value.</em><br/><br/><img alt="" class="layerphoto" height="176" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201014/20201014111730_8048.jpg" width="300"/><br/><strong>Figure 3. The sampling parameters and the parameters for clustering can be changed by the user in the Guru Interface level</strong><br/><br/></li><li>It is recommended to set the âAdvanced sampling parametersâ (Figure 4) according to the specific protein-peptide docking calculation following the settings reported in Trellet <em>et al.</em>, 2013.<br/><em>Note: In the representative case described in this protocol, we have quadrupled the default number of molecular dynamics integrations steps for the rigid body dynamics at high temperature and the subsequent cooling stages with flexibility at the interface (from 500/500/1,000/1,000 to 2,000/2,000/4,000/4,000) (Figure 4, âAdvanced sampling parametersâ tab). The decision to increase the values is based on the fact that for one of the two component of the complex, TRIP8b<sub>nano</sub>, the number of the active residues was significantly limited (4 residues out of 40 forming TRIP8b<sub>nano</sub>). This can affect the sampling of the conformational space during the rigid body docking calculation phase. Furthermore, because CNBD contains a disordered C-terminal tail, which is necessary for the binding (Saponaro </em><em>et al.</em><em>, 2014), the number of MD steps is also increased to allow the protein to adapt its conformation during the flexible refinement stage.</em><br/><br/><img alt="" class="layerphoto" height="294" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201014/20201014111715_6460.jpg" width="300"/><br/><strong>Figure 4. Advanced sampling parameters used for CNBD-TRIP8b<sub>nano</sub> complex</strong><br/><div><br/></div></li></ol></li><li>Provide both username and password to submit a job.<br/></li></ol><p class="pptt" id="biaoti38443">### Data analysis</p><ol class="npwtwok"><li>Submission data<br/>Once all data have been uploaded and the run submitted, the web-server offers the option to download a parameter file and provides a link to the results page. This link is also e-mailed to the user.</li><li>Analysis of the docking results<br/><ol class="npwthree"><li>The result page of a successful docking run presents the statistics for the top10 clusters ranked by HADDOCK. Those are ranked based on the average HADDOCK score of the top4 member of each cluster, which is a weighted sum of electrostatics, van der Waals, restraints energy, buried surface area and empirical desolvation energy terms (Fernandez-Recio <em>et al.</em>, 2004). The page indicates how many water-refined models could be clustered and displays at the bottom a graphical overview of the results.<br/><em>Note: For the specific case of CNBD-TRIP8b<sub>nano</sub> complex calculation, seventeen clusters are obtained and ranked according to their HADDOCK score.</em><br/></li><li>For each cluster, detailed statistics are displayed, representing the average values calculated over the top four best-scoring structures within each cluster. The clusters require to be also manually screened for the relative orientation of the two components of the complex on the basis of available experimental data.<br/><em>Note: In the specific case of CNBD-TRIP8b<sub>nano</sub>, Double Electron-Electron Resonance (DEER) data are available (DeBerg </em><em>et al.</em><em>, 2015). They provide nanometer-scale distances between unpaired electrons in paramagnetic systems or tagged with paramagnetic probes. DEER data were used to screen the relative orientation of TRIP8b<sub>nano</sub> and CNBD and select two out of the seventeen clusters obtained (Figure 5). The two selected clusters lay within the 30% top ranked clusters. Table 1 summarizes the HADDOCK statistics of the two selected clusters.</em> </li></ol><br/><img alt="" class="layerphoto" height="139" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201014/20201014111656_9478.jpg" width="300"/><br/><strong>Figure 5. Representative poses of cluster families obtained from the first docking calculation.</strong> CNBD is colored in gray; TRIP8b<sub>nano</sub> is colored in light blue in cluster 1 and in green in cluster 2. Residues forming cluster-specific contacts are shown and labeled (modified from Saponaro <em>et al.</em>, 2018 with permission).<br/><br/><strong>Table 1. HADDOCK Score and energetic values for the selected clusters</strong><br/><img alt="" class="layerphoto" height="264" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201015/20201015102206_2435.png" width="700"/><br/><br/></li><li>Validation of the generated models by mutagenesis<br/>Rational mutagenesis studies, coupled with an appropriate protein-peptide assay, can be used to validate a cluster-specific contact and eventually perform a second round of modelling introducing as active those residues whose mutations abolish the interaction.<br/><br/><em>Note: This approach was used for determining the structural model of CNBD-TRIP8b<sub>nano</sub> complex.</em><br/><em>In order to select residues for mutagenesis we examine the intermolecular contacts formed in each of the two selected clusters. This reveals that:</em><br/><ol class="npwthree"><li><em>Residues E264 or E265 of TRIP8b<sub>nano</sub> are found to interact with residues K665 or K666 of CNBD in both clusters (see Figure 5).</em><br/></li><li><em>In cluster 1, TRIP8b<sub>nano</sub> is oriented in such a way that allows E239-E243 residues to interact with R650 of CNBD; whereas, in cluster 2, TRIP8b<sub>nano</sub> is oriented, in almost all the conformers of the cluster, in such a way that CNBD R650 cannot developed contacts with TRIP8b<sub>nano</sub>.</em> </li></ol><em>Rational mutagenesis is performed to validate the above mentioned cluster-specific protein-peptide contacts a posteriori. The effect of the mutations on the complex formation was tested by means of Isothermal Titration Calorimetry (ITC) assay (Saponaro </em><em>et al.</em><em>, 2017 and 2018). Reverse charge mutations K665E or K666E of CNBD, and E264K or E265K of TRIP8b<sub>nano</sub> each significantly reduced the binding affinity between the two proteins, thus indicating that this set of residues are involved in the CNBD - TRIP8b<sub>nano</sub> complex formation. Moreover, the reversion of the charge of R650 (R650E) causes a significant reduction in binding affinity for TRIP8b<sub>nano</sub> (for details see Saponaro </em><em>et al.</em><em>, 2018).</em> </li><li>Refining the model by a second round of docking<br/>On the basis of the experimental observations described above, we performed a second molecular docking calculation including E264 and E265 as additional active residues for TRIP8b<sub>nano</sub>. The docking setup was similar to the first docking calculation described above.<br/>   From this second HADDOCK calculation fourteen clusters are obtained. Four of these are consistent with the DEER and mutagenesis data (Figure 6). In order to validate these models, we performed a second round of mutagenesis based again on identified unique contacts. In particular, in cluster 1 D252 of TRIP8b<sub>nano</sub>, located at the junction between helix N and helix C, contacts residue N547 of the CNBD, while this contact is absent in the other clusters.<br/><br/><img alt="" class="layerphoto" height="273" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201014/20201014111639_2233.jpg" width="300"/><br/><strong>Figure 6. Representative poses of cluster families obtained from the second docking calculation.</strong> CNBD is colored in gray in all the four poses; TRIP8b<sub>nano</sub> is colored in blue in cluster 1, in magenta in cluster 2, in green in cluster 3 and in yellow in cluster 4. D252 of TRIP8b<sub>nano</sub> is shown in all the four poses; N547 of CNBD is shown in the pose representing cluster 1 as in this cluster N547 contacts D252. Helices N and C of TRIP8b<sub>nano</sub> are labeled.<br/><br/>N547 in CNBD was mutated into aspartate (N547D) to generate an electrostatic repulsion with the supposed partner residue D252, while the carboxyl group in D252 of TRIP8b<sub>nano</sub> was removed by mutating it into an asparagine (D252N). Both mutations reduced binding to TRIP8b (for details see Saponaro <em>et al.</em>, 2018). These results suggest that Cluster 1 can be chosen as the <em>bona fide</em> model for the CNBD-TRIP8b<sub>nano</sub> complex.<br/>   It is worth noting that cluster 1 represents also the top-ranking cluster for energetic, scoring function and number of structures populating the cluster (Table 2). This is explained by the fact that the second docking calculation was further implemented with more ambiguous interaction restraints, which significantly improved HADDOCK results.<br/><br/><strong>Table 2. HADDOCK Score and energetic values for the selected clusters</strong><br/><img alt="" class="layerphoto" height="302" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201015/20201015102133_5322.png" width="700"/><br/></li></ol><p class="pptt" id="biaoti38444">### Notes</p><p class="ppzz">Tutorials and video describing HADDOCK program and its use for various scenarios can be found at <a href="http://www.bonvinlab.org/education" target="_blank">http://www.bonvinlab.org/education</a>.</p><p class="pptt" id="biaoti38445">### Acknowledgments</p><p class="ppzz">This protocol was adapted from Saponaro <em>et al.</em> (2018). A.M.J.J.B. acknowledges financial support from the BioExcel CoE (<a href="www.bioexcel.eu" target="_blank">www.bioexcel.eu</a>), a project funded by the European Union Horizon 2020 program under grant agreements 675728 and 823820.</p><p class="pptt" id="biaoti38446">### Competing interests</p><p class="ppzz">There are no conflicts of interest or competing interest.</p><p class="pptt" id="biaoti38447">### References</p><ol class="npwtwok"><li>DeBerg, H. A., Bankston, J. R., Rosenbaum, J. C., Brzovic, P. S., Zagotta, W. N. and Stoll, S. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25800552" target="_blank">Structural mechanism for the regulation of HCN ion channels by the accessory protein TRIP8b.</a> <em>Structure</em> 23(4): 734-744.</li><li>Dominguez, C., Boelens, R. and Bonvin, A. M. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12580598" target="_blank">HADDOCK: a protein-protein docking approach based on biochemical or biophysical information.</a> <em>J Am Chem Soc</em> 125(7): 1731-1737.</li><li>Fernandez-Recio, J., Totrov, M. and Abagyan, R. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14687579" target="_blank">Identification of protein-protein interaction sites from docking energy landscapes.</a> <em>J Mol Biol</em> 335(3): 843-865.</li><li>GroÃ, C., Saponaro, A., Santoro, B., Moroni, A., Thiel, G. and Hamacher, K. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29936413" target="_blank">Mechanical transduction of cytoplasmic-to-transmembrane-domain movements in a hyperpolarization-activated cyclic nucleotideâgated cation channel.</a> <em>J Biol Chem</em> 293(33): 12908-12918.</li><li>Porro, A., Saponaro, A., Gasparri, F., Bauer, D., Gross, C., Pisoni, M., Abbandonato, G., Hamacher, K., Santoro, B., Thiel, G. and Moroni, A. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31769408" target="_blank">The HCN domain couples voltage gating and cAMP response in hyperpolarization-activated cyclic nucleotide-gated channels.</a> <em>Elife</em> 8: e49672.</li><li>Santoro, B., Hu, L., Liu, H., Saponaro, A., Pian, P., Piskorowski, R. A., Moroni, A. and Siegelbaum, S. A. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21411649" target="_blank">TRIP8b regulates HCN1 channel trafficking and gating through two distinct C-terminal interaction sites.</a> <em>J Neurosci</em> 31(11): 4074-4086.</li><li>Saponaro, A. (2018). <a href="https://bio-protocol.org/e2957" target="_blank">Isothermal titration calorimetry: a biophysical method to characterize the interaction between label-free biomolecules in solution.</a> <em>Bio-protocol</em> 8(15): e2957.</li><li>Saponaro, A., Cantini, F., Porro, A., Bucchi, A., DiFrancesco, D., Maione, V., Donadoni, C., Introini, B., Mesirca, P., Mangoni, M. E., Thiel, G., Banci, L., Santoro, B. and Moroni, A. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29923826" target="_blank">A synthetic peptide that prevents cAMP regulation in mammalian hyperpolarization-activated cyclic nucleotide-gated (HCN) channels.</a> <em>Elife</em> 7: 35753. </li><li>Saponaro, A., Pauleta, S. R., Cantini, F., Matzapetakis, M., Hammann, C., Donadoni, C., Hu, L., Thiel, G., Banci, L., Santoro, B. and Moroni, A. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25197093" target="_blank">Structural basis for the mutual antagonism of cAMP and TRIP8b in regulating HCN channel function.</a> <em>Proc Natl Acad Sci U S A</em> 111(40): 14577-14582.</li><li>Saponaro, A., Porro, A., Chaves-Sanjuan, A., Nardini, M., Rauh, O., Thiel, G. and Moroni, A. (2017). Fusicoccin activates KAT1 channels by stabilizing their interaction with 14-3-3 proteins. <em>Plant Cell</em> 29(10): 2570-2580.</li><li>Trellet, M., Melquiond, A. S. and Bonvin, A. M. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23516555" target="_blank">A unified conformational selection and induced fit approach to protein-peptide docking.</a> <em>PLoS One</em> 8(3): e58769.</li><li>van Zundert, G. C. P., Rodrigues, J. P. G. L. M., Trellet, M., Schmitz, C., Kastritis, P. L., Karaca, E., Melquiond, A. S. J., van Dijk, M., de Vries, S. J. and Bonvin, A. M. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26410586" target="_blank">The HADDOCK2.2 Web Server: User-Friendly Integrative Modeling of Biomolecular Complexes.</a> <em>J Mol Biol</em> 428(4):720-725.</li><li>WeiÃgraeber, S, Saponaro, A., Thiel, G. and Hamacher, K. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28074902" target="_blank">A reduced mechanical model for cAMP-modulated gating in HCN channels.</a> <em>Sci Rep</em> 7:40168.<br/></li></ol>