<body>
            <h1>Dual sgRNA-based Targeted Deletion of Large Genomic Regions and Isolation of Heritable <em>Cas9-free</em> Mutants in <em>Arabidopsis</em></h1>
            <p><strong>Authors:</strong> Yu  Jin, Sebastian Marquardt</p>
            <h2>Abstract</h2>
            <p><em>CRISPR/Cas9</em> system directed by a gene-specific single guide RNA (sgRNA) is an effective tool for genome editing such as deletions of few bases in coding genes. However, targeted deletion of larger regions generate loss-of-function alleles that offer a straightforward starting point for functional dissections of genomic loci. We present an easy-to-use strategy including a fast cloning dual-sgRNA vector linked to efficient isolation of heritable <em>Cas9-free</em> genomic deletions to rapidly and cost-effectively generate a targeted heritable genome deletion. This step-by-step protocol includes gRNA design, cloning strategy and mutation detection for <em>Arabidopsis</em> and may be adapted for other plant species.</p>
            <p><strong>Keywords:</strong> <em>CRISPR/Cas9</em>, <em>Cas9-free</em>, dual-sgRNA, gBlock, Genomic deletion</p>
            <h2>Background</h2><p>Dual sgRNA-directed gene knockout by <em>CRISPR/Cas9</em> has been successfully used for genome editing in a variety of organisms (Wang <em>et al.</em>, 2013; Chen <em>et al.</em>, 2014; Char <em>et al.</em>, 2017; Cai <em>et al.</em>, 2018; Durr <em>et al.</em>, 2018; Cui <em>et al.</em>, 2019; Do <em>et al.</em>, 2019; Liu <em>et al.</em>, 2020). Targeted deletions of genomic DNA regions offer a valuable starting point for functional genomics studies (Hilton and Gersbach, 2015; Ford <em>et al.</em>, 2019; Gowthaman <em>et al.</em>, 2020). <em>CRISPR/Cas9</em>-based methods to delete genomic regions benefit from two gRNAs flanking the target DNA regions (Xiao <em>et al.</em>, 2013; Canver <em>et al.</em>, 2014; Kistler <em>et al.</em>, 2015; Song <em>et al.</em>, 2016). In plants, one key bottleneck to perform multiplex gene targeting from a single transformation event is to include multiple gBlocks into one binary vector. A gBlock is composed of a RNA Polymerase III (RNAPIII) promoter, a gene specific sgRNA protospacer, a sgRNA scaffold and a RNAPIII terminator. However, gBlock DNA sequences are usually long and repetitive, rendering design expensive for synthetic DNA and laborious by traditional assembly methods (Gao <em>et al.</em>, 2016; Peterson <em>et al.</em>, 2016; Zhang <em>et al.</em>, 2016; Char <em>et al.</em>, 2017; Durr <em>et al.</em>, 2018; Pauwels <em>et al.</em>, 2018; Schuster, 2018; Wu <em>et al.</em>, 2018; Hui <em>et al.</em>, 2019; Fonseca <em>et al.</em>, 2020). For example, Durr <em>et al.</em> (2018) developed a dual-sgRNA vector by first modifying a pEN-Chimera entry vector to generate two gBlocks, then inserting two gRNAs into the modified entry vector by restriction enzymes, respectively, and finally cloning two gBlocks into the binary vectors. The multiple steps necessary are laborious and time-consuming. Although multiplex <em>CRISPR/Cas9</em> platforms by introducing repetitive gBlocks for targeted genome editing were reported (Ordon <em>et al.</em>, 2016; Schuster, 2018), several studies have noted that transformation of highly repetitive DNA sequences can trigger recombination and silencing of the RNA expression cassettes in a variety of species (Ma and Mitra, 2002; Lovett <em>et al.</em>, 2004; Brake <em>et al.</em>, 2008). To simplify targeted genomic regions and minimize potential recombination in <em>Arabidopsis</em>, we combined and modified existing cloning-based assembly steps. First, we amplified the middle border of our target specific two-gBlocks from a previously developed vector pHEE2E-TRI (Wang <em>et al.</em>, 2015) in a single step. Second, we cloned the middle border into a known <em>CRISPR/Cas9</em> binary vector pKIR1.1 (Maruyama <em>et al.</em>, 2013; Tsutsui and Higashiyama, 2017), allowing two-gBlocks with different Poll III-dependent promoters to reduce the repetitiveness. This vector harbors an RPS5A-Cas9 cassette driving high constitutive expression of Cas9 protein at all developmental stages including egg cells, thus achieves highly efficient mutation in the T1 generation of <em>Arabidopsis</em>. In addition, the expression cassette OLE1-tagRFP in this system shows red fluorescence in seeds, allowing rapid screening for heritable <em>Cas9-free</em> <em>Arabidopsis</em> mutants in the seed of primary transformants. We combined the advantages of the two vectors by a single PCR and a single cloning step, thus providing a simple and reliable protocol to generate stable inherited deletion mutants. Our strategy promises to save cost and time to delete any chromosomal region in <em>Arabidopsis</em>, and can be likely adapted for genome editing of multiple genes simultaneously. It also has the potential to simplify genomic deletion in other plant species.</p>
        </body>

<p class="pptt" id="biaoti38513">## Materials and Reagents</p><ol class="npwone"><li>Consumables<ol class="npwtwo"><li>Sterile pipette tips (Axygen, catalog numbers: TF-300-R-S [10 Âµl], T-350-C-L-R-S [300 Âµl], TF-1000-R-S [1,000 Âµl])</li><li>PCR microtubes (BioExpress, catalog number: <a class="blackNoUnder" target="_blank"> T-3135-2 </a> )</li><li>60 mm x 15 mm round Petri dishes (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 100488-404 </a> )<br/><br/></li></ol></li><li>Competent cells<ol class="npwtwo"><li><em>Escherichia coli</em> HST08 competent cell (homemade, <a href="https://protocols.io/view/preparation-of-chemical-competent-e-coli-cells-w-c-qe4dtgw" target="_blank">protocol 1</a>), store at -80 Â°C</li><li><em>Agrobacterium tumefaciens</em> GV3101 competent cells (homemade, <a href="https://www.protocols.io/view/making-electrocompetent-agrobacterium-tumefaciens-xyqfpvw" target="_blank">protocol 2</a>), store at -80 Â°C<br/><br/></li></ol></li><li>Vectors<br/><ol class="npwtwo"><li>pHEE2E-TRI (Addgene, catalog number: <a class="blackNoUnder" target="_blank"> 71288 </a> ), store at -20 Â°C </li><li>pKIR1.1 (Addgene, catalog number: <a class="blackNoUnder" target="_blank"> 85758 </a> ), store at -20 Â°C</li></ol><em>Note: Sequencs of both vectors can be found in Addgene online.</em><br/><em> pHEE2E-TRI sequence: </em><a href="https://www.addgene.org/71288/sequences/" target="_blank"><em>https://www.addgene.org/71288/sequences/</em></a><br/><em> pKIR1.1 sequence: </em><a href="https://www.addgene.org/85758/sequences/" target="_blank"><em>https://www.addgene.org/85758/sequences/</em></a><br/><br/></li><li>Oligonucleotides 10 ÂµM<br/>Dual-sgRNA1_F: 5â²-CACCTGCATACATTGN20(<sub>protospacer 1</sub>)GTTTTAGAGCTAGAAATAGC-3â²<br/>Dual-sgRNA2_R: 5â²-CACCTGCATACAAACN20(<sub>protospacer 2 reverse complement</sub>)CAATCTCTTAGTCGA CTCTAC-3â²<br/>Mlo 1938: 5â²-TCCCAGGATTAGAATGATTAGG-3â²<br/>Primer_F: 5â²-TTCTCTCTTCGCTCTCTCGTAG-3â²<br/>Primer_R: 5â²-GGCCCAAATACTCTTTTCCAAGAC-3â²<br/>Cas9_F: 5â²-CAGCCGACAAGAAGTACAGC-3â²<br/>Cas9_R: 5â²-ATGGTGGGGTACTTCTCGTG-3â²<br/><br/></li><li>Enzymes and buffers<ol class="npwtwo"><li>AarI (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> ER1581 </a> ), store at -20 Â°C</li><li>T4 DNA Ligase (NEB, catalog number: <a class="blackNoUnder" target="_blank"> M0202L </a> ), store at -20 Â°C</li><li>T4 Polynucleotide Kinase (NEB, catalog number: <a class="blackNoUnder" target="_blank"> M0201L </a> ), store at -20 Â°C</li><li>T4 DNA Ligase Reaction Buffer (10x) (NEB, catalog number: <a class="blackNoUnder" target="_blank"> B0202S </a> ), store at -20 Â°C</li><li>Alkaline Phosphatase, Calf Intestinal (CIP) (NEB, catalog number: <a class="blackNoUnder" target="_blank"> M0290 </a> ), store at -20 Â°C</li><li>Phusion High-Fidelity DNA Polymerase Kits (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M0530S </a> ), store at -20 Â°C </li><li>HotMaster Taq DNA Polymerase (VWR, catalog number: <a class="blackNoUnder" target="_blank"> QUNT2200330 </a> ), store at -20 Â°C</li><li>Wizard<sup>Â®</sup> SV Gel and PCR Clean-Up System (Promega, catalog number: <a class="blackNoUnder" target="_blank"> A9282 </a> ), store at room temperature</li><li>DNA plasmid kit (VWR, catalog number: <a class="blackNoUnder" target="_blank"> D6943-02 </a> ), store at room temperature<br/><br/></li></ol></li><li>Reagents<ol class="npwtwo"><li>MES (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 4432-31-9 </a> ), store at room temperature</li><li>KOH (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 1310-58-3 </a> ), store at room temperature</li><li>Sucrose (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 57-50-1 </a> ), store at room temperature</li><li>Plant agar (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 9002-18-0 </a> ), store at room temperature</li><li>Bacto agar (BD Biosciences, catalog number: <a class="blackNoUnder" target="_blank"> 214030 </a> ), store at room temperature</li><li>Bacto tryptone (BD Biosciences, catalog number: <a class="blackNoUnder" target="_blank"> 211699 </a> ), store at room temperature</li><li>Bacto yeast extract (BD Biosciences, catalog number: <a class="blackNoUnder" target="_blank"> 212730 </a> ), store at room temperature</li><li>Sodium chloride (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 7647-14-5 </a> ), store at room temperature</li><li>Murashige &amp; Skoog medium (Duchefa Biochemie, catalog number: <a class="blackNoUnder" target="_blank"> M524 </a> ), store at 4 Â°C<br/><br/></li></ol></li><li>Antibiotics<ol class="npwtwo"><li>Spectinomycin (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 101454-196 </a> ), store at -20 Â°C</li><li>Rifampicin (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 13292-46-1 </a> ), store in a dry and well-ventilated place</li><li>Gentamycin (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 97062-974 </a> ), store at 4 Â°C</li><li>Kanamycin (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 25389-94-09 </a> ), store at 4 Â°C<br/><br/></li></ol></li><li>Media (see Recipes)<br/><ol class="npwtwo"><li>LB liquid medium + 100 mg/L spectinomycin (store at 4 Â°C for one month)</li><li>LB agar plates + 100 mg/L spectinomycin (store at 4 Â°C for one month)</li><li>LB liquid medium + 100 mg/L spectinomycin + 20 mg/L rifampicin + 25 mg/L gentamycin + 25 mg/L kanamycin (store at 4 Â°C for one month)</li><li>LB agar plates + 100 mg/L spectinomycin + 20 mg/L rifampicin + 25 mg/L gentamycin + 25 mg/L</li><li>Â½ MS-medium plates<br/></li></ol><em>Note: All media need to be autoclaved before adding antibiotics. For plates, 20 ml of media should be used per Petri dish.</em><br/></li></ol><p class="pptt" id="biaoti38514">### Equipment</p><ol class="npwtwok"><li>Pipettes (Thermo Scientific, Finnpipette<sup>TM</sup> F2, catalog numbers: <a class="blackNoUnder" target="_blank"> 4642010 </a> [0.2-2 Âµl], <a class="blackNoUnder" target="_blank"> 4642030 </a> [1-10 Âµl], <a class="blackNoUnder" target="_blank"> 4642060 </a> [2-20 Âµl], <a class="blackNoUnder" target="_blank"> 4642080 </a> [20-200 Âµl], <a class="blackNoUnder" target="_blank"> 4642090 </a> [100-1,000 Âµl])</li><li>Incubator (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 51028132 </a> )</li><li>Shakers (Eppendorf, model: <a class="blackNoUnder" target="_blank"> New Brunswick<sup>TM</sup> Innova<sup>Â®</sup> 44 </a> , catalog number: M1282-0000)</li><li>Stereo Fluorescence Microscope (Leica, model: <a class="blackNoUnder" target="_blank"> M205FA </a> )</li><li>Gel Doc EQ System (Bio-Rad, Universal Hood II, model: <a class="blackNoUnder" target="_blank"> BGDII </a> )</li><li>Heating blocks (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> T1317-1EA </a> )</li><li>ThermoMixer<sup>Â®</sup> C (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 5382000015 </a> )</li><li>PCR Thermo Cycler (Bio-Rad, model: <a class="blackNoUnder" target="_blank"> T100 </a> , catalog number: <a class="blackNoUnder" target="_blank"> 1861096 </a> )</li><li>Tabletop centrifuge (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 75008801 </a> )</li><li>NanoDrop (Thermo Scientific, model: <a class="blackNoUnder" target="_blank"> NanoDrop<sup>TM</sup> 2000C </a> , catalog number: ND2000)</li><li>Agarose gel electrophoresis equipment (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1704489EDU </a> )</li><li>Plant growth chamber (photoperiod: 16âh light/ 8âh dark, temperature: 22 Â°C in the day/20 Â°C in the darkness, humidity: 65%, light intensity: 100âÎ¼Eâm<sup>â2</sup>âs<sup>â1</sup>)<br/></li></ol><p class="ppzzs"><em>Note: No equipment from specific manufacturers is required. Any equivalent device can be used.</em> </p><p class="pptt" id="biaoti38515">### Software</p><ol class="npwtwok"><li>SnapGene<sup>Â®</sup> (SnapGene, <a href="https://www.snapgene.com/" target="_blank">https://www.snapgene.com/</a>)</li><li>Image Lab Software (Bio-Rad, <a href="https://www.bio-rad.com/" target="_blank">https://www.bio-rad.com/</a>)<br/></li></ol><p class="pptt" id="biaoti38516">### Procedure</p><ol class="npwone"><li>Overview<br/>The process that generates dual sgRNA-directed large deletion by <em>CRISPR/Cas9</em> is shown in Figure 1. The vector pKIR1.1 can be ordered from Addgene. A gBlock is composed of a U6 promoter, a gene specific sgRNA protospacer, and a sgRNA scaffold and a terminator.<br/><br/><img alt="" class="layerphoto" height="75" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201018/20201018122911_4417.jpg" width="300"/><br/><strong>Figure 1. Flowchart for isolation of <em>Cas9-free</em> target mutants.</strong> 1. Cloning strategy for generation of two gBlocks. 2. Transformation by floral dipping. 3. Screening T1 seeds by red fluorescence. 4. Genotyping and sequencing T1 plants with red fluorescence to identify candidates and harvest seeds from each individual plant. 5. Isolating <em>Cas9-free</em> T2 seeds without red fluorescence. 6. Genotyping T2 plants to obtain stable and heritable null mutants.<br/><br/></li><li>Generate the middle border of two-gBlocks<ol class="npwtwo"><li>Design two gRNAs to target the same gene of interest (<em>GOI</em>) with CRISPRdirect (Naito <em>et al.</em>, 2015) (Figure 2).<br/><br/><img alt="" class="layerphoto" height="185" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201018/20201018122927_4584.jpg" width="300"/><br/><strong>Figure 2. Overview of dual-sgRNAs cloning strategy.</strong> Schematic representation of the cloning strategy for the introduction of the middle border of two-gBlocks into the pKIR1.1 backbone. pHEE2E-TRI harbors two sets of gBlocks including gBlock1 (a U6-26 promoter, a 19 bp target sequence 1, a sgRNA scaffold, a terminator) and gBlock2 (a U6-29 promoter, a 19 bp target sequence 2, a sgRNA scaffold, a terminator). Therefore, pHEE2E-TRI can serve as a template to amplify the middle border (AarI-overhang1-protospacer1-scaffold-terminator-U6-29 promoter protospacer2-overhang2-AarI) using a pair of dual-sgRNA primers. After digestion of pKIR1.1 plasmid and the middle border both by AarI, the middle border can be integrated into the linearized pKIR1.1 backbone to generate pKIR1.1-dual-sgRNA for plant transformation.<br/><br/><ol class="npwthree"><li>Open the CRISPRdirect webpage (<a href="http://crispr.dbcls.jp/" target="_blank">http://crispr.dbcls.jp/</a>) and paste a target genomic sequence into text field in a FASTA format or a plain nucleotide sequence up to 10 kb.<br/><em>Note: You can also enter an accession number (e.g., NM_001187) or genome location (e.g., hg19: chr7: 900000-901000) to retrieve sequence, or upload a sequence plain text file in a FASTA format or a plain nucleotide sequence up to 10 kb.</em> </li><li>NGG is selected on the âPAM sequence requirementâ panel.</li><li>Select Thale cress (<em>Arabidopsis thaliana</em>) genome, TAIR10 as the organism.</li><li>Click on âdesignâ.</li><li>Select two highlighted protospacer sequence by clicking on âshow highly specific target onlyâ as shown in the screen shot below (Figure 3).<br/></li></ol><br/><img alt="" class="layerphoto" height="226" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201018/20201018122946_4071.jpg" width="300"/><br/><strong>Figure 3. gRNA selection.</strong> Example screenshot for gRNA selection using the CRISPRdirect webpage.<br/><br/><em>Note: One limitation is to avoid presence of an AarI restriction site on your protospacer sequence. In addition, target sequence can be selected from both DNA strands and should be devoid of TTTTs. The distance between the two sgRNAs depends on your expected deletion regions. Small deletions (&lt;100 bp) can be induced with relatively high frequencies and large one (up to 120 kb) with low frequencies (Ordon et al., 2016) in Arabidopsis. The deletion area at 5â² end or 3â end of the non-coding region doesnât show a frequency difference in this sequence from screen shot (NC_003071.7:8122119-8123076 Arabidopsis thaliana chromosome 2 sequence). The 20 nt-protospacer sequence does not have to start with a G, because our linearized plasmids retain a G overhang at the 3â² end of the U6 promoter that serves as the first G preferred for initiating transcription at the U6 promoter.</em> </li><li>Amplify the middle border from the templet pHEE2E-TRI (Figure 2)<br/><ol class="npwthree"><li>Order forward and reverse primers from idtDNA (<a href="https://www.idtdna.com/" target="_blank">https://www.idtdna.com/</a>) and dilute them to a final concentration of 10 Î¼M.<br/>Dual-sgRNA1Â¬_F: 5â²-<u><span style="color:#64451D;">CACCTGC</span></u><span style="color:#64451D;">ATAC<span style="background-color:#FFE500;">ATTG</span></span><span style="color:#00D5FF;">N</span><sub><span style="color:#00D5FF;">20</span></sub><span style="color:#FF9900;">GTTTTAGAGCTAGAAATAGC</span>-3â²<br/>Dual-sgRNA2Â¬_R: 5â²-<u><span style="color:#64451D;">CACCTGC</span></u><span style="color:#64451D;">ATAC</span><span style="color:#64451D;background-color:#FFE500;">AAAC</span><span style="color:#9933E5;">N</span><sub><span style="color:#9933E5;">20</span></sub><span style="color:#009900;">CAATCTCTTAGTCGACTCTAC</span>-3â²<br/>AarI: <u><span style="color:#64451D;">CACCTGC</span></u><br/>20 nt-protospacer 1 sequence: <span style="color:#00D5FF;">N</span><sub><span style="color:#00D5FF;">20</span></sub><br/>20 nt-protospacer 2 reverse complement sequence: <span style="color:#9933E5;">N</span><sub><span style="color:#9933E5;">20</span></sub><br/><span style="background-color:#FFE500;color:#64451D;">Overhang1 (ATTG) and overhang2 (AAAC)</span> in yellow<br/><span style="color:#FF9900;">Template specific forward sequence</span> in orange<br/><span style="color:#009900;">Template specific reverse sequence</span> in green<br/><em>Note: Here we take the first and the last gRNA designs in the screen shot as an example:</em><br/><em>protospacer 1: TAAACGGGACGGCGACTCAC</em><br/><em> protospacer 2: CAGAATCGTCAGATCTGAAG</em><br/><em> protospacer 2 reverse complement sequence: CTTCAGATCTGACGATTCTG</em><br/></li><li>PCR amplification with the following setup:<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>5x HF buffer<br/></td><td>5 Âµl<br/></td></tr><tr><td>dNTP (10 mM)<br/></td><td>1 Âµl<br/></td></tr><tr><td>Dual-sgRNA1_F (10 ÂµM)<br/></td><td>1.25 Âµl<br/></td></tr><tr><td>Dual-sgRNA2_R (10 ÂµM)<br/></td><td>1.25 Âµl<br/></td></tr><tr><td>pHEE2E-TRI (100 ng/Âµl)<br/></td><td>1 Âµl<br/></td></tr><tr><td>Phusion<br/></td><td>0.25 Âµl<br/></td></tr><tr><td>Water<br/></td><td>15.25 Âµl<br/></td></tr><tr><td><strong>total</strong><br/></td><td><strong>25 Âµl</strong><br/></td></tr></tbody></table>PCR program:<br/><img alt="" class="layerphoto" height="178" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201018/20201018120242_9201.png" width="550"/><br/><br/></li><li>Run 5 Âµl PCR product on an agarose gel to check a band with the expected size 620 bp and clean up the rest PCR product using Wizard<sup>Â®</sup> SV Gel and PCR Clean-Up System.<br/><br/></li></ol></li></ol></li><li>Ligate the middle border into pKIR1.1 (Figure 2)<ol class="npwtwo"><li>Digest and phosphorylate PCR product (the middle border)<br/><ol class="npwthree"><li>Digest the purified PCR product with AarI for 6 hrs at 37 Â°C.<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>10x Aarl buffer<br/></td><td>5.0 Âµl<br/></td></tr><tr><td>Middle border<br/></td><td>X Âµl (=1.5 Âµg)<br/></td></tr><tr><td>AarI<br/></td><td>1.5 Âµl<br/></td></tr><tr><td>50x oligo<br/></td><td>1.0 Âµl<br/></td></tr><tr><td>Water<br/></td><td>42.4 â X Âµl<br/></td></tr><tr><td><strong>Total</strong><br/></td><td><strong>50 Âµl</strong><br/></td></tr></tbody></table><em>Note: 50x oligo contains the AarI recognition sequence for stimulating AarI activity (GrigaitÄ et al., 2002). Both 10x Aarl buffer and 50x oligo are supplied with AarI enzyme from Thermo Fisher.</em> </li><li>Phosphorylate the digestion mixture for 30 min at 37 Â°C in the following step, then clean up the mixture using Wizard<sup>Â®</sup> SV Gel and PCR Clean-Up System.<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>Digestion mixture<br/></td><td>25 Âµl<br/></td></tr><tr><td>10x T4 ligase buffer<br/></td><td>2.83 Âµl<br/></td></tr><tr><td>T4 Polynucleotide Kinase <br/></td><td>0.5 Âµl<br/></td></tr></tbody></table></li></ol><em>Note: Skipping the phosphorylation step will lower the ligation efficiency.</em> </li><li>Digest and dephosphorylate pKIR1.1<br/><ol class="npwthree"><li>Digest pKIR1.1 with AarI for 6 h at 37 Â°C.<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>10x Aarl buffer<br/></td><td>5.0 Âµl<br/></td></tr><tr><td>pKIR1.1<br/></td><td>X Âµl (= 1.5 Âµg)<br/></td></tr><tr><td>AarI<br/></td><td>1.5 Âµl<br/></td></tr><tr><td>50x oligo<br/></td><td>1.0 Âµl<br/></td></tr><tr><td>Water<br/></td><td>42.4 â X Âµl<br/></td></tr><tr><td><strong>Total</strong><br/></td><td><strong>50 Âµl</strong><strong></strong><br/></td></tr></tbody></table></li><li>Dephosphorylate the pKIR1.1 digestion mixture by 1 Î¼l of phosphatase (CIP) for 30 min at 37 Â°C and for 10 min at 80 Â°C.<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>Digestion mixture<br/></td><td>50 Âµl<br/></td></tr><tr><td>CutSmart<sup>Â®</sup> Buffer (10x)<br/></td><td>5.67 Âµl<br/></td></tr><tr><td>CIP<br/></td><td>1 Âµl<br/></td></tr></tbody></table></li><li>Perform agarose gel electrophoresis with 5 Âµl of digestion mixture to test the digest. The expected size of digested pKIR1.1 is around 18.5 kb. Clean up the rest digestion mixture using WizardÂ® SV Gel and PCR Clean-Up System.<br/></li></ol><em>Note: We recommend using pKIR1.1 plasmid as negative control. The digested linear plasmid fragment runs more slowly with a larger band size and can therefore be distinguished from negative control.</em> </li><li>Ligate the digested middle border with the linearized pKIR1.1 at 16 Â°C for 30 min to generate the final vector pKIR1.1-dual-sgRNA:<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>AarI-digested pKIR1.1 vector<br/></td><td>X Âµl (50 ng)<br/></td></tr><tr><td>AarI-digested middle border<br/></td><td>X Âµl (5 ng)<br/></td></tr><tr><td>10x T4 ligase buffer<br/></td><td>1.0 Âµl<br/></td></tr><tr><td>Water<br/></td><td>X Âµl<br/></td></tr><tr><td>T4 Ligase<br/></td><td>1.0 Âµl<br/></td></tr><tr><td><strong>Total</strong><br/></td><td><strong>10.0 Âµl</strong><strong></strong><br/></td></tr></tbody></table><em>Note: Here we used vector: insert molar ratio 1:3.</em> </li><li>Transform <em>E. coli</em> competent cells using 10 Âµl of ligation product and spread the transformed cells on LB agar plates with 100 mg/L Spectinomycin, then incubate the selection plates overnight at 37 Â°C (transformation protocol is provided by Clontech: <a href="http://www.takara.co.kr/file/manual/pdf/tr_PT5055-2.pdf" target="_blank">http://www.takara.co.kr/file/manual/pdf/tr_PT5055-2.pdf</a>).</li><li>Pick 8 colonies to verify the correct insertion (Figure 4) by colony PCR using forward primer Mlo 1938 and your own protospacer reverse primer with the following setup:<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>10x HotMaster<sup>TM</sup> Taq Buffer with Mg<sup>2+</sup><br/></td><td>2 Âµl<br/></td></tr><tr><td>dNTP (10 mM)<br/></td><td>0.4 Âµl<br/></td></tr><tr><td>Mlo 1938 (10 ÂµM)<br/></td><td>1 Âµl<br/></td></tr><tr><td>Dual-sgRNA2_R (10 ÂµM)<br/></td><td>1 Âµl<br/></td></tr><tr><td>HotMaster<sup>TM</sup> Taq DNA Polymerase<br/></td><td>0.1 Âµl<br/></td></tr><tr><td>Water<br/></td><td>15.5 Âµl<br/></td></tr></tbody></table>Pick half of a single colony with a sterile pipette tip and swirl in the PCR reaction.<br/>Use the other half for inoculation.<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td><strong>Total</strong><br/></td><td><strong>20 Âµl</strong><br/></td></tr></tbody></table>PCR program:<br/><img alt="" class="layerphoto" height="176" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201018/20201018121642_6260.png" width="510"/><br/><br/><img alt="" class="layerphoto" height="217" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201018/20201018123019_0567.jpg" width="300"/><br/><strong>Figure 4. Electrophoresis of the colony-PCR products.</strong> The positive colonies show the correct insertion size with 868 bp. P.C: positive colony, N.C: negative colony.<br/><br/></li><li>Inoculate the positive colony in LB medium with 100 mg/L Spectinomycin and grow overnight.</li><li>Extract the plasmid using the DNA plasmid kit.</li><li>Sequence the plasmid and verify the sequence of the middle border insertion using primer Mlo 1938.<br/><br/></li></ol></li><li><em>Arabidopsis</em> transformation with pKIR1.1-dual-sgRNA<br/><ol class="npwtwo"><li>Transform competent <em>A. tumefaciens</em> GV3101cells with 1 Î¼g of pKIR1.1-dual-sgRNA and spread the transformed cells on LB agar plates supplemented with 100 mg/L spectinomycin, 20 mg/L rifampicin, 25 mg/L gentamycin and 25 mg/L kanamycin (HÃ¶fgen and Willmitzer, 1988).</li><li>After two days of growth at 30 Â°C, verify plasmid presence in at least three colonies by colony PCR as described before.</li><li>Transform <em>Arabidopsis</em> plants using <em>Agrobacteriu</em><em>m</em>-mediated T-DNA transfer with the floral dipping method (Clough and Bent, 1998).<br/></li></ol><em>Note: Per construct, we usually transform at least ten Arabidopsis plants. We recommend removing of all present siliques before transformation to avoid excessive screening for transformation events afterwards.</em><br/><br/></li><li>Detect dual sgRNA-directed deletion by Cas9<ol class="npwtwo"><li>Place the T1 seeds in a glass plate and pick up red fluorescent seeds with a slightly wet toothpick under a Stereo Fluorescence Microscope using an EL6000 external light source with magnification 10Ã and DsRED filter set (excitation, 546/10ânm; emission, 600/40ânm) (Figure 5).<br/><br/><img alt="" class="layerphoto" height="147" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201018/20201018123040_8436.jpg" width="300"/><br/><strong>Figure 5. Visual screen for T1 seeds that harbor the <em>CRISPR/Cas9</em> construct.</strong> pKIR1.1 vector can express the s.p. Cas9 protein driven by the RPS5A promoter, that allows high constitutive expression at all developmental stages including the germ cells. pKIR1.1 also contains an expression cassette of OLE1âTagRFP (red fluorescent protein) that exhibits red fluorescence in seeds. Therefore, the transformants containing <em>CRISPR/Cas9</em> construct can be observed with red fluorescence in T1 seeds as indicated by circles. Scale bar = 1 mm.<br/><em>Note: The transformation efficiency in this system is around 2.5%, which allows picking up 25 red T1 seeds within 10 min.</em><br/><br/></li><li>Genotype the targeted deletion of T1 plants observed with red fluorescence in seeds by isolating leaf genomic DNA (gDNA) and subsequent PCR (Edwards <em>et al.</em>, 1991) using oligonucleotides flanking the deletion site (Â± 300-500 bp upstream and downstream) (Figure 6):<br/><br/><img alt="" class="layerphoto" height="60" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201018/20201018123113_8559.jpg" width="300"/><br/><strong>Figure 6. Genotyping the dual-sgRNA induced mutation on <em>GOI</em>.</strong> The large deletion of the <em>GOI</em> can be detected by PCR with a pair of primers flanking the deletion site (Primer F and Primer R). A shorter amplicon size can be detected in <em>goi</em> mutant compared to WT. <br/><br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>10x HotMaster<sup>TM</sup> Taq Buffer with Mg<sup>2+</sup><br/></td><td>2 Âµl<br/></td></tr><tr><td>dNTP (10 mM)<br/></td><td>0.4 Âµl<br/></td></tr><tr><td>Primer F (10 ÂµM)<br/></td><td>1 Âµl <br/></td></tr><tr><td>Primer R (10 ÂµM)<br/></td><td>1 Âµl <br/></td></tr><tr><td>Leaf gDNA<br/></td><td>1 Âµl (10 ng)<br/></td></tr><tr><td>HotMaster<sup>TM</sup> Taq DNA Polymerase<br/></td><td>0.1 Âµl<br/></td></tr><tr><td>Water<br/></td><td>14.5 Âµl<br/></td></tr><tr><td><strong>Total</strong><br/></td><td><strong>20 Âµl</strong><strong></strong> </td></tr></tbody></table><br/>PCR program:<br/><img alt="" class="layerphoto" height="176" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201018/20201018122140_1098.png" width="550"/><br/><br/></li></ol></li><li>Isolate <em>Cas9-Free</em> heritable mutation<ol class="npwtwo"><li>Harvest T2 seeds from individual genotyped T1 plants for the second round of observation under the Stereo Fluorescence Microscope to select the desired progeny. T2 seeds that do not contain the <em>CRISPR/Cas9</em> construct can be identified since they lack red fluorescence and isolated.</li><li>Genotype 10 <em>Cas9-free</em> T2 plants by PCR using oligonucleotides flanking the deletion site using phusion polymerase (Figure 7).<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>5x HF buffer<br/></td><td>5 Âµl<br/></td></tr><tr><td>dNTP (10 mM<br/></td><td>1 Âµl<br/></td></tr><tr><td>Primer F (10 ÂµM)<br/></td><td>1.25 Âµl<br/></td></tr><tr><td>Primer R (10 ÂµM)<br/></td><td>1.25 Âµl<br/></td></tr><tr><td>Leaf gDNA<br/></td><td>1 Âµl (10 ng)<br/></td></tr><tr><td>Phusion<br/></td><td>0.25 Âµl<br/></td></tr><tr><td>Water<br/></td><td>15.25 Âµl<br/></td></tr><tr><td><strong>Total</strong><br/></td><td><strong>25 Âµl</strong><strong></strong> </td></tr></tbody></table><br/>PCR program:<br/><img alt="" class="layerphoto" height="176" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201018/20201018122430_6845.png" width="550"/><br/><br/></li><li>Separate the PCR products on an agarose gel and extract the shortened <em>goi</em> band with the correct size from the gel using the Wizard<sup>Â®</sup> SV Gel and PCR Clean-Up System.</li><li>Sequence PCR product with Sanger sequencing and verify the sequence of the <em>goi</em> mutation. Usually heritable deletion events can be identified in plants lacking Cas9 activity after selection of seeds lacking the fluorescent reporter.<br/></li></ol></li></ol><p class="pptt" id="biaoti38517">### Data analysis</p><p class="ppzz">The sequencing data can be analyzed by alignment against the wild-type sequence using software SnapGene. Large deletion is generated in <em>goi</em> mutant of T2 (Figure 7).<br/><br/><img alt="" class="layerphoto" height="153" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201018/20201018123132_9654.jpg" width="300"/><br/><strong>Figure 7. Representative chromatogram of PCR product from <em>goi</em> mutant.</strong> A. Genotyping of individual T1 plants that display red fluorescence. <em>goi-3</em> in red box shows two bands, WT <em>GOI</em> amplicon and <em>goi</em> amplicon. This representative <em>goi</em> mutant with an expected 229 bp deletion is generated by dual-sgRNA targeting at the genomic region Chr2: 8122228-8122453. PCR product with 1 kb in WT represents no deletion of the <em>GOI</em>. B. In the T2 generation, seeds from T1 <em>goi-3</em> are harvested and grown for screening <em>Cas9-free</em> plants. <em>goi3-10</em> in red box (without red fluorescence in seeds) shows only the expected size of the <em>goi</em> amplicon. C. Bulked seedlings of <em>goi-3-10</em> in the T3 generation are genotyped. Top, only the expected <em>goi</em> amplicon is amplified, suggesting a heritable homozygous deletion. Bottom, no amplification of the Cas9-specific band (406 bp) validates fluorescence-based counter selection of the Cas9 transgene. <em>goi-3</em> in the T2 generation (no fluorescence selection) and WT are used as a positive control and negative control, respectively. D. Sequences of the representative deletion fragment of <em>GOI</em> (â229 bp). The alignment is generated with SnapGene. PAM are in blue, sgRNA protospacers are in red, and deleted bases are replaced by a dash in <em>goi3-10</em>.<br/><em>Note: It is mandatory to inspect the sequencing chromatograms carefully and check for the presence of overlapping peaks. Overlapping peaks are a sign of genetic heterogeneity in the sequenced sample.</em> </p><p class="pptt" id="biaoti38518">### Recipes</p><ol class="npwtwok"><li>LB-liquid medium<br/>10 g/L Bacto-tryptone<br/>5 g/L yeast extract<br/>5 g/L NaCl </li><li>LB-agar medium<br/>10 g/L Bacto tryptone <br/>5 g/L Bacto yeast extract<br/>5 g/L Sodium chloride (NaCl)<br/>15 g/L Bacto agar for plates</li><li>Â½ MS-medium plate<br/>4 g/L Murashige &amp; Skoog medium<br/>0.5 g/L MES<br/>10 g/L Sucrose<br/>Adjust pH to 5.7 with KOH<br/>10 g/L plant agar for plates</li></ol><p class="pptt" id="biaoti38519">### Acknowledgments</p><p class="ppzz">This work was funded by the Novo Nordisk Foundation NNF15OC0014202, Copenhagen Plant Science Centre Young Investigator Starting grant and the European Research Council (ERC) under the European Unionâs Horizon 2020 Research and Innovation Programme StG2017-757411 (S.M.).</p><p class="pptt" id="biaoti38520">### Competing interests</p><p class="ppzz">No competing interests.</p><p class="pptt" id="biaoti38521">### References</p><ol class="npwtwok"><li>Brake, T. O., Liu, Y. P., Centlivre, M., Von Eije, K. J., and Berkhout, B. (2008). <a href="https://www.sciencedirect.com/science/article/pii/S1525001616314307?via%3Dihub" target="_blank">Lentiviral vector design for multiple shRNA expression and durable HIV-1 inhibition.</a> <em>Mol Therapy</em> 16(3): 557-564.</li><li>Cai, Y., Chen, L., Sun, S., Wu, C., Yao, W., Jiang, B., Han, T. and Hou, W. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30513774" target="_blank">CRISPR/Cas9-mediated deletion of large genomic fragments in soybean.</a> <em>Int J Mol Sci</em> 19(12).</li><li>Canver, M. C., Bauer, D. E., Dass, A., Yien, Y. Y., Chung, J., Masuda, T., Maeda, T., Paw, B. H. and Orkin, S. H. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24907273" target="_blank">Characterization of genomic deletion efficiency mediated by clustered regularly interspaced short palindromic repeats (CRISPR)/Cas9 nuclease system in mammalian cells.</a> <em>J Biol Chem</em> 289(31): 21312-21324. </li><li>Char, S. N., Neelakandan, A. K., Nahampun, H., Frame, B., Main, M., Spalding, M. H., Becraft, P. W., Meyers, B. C., Walbot, V., Wang, K. and Yang, B. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27510362" target="_blank">An <em>Agrobacterium</em>-delivered CRISPR/Cas9 system for high-frequency targeted mutagenesis in maize.</a> <em>Plant Biotechnol J</em> 15(2): 257-268. </li><li>Chen, X., Xu, F., Zhu, C., Ji, J., Zhou, X., Feng, X. and Guang, S. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25531445" target="_blank">Dual sgRNA-directed gene knockout using CRISPR/Cas9 technology in <em>Caenorhabditis</em> <em>elegans</em><em>.</em></a> <em>Sci Rep</em> 4: 7581. </li><li>Clough, S. J. and Bent, A. F. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10069079" target="_blank">Floral dip: a simplified method for <em>Agrobacterium</em>-mediated transformation of <em>Arabidopsis thaliana</em>.</a> <em>Plant J</em> 16(6): 735-743.</li><li>Cui, X., Balcerzak, M., Schernthaner, J., Babic, V., Datla, R., Brauer, E. K., Labbe, N., Subramaniam, R. and Ouellet, T. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31673276" target="_blank">An optimised CRISPR/Cas9 protocol to create targeted mutations in homoeologous genes and an efficient genotyping protocol to identify edited events in wheat.</a> <em>Plant Methods</em> 15: 119. </li><li>Do, P. T., Nguyen, C. X., Bui, H. T., Tran, L. T. N., Stacey, G., Gillman, J. D., Zhang, Z. J. and Stacey, M. G. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31307375" target="_blank">Demonstration of highly efficient dual gRNA CRISPR/Cas9 editing of the homeologous GmFAD2-1A and GmFAD2-1B genes to yield a high oleic, low linoleic and alpha-linolenic acid phenotype in soybean.</a> <em>BMC Plant Biol</em> 19(1): 311.</li><li>Durr, J., Papareddy, R., Nakajima, K. and Gutierrez-Marcos, J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29535386" target="_blank">Highly efficient heritable targeted deletions of gene clusters and non-coding regulatory regions in <em>Arabidopsis</em> using CRISPR/Cas9.</a> <em>Sci Rep</em> 8(1): 4443. </li><li>Edwards, K., Johnstone, C., and Thompson, C. (1991). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC333874/pdf/nar00242-0175.pdf" target="_blank">A simple and rapid method for the preparation of plant genomic DNA for PCR analysis.</a> <em>Nucleic Acids Res</em> 19(6): 1349.</li><li>Fonseca, J. P., Bonny, A. R., Town, J. and El-Samad, H. (2020). <a href="https://en.bio-protocol.org/e3547" target="_blank">Assembly of Genetic Circuits with the Mammalian ToolKit.</a> <em>Bio-Protocol</em> 10(5): e3547.</li><li>Ford, K., McDonald, D. and Mali, P. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29959923" target="_blank">Functional Genomics via CRISPR-Cas.</a> <em>J Mol Biol</em> 431(1): 48-65.</li><li>Gao, X., Chen, J., Dai, X., Zhang, D. and Zhao, Y. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27208253" target="_blank">An effective strategy for reliably isolating heritable and Cas9-Free <em>Arabidopsis</em> mutants generated by CRISPR/Cas9-mediated genome editing.</a> <em>Plant Physiol</em> 171(3): 1794-1800.</li><li>Gowthaman, U., GarcÃ­a-Pichardo, D., Jin, Y., Schwarz, I., and Marquardt, S. (2020). <a href="https://www.preprints.org/manuscript/202007.0744/v1" target="_blank">DNA processing in the context of non-coding transcription.</a> doi: 10.20944/preprints202007.0744.v1.</li><li>GrigaitÄ, R., ManelienÄ, Z., and Janulaitis, A. (2002). <a href="https://academic.oup.com/nar/article/30/21/e123/1105496" target="_blank">Aar I, a restriction endonuclease from Arthrobacter aurescens SS2-322, which recognizes the novel non-palindromic sequence 5â-CACCTGC (N) 4/8-3.</a> <em>Nucleic Acids Res</em> 30(21): e123-e123.</li><li>Hilton, I. B. and Gersbach, C. A. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26430154" target="_blank">Enabling functional genomics with genome engineering.</a> <em>Genome Res</em> 25(10): 1442-1455. </li><li>HÃ¶fgen, R., and Willmitzer, L. (1988). <a href="https://academic.oup.com/nar/article-abstract/16/20/9877/2378737?redirectedFrom=fulltext" target="_blank">Storage of competent cells for <em>Agrobacterium</em> transformation.</a> <em>Nucleic Acids Res</em> 16(20): 9877.</li><li>Hui, L., Zhao, M., He, J., Hu, Y., Huo, Y., Hao, H., Hao, Y., Zhu, W., Wang, Y. and Xu, M. (2019). <a href="https://link.springer.com/article/10.1007/s11738-019-2961-3" target="_blank">A simple and reliable method for creating PCR-detectable mutants in <em>Arabidopsis</em> with the polycistronic tRNAâgRNA CRISPR/Cas9 system.</a> <em>Acta Physiologiae Plantarum</em> 41(10): 170. </li><li>Kistler, K. E., Vosshall, L. B. and Matthews, B. J. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25818303" target="_blank">Genome engineering with CRISPR-Cas<em>9</em> in the mosquito <em>Aedes aegypti</em>.</a> <em>Cell Rep</em> 11(1): 51-60.</li><li>Liu, K., Sun, B., You, H., Tu, J. L., Yu, X., Zhao, P. and Xu, J. W. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31958883" target="_blank">Dual sgRNA-directed gene deletion in basidiomycete <em>Ganoderma</em> <em>lucidum</em> using the CRISPR/Cas9 system.</a> <em>Microb Biotechnol</em> 13(2): 386-396.</li><li>Lovett, S. T. (2004). <a href="https://onlinelibrary.wiley.com/doi/full/10.1111/j.1365-2958.2004.04076.x" target="_blank">Encoded errors: mutations and rearrangements mediated by misalignment at repetitive DNA sequences.</a> <em>Mol Microbio</em> 52(5): 1243-1253.</li><li>Ma, C. and Mitra, A. (2002). <a href="https://onlinelibrary.wiley.com/doi/full/10.1046/j.1365-313X.2002.01332.x" target="_blank">Intrinsic direct repeats generate consistent post-transcriptional gene silencing in tobacco.</a> <em>Plant J</em> 31 (1): 37-49.</li><li>Maruyama, D., Hamamura, Y., Takeuchi, H., Susaki, D., Nishimaki, M., Kurihara, D., Kasahara, R.D. and Higashiyama, T. (2013). <a href="https://www.sciencedirect.com/science/article/pii/S1534580713001846?via%3Dihub" target="_blank">Independent control by each female gamete prevents the attraction of multiple pollen tubes.</a> <em>Dev Cell</em> 25(3): 317-23.</li><li>Naito, Y., Hino, K., Bono, H. and Ui-Tei, K. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25414360" target="_blank">CRISPRdirect: software for designing CRISPR/Cas guide RNA with reduced off-target sites.</a> <em>Bioinformatics</em> 31(7): 1120-1123. </li><li>Ordon, J., Gantner, J., Kemna, J., Schwalgun, L., Reschke, M., Streubel, J., Boch, J. and Stuttmann, J. (2017). <a href="https://doi.org/10.1111/tpj.13319" target="_blank">Generation of chromosomal deletions in dicotyledonous plants employing a user-friendly genome editing toolkit.</a> <em>Plant J</em> 89:155-168.</li><li>Pauwels, L., De Clercq, R., Goossens, J., Inigo, S., Williams, C., Ron, M., Britt, A. and Goossens, A. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29884615" target="_blank">A Dual sgRNA approach for functional genomics in <em>Arabidopsis</em> <em>thaliana</em>.</a> <em>G3 (Bethesda)</em> 8(8): 2603-2615. </li><li>Peterson, B. A., Haak, D. C., Nishimura, M. T., Teixeira, P. J., James, S. R., Dangl, J. L. and Nimchuk, Z. L. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27622539" target="_blank">Genome-wide assessment of efficiency and specificity in CRISPR/Cas9 mediated multiple site targeting in <em>Arabidopsis</em>.</a> <em>PLoS One</em> 11(9): e0162169.</li><li>Schuster, M., Trippel, C., Happel, P., Lanver, D., Reissmann, S. And Kahmann, R. (2018). <a href="https://bio-protocol.org/e2928" target="_blank">Single and multiplexed gene editing in <em>Ustilago maydis</em> using CRISPR-Cas9.</a> <em>Bio-Protocol</em> 8(14): e2928.</li><li>Song, Y., Yuan, L., Wang, Y., Chen, M., Deng, J., Lv, Q., Sui, T., Li, Z. and Lai, L. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26817461" target="_blank">Efficient dual sgRNA-directed large gene deletion in rabbit with CRISPR/Cas9 system.</a> <em>Cell Mol Life Sci</em> 73(15): 2959-2968.</li><li>Tsutsui, H. and Higashiyama, T. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27856772" target="_blank">pKAMA-ITACHI vectors for highly efficient CRISPR/Cas9-mediated gene knockout in <em>Arabidopsis thaliana</em>.</a> <em>Plant Cell Physiol</em> 58(1): 46-56.</li><li>Wang, H., Yang, H., Shivalila, C. S., Dawlaty, M. M., Cheng, A. W., Zhang, F. and Jaenisch, R. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23643243" target="_blank">One-step generation of mice carrying mutations in multiple genes by CRISPR/Cas-mediated genome engineering.</a> <em>Cell</em> 153(4): 910-918.</li><li>Wang, Z. P., Xing, H. L., Dong, L., Zhang, H. Y., Han, C. Y., Wang, X. C. and Chen, Q. J. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26193878" target="_blank">Egg cell-specific promoter-controlled CRISPR/Cas9 efficiently generates homozygous mutants for multiple target genes in <em>Arabidopsis</em> in a single generation.</a> <em>Genome Biol</em> 16: 144. </li><li>Wu, R., Lucke, M., Jang, Y. T., Zhu, W., Symeonidi, E., Wang, C., Fitz, J., Xi, W., Schwab, R. and Weigel, D. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30083222" target="_blank">An efficient CRISPR vector toolbox for engineering large deletions in <em>Arabidopsis thaliana</em>.</a> <em>Plant Methods</em> 14: 65. </li><li>Xiao, A., Wang, Z., Hu, Y., Wu, Y., Luo, Z., Yang, Z., Zu, Y., Li, W., Huang, P., Tong, X., Zhu, Z., Lin, S. and Zhang, B. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23748566" target="_blank">Chromosomal deletions and inversions mediated by TALENs and CRISPR/Cas in zebrafish.</a> <em>Nucleic Acids Res</em> 41(14): e141. </li><li>Zhang, Z., Mao, Y., Ha, S., Liu, W., Botella, J. R. and Zhu, J. K. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26661595" target="_blank">A multiplex CRISPR/Cas9 platform for fast and efficient editing of multiple genes in <em>Arabidopsis</em>.</a> <em>Plant Cell Rep</em> 35(7): 1519-1533. <br/></li></ol>