<body>
            <h1>Quantification of Salivary Charged Metabolites Using Capillary Electrophoresis Time-of-flight-mass Spectrometry</h1>
            <p><strong>Authors:</strong> Masahiro Sugimoto, Sana Ota, Miku Kaneko, Ayame Enomoto, Tomoyoshi Soga</p>
            <h2>Abstract</h2>
            <p>Salivary metabolomics have provided the potentials to detect both oral and systemic diseases. Capillary electrophoresis time-of-flight-mass spectrometry (CE-TOFMS) enables the identification and quantification of various charged metabolites. This method has been employed to biomarker discoveries using human saliva samples, especially for various types of cancers. The untargeted analysis contributes to finding new biomarkers. <em>i.e.</em>, the analysis of all detectable signals including both known and unknown metabolites extends the coverage of metabolite to be observed. However, the observed data includes thousands of peaks. Besides, non-linear migration time fluctuation and skewed peaks are caused by the sample condition. The presented pretreatment protocols of saliva samples enhance the reproducibility of migration time drift, which facilitates the matching peaks across the samples and also results in reproducible absolute concentrations of the detected metabolites. The described protocols are utilized not only for saliva but for any liquid samples with slight modifications.</p>
            <p><strong>Keywords:</strong> Metabolomics, Capillary electrophoresis-mass spectrometry, Saliva, Polyamine, Cancer</p>
            <h2>Background</h2><p>Saliva is one of the biofluids suitable for monitoring the systemic conditions. The non-invasive availability of saliva samples enables frequent, timely, and cost-effective diagnostics, which would contribute to realizing personalized medicine. Therefore biomarker discoveries for various diseases have been reported (Wang <em>et al.</em>, 2017). Saliva contains microorganisms and also a wide variety of components, such as genome, coding and non-coding RNA, proteins, and metabolites (Bonne and Wong, 2012; Yoshizawa <em>et al.</em>, 2013). Traditionally, the profiling of these components has been utilized for the diagnostic of oral diseases (Dawes and Wong, 2019; Martina <em>et al.</em>, 2020). Recently, accumulated evidence has revealed the potential to monitor systematic health conditions (Sugimoto <em>et al.</em>, 2013; Kaczor-Urbanowicz <em>et al.</em>, 2017).<br/><br/>Among various omics technologies, metabolomics has been utilized to discover the biomarkers for metabolic diseases, including cancers (Trezzi <em>et al.</em>, 2015). The metabolic aberrance in cancer cells, such as the Warburg effect (Warburg, 1956), potentially reflects metabolite profiles in biofluids. Metabolomics-based liquid biopsy is therefore intensively developed (Armitage and Ciborowski, 2017). In comparison to the blood and urine-based diagnostics, the use of saliva is an emerging approach while salivary metabolic profiles showed the potential to detect various cancers (Sugimoto <em>et al.</em>, 2010a).<br/><br/>The salivary diagnostics of oral cancer are the most reported among various cancers (Washio and Takahashi, 2016; Chattopadhyay and Panda, 2019). We previously confirmed the consistently elevated metabolites in saliva and oral cancer tissues (Ishikawa <em>et al.</em>, 2016). The effect of saliva collection after various fasting duration on these markers was also evaluated (Ishikawa <em>et al.</em>, 2017). The evaluation of the specificity of oral cancer-specific markers against various diseases in the oral cavity, <em>e.g.</em>, periodontal diseases, is also conducted (Mikkonen <em>et al.</em>, 2016). Discrimination of oral leukoplakia, oral lichen planus, and oral cancer is practically useful in clinical settings (Sridharan <em>et al.</em>, 2019; Ishikawa <em>et al.</em>, 2020). The storage condition of the saliva samples affected the discrimination abilities of metabolite markers (Wang <em>et al.</em>, 2014), which requires the standard of the protocol to deal with the saliva samples for realizing reproducible diagnostics.<br/><br/>Recently, salivary metabolite biomarkers to diagnose the cancers far from the oral cavity have been reported. The quantified polyamines in saliva were utilized for the detection of breast cancers (Takayama <em>et al.</em>, 2016; Murata <em>et al.</em>, 2019) using liquid chromatography-mass spectrometry (LC-MS). These analyses were conducted using triple-quadrupole-MS (MS) for targeted analyses. We utilized capillary electrophoresis time-of-flight-MS (CE-TOFMS) to conduct untargeted analyses to quantify hundreds of salivary metabolites for pancreatic cancer detections (Asai <em>et al.</em>, 2018). MS requires the ionization of metabolites to be detected and therefore various separation system, detection, and their interface are available (Monton and Soga, 2007). Among these systems, CE-MS is one of a powerful tool that can analyze the charged metabolites using only two modes, cation and anion, <em>i.e.</em>, positively and negatively charged metabolites (Soga <em>et al.</em>, 2003; Soga, 2007). Here, the metabolite profiling protocol for saliva samples using CE-TOFMS is described.</p>
        </body>

<p class="pptt" id="biaoti38470">## Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips 100 Î¼l, 10 Î¼l (Eppendorf, catalog numbers: 0030.073.428, 0030 073.363)</li><li>1.5 ml reaction tube (Greiner-Bio-One, catalog number: <a class="blackNoUnder" target="_blank"> 616201 </a> )</li><li>Nanosep Centrifugal Devices With Omega Membrane 3K, Gray (PALL, catalog number: <a class="blackNoUnder" target="_blank"> OD003C34 </a> )</li><li>Polypropylene vial (Agilent, catalog number: <a class="blackNoUnder" target="_blank"> 5190-3155 </a> )</li><li>Snap Cap (Agilent, catalog number: <a class="blackNoUnder" target="_blank"> 5042-6491 </a> )</li><li>Nitrile gloves (TOP, catalog number: <a class="blackNoUnder" target="_blank"> 15731 </a> )</li><li>2 ml Crimp/Snap Top Vials &amp; Caps (Agilent, catalog number: <a class="blackNoUnder" target="_blank"> 5182-9697 </a> )</li><li>Fused Silica Capillary (Polymicro Technologies, catalog number: <a class="blackNoUnder" target="_blank"> TSP050375 </a> )</li><li>COSMO(+)Capillary (NACALAI TESQUE, catalog number: <a class="blackNoUnder" target="_blank"> 07584-44 </a> )</li><li>L-Methionine sulfone, 99+% (Thermo Fisher Scientific Inc, catalog number: <a class="blackNoUnder" target="_blank"> A17027 </a> )</li><li>3-Aminopyrrolidine dihydrochloride (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 404624 </a> )</li><li>2-(N-Morpholino)ethanesulfonic Acid (MES) (FUJIFILM Wako Pure Chemical, catalog number: <a class="blackNoUnder" target="_blank"> 341-01622 </a> )</li><li>D-Camphor-10-sulfonic Acid Sodium Salt (CSA) (FUJIFILM Wako Pure Chemical, catalog number: <a class="blackNoUnder" target="_blank"> 037-01032 </a> )</li><li>1,3,5-Benzenetricarboxylic Acid (Trimesate) (FUJIFILM Wako Pure Chemical, catalog number: <a class="blackNoUnder" target="_blank"> 206-03641 </a> )</li><li>Spermidine (Merk/Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S2626 </a> )</li><li>Spermine (FUJIFILM Wako Pure Chemical, catalog number: <a class="blackNoUnder" target="_blank"> 194-09813 </a> )</li><li>N1-Acetylspermine (FUJIFILM Wako Pure Chemical, catalog number: <a class="blackNoUnder" target="_blank"> 014-20421 </a> )</li><li>N1-Acetylspermidine (Sigma-Aldrich Japan, 01467)</li><li>N8-Acetylspermidine (Fluka, catalog number: <a class="blackNoUnder" target="_blank"> A3658 </a> )</li><li>DL-a-Amino-n-butyric acid (2AB) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> A1754 </a> )</li><li>Methanol (FUJIFILM Wako Pure Chemical, catalog number: <a class="blackNoUnder" target="_blank"> 134-14523 </a> )</li><li>Ammonium Acetate (FUJIFILM Wako Pure Chemical, catalog number: <a class="blackNoUnder" target="_blank"> 019-02835 </a> )</li><li>Hexakis(2,2-difluoroethoxy)phosphazene (Synquest Laboratories, catalog number: <a class="blackNoUnder" target="_blank"> 8H79-3-02 </a> )</li><li>Formic Acid Formate (FUJIFILM Wako Pure Chemical, catalog number: <a class="blackNoUnder" target="_blank"> 063-05895 </a> )</li><li>Ammonium Formate (Kanto Kagaku, catalog number: <a class="blackNoUnder" target="_blank"> 01294-00 </a> )</li><li>Acetic Acid (FUJIFILM Wako Pure Chemical, catalog number: <a class="blackNoUnder" target="_blank"> 012-00245 </a> )</li><li>Milli-Q water (MERCK, Tokyo, Japan, Milli-Q IQ 7000 ICP-MS)</li><li>Internal standards mixture (see Recipes)</li><li>Sheath liquid (see Recipes)</li><li>Run buffer (see Recipes)</li><li>Standard mixture (see Recipes)</li></ol><p class="pptt" id="biaoti38471">### Equipment</p><ol class="npwtwok"><li>Measuring flask 5 ml (IWAKI, catalog number: <a class="blackNoUnder" target="_blank"> 5640FK5S </a> )</li><li>Laboratory Bottles 100 ml (SCHOTT/DURAN, catalog number: <a class="blackNoUnder" target="_blank"> 017200-100A </a> )</li><li>Pipettes 100 Î¼l, and 10 Î¼l (Eppendorf, catalog numbers: 3121 000.074, 3121 000.015)</li><li>Centrifuge (TOMY, model: <a class="blackNoUnder" target="_blank"> MX-307 </a> )</li><li>Roter (TOMY, model: <a class="blackNoUnder" target="_blank"> AR015-24 </a> )</li><li>MicroMixer (TAITEC, model: <a class="blackNoUnder" target="_blank"> E-36 </a> )</li><li>CE system (Agilent Technologies, model: <a class="blackNoUnder" target="_blank"> 7100 </a> )</li><li>LC/MSD TOF system (Agilent, catalog number: <a class="blackNoUnder" target="_blank"> G1969A </a> )</li><li>Isocratic HPLC pump (Agilent, model: <a class="blackNoUnder" target="_blank"> 1260 series </a> )</li><li>CEMS adapter kit (Agilent, catalog number: <a class="blackNoUnder" target="_blank"> G1603A </a> )</li><li>CE-ESI-MS sprayer kit (Agilent, catalog number: <a class="blackNoUnder" target="_blank"> G1607A </a> )</li><li>Circulation type hand cooler (THOMAS, catalog number: <a class="blackNoUnder" target="_blank"> 2241059 </a> )<br/></li></ol><p class="pptt" id="biaoti38472">### Software</p><ol class="npwtwok"><li>Agilent Chemstation software was used for CE, ver. B.02.01.SR1 (Agilent Technologies, Santa Clara, CA, USA)</li><li>Agilent MassHunter software, ver. B.02.01.SR1 (Agilent) for TOF-MS data analyses</li><li>R, ver. 3.4.3 (R Foundation for Statistical Computing, Vienna, Austria)</li><li>JMP, ver. 13.2.0 (SAS Institute Inc., Cary, NC, USA)</li><li>WEKA, ver. 3.6.13 (The University of Waikato, Hamilton, New Sealand)</li><li>GraphPad Prism, ver 7.03 (GraphPad Software Inc., San Diego, CA, USA)</li><li>MasterHands, ver 2.17.3.18 (Keio University, Tsuruoka, Japan)<br/></li></ol><p class="pptt" id="biaoti38473">### Procedure</p><p class="ppzz">Prepare all solutions and mixtures at room temperature. All sample processing is conducted using Nitrile gloves. Anaerobic condition is not necessary. <br/></p><ol class="npwone"><li>Saliva collection<ol class="npwtwo"><li>Subjects with a primary disease without a history of prior malignancy.</li><li>Subjects who did not receive any prior treatment in the form of chemotherapy, radiotherapy, surgery, or alternative therapy.</li><li>The saliva providers were not allowed to take any food except water intake after 9:00 p.m. on the previous day.</li><li>All samples were collected from 8:00 to 11:00 a.m.</li><li>The subjects were required to brush their teeth without toothpaste on the day of saliva collection and had to refrain from drinking water, smoking, tooth-brushing, and intense exercise 1 h before saliva collection.</li><li>They were required to gargle with water just before saliva collection.</li><li>Approximately 400 Î¼l of unstimulated saliva was collected in a 50 cc polypropylene tube (see Note 1).</li><li>A polypropylene straw 1.1 cm in diameter was used to assist the saliva collection.</li><li>After collection, saliva samples were immediately stored at -80 Â°C.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Processing of saliva samples.<br/>The overview of saliva processing is depicted in Figure 1.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201014/20201014020124_9704.jpg" width="300"/><br/><strong>Figure 1. A protocol of saliva processing</strong><br/><br/><ol class="npwtwo"><li>A frozen saliva sample is thawed at 4 Â°C.</li><li>100 Î¼l of the saliva sample is moved to a 1.5-ml tube with a 5-kDa cutoff filter using a pipet.</li><li>The tube is centrifuged at 9,100 <em>Ã g</em> for 3 h at 4 Â°C to eliminate large molecules.</li><li>If the filtrate is not less than 45 Î¼l (Figure 2), the rest of the raw saliva samples should be processed independently (Steps B2 and B3) and the filtrate should be merged.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201014/20201014020320_4563.jpg" width="300"/><br/><strong>Figure 2. A tube with a filter.</strong> The filtrate of 45 Î¼l is shown.<br/><br/></li><li>45 Î¼l of the filtrate is moved to a 1.5 ml of microtube using a pipet.</li><li>5 Î¼l of water containing an internal standards mixture (Recipe 1) was added to the tube using a pipet, mixed 30 s using vortex, and centrifuged at 4,550 <em>Ã g</em> for 1 min at 4 Â°C to mix well.</li><li>7 Î¼l of the mixture is transferred to a vial and the vial is covered with a snap cap.</li></ol><br/></li><li>Setup for the cation measurement<ol class="npwtwo"><li>CE<ol class="npwthree"><li>Capillary: Fused-silica, i.d. 50 Î¼m Ã 100 cm</li><li>Buffer: 1 M Formic acid</li><li>Voltage: Positive, 30 kV</li><li>Temperature: 20 Â°C</li><li>Injection: Pressure injection 50 mbar, 5 s (approximately 5 nl)</li></ol></li><li>TOFMS<ol class="npwthree"><li>Ion Source: Electrospray ionization (ESI)</li><li>Polarity: Positive</li><li>Capillary voltage: 4,000 V</li><li>Fragmentor: 75 V</li><li>Skimmer: 50 V</li><li>OCT RFV: 500 V</li><li>Drying gas: N<sub>2</sub>, 10 L/min</li><li>Drying gas temp.: 300 Â°C</li><li>Nebulizer gas press.: 7 psig</li><li>Sheath liquid: Cation sheath liquid (Recipe 2)</li><li>Flow rate: 10 Î¼l/min</li><li>Lock mass: 2MeOH<sub>13</sub>Cisotope <em>m/z</em> 66.063061 and Hexakis(2,2-difluoroethoxy)phosphazene <em>m/z</em> 622.028963</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Setup for the anion measurement<ol class="npwtwo"><li>CE<ol class="npwthree"><li>Capillary: COSMO(+), i.d. 50 Î¼m Ã 105 cm</li><li>Buffer: 50 mM Ammonium acetate, pH 8.5</li><li>Voltage: Negative, 30 kV</li><li>Temperature: 20 Â°C</li><li>Injection: Pressure injection 50 mbar, 30 s (approximately 30 nl)</li><li>Preconditioning: 2 min at 50 mM Ammonium acetate, pH 3.4 and 5 min at run buffer</li></ol></li><li>TOFMS<ol class="npwthree"><li>Ion Source: ESI</li><li>Polarity: Negative</li><li>Capillary voltage: 3,500 V</li><li>Fragmentor: 100 V</li><li>Skimmer: 50 V</li><li>OCT RFV: 200 V</li><li>Drying gas: N<sub>2</sub>, 10 L/min</li><li>Drying gas temp.: 300 Â°C</li><li>Nebulizer gas press.: 7 psig</li><li>Sheath liquid: Anion sheath liquid (see Recipes)</li><li>Flow rate: 10 Î¼l/min</li><li>Lock mass: 2CH<sub>3</sub>COOH<sup>13</sup>C isotope <em>m/z</em> 120.038339, Hexakis(2,2-difluoroethoxy)phosphazene+CH<sub>3</sub>COOH <em>m/z</em> 680.035541</li><li>ESI needle: Platinum</li></ol></li><li>Measurement<ol class="npwthree"><li>Sample preconditioning<ol class="npwfour"><li>Cation for standard mixtures: 4 min at a cation run buffer (see Recipes).</li><li>Cation for samples: 5 min at Ammonium Formate, 5 min at Milli-Q, and 5 min at an anion run buffer (Recipe 3).</li><li>Anion for standard mixtures and samples: 2 min at 50 mM Ammonium acetate, pH 3.4, and 5 min at a cation run buffer (Recipe 3).</li></ol></li><li>Three samples and a standard mixture are measured for accessing the quality.</li><li>Evaluate the stability of the total ion electrophoresis (TIE) (Figures 3 and 4). If the TIE is unstable, adjust the ESI sprayer until the intensities of the TIE becomes steady.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201014/20201014020350_6340.jpg" width="300"/><br/><strong>Figure 3. An example of the stable TIE of the cation measurement.</strong> The X-axis indicates the migration time. The Y-axis indicates the intensities of TIE, the sum of intensities over the <em>m/z</em> range in the observed data. Under the successful conditioning, this line becomes flat except for a few parts. Two significant drops of the TIE are observed at the left because of the existence of the inorganic cations, such as Na<sup>+</sup> and K<sup>+</sup>. Two large peaks are observed of the existence of neutral molecules. The other small alternations are negligible and the flat of the other part indicates the stability of the conditioning.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201014/20201014020459_8461.jpg" width="300"/><br/><strong>Figure 4. An example of the unstable TIE of the cation measurement.</strong> Compared to the TIE line in Figure 3, the line here fluctuates, which indicate the unsuccessful conditioning. Most of the cases, the adjustment of ESI sprayer is not suitable and the electric current fluctuates, which causes such TIE fluctuations.<br/><br/></li><li>Evaluate the stability of electric current during the measurement (Figures 5 and 6). Dilute the final solution and measure the sample again if the TIE is unstable.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201014/20201014020542_3300.jpg" width="300"/><br/><strong>Figure 5. An example of the stable electric current of the cation measurement.</strong> The X-axis indicates the migration time. The Y-axis indicates the electric current during the measurement. Under the successful measurement, this line becomes flat except for the start of the measurement.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201014/20201014020556_9505.jpg" width="300"/><br/><strong>Figure 6. An example of the unstable electric current of the cation measurement.</strong> The X-axis indicates the migration time. The Y-axis indicates the electric current during the measurement. This example shows many spike peaks. When the condition of capillary becomes abnormal, the line doesn't stay flat.<br/><br/></li><li>Measure saliva samples.</li></ol></li></ol></li></ol><p class="pptt" id="biaoti38474">### Data analysis</p><p class="ppzz">Data analysis includes data processing and statistical analysis (Figure 7). The data processing includes data conversion and peak integration for each raw data. The process data were aligned and a data matrix is yielded after filling missing peaks (Sugimoto <em>et al.</em>, 2012). This matrix is used for the subsequent data analysis, <em>i.e.</em>, statistical analysis.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201014/20201014020617_0698.jpg" width="300"/><br/><strong>Figure 7. Overview of data processing and data analysis<br/><br/></strong> </p><ol class="npwtwo"><li>For each sample, a data folder (.d extension) is yielded by MassHunter, a vendor-supplied software for CE-MS controls. Data folders of the standard mixture, quality control, and saliva samples are prepared.</li><li>Each data folder is converted to another format file (.kiff extension) using a data conversion function of MasterHands (Sugimoto <em>et al.</em>, 2012)</li><li>Subsequent data processing will be conducted using MasterHands</li><li>Detect and integrate peaks with default options of all data files; signal/noise (S/N) radio &gt; 2.0 for peak detection and 50% of the depth between two peaks for peak separation (see Note 2).</li><li>Run alignment function to correct and matching of the peaks across all samples is conducted with default options. If the alignment function is not stable (Figure 8), rerun the alignment with the different gap penalty option (Sugimoto <em>et al.</em>, 2010b).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201014/20201014020639_7390.jpg" width="300"/><br/><strong>Figure 8. Examples of alignment functions. </strong>Stable (left) and unstable (right) cases.<br/><br/></li><li>Evaluate the overlaid peaks for internal standards. If the overlaid peaks are not observed, rerun the alignment with the different options (Figure 9).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201014/20201014020701_4262.jpg" width="300"/><br/><strong>Figure 9. Examples of overlaid peaks.</strong> Successful (left) and unsuccessful (right) results.<br/><br/></li><li>Run reintegration function to filling missing peaks based on the aligned peak matrix.</li><li>Exchange the peak area less than the lower limit of quantification to 0.</li><li>The peak area of all peaks will be generated and these peak areas are divided by that of internal standard (methionine sulfone for cation and CSA for anion).</li><li>Based on the corrected migration times and <em>m/z</em> values among standard mixture and samples, the metabolite names will be assigned.</li><li>Export the peak matrix as a CSV file.</li><li>Calculate the absolute concentration of the salivary metabolite concentrations.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201014/20201014021108_3030.jpg" width="600"/><br/>where, M, IS, STD, PA, V indicates metabolite, internal standard, standard, peak area, and volume, respectively.</li></ol><br/><strong> Statistical analysis</strong> <ol class="npwtwo"><li>Conduct principal component analysis (PCA) using JMP to visualize the overall similarity and discrepancy of the metabolite concentration patterns among salivary samples (see Note 3).</li><li>Conduct the MannâWhitney test using R to access the difference of metabolite concentrations between 2 groups. The R function of wilcox.test() with correct = F options is used (see Note 4).</li><li>Correct <em>P</em>-values using R using false discovery rate (Benjamini and Hochberg methods), considering multiple independent tests. The adjust() method of R vector with method = âfdrâ options is used.</li><li>Select only metabolite showing corrected <em>P</em> &lt; 0.05 and fold change (F.C.) &gt; 4.0 of the averaged concentrations between pancreatic patients (PC) and non-PC groups (healthy controls with chronic pancreatitis).</li><li>Select the minimum metabolite set for MLR using JMP. Select the option of mixed selection (the combination of backward and forward selections with <em>P</em> &lt; 0.05) to eliminate multicollinearity. Select [analysis]-[fit model menu] on JMP, and set stepwise options for personality. Select the â<em>P</em>-value Thresholdâ for stopping rule and select âmixedâ for the direction. Input 0.05 for the threshold and press the âGoâ button.</li><li>Develop a multiple logistic regression (MLR) model to discriminate patients from non-PC and calculate an area under the receiver operating characteristic curves (AUC). Press the âRun modelâ button.</li><li>Evaluate the generalization ability of the MLR model using Weka. Run <em>k</em>-fold cross-validations (CVs) to evaluate the generalization ability of the developed MLR. Repeat a CV test with 200 random values and <em>k</em> = 5, 10, and 20 (totally, 600 tests). Run weka.classifiers.functions.Logistic class of weka package with âx k (k is replaced to numerical values).</li><li>Correct AUC values to calculate its mean and 95% confidential intervals. Select [Analyze Data]-[ROC Curve] menu using GraphPad.</li></ol><p class="pptt" id="biaoti38475">### Notes</p><ol class="npwtwok"><li>The use of polypropylene should be evaluated beforehand. Using the presented protocol, we confirmed that no effect of eluted compounds on the quantification quality of the saliva metabolites. However, this confirmation is limited to our standard library. Various condition of storage and solvent was compared (Tomita <em>et al.</em>, 2018). Based on the observed fluctuation of these data, we computationally produced noise on the quantification level of the salivary metabolite and evaluated the effect of salivary biomarkers (Asai <em>et al.</em>, 2018) on their discrimination abilities. Based on these data, standardization of operating procedures should be defined.</li><li>Calibration ranges for each metabolite should be defined using a standard mixture before the analysis of saliva samples. In our protocol, we confirmed that the metabolites in our standard library showed high linearity between the peak area and the concentration up to 1 mM. Based on these data, the upper quantification range was defined 500 M, and the corresponding peak area was calculated beforehand. The peak areas showing larger than this threshold are considered as saturated peaks. The peaks showing S/N &lt; 1.5 are considered as lower quantification limit.</li><li>For PCA analyses, only frequently observed metabolite data should be used. We used metabolites detected at least 50% per group (Asai <em>et al.</em>, 2018), while this threshold depends on the number of samples and the number of metabolites.</li><li>The comparison of multiple groups depends on the study design. The salivary biomarker discovery study to detect pancreatic cancer included control (C), pancreatic cancer (PC), and chronic pancreatitis (CP) (Asai <em>et al.</em>, 2018). Usually, a Kruskal-Wallis test should be used for the comparison of three groups. Here, we aimed to find metabolite showing high concentration only in a PC group. Therefore, we treated C and CP as one group and used a Mann-Whitney test to compare PC vs (C + CP) groups.<br/></li></ol><p class="pptt" id="biaoti38476">### Recipes</p><ol class="npwtwok"><li>Internal standards mixture<br/>2 mM of Methionine sulfone<br/>2 mM of MES<br/>2 mM of CSA<br/>2 mM of 3-Aminopyrrolidine<br/>2 mM of Trimesate</li><li>Sheath liquid<br/>Cation: Methanol/water (50% v/v) containing 0.1 Î¼M Hexakis (2,2-difluorothoxy) phosphazene<br/>Anion: Ammonium acetate (5 mmol/L) in 50% methanol/water (50% v/v) containing 0.1 Î¼M Hexakis (2,2-difluorothoxy) phosphazene</li><li>Run buffer<br/>Cation: 1 M of Formic acid<br/>Anion: 50 mM Ammonium acetate (pH 3.4 and pH 8.5)</li><li>Standard mixture<br/>Cation: 200 Î¼M of internal standards and 20 Î¼M of each metabolite<br/>Anion: 200 Î¼M of internal standards and 50 Î¼M of each metabolite<br/></li></ol><p class="pptt" id="biaoti38477">### Acknowledgments</p><p class="ppzz">This work was supported by a grant from the Program on Open Innovation Platform with Enterprises, Research Institute and Academia (OPERA, JPMJOP1842), and grants from Tsuruoka City and Yamagata Prefecture.</p><p class="pptt" id="biaoti38478">### Competing interests</p><p class="ppzz">There are no competing interests to be declared.</p><p class="pptt" id="biaoti38479">### Ethics</p><p class="ppzz">This study was approved by the ethics committee of Tokyo Medical University (approval no. 1560, 30 September 2010). Written informed consent was obtained from all patients and from volunteers who agreed to serve as saliva donors. Our study was carried out following the Helsinki Declaration.</p><p class="pptt" id="biaoti38480">### References</p><ol class="npwtwok"><li>Armitage, E. G. and Ciborowski, M. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28132182" target="_blank">Applications of metabolomics in cancer studies.</a> <em>Adv Exp Med Biol</em> 965: 209-234. </li><li>Asai, Y., Itoi, T., Sugimoto, M., Sofuni, A., Tsuchiya, T., Tanaka, R., Tonozuka, R., Honjo, M., Mukai, S., Fujita, M., Yamamoto, K., Matsunami, Y., Kurosawa, T., Nagakawa, Y., Kaneko, M., Ota, S., Kawachi, S., Shimazu, M., Soga, T., Tomita, M. and Sunamura, M. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29401744" target="_blank">Elevated polyamines in saliva of pancreatic cancer.</a> <em>Cancers (Basel)</em> 10(2). </li><li>Bonne, N. J. and Wong, D. T. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23114182" target="_blank">Salivary biomarker development using genomic, proteomic and metabolomic approaches.</a> <em>Genome Med</em> 4(10): 82.</li><li>Chattopadhyay, I. and Panda, M. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31109866" target="_blank">Recent trends of saliva omics biomarkers for the diagnosis and treatment of oral cancer.</a> <em>J Oral Biosci</em> 61(2): 84-94.</li><li>Dawes, C. and Wong, D. T. W. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30782091" target="_blank">Role of saliva and salivary diagnostics in the advancement of oral health.</a> <em>J Dent Res</em> 98(2): 133-141. </li><li>Ishikawa, S., Sugimoto, M., Edamatsu, K., Sugano, A., Kitabatake, K. and Iino, M. (2020). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31602722" target="_blank">Discrimination of oral squamous cell carcinoma from oral lichen planus by salivary metabolomics.</a> <em>Oral Dis</em> 26(1): 35-42. </li><li>Ishikawa, S., Sugimoto, M., Kitabatake, K., Sugano, A., Nakamura, M., Kaneko, M., Ota, S., Hiwatari, K., Enomoto, A., Soga, T., Tomita, M. and Iino, M. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27539254" target="_blank">Identification of salivary metabolomic biomarkers for oral cancer screening.</a> <em>Sci Rep</em> 6: 31520.</li><li>Ishikawa, S., Sugimoto, M., Kitabatake, K., Tu, M., Sugano, A., Yamamori, I., Iba, A., Yusa, K., Kaneko, M., Ota, S., Hiwatari, K., Enomoto, A., Masaru, T. and Iino, M. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28101653" target="_blank">Effect of timing of collection of salivary metabolomic biomarkers on oral cancer detection.</a> <em>Amino Acids</em> 49(4): 761-770.</li><li>Kaczor-Urbanowicz, K. E., Martin Carreras-Presas, C., Aro, K., Tu, M., Garcia-Godoy, F. and Wong, D. T. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27903834" target="_blank">Saliva diagnostics - Current views and directions.</a> <em>Exp Biol Med (Maywood)</em> 242(5): 459-472.</li><li>Martina, E., Campanati, A., Diotallevi, F. and Offidani, A. (2020). <a href="https://www.ncbi.nlm.nih.gov/pubmed/32046271" target="_blank">Saliva and oral diseases.</a> <em>J Clin Med</em> 9(2). </li><li>Mikkonen, J. J., Singh, S. P., Herrala, M., Lappalainen, R., Myllymaa, S. and Kullaa, A. M. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26446036" target="_blank">Salivary metabolomics in the diagnosis of oral cancer and periodontal diseases.</a> <em>J Periodontal Res</em> 51(4): 431-437. </li><li>Monton, M., R., and Soga, T. (2007). <a href="https://pubmed.ncbi.nlm.nih.gov/17376458/" target="_blank">Metabolome analysis by capillary electrophoresis-mass spectrometry.</a> <em>J Chromatogr A</em> 168(1-2): 237-246.</li><li>Murata, T., Yanagisawa, T., Kurihara, T., Kaneko, M., Ota, S., Enomoto, A., Tomita, M., Sugimoto, M., Sunamura, M., Hayashida, T., Kitagawa, Y. and Jinno, H. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31286302" target="_blank">Salivary metabolomics with alternative decision tree-based machine learning methods for breast cancer discrimination.</a> <em>Breast Cancer Res Treat</em> 177(3): 591-601. </li><li>Soga, T., Ohashi, Y., Ueno, Y., Naraoka, H., Tomita, M., and Nishioka, T. (2003). <a href="https://pubmed.ncbi.nlm.nih.gov/14582645/" target="_blank">Quantitative metabolome analysis using capillary electrophoresis mass spectrometry.</a> <em>J Proteome Res</em> 2(5): 488-494.</li><li>Soga, T. (2007). <a href="https://pubmed.ncbi.nlm.nih.gov/17035684/" target="_blank">Capillary electrophoresis-mass spectrometry for metabolomics.</a> <em>Methods Mol Biol</em> 358:129-137.</li><li>Sridharan, G., Ramani, P., Patankar, S. and Vijayaraghavan, R. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30714209" target="_blank">Evaluation of salivary metabolomics in oral leukoplakia and oral squamous cell carcinoma.</a> <em>J Oral Pathol Med</em> 48(4): 299-306.</li><li>Sugimoto, M., Wong, D. T., Hirayama, A., Soga, T. and Tomita, M. (2010a). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20300169" target="_blank">Capillary electrophoresis mass spectrometry-based saliva metabolomics identified oral, breast and pancreatic cancer-specific profiles.</a> <em>Metabolomics</em> 6(1): 78-95.</li><li>Sugimoto, M., Kawakami, M., Robert, M., Soga, T., Tomita, and M. (2012). <a href="https://pubmed.ncbi.nlm.nih.gov/22438836/" target="_blank">Bioinformatics tools for mass spectroscopy-based metabolomic data processing and analysis.</a> <em>Curr Bioinform</em> 7(1): 96-108.</li><li>Sugimoto, M., Saruta, J., Matsuki, C., To, M., Onuma, H., Kaneko, M., Soga, T., Tomita, M. and Tsukinoki, K. (2013). <a href="https://link.springer.com/article/10.1007/s11306-012-0464-y" target="_blank">Physiological and environmental parameters associated with mass spectrometry-based salivary metabolomic profiles.</a> <em>Metabolomics</em> 9(2): 454-463. </li><li>Sugimoto, M., Hirayama, A., Ishikawa, T., Robert, M., Baran, R., Uehara, K., Kawai, K., Soga, T. and Tomita, M. (2010b). <a href="https://link.springer.com/article/10.1007/s11306-009-0175-1" target="_blank">Differential metabolomics software for capillary electrophoresis-mass spectrometry data analysis.</a> <em>Metabolomics</em> 6(1): 27-41. </li><li>Takayama, T., Tsutsui, H., Shimizu, I., Toyama, T., Yoshimoto, N., Endo, Y., Inoue, K., Todoroki, K., Min, J. Z., Mizuno, H. and Toyo'oka, T. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26523874" target="_blank">Diagnostic approach to breast cancer patients based on target metabolomics in saliva by liquid chromatography with tandem mass spectrometry.</a> <em>Clin Chim Acta</em> 452: 18-26. </li><li>Tomita, A., Mori, M., Hiwatari, K., Yamaguchi, E., Itoi, T., Sunamura, M., Soga, T., Tomita, M,, Sugimoto, M. (2018). <a href="https://pubmed.ncbi.nlm.nih.gov/30104641/" target="_blank">Effect of storage conditions on salivary polyamines quantified via liquid chromatography-mass spectrometry.</a> <em>Sci Rep</em> 8(1):12075.</li><li>Trezzi, J. P., Vlassis, N. and Hiller, K. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26530359" target="_blank">The role of metabolomics in the study of cancer biomarkers and in the development of diagnostic tools.</a> <em>Adv Exp Med Biol</em> 867: 41-57.</li><li>Wang, Q., Gao, P., Wang, X. and Duan, Y. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24144867" target="_blank">Investigation and identification of potential biomarkers in human saliva for the early diagnosis of oral squamous cell carcinoma.</a> <em>Clin Chim Acta</em> 427: 79-85. </li><li>Wang, X., Kaczor-Urbanowicz, K. E. and Wong, D. T. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27943101" target="_blank">Salivary biomarkers in cancer detection.</a> <em>Med Oncol</em> 34(1): 7. </li><li>Warburg, O. (1956). <a href="https://www.ncbi.nlm.nih.gov/pubmed/13351639" target="_blank">On respiratory impairment in cancer cells.</a> <em>Science</em> 124(3215): 269-270.</li><li>Washio, J. and Takahashi, N. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27271597" target="_blank">Metabolomic studies of oral biofilm, oral cancer, and beyond.</a> <em>Int J Mol Sci</em> 17(6). </li><li>Yoshizawa, J. M., Schafer, C. A., Schafer, J. J., Farrell, J. J., Paster, B. J. and Wong, D. T. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24092855" target="_blank">Salivary biomarkers: toward future clinical and diagnostic utilities.</a> <em>Clin Microbiol Rev</em> 26(4): 781-791.<br/></li></ol>