<body>
            <h1>Estimation of the Minimum Number of Replication Origins Per Chromosome in any Organism</h1>
            <p><strong>Authors:</strong> Marcelo S. da Silva</p>
            <h2>Abstract</h2>
            <p>Eukaryote nuclear genomes predominantly replicate through multiple replication origins. The number of replication origins activated per chromosome during the S-phase duration may vary according to many factors, but the predominant one is replication stress. Several studies have applied different approaches to estimate the number and map the positions of the replication origins in various organisms. However, without a parameter to restrict the minimum of necessary origins, less sensitive techniques may suggest conflicting results. The estimation of the <u>m</u>inimum number of replication <u>o</u>rigins (MO) per chromosome is an innovative method that allows the establishment of a threshold, which serves as a parameter for genomic approaches that map origins. For this, the MO can be easily obtained through a formula that requires as parameters: chromosome size, S-phase duration, and replication rate. The chromosome size for any organism can be acquired in genomic databanks (such as NCBI), the S-phase duration can be estimated by monitoring DNA replication, and the replication rate is obtained through the DNA combing approach. The estimation of MO is a simple, quick, and easy method that provides a new methodological framework to assist studies of mapping replication origins in any organism.</p>
            <p><strong>Keywords:</strong> DNA replication, Replication origins, Replication rate, S-phase duration, Chromosome size</p>
            <h2>Background</h2><p>For all living organisms, DNA replication is a key and highly regulated process of paramount importance for biological inheritance. The first step of DNA replication is the establishment of the genomic loci, where DNA synthesis begins. These loci are called replication origins (or just origins) (MÃ©chali, 2010). In general, the beginning of DNA synthesis occurs after an initiator binds to an origin and recruits specific proteins that will result in the formation of the replisome in a process called origin firing. Each origin fired generates two replication forks that move in opposite directions. Replication forks synthesize DNA at a velocity (rate) that varies according to the organism and cell lineage (Myllykallio <em>et al.</em>, 2000; Stanojcic <em>et al.</em>, 2016; da Silva <em>et al.</em>, 2017). The time needed for all chromosomes to replicate determines the S-phase duration, which seems to be robust for certain cell types and organisms (Zhang <em>et al.</em>, 2017; da Silva <em>et al.</em>, 2020).<br/> <br/>Although prokaryotes from the Bacteria domain usually replicate their genome using only one origin, the vast majority of organisms (Archaea and Eukarya domains, in general) replicate their chromosomes from multiple origins (Leonard and MÃ©chali, 2013). However, the exact number of origins fired per chromosome can vary according to cell type and the cellular environment (da Silva <em>et al.</em>, 2020). In a recent study (da Silva <em>et al.</em>, 2019), I developed a formula capable of estimating the <u>m</u>inimum number of <u>o</u>rigins (MO) required to duplicate an entire chromosome within the S-phase duration. The principle of the formula is the bidirectional movement of the replication forks. Moreover, the S-phase duration, the size of the chromosome in question, and the average replication rate are required as parameters of the equation.<br/> <br/>The estimation of MO per chromosome is an innovative method that allows the establishment of a threshold, which serves as a parameter to assist (or validate) genomic approaches that map origins, such as marker frequency analysis coupled to next-generation sequencing (MFA-seq), small leading nascent strand purification coupled to next-generation sequencing (SNS-seq), DNA microarray, and DNA combing. Under standard conditions, MO values show minimal variation since the variables used in the formula are, in general, stable. The MO formula is universal and can be applied for any organism, even procaryotes. In the analyzes presented here, the trypanosomatid parasite <em>Trypanosoma brucei</em> was used as a model. In this organism, the protocol typically takes 2-3 days from the estimation of the variables to results. However, if the necessary variables are available (<em>i.e.</em>, they can be obtained from other studies), the estimation of MO per chromosome can be carried out immediately.</p>
        </body>

<p class="pptt" id="biaoti38524">## Materials and Reagents</p><ol class="npwtwok"><li>Microtubes 1.5 ml (Axygen, Maxyclear, catalog number: <a class="blackNoUnder" target="_blank"> MCT-150-C-S </a> )</li><li>Centrifuge tubes 15 ml (Corning, catalog number: <a class="blackNoUnder" target="_blank"> CLS430791 </a> )</li><li>Culture flasks 25 cm<sup>2</sup> (Corning, canted neck, cap plug seal, catalog number: <a class="blackNoUnder" target="_blank"> CLS430168 </a> )</li><li>Syringe filter 0.22 Âµm (Sartori, Minisart Syringe filter, catalog number: <a class="blackNoUnder" target="_blank"> 16534 </a> )</li><li>Micropipette tips (Axygen, 10 Âµl, 200 Âµl and 1,000 Âµl)</li><li>Serological pipettes (Costar Sterile, 10 ml)</li><li>Microscope Slides (Knitell glass, non-color)</li><li>Coverslips (Knitell glass, 22 x 22 mm)</li><li>Combicoverslips (Genomic vision, catalog number: <a class="blackNoUnder" target="_blank"> COV-002-RUO </a> )</li><li>Reusable plug modes (from Molecular combing systemâGenomic vision)</li><li>6-well flat-bottom plate (Costar, catalog number: <a class="blackNoUnder" target="_blank"> 38015 </a> )</li><li>Plastic coverslips (use a cut plastic pocket for binder)</li><li>Nail varnish (any brand, preferably colorless)</li><li>Click-iT EdU Cell Proliferation Kit, Alexa Fluor 488 dye (ThermoFisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> C10337 </a> )</li><li>5â²-Iodo-2â-deoxyuridine (IdU) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> I7125 </a> )</li><li>5â²-Chloro-2â²-deoxyuridine (CldU) (Abcam, catalog number: <a class="blackNoUnder" target="_blank"> ab213715 </a> )</li><li>Mouse Î±-BrdU/Î±-IdU monoclonal antibody (BD, catalog number: <a class="blackNoUnder" target="_blank"> 347580 </a> )</li><li>Rat Î±-BrdU/Î±-CldU monoclonal antibody (Accurate, catalog number: <a class="blackNoUnder" target="_blank"> YSRTMCA2060GA </a> )</li><li>Goat Î±-mouse IgG1 secondary antibody, Alexa Fluor 568 (ThermoFisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A-21124 </a> )</li><li>Goat Î±-rat IgG (H+L) secondary antibody, Alexa Fluor 488 (ThermoFisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A-11006 </a> )</li><li>Paraformaldehyde (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 158127 </a> )</li><li>Poly-L-lysine hydrochloride (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P2658 </a> )</li><li>Bovine serum albumin (Sigma-Aldrich, catalog number: 0<a class="blackNoUnder" target="_blank"> 5470 </a> )</li><li>Triton X-100 (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> T8787 </a> )</li><li>NaCl (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S9888 </a> )</li><li>Na<sub>2</sub>HPO<sub>4</sub> (Sigma Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 255793 </a> )</li><li>Ethanol absolute (Merck Millipore, catalog number: <a class="blackNoUnder" target="_blank"> 100983 </a> )</li><li>Vectashield Mounting Medium with DAPI (Vector Labs, catalog number: <a class="blackNoUnder" target="_blank"> H-1200 </a> )</li><li>ProLong Gold Antifade Mountant (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> P36930 </a> )</li><li>Hydrochloric acid fuming 37% (Merck, catalog number: <a class="blackNoUnder" target="_blank"> 100317 </a> )</li><li>Sodium hydroxide (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 221465 </a> )</li><li>SDM-79 medium (LGC Biotecnologia, catalog number: <a class="blackNoUnder" target="_blank"> BR30079-05 </a> )</li><li>Hemin (Merck, catalog number: <a class="blackNoUnder" target="_blank"> H9039 </a> )</li><li>KCl (for molecular biology, any brand)</li><li>KH<sub>2</sub>PO<sub>4</sub> (for molecular biology, any brand)</li><li>Fetal Bovine Serum (FBS) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> F7524 </a> )</li><li>Streptomycin sulfate salt (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S6501 </a> )</li><li>Penicillin G sodium salt (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P3032 </a> )</li><li>EDTAÂ·2H<sub>2</sub>O (Ethylenediaminetetraacetic acid disodium salt dihydrate) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> E5134 </a> )</li><li>NaOH (ACS reagent, any brand)</li><li>Ethanol (Absolute, Merck, catalog number: <a class="blackNoUnder" target="_blank"> 100983 </a> )</li><li>N-Laurylsarcosine sodium salt (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> L5125 </a> )</li><li>Proteinase K (ThermoFischer Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM2544 </a> )</li><li>Anti-DNA Antibody, single stranded (mouse anti-ssDNA) (Millipore, catalog number: <a class="blackNoUnder" target="_blank"> MAB3868 </a> )</li><li>Goat anti-Mouse IgG2b Cross-Adsorbed Secondary Antibody, Alexa Fluor 350 (anti-mouse Alexa Fluor 350) (ThermoFischer Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A-21140 </a> )</li><li>MES, free acid (ULTROL grade, Calbiochem, catalog number: <a class="blackNoUnder" target="_blank"> 475893 </a> )</li><li>Phosphate buffered saline (1x PBS) (see Recipes)</li><li>Hemin Solution (HS) (see Recipes)</li><li>SDM-79 medium (for cultivate <em>T. brucei</em> procyclic cells) (see Recipesï¼</li><li>5â²-chloro-2â²-deoxyuridine solution (CldU-S) (see Recipes)</li><li>5-ethynyl-2â-deoxyuridine solution (EdU-S) (see Recipes)</li><li>5â²-iodo-2â²-deoxyuridine solution (IdU-S) (see Recipes)</li><li>0.5 M EDTA (see Recipes)</li><li>0.65 M EDTA (see Recipes)</li><li>1.5 M Tris-HCl (see Recipes)</li><li>1 M NaCl (see Recipes)</li><li>70% ethanol (see Recipes)</li><li>90% ethanol (see Recipes)</li><li>DNA combing washing buffer (DC-WB) (see Recipes)</li><li>DNA combing lysis buffer (DC-LB) (see Recipes)</li><li>DNA combing blocking solution (DC-BS) (see Recipes)</li><li>DNA combing denaturation buffer (DC-DB) (see Recipes)</li><li>DNA combing primary antibodies solution (DC-PAS) (see Recipes)</li><li>DNA combing secondary antibodies solution (DC-SAS) (see Recipes)</li><li>DNA combing anti-ssDNA solution (DC-anti-ssDNA) (see Recipes)</li><li>DNA combing Alexafluor 350 (DC-alexa350) (see Recipes)</li><li>T<sub>10</sub>E<sub>1</sub> buffer (see Recipes)</li><li>0.5 M MES buffer (see Recipes)</li><li>Fixation buffer (FB) (see Recipes)</li><li>Poly-L-lysine solution (PLS) (see Recipes)</li><li>Permeabilization solution (PS) (see Recipes)<br/></li></ol><p class="pptt" id="biaoti38525">### Equipment</p><ol class="npwtwok"><li>Microcentrifuge (Eppendorf, model: <a class="blackNoUnder" target="_blank"> 5424 R </a> )</li><li>Motorized pipet dispenser (Fisher Scientific, Fisherbrand, catalog number: <a class="blackNoUnder" target="_blank"> 03-692-172 </a> )</li><li>Water bath (Cientec, model: <a class="blackNoUnder" target="_blank"> CT-226 </a> )</li><li>Magnetic stirrer (Fisatom, model: <a class="blackNoUnder" target="_blank"> 753A </a> )</li><li>Incubator BOD (Vitrex, model: <a class="blackNoUnder" target="_blank"> NI1705 </a> )</li><li>Fluorescence Microscope [Olympus, model: <a class="blackNoUnder" target="_blank"> BX51 </a> , coupled to an XM10 digital camera. Filters specifications: U-MWU2 (excitation = 330-385 nm; emission = 420 nm), U-MWIBA3 (excitation = 460-495 nm; emission = 510-550 nm), and U-MWG2 (excitation = 510-550 nm; emission = 590 nm)]</li><li>Micropipettes (Gilson, models: Pipetman P10, P20, P200, and P1000)</li><li>Centrifuge (Eppendorf, model: <a class="blackNoUnder" target="_blank"> 5810 R </a> ), equipped with 4 x 250 ml Swing-Bucket Rotor</li><li>Neubauer chamber with cover glass (Sigma-Aldrich, model: <a class="blackNoUnder" target="_blank"> Bright-Line<sup>TM</sup> Hemacytometer </a> )</li><li>Biosafety Class II A2 cabinet (Pachane, model: <a class="blackNoUnder" target="_blank"> PA 700 </a> )</li><li>pH meter (Gehaka, model: <a class="blackNoUnder" target="_blank"> PG1800 </a> )</li><li>Autoclave (Tomy Seiko, model: <a class="blackNoUnder" target="_blank"> SS-245 </a> )</li><li>FiberCombâMolecular combing system (Genomic Vision, model: <a class="blackNoUnder" target="_blank"> MCS-001 </a> )<br/></li></ol><p class="pptt" id="biaoti38526">### Software</p><ol class="npwtwok"><li>CeCyD (Butantan Institute, published in da Silva <em>et al.</em>, 2020, <a href="http://cecyd.vital.butantan.gov.br/" target="_blank">http://cecyd.vital.butantan.gov.br/</a>)</li><li>Doubling Time Calculator (2006) (version 3.1.0, <a href="https://doubling-time.com/compute_more.php" target="_blank">https://doubling-time.com/compute_more.php</a>)</li><li>Olympus Cell F software (Olympus, version 5.1.2640)</li><li>ImageJ (NIH, version 1.47t)</li><li>Microsoft Excel (Microsoft Office-any version) or GraphPad Prism (GraphPad software, Inc.)</li></ol><p class="pptt" id="biaoti38527">### Procedure</p><p class="ppzzs"><em>Notes:</em> </p><ol class="npwtwok"><li><em>If you already have the values for chromosomes size, S-phase duration, and replication rate for the organism in question, the following steps (A-C) are not necessary.</em> </li><li><em>The description of the steps B (estimation of the S-phase) and C (estimation of the replication rate) uses as example the organism </em><em>T. brucei</em><em>. However, for other cell types these parameters can be found in specific studies when available [e.g., S. cerevisiae (Brewer </em><em>et al.</em><em>, 1984; Sekedat </em><em>et al.</em><em>, 2010; Ivanova </em><em>et al.</em><em>, 2020); mammalian cellsâMEFs cell (Ishida </em><em>et al.</em><em>, 2001; Stanojcic </em><em>et al.</em><em>, 2016; Pereira </em><em>et al.</em><em>, 2017)].</em><br/><br/></li></ol><ol class="npwone"><li>Obtaining values for chromosome size<br/>First option (for any organism):<br/><ol class="npwtwo"><li>Access the website <a href="https://www.ncbi.nlm.nih.gov/" target="_blank">https://www.ncbi.nlm.nih.gov/</a>.<br/></li><li>In âpopular resourcesâ option (right side), click on genome.<br/></li><li>Type the name of the organism of interest.<br/></li><li>Obtain the values of chromosome size from the table.</li></ol>Second option (only for trypanosomatids organisms):<br/><ol class="npwtwo"><li>Access the website https://tritrypdb.org/tritrypdb/<a href="https://tritrypdb.org/tritrypdb/" target="_blank">https://tritrypdb.org/tritrypdb/</a>.</li><li>In âsearch for other data typesâ option (middle column), click on âgenomic sequencesâ, then 'organismâ.</li><li>Choose the organism of interest (<em>e.g.</em>, <em>Trypanosoma brucei</em>).</li><li>Click on âGet Answerâ.</li><li>Obtain the values of chromosome size (length) from the table.<br/><br/></li></ol></li><li>Estimation of the S-phase duration using CeCyD software<br/>CeCyD is a user-friendly website able to calculate the values of cytokinesis (C), mitosis (M), G2, S, and G1 phases of the cell cycle, for any organism. For this, the user must enter the following parameters on the website: <u>doubling time</u>, <u>percentage of cells in cytokinesis</u>, <u>percentage of cells in mitosis</u>, <u>minimum time to detect two EdU-labeled nuclei in the same cell</u>, <u>percentage of cells EdU-labeled after EdU pulse</u>, and the <u>duration of this EdU pulse</u>. The following steps of this section describe how these parameters are obtained in <em>T. brucei</em> (used as an example). However, for other cell types, the S-phase duration can be obtained directly from specific studies when available [<em>e.g.</em>, <em>S. cerevisiae</em> (Brewer <em>et al.</em>, 1984; Ivanova <em>et al.</em>, 2020); mammalian cells â MEFs cells (Ishida <em>et al.</em>, 2001; Pereira <em>et al.</em>, 2017)].<br/><br/><ol class="npwtwo"><li>Prepare a set of <em>T. brucei</em> culture (here, it was used the procyclic forms) to estimate de doubling time. Initiates a growth curve with 1 x 10<sup>6</sup> cells/ml at 28 Â°C in 10 ml of culture.<br/><em>Note: The cell density for the beginning of the curve varies according to the cell type.</em> </li><li>Harvest cell samples and count daily, until it reached the stationary phase.</li><li>Insert the values counted on <a href="https://doubling-time.com" target="_blank">https://doubling-time.com</a> and take note of the <u>doubling time</u> value.</li><li>Prepare a new set of <em>T. brucei</em> culture (procyclic forms) to estimate the percentage of cells in mitosis and cytokinesis. Incubate cells (28 Â°C) until they reach exponential phase (~1 x 10<sup>7</sup> cells/ml).<br/><em>Note: The cell density for the exponential phase varies according to the cell type used.</em> </li><li>Harvest 1 ml of the culture by centrifugation at 800 <em>x g</em> for 5 min at 4 Â°C and wash twice using 1 ml of 1x PBS.</li><li>Remove the 1x PBS from cells carefully to preserve the pellet.</li><li>Suspend the pellet in 1 ml of Fixation Buffer (FB) and transfer to a 1.5 ml microcentrifuge tube.</li><li>Incubate at 4 Â°C for 7 min and wash three times using 1 ml of cold 1x PBS (centrifuging at 800 <em>x g</em> for 5 min in each wash). For the last wash, suspend the pellet carefully into 500 Î¼l of 1x PBS. If the pellet is too small (<em>i.e.</em>, almost invisible to the naked eye), decrease the volume to 100 Î¼l of 1x PBS.</li><li>Prepare the slides to receive the cells by spreading 2.5 Î¼l of Poly-L-lysine solution (PLS) onto slide surface until PLS dry out. Prepare three slides for each sample from Step B8.<br/><em>Note: Use a coverslip to spread PLS onto slide (see da Silva </em><em>et al.</em><em>, 2018 for more detail).</em> </li><li>Spread the suspended-pellet (from Step B8) carefully in each of the three slides. Use 25-30 Î¼l and save the remaining suspended-pellet volume in case you need to remake some slides.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Use the same surface where PLS was previous spread.</em><br/></li><li><em>Each slide is one technical replicate.</em> </li></ol></li><li>Wait for the cells to precipitate and settle on slide for 10-15 min at room temperature. Ensure that the cells do not dry out.</li><li>Permeabilize the cells by adding 50 Î¼l of permeabilization solution (PS) for 10 min at room temperature.</li><li>Wash the slide containing cells three times using 1x PBS.<br/><em>Note: Use a P1000 micropipette to spread (by sneezing) 1x PBS (1 ml) onto slide three times.</em> </li><li>Ensure all the liquid was removed from the slide surface containing cells and add 4 Î¼l of Vectashield mounting medium containing DAPI.<br/><em>Note: This reagent is used as anti-fade mounting solution and to stain organelles containing DNA.</em> </li><li>Add a glass coverslip and seal using colorless nail varnish. Wait the varnish dry out for 5 min. The slide can be analyzed under a fluorescence microscope immediately or stored at 4 Â°C up to one month.</li><li>Count at least 100 cells and identify cells in mitosis (with the nucleus in division) and cytokinesis (with two separated nuclei in the same cell). Figure 1 shows representative images of <em>T. brucei</em> cells in mitosis (A) and cytokinesis (B).<br/><em>Note: Capture images using the differential interference contrast (DIC) (if available) or phase contrast. Merge with DAPI to assist in the identification of cells in mitosis/cytokinesis.</em><br/><br/><img alt="" class="layerphoto" height="84" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201017/20201017092159_8630.jpg" width="300"/><br/><strong>Figure 1. Representative images showing <em>T. brucei</em> cells in mitosis (left) and cytokinesis (right).</strong> It is worth to mentioning that in mitosis, the nucleus is dividing, while in cytokinesis the nucleus has already been divided. DIC was used to show the morphology of the cells, while DAPI was used to stain organelles containing DNA [nucleus (N) and kinetoplast (K)]. Scale bars = 10 Î¼m.<br/><br/></li><li>Take note the <u>percentage of cells in mitosis and citokynesis</u>.</li><li>Prepare a new set of <em>T. brucei</em> culture to estimate the minimum time to detect two EdU-labeled nuclei in the same cell. Incubate cells (28 Â°C) until they reach exponential phase (~1 x 10<sup>7</sup> cells/ml).<br/><em>Note: The cell density for the exponential phase varies according to the cell type used.</em> </li><li>Add 100 ÂµM of 5-ethynyl-2â-deoxyuridine solution (EdU-S) in the culture containing <em>T. brucei</em> cells and wait for 30 min.</li><li>Harvest 500 Âµl of the culture by centrifugation (800 <em>x g</em> for 3 min at 4 Â°C) and wash twice using 1 ml of 1x PBS.</li><li>Follow the Steps B6-B7 exactly as previously described.</li><li>Incubate at 4 Â°C.</li><li>Continue collecting samples every 15 min until reach 3 h of EdU incorporation (11 samples). Follow the Steps B20-B22 for each sample.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Each sample should be incubated in FB for, at least, 7 min on ice.</em> </li><li><em>At the end of this step, there should be 12 samples collected every 15 min.</em> </li></ol></li><li>Take the 12 samples and wash three times each using 1 ml of cold 1x PBS (centrifuging at 800 <em>x g</em> for 5 min in each wash). For the last wash, suspend the pellet of each sample carefully into 500 Î¼l of 1x PBS. If the pellet is too small (<em>i.e.</em>, almost invisible to the naked eye), decrease the volume to 100 Î¼l of 1x PBS.</li><li>Prepare 12 slides by spreading 2.5 Î¼l of PLS onto slide surface until it dry out.<br/><em>Note: Use a coverslip to spread PLS onto slide. For more details, see da Silva </em><em>et al.</em><em> (2018).</em> </li><li>Spread the suspended-pellets (from Step B23) carefully in each slide. Use 25-30 Î¼l and save the remaining volume in case you need to remake some slides.<br/><em>Note: Use the same surface where PLS was previous spread.</em> </li><li>Wait for the cells to decant and settle on slide for 10-15 min at room temperature. Ensure that the cells do not dry out.</li><li>Permeabilize each sample by adding 50 Î¼l of PS for 10 min at room temperature.</li><li>Wash each slide containing cells three times using 1x PBS.<br/><em>Note: Use a P1000 micropipette to spread (by sneezing) 1x PBS (1 ml) onto slide three times.</em> </li><li>Follow the protocol of Click-iT EdU Imaging Kit, established by the manufacturer.<br/><em>Note: The protocol can be accessed here, on the topic Documents/Manuals and protocols:</em><br/><a href="https://www.thermofisher.com/order/catalog/product/C10337#/C10337" target="_blank"><em>https://www.thermofisher.com/order/catalog/product/C10337#/C10337</em></a><em>.</em> </li><li>At the end of the protocol established by the manufacturer, ensure all the liquid was removed from the slide surface containing cells and add 4 Î¼l of Vectashield mounting medium containing DAPI.<br/><em>Note: This solution is used as the anti-fade mounting solution and to stain organelles containing DNA.</em> </li><li>Add a glass coverslip and seal using colorless nail varnish. Wait the varnish dry out for 5 min. The slide can be analyzed under a fluorescence microscope immediately or stored at 4 Â°C up to one month.</li><li>Search (in each sample) for a parasite containing two EdU-labeled nuclei in the same cell (<em>i.e.</em>, cell in citokynesis). Start from the shortest time sample, <em>i.e.</em>, 15 min after 30 min of EdU incorporation, and proceed with the other samples (<em>i.e.</em>, 30 min after 30 min of EdU incorporation, 45 min after 30 min of EdU incorporation, <em>etc.</em>).<br/><em>Note: For more details, see the description and images on Figure 1D from da Silva </em><em>et al.</em><em> (2019).</em> </li><li>Once observed, take note the time required for this. This is the <u>minimum time to detect two EdU-labeled nuclei in the same cell</u>.<br/><em>Note: The time obtained must be the sum of the initial time for EdU incorporation (30 min) plus the shortest time in which the two EdU-labeled nuclei were observed. In our case, this time was 2 h. For details, see da Silva </em><em>et al.</em><em> (2019).</em> </li><li>Prepare a new set of <em>T. brucei</em> culture to estimate the percentage of EdU-labeled cells during a EdU pulse. Incubate cells (28 Â°C) until they reach exponential phase (~1 x 10<sup>7</sup> cells/ml). <br/><em>Note: The cell density for the exponential phase varies according to the cell type used.</em> </li><li>Add 100 ÂµM of EdU-S in the culture containing <em>T. brucei</em> cells and wait for 1 h.<br/><em>Note: This will be the <u>duration of the EdU pulse</u>, i.e., 1 h.</em> </li><li>Harvest 500 Âµl of the culture by centrifugation (800 <em>x g</em> for 3 min at 4 Â°C) and wash twice using 1 ml of 1x PBS.</li><li>Follow the Steps B6-B13 exactly as previously described.</li><li>Follow the protocol of Click-iT EdU Imaging Kit, established by the manufacturer.<br/><em>Note: The protocol can be accessed here, on the topic Documents/Manuals and protocols:</em><br/><a href="https://www.thermofisher.com/order/catalog/product/C10337#/C10337" target="_blank"><em>https://www.thermofisher.com/order/catalog/product/C10337#/C10337</em></a><em>.</em> </li><li>At the end of the protocol established by the manufacturer, ensure all the liquid was removed from the slide surface containing cells and add 4 Î¼l of Vectashield mounting medium containing DAPI.</li><li>Add a glass coverslip and seal using colorless nail varnish. Wait the varnish dry out for 5 min. The slide can be analyzed under a fluorescence microscope immediately or stored at 4 Â°C up to one month.</li><li>Count at least 100 cells (total) and identify cells EdU-labeled. Figure 2 shows representative images of <em>T. brucei</em> EdU-positive cells.</li><li>Measure the <u>percentage of cells EdU-positive</u>.</li><li>Once you have all the required parameters, access the CeCyD website: <a href="http://cecyd.vital.butantan.gov.br/" target="_blank">http://cecyd.vital.butantan.gov.br/</a>, and input the values to obtain the duration of all the cell cycle phases. Take note of the S-phase duration.<br/><br/><img alt="" class="layerphoto" height="287" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201017/20201017091412_3518.jpg" width="300"/><br/><strong>Figure 2. Representative image showing EdU-positive <em>T. brucei</em> cells.</strong> The representative image is organized in four squares: DIC (indicating the morphology of the cells), DAPI (staining organelles containing DNA), EdU (showing cells that uptaked EdU during the S-phase), and Merged (overlay between DAPI and EdU). The white arrows indicate EdU-positive cells. Scale bar = 10 Î¼m.<br/><br/></li></ol></li><li>Estimation of the replication rate using DNA combing<br/>The following steps of this section describe how the replication rate is obtained in <em>T. brucei</em> (used as an example). However, for other cell types, the replication rate can be obtained directly from specific studies when available [<em>e.g.</em>, <em>S. cerevisiae</em> (Sekedat <em>et al.</em>, 2010); mammalian cellsâMEFs cells (Stanojcic <em>et al.</em>, 2016)].<br/><br/><ol class="npwtwo"><li>Prepare a set of <em>T. brucei</em> culture (here, it was used the procyclic forms) to perform the DNA combing assay. Initiates the culture with 2 x 10<sup>6</sup> cells/ml in 10 ml of culture. Incubate at 28 Â°C for 24 h. Count the cell density. The culture must be in the concentration of ~1-2 x 10<sup>7</sup> cells/ml, totalizing ~1-2 x 10<sup>8</sup> cells.<br/><em>Note: The cell density for the beginning of the DNA combing assay varies according to the cell type.</em> </li><li>Add 300 ÂµM of 5â²-iodo-2â²-deoxyuridine solution (IdU-S) and wait for 30 min.</li><li>Add 500 ÂµM of 5â²-chloro-2â²-deoxyuridine solution (CldU-S) and wait for 30 min.<br/><em>Note: Do not wash between the halogenated thymidine analogs (IdU and CldU) incorporation.</em> </li><li>Harvest the whole culture by centrifugation (1,600 <em>x g</em> for 3 min at 4 Â°C) and wash twice using 1 ml of 1x PBS. In the second wash, transfer the suspended-pellet to a 1.5 ml microcentrifuge tube.</li><li>Wash using 500 Âµl of DNA combing washing buffer (DC-WB) and suspend the pellet in 50 Âµl of DC-WB. Keep at room temperature.</li><li>Prepare 2% agarose low-melting: Weigh 10 mg of low melting agarose in a 1.5 ml centrifuge tube. Using another 1.5 ml centrifuge tube, heat 500 Âµl of DC-WB (90 Â°C for 5 min). Resuspend the 10 mg of low melting agarose using the heated 500 Âµl of DC-WB. Mix well to solubilize the agarose. Keep the solution heated at 65 Â°C.</li><li>Prepare the reusable plug modes to receive 100 Âµl of the agarose + cells mixture.</li><li>Gently mix 50 Âµl of the agarose solution (from Step C6) in the 50 Âµl of resuspended pellet (from Step C5), totalizing 100 Âµl. Add immediately to the previously prepared reusable plug model (from Step C7). Wait to cool and incubate at 4 Â°C for 1 h.</li><li>Carefully unmold the plug in a 1.5 ml epp containing 500 Âµl of DNA combing lysis buffer (DC-LB).</li><li>Incubate for 24 h at 50 Â°C.</li><li>Replace the DC-LB (gently) by a new one (500 Âµl) and incubate for another 24 h at 50 Â°C.</li><li>Wash twice using 1 ml of T<sub>10</sub>E<sub>1</sub> buffer for 1 h (each wash) at room temperature.</li><li>Add 1 ml of T<sub>10</sub>E<sub>1</sub> buffer (gently) and incubate overnight at 4 Â°C.</li><li>Discard the T<sub>10</sub>E<sub>1</sub> buffer and incubate the plug in 800 Âµl of 0.5 M MES, pH 5.5 at 68 Â°C for 20 min. Next, remove the tube containing the plug from 68 Â°C and incubate it immediately at 42 Â°C for 10 min.</li><li>Add 2 Î¼l of Î²-agarase I (1,000 units/ml) and incubate overnight at 42 Â°C.<br/><em>Note: In this step, add the Î²-agarase I in 200 Âµl of 0.5 M MES. Mix well and add it carefully in the plug dissolved with 800 Âµl 0.5 M MES.</em> </li><li>Prepare the molecular combing system by adding 1 ml of 0.5 M MES into the reservoir of the FiberComb apparatus. Next, carefully pour the tube containing the plug dissolved (from Step C15) into the reservoir containing 1 ml of 0.5 M MES.<br/><em>Note: At this stage, it is normal to form a viscosity in the liquid coming from the tube.</em> </li><li>Immediately, place the Combicoverslip on the FiberComb apparatus and stretch the DNA according to equipment's instructions.<br/><em>Note: The solution containing the plug dissolved (i.e., incorporated DNA) inside the reservoir can be stored up to 1 week at 4 Â°C protected from light.</em> </li><li>After stretch the DNA onto Combicoverslip, let it dry for a few seconds and incubate it at 65 Â°C for 2-4 h.</li><li>Place the Combicoverslip containing the DNA stretched in an appropriate container (<em>e.g.</em>, use a 6-well flat-bottom plate). Carefully add 3 ml of DNA combing denaturation buffer (DC-DB) and incubate for 20 min.</li><li>Wash three times using 1 ml of 1x PBS. Dehydrate the coverslip containing the DNA stretched successively using 2 ml of: 70% ethanol, then 90% ethanol, and 100% ethanol (1 min each). Remove the alcohol and let the Combicoverslip dry completely at room temperature.</li><li>Block using 3 ml of DNA combing blocking solution (DC-BS) for 20 min at 37 Â°C.</li><li>Prepare the DNA combing primary antibodies solution (DC-PAS).<br/><em>Note: The DC-PAS must be freshly prepared, as it is for immediate use.</em> </li><li>Add 20 Âµl of DC-PAS on the top of the Combicoverslip containing the DNA stretched (from Step C21). Cover with plastic coverslip. Incubate at 37 Â°C for 1 h protected from light.</li><li>Wash twice using 1x PBS for 3 min each.</li><li>Prepare the DNA combing secondary antibodies solution (DC-SAS).<br/><em>Note: The DC-SAS must be freshly prepared, as it is for immediate use.</em> </li><li>Add 20 Âµl of DC-SAS on the top of the Combicoverslip containing the DNA stretched (from Step C24). Cover with a plastic coverslip. Incubate at 37 Â°C for 1 h protected from light.</li><li>Wash twice using 1x PBS for 3 min each.</li><li>Prepare the DNA combing anti-ssDNA solution (DC-anti-ssDNA).<br/><em>Note: The DC-anti-ssDNA must be freshly prepared, as it is for immediate use.</em> </li><li>Add 20 Âµl of DC-anti-ssDNA on the top of the coverslip containing the DNA stretched (from Step C27). Cover with plastic coverslip. Incubate at 37 Â°C for 2 h protected from light.</li><li>Wash twice using 1x PBS for 3 min each.</li><li>Prepare the DNA combing Alexafluor 350 (DC-alexa350).<br/><em>Note: The DC-alexa350 must be freshly prepared, as it is for immediate use.</em> </li><li>Add 20 Âµl of DC-alexa350 on the top of the coverslip containing the DNA stretched (from Step C30). Cover with plastic coverslip. Incubate at 37 Â°C for 1 h protected from light.</li><li>Wash twice using 1x PBS for 3 min each.</li><li>Ensure all the liquid was removed from the coverslip containing the DNA stretched. Mount the coverslip on a slide adding 20 Âµl of ProLong Gold antifade. Seal using colorless nail varnish and let dry for at least 1 h at room temperature.</li><li>Acquire images of DNA fibers using a fluorescence microscope. Merge acquired images using the Cell F Olympus or ImageJ software.<br/><em>Notes: </em><br/><ol class="npwthree"><li><em>The observation of longer DNA fibers requires the capture of adjacent fields.</em><br/></li><li><em>The stretching factor along each molecule is contant (1 Âµm = 2 kb).</em> </li><li><em>Fibers &lt; 100 kb must be excluded from the analysis to improve the accuracy of the assay.</em> </li><li><em>Only green tracks (containing CldU, the second analog added) continuing from red tracks (containing IdU, the first analog added) or blue tracks (non-incorporated DNA) must be used to determine replication rate.</em> </li><li><em>The length of the green track must be measured from intact green fibers: blue-green-red (BGR), red-green-blue (RGB), and blue-green-red-green-blue (BGRGB) events.</em> </li></ol></li><li>Estimate the average replication rate using the formula:<br/><br/><div style="text-align:center;"><img alt="" class="layerphoto" height="56" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201017/20201017091620_2675.png" width="280"/><br/></div><br/>where <em>v</em> is the average replication rate (in kb.min<sup>-1</sup>), <em>n</em> is the number of DNA fibers analyzed, and <em>t</em> is the time of CldU pulse (in min).<br/><em>Note: At least three independent assays must be performed to improve the accuracy of the assay</em><br/></li></ol></li></ol><p class="pptt" id="biaoti38528">### Data analysis</p><ol class="npwtwok"><li>Data analysis<br/>With the possession of all variables (parameters) obtained previously, <em>i.e.</em>, chromosome size, S-phase duration, and average replication rate, insert them in the formula and calculate the MO per chromosome analyzed.<br/><br/><div style="text-align:center;"><img alt="" class="layerphoto" height="47" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201017/20201017091818_5101.png" style="font-size:16px;" width="255"/> </div><br/>of note, <em>N</em> is the size of the chromosome in question, <em>v</em> is the average replication rate, and <em>S</em> is the S-phase duration.<br/><em>Note: If the right-hand side of the formula results in a fraction of a unit, then the next higher integer unit must be taken as the result of the formula, which is represented by the ceiling function </em>(â â)<em>.</em><br/><br/></li><li>Prepare a table to add the MO values found and compare them with the number of replication origins estimated/mapped by other approaches (<em>e.g.</em>, DNA combing, MFA-seq analysis). In the current protocol, Table 1 was prepared for <em>T. brucei</em> and Table 2 for <em>S. cerevisiae</em>, which were used to demonstrate that the MO estimation can be applied in any organism.<br/><br/><strong>Table 1. Comparison among the MO, origins mapped by MFA-seq technique (Tiengwe </strong><em><strong>et al.</strong></em><strong>, 2012), and origins estimated by DNA combing (Stanojcic </strong><em><strong>et al.</strong></em><strong>, 2016) in </strong><em><strong>T. brucei</strong></em><br/><img alt="" class="layerphoto" height="410" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201017/20201017091908_3873.png" width="750"/><br/>S-phase duration = 138.6 min (da Silva <em>et al.</em>, 2019), replication rate = 1.84 kbÂ·min<sup>-1</sup> (Stanojcic <em>et al.</em>, 2016). The values presented in this table were originally published in (da Silva <em>et al.</em>, 2019).<br/><br/><strong>Table 2. Comparison among the MO, known replication origins (Nieduszynski </strong><em><strong>et al.</strong></em><strong>, 2006), and origins estimated by DNA combing (Lengronne </strong><em><strong>et al.</strong></em><strong>, 2001) in <em>S. cerevisiae</em></strong><br/><img alt="" class="layerphoto" height="561" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201017/20201017091938_2048.png" width="750"/><br/>S-phase duration = 30 min (Brewer <em>et al.</em>, 1984; Ivanova <em>et al.</em>, 2020), replication rate = 1.6 kbÂ·min<sup>-1</sup> (Sekedat <em>et al.</em>, 2010). The values presented in this table were originally published in (da Silva <em>et al.</em>, 2020).<br/><br/></li><li>Set up a graph in order to show trend lines for each approach analyzed (Figure 3 was made for <em>T. brucei</em> and Figure 4 for <em>S. cerevisiae</em>).<br/><br/><img alt="" class="layerphoto" height="174" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201017/20201017092020_3830.jpg" width="300"/><br/><strong>Figure 3. Comparative analysis among different approaches used to estimate origins in <em>T. brucei</em>.</strong> The graph shows positive correlations between chromosome size and the number of origins estimated by: MO (black), origins estimated by MFA-seq (blue), and origins estimated by DNA combing (red). The trend lines for the approaches, as well as the equations, are shown. Studies are referenced in the legend of the graph. The data presented in this graph were originally published in da Silva <em>et al.</em> (2019).<br/><br/><img alt="" class="layerphoto" height="176" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201017/20201017092039_1651.jpg" width="300"/><br/><strong>Figure 4. Comparative analysis among different approaches used to estimate origins in <em>S. cerevisiae</em>.</strong> The graph shows positive correlations between chromosome size and the number of origins estimated by: MO (black), known replication origins (green), and origins estimated by DNA combing (red). The trend lines for the approaches, as well as the equations, are shown. Studies are referenced in the legend of the graph. The data presented in this graph were originally published in da Silva <em>et al.</em> (2020).<br/><br/>    It is possible to observe an expected positive correlation between the number of origins and the size of each chromosome, <em>i.e.</em>, the larger the chromosome, the more origins are required to replicate it within the S-phase duration. The MO allows the establishment of a threshold that can serve as a parameter by other methods. For instance, in <em>T. brucei</em> (Figure 3), when comparing the MO values with the MFA-seq ones, it is clear that some chromosomes are not able to be duplicated only with the origins mapped by MFA-seq, <em>i.e.</em>, some blue circles are above from the threshold established by the MO values (black dotted line). Briefly, the reason is due MFA-seq approach has low resolution and map mainly constitutive (frequent) origins (da Silva <em>et al.</em>, 2019 and 2020). Further analysis and a complete explanation can be found in the original publication from which this method was derived (da Silva <em>et al.</em>, 2019).<br/></li></ol><p class="pptt" id="biaoti38529">### Notes</p><ol class="npwone"><li>General notes<ol class="npwtwo"><li>If the values for the chromosomes size, S-phase duration, and replication rate are already available in the literature (<em>i.e.</em>, they can be acquired from other studies), the Steps A-C described in âprocedureâ are not necessary.</li><li>The number of DNA fibers (<em>n</em>) analyzed in step c described on procedure must be as high as possible, since the higher the <em>n</em>, the more accurate the estimation of the replication rate will be.<br/><br/></li></ol></li><li>Technical tips<ol class="npwtwo"><li>To avoid false positives during the estimation of EdU-positive cells (Step B from procedure), do not overexpose fluorescence images during acquisition step.</li><li>The incubation time of the halogenated thymidine analogs (IdU and CldU) varies according to the doubling-time (Step C from procedure) and it can be optimized for other cell types. The same is valid for the incubation time of the EdU (EdU pulse) (Step B from procedure).</li><li>If it is possible, perform the whole protocol of DNA combing (Step C from procedure) in the dark, because DNA with halogenated thymidine analogs incorporated is more sensitive to the light.<br/></li></ol></li></ol><p class="pptt" id="biaoti38530">### Recipes</p><ol class="npwtwok"><li>Phosphate buffered saline (1x PBS)<br/>137 mM NaCl<br/>2.7 mM KCl<br/>10 mM Na<sub>2</sub>HPO<sub>4</sub><br/>2 mM KH<sub>2</sub>PO<sub>4</sub><br/><ol class="npwthree"><li>Prepare the buffer by adding 8 g of NaCl, 0.2 g of KCl, 1.44 g of Na<sub>2</sub>HPO<sub>4</sub>, and 0.24 g of KH<sub>2</sub>PO<sub>4</sub> in 800 ml of destilled water</li><li>Adjust the pH to 7.4 with HCl</li><li>Complete the volume to 1 L with destilled water</li><li>Dispense the solution into aliquots (<em>e.g.</em>, 250 ml) and sterilize by autoclaving (20 min, 120 Â°C, liquid cycle)</li><li>Store at room temperature for up to six months</li><li>Check the pH after prolonged use<br/></li></ol></li><li>Hemin Solution (HS)<br/>50 mg/ml Hemin<br/><ol class="npwthree"><li>Prepare the solution by adding 50 mg of Hemin and 1 ml of 1 M NaOH in 80 ml of MilliQ water</li><li>Stir vigorously on a magnetic stirrer</li><li>Bring up the volume to 100 ml with MilliQ water</li><li>Sterilize by filtering (using 0.22 Î¼m filter)</li><li>Store at 4 Â°C for up to one month<br/></li></ol></li><li>SDM-79 medium (for cultivate <em>T. brucei</em> procyclic cells)<br/>90% (v/v) SDM-79 medium (incomplete)<br/>10% (v/v) heat-inactivated Fetal Bovine Serum (FBS)<br/>7.5 Âµg/ml Hemin<br/>0.134 Âµg/ml Streptomycin sulfate salt<br/>0.060 Âµg/ml Penicillin G sodium salt<br/><ol class="npwthree"><li>Prepare the medium by adding 50 ml of heat-inactivated FBS, 75 Âµl of HS, 67 mg of Streptomycin sulfate salt, and 30 mg of Penicillin G sodium salt in 400 ml of SDM-79 medium (incomplete)</li><li>Adjust the pH to 7.3 and complete the volume to 500 ml with SDM-79 medium (incomplete)</li><li>Sterilize by filtering (using 0.22 Î¼m filter) and dispense into aliquots (<em>e.g.</em>, 100 ml)</li><li>Store at 4 Â°C for up to one month<br/></li></ol></li><li>5â²-chloro-2â²-deoxyuridine solution (CldU-S)<br/>10 mM of CldU diluted in water<br/><ol class="npwthree"><li>Prepare the buffer by adding 26.26 mg of 5â²-chloro-2â²-deoxyuridine in 9 ml of water</li><li>Solubilizes and complete the volume to 10 ml</li><li>Sterilize by filtering (using 0.22 Î¼m syringe filter) and dispense the solution into aliquots (<em>e.g.</em>, 1 ml)</li><li>Store at -20 Â°C for up to six months<br/></li></ol></li><li>5-ethynyl-2â-deoxyuridine solution (EdU-S)<br/>10 mM of EdU diluted in 1x PBS<br/><ol class="npwthree"><li>Prepare the solution by adding 2 ml of 1x PBS to component A from Click-iT<sup>Â®</sup> EdU Imaging Kit</li><li>Mix well to solubilize it</li><li>Sterilize by filtering (using 0.22 Î¼m syringe filter) and dispense the solution into aliquots (<em>e.g.</em>, 1 ml)</li><li>Store at -20 Â°C for up to six months<br/></li></ol></li><li>5â²-iodo-2â²-deoxyuridine solution (IdU-S)<br/>10 mM of IdU diluted in water alkalinized<br/><em>Note: IdU is not readily soluble at physiological pH.</em><br/><ol class="npwthree"><li>Prepare the buffer by adding 35.4 mg of 5â²-iodo-2â²-deoxyuridine in 5 ml of alkalinized MilliQ water (adjust the Milli-Q water pH to 9.5 using 5 M NaOH with the aid of a pH meter)</li><li>Solubilize and complete the volume to 10 ml with Milli-Q water</li><li>Sterilize by filtering (using 0.22 Î¼m syringe filter) and dispense the solution into aliquots (<em>e.g.</em>, 1 ml)</li><li>Store at -20 Â°C for up to six months<br/></li></ol></li><li>0.5 M EDTA, pH 8.0<br/><ol class="npwthree"><li>Prepare the solution by adding 186.1 g of disodium EDTAÂ·2H<sub>2</sub>O to 800 ml of distilled H<sub>2</sub>O </li><li>Stir vigorously on a magnetic stirrer while adjust the pH to 8.0 with NaOH (~20 g of NaOH pellets)</li><li>Bring up the volume to 1 L with distilled water</li><li>Dispense into aliquots (<em>e.g.</em>, 250 ml) and sterilize by autoclaving</li><li>Store at room temperature for up to six months<br/></li></ol></li><li>0.65 M EDTA, pH 8.0<br/><ol class="npwthree"><li>Prepare the solution by adding 24.2 g of disodium EDTAÂ·2H<sub>2</sub>O to 80 ml of distilled H<sub>2</sub>O </li><li>Stir vigorously on a magnetic stirrer while adjust the pH to 8.0 with NaOH</li><li>Bring up the volume to 100 ml with distilled water</li><li>Sterilize by autoclaving</li><li>Store at room temperature for up to one month<br/></li></ol></li><li>1.5 M Tris-HCl, pH 8.8<br/><ol class="npwthree"><li>Prepare the solution by adding 181.65 g Tris base in 800 ml of MilliQ water</li><li>Adjust the pH to 8.8 using concentrated HCl</li><li>Bring up the volume to 1 L with MilliQ water</li><li>Dispense into aliquots (<em>e.g.</em>, 250 ml) and sterilize by autoclaving</li><li>Store at room temperature for up to six months<br/></li></ol></li><li>1 M NaCl<br/><ol class="npwthree"><li>Prepare the solution by adding 58.44 g of NaCl in 800 ml of distilled H<sub>2</sub>O </li><li>Stir vigorously on a magnetic stirrer (heat to help dissolve)</li><li>Bring up the volume to 1 L with distilled water</li><li>Dispense into aliquots (<em>e.g.</em>, 250 ml) and sterilize by autoclaving</li><li>Store at room temperature for up to one year<br/></li></ol></li><li>70% ethanol<br/><ol class="npwthree"><li>Prepare the solution by adding 35 ml of ethanol absolute in 15 ml of distilled H<sub>2</sub>O</li><li>Mix well and store at 4 Â°C for up to one year<br/></li></ol></li><li>90% ethanol<br/><ol class="npwthree"><li>Prepare the solution by adding 45 ml of ethanol absolute in 5 ml of distilled H<sub>2</sub>O</li><li>Mix well and store at 4 Â°C for up to one year<br/></li></ol></li><li>DNA combing washing buffer (DC-WB)<br/>100 mM EDTA, pH 8.0<br/>10 mM Tris-HCl <br/>20 mM NaCl <br/><ol class="npwthree"><li>Prepare the buffer by adding 20 ml of 500 mM EDTA, 666 Âµl of Tris-HCl, and 2 ml of NaCl in 77.33 ml of MilliQ water</li><li>Mix well and sterilize by filtering (using 0.22 Î¼m filter)</li><li>Store at 4 Â°C for up to one month<br/></li></ol></li><li>DNA combing lysis buffer (DC-LB)<br/>0.65 EDTA, pH 8.0<br/>1% N-Laurylsarcosine <br/>100 Î¼g/ml Proteinase K<br/><ol class="npwthree"><li>Prepare the buffer by adding 50 Âµg of N-Laurylsarcosine and 500 Âµg of proteinase K in 4 ml of 0.65 M EDTA, pH 8.0</li><li>Mix well and bring up the volume to 5 ml using 0.65 M EDTA, pH 8.0</li><li>Store at room temperature up to one week<br/></li></ol></li><li>DNA combing blocking solution (DC-BS)<br/>1x PBS<br/>1% Bovine Serum Albumin (BSA)<br/>0.1% Triton X-100<br/><ol class="npwthree"><li>Prepare the solution by adding 100 Âµg of BSA and 10 Âµl of Triton X-100 in 9,880 Âµl of 1x PBS</li><li>Mix well and sterilize by filtering (using 0.22 Î¼m filter)</li><li>Store at 4 Â°C up to one week<br/></li></ol></li><li>DNA combing denaturation buffer (DC-DB)<br/>0.5 M NaOH<br/>1 M NaCl<br/><ol class="npwthree"><li>Prepare the solution by adding 200 mg of NaOH and 58.4 mg of NaCl in 8 ml of distilled H<sub>2</sub>O</li><li>Mix well and bring up the volume to 10 ml using distilled H<sub>2</sub>O</li><li>Store at room temperature up to one week<br/></li></ol></li><li>DNA combing primary antibodies solution (DC-PAS)<br/>1/10 anti-IdU (mouse anti-BrdU)<br/>1/10 anti-CldU (rat anti-BrdU)<br/>100% DC-BS<br/><ol class="npwthree"><li>Prepare the solution by adding 2 Âµl of anti-IdU, 2 Âµl of anti-CldU in 16 Âµl of DC-BS</li><li>Mix well and use immediately<br/></li></ol></li><li>DNA combing secondary antibodies solution (DC-SAS)<br/>1/20 anti-mouse Alexafluor 568<br/>1/20 anti-rat Alexafluor 488<br/>100% DC-BS<br/><ol class="npwthree"><li>Prepare the solution by adding 1 Âµl of anti-mouse Alexafluor 568, 1 Âµl of anti-rat Alexafluor 488 in 18 Âµl of DC-BS<br/></li><li>Mix well and use immediately<br/></li></ol></li><li>DNA combing anti-ssDNA solution (DC-anti-ssDNA)<br/>1/5 anti-ssDNA (mouse anti-ssDNA)<br/>100% DC-BS<br/><ol class="npwthree"><li>Prepare the solution by adding 4 Âµl of anti-ssDNA in 16 Âµl of DC-BS<br/></li><li>Mix well and use immediately</li></ol></li><li>DNA combing Alexafluor 350 (DC-alexa350)<br/>1/10 anti-mouse Alexafluor 350<br/>100% DC-BS<br/><ol class="npwthree"><li>Prepare the solution by adding 2 Âµl of anti-mouse Alexafluor 350 in 18 Âµl of DC-BS<br/></li><li>Mix well and use immediately</li></ol></li><li>T<sub>10</sub>E<sub>1</sub> buffer<br/>10 mM Tris-HCl, pH 8.8<br/>1 mM EDTA, pH 8.0<br/><ol class="npwthree"><li>Prepare the buffer by mixing 66.6 Âµl of 1.5 M Tris-HCl, 20 Âµl of 0.5 M EDTA, and 9,913 Âµl of MilliQ H<sub>2</sub>O<br/></li><li>Mix well and sterilize by filtering (using 0.22 Î¼m filter)<br/></li><li>Store at 4 Â°C up to one month</li></ol></li><li>0.5 M MES buffer, pH 5.5<br/><ol class="npwthree"><li>Prepare the solution by adding 9.76 g of MES free acid in 80 ml of distilled H<sub>2</sub>O<br/></li><li>Stir vigorously on a magnetic stirrer<br/></li><li>Adjust the pH to 5.5 using NaOH<br/></li><li>Bring up the volume to 100 ml using distilled H<sub>2</sub>O<br/></li><li>Sterilize by autoclaving<br/></li><li>Store at room temperature for up to one month</li></ol></li><li>Fixation buffer (FB)<br/>4% (w/v) Paraformaldehyde in 1x PBS<br/><ol class="npwthree"><li>Prepare the buffer by adding 2.1 g of Paraformaldehyde (powder, 95%) in 40 ml 1x PBS<br/></li><li>Solubilize and complete the volume to 50 ml with 1x PBS<br/></li><li>Store at 4 Â°C for up to one month</li></ol></li><li>Poly-L-lysine solution (PLS)<br/>0.1% (w/v) Poly-L-Lysine hydrochloride diluted in water<br/><ol class="npwthree"><li>Prepare the buffer by adding 10 mg of Poly-L-Lysine hydrochloride in 9 ml of water<br/></li><li>Solubilize and complete the volume to 10 ml with water<br/></li><li>Sterilize by filtering (using 0.22 Î¼m syringe filter) and dispense the solution into aliquots (<em>e.g.</em>, 1 ml)<br/></li><li>Store at 4 Â°C for up to six months</li></ol></li><li>Permeabilization solution (PS)<br/><ol class="npwthree"><li>Prepare the buffer by adding 10 Î¼l of Triton X-100 in 9,990 Î¼l of 1x PBS<br/></li><li>Mix well, sterilize by filtering (using 0.22 Î¼m syringe filter), and dispense the solution into aliquots (<em>e.g.</em>, 1 ml)<br/></li><li>Store at 4 Â°C for up to six months</li></ol></li></ol><p class="pptt" id="biaoti38531">### Acknowledgments</p><p class="ppzz">This work has been derived from da Silva <em>et al.</em> (2019) and da Silva <em>et al.</em> (2020). The development of the protocol presented here was supported by SÃ£o Paulo Research Foundation (FAPESP) â Center of Toxins, Immune Response and Cell Signaling (CeTICS) under grants 2014/24170-5 and 2017/18719-2. Currently, Marcelo S. da Silva is fellow from FAPESP (grants 2019/10753-2 and 2020/10277-3).</p><p class="pptt" id="biaoti38532">### Competing interests</p><p class="ppzz">The author declares that there is no conflict of interest regarding the publication of this article.</p><p class="pptt" id="biaoti38533">### References</p><ol class="npwtwok"><li>Brewer, B.J., Chlebowicz-Sledziewska, E., Fangman, W.L. (1984). <a href="https://pubmed.ncbi.nlm.nih.gov/6392855/" target="_blank">Cell cycle phases in the unequal mother/daughter cell cycles of <em>Saccharomyces cerevisiae</em>.</a> <em>Mol Cell Biol</em> 4: 2529-2531.</li><li>da Silva, M. S., Marin, P. A., RepolÃªs, B. M., Elias, M. C. and Machado, C. R. (2018). <a href="https://doi.org/10.21769/BioProtoc.3125" target="_blank">Analysis of DNA Exchange Using Thymidine Analogs (ADExTA) in <em>Trypanosoma cruzi</em>.</a> <em>Bio-protocol</em> 8(24): e3125.</li><li>da Silva, M. S., Cayres-Silva, G. R., Vitarelli, M. O., Marin, P. A., Hiraiwa, P. M., Araujo, C. B., Scholl, B. B., Avila, A. R., McCulloch, R., Reis, M. S. and Elias, M. C. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31811174" target="_blank">Transcription activity contributes to the firing of non-constitutive origins in African trypanosomes helping to maintain robustness in S-phase duration.</a> <em>Sci Rep</em> 9(1): 18512.</li><li>da Silva, M. S., Pavani, R. S., Damasceno, J. D., Marques, C. A., McCulloch, R., Tosi, L. R. O. and Elias, M. C. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28844718" target="_blank">Nuclear DNA replication in trypanosomatids: There are no easy methods for solving difficult problems.</a> <em>Trends Parasitol</em> 33(11): 858-874.</li><li>da Silva, M. S., Vitarelli, M. O., Souza, B. F. and Elias, M. C. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/32397111" target="_blank">Comparative analysis of the minimum number of replication origins in trypanosomatids and yeasts.</a> <em>Genes (Basel)</em> 11(5).</li><li>Ishida, S., Huang, E., Zuzan, H., Spang, R., Leone, G., West, M., and Nevins, J. R. (2001). <a href="https://pubmed.ncbi.nlm.nih.gov/11416145/" target="_blank">Role for E2F in control of both DNA replication and mitotic functions as revealed from DNA microarray analysis.</a> <em>Mol Cell Biol</em> 21(14): 4684-4699.</li><li>Ivanova, T., Maier, M., Missarova, A., Ziegler-Birling, C., Carey, L.B., Mendoza, M. (2020). <a href="https://pubmed.ncbi.nlm.nih.gov/32385287/" target="_blank">Budding yeast complete DNA replication after chromosome segregation begins.</a> <em>Nat Commun</em> 11(1): 2267.</li><li>Lengronne, A., Pasero, P., Bensimon, A., Schwob E. (2001). <a href="https://pubmed.ncbi.nlm.nih.gov/11266543/" target="_blank">Monitoring S phase progression globally and locally using BrdU incorporation in TK+ yeast strains.</a> <em>Nucleic Acids Res</em> 2001, 29(7): 1433-1442.</li><li>Leonard, A. C. and Mechali, M. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23838439" target="_blank">DNA replication origins.</a> <em>Cold Spring Harb Perspect Biol</em> 5(10): a010116.</li><li>Mechali, M. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20861881" target="_blank">Eukaryotic DNA replication origins: many choices for appropriate answers.</a> <em>Nat Rev Mol Cell Biol</em> 11(10): 728-738.</li><li>Myllykallio, H., Lopez, P., Lopez-Garcia, P., Heilig, R., Saurin, W., Zivanovic, Y., Philippe, H. and Forterre, P. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10864870" target="_blank">Bacterial mode of replication with eukaryotic-like machinery in a hyperthermophilic archaeon.</a> <em>Science</em> 288(5474): 2212-2215.</li><li>Nieduszynski, C.A., Knox, Y., Donaldson, A.D. (2006). <a href="https://pubmed.ncbi.nlm.nih.gov/16847347/" target="_blank">Genome-wide identification of replication origins in yeast by comparative genomics.</a> <em>Genes Dev</em> 20(14): 1874-1879.</li><li>Pereira, P. D., Serra-Caetano, A., Cabrita, M., Bekman, E., Braga, J., Rino, J., Santus, R., Filipe, P. L., Sousa, A. E., and Ferreira, J. A. (2017). <a href="https://pubmed.ncbi.nlm.nih.gov/28465489/" target="_blank">Quantification of cell cycle kinetics by EdU (5-ethynyl-2â-deoxyuridine)-coupled-fluorescence-intensity analysis.</a> <em>Oncotarget</em> 8(25): 40514-40532.</li><li>Sekedat, M. D., FenyÃ¶, D., Rogers, R. S., Tackett, A. J., Aitchison, J. D. and Chait, B. T. (2010). <a href="https://pubmed.ncbi.nlm.nih.gov/20212525/" target="_blank">GINS motion reveals replication fork progression is remarkably uniform throughout the yeast genome.</a> <em>Mol Syst Biol</em> 6: 353.</li><li>Stanojcic, S., Sollelis, L., Kuk, N., Crobu, L., Balard, Y., Schwob, E., Bastien, P., Pages, M. and Sterkers, Y. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26976742" target="_blank">Single-molecule analysis of DNA replication reveals novel features in the divergent eukaryotes Leishmania and <em>Trypanosoma brucei</em> versus mammalian cells.</a> <em>Sci Rep</em> 6: 23142.</li><li>Tiengwe, C., Marcello, L., Farr, H., Dickens, N., Kelly, S., Swiderski, M., Vaughan, D., Gull, K., Barry, J. D., Bell, S. D. and McCulloch, R. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22840408" target="_blank">Genome-wide analysis reveals extensive functional interaction between DNA replication initiation and transcription in the genome of <em>Trypanosoma brucei</em>.</a> <em>Cell Rep</em> 2(1): 185-197.</li><li>Zhang, Q., Bassetti, F., Gherardi, M. and Lagomarsino, M. C. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28854733" target="_blank">Cell-to-cell variability and robustness in S-phase duration from genome replication kinetics.</a> <em>Nucleic Acids Res</em> 45(14): 8190-8198.<br/></li></ol>