<body>
            <h1>Colorimetric RT-LAMP Methods to Detect Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)</h1>
            <p><strong>Authors:</strong> Gun-Soo Park, Seung-Hwa Baek, Keunbon  Ku, Seong Jun  Kim, Seung Il Kim, Bum-Tae Kim, Jin-Soo  Maeng</p>
            <h2>Abstract</h2>
            <p>Standard diagnostic methods of Coronavirus Disease 2019 (COVID-19) rely on RT-qPCR technique which have limited point-of-care test (POCT) potential due to necessity of dedicated equipment and specialized personnel. LAMP, an isothermal nucleic acid amplification test (NAAT), is a promising technique that may substitute RT-qPCR for POCT of genomic materials. Here, we provide a protocol to perform reverse transcription LAMP targeting SARS-CoV-2. We adopted both real-time fluorescence detection and end-point colorimetric detection approaches. Our protocol would be useful for screening diagnosis of COVID-19 and be a baseline for development of improved POCT NAAT.</p>
            <p><strong>Keywords:</strong> SARS-CoV-2, COVID-19, LAMP, RT-LAMP, Colorimetric detection</p>
            <h2>Background</h2><p>Fast and sensitive detection of SARS-CoV-2, the etiologic agent of COVID-19, is important to control current pandemic situation as it enables early detection, isolation and treatment as well as monitoring screening. Current standard methods for detection of SARS-CoV-2 utilize RT-qPCR as World Health Organization recommends (WHO, 2020) However, proper performance of RT-qPCR diagnosis requires high profile facilities and specialists, often not available in the hospital/sampling places, thus, lacking POCT suitability.<br/><br/>Isothermal amplification methods are developed to accomplish diagnosis by nucleic acid detection in various point-of-care as it can be performed with relatively simple instruments (Niemz <em>et al.</em>, 2011). Loop-mediated isothermal amplification (LAMP) is one of such isothermal NAAT (Notomi <em>et al.</em>, 2000). Amplification by LAMP reaction can be observed through fluorescent dye with real-time PCR instruments (Oscorbin <em>et al.</em>, 2016), or through cost effective colorimetric detection methods (Goto <em>et al.</em>, 2009; Miyamoto <em>et al.</em>, 2015; Tanner <em>et al.</em>, 2015). Indeed reverse transcription LAMP (RT-LAMP) may be considered a promising tool as several other groups are employing this technique for SARS-CoV-2 detection (Yan <em>et al.</em>, 2020; Zhang <em>et al.</em>, 2020).<br/><br/>Here, we present our protocol used to develop RT-LAMP assays targeting SARS-CoV-2. Candidate primer sets are designed using web based tool PrimerExplorerV5 (<a href="https://primerexplorer.jp/e/">https://primerexplorer.jp/e/</a>) targeting SARS-CoV-2 specific regions compare to SARS-CoV and SARS-CoV-2 <em>Nucleocapsid</em>. The primer sets are screened for limit of detection and threshold time measured through real time fluorescent method. Below illustrated protocol includes RNA standard preparation, titration of cultured viral RNA and LAMP reaction which are used during primer screening (Figure 1). Optimized RT-LAMP condition for finally selected two primer sets, namely “Nsp3_1-61” and “Nsp3_2-24”, are representatively provided. We also adapted leuco crystal violet (LCV) colorimetric detection method optimized for Bst 3.0 buffer system (Miyamoto <em>et al.</em>, 2015). A commercially available colorimetric RT-LAMP premix uses pH sensitive dye so that the premix is not compatible with viral genome extraction methods which affect pH of weakly buffered reaction solution (Lalli <em>et al.</em>, 2020). However, LCV method is compatible with such viral genome extraction methods because the color change depends on dsDNA product.<br/><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3804/bioprotoc-10-21-3804-g001.jpg"/><br/><strong>Figure 1. Protocol overview</strong><br/></p>
        </body>

<p class="pptt" id="biaoti38585">## Materials and Reagents</p><ol class="npwtwok"><li><p>LightCycler<sup>Â®</sup> 8-Tube Strips, Clear (Roche, catalog number: 06327672001)</p></li><li><p>LightCycler<sup>Â®</sup> 480 Multiwell Plate 96, Clear (Roche, catalog number: 05102413001)</p></li><li><p>AccuPower PCR PreMix (-dye) kit (Bioneer, catalog number: <a class="blackNoUnder" target="_blank"> K-2016 </a> )</p></li><li><p>Agarose (Bio Basic, catalog number: <a class="blackNoUnder" target="_blank"> D0012 </a> )</p></li><li><p>50x TAE (Biosesang, catalog number: <a class="blackNoUnder" target="_blank"> T2002 </a> )</p></li><li><p>MEGAscript<sup>TM</sup> T7 Transcription Kit (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> AM1334 </a> )</p></li><li><p>Formaldehyde, 37% w/w aq. Soln. (Alfa Aesar, catalog number: <a class="blackNoUnder" target="_blank"> A16163 </a> )</p></li><li><p>10x MOPS (Bioneer, catalog number: <a class="blackNoUnder" target="_blank"> C-9031 </a> )</p></li><li><p>SYBR<sup>TM</sup> Green II RNA Gel Stain (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> S7564 </a> )</p></li><li><p>RiboRuler Low Range RNA Ladder (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> SM1831 </a> )</p></li><li><p>QuantiFluor<sup>Â®</sup> RNA System (Promega, catalog number: <a class="blackNoUnder" target="_blank"> E3310 </a> )</p></li><li><p>1 M Tris-HCl, pH 7.5 (Biosesang, catalog number: <a class="blackNoUnder" target="_blank"> T2016-7.5 </a> )</p></li><li><p>0.5 M EDTA, pH 8 (Biosesang, catalog number: <a class="blackNoUnder" target="_blank"> E2002 </a> )</p></li><li><p>QIAamp Viral RNA Mini Kit (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 52906 </a> )</p></li><li><p>Luna<sup>Â®</sup> Universal Probe One-Step Reaction Mix (NEB, catalog number: <a class="blackNoUnder" target="_blank"> E3006 </a> )</p></li><li><p>DEPC treated water (Biosesang, catalog number: <a class="blackNoUnder" target="_blank"> W2004 </a> )</p></li><li><p>WarmStartÂ® Colorimetric LAMP 2x Master Mix (NEB, catalog number: <a class="blackNoUnder" target="_blank"> E1800 </a> )</p></li><li><p>SYTO<sup>TM</sup> 9 (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> S34854 </a> )</p></li><li><p>Bst 3.0 DNA polymerase (NEB, catalog number: <a class="blackNoUnder" target="_blank"> M0374 </a> , The product includes Isothermal Amplification Buffer II and Magnesium Sulfate solution.)</p></li><li><p>dNTP Mixture, 10 mM ea. (Enzynomics, catalog number: <a class="blackNoUnder" target="_blank"> N002 </a> )</p></li><li><p>SuperScript<sup>TM</sup> IV Reverse Transcriptase (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> 18090050 </a> )</p></li><li><p>Crystal Violet (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> C0775 </a> )</p></li><li><p>Sodium Sulfite (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> S0505 </a> )</p></li><li><p>Î²-Cyclodextrin (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> C4767 </a> )</p></li><li><p>pET21a plasmid containing SARS-CoV-2 <em>Envelope</em> gene (Bionics, custom order)</p></li><li><p>T7 promoter primer (5â²-AATACGACTCACTATAG-3â²) and T7 terminator primer (5â²-GCTAGTTATTGCTCAGCGG-3â²) (Macrogen) (Table 1)</p></li><li><p>Denaturing agarose gel (1%, 100 ml) (see Recipes)</p></li><li><p>20 Î¼M SYTO 9 (see Recipes)</p></li><li><p>5x LCV solution (see Recipes)</p></li></ol><br/><p><strong>Table 1. Primer sequences used for RT-qPCR and RT-LAMP</strong> </p><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201029/20201029005105_8699.jpg" width="600"/> <p class="pptt" id="biaoti38586">### Equipment</p><ol class="npwtwok"><li><p>Mupid-One (ADVANCE, catalog number: <a class="blackNoUnder" target="_blank"> AD160 </a> )</p></li><li><p>ChemiDoc<sup>TM</sup> Touch Imaging System (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1708370 </a> )</p></li><li><p>Quantus<sup>TM</sup> Fluorometer (Promega, catalog number: <a class="blackNoUnder" target="_blank"> E6150 </a> )</p></li><li><p>LightCycler<sup>Â®</sup> 96 Instrument (Roche, catalog number: 0<a class="blackNoUnder" target="_blank"> 5815916001 </a> )</p></li></ol><p class="pptt" id="biaoti38587">### Software</p><ol class="npwtwok"><li><p>Polynucleotide Molecular Weight Calculator (Developed by Andrew Staroscik, <a href="http://scienceprimer.com/nucleotide-molecular-weight-calculator">http://scienceprimer.com/nucleotide-molecular-weight-calculator</a>)</p></li><li><p>LightCycler<sup>Â®</sup> 96 Software (Roche)</p></li></ol><p class="pptt" id="biaoti38588">### Procedure</p><ol class="npwone"><li><p>Preparation of <em>in vitro</em> transcribed RNA standards for SARS-CoV-2 <em>E</em> gene</p><ol class="npwtwok"><li><p>Prepare PCR product as template for in vitro transcription: Run PCR with AccuPower<sup>Â®</sup> PCR PreMix (-dye) kit as manufacturerâs instructions. Use pET21a plasmid containing SARS-CoV-2 <em>Envelope</em> gene as template and T7 promoter primer and T7 terminator primer.</p></li><li><p>Perform <em>in vitro</em> transcription using MEGAscript<sup>TM</sup> T7 Transcription Kit as manufacturerâs instructions.</p></li><li><p>Confirm RNA product by subjecting a portion to formaldehyde-MOPS denaturing agarose gel electrophoresis. The same volume of RNA loading buffer II included in MEGAscript<sup>TM</sup> T7 Transcription Kit was added and heated for 10 min at 65 Â°C for sampling. Post-stain gel with 2x SYBR green II in 1x TAE and obtain image using ChemiDoc<sup>TM</sup> Touch Imaging System (Figure 2).</p><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3804/bioprotoc-10-21-3804-g002.jpg"/><br/><strong>Figure 2. Agarose gel image of <em>in vitro</em> transcribed SARS-CoV-2 <em>E</em> gene RNA. </strong>Two 10-fold serial dilutions are loaded. Ladder sizes are corresponding to 1,000, 800, 600, 400, 300, and 200 bp from up.<br/><br/></li><li>Measure RNA concentration with Quantus<sup>TM</sup> Fluorometer.</li><li>Calculate RNA copy number.<ol class="npwthree"><li>Obtain molecular weight (M.W.) of the transcribed RNA (corresponding sequence: 5â²-GGGGAAUUGUGAGCGGAUAACAAUUCCCCUCUAGAAAUAAUUUUGUUUAACUUUAAGAAGGAGAUAUACAUAUGUACUCAUUCGUUUCGGAAGAGACAGGUACGUUAAUAGUUAAUAGCGUACUUCUUUUUCUUGCUUUCGUGGUAUUCUUGCUAGUUACACUAGCCAUCCUUACUGCGCUUCGAUUGUGUGCGUACUGCUGCAAUAUUGUUAACGUGAGUCUUGUAAAACCUUCUUUUUACGUUUACUCUCGUGUUAAAAAUCUGAAUUCUUCUAGAGUUCCUGAUCUUCUGGUCUAACUCGAGCACCACCACCACCACCACUGAGAUCCGGCUGCUAACAAAGCCCGAAAGGAAGCUGAGUUGGCUGCUGCCACCGCUGAGCAAUAACUAGC-3â²) using Polynucleotide Molecular Weight Calculator, or using following formula where â#Nâ represents the number of each nucleotide.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201029/20201029005719_0734.jpg" width="500"/><br/><br/></li><li>Calculate copy number using following formula: <br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201029/20201029005827_8873.jpg" width="450"/> <br/><br/></li></ol></li><li><p>Make 10-fold serial dilutions in TE buffer as RT-qPCR standards.</p><br/></li></ol></li><li><p>Viral RNA Preparation and Titration Using <em>E</em> gene</p><ol class="npwtwok"><li><p>Extract viral RNA from debris-free culture media with QIAamp Viral RNA Mini Kit following manufacturerâs instructions.</p><p>Aliquot viral RNA extract and store at -80 Â°C until use.</p></li><li><p>Titrate cultured viral RNA copy number by RT-qPCR</p><ol class="npwthree"><li><p>Prepare primer mix as follows: for 100 Î¼l, mix 10 Î¼l of forward and reverse primer, 5 Î¼l of probe, 75 Î¼l of DEPC treated water. Stock concentration of each primer/probe is 100 Î¼M. Names and sequences of primers are listed in Table 1. Primer mix may be prepared in a batch and stored in -20 Â°C for later use.</p></li><li><p>Prepare master mix except template as follow : 7.5 Î¼l of Luna<sup>Â®</sup> Universal Probe One-Step Reaction Mix, 0.75 Î¼l of Luna<sup>Â®</sup> WarmStart<sup>Â®</sup> RT Enzyme Mix, 0.6 Î¼l of primer mix, and 4.15 Î¼l of DEPC treated water per reaction.</p></li><li><p>Aliquot 13 Î¼l of master mix to each well of PCR tube or plate.</p></li><li><p>Add 2 Î¼l of template (viral RNA dilutions, <em>in vitro</em> transcribed RNA standards or no template control) to each well. Close lid of PCR tube or cover film on plate.</p></li><li><p>Run PCR reaction as following temperature and time setting: 10 min at 55 Â°C, 1 min at 95 Â°C, and 45 cycles of 10 s at 95 Â°C and 30 s at 60 Â°C.</p></li><li><p>Copy number of viral RNA can be calculated by substituting threshold cycle value (Ct) to the equation of standard curve of following form:<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201029/20201029010249_2927.jpg" width="450"/> <br/><br/>Standard curve generation and sample copy number calculation can be done using each software coupled with real-time PCR instrument.<br/></p></li></ol></li></ol></li><li><p>RT-LAMP reaction</p><ol class="npwtwok"><li><p>Template viral RNA may prepared from cultured virus or biological specimen from COVID-19 patients. Selection of proper RNA extraction method is up to performers.</p></li><li><p>Prepare 10x LAMP primer mix as follows: for 100 Î¼l, mix 16 Î¼l of FIP/BIP, 2 Î¼l of F3/B3 primers, 4 Î¼l of LF/LB primers and 56 Î¼l of DEPC treated water. Stock concentration of each primers is 100 Î¼M. Sequences of primers are listed in Table 1. The LAMP primer mix may be prepared in a batch and stored in -20 Â°C for later use.</p></li><li><p>Prepare RT-LAMP reaction mix. Master mix except template can be prepared in a batch.</p><ol class="npwthree"><li><p>For LCV method, Reaction components, stock concentration and volume to use are as listed in Table 2.</p></li><li><p>When using WarmStart<sup>Â®</sup> Colorimetric LAMP 2X Master Mix, prepare reaction mix as follows: 2 Î¼l of template, 1.5 Î¼l of 10x LAMP primer mix, 0.3 Î¼l of 20 Î¼M SYTO9, 7.5 Î¼l of LAMP mix, and 3.7 Î¼l of DEPC treated water.</p><br/></li></ol><p><strong>Table 2. Components and volume used (Î¼l) for RT-LAMP with designated LAMP primer sets</strong> </p><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201029/20201029010457_4823.jpg" width="450"/><br/><br/></li><li><p>Aliquot 13 Î¼l of reaction mix and 2 Î¼l of template to each well of PCR tube or plate and close lid or cover film.</p></li><li><p>Run isothermal amplification. Following example settings are for the LightCycler<sup>Â®</sup> 96 instrument SYTO 9 as fluorescent dye.</p><ol class="npwthree"><li><p>Select SYBR Green I and set âIntegration Timeâ to 1 s.</p></li><li><p>Set cycling condition to 59 s at 65 Â°C for Nsp3_1-61 primer set or 69 Â°C for Nsp3_2-24 primer set. Use 65 Â°C for reaction with WarmStart<sup>Â®</sup> Colorimetric LAMP 2x Master Mix. Set 59 s of incubation time and 1 s of data acquisition time for each cycle to make actual incubation time per cycle as 1 min. Set number of cycles to the desired incubation time in minutes (Figure 3).</p><br/></li></ol><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3804/bioprotoc-10-21-3804-g003.jpg"/><br/><strong>Figure 3. Example settings for the primer set Nsp3_1-61 using LightCycler<sup>Â®</sup> 96 software </strong> </p></li></ol></li></ol><p class="pptt" id="biaoti38589">### Data analysis</p><ol class="npwtwok"><li><p>Real-time amplification of LAMP reactions can be observed via LightCycler 96 software or other program accompanied with each qPCR machines. Sigmoidal amplification curve is shown for samples with positive result while no amplification is shown for negative samples or samples with negative results. Non-specific amplification curve may be observed with significantly delayed threshold time and/or shifted melting temperature (Figure 4).</p></li><li><p>For tests for which WarmStart Colorimetric LAMP 2x Master Mix is used, samples with positive result show yellow color while samples with negative result show pink color. For tests to which LCV method is adapted, samples with positive result show clear blue color from blueish transparent color (Figure 5).</p><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3804/bioprotoc-10-21-3804-g004.jpg"/><br/><strong>Figure 4. Amplification curves of RT-LAMP using Nsp3_1-61 primer set on LightCycler 96 system.</strong> Representative amplification curves of one replicate sample with designated RNA copies are shown. NTC, No Template Control.<br/><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3804/bioprotoc-10-21-3804-g005.jpg"/><br/><strong>Figure 5. The color changes of LCV method for RT-LAMP.</strong> LCV color changes to blue by positive RT-LAMP amplification. RT-LAMP reaction was performed for 60 min at 69 Â°C with the half amount of SuperScript IV reverse transcriptase described in the main protocol as the result is from an experiment during reaction optimization. NTC, No Template Control.<p><br/></p><p class="pptt" id="biaoti38590">### Notes</p><ol class="npwtwok"><li><p>Here we targeted <em>E</em> gene for RT-qPCR titration while LAMP primer targets <em>Nps3</em>. Therefore titrated RNA copy number does not perfectly match with LAMP target copy number.</p></li><li><p>Denaturing agarose gel for RNA electrophoresis should be prepared in fume hood as vapor with formaldehyde is toxic. It would be better to do electrophoresis in fume hood too.</p></li><li><p><em>In vitro</em> transcription usually give micrograms of RNA products.</p></li><li><p>Same as or more than seven points with 5 to 10 fold serial dilution are generally recommended for standards of RT-qPCR. We used 10<sup>8</sup> to 10<sup>0</sup> copies per reaction and no template control (NTC) for standard curve generation. Triplicate for each dilution is sufficient.</p></li><li><p>While we obtained about 10<sup>7</sup> copies/Î¼l concentration from cultured virus, the number may differ by incubation period and virus species. Non-infected cells work as the negative control for viral infection yet the derived RNA is not necessarily included for RT-qPCR as no template control can serve as the negative control of RT-qPCR.</p></li><li><p>When preparing a master mix for RT-qPCR and LAMP/RT-LAMP in bulk for a batch of experiment, add extra volume to prevent shortage of the master mix from pipette error or other reasons. For 15 Î¼l reaction described in this protocol, 0.5 to 1 extra amount for &lt; 20 reactions or about 10% extra amount for more reactions is usually enough.</p></li><li><p>For RT-qPCR and real-time RT-LAMP, other qPCR instruments than LightCycler<sup>Â®</sup> 96 Software can be used. For real-time RT-LAMP, the time required for measuring fluorescence should be accounted for setting incubation time and cycle scheme and for the calculation between Ct and threshold time. For example, Bio-Rad CFX systems need rather unpredictable data acquisition time as the systemâs detection unit scan each well one by one.</p></li><li><p>A melting step may be added to the RT-LAMP reaction since non-specific amplifications may be distinguished from their melting curves. Addition of an incubation step for 5 min at 95 Â°C between amplification and melting steps is rather recommended for polymerase inactivation.</p></li><li><p>When using LCV method for colorimetric detection, color varies by temperature. In high temperature, color of reaction mixture may turn to blue without nucleic acid amplification. Color of the mixture will back to normal in room temperature (18-25 Â°C) within 5 min.</p></li><li><p>The sensitivity of LAMP is bottlenecked by formation of âdumbbell intermediateâ and it is usually lower than that of PCR. However, LAMP is highly vulnerable to cross contamination because LAMP yield is high and the bottleneck is not applied. Therefore, we strongly discourage to open reaction tube after LAMP reaction. Otherwise, using separated space for analysis of reaction product (<em>e.g.</em>, agarose gel electrophoresis) is recommended.</p></li><li><p>Typical LAMP optimization includes changing concentration of dNTPs or Mg<sup>2+</sup>. We suggest to test each concentration in combination because dNTPs can chelate Mg<sup>2+</sup>.</p></li><li><p>Since EDTA in TE buffer chelate Mg<sup>2+</sup> ion, using DEPC treated water for RNA elution and dilution may be beneficial when testing larger volume of template.</p></li></ol><p class="pptt" id="biaoti38591">### Recipes</p><ol class="npwtwok"><li><p>Denaturing agarose gel (1%, 100 ml)</p><p>1g agarose</p><p>10 ml 10x MOPS</p><p>18 ml 37% formaldehyde</p><p>72 ml DEPC treated water</p></li><li><p>20 Î¼M SYTO 9</p><p>Dissolve 5 mM SYTO 9 original stock in DEPC treated water with 1/250 dilution. Aliquot in small volume (~100 Î¼l) and store at -20 Â°C</p></li><li><p>5x LCV solution</p><p>10.20 mg Crystal Violet</p><p>378.12 mg Sodium Sulfite</p><p>283.75 mg Î²-Cyclodextrin</p><p>Dissolved in 50 ml DEPC treated water</p></li></ol><p class="pptt" id="biaoti38592">### Acknowledgments</p><p>Authors thank to Korea Centers for Disease Control and Prevention (KCDC) for kind and rapid sharing of isolated strain of SARS-CoV-2. This work was supported by the National Research Council of Science and Technology (NST) grant by the Ministry of Science and ICT (Grant No. CRC-16-01-KRICT). The protocol provided is from our original article previously published (     Park <em>et al.</em>, 2020    ).</p><p class="pptt" id="biaoti38593">### Competing interests</p><p>The authors declare no conflict of interests.</p><p class="pptt" id="biaoti38594">### References</p><ol class="npwtwok"><li>Corman, V. M., Landt, O., Kaiser, M., Molenkamp, R., Meijer, A., Chu, D. K., Bleicker, T., Brunink, S., Schneider, J., Schmidt, M. L., Mulders, D. G., Haagmans, B. L., van der Veer, B., S., den Brink, Wijsman, L., Goderski, G., Romette, J. L., Ellis, J., Zambon, M., Peiris, M., Goossens, H., Reusken, C., Koopmans, M. P. and Drosten, C. (2020). <a href="https://pubmed.ncbi.nlm.nih.gov/31992387/" target="_blank"> Detection of 2019 novel coronavirus(2019-nCoV) by real-time RT-PCR.</a> <em>Euro Surveill</em> 25(3).        doi:         10.2807/1560-7917.ES.2020.25.3.2000045..</li><li>Goto, M., Honda, E., Ogura, A., Nomoto, A. and Hanaki, K. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19317660" target="_blank"> Colorimetric detection of loop-mediated isothermal amplification reaction by using hydroxy naphthol blue.</a> <em>Biotechniques</em> 46(3): 167-172.</li><li>Lalli, M. A., Chen, X., Langmade, S. J., Fronick, C. C., Sawyer, C. S., Burcea, L. C., Fulton, R. S., Heinz, M., Buchser, W. J., Head, R. D., Mitra, R. D. and Milbrandt, J. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/32511508" target="_blank"> Rapid and extraction-free detection of SARS-CoV-2 from saliva with colorimetric LAMP.</a> <em>medRxiv</em> .</li><li>Miyamoto, S., Sano, S., Takahashi, K. and Jikihara, T. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25575759" target="_blank"> Method for colorimetric detection of double-stranded nucleic acid using leuco triphenylmethane dyes.</a> <em>Anal Biochem</em> 47328-33.</li><li>Niemz, A., Ferguson, T. M. and Boyle, D. S. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21377748" target="_blank"> Point-of-care nucleic acid testing for infectious diseases.</a> <em>Trends Biotechnol</em> 29(5): 240-250.</li><li>Notomi, T., Okayama, H., Masubuchi, H., Yonekawa, T., Watanabe, K., Amino, N. and Hase, T. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10871386" target="_blank"> Loop-mediated isothermal amplification of DNA.</a> <em>Nucleic Acids Res</em> 28(12): E63.</li><li>Oscorbin, I. P., Belousova, E. A., Zakabunin, A. I., Boyarskikh, U. A. and Filipenko, M. L. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27401670" target="_blank"> Comparison of fluorescent intercalating dyes for quantitative loop-mediated isothermal amplification(qLAMP).</a> <em>Biotechniques</em> 61(1): 20-25.</li><li>Park, G. S., Ku, K., Baek, S. H., Kim, S. J., Kim, S. I., Kim, B. T. and Maeng, J. S. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/32276051" target="_blank"> Development of Reverse Transcription Loop-Mediated Isothermal Amplification Assays Targeting Severe Acute Respiratory Syndrome Coronavirus 2(SARS-CoV).</a> <em>J Mol Diagn</em> 22(6): 729-735.</li><li>Tanner, N. A., Zhang, Y. and Evans, T. C. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25652028" target="_blank"> Visual detection of isothermal nucleic acid amplification using pH-sensitive dyes.</a> <em>Biotechniques</em> 58(2): 59-68.</li><li>WHO. (2020). Coronavirus disease(COVID-19) technical guidance: Laboratory testing for 2019-nCoV in humans. Retrieved from <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/laboratory-guidance" target="_blank">https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/laboratory-guidance.</a> </li><li>Yan, C., Cui, J., Huang, L., Du, B., Chen, L., Xue, G., Li, S., Zhang, W., Zhao, L., Sun, Y., Yao, H., Li, N., Zhao, H., Feng, Y., Liu, S., Zhang, Q., Liu, D. and Yuan, J. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/32276116" target="_blank"> Rapid and visual detection of 2019 novel coronavirus(SARS-CoV-2) by a reverse transcription loop-mediated isothermal amplification assay.</a> <em>Clin Microbiol Infect</em> 26(6): 773-779.</li><li>Zhang, Y., Odiwuor, N., Xiong, J., Sun, L., Nyaruaba, R. O., Wei, H. and Tanner, N. A. (2020). <a href="https://www.medrxiv.org/content/10.1101/2020.02.26.20028373v1" target="_blank"> Rapid Molecular Detection of SARS-CoV-2(COVID-19) Virus RNA Using Colorimetric LAMP.</a> <em>medRxiv</em> .</li></ol></li></ol>