<body>
            <h1>Advances in Proximity Ligation <em>in situ</em> Hybridization (PLISH)</h1>
            <p><strong>Authors:</strong> Monica Nagendran, Adam M Andruska, Pehr B Harbury, Tushar J Desai</p>
            <h2>Abstract</h2>
            <p>Understanding tissues in the context of development, maintenance and disease requires determining the molecular profiles of individual cells within their native <em>in vivo</em> spatial context. We developed a Proximity Ligation <em>in situ</em> Hybridization technology (PLISH) that enables quantitative measurement of single cell gene expression in intact tissues, which we have now updated. By recording spatial information for every profiled cell, PLISH enables retrospective mapping of distinct cell classes and inference of their <em>in vivo</em> interactions. PLISH has high sensitivity, specificity and signal to noise ratio. It is also rapid, scalable, and does not require expertise in molecular biology so it can be easily adopted by basic and clinical researchers.</p>
            <p><strong>Keywords:</strong> <em>In situ</em> hybridization, Spatial transcriptomics, RNA localization, Single cell analysis</p>
            <h2>Background</h2><p>We recently developed a multiplexed <em>in situ</em> hybridization technique called PLISH (Proximity Ligation <em>in situ</em> Hybridization) (Nagendran <em>et al.</em>, 2018). PLISH is distinct from other existing spatial transcriptomic technologies because it combines high performance, rapid multiplexing, low cost, and technical simplicity (Wilbrey-Clark <em>et al.</em>, 2020). PLISH results can be analyzed by automated calculation of single cell expression profiles in intact cryo- or paraffin-embedded tissue and it is compatible with concurrent immunostaining.<br/><br/>PLISH integrates several molecular biology techniques to provide sensitive and quantitative localization of single RNA molecules in cells and intact tissues. It is broadly accessible to the research community for small- and large-scale applications because it does not employ costly reagents or require up-front investment in specialized equipment. The technology involves rolling circle amplification (RCA) at sites of RNA-catalyzed proximity ligation, which ensures high specificity and signal-to-background with easy discrimination of bright punctal signal from autofluorescence. PLISH has several features that make it adaptable to different experimental goals. First is the ability to ‘tile’ multiple probes along the length of a target RNA. The extent of tiling can be tuned based on the expression level of each transcript, allowing for co-localization of high and low abundance RNAs in a single stain. Second, PLISH's gentle signal 'erasure' step confers a simple multiplexing capability without damaging the tissue sample, allowing for iteration of the fluorescence labeling and imaging steps. Thus, for example, 12 transcripts can be co-localized in a sample with three label-image-erase rounds of four transcripts per cycle following the initial hybridization and RCA. Third, the exceptional signal-to-background of PLISH puncta facilitates automated quantitation of transcripts in each cell using freely available image-processing software packages like CellProfiler. Fourth, the simplicity of PLISH data processing enables researchers without programming expertise to generate and interrogate largescale single cell datasets, and to quantify and spatially resolve known and novel cell classes. Last, PLISH is compatible with fluorescence immunohistochemistry, which enables co-detection of proteins and transcripts in individual cells. This feature broadens the experimental scope of PLISH and on a practical level may be useful for validation of staining patterns.<br/><br/>A PLISH probeset consists of a pair of DNA Hybridization probes (H probes) that target a pre-selected region of an RNA transcript. The right probe (Hybridization Right probe, HR) and left probe (Hybridization Left probe, HL) recognize adjacent sequences in the targeted region (Figure 1). Each H probe comprises a ~20 bp region complementary to the target RNA, and a single stranded overhang at the 3' (HR) or 5' (HL) end. The overhangs are complementary to two oligonucleotides, a Variable Bridge (VB) oligonucleotide and a Common Connector Circle 2.1 (CCC2.1) oligonucleotide. VB and CCC2.1 are designed to hybridize cooperatively with two Hprobe overhangs to form a stable Holliday junction structure, but only at sites where the HR and HL probe have been brought into close proximity by their interaction with the target RNA (Figure 1). A successfully assembled Holliday junction catalyzes ligation of VB and CCC2.1 into a closed circle, which then serves as a substrate for rolling circle amplification (RCA, Figure 1). The RCA amplicons are ~60 kilobase long stretches of single-stranded DNA, consisting of tandem repeats that are complementary to the VB and CCC2.1 sequences. The amplicons are detected with fluorescently tagged VB oligonucleotides (Label probes, LPs). This labeling strategy produces localized fluorescence at the site of the target transcript, which is easily distinguishable from background because of its high intensity and punctal shape (Figure 1). The detection sensitivity can be increased as desired by ‘tiling’ one or more additional H probesets along the length of a given target RNA.<br/><br/>For multiplexed PLISH, a unique VB barcode sequence is inserted into the probesets targeting each different transcript. We have designed and tested 24 distinguishable VB barcodes, supporting up to 24-plex PLISH experiments. Typically, three to five target RNAs are imaged per round, depending on the number of fluorophores the investigator's microscope can resolve. Selective detection of a subset of RNAs is accomplished by 'lighting up' only their VB barcodes with the corresponding label probes. Importantly, the label probes contain deoxyuracil bases, enabling gentle 'erasure' of their fluorescence by enzymatic degradation with uracil DNA glycosylase. New label probes are then applied to detect a second subset of the VB barcodes. The process of label probe hybridization, photo-documentation, and enzymatic erasure is repeated until all the transcripts have been localized, after which expression profiles for every imaged cell can be compiled.<br/><br/></p><ol class="npwone"><li><p>Advantages and Adaptation</p><p>PLISH is a high-performance, rapid, technically simple and low-cost technology for transcriptional profiling of single cells in intact tissue that exploits RNA-templated proximity ligation at Holliday junctions. Our adaptation significantly improves the specificity, sensitivity, and ease of multiplexing with PLISH. Since the initial publication of PLISH (Nagendran <em>et al.</em>, 2018) we have made three important modifications to the protocol (<em>see</em> “<em>Modifications from original PLISH protocol”</em>). In its current iteration, PLISH remains technically simple and adoptable without having to invest in specialized equipment or purchase expensive reagents. It should be straightforward for any researcher to identify optimal conditions for the tissue(s) of interest, for which we include guidelines below, after which they may embark on data collection.</p><br/><br/></li><li>Applications <br/>Molecular profiling of single cells in intact tissues by PLISH has applications that span from co-staining several transcripts and proteins in a single tissue section to high-level multiplexing in a massive number of cells. The former approach facilitates high-resolution mapping of cell-cell interactions, for instance between stem cells and the niche cells that support them, using a small set of curated transcripts (Nabhan <em>et al.</em>, 2018). The latter approach enables unbiased identification and <em>in vivo</em> mapping of molecularly distinct cell populations, which can reveal new and unexpected biology (Nagendran <em>et al.</em>, 2018). PLISH also allows for the incorporation of cytologic features of the profiled cells along with the molecular and spatial characteristics, which may be important in disease states like cancer.<br/><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3808/bioprotoc-10-21-3808-g001.jpg"/><br/><strong>Figure 1. Overview of the PLISH protocol for two targets (Transcript A and Transcript B).</strong> Each pair of H probes in a probeset hybridize at adjacent sites on the target transcript. H probes for Transcript A are designed to work with variable bridge VB01, and probes for Transcript B with variable bridge VB02. CCC2.1 is the common connector circle used for both transcripts. CCC2.1 and the variable bridge hybridize to the single-stranded overhang regions of the H probes. T4 DNA ligase seals the nicks between the variable bridge and CCC2.1 to form a closed circle, and phi29 DNA polymerase copies the circle by rolling circle amplification. Label probes conjugated to fluorophores (LP1 for Transcript A and LP2 for Transcript B) are hybridized to complementary barcodes (colored segments) in the single-stranded RCA amplicons.<br/></li></ol>
        </body>

<p class="pptt" id="biaoti38627">## Materials and Reagents</p><ol class="npwone"><li><p>Consumables</p><ol class="npwtwok"><li><p>Sterile microcentrifuge tubes, 1.5 ml (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 05-408-130 </a> )</p></li><li><p>0.2 Âµm syringe filter (Thermo Scientific, Nalgene, catalog number: <a class="blackNoUnder" target="_blank"> 194-2520 </a> )</p></li><li><p>Microscope slides (Thermo Fisher Superfrost Plus, catalog number: <a class="blackNoUnder" target="_blank"> 4951PLUS4 </a> )</p></li><li><p>Kimwipes (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 06-666 </a> )</p></li><li><p>Rectangular coverslips (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 22X60-1 </a> )</p></li><li><p>Petri dish (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 08-757-099 </a> )</p><br/></li></ol></li><li><p>Reagents</p><ol class="npwtwok"><li><p>Diethyl Pyrocarbonate (DEPC) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> D5758 </a> ); DEPC is a suspected carcinogen. Use a fume hood and wear gloves while working with DEPC</p></li><li><p>Milli Q water</p></li><li><p>Nuclease-free water (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9920 </a> ); can be used instead of preparing DEPC-treated water in the lab (Recipe below)</p></li><li><p>Paraformaldehyde (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P6148 </a> )</p><p><em>Caution: Paraformaldehyde fumes are toxic. Wear gloves and use a fume hood while working with paraformaldehyde.</em> </p></li><li><p>10% Neutral Buffered Formalin (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 5705 </a> )</p><p><em>Caution: Wear gloves and use a fume hood while working with formalin.</em> </p></li><li><p>Sodium Chloride (NaCl) (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP3581 </a> )</p></li><li><p>Sodium Phosphate Dibasic (Na<sub>2</sub>HPO<sub>4</sub>) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S7907 </a> )</p></li><li><p>Sodium Phosphate Monobasic (NaH<sub>2</sub>PO<sub>4</sub>) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S8282 </a> )</p></li><li><p>Dimethyl Sulfoxide (DMSO) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> D8418 </a> )</p></li><li><p>Sodium Trichloroacetic acid (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AC184155000 </a> )</p></li><li><p>Ethanol (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> E7023 </a> )</p></li><li><p>Tween 20 (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P1379 </a> )</p></li><li><p>Formamide (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> F9037 </a> )</p><p><em>Caution: Formamide is a skin, eye, nose and throat irritant and possible teratogen. Wear gloves and use a fume hood while working with formamide.</em> </p></li><li><p>Sodium Citrate Dehydrate (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> S279 </a> )</p></li><li><p>Heparin Sodium salt (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> H3393 </a> )</p></li><li><p>Tris Base (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP152-1 </a> )</p></li><li><p>Glycerol (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 5516 </a> )</p></li><li><p>DAPI (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> D9542 </a> )</p><p><em>Caution: DAPI is a potential carcinogen. Wear gloves while working with DAPI.</em> </p></li><li><p>Triton X-100 (Fisher Scientific, Acros organics, catalog number: <a class="blackNoUnder" target="_blank"> 215680010 </a> )</p></li><li><p>Xylene (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> XS-1 </a> )</p><p><em>Caution: Xylenes affect the central nervous system. Vapors of xylenes are flammable. Use a fume hood and wear gloves while working with this reagent.</em> </p></li><li><p>Ethylenediaminetetraacetic Acid Disodium Salt Dehydrate (EDTA) (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> S312-500 </a> )</p></li><li><p>Sodium Hydroxide (NaOH) (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> S318-500 </a> )</p></li><li><p>Hydrochloric Acid (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A144S-500 </a> )</p><p><em>Caution: Work in a fume hood, wear gloves, and exercise extreme caution when handling concentrated hydrochloric acid.</em> </p></li><li><p>OCT or Optimal Cutting Temperature (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 23-730-571 </a> )</p></li><li><p>dNTP mix, 10 mM each (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> R0192 </a> )</p></li><li><p>ELIMINase (Deacon labs, catalog number: <a class="blackNoUnder" target="_blank"> 1101 </a> )</p></li><li><p>Poly-L-Lysine solution (Sigma Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P4707 </a> )</p></li><li><p>Sterile filtered water for cell culture (Sigma Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> W3500 </a> )</p></li><li><p>Sucrose (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> ICN821713 </a> )</p><br/></li></ol></li><li><p>Enzymes</p><ol class="npwtwok"><li><p>T4 Poly Nucleotide Kinase (T4 PNK) (New England Biolabs catalog number: <a class="blackNoUnder" target="_blank"> M0201L </a> ), store at -20 Â°C</p></li><li><p>Pepsin (Sigma-Aldrich-Roche, catalog number: <a class="blackNoUnder" target="_blank"> 10108057001 </a> ), store at 4 Â°C</p></li><li><p>T4 DNA Ligase (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M0202T </a> ), store at -20 Â°C</p></li><li><p>RNaseIN, Recombinant (Promega, catalog number: <a class="blackNoUnder" target="_blank"> N2511 </a> ), store at -20 Â°C</p></li><li><p>phi29 DNA polymerase, Nxgen (Lucigen, catalog number: <a class="blackNoUnder" target="_blank"> 30221-2 </a> ), store at -20 Â°C</p></li><li><p>USER enzyme (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M5508 </a> ), store at -20 Â°C</p></li><li><p>Uracil Glycosylase Inhibitor (UGI) (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M0281 </a> ), store at -20 Â°C</p><br/></li></ol></li><li><p>Reagent set upâ<em>see corresponding recipe for each reagent in the Recipes section below</em> </p></li></ol><ol class="npwtwok"><strong>Stock Solution Recipes</strong> <li><p>Nuclease-free water</p></li><li><p>3 M Sodium Chloride (NaCl)</p></li><li><p>1 M Disodium Hydrogen Phosphate (Na<sub>2</sub>HPO<sub>4</sub>)</p></li><li><p>1 M Sodium Dihydrogen Phosphate (NaH<sub>2</sub>PO<sub>4</sub>)</p></li><li><p>10x Phosphate Buffered Saline (PBS)</p></li><li><p>1x Phosphate Buffered Saline (PBS) pH 7.4</p></li><li><p>30% Sucrose solution</p></li><li><p>5% Tween-20</p></li><li><p>0.5% Tween-20</p></li><li><p>4% Paraformaldehyde</p></li><li><p>1x Phosphate Buffered Saline with 0.05% Tween-20 (PBST)</p></li><li><p>0.01 M Citric Acid</p></li><li><p>0.01 M Sodium Citrate</p></li><li><p>10% Lithium Dodecyl Sulfate (LDS)</p></li><li><p>0.1 M Hydrocholoric acid (HCl)</p></li><li><p>Pepsin stock solution from powdered enzyme</p></li><li><p>2 M Sodium Trichloroacetate</p></li><li><p>0.5 M Ethylenediaminetetraacetic Acid Disodium Salt Dehydrate (EDTA)</p></li><li><p>1 M Tris pH 7.4</p></li><li><p>1 M Tris pH 8.0</p></li><li><p>20 mg/ml Heparin</p></li><li><p>50% and 80% Glycerol</p></li><li><p>20 mg/ml Bovine Serum Albumin (BSA)</p></li><li><p>20x Saline Sodium Citrate (SSC) pH 7.0</p></li><li><p>DAPI</p></li><li><p>Blocking serum</p><br/></li></ol><ol class="npwtwok"><strong>Buffer Recipes</strong> <li><p>10 mM Citrate buffer with 0.05% LDS</p></li><li><p>TE buffer pH 8.0 contains 100 mM Tris pH 8.0 and 10 mM EDTA pH 8.0</p></li><li><p>Hybridization buffer</p></li><li><p>High Salt Buffer</p></li><li><p>Circle hybridization buffer</p></li><li><p>1x Ligase Buffer</p></li><li><p>1x phi29 DNA polymerase buffer</p></li><li><p>Label probe buffer</p></li></ol><p class="pptt" id="biaoti38628">### Equipment</p><ol class="npwtwok"><li><p>Heat block set at 37 Â°C</p></li><li><p>Water bath set at 100 Â°C</p></li><li><p>Seal chambers (Grace Biolabs, Sigma, catalog number: <a class="blackNoUnder" target="_blank"> GBL621505 </a> )</p></li><li><p>Laser confocal microscope (we acquire the images using a Leica Sp8 confocal microscope)</p></li><li><p>P-2, P-10, P-20, P-200, and P-1000 pipettors</p></li><li><p>Rotating platform (rocker)</p></li><li><p>4 Â°C refrigerator</p></li><li><p>-20 Â°C and -80 Â°C freezer</p></li><li><p>Water bath set at 100 Â°C</p></li><li><p>Hybridization oven or humidified chamber set at 37 Â°C</p></li></ol><p class="pptt" id="biaoti38629">### Software</p><ol class="npwtwok"><li><p>Fiji (<a href="https://fiji.sc/">https://fiji.sc/</a>)</p></li><li><p>Cell Profiler (<a href="https://cellprofiler.org/">https://cellprofiler.org/</a>)</p></li><li><p>MATLAB (<u><a href="https://www.mathworks.com/products/matlab.html">https://www.mathworks.com/products/matlab.html</a></u>)</p></li></ol><p class="pptt" id="biaoti38630">### Procedure</p><ol class="npwone"><li><p>Probe design, sequences and templates</p><ol class="npwtwok"><li><p>Label probes (oligonucleotides conjugated to fluorophores) (Table 1)</p><p>Label probes (LPs) bind to the VB barcodes within the single-stranded RCA amplicon. Each LP consists of 15-20 bases that are identical to the barcode region of the corresponding VB (and are therefore complementary to the RCA amplicon).</p><br/><p><strong>Table 1. Label probe sequences for a 4 channel PLISH</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102003744_5451.jpg" width="450"/> </p><ol class="npwthree"><li><p>Order LPs from a vendor (<em>e.g.</em>, IDT) on the 100 nanomole scale, with 5â conjugation of the fluorophore and HPLC purification. Resuspend the LPs in TE buffer to a stock concentration of 100 ÂµM and store at -20 Â°C. Prepare 10 ÂµM working solutions in nuclease-free water and store at 4 Â°C.</p></li><li><p>Plan your experiment so that you will use Cy5 for the most important target RNA. The simultaneous use of Cy3 and Texas red readouts may require linear unmixing to maximize the accurate discrimination of these two fluorophores, given their overlapping emission spectra. Consult your microscope software manual for instructions on linear unmixing, if available.</p></li><li><p>If you plan to perform PLISH for more than 4 transcripts in a specimen, order LPs with the âTâ nucleotides substituted with deoxyuracil (Table 2). This enables the enzymatic erasure of LPs by USER (Uracil specific excision reagent) after photo-documentation (<em>Refer to detailed protocol in Step C5b, in situ hybridization</em>).</p></li><li>Some LPs have additional nucleotides on the 5â² or 3â² ends to facilitate fluorophore conjugation and/or compatibility with the USER digestion protocol.<br/><br/><p><strong>Table 2. Label probe sequences for iterative PLISH</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102003807_3733.jpg" width="600"/> </p></li></ol></li><li>Circle components<br/>CCC2.1 is the Common Connector Circle<br/>CCC2.15â² ATTCCTGACCTAACAAACATGCGTCTATAGTGGAGCCACATAATTAAACCTGGCTAT 3'<br/>(CCC2.1 is compatible with all Variable Bridges).<p>VB barcodes that are identical to the corresponding label probe are in bold (Table 3).</p><br/><p><strong>Table 3. Variable bridge sequences for PLISH</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102011754_3461.jpg" width="600"/> </p><p>Order CCC2.1 and VB on a 100 nanomole scale as 5â² phosphorylated oligonucleotides with HPLC purification. Prepare 10 ÂµM stock aliquots in nuclease-free water and store at -20 Â°C.</p><br/></li><li><p>Hybridization probe design overview</p><p>Stringent probe design is critical for optimizing transcript detection efficiency and minimizing the risk of false positive signal. A single PLISH probeset consists of one right probe (HR) and one left probe (HL). Each H probe is ~46 bases long, with an ~20 base region that is the reverse complement of the target RNA. The other ~26 base pairs comprise an âoverhangâ sequence that is the reverse complement to the variable bridge (VB) and common connector circle (CCC, <em>see Figure 1</em>) sequences. The design process starts by choosing an RNA transcript of interest from the National Center for Biotechnology Information (NCBI) Nucleotide database. We generally name the probeset using the HUGO (Human Genome Organization) Gene Nomenclature Committee (HGNC) approved gene symbol found at <a href="https://www.genenames.org">https://www.genenames.org</a>.</p><p>  Dinucleotides in the target transcript that can serve as the center of a Holliday junction are first identified, and the surrounding ~40 base RNA sequence is nominated as a candidate probe site. The candidate probe site sequence is then fed to the NCBI Basic Local Alignment Search Tool (BLAST) both to quantify homology to the target RNA and its transcript variants, and to determine if the degree of homology to off-target RNAs is below a minimum threshold (<em>see âCriteria for Off-Target Homologyâ below</em>). When the specificity criteria are met, the reverse complement of the RNA sequence on the 5' side of the junction center (the HL homology region) and the reverse complement of the RNA sequence on the 3' side of the junction center (the HR homology region) are catenated to an appropriate overhang sequence to produce complete H probes. The H probeset is analyzed using an online oligo calculator (refer to Step A3b.xvi) to determine whether the melting temperature of each arm of the prospective junction is acceptable, and to rule out the possibility of hairpin or dimer formation. Between five and ten H probeset pairs are designed per transcript. Cognate VB, CCC2.1 and LP oligonucleotides are ordered along with the H probes.</p><p>  The RNA dinucleotide sequence at the junction center is intended to force the Holliday junction into a conformational isomer that is favorable for PLISH (Miick <em>et al.</em>, 1997; Eichman <em>et al.</em>, 2000; Liu <em>et al.</em>, 2005; Hays <em>et. al.</em>, 2006; Khuu <em>et al.</em>, 2006). In general, we recommend using RNA dinucleotides 5â²-AG-3â² or 5â²-UA-3â² at base pairs 20 and 21 of candidate probe sites. However, requiring the optimal central dinucleotide sequence can overly restrict the number of candidate probe sites for short transcripts, for which the criterion may be relaxed.</p><p>  In our experience, the major challenges in PLISH probe design relate to off-target homology and RNA access, since the tertiary and quaternary structure of endogenous target RNAs likely limits the accessibly of some candidate probe sites more than others. Probe dimerization and hairpin formation are less significant concerns, possibly due to the use of strong denaturants in the hybridization buffer.</p><p>  We include a MATLAB script that was written to automate the above steps (<em>see Appendix for the MATLAB file</em>â<em>PLISH_Probe_Design_BLAST210.m and accompanying instructions âHow_to_use_the_MATLAB_Probe_Design_Script.pdfâ</em>), allowing the user to quickly identify candidate H-probe binding regions that do not have a high level of off-target homology. It also generates lists of H probes with overhang sequences corresponding to a specified VB barcode.</p><p>  Please note that the MATLAB script is not required to perform PLISH. It is included as a potential convenience for some users. We recommend first designing probes by hand and optimizing the entire PLISH protocol prior to moving on to an automated design script.</p><p>  Additional considerations are required prior to using the MATLAB script. First, if PLISH will be  used for a low number of genes (<em>e.g.</em>, less than 10) then it is likely more efficient to design the probes by hand rather than investing in the required ancillary software needed for the MATLAB script. Second, the MATLAB script requires that the user have a stand-alone, or âlocalâ installation of the NCBI BLAST search tool and Refseq database, which may necessitate additional computing resources. It also requires the user to set up the local BLAST search tool and related Refseq databases such that they can be âcalledâ by MATLAB (and other programs) in the userâs operating system (usually Windows). Although we provide basic guidelines on how to install these NCBI tools, it is beyond the scope of this protocol to guide the user through this process, especially as updates and newer versions of BLAST are regularly released by the NCBI. Instead, the user is directed to the NCBIâs current online instructions and help resources for installation of both the search tool and the databases. Finally, updates in MATLAB or the NCBI tools could render the current script obsolete over time.</p><p>  Our MATLAB script determines the specificity of a candidate probe to its probesite based on the number of significant BLAST alignments to genes other than the one from which the probesite is derived. The design of oligonucleotide probes specific for a transcript of interest at a genome-scale is an active area of research, with new tools emerging. One solution, OligoMiner (Beliveau <em>et al.</em>, 2018), provides a python-based pipeline which leverages both next-generation sequencing (NGS) alignment tools (such as Bowtie2) and linear discriminate analysis machine learning to determine target specificity.</p><p>  If electing to use the MATLAB script, it is still recommended that the user manually verify (by entering the region of the RNA sequence targeted by the H-probe into NCBI BLAST) that off-target homology is acceptably low. The script will provide a FASTA file of these sequences which can be entered directly into the online NCBI BLAST search query for this purpose.</p><ol class="npwthree"><li><p>Criteria for off-target homology</p><p>When performing a BLAST query, the settings outlined in Table 4 should be applied. Based on our experience with the hybridization conditions detailed in the new protocol, an alignment with a BlastN bit score of greater than 30 should be considered to have significant homology, irrespective of the number of base pair matches or gaps in the alignment. Note that when parsing the search results, only alignments to RNA (and not genomic DNA) are considered. Also, because the target RNA transcript sequence is used as input for the BLAST query, only "Plus/Plusâ (and not âPlus/Minusâ) strand alignments are relevant. For example, a candidate sequence that has an alignment to an off-target transcript with a "Plus/Plusâ bit score of 35 would be deemed to have unacceptable off-target homology.</p><p><em>Note: Plus/Plus vs. Plus/Minus Convention:</em> </p><ol class="npwfour"><li><p><em>For every Blast search, the result report will include 2 strands: The query (~40 bp candidate RNA sequence) and the subject (the RNA transcript pool database). Metrics describe how good the alignment is between the two strands.</em> </p></li><li><p><em>The Query is always presented as a âPlusâ strand with a 5â-3â orientation.</em> </p></li><li><p><em>Although the subject is also written in 5â²-3â² orientation, BLAST will consider alignments to both the sense mRNA strand (âPlusâ) and the anti-sense mRNA strand (âMinusâ). The BLAST algorithm compares strands for similarity in a base-for-base fashion.</em> </p></li><li><p><em>After performing a BLAST query for a 40 bp query sequence, clicking on the âAlignmentâ tab in the results page will show the pairwise alignment of sequences. The first few BLAST hits generally align to the sequence of the RNA of interest and its isoforms, with identitiesâ40/40 (100%) and StrandâPlus/Plus indicating a base-for-base, query-to-subject alignment to the sense strand. In some of the subsequent hits (possible off-target RNA sequences) the strand alignments may be Plus/Plus (i.e., the query sequence aligns entirely or partly to the sense mRNA subject strand) or Plus/Minus (i.e., the query sequence aligns to the reverse complement of the sense mRNA subject strand). You may ignore hits that have Plus/Minus, because they are reverse complementary to your candidate RNA sequence (i.e., the aligned subject sequence is the same as the H probe sequence so there will be no off-target binding).</em> </p><br/><p><strong>Table 4. Settings for performing NCBI Blast search</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102003853_3598.jpg" width="600"/> </p><br/></li></ol></li><li><p>Detailed H Probe Design Protocol for mRNA</p><ol class="npwfour"><li><p>Navigate to the NCBI Nucleotide Database (<a href="https://www.ncbi.nlm.nih.gov/nucleotide?cmd">https://www.ncbi.nlm.nih.gov/nucleotide?cmd</a>=search).</p></li><li><p>Search for the transcript of interest. On the results page, first filter the results by the taxon of interest using the menu in the top right corner (<em>e.g.</em>, <em>Homo sapiens</em> or <em>Mus musculus</em>) and then by the molecule type using the menu in the left sidebar (<em>RNA</em>).</p></li><li><p>Record the number of transcript variants, predicted or otherwise. Choose the search result for the primary transcript entry or for the lowest numbered transcript variant.</p></li><li><p>Navigate to the bottom of the web page and examine the sequence (denoted by ORIGIN). Paste the full sequence into a word processor document such as Microsoft Word.</p></li><li><p>Use the <em>Control-F</em> âfindâ function to identify suitable central dinucleotides for a prospective Holliday junction. As noted above, these sites are 5â²-AG-3â² or 5â²-UA-3â².</p></li><li><p>Once a Holliday junction site is found, use the <em>Control-C</em> function to âcopyâ the 40 base pair sequence.</p></li><li><p>Open the NCBI Basic Local Alignment Search Tool website (<a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">https://blast.ncbi.nlm.nih.gov/Blast.cgi</a>). Click on the âNucleotide Blastâ button.</p></li><li><p>Paste the candidate sequence in the âEnter Query Sequenceâ Box at the top of the BlastN suite webpage.</p></li><li><p>Next, choose the appropriate database. For <em>Homo sapiens</em> or <em>Mus musculus</em> the curated species-specific genome + transcript databases can be selected from the appropriate dropdown menu located to the right of âDatabaseâ. For other species, or for performing local blast queries, we recommend using the Referenced RNA sequences (Refseq_rna) database. This option will require the user to enter the organism name or Taxid (<a href="https://www.ncbi.nlm.nih.gov/taxonomy">https://www.ncbi.nlm.nih.gov/taxonomy</a>) in the box next to Organism (<em>e.g.</em>, âZebrafishâ Taxid â7955â).</p></li><li><p>Under âProgram Selectionâ to the left of âOptimize forâ, choose the âSomewhat similar sequencesâ or âblastnâ option.</p></li><li><p>Click on the â+â next to Algorithm Parameters.</p></li><li><p>Change the Expect Threshold to 100 and the Word Size to 7. Match and Mismatch scores should be set to 2 and -3, and gap costs should be +5 and +2 for existence and extension, respectively. De-select the check box for Low complexity regions. Finally, click on the bottom BLAST button to run the search.</p></li><li><p>Parse the Blast search results page. First ensure that the transcript of interest and all of its variants appear in the search results. Next, examine the off-target alignments. If the bit score of the alignment is over 30 and the strand result is âPlus/Plusâ, then the candidate sequence has unacceptable off-target homology.<br/>1)If unacceptable off-target homology exists, the candidate sequence should be abandoned in favor of an alternative sequence. Return to Step A3bv. and try a new candidate sequence.<br/>2)If it is known with high confidence that an off-target transcript is not expressed in the tissue or cells that you plan to stain, the probe can still be considered for use in this context.</p></li><li><p>Continue with Steps A3bv. through A3Bxiii. until a pool of 5-10 candidate RNA sequences have been identified.</p></li><li><p>Divide each candidate RNA sequence in half. Base pairs 1-19 form the target site of HL and base pairs 20-40 form the target site of HR. Determine the reverse complements of the two 20 base target site sequences, which will constitute the target homology regions of HL and HR, respectively. Online tools such as âSequence Massagerâ can be used.</p></li><li><p>Analyze the HL and HR homology region sequences for Tm, hairpin formation, self-annealing, and 3â² complementarity. A Tm range of 45-65 Â°C is recommended. Acceptable deltaG values for hairpin formation should be greater than -3 kcal/mole, and for self-dimerization greater than -10 kcal/mole. The calculations can be performed with the online âOligo Calculatorâ, with proprietary packages such as âOligocalcâ (MATLAB) or with the online IDT âOligoAnalyzerâ tool. Since H probe hybridization is conducted in a denaturing high salt buffer, weak self-dimerization or hairpin formation should not necessarily rule out a candidate H probe.</p></li><li><p>An appropriate left overhang sequence is catenated to the 3â² end of the HL homology region sequence, and a corresponding right overhang sequence is catenated to the 5â² end of the HR homology region sequence.</p><p><em>Notes:<br/>1)When designing probes for transcripts that will be imaged together, it is essential to choose VR barcodes that are read out in distinct color channels.<br/>2)Pick at least 5 pairs of probes for abundantly expressed transcripts and 10 pairs for low or moderately expressed transcripts.<br/>3)In some cases, we have observed specificity issues when using probes with partial homology to an off-target RNA. The off-target signal is usually weaker than the on-target signal. Some probes with partial off-target homology may not be problematic, for instance when the off-target RNA is very lowly expressed or absent in the stained tissue.<br/>4)It is not essential that the H probes have exactly 20 bases of homology to the target RNA. Depending on the sequence, 19-25 bases of homology can provide an acceptable melting temperature. Melting temperatures should be at least 9 Â°C above the temperature at which the probes are incubated with the tissue sample, which is 37 Â°C in this protocol.<br/>5)For an initial test experiment, select one abundant transcript (5 pairs of H probes) and one or two RNAs that are expressed at a lower level (10 pairs of H probes) in the tissue you will be staining. It is most useful to select RNAs for which you already know the proper expression pattern, for instance cell type markers, to help gauge which condition provides the optimal sensitivity and specificity in the target tissue.</em> </p><div style="white-space:nowrap;"></div><p><br/></p></li></ol></li><li><p>Preparing individual and pooled stocks of H probes</p><p>Order H probes on the 25 nanomole scale as standard desalted oligonucleotides. Dissolve each H probe in TE buffer (pH 8.0) to make a 100 ÂµM stock solution and store at -20 Â°C. Depending on the number of probe pairs designed for your transcript of interest, prepare pooled stocks as described below (Tables 5-7) and store at -20 Â°C.</p><br/><p><strong>Table 5. Preparing 10 ÂµM pooled stock for 5 pairs of H probes targeting Transcript A in a template for variable bridge VB01</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102003915_5628.jpg" width="600"/> </p><br/><p><strong>Table 6. Preparing 5 ÂµM pooled stock for more than 5 pairs of H probes targeting Transcript A in variable bridge VB01</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102003934_5929.jpg" width="600"/> </p><br/><p><strong>Table 7. Preparing 5 ÂµM pooled stock for 10 pairs of H probes targeting Transcript A in variable bridge VB01</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102003951_9683.jpg" width="600"/> </p></li><li><p>Template for designing H probes</p>Every H probe contains an ~20 base homology region that is complementary to the target RNA. The overhang on H probes consists of a 16-base region (<em>highlighted in red</em>) that binds to the VB oligonucleotide, and an 11-base region (<em>highlighted in blue</em>) that binds to the common connector circle oligonucleotide (Tables 8-10). Table 9 shows a 40 nucleotide region on the mouse Nkx 2.1 transcript that was used to design a pair H probes (Table 10).<p><br/></p><p>When designing H probes for one round of PLISH (1-4 genes), refer to Table 8 for templates. For iterative PLISH (&gt; 4 genes), refer to Table 11.</p><br/><p><strong>Table 8. Template for designing H probes for simultaneous four-plex PLISH</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102004012_3443.jpg" width="600"/> </p><p>HLC2-VB01 denotes an HL probe that works with the CCC2.1 circle and VB01.</p><p>LLLLL and RRRRR designate the regions where the ~20 base target homology sequences are placed.</p><br/><p><strong>Table 9. Identifying a candidate probe site and prospective Holliday junction for the mouse Nkx2.1 transcript (NCBI reference number NM- 009385.3)</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102004031_5323.jpg" width="600"/> </p><p>The dinucleotide at the junction center (AG or UA in the RNA sequence) is highlighted in green.</p><br/><p><strong>Table 10. Designing a pair of H probes targeting the mouse Nkx2.1 transcript (NCBI reference number NM- 009385.3)</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102005011_8268.jpg" width="600"/> </p><p>The two nucleotides complementary to the junction center (AG or UA in the RNA sequence) are highlighted in green. mm, <em>Mus Musculus.</em> </p><br/><p><strong>Table 11. H probe template sequences for iterative PLISH</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102004107_4842.jpg" width="600"/> </p><p>LLLLL and RRRRR indicate the regions where the ~20 bp probe sequences are inserted.</p><br/></li></ol></li></ol></li><li><p>Sample preparation</p><ol class="npwtwok"><li><p>Cells grown on coverslips</p><ol class="npwthree"><li><p>Preparing poly-L-Lysine coated coverslips in a tissue culture hood:</p><ol class="npwfour"><li><p>Wash coverslips in 70% ethanol briefly then transfer them into sterile plates (that have also been pre-washed with 70% ethanol).</p></li><li><p>Rinse once with tissue culture grade sterile water (Pyrogen-free).</p></li><li><p>Incubate the coverslips in the poly-L-Lysine solution for 20 min.</p></li><li><p>Rinse twice with tissue culture grade sterile water.</p></li><li><p>Aspirate off all remaining water and allow the coverslips to air dry for 10 min in the tissue culture hood.</p></li></ol></li><li><p>Splitting cells growing in a flask (perform steps in a tissue culture hood):</p><ol class="npwfour"><li><p>Aspirate off the media and discard it. For a medium-sized flask, wash once in 5 ml of a solution containing 1x PBS and 1 mM EDTA. Alternatively, you may use Trypsin.</p></li><li><p>Cover the cells in 1 ml of a solution containing 1x PBS and 1 mM EDTA, then incubate the cells at 37 Â°C for 2-3 min (no more than 5 min). Gently tap the flask to ensure that the cells have detached into solution (cells should be visibly floating).</p></li><li><p>Pre-dilute the cells at least ten-fold into fresh, pre-warmed media. Perform a final dilution (<em>e.g.</em>, 1:20) with fresh media then gently pipet the cells onto Petri plates containing coverslips until they are submerged. Move Petri plates into 37 Â°C incubator where cells will settle onto coverslips and adhere. Grow the cells until they reach the desired level of confluency.</p></li><li><p>When the cells are ready for analysis, rinse the coverslips in 1x PBS. Fix cells in 3.7% Formaldehyde (prepared in 1x PBS using 37% molecular biology grade Formaldehyde) at room temperature for 20 min.</p></li><li><p>Pre-treat the coverslips with 10 mM Citrate pH 6.0 + 0.05% LDS at 70 Â°C for 30 min, followed by air drying (~5 min).</p></li><li><p>Apply seal chambers. Proceed to the H probe hybridization step.</p></li></ol></li></ol></li><li><p>Tissue sections collected on Superfrost glass slides</p><p>Prepare sections from OCT embedded cryoblocks or from paraffin blocks of Paraformaldehyde (PFA) fixed tissue. To cryoembed adult mouse lungs, immersion fix in 4% PFA with 1x PBS, 5 mM EDTA and 10 mM Citrate at 4 Â°C overnight. After fixation, transfer the lungs into a 30% Sucrose solution prepared in 1x PBS with 5 mM EDTA and 10 mM Citrate, then incubate at 4 Â°C overnight. Embed the lungs in OCT, freeze them, and store the cryoblock at -80 Â°C.</p><ol class="npwthree"><li><p><strong>For OCT blocks:</strong> Cut 14 Âµm sections and collect on clean Superfrost glass slides. Air dry for 5-10 min then proceed to sample treatment step.</p></li><li><p><strong>For FFPE samples:</strong> Cut 10 Âµm paraffin sections and collect them on clean Superfrost glass slides. Air dry the sections for 30 min then bake them at 60 Â°C for 1 h. To deparaffinize, immerse in Xylene twice for 5 min each; immerse in 100% ethanol twice for 5 min each; air dry slides for 5 min. Rehydrate the sections in a graded ethanol series (80% ethanol, 50% ethanol, then 1x PBS for 5 min each). Proceed to the sample treatment step.</p></li></ol></li><li><p>Tissue sections collected on poly-L-Lysine coated coverslips</p><ol class="npwthree"><li><p>Collecting tissue sections on coverslips facilitates iterated labelimage-erase cycles because the seal chambers do not have to be removed for imaging each cycle (which is the case when a slide is used).</p></li><li><p>After imaging, LPs can be erased by USER enzyme (<em>see Step C5b for protocol</em>) then the specimen incubated with a new set of LPs to image another subset of transcripts.</p></li><li><p>Wash the coverslips in 70% ethanol for 2 min and rinse them with nuclease-free water. Incubate the clean coverslips with poly-L-Lysine solution for 20 min with gentle rocking (the solution should uniformly coat the coverslips), remove the excess solution and air dry the coverslips for 10 min or until all of the moisture has evaporated. Store the coverslips protected from dust.</p></li><li><p>Collect tissue sections on poly-L-Lysine coated coverslips. Air dry for 5 min at room temperature. Bake coverslips at 37 Â°C for 15-30 min for strong adhesion of the tissue. Proceed to the sample treatment step (Figure 2).</p><p><em>Note: Consider the following when preparing tissue samples for PLISH.</em> </p></li></ol><ol class="npwfour"><li><p><em>Prior to fixation, trim each tissue piece so its size does not exceed 10 mm.</em> </p></li><li><p><em>During collection, transfer tissues immediately to fixative solution to minimize RNA degradation.</em> </p></li><li><p><em>For cells, only perform the sample pretreatment step 1. Skip the sample pretreatment step 2.</em> </p></li><li><p><em>All reagents should be prepared using nuclease-free water. Maintain a clean work area and use ELIMINase to minimize risk of exposure of cells or tissue to RNases.</em> </p></li></ol></li><li><p>Sample treatment for adult mouse lung</p><ol class="npwthree"><li><p>Sample treatment step 1: Prepare 10 mM Citrate pH 6.0 + 0.05% LDS in nuclease-free water and pour into a slide jar. Pre-warm this buffer in a water bath at 100 Â°C for 5 min. Place the air-dried slides into the jar and incubate for 5 min at 100 Â°C. Use forceps to transfer slides into a jar containing nuclease-free water. Rinse by lifting then re-immersing the slides in the water 4 times. Air dry the slides for 10 min. Use a Kimwipe to remove excess moisture. Apply seal chambers over the tissue. Rehydrate the sections with nuclease-free water for 1 min before proceeding to the next step. Each seal chamber accommodates ~50 Âµl volume (Figure 2).</p></li><li><p>Sample treatment step 2: Prepare 0.025 mg/ml Pepsin (in 0.1 M HCl). Add this solution to the seal chamber wells and incubate at 37 Â°C in a humidified chamber for 5 min. Quickly remove the Pepsin solution and add 1x PBS to inactivate residual Pepsin (Figure 2). Proceed to hybridization of H probes.</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3808/bioprotoc-10-21-3808-g002.jpg"/><br/><strong>Figure 2. An overview of the sample preparation protocol for PLISH.</strong> A fresh tissue specimen is chopped into smaller pieces (for full penetration of the fixative solution) then rapidly fixed. The tissues are then used to prepare OCT embedded cryoblocks or Formalin Fixed Paraffin Embedded (FFPE) blocks. Sections from these blocks are collected on Superfrost slides, deparaffinized (if required), subjected to de-crosslinking by boiling in 10 mM Citrate buffer at 100 Â°C, followed by proteolysis using Pepsin and hybridization of H probes. Cells grown on coverslips or slides are fixed, subjected to de-crosslinking by boiling in 10mM Citrate buffer at 100 Â°C, then H probes are hybridized.</p></li></ol><p><br/></p><p><em>Note: The provided sample treatment is optimized for 14 Âµm adult mouse lung cryosections, but in our experience each tissue requires a customized treatment protocol that varies depending on the tissue under study, the fixation and embedding conditions, and the species of origin. Use the guidelines below to determine the optimal sample treatment conditions for your tissue of interest (Table 12; Figure 3). Note that tissue thickness can also impact on PLISH performance, so if the initial trials are suboptimal, you may wish to empirically test the effect of varying the section thickness.</em> </p><br/><p><strong>Table 12. Range of suggested conditions for determining optimal pretreatment for fixed frozen or FFPE samples of your tissue of interest</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102004338_7807.jpg" width="600"/> </p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3808/bioprotoc-10-21-3808-g003.jpg"/><br/><strong>Figure 3. Utilizing guidelines to identify optimal pretreatment conditions for normal human lung FFPE sections.</strong> A-F, 10 Î¼m FFPE sections of normal human lung were processed for PLISH. The sample pretreatment guidelines mentioned in Table 12 were followed and PLISH was performed to detect human Surfactant protein C (SFTPC, red) using Cy5-conjugated label probes. Samples (A-F) were boiled in 10mM citrate buffer with 0.05% LDS at 100 Â°C for 5 min. This was followed by no pepsin treatment (A) or 5, 10, 20, 30 and 40 min of pepsin (0.1 mg/ml in 0.1 M HCl) digestion at 37 Â°C (B-F). A-C show weak detection of SFTPC. D shows maximum SFTPC detection indicating optimal pretreatment conditions. E and F show diminished SFTPC signal due to excess proteolytic digestion leading to loss of RNA. DAPI, blue. Scale, 50 Î¼m. Images were acquired on a Leica Sp8 confocal microscope using a 633 nm laser (with a laser power level of 10%; PMT detector gain of 750 V and offset ofâ0.5%) and a 25x water immersion objective.</p><p><br/></p></li></ol></li><li><p><em>In situ</em> hybridization</p><p>The example below shows how to perform PLISH for 2 transcripts. Transcript A has 5 pairs of H probes designed to work with the VB01 bridge and LP1 label probe. Transcript B has 10 pairs of H probes designed to work with the VB02 bridge and LP2 label probe.</p><br/><ol class="npwtwok"><li><p>H probe Hybridization (Table 13)</p><p>If using 5 pairs of H probes to target a transcript of interest (as described in Table 5), the final concentration of each probe in the hybridization reaction should be 100 nM. If using more than 5 pairs, the final concentration of each H probe in the hybridization reaction should be 50 nM. Therefore, for Transcript A (targeted using 5 pairs of H probes or a total of 10 probes) the pooled stock is 10 ÂµM (Table 5), the final concentration of each H probe in the hybridization reaction is 100 nM and the total final concentration of all H probes targeting Transcript A is 1,000 nM (100 nM x 10 probes = 1,000 nM). For Transcript B (targeted using 10 pairs of H probes or a total of 20 probes) the pooled stock is 5 ÂµM (Table 7), the final concentration of each H probe in the hybridization reaction is 50 nM and the total final concentration of all H probes targeting Transcript B is 1,000 nM (50 nM x 20 probes = 1,000 nM).</p><br/><p><strong>Table 13. Hybridization buffer with probes targeting Transcript A and Transcript B</strong> <br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102004402_3918.jpg" width="600"/> </p><p>Add the hybridization buffer with probes to the sample and incubate at 37 Â°C in a humid chamber for 2 h. Wash four times for 5 min each in pre-warmed hybridization buffer at 37 Â°C, followed by one wash for 10 min with pre-warmed high salt buffer at 37 Â°C.</p><br/></li><li><p>Assembly of circle components (Table 14)</p><p>Use each phosphorylated oligonucleotide (CCC2.1 and each VB) at a final concentration of 100 nM.</p><br/><p><strong>Table 14. Circle hybridization buffer with phosphorylated circle and bridge oligonucleotides</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102004424_4568.jpg" width="500"/> </p><p>Add circle hybridization buffer containing phosphorylated VB and CCC2.1 to the sample and incubate at 37 Â°C in humid chamber for 60 min. Wash twice for 5 min each in pre-warmed circle hybridization buffer at 37 Â°C. Wash for 2 min in pre-warmed 1x ligase buffer at 37 Â°C.</p><br/></li><li><p>Ligation Reaction (Table 15)</p><br/><p><strong>Table 15. T4 DNA Ligase reaction mix</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102004441_8420.jpg" width="500"/> </p><p>Add the ligation reaction mix to the sample and incubate at 37 Â°C in a humid chamber for 2 h. Wash twice for 5 min each in pre-warmed circle hybridization buffer at 37 Â°C. Wash for 2 min with pre-warmed 1x phi29 DNA polymerase buffer at 37 Â°C.</p><br/></li><li><p>Rolling Circle Amplification (RCA) Reaction (Table 16)</p><br/><p><strong>Table 16. Rolling circle amplification reaction mix</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102004558_2499.jpg" width="500"/> </p><p>Add the RCA reaction mix to the sample and incubate at 37 Â°C in a humid chamber overnight. Wash twice for 5 min each in pre-warmed label probe hybridization buffer at 37 Â°C.</p><br/></li><li><p>Label Probe Hybridization (Table 17)</p><p>Use label probes at a final concentration of 100 nM each.</p><br/><p><strong>Table 17. Label probe hybridization mix</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102004802_1571.jpg" width="300"/> </p><p>Add the label probes to the sample and incubate at 37 Â°C in humid chamber for 30 min. From this point, the protocol diverges to three possible steps, depending on the experimental design.</p><br/></li></ol></li></ol><ol class="npwthree"><li><p>If no additional staining is planned</p><p>Wash samples twice for 5 min each in PBST, then incubate for 5 min with DAPI in PBS (DAPI stock is 1 mg/ml; dilute the stock 1:1,000 in 1x PBS to prepare a working solution). Peel off the seal chamber and mount the sample in 1x PBS if using a water objective, or 80% glycerol if using a glycerol objective.</p></li><li><p>If conducting additional round(s) of label probe hybridization for iterative PLISH</p><p>Wash samples twice for 5 min each in PBST, then incubate for 5 min with DAPI in PBS (the DAPI stock is 1 mg/ml; dilute stock 1:1,000 in 1x PBS to prepare a working solution). Image the specimen in PBS. After photo-documentation of labeled transcripts, proceed with the enzymatic erasure of label probes as detailed below.</p><ol class="npwfour"><li><p>USER digestion (Table 18)</p><br/><p><strong>Table 18. USER digestion mix</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102004818_1878.jpg" width="300"/> </p><p>Incubate at 37 Â°C in a humidified chamber for 20 min. Wash samples twice for 5 min each in PBST.</p><br/></li><li><p>UDG (Uracil DNA Glycosylase) Inhibition (Table 19)</p><br/><p><strong>Table 19. Uracil DNA Glycosylase Inhibitor reaction mix</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201102/20201102004833_7344.jpg" width="300"/> </p><p>Incubate the samples at 37 Â°C in a humidified chamber for 20 min then remove the digestion solution and add the next round of LPs in label probe buffer. Incubate at 37 Â°C in humid chamber for 30 min.</p><br/></li></ol><p><em>Note: If performing more than one label-image-erase cycle, we recommend fixing the samples after the RCA reaction step using 4% Paraformaldehyde for 10 min at room temperature, followed by washing twice for 5 min each in 1x PBS.</em> </p><br/></li><li><p>If conducting combined PLISH and immunohistochemistry:</p><p>Add 5 mM EDTA to all of the solutions used for the immunohistochemistry steps. Following label probe hybridization, wash samples twice for 5 min each in at room temperature. Add blocking solution (50 Âµl/ml serum, 1 Âµl/ml Triton X-100 and 0.03 g/ml BSA in PBS) and incubate for 1 h at room temperature with gentle rocking. Remove blocking solution and add primary antibody (diluted in blocking solution). Incubate samples for 2 h at room temperature or overnight at 4 Â°C with gentle rocking.</p><p>  Wash samples four times for 5 min each in PBST at room temperature. Add secondary antibody (diluted in blocking solution) and incubate samples for 1 h at room temperature with gentle rocking. Wash samples four times for 5 min each in PBST at room temperature, then incubate for 5 min with DAPI in PBS at room temperature. Peel off the seal chamber and mount the sample in 1x PBS if using a water objective, or 80% glycerol if using a glycerol objective.</p><p>  If you plan to store samples for imaging later, seal the coverslips with a thin coat of clear sealant along the edges and allow to dry completely in a dark chamber at room temperature. Store slides at 4 Â°C in a slide box protected from light.</p></li></ol><p class="pptt" id="biaoti38631">### Data analysis</p><p>For detailed information on data analyses, refer the original article (Nagendran <em>et al.</em>, 2018).</p><p><em>Modifications from original PLISH protocol<br/>1)Buffers: Hybridization buffer contains the detergent Lithium Dodecyl Sulfate (LDS). All washes are carried out at 37 Â°C and a high salt buffer wash is included in the steps after H probe hybridization to increase the stringency. Circle and bridge oligonucleotides are added in 20% formamide and 2x SSC buffer with LDS.<br/>2)Due to variability in the performance of circles used in the published version (CC06 performed very well while CC03 and CC01 performed relatively poorly), we implemented a modified version of CC06 for use as a common connector circle (referred to here as CCC2.1) and incorporated the barcode into the bridge oligonucleotides (referred to here as VB). As a result, all oligonucleotide sequences in the new protocol differ from the ones in the original publication. The new protocol utilizes 24 unique variable bridges for iterative (label-image-erase) multiplexing experiments. This modification eliminates the need for up-front testing of each connector circle, which are more prone to variation in performance than bridges.<br/>3)We increased the stringency of the criteria used to select probe sites in order to minimize the risk of cross hybridization to off-target transcripts with partial sequence homology.</em> </p><p class="pptt" id="biaoti38632">### Recipes</p><ol class="npwtwok"><strong>Stock Solution Recipes</strong> <li><p>Nuclease-free water</p><ol class="npwthree"><li><p>Add 1 ml of Diethyl pyrocarbonate (DEPC) to 1 liter of Milli-Q water in a glass bottle</p></li><li><p>Close lid tightly and mix well by inverting or shaking the bottle several times</p></li><li><p>Keep the bottle in a dark room and leave overnight</p></li><li><p>Autoclave on liquid cycle for 30 min to inactivate DEPC</p></li><li><p>Store at room temperature</p></li></ol></li><li><p>3 M Sodium Chloride (NaCl)</p><ol class="npwthree"><li><p>Dissolve 17.53 g of Sodium Chloride in 100 ml nuclease-free water</p></li><li><p>Mix well</p></li><li><p>Autoclave on liquid cycle for 30 min</p></li><li><p>Store at room temperature</p></li></ol></li><li><p>1 M Disodium Hydrogen Phosphate (Na<sub>2</sub>HPO<sub>4</sub>)</p><ol class="npwthree"><li><p>Dissolve 141.96 g of Na<sub>2</sub>HPO<sub>4</sub> in 1,000 ml of nuclease-free water</p></li><li><p>Mix well</p></li><li><p>Store at room temperature</p></li></ol></li><li><p>1 M Sodium Dihydrogen Phosphate (Na<sub>2</sub>HPO<sub>4</sub>)</p><ol class="npwthree"><li><p>Dissolve 119.98 g of Na<sub>2</sub>HPO<sub>4</sub> in 1,000 ml of nuclease-free water</p></li><li><p>Mix well</p></li><li><p>Store at room temperature</p></li></ol></li><li><p>10x Phosphate Buffered Saline (PBS)</p><p>10x PBS contains 1.5 M NaCl and 100 mM NaPO<sub>4</sub> (pH 6.8)</p><p>To make 1,000 of 10x PBS:</p><ol class="npwthree"><li><p>Mix 500 ml of 3 M NaCl</p></li><li><p>46.3 ml of 1 M Na<sub>2</sub>HPO<sub>4</sub> </p></li><li><p>53.7 ml of 1 M Na<sub>2</sub>HPO<sub>4</sub> </p></li><li><p>Make up the volume to 1,000 ml with nuclease-free water</p></li><li><p>Autoclave on liquid cycle for 30 min</p></li></ol><p>When diluted to 1x, the pH of the solution will be 7.4</p><p>Store at room temperature</p></li><li><p>1x Phosphate Buffered Saline (PBS) pH 7.4</p><ol class="npwthree"><li><p>Dilute 10 ml of 10x PBS in 90 ml of nuclease-free water</p></li><li><p>Store at room temperature</p></li></ol></li><li><p>30% Sucrose solution</p><ol class="npwthree"><li><p>Dissolve 30 g of Sucrose in 100 ml of 1x PBS.</p></li><li><p>Mix well.</p></li><li><p>Make 10 ml aliquots and store at -20 Â°C</p></li></ol></li><li><p>5% Tween-20</p><ol class="npwthree"><li><p>Add 500 Âµl of Tween-20 in 10 ml nuclease-free water</p></li><li><p>Mix thoroughly until completely dissolved</p></li><li><p>Store at room temperature</p></li></ol></li><li><p>0. 5% Tween-20</p><ol class="npwthree"><li><p>Add 1,000 Âµl of 5% Tween-20 in 10 ml nuclease-free water</p></li><li><p>Mix thoroughly until completely dissolved</p></li><li><p>Store at room temperature</p></li></ol></li><li><p>4% Paraformaldehyde</p><ol class="npwthree"><li><p>Add 40 g of Paraformaldehyde to 500 ml of nuclease-free water</p></li><li><p>Add NaOH pellets and stir gently on a heating block at ~60 Â°C until the Paraformaldehyde is dissolved</p></li><li><p>Add 100 ml of 10x PBS and allow the mixture to cool to room temperature</p></li><li><p>Adjust the pH to 7.4 with HCl, then adjust the final volume to 1,000 ml with nuclease-free water</p></li><li><p>Store in aliquots at -20 Â°C for several months. Avoid repeated freeze/thawing</p></li></ol><p><em>Caution: Wear gloves, work in a fume hood and exercise extreme caution when preparing paraformaldehyde.</em> </p></li><li><p>1x Phosphate Buffered Saline with 0.05% Tween-20 (PBST)</p><ol class="npwthree"><li><p>Dissolve 100 Âµl of 5% Tween-20 in 10 ml 1x PBS</p></li><li><p>Store at room temperature</p></li></ol></li><li><p>0.01 M Citric Acid</p><ol class="npwthree"><li><p>Dissolve 1.92 g of Citric Acid in 1,000 ml of nuclease-free water</p></li><li><p>Autoclave on liquid cycle for 30 min</p></li><li><p>Store at room temperature</p></li></ol></li><li><p>0.01 M Sodium Citrate</p><ol class="npwthree"><li><p>Dissolve 2.58 g of Sodium Citrate in 1,000 ml of nuclease-free water</p></li><li><p>Autoclave on liquid cycle for 30 min</p></li><li><p>Store at room temperature</p></li></ol></li><li><p>10% Lithium Dodecyl Sulfate (LDS)</p><ol class="npwthree"><li><p>Dissolve 1 g of Lithium Dodecyl Sulfate in 10 ml nuclease-free water</p></li><li><p>Filter sterilize using a 0.2 Âµm syringe filter</p></li><li><p>Store at room temperature</p></li></ol></li><li><p>0.1 M Hydrocholoric acid (HCl)</p><ol class="npwthree"><li><p>Add 817 Âµl of concentrated Hydrochloric Acid (10 N) to 100 ml nuclease-free water</p></li><li><p>Store at room temperature</p></li></ol><p><em>Caution: Wear gloves, work in a fume hood and exercise extreme caution when handling concentrated Hydrocholoric acid.</em> </p></li><li><p>Pepsin stock solution from powdered enzyme</p><ol class="npwthree"><li><p>Weigh 1 mg of Pepsin in a 1.5 ml microcentrifuge tube</p></li><li><p>Add 1 ml of 0.1 M HCl, mix thoroughly</p></li><li><p>This is a 1 mg/ml Pepsin stock solution</p></li><li><p>Dilute it to the required concentration in 0.1 M HCl, for example 1:10 (100 Âµl stock solution in 900 Âµl 0.1M HCl) for making 0.1 mg/ml Pepsin working solution. Use immediately then discard leftover working and stock solutions</p></li></ol><p>Alternatively, you can prepare a 10 mg/ml stock of Pepsin in glycerol and store at -20 Â°C. When needed, dilute this stock to the required concentration in 0.1 M HCl.</p><p><em>Note: Pepsin activity may differ between manufactured batches, so it is recommended to test the enzyme performance when you receive a new batch. Some PLISH users have also used Proteinase K successfully.</em> </p></li><li><p>2 M Sodium Trichloroacetate</p><ol class="npwthree"><li><p>Dissolve 185.37 g of Sodium Trichloroacetate in 500 ml nuclease-free water</p></li><li><p>Autoclave on liquid cycle for 30 min</p></li><li><p>Store at room temperature</p></li></ol></li><li><p>0.5 M Ethylenediaminetetraacetic Acid Disodium Salt Dehydrate (EDTA)</p><p><table border="0" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:50%;"><tbody><tr><td>Reagent</td><td>Amount needed</td></tr><tr><td>EDTA, disodium salt</td><td>9.3 g</td></tr><tr><td>Sodium hydroxide pellets</td><td>Approximately 1 g</td></tr><tr><td>nuclease-free water</td><td>Approximately 50 ml</td></tr></tbody></table></p><p>Add EDTA to approximately 45 ml of nuclease-free water with stirring. Monitor pH and slowly add 1 or 2 Sodium Hydroxide pellets with stirring until the pH reaches 8.0. Make up the volume to 50 ml. Filter sterilize using a 0.2 Âµm syringe filter. Store at room temperature.</p></li><li><p>1 M Tris pH 7.4</p><p><table style="width:50%;"><tbody><tr><td>Reagent</td><td>Amount needed</td></tr><tr><td>Tris Base</td><td>12.114 g</td></tr><tr><td>Concentrated Hydrochloric acid</td><td>7-8 ml</td></tr><tr><td>Nuclease-free water</td><td>Approximately 100 ml</td></tr></tbody></table></p><p>Add Tris Base to approximately 85 ml of nuclease-free water with stirring. Monitor pH and slowly add concentrated Hydrochloric acid with stirring until the pH reaches 7.4. Make up the volume to 100 ml. Mix well. Autoclave on liquid cycle for 30 min. Store at room temperature.</p></li><li><p>1 M Tris pH 8.0</p><p><table style="width:50%;"><tbody><tr><td>Reagent</td><td>Amount needed</td></tr><tr><td>Tris Base</td><td>12.114 g</td></tr><tr><td>Concentrated Hydrochloric acid</td><td>7-8 ml</td></tr><tr><td>Nuclease-free water</td><td>Approximately 100 ml</td></tr></tbody></table></p><p>Add Tris Base to approximately 85 ml of nuclease-free water with stirring. Monitor pH and slowly add concentrated Hydrochloric acid with stirring until the pH reaches 8.0. Make up the volume to 100 ml. Mix well. Autoclave on liquid cycle for 30 min. Store at room temperature.</p></li><li><p>20 mg/ml Heparin</p><ol class="npwthree"><li><p>Dissolve 0.2 g of Heparin Sodium salt in 10 ml nuclease-free water</p></li><li><p>Mix well</p></li><li><p>Filter sterilize using a 0.2 Âµm syringe filter</p></li><li><p>Make aliquots and store at -20 Â°C</p></li></ol></li><li><p>50% and 80% Glycerol</p><ol class="npwthree"><li><p>Add 25 ml Glycerol to 25 ml nuclease-free water (for 50%) or 8 ml Glycerol to 2 ml 1x PBS (for 80%)</p></li><li><p>Mix well until dissolved</p></li><li><p>Store at room temperature</p></li></ol></li><li><p>20 mg/ml Bovine Serum Albumin (BSA)</p><ol class="npwthree"><li><p>Dissolve 0.2 g of bovine serum albumin in 10 ml nuclease-free water</p></li><li><p>Mix well</p></li><li><p>Filter sterilize using a 0.2 Âµm syringe filter</p></li><li><p>Make aliquots and store at -20 Â°C</p></li></ol></li><li><p>20x Saline Sodium Citrate (SSC) pH 7.0</p><p>20x SSC contains 3 M Sodium Chloride in 0.3 M Sodium Citrate</p><p><table style="width:50%;"><tbody><tr><td>Reagent</td><td>Amount needed</td></tr><tr><td>Sodium Chloride</td><td>17.53 g</td></tr><tr><td>Sodium Citrate</td><td>8.82 g</td></tr><tr><td>nuclease-free water</td><td>Approximately 100 ml</td></tr></tbody></table></p><p>Add Sodium Chloride and Sodium Citrate to 80 ml nuclease-free water. Monitor pH and slowly add concentrated Hydrochloric acid with stirring until the pH reaches 7.0. Make up the volume to 100 ml. Mix well. Autoclave on liquid cycle for 30 min. Store at room temperature</p></li><li><p>DAPI</p><p><strong>Stock solution:</strong> </p><p>Dissolve 1 mg of DAPI in 1 ml of Dimethyl Sulfoxide (DMSO). Store protected from light at -20 Â°C</p><p><strong>Working solution:</strong> </p><p>Dissolve 10 Âµl of the stock solution in 10 ml of 1x PBS. Mix well. Make aliquots and store protected from light at 4 Â°C</p><p><strong>Precaution</strong> </p><p>Handle DMSO with care in a fume hood while wearing gloves</p></li><li><p>Blocking serum</p><p>Blocking serum contains 5% Serum, 0.1% Triton X-100, 5 mM EDTA and 3% BSA in 1x PBS</p><p>To make 10 ml of blocking serum, place 0.3 g of BSA in a 15 ml tube. Add 10 ml of nuclease-free water. Mix thoroughly. Add 10 Âµl of Triton X-100, 100 Âµl of 0.5 M EDTA and 500 Âµl of serum. Mix well. Always prepare fresh</p><br/></li></ol><ol class="npwtwok"><strong>Buffer Recipes</strong> <li><p>10 mM Citrate buffer with 0.05% LDS</p><p>Mix 0.01 M Sodium Citrate with 0.01 M Citric acid at a 1:4 ratio by volume. Mix well and adjust the pH to 6.0 using HCl. Store at room temperature.</p></li><li><p>TE buffer pH 8.0 contains 100 mM Tris pH 8.0 and 10 mM EDTA pH 8.0</p><p><table style="width:60%;"><tbody><tr><td>Stock solution</td><td>Volume required</td></tr><tr><td>1 M Tris pH 8.0</td><td>10 ml</td></tr><tr><td>0.5 M EDTA pH 8.0</td><td>2 ml</td></tr><tr><td>Nuclease-free water</td><td>Make up volume to 100 ml</td></tr></tbody></table></p><p>Mix well and store at room temperature.</p></li><li><p>Hybridization buffer</p><p>Contains 1 M Sodium Trichloroacetate, 5 mM EDTA, 50 mM Tris pH 7.4, 0.2 mg/ml Heparin and 0.1% LDS</p><p><table style="width:50%;"><tbody><tr><td>Stock solution</td><td>Volume required</td></tr><tr><td>2 M Sodium trichloroacetate</td><td>500 ml</td></tr><tr><td>0.5 M EDTA</td><td>10 ml</td></tr><tr><td>1 M Tris pH 7.4</td><td>50 ml</td></tr><tr><td>Nuclease-free water</td><td>Make up volume to 1,000 ml</td></tr></tbody></table></p><p>Mix well and store at room temperature. When you start a PLISH stain, take the required volume of buffer, add Heparin (final concentrationâ0.2 mg/ml) and LDS (final concentration 0.1%). Mix well</p><p><em>Note: LDS does not fully dissolve in this buffer at room temperature. Warm buffer in a water bath or oven at 37 Â°C to ensure LDS dissolves completely. Keep this buffer at 37 Â°C to avoid precipitation.</em> </p></li><li><p>High Salt Buffer</p><p>Contains 0.25 M NaCl, 50 mM Tris pH 7.4, 2 mM EDTA, 0.1% LDS in nuclease-free water</p><p><table><tbody><tr><td>Stock solution</td><td>Volume required</td></tr><tr><td>3 M NaCl</td><td>83.33 ml</td></tr><tr><td>1 M Tris pH 7.4</td><td>50 ml</td></tr><tr><td>0.5 M EDTA</td><td>10 ml</td></tr><tr><td>10% LDS</td><td>10 ml</td></tr><tr><td>Nuclease-free water</td><td>Make up volume to 1,000 ml</td></tr></tbody></table></p><p>Mix well and store at room temperature</p></li><li><p>Circle hybridization buffer</p><p>Contains 2x SSC, 20% Formamide, 0.1% LDS and 0.2 mg/ml Heparin in nuclease-free water</p><p><table><tbody><tr><td>Stock solution</td><td>Volume required</td></tr><tr><td>20x SSC</td><td>1 ml</td></tr><tr><td>Formamide</td><td>2 ml</td></tr><tr><td>10% LDS</td><td>100 Âµl</td></tr><tr><td>20 mg/ml Heparin</td><td>100 Âµl</td></tr><tr><td>Nuclease-free water</td><td>Make up volume to 10 ml</td></tr></tbody></table></p><p>Prepare fresh on the day of PLISH, mix well and pre-warm at 37 Â°C before use</p></li><li><p>1x Ligase Buffer</p><p>10x T4 Ligase buffer is supplied along with the T4 DNA ligase enzyme. Dilute the 10x stock buffer in nuclease-free water to 1x. Prepare fresh on the day of PLISH, mix well and pre-warm at 37 Â°C before use</p></li><li><p>1x phi29 DNA polymerase buffer</p><p>10x phi29 DNA polymerase buffer is supplied along with the phi29 DNA polymerase enzyme. Dilute the 10x stock buffer in nuclease-free water to 1x. Prepare fresh on the day of PLISH, mix well and pre-warm at 37 Â°C before use</p></li><li><p>Label probe buffer</p><p>Contains 2x SSC, 20% Formamide and 0.2 mg/ml Heparin in nuclease-free water</p><p><table><tbody><tr><td>Stock solution</td><td>Volume required</td></tr><tr><td>20x SSC</td><td>1 ml</td></tr><tr><td>Formamide</td><td>2 ml</td></tr><tr><td>20 mg/ml Heparin</td><td>100 Âµl</td></tr><tr><td>Nuclease-free water</td><td>Make up volume to 10 ml</td></tr></tbody></table></p><p>Prepare fresh on the day of PLISH, mix well and pre-warm at 37 Â°C before use</p></li></ol><p class="pptt" id="biaoti38633">### Acknowledgments</p><p>We thank Andres Andalon and Yana Kazadaeva for technical assistance. This work was supported by the NIH HubMap Consortium 1UG3HL14562301 (PH, TD), NHLBI 1R01HL14254901 (TD), Chan Zuckerberg Initiative (TD), and a Stanford Discovery Innovation Fund award (PH). TD is the Woods Family Endowed Faculty Scholar in Pediatric Translational Medicine of the Stanford Child Health Research Institute.</p><p class="pptt" id="biaoti38634">### Competing interests</p><p>Tushar J Desai, Pehr B Harbury, and Monica Nagendran have filed a patent âMolecular Profiling Using Proximity Ligation <em>in situ</em> Hybridizationâ (Application #62/475,090).</p><p><br/></p><p class="pptt" id="biaoti38635">### Ethics</p><p>Human subjects: Adult human lung was obtained from Stanford Healthcare with patient informed consent and consent to publish in strict accordance with protocol 18891, approved by the Institutional Review Board Administrative Panel on Human Subjects in Medical Research of Stanford University, in compliance with requirements for protection of human subjects.</p><p class="pptt" id="biaoti38636">### References</p><ol class="npwtwok"><li>Beliveau, B. B., Kishi, J. Y., Nir, G., Sasaki, H. M., Saka, S. K., Nguyen, S. C., Wu, C. and Yin, P. (2018). <a href="https://doi.org/10.1073/pnas.1714530115" target="_blank"> OligoMiner provides a rapid, flexible environment for the design of genome-scale oligonucleotide in situ hybridization probes.</a> <em>Proc Natl Acad Sci U S A</em> 115(10): E2183-E2192.</li><li>Eichman, B. F., Vargason, J. M., Mooers, B. H. and Ho, P. S. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10760268" target="_blank"> The Holliday junction in an inverted repeat DNA sequence: sequence effects on the structure of four-way junctions.</a> <em>Proc Natl Acad Sci U S A</em> 97(8): 3971-3976.</li><li>Hays, F. A., Schirf, V., Ho, P. S. and Demeler, B. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16489738" target="_blank"> Solution formation of Holliday junctions in inverted-repeat DNA sequences.</a> <em>Biochemistry</em> 45(8): 2467-2471.</li><li>Khuu, P. A., Voth, A. R., Hays, F. A. and Ho, P. S. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16575941" target="_blank"> The stacked-X DNA Holliday junction and protein recognition.</a> <em>J Mol Recognit</em> 19(3): 234-242.</li><li>Liu, J., Declais, A. C., McKinney, S. A., Ha, T., Norman, D. G. and Lilley, D. M. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15734649" target="_blank"> Stereospecific effects determine the structure of a four-way DNA junction.</a> <em>Chem Biol</em> 12(2): 217-228.</li><li>Miick, S. M., Fee, R. S., Millar, D. P. and Chazin, W. J. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9256438" target="_blank"> Crossover isomer bias is the primary sequence-dependent property of immobilized Holliday junctions.</a> <em>Proc Natl Acad Sci U S A</em> 94(17): 9080-9084.</li><li>Nabhan, A. N., Brownfield, D. G., Harbury, P. B., Krasnow, M. A. and Desai, T. J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29420258" target="_blank"> Single-cell Wnt signaling niches maintain stemness of alveolar type 2 cells.</a> <em>Science</em> 359(6380): 1118-1123.</li><li>Nagendran, M., Riordan, D. P., Harbury, P. B. and Desai, T. J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29319504" target="_blank"> Automated cell-type classification in intact tissues by single-cell molecular profiling.</a> <em>Elife</em> 7: e30510.</li><li>Wilbrey-Clark, A., Roberts, K. and Teichmann, S. A. (2020). <a href="https://pubmed.ncbi.nlm.nih.gov/32339226/" target="_blank"> Cell atlas technologies and insights into tissue architecture.</a> <em>Biochem J</em> 477(8): 1427-1442.</li></ol>