<body>
            <h1>DigiTAGâa RNA Sequencing Approach to Analyze Transcriptomes of Rare Cell Populations in<em> Drosophila melanogaster</em></h1>
            <p><strong>Authors:</strong> Lisa Landskron, Francois Bonnay, Thomas R. Burkard, JÃ¼rgen A. Knoblich</p>
            <h2>Abstract</h2>
            <p><p>Cell-type specific transcriptional programs underlie the development and maintenance of organs. Not only distinct cell types within a tissue, even cells with supposedly identical cell fates show a high degree of transcriptional heterogeneity. Inevitable, low cell numbers are a major hurdle to study transcriptomes of pure cell populations. Here we describe DigiTAG, a high-throughput method that combines transposase fragmentation and molecular barcoding to retrieve high quality transcriptome data of rare cell types in <em>Drosophila melanogaster</em>. The protocol showcases how DigiTAG can be used to analyse the transcriptome of rare neural stem cells (type II neuroblasts) of <em>Drosophila</em> larval brains, but can also be utilized for other cell types or model systems.</p></p>
            <p><strong>Keywords:</strong> RNA Sequencing, Transcriptome, Low input RNA-seq, <em>Drosophila</em>, DigiTAG, Neuroblasts</p>
            <h2>Background</h2><p><p>Transitions between different cell types during development and tissue homeostasis are orchestrated by a plethora of transcription factors and their induced transcriptional changes. In the last decade, RNA sequencing (RNA-seq) has become the classical approach to measure transcriptional dynamics across the genome (Stark <em>et al.</em>, 2019). Bulk RNA-seq on tissues does not allow to investigate transcriptional networks of different cell populations, specifically those of rare cell types. Thus, RNA-seq protocols that deliver high quality transcriptomes of low input samples are required.</p><br/><p> In <em>Drosophila,</em> limited material often constitutes a hurdle to analyse particular tissues or cell types. This is well illustrated by <em>Drosophila</em> neural stem cells, called neuroblasts (Homem and Knoblich, 2012). Several distinct subpopulations of neuroblasts exist. For example, in the larval <em>Drosophila</em> brain only sixteen type II neuroblasts produce neurons innervating brain regions required for locomotion and sensory processing (Walsh and Doe, 2017). Defects in type II neuroblasts do not only result in aberrant neurogenesis but can also induce tumor growth (Knoblich, 2010).</p><br/><p>To overcome low cell numbers, mutants displaying tumor-like overgrowth of particular neuroblast subpopulations have been used for bulk RNA-seq to enrich for cells-of-interest (Carney <em>et al.</em>, 2012). Although such an approach has given insights into the heterogeneity of neural stem cells, it eliminates the possibility to distinguish between tumor- and cell type-specific changes. Cell isolation by flow cytometry (Berger <em>et al.</em>, 2012; Harzer <em>et al.</em>, 2013) or by robotic single-cell picking (Yang <em>et al.</em>, 2016) have greatly advanced the purity of the isolated cell populations, but manual dissection or the navigation of special equipment still limit cell numbers. Recently, single-cell RNA-seq has been used to map the <em>Drosophila</em> brain (Brunet Avalos <em>et al.</em>, 2019), however such high resolution and complex datasets might not be required to answer many scientific questions and is suboptimal due to costs, non-standard analysis, higher technical noise and loss of lowly expressed genes. Therefore, we present here a RNA-seq protocol for <em>Drosophila</em> neuroblasts that can be performed in any standard molecular biology laboratory using commercially available reagents (Figure 1). First, RNA is isolated and polyA-containing RNA molecules are converted into double-stranded cDNA (Figure 1A). Then the simultaneous cDNA fragmentation and ligation of sequencing adapters to each cDNA molecule by the use of transposases (Figure 1B), is time-efficient and highly reproducible. Primers tagged with unique barcodes are subsequently used to amplify the fragmented cDNA molecules and allow the identification of PCR biases, which can be removed during analysis (Figure 3). The combination of transposon-mediated library preparation with molecular barcoding to quantify the original library molecules rather than their amplicons (Figure 1) ensures high quality of the transcriptome data. Recently, DigiTAG has been successfully used to compare tumor cells to their rare cells-of-origin (Landskron <em>et al.</em>, 2018), identify growth regulators of differently sized cells (Wissel <em>et al.</em>, 2018) and decipher the temporal patterning of transient-amplifying progenitors (Abdusselamoglu <em>et al.</em>, 2019).</p><br/><p><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3809/bioprotoc-10-21-3809-g001.jpg"/><br/><strong>Figure 1. Overview of the DigiTAG workflow.</strong> A. RNA is isolated from the purified cells-of-interest and converted into double-stranded cDNA. To generate the final sequencing library, cDNA molecules are fragmented and amplified (for more detail see B). B. Generation of sequencing libraries from cDNA. By using transposases, cDNA is simultaneously fragmented and adapters (dark grey and brown) are ligated. In the following amplification step these adapters serve as binding sites for two types of primers. Forward primers include the Illumina flow cell attachment sequence P5 (red) together with a sample-specific index sequence (green, allowing to multiplex several RNA-seq samples if desired), whereas reverse primers contain the Illumina flow cell attachment sequence P7 (purple) with a molecule-specific index sequence (rainbow-colored, allowing to barcode each amplified molecule uniquely).</p><br/></p>
        </body>

<p class="pptt" id="biaoti38639">## Materials and Reagents</p><ol class="npwtwok"><li><p>Hardshell 96-Well PCR Plate, low profile, thin wall, skirted, white/clear (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> HSP9601 </a> )</p></li><li><p>Microseal âBâ PCR Plate Sealing Film, adhesive, optical (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> MSB1001 </a> )</p></li><li><p>Refrigerated centrifuge</p></li><li><p>DNA LoBind tubes 1.5 ml (Eppendorf, catalog number: 0030108051)</p></li><li><p>Filter tips for micropipettes</p></li><li><p><em>Drosophila</em> </p></li><li><p>Trizol<sup>TM</sup> LS (ThermoFisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10296028 </a> ) (Storage: 4 Â°C until expiration date)</p></li><li><p>Nuclease-free Water (ThermoFisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9932 </a> ) (Storage: Room temperature until expiration date)</p></li><li><p>Invitrogen<sup>TM</sup> GlycoBlue<sup>TM</sup> Coprecipitant (ThermoFisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9515 </a> ) (Storage: -20 Â°C until expiration date)</p></li><li><p>Chloroform (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C2432 </a> , handle in fume hood!)</p></li><li><p>70% and 75% ethanol, prepared from absolute Ethanol (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 32205 </a> , store at room temperature in fireproof cabinet), diluted with Nuclease-free water.</p></li><li><p>Isopropanol (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 278475 </a> , store at room temperature in fireproof cabinet)</p></li><li><p>Oligo(dT)<sub>20</sub> (50 Î¼M) (ThermoFisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 18418020 </a> ) (Storage: -20 Â°C until expiration date)</p></li><li><p>dNTPs (10 mM each) (ThermoFisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> R0193 </a> ) (Storage: -20 Â°C until expiration date)</p></li><li><p>5x First Strand Buffer, DTT (100 mM) and SuperScript III (200 U/Î¼l) are all part of the kit: SuperScript<sup>TM</sup> III Reverse Transcriptase (ThermoFisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 18080093 </a> ) (Storage: -20 Â°C until expiration date)</p></li><li><p>MgCl<sub>2</sub> (25 mM) (ThermoFisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> R0971 </a> ) (Storage: -20 Â°C until expiration date)</p></li><li><p>RNaseOUT (40 U/Î¼l) (ThermoFisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10777019 </a> ) (Storage: -20 Â°C until expiration date)</p></li><li><p>5x Second Strand Buffer (ThermoFisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10812014 </a> ) (Storage: -20 Â°C until expiration date)</p></li><li><p>RNase H (5 U/Î¼l) (ThermoFisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> EN0202 </a> ) (Storage: -20 Â°C until expiration date)</p></li><li><p>DNA Polymerase I (10 U/Î¼l) (ThermoFisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 18010017 </a> ) (Storage: -20 Â°C until expiration date)</p></li><li><p>AMPure XP Beads (Beckman Coulter, catalog number: <a class="blackNoUnder" target="_blank"> A63880 </a> ) (Storage: 4 Â°C until expiration date)</p></li><li><p>EB Buffer (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 19086 </a> ) (Storage: Room temperature until expiration date)</p></li><li><p>2x Phusion HF master mix (ThermoFisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> F531S </a> ) (Storage: -20 Â°C until expiration date)</p></li><li><p>TDE Tagment DNA buffer and TDE1 Tagment DNA enzyme are all part of the kit: Illumina Tagment DNA TDE1 Enzyme and Buffer Kits (Illumina, catalog number: <a class="blackNoUnder" target="_blank"> 20034197 </a> ) (Storage: -20 Â°C until expiration date)</p></li><li><p>20x Eva Green (Biotum, catalog number: <a class="blackNoUnder" target="_blank"> 31000 </a> ) (Storage: -20 Â°C until expiration date)</p></li><li><p>Nextera PCR, modified Index one and Index two primers are all part of the kit: Nextera DNA Library Preparation Kit (24 samples) (Illumina, catalog number: <a class="blackNoUnder" target="_blank"> FC-121-1030 </a> ) (Storage: -20 Â°C until expiration date)</p></li><li><p>Real-Time Standard 1-4 are all part of the kit: KAPA Real-time Library Amplification kit (KAPA Biosystems, catalog number: <a class="blackNoUnder" target="_blank"> KK2709 </a> ) (Storage: -20 Â°C until expiration date)</p></li></ol><p class="pptt" id="biaoti38640">### Equipment</p><ol class="npwtwok"><li><p>Touch Real-Time PCR Detection System (Bio-Rad, model: <a class="blackNoUnder" target="_blank"> CFX96 </a> )</p></li><li><p><a class="blackNoUnder" target="_blank"> Thermomixer Compact </a> with 1.5 ml block (Eppendorf, model: <a class="blackNoUnder" target="_blank"> Thermomixer Compact </a> )</p></li><li><p>Magnet rack (Fisher Scientific, Invitrogen<sup>TM</sup> DYNAL<sup>TM</sup> DynaMag<sup>TM</sup> Dynabeads<sup>TM</sup> DynaMag-2 Magnet, catalog number: <a class="blackNoUnder" target="_blank"> 10723874 </a> )</p></li><li><p>0.2-2 Î¼l micropipette</p></li><li><p>10-100 Î¼l micropipette</p></li><li><p>20-200 Î¼l micropipette</p></li><li><p>100-1,000 Î¼l micropipette</p></li></ol><p class="pptt" id="biaoti38641">### Software</p><ol class="npwtwok"><li><p>CFX Maestro<sup>TM</sup> Software (Bio-Rad)</p></li></ol><p class="pptt" id="biaoti38642">### Procedure</p><ol class="npwone"><li><p>Isolation of cells-of-interest</p><p>The cell type-of-interest can be isolated through various different methods. For <em>Drosophila</em> neuroblasts, isolation by flow cytometry is recommended. Briefly, <em>Drosophila</em> brain cells as for example type II neuroblasts cells are labeled using fluorophore-expressing driver lines (<em>e.g.</em>, worniu-GAL4, ase-GAL80, UAS-CD8::GFP for type II neuroblasts). Brains are manually dissected and dissociated using enzymatic treatment followed by mechanical disruption. Single-cell suspensions are then subjected to flow cytometry. Neuroblasts identified as a separate cell population, which includes the largest GFP positive cells, are isolated and either stored at -80 Â°C in Trizol<sup>TM</sup> or immediately used for RNA isolation. For a step-by-step protocol see Harzer <em>et al.</em> (2013).</p></li><br/><li><p>RNA isolation</p><ol class="npwtwo"><li>Collect a minimum of 100 and up to 10<sup>5</sup> cells of interest with a minimal estimated volume of ~524 Î¼m<sup>3</sup> (or ~10Î¼m of diameter) per cell (<em>e.g.</em>, type II Neuroblasts) by FACS directly into 750 Î¼l of Trizol<sup>TM</sup> LS in a 1.5 ml DNA LoBind tube.<br/>Importantly, using less than this amount of initial material will lead to a significantly lowered power to detect transcripts. We advise users experimenting on other cell types to adjust the minimal number of cells proportionally in consideration with their cells-of-interest estimated volume.</li><li>Adjust the total volume of each sample to 1 ml with Nuclease-free Water.</li><li>Add 1 Î¼l of GlycoBlue<sup>TM</sup> Coprecipitant (15 mg/ml) to allow low amount of RNA to be recovered from the RNA extraction.</li><li>Mix by inverting the tubes 4-6 times.</li><li>Optional: store your samples at -80 Â°C for up to one month.</li><li>Add 200 Î¼l of ~100% pure chloroform (safety note: handle chloroform in fume hood).</li><li>Vortex the tubes for 15 s.</li><li>Incubate for 2 min at room temperature.</li><li>Centrifuge the samples at maximal speed (~24,000 <em>x g</em>) for 15 min at 4 Â°C in a refrigerated centrifuge.</li><li>Carefully aspirate most of the upper aqueous phase (leave a small fraction to prevent unwanted contamination from the organic phase) preferably with a 20-200 Î¼l micropipette, and transfer it into a new 1.5 ml DNA LoBind tube.</li><li>Add 500 Î¼l of ~100% pure isopropanol to precipitate the RNA.</li><li>Incubate the samples for 15 min at -20 Â°C to optimize RNA precipitation.</li><li>Optional: keep your samples at -20 Â°C for up to one week.</li><li>Vortex the tubes for 15 s.</li><li>Centrifuge the samples at maximal speed (~24,000<em> x g</em>) for 15 min at 4 Â°C in a refrigerated centrifuge.</li><li>Carefully remove all the supernatant.</li><li>Add 1 ml of ice-cold 75% ethanol to improve the purity of the RNA pellet.</li><li>Vortex the tubes for 15 s.</li><li>Centrifuge the samples at maximal speed (~24,000 <em>x g</em>) for 15 min at 4 Â°C in a refrigerated centrifuge.</li><li>Carefully remove all the supernatant.</li><li>Remove the very last drop of liquid from each tube with a micropipette but importantly do not let the RNA pellet dry (it will prevent its subsequent solubilization).</li><li>Resuspend the RNA pellet in 7 Î¼l of EB Buffer. Such a low volume is required to enable subsequent reactions to be performed on the whole sample in the situation when the minimal amount of cells was used for the sampleâs preparation.</li></ol><p><br/></p></li><li><p>Reverse Transcription</p><ol class="npwtwo"><li>In order to synthesize the first strand of cDNA, add the following reagents to the 7 Î¼l of RNA:<br/>1 Î¼l of oligo(dT)<sub>20</sub> (50 Î¼M)<br/>1 Î¼l of dNTPs (10 mM each)<br/>4 Î¼l of 5x First Strand Buffer<br/>4 Î¼l of MgCl<sub>2</sub> (25 mM)<br/>2 Î¼l of DTT (100 mM)<br/>0.5 Î¼l of RNaseOUT (40 U/Î¼l)<br/>0.5 Î¼l of SuperScript III (200 U/Î¼l)</li><li>Mix well and spin down.</li><li>Incubate 50 min at 50 Â°C and then 5 min at 85 Â°C.</li></ol><p><br/></p></li><li><p>Second Strand Synthesis</p><ol class="npwtwo"><li>In order to degrade the RNA strand and synthesize a second strand of cDNA instead, add the following reagents to the first strand reaction and mix well:<br/>5 Î¼l of 5x Second Strand Buffer<br/>3 Î¼l of dNTPs (10 mM each)<br/>2 Î¼l of RNase H (5 U/Î¼l)<br/>1 Î¼l of DNA Polymerase I (10 U/Î¼l)<br/>19 Î¼l of Nuclease-free Water</li><li>Incubate 2.5 h at 16 Â°C. The sample contains now double-stranded cDNA.</li><li>Optional: store your samples at 4 Â°C for up to one day.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li><p>DNA Purification</p><ol class="npwtwo"><li>To purify double-stranded cDNA molecules from the previous reaction mixes, add 90 Î¼l of well-suspended AMPure XP Beads.</li><li>Mix by pipetting 10 times. The reaction turns brownish due to the color of the beads. A homogenous color indicates that the reaction is well mixed (Figure 2).<br/><br/><img alt="" class="layerphoto" height="350" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201027/20201027210215_6847.jpg" width="991"/><br/><strong>Figure 2. DNA purification using magnetic beads.</strong> A. Brownish beads are added to the reaction mix. B. After thoroughly mixing the reaction turns homogenously brown. C. Tube is placed into the magnet rack. D. Shortly after, the reaction mix becomes clear and the brown beads accumulate at the wall closest to the magnet.</li><br/><li> Incubate for 5 min at Room-Temperature.</li><li> Put the samples on a magnet rack. Wait 2-3 min or until all beads have migrated to the wall of the tube closest to the magnet (Bead migration is completed when brown beads accumulate clearly visible on the wall of the tube next to the magnet and the reaction is cleared and thus appears transparent, see also Figure 2).</li><li>Wash twice with 70% ethanol while leaving the tube on the magnet rack. Take care not to disturb the beads. If the pellet of beads is disturbed, wait for 1-2 min to let the beads migrate again to the tubeâs wall close to the magnet.</li><li>After washing remove all ethanol using a pipette and let all ethanol evaporate for 2 min at Room-Temperature.</li><li>Remove samples from magnet and elute the DNA by resuspending the beads in 15.5 Î¼l of EB buffer by pipetting up and down. Wait one min and place the tubes back on the magnetic rack for 2-3 min to allow all beads to migrate to the wall of the tube closest to the magnet. Transfer 15 Î¼l of the completely cleared eluate to a new DNA low binding 1.5 ml tube.<br/></li></ol><p><br/></p></li><li><p>Tagmentation</p><ol class="npwtwo"><li><!--[if gte mso 9]><xml><br /><o:documentproperties><br /><o:version>15.00</o:version><br /></o:documentproperties><br /><o:officedocumentsettings><br /><o:relyonvml /><br /><o:allowpng /><br /></o:officedocumentsettings><br /></xml><![endif]--><!--[if gte mso 9]><xml><br /><w:worddocument><br /><w:view>Normal</w:view><br /><w:zoom>0</w:zoom><br /><w:trackmoves /><br /><w:trackformatting /><br /><w:donotshowrevisions /><br /><w:donotprintrevisions /><br /><w:donotshowcomments /><br /><w:donotshowinsertionsanddeletions /><br /><w:donotshowpropertychanges /><br /><w:punctuationkerning /><br /><w:drawinggridverticalspacing>7.8 ç£</w:drawinggridverticalspacing><br /><w:displayhorizontaldrawinggridevery>0</w:displayhorizontaldrawinggridevery><br /><w:displayverticaldrawinggridevery>2</w:displayverticaldrawinggridevery><br /><w:validateagainstschemas /><br /><w:saveifxmlinvalid>false</w:saveifxmlinvalid><br /><w:ignoremixedcontent>false</w:ignoremixedcontent><br /><w:alwaysshowplaceholdertext>false</w:alwaysshowplaceholdertext><br /><w:donotpromoteqf /><br /><w:lidthemeother>EN-US</w:lidthemeother><br /><w:lidthemeasian>ZH-CN</w:lidthemeasian><br /><w:lidthemecomplexscript>AR-SA</w:lidthemecomplexscript><br /><w:compatibility><br /><w:spaceforul /><br /><w:balancesinglebytedoublebytewidth /><br /><w:donotleavebackslashalone /><br /><w:ultrailspace /><br /><w:donotexpandshiftreturn /><br /><w:adjustlineheightintable /><br /><w:breakwrappedtables /><br /><w:snaptogridincell /><br /><w:wraptextwithpunct /><br /><w:useasianbreakrules /><br /><w:dontgrowautofit /><br /><w:splitpgbreakandparamark /><br /><w:enableopentypekerning /><br /><w:dontflipmirrorindents /><br /><w:overridetablestylehps /><br /><w:usefelayout /><br /></w:compatibility><br /><m:mathpr><br /><m:mathfont m:val="Cambria Math" /><br /><m:brkbin m:val="before" /><br /><m:brkbinsub m:val="&#45;-" /><br /><m:smallfrac m:val="off" /><br /><m:dispdef /><br /><m:lmargin m:val="0" /><br /><m:rmargin m:val="0" /><br /><m:defjc m:val="centerGroup" /><br /><m:wrapindent m:val="1440" /><br /><m:intlim m:val="subSup" /><br /><m:narylim m:val="undOvr" /><br /></m:mathpr></w:worddocument><br /></xml><![endif]--><!--[if gte mso 9]><xml><br /><w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="false"<br />DefSemiHidden="false" DefQFormat="false" DefPriority="99"<br />LatentStyleCount="371"><br /><w:lsdexception locked="false" priority="0" qformat="true" name="Normal" /><br /><w:lsdexception locked="false" priority="9" qformat="true" name="heading 1" /><br /><w:LsdException Locked="false" Priority="9" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="heading 2"/><br /><w:LsdException Locked="false" Priority="9" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="heading 3"/><br /><w:LsdException Locked="false" Priority="9" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="heading 4"/><br /><w:LsdException Locked="false" Priority="9" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="heading 5"/><br /><w:LsdException Locked="false" Priority="9" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="heading 6"/><br /><w:LsdException Locked="false" Priority="9" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="heading 7"/><br /><w:LsdException Locked="false" Priority="9" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="heading 8"/><br /><w:LsdException Locked="false" Priority="9" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="heading 9"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 5"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 6"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 7"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 8"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 9"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 1"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 2"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 3"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 4"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 5"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 6"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 7"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 8"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 9"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Normal Indent"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="footnote text"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="annotation text"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="header"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="footer"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index heading"/><br /><w:LsdException Locked="false" Priority="35" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="caption"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="table of figures"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="envelope address"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="envelope return"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="footnote reference"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="annotation reference"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="line number"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="page number"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="endnote reference"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="endnote text"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="table of authorities"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="macro"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="toa heading"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Bullet"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Number"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List 5"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Bullet 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Bullet 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Bullet 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Bullet 5"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Number 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Number 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Number 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Number 5"/><br /><w:lsdexception locked="false" priority="10" qformat="true" name="Title" /><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Closing"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Signature"/><br /><w:LsdException Locked="false" Priority="1" SemiHidden="true"<br />UnhideWhenUsed="true" Name="Default Paragraph Font"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Body Text"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Body Text Indent"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Continue"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Continue 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Continue 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Continue 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Continue 5"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Message Header"/><br /><w:lsdexception locked="false" priority="11" qformat="true" name="Subtitle" /><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Salutation"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Date"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Body Text First Indent"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Body Text First Indent 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Note Heading"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Body Text 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Body Text 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Body Text Indent 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Body Text Indent 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Block Text"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Hyperlink"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="FollowedHyperlink"/><br /><w:lsdexception locked="false" priority="22" qformat="true" name="Strong" /><br /><w:lsdexception locked="false" priority="20" qformat="true" name="Emphasis" /><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Document Map"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Plain Text"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="E-mail Signature"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Top of Form"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Bottom of Form"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Normal (Web)"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Acronym"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Address"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Cite"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Code"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Definition"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Keyboard"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Preformatted"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Sample"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Typewriter"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Variable"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Normal Table"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="annotation subject"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="No List"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Outline List 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Outline List 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Outline List 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Simple 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Simple 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Simple 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Classic 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Classic 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Classic 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Classic 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Colorful 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Colorful 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Colorful 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Columns 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Columns 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Columns 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Columns 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Columns 5"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Grid 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Grid 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Grid 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Grid 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Grid 5"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Grid 6"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Grid 7"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Grid 8"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table List 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table List 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table List 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table List 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table List 5"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table List 6"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table List 7"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table List 8"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table 3D effects 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table 3D effects 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table 3D effects 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Contemporary"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Elegant"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Professional"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Subtle 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Subtle 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Web 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Web 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Web 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Balloon Text"/><br /><w:lsdexception locked="false" priority="39" name="Table Grid" /><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Theme"/><br /><w:lsdexception locked="false" semihidden="true" name="Placeholder Text" /><br /><w:lsdexception locked="false" priority="1" qformat="true" name="No Spacing" /><br /><w:lsdexception locked="false" priority="60" name="Light Shading" /><br /><w:lsdexception locked="false" priority="61" name="Light List" /><br /><w:lsdexception locked="false" priority="62" name="Light Grid" /><br /><w:lsdexception locked="false" priority="63" name="Medium Shading 1" /><br /><w:lsdexception locked="false" priority="64" name="Medium Shading 2" /><br /><w:lsdexception locked="false" priority="65" name="Medium List 1" /><br /><w:lsdexception locked="false" priority="66" name="Medium List 2" /><br /><w:lsdexception locked="false" priority="67" name="Medium Grid 1" /><br /><w:lsdexception locked="false" priority="68" name="Medium Grid 2" /><br /><w:lsdexception locked="false" priority="69" name="Medium Grid 3" /><br /><w:lsdexception locked="false" priority="70" name="Dark List" /><br /><w:lsdexception locked="false" priority="71" name="Colorful Shading" /><br /><w:lsdexception locked="false" priority="72" name="Colorful List" /><br /><w:lsdexception locked="false" priority="73" name="Colorful Grid" /><br /><w:lsdexception locked="false" priority="60" name="Light Shading Accent 1" /><br /><w:lsdexception locked="false" priority="61" name="Light List Accent 1" /><br /><w:lsdexception locked="false" priority="62" name="Light Grid Accent 1" /><br /><w:lsdexception locked="false" priority="63" name="Medium Shading 1 Accent 1" /><br /><w:lsdexception locked="false" priority="64" name="Medium Shading 2 Accent 1" /><br /><w:lsdexception locked="false" priority="65" name="Medium List 1 Accent 1" /><br /><w:lsdexception locked="false" semihidden="true" name="Revision" /><br /><w:LsdException Locked="false" Priority="34" QFormat="true"<br />Name="List Paragraph"/><br /><w:lsdexception locked="false" priority="29" qformat="true" name="Quote" /><br /><w:LsdException Locked="false" Priority="30" QFormat="true"<br />Name="Intense Quote"/><br /><w:lsdexception locked="false" priority="66" name="Medium List 2 Accent 1" /><br /><w:lsdexception locked="false" priority="67" name="Medium Grid 1 Accent 1" /><br /><w:lsdexception locked="false" priority="68" name="Medium Grid 2 Accent 1" /><br /><w:lsdexception locked="false" priority="69" name="Medium Grid 3 Accent 1" /><br /><w:lsdexception locked="false" priority="70" name="Dark List Accent 1" /><br /><w:lsdexception locked="false" priority="71" name="Colorful Shading Accent 1" /><br /><w:lsdexception locked="false" priority="72" name="Colorful List Accent 1" /><br /><w:lsdexception locked="false" priority="73" name="Colorful Grid Accent 1" /><br /><w:lsdexception locked="false" priority="60" name="Light Shading Accent 2" /><br /><w:lsdexception locked="false" priority="61" name="Light List Accent 2" /><br /><w:lsdexception locked="false" priority="62" name="Light Grid Accent 2" /><br /><w:lsdexception locked="false" priority="63" name="Medium Shading 1 Accent 2" /><br /><w:lsdexception locked="false" priority="64" name="Medium Shading 2 Accent 2" /><br /><w:lsdexception locked="false" priority="65" name="Medium List 1 Accent 2" /><br /><w:lsdexception locked="false" priority="66" name="Medium List 2 Accent 2" /><br /><w:lsdexception locked="false" priority="67" name="Medium Grid 1 Accent 2" /><br /><w:lsdexception locked="false" priority="68" name="Medium Grid 2 Accent 2" /><br /><w:lsdexception locked="false" priority="69" name="Medium Grid 3 Accent 2" /><br /><w:lsdexception locked="false" priority="70" name="Dark List Accent 2" /><br /><w:lsdexception locked="false" priority="71" name="Colorful Shading Accent 2" /><br /><w:lsdexception locked="false" priority="72" name="Colorful List Accent 2" /><br /><w:lsdexception locked="false" priority="73" name="Colorful Grid Accent 2" /><br /><w:lsdexception locked="false" priority="60" name="Light Shading Accent 3" /><br /><w:lsdexception locked="false" priority="61" name="Light List Accent 3" /><br /><w:lsdexception locked="false" priority="62" name="Light Grid Accent 3" /><br /><w:lsdexception locked="false" priority="63" name="Medium Shading 1 Accent 3" /><br /><w:lsdexception locked="false" priority="64" name="Medium Shading 2 Accent 3" /><br /><w:lsdexception locked="false" priority="65" name="Medium List 1 Accent 3" /><br /><w:lsdexception locked="false" priority="66" name="Medium List 2 Accent 3" /><br /><w:lsdexception locked="false" priority="67" name="Medium Grid 1 Accent 3" /><br /><w:lsdexception locked="false" priority="68" name="Medium Grid 2 Accent 3" /><br /><w:lsdexception locked="false" priority="69" name="Medium Grid 3 Accent 3" /><br /><w:lsdexception locked="false" priority="70" name="Dark List Accent 3" /><br /><w:lsdexception locked="false" priority="71" name="Colorful Shading Accent 3" /><br /><w:lsdexception locked="false" priority="72" name="Colorful List Accent 3" /><br /><w:lsdexception locked="false" priority="73" name="Colorful Grid Accent 3" /><br /><w:lsdexception locked="false" priority="60" name="Light Shading Accent 4" /><br /><w:lsdexception locked="false" priority="61" name="Light List Accent 4" /><br /><w:lsdexception locked="false" priority="62" name="Light Grid Accent 4" /><br /><w:lsdexception locked="false" priority="63" name="Medium Shading 1 Accent 4" /><br /><w:lsdexception locked="false" priority="64" name="Medium Shading 2 Accent 4" /><br /><w:lsdexception locked="false" priority="65" name="Medium List 1 Accent 4" /><br /><w:lsdexception locked="false" priority="66" name="Medium List 2 Accent 4" /><br /><w:lsdexception locked="false" priority="67" name="Medium Grid 1 Accent 4" /><br /><w:lsdexception locked="false" priority="68" name="Medium Grid 2 Accent 4" /><br /><w:lsdexception locked="false" priority="69" name="Medium Grid 3 Accent 4" /><br /><w:lsdexception locked="false" priority="70" name="Dark List Accent 4" /><br /><w:lsdexception locked="false" priority="71" name="Colorful Shading Accent 4" /><br /><w:lsdexception locked="false" priority="72" name="Colorful List Accent 4" /><br /><w:lsdexception locked="false" priority="73" name="Colorful Grid Accent 4" /><br /><w:lsdexception locked="false" priority="60" name="Light Shading Accent 5" /><br /><w:lsdexception locked="false" priority="61" name="Light List Accent 5" /><br /><w:lsdexception locked="false" priority="62" name="Light Grid Accent 5" /><br /><w:lsdexception locked="false" priority="63" name="Medium Shading 1 Accent 5" /><br /><w:lsdexception locked="false" priority="64" name="Medium Shading 2 Accent 5" /><br /><w:lsdexception locked="false" priority="65" name="Medium List 1 Accent 5" /><br /><w:lsdexception locked="false" priority="66" name="Medium List 2 Accent 5" /><br /><w:lsdexception locked="false" priority="67" name="Medium Grid 1 Accent 5" /><br /><w:lsdexception locked="false" priority="68" name="Medium Grid 2 Accent 5" /><br /><w:lsdexception locked="false" priority="69" name="Medium Grid 3 Accent 5" /><br /><w:lsdexception locked="false" priority="70" name="Dark List Accent 5" /><br /><w:lsdexception locked="false" priority="71" name="Colorful Shading Accent 5" /><br /><w:lsdexception locked="false" priority="72" name="Colorful List Accent 5" /><br /><w:lsdexception locked="false" priority="73" name="Colorful Grid Accent 5" /><br /><w:lsdexception locked="false" priority="60" name="Light Shading Accent 6" /><br /><w:lsdexception locked="false" priority="61" name="Light List Accent 6" /><br /><w:lsdexception locked="false" priority="62" name="Light Grid Accent 6" /><br /><w:lsdexception locked="false" priority="63" name="Medium Shading 1 Accent 6" /><br /><w:lsdexception locked="false" priority="64" name="Medium Shading 2 Accent 6" /><br /><w:lsdexception locked="false" priority="65" name="Medium List 1 Accent 6" /><br /><w:lsdexception locked="false" priority="66" name="Medium List 2 Accent 6" /><br /><w:lsdexception locked="false" priority="67" name="Medium Grid 1 Accent 6" /><br /><w:lsdexception locked="false" priority="68" name="Medium Grid 2 Accent 6" /><br /><w:lsdexception locked="false" priority="69" name="Medium Grid 3 Accent 6" /><br /><w:lsdexception locked="false" priority="70" name="Dark List Accent 6" /><br /><w:lsdexception locked="false" priority="71" name="Colorful Shading Accent 6" /><br /><w:lsdexception locked="false" priority="72" name="Colorful List Accent 6" /><br /><w:lsdexception locked="false" priority="73" name="Colorful Grid Accent 6" /><br /><w:LsdException Locked="false" Priority="19" QFormat="true"<br />Name="Subtle Emphasis"/><br /><w:LsdException Locked="false" Priority="21" QFormat="true"<br />Name="Intense Emphasis"/><br /><w:LsdException Locked="false" Priority="31" QFormat="true"<br />Name="Subtle Reference"/><br /><w:LsdException Locked="false" Priority="32" QFormat="true"<br />Name="Intense Reference"/><br /><w:lsdexception locked="false" priority="33" qformat="true" name="Book Title" /><br /><w:LsdException Locked="false" Priority="37" SemiHidden="true"<br />UnhideWhenUsed="true" Name="Bibliography"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="TOC Heading"/><br /><w:lsdexception locked="false" priority="41" name="Plain Table 1" /><br /><w:lsdexception locked="false" priority="42" name="Plain Table 2" /><br /><w:lsdexception locked="false" priority="43" name="Plain Table 3" /><br /><w:lsdexception locked="false" priority="44" name="Plain Table 4" /><br /><w:lsdexception locked="false" priority="45" name="Plain Table 5" /><br /><w:lsdexception locked="false" priority="40" name="Grid Table Light" /><br /><w:lsdexception locked="false" priority="46" name="Grid Table 1 Light" /><br /><w:lsdexception locked="false" priority="47" name="Grid Table 2" /><br /><w:lsdexception locked="false" priority="48" name="Grid Table 3" /><br /><w:lsdexception locked="false" priority="49" name="Grid Table 4" /><br /><w:lsdexception locked="false" priority="50" name="Grid Table 5 Dark" /><br /><w:lsdexception locked="false" priority="51" name="Grid Table 6 Colorful" /><br /><w:lsdexception locked="false" priority="52" name="Grid Table 7 Colorful" /><br /><w:LsdException Locked="false" Priority="46"<br />Name="Grid Table 1 Light Accent 1"/><br /><w:lsdexception locked="false" priority="47" name="Grid Table 2 Accent 1" /><br /><w:lsdexception locked="false" priority="48" name="Grid Table 3 Accent 1" /><br /><w:lsdexception locked="false" priority="49" name="Grid Table 4 Accent 1" /><br /><w:lsdexception locked="false" priority="50" name="Grid Table 5 Dark Accent 1" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="Grid Table 6 Colorful Accent 1"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="Grid Table 7 Colorful Accent 1"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="Grid Table 1 Light Accent 2"/><br /><w:lsdexception locked="false" priority="47" name="Grid Table 2 Accent 2" /><br /><w:lsdexception locked="false" priority="48" name="Grid Table 3 Accent 2" /><br /><w:lsdexception locked="false" priority="49" name="Grid Table 4 Accent 2" /><br /><w:lsdexception locked="false" priority="50" name="Grid Table 5 Dark Accent 2" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="Grid Table 6 Colorful Accent 2"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="Grid Table 7 Colorful Accent 2"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="Grid Table 1 Light Accent 3"/><br /><w:lsdexception locked="false" priority="47" name="Grid Table 2 Accent 3" /><br /><w:lsdexception locked="false" priority="48" name="Grid Table 3 Accent 3" /><br /><w:lsdexception locked="false" priority="49" name="Grid Table 4 Accent 3" /><br /><w:lsdexception locked="false" priority="50" name="Grid Table 5 Dark Accent 3" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="Grid Table 6 Colorful Accent 3"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="Grid Table 7 Colorful Accent 3"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="Grid Table 1 Light Accent 4"/><br /><w:lsdexception locked="false" priority="47" name="Grid Table 2 Accent 4" /><br /><w:lsdexception locked="false" priority="48" name="Grid Table 3 Accent 4" /><br /><w:lsdexception locked="false" priority="49" name="Grid Table 4 Accent 4" /><br /><w:lsdexception locked="false" priority="50" name="Grid Table 5 Dark Accent 4" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="Grid Table 6 Colorful Accent 4"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="Grid Table 7 Colorful Accent 4"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="Grid Table 1 Light Accent 5"/><br /><w:lsdexception locked="false" priority="47" name="Grid Table 2 Accent 5" /><br /><w:lsdexception locked="false" priority="48" name="Grid Table 3 Accent 5" /><br /><w:lsdexception locked="false" priority="49" name="Grid Table 4 Accent 5" /><br /><w:lsdexception locked="false" priority="50" name="Grid Table 5 Dark Accent 5" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="Grid Table 6 Colorful Accent 5"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="Grid Table 7 Colorful Accent 5"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="Grid Table 1 Light Accent 6"/><br /><w:lsdexception locked="false" priority="47" name="Grid Table 2 Accent 6" /><br /><w:lsdexception locked="false" priority="48" name="Grid Table 3 Accent 6" /><br /><w:lsdexception locked="false" priority="49" name="Grid Table 4 Accent 6" /><br /><w:lsdexception locked="false" priority="50" name="Grid Table 5 Dark Accent 6" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="Grid Table 6 Colorful Accent 6"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="Grid Table 7 Colorful Accent 6"/><br /><w:lsdexception locked="false" priority="46" name="List Table 1 Light" /><br /><w:lsdexception locked="false" priority="47" name="List Table 2" /><br /><w:lsdexception locked="false" priority="48" name="List Table 3" /><br /><w:lsdexception locked="false" priority="49" name="List Table 4" /><br /><w:lsdexception locked="false" priority="50" name="List Table 5 Dark" /><br /><w:lsdexception locked="false" priority="51" name="List Table 6 Colorful" /><br /><w:lsdexception locked="false" priority="52" name="List Table 7 Colorful" /><br /><w:LsdException Locked="false" Priority="46"<br />Name="List Table 1 Light Accent 1"/><br /><w:lsdexception locked="false" priority="47" name="List Table 2 Accent 1" /><br /><w:lsdexception locked="false" priority="48" name="List Table 3 Accent 1" /><br /><w:lsdexception locked="false" priority="49" name="List Table 4 Accent 1" /><br /><w:lsdexception locked="false" priority="50" name="List Table 5 Dark Accent 1" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="List Table 6 Colorful Accent 1"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="List Table 7 Colorful Accent 1"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="List Table 1 Light Accent 2"/><br /><w:lsdexception locked="false" priority="47" name="List Table 2 Accent 2" /><br /><w:lsdexception locked="false" priority="48" name="List Table 3 Accent 2" /><br /><w:lsdexception locked="false" priority="49" name="List Table 4 Accent 2" /><br /><w:lsdexception locked="false" priority="50" name="List Table 5 Dark Accent 2" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="List Table 6 Colorful Accent 2"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="List Table 7 Colorful Accent 2"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="List Table 1 Light Accent 3"/><br /><w:lsdexception locked="false" priority="47" name="List Table 2 Accent 3" /><br /><w:lsdexception locked="false" priority="48" name="List Table 3 Accent 3" /><br /><w:lsdexception locked="false" priority="49" name="List Table 4 Accent 3" /><br /><w:lsdexception locked="false" priority="50" name="List Table 5 Dark Accent 3" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="List Table 6 Colorful Accent 3"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="List Table 7 Colorful Accent 3"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="List Table 1 Light Accent 4"/><br /><w:lsdexception locked="false" priority="47" name="List Table 2 Accent 4" /><br /><w:lsdexception locked="false" priority="48" name="List Table 3 Accent 4" /><br /><w:lsdexception locked="false" priority="49" name="List Table 4 Accent 4" /><br /><w:lsdexception locked="false" priority="50" name="List Table 5 Dark Accent 4" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="List Table 6 Colorful Accent 4"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="List Table 7 Colorful Accent 4"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="List Table 1 Light Accent 5"/><br /><w:lsdexception locked="false" priority="47" name="List Table 2 Accent 5" /><br /><w:lsdexception locked="false" priority="48" name="List Table 3 Accent 5" /><br /><w:lsdexception locked="false" priority="49" name="List Table 4 Accent 5" /><br /><w:lsdexception locked="false" priority="50" name="List Table 5 Dark Accent 5" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="List Table 6 Colorful Accent 5"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="List Table 7 Colorful Accent 5"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="List Table 1 Light Accent 6"/><br /><w:lsdexception locked="false" priority="47" name="List Table 2 Accent 6" /><br /><w:lsdexception locked="false" priority="48" name="List Table 3 Accent 6" /><br /><w:lsdexception locked="false" priority="49" name="List Table 4 Accent 6" /><br /><w:lsdexception locked="false" priority="50" name="List Table 5 Dark Accent 6" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="List Table 6 Colorful Accent 6"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="List Table 7 Colorful Accent 6"/><br /></w:latentstyles><br /></xml><![endif]--><!--[if gte mso 10]><br /><style><br />/* Style Definitions */<br />table.MsoNormalTable<br />{mso-style-name:æ®éè¡¨æ ¼;<br />mso-tstyle-rowband-size:0;<br />mso-tstyle-colband-size:0;<br />mso-style-noshow:yes;<br />mso-style-priority:99;<br />mso-style-parent:"";<br />mso-padding-alt:0cm 5.4pt 0cm 5.4pt;<br />mso-para-margin:0cm;<br />mso-para-margin-bottom:.0001pt;<br />mso-pagination:widow-orphan;<br />font-size:10.0pt;<br />font-family:"Calibri","sans-serif";<br />mso-bidi-font-family:"Times New Roman";}<br /></style><br /><![endif]--> In order to uniquely tag each double-stranded DNA molecule, add on ice the following reagents to the15 Î¼l of double-stranded and purified cDNA from Step E7, and mix:<br/>15 Î¼l of TDE Tagment DNA buffer<br/>0.2 Î¼l of TDE1 Tagment DNA enzyme</li><li> Incubate 5 min at 55 Â°C.</li><li> Purify using 54 Î¼l of AMPure XP beads (see <b>P</b><b>rocedure</b><b> E. DNA Purification</b>).</li><li>Elute in 20 Î¼l EB Buffer.</li></ol><p><br/></p></li><li><p>PCR amplification</p><ol class="npwtwo"><li>In order to amplify each uniquely tagmented double-stranded DNA molecule to the concentrations required for RNA-sequencing, add the following reagents to the 19.5 Î¼l of tagmented and purified double-stranded cDNA from Step F4 in a single well of a 96-Well qPCR plate and mix well:<br/>25 Î¼l of 2x Phusion HF master mix<br/>2.5 Î¼l 20x Eva Green<br/>1 Î¼l of Nextera PCR primers (10 Î¼M each)<br/>1 Î¼l of Nextera modified Index one primers (âN7XXâ; includes a random 8-mer tags for barcoding; 0.5 Î¼M each)<br/>1 Î¼l of Nextera Index two primer (âN501-506â, for multiplexing; 0.5 Î¼M)</li><li>Include the four Real-Time fluorescent Standards in duplicates by directly pipetting 50 Î¼l from each Real-Time fluorescent Standardsâ tube in separated wells of the qPCR plate.</li><li>Incubate in a <a class="blackNoUnder" target="_blank"> CFX96 </a> Touch Real-Time PCR Detection System:<br/><ol class="npwthree"><li>3 min at 72 Â°C</li><li>30 s at 98 Â°C</li><li>10 s at 98 Â°C</li><li>30 s at 63 Â°C</li><li>3 min at 72 Â°C (plate read)</li><li>Repeat steps c) to e) up to 25 times (see also Step G5)<br/></li></ol></li><li>Use the âno baseline subtractionâ setting (in User Preference Settings in the Bio-Rad, CFX Maestro<sup>TM</sup> Software) in order to monitor raw fluorescence amplifications in parallel to the unchanging Real-Time Fluorescent Standards.</li><li>Stop the PCR reaction right after plate read (3.e) and before denaturation (3.c) when an optimal amplification is reached (when the curve reaches a fluorescence level between standard 2 and standard 4; stopping the reaction at this point is key to reach a compromise between sufficient specific amplification of the sample libraries and minimized amplification of non-specific PCR products such as primers dimers) and recover the samples in 1.5 ml DNA LoBind tubes on ice. Optimal amplification is typically reached after 18 to 20 cycles when using a starting material of 100 Neuroblasts or after 14 to 15 cycles when using a starting material of 1,000 Neuroblasts.</li><li>Purify using 90 Î¼l of AMPure XP beads (see <strong>Procedure E. DNA Purification</strong>).</li><li>Elute in 20 Î¼l EB buffer and send for 50 base-pair Illumina Single-end sequencing. Prior to sending to sequencing, the quality of libraries can be checked by performing quantitative PCR on a house-keeping gene and the resulting amplification should be comparable to an amplification performed in the same settings, using the same DNA amount from a reference cDNA sample (for example, from a whole-tissue extraction).<br/></li></ol></li></ol><p class="pptt" id="biaoti38643">### Data analysis</p><ol class="npwone"><li><p>Summary of the analysis</p><p>Most critical analysis steps are the following (see also Figure 3): In a first step, sequencing reads are aligned to the genome. Ribosomal RNA reads are subtracted from all aligned reads. Next, reads that arose due to biased amplification (in the step shown in Figure 1B) are identified using the molecule-specific barcode. At each position in the genome the different barcodes are counted. Reads sharing the same barcode are marked as âduplicated readsâ. Only the read with the highest quality (this can be assessed with the PHRED score) is kept while all others are removed. The aligned and deduplicated reads are then counted and polyA-containing transcripts are subjected to a differential expression analysis (for example DESeq, Anders and Huber, 2010).</p><p><br/></p><p class="ppzz" style="margin-left:30px;"><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3809/bioprotoc-10-21-3809-g003.jpg"/><br/><strong>Figure 3. Barcoding strategy for data analysis.</strong> The sequencing library (see also Figure 1) contains DNA molecules with sequences allowing the attachment to the flow cell (P5/P7) and indices. The sample-specific index (green) can be used to pool several sequence libraries for a single sequencing run (multiplexing). The molecule-specific barcode is unique for each fragmented, double-stranded DNA molecule. Sequencing reads are mapped to the reference genome. The molecule-specific barcodes identify PCR amplicons derived from the same DNA molecule and thus allow the correction of such PCR bias by filtering duplicates.</p><p><br/></p></li><li><p>Step by step procedure of the RNA-seq data analysis</p><ol class="npwtwo"><li>Demultiplex reads based on âmultiplex BCâ:<br/><ol class="npwthree"><li>Create a FASTQ file for each multiplex BC (exact match or edit distance â¤ 2 based)</li><li>Each sequence ID must have following structure: â@readID:multiplexBC:indexMoleculeâ<br/></li></ol></li><li>Remove reads of ribosomal RNA origin:<br/><ol class="npwthree"><li>Align against rRNA sequences derived from RefSeq (BWA, <a href="http://bio-bwa.sourceforge.net/bwa.shtml" target="_blank">http://bio-bwa.sourceforge.net/bwa.shtml</a>)</li><li>Retrieve un-aligned reads (samtools; <a href="http://www.htslib.org/doc/samtools.html" target="_blank">http://www.htslib.org/doc/samtools.html</a>)</li><li>Convert un-aligned reads to FASTQ format<br/></li></ol></li><li>Align reads to the genome with TopHat (<a href="https://ccb.jhu.edu/software/tophat/manual.shtml" target="_blank">https://ccb.jhu.edu/software/tophat/manual.shtml</a>):<br/><ol class="npwthree"><li>Use the <em>Drosophila melanogaster</em> genome (FlyBase)</li><li>Use min/max intron range of 20-150,000 bp (FlyBase statistics)</li><li>Enable microexon-search <br/></li><li>Provide a gene model as GTF (FlyBase)<br/></li></ol></li><li>Use our provided custom perl script (digiTag.pl; <a href="https://gitlab.com/ii-bioinfo-thomas/ii-digitag" target="_blank">https://gitlab.com/ii-bioinfo-thomas/ii-digitag</a>) to mark reads arising from duplication events. The index molecule (barcode) must be stored in ID (see 1b). In short the script does following:<br/><p class="ppzz" style="margin-left:-45px;"><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201028/20201028002216_8364.jpg" width="761"/> </p><ol class="npwthree"><li>The different barcodes are counted at each genomic position.</li><li>The diversity of barcodes at each position is examined.<br/><ol class="npwfour"><li>Barcodes are sorted descending by their count.</li><li>If several barcodes have the same occurrence, they are further sorted alphanumerically.</li><li>Reads sharing the same barcode, are sorted by the average PHRED quality.</li><li>If several reads have the same quality, they are further sorted alphanumerically.</li><li>The barcodes are cycled through by their counts.</li><li>Within one barcode, the read with the highest average PHRED quality is the unique correct read and all subsequent reads with the same barcode are marked as duplicates.</li><li>All reads which have barcodes with one mismatch difference compared the pool of valid read barcodes are also marked as duplicates.<br/></li></ol></li></ol></li><li>Count with HTSeq (<a href="https://htseq.readthedocs.io/en/latest/count.html" target="_blank">https://htseq.readthedocs.io/en/latest/count.html</a>):<br/><ol class="npwthree"><li>Set stranded to ânoâ</li><li>Set mode to âintersection-nonemptyâ</li><li>use a GTF (FlyBase) without snRNA, rRNA, tRNA, snoRNA and pseudogenes<br/></li></ol></li><li>Perform standard differential expression analysis with DESeq2 (<a href="ages/release/bioc/vignette" target="_blank">https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html</a>)<br/><br/><p class="ppzz" style="margin-left:-20px;">Further documentation of the analysis can be found in the method section of Landskron <em>et al.</em> (2018) and all data are available on this link: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87085" target="_blank">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87085</a>.</p></li></ol></li></ol><p class="pptt" id="biaoti38644">### Notes</p><ol class="npwtwok"><li><p>Always use 1.5 ml DNA LoBind tube along the protocol to limit DNA loss.</p></li><li><p>Always use Nuclease-free Water to prepare ethanol solutions.</p></li></ol><p class="pptt" id="biaoti38645">### Acknowledgments</p><p>We would like to particularly thank Jonas Steinman and Heike Harzer for pioneering DigiTAG. This work was funded by the Austrian Academy of Sciences, Austrian science Fund and the European Commission. F.B. was supported by the European Molecular Biology Organization. This protocol is derived from Landskron <em>et al.</em> (2018).</p><p class="pptt" id="biaoti38646">### Competing interests</p><p>Authors declare no competing interests.</p><p class="pptt" id="biaoti38647">### References</p><ol class="npwtwok"><li>Abdusselamoglu, M. D., Eroglu, E., Burkard, T. R. and Knoblich, J. A. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31329099" target="_blank">The transcription factor odd-paired regulates temporal identity in transit-amplifying neural progenitors via an incoherent feed-forward loop.</a> <em>Elife</em> 8: e46566.</li><li>Anders, S. and Huber, W. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20979621" target="_blank">Differential expression analysis for sequence count data.</a> <em>Genome Biol</em> 11(10): R106.</li><li>Brunet Avalos, C., Maier, G. L., Bruggmann, R.and Sprecher, S. G. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31746739" target="_blank">Single cell transcriptome atlas of the <em>Drosophila </em>larval brain. </a><em><i>Elife</i> 8: e50354</em>.</li><li>Berger, C., Harzer, H., Burkard, T. R., Steinmann, J., van der Horst, S., Laurenson, A. S., Novatchkova, M., Reichert, H. and Knoblich, J. A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22884370" target="_blank"> FACS purification and transcriptome analysis of<em> Drosophila</em> neural stem cells reveals a role for Klumpfuss in self-renewal.</a> <em>Cell Rep</em> 2(2): 407-418.</li><li>Carney, T. D., Miller, M. R., Robinson, K. J., Bayraktar, O. A., Osterhout, J. A. and Doe, C. Q. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22061480" target="_blank"> Functional genomics identifies neural stem cell sub-type expression profiles and genes regulating neuroblast homeostasis.</a> <em>Dev Biol</em> 361(1): 137-146.</li><li>Harzer, H., Berger, C., Conder, R., Schmauss, G. and Knoblich, J. A. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23660757" target="_blank">FACS purification of <em>Drosophila</em> larval neuroblasts for next-generation sequencing.</a> <em>Nat Protoc</em> 8(6): 1088-1099.</li><li>Homem, C. C. and Knoblich, J. A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23132240" target="_blank"><em>Drosophila</em> neuroblasts: a model for stem cell biology.</a> <em>Development</em> 139(23): 4297-4310.</li><li>Knoblich, J. A., (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21102610" target="_blank"> Asymmetric cell division: recent developments and their implications for tumour biology.</a> <em>Nat Rev Mol Cell Biol</em> 11(12): 849-860.</li><li>Landskron, L., Steinmann, V., Bonnay, F., Burkard, T. R., Steinmann, J., Reichardt, I., Harzer, H., Laurenson, A. S., Reichert, H. and Knoblich, J. A. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29580384" target="_blank">The asymmetrically segregating lncRNA cherub is required for transforming stem cells into malignant cells.</a> <i style="font-style:italic;">Elife</i> 7: e31347.<em></em><span></span> </li><li>Stark, R., Grzelak, M. and Hadfield, J. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31341269" target="_blank">RNA sequencing: the teenage years.</a> <em>Nat Rev Genet</em> 20(11): 631-656.</li><li>Walsh, K. T. and Doe, C. Q. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29158446" target="_blank"> <em>Drosophila </em>embryonic type II neuroblasts: origin, temporal patterning, and contribution to the adult central complex.</a> <em>Development</em> 144(24): 4552-4562.</li><li>Wissel, S., Harzer, H., Bonnay, F., Burkard, T. R., Neumuller, R. A. and Knoblich, J. A. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29959232" target="_blank">Time-resolved transcriptomics in neural stem cells identifies a v-ATPase/Notch regulatory loop.</a> <em>J Cell Biol</em> 217(9): 3285-3300.</li><li>Yang, C. P., Fu, C. C., Sugino, K., Liu, Z., Ren, Q., Liu, L. Y., Yao, X., Lee, L. P. and Lee, T. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26700685" target="_blank">Transcriptomes of lineage-specific<em> Drosophila</em> neuroblasts profiled by genetic targeting and robotic sorting.</a> <em>Development</em> 143(3): 411-421.</li></ol>