<body>
            <h1><em>In vitro</em> Glutamylation Inhibition of Ubiquitin Modification and Phosphoribosyl-Ubiquitin Ligation Mediated by <em>Legionella pneumophila</em> Effectors</h1>
            <p><strong>Authors:</strong> Alan G. Sulpizio, Marena E. Minelli, Yuxin Mao</p>
            <h2>Abstract</h2>
            <p><p>Glutamylation is a posttranslational modification where the amino group of a free glutamate amino acid is conjugated to the carboxyl group of a glutamate side chain within a target protein. SidJ is a <em>Legionella</em> kinase-like protein that has recently been identified to perform protein polyglutamylation of the <em>Legionella</em> SdeA Phosphoribosyl-Ubiquitin (PR-Ub) ligase to inhibit SdeAâs activity. The attachment of multiple glutamate amino acids to the catalytic glutamate residue of SdeA by SidJ inhibits SdeAâs modification of ubiquitin (Ub) and ligation activity. In this protocol, we will discuss a SidJ non-radioactive, <em>in vitro</em> glutamylation assay using its substrate SdeA. This will also include a second reaction to assay the inhibition of SdeA by using both modification of free Ub and ligation of ADP-ribosylated Ubiquitin (ADPR-Ub) to SdeA’s substrate Rab33b. Prior to the identification and publication of SdeAâs activity, no SdeA inhibition assays existed. Our group and others have demonstrated various methods to display inhibition of SdeAâs activity. The alternatives include measurement of ADP-ribosylation of Ub using radioactive NAD, NAD hydrolysis, and Western blot analysis of HA-Ub ligation by SdeA. This protocol will describe the inhibition of both ubiquitin modification and the PR-Ub ligation by SdeA using inexpensive standard gels and Coomassie staining.</p></p>
            <p><strong>Keywords:</strong> SidJ, SdeA, Glutamylation, PR-Ubiquitination, <em>Legionella</em>, Pseudokinase, ADP-ribosylation</p>
            <h2>Background</h2><p><p><em>Legionella pneumophila</em> is an infectious bacterium that opportunistically infects alveolar macrophages. This occurs through the inhalation of contaminated water aerosols, causing a potentially lethal form of pneumonia known as Legionnairesâdisease (McDade <em>et al.</em>, 1977). <em>Legionella</em> infects host cells by the secretion of over 300 effector proteins that are used to hijack many host cellular processes and prevent their lysosomal degradation (Hubber and Roy, 2010). One process hijacked by <em>Legionella</em> is the ubiquitination system (Hubber <em>et al.</em>, 2013). Ubiquitination is a eukaryotic posttranslational modification, that regulates a variety of cellular processes (Hershko and Ciechanover, 1998; Chen and Sun, 2009; Hurley and Stenmark, 2011; Haglund and Dikic, 2012). This requires a concerted effort of E1, E2, and E3 enzymes to carefully regulate which proteins are ubiquitinated (Scheffner <em>et al.</em>, 1995). However, <em>Legionella</em> have co-opted this modification with the SidE family of phosphoribosyl-ubiquitin ligases that act independently of E1 and E2 enzymes. This family uses two catalytic domains to ADP-ribosylate ubiquitin using the mono-ADP-ribosyl transferase (mART) domain, and then, ligates ADPR-Ub to a host protein serine residues using a phosphodiesterase (PDE) domain (Bhogaraju <em>et al.</em>, 2016; Qiu <em>et al.</em>, 2016; Kotewicz <em>et al.</em>, 2017). These previously mentioned studies include means for assaying SidE activity. The SidE family includes a member, SdeA, which has been identified to be spatiotemporally regulated by SidJ (Havey and Roy, 2015; Jeong <em>et al.</em>, 2015; Urbanus <em>et al.</em>, 2016); although, the mechanism of this regulation was not completely understood. It was suggested that SidJ may act as a PR-deubiquitinase (Qiu <em>et al.</em>, 2017) when using <em>Legionella</em> purified SidJ (Qiu and Luo, 2019); however, recent studies do not recapitulate these results (Bhogaraju <em>et al.</em>, 2019; Wan <em>et al.</em>, 2019; Shin <em>et al.</em>, 2020).</p><br/><p> Our group (Sulpizio <em>et al.</em>, 2019), and several others (Bhogaraju <em>et al.</em>, 2019; Black <em>et al.</em>, 2019; Gan <em>et al.</em>, 2019), have demonstrated that SidJ is a polyglutamylase that adds multiple glutamate amino acids to SdeA to inhibit SdeA’s function. These studies provided structural data that identified SidJ contains a kinase-like domain and binds the eukaryotic protein calmodulin. Furthermore, mass spectrometry studies have identified that the mechanism of SdeA’s inhibition is polyglutamylation of SdeAâs catalytic mART glutamate residue. Based on these results, an <em>in vitro</em> glutamylation assay was developed using the substrate SdeA, calmodulin, ATP/MgCl<sub>2</sub>, and glutamate. To demonstrate the inhibitory effect of glutamylation, <em>in vitro</em> glutamylation was followed by a SdeA activity assay. Other groups have also described SdeA inhibition using NAD hydrolysis (Bhogaraju <em>et al.</em>, 2019), radioactive NAD (Black <em>et al.</em>, 2019), Flag-tagged SdeA substrates (Gan <em>et al.</em>, 2019), HA-Ub, and HA-Ub variants resistant to canonical ubiquitination by these groups. These alternative methods are suitable for identifying inhibition and may provide more quantitative detection. However, some of these experiments did not include <em>in vitro</em> inhibited SdeA, and those that monitor ubiquitin modification require the use of expensive reagents such as radioactive NAD. This protocol discusses a glutamylation assay developed for SidJ and assays <em>in vitro</em> inhibition of both SdeA PR-ubiquitin ligation and ubiquitin modification using standard gels and Coomassie staining. This can be used to identify the effects of mutations on activity, assay inhibition of both ubiquitin modification and PR-Ubiquitin ligation, and may more generally be adopted to determine inhibition of other ADP-ribosyl transferases by glutamylation.</p></p>
        </body>

<p class="pptt" id="biaoti38658">## Materials and Reagents</p><ol class="npwtwok"><li><p>1.7 ml Microtubes (Corning Incorporated, Axygen, catalog number: <a class="blackNoUnder" target="_blank"> MCT-175-C </a> )</p></li><li><p>50 ml Centrifuge tubes (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 525-0637 </a> )</p></li><li><p>Gloves (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 89038-270 </a> )</p></li><li><p>Kim wipes (Kimberly-Clark Professional, catalog number: <a class="blackNoUnder" target="_blank"> 34120 </a> )</p></li><li><p>Pipette tips:</p><p>10 Î¼l XL Graduated Tips (USA Scientific, Tip One, catalog number: <a class="blackNoUnder" target="_blank"> 1110-3700 </a> )</p><p>200 Î¼l Graduated Quick Rack (Laboratory Products Sales, catalog number: <a class="blackNoUnder" target="_blank"> 130430 </a> )</p><p>1,250 Î¼l Pipette tips (Laboratory Product Sales, catalog number: L134770)</p></li><li><p>Recombinant proteins</p><p>SidJ 89-853 truncation, SdeA 211-1152 truncation, human calmodulin 2, were expressed in <em>Escherichia coli</em> with an N-terminal 6XHis-SUMO tag and purified as described previously (Sulpizio <em>et al.</em>, 2019). Rab33b (1-200) was also purified as proteins mentioned in (Sulpizio <em>et al.</em>, 2019) and Ub was purified as described in Akturk <em>et al.</em> (2018). Final purified proteins were stored in buffer (20 mM Tris pH 7.5, 150 mM NaCl) without glycerol, aliquoted, flash-frozen, and stored at -80 Â°C.</p></li><li><p>Î²-Nicotinamide adenine dinucleotide sodium salt (NAD) (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> N0632-1G </a> )</p></li><li><p>L-Glutamic Acid Monosodium Salt, Monohydrate (USB Corporation, catalog number: <a class="blackNoUnder" target="_blank"> 16245 </a> )</p></li><li><p>2-Mercaptoethanol (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> M3148-100ML </a> )</p></li><li><p>30% Acrylamide/Bis solution 37.5:1 (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1610158 </a> )</p></li><li><p>Acetic acid, glacial (J.T. Baker, catalog number: 9508-06)</p></li><li><p>Adenosine 5â-triphosphate disodium salt hydrate (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> A2382-10G </a> )</p></li><li><p>Ammonium persulfate (APS) (Amresco, catalog number: <a class="blackNoUnder" target="_blank"> 0486-100G </a> )</p></li><li><p>Brilliant Blue R-250 (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> BP101-50 </a> )</p></li><li><p>Bromophenol Blue sodium salt (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> BP114-25 </a> )</p></li><li><p>DL-Dithiothreitol (DTT) (Amresco, catalog number: <a class="blackNoUnder" target="_blank"> M109-25g </a> )</p></li><li><p>Ethanol 200-proof (Koptec, catalog number: <a class="blackNoUnder" target="_blank"> V1001 </a> )</p></li><li><p>Glycerol (Mallinckrodt Chemicals, catalog number: <a class="blackNoUnder" target="_blank"> 5092-16 </a> )</p></li><li><p>Glycine (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 0167-5KG </a> )</p></li><li><p>Magnesium chloride, 6-hydrate (Mallinckrodt Chemicals, catalog number: <a class="blackNoUnder" target="_blank"> 5958-04 </a> )</p></li><li><p>Methanol (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> A454SK-4 </a> )</p></li><li><p>N,N,Nâ,Nâ-tetramethylethylene-diamine (TEMED) (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 161-0800 </a> )</p></li><li><p>Precision Plus Protein All Blue Standards Protein Ladder (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 161-0373 </a> )</p></li><li><p>Sodium chloride (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 0241-10KG </a> )</p></li><li><p>Sodium dodecyl sulfate (SDS) (VWR Life Sciences, catalog number: <a class="blackNoUnder" target="_blank"> 0227-1KG </a> )</p></li><li><p>Tris (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 0497-5KG </a> )</p></li><li><p>Reaction Buffer (see Recipes)</p></li><li><p>MgCl<sub>2</sub> 1 M Solution (see Recipes)</p></li><li><p>ATP 100 mM pH 7.5 Solution (see Recipes)</p></li><li><p>Glutamic Acid 1 M Solution (see Recipes)</p></li><li><p>10x SDS-PAGE Running Buffer (see Recipes)</p></li><li><p>10x Native-PAGE Running Buffer (see Recipes)</p></li><li><p>SDS Sample Buffer (see Recipes)</p></li><li><p>Native Sample Buffer (see Recipes)</p></li><li><p>Coomassie Stain (see Recipes)</p></li><li><p>Coomassie Destaining Solution (see Recipes)</p></li><li><p>12% SDS-PAGE Resolving Gel (see Recipes)</p></li><li><p>4% SDS-PAGE Stacking Gel (see Recipes)</p></li><li><p>8% Native PAGE Resolving Gel (see Recipes)</p></li><li><p>3% Native PAGE Stacking Gel (see Recipes)</p><p class="ppzz" style="margin-left:-30px;"><em>Note: Products were stored as suggested by manufacture except where listed.</em> </p></li></ol><p class="pptt" id="biaoti38659">### Equipment</p><ol class="npwtwok"><li><p>-80 Â°C freezer (So-Low, model: <a class="blackNoUnder" target="_blank"> PV85-21 </a> )</p></li><li><p>Computer</p></li><li><p>Dry Bath (Benchmark, model: <a class="blackNoUnder" target="_blank"> BSH1001 </a> )</p></li><li><p>Fixed speed centrifuge (Benchmark, model: <a class="blackNoUnder" target="_blank"> C1008-C </a> )</p></li><li><p>Forceps</p></li><li><p>Gel electrophoresis apparatus (Bio-Rad, model: <a class="blackNoUnder" target="_blank"> Mini-PROTEAN Tetra System </a> )</p></li><li><p>Gel electrophoresis power supply (Bio-Rad, model: <a class="blackNoUnder" target="_blank"> PowerPac Basic </a> )</p></li><li><p>Gel imager (Bio-Rad, model: <a class="blackNoUnder" target="_blank"> Chemidoc MP Imaging System </a> )</p></li><li><p>Ice bucket</p></li><li><p>Labcoat (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 10141-306 </a> )</p></li><li><p>Laboratory tape (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 89098-062 </a> )</p></li><li><p>Microwave (Sharp, model: <a class="blackNoUnder" target="_blank"> R230KW </a> )</p></li><li><p>Pipettes (Gilson, model: Pipetman classic P2, P20, P200, P1000, catalog numbers: <a class="blackNoUnder" target="_blank"> F144801 </a> , <a class="blackNoUnder" target="_blank"> F123600 </a> , <a class="blackNoUnder" target="_blank"> F123601 </a> , <a class="blackNoUnder" target="_blank"> F123602 </a> )</p></li><li><p>Rocking shaker (Reliable Scientific, Inc., model: <a class="blackNoUnder" target="_blank"> 55D 12 x 16 </a> )</p></li><li><p>Sheet protectors (Clear file, Archival Plus 5x7 Print, catalog number: <a class="blackNoUnder" target="_blank"> 370100B </a> )</p></li><li><p>Vortex mixer 120V (Corning LSE, model: <a class="blackNoUnder" target="_blank"> 6775 </a> )</p></li></ol><p class="pptt" id="biaoti38660">### Procedure</p><ol class="npwone"><li><p>SidJ <em>in vitro</em> Glutamylation Reaction</p><ol class="npwtwok"><li><p>Review the flowchart of the experimental outline of SidJ glutamylation and SdeA activity assays before beginning the procedure (Figure 1).</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3811/bioprotoc-10-21-3811-g001.jpg"/><br/><strong>Figure 1. Experimental outline for SidJ <em>in vitro</em> Glutamylation and SdeA Inhibition assay. </strong>A flow chart of the general experimental steps described in this procedure. The left branch is to assay the inhibition of SdeA ADP-ribosylation or Phospho-Ribosylation of Ub. The right branch is the experiments to assay PR-Ub ligation to substrates.</p><p><br/></p></li><li><p>Thaw recombinantly purified SidJ 89-853, SdeA Core (truncation 211-1152), calmodulin, ubiquitin, and optionally, Rab33b (1-200) on ice.</p><p><em>Note: Rab33b 1-200 was used in this assay due to the prominence of a single PR-Ubiquitination band and increased protein stability (Jon Wasilko, Mao Lab, unpublished results). Full-length Rab33b may also be used instead.</em> </p></li><li><p>Preheat dry bath to 37 Â°C.</p></li><li><p>Prepare stock solutions, on ice, listed in Table 1 by diluting in reaction buffer (20 mM Tris pH 7.5, 50 mM NaCl). Prepare NAD solution fresh and use glutamic acid and ATP stocks stored at -80 Â°C.</p><br/><p><strong>Table 1. </strong><strong>Stock solution concentrations for SidJ <em>in vitro</em> glutamylation and SdeA inhibition reaction<br/></strong> </p><p class="ppzz" style="margin-left:-20px;"><strong><img alt="" class="layerphoto" height="350" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201028/20201028194846_7407.jpg" width="557"/></strong> </p></li><li><p>Pipette the volumes of stock solution listed in SidJ glutamylation reaction row for a 25 Î¼l reaction listed in Table 2 into a chilled 1.7 ml microcentrifuge tube on ice. Pipette ATP last to initiate the reaction. Immediately vortex, centrifuge briefly (~10 s max speed), and incubate the samples at 37 Â°C in a dry bath for 30 min.</p><p><em>Note: It is important that samples are mixed thoroughly and centrifuged. SdeA is very active and the reaction mixture needs to be homogenous for maximal inhibition.</em> </p></li><li><p>Optional: Prepare a master mix for Step A5. Combine components contained in all reactions by pipetting stock solutions and 10 Î¼l of reaction buffer per reaction. Prepare approximately 10% more reaction mix than needed for samples. The addition of reaction buffer to the master mix dilutes components to maintain protein stability. If a master mix is prepared, for each reaction, subtract the volumes of reaction components included in the mix and 10 Î¼l of reaction buffer from the amount used in Table 2.</p><br/><p><strong>Table 2. SidJ <em>in vitro</em> glutamylation and SdeA modification reaction components and concentrations<br/></strong> </p><p class="ppzz" style="margin-left:-20px;"><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201028/20201028195423_8534.jpg" width="800"/></p></li><li><p>Pipette remaining reaction components for the SdeA activity portion of the assay into the tube. If assaying ubiquitin modification, replace Rab33b with reaction buffer. Immediately vortex, centrifuge briefly (~10 s max speed), and incubate samples at 37 Â°C in a dry bath for 30 min.</p><p><em>Note: A master mix containing SdeA modification components and 3 Î¼l reaction buffer per sample can be prepared to minimize pipetting inaccuracies. A negative control excluding NAD would be beneficial for the visualization of the absence of SdeA activity.</em> </p></li><li><p>If assaying ubiquitin modification, separate each reaction into two tubes by pipetting 12.5 Î¼l into another 1.7 ml microcentrifuge tube. Label tubes and pipette 3 Î¼l of native sample buffer into one tube for native-PAGE analysis, and 3 Î¼l of SDS sample buffer into the other for SDS-PAGE analysis and visualization of protein loading. Vortex to mix and centrifuge briefly (10 s at max speed). If only assaying PR-ubiquitin ligation activity, the reaction can be halted by the addition of 6 Î¼l of SDS sample buffer.</p><p><em>Note: Do not include SDS in native sample buffer, native-PAGE gels, or native-PAGE running buffer.</em> </p></li></ol></li><br/><li><p>Detection of ubiquitin modification and PR-Ubiquitin ligation to Rab33b by gel electrophoresis</p><ol class="npwtwok"><li><p>For detection of ubiquitin modification, electrophorese 13 Î¼l of the portion of each sample in native sample buffer using a native-PAGE gel and native-PAGE running buffer at 80 V. Once samples have migrated through stacking gel, increase the voltage to 120 V and electrophorese until dye front migrates 50-75% through the gel.</p><p><em>Note: Native-PAGE is required to detect modification of ubiquitin. The use of cold native-PAGE running buffer, placement of gel apparatus on ice, and shortened electrophoresis distance may provide better separation and clarity of protein bands.</em> </p></li><li><p>For detection of PR-Ubiquitin ligation and visualization of protein loading for ubiquitin modification gel, electrophorese 2.5 Î¼l of protein ladder and 13 Î¼l of each reaction in SDS sample buffer using an SDS-PAGE gel (4% separating gel, 12% resolving gel) and SDS-PAGE running buffer at 80 V. Once samples have migrated through stacking gel, increase the voltage to 150 V and electrophorese until dye front reaches the bottom of the gel.</p></li><li><p>Separate the gel from the casting glass and remove the stacking gel. Transfer the gel to a microwave-safe, plastic container with a lid. Pour Coomassie stain to cover gel and microwave briefly in a covered container, until boiling. Ensure not to inhale fumes when moving the container by maximizing distance from the container. Stain gel by rocking for a few hours, to overnight, at room temperature.</p><p><em>Note: There was some difficulty visualizing the calmodulin band using Coomassie staining. Staining temperature and time may be decreased if calmodulin can be adequately stained.</em> </p></li><li><p>Discard stain and rinse with the destaining solution and incubate in the destaining solution for approximately 30 min to 1 h. Discard solution and repeat incubation. Repeat until protein bands are visible. If some background staining persists, allow longer water destaining in Step B5.</p></li><li><p>Rehydrate gel and destain further by incubation in ddH<sub>2</sub>O while rocking for 1-2 h. Remove water and repeat if necessary. The addition of a Kim wipe can assist in destaining and provide cleaner gel images.</p></li><li><p>After the gel is rehydrated, transfer the gel to a sheet protector and image using the gel imager. Wiping dust and staining imperfections with Kim wipe may help obtain clearer images (Figure 2).</p></li><li><p>Analyze results. The addition of phosphoribose or ADP-ribose to ubiquitin by SdeA causes a significant charge alteration relative to the overall size of the small ubiquitin protein. As a result, this alteration greatly shifts the electrophoretic mobility on a native gel, while this modification is not visualized if separating only by protein size as with SDS-PAGE. The inhibition of SdeA prevents the migration shift of ubiquitin (Figure 2A). This assay does not distinguish between the addition of ADP-ribose and further modification to PR-Ub. If assaying the PR-Ubiquitin ligation to Rab33b by SdeA, ligation is detected by incremental 8 kDa band shifts above the unmodified Rab33b band in the presence of NAD. This appearance of increased molecular weight bands corresponds to the attachment of one or multiple PR-Ub to Rab33b. Inhibition of SdeAâs PR-Ubiquitin ligase activity should decrease the intensity of the modified Rab33b compared to the uninhibited SdeA reaction (Figure 2B).</p></li></ol><br/></li></ol><p class="ppzz" style="margin-left:60px;"><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3811/bioprotoc-10-21-3811-g002.jpg"/><br/><strong>Figure 2. Reaction components required for SidJ mediated glutamylation and inhibition of SdeA. </strong>A. SidJ inhibition of the ability of SdeA to modify ubiquitin. SidJ glutamylation and SdeA modification were conducted with the reaction components and concentrations listed in Table 2, with Rab33b 1-200 replaced with reaction buffer. Reaction components were excluded as indicated. SidJ <em>in vitro</em> glutamylation assay was conducted for 30 min at 37 Â°C followed by SdeA modification for 30 min at 37 Â°C. Proteins were electrophoresed by native-PAGE and stained with Coomassie stain. B. SidJ inhibition of the ability of SdeA to PR-Ubiquitinate Rab33b 1-200. The reaction was conducted with concentrations listed in Table 2. SidJ <em>in vitro</em> glutamylation assay was conducted for 30 min at 37 Â°C followed by SdeA PR-Ubiquitination for 30 min at 37 Â°C. Proteins were electrophoresed by SDS-PAGE and the gel was stained with Coomassie stain. This figure is from the original research article (Sulpizio <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti38661">### Recipes</p><ol class="npwtwok"><li><p>Reaction Buffer</p><p>50 mM Tris pH 7.5</p><p>50 mM NaCl</p><p>Stored at room temperature</p></li><li><p>MgCl<sub>2 </sub>1 M Solution</p><p>Weigh 2.033 g of magnesium chloride, 6-hydrate and dilute to 10 ml with ddH<sub>2</sub>O</p><p>Store at room temperature</p></li><li><p>ATP 100 mM pH 7.5 Solution</p><p>Weigh 551.14 mg adenosine 5â²-triphosphate disodium salt hydrate and dissolve in 8 ml of ddH<sub>2</sub>O</p><p>Adjust the pH to 7.5 with 10 M sodium hydroxide and dilute to a final volume of 10 ml with ddH<sub>2</sub>O</p><p>Aliquot and store at -80 Â°C</p></li><li><p>Glutamic Acid 1 M Solution</p><p>Weigh 0.936 g of L-glutamic acid monosodium salt, monohydrate and dilute to 5 ml with ddH<sub>2</sub>O</p><p>Aliquot and store at -80 Â°C or prepare fresh</p></li><li><p>10x SDS-PAGE Running Buffer</p><p>Store at room temperature and dilute 10-fold with ddH<sub>2</sub>O for use</p><p><table><tbody><tr><th>Component</th><th>  1 L</th></tr><tr><td>Tris-Base</td><td>30 g</td></tr><tr><td>Glycine</td><td>140 g</td></tr><tr><td>SDS</td><td>10 g</td></tr><tr><td>ddH<sub>2</sub>O</td><td>To 1 L</td></tr></tbody></table></p></li><li><p>10x Native-PAGE Running Buffer</p><p>Store at room temperature and dilute 10-fold with ddH<sub>2</sub>O for use</p><p><table><tbody><tr><th>Component</th><th>  1 L</th></tr><tr><td>Tris-Base</td><td>35 g</td></tr><tr><td>Glycine</td><td>144 g</td></tr><tr><td></td><td></td></tr></tbody></table>ddH<sub>2</sub>O         To 1 L</p></li><li><p>SDS Sample Buffer</p><p>Store at room temperature, freeze aliquots -20 Â°C for extended storage</p><p><strong>Component             Concentration</strong><br/>Bromophenol Blue     0.25% (w/v)<br/>DTT                                0.5 M<br/>Glycerol                         50% (v/v)<br/>SDS                                10% (w/v)<br/>2-Mercaptoethanol     10% (v/v)</p></li><li><p>6x Native Sample Buffer</p><p>Store at room temperature</p><p><strong>Component               </strong><strong> 50 ml </strong><br/>ddH<sub>2</sub>O                         35 ml<br/>Glycerol                       15 ml<br/>Bromophenol Blue    0.125 g</p></li><li><p>Coomassie Stain</p><p>Store at room temperature</p><p><strong>Component                 </strong><strong>500 ml</strong><br/>Methanol                     225 ml<br/>ddH<sub>2</sub>O                          225 ml<br/>Glacial Acetic Acid      50 ml<br/>Brilliant Blue R250      1.25 g<span style="white-space:nowrap;"></span> </p></li><li><p>Coomassie Destaining Solution</p><p>Store at room temperature</p><p><strong>Component               </strong><strong>   1 L</strong><br/>Ethanol                        450 ml<br/>ddH<sub>2</sub>O                         450 ml<br/>Glacial Acetic Acid      100 ml</p><br/><p class="ppzz" style="margin-left:-30px;"><strong>SDS-PAGE Gel</strong> </p></li><li><p>12% Resolving Gel</p><p><strong>Component                                                 </strong> <strong>8 gels                                   </strong><strong> Final Conc.</strong><br/>ddH<sub>2</sub>O                                                           13.3 ml <br/>30% Acrylamide/Bis Solution                     16 ml                                       12%<br/>1.5 M Tris pH 8.8                                          10 ml                                        375 mM<br/>10% SDS                                                        400 Î¼l                                        0.1%<br/>10% APS                                                        300 Î¼l                                         0.075%<br/>TEMED                                                           24 Î¼l                                           0.06%</p></li><li><p>4% Stacking Gel</p><p><strong>Component                                                   8 gels                                     Final Conc.</strong><br/>ddH<sub>2</sub>O                                                            11.2 ml <br/>30% Acrylamide/Bis Solution                      2.16 ml                                     4.14%<br/>1.0 M Tris pH 6.8                                           2 ml                                          128 mM<br/>10% SDS                                                         160 Î¼l                                        0.1%<br/>10% APS                                                          110 Î¼l                                       0.07%<br/>TEMED                                                             16 Î¼l                                           0.1%</p><br/><p class="ppzz" style="margin-left:-30px;"><strong>Native-PAGE Gel</strong> </p></li><li><p>8% Resolving Gel</p><p><strong>Component                                                 </strong><strong>20 ml (4 Gels)                            </strong><strong>Final Conc.</strong><br/>ddH<sub>2</sub>O                                                               9.34 ml <br/>1.5 M Tris, pH 8.8                                             5 ml                                        375 mM<br/>30% Acrylamide/Bis Solution                         5.34 ml                                   8%<br/>10% APS                                                            100 Î¼l                                      0.05%<br/>TEMED                                                               20 Î¼l                                        0.1%<span style="white-space:nowrap;"></span> </p></li><li><p>3% Stacking Gel</p><p><strong>Component                                                 10 ml (4 Gels)                             Final Conc.</strong><br/>ddH<sub>2</sub>O                                                           8.32 ml <br/>1.0 M Tris, pH 6.8                                         500 Î¼l                                          50 mM<br/>30% Acrylamide/Bis Solution                    1.02 ml                                         3%<br/>10% APS                                                        50 Î¼l                                            0.05%<br/>TEMED                                                           10 Î¼l                                            0.1%<span style="white-space:nowrap;"></span> </p></li></ol><p class="pptt" id="biaoti38662">### Acknowledgments</p><p>This work was supported by National Institute of Health (NIH) grant 5R01GM116964 (to YM), the Cornell University Harry and Samuel Mann Outstanding Graduate Student Award (to AGS), and by the NIH under Ruth L Kirschstein National Research Service Award (6T32GM008267) from the NIGMS (to MEM).</p><p>   Protocol from original research article: Sulpizio, A., M. E. Minelli, M. Wan, P. D. Burrowes, X. Wu, E. J. Sanford, J.-H. Shin, B. C. Williams, M. L. Goldberg, M. B. Smolka, and Y. Mao (2019). "Protein polyglutamylation catalyzed by the bacterial calmodulin-dependent pseudokinase SidJ." eLife 8: e51162.</p><p class="pptt" id="biaoti38663">### Competing interests</p><p>The authors declare no competing interests.</p><p class="pptt" id="biaoti38664">### References</p><ol class="npwtwok"><li>Akturk, A., Wasilko, D. J., Wu, X., Liu, Y., Zhang, Y., Qiu, J., Luo, Z. Q., Reiter, K. H., Brzovic, P. S., Klevit, R. E. and Mao, Y. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29795346" target="_blank"> Mechanism of phosphoribosyl-ubiquitination mediated by a single <em>Legionella</em> effector.</a> <em>Nature</em> 557(7707): 729-733.</li><li>Bhogaraju, S., Bonn, F., Mukherjee, R., Adams, M., Pfleiderer, M. M., Galej, W. P., Matkovic, V., Lopez-Mosqueda, J., Kalayil, S., Shin, D. and Dikic, I. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31330532" target="_blank"> Inhibition of bacterial ubiquitin ligases by SidJ-calmodulin catalysed glutamylation.</a> <em>Nature</em> 572(7769): 382-386.</li><li>Bhogaraju, S., Kalayil, S., Liu, Y., Bonn, F., Colby, T., Matic, I. and Dikic, I. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27912065" target="_blank"> Phosphoribosylation of Ubiquitin Promotes Serine Ubiquitination and Impairs Conventional Ubiquitination.</a> <em>Cell</em> 167(6): 1636-1649e1613.</li><li>Black, M. H., Osinski, A., Gradowski, M., Servage, K. A., Pawlowski, K., Tomchick, D. R. and Tagliabracci, V. S. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31123136" target="_blank"> Bacterial pseudokinase catalyzes protein polyglutamylation to inhibit the SidE-family ubiquitin ligases.</a> <em>Science</em> 364(6442): 787-792.</li><li>Chen, Z. J. and Sun, L. J. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19217402" target="_blank"> Nonproteolytic functions of ubiquitin in cell signaling.</a> <em>Mol Cell</em> 33(3): 275-286.</li><li>Gan, N., Zhen, X., Liu, Y., Xu, X., He, C., Qiu, J., Liu, Y., Fujimoto, G. M., Nakayasu, E. S., Zhou, B., Zhao, L., Puvar, K., Das, C., Ouyang, S. and Luo, Z. Q. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31330531" target="_blank"> Regulation of phosphoribosyl ubiquitination by a calmodulin-dependent glutamylase.</a> <em>Nature</em> 572(7769): 387-391.</li><li>Haglund, K. and Dikic, I. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22357968" target="_blank"> The role of ubiquitylation in receptor endocytosis and endosomal sorting.</a> <em>J Cell Sci</em> (2): 265-275.</li><li>Havey, J. C. and Roy, C. R. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26099583" target="_blank"> Toxicity and SidJ-Mediated Suppression of Toxicity Require Distinct Regions in the SidE Family of <em>Legionella pneumophila</em> Effectors.</a> <em>Infect Immun</em> 83(9): 3506-3514.</li><li>Hershko, A. and Ciechanover, A. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9759494" target="_blank"> The ubiquitin system.</a> <em>Annu Rev Biochem</em> 67425-479.</li><li>Hubber, A., Kubori, T. and Nagai, H. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23918174" target="_blank"> Modulation of the ubiquitination machinery by <em>Legionella</em>.</a> <em>Curr Top Microbiol Immunol</em> 376227-247.</li><li>Hubber, A. and Roy, C. R. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20929312" target="_blank"> Modulation of host cell function by <em>Legionella pneumophila</em> type IV effectors.</a> <em>Annu Rev Cell Dev Biol</em> 26261-283.</li><li>Hurley, J. H. and Stenmark, H. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21332354" target="_blank"> Molecular mechanisms of ubiquitin-dependent membrane traffic.</a> <em>Annu Rev Biophys</em> 40119-142.</li><li>Jeong, K. C., Sexton, J. A. and Vogel, J. P. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25774515" target="_blank"> Spatiotemporal regulation of a <em>Legionella pneumophila</em> T4SS substrate by the metaeffector SidJ.</a> <em>PLoS Pathog</em> 11(3): e1004695.</li><li>Kotewicz, K. M., Ramabhadran, V., Sjoblom, N., Vogel, J. P., Haenssler, E., Zhang, M., Behringer, J., Scheck, R. A. and Isberg, R. R. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28041930" target="_blank"> A Single <em>Legionella</em> Effector Catalyzes a Multistep Ubiquitination Pathway to Rearrange Tubular Endoplasmic Reticulum for Replication.</a> <em>Cell Host Microbe</em> 21(2): 169-181.</li><li>McDade, J. E., Shepard, C. C., Fraser, D. W., Tsai, T. R., Redus, M. A. and Dowdle, W. R. (1977). <a href="http://www.ncbi.nlm.nih.gov/pubmed/335245" target="_blank"> Legionnaires' disease: isolation of a bacterium and demonstration of its role in other respiratory disease.</a> <em>N Engl J Med</em> 297(22): 1197-1203.</li><li>Qiu, J. and Luo, Z. Q. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30694498" target="_blank"> Methods for Noncanonical Ubiquitination and Deubiquitination Catalyzed by <em>Legionella pneumophila</em> Effector Proteins.</a> <em>Methods Mol Biol</em> 1921267-276.</li><li>Qiu, J., Sheedlo, M. J., Yu, K., Tan, Y., Nakayasu, E. S., Das, C., Liu, X. and Luo, Z. Q. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27049943" target="_blank"> Ubiquitination independent of E1 and E2 enzymes by bacterial effectors.</a> <em>Nature</em> 533(7601): 120-124.</li><li>Qiu, J., Yu, K., Fei, X., Liu, Y., Nakayasu, E. S., Piehowski, P. D., Shaw, J. B., Puvar, K., Das, C., Liu, X. and Luo, Z. Q. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28497808" target="_blank"> A unique deubiquitinase that deconjugates phosphoribosyl-linked protein ubiquitination.</a> <em>Cell Res</em> 27(7): 865-881.</li><li>Scheffner, M., Nuber, U. and Huibregtse, J. M. (1995). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7800044" target="_blank"> Protein ubiquitination involving an E1-E2-E3 enzyme ubiquitin thioester cascade.</a> <em>Nature</em> 373(6509): 81-83.</li><li>Shin, D., Mukherjee, R., Liu, Y., Gonzalez, A., Bonn, F., Liu, Y., Rogov, V. V., Heinz, M., Stolz, A., Hummer, G., Dotsch, V., Luo, Z. Q., Bhogaraju, S. and Dikic, I. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31732457" target="_blank"> Regulation of Phosphoribosyl-Linked Serine Ubiquitination by Deubiquitinases DupA and DupB.</a> <em>Mol Cell</em> 77(1): 164-179e166.</li><li>Sulpizio, A., Minelli, M. E., Wan, M., Burrowes, P. D., Wu, X., Sanford, E. J., Shin, J. H., Williams, B. C., Goldberg, M. L., Smolka, M. B. and Mao, Y. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31682223" target="_blank"> Protein polyglutamylation catalyzed by the bacterial calmodulin-dependent pseudokinase SidJ.</a> <i style="font-style:italic;">Elife</i> 8:<em> </em><span></span>e51162<em>.</em> </li><li>Urbanus, M. L., Quaile, A. T., Stogios, P. J., Morar, M., Rao, C., , R., Evdokimova, E., Lam, M., Oatway, C., Cuff, M. E., Osipiuk, J., Michalska, K., Nocek, B. P., Taipale, M., Savchenko, A. and Ensminger, A. W. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27986836" target="_blank"> Diverse mechanisms of metaeffector activity in an intracellular bacterial pathogen, <em>Legionella pneumophila</em>.</a> <em>Mol Syst Biol</em> 12(12): 893.</li><li>Wan, M., Sulpizio, A. G., Akturk, A., Beck, W. H. J., Lanz, M., Faca, V. M., Smolka, M. B., Vogel, J. P. and Mao, Y. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31690664" target="_blank"> Deubiquitination of phosphoribosyl-ubiquitin conjugates by phosphodiesterase-domain-containing <em>Legionella</em> effectors.</a> <em>Proc Natl Acad Sci U S A</em> 116(47): 23518-23526.</li></ol>