<body>
            <h1>Long-distance Transport in Bacterial Swarms Revealed by Single Nanoparticle Tracking</h1>
            <p><strong>Authors:</strong> Jingjing Feng, Yan He</p>
            <h2>Abstract</h2>
            <p><p>During swarming, high density flagella-driven bacteria migrate collectively in a swirling pattern on wet agar surfaces, immersed in a thin viscous fluid layer called “swarm fluid”. Though the fluid environment has essential role in the emergence of swarming behavior, the microscopic mechanisms of it in mediating the cooperation of bacteria populations are not fully understood. Here, instead of micro-sized tracers used in previous research, we use gold nanorods as single particle tracers to probe the dynamics of the swarm fluid. This protocol includes five major parts: (1) the culture of swarming bacterial colony; (2) the preparations of gold nanorod tracers and the micro-spraying technique which are used to put the nanotracers into the upper fluid of bacterial swarms; (3) imaging and tracking; (4) other necessary control experiments; (5) data analysis and fitting of physical models. With this method, the nano-sized tracers could move long distances above motile cells without direct collisions with the bacteria bodies. In this way, the microscopic dynamics of the swarm fluid could be tracked with high spatiotemporal resolution. Moreover, the comprehensive analysis of multi-particle trajectories provides systematic visualization of the fluid dynamics. The method is promising to probe the fluid dynamics of other natural or artificial active matter systems.</p></p>
            <p><strong>Keywords:</strong> Collective motion, Active matter, Bacteria swarming, Gold nanorods, Single particle tracking</p>
            <h2>Background</h2><p> Recently, collective motions of bacteria have attracted a lot of attention in the field of active matter/fluid (Rabani <em>et al.</em>, 2013 ; Zhang, H. <em>et al.</em>, 2010; Marchetti <em>et al.</em>, 2013). Due to the self-driven motion of individual bacterium propelled by flagella, high density bacteria suspensions would fall into a non-equilibrium state and emerge collective structures such as vortices, jets ( Mendelson <em>et al.</em>, 1999 ) or turbulences (Wensink <em>et al.</em>, 2012 ) in a scale far larger than the size of an individual bacterium. As one of the most widely used model system for the study of bacteria collection motions (Copeland and Weibel, 2009), bacteria swarming refers to coordinated migrations of cells across wet solid agar surfaces with swirling patterns (Kearns, 2010). During swarming, motile cells are immersed in a thin layer of highly viscous fluid called “swarm fluid” (Wu and Berg, 2012). Physically, the interplay between the fluid medium and the stirring flagella provides the dynamic origins of the collective motion though far-field hydrodynamic interactions (Koch and Subramanian, 2011). Biologically, the fluid environment enhances the mixing and diffusion of oxygen, nutrition as well as the signaling molecules involved in Quorum sensing (Hardman <em>et al.</em>, 1998 ) and chemotaxis (Taktikos <em>et al.</em>, 2012 ). Although the swarm fluid is important for the generation of large-scale synergetic patterns, how it works and its relationships with fast moving cells remain unclear especially at the microscopic scale.<br/> <br/>Single particle tracking has been regarded as a powerful technique to investigate the complex fluid (Burov <em>et al.</em>, 2011 ). Previous studies have used micro-sized particles as single particle tracers to characterize the properties of swarm fluid. Wu <em>et al.</em> used 1~2 µm micro-bubbles produced by explosive transformation of the water insoluble surfactant Span-83 droplets as tracers to reveal the intensive matter transfer flow in the leading edge of the swarming colony (Wu <em>et al.</em>, 2011). Zhang <em>et al.</em> fabricated MgO particles as tracer particles through burning magnesium ribbons. They found that 0.2 µm MgO particles only diffused normally within a small region (~4 µm<sup>2</sup>) as the swarm front approaches (Zhang, R. <em>et al.</em>, 2010). However, these methods of producing tracer particles are complicated and difficult to control. In addition, the prepared gold nanorods are often uneven in size. On the other hand, micro-sized tracers with various sizes and materials would either inevitably collide with bacteria bodies or be trapped at the liquid-air surface, making the tracers’ motions incapable of revealing real dynamics of the fluid motions. Plasmonic gold nanorods (AuNRs) have long been used as nanotracers for long-duration observations in biological studies with high spatio-temporal resolution. The AuNRs have good photostability and low cytotoxicity. Moreover, the methods of preparing stable, uniform, and monodispersed AuNRs are very mature. Here, with <em>Bacillus subtilis</em> as a model strain, we introduced 40 x 84 nm AuNRs as tracers into the bacterial swarm fluid. The AuNRs tracers could move freely in the upper layer without obvious physical contacts with the cell bodies. Combined with high spatiotemporal resolution of imaging and tracking technique, the trajectories of multi-nanoparticles in the large field of view could be obtained. By detailed diffusion analysis of tracers’ trajectories, we found that the swarm fluid could transport the AuNRs to long distances in a super diffusive, Lévy mode (Figure 1). This result is consistent with previous studies that individual bacterium in swarming would exhibit characteristic motions of Lévy walk ( Ariel <em>et al.</em>, 2015 ).<br/><br/> <p class="ppzz" style="margin-left:30px;"> <img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3812/bioprotoc-10-21-3812-g001.jpg"/><br/><strong>Figure 1. The gold nanorods move in high-efficiency on the upper fluid layer of the swarming bacteria.</strong> A. A captured image of the swarming bacteria and AuNRs. The brown rods are bacteria. The red spots which are indicated by white arrows are AuNRs. The scale bar is 10 μm. B. A typical trajectory illustrates that AuNRs move in a super-diffusive, Lévy mode with heterogeneous MSD local exponents.<br/></p></p>
        </body>

<p class="pptt" id="biaoti38667">## Materials and Reagents</p><ol class="npwtwok"><li><p>Plastic Petri dishes, 60 mm and 90 mm. Brand is not critical</p></li><li><p>Serum bottles, 250 and 500 ml (brand is not critical)</p><p><em>Note: Ensure that the bottles could withstand temperatures of at least</em> <em>121 </em><em>Â°C.</em> </p></li><li><p>Square cover glass slides, 22 x 22 mm (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 2870-25 </a> )</p></li><li><p>Single concave microscope glass slide, 25.4 x 76.2 mm (Sail Brand, catalog number: CAT.NO.<a class="blackNoUnder" target="_blank"> 7101 </a> )</p></li><li><p>Disposable sterile syringe filters, 0.22 Âµm diameter (Bioland, catalog number: <a class="blackNoUnder" target="_blank"> PT13-022 </a> )</p></li><li><p>230 mesh carbon film, copper grids (Zhongjing Keyi, catalog number: <a class="blackNoUnder" target="_blank"> BZ110223b </a> )</p></li><li><p><em>Bacillus subtilis</em> NCIB 3610 (purchased from China General Microbiological Culture Collection center, catalog number: <a class="blackNoUnder" target="_blank"> 1.3358 </a> )</p></li><li><p>Tryptone, Bacto (BD Difco, catalog number: <a class="blackNoUnder" target="_blank"> 211705 </a> )</p></li><li><p>Yeast extract, LD, Bacto (BD Difco, catalog number: <a class="blackNoUnder" target="_blank"> 210933 </a> )</p></li><li><p>Bacto Agar (BD Difco, catalog number: <a class="blackNoUnder" target="_blank"> 214010 </a> )</p></li><li><p>NaCl (Beijing Chemical Works, catalog number: <a class="blackNoUnder" target="_blank"> A1060005 </a> )</p></li><li><p>LB agar, powder (Solarbio, catalog number: <a class="blackNoUnder" target="_blank"> L1015 </a> , storage: 2-8 Â°C)</p></li><li><p>SH-PEG modified gold nanorods 40 x 84 nm (NanoSeedz, catalog number: <a class="blackNoUnder" target="_blank"> PEG-40-650-50 </a> )</p></li><li><p>COOH-modified polystyrene microspheres, 0.5 Âµm (Dae technique, catalog number: <a class="blackNoUnder" target="_blank"> PSC-00500 </a> )</p></li><li><p>Gold nanospheres 90 nm and 120 nm</p><p><em> Note: We synthesized the gold nanospheres according to the methods in the reference paper (Zhou et al., 2011). </em> </p></li><li><p>Milli-Q H<sub>2</sub>O (from Milli-Q Water Purification System, Millipore Sigma)</p></li><li><p>Soft LB agar for swarming (see Recipes)</p></li><li><p>LB broth, solution (see Recipes)</p></li></ol><p class="pptt" id="biaoti38668">### Equipment</p><ol class="npwtwok"><li><p>Micro pipettes, 5, 10, 100, 200 and 1,000 Î¼l (Eppendorf)</p></li><li><p>Autoclave (purchased from Alibaba) (brand is not critical)</p></li><li><p>Microwave oven (Meidi) (brand is not critical)</p></li><li><p>4 Â°C and -80 Â°C refrigerator</p></li><li><p>Shaker incubator (Jiecheng Experimental Apparatus, model: <a class="blackNoUnder" target="_blank"> TS-100B </a> )</p></li><li><p>Constant temperature and humidity incubator (Yiheng Instrument, model: <a class="blackNoUnder" target="_blank"> LHS-80HC- </a> )</p></li><li><p>UV-visible spectrophotometer (Shimadzu Corporation, model: <a class="blackNoUnder" target="_blank"> UV1800 </a> )</p></li><li><p>Hand ultraviolet ray examining lamp, 365 nm (Yuhua Instrument, model: <a class="blackNoUnder" target="_blank"> ZF-7 </a> ) (brand is not critical)</p></li><li><p>High-frequency vibrating atomizer, 5 W, 5 V (Purchase from Alibaba, SKG, catalog number: <a class="blackNoUnder" target="_blank"> 37288557693 </a> )</p></li><li><p>Inverted microscope equipped with a 100 W halogen tungsten lamp, a multipurpose dry condenser and polarizer/analyzer combination (Nikon, model: <a class="blackNoUnder" target="_blank"> Ellipse Ti-U </a> )</p></li><li><p>Long working distance objective, CFI TU Plan FLUOR BD 20x, NA/WD 0.45/4.5 mm (Nikon)</p></li><li><p>20x objective, Plan Fluor 20x, NA/WD 0.5/ 2.1 mm (Nikon)</p></li><li><p>Upright metallographic microscope equipped with a 100 W halogen tungsten lamp, and an Epi illuminator containing an orthogonal polarizer/analyzer module (Nikon, model: <a class="blackNoUnder" target="_blank"> LV100D </a> )</p></li><li><p>High-precision piezo-Z positioning stage (PZ-2150)</p></li><li><p>Color CMOS camera (Olympus, model: <a class="blackNoUnder" target="_blank"> DP74 </a> )</p></li><li><p>Upright dark-field microscope with a 100 W halogen tungsten lamp and a dark-field oil condenser (Nikon, model: <a class="blackNoUnder" target="_blank"> 80i </a> )</p></li><li><p>Inverted phase contrast microscope (Olympus, model: <a class="blackNoUnder" target="_blank"> CKX53 </a> )</p></li></ol><p class="pptt" id="biaoti38669">### Software</p><ol class="npwtwok"><li><p>ImageJ (An open source software written in Java, <a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a>)</p></li><li><p>Matlab (The MathWorks, Inc., <a href="https://www.mathworks.com/">https://www.mathworks.com/</a>)</p></li><li><p>Multi-particle tracking software (self-developed in MATLAB, Github: <a href="https://github.com/threebullets/ParDet">https://github.com/threebullets/ParDet</a>)</p></li><li><p>Origin 8 (OriginLab, <a href="https://www.originlab.com/">https://www.originlab.com/</a>)</p></li><li><p>PIVlab-particle image velocimetry (PIV) tool (William Thielicke, 2014) (GUI based tool developed by William Thielicke<u>, </u><a href="https://pivlab.blogspot.com/">https://pivlab.blogspot.com/</a>)</p></li><li><p>SPSS (IBM, <a href="https://www.ibm.com/analytics/spss-statistics-software/">https://www.ibm.com/analytics/spss-statistics-software/</a>)</p></li></ol><p class="pptt" id="biaoti38670">### Procedure</p><ol class="npwone"><li><p>Swarming bacteria culture</p><ol class="npwtwok"><li><p>Prepare the LB broth, Solarbio LB agar and soft LB agar culture solution according to the <strong>Recipes</strong>. First, add weighed reagents into 300 ml Milli-Q H<sub>2</sub>O until completely dissolved. After that, put the dissolved culture solution into the autoclave under 121 Â°C, 20 min. Next put them in the microwave until boiling. Finally place them in the 4 Â°C refrigerator for later use.</p><p><em>Note: While putting the glass bottles into the autoclave, for safety make sure the caps are not too tight. In addition, the prepared culture medium has a shelf-life even if stored in the refrigerator. Stop use it if you find that it cannot maintain the quality as before.</em> </p></li><li><p>Prepare the soft agar plates for bacterial swarming. First boil the soft LB agar solution to 100 Â°C in the microwave oven. After cooling down to 60 Â°C, add 3 ml solution to the 60 mm Petri dishes. Continue to cool down in the ultra-clean workbench until the liquid solidifies into the gel state.</p></li><li><p>Prepare the solid agar plates with Solarbio LB agar solution for bacterial strain activation and transmission. The procedures are like Step A2.</p><p><em>Note: The flatness, hardness as well as the wetness of the agar surface have great influences on the growth and the movements of bacterial colonies. In addition, the thickness of the agar plates is essential for the bacteria culture. For the LB solid agar plates, the thickness is usually 2/3 height of the Petri dishes. For the soft agar plates, we need ensure that the thickness of soft agar plates fits the working distance of the objectives. In addition, the degree of transparency of agar is an important factor affecting microscopic observation.</em> </p></li><li><p>To revive the frozen bacteria strains, dissolve the purchased <em>B. subtilis</em> 3610 powder which were sealed in a vacuum glass vial in the proper amount of LB broth solution and put it into the bacterial culture shaker under 37 Â°C, 200 rpm overnight. Then incubate the next generation on the Solarbio LB agar with streaking method using transfer needles.</p></li><li><p>To obtain the swarming bacterial colony, 5 Âµl of the bacteria overnight culture is inoculated at the center of the soft agar plates. Then the plates are placed in the laminar flow bench air drying for 5-8 min. After that, store it in the incubator at 30 Â°C and at least 90 %rh humidity for about 2-4 h.</p><p><em>Note: Make sure the humidity in the incubator are more than 90 %rh. Because the bacterial colony would not exhibit collective swarming behavior in the environment without enough humidity.</em> </p></li></ol></li><br/><li><p>Preparation of AuNR solutions</p><ol class="npwtwok"><li><p>To examine uniformity of the size and shapes of the particles, measure the UV-Vis spectrum of the SH-PEG AuNRs to characterize the positions of plasmonic resonance absorption peaks and acquire the transmission electron microscope images of the SH-PEG AuNRs. There exist two peaks in the UV spectrum. One is located around 650 nm and the other is around 522 nm. Under electron microscopy, AuNRs should be rod-shaped and uniform in size, with an average size of 40 Ã 84 nm.</p><p><em>Note: To prepare AuNRs samples for TEM examination, pipette 5 Î¼l of undiluted AuNRs (145.36 Î¼g/ml) and drop it on the copper grid. Evaporate naturally at room temperature. Repeat 2 or 3 times.</em> </p></li><li><p>Dilute the SH-PEG AuNR solutions (145.36 Î¼g/ml) for 20-fold with Milli-Q H<sub>2</sub>O.</p></li></ol></li><br/><li><p>Micro-spraying technique</p><ol class="npwtwok"><li><p>Put the AuNRs tracers into the upper surface of swarming bacteria fluid. First add 50 Âµl of the diluted AuNRs solution on the high-frequency vibrating atomizer.</p></li><li><p>Spray the atomized AuNRs aerosol out in a direction parallel to the surface of the swarming colony.</p><p><em>Note: The spraying direction and the distance to the surface of the colony should be adjusted to avoid the damages to the collective moving bacteria. In general, the automizer are placed about 8 cm above the Petri dish. The direction of the spraying is preferably parallel to the colony surface or slightly inclined so that the aerosol could fall naturally onto the surface of the colony. Avoid spraying vertically to the colony.</em> </p></li></ol></li><br/><li><p>Imaging</p><ol class="npwtwok"><li><p>Examine the moving states of the bacterial colony using phase contrast microscope under 20x or 40x objectives.</p></li><li><p>The area of the swarming bacterial colony will continuously expand with the number of bacteria increasing gradually. In the lag phase, the bacteria cells remain stationary. After a lag time of about 2 h, we can observe that the bacteria near the edge of the colony begin to swarm actively in swirls and jets. Once the bacteria start swarming, immediately take the specimen to the inverted microscope (Figure 2A).</p><p><em>Note: Special imaging dishes are not needed.</em> </p></li><li><p>Tune the positions of the dark-field condenser and the working distance of the 20x objectives (Figure 2B) until the red color AuNRs and brown color bacteria could be imaged simultaneously.</p></li><li><p>Capture the images for 40 s with a frame rate of 60 fps using <a class="blackNoUnder" target="_blank"> DP74 </a> color CMOS camera (Figure 2C) in RGB channel under dark-field mode. Three parallel experiments are required.</p><p class="ppzz" style="margin-left:-20px;"><em>Note: The environmental temperature and humidity (30 Â°C, 90 %rh) should be maintained just as in the incubator during the process of imaging. In addition, the intensity of light source of the microscope should not be too high to avoid the photodamages to the specimens. Damage to the cells would cease the collective movements of bacteria.</em> </p></li></ol><p><br/></p><p class="ppzz" style="margin-left:30px;"><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3812/bioprotoc-10-21-3812-g002.jpg"/><br/><strong>Figure 2. Experimental setups for imaging. </strong>A. The inverted Nikon Eclipse Ti-U microscope. The specimens are placed under a dark-field air condenser. B. 20x long working distance objective. C. The Olympus <a class="blackNoUnder" target="_blank"> DP74 </a> color camera.</p></li><br/><li><p>Localize the z-axis position of AuNRs</p><ol class="npwtwok"><li><p>Measure the distance between the bacteria layer and the AuNRs layer under the cross-polarization mode of the upright metallographic microscope.</p></li><li><p>Capture the time-lapsed images of the focal plane of the bacteria layer and the AuNRs layer respectively. Record their z-axis positions.</p></li><li><p>Record the 3D images of optical slicing starting from the bacteria layer to the AuNRs layer through tuning the piezo-Z positioning stage. The z-axis resolution of the piezo-Z positioning stage is 0.01 Âµm. The z-increment steps typically chosen are about 0.2 Âµm.</p></li><li><p>Repeat the experiments in at least three different colonies.</p><p class="ppzz" style="margin-left:-30px;"><em>Note: More elaborate procedures could be found in the section of âtransparent methodsâ-3. Cross polarization microscopy for distance measurements between the AuNRs layer and the bacteria layerâ of original paper.</em> </p></li></ol></li><br/><li><p>Other control experiments</p><ol class="npwtwok"><li><p>To obtain the motions of 90 and 120 nm gold nanospheres on the upper fluid of swarming bacteria, use the same tracing and imaging method as AuNRs particles.</p></li><li><p>To obtain the motions of PS spheres on the upper fluid of swarming bacteria, use the same tracing and imaging method as AuNRs particles.</p></li><li><p>To obtain the motions of AuNRs in the suspensions which are filtered out of bacteria, first filter the cells from the cultured bacteria suspensions with a 0.22 Î¼m diameter syringe filter. Then pipette the 50-fold diluted AuNRs solution to the single concave glass slide. After covering the cover slides, take the specimen to the upright dark-field microscope equipped with 20x Nikon objectives. Then acquire the images with <a class="blackNoUnder" target="_blank"> DP74 </a> color CMOS camera.</p></li><li><p>To obtain the motions of AuNRs in the non-swarming bacterial colony without swarming, irradiate the swarming bacterial colony with a UV lamp for at least 10 h until the collective motions completely stops. Then spray the AuNRs aerosols as the procedures described in Procedure C. Observe the colony under Nikon LV 100D microscope in the reflective dark-field mode.</p></li></ol></li></ol><p class="pptt" id="biaoti38671">### Data analysis</p><ol class="npwone"><li><p>PIV analysis</p><p>Perform the flow field analysis of bacteria swarming on PIVlab software according to the steps below. The work flow is also shown in the Figure 3A (a-f) in detail.</p><ol class="npwtwok"><li><p>Run PIVlabGui.m in MATLAB.</p></li><li><p>Load and import images. Generally, choose the 1-2, 2-3, 3-4 sequencing style unless there are other requirements.</p></li><li><p>Set analysis parameters by Analyses settings &gt; PIV settings &gt; Pass 1 &gt; Pass 2â¦4. Set the PIV algorithm to the FFT window. Define the size of interrogation areas and the steps in pixels.</p><p><em>Note: Use the options for simple image pre-processing if necessary, such as selection of ROI in âExclusionsâ and noise reduction by âEnable CLAHEâ, âEnable highpassâ and other denoise filters.</em> </p></li><li><p>Run PIV analysis by Analysis &gt; Analyze! &gt; Analyze current frame &gt; Analyze all frames.</p></li><li><p>Refine velocity limits by Post processing &gt; Vector validation.</p></li><li><p>Plot PIV images though Plot &gt; Derive parameters/ modify data.</p></li><li><p>Export the txt file, videos, and images Figure 3B by File &gt; save.</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3812/bioprotoc-10-21-3812-g003.jpg"/><br/><strong>Figure 3. PIV analysis of the flow field of collective swarming bacteria.</strong> A. The general work flow while performing PIV analysis. (a) Import the image. (b) Set the PIV interrogation areas and number of passes. (c) Perform the PIV analysis of successive frames. (d) Manually refine the vector limits in the scatterplot of vector field. (e) Extract the parameters on demand. (f) Modify the plot appearance. B. A typical image of the vorticity field overlaid by the velocity vector needles.</p></li></ol></li><br/><li><p>Particle tracking</p><ol class="npwtwok"><li><p>Single particle tracking by ImageJ (Figure 4A)</p><p>Load image sequences by File &gt; import &gt; image sequences. Open the particle tracking plugin by ImageJ &gt; Plugins &gt; Particle Trackers Classic &gt; Particle tracker. Set the particle detection parameters including the particle radius, cutoff, link range and the displacement Figure 4B (a), <em>etc.</em> Visualize and select trajectories Figure 4B (b). Save the tables and images Figure 4B (c-d).</p><p><em>Note: while using ImageJ, only some of the tracers could be tracked easily due to the interference of the cells in the background.</em> </p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3812/bioprotoc-10-21-3812-g004.jpg"/><br/><strong>Figure 4. Single particle tracking analysis.</strong> A.The work panel of ImageJ. B. Figure (a)-(b) exhibit the work flow when using âParticle trackerâ plugins. (c) The results show the coordinates of AuNRs in each frame. (d) The panel shows the overlay of the AuNRs trajectory on the original image. The scale bar is 5 Î¼m.</p></li><br/><li><p>Multi-particle tracking</p><p>To obtain the trajectories of multiple particles in successive frames, we developed a software in MATLAB. The main procedures include image preprocessing, particle recognition based on the local intensity threshold and inter-frame tracking. The codes and detailed instructions are shared in GitHub as indicated in Software.</p><p><em>Note: In the multi-particle tracking software, a local dynamic intensity threshold instead of a global one was set to recognize the particles from the background of cells.</em> </p></li></ol></li><br/><li><p>Diffusion analysis of tracersâ trajectories</p><ol class="npwtwok"><li><p>Mean squared displacements (MSDs)</p><p>Compute the mean squared displacementâ©âr<sup>2</sup> (Ï)âªof each trajectory over different lag times        Then compute the ensemble averaged MSDs for multiple trajectories in the field of view. Plot in log-log scale. See also in the Figure 3C of original paper.</p></li><li><p>The distributions of the direction of motions and turning angles</p><p>Make the right side of the x-axis as positive direction arbitrarily. Define the direction of motions as the direction of each step relative to the x-axis and the turning angles as the directions between successive steps. Plot the probability distribution function (PDF) in the semi-log scale in the Origin employing the calculated data of all trajectories in the field of view. See also the Figure 3A and 3B of original paper.</p></li><li><p>PDF of normalized displacements</p><p>Compute the displacements <em>Îx</em> over different lag times Ît. While comparing the PDF of displacements of different <em>Ît</em>, <em>Îx</em> should be normalized by <em>â2DÎt</em> where D is the diffusion coefficient calculated from the fitting of MSD curves. Plot the PDF in semi-log scale. See also the Figure 3D of original paper.</p></li><li><p>Correlation analysis of angular changes and speed of tracer particles</p><p>Calculate the instantaneous velocity of a particle over time. Perform the correlation analysis of the tracersâ time-dependent angular change (turning angles) and velocity in the SPSS utilizing Pearson correlation function. The p-value are less than 0.05 with the confidence level of 95%. See also the Figures 3E and 3F of original paper.</p></li><li><p>Calculation of local MSD exponents with the slicing window method</p><p>Slice the trajectories into different lengths according to the chosen window size. Then calculate the MSD exponents of each trajectory segment. See the boxplot in Figure 3H of original paper.</p></li></ol></li><br/><li><p>Fitting of the LÃ©vy walk model</p><p>Segment the long trajectory into âflightsâ which consists of several steps by âAngle methodâ modified from Turchin (1998). Next fit the PDF of âflightsâ to the pow-law function, P(x) = C x<sup>-Î¼</sup>,xâ¥ x<sub>min </sub>to get the exponent <em>Î¼</em> through maximum likelihood estimation method. Then justify the power-law model over the exponential model by Akaike weights (       Edwards <em>et al.</em>, 2007      ). The detailed analysis and explanations could be found in Figure 4 and transparent methodsââ6. LÃ©vy walk models and the power law fittingâ of original paper (       Feng <em>et al.</em>, 2019      ).</p></li></ol><p class="pptt" id="biaoti38672">### Recipes</p><ol class="npwtwok"><li><p>Soft LB agar for swarming</p><p>10 g/L Tryptone, Bacto</p><p>5 g/L Yeast extract, LD, Bacto</p><p>5 g/L NaCl</p><p>0.5% Bacto Agar</p></li><li><p>LB broth, solution</p></li></ol><p>       10 g/L Tryptone, Bacto</p><p>        5 g/L Yeast extract, LD, Bacto</p><p>        5 g/L NaCl</p><p class="pptt" id="biaoti38673">### Acknowledgments</p><p>This research was originally published in the Journal of iScience: Feng, J., Zhang, Z., Wen, X., Xue, J. and He, Y. (2019). Single nanoparticle tracking reveals efficient long-distance undercurrent transport in upper fluid of bacterial swarms, iScience 22, 123-132. This work was supported by the National Natural Science Foundation of China with grant numbers of 21425519. The authors of this protocol are grateful to all the contributors of the original research paper.</p><p class="pptt" id="biaoti38674">### Competing interests</p><p>The authors have no competing interests.</p><p class="pptt" id="biaoti38675">### References</p><ol class="npwtwok"><li>Ariel, G., Rabani, A., Benisty, S., Partridge, J. D., Harshey, R. M. and Be'er, A. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26403719" target="_blank">Swarming bacteria migrate by Levy Walk.</a> <em>Nat Commun</em> 68396.</li><li>Be'er and Harshey, R. M. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21889437" target="_blank">Collective motion of surfactant-producing bacteria imparts superdiffusivity to their upper surface.</a> <em>Biophys J</em> 101(5): 1017-1024.</li><li>Burov, S., Jeon, J. H., Metzler, R. and Barkai, E. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21203639" target="_blank">Single particle tracking in systems showing anomalous diffusion: the role of weak ergodicity breaking.</a> <em>Phys Chem Chem Phys</em> 13(5): 1800-1812.</li><li>Copeland, M. F. and Weibel, D. B. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23926448" target="_blank">Bacterial Swarming: A Model System for Studying Dynamic Self-assembly.</a> <em>Soft Matter</em> 5(6): 1174-1187.</li><li>Edwards, A. M., Phillips, R. A., Watkins, N. W., Freeman, M. P., Murphy, E. J., Afanasyev, V., Buldyrev, S. V., da Luz, M. G. E., Raposo, E. P., Stanley, H. E. and Viswanathan, G. M. (2007). <a href="https://pubmed.ncbi.nlm.nih.gov/17960243/" target="_blank">Revisiting LÃ©vy flight search patterns of wandering albatrosses, bumblebees and deer.</a> <em>Nature</em> 4491044-1048.</li><li>Feng, J., Zhang, Z., Wen, X., Xue, J. and He, Y. (2019). <a href="https://www.sciencedirect.com/science/article/pii/S2589004219304626" target="_blank">Single nanoparticle tracking reveals efficient long-distance undercurrent transport in upper fluid of bacterial swarms.</a> <em>iScience</em> 22123-132.</li><li>Hardman, A. M., Stewart, G. S. and Williams, P. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10081580" target="_blank">Quorum sensing and the cell-cell communication dependent regulation of gene expression in pathogenic and non-pathogenic bacteria.</a> <em>Antonie Van Leeuwenhoek</em> 74(4): 199-210.</li><li>Kearns, D. B., (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20694026" target="_blank">A field guide to bacterial swarming motility.</a> <em>Nat Rev Microbiol</em> 8(9): 634-644.</li><li>Koch, D. L. and Subramanian, G. (2011). <a href="https://www.annualreviews.org/doi/abs/10.1146/annurev-fluid-121108-145434" target="_blank">Collective hydrodynamics of swimming microorganisms: living fluids.</a> <em>Annu Rev Fluid Mech</em> 43(1): 637-659.</li><li>Marchetti, M. C., Joanny, J. F., Ramaswamy, S., Liverpool, T. B., Prost, J., Rao, M. and Simha, R. A. (2013). <a href="https://journals.aps.org/rmp/abstract/10.1103/RevModPhys.85.1143" target="_blank">Hydrodynamics of soft active matter.</a> <em>Rev Mod Phys</em> 85(3): 1143-1189.</li><li>Mendelson, N. H., Bourque, A., Wilkening, K., Anderson, K. R. and Watkins, J. C. (1999). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9882676" target="_blank">Organized cell swimming motions in <em>Bacillus subtilis</em> colonies: patterns of short-lived whirls and jets.</a><em>J Bacteriol</em> 181(2): 600-609.</li><li>Rabani, A., Ariel, G. and Be'er, A. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24376741" target="_blank">Collective motion of spherical bacteria.</a> <em>PLoS One</em> 8(12): e83760.</li><li>Taktikos, J., Zaburdaev, V. and Stark, H. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23004782" target="_blank">Collective dynamics of model microorganisms with chemotactic signaling.</a> <em>Phys Rev E Stat Nonlin Soft Matter Phys</em> (1): 051901.</li><li>Turchin, P., (1998). <a href="https://www.journals.uchicago.edu/doi/abs/10.1086/393125" target="_blank">Quantitative analysis of movement: measuring and modeling population redistribution in animals and plants.</a> <em>Q Rev Biol</em> 74(2): 535-550.</li><li>Wensink, H. H., Dunkel, J., Heidenreich, S., Drescher, K., Goldstein, R. E., Lowen, H. and Yeomans, J. M. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22908244" target="_blank">Meso-scale turbulence in living fluids.</a> <em>Proc Natl Acad Sci U S A</em> 109(36): 14308-14313.</li><li>William Thielicke, E. J. S., (2014). <a href="https://openresearchsoftware.metajnl.com/articles/10.5334/jors.bl/" target="_blank">PIVlab â Towards user-friendly, aï¬ordable and accurate digital particle image velocimetry in MATLAB.</a> <em>J Open Resea Soft</em> 2e30.</li><li>Wu, Y. and Berg, H. C. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22371567" target="_blank">Water reservoir maintained by cell growth fuels the spreading of a bacterial swarm.</a> <em>Proc Natl Acad Sci U S A</em> 109(11): 4128-4133.</li><li>Wu, Y., Hosu, B. G. and Berg, H. C. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21300887" target="_blank">Microbubbles reveal chiral fluid flows in bacterial swarms.</a> <em>Proc Natl Acad Sci U S A</em> 108(10): 4147-4151.</li><li>Zhang, H. P., A., Be'er, Florin, E. L. and Swinney, H. L. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20643957" target="_blank">Collective motion and density fluctuations in bacterial colonies.</a> <em>Proc Natl Acad Sci U S A</em> 107(31): 13626-13630.</li><li>Zhang, R., Turner, L. and Berg, H. C. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19966294" target="_blank">The upper surface of an <em>Escherichia coli </em>swarm is stationary.</a> <em>Proc Natl Acad Sci U S A</em> 107(1): 288-290.</li><li>Zhou, R., Xiong, B., He, Y. and Yeung, E. S. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21052644" target="_blank">Slowed diffusion of single nanoparticles in the extracellular microenvironment of living cells revealed by darkfield microscopy.</a> <em>Anal Bioanal Chem</em> 399(1): 353-359.</li></ol>