<body>
            <h1>Determination of Microtubule Lattice Parameters from Cryo-electron Microscope Images Using TubuleJ</h1>
            <p><strong>Authors:</strong> Siou Ku, CÃ©dric Messaoudi, Charlotte Guyomar, Charles Kervrann, Denis ChrÃ©tien</p>
            <h2>Abstract</h2>
            <p><p>The α-β tubulin heterodimer undergoes subtle conformational changes during microtubule assembly. These can be modulated by external factors, whose effects on microtubule structure can be characterized on 2D views obtained by cryo-electron microscopy. Analysis of microtubule images is facilitated if they are straight enough to interpret and filter their image Fourier transform, which provide useful information concerning the arrangement of tubulin molecules inside the microtubule lattice. Here, we describe the use of the TubuleJ software to straighten microtubules and determine their lattice parameters. Basic 3D reconstructions can be performed to evaluate the relevance of these parameters. This approach can be used to analyze the effects of nucleotide analogues, drugs or MAPs on microtubule structure, or to select microtubule images prior to high-resolution 3D reconstructions.</p></p>
            <p><strong>Keywords:</strong> Tubulin, Microtubules, Microtubule lattice parameters, Microtubule polarity, Cryo-electron microscopy, Image analysis, Helical assemblies, Three-dimensional reconstructions</p>
            <h2>Background</h2><p>Microtubules are polymers of the α-β tubulin heterodimer that form tubes of about 25 nm in diameter and several µm in length. Tubulin binds two molecules of guanosine triphosphate (GTP), one of which is hydrolyzed to GDP during assembly. GTP-hydrolysis is key to microtubule dynamics, since it destabilizes their lattice and allows their fast rearrangements and turn-over during cell activity. Works with GTP analogues such as guanylyl-(α, β)-methylene-diphosphonate (GMPCPP) or guanosine 5'-(γ-thio)-triphosphate (GTPγS) gave rise to a model in which GTP-tubulin undergoes a compaction and rotation between its subunits during assembly and GTP-hydrolysis (Zhang <em>et al.</em>, 2015). This model was recently challenged by a study that involved a range of nucleotide analogues and structural approaches, including X-ray crystallography, small angle X-ray diffraction and cryo-electron microscopy (Estévez-Gallego <em>et al.</em>, 2020 ). This study led to the proposal that if tubulin sustains an expansion-compaction conformational change, this would occur after GTP-hydrolysis to facilitate inorganic phosphate release.<br/><br/>Other factors have been shown to modulate the tubulin compaction state or the protofilament skew angle of microtubules (Manka and Moores, 2018). End-binding proteins (EBs) that bind in-between protofilaments have been proposed to accelerate GTP-hydrolysis by compacting tubulin and to induce a left-handed lattice twist (Zhang <em>et al.</em>, 2018 ). Taxol has been reported to give rise to an expanded state similar to that found in GMPCPP microtubules (Vale <em>et al.</em>, 1994; Estevez-Gallego <em>et al.</em>, 2020 ), suggesting that this state reflects a stable conformation of tubulin in microtubules.<br/><br/>While high-resolution analysis of microtubules is prone to describe these conformational changes at the near-atomic level, it requires a careful selection of homogeneous types of microtubules. When reassembled <em>in vitro</em> from pure tubulin, microtubules form a range of structures that differ in terms of protofilament and/or lateral helical start numbers (Chrétien and Fuller, 2000). To accommodate these variations, the protofilaments skew relative to the microtubule longitudinal axis (Langford, 1980). This produces moiré patterns in 2D projection views obtained by cryo-electron microscopy, which provide a direct measure of their protofilament skew angle (Chrétien and Wade, 1991). Moreover, this moiré pattern has an arrowhead shape that reflects microtubule polarity, providing that the protofilament handedness has been unambiguously determined ( Chrétien <em>et al.</em>, 1996 ; Sosa and Chrétien, 1998). The compaction state of tubulin can be accurately measured on diffraction patterns of the microtubule images (Vale <em>et al.</em>, 1994; Hyman <em>et al.</em>, 1995), and the helical rise of the tubulin subunits, which reflects how they interact laterally, can be deduced when all other parameters are known (Chrétien and Fuller, 2000; Estévez-Gallego <em>et al.</em>, 2020).<br/><br/>To facilitate analysis of microtubule images obtained by cryo-electron microscopy, we have developed TubuleJ as a plugin to the multi-platform ImageJ software ( Schneider <em>et al.</em>, 2012). The basic principles of TubuleJ were originally described in (Blestel <em>et al.</em>, 2009 ). The current version has been updated with new routines, and incorporates TomoJ ( Messaoudi <em>et al.</em>, 2007) to perform fast 3D reconstructions of microtubules. The workflow of TubuleJ is presented in Figure 1.<br/><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3814/bioprotoc-10-21-3814-g001.jpg"/><br/><strong>Figure 1. TubuleJ workflow. </strong>A. TubuleJ menu. B. TubuleJ processing routines. Only the steps in black are described in this protocol. The steps âUntwist fiber from fileâ and âCreate stack from curved fiberâ are not described in this protocol. TubuleJ is divided in three âmodulesâ. Module 1 (left) can be used with any type of fiber, while modules 2 (middle) and 3 (right) are specific to microtubules. Module 2 allows determination of microtubule lattice parameters, while module 3 allows basic 3D reconstructions, without differentiating the Î±-Î² tubulin monomers.</p>
        </body>

<p class="pptt" id="biaoti38689">### Equipment</p><ol class="npwtwok"><li><p>Mac-Pro (mi-2010), 2 x 3.46 GHz, 2 x 6-Core Intel Xeon, 96 Go 1333 MHz DDR3 RAM, SSD 1 To, 3 x 6 To Sata disks, 2 x NVIDIA Quadro K5000 graphic cards, macOS version 10.13.6 (Apple Inc.)</p><p><em>Notes:</em> </p><ol class="npwthree"><li><p><em>With the current configuration, determination of microtubule lattice parameters and calculation of a 3D reconstruction can be performed in less than 10 min.</em> </p></li><li><p><em>Since ImageJ is multi-platform, the TubuleJ plugin can be installed and run on any computer equipped with compatible operating systems (MacOs, Linux, Windows). TubuleJ takes advantage of parallel processing (fast Fourier transforms with parallel FFTJ, 3D reconstructions with TomoJ). Therefore, mutli-core processors will speed up the calculations. Allocate a sufficient amount of RAM to ImageJ depending on your configuration and make sure it uses parallel processing (âEdit -&gt; Options -&gt; Memory &amp; Threadsâ¦â).</em> </p></li></ol></li></ol><p class="pptt" id="biaoti38690">### Software</p><ol class="npwtwok"><li><p>ImageJ software: <a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a> </p></li><li><p>TubuleJ plugin to ImageJ: <a href="https://igdr.univ-rennes1.fr/TubuleJ/">https://igdr.univ-rennes1.fr/TubuleJ/</a> </p></li><li><p>FFTJ (implemented in TubuleJ)<u>:</u><a href="https://sites.google.com/site/piotrwendykier/software/parallelfftj">https://sites.google.com/site/piotrwendykier/software/parallelfftj</a> </p><p>Parallel FFTJ is a multithreaded Fast Fourier Transform plugin for ImageJ.</p></li><li><p>TomoJ (implemented in TubuleJ)<u>: </u><a href="https://sourceforge.net/projects/tomoj/">https://sourceforge.net/projects/tomoj/</a> </p><p>TomoJ allows the preprocessing and registration of tilt series before performing 3D reconstructions.</p></li><li><p>UCSF Chimera: <a href="https://www.cgl.ucsf.edu/chimera/">https://www.cgl.ucsf.edu/chimera/</a> </p></li></ol><p class="pptt" id="biaoti38691">### Procedure</p><ol class="npwone"><li><p>TubuleJ install</p><ol class="npwtwok"><li><p>Download TubuleJ and the test image from <a href="https://igdr.univ-rennes1.fr/TubuleJ/">https://igdr.univ-rennes1.fr/TubuleJ/</a>.</p></li><li><p>Unzip TubuleJ.zip and install the TubuleJ folder inside the plugin folder of ImageJ.</p></li></ol></li><br/><li><p>Microtubule selection</p><ol class="npwtwok"><li><p>Open the electron microscope image in ImageJ.</p></li><li><p>Select âPlugins -&gt; TubuleJ -&gt; TubuleJâ. The interface of TubuleJ opens (Figure 1A).</p></li><li><p>Click on âSelect fiberâ and create a folder named âMTaâ. Select this folder as your working directory.</p><p><em>Note: Several microtubules are frequently present in a single image, which requires creating a separate folder for each microtubule that will be analyzed.</em> </p></li><li><p>Fix the pixel size to 2.16 Ã (Figure 2A) and press âOKâ.</p><p><em> Note: The pixel size is automatically recovered from the header if present. If absent or inaccurate, it must be set at this stage. In the present case, the image header proposes 2.21 Ã, but the actual magnification was calibrated using Tobacco Mosaic Virus, which gave a pixel size of 2.16 Ã. If one aims to analyze the compaction state of tubulin (Hyman et al., 1995), it is important to calibrate precisely the images at the working magnifications, since changes in subunit monomer repeats are on the order of a few Ã. </em> </p></li><li><p>Draw a line on the fiber of interest (Figure 2B) and press âOKâ. This defines the rotation angle of the image.</p></li><li><p>Draw a box around the fiber of interest (Figure 2C) and press âOKâ. An image named âimage_orig.tifâ is presented (Figure 2D). Select the whole image (Edit-&gt;Select All) and resize the image to, <em>e.g.</em>, 189 in height, to minimize background (Image-&gt;Crop; Figure 2E). Save this image in the working directory.</p><p><em>Notes:</em> </p><ol class="npwthree"><li><p><em> The procedure from microtubule selection (B) to 3D reconstruction (I) is provided in Video 1. </em> </p></li><li><p><em>As the plugins starts with the current image opened in ImageJ, any file format that can be opened in ImageJ can be used with TubuleJ. It includes classical formats such as tiff. With the install of TomoJ, formats dedicated to electron microscopy such as dm3, mrc, spider or mrc.bz2 are also accepted.</em> </p></li></ol><p><br/></p></li></ol><p class="ppzz" style="margin-left:30px;"><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3814/bioprotoc-10-21-3814-g002.jpg"/><br/><strong>Figure 2. Microtubule selection. </strong>A. Pixel size setting. B. Rotation of the microtubule image. C. Microtubule image boxing. D. Extracted region. E. Extracted region cropped to minimize background.</p><br/><p><!--video3814-1å¼å§--></p><div class="videodiv" id="vd3814-1" style="height:480px;"></div><div class="videophoto" id="vp3814-1"><p class="ppzz" style="margin-left:40px;"><a class="newapic" href="javascript:;" onclick="openvideo(1)" style="background:url('https://en-cdn.bio-protocol.org/video/3814/3814-1s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/></a> </p></div><p><br/></p><p class="ppzz" style="margin-left:40px;"><strong>Video 1.Microtubule image analysis and 3D reconstruction</strong> </p></li><br/><li><p>Microtubule straightening</p><ol class="npwtwok"><li><p>Select âimage_orig.tifâ and click on âUntwist fiberâ. Set the following parameters (Figure 3A): âEnter the width of the sub-images (pixels): 384â; âEnter the step size (pixels): 192â; âEnter the height of the straight fiber image (pixels): 185â. Let checked âCorrect contrastâ; âradius: 60â, let unchecked âDisplay middle step imagesâ and press âOKâ. Orientations of the microtubule sub-regions with respect to the horizontal are indicated in the log window. The extracted image corrected for contrast variations is presented (Figure 3B), together with the first sub-region from the left roughly aligned horizontally (Figure 3C), and a line profile of the equator of the microtubule sub-region Fourier transform (Figure 3D).</p><p><em>Notes:</em> </p><ol class="npwthree"><li><p><em>The width of the sub-regions will depend on the pixel size and the signal/noise ratio present in the images. We commonly use sub-image lengths comprised between 50 and 100 nm at the specimen level. The step size can be set to half the width size, and the height of the straight fiber about 65% larger than the width of a microtubule image (in pixels). Use an odd value for the width size to allow a proper centering of the microtubule image.Note: Grey gradients are essentially due to variations in ice thickness, e.g., from the border to the center of the carbon holes. These must be corrected for proper filtering of the microtubule images. The âradiusâ option corresponds to the value of the Gaussian filter in ImageJ, which is used to produce a background image that is subtracted from the original image.</em> </p></li><li><p><em>The option âDisplay middle step imagesâ presents all the extracts (image named âHorizontal and centered windowsâ), as well as an image showing the local centers (white crosses) connected with a dark line. Analysis of these images can be useful to determine why the algorithm failed during the local determination of microtubule centers.</em> </p></li></ol></li><li><p>Click at the level of the second large peak starting from the left (around 10 in frequency, Figure 3D). The progress of local center calculation can be followed in the log window. A straightened image corrected for contrast variations named âimage_orig-straightâ is presented (Figure 3E).</p><p><em>Notes:</em> </p><ol class="npwthree"><li><p><em> TubuleJ uses the phase of the J0 Bessel function to center the microtubule image (Blestel et al., 2009), which correspond to the 3 to 4 first large peaks in Figure 3D. The higher resolution peaks may overlap with J<sub>N</sub> (N = number of protofilaments, here N = 15). Selection of the second large peak at ~10 in frequency limits the contribution of the JN term that could corrupt the centering process. </em> </p></li><li><p><em>The straightened image is shorter than the original image, since point centers are taken from the middle of the first sub-region to that of the last sub-region. A procedure to straighten the entire image is provided in the TubuleJ manual. This procedure uses the âUntwist fiber from fileâ routine not described in this protocol. This routine can also be used to adjust mis-centered points.</em> </p></li></ol><p><br/></p></li></ol><p class="ppzz" style="margin-left:30px;"><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3814/bioprotoc-10-21-3814-g003.jpg"/><br/><strong>Figure 3. Microtubule straightening. </strong>A. Straightening parameters. B. Contrast corrected image. C. First sub-region from the left. D. Line profile of the equator of the microtubule sub-region Fourier transform. The cross is placed at the level of the second large peak of the <em>J<sub>0</sub></em> Bessel term (red arrow). E. Straightened microtubule image.</p><p><br/></p></li><li><p>Microtubule centering</p><ol class="npwtwok"><li><p>Select âimage_orig-straightâ and click on âCenter straight fiberâ, set the max width of the FFT to 2048 pixels (Figure 4A) and press âOKâ. The Fourier transform of the image is presented (not shown) together with a profile on the equator of the microtubule image FFT (Figure 4B).</p></li><li><p>Click at the level of the third large peak starting from the left (around 55 in frequency). An image named âimage_orig-straight-Centeredâ is presentedâ (Figure 4C), and the translation along the âY-axisâ is indicated in the log window. It should be minimal (less than 1 pixel) if the âimage_orig-straightâ image has not been cropped.</p><p><em>Notes:</em> </p><ol class="npwthree"><li><p><em>When protofilaments are skewed with respect to the longitudinal axis of the microtubule, the profile plot of the equator should be composed only of J<sub>0</sub>, without contribution of J<sub>N</sub> (see the power spectra in Figures 5A-5B and 6A). Hence, it is safe to click on a peak at higher resolution than in the straightening process.</em> </p></li><li><p><em>We have limited the size of the Fourier transform to 4,096 x 4,096 pixels since the centering procedure is computationally intensive. This process can take time, wait until it is finished.</em> </p></li></ol><p><br/></p></li></ol><p class="ppzz" style="margin-left:30px;"><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3814/bioprotoc-10-21-3814-g004.jpg"/><br/><strong>Figure 4. Microtubule centering. </strong>A. Setting of the FFT size. B. Line profile of the equator of the straightened microtubule image Fourier transform. The cross is placed at the level of the third large peak of the <em>J<sub>0</sub></em> Bessel term (red arrow). C. Centered image.</p><p><br/></p></li><li><p>Determination of protofilament handedness and tubulin subunit repeat along protofilaments</p><ol class="npwtwok"><li><p>Select âimage_orig-straight-Centeredâ and click on âAnalyze FFTâ. The Fourier transform of the image is presented together with a horizontal line crossing the power spectrum. Select the FFT window and zoom in until the diffraction pattern fills the image. Adjust the contrast to emphasize the layer-line pattern (<em>e.g.</em>, from 0 to ~3.0E13 in the B&amp;C window).</p></li><li><p>Move the yellow line to cross the J<sub>S</sub> layer line (Figure 5A) and press âOKâ.</p></li><li><p>Move the yellow line to cross the <em>J<sub>N-S</sub></em> layer line (Figure 5B) and press âOKâ. Line profiles of <em>J<sub>S</sub></em> and <em>J<sub>N-S</sub></em> are presented (Figures 5C-5D).</p></li><li><p>In the window âAnalyze FFT parametersâ (Figure 5E), let the default values (âapproximate size in Ã: 40.0â, and ârange of search (in Ã): 10â), and press âOKâ. This sets the interval for searching peaks in the FFT. A window opens asking if you want to close intermediate windows. Press âOKâ. Results are provided in the log window: The âmonomer repeatâ is 40.87 Ã and the protofilament skew angle âthetaâ is positive. These values will be used to determine the protofilament skew angle (G) and to build image stacks suitable for 3D reconstruction (H).</p><p><em>Notes:</em> </p><ol class="npwthree"><li><p><em>The Fourier transform of a helical fiber is characterized by layer lines that correspond to the different helical families present in this fiber. These layer lines are mathematically described by Bessel functions. In the case of microtubules, the Bessel function of order â0â (J<sub>0</sub>) is due to the tubular nature of microtubules. It lies on the âequatorâ of the Fourier transform, and is used to precisely center microtubule images (see Blestel et al., 2009, for details). The Bessel function of order âNâ (J<sub>N</sub>) arises from the linear arrangement of the tubulin molecules along the N protofilaments. Since the protofilaments are almost parallel to the microtubule longitudinal axis, J<sub>N</sub> lies close to J<sub>0</sub> (Figure 5A). The Bessel function of order âSâ is due to the lateral interactions between tubulin monomers around the microtubule lattice. A full turn of this helix includes S monomers in height (between S = 2 and S = 4 in most microtubule types). J<sub>S</sub> lies close to the âmeridianâ of the Fourier transform of microtubule images (Figure 5A, the meridian is perpendicular to the equator). The Bessel function of order âN-Sâ is geometrically related to J<sub>N</sub> and J<sub>S</sub>. J<sub>N-S</sub> is further away from the meridian than J<sub>S</sub> (Figure 5A).</em> </p></li><li><p><em> Determination of the protofilament skew angle sign is based on the following rule (ChrÃ©tien et al., 1996): When J<sub>S</sub> is farther away from the equator than J<sub>N-S</sub>, theta is negative, and when JS is closer to the equator than J<sub>N-S</sub>, theta is positive.</em> </p></li></ol></li></ol><p><br/></p><p class="ppzz" style="margin-left:30px;"><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3814/bioprotoc-10-21-3814-g005.jpg"/><br/><strong>Figure 5. Analysis of the microtubule Fourier transform. </strong>A. Selection of the <em>J<sub>S</sub></em> layer line. The main layer lines are indicated. <em>J<sub>0</sub></em> lies on the equator of the Fourier transform. <em>J<sub>S</sub></em> is closer to the equator than <em>J<sub>N-S</sub></em>, indicating that the microtubule has right-handed protofilaments. B. Selection of the <em>J<sub>N-S</sub></em> layer line. C. Line profile along <em>J<sub>S</sub></em>. D. Line profile along <em>J<sub>N-S</sub></em>. E. Peak search interval.</p><p><br/></p></li><li><p>Microtubule image filtering and determination of microtubule polarity</p><ol class="npwtwok"><li><p>Select âimage_orig-straight_Centeredâ and click on âFilter fiberâ.</p></li><li><p>Select the FFT window (Figure 6A) and zoom in until the diffraction pattern fills the image.</p></li><li><p>Select the rectangular tool of ImageJ and draw a rectangle that incorporate the <em>J<sub>0</sub></em> and the first peak of the <em>J<sub>N</sub></em> term on each side of the equator and press âOKâ.</p></li><li><p>Enter 21 lines and press âOKâ. Click âYesâ to accept the selected region and answer âNoâ in the window âDo you want to select other layer lines?â. An image named âfiltered imageâ is presented (Figure 6B), which emphasizes the moirÃ© pattern originating from the skewed protofilaments imaged in projection.</p></li><li><p>The moirÃ© pattern has an âarrowheadâ shape pointing to the left of the image. Since the protofilaments are right-handed (theta is positive), the microtubule is oriented with its plus end pointing to the left.<em> <br/>Note: The arrowhead pattern is due to the asymmetric shape of tubulin when viewed along the microtubule axis (see Figures 9D and 10). The directionality of this pattern reverses when protofilament have opposite chirality (ChrÃ©tien et al., 1996; Sosa and ChrÃ©tien, 1998). The rule to determine microtubule polarity is the following: The fringe pattern points towards the plus end of microtubules with right-handed protofilaments (Î¸ &gt; 0), and towards the minus end of microtubules with left-handed protofilaments (Î¸ &lt; 0). </em> </p></li></ol><p><br/></p><p class="ppzz" style="margin-left:30px;"><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3814/bioprotoc-10-21-3814-g006.jpg"/><br/><strong>Figure 6. Microtubule image filtering. </strong>A. Selection of <em>J<sub>0</sub></em> and of the first peak <em>J<sub>N</sub></em>. B. Filtered image. The moirÃ© pattern displays an arrowhead shape that points toward the left of the image.</p><p><br/></p></li><li><p>Determination of the protofilament skew angle</p><ol class="npwtwok"><li><p>Select âfiltered imageâ and click on âDetermine protofilament angleâ. Select the âfiltered imageâ (Figure 7A) and move the line with the arrow key so that it crosses minima and maxima of the fringe pattern (6 pixels up) and press âOKâ.</p><p><em>Note: In the case of a microtubule with an even protofilament number, the line should be adequately placed in the middle of the filtered image. Here, the protofilament number is odd (N = 15), which necessitates moving the line slightly off center to cross maxima and minima of the moirÃ© pattern.</em> </p></li><li><p>Three line plots open corresponding to the line profile (âline intensitiesâ, Figure 7B), a smoothed version to attenuate local variations if present (not shown), and the absolute value of the smoothed line profile (âline intensities after abs()â Figure 7C). A window opens providing the measured period (in pixels, Figure 7D), the pixel size and the protofilament separation (set as default to 48.95 Ã). Let the ânegative pf angleâ unchecked since it has been previously determined to be positive and press âOKâ. A window opens to close intermediate windows, press âOKâ.</p></li><li><p>Close the filtered image.</p></li></ol><p class="ppzz" style="margin-left:30px;">The moirÃ© period <em>L</em> (1906.56 Ã) and the protofilament skew angle theta (+1.47 Ã) are given in the log window.</p><p class="ppzz" style="margin-left:30px;"><em>Note: The protofilament skew angle Î¸ is calculated according to the following formula:<br/><img alt="" class="layerphoto" height="50" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201029/20201029184135_2591.jpg" width="147"/></em> </p><p class="ppzz" style="margin-left:30px;"><em> where  Î´x denotes the separation between adjacent protofilaments. The separation between protofilaments x is difficult to retrieve from individual images. If several types of microtubules are present in the specimen, this can be estimated by measuring the change in image width with protofilament number (ChrÃ©tien and Wade, 1991). The default value of 48.95 Ã was derived from medium-resolution 3D maps of microtubules with from 11 to 16 protofilaments (Sui and Downing, 2010). </em> </p><p><br/></p><p class="ppzz" style="margin-left:30px;"><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3814/bioprotoc-10-21-3814-g007.jpg"/><br/><strong>Figure 7. Determination of the protofilament skew angle. </strong>A. Selection of the moirÃ© profile. B. Line profile revealing the moirÃ© period (distance between two maxima or minima). C. Absolute value of the moirÃ© period. This strategy has been implemented to measure low protofilament skew angles where only half the moirÃ© repeat is present (like in most 13_3 microtubules).</p><p><br/></p></li><li><p>Stack generation</p><ol class="npwtwok"><li><p>Select âimage_orig-straight_Centeredâ and press âCreate stack from straight MTâ. Fix the protofilament number to 15 and the helical-start number to 4 (Figure 8A). Let the other parameters as provided and press âOKâ.</p><p><em>Notes:</em> </p><ol class="npwthree"><li><p><em>A âCreate stack from curved MTâ routine is provided to extract stack images from the original (contrast corrected) microtubule image (saved as âorig_corrected.tifâ in the working directory). This option can be useful if one wishes to process the image stack with other 3D reconstruction software.</em> </p></li><li><p><em> A rigorous determination of N and S would require a detailed analysis of the positions and phases of the layer lines in the Fourier transform of the images (Stewart, 1988), which goes beyond the scope of this protocol. The parity of N can be determined from the symmetry of the fringe pattern: it is centered for even protofilament numbers, and off-center for odd protofilament numbers. The value of N can then be estimated by comparing the microtubule diameter with that of other microtubules present in the specimen, and also by comparing the fringe pattern with published images of microtubules with different N_S configurations (ChrÃ©tien and Fuller, 2000). The helical start number S can be estimated from the sign of the protofilament skew angle (Table 1). In the present case, N = 15 and Î¸ is positive, hence S = 4. </em> </p></li></ol><p><br/></p><p><strong>Table 1.</strong><strong> Sign of <em>Î¸</em> for the main microtubule types assembled <em>in vitro</em> from purified tubulin (taken from ChrÃ©tien and Fuller, 2000), n. o.: not observed. </strong> </p><p>These latter N_S configurations require protofilament skew angles &gt; Â± 4Â°, which may represent a limit in the twist that the tubulin molecule can accommodate.</p><p class="ppzz" style="margin-left:-20px;"><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201029/20201029190946_0411.jpg" width="500"/> </p></li><li><p>A window opens asking the approximate MT length (86 pixels; Figure 8B). Let this value and press âOKâ. The incremental âZ-shift â(1.26 pixels) and âAngular shiftâ (-95.966Â°) assigned to the stack images are given in the result window.</p><p><em>Notes:</em> </p><ol class="npwthree"><li><p><em>The âHelical riseâ (r = 9.64 Ã) corresponds to the longitudinal stagger between tubulin monomers in adjacent protofilaments. It is calculated according to the following formula: <br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201029/20201029191737_0224.jpg" width="347"/><br/>where S denotes the monomer helical start number, âaâ the tubulin subunits repeat along protofilaments, N the protofilament number,x the separation between protofilaments, and L the moirÃ© period (with the sign of Î¸).</em> </p></li><li><p><em>The âapproximate MT lengthâ value determines the height of the volume that will be calculated at the following step so that it encompasses at least 4 monomer helical starts.</em> </p></li></ol></li><li><p>A histogram opens showing the angle distribution assigned to the images of the stack (âAngle distributionâ), Figure 8C. A stack is presented (âimage_orig-straight_Centered_angle_-95.966â, Figure 8D), which is composed of 2214 images corresponding to views extracted at each subunit monomer position along the microtubule image.</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3814/bioprotoc-10-21-3814-g008.jpg"/><br/><strong>Figure 8. Stack generation. </strong> A. Stack generation parameters. B. Stack height. C. Angular distribution of the stack images. D. Image stack.</p><br/></li><li><p>Table 2 summarizes the microtubule lattice parameters derived from this analysis:<br/><br/><strong>Table 2. Microtubule lattice parameters. </strong>Polarity is from plus end (left) to minus end (right) in the original image.<br/><span style="color:#666666;font-family:Arial;font-size:10pt;font-weight:lighter;"><img alt="" class="layerphoto" height="80" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201029/20201029193254_0721.jpg" style="color:#666666;font-family:Arial;font-size:13.3333px;text-align:justify;white-space:normal;" width="383"/> </span> </p></li></ol></li><li>3D reconstruction<ol class="npwtwok"><li><p>Click on âCompute MT 3D reconstructionâ. Let âInvert intensities?â checked and the âweighting radiusâ to 0.0 (Figure 9A), then press âOKâ.</p><p><em>Notes:</em> </p><ol class="npwthree"><li><p><em>Images taken on a camera under defocused conditions display protein as dark. Contrast inversion is thus necessary to examine these data in UCSF Chimera (J). Scanned images recorded on negatives display the protein density as white. In that case, uncheck the âInvert intensities?â button.</em> </p></li><li><p><em> The weighting radius can be set to higher values (between 0 and 0.5) in order to attenuate the contribution of low frequencies, which are amplified during back projection (Radermacher, 1992). Yet, this is done at the expense of increasing the noise in the final 3D reconstruction.</em> </p></li></ol></li><li><p>A âProgressionâ¦â window opens (Figure 9B): wait until the âstack_straight_3dReconstructionâ is calculated (Figure 9C). Scroll through this volume, the tubulin subunits (shown in white) are clearly visible in the volume. Select âImage -&gt; Stacks -&gt; Reslice [/]â¦â then select âStart at: topâ and press âOKâ (Figure 9D). The asymmetry of the tubulin subunits indicates that we are looking at this map from the plus end (         ChrÃ©tien <em>et al.</em>, 1996        ; Sosa and ChrÃ©tien, 1998; see also (J)). The top-to-bottom orientation of the 3D map corresponds to the left-to-right orientation in the original image, which indicates that the polarity was well assigned.</p></li><li><p>Select âstack_straight_3dReconstructionâ and click on âElongate 3D MTâ (Figure 9E). Let the 3 first parameters as default, fix the final length to 1024 and press âOKâ. This generates an elongated version of the microtubule where the protofilament skew and its handedness can be observed (Figure 9F).</p><p><em>Note: The z shift and angle shift used to elongate the microtubule correspond to the parameters of the basic one start helix that join all the tubulin subunits within the microtubule lattice. These values are also used to perform the 3D reconstruction of the microtubule, see Blestel et al. (2009) for deeper details.</em> </p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3814/bioprotoc-10-21-3814-g009.jpg"/><br/><strong>Figure 9. Microtubule 3D reconstruction. </strong>A. 3D reconstruction parameters. B. Progression bar. C. 3D reconstruction. D. 3D reconstruction observed from the top. E. Elongation parameters. F. Elongated version of the microtubule. The right-handed skew of the protofilaments is clearly visible.</p><p><br/></p></li><li><p>Examine the working directory, several files have been created (listed in chronological order):</p><ol class="npwthree"><li><p>calibration.cvs: pixelSize, the pixel size in Ã.</p></li><li><p>image_orig.tif: the region extracted from the original image.</p></li><li><p>image_corrected.tif: the region extracted from the original image corrected for contrast variations.</p></li><li><p>centers.txt: number of local centers and their x, y coordinates.</p></li><li><p>straight_image.tif: the straightened image.</p></li><li><p>image_orig-straight_Centered.tif: the straight image centered.</p></li><li><p>analyzeFFT.csv: pixelSize, the pixel size in Ã; monomerRepeat, the monomer repeat along protofilaments (in Ã); thetaSign, the sign of theta (1 positive, -1 negative).</p></li><li><p>filtered image.tif: the filtered image.</p></li><li><p>protofilament.csv: l(pixel), the moirÃ© period in pixels; pixelSize, the pixel size (in Ã); L(angstrom), the moirÃ© period (in Ã), Pf separation(angstrom); the protofilament separation (in Ã); theta, the protofilament skew angle (in Â°).</p></li><li><p>angle_distribution.csv: the number of views assigned in the stack for each angle (Y), from 0Â° to 359Â° (X).</p></li><li><p>angles.txt: the angular orientation assigned to the images of the stack.</p></li><li><p>elongation.csv: zshift, the Z-rise of tubulin monomers with respect to the longitudinal axis of the microtubule; angleshift, the angular increment of tubulin monomers around the microtubule Z axis; nbProtofilaments, the protofilament number; helicalRise, the helical rise of tubulin subunits along adjacent protofilaments.</p></li><li><p>stack_straight.tif: the image stack used for 3D reconstruction.</p></li><li><p>stack_straight_3dreconstruction.tif: the 3D reconstruction of the microtubule.</p></li><li><p>3dReconstruction_elongated.tif: the elongated version of the 3D reconstruction.</p></li></ol></li></ol></li><br/><li><p>Observation of the 3D reconstructions with UCSF Chimera</p><ol class="npwtwok"><li><p>Open the âstack_straight_3dreconstruction.tifâ volume saved in the working directory into UCSF Chimera.</p></li><li><p>Display the command line: âMenu-&gt;Favorites-&gt;Command lineâ.</p></li><li><p>Open the camera viewer: âMenu-&gt;Tools-&gt;Viewing Controls-&gt;Cameraâ and select âorthographicâ.</p></li><li><p>Open the Volume viewer: âMenu-&gt;Tools-&gt;Volume Data-&gt;Volume Viewerâ, and in the âFeaturesâ menu, check âBrightness and Transparencyâ, âCoordinatesâ, and âThreshold and Colorâ.</p></li><li><p>Adjust the âLevelâ to enlarge tubulin densities (<em>e.g.</em>, to âLevel = 300â) and set the âTransparencyâ to 0.50.</p></li><li><p>In the command line, enter âturn x 90 1â.</p></li><li><p>Set the voxel size to 2.16 Ã and press center.</p></li><li><p>Select the âSide Viewâ tab of the âViewing windowâ and press âView Allâ.</p></li><li><p>Open the tubulin map 1tub: âMenu-&gt;File-&gt;Fetch by IDâ¦â, check the âPDBâ button, enter 1tub and press âFetchâ.</p></li><li><p>In the command line, enter âmove 62,-30,-30 mod #1; turn y 90 1 mod #1 center #1â. The tubulin map is oriented with its plus end viewed from the top (Figure 10A). Comparison of the overall shape of the 1tub model with that of the microtubule map confirms that the plus end points towards the top of the reconstruction.</p></li><li><p>In the command line, enter âmove -45,-110,0 mod #0â. Select âVolume Viewer-&gt;Tools-&gt;Fit in Mapâ and fit â1tub (#1) in map âstack_straight_3dreconstruction.tif (#0)â. Inspect the fit of the 1tub model inside the 3D density of the microtubule (Figures 10B-10F).</p><p><em>Notes:</em> </p><ol class="npwthree"><li><p><em>The good agreement between the overall shape of the 1tub model and the 3D reconstruction, including the correspondence of structural motifs such as densities corresponding to the lateral interactions, the H1-S2-loop pointing inward and the C-terminal</em> Î±<em>-helices pointing outward (Figure 10B), give a fair confidence that the parameters derived from the analysis were adequately determined.</em> </p></li><li><p><em>The original image has not been corrected for the effect of the Contrast Transfer Function. This can be performed before processing the images in TubuleJ in order to improve the quality of the final reconstruction.</em> </p></li></ol></li><li><p>Save the session in the working directory as âMTa.pyâ.</p></li><li><p>Close the session and open â3dReconstruction_elongated.tifâ.</p></li><li><p>Set the step size to 2 and adjust the âLevelâ to 300. The right-handed skew of the protofilaments is clearly visualized in this elongated version of the microtubule 3D reconstruction (Figure 10G).</p></li><li><p>Save the session as âMTa_Elongated.pyâ.</p><p><em> Note: The visualization procedure using UCSF Chimera is provided in Video 2. </em> </p></li></ol></li></ol><br/><p class="ppzz" style="margin-left:50px;"><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3814/bioprotoc-10-21-3814-g010.jpg"/><br/><strong> Figure 10. Visualization of the 3D reconstruction with UCSF Chimera. </strong> A. Comparison of the 3D reconstruction with the 1tub tubulin model. B. Fit of 1tub into the 3D reconstruction. Plus end view. Structural features that allow a good match between the tubulin model and the 3D reconstruction are indicated. C. Minus end view. D. Inside view. E. Outside view. F. Side view. G. Elongated version of the 3D reconstruction.</p><br/><p><!--video3814-2å¼å§--></p><div class="videodiv" id="vd3814-2" style="height:480px;"></div><div class="videophoto" id="vp3814-2"><p class="ppzz" style="margin-left:40px;"><a class="newapic" href="javascript:;" onclick="openvideo(2)" style="background:url('https://en-cdn.bio-protocol.org/video/3814/3814-2s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/></a> </p></div><p class="ppzz" style="margin-left:40px;"><strong>Video 2. </strong><strong>Observation of the 3D reconstruction using UCSF Chimera</strong></p><p class="pptt" id="biaoti38692">### Acknowledgments</p><p>This work was funded by the Agence Nationale de la Recherche (ANR-16-C11-0017-01). We thank S. Blestel who wrote the original version of TubuleJ. The determination of microtubule centers used in TubuleJ was published in Blestel <em>et al.</em> (2009). TubuleJ was used in EstÃ©vez-Gallego <em>et al.</em> (2020) to analyze the structure of microtubules assembled in the presence of GTP, GMPCPP and GDP-BeF3-.</p><p class="pptt" id="biaoti38693">### Competing interests</p><p>TubuleJ was deposited to the Agency for the Protection of programs (APP) under the code number IDDN.FR.001.240023.000.S.P.2011.000.21000.</p><p class="pptt" id="biaoti38694">### References</p><ol class="npwtwok"><li>Blestel, S., Kervrann, C. and ChrÃ©tien, D. (2009). <a href="https://ieeexplore.ieee.org/abstract/document/5193043/" target="_blank">A Fourier-based method for detecting curved microtubule centers: Application to straightening of cryo-electron microscope images. </a><em>Proceedings of the IEEE International Symposium on Biomedical Imaging: From Nano to Macro, Boston. 2009, June 28-July 1</em> 298-301.</li><li>ChrÃ©tien, D. and Fuller, S. D. (2000). <a href="https://www.sciencedirect.com/science/article/abs/pii/S0022283600936969" target="_blank">Microtubules switch occasionally into unfavorable configurations during elongation.</a> <em>J Mol Biol</em> 298(4): 663-676.</li><li>ChrÃ©tien, D., Kenney, J. M., Fuller, S. D. and Wade, R. H. (1996). <a href="https://pubmed.ncbi.nlm.nih.gov/8805589/" target="_blank">Determination of microtubule polarity by cryo-electron microscopy.</a> <em>Structure</em> 4(9): 1031-1040.</li><li>ChrÃ©tien, D. and Wade, R. H. (1991). <a href="https://onlinelibrary.wiley.com/doi/abs/10.1016/0248-4900(91)90062-R" target="_blank">New data on the microtubule surface lattice.</a> <em>Biol Cell</em> 71(1-2): 161-174.</li><li>Hyman, A. A., ChrÃ©tien, D., Arnal, I. and Wade, R. H. (1995). <a href="https://rupress.org/jcb/article/128/1/117/20222/Structural-changes-accompanying-GTP-hydrolysis-in" target="_blank">Structural changes accompanying GTP hydrolysis in microtubules: information from a slowly hydrolyzable analogue guanylyl-(alpha, beta)-methylene-diphosphonate.</a> <em>J Cell Biol</em> 128(1): 117-125.</li><li>EstÃ©vez-Gallego, J., Josa-Prado, F., Ku, S., Buey, R. M., Balaguer, F. A., Prota, A. E., Lucena-Agell, D., Kamma-Lorger, C., Yagi, T., Iwamoto, H., Duchesne, L., Barasoain, I., Steinmetz, M. O., ChrÃ©tien, D., Kamimura, S., DÃ­az, J. F. and Oliva, M. A. (2020). <a href="https://doi.org/10.7554/eLife.50155" target="_blank">Structural model for differential cap maturation at growing microtubule ends.</a> <em>eLife</em> 9: e50155.</li><li>Langford, G. M., (1980). <a href="https://rupress.org/jcb/article-lookup/doi/10.1083/jcb.87.2.521" target="_blank">Arrangement of subunits in microtubules with 14 protofilaments.</a> <em>J Cell Biol</em> (1): 521-526.</li><li>Manka, S. W. and Moores, C. A. (2018). <a href="https://portlandpress.com/essaysbiochem/article-lookup/doi/10.1042/EBC20180031" target="_blank">Microtubule structure by cryo-EM: snapshots of dynamic instability.</a> <em>Essays Biochem</em> 62(6): 737-751.</li><li>Messaoudi, C., Boudier, T., , C. O. and Marco, S. (2007). <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-8-288" target="_blank">TomoJ: tomography software for three-dimensional reconstruction in transmission electron microscopy.</a> <em>BMC Bioinformatics</em> 8288-296.</li><li>Radermacher, M., (1992). <a href="https://link.springer.com/chapter/10.1007%2F978-1-4757-2163-8_5" target="_blank">Weighted Back-Projection Methods.</a> In: Frank, J. (Ed.). <em>Electron Tomography.</em> Springer, Boston, MA: 91-115.</li><li>Stewart, M., (1988). <a href="https://onlinelibrary.wiley.com/doi/10.1002/jemt.1060090404" target="_blank">Computer image processing of electron micrographs of biological structures with helical symmetry</a>. <em>J Electron Microsc Tech</em> 9325-358.</li><li>Schneider, C. A., Rasband, W. S. and Eliceiri, K. W. (2012). <a href="http://www.nature.com/nmeth/journal/v9/n7/full/nmeth.2089.html" target="_blank">NIH Image to ImageJ: 25 years of image analysis.</a> <em>Nat Methods</em> 9671-675.</li><li>Sosa, H. and ChrÃ©tien, D. (1998). <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/(SICI)1097-0169(1998)40:1%3C38::AID-CM4%3E3.0.CO;2-B" target="_blank">Relationship between moire patterns, tubulin shape, and microtubule polarity.</a> <em>Cell Motil Cytoskeleton</em> 40(1): 38-43.</li><li>Sui, H. and Downing, K. H. (2010). <a href="https://www.cell.com/structure/fulltext/S0969-2126(10)00221-2?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0969212610002212%3Fshowall%3Dtrue" target="_blank">Structural basis of interprotofilament interaction and lateral deformation of microtubules.</a> <em>Structure</em> 181022-1031.</li><li>Vale, R. D., Coppin, C. M., Malik, F., Kull, F. J. and Milligan, R. A. (1984). <a href="https://pubmed.ncbi.nlm.nih.gov/7916345/" target="_blank">Tubulin GTP hydrolysis influences the structure, mechanical properties, and kinesin-driven transport of microtubules.</a> <em>J Biol Chem</em> 269(38): 23769-23775.</li><li>Zhang, R., Alushin, G. M., Brown, A. and Nogales, E. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26234155" target="_blank">Mechanistic origin of microtubule dynamic instability and its modulation by EB proteins.</a> <em>Cell</em> 162(4): 849-859.</li><li>Zhang, R., LaFrance, B. and Nogales, E. (2018). <a href="https://www.pnas.org/content/115/27/E6191.long" target="_blank">Separating the effects of nucleotide and EB binding on microtubule structure.</a> <em>Proc Natl Acad Sci U S A</em> 115(27): E6191-E6200.</li></ol>