<body>
            <h1>Reference-free Association Mapping from Sequencing Reads Using k-mers</h1>
            <p><strong>Authors:</strong> Zakaria Mehrab, Jaiaid Mobin, Ibrahim Asadullah Tahmid, Lior Pachter, Atif Rahman</p>
            <h2>Abstract</h2>
            <p>Association mapping is the process of linking phenotypes with genotypes. In genome wide association studies (GWAS), individuals are first genotyped using microarrays or by aligning sequenced reads to reference genomes. However, both these approaches rely on reference genomes which limits their application to organisms with no or incomplete reference genomes. To address this, reference free association mapping methods have been developed. Here we present the protocol of an alignment free method for association studies which is based on counting k-mers in sequenced reads, testing for associations between k-mers and the phenotype of interest, and local assembly of the k-mers of statistical significance. The method can map associations of categorical phenotypes to sequence and structural variations without requiring prior sequencing of reference genomes.</p>
            <p><strong>Keywords:</strong> Association mapping, Genome wide association studies (GWAS), Reference free, k-mer</p>
            <h2>Background</h2><p>Association mapping, <em>i.e.</em>, the process of associating genotypes to phenotypes is most frequently performed in the form of genome wide association studies (GWAS) with single nucleotide polymorphisms (SNP). Microarrays are used to genotype individuals at a large number of known SNP locations and each SNP is tested for association with the phenotype of interest. But this approach requires prior sequencing of a reference genome and determining the locations of the SNPs. Moreover, this precludes mapping associations to structural variations such as insertion-deletions (indels) and copy number variations, and to variations outside of the reference genome.<br/> <br/>With advances in sequencing technologies, the use of whole genome sequenced reads for association mapping is increasingly becoming more widespread. This is most commonly done by mapping the reads to a reference genome, calling variants, and testing for association between the variants and the phenotype. However, this approach also requires a reference genome and regions missing from the reference are not included in the study.<br/><br/>To address these issues, a number of reference free methods for association mapping have been developed. They are based on testing for association between k-mers, <em>i.e.</em>, contiguous sequence of length k in sequenced reads and the phenotype. Sheppard <em>et al.</em>, 2013, Earle <em>et al.</em>, 2016, Lees <em>et al.</em>, 2016 and Jaillard <em>et al.</em>, 2018 presented methods for association mapping in bacterial genomes where high plasticity makes application of reference based methods difficult. Rahman <em>et al.</em>, 2018 introduced a method for mapping associations to categorical phenotypes applicable to organisms with large genomes and more recently Voichek <em>et al.</em>, 2020 presented a method for both categorical and quantitative phenotypes.<br/> <br/>Here we present the protocol of the reference free association mapping tool HAWK, which was developed by Rahman <em>et al.</em>, 2018 and extended by Mehrab <em>et al.</em>, 2020. It works by first counting k-mers in reads from each individual using Jellyfish (Marçais and Kingsford, 2011). Then likelihood ratio test is used to find k-mers with significantly different counts in case and control samples. Next, population structure is determined using Eigenstrat (Patterson <em>et al.</em>, 2006, Price <em>et al.</em>, 2006). Finally, k-mers associated with the phenotype are identified and the k-mers are locally assembled to get a sequence for each associated loci. The results found by HAWK were found to be largely in agreement with reference based methods. Moreover, HAWK was able to map associations to structural variants and to variants in regions not present in the reference. It is worth reiterating that the method is applicable for any genetic diseases or traits. However, it currently only supports categorical, <em>i.e.</em>, binary phenotypes although work is ongoing to extend it to quantitative phenotypes.<br/></p>
        </body>

<p class="pptt" id="biaoti38697">### Equipment</p><ol class="npwtwok"><li><p>Computer (We recommend at least 16GB RAM and multiple cores)</p></li></ol><p class="pptt" id="biaoti38698">### Software</p><ol class="npwtwok"><li><p>HAWK (       Rahman <em>et al.</em>, 2018      ;        Mehrab <em>et al.</em>, 2020      )</p><p>The primary software for association mapping from sequencing reads using k-mers. The software is available for download at <u><a href="https://github.com/atifrahman/HAWK/releases">https://github.com/atifrahman/HAWK/releases</a></u> and installation instructions are at <u><a href="https://github.com/atifrahman/HAWK">https://github.com/atifrahman/HAWK</a></u>.</p></li><li><p>Modified version of Jellyfish/Jellyfish 2 (MarÃ§ais and Kingsford, 2011)</p><p>For k-mer counting. Modified versions available for download at <a href="https://github.com/atifrahman/HAWK/tree/master/supplements">https://github.com/atifrahman/HAWK/tree/master/supplements</a> and installation instructions are in the README.md file.</p></li><li><p>Modified version of EIGENSTRAT (       Patterson <em>et al.</em>, 2006      ,        Price <em>et al.</em>, 2006      )</p><p>For population structure determination. A modified version can be downloaded from <a href="https://github.com/atifrahman/HAWK/tree/master/supplements">https://github.com/atifrahman/HAWK/tree/master/supplements</a> and installation instructions are available in the README file.</p></li><li><p>ABySS (       Simpson <em>et al.</em>, 2009      )</p><p>To assemble k-mers of statistical significance. Download and installation instructions are at <a href="https://github.com/bcgsc/abyss">https://github.com/bcgsc/abyss</a>.</p></li><li><p>GNU sort with parallel support. Usually included with Linux distribution</p></li><li><p>R. Available for download at <u><a href="https://cran.r-project.org/">https://cran.r-project.org/</a></u> </p></li></ol><p class="pptt" id="biaoti38699">### Procedure</p><p>The overview of the procedure of association mapping from sequencing reads with k-mers using HAWK is shown in Figure 1. The steps are described in more details below. More information on the process is available at <a href="https://github.com/atifrahman/HAWK/blob/master/README.md">https://github.com/atifrahman/HAWK/blob/master/README.md</a>.</p><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3815/bioprotoc-10-21-3815-g001.jpg"/><br/><strong>Figure 1. Overview of association mapping from sequencing reads with k-mers using HAWK</strong> <br/><br/><ol class="npwone"><li><p>Counting and sorting k-mers</p><ol class="npwtwok"><li><p>Count k-mers in each sample by running Jellyfish or Jellyfish 2. To do this, modify the example scripts available at <u><a href="https://github.com/atifrahman/HAWK/tree/master/supplements">https://github.com/atifrahman/HAWK/tree/master/supplements</a></u>.</p></li><li><p>countKmers_jf2 or countKmers_jf1 can be used to count and sort k-mers using Jellyfish 2 or Jellyfish respectively when the reads from each sample are in a separate folder whereas countKmers_jf2_sra or countKmers_jf1_sra can be used when reads first need to be downloaded from the sequence read archive (SRA).</p></li><li><p>The scripts will write the names of the sorted k-mer count files in 'sorted_files.txt' and the total k-mer counts in each sample in 'total_kmer_counts.txt'.</p><br/></li></ol></li><li><p>Running HAWK</p><ol class="npwtwok"><li><p>Create a file named 'gwas_info.txt' containing three columns separated by tabs in each row giving a sample ID, male/female/unknown denoted by M/F/U and Case/Control status of the sample for each sample.</p></li><li><p>Copy the files 'sorted_files.txt' and 'total_kmer_counts.txt' into a folder as well as the file 'gwas_info.txt'</p></li><li><p>Copy the scripts 'runHawk' and 'runAbyss' into the folder, edit the variables hawkDir, eigenstratDir and isDiploid, and run ./runHawk</p><p>This will complete the following steps:</p><ol class="npwthree"><li><p>Identifying k-mers associated with cases and controls. This will be done by running âhawkâ and will generate the initial lists of k-mers associated with cases and controls in files âcase_out_w_bonf.kmerDiffâ and âcontrol_out_w_bonf.kmerDiffâ respectively as well as the files to be used by Eigenstrat to determine population structure.</p></li><li><p>Determining population structure. Population structure will be detected by running Eigenstrat. Optionally, the population structure can be investigated by running the R script pca_plot.R. This will read the âgwas_eigenstrat.evecâ file generated by Eigenstrat and output the principal component analysis (PCA) plot in âpca_plot.epsâ. Adjust the variables PC1 and PC2 to select along which principal components the data will be plotted.</p></li><li><p>Correcting for population structure. The p-values for the k-mers identified in Step B3a will be adjusted for confounding factors such as population structure, total number of k-mers and sex. The k-mers found significantly associated with cases and controls will be in files 'case_kmers.fasta' and 'control_kmers.fasta'. Additional information about the k-mers will be in âpvals_case_top_merged.txtâ and âpvals_control_top_merged.txt.â</p><br/></li></ol></li></ol></li></ol><ol class="npwone" start="3"><li><p>Assembling k-mers</p><ol class="npwtwok"><li><p>Edit the variable âabyssDirâ in the script ârunAbyssâ and run ./runAbyss<br/>This will assemble the k-mers to generate one sequence for each associated region and the sequences associated with cases and controls will be in âcase_abyss.25_49.faâ and âcontrol_abyss.25_49.faâ respectively.</p></li></ol></li><li><p>Downstream analysis</p><p>Once the k-mers or the assembled sequences are obtained, they can be analyzed in a number of ways.</p><ol class="npwtwok"><li><p>To obtain summary stats such as average p-values, average counts of constituent k-mers and average number of times they are present in cases and controls edit the HAWK directory, input filename, and whether the sequences are from case or control in the script ârunKmerSummaryâ (at <u><a href="https://github.com/atifrahman/HAWK/tree/master/supplements">https://github.com/atifrahman/HAWK/tree/master/supplements</a></u>) and run ./runKmerSummary (see <u><a href="https://github.com/atifrahman/HAWK">https://github.com/atifrahman/HAWK</a></u> for details).</p></li><li><p>If no reference genome is available, BLAST (         Altschul <em>et al.</em>, 1990        ) the sequences to check for hits to sequences in related organisms and analyze the matched sequences.</p></li><li><p>If a reference genome is available, the k-mers can be mapped to the reference using a tool such as Bowtie 2 (Langmead and Salzberg, 2012) and their positions and p-values can be visualized using Manhattan plots. Edit the shell script ârunBowtie2â and the R script âmanhattan_plasmid.Râ, available at <a href="https://github.com/atifrahman/HAWK/tree/master/ecoli_analysis">https://github.com/atifrahman/HAWK/tree/master/ecoli_analysis</a> to align the k-mers to a reference and generate Manhattan plots respectively.</p></li></ol></li></ol><p class="pptt" id="biaoti38700">### Data analysis</p><p>To identify the k-mers present significantly more times in cases or controls compared to the other, HAWK assumes that k-mer counts are Poisson distributed and performs a likelihood ratio test. Population structure is determined by running Eigenstrat to do a principal component analysis on the binary matrix denoting presence or absence of a random set of k-mers. Correction of population structure and other confounders is done by fitting logistic regression models of the phenotype against the confounders as well as a k-mer count vector and the confounders and adjusted p-values for the k-mers identified in the first step are identified. Bonferroni correction is performed to correct for multiple testing. See      Rahman <em>et al.</em>, 2018     and      Mehrab <em>et al.</em>, 2020     for a detailed description of the methods and supporting results.</p><p>    Next we present an example data analysis using the pipeline. We use the <em>E. coli</em> ampicillin resistance dataset from      Earle <em>et al.</em>, 2016    , which was also analyzed by      Rahman <em>et al.</em>, 2018    . The dataset contains sequenced reads from 241 strains, of which 189 were ampicillin resistant and the rest were not. First, Jellyfish 2 was used to count k-mers in reads from each sample. Of the 176,284,643 distinct k-mers in total, the first step of HAWK identified 4,752,738 and 4,007,202 k-mers to be associated with cases and controls respectively before correcting for confounders. Next, Eigenstrat was run on 342,988 randomly chosen k-mers to detect population structure. Figure 2A shows the PCA plot of the samples along the first two principal components revealing population stratification.</p><p>    We then adjust the p-values using the first ten principal components and total number of k-mers in each sample. After correcting for confounders, we get 4,125 k-mers associated with cases and none associated with controls. The k-mers associated with cases were then assembled with ABySS, revealing 11 sequences.</p><p>    The k-mers found associated with ampicillin resistance were mapped to the <em>E. coli</em> strain DTU-1 genome [GenBank: CP026612.1] and the <em>E. coli</em> strain KBN10P04869 plasmid pKBN10P04869A sequence [GenBank: CP026474.1] using Bowtie 2. Manhattan plots in Figures 2B and 2C show <strong>-log</strong><sub><strong>10 </strong></sub><strong>â¡(</strong><em><strong>p-values</strong></em><strong>) </strong>of the k-mers against their locations in <em>E. coli</em> strain DTU-1 genome and plasmid pKBN10P04869A sequence respectively. The vertical lines denote locations of the Î²-lactamase TEM-1 gene. We observe that no k-mers map to the <em>E. coli</em> strain DTU-1 genome. However, k-mers map to the plasmid pKBN10P04869A sequence near the Î²-lactamase TEM-1 gene, the presence of which is known to provide ampicillin resistance.</p><br/><p><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201029/20201029021635_9080.png"/><br/><strong>Figure 2. Association mapping of ampicillin resistance in <em>E. coli</em></strong> (A) Plots of the first two principal components of the <em>E. coli</em> strains in the ampicillin resistance dataset. Manhattan plots showing -log<sub>10â¡</sub>(<em>p-values</em>) of k-mers associated with ampicillin resistance and their locations in (B) Escherichia coli strain DTU-1 genome and (C) plasmid pKBN10P04869A sequence.</p><p class="pptt" id="biaoti38701">### Notes</p><ol class="npwtwok"><li><p>By default, HAWK uses Bonferroni correction to address the issue of multiple testing. If the study is underpowered for Bonferroni correction, the Benjamini-Hochberg procedure can be used instead. For this after executing ârunHawkâ, run ./runBHCorrection. The resulting k-mers will be in âcase_kmers_bh_correction.fastaâ and âcontrol_kmers_bh_correction.fastaâ.</p></li><li><p>HAWK uses first two principal components found using Eigenstrat, sex of samples and sequencing depth in the form of total k-mer counts as confounders, by default. To change the default settings, edit the variables ânoPCâ and âuseSexConfounderâ in the script ârunHawkâ. In order to provide additional confounders, create a file with the number of lines equal to the number of samples and in each line specify the covariates, given by numbers and separated by spaces or tabs. Edit the variable âcovFileâ in ârunHawkâ with the name of the confounder file.</p></li></ol><p class="pptt" id="biaoti38702">### Recipes</p><p>The HAWK pipeline can be used to find sex specific k-mers. In order to do this, in the file 'gwas_info.txt', provide the sample IDs in the first column, write U's in the second column and specify contain Case/Control status depending on whether the sample is Male/Female in the third column. For example if SRR3050845 and SRR3050847 are female and SRR3050846 is male, the file will be:</p><p>SRR3050845      U      Control</p><p>SRR3050846      U      Case</p><p>SRR3050847      U      Control</p><p class="pptt" id="biaoti38703">### Acknowledgments</p><p>Lior Pachter, and Atif Rahman were funded in part by NIH R21 HG006583. This paper describes protocol of a method originally presented in the paper âAssociation mapping from sequencing reads using k-mersâ by Atif Rahman, Ingileif HallgrÃ­msdÃ³ttir, Michael Eisen and Lior Pachter, and extended in âA faster implementation of association mapping from k-mersâ by Zakaria Mehrab, Jaiaid Mobin, Ibrahim Asadullah Tahmid and Atif Rahman.</p><p class="pptt" id="biaoti38704">### Competing interests</p><p>The authors declare no competing interests.</p><p class="pptt" id="biaoti38705">### References</p><ol class="npwtwok"><li>Altschul, S. F., Gish, W., Miller, W., Myers, E. W. and Lipman, D. J. (1990). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" target="_blank"> Basic local alignment search tool.</a> <em>J Mol Biol</em> 215(3): 403-410.</li><li>Earle, S. G., Wu, C. H., Charlesworth, J., Stoesser, N., Gordon, N. C., Walker, T. M., Spencer, C. C., Iqbal, Z., Clifton, D. A., Hopkins, K. L. and Woodford, N. (2016). <a href="https://pubmed.ncbi.nlm.nih.gov/27572646/" target="_blank"> Identifying lineage effects when controlling for population structure improves power in bacterial association studies.</a> <em>Nature Microbiol</em> 1(5): 1-8.</li><li>Jaillard, M., Lima, L., Tournoud, M., Mahe, P., van Belkum, A., Lacroix, V. and Jacob, L. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30419019" target="_blank"> A fast and agnostic method for bacterial genome-wide association studies: Bridging the gap between k-mers and genetic events.</a> <em>PLoS Genet</em> 14(11): e1007758.</li><li>Langmead, B. and Salzberg, S. L. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" target="_blank"> Fast gapped-read alignment with Bowtie.</a> <em>Nat Methods</em> 9(4): 357-359.</li><li>Lees, J. A., Vehkala, M., Valimaki, N., Harris, S. R., Chewapreecha, C., Croucher, N. J., Marttinen, P., Davies, M. R., Steer, A. C., Tong, S. Y., Honkela, A., Parkhill, J., Bentley, S. D. and Corander, J. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27633831" target="_blank"> Sequence element enrichment analysis to determine the genetic basis of bacterial phenotypes.</a> <em>Nat Commun</em> 712797.</li><li>MarÃ§ais, G. and Kingsford, C. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122" target="_blank"> A fast, lock-free approach for efficient parallel counting of occurrences of k-mers.</a> <em>Bioinformatics</em> 27(6): 764-770.</li><li>Mehrab, Z., Mobin, J., Tahmid, I. A. and Rahman, A. (2020). <a href="https://www.biorxiv.org/content/10.1101/2020.04.14.040675v1" target="_blank"> A faster implementation of association mapping from k-mers.</a> <em>bioRxiv.</em> .</li><li>Patterson, N., Price, A. L. and Reich, D. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17194218" target="_blank"> Population structure and eigenanalysis.</a> <em>PLoS Genet</em> 2(12): e190.</li><li>Price, A. L., Patterson, N. J., Plenge, R. M., Weinblatt, M. E., Shadick, N. A. and Reich, D. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16862161" target="_blank"> Principal components analysis corrects for stratification in genome-wide association studies.</a> <em>Nat Genet</em> 38(8): 904-909.</li><li>Rahman, A., Hallgrimsdottir, I., Eisen, M. and Pachter, L. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29897334" target="_blank"> Association mapping from sequencing reads using k-mers.</a> <em>Elife</em> 7: e32920.</li><li>Sheppard, S. K., Didelot, X., Meric, G., Torralbo, A., Jolley, K. A., Kelly, D. J., Bentley, S. D., Maiden, M. C., Parkhill, J. and Falush, D. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23818615" target="_blank"> Genome-wide association study identifies vitamin B5 biosynthesis as a host specificity factor in Campylobacter.</a> <em>Proc Natl Acad Sci U S A</em> 110(29): 11923-11927.</li><li>Simpson, J. T., Wong, K., Jackman, S. D., Schein, J. E., Jones, S. J. and Birol, I. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19251739" target="_blank"> ABySS: a parallel assembler for short read sequence data.</a> <em>Genome Res</em> 19(6): 1117-1123.</li><li>Voichek, Y. and Weigel, D. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/32284578" target="_blank"> Identifying genetic variants underlying phenotypic variation in plants without complete genomes.</a> <em>Nat Genet</em> 52(5): 534-540.</li></ol>