<body>
            <h1>Giant Mimiviridae CsCl Purification Protocol</h1>
            <p><strong>Authors:</strong> Lionel Bertaux, Audrey Lartigue, Sandra Jeudy</p>
            <h2>Abstract</h2>
            <p>While different giant viruses’ purification protocols are available, they are not fully described and they use sucrose gradient that does not reach an equilibrium. Here, we report a protocol for the purification of members of the <em>Mimiviridae</em> family virions resulting from <em>Acanthamoeaba castellanii</em> infections. Viruses are harvested after cell lysis and purified through a high density CsCl gradient to optimize the isolation of the virus from the cell debris or other potential contaminants. Due to the large size of the virion capsids, reaching half a micrometer diameter, the quality of the process can be monitored by light microscopy. The resulting purified particles can then be used to perform new infections, DNA extraction, structural studies, sugar composition analyses, sub-compartment characterization or proteomic experiments.</p>
            <p><strong>Keywords:</strong> Giant viruses, Mimivirus, CsCl density gradient, Virion purification, Ultracentrifugation</p>
            <h2>Background</h2><p>The discovery of Mimivirus, the first virus visible under a light microscope, overlapping in size and genome complexity with unicellular organisms, initiated a new research area in virology (La Scola <em>et al.</em>, 2003 ; Raoult <em>et al.</em>, 2004). Over the past 15 years, many additional members of the Mimiviridae family have been isolated from various environments and several protocols have been published to purify the virions (La Scola <em>et al.</em>, 2003; Byrne <em>et al.</em>, 2009; Arslan <em>et al.</em> ; Philippe <em>et al.</em>, 2013 ; Campos <em>et al.</em>, 2014 ; Andrade <em>et al.</em>, 2017). Different approaches were developed, mostly involving sucrose cushion (Campos <em>et al.</em>, 2014 ; Andrade <em>et al.</em>, 2017), or sucrose discontinuous gradients (Arslan <em>et al.</em>, 2011). However, these protocols are not optimal since the density of the virus is higher than the maximum density of a sucrose solution (1.36 g/cm<sup>3</sup> and 1.3 g/cm<sup>3</sup>, respectively – ICTV 9<sup>th</sup> report, 2011), meaning the equilibrium cannot be reached. Thus, a long time or high speed centrifugation results in a viral pellet instead of a ring or both depending on the conditions used. We recently reported (Jeudy <em>et al.</em>, 2019) an optimized version of our previously published protocol using a CsCl discontinuous gradient (Byrne <em>et al.</em>, 2009). Here, we provide the detailed protocol for the purification of Mimiviridae particles.</p>
        </body>

<p class="pptt" id="biaoti38907">## Materials and Reagents</p><p><strong>General use</strong> </p><ol class="npwtwo"><li>Standard pipette filter tips (TipOne filter tips) (StarLab, catalog numbers: <a class="blackNoUnder" target="_blank"> S1120-3810 </a> , <a class="blackNoUnder" target="_blank"> S1120-1810 </a> , <a class="blackNoUnder" target="_blank"> S1120-8810 </a> and <a class="blackNoUnder" target="_blank"> S1112-1720 </a> or equivalent)</li><li>Pipettes (10 ml serological pipette) (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 86.1254.001 </a> )</li><li>Ultrapure water<br/></li></ol><br/><strong>Cell culture</strong><br/><ol class="npwtwo"><li>175 cm<sup>2</sup> flasks (Greiner Bio-One, catalog number: <a class="blackNoUnder" target="_blank"> 660175 </a> , or equivalent)</li><li>Filtropur S 0.2 (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 83.1826.001 </a> )</li><li>Stericup Quick Release Express Plus 0.22 Âµm PES (Merck Millipore, catalog number: <a class="blackNoUnder" target="_blank"> S2GPU11RE </a> )</li><li><em>Acanthamoeba castellanii</em> (Douglas) Page (ATCC, catalog number: <a class="blackNoUnder" target="_blank"> 30010 </a> )</li><li>D-(+)-glucose (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> G8270 </a> )</li><li>Sodium citrate tribasic trihydrate (Sigma, catalogue number: <a class="blackNoUnder" target="_blank"> C7254 </a> )</li><li>Proteose-Peptone (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 82450 </a> )</li><li>Yeast extract (Fisher BioReagent, catalog number: <a class="blackNoUnder" target="_blank"> BP1422 </a> )</li><li>Ampicillin Sodium Salt, Cell Culture/Molecular Biology Grade (Euromedex, catalog number: <a class="blackNoUnder" target="_blank"> EU0400 </a> )</li><li>Kanamycin Sulfate, Cell Culture Grade (Euromedex, catalog number: <a class="blackNoUnder" target="_blank"> UK0010 </a> )</li><li>D-(+)-Glucose (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> G8270 </a> )</li><li>Proteose Peptone Yeast Extract medium (PPYG; see Recipes)<br/>Base medium<br/>Complete medium</li><li>36% Glucose solution (see Recipes)</li><li>Ampicillin stock solution (100 mg/ml) (see Recipes)</li><li>Kanamycin stock solution (25 mg/ml) (see Recipes)<br/><div style="white-space:nowrap;"><br/></div></li></ol><strong>Gradient preparation and recovery</strong><br/><ol class="npwtwo"><li>Plastic Syringes (5 ml syringe) (Terumo, catalog number: <a class="blackNoUnder" target="_blank"> SS*05SE1 </a> )</li><li>21G x 1<sup>1/2â</sup> needle (Terumo, catalog number: <a class="blackNoUnder" target="_blank"> AN*2138R1 </a> or equivalent)</li><li>Sterile 50 ml conical tubes (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 62.547.254 </a> , or equivalent)</li><li>Polyallomer Tubes, 38.5 ml (Beckman, catalog number: <a class="blackNoUnder" target="_blank"> 326823 </a> )</li><li>UltraPure Cesium chloride Optical Grade (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> 15507-023 </a> )</li><li>Cesium chloride solutions (1.2 g/cm<sup>3</sup>, 1.3 g/cm<sup>3</sup>, 1.4 g/cm<sup>3</sup> and 1.5 g/cm<sup>3</sup>; see Recipes)</li><li>K36 buffer (see Recipes)<br/></li></ol><br/><strong>Purification quality control<br/></strong> <ol class="npwtwo"><li>Glass microscope slide (Biosigma, catalog number: <a class="blackNoUnder" target="_blank"> VBS653/A </a> )</li><li>Coverslip (Biosigma, catalog number: <a class="blackNoUnder" target="_blank"> VBS636 </a> )</li><li>UVette 220-1600 nm (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 952010051 </a> )</li><li>Formvar and carbon coated 200 mesh copper/rhodium grids (Electron Microscopy Sciences, catalog number: <a class="blackNoUnder" target="_blank"> FCF200-Cu </a> )</li><li>Uranyl acetate 1% in water<br/><em>Note: Uranyl acetate is a radioactive chemical. It is provided by electron microscopy facilities as we are not allowed to use it in the lab.</em> </li></ol><p class="pptt" id="biaoti38908">### Equipment</p><ol class="npwtwok"><li><p>Microbiological safety hood (ADS laminaire, model: <a class="blackNoUnder" target="_blank"> Optimal 12 </a> , or equivalent)</p></li><li><p>Incubator Bio Performance (Froilabo, catalog number: <a class="blackNoUnder" target="_blank"> BP240 </a> , or equivalent)</p></li><li><p>Centrifuge (Beckman Coulter, model: <a class="blackNoUnder" target="_blank"> JXN-30 </a> )</p></li><li><p>Swinging-Bucket Rotor (Beckman Coulter, model: <a class="blackNoUnder" target="_blank"> JS24-38 </a> )</p></li><li><p>Spectrophotometer (Eppendorf, model: <a class="blackNoUnder" target="_blank"> Biophotometer </a> )</p></li><li><p>Inverted microscope (Zeiss, model: <a class="blackNoUnder" target="_blank"> Axio Observer.Z1 </a> )</p></li><li><p>Transmission Electron Microscope (FEI, model: <a class="blackNoUnder" target="_blank"> Tecnai G2 </a> )</p></li><li><p>Autoclave (SHP Steriltechnik SG, model: <a class="blackNoUnder" target="_blank"> Laboklav 135 </a> )</p></li><li><p>Peristaltic pump (KNF Lab, model: <a class="blackNoUnder" target="_blank"> Laboport </a> )</p></li></ol><p class="pptt" id="biaoti38909">### Procedure</p><ol class="npwtwok"><li><p>Seed 20 175 cm<sup>2</sup> flasks with 1,000 <em>Acanthamoeba castellanii</em> cells per cm<sup>2</sup> in 20 ml of PPYG medium. Incubate at 32 Â°C for two days.</p><p><em>Note: You can add ampicillin and kanamycin at final concentrations of 0.1 mg/ml and 0.025 mg/ml, respectively, in each flask to avoid bacterial contamination.</em> </p></li><li><p>After two days, the cell confluence should be between 130,000 and 180,000 cells/cm<sup>2</sup>. Infect the cells with a Mimivirus stock at a multiplicity of infection of 0.2.</p><p>Incubate at 32 Â°C for two days or until cell lysis is complete.</p><p><em>Note: There is no need to change the medium before infection or to remove the virus from the flasks.</em> </p></li><li><p>Harvest the cell debris-containing supernatants from the flasks and pool them into 50 ml conical tubes.</p><p><em> Note: For 20 175 cm<sup>2</sup> flasks containing 20 ml of medium you will need 7 tubes. </em> </p></li><li><p>Centrifuge at 500 <em>x g</em> for 10 min at room temperature (~25 Â°C, RT) to remove the cell debris.</p></li><li><p>Decant the supernatant into new 50 conical tubes.</p></li><li><p>Centrifuge at 10,000 <em>x g</em> for 25 min at RT to pellet the virus.</p></li><li><p>Discard the supernatant.</p></li><li><p>Resuspend the resulting virus pellets in 10 ml K36 buffer by pipeting and pool in two new 50 ml conical tubes (35 ml per tube). Complete the volume to 50 ml with K36 buffer.</p></li><li><p>Centrifuge at 10,000 <em>x g</em> for 25 min at RT to pellet the virus.</p></li><li><p>During the centrifugation, prepare four CsCl gradients in Beckman polyallomer tubes. Add 7.5 ml of CsCl solution density 1.5 g/cm<sup>3</sup> at the bottom of each tube, then carefully overlay with 9 ml of 1.4 g/cm<sup>3</sup> CsCl solution and 9 ml of 1.3 g/cm<sup>3</sup> CsCl solution, dropwise. Hold for the final virus-containing 1.2 g/cm<sup>3</sup> density layer.</p></li><li><p>Discard the supernatant from the centrifugation in Step 9.</p></li><li><p>Resuspend the two viral pellets with 16 ml of 1.2 g/cm<sup>3</sup> CsCl solution for each pellet by pipeting.</p></li><li><p>Overlay 8 ml of resuspended virus pellets on top of each of the four gradients, dropwise (Figure1A).</p></li><li><p>Centrifuge overnight (between 16 and 18 h) at 100,000 <em>x g</em> at 20 Â°C in a Beckman Coulter <a class="blackNoUnder" target="_blank"> JS24-38 </a> rotor.</p></li><li><p>After centrifugation, harvest the white ring corresponding to the virus fraction (Figure 1B) by carefully pipetting through the gradient or by puncturing the polyallomer tube and aspirating with a needle and a syringe (Figure 2). Transfer the virus fraction to two new 50 ml conical tubes, add K36 buffer up to 50 ml in each tube and mix by inverting the tubes.</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3827/bioprotoc-10-22-3827-g001.jpg"/><br/><strong>Figure 1. Centrifuge tubes before (A) and after (B) separation of Mimivirus particles on CsCl gradient.</strong> The white ring indicated by the black arrow corresponds to Mimivirus particles.</p><p><br/></p><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3827/bioprotoc-10-22-3827-g002.jpg"/><br/><strong>Figure 2. Collection of the viral fraction by side puncture</strong> </p></li><br/><li><p>Centrifuge at 10,000 <em>x g</em> for 25 min at RT to pellet the virus.</p></li><li><p>Resuspend the two pellets with 10 ml of K36 buffer each, complete the volume to 50 ml with K36 and centrifuge them at 10,000 <em>x g</em> for 25 min at RT to wash the virus.</p></li><li><p>Repeat Steps 16 and 17 twice. For the last wash, pool the resuspended virus in a single 50 ml conical tube.</p></li><li><p>Resuspend the final pellet in 10 ml of K36 buffer.</p></li><li><p>The virus is ready to be titrated or stored at -80 Â°C until further use.</p></li></ol><br/><p><em>Notes</em>:</p><ol class="npwthree"><li><p><em> Typically, when purifying Acanthamoeba polyphaga Mimivirus using that protocol, we recover 10 ml of viral solution containing 1 x 10<sup>10</sup> to 4 x 10<sup>10</sup> particles/ml. </em> </p></li><li><p><em>The purified particles are infectious and can be used to performed new infections or any other experiments.</em> </p></li></ol><p class="pptt" id="biaoti38910">### Data analysis</p><ol class="npwtwok"><li><p>Light microscopy</p><p>Drop 4.5 Âµl of the purified virus suspension on a microscope slide and overlay with a coverslip. Invert for a few hours in order to let the virus settle onto the coverslip for easier focusing. Observe with a light microscope using the 63x objective to confirm the purified virus does not contain any obvious contaminants (Figure 3).</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3827/bioprotoc-10-22-3827-g003.jpg"/><br/><strong>Figure 3. Observation of Mimivirus purified particles with a light microscope (63x objective, 1.6x optovar)</strong></p></li><br/><li><p>Transmission Electron Microscopy (TEM)</p><p>For TEM, dilute the purified Mimivirus sample 1:2 (v/v) in K36 buffer and incubate a 10 Âµl droplet on a formvar- and carbon-coated 200 mesh copper/rhodium grid for 1 min at RT. Wash the grid with three successive 1% uranyl acetate droplets. Leave the residual uranyl acetate on the grid for 1 min and remove by gently touching the edge of the grid with a filter paper. After drying, examine the grid using an electron microscope (Figure 4).</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3827/bioprotoc-10-22-3827-g004.jpg"/><br/><strong>Figure 4. Observations of Mimivirus purified particles by negative staining</strong> </p></li></ol><p class="pptt" id="biaoti38911">### Recipes</p><ol class="npwtwok"><li><p>Glucose solution</p><p>Dissolve 36 g of glucose and 2 g of sodium citrate in 100 ml of slightly warm water.</p><p>When the solution is clear, use it to prepare the complete PPYG medium.</p></li><li><p>Proteose Peptone Yeast Extract (PPYG) medium</p><p><strong>Base medium:</strong> </p><p>Autoclave 40 g of proteose peptone and 2 g of yeast extract diluted in 1.8 L of ultrapure water in a 2 L glass bottle.</p><p>Let it cool at room temperature.</p><p><strong>Complete medium:</strong> </p><ol class="npwthree"><li><p>Add the following to the basal medium</p><p>20 ml of MgSO<sub>4</sub> 400 mM</p><p>16 ml of CaCl<sub>2</sub> 50 mM</p><p>20 ml of Fe(NH<sub>4)2</sub>(SO<sub>4)2</sub> 5 mM</p><p>20 ml of Na<sub>2</sub>HPO<sub>4</sub> 250 mM</p><p>20 ml of KH<sub>2</sub>PO<sub>4</sub> 250 mM</p><p>and 100 ml of glucose solution.</p></li><li><p>Filter through 0.22 Âµm Stericups using a peristaltic pump and store at room temperature</p></li></ol></li><li><p>Ampicillin stock solution (100 mg/ml)</p><ol class="npwthree"><li><p>Dissolve 1 g of ampicillin in 10 ml of ultrapure water</p></li><li><p>Filter through a 0.22 Âµm filter plugged on a syringe</p></li><li><p>Aliquot in 1.5 ml tubes and store at -20 Â°C</p></li></ol></li><li><p>Kanamycin stock solution (25 mg/ml)</p><ol class="npwthree"><li><p>Dissolve 250 mg of kanamycin in 10 ml of ultrapure water</p></li><li><p>Filter through a 0.22 Âµm filter plugged on a syringe</p></li><li><p>Aliquot in 1.5 ml tubes and store at -20 Â°C</p></li></ol></li><li><p>K36 buffer</p><ol class="npwthree"><li><p>Dissolve 2.25 g of KH<sub>2</sub>PO<sub>4</sub>, 5.8 g of K<sub>2</sub>HPO<sub>4</sub>Â·3H<sub>2</sub>O, 7.4 g of KCl and 0.9 g of NaCl in 1 L of ultrapure water</p></li><li><p>Filter through a 1 L 0.22 Âµm Stericup linked to a peristaltic pump and store at room temperature</p></li></ol></li><li><p>Cesium chloride solutions</p><p>Density 1.5 g/cm<sup>3</sup>: dissolve 22.7 g of CsCl in 27.3 ml of ultra pure water</p><p>Density 1.4 g/cm<sup>3</sup>: dissolve 19.4 g of CsCl in 30.6 ml of ultra pure water</p><p>Density 1.3 g/cm<sup>3</sup>: dissolve 15.62 g of CsCl in 34.38 ml of ultra pure water</p><p>Density 1.2 g/cm<sup>3</sup>: dissolve 11.2 g of CsCl in 38.8 ml of ultra pure water</p><p>Once the solutions appear clear, filter through a 0.22 Âµm filter plugged on a syringe</p></li></ol><p class="pptt" id="biaoti38912">### Acknowledgments</p><p>This work was supported by the French National Center for Scientic Research and the âcoup de chapeauâ of the Mediterranneen Institute of Microbiology.</p><p class="pptt" id="biaoti38913">### Competing interests</p><p>There are no conflicts of interest or competing interest.</p><p class="pptt" id="biaoti38914">### References</p><ol class="npwtwok"><li>Andrade, A., Rodrigues, R. A. L., Oliveira, G. P., Andrade, K. R., Bonjardim, C. A., La Scola, B., Kroon, E. G. and AbrahÃ£o, J. S. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28878069" target="_blank">Filling knowledge gaps for mimivirus entry, uncoating, and morphogenesis.</a> <em>J Virol</em> 91(22).</li><li>Arslan, D., Legendre, M., Seltzer, V., Abergel, C. and Claverie, J. M. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21987820" target="_blank">Distant Mimivirus relative with a larger genome highlights the fundamental features of Megaviridae.</a> <em>Proc Natl Acad Sci U S A</em> 108(42): 17486-17491.</li><li>Byrne, D., Grzela, R., Lartigue, A., Audic, S., Chenivesse, S., Encinas, S., Claverie, J. M. and Abergel, C. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19403753" target="_blank">The polyadenylation site of Mimivirus transcripts obeys a stringent'hairpin rule.</a> <em>Genome Res</em> 19(7): 1233-1242.</li><li>Campos, R. K., Boratto, P. V., Assis, F. L., Aguiar, E. R., Silva, L. C., Albarnaz, J. D., Dornas, F. P., Trindade, G. S., Ferreira, P. P., Marques, J. T., Robert, C., Raoult, D., Kroon, E. G., La Scola, B. and AbrahÃ£o, J. S. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24886672" target="_blank">Samba virus: a novel mimivirus from a giant rain forest, the Brazilian Amazon.</a> <em>Virol J</em> 11: 95.</li><li>ICTV 9<sup>th</sup> report. (2011). <a href="https://talk.ictvonline.org/ictv-reports/ictv_9th_report/dsdna-viruses-2011/w/dsdna_viruses/117/mimiviridae" target="_blank">https://talk.ictvonline.org/ictv-reports/ictv_9th_report/dsdna-viruses-2011/w/dsdna_viruses/117/mimiviridae.</a> </li><li>Jeudy, S., Bertaux, L., Alempic, J. M., Lartigue, A., Legendre, M., Belmudes, L., Santini, S., Philippe, N., Beucher, L., Biondi, E. G., Juul, S., Turner, D. J., CoutÃ©, Y., Claverie, J. M. and Abergel, C. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31822788" target="_blank">Exploration of the propagation of transpovirons within Mimiviridae reveals a unique example of commensalism in the viral world.</a> <em>ISME J</em> 14(3): 727-739.</li><li>La Scola, B., Audic, S., Robert, C., Jungang, L., de Lamballerie, X., Drancourt, M., Birtles, R., Claverie, J. M. and Raoult, D. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12663918" target="_blank">A giant virus in amoebae.</a> <em>Science</em> 299(5615): 2033.</li><li>Philippe, N., Legendre, M., Doutre, G., Coute, Y., Poirot, O., Lescot, M., Arslan, D., Seltzer, V., Bertaux, L., Bruley, C., Garin, J., Claverie, J. M. and Abergel, C. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23869018" target="_blank">Pandoraviruses: amoeba viruses with genomes up to 2.5 Mb reaching that of parasitic eukaryotes.</a> <em>Science</em> 341(6143): 281-286.</li><li>Raoult, D., Audic, S., Robert, C., Abergel, C., Renesto, P., Ogata, H., La Scola, B., Suzan, M. and Claverie, J. M. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15486256" target="_blank">The 1.2-megabase genome sequence of Mimivirus.</a> <em>Science</em> 306(5700): 1344-1350.</li></ol>