<body>
            <h1>Chromatin Immunoprecipitation (ChIP) to Assess Histone Marks in Auxin-treated <em>Arabidopsis thaliana</em> Inflorescence Tissue</h1>
            <p><strong>Authors:</strong> AndrÃ© Kuhn, Lars Ãstergaard</p>
            <h2>Abstract</h2>
            <p>Chromatin immunoprecipitation coupled with quantitative PCR (ChIP-qPCR) or high-throughput sequencing (ChIP-seq) has become the gold standard for the identification of binding sites of DNA binding proteins and the localization of histone modification on a locus-specific or genome-wide scale, respectively. ChIP experiments can be divided into seven critical steps: (A) sample collection, (B) crosslinking of proteins to DNA, (C) nuclear extraction, (D) chromatin isolation and fragmentation by sonication, (E) immunoprecipitation of histone marks by appropriate antibodies, (F) DNA recovery, and (G) identification of precipitated protein-associated DNA by qPCR or high-throughput sequencing. Here, we describe a time-efficient protocol that can be used for ChIP-qPCR experiments to study the localization of histone modifications in young inflorescences of the model plants <em>Arabidopsis thaliana</em>.</p>
            <p><strong>Keywords:</strong> ChIP, <em>Arabidopsis thaliana</em>, Epigenetic mark, Histone modification, Inflorescence</p>
            <h2>Background</h2><p>Eukaryotic genomes are organised in chromosomes in which DNA associates with histones to form chromatin. Tight interactions between histones and DNA impedes the accessibility of DNA to other factors. Consequently, the location of histones relative to important regulatory DNA sequences and the strength of histone-DNA contacts can either hide or expose genes providing yet another layer of gene regulation. In chromatin, both histones and DNA can be chemically modified (Zhou <em>et al.</em>, 2010; Schübeler, 2015). Depending on the physical properties of the modification, the chromatin state can either prevent or enhance the transcription of the underlying genes (Kouzarides, 2007; Yang <em>et al.</em>, 2014; Wu <em>et al.</em>, 2015). In plants, the epigenetic state of the chromatin has been shown to be a crucial determinant of gene expression in response to developmental or environmental stimuli (Yang <em>et al.</em>, 2014; Wu <em>et al.</em>, 2015; Chung <em>et al.</em>, 2019; Kuhn <em>et al.</em>, 2020).<br/><br/>The plant hormone auxin regulates almost all aspect of a plant’s life cycle. A mechanism for auxin perception has been established involving a nuclear signalling pathway. This pathway is based on the de-repression of Auxin Response Factors (ARFs) via auxin-induced degradation of Aux/IAA transcriptional repressors. Aux/IAAs inhibit ARF activity towards target genes by recruiting the co-repressor TOPLESS (TPL) to repress auxin-dependent gene expression. TPL/TPRs mediate their repressive effect by attracting histone deacetylases (HDACs) (Krogan <em>et al.</em>, 2012; Weijers and Wagner, 2016). We have identified an alternative auxin signalling mechanism which is essential for gynoecium development in <em>Arabidopsis thaliana</em>. In this noncanonical pathway, auxin directly affects the activity of complexes involving an atypical ARF, called ETTIN (ETT/ARF3) towards downstream targets (Simonini <em>et al.</em>, 2016 and 2017, Kuhn <em>et al.</em>, 2020). Recently, we found that direct auxin binding by ETT leads to dissociation from co-repressor proteins of the TPL/TPR family followed by induction of gene expression. Moreover, we showed that this correlates with ETT-dependent auxin-responsive histone acetylation of the target genes (Kuhn <em>et al.</em>, 2020).<br/><br/>Chromatin immunoprecipitation coupled with quantitative PCR (ChIP-qPCR) has become the gold standard for the identification and characterisation of histone modifications at given gene loci. Since ETT is primarily involved in floral reproductive development, we optimized existing ChIP protocols (Yang <em>et al.</em>, 2014, Qüesta <em>et al.</em>, 2016) to study the effect of auxin on histone modifications in inflorescence tissue. The protocol that we present here has been successfully applied to evaluate several epigenetic marks including H3K27Ac (Kuhn <em>et al.</em>, 2020) in <em>Arabidopsis</em> inflorescence tissue.</p>
        </body>

<p class="pptt" id="biaoti38982">## Materials and Reagents</p><ol class="npwtwok"><li><p>Spray bottle TurnNSpray 500 ml (R&amp;L Slaughter, catalog number: <a class="blackNoUnder" target="_blank"> 215-2962 </a> )</p></li><li><p>Corning 50 ml centrifuge tubes (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> CLS430829-500E </a> )</p></li><li><p>Elastic rubber bands</p></li><li><p>Paper towel</p></li><li><p>Aluminium Foil</p></li><li><p>Funnel (6 cm diameter)</p></li><li><p>2 ml DNA low binding (LoBind) tubes (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 0030108078 </a> )</p></li><li><p>1.5 ml DNA low binding (LoBind) tubes (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 022431021 </a> )</p></li><li><p><em>Arabidopsis thaliana</em> </p></li><li><p>Indole-3-acetic acid (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> I5148 </a> )</p></li><li><p>Silwet L-77 (De Sangosse Ltd., catalog number: <a class="blackNoUnder" target="_blank"> 0640 </a> )</p></li><li><p>Miracloth (MERK, catalog number: <a class="blackNoUnder" target="_blank"> 475855 </a> )</p></li><li><p>Liquid nitrogen</p></li><li><p>10x Phosphate-buffered saline (PBS) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 11666789001 </a> )</p></li><li><p>Formaldehyde (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> F8775 </a> )</p></li><li><p>Glycine (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> G/0800/60 </a> )</p></li><li><p>Sterile water</p></li><li><p>Succrose (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> S/8600/60 </a> )</p></li><li><p>Ficoll 400 (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> F8636 </a> )</p></li><li><p>Dextran from <em>Leuconostoc</em> spp. (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 31389 </a> )</p></li><li><p>HEPES (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> H4034 </a> )</p></li><li><p>Magnesium chloride (MgCl<sub>2</sub>) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M8266 </a> )</p></li><li><p>Triton X-100 (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> T9284 </a> )</p></li><li><p>DL-Dithiothreitol (DTT; Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> D0632 </a> )</p></li><li><p>cOmplete EDTA-free protease inhibitor cocktail (Roche Molecular Systems, catalog number: <a class="blackNoUnder" target="_blank"> 4693132001 </a> )</p></li><li><p>Tris(hydroxymethyl)aminomethane (TRIS; Sigmal-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> B2005 </a> )</p></li><li><p>Ethylenediaminetetraacetic acid (EDTA; Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> E6758 </a> )</p></li><li><p>20% Sodium Dodecyl Sulfate (SDS) solution (Severn Biotech; catalog number: <a class="blackNoUnder" target="_blank"> 20-4002-05 </a> )</p></li><li><p>Sodium Chloride (NaCl) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S7653 </a> )</p></li><li><p>Dynabeads Protein A (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> 10001D </a> )</p></li><li><p>Anti-H3 Antibody (Rabbit polyclonal Antibody; Abcam, catalog number: <a class="blackNoUnder" target="_blank"> ab1791 </a> )</p></li><li><p>Anti-H3K27ac Antibody (Rabbit polyclonal Antibody; Abcam, catalog number: <a class="blackNoUnder" target="_blank"> ab4729 </a> )</p></li><li><p>Sodium bicarbonate (NaHCO<sub>3</sub>) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S5761 </a> )</p></li><li><p>Proteinase K (Roche Molecular Systems, catalog number: <a class="blackNoUnder" target="_blank"> 40693800 </a> )</p></li><li><p>Phenol:Chloroform:Isoamyl alcohol (25:24:1) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 77617 </a> )</p></li><li><p>Sodium acetate (NaAc) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S7670 </a> )</p></li><li><p>Ethanol absolute (VWR Chemicals, catalog number: <a class="blackNoUnder" target="_blank"> 20821.330 </a> )</p></li><li><p>LightCycler 480 Multiwell Plate 96, white (Roche Molecular Systems, catalog number: <a class="blackNoUnder" target="_blank"> 04729692001 </a> )</p></li><li><p>LightCycler<sup>Â®</sup> 480 Sealing Foil (Roche Molecular Systems, catalog number: <a class="blackNoUnder" target="_blank"> 04729757001 </a> )</p></li><li><p>Oligo-nucleotide primers (Integrated DNA Technologies)</p></li><li><p>SYBR Green JumpStart Taq Ready Mix (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S4438 </a> )</p></li><li><p>RNase A (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> EN0531 </a> )</p></li><li><p>Agarose (Melford Biolaboratories Ltd., catalog number: <a class="blackNoUnder" target="_blank"> A20080-1000.0 </a> )</p></li><li><p>100 bp DNA ladder (New England Biolabs Ltd., catalog number: <a class="blackNoUnder" target="_blank"> N3221S </a> )</p></li><li><p>Stock Solutions (see Recipes)</p><p>2 M Glycine</p><p>0.5 M HEPES KOH pH 7.4</p><p>1 M MgCl<sub>2</sub> </p><p>1 M Tris-HCl pH 8</p><p>0.5 M EDTA</p><p>5 M NaCl</p><p>3 M NaAc</p><p>1 M DTT</p><p>20% (v/v) Triton X-100</p><p>10% (v/v) SDS</p></li><li><p>ChIP Buffers (see Recipes)</p><p>Honda Buffer</p><p>Nuclei Lysis Buffer</p><p>ChIP Dilution Buffer</p><p>Low Salt Wash Buffer</p><p>High Salt Wash Buffer</p><p>TE Buffer</p><p>Elution Buffer</p></li></ol><p class="pptt" id="biaoti38983">### Equipment</p><ol class="npwtwok"><li><p>Tweezers (DUMONT BIOLOGICAL TYPE 5 SS-1, TAAB Laboratories Equipment, catalog number: <a class="blackNoUnder" target="_blank"> T083 </a> )</p></li><li><p>Glass beaker 100 ml</p></li><li><p>Mortar and pistil</p></li><li><p>Bel-Art Space Saver Vacuum Desiccator (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11852732 </a> )</p></li><li><p>Vacuum rump (Edwards, model: <a class="blackNoUnder" target="_blank"> IEC34-1 </a> )</p></li><li><p>Tube rotor (Stuart Scientific, model: <a class="blackNoUnder" target="_blank"> SB1 </a> )</p></li><li><p>Centrifuge (Eppendorf, model: <a class="blackNoUnder" target="_blank"> 5810R </a> )</p></li><li><p>Swing-bucket rotor with 50 ml tube adaptors (Eppendorf, model: <a class="blackNoUnder" target="_blank"> A-4-81 </a> )</p></li><li><p>Fixed-angle rotor for 48 x 1.5/2.0 ml tubes (Eppendorf, model: <a class="blackNoUnder" target="_blank"> FA-45-48-11 </a> )</p></li><li><p>Sonicator (Diagenode, Model <a class="blackNoUnder" target="_blank"> Bioruptor Plus </a> )</p></li><li><p>Magnetic separator rack (GE Healthcare, MagRack6, catalog number <a class="blackNoUnder" target="_blank"> 28-9489-64 </a> )</p></li><li><p>Thermomixer (Eppendorf, Model <a class="blackNoUnder" target="_blank"> Eppendorf ThermoMixer C </a> )</p></li><li><p>CFX96 touch real-time PCR detection system (Bio-Rad)</p></li><li><p>Vacuum concentrator (Eppendorf, model: <a class="blackNoUnder" target="_blank"> Concentator Plus </a> )</p></li><li><p>Gel electrophoresis system (Thermo Electric Corporation, model: <a class="blackNoUnder" target="_blank"> CSSU1214 </a> )</p></li><li><p>Power pack (Kikusui Electronics Corporation, model: <a class="blackNoUnder" target="_blank"> PAB 350-0.2 </a> )</p></li></ol><p class="pptt" id="biaoti38984">### Procedure</p><ol class="npwone"><li><p>Sample treatment and collection</p><p>This section describes the plant cultivation and treatment procedures for <em>Arabidopsis thaliana</em> and <em>Capsella rubella</em> inflorescence tissue subjected to auxin treatments. Cultivation and treatment procedures may differ for other species, tissues and treatments.</p><ol class="npwtwok"><li><p>Grow <em>Arabidopsis</em> plants on soil at 22 Â°C in long day conditions (16 h day/8 h dark) until inflorescences emerge (3-4 weeks).</p></li><li><p>Treat inflorescences by spraying the full tray of plants with a solution containing 100 ÂµM Indole-3-acetic acid (IAA, auxin) and 0.015% Silwet L-77 or MOCK (only 0.015% Silwet L-77) in water (<strong>Note 1</strong>).</p></li><li><p>Return treated plants to the growth room for two hours.</p></li><li><p>Using tweezers collect 3 g of young inflorescence tissue in a 50 ml Falcon Tube (<strong>Note 2</strong>).</p></li><li><p>Collect a total of three biological replicates (3 times 3 g) for one experiment (<strong>Note 3</strong>).</p><br/></li></ol></li><li><p>Crosslinking</p><ol class="npwtwok"><li><p>Transfer the collected samples onto Miracloth squares (~5 x 5 cm), fold into small bags and close them securely with an elastic band (<strong>Note 4</strong>).</p></li><li><p>Place samples in a glass beaker (100 ml) and add 50 ml 1% Formaldehyde solution (<strong>Note 5</strong>).</p><p><em>Note: Formaldehyde is toxic, work in fume cabinet.</em> </p></li><li><p>Place the glass beakers in the vacuum desiccator and turn it on.</p></li><li><p>Crosslink samples in 1% formaldehyde solution for 3 times 5 min, releasing the vacuum after 5 min and reapply when fully released (<strong>Note 6</strong>).</p></li><li><p>Quench the reaction by adding 2 M glycine solution to a final concentration of 125 mM and apply vacuum for 5 min.</p></li><li><p>Release vacuum and discard quenched 1% Formaldehyde solution.</p></li><li><p>Wash samples twice with 1x PBS and twice with water.</p></li><li><p>Blot them dry on paper towel and then fold the tissue into aluminium foil.</p></li><li><p>Flash freeze packets in liquid nitrogen.</p></li></ol><p>*Stopping point (store at -80 Â°C).</p><br/></li></ol><p><em>Note: Unless stated otherwise all subsequent steps should be carried out on ice using pre-cooled buffers.</em> </p><ol class="npwone" start="3"><li><p>Nuclear extraction</p><ol class="npwtwok"><li><p>Using pre-cooled mortar and pistil, grind the plant material in liquid nitrogen into fine powder.</p></li><li><p>Suspend ground up plant material in 25 ml Honda Buffer and homogenise for 10 min at 4 Â°C on a rotor.</p></li><li><p>Filter through one layer of Miracloth (~7 x 7 cm) using a funnel into new 50 ml Falcon tube. Squeeze the Miracloth to maximise the yield.</p></li><li><p>Centrifuge at 850 <em>x g</em> for 20 min (4 Â°C).</p></li><li><p>Gently resuspend the pellet in 1 ml Honda Buffer (<strong>Note 7</strong>).</p></li><li><p>Centrifuge at 850 <em>x g</em> for 5 min (4 Â°C).</p></li><li><p>Repeat step 5 and 6 twice to clear the extract.</p></li></ol><p>*Stopping point (pellet can be stored overnight at -20 Â°C).</p><br/></li></ol><ol class="npwone" start="4"><li><p>Chromatin fragmentation by sonication</p><ol class="npwtwok"><li><p>Centrifuge 4,000 <em>x g</em> for 1 min (4 Â°C).</p></li><li><p>Remove residual Honda Buffer and resuspend pellet in 300 Î¼l of Nuclei Lysis Buffer.</p></li><li><p>Split the sample in two 2 ml Lobind tubes (~250 Î¼l per tube).</p></li><li><p>Take a 25 Î¼l of this new suspension for your non-sonicated control and store at -20 Â°C.</p></li><li><p>Sonicate 3 times for 5 min on medium strength using a Bioruptor water bath sonicator (Diagenode) using a 30 s ON/OFF cycle, to fragment the chromatin. Mix samples between each 5 min period (<strong>Note 8</strong>).</p></li><li><p>Centrifuge the samples for 10 min, 4,000 <em>x g</em>, 4 Â°C.</p></li><li><p>Transfer the supernatant to fresh 2 ml Lobind tubes.</p></li><li><p>Take a 25 Î¼l to check sonication efficiency and store at -20 Â°C (<strong>Note 9</strong>).</p></li></ol><p>*Stopping point (samples can be stored overnight at -20 Â°C).</p><br/></li></ol><ol class="npwone" start="5"><li><p>Immunoprecipitation (IP)</p><ol class="npwtwok"><li><p>Dilute the samples ten-fold with ChIP Dilution Buffer (~2 ml).</p><p><em>Note: Take 20 Î¼l sonicated homogenate as Input control.</em> </p></li><li><p>Prepare the antibody coated magnetic beads.</p><ol class="npwthree"><li><p>Take 15 Î¼l of Protein A magnetic beads for each IP and wash three times with 1 ml ChIP dilution buffer.</p></li><li><p>Resuspend magnetic beads in 50 Î¼l ChIP Dilution Buffer per IP.</p></li><li><p>Bind 1 Î¼g of antibody per IP to the beads.</p></li><li><p>To IP H3 we use the anti-H3 antibody and to IP H3K27Ac we use the anti-H3K27ac Antibody.</p></li><li><p>Incubate on a rotor for an hour at 4 Â°C.</p></li></ol></li><li><p>Add 50 Î¼l anti-H3 antibody-coated beads to one half of the samples and 50 Î¼l antibody-coated beads to the other half of the samples.</p></li><li><p>Incubate on a rotor at 4 Â°C for at least 4 h.</p><p>*Stopping point (this step can also be done overnight).</p></li><li><p>Wash the beads sequentially (each time 5 min at 4 Â°C incubation on the rotor) with 1 ml with:</p><ol class="npwthree"><li><p>Twice Low Salt Washing Buffer.</p></li><li><p>Twice High Salt Washing Buffer.</p></li><li><p>Twice TE Buffer.</p></li></ol><p><em>Note: For each of these washes, leave on the magnetic stand for one minute at a time.</em> </p></li><li><p>After the last TE wash transfer the sample to a fresh 1.5 ml Lobind tube then separate on the magnetic stand and remove TE.</p></li><br/></ol></li></ol><ol class="npwone" start="6"><li><p>DNA recovery</p><ol class="npwtwok"><li><p>Elute from beads twice with 200 Î¼l pre-heated Elution Buffer. Incubate for 15 min at 65 Â°C with 800 rpm agitation. Collect the two elutions (400 Î¼l total volume) in one fresh 1.5 ml Lobind tube and discard the magnetic antibody coated beads.</p></li><li><p>Bring the Input samples to 400 Î¼l using Elution Buffer.</p></li><li><p>To reverse crosslink, add 16 Î¼l 5M NaCl to the eluate and incubate at 65 Â°C and 600 rpm agitation overnight.</p></li><li><p>After reverse crosslinking, add 3 Î¼l Proteinase K (20 mg/ml) and incubate for 1 h at 45 Â°C.</p></li><li><p>To extract DNA, add 400 Î¼l Phenol:Chloroform:Isoamyl alcohol (25:24:1), vortex for 30 s and centrifuge at 4,000 <em>x g</em> for 10 min at room temperature.</p></li><li><p>Transfer the aqueous layer (~400 Î¼l) in a fresh 1.5 ml Lobind tube.</p></li><li><p>To precipitate DNA, add 0.1 volume (40 Î¼l) 3 M NaAc and 2 volumes (800 Î¼l) 100% ethanol (mix well) and incubate at -20 Â°C for 1.5 h.</p></li><li><p>Centrifuge at 4,000 <em>x g</em> for 30 min (4 Â°C).</p></li><li><p>Wash pellet with 500 Î¼l ice cold 70% ethanol, centrifuge for 10 min at 4,000 <em>x g</em> (4 Â°C).</p></li><li><p>Remove all traces of ethanol and dry the pellet in vacuum concentrator for 10 min (65 Â°C).</p></li><li><p>Resuspend the pellet in the required volume of nuclease free water (<strong>Note 10</strong>).</p></li><br/></ol></li></ol><ol class="npwone" start="7"><li><p>Identification of precipitated protein-associated DNA by qPCR</p><ol class="npwtwok"><li><p>Setup qPCR reactions in triplicate per antibody, treatment and primer pair as follows (<strong>Note 11</strong>):</p><p>Per reaction (10 Î¼l final volume): 3 Î¼l DNA, 5 Î¼l SYBR Green JumpStart Taq ReadyMix (Sigma), 0.125 Î¼l Forward Primer (10 Î¼M), 0.125 Î¼l Reverse Primer (10 Î¼M), 1.75 Î¼l water.</p></li><li><p>Run the reaction on qPCR machine using the following conditions: 95 Â°C for 2 min followed by 60 cycles at 95 Â°C for 10 s, 60 Â°C for 10 s and 72 Â°C for 30 s.</p></li><li><p>Analyse the data as described under âData analysisâ.</p></li></ol></li></ol><p class="pptt" id="biaoti38985">### Data analysis</p><p><em>Note: We have analysed the data published in Kuhn et al. (2020) using described protocol.</em> </p><ol class="npwtwok"><li><p>Once the Ct (Cycle threshold) values have been obtained calculate the mean between the three technical replicates.</p></li><li><p>Calculate the ratio between each IP with the corresponding Input for all target regions:</p><p><strong> MEAN = 2<sup>-(MEAN(IP)-MEAN(Input))</sup></strong> to transform to linear scale.</p></li><li><p>Calculate the enrichment of the Histone H3 modification of interest (<em>e.g.</em>, H3K27Ac) as the ratio between the H3K27Ac and H3 for each biological replicate at each target region.</p></li><li><p>Calculate mean and standard deviation between all biological replicates at each target region.</p></li><li><p>Analyse the statistical differences between hormone treated and MOCK samples at each target region using two-way ANOVA with <em>post hoc</em> Bonferroni multiple comparison test.</p></li><li><p>Plot the mean and standard deviation of the ratio (H3K27Ac/H3) along the length of the examined target region.</p></li></ol><p class="pptt" id="biaoti38986">### Notes</p><ol class="npwtwok"><li><p>Indole-3-acetic acid (IAA) is insoluble in water and needs to be freshly prepared in ethanol to a stock concentration of 100 mM. Mock treatment has to contain the same amount of ethanol as the auxin treatment. When treating the plants, make sure that you spray equally with treatment and mock solution. Treatment of ~100 <em>Arabidopsis</em> plants requires ~100 ml solution.</p></li><li><p>We only harvested inflorescence tissue that contained closed flower buds. Flowers that were opened or young siliques were removed.</p></li><li><p>To obtain 3 g sample inflorescences of ~100 plants need to be harvested. We recommend to collect one pair of biological replicates (one replicate per genotype, mock and IAA-treatment) at the same time and to repeat collection three times to obtain three replicates for an experiment. We recommend having two people collecting to reduce the time.</p></li><li><p>If including more than one genotype or treatment, it is advised to use one glass beaker per treatment and genotype to prevent mixing of samples.</p></li><li><p>Crosslinking time may need optimisation depending on protein and tissue. The optimal crosslink time crosslinks your protein of interest to the DNA but does not block the epitope. This can be tested by Western blotting.</p></li><li><p>Do not start timing the 5 min before the first bubbles emerge.</p></li><li><p>This can be done with a 100 Î¼l pipette. We recommend cutting the pipette tip to prevent damaging the nuclei.</p></li><li><p>Sonication conditions can differ between sonicators and depend on samples. Generally four parameters are essential for the efficiency of the DNA fragmentation: 1) the temperature of the sample (we recommend to maintain a temperature of 4 Â°C throughout the sonication), 2) the concentration of formaldehyde used for crosslinking (the higher the concentration leads increases the sonication time), 3) the volume in which sonication is performed (we recommend not to sonicate more than 250 Î¼l of lysate in a 1.5 ml tube), 4) the concentration of SDS in the lysis buffer (more SDS leads to easier lysis during sonication).</p></li><li><p>Sonication test:</p><p>Once your sonication conditions are well established, you can skip this step. Bring sonication and non-sonication aliquots to 100 Î¼l volume with elution buffer.</p><ol class="npwthree"><li><p>Add 1 Î¼l of 10 mg/ml RNase A, mix well and incubate for 1 h at 37 Â°C.</p></li><li><p>Add 100 Î¼l of Phenol:Chloroform:Isoamyl alcohol (25:24:1), vortex for 30 s and centrifuge 4,000 <em>x g</em> for 10 min at room temperature.</p></li><li><p>Transfer the aqueous phase to a fresh 1.5 ml Lobind tube and add 0.1 volumes (10 Î¼l) of 3 M NaAc (pH 5.5) and 2 volumes (200 Î¼l) of 100% EtOH. Vortex and incubate for 1.5h at -20 Â°C.</p></li><li><p>Centrifuge for 4,000 <em>x g</em> for 20 min at 4 Â°C.</p></li><li><p>Remove the supernatant and dry the pellet for 10 min in vacuum concentrator (65 Â°C).</p></li><li><p>Resuspend the pellet in 10 Î¼l water.</p></li><li><p>Run samples on a 1.5% Agarose gel.</p></li><li><p>Visualise the gel. The DNA should be visible as a smear between 100 bp and 500 bp (Figure 1). The smaller the DNA is fragmented the higher the resolution of the experiment.</p><br/></li></ol><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201204/20201204193501_9596.jpg" width="200"/><br/><strong>Figure 1. Sonication test.</strong> Marker (NEB 100 bp ladder); Sample showing the bulk of DNA fragments range from 100-500 bp.</p><br/></li><li><p>The volume of water depends on the number of qPCR reactions that will be run (3 Î¼l DNA per reaction).</p></li><li><p>Primers should be designed to amplify fragments (the ideal fragment length is ~100 bp) with regular spacing across the whole gene locus of interest. The resolution of the experiments increases with increasing amplicon density. Before use, all qPCR primers should be tested for their efficiency and specificity. The amplification efficiency of primers can be tested using a genomic DNA dilution series (1/4 to 1/4096) (Figure 2A). The specificity of primers was confirmed by analysing the melting curves (65 Â°C to 95 Â°C) (Figure 2B). qPCR reactions should be prepared and performed as described in the experimental procedures (G1-3).</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201204/20201204193522_2158.jpg"/><br/><strong>Figure 2. Example a suitable qPCR primer.</strong> A. Amplification efficiency of a suitable primer should show a correlation between amplification and dilution (R<sup>2</sup>) of 0.95-1 and a primer efficiency (10<sup>-1/slope</sup>) of 2 Â± 0.1. B. Melt peak indicating high specificity of the primer pair.</p></li></ol><p class="pptt" id="biaoti38987">### Recipes</p><ol class="npwone"><li><p>Stock Solutions</p><ol class="npwtwok"><li><p>2 M Glycine<br/>Autoclave</p></li><li><p>0.5 M HEPES KOH pH 7.4</p><p>Autoclave</p></li><li><p>1 M MgCl<sub>2</sub> </p><p>Autoclave</p></li><li><p>1 M Tris-HCl pH 8<br/>Autoclave</p></li><li><p>0.5 M EDTA<br/>Autoclave</p></li><li><p>5 M NaCl<br/>Autoclave</p></li><li><p>3 M NaAc<br/>Autoclave</p></li><li><p>1 M DTT</p><p>Weigh 1.54 g of DTT and dissolve in 10 ml water</p><p>Filter sterilize after all DTT is dissolved and make 1 ml aliquots</p><p>Store at -20 Â°C</p></li><li><p>20% (v/v) Triton X-100</p><p>Dilute Triton X-100 five times with water to obtain 20% Triton X-100</p></li><li><p>10% (v/v) SDS</p><p>Dilute 20% (v/v) SDS with 1 volume water to obtain 10% (v/v) SDS solution</p><br/></li></ol></li><li><p>ChIP Buffers</p><ol class="npwtwok"><li><p>Honda Buffer</p><p>0.44 M sucrose</p><p>1.25% (w/v) Ficoll</p><p>2.5% (w/v) Dextran T40</p><p>20 mM HEPES KOH pH 7.4</p><p>0.5% (v/v) Triton X-100</p><p>10 mM MgCl<sub>2</sub> </p><p>0.5 mM DTT</p><p>1 tablet per 50 ml cOmplete EDTA-free protease inhibitor cocktail (Roche)</p></li><li><p>Nuclei Lysis Buffer</p><p>50 mM Tris-HCl pH 8.0</p><p>10 mM EDTA</p><p>1% (v/v) SDS</p><p>1 tablet per 50 ml cOmplete EDTA-free protease inhibitor cocktail (Roche)</p></li><li><p>ChIP Dilution Buffer</p><p>16.7 mM Tris-HCl pH 8.0</p><p>1.2 mM EDTA</p><p>1.1% (v/v) Triton X-100</p><p>167 mM NaCl</p><p>1 tablet per 50 ml cOmplete EDTA-free protease inhibitor cocktail (Roche)</p></li><li><p>Low Salt Wash Buffer</p><p>20 mM Tris-HCl pH 8.0</p><p>2 mM EDTA</p><p>150 mM NaCl</p><p>0.1% (v/v) SDS</p><p>1% (v/v) Triton X-100</p><p>1 tablet per 50 ml cOmplete EDTA-free protease inhibitor cocktail (Roche)</p></li><li><p>High Salt Wash Buffer</p><p>20 mM Tris-HCl pH 8.0</p><p>2 mM EDTA</p><p>500 mM NaCl</p><p>0.1% (v/v) SDS</p><p>1% (v/v) Triton X-100</p><p>1 tablet per 50 ml cOmplete EDTA-free protease inhibitor cocktail (Roche)</p></li><li><p>TE Buffer</p><p>10 mM Tris-HCl pH 8.0</p><p>1 mM EDTA</p><p>1 tablet per 50 ml cOmplete EDTA-free protease inhibitor cocktail (Roche)</p></li><li><p>Elution Buffer</p><p>0.1 M NaHCO<sub>3</sub> </p><p>1% (v/v) SDS</p></li></ol></li></ol><p class="pptt" id="biaoti38988">### Acknowledgments</p><p>We are grateful to Yang Dong and Emilie Knight for critical comments on the protocol. We also express our gratitude to Caroline Dean, Hongchun Yang and Julia QÃ¼esta for their helpful advice throughout the development of this protocol. This protocol was adapted from      Kuhn <em>et al.</em>, 2020    . This work was supported by grant BB/S002901/1 to L.Ã., the Norwich Research Park Biosciences Doctoral Training Partnership [grant number BB/M011216/1 to A.K.] and by the Institute Strategic Programme grant (BB/P013511/1) to the John Innes Centre all from the UKRI Biotechnological and Biological Sciences Research Council.</p><p class="pptt" id="biaoti38989">### Competing interests</p><p>The authors declare no competing interests.</p><p class="pptt" id="biaoti38990">### References</p><ol class="npwtwok"><li>Chung, Y., Zhu, Y., Wu, M. F., Simonini, S., Kuhn, A., Armenta-Medina, A., Jin, R., Ostergaard, L., Gillmor, C. S. and Wagner, D. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30792395" target="_blank">Auxin Response Factors promote organogenesis by chromatin-mediated repression of the pluripotency gene </a><em><a href="http://www.ncbi.nlm.nih.gov/pubmed/30792395" target="_blank">SHOOTMERISTEMLESS.</a> Nat Commun</em> 10(1): 886.</li><li>Kouzarides, T. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17320507" target="_blank">Chromatin modifications and their function.</a> <em>Cell</em> 128(4): 693-705.</li><li>Krogan, N. T., Hogan, K. and Long, J. A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23034631" target="_blank">APETALA2 negatively regulates multiple floral organ identity genes in <em>Arabidopsis</em> by recruiting the co-repressor TOPLESS and the histone deacetylase HDA19.</a> <em>Development</em> 139(22): 4180-4190.</li><li>Kuhn, A., Ramans Harborough, S., McLaughlin, H. M., Natarajan, B., Verstraeten, I., Friml, J., Kepinski, S. and Ostergaard, L. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/32267233" target="_blank">Direct ETTIN-auxin interaction controls chromatin states in gynoecium development.</a> <em>Elife</em> 9: e51787.</li><li>QÃ¼esta, J. I., Song, J., Geraldo, N., An, H. and Dean, C. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27471304" target="_blank"><em>Arabidopsis</em> transcriptional repressor VAL1 triggers Polycomb silencing at FLC during vernalization.</a> <em>Science</em> 353(6298): 485-488.</li><li>SchÃ¼beler, D. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25592537" target="_blank">Function and information content of DNA methylation.</a> <em>Nature</em> 517(7534): 321-326.</li><li>Simonini, S., Deb, J., Moubayidin, L., Stephenson, P., Valluru, M., Freire-Rios, A., Sorefan, K., Weijers, D., Friml, J. and Ostergaard, L. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27898393" target="_blank">A noncanonical auxin-sensing mechanism is required for organ morphogenesis in </a><em><a href="http://www.ncbi.nlm.nih.gov/pubmed/27898393" target="_blank">Arabidopsis.</a> Genes Dev</em> 30(20): 2286-2296.</li><li>Simonini, S., Bencivenga, S., Trick, M. and Ostergaard, L. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28804059" target="_blank">Auxin-Induced Modulation of ETTIN Activity Orchestrates Gene Expression in</a> <em><a href="http://www.ncbi.nlm.nih.gov/pubmed/28804059" target="_blank">Arabidopsis.</a> Plant Cell</em> 29(8): 1864-1882.</li><li>Weijers, D. and Wagner, D. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26905654" target="_blank">Transcriptional Responses to the Auxin Hormone.</a> <em>Annu Rev Plant Biol</em>  67: 539-574.</li><li>Wu, M. F., Yamaguchi, N., Xiao, J., Bargmann, B., Estelle, M., Sang, Y. and Wagner, D. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26460543" target="_blank">Auxin-regulated chromatin switch directs acquisition of flower primordium founder fate.</a> <em>Elife</em> 4: e09269.</li><li>Yang, H., Howard, M. and Dean, C. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25065750" target="_blank">Antagonistic roles for H3K36me3 and H3K27me3 in the cold-induced epigenetic switch at <em>Arabidopsis</em> FLC.</a> <em>Curr Biol</em> 24(15): 1793-1797.</li><li>Zhou, V. W., Goren, A. and Bernstein, B. E. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21116306" target="_blank">Charting histone modifications and the functional organization of mammalian genomes.</a> <em>Nat Rev Genet</em> 12(1): 7-18.</li></ol>