<body>
            <h1>Charging State Analysis of Transfer RNA from an Î±-proteobacterium</h1>
            <p><strong>Authors:</strong> Liang Yin, Caroline S. Harwood</p>
            <h2>Abstract</h2>
            <p>Transfer RNA (tRNA) is an essential link between the genetic code and proteins. During the process of translation, tRNA is charged with its cognate amino acid and delivers it to the ribosome, thus serving as a substrate of protein synthesis. To analyze the charging state of a particular tRNA, total RNA is purified and analyzed on an acid-urea gel. Separated RNA is then transferred to a membrane and detected with a probe for the tRNA of interest. Here, we present an improved protocol to analyze the tRNA charging state in the α-proteobacterium <em>Rhodopseudomonas palustris</em>. Compared to the classical method, the RNA isolation step is optimized to suit this organism. Additionally, a non-radioactive platform is used for electrophoresis and Northern blots. This significantly reduces the time and the effort required for this protocol.</p>
            <p><strong>Keywords:</strong> Translation, tRNA charging, Aminoacyl-tRNA, Acid urea gel, <em>Rhodopseudomonas palustris</em></p>
            <h2>Background</h2><p>The primary function of tRNA is, with the help of other translation factors, to ensure the accurate translation of mRNA to protein. Aminoacyl-tRNAs (charged) bring amino acids to the ribosome for peptide elongation, and then the uncharged tRNAs are released. The charging state of tRNA is largely determined by the available resource (<em>i.e.</em>, amino acids) and their consumption by the ribosome. To analyze the charging state of cellular tRNA, methods have been developed using acid-urea gels to separate the total RNA and to detect the tRNA of interest by Northern blot (Janssen <em>et al.</em>, 2012; Bullwinkle and Ibba, 2016). Recently, we explored the relationship between tRNA charging states and survival of starving bacteria (Yin <em>et al.</em>, 2019). The method used in that study and elaborated here, has optimized the RNA purification step to better capture the tRNA charging state in bacteria other than <em>E. coli</em>. It also utilizes a commercially available system to perform the analysis without using radioactive materials, which could be helpful for some research groups.</p>
        </body>

<p class="pptt" id="biaoti39004">## Materials and Reagents</p><p><em>Notes:</em> </p><ol class="npwthree"><li><p><em>It is crucial to keep all the materials and reagents RNase-free for this protocol. To this end, the reagents and materials are all RNase-free, and are purchased pre-made or ready-to-dissolve in order to simplify the procedure as much as possible. Disposable RNase-free centrifuge tubes are used to make reagents whenever applicable.</em> </p></li><li><p><em>Multiple kits are available for DIG reagents. Items 19-23 are listed here for readers to choose from to suit the scale required.</em> </p><br/></li></ol><ol class="npwtwok"><li><p>Centrifuge tubes, RNase free (1.5 ml, 2 ml, 15 ml and 50 ml, as needed)</p></li><li><p>QIAzol Lysis Reagent (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 79306 </a> )</p></li><li><p>Chloroform, molecular biology grade</p></li><li><p>Isopropanol, molecular biology grade</p></li><li><p>Ethanol, molecular biology grade</p></li><li><p>UltraPure DNase/RNase-Free Distilled Water (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> 10977-023 </a> )</p></li><li><p>TEMED (National Diagnostics, catalog number: <a class="blackNoUnder" target="_blank"> EC-503 </a> )</p></li><li><p>Ammonium persulfate (National Diagnostics, catalog number: <a class="blackNoUnder" target="_blank"> EC-504 </a> )</p></li><li><p>Urea (National Diagnostics, catalog number: <a class="blackNoUnder" target="_blank"> EC-605 </a> )</p></li><li><p>AccuGel 29:1 (30%) (National Diagnostics, catalog number: <a class="blackNoUnder" target="_blank"> EC-851 </a> )</p></li><li><p>Sodium acetate, 3 M solution, pH 4.5 (National Diagnostics, catalog number: <a class="blackNoUnder" target="_blank"> EC-905 </a> )</p></li><li><p>Sodium acetate, 3 M solution, pH 5.2 (National Diagnostics, catalog number: <a class="blackNoUnder" target="_blank"> EC-906 </a> )</p></li><li><p>EDTA, 0.5 M, pH 8.0 (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9260G </a> )</p></li><li><p>Xylene cyanol, molecular biology grade</p></li><li><p>Bromophenol blue, molecular biology grade</p></li><li><p>Tris-HCl solution, 1 M, pH 9.0 (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> T2819-100ML </a> )</p></li><li><p>Amersham Hybond-XL membrane (GE Life Sciences, catalog number: <a class="blackNoUnder" target="_blank"> 45-001-148 </a> )</p></li><li><p>Tris-broate-EDTA (TBE) buffer, 10x (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9863 </a> )</p></li><li><p>DIG RNA labeling Mix (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 11277073910 </a> )</p></li><li><p>DIG RNA labeling kit (Roche, catalog number: <a class="blackNoUnder" target="_blank"> 11175025910 </a> )</p></li><li><p>DIG system (Roche, catalog number: <a class="blackNoUnder" target="_blank"> 12039672910 </a> )</p></li><li><p>DIG Easy Hyb (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 11603558001 </a> )</p></li><li><p>DIG Wash and Block Buffer Set (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 11585762001 </a> )</p></li><li><p>Anti-Digoxigenin-AP, Fab fragments (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 11093274910 </a> )</p></li><li><p>CDP-Star (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 11685627001 </a> )</p></li><li><p>RNase Zap (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> AM9780 </a> )</p></li><li><p>Growth medium for <em>R. palustris</em> (PM medium) (see Recipes)</p></li><li><p>Acid-urea gel (see Recipes)</p></li><li><p>Sodium acetate-EDTA running buffer (see Recipes)</p></li><li><p>Sodium acetate-EDTA loading buffer (see Recipes)</p></li></ol><p class="pptt" id="biaoti39005">### Equipment</p><p><em>Note: The working bench as well as electrophoresis, transfer and Northern blot systems should be cleaned with RNaseZap or equivalent to decontaminate RNase before use.</em> </p><br/><ol class="npwtwok"><li><p>Zirconia/Silica Beads, 0.1 mm dia (Bio Spec Products Inc, manufacture number: <a class="blackNoUnder" target="_blank"> 11079101z </a> )</p></li><li><p>Mini-Beadbeater-24 (Bio Spec Products Inc, catalog number: <a class="blackNoUnder" target="_blank"> 112011 </a> )</p></li><li><p>Electrophoresis, transfer and Northern blot system that fits a gel 20 cm in height</p></li><li><p>Temperature-controlled tabletop centrifuge</p></li><li><p>PCR machine for temperature-controlled incubation or equivalent incubator</p></li></ol><p class="pptt" id="biaoti39006">### Software</p><ol class="npwtwok"><li><p>ImageJ</p></li></ol><p class="pptt" id="biaoti39007">### Procedure</p><ol class="npwone"><li><p>Isolation of total RNA from <em>R. palustris</em> </p><p><em>Notes:</em> </p><ol class="npwthree"><li><p><em>It is crucial to choose the proper cell lysis method for the organism of interest in order to capture the native charging state of tRNA. The method described here, based on micro beads and a beadbeater, is routinely used in our group to isolate RNA from R. palustris and related Î±-proteobacteria. We strongly advise the reader to test different lysis methods for their organism of interest (e.g., the ratio between bead volume and medium volume, the number of cell beating cycles, etc.).</em> </p></li><li><p><em> Another note about tRNA charging states is that, as elaborated elsewhere (Janssen et al., 2012; Bullwinkle and Ibba, 2016), the sample needs to be kept in acidic conditions all the time. Acidic buffers and gel systems are used throughout this protocol. </em> </p></li><li><p><em>RNA purified from Procedure A can be used directly in the following steps in this protocol. This is because the charging state of tRNA is calculated based on the ratio of charged vs uncharged tRNA. Slight contamination of DNA and/or protein, therefore, will not affect the end result after the purification described here. If the purified RNA is to be used in other procedure such as RNAseq or in vitro translation, further purification would be required.</em> </p><br/></li></ol><ol class="npwtwok"><li><p>Before the experiment, prepare one screw-capped 2 ml centrifuge tube for each sample. Add ~500 Âµl of Zirconia/Silica Beads into each tube and autoclave all of the tubes.</p></li><li><p>Grow <em>R. palustris</em> cultures as required by the experimental question. Typical growth conditions that we use are given in Kim and Harwood (1991). However, the composition of the cultivation medium is not important for the protocol we describe here. The typical sample contains 3-5 ml of culture at Abs<sub>660</sub> ~1.2. Collect the cells at 4 Â°C by centrifugation at 4,000 rpm and store the pellets at -80 Â°C.</p></li><li><p>Resuspend each frozen cell pellet in 1 ml of QIAzol Lysis Reagent. Mix gently with pipetting and transfer the resuspended samples into the tubes with beads. This step should be performed on ice.</p></li><li><p>Break the cells with a beadbeater at 3,500 rpm for 1 min and then place the samples on ice for another 1 min. Repeat this cycle 3 more times.</p></li><li><p>Place all the samples at room temperature for 5 min.</p></li><li><p>Add 200 Âµl chloroform to each sample. Shake the tube vigorously by hand for 15 s.</p></li><li><p>Place the tube at room temperature for 3 min.</p></li><li><p>Centrifuge the samples for 15 min at 12,000 <em>x g</em> at 4 Â°C. Carefully transfer 560 Âµl of the colorless upper aqueous phase to a new 1.5 ml centrifuge tube.</p></li><li><p>Add 0.5 ml isopropanol and mix thoroughly by vortexing.</p></li><li><p>Centrifuge for 15 min at 12,000 <em>x g</em> at 4 Â°C. Discard the supernatant.</p></li><li><p>Add 1 ml of 75% ethanol. Centrifuge for 5 min at 7,500 <em>x g</em> at 4 Â°C.</p></li><li><p>Remove the supernatant completely. Air-dry the RNA pellet in a fume hood briefly (5-10 min).</p></li><li><p>Re-dissolve each RNA pellet in 10 mM of sodium acetate, pH 4.5. The purified RNA can be stored at -80 Â°C for at least two weeks. A typical yield is 50 Âµl of 5 Âµg/Âµl total RNA. This number can vary depending on factors such as species, growth states and sampling volume.</p><br/></li></ol></li><li><p>Alkaline pH treatment to produce a deacylated (uncharged) tRNA control</p><ol class="npwtwok"><li><p>Take ~10 Âµg of purified RNA. Add stock solution of 0.5 M EDTA and 1 M Tris-HCl (pH 9.0) directly to the sample in order to adjust the final buffer to 1 mM EDTA and 100 mM Tris-HCl, pH 9.0. Adjust final volume to 50 Âµl. Mix well by pipetting.</p></li><li><p>Incubate the sample at 42 Â°C for 60 min.</p></li><li><p>Add 5 Âµl of 3 M sodium acetate, pH 5.2 to the sample. Reverting the buffer back to pH 5.2 will not make the sample acylated again.</p></li><li><p>Add 137.5 Âµl of ice-cold 95% ethanol to the sample, then incubate on ice for 10 min.</p></li><li><p>Centrifuge at 13,000 <em>x g</em> for 10 min at 4 Â°C to precipitate RNA.</p></li><li><p>Remove the supernatant and wash the RNA pellet with 1 ml of 70% ethanol.</p></li><li><p>Centrifuge at 13,000 <em>x g</em> for 10 min at 4 Â°C.</p></li><li><p>Remove the supernatant completely. Air-dry the RNA pellet as needed.</p></li><li><p>Dissolve deacylated RNA with ~30 Âµl 10 mM sodium acetate, pH 4.5.</p><br/></li></ol></li><li><p>Making DIG-labeled probe for tRNA-Trp</p><p>DIG RNA labeling kit was used in our project to generate labeled probe for subsequent experiments. Equivalent reagents, such as DNase I and T7 RNA polymerase, can be substituted where suitable. Other strategies such as the classical P<sup>32</sup>-labeling can also be used for this step. If so, the following experiments should be adjusted accordingly. As the detailed protocol for the DIG kit is commercially available, this section will focus on how we adapted it to detect tRNA<sub>Trp</sub> in <em>R. palustris</em> CGA009.</p><br/><ol class="npwtwok"><li><p>A stretch of 31 nucleotides in <em>R. palustris</em> CGA009 tRNA<sub>Trp</sub> was chosen as the target of the probe. The following dsDNA was commercially made (IDT) as the template of <em>in vitro</em> transcription:</p><p>TGAATTG<span style="background-color:#CCCCCC;">TAATACGACTCACTATAGGG</span><u>GGTTTTGGAGACCGGTGCTCTACCAATTGAG</u> </p><p>A basic purification level, which should be molecular biology grade by default, was sufficient for dsDNA in our experience. We have also ordered two complementing ssDNA and annealed them together to form the dsDNA (see Reference 5 for more information), and it worked as well.</p><p>    Within the above sequence, the shaded part is the promoter for T7 RNA polymerase and the underscored part is the reverse-complement of our target on tRNA<sub>Trp</sub>. A 7-bp clamp is added before the promoter to facilitate polymerase binding. The selection of tRNA target-sequence needs to be done case by case and the readers are encouraged to try several candidate sequences.</p></li><li><p>Make the reaction mixture following the instructions for the DIG RNA labeling kit. Below is an example we used for our project (all reagents here except the template are included in the kit). The volume can be scaled up or down as needed.</p><p>9 Âµl RNase-free water</p><p>4 Âµl transcription (5x) buffer</p><p>4 Âµl DIG labeling mix</p><p>2 Âµl T7 polymerase</p><p>1 Âµl template of dsDNA (10 ng/Âµl is sufficient)</p></li><li><p>Incubate the reaction mixture at 42 Â°C for 1.5 h.</p></li><li><p>For each reaction, add 2 Âµl DNaseI and incubate at 37 Â°C for 15 min.</p></li></ol>There is no need to further purify the probe. The reaction product is diluted and ready for further steps. A testing Northern blot, such as a dot blot with total RNA, can be used to determine the optimal dilution factor for the probe to be used in actual experiments (<em>e.g.</em>, make a serial dilution of the probe by the factor of 10, then test which concentration generates the best signal-noise ratio in readerâs detection system).<p><br/></p></li><li><p>Acid-urea polyacrylamide gel electrophoresis</p><p>An electrophoresis system of at least 20 cm in height is recommended to analyze the charging states of tRNAs. The thickness of gel/spacer does not matter as long as the well can be sufficiently cleaned of urea. The difference between charged and uncharged tRNA is small in size. A smaller gel might be suitable to analyze some peptidyl-tRNAs, but in general does not provide the resolution to analyze aminoacyl-tRNAs. For each run, a sample of uncharged tRNA must be included as the negative control.</p><ol class="npwtwok"><li><p>Assemble the electrophoresis system with acid-urea gel. Add the cold sodium acetate-EDTA running buffer and pre-run the gel at 20 V for 30 min at 4 Â°C. The purpose of pre-running is to bring up the temperature of the gel. See more information at companyâs page (see Reference 6).</p></li><li><p>Incubate the sample at 72 Â°C for 3 min. Briefly spin the sample and mix it with the loading buffer at a 1:1 ratio. We usually loaded ~20 Âµl of sample, a total of 2.5-5 Âµg total tRNA, per well for a gel of 20 cm height. The minimum and maximum amounts of sample required need to be tested and determined case by case.</p></li><li><p>Clean the loading wells with syringe and needle. This needs to be done immediately before applying the sample to the well. For acid-urea gel, urea continues to accumulate in the well. This can prevent the sample from settling at the bottom of the well if not properly cleaned.</p></li><li><p>Load the sample. There should be no change of dye color. Run the gel at 15 W for 12 h (for a gel of 20 cm height) at 4 Â°C. It is crucial to bring up and maintain the operating temperature of the gel during the run, hence the setting of constant power as well as the pre-run specified above (see Reference 5 for more information).</p><br/></li></ol></li><li><p>Northern blot analysis</p><ol class="npwtwok"><li><p>After the electrophoresis, any standard protocol suiting the available system can be used to transfer RNA from the gel to a membrane. In our case, we used 0.5x TBE as the transfer buffer. We did a standard wet-transfer was at 4 Â°C, either at 20 V overnight or at 650 mA for 30 min. No obvious difference was observed between the two settings. The membrane is then ready for cross-link and following Northern blot. In our experience, exposure to a UV-light box for 10-15 min was sufficient and could serve as a start point.</p></li><li><p>The standard protocol of the DIG system (Roche, see Reference 7 for more information) was used for Northern blot blocking, washing and detection steps. Briefly, the probe for tRNA<sub>Trp</sub> is used to detect tRNA<sub>Trp</sub> on the membrane, followed by interaction with Anti-Digoxigenin-AP and then CDP-Star to generate chemiluminescence signals. Any suitable imaging system, such as film or digital imager, can be used to detect and quantify the results.</p></li></ol></li></ol><p class="pptt" id="biaoti39008">### Data analysis</p><p>ImageJ was used for quantification of the image on films. For results obtained with a digital imager, the accompanying software was used for quantification. The charging rate of tRNA is calculated as:<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201130/20201130220317_9271.png" width="600"/> </p><p>A brief walkthrough of tRNA charging gel analysis is provided here. As shown in Figure 1; lane 1 contains untreated total RNA from growing <em>R. palustris</em>, which mainly contains charged tRNA. The same sample is deacylated as described in this protocol and analyzed in lane 2. The sample to be analyzed is in lane 3. The bands corresponding to charged and uncharged tRNA are determined by referring to the bands in lanes 1 and 2. The intensity of the individual bands is the analyzed with ImageJ. An example can also be found in Figure 5d and 5e from Yin <em>et al.</em> (2019).</p><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3834/bioprotoc-10-23-3834-g001.jpg"/><br/><strong>Figure 1. Northern blot showing separation of charged and unchaged tRNA<sub>Trp</sub></strong> <p class="pptt" id="biaoti39009">### Recipes</p><ol class="npwtwok"><li><p>Growth medium for <em>R. palustris</em> (PM medium)</p><p><strong>For 1,000 ml PM</strong> </p><p>800 ml distilled water</p><p>25 ml 0.5 M Na<sub>2</sub>HPO<sub>4</sub> </p><p>25 ml 0.5 M KH<sub>2</sub>PO<sub>4</sub> </p><p>10 ml 10% (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> </p><p>1 ml Concentrated base (see below)</p><p>0.1 M Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub>Â·5H<sub>2</sub>O</p><p>2 mg/ml <em>p</em>-aminobenzoic acid</p><p>Bring volume to 1,000 ml</p><br/><p><strong>For 1,000 ml Concentrated base</strong> </p><p>20 g Nitrilotriacetic acd (NTA-acid free)</p><p>28.9 g MgSO<sub>4</sub> anhydrous</p><p>6.7 g CaCl<sub>2</sub>Â·2H<sub>2</sub>O</p><p>18.5 mg (NH<sub>4</sub>)<sub>6</sub>Mo<sub>7</sub>O<sub>24</sub>Â·4H<sub>2</sub>O</p><p>198 mg FeSO<sub>4</sub>Â·7H<sub>2</sub>O</p><p>100 ml Metals 44 (see below)</p><p>Dissolve NTA separately in 600 ml water and neutralize with KOH (14.6 g KOH). Add other components, and dissolve in the order given. Adjust to pH 6.8 before bringing to final volume of 1,000 ml. A precipitate forms when adjusting the pH from the acid side of 6.8 with KOH (need about 100 ml of 1M KOH) but will eventually redissolve with stirring. When the pH is near 6.8, the color of the solution changes from a deep yellow to straw color. Then, filter steriize and store in a glass bottle wrapped in aluminum fold. Store in refrigerator. It is good for at least one year.</p><br/><p><strong>For 1,000 ml Metals 44</strong> </p><p>2.5 g (free acid, not sodium salt)</p><p>10.95 g ZnSO<sub>4</sub>Â·7H<sub>2</sub>O</p><p>5 g FeSO<sub>4</sub>Â·7H<sub>2</sub>O</p><p>1.54 g MnSO<sub>4</sub>Â·H<sub>2</sub>O</p><p>392 mg CuSO<sub>4</sub>Â·5H<sub>2</sub>O</p><p>250 mg Co(NO<sub>3</sub>)<sub>2</sub>Â·6H<sub>2</sub>O</p><p>177 mg Na<sub>2</sub>B<sub>4</sub>O<sub>7</sub>Â·10H<sub>2</sub>O</p><p>Add EDTA to 800 ml distilled water with stirring and adjust pH to about 5.0 with 10 M NaOH to get EDTA dissoved. Add the other metals in the order given. Do not add components until the previous one has dissolved completely. Then bring to a final volume of 1,000 ml (the final pH is around 2.4, a clear and lime green solution). Then filter sterilize and store in a glass bottle wrapped in aluminum foil. Can store in refiigerator indefinitely.</p></li><li><p>Acid-urea gel</p><ol class="npwthree"><li><p>Make acid-urea gel solution in a 50 ml centrifuge tube. A recipe for 50 ml of 12% acid-urea gel solution is as follows:</p><p>22.5 g of urea</p><p>1.5 ml of 3 M sodium acetate, pH 5.2</p><p>93.8 Âµl of 0.5M EDTA, pH 8.0</p><p>18 ml of 30% acrylamide (29:1)</p></li><li><p>Add RNase-free water and bring the total volume to 45 ml</p></li><li><p>Add 1 ml of 10% APS</p></li><li><p>Invert the tube gently to mix</p></li><li><p>Then add 20 Âµl of TEMED and invert the tube gently to mix</p></li><li><p>Pour the solution immediately into the gel cassette</p></li></ol></li><li><p>Sodium acetate-EDTA running buffer</p><p>Prepare sodium acetate-EDTA running buffer as follows. The buffer should be pre-chilled to 4 Â°C before the run.</p><p>100 mM sodium acetate, pH 5.2</p><p>1 mM EDTA</p></li><li><p>Sodium acetate-EDTA loading buffer</p><p>Prepare sodium acetate-EDTA loading buffer with the following recipe:</p><p>10 ml of RNase-free water</p><p>4.8 g of urea</p><p>20 Î¼l of 0.5 M EDTA, pH 8.0</p><p>33 Î¼l of sodium acetate, pH 5.2</p><p>Pinch of xylene cyanol and bromophenol blue</p></li></ol><p class="pptt" id="biaoti39010">### Acknowledgments</p><p>This work was supported by grant W911NF-15-1-0150 to C.S.H. from the U.S. Army Research Office. We thank Dr. Yasuhiro Oda, University of Washington, for his extensive RNA work in <em>R. palustris</em>. We also thank Prof. Michael Ibba, Ohio State University for his excellent advice about tRNA. This protocol is derived from our previous work published in 2019 (Yin <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti39011">### Competing interests</p><p>The authors claim no financial or non-financial competing interest.</p><p class="pptt" id="biaoti39012">### References</p><ol class="npwtwok"><li>Bullwinkle, T. J. and Ibba, M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26858451" target="_blank">Translation quality control is critical for bacterial responses to amino acid stress.</a> <em>Proc Natl Acad Sci U S A</em> 113(8): 2252-2257.</li><li>Janssen, B. D., Diner, E. J. and Hayes, C. S. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22736012" target="_blank">Analysis of aminoacyl- and peptidyl-tRNAs by gel electrophoresis.</a> <em>Methods Mol Biol</em> 905291-309.</li><li>Yin, L., Ma, H., Nakayasu, E. S., Payne, S. H., Morris, D. R. and Harwood, C. S. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31615958" target="_blank">Bacterial longevity requires protein synthesis and a stringent response.</a> <em>mBio</em> 10(5): .</li><li>Kim, M. K. and Harwood, C. S. (1991). <a href="https://academic.oup.com/femsle/article/83/2/199/579254" target="_blank">Regulation of benzoate-CoA ligase in. </a><em><a href="https://academic.oup.com/femsle/article/83/2/199/579254" target="_blank">Rhodopseudomonas palustris.</a> FEMS Microbiology Letters</em> 83199-203.</li><li><a href="https://www.idtdna.com/pages/education/decoded/article/annealing-oligonucleotides" target="_blank">https://www.idtdna.com/pages/education/decoded/article/annealing-oligonucleotides.</a> </li><li><a href="https://www.nationaldiagnostics.com/electrophoresis/article/run-conditions-denaturing-page" target="_blank">https://www.nationaldiagnostics.com/electrophoresis/article/run-conditions-denaturing-page.</a> </li><li><a blacknounder"="" href="https://www.sigmaaldrich.com/content/dam/sigma-aldrich/docs/Roche/Bulletin/1/&lt;a class=" target="_blank"> 12039672910 </a> bul.pdf" target="_blank"&gt;https://www.sigmaaldrich.com/content/dam/sigma-aldrich/docs/Roche/Bulletin/1/<a class="blackNoUnder" target="_blank"> 12039672910 </a> bul.pdf.</li></ol>