<body>
            <h1>Combining 	Gel Retardation and Footprinting to Determine Protein-DNA Interactions of Specific and/or Less Stable Complexes</h1>
            <p><strong>Authors:</strong> Meng-Lun  Hsieh, Alice  Boulanger, Leslie G.  Knipling, Deborah M. Hinton</p>
            <h2>Abstract</h2>
            <p><p>DNA footprinting is a classic technique to investigate protein-DNA interactions. However, traditional footprinting protocols can be unsuccessful or difficult to interpret if the binding of the protein to the DNA is weak, the protein has a fast off-rate, or if several different protein-DNA complexes are formed. Our protocol differs from traditional footprinting protocols, because it provides a method to isolate the protein-DNA complex from a native gel after treatment with the footprinting agent, thus removing the bound DNA from the free DNA or other protein-DNA complexes. The DNA is then extracted from the isolated complex before electrophoresis on a sequencing gel to determine the footprinting pattern. This analysis provides a possible solution for those who have been unable to use traditional footprinting methods to determine protein-DNA contacts.</p></p>
            <p><strong>Keywords:</strong> DNase I footprinting, KMnO<sub>4</sub> footprinting, Gel retardation assays, EMSAs, Protein-DNA interactions</p>
            <h2>Background</h2><p>Nuclease/chemical footprinting is a classic method to probe protein-DNA interactions (Galas and Schmitz, 1978; Sasse-Dwight and Gralla, 1989; Hampshire <em>et al.</em>, 2007). In this technique, the binding of a protein to a particular region of DNA inhibits (protection) or increases (enhancement) the ability of the nuclease or chemical to cleave the DNA. Consequently, if using 5’-end labeled DNA, a specific pattern of protection/enhancement will be revealed by incubation of the protein-DNA complex with the cleaving reagent and the subsequent electrophoresis of the DNA on a sequencing gel. However, in order to obtain a clear, interpretable pattern, the protein must bind the DNA relatively tightly during the cleavage process. If the protein(s) of interest binds to the DNA weakly and/or if the protein has a fast off-rate, traditional footprinting methods are likely to fail. Furthermore, sometimes multiple complexes can be formed with differing protein-DNA contacts. In this case, the footprint pattern will only reveal the full ensemble of interactions rather than those associated with a discrete complex.<br/><br/>Here, we have developed a straightforward method to treat protein-radiolabeled DNA complexes with DNase I or KMnO<sub>4</sub> followed by isolation of the bound complex using an Electrophoretic Mobility Shift Assay (EMSA). In this assay, the DNA is incubated with the protein(s) of interest, and free DNA is separated from protein-bound DNA by electrophoresis on a native gel. This separation occurs because the protein-DNA complex migrates more slowly than the free DNA on the gel. The protein-DNA complex of interest is then extracted from the excised polyacrylamide gel slice, and the DNA is isolated and electrophoresed on a sequencing gel to determine the footprint. In many instances, this protocol allows users to obtain clear footprint patterns even when investigating less stable and/or weakly bound complexes. For DNase I reactions, the protection pattern with or without protein(s) indicates protein binding site(s) while the hypersensitivity site(s)/enhancement pattern represents DNA bending, kinking, and/or looping. For KMnO<sub>4</sub> reactions, enhancement bands represent regions containing single-stranded thymine residues, such as within an open transcription bubble. However, the procedure could be adapted to other types of enzymatic or chemical reagents, which would yield different types of patterns.<br/><br/>In the lab, we have successfully used this protocol to obtain clear footprints of protein-DNA complexes for transcriptional activators in two different bacterial systems: <em>Bordetella pertussis</em> BvgA (Boulanger <em>et al.</em>, 2015) and <em>Vibrio cholerae</em> VpsR (Hsieh <em>et al.</em>, 2018 and 2020) in the presence and absence of RNA polymerase (RNAP). In the case of VpsR, we separated the protein-DNA complexes formed in the presence or absence of the small molecule cyclic-di-GMP (c-di-GMP) from the free DNA (Figure 1A, 1B) in order to obtain direct patterns of the complexes themselves (Figure 1C). In the case of phosphorylated response regulator BvgA (BvgA~P), while only one complex, the open complex (RPo), was formed by RNAP in the absence of ribonucleoside triphosphates (rNTPs) (Figure 2A, lanes 1 vs. 2), two complexes, both RPo and the initiating complex (RPi), were formed by RNAP in the presence of rNTPs (Figure 2A, lane 3). Consequently, we used this technique to separate RPi from the free DNA and from RPo, allowing us to visualize the DNase I footprint of each complex separately (Figure 2B, lanes 2 and 3). We also employed this method together with KMnO<sub>4</sub> footprinting to determine the position of the transcription bubble within RPo formed by RNAP with non-phosphorylated BvgA (Figure 2A, lane 4) or within RPo formed by RNAP and BvgA~P (Figure 2A, lane 2). This yielded the KMnO<sub>4</sub> footprints seen in Figure 2C, lanes 1 and 2, respectively. We were then able to compare these results to those obtained without the complex isolation step (Figure 2D, lanes 1 and 2), allowing us to determine how the stable complexes differed from the mixture of complexes that were formed in solution.<br/><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3843/bioprotoc-10-23-3843-g001.jpg"/><br/><strong>Figure 1. Example of EMSA/DNase I footprinting.</strong> Figure has been adapted and reprinted with permission from Hsieh <em>et al.</em>, 2018. (A) and (B) EMSA gels showing complexes formed by the <em>V. cholerae</em> regulator VpsR bound to the promoter DNA for the <em>V. cholerae</em> gene <em>vpsL</em> (P<em><sub>vpsL</sub></em>) in the presence and absence of the small molecule c-di-GMP and in the absence (A) and presence (B) of RNA polymerase (RNAP), as indicated. The position of the free DNA is indicated by grey arrows while shifted complexes are represented by black arrows. Binding reactions in (A) contained 5 nM<sup> 32</sup>P-labeled nontemplate P<em><sub>vpsL</sub></em> harboring -97 to +103 relative to the transcription start site, 1 μg poly (dI-dC) as competitor, and increasing amounts of VpsR from 0 to 2 μM in the absence (lanes 1-5) and presence (lanes 6-10) of 50 μM c-di-GMP. Binding reactions in (B) contained 0.04 μM P<em><sub>vpsL</sub></em>, 0.14 μM reconstituted RNAP (σ:core ratio of 2.5:1), 1.4 μM VpsR, and 50 μM c-di-GMP, as indicated, and the reactions were competed with 500 ng heparin for 15 s. (C) Sequencing gels showing the DNase I products from isolated complexes. In these cases, reactions were incubated with 0.3 U of DNase I at 37 °C for 30 s before loading on the EMSA gel, the free DNA and complexes were excised and isolated, and after extraction, the DNA was electrophoresed on the gel. GA indicates G+A ladder. A schematic of the -10 element, -35 element, and the +1 position of P<em><sub>vpsL</sub></em> is shown to the right of the image. Protection pattern and hypersensitivity sites are indicated by black rectangular boxes and thin black arrows, respectively. Dashed red box represents the VpsR binding site. <br/><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3843/bioprotoc-10-23-3843-g002.jpg"/><br/><strong>Figure 2. Example of EMSA/DNase I footprinting and of KMnO<sub>4 </sub>footprinting with and without isolation of protein-DNA complexes from EMSA gel.</strong> Figure has been adapted and reprinted with permission from Boulanger <em>et al.</em>, 2015. (A) EMSA gel showing complexes formed by the <em>B. pertussis</em> response regulator BvgA either phosphorylated (BvgA~P) or non-phosphorylated (BvgA), the promoter for the <em>B. pertussis</em> gene <em>fim3</em> (P<em><sub>fim3</sub></em>), and RNAP. Binding reactions contained 0.05 pmol DNA, 5 pmol BvgA or BvgA~P, and 0.75 pmol reconstituted RNAP (σ:core ratio of 2.5:1). All samples were competed with 200 μg/ml heparin and as indicated, rNTPs (GTP, ATP, and CTP) were added. Free DNA is represented by a grey arrow while the open complex (RP<sub>O</sub>) is labeled with a black arrow and the initiating complex (RP<sub>I</sub>) is represented by a dashed black arrow. (B) Sequencing gels showing DNase I footprints at P<em><sub>fim3</sub></em> obtained from binding reactions with 0.36 U of DNase I and the subsequent EMSA isolation step. A schematic of the binding region as well as -35 , -10, and +1 is to the right of the image. Hypersensitive sites at -15 and -80, observed with RPi, but not with RPo, are indicated with the ‘*’. (C) and (D) Sequencing gels showing KMnO<sub>4</sub> footprints in the region of position +1 of P<em><sub>fim3</sub></em> obtained by treating the binding reactions with 2.5 mM KMnO<sub>4</sub> for 2.5 min at 37 °C and then either obtaining complexes from an EMSA gel and isolating the DNA before running the sequencing gel (C) or using the binding reaction products directly without the EMSA isolation step (D). Binding reactions contained 0.5 pmol DNA, 12 pmol BvgA or BvgA~P and 1.125 pmol reconstituted RNAP (σ:core ratio of 2.5:1). GA represents G+A ladder. <br/></p>
        </body>

<p class="pptt" id="biaoti39100">## Materials and Reagents</p><p><em>Note: All reagents are stored at room temperature unless otherwise indicated.</em> </p><ol class="npwtwok"><li><p>UltraCruz<sup>Â®</sup> Autoradiography Tape (Santa Cruz Biotechnology, catalog number: sc-200214)</p></li><li><p>Clear plastic wrap, such as Saran Wrap or Glad Cling Wrap</p></li><li><p>Razor blade (Personna Gem, catalog number: <a class="blackNoUnder" target="_blank"> 62-0179 </a> or equivalent)</p></li><li><p>1.7 ml microtubes (sterile, RNase- and DNase-free) (Genesee Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 24-282S </a> )</p></li><li><p>0.2 ml PCR tubes (Axygen, catalog number: <a class="blackNoUnder" target="_blank"> PCR-02-C </a> )</p></li><li><p>1.5 ml Pestle (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 12-141-364 </a> )</p></li><li><p>50 ml, 0.22 micron Millipore Steriflip filtration system (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> SCGP00525 </a> )</p></li><li><p>Ultrafree MC Centrifugal Filters (Millipore, catalog number: <a class="blackNoUnder" target="_blank"> UFC30HV00 </a> )</p></li><li><p>8â x 10â Hyblot CL Autoradiography Film (Denville Scientific, catalog number: <a class="blackNoUnder" target="_blank"> E3018 </a> )</p></li><li><p>14â x 17â Hyblot CL Autoradiography Film (Denville Scientific, catalog number: <a class="blackNoUnder" target="_blank"> E3031 </a> )</p></li><li><p>TranScreen-HE (Kodak, catalog number: <a class="blackNoUnder" target="_blank"> 881-1457 </a> )</p></li><li><p>8â x 10â Film cassettes (Research Products International, catalog number: <a class="blackNoUnder" target="_blank"> 420180 </a> )</p></li><li><p>14â x 17â Film cassettes (Research Products International, catalog number: <a class="blackNoUnder" target="_blank"> 421417 </a> )</p></li><li><p>8â x 10â Film cassette security bag (Thomas Scientific, catalog number: <a class="blackNoUnder" target="_blank"> E3753-1 </a> )</p></li><li><p>14â x 17â Film cassette security bag (Thomas Scientific, catalog number: <a class="blackNoUnder" target="_blank"> E3753-0 </a> )</p></li><li><p>Purified proteins to be tested and stored at the appropriate temperature for the particular protein</p></li><li><p>Oligonucleotides that anneal to the 5â-end of the DNA regions (usually 20-30 basepairs (bp)) and are used in PCR to amplify the DNA region needed for footprinting, dissolved in ddH<sub>2</sub>O at 50 pmol/Î¼l and stored at -20 Â°C (We recommend using PCR to generate a DNA fragment of 200 to 300 bp). We have obtained oligonucleotides from Integrated DNA Technologies. However, oligonucleotides can be purchased from many sources.</p></li><li><p>Genomic DNA or plasmid DNA containing the protein-binding DNA region that can be used as a template for PCR amplification, dissolved in ddH<sub>2</sub>O or TE (see below) at 100 ng/ml and stored at 4 Â°C (genomic DNA) or at -20 Â°C (plasmid DNA)</p></li><li><p>[Î³-<sup>32</sup>P]-ATP (3,000 Ci/mmol, 10 mCi/ml, 250 Î¼Ci) (PerkinElmer, catalog number: <a class="blackNoUnder" target="_blank"> BLU002A; </a> stored at -20 Â°C)</p></li><li><p>Optikinase (10 U/Î¼l) and 10x Optikinase buffer (USB Affymetrix, catalog number: <a class="blackNoUnder" target="_blank"> 78334Y </a> ; stored at -20 Â°C)</p></li><li><p>Pfu Turbo polymerase (2.5 U/Î¼l) and 10x Pfu buffer (Agilent Technologies, catalog number: <a class="blackNoUnder" target="_blank"> 600252 </a> ; stored at -20 Â°C)</p></li><li><p>dNTP mix (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> N0447, </a> 10 mM; stored at -20 Â°C)</p></li><li><p>DNase I (2 U/Î¼l) and 10x DNase I buffer (Life Technologies, catalog number: <a class="blackNoUnder" target="_blank"> AM2222 </a> ; stored at -2 Â°C)</p></li><li><p>Heparin (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> H3149 </a> , 500 Î¼g/ml dissolved in ddH<sub>2</sub>O; stored at -20 Â°C)</p></li><li><p>Poly(deoxyinosinic-deoxycytidylic) (Poly(dI-dC)) acid sodium salt (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P4929 </a> , 1 mg/ml dissolved in ddH<sub>2</sub>O; stored at -20 Â°C)</p></li><li><p>Urea Ultrapure (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> 15505-050 </a> )</p></li><li><p>Petroleum jelly</p></li><li><p>Large binder clips</p></li><li><p>40% 19:1 Acrylamide:bis solution (Bio-Rad Laboratories, catalog number: <a class="blackNoUnder" target="_blank"> 1610144 </a> ; stored at 4 Â°C)</p></li><li><p>40% 37.5:1 Acrylamide:bis solution (Bio-Rad Laboratories, catalog number: <a class="blackNoUnder" target="_blank"> 1610148 </a> ; stored at 4 Â°C)</p></li><li><p>Tetramethylethylenediamine (TEMED) (Bio-Rad Laboratories, catalog number: <a class="blackNoUnder" target="_blank"> 1610800 </a> ; stored at 4 Â°C)</p></li><li><p>TE-saturated Phenol, pH 8.0 (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 77607 </a> ; stored at 4 Â°C)</p></li><li><p>Phenol:chloroform:isoamyl alcohol (25:24:1 v/v, pH 7.9) (Life Technologies, catalog number: <a class="blackNoUnder" target="_blank"> AM9732 </a> ; stored at 4 Â°C)</p></li><li><p>190 proof ethanol (Warner-Graham Company; stored at -20 Â°C)</p></li><li><p>Drawn-out plastic pipet (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 13-711-27 </a> )</p></li><li><p>GlycoBlue<sup>TM</sup> Coprecipitant (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> AM9515 </a> ; stored at -20 Â°C)</p></li><li><p>10x TBE Buffer (Quality Biological, catalog number: <a class="blackNoUnder" target="_blank"> 351-001-131 </a> )</p></li><li><p>50x TAE Buffer (Quality Biological, catalog number: <a class="blackNoUnder" target="_blank"> 351-008-131 </a> )</p></li><li><p>TE pH 8.0 (Quality Biological, catalog number: <a class="blackNoUnder" target="_blank"> 351-011-131 </a> )</p></li><li><p>0.5 M EDTA, pH 8.0 (Quality Biological, catalog number: <a class="blackNoUnder" target="_blank"> 351-027-101 </a> )</p></li><li><p>Formamide (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 47670 </a> )</p></li><li><p>1-Butanol (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A399-500 </a> )</p></li><li><p>3 M Sodium Acetate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 71196 </a> )</p></li><li><p>Dry ice</p></li><li><p>6x Dye Loading solution for native gel (Fermentas, catalog number: <a class="blackNoUnder" target="_blank"> R0611 </a> )</p></li><li><p>Xylene cyanol (XC) dye (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> X4126 </a> )</p></li><li><p>Bromophenol blue (BPB) dye (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> B0126 </a> )</p></li><li><p>AG<sup>Â® </sup>501-X8(D) mixed bed resin (Bio-Rad Laboratories, catalog number: <a class="blackNoUnder" target="_blank"> 1426425 </a> )</p></li><li><p>Salmon sperm DNA (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> D1626 </a> ; 10 mg/ml, 5 mg/ml, and 1 mg/ml dissolved in ddH<sub>2</sub>O; stored at -20 Â°C)</p></li><li><p>ddH<sub>2</sub>O</p></li><li><p>APS (Bio-Rad Laboratories, catalog number: <a class="blackNoUnder" target="_blank"> 1610700 </a> )</p></li><li><p>Ammonium acetate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> A7330 </a> )</p></li><li><p>Calcium chloride dihydrate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C3306 </a> )</p></li><li><p>KMnO<sub>4</sub> (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 223468 </a> )</p></li><li><p>Magnesium acetate tetrahydrate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M5661 </a> )</p></li><li><p>Formic acid stock (88%) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 399388 </a> ). 4% diluted in ddH<sub>2</sub>O (see Recipes)</p></li><li><p>50 mM KMnO<sub>4</sub> (Potassium permanganate) (see Recipes)</p></li><li><p>10% Ammonium persulfate (APS) (see Recipes)</p></li><li><p>10 M Ammonium acetate (see Recipes)</p></li><li><p>1 M Magnesium acetate (see Recipes)</p></li><li><p>20% SDS (Quality Biological, catalog number: <a class="blackNoUnder" target="_blank"> 351-066-721 </a> ); 1% SDS diluted in ddH<sub>2</sub>O (see Recipes)</p></li><li><p>2 mM CaCl<sub>2</sub> (Calcium chloride) (see Recipes)</p></li><li><p>Piperidine stock (10 M) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 104094 </a> ); 2 M and 100 mM diluted in ddH<sub>2</sub>O (see Recipes)</p></li><li><p>Formamide loading dye; stored at -20 Â°C (see Recipes)</p></li><li><p>2-mercaptoethanol stock (14.3 M) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M3148 </a> ). 1 M diluted in ddH<sub>2</sub>O (see Recipes)</p></li><li><p>70% ethanol (see Recipes; stored at -20 Â°C)</p></li><li><p>Diffusion buffer (see Recipes)</p></li></ol><p class="pptt" id="biaoti39101">### Equipment</p><ol class="npwtwok"><li><p>Suitable space for working with <sup>32</sup>P radioactivity</p></li><li><p>Geiger counter to monitor radioactivity and contamination</p></li><li><p>Plexiglass shield to protect user from radioactivity</p></li><li><p>Plexiglass box for <sup>32</sup>P waste</p></li><li><p>150 ml, 0.22 micron filter apparatus (Nalgene, catalog number: <a class="blackNoUnder" target="_blank"> 125-0020 </a> or equivalent)</p></li><li><p>PCR Machine</p></li><li><p>Vortex (such as Vortex Genie 2, Scientific Industries)</p></li><li><p>Heating block that that can warm up to 95 Â°C</p></li><li><p>Vertical gel box - medium size (~7âW x ~9âH) (such as Cole-Parker Vertical Single Adjustable Slab Gel System, catalog number: <a class="blackNoUnder" target="_blank"> EW-28570-00 </a> or Hoefer<sup>TM</sup> Air-cooled Vertical Electrophoresis Unit, catalog number: <a class="blackNoUnder" target="_blank"> SE400 </a> )</p></li><li><p>Vertical gel box- large size (~14âW x ~17âH) (such as LABRepCo Model S2 Sequencing gel Electrophoresis Apparatus, catalog number: <a class="blackNoUnder" target="_blank"> 21105036 </a> or BTLab Systems Nucleic Acid Sequencing Electrophoresis cell (330 x 420 mm), catalog number: <a class="blackNoUnder" target="_blank"> BT210 </a> )</p></li><li><p>Elutrap electroelution system (GE Healthcare, catalog number: <a class="blackNoUnder" target="_blank"> 10447711 </a> )</p></li><li><p>Power supply box for electrophoresis (Bio-Rad Laboratories)</p></li><li><p>Small Microcentrifuge, such as Benchmark MyFuge Mini that spins at 6,000 rpm/2,000 <em>x g</em> </p></li><li><p>Benchtop Microcentrifuge, such as Eppendorf Centrifuge 5425 that can spin at â¥ 14,000 <em>x g</em> (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 5405000646 </a> )</p></li><li><p>Speed Vacuum (we use Thermo Electron Corp. Savant DNA 120 SpeedVac System, whose speed is 1600 rpm; Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 13442549 </a> )</p></li><li><p>Scintillation Counter</p></li><li><p>Film Developer (or phosphoimager)</p></li><li><p>Densitometer to scan autoradiographs [we use a GS-800 Calibrated Imaging Densitometer from Bio-Rad Laboratories, catalog number: 170-7983 (discontinued); new model is GS-900, catalog number: <a class="blackNoUnder" target="_blank"> 170-7989 </a> ]</p></li><li><p>Glass plates with spacers and combs to fit medium sized gel apparatus (Gel Company; we suggest a gel thickness of 1 mm)</p></li><li><p>Glass plates with spacers and combs to fit large sized gel apparatus (Gel Company; we suggest a gel thickness of 1 mm)</p></li></ol><p class="pptt" id="biaoti39102">### Software</p><ol class="npwtwok"><li><p>Software to operate densitometer (We use Quantity One from Bio-Rad Laboratories, catalog number: 1709601)</p></li></ol><p class="pptt" id="biaoti39103">### Procedure</p><p>Unless otherwise indicated, all reactions are assembled on ice, and centrifugations are performed at room temperature.</p><p>  When working with <sup>32</sup>P, use safety protocols for handling radioactivity, checking surfaces for contamination, and disposal of waste. We recommend using RNase/DNase-free, sterile ddH<sub>2</sub>O for solutions and appropriate precautions to minimize the possibility of DNase contamination.</p><ol class="npwone"><li><p>Make 5â²-<sup>32</sup>P DNA end-labeled on either the nontemplate or template strand</p><ol class="npwtwok"><li><p>Kinase reaction: 5â-<sup>32</sup>P end-label either template or nontemplate strand oligonucleotide.</p><p>In the 0.2 ml PCR tube combine the following:</p><p>7 Î¼l [Î³-<sup>32</sup>P]-ATP</p><p>1 Î¼l Optikinase enzyme</p><p>1 Î¼l 10x Optikinase buffer</p><p>1 Î¼l oligonucleotide (50 pmol/Î¼l)</p><p>Incubate at 37 Â°C for 30 min. Inactivate enzyme by incubating at 65 Â°C for 10 min.</p></li><li><p>Assemble PCR reaction.</p><p>Add the following to the kinase reaction:</p><p>76 Î¼l ddH<sub>2</sub>O</p><p>10 Î¼l 10x Pfu buffer</p><p>1 Î¼l 10 mM dNTP mix</p><p>1 Î¼l plasmid DNA or gDNA (100 ng/ml)</p><p>1 Î¼l other (nonlabeled) oligonucleotide (50 pmol/Î¼l)</p><p>1 Î¼l Pfu Turbo enzyme (2.5 U/Î¼l).</p></li><li><p>Perform PCR to generate labeled DNA fragment with the following cycles:</p><ol class="npwthree"><li><p>95 Â°C for 2 min to denature the DNA</p></li><li><p>95 Â°C for 30 s</p></li><li><p>Temperature of (m-5 Â°C)* for 30 s</p></li><li><p>72 Â°C for 30 s</p></li><li><p>Repetition of steps b-d for 25-35 cycles</p></li><li><p>72 Â°C for 10 min</p></li><li><p>4 Â°C for storage.</p></li></ol><p>* m-5 Â°Câtemperature that is 5 Â°C lower than the lowest primer melting temperature.</p></li><li><p>Prepare 4% polyacrylamide gel for the medium-sized gel apparatus the day before to ensure complete gel solidification and improved resolution.</p><p>Assemble glass plates using petroleum jelly to seal at corners of side and bottom spacers and large binder clips to keep plates together.</p><p>Mix together:</p><p>4 ml 40% acrylamide:bis 19:1</p><p>34.8 ml ddH<sub>2</sub>O</p><p>0.8 ml 50x TAE</p><p>0.4 ml 10% APS</p><p>Filter solution through a 150 ml, 0.22 micron filter apparatus. Add 12 Î¼l TEMED to the filtrate, swirl gently to mix (avoid introducing air bubbles), and immediately pour gel and insert comb. Carefully cover comb area with plastic wrap. Use binder clips over the plastic wrap at the top of gel to make a tight seal between the glass plates and the comb. Allow gel to solidify overnight at room temperature.</p></li><li><p>Electrophorese PCR-generated labeled DNA fragment on the 4% vertical polyacrylamide gel.</p><p>Place the gel into the medium-sized gel apparatus. After removing comb, mark the well positions on the front of the glass plates with a permanent marker. (This significantly improves slot visualizing during loading.) Pre-run gel at 100 V in 1x TAE for 1 h. Add 6x Dye Loading solution to the PCR sample volume. Load all of the sample on the gel. Run at 140 V in 1x TAE until the BPB component of the 6x loading dye reaches ~13 cm from the origin (~2 h).</p></li><li><p>Excise labeled DNA from gel.</p><p>Dismantle gel. Remove top glass plate. Wrap bottom plate/gel with plastic wrap. Mark corners of plastic-wrapped gel with autoradiography tape. Follow instructions for marking the tape. Expose gel to film for a few min. (Approximately 2-3 min should be sufficient.) Develop film and determine the position of the radiolabeled DNA on the gel by aligning film to the gel that is still wrapped. Make a stencil by using a razor blade to cut out desired labeled DNA band(s) on the developed film. Align stencil to plastic-wrapped gel. Using the stencil, mark the position of the labeled DNA on the plastic-wrapped gel with a permanent marker. Excise band with a razor blade. Use a clean razor for each sample. Be sure to remove the plastic wrap once labeled DNA is excised from gel. Note orientation of the gel as you remove the slice (- vs + end).</p></li><li><p>Isolate the labeled DNA by electroelution.</p><p>Assemble Elutrap per instruction manual. Fill chambers with X TAE. Load excised labeled DNA in the same orientation as it was running in the gel. (Be sure the âââ end of gel is towards the âââ end of the Elutrap and the â+â end of the gel is towards the â+â end of the Elutrap.) Electrophorese at 200 V for 1 h. Collect solution containing the labeled DNA and place it into a 1.7 ml microtube. Repeat electrophoresis, collect solution, and add it to the same tube.</p></li><li><p>Precipitate labeled DNA.</p><p>Evaporate labeled DNA solution to approximately 200 Î¼l in the speed vacuum. (<em>Note: This can take up to an hour or longer; use the ambient temperature setting and do NOT turn on heat to expedite drying).</em> Add 200 Î¼l phenol (TE-saturated). Mix 30 s, centrifuge at 2,000 <em>x g</em> for 1 min in small microcentrifuge and transfer the labeled DNA in the top (aqueous) layer to a clean microtube. Add 5x volume of 190 proof ethanol and volume of 10 M ammonium acetate to the phenol-extracted, aqueous layer containing the labeled DNA, mix well, and incubate on dry ice for 1 h or at -20 Â°C overnight. Centrifuge in benchtop microcentrifuge at highest speed (â¥14,000 <em>x g</em>) for 30 min, remove ethanol solution with drawn-out pipet, and wash precipitated labeled DNA pellet with 100 Î¼l ice cold 70% ethanol. Centrifuge as above for 5 min, remove 70% ethanol, and dry the labeled DNA in the speed vacuum for 2-3 min at room temperature. Resuspend pellet in 20 Î¼l TE. This assumes a 10% to 40% incorporation of <sup>32</sup>P into the DNA and a yield of &gt; 25%. However, if desired, the incorporation rate and yield can be quantified by using TCA (trichloroacetic acid) precipitation (see <a href="https://www.thermofisher.com/us/en/home/references/protocols/nucleic-acid-purification-and-analysis/rna-protocol/protocol-for-trichloroacetic-acid-tca-precipitation.html">here</a>).</p></li></ol></li><br/><li><p>Prepare a G+A sequencing ladder (Maxam and Gilbert,1977)</p><ol class="npwtwok"><li><p>In a 1.7 ml microtube, combine the following:</p><p>1 Î¼l labeled DNA</p><p>1 Î¼l 1 mg/ml salmon sperm DNA</p><p>8 Î¼l TE</p></li><li><p>Add 1 Î¼l 4% formic acid (see Recipes).</p></li><li><p>Incubate at 37 Â°C for 45 min.</p></li><li><p>Place tube on ice and add 150 Î¼l 2 M piperidine (see Recipes).</p></li><li><p>Incubate at 90 Â°C for 30 min.</p></li><li><p>Put tube on ice, and add 5 Î¼l 10 mg/ml salmon sperm DNA.</p></li><li><p>Add 1 ml 1-butanol, vortex to mix thoroughly, and then centrifuge in the benchtop microcentrifuge at â¥ 14,000 <em>x g</em> for 5 min. Carefully remove and discard supernatant.</p><p><em>Note: Pellets formed by butanol precipitation have a tendency to float away from the side of the tube so it is important to make sure that the pellet is not discarded when removing the supernatant.</em> </p></li><li><p>Add 150 Î¼l of 1% SDS to the pellet (see Recipes).</p></li><li><p>Add 1 ml butanol, vortex to mix thoroughly, and then centrifuge in the benchtop microcentrifuge at â¥ 14,000 <em>x g</em> for 5 min. Carefully remove and discard supernatant.</p></li><li><p>Wash pellet two times with 0.5 ml butanol, centrifuging for 1 min in the benchtop microcentrifuge at â¥ 14,000 <em>x g</em> after each wash. Carefully remove and discard supernatant.</p></li><li><p>Dry pellet in the speed-vac for 1-2 min at room temperature.</p></li><li><p>Add 10 Î¼l formamide load solution to dissolve and store at -20 Â°C.</p></li></ol></li><br/><li><p>Perform DNase I or KMnO<sub>4</sub> reaction</p><p><em>Note: The amount of protein and the specific binding buffer will be dependent on the protein being used. As a starting point, use conditions in which you know that your protein is active or see conditions listed in Boulanger et al. (2015) and Hsieh et al. (2018 and 2020).</em> </p><ol class="npwtwok"><li><p>DNase I reactions</p><p>In a total 10 Î¼l volume, incubate protein of interest (or just the proteinâs buffer as the negative control), the labeled DNA, and binding buffer appropriate for the protein to bind to the DNA. Additionally, the buffer should contain 2 mM CaCl<sub>2</sub> for the DNase I activity. The temperature should be appropriate for the protein being tested, but typically will range between room temperature and 37Â°C. We recommend using 0.1 to 0.5 pmol of DNA per reaction. The ratio of protein:DNA can be altered depending on what conditions are being tested. For good binding, we recommend starting with at least a 5 to 10 fold excess of protein to the labeled DNA. To eliminate unstable and non-specific complexes, we also recommend the addition of 1 Î¼l of 1 mg/ml poly (dI-dC) in the binding reaction or 0.5 to 1 Î¼l of 500 Î¼g/ml heparin after the binding reaction is completed. Dilute the stock DNase I enzyme (2 U/Î¼l) in 1x DNase I buffer to the desired concentration. We recommend trying a range of DNase I concentrations. We have previously used ~0.3 U total. To initiate the DNase I reaction, add 1 Î¼l diluted DNase I enzyme (or 1 Î¼l 1x DNase I buffer as the control). Final reaction volume is 11 Î¼l. Reactions can also be scaled up. Mix reaction components by tapping finger on tube 2-3 times and quickly centrifuge the sample using the small microcentrifuge at 2,000 <em>x g</em>. This step should take only ~15 s. Place sample in 37Â°C heating block, and incubate for 30 s. (This time can be varied if more or less cleavage is desired.) Immediately load the sample on gel that is already running at 100 V/h. See below: Step D3.</p></li><li><p>KMnO<sub>4</sub> reactions</p><p>Incubate protein of interest (or just the proteinâs buffer as the negative control) with the labeled DNA in an appropriate binding buffer (10 Î¼l total volume) at 37Â°C to initiate formation of single-stranded regions of the DNA, such as in an open transcription complex when using RNAP. We recommend using 0.1 to 0.5 pmol of labeled DNA. After this incubation, a competitor, such as heparin (1 Î¼l of 500 Î¼g/ml solution), can be added with an additional incubation of ~1 min to remove unstable complexes. Add 0.5 Î¼l of 50 mM KMnO<sub>4 </sub>(see Recipes) (or 0.5 Î¼l of ddH<sub>2</sub>O as the negative control). Be sure to make the KMnO<sub>4</sub> solution on the day of use. Incubate for 2.5 min at 37Â°C. Quench the reaction by adding 5 Î¼l 1 M 2-mercaptoethanol (see Recipes). Immediately load sample onto gel that is already running at 100 V/h. See below: Step D3.</p></li></ol></li><br/><li><p>Electrophorese DNase I-treated or KMnO<sub>4</sub>-treated complexes on a 4% acrylamide native gel</p><ol class="npwtwok"><li><p>Prepare 4% polyacrylamide gel for the large-sized gel apparatus the day before to ensure complete gel solidification and improved resolution.</p><p>Assemble glass plates using petroleum jelly to seal at corners of spacers and large binder clips to keep plates together.</p><p>Mix together:</p><p>12 ml 40% acrylamide:bis 37.5:1</p><p>12 ml 10x TBE</p><p>95.28 ml ddH<sub>2</sub>O</p><p>0.72 ml 10% APS.</p><p>Filter solution through a 150 ml 0.22 micron filter unit. Remove 4 ml of gel solution, add 4 Î¼l TEMED to this aliquot, mix well, and pour into plates. Once this portion of the gel solidifies as a plug (~15 min), add 20 Î¼l TEMED to remaining gel solution, mix well by inverting, but do not introduce air bubbles. Immediately pour into plates. Remove any air bubbles by tapping on glass plates. Lay horizontally on a support and immediately insert comb. Carefully cover comb area with plastic wrap. Use binder clips over the plastic wrap at the top of gel to make a tight seal between the glass plates and the comb. Allow gel to solidify overnight at room temperature.</p></li><li><p>Pre-run gel at 100 V/h for 2 h. (Be sure to mark the well positions on the front of the glass plates with a permanent marker. This significantly improves slot visualizing during loading.)</p></li><li><p>While gel is running at 100 V/h, load all of sample for one reaction in one lane. To ensure good separation, skip one lane when loading each new sample. (If more sample will be needed, the same reaction can be performed multiple times and loaded into multiple lanes. The products are then combined after excision of the gel slices.)</p></li><li><p>Run the gel at 380 V for 3 h.</p></li><li><p>Dismantle gel. Remove top plate and wrap gel/bottom plate with plastic wrap.</p></li></ol></li><br/><li><p>Isolate and extract labeled DNA from 4% acrylamide native gel</p><ol class="npwtwok"><li><p>Using autoradiography tape, mark each corner of the plastic-wrapped gel. This will help with stencil alignment.</p></li><li><p>Place unexposed film on gel in a film cassette in the dark room.</p></li><li><p>Expose film overnight.</p></li><li><p>Develop film.</p></li><li><p>As detailed in Step A6 make a stencil, but in this case, cut out both the free labeled DNA band and any desired complexes from the film. Take the minimum amount of gel needed to obtain all of the radioactivity. Slices are typically about 4-5 mm wide (the width of a gel lane) and 3 mm long. However, longer slices can be taken if necessary. Be sure to label.</p></li><li><p>Place each gel slice in an empty 1.7 ml microtube.</p></li><li><p>Crush the polyacrylamide gel with a 1.5 ml pestle in the microtube.</p></li><li><p>Add 200 Î¼l diffusion buffer (see Recipes) per sliced band. (If multiple slices were combined, add the appropriate amount of diffusion buffer.)</p></li><li><p>Incubate tubes at 60 Â°C for at least 2 h or at room temperature overnight.</p></li><li><p>Centrifuge in the benchtop microcentrifuge at â¥ 14,000 <em>x g</em> for 5 min.</p></li><li><p>Decant or carefully remove solution with pipet tip.</p></li><li><p>Transfer solution to a MC centrifugal filter.</p></li><li><p>Centrifuge the MC centrifugal filter in benchtop microcentrifuge at â¥14,000 <em>x g</em> for 5 min to remove traces of polyacrylamide.</p></li><li><p>Transfer flow-through to a clean 1.7 ml microtube.</p></li><li><p>In a speed vacuum, reduce volume to ~200 Î¼l for DNase I reactions and ~100 Âµl for KMnO<sub>4</sub> reactions, if necessary.</p><p><em>Note: This can take up to an hour or longer; use the ambient temperature setting and do NOT turn on heat to expedite drying.</em> </p></li><li><p>For DNase I reactions, phenol-extract solution. Add 200 Î¼l phenol:chloroform:isoamyl alcohol, mix 30 s, centrifuge in small microcentrifuge at 2,000 <em>x g</em> for 1 min, and transfer the top (aqueous) layer containing the labeled DNA to a clean 1.7 ml microtube.</p><p>Ethanol precipitate DNA. Add 5x volume of 190 proof ethanol, DNA volume of 10 M ammonium acetate (see Recipes), and 1 Î¼l GlycoBlue to the phenol-extracted aqueous DNA layer, mix well, incubate on dry ice for 1 h or at -20 Â°C overnight, centrifuge in benchtop microcentrifuge at â¥ 14,000 <em>x g</em> for 30 min, remove ethanol/ammonium acetate/GlycoBlue solution with drawn-out pipet, wash labeled DNA pellet with 100 Î¼l ice cold 70% ethanol, centrifuge as above for 5 min, remove 70% ethanol, and dry in speed vacuum for 2-3 min at room temperature.</p></li><li><p>For KMnO<sub>4</sub> reactions, ethanol precipitate the labeled DNA. Add 10x volume of 190 proof ethanol, mix well, incubate at -20 Â°C for 20 min, centrifuge in benchtop microcentrifuge at â¥ 14,000 <em>x g</em> for 20 min, remove ethanol solution with drawn-out pipet, wash labeled DNA pellet with 100 Î¼l ice cold 70% ethanol, centrifuge in benchtop microcentrifuge at â¥ 14,000 <em>x g</em> for 5 min, remove 70% ethanol, and dry in speed vacuum for 2-3 min. Resuspend pellet in 100 Î¼l 100 mM piperidine (see Recipes) and incubate at 90 Â°C for 30 min to perform cleavage reaction. Place the sample on ice for 5 min prior to the addition of 2 Î¼l of 5 mg/ml salmon sperm DNA. Add 1 ml 1-butanol, vortex sample to mix thoroughly, and centrifuge in benchtop microcentrifuge at â¥14,000 <em>x g</em> for 5 min (<em>Note: Pellets formed by butanol precipitation have a tendency to float away from the side of the tube so it is important to make sure that the pellet is not discarded when removing the supernatant.).</em> Wash with 10 Î¼l 1-butanol, spin in benchtop microcentrifuge at â¥14,000 <em>x g</em> for 5 min and remove butanol wash. Dry samples in speed vacuum for 2-3 min. Resuspend pellet in 18.75 Î¼l TE and 6.25 Î¼l 3 M sodium acetate. Repeat ethanol precipitation as described at the beginning of step 17.</p></li><li><p>Resuspend pellet in 10 Î¼l formamide loading dye (see Recipes).</p></li><li><p>Count samples in scintillation counter.</p></li></ol></li><br/><li><p>Electrophorese labeled DNA and the G+A ladder on an 8% denaturing sequencing gel and image gel</p><ol class="npwtwok"><li><p>Prepare 8% denaturing, sequencing gel for the large-sized gel apparatus the day before to ensure complete gel solidification and improved resolution.</p><p>Assemble glass plates using petroleum jelly to seal at corners of spacers and large binder clips to keep plates together.</p><p>Combine in a 250 ml beaker:</p><p>24 ml 40% acrylamide:bis 19:1</p><p>53.3 g urea</p><p>50.3 ml ddH<sub>2</sub>O</p><p>1 g mixed bed resin</p><p>Place beaker at 37Â°C until all of the urea has dissolved (This can require â¥20 min). The mixed bed resin will remain. Swirl to mix, and filter through a 150 ml 0.22 micron filter unit. Add 6 ml 10x TBE and 0.72 ml 10% APS to the filtrate. Pour a gel âplugâ by adding 4 Î¼l TEMED to 4 ml of the above gel solution. Mix well and pour into glass plates. Wait to solidify (~15 min). Add 20 Î¼l TEMED to remaining gel solution, mix well by inverting but do not introduce bubbles. Immediately pour gel. Remove air bubbles by tapping on glass plates. Lay horizontal on a support and immediately insert comb. Carefully cover comb area with plastic wrap. Use binder clips over the plastic wrap at the top of gel to make a tight seal between the glass plates and the comb. Allow gel to solidify overnight at room temperature.</p></li><li><p>Pre-run gel at 500 V for â¥ 30 min, 750 V for â¥ 30 min, and then 1,000 V for â¥ 30 min in x TBE.</p><p><em>Note: Be sure to mark the well positions on the front of the glass plates with a permanent marker. This significantly improves slot visualizing during loading.</em> </p></li><li><p>Stop pre-running gel. Heat samples at 95 Â°C for 2 min. During this time, rinse out each well with the running buffer using a drawn-out pipet to remove any urea that has diffused into the well.</p></li><li><p>Load samples (as well as the G+A ladder lane to determine the sequence of the DNA). Load approximately the same cpm in each lane (based on the scintillation counter values). We recommend loading a minimum of 8,000 counts per lane. Lower amounts can be used, but will require extended film exposure times.</p></li><li><p>Run gel at 1,000 V/h for ~3 h. (The exact time will depend on the region of the labeled DNA that needs to have the greatest resolution to observe the footprint.)</p></li><li><p>Remove gel from apparatus. Remove the top plate and wrap the gel/bottom plate with plastic wrap.</p></li><li><p>Place unexposed film on the plastic-wrapped gel/bottom plate in the dark room and store gel plate/gel/film in a film cassette placed in the cassette security bag in the -80 Â°C freezer. To help decrease static, you can also add an extra exposed film in between the unexposed film and the plastic-wrapped gel. Alternatively, a phosphor imaging screen and phosphor imager can be used instead.</p></li><li><p>After desired amount of days, remove film from gel and develop. Add new film if additional exposures are needed. Consider using a TranScreen if the signal is weak, and signal amplification is needed.</p></li><li><p>Scan autoradiograph on a densitometer. We use a GS-800 densitometer from Bio-Rad Laboratories.</p></li></ol></li></ol><p class="pptt" id="biaoti39104">### Recipes</p><p>We recommend using RNase/DNase-free, sterile ddH<sub>2</sub>O for solutions to minimize the possibility of RNase or DNase contamination. Unless indicated otherwise, solutions are made and stored at room temperature.</p><ol class="npwtwok"><li><p>4% formic acid</p><p>9.1 Î¼l of 88% stock plus 190.9 Î¼l ddH<sub>2</sub>O</p><p><em>Note: Make fresh right before use.</em> </p></li><li><p>1% SDS</p><p>7.5 Î¼l of 20% SDS stock plus 142.5 Î¼l ddH<sub>2</sub>O</p></li><li><p>10% APS</p><p>1 g ammonium persulfate dissolved in 10 ml ddH<sub>2</sub>O</p><p>Solution is freshly prepared before use.</p></li><li><p>Formamide loading dye</p><p>Mix 1 ml deionized formamide and 10 Î¼l saturated solution of XC and 10 Î¼l saturated solution BPB. Solution can be stored at -20Â°C for 1 week.</p><p><em>Note: To deionize the formamide, add ~0.5 g of AG<sup>Â®</sup> 501-X8(D) mixed bed resin to 5 ml of formamide, incubate at room temperature for at least 10 min, and filter using a 0.22 micron Millipore Steriflip. To make saturated solution of either XC or BPB, add ~0.2 g of the dye to a sterile 1.7 ml tube and add 1 ml of ddH<sub>2</sub>O, vortex vigorously, and centrifuge briefly using the small microcentrifuge at 2,000 x g to pellet the undissolved powder. Remove 10 Î¼l from the liquid above the remaining powder to add to the loading dye solution.</em> </p></li><li><p>10 M Ammonium acetate</p><p>Dissolve 77 g of ammonium acetate in 100 ml ddH<sub>2</sub>O and filter with 0.22 micron filter unit</p></li><li><p>2 mM CaCl<sub>2 </sub> </p><p>Dissolve 0.029 g calcium chloride dehydrate in 100 ml ddH<sub>2</sub>O and filter with 0.22 micron filter unit</p></li><li><p>50 mM KMnO<sub>4 </sub> </p><p>For the 250 mM stock, dissolve 3.95 g KMnO<sub>4</sub> in 100 ml ddH<sub>2</sub>O</p><p><em>Note: This can be stored at room temperature for several weeks.</em> </p><p>For 50 mM KMnO<sub>4</sub>, which should be made fresh on the day of use, mix 10 Î¼l 250 mM KMnO<sub>4 </sub>with 40 Î¼l ddH<sub>2</sub>O</p></li><li><p>100 mM piperidine</p><p><em>Note: Solution is freshly prepared before use.</em> </p><p>10 Î¼l of piperidine stock (10 M)</p><p>990 Î¼l ddH<sub>2</sub>O</p></li><li><p>2 M piperidine</p><p><em>Note: Solution is freshly prepared before use.</em> </p><p>30 Î¼l of piperidine stock (10 M)</p><p>120 Î¼l H<sub>2</sub>O</p></li><li><p>1 M 2-mercaptoethanol</p><p><em>Note: Solution is stored at -20 Â°C and can be thawed/refrozen for months.</em> </p><p>7 Î¼l of 2-mercaptoethanol stock (14.3 M)</p><p>93 Î¼l ddH<sub>2</sub>O</p></li><li><p>1 M Magnesium acetate</p><p>Dissolve 21.45 g magnesium acetate tetrahydrate in 100 ml ddH<sub>2</sub>O and filter with 0.22 micron filter unit</p></li><li><p>70% Ethanol</p><p><em>Note: Solution is freshly prepared before use.</em> </p><p>7.37 ml 190 proof ethanol</p><p>2.63 ml ddH<sub>2</sub>O</p></li><li><p>Diffusion buffer (100 ml)</p><p>0.5 M ammonium acetate (5 ml of 10 M ammonium acetate)</p><p>10 mM magnesium acetate (1 ml of 1 M magnesium acetate)</p><p>1 mM EDTA, pH 8.0 (0.2 ml of 0.5 M EDTA, pH 8.0)</p><p>0.1% SDS (0.5 ml of 20% SDS stock)</p><p>ddH<sub>2</sub>O (93.3 ml ddH<sub>2</sub>O)</p></li></ol><p class="pptt" id="biaoti_no">### Acknowledgements</p><p>Funding sources include the National Institutes of Health (NIH) [F30GM123632 to M.L.H]; Michigan State University DO/PhD Program (to M.L.H.); Intramural Research Program of the NIH, National Institute of Diabetes and Digestive and Kidney Diseases (to M.L.H., A.B.C, L.G.K., and D.M.H.). Protocol is based on the previously published papers (Boulanger <em>et al.</em>, 2015; Hsieh <em>et al.</em>, 2015; Hsieh <em>et al.</em>, 2020).</p><p class="pptt" id="biaoti39105">### Competing interests</p><p>There are no financial or non-financial competing interests for any of the authors.</p><p class="pptt" id="biaoti39106">### References</p><ol class="npwtwok"><li>Boulanger, A., Moon, K., Decker, K. B., Chen, Q., Knipling, L., Stibitz, S. and Hinton, D. M. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25624471" target="_blank"><em>Bordetella pertussis fim3</em> gene regulation by BvgA: phosphorylation controls the formation of inactive vs. active transcription complexes.</a> <em>Proc Natl Acad Sci U S A</em> 112(6): E526-535.</li><li>Galas, D. J. and Schmitz, A. (1978). <a href="http://www.ncbi.nlm.nih.gov/pubmed/212715" target="_blank">DNAse footprinting: a simple method for the detection of protein-DNA binding specificity.</a> <em>Nucleic Acids Res</em> 5(9): 3157-3170.</li><li>Hampshire, A. J., Rusling, D. A., Broughton-Head, V. J. and Fox, K. R. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17472895" target="_blank">Footprinting: a method for determining the sequence selectivity, affinity and kinetics of DNA-binding ligands.</a> <em>Methods</em> 42(2): 128-140.</li><li>Hsieh, M. L., Waters, C. M. and Hinton, D. M. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/32661076" target="_blank">VpsR Directly Activates Transcription of Multiple Biofilm Genes in <em>Vibrio cholerae</em>.</a> <em>J Bacteriol</em> 202(18): e00234-20.</li><li>Hsieh, M. L., Hinton, D. M. and Waters, C. M. (2018). <a href="https://pubmed.ncbi.nlm.nih.gov/30007313/" target="_blank">VpsR and cyclic di-GMP together drive transcription initiation to activate biofilm formation in <em>Vibrio cholerae</em>.</a> <em>Nucleic Acids Res.</em> 46(17): 8876-8887.</li><li>Maxam, A. M. and Gilbert, W. (1977). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC392330/" target="_blank">A new method for sequencing DNA.</a> <em>Proc Natl Acad Sci U S A</em> 74(2): 560-564.</li><li>Sasse-Dwight, S. and Gralla, J. D. (1989). <a href="https://europepmc.org/article/med/2722774" target="_blank">KMnO<sub>4</sub> as a probe for lac promoter DNA melting and mechanism<em> in vivo</em>.</a> <em>J Biol Chem</em> 264(14): 8074-8081.</li></ol>