<body>
            <h1>A Quantitative Assay to Measure Stress Granule Association of Proteins and Peptides in Semi-permeabilized Human Cells</h1>
            <p><strong>Authors:</strong> Saskia Hutten, Dorothee Dormann</p>
            <h2>Abstract</h2>
            <p><p>Stress granules (SGs) are membrane-less organelles that form in the cytoplasm through phase separation, in response to diverse stressors. SGs contain translationally stalled mRNAs, proteins involved in translation, and various RNA-binding proteins (RBPs). Due to the high local concentration of aggregation-prone RBPs, SGs might act as condensation sites for aberrant phase transitions of RBPs and could favor formation of solid protein aggregates underlying the pathological cytoplasmic inclusions found in numerous neurodegenerative diseases. Most assays aiming at studying the recruitment of RBPs into SGs are based on overexpression and SG recruitment of RBPs in intact cells. These approaches are, however, often limited by the predominantly nuclear localization of many RBPs, which precludes cytoplasmic RBP concentrations sufficient for SG localization, and does not address RBP recruitment independent of SG formation. Here, we present a quantitative method to assess recruitment of recombinant RBPs into pre-formed SGs, independent of the RBP’s nuclear localization, using semi-permeabilized cells and fluorescence microscopy. In this assay, SGs are firstly induced by a stressor, and then the plasma membrane of the stressed cells is subsequently selectively permeabilized to provide access of the recombinant protein to SGs. Nuclear import of the protein-of-interest is prevented by blocking nuclear pores with wheat germ agglutinin. This assay allows one to study the molecular mechanisms underlying recruitment of RBPs into SGs quantitatively, in absence of their nuclear import and under controlled conditions. The method allows for a direct comparison of wildtype, mutant or posttranslationally modified RBPs, for addressing the influence of other proteins’ preventing or promoting SG association of RBPs, and is also applicable to synthetic peptides.</p><br/><strong>Graphic abstract</strong><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201215/20201215014034_4435.jpg"/><br/><strong>Workflow overview for analysis of SG recruitment of recombinant proteins or peptides in semi-permeabilized cells</strong><br/></p>
            <p><strong>Keywords:</strong> RNA-binding protein (RBP), Peptides, Stress granules (SGs), Semi-permeabilized cells, Neurodegeneration</p>
            <h2>Background</h2><p><p>Stress granules (SGs), membrane-less organelles that contain translationally stalled mRNAs and several RNA-binding proteins (RBPs), form by phase separation as part of a cytoprotective mechanism in response to cellular stress (Protter and Parker, 2016; Gomes and Shorter, 2019). However, SGs have also been suggested to act as precursors for pathological protein aggregates in neurodegenerative diseases, in particular amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD) (Bentmann <em>et al.</em>, 2013; Alberti and Dormann, 2019; Wolozin and Ivanov, 2019; Zhang <em>et al.</em>, 2019). First, pathological inclusions containing the RBPs Fused in Sarcoma (FUS) or TAR DNA-binding protein 43 (TDP-43) in ALS/FTD <em>post mortem</em> tissue have been shown to harbor proteins commonly found in SGs, such as PABPC1 or TIA-1 (Dormann <em>et al.</em>, 2010; Liu-Yesucevitz <em>et al.</em>, 2010; Bentmann <em>et al.</em>, 2012; Mackenzie <em>et al.</em>, 2017). Second, ALS-associated mutations in RBPs, such as FUS, TDP-43, several hnRNPs and TIA-1, were demonstrated to promote SG localization of these proteins and/or to reduce SG dynamics (Bosco <em>et al.</em>, 2010; Dewey <em>et al.</em>, 2011; Dormann <em>et al.</em>, 2010; Kim <em>et al.</em>, 2013; Mackenzie <em>et al.</em>, 2017). Third, optogenetic approaches have elegantly demonstrated that persistent or repetitive formation of SGs can lead to cytoplasmic assemblies resembling pathological inclusions (Zhang <em>et al.</em>, 2019). Disease-linked RBPs have the tendency to phase separate at high concentration <em>in vitro</em> into liquid-like condensates that can solidify over time, a process promoted by distinct ALS-associated mutations (Murakami <em>et al.</em>, 2012; Molliex <em>et al.</em>, 2015; Patel <em>et al.</em>, 2015; Mackenzie <em>et al.</em>, 2017). As SGs contain a high local protein concentration, they could act as condensation sites for disease-linked RBPs and thus promote their pathological aggregation.<br/> </p><p><br/>What are the molecular mechanisms that determine SG association of RBPs and other proteins? In the past years, the role of individual domains or mutations of RBPs for SG localization was mainly studied by cellular (over) expression of RBPs. However, this approach does not distinguish (i) whether the ectopically expressed protein might promote or suppress the process of SG formation <em>per se</em>, (ii) whether reduced nuclear import of the protein-of-interest might increase its cytoplasmic concentration, and thereby its partitioning into SGs, or (iii) whether a specific mutation directly promotes or suppresses SG association of the protein-of-interest.<br/> </p><p><br/>    Here, we report an assay to study association of recombinant proteins or peptides with pre-formed SGs in semi-permeabilized cells by quantitative fluorescence microscopy. It has been adapted from an original protocol by Adam <em>et al.</em> (1992) who first established selective permeabilization of the plasma membrane by the detergent digitonin to study mechanisms of nuclear import of proteins. Our assay can determine SG association of recombinant proteins or peptides irrespective of their nuclear import, as nuclear pore complexes are selectively blocked by wheat germ agglutinin (WGA), known to bind to nucleoporins (Finlay <em>et al.</em>, 1987). By the use of recombinant proteins or peptides, either as fusion protein or untagged, SG association in dependence of the protein sequence or post-translational modifications can directly be correlated with further biochemical characterization <em>in vitro</em> (Hofweber and Dormann, 2018; Ukmar-Godec <em>et al.</em>, 2019; Bourgeois <em>et al.</em>, 2020). Furthermore, the influence of other factors, such as molecular chaperones or interacting proteins, can be studied under controlled conditions in a quantitative manner (Hofweber and Dormann, 2018; Bourgeois <em>et al.</em>, 2020). This protocol was first published by Hofweber and Dormann (2018) to study how SG association of FUS is altered by its nuclear import receptor and chaperone Transportin (TNPO1) as well as arginine methylation.</p><br/></p>
        </body>

<p class="pptt" id="biaoti39167">## Materials and Reagents</p><ol class="npwtwok"><li><p>High precision coverslips, 12 mm No 1.5 (Marienfeld, catalog number: 0<a class="blackNoUnder" target="_blank"> 117520 </a> )</p></li><li><p>Microscopy slides, 76 x 26 mm (any commercial available, <em>e.g.</em>, Marienfeld, catalog number: <a class="blackNoUnder" target="_blank"> 1000200 </a> )</p></li><li><p>6- or 10-cm cell culture dishes (any commercially available, <em>e.g.</em>, Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 150288 </a> or Greiner Bio-One, catalog number: <a class="blackNoUnder" target="_blank"> 664160 </a> , respectively)</p></li><li><p>Humid-chamber (<em>e.g.</em>, wet Whatman paper covered by parafilm in a 10- or 15 cm cell culture dish) (sterile)</p></li><li><p>Custom made recombinant protein or peptide (either fluorescently labelled or tagged suitably for immunostaining)</p></li><li><p>Milli-Q water</p></li><li><p>HEPES (any commercially available, <em>e.g.</em>, Sigma, catalog number: <a class="blackNoUnder" target="_blank"> H3375 </a> )</p></li><li><p>Potassium acetate (any commercially available, <em>e.g.</em>, Roth, catalog number: <a class="blackNoUnder" target="_blank"> T874.2 </a> )</p></li><li><p>Magnesium acetate (any commercially available, <em>e.g.</em>, MERCK, catalog number: <a class="blackNoUnder" target="_blank"> 7463348 </a> )</p></li><li><p>EGTA (any commercially available, <em>e.g.</em>, GERBU, catalog number: <a class="blackNoUnder" target="_blank"> 1310 </a> )</p></li><li><p>Potassium phosphate monobasic (KH<sub>2</sub>PO<sub>4</sub>) (any commercially available, <em>e.g.</em>, Roth, catalog number: <a class="blackNoUnder" target="_blank"> 3904.1 </a> )</p></li><li><p>Potassium phosphate dibasic (K<sub>2</sub>HPO<sub>4</sub>Â·3H<sub>2</sub>O) (any commercially available, <em>e.g.</em>, Roth, catalog number: <a class="blackNoUnder" target="_blank"> 6878.1 </a> )</p></li><li><p>Magnesium chloride (MgCl<sub>2</sub>) (any commercially available, <em>e.g.</em>, Sigma, catalog number: <a class="blackNoUnder" target="_blank"> M2670 </a> )</p></li><li><p>Sodium chloride (any commercially available, <em>e.g.</em>, Roth, catalog number: 0<a class="blackNoUnder" target="_blank"> 601.2 </a> )</p></li><li><p>Potassium chloride (KCl) (any commercially available, <em>e.g.</em>, Roth, catalog number: <a class="blackNoUnder" target="_blank"> 6781.1 </a> )</p></li><li><p>Disodiumhydrogen phosphate (Na<sub>2</sub>HPO<sub>4</sub>) (any commercially available, <em>e.g.</em>, Roth, catalog number: <a class="blackNoUnder" target="_blank"> P030.2 </a> )</p></li><li><p>DMSO (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> D2438 </a> )</p></li><li><p>MG-132 (10 mM in DMSO) (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> C2211 </a> )</p></li><li><p>DMEM high glucose (Thermo Fisher Scientific, Invitogen, catalog number: <a class="blackNoUnder" target="_blank"> 61995059 </a> )</p></li><li><p>10x Trypsin-EDTA (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 59418C </a> )</p></li><li><p>Sterile PBS solution (see solutions)</p></li><li><p>Fetal bovine serum (FBS) (Thermo Fisher Scientific, Invitogen, catalog number: <a class="blackNoUnder" target="_blank"> 10270-106 </a> )</p></li><li><p>Gentamycin (<em>e.g.</em>, Thermo Fisher Scientific, Invitogen, catalog number: <a class="blackNoUnder" target="_blank"> 15710049 </a> )</p></li><li><p>Digitonin, high purity (Milipore, CAS: <a class="blackNoUnder" target="_blank"> 11024-24-1 </a> )</p></li><li><p>Trypan Blue solution (Thermo Fisher Scientific, Invitogen, catalog number: <a class="blackNoUnder" target="_blank"> 152 500 61 </a> )</p></li><li><p>Wheat germ agglutinin (WGA) (lectin from Triticum vulgaris) solution (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> A2408 </a> )</p></li><li><p>37% Formaldehyde (AppliChem, catalog number: <a class="blackNoUnder" target="_blank"> 131328.1211 </a> )</p></li><li><p>TX-100 (any commercially available, <em>e.g.</em>, AppliChem, catalog number: <a class="blackNoUnder" target="_blank"> 9002-93-1 </a> )</p></li><li><p>Serum matching the species that the secondary antibodies was raised in, <em>e.g.</em>, donkey serum (Milipore, catalog number: <a class="blackNoUnder" target="_blank"> S30 </a> )</p></li><li><p>Tween-20 (Roth, catalog number: <a class="blackNoUnder" target="_blank"> 9127.2 </a> )</p></li><li><p>DAPI (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> D9542 </a> )</p></li><li><p>ProLong Diamond Antifade mountant (Thermo Fisher Scientific, Invitogen, catalog number: <a class="blackNoUnder" target="_blank"> P36961 </a> )</p></li><li><p>DL-Dithiothreitol (DTT, Applichem, catalog number: <a class="blackNoUnder" target="_blank"> A1101.0025 </a> )</p></li><li><p>Aprotinin (Roth, catalog number: <a class="blackNoUnder" target="_blank"> A162.2 </a> )</p></li><li><p>Leupepstin hemisulfate (Roth, catalog number: <a class="blackNoUnder" target="_blank"> CN33.2 </a> )</p></li><li><p>Pepstatin A (Roth, catalog number: <a class="blackNoUnder" target="_blank"> 3083.2 </a> )</p></li><li><p>0.1% (w/v) poly-L-lysine solution (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> P8920 </a> )</p></li><li><p>Antibody to detect SG core components, <em>e.g.</em>, Rabbit anti-G3BP1 antibody (ProteinTech, catalog number: <a class="blackNoUnder" target="_blank"> 13057-2-AP </a> )</p></li><li><p>Optional: Antibody to detect tag of recombinant protein, <em>e.g.</em>, Mouse anti-MBP antibody (ProteinTech, catalog number: <a class="blackNoUnder" target="_blank"> 66003-1-1g </a> )</p></li><li><p>Optional: Protein labeling kit (any commercially available, <em>e.g.</em>, Thermo Fisher Scientific, Invitogen, catalog number: <a class="blackNoUnder" target="_blank"> 46403 </a> )</p></li><li><p>Fluorescently labelled secondary antibodies (<em>e.g.</em>, donkey anti-rabbit Alexa 555 or Alexa 647; Thermo, catalog number: A-31572 or <a class="blackNoUnder" target="_blank"> A-31573 </a> ; donkey anti-mouse A-21202)</p></li><li><p>DMEM/10% FBS/10 Âµg/ml gentamycin (see Recipes)</p></li><li><p>1x Trypsin/EDTA (see Recipes)</p></li><li><p>10 mM MG-132 in DMSO (see Recipes)</p></li><li><p>Reaction buffer, either transport buffer (TPB) or potassium-phosphate buffer (KPB) (see Recipes)</p></li><li><p>10% digitonin/reaction buffer (see Recipes)</p></li><li><p>2 mg/ml WGA in reaction buffer (see Recipes)</p></li><li><p>3.7% formaldehyde in PBS (see Recipes)</p></li><li><p>10% and 0.5% TX-100/PBS solution (see Recipes)</p></li><li><p>PBS/0.1% Tween-20 (see Recipes)</p></li><li><p>1% donkey serum in PBS/0-1% Tween or alternative blocking solution for immunostaining (see Recipes)</p></li><li><p>DAPI solution (see Recipes)</p></li></ol><p class="pptt" id="biaoti39168">### Equipment</p><ol class="npwtwok"><li><p>P1000, P200, P20, P10 pipettes</p></li><li><p>Fine tip curved tweezers</p></li><li><p>Cell culture hood</p></li><li><p>CO<sub>2</sub> incubator</p></li><li><p>pH meter</p></li><li><p>Vacuum pump</p></li><li><p>Optional: upright microscope for brightfield microscopy</p></li><li><p>Confocal scanning microscope (<em>e.g.</em>, Leica SP8 with 63x/1.4 oil objective or equivalent) equipped with laser lines matching chosen secondary antibodies and protein/peptide labels. Detection by standard photomultiplier tube (PMT) has been sufficient for visualization and quantification of protein localization to SGs in our hands.</p></li></ol><p class="pptt" id="biaoti39169">### Software</p><ol class="npwtwok"><li><p>LASX software (Leica)</p></li><li><p>Fiji/ImageJ (NIH, <a href="https://imagej.net/Fiji" target="_blank">https://imagej.net/Fiji</a>)</p></li><li><p>Excel (Microsoft)</p></li><li><p>GraphPad Prism (GrapPad)</p></li></ol><p class="pptt" id="biaoti39170">### Procedure</p><ol class="npwone"><li><p>Prepare recombinant protein of choice by established purification protocols from bacteria (or equivalent expression system)</p><p>For subsequent detection, the protein should either be fused to a fluorescent protein (<em>e.g.</em>, EGFP) or another tag suitable for subsequent immunostaining. We have had good experience using a MBP-tag, as it also serves as a solubility tag for aggregation prone proteins and does not interfere with SG recruitment. Alternatively, proteins can also be labeled using commercially available labeling kits. Peptides can already be purchased fluorescently labeled.</p><p><em>Notes:</em> </p><ol class="npwtwok"><li><p><em> The recombinant protein should be highly pure, i.e., free of RNA (A<sub>260/280</sub> ratio ~0.6-0.7) and of contaminating proteins and/or degradation products. Endotoxin removal is not necessary. </em> </p></li><li><p><em>In order to avoid major changes to the buffer composition of the assay, one needs to consider that depending on the storage buffer and concentration of the protein/ peptide stock, additional salt can be introduced into the reaction. If possible, dialyze the recombinant protein(s) into the assay buffer. Alternatively, preparing a highly concentrated stock (e.g., â¥ 100x) of protein/peptide can help to prevent unphysiological salt concentrations in the final assay.</em> </p></li><li><p><em>Presence of high concentrations of detergent might interfere with cellular physiology and integrity of the nuclear membrane. The storage buffer should therefore not contain any detergents to improve protein/ peptide solubility.</em> </p></li><li><p><em>Protein stabilizing agents in the protein storage buffer, such as glycerol or sucrose, have not interfered with the assay in our hands, provided the protein stock is sufficiently highly concentrated (see point 2 above).</em> </p><br/></li></ol></li><li><p>Coat coverslips with poly-L-lysine (optional; 30-60 min)</p><p><em>Note: This step can be omitted if working with well adherent cell lines, such as HeLa cells. However, it can improve cell retention on coverslips and hence increase cell numbers for subsequent analysis.</em> </p><ol class="npwtwok"><li><p>Work under sterile conditions in cell culture hood.</p></li><li><p>Spread out coverslips either directly in appropriate cell culture dish (if subsequently used directly) or on parafilm in cell culture dish (if prepared in advance). If coverslips are not considered to be sterile, dip them once into 70% ethanol or absolute isopropanol before spreading them (on parafilm) in cell culture dish. Wait for EtOH/isopropanol to evaporate before proceeding.</p></li><li><p>Cover each coverslip with 0.1% poly-L-lysine solution using P1000 pipette and incubate for 20-30 min at room temperature.</p></li><li><p>Collect poly-L-Lysine solution using P1000 pipette (solution can be re-used up to 4 times) or aspirate using vacuum pump.</p></li><li><p>Wash coverslips once by covering them with sterile MilliQ water.</p></li><li><p>Aspirate water and let coverslips dry. Coverslips can be stored in a dry and dark place up to 2 weeks. Alternatively, rinse coverslips once with 3 ml medium (in 6 cm dish) if cells are to be seeded directly (Procedure C).</p><br/></li></ol></li><li><p>Seed cells onto coverslips (15-20 min)</p><ol class="npwtwok"><li><p>Estimate confluency of HeLa cell culture.</p></li><li><p>Aspirate medium and gently wash cells once with sterile PBS. Aspirate PBS and dissociate cells off the dish using pre-warmed 1x trypsin and 2-5 min incubation at 37 Â°C. Resuspend cells thoroughly in appropriate medium (<em>e.g.</em>, DMEM supplemented with 10% FBS and 10 Âµg/ml gentamicin) to inhibit trypsin activity and to achieve single cells in suspension.</p><p><em>Note: In order to ensure optimal and equal permeabilization of all cells, it is important to avoid cell clusters that would interfere with efficient permeabilization by digitonin. Also, cell density should be kept similar between individual replicates to minimize variability in cell permeabilization.</em> </p></li><li><p>Calculate cell concentration following standard procedures (<em>e.g.</em>, hemocytometer) or based on cell confluency. Seed cells onto (poly-L-lysine coated) coverslips so they reach ~60% confluency at the time of the assay (<em>e.g.</em>, HeLa cells double every 20-23 h, so seed them at ~30% confluency). Ensure that coverslips do not float on top of the medium.</p></li><li><p>Let cells adhere onto coverslips (overnight [12-20 h]).</p><br/></li></ol></li><li><p>Stress granule (SG) association assay (~1.5 h)</p><ol class="npwtwok"><li><p>Incubate cells with 10 ÂµM MG-132 for 2-3 h at 37 Â°C and 5% CO<sub>2</sub> to induce SGs.</p></li><li><p>Cool humidified chamber on ice.</p></li><li><p>Place coverslips with cells facing upwards well separated into the humidified chamber (<em>e.g.</em>, draw a grid onto the parafilm to keep individual reactions in order).</p></li><li><p>Aspirate left over medium off the coverslips and wash cells once in 50-100 Âµl reaction buffer. <br/><em>Notes:</em> </p><ol class="npwthree"><li><p><em>To cover a 12 mm coverslip completely, 40-50 Âµl solution are usually sufficient. For the washing steps this however should be increased to at least 100 Âµl per coverslip.</em> </p></li><li><p><em> Choice of the reaction buffer depends largely on the nature of the protein or peptide used. We have made good experience with physiological buffers, such as transport buffer (TPB) or potassium phosphate buffer (KPB) that are also commonly used in nuclear import assays using semi-permeabilized cells (Adam et al., 1990; Arnold, 2006). Choose the buffer that results in the best signal-to-noise ratio, as judged by enrichment of your protein-of-interest in SGs compared to the surrounding cytoplasm. </em> </p></li></ol></li><li><p>Selectively permeabilize the plasma membrane of cells for 4-5 min on ice using 0.004-0.01% digitonin diluted in your reaction buffer. For this step, place a drop of the digitonin solution (50 Âµl per 12 mm coverslip) onto the coverslip. Renewing the digitonin solution after 2 min can improve permeabilization.</p><p><em>Note: The exact incubation time and final digitonin concentration largely depends on the digitonin batch, cell type and cell density and needs to be experimentally determined by trypan-blue staining. Insufficient permeabilization will interfere with protein access to SGs. Overpermeabilization can result in a leaky nuclear membrane and, particularly for RBPs, enrichment of proteins in nuclei, which correlates to an underrepresenation of the protein in the cytoplasm (see Notes 1-3 below).</em> </p></li><li><p>Aspirate the digitonin solution, wash coverslips once briefly and then 4x for 4-5 min on ice in your reaction buffer.</p></li><li><p>Block nuclear pores by applying 40 Âµl of 0.2 mg/ml wheat germ agglutinin (WGA) diluted in your reaction buffer onto coverslips. Incubate on ice for 15 min.</p></li><li><p>Aspirate the WGA solution and replace with your protein(s) in reaction buffer (40 Âµl final volume/12 mm coverslip). For subsequent detection of recombinant proteins in SGs, ~100-200 nM final protein concentration in our hands is usually sufficient, but this might need to be optimized on an individual basis. In contrast to recombinant proteins, peptides can require higher concentrations, likely due to smaller molecular size and lack of signal amplification by a direct fluorescent label. If the influence of a second protein (such as a protein suspected to act as a molecular chaperone) on SG association is to be tested, a titration of the second protein with the first one held at a constant concentration is recommended.</p><p><em>Note: Include a âbuffer onlyâ control to determine bleed-through from the antibody staining for the SG marker (G3BP1) and/or unspecific binding of the antibody used to detect binding of the recombinant protein to SGs.</em> </p></li><li><p>Incubate reactions for 30 min at room temperature by placing the humidified chamber from the ice onto the bench. Avoid evaporation, as this would result in changes in protein/peptide concentration, and/or drying out of coverslips.</p></li><li><p>Place humified chamber back on ice and aspirate reaction mix. Wash cells 3 x for 5 min (100 Âµl final volume/12 mm coverslip) each with reaction buffer (continue immediately with Procedure E).</p><br/></li></ol></li><li><p>Immunostaining of SGs (and recombinant protein) (~3 h or overnight)</p><ol class="npwtwok"><li><p>Fix cells with 3.7% formaldehyde/PBS solution (50 Âµl final volume/12 mm coverslip) for 7-10 min at room temperature.</p><p><em>Note: Formaldehyde is toxic and should be handled and disposed of with necessary caution.</em> </p></li><li><p>Wash cells 2 x in PBS (100 Âµl final volume/12 mm coverslip). (Possible break point; cells can be stored in PBS overnight and immunostaining continued on the following day, although we prefer to perform the primary antibody incubation step overnight, if the protocol cannot be completed on the same day.)</p></li><li><p>Permeabilize cells using 0.5% TX-100/PBS (50 Âµl final volume/12 mm coverslip) for 5 min at room temperature. This step ensures access of antibodies and efficient SG staining.</p></li><li><p>Wash cells 2 x in PBS (100 Âµl final volume/12 mm coverslip).</p></li><li><p>Block unspecific binding sites using 1% donkey serum (50 Âµl final volume/12 mm coverslip) in PBS/0.1% Tween-20 (or alternative blocking buffer, such as 5% BSA or similar) for 10 min at room temperature.</p></li><li><p>Stain for SGs using G3BP1 antibody (1:1,000 in blocking buffer) and, if required, recombinant protein (<em>e.g.</em>, using anti-MBP antibody; 1:1,000 in blocking buffer) for 1-2 h at RT or overnight at 4 Â°C (50 Âµl final volume antibody dilution/12 mm coverslip).</p></li><li><p>Wash coverslips 3 x 5 min each in PBS/ 0.1% Tween-20 (100 Âµl final volume/12 mm coverslip).</p></li><li><p>Incubate with secondary antibodies for 30-60 min at room temperature (diluted 1:1,000 in blocking buffer; 100 Âµl final volume/12 mm coverslip). If bleed-through is a concern, use spectrally well separated dyes (<em>e.g.</em>, Alexa 488 and Alexa 647). Staining for SGs in the visible range, however, allows for easier judgement of permeabilization efficiency (see Notes below).</p></li><li><p>Wash 3x 5 min in PBS/0.1% Tween-20 (100 Âµl final volume/12 mm coverslip).</p></li><li><p>Stain nuclei using 0.5 Âµg/ml DAPI freshly diluted in PBS for 5 min at room temperature (50 Âµl final volume/12 mm coverslip).</p></li><li><p>Wash cells twice in PBS (100 Âµl final volume/12 mm coverslip).</p></li><li><p>Place a drop (5-10 Âµl/coverslip) of ProLong Diamond mounting medium onto microscopy slide, avoid air bubbles.</p></li><li><p>Use fine tip curved tweezers to take individual coverslips, dip excess PBS off using a paper towel and invert coverslips into the mounting medium.</p></li><li><p>Allow mounting medium to polymerize and dry in a dark place in a horizontal position (overnight). Mounted coverslips can be stored at 4 Â°C for up to several months.</p><br/></li></ol></li><li><p>Imaging (1-4 h, depending on number of samples)</p><p><em>Note: Acquisition settings (laser power, gain, detector settings) need to be determined individually. It is important to avoid pixel saturation in order to allow for quantitative image analysis. Also, samples to be compared directly with each other need to be recorded using identical settings. Check for bleed-through and cross talk applying standard protocols (e.g., by including a âbuffer onlyâ control in the subsequent immunostaining or by applying secondary antibody in absence of primary antibody etc).</em> </p><ol class="npwtwok"><li><p>Establish acquisition settings, using the sample showing the strongest SG recruitment of your protein/peptide.</p></li><li><p>Avoid imaging of the coverslip edge, as cells here are prone to dry out more easily, resulting in unspecific signal or leaky nuclei.</p></li><li><p>Choose (equally) well-permeabilized cells. These cells usually display little or no diffuse G3BP1 staining in the cytoplasm and only strong SG enrichment of G3BP1 (Figure 1).</p><p><em> Note: If the experiment is performed the first time, it can be helpful to include one MG-132-treated, non-permeabilized coverslip in the first experiment, in order to distinguish well from poorly permeabilized cells (Figure 1). Staining for TIA-1, another commonly used SG marker, is not well suitable to appropriately judge cell permeabilization, due to its predominantly nuclear localization. </em> </p></li><li><p>Find ideal z-position by focusing on SGs stained by G3BP1 (or alternative SG marker) antibody. Imaging an area of 73.81 x 73.81 Âµm (0.059 Âµm resolution) with a 63x objective results in sufficient pixel area per SG for subsequent analysis. Record several independent fields of view as single z-planes in all channels with a total of at least 10 cells per condition.</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3846/bioprotoc-10-24-3846-g002.jpg"/><br/><strong>Figure 1. Immunostaining for the SG marker G3BP1 in permeabilized vs. unpermeabilized cells.</strong> Unstressed (untreated) cells or MG-132-treated cells were fixed and stained for G3BP1 either before (MG-132) or after the cells have been permeabilized using digitonin (MG-132 + perm). Note that well-permeabilized cells (MG-132 + perm) are characterized by strong SG formation in comparison to untreated cells, but display reduced cytoplasmic staining for G3BP1 in comparison to cells that have been fixed directly after MG-132 treatment. Scale bar, 20 Âµm.</p></li></ol></li></ol><p class="pptt" id="biaoti39171">### Data analysis</p><ol class="npwtwok"><li><p>Open image files in Fiji/ImageJ as hyperstack and select G3BP1 (SG) channel.<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201217/20201217173046_2233.png" width="200"/> </p></li><li><p>Choose the magic wand tool. Double-click will allow one to adjust the threshold in order for a SG to be recognized as a ROI (marked by solid line).</p></li><li><p>Pressing [t] will add this ROI to the ROI manager (alternatively, âAnalyze â Tools â ROI Manager â Add [t]â).</p></li><li><p>Mark all SGs in the image and add them to the ROI manager. ROIs can be renamed as âSG1â, âSG2â <em>etc</em>. by choosing âRenameâ in the ROI manager.</p></li><li><p>Finally draw a ROI outside the area of any cell to determine the background fluorescence intensity. Rename this ROI âbackgroundâ.</p></li><li><p>Transfer ROIs into the channel to be quantified. Select each ROI, choose âPropertiesâ in the ROI manager and enter the channel number corresponding to the channel you want to quantify (usually numbered in sequence of recording).</p></li><li><p>Set measurements by âAnalyzeâ â âSet measurementsâ. Make sure âAreaâ, âMean gray valueâ, âMin &amp; max gray valueâ and âStack positionâ and âDisplay labelâ is generated.</p></li><li><p>Mark all ROIs and press âMeasureâ. ROIs can be saved for documentation (âFileâ â âSaveâ).</p></li><li><p>Save generated measurements as an excel file.</p></li><li><p>Open Excel file. For each image, subtract the âmeanâ measured for the respective background from the âmeanâ measured for the individual SG ROIs.</p></li><li><p>Transfer background corrected measurements into GraphPad Prism and plot the values, <em>e.g.</em> as dot or violin plot, displaying the mean Â± SEM in addition to the individual values. To combine several replicates (at least 3 independent replicates are recommended), calculate the mean of the individual experiments and display those as mean Â± SEM.</p></li><li><p>Apply appropriate statistical tests. If two groups are to be compared with each other, apply Shapiro-Wilk normality test to check data distribution. Apply Paired <em>t</em>-test for samples with normal distribution, Wilcoxon-Mann-Whitney test for non-normal distribution. If &gt;3 conditions are to be compared, apply 1-way ANOVA with appropriate <em>post-hoc</em> test. A typical outcome for the SG association of the RBP FUS in absence or presence of its import receptor and chaperone is shown in Figure 2.</p><br/><p><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201215/20201215020802_0262.jpg"/><br/><strong>Figure 2. Typical data set obtained by the SG association assay in semi-permeabilized cells.</strong> A. MBP-FUS-EGFP (green in merged image) associates with SGs (marked by G3BP1 stain, red in merged image) in the absence, but not presence, of its nuclear import receptor and chaperone TNPO1. DAPI is shown in turquoise in the merge. Scale bar: 10 Âµm. B. Quantification of FUS SG association of a single replicate shown in (B), ***<em>P</em> &lt; 0.001 by Wilcoxon paired <em>t</em>-test.</p></li></ol><p class="pptt" id="biaoti39172">### Notes</p><ol class="npwtwok"><li><p>The digitonin concentration and incubation time to achieve efficient permeabilization without causing nuclei to get leaky needs to be experimentally determined in advance. Successful permeabilization can be controlled by inverting the coverslip in trypan blue solution. Sufficient permeabilization will cause the nuclei to be stained blue. The incubation time should be kept to a minimum, aiming for the maximum number of cells being permeabilised. Longer digitonin incubation might result in a leaky nuclear membrane, preventing a successful block of nuclear pores by WGA and resulting in a strong nuclear localization of the recombinant protein (particularly if it has nucleic acid binding capacity).</p></li><li><p>We have observed that the 10% digitonin stock solution in DMSO loses permeabilization efficiency after several freeze-thaw cycles. Therefore, prepare small aliquots (10 Âµl) and use a fresh aliquot once permeabilization efficiency drops.</p></li><li><p>The protocol described here uses HeLa cells, as they are well-adherent and their plasma membrane can be selectively permeabilized without compromising the nuclear membrane. However, it is applicable to any other cell line meeting these criteria. Our own observations suggest that HEK293 (T) cells are too fragile and not suitable for selective permeabilization by digitonin.</p></li><li><p>Cells towards the edge of the coverslip might suffer from evaporation artefacts. Hence, be sure to image cells located in the center of the coverslip and do not allow the coverslip to dry out during the procedure.</p></li><li><p>We have successfully used both TPB and KPB for SG association of recombinant proteins and peptides. However, we have, on occasion, observed differences in the level of SG association over the surrounding cytoplasmic background signal and recommend a pilot experiment to determine ideal buffer conditions for each protein/peptide.</p></li><li><p>SGs are RNA-dependent MLOs and dissolve in presence of RNase, however in our experience it is not necessary to prepare RNase-free solutions.</p></li><li><p>We have made good experience using G3BP1 as a SG marker to verify SG localization of our protein-of-interest. G3BP1 represents a core SG protein and its signal is stable upon digitonin-mediated permeabilization of the plasma membrane. TIA-1, another common marker for SGs, might also be used, but we have observed significant reduction and stronger variability of its intensity in SGs, likely due to wash out after permeabilization. Additionally, unpermeabilized cells display a substantial level of diffuse G3BP1 in the cytoplasm surrounding SGs, allowing to distinguish unpermeablized/not well-permeabilized cells from those suitable for analysis. Unexperienced users can easily learn to judge the level of permeabilization by including a coverslip that had not been treated with digitonin in the G3BP1 immunostaining (Figure 1).</p></li><li><p>Antibodies against tags might cause considerable background staining, due to cross-reactivity with residual cellular proteins in SGs. Make sure to test for unspecific SG signal caused by your antibody of choice, by performing immunostaining on MG-132-treated cells in the absence of exogenously added recombinant protein.</p></li><li><p>SG composition has been described to vary depending on the stressor used for SG induction (Aulas <em>et al.</em>, 2017). Hence, SG association of the protein/peptide-of-interest might vary depending on reagent/treatment used for SG induction. We have mainly used MG-132, as this treatment is efficient and does not interfere with cell morphology (<em>e.g.</em>, rounding up of cells, which might hamper subsequent analysis).</p></li><li><p>When comparing different proteins that have been labelled using commercially available labeling kits, it is essential to check the degree-of-labeling (DOL) of each protein and to compare only proteins with similar DOL. As labelling efficiency can vary considerably between individual proteins/experiments, this can for example be achieved by doping in unlabelled protein until similar DOLs are reached.</p></li></ol><p class="pptt" id="biaoti39173">### Recipes</p><ol class="npwtwok"><li><p>DMEM containing 10% FBS, 10 Âµg/ml gentamycin</p><ol class="npwthree"><li><p>Add 50 ml FBS to 500 ml of DMEM</p></li><li><p>Add 0.5 ml of 10 mg/ml gentamycin</p></li><li><p>Mix well and store at 4 Â°C (keep sterile)</p></li></ol></li><li><p>1x Trypsin-EDTA</p><p>Dilute 5 ml 10x trypsin-EDTA in 45 ml 1x sterile PBS (keep sterile)</p></li><li><p>1x sterile PBS (can also be stored as 10x solution or purchased)</p><ol class="npwthree"><li><p>For 1 L, dissolve 8 g NaCl, 17.8 g Na<sub>2</sub>HPO<sub>4</sub>, 2 g KCl and 2.4 g KH<sub>2</sub>PO<sub>4</sub> in 800 ml MilliQ H<sub>2</sub>O</p></li><li><p>Adjust pH to 7.4 and adjust volume to 1 L</p></li><li><p>Autoclave and keep sterile</p></li></ol></li><li><p>10 mM MG-132 in DMSO</p><ol class="npwthree"><li><p>Prepare 10 mM MG-132 stock solution in DMSO</p></li><li><p>Aliquot and store at -20 Â°C</p></li></ol></li><li><p>Prepare stocks of DTT and protease inhibitors</p><ol class="npwthree"><li><p>Prepare 1 M DTT in MilliQ H<sub>2</sub>O</p></li><li><p>Prepare 1 mg/ml each of aprotinin and leupeptin hemisulfate in MilliQ H<sub>2</sub>O</p></li><li><p>Prepare 1 mg/ml pepstatin in either DMSO or ethanol</p></li><li><p>Aliquot and store at -20 Â°C (can be thawed/frozen multiple times)</p></li></ol></li><li><p>1x Transport Buffer (TPB) (can also be prepared and stored as 10x solution)</p><ol class="npwthree"><li><p>Prepare 20 mM HEPES pH 7.3-7.4, 110 mM KOAc, 2 mM Mg(OAC)<sub>2</sub>, 1 mM EGTA</p></li><li><p>Filter through 0.2 Âµm filter and store at 4 Â°C</p></li><li><p>Directly add 2 mM DTT and protease inhibitors right before use (1 Âµg/ml each of aprotinin, leupeptin and pepstatin)</p></li></ol></li><li><p>1x KPB (can be stored at room temperature)</p><ol class="npwthree"><li><p>Prepare 1 M stock solution of KH<sub>2</sub>PO<sub>4</sub> </p></li><li><p>Prepare 1 M stock solution of K<sub>2</sub>HPO<sub>4</sub> </p></li><li><p>Mix 19 ml 1 M KH<sub>2</sub>PO<sub>4</sub> with 81 ml 1 M K<sub>2</sub>HPO<sub>4</sub> to achieve 1 M K-phosphate buffered solution at pH 7.4</p></li><li><p>Prepare 20 mM potassium phosphate buffer pH 7.4, 5 mM MgCl<sub>2</sub>, 200 mM KOAc, 1 mM EGTA</p></li></ol></li><li><p>10% digitonin in DMSO</p><ol class="npwthree"><li><p>Prepare 10% stock solution in DMSO</p><p><em>Note: Digitonin is toxic, weigh in with care, e.g., by wearing a mask!</em> </p></li><li><p>Prepare 10 Âµl aliquots and store at -20Â°C</p></li><li><p>If required, 1% DMSO stock solutions can be prepared and stored in small aliquots</p></li><li><p>Prepare digitonin working solution freshly at the digitonin concentration tested to be efficient (see Note 1 above)</p></li></ol></li><li><p>2 mg/ml WGA solution</p><ol class="npwthree"><li><p>Prepare 2 mg/ml WGA solution in reaction buffer. Aliquot and store at -20 Â°C</p></li><li><p>Prepare working dilution freshly (final 0.2 mg/ml) in reaction buffer</p></li></ol></li><li><p>3.7% Formaldehyde/PBS</p><p>Dilute 37% formaldehyde solution in PBS freshly before use</p><p><em>Note: Formaldehyde is toxic and needs to be handled and disposed of with care (e.g., under fume hood).</em> </p></li><li><p>0.5% TX-100/PBS</p><ol class="npwthree"><li><p>Dissolve 10 ml TX-100 stock solution in 100 ml PBS. Solution might need extended stirring and can be stored at room temperature</p></li><li><p>Use 10% TX-100/PBS solution to prepare 0.5% TX-100/PBS working solution freshly</p></li></ol></li><li><p>1x PBS/0.1% Tween</p><p>Dissolve 1 ml Tween-20 in 1 L PBS and stir thoroughly. Solution can be stored at room temperature</p></li><li><p>0.5 Âµg/ml DAPI working solution</p><ol class="npwthree"><li><p>Dissolve DAPI in MilliQ water (DAPI does not dissolve in PBS!) at 1 mg/ml final concentration</p></li><li><p>Aliquot and store at -20 Â°C</p></li><li><p>Prepare freshly: 0.5 Âµg/ml DAPI working solution in PBS</p></li></ol></li></ol><p class="pptt" id="biaoti39174">### Acknowledgments</p><p>We thank Erin Sternburg for comments on the manuscript and acknowledge the BioImaging Facility of the BioMedical Center. This work was supported by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) within Emmy Noether grants DO1804/1-1 and DO1804/1-2, research grant DO1804/3-1, the priority programme SPP2191 (ID 402723784), the Munich Cluster for Systems Neurology (EXC2145 SyNergy-ID 390857198), the Fritz Thyssen Foundation (Az. 10.19.1.001MN) and the Junior Researcher Fund of Ludwig-Maximilians-UniversitÃ¤t MÃ¼nchen.</p><p>    This protocol has been first published for FUS (Hofweber and Dormann, 2018), and subsequently for Tau and lysine-rich peptides (Ukmar-     Godec <em>et al.</em>, 2019), as well as for the RBP Cold-inducible RNA-binding protein (CIRBP) (Bourgeois <em>et al.</em>, 2020). We acknowledge Adam <em>et al.</em> (1992), who first established the semi-permeabilized cell assay to study nuclear import of proteins, which served as basis for this protocol.</p><p class="pptt" id="biaoti39175">### Competing interests</p><p>The authors declare no competing interests.</p><p class="pptt" id="biaoti39176">### References</p><ol class="npwtwok"><li>Adam, S. A., Sterne-Marr, R. and Gerace, L. (1992). <a href="https://pubmed.ncbi.nlm.nih.gov/1488017/" target="_blank">Nuclear protein import using digitonin-permeabilized cells.</a> <em>Methods Enzymol</em> 219: 97-110.</li><li>Alberti, S. and Dormann, D. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31430179" target="_blank">Liquid-Liquid Phase Separation in Disease.</a> <em>Annu Rev Genet</em> 53: 171-194.</li><li>Aulas, A., Fay, M. M., Lyons, S. M., Achorn, C. A., Kedersha, N., Anderson, P. and Ivanov, P. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28096475" target="_blank">Stress-specific differences in assembly and composition of stress granules and related foci.</a> <em>J Cell Sci</em> 130(5): 927-937.</li><li>Bentmann, E., Haass, C. and Dormann, D. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23587065" target="_blank">Stress granules in neurodegeneration--lessons learnt from TAR DNA binding protein of 43 kDa and fused in sarcoma.</a> <em>FEBS J</em> 280(18): 4348-4370.</li><li>Bentmann, E., Neumann, M., Tahirovic, S., Rodde, R., Dormann, D. and Haass, C. (2012). <a href="https://pubmed.ncbi.nlm.nih.gov/22563080/" target="_blank">Requirements for stress granule recruitment of fused in sarcoma(FUS) and TAR DNA-binding protein of 43 kDa(TDP-43).</a> <em>J Biol Chem</em> 287(27): 23079-23094.</li><li>Bosco, D. A., Lemay, N., Ko, H. K., Zhou, H., Burke, C., Kwiatkowski, T. J., Sapp, P., McKenna-Yasek, D., Brown, R. H. and Hayward, L. J. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20699327" target="_blank"> Mutant FUS proteins that cause amyotrophic lateral sclerosis incorporate into stress granules.</a> <em>Hum Mol Genet</em> 19(21): 4160-4175.</li><li>Bourgeois, B., Hutten, S., Gottschalk, B., Hofweber, M., Richter, G., Sternat, J., Abou-Ajram, C., Gobl, C., Leitinger, G., Graier, W. F., Dormann, D. and Madl, T. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/32234784" target="_blank">Nonclassical nuclear localization signals mediate nuclear import of CIRBP.</a> <em>Proc Natl Acad Sci U S A</em> 117(15): 8503-8514.</li><li>Dewey, C. M., Cenik, B., Sephton, C. F., Dries, D. R., Mayer, P., Good, S. K., Johnson, B. A., Herz, J. and Yu, G. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21173160" target="_blank">TDP-43 is directed to stress granules by sorbitol, a novel physiological osmotic and oxidative stressor.</a> <em>Mol Cell Biol</em> 31(5): 1098-1108.</li><li>Dormann, D., Rodde, R., Edbauer, D., Bentmann, E., Fischer, I., Hruscha, A., Than, M. E., Mackenzie, I. R., Capell, A., Schmid, B., Neumann, M. and Haass, C. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20606625" target="_blank"> ALS-associated fused in sarcoma(FUS) mutations disrupt Transportin-mediated nuclear import.</a> <em>EMBO J</em> 29(16): 2841-2857.</li><li>Finlay, D. R., Newmeyer, D. D., Price, T. M. and Forbes, D. J. (1987). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3805121" target="_blank"> Inhibition of <em>in vitro</em> nuclear transport by a lectin that binds to nuclear pores.</a> <em>J Cell Biol</em> 104(2): 189-200.</li><li>Gomes, E. and Shorter, J. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30045872" target="_blank"> The molecular language of membraneless organelles.</a> <em>J Biol Chem</em> 294(18): 7115-7127.</li><li>Hofweber, M. and Dormann, D. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30587571" target="_blank">Friend or foe-Post-translational modifications as regulators of phase separation and RNP granule dynamics.</a> <em>J Biol Chem</em> 294(18): 7137-7150.</li><li>Kim, H. J., Kim, N. C., Wang, Y. D., Scarborough, E. A., Moore, J., Diaz, Z., MacLea, K. S., Freibaum, B., Li, S. and Molliex, A. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23455423" target="_blank">Mutations in prion-like domains in hnRNPA2B1 and hnRNPA1 cause multisystem proteinopathy and ALS.</a> <em>Nature</em> 495(7442): 467-473.</li><li>Liu-Yesucevitz, L., Bilgutay, A., Zhang, Y. J., Vanderweyde, T., Citro, A., Mehta, T., Zaarur, N., McKee, A., Bowser, R., Sherman, M., Petrucelli, L. and Wolozin, B. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20948999" target="_blank">Tar DNA binding protein-43(TDP-43) associates with stress granules: analysis of cultured cells and pathological brain tissue.</a> <em>PLoS One</em> 5(10): e13250.</li><li>Mackenzie, I. R., Nicholson, A. M., Sarkar, M., Messing, J., Purice, M. D., Pottier, C., Annu, K., Baker, M., Perkerson, R. B. and Kurti, A. (2017). <a href="https://pubmed.ncbi.nlm.nih.gov/28817800/" target="_blank">TIA1 Mutations in Amyotrophic Lateral Sclerosis and Frontotemporal Dementia Promote Phase Separation and Alter Stress Granule Dynamics.</a> <em>Neuron</em> 95(4): 808-816e809.</li><li>Molliex, A., Temirov, J., Lee, J., Coughlin, M., Kanagaraj, A. P., Kim, H. J., Mittag, T. and Taylor, J. P. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26406374" target="_blank">Phase separation by low complexity domains promotes stress granule assembly and drives pathological fibrillization.</a> <em>Cell</em> 163(1): 123-133.</li><li>Murakami, T., Yang, S. P., Xie, L., Kawano, T., Fu, D., Mukai, A., Bohm, C., Chen, F., Robertson, J., Suzuki, H., Tartaglia, G. G., Vendruscolo, M., Kaminski Schierle, G. S., Chan, F. T., Moloney, A., Crowther, D., Kaminski, C. F., Zhen, M. and St George-Hyslop, P. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21949354" target="_blank">ALS mutations in FUS cause neuronal dysfunction and death in Caenorhabditis elegans by a dominant gain-of-function mechanism.</a> <em>Hum Mol Genet</em> 21(1): 1-9.</li><li>Patel, A., Lee, H. O., Jawerth, L., Maharana, S., Jahnel, M., Hein, M. Y., Stoynov, S., Mahamid, J., Saha, S., Franzmann, T. M., Pozniakovski, A., Poser, I., Maghelli, N., Royer, L. A., Weigert, M., Myers, E. W., Grill, S., Drechsel, D., Hyman, A. A. and Alberti, S. (2015). <a href="https://pubmed.ncbi.nlm.nih.gov/26317470/" target="_blank"> A Liquid-to-Solid Phase Transition of the ALS Protein FUS Accelerated by Disease Mutation.</a> <em>Cell</em> 162(5): 1066-1077.</li><li>Protter, D. S. W. and Parker, R. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27289443" target="_blank">Principles and Properties of Stress Granules.</a> <em>Trends Cell Biol</em> 26(9): 668-679.</li><li>Ukmar-Godec, T., Hutten, S., Grieshop, M. P., Rezaei-Ghaleh, N., Cima-Omori, M. S., Biernat, J., Mandelkow, E., Soding, J., Dormann, D. and Zweckstetter, M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31266957" target="_blank">Lysine/RNA-interactions drive and regulate biomolecular condensation.</a> <em>Nat Commun</em> 10(1): 2909.</li><li>Wolozin, B. and Ivanov, P. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31582840" target="_blank">Stress granules and neurodegeneration.</a> <em>Nat Rev Neurosci</em> 20(11): 649-666.</li><li>Zhang, P., Fan, B., Yang, P., Temirov, J., Messing, J., Kim, H. J. and Taylor, J. P. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30893049" target="_blank">Chronic optogenetic induction of stress granules is cytotoxic and reveals the evolution of ALS-FTD pathology.</a> <em>Elife</em> 8: 39578.</li></ol>