<body>
            <h1>Mechanical Characterization of Glandular Acini Using a Micro-indentation Instrument</h1>
            <p><strong>Authors:</strong> Christopher S. OâBryan, Qiao  Zhang, Tanmay P. Lele, Thomas E. Angelini</p>
            <h2>Abstract</h2>
            <p><p>The linker of nucleoskeleton and cytoskeleton (LINC) complex is responsible for tethering the nucleus to the cytoskeleton, providing a pathway for the cell’s nucleus to sense mechanical signals from the environment. Recently, we explored the role of the LINC complex in the development of glandular epithelial acini, such as those found in kidneys, breasts, and other organs. Acini developed with disrupted LINC complexes exhibited a loss of structural integrity, including filling of the lumen structures. As part of our investigation, we performed a mechanical indentation assay of LINC disrupted and undisrupted MDCK II cells using a micro-indentation instrument mounted above a laser-scanning confocal microscope. Through a combination of force measurements acquired from the micro-indentation instrument and contact area measurements taken from fluorescence images, we determined the average contact pressure at which the acini structure ruptured. Here, we provide a detailed description of the design of the micro-indentation instrument, as well as the experimental steps developed to perform these bio-indentation measurements. Furthermore, we discuss the data analysis steps necessary to determine the rupture pressure of the acini structures. While this protocol is focused on the indentation of individual glandular acini, the methods presented here can be adapted to perform a variety of mechanical indentation experiments for both 2D and 3D biological systems.</p></p>
            <p><strong>Keywords:</strong> Bio-indentation, Micro-indentation, Biomechanics, Tissue mechanics, Nuclear mechanics, LINC Complex, Acinar development</p>
            <h2>Background</h2><p>Bio-indentation measurements have emerged as means to measure the material properties of biological systems at length scales ranging from sub-cellular biopolymers to multi-cellular tissue structures. Nano-indentation instruments with micron sized probes and contact areas on the order of 10-100 nm<sup>2</sup> have been used to measure the material properties of biopolymers and individual cells (Stolz <em>et al.</em>, 2004; Sen <em>et al.</em>, 2005; Strasser <em>et al.</em>, 2007; Li <em>et al.</em>, 2008). Similarly, micro-indentation instrument with millimeter sized indentation tips and contact areas on the order of 10-100 µm<sup>2</sup> have been used to study the collective behavior and responses of cellular structures to external mechanical forces (Ahn <em>et al.</em>, 2010; Levental <em>et al.</em>, 2010; Schulze <em>et al.</em>, 2017). Exploring cellular responses to mechanical queues from the micro-environment is crucial toward understanding how cells regulate growth, undergo differentiation or morphological changes, and express various genes (Vogel and Sheetz, 2006).<br/><br/>Transmission of mechanical signals from the cytoplasmic cytoskeleton to the nuclear envelope occurs through an assembly of proteins collectively referred to as the linker of nucleoskeleton and cytoskeleton (LINC) complex (Crisp <em>et al.</em>, 2006). In addition to facilitating the mechanotransduction of signals to the nucleus, the LINC complex is responsible for tethering the nucleus to the cytoskeleton and regulating cytoplasmic filament organization (Crisp <em>et al.</em>, 2006; Lei <em>et al.</em>, 2009; Mellad <em>et al.</em>, 2011; Tapley and Starr, 2013). Disruption of the LINC complex has been proposed as a potential pathway in the development of a variety of human diseases including cancerous development in glandular epithelia; a reduction in the expression of LINC complex proteins, including SUN1, SUN2, nesprin-2, and lamin A/C, has been observed in breast cancer tissues (Debnath and Brugge, 2005; Horn <em>et al.</em>, 2013; Meinke <em>et al.</em>, 2014; Matsumoto <em>et al.</em>, 2015). However, the precise role that the loss of the LINC complex plays during malignant transformation is not well understood. Recently, we explored the effect of LINC complex disruption on the development and maintenance of higher-order cellular structures (Zhang <em>et al.</em>, 2019). Development of glandular acini were hindered when the LINC complex was disrupted resulting in the loss of structural integrity and the filling of the lumens in MDCK II and MCF-10A acini. As part of our investigation into the role of the LINC complex in acinar development, we performed a bio-indentation assay of MDCK II acini to determine the average contact pressure required to induce rupture of individual acini structures. MDCK II acini with filled lumens resulting from the disruption of the LINC complex were found to require higher contact pressures to induce rupture compared to undisrupted MDCK II acini.<br/><br/>Here, we present the protocol for performing the mechanical indentation assay of individual MDCK II acini that was developed as part of our exploration into the role the LINC complex plays in acinar development. Through a combination of force measurements recorded using a micro-indentation instrument and fluorescence images captured through confocal microscopy, we measure the pressure necessary to mechanically rupture individual glandular acini. We discuss the design and operation of a micro-indentation instrument capable of applying vertical displacements on the order nanometers and measuring normal forces on the order of micro-Newtons. Additionally, we discuss the experimental protocol necessary to perform these indentation measurements and the data analysis necessary to determine the average contact pressure to rupture the acini structure. While this manuscript is focused on the indentation of individual acini, these methods can be adapted to perform a variety of mechanical indentation explorations of both 2D and 3D biological systems.</p>
        </body>

<p class="pptt" id="biaoti39120">## Materials and Reagents</p><p>The following materials and reagents are used as part of this bio-indentation protocol:</p><ol class="npwtwok"><li><p>8-well chamber slide (Nunc Lab-Tek, catalog <a class="blackNoUnder" target="_blank"> number:154534 </a> )</p></li><li><p>35 mm glass bottom Petri dish, 20 mm well size #0 thickness (Cell Vis: D35-20-0-N)</p></li><li><p>Matrigel matrix basement membrane (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 35623 </a> )</p></li><li><p>MDCK II cell line (gifted from Jennifer Lippincott-Schwartz)</p></li><li><p>MCF-10A cell line (ATCC, <a class="blackNoUnder" target="_blank"> CRL-10317 </a> ; RRID: CVCL_0598)</p></li><li><p>DMEM medium w/ 4.5 g/L glucose (Mediatech Cellgro, catalog number: <a class="blackNoUnder" target="_blank"> 23-10-013-CM </a> )</p></li><li><p>Donor bovine serum (DBS) (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 16030074 </a> )</p></li><li><p>Ethylenediaminetetraacetic acid (EDTA) (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 46034CI </a> )</p></li><li><p>Sodium orthovanadate (Na<sub>3</sub>VO<sub>4</sub>) (Sigma Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S6508 </a> )</p></li><li><p>Sodium fluoride (NaF) (Sigma Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 67414 </a> )</p></li><li><p>Calcein AM (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> C3100MP </a> )</p></li><li><p>0.25% trypsin (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 25053CI </a> )</p></li><li><p>Poly-L-lysine (Sigma Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P4832 </a> )</p></li><li><p>Phosphate buffered saline (PBS) solution 1x (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP243820 </a> )</p></li><li><p>F-127 pluronic (Sigma Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P2443 </a> )</p></li><li><p>Millipore water</p></li><li><p>MDCK cell growth medium (see Recipes)</p></li><li><p>MDCK acini indentation medium (see Recipes)</p></li><li><p>Matrigel wash solution (see Recipes)</p></li><li><p>Acini dye medium (see Recipes)</p></li><li><p>F-127 pluronic solution (see Recipes)</p></li></ol><p class="pptt" id="biaoti39121">### Equipment</p><p>The following equipment is used as part of this bio-indentation protocol:</p><ol class="npwtwok"><li><p>Micro-indentation Instrument:</p><ol class="npwthree"><li><p>Sapphire indentation tip (Edmund Optics, catalog number: <a class="blackNoUnder" target="_blank"> 48-430 </a> )</p></li><li><p>Capacitance load cell</p><ol class="npwfour"><li><p>Double-leaf cantilever beam (custom-built; spring stiffness <em>k</em> = 49.94 N/m)</p></li><li><p>Capacitive probe (Lion Precision, model: <a class="blackNoUnder" target="_blank"> C5R-0.8 </a> )</p></li><li><p>Capacitive sensor (Lion Precision, model: <a class="blackNoUnder" target="_blank"> CPL290 </a> )</p></li></ol></li><li><p>Nylon M6 threaded rod (McMaster-Carr, catalog number: <a class="blackNoUnder" target="_blank"> 98882A249 </a> )</p></li><li><p>Nylon M6 hexagonal nut (McMaster-Carr, catalog number: <a class="blackNoUnder" target="_blank"> 93800A600 </a> )</p></li><li><p>Aluminum mount for cantilever beam (custom-built)</p></li><li><p>Vertical precision Z-positioner (Physik Instrumente, model: <a class="blackNoUnder" target="_blank"> P-622.ZCL PIHera </a> )</p></li><li><p>Aluminum mounting adapter (custom-built)</p></li><li><p>25 mm compact XY linear translational stage (Thor Lab, model: <a class="blackNoUnder" target="_blank"> LX20 </a> )</p></li><li><p>Aluminum mounting flange (custom-built)</p></li><li><p>Super glue gel control (Locktite)</p></li></ol></li><li><p>Nikon Eclipse Ti2-E Inverted Confocal Microscope:</p><ol class="npwthree"><li><p>Nikon Ti2-Eclipse inverted microscope (Nikon)</p></li><li><p>Nikon C2+ confocal scan head (Nikon)</p></li><li><p>C2-DU3 detector unit (Nikon)</p></li><li><p>DAPI filter cube 438/24 (Nikon)</p></li><li><p>FITC filter cube 525/50 (Nikon)</p></li><li><p>561 long pass filter (Nikon, model: <a class="blackNoUnder" target="_blank"> 561LP </a> )</p></li><li><p>Reflectance filter cube (Nikon, model: <a class="blackNoUnder" target="_blank"> A1-C2 C165305 </a> )</p></li><li><p>LU-N4 laser unit (Nikon)</p></li><li><p>4x microscope objective (Nikon, model: <a class="blackNoUnder" target="_blank"> MRD00045 </a> )</p></li><li><p>10x microscope objective (Nikon, model: <a class="blackNoUnder" target="_blank"> MRD00101 </a> )</p></li><li><p>20x microscope objective (Nikon, model: <a class="blackNoUnder" target="_blank"> MRD00205 </a> )</p></li><li><p>Automated translational stage (Nikon, model: <a class="blackNoUnder" target="_blank"> 761794 </a> )</p></li><li><p>perfect focus (Nikon)</p></li></ol></li><li><p>Petri dish support plate:</p><ol class="npwthree"><li><p>Aluminum Petri dish support plate (custom-built)</p></li><li><p>Thermocouple (Omega)</p></li><li><p>Power supply (Velleman, model: <a class="blackNoUnder" target="_blank"> PS613U </a> )</p></li><li><p>Temperature controller (Omega, model: <a class="blackNoUnder" target="_blank"> CN7800 </a> )</p></li><li><p>Nichrome heating wire (Pelican Wire, model: <a class="blackNoUnder" target="_blank"> 2130N80DGS </a> )</p></li></ol></li><li><p>O<sub>2</sub> plasma cleaner (Plasma Etch, model: <a class="blackNoUnder" target="_blank"> PE-50 </a> )</p></li><li><p>Cell incubator</p></li><li><p>Cell centrifuge</p></li></ol><br/><p><strong>Description of Indentation Instrument</strong> </p><p>Bio-indentation measurements of individual acini structures are performed using a micro-indentation instrument mounted above an inverted microscope (Figure 1A). The micro-indentation instrument consists of a hemispherical indentation tip, a precision load cell, a vertical Z-positioner, an XY translational stage, and a male dovetail mounting flange (Figure 1B). The indentation instrument is mounted into the condenser slot of the inverted microscope and secured using a hexagonal set screw (Figure 1C). Below, we describe the individual components of the micro-indentation instrument used to perform the indentation assay reported by Zhang <em>et al.</em>, 2019 (A detailed CAD rendering of the indentation instrument can be found in the work presented by Schulze <em>et al.</em>, 2017). Alternatively, there are several commercially available micro-indentation instruments, including the Bruker Hysitron Biosoft indenter, that provide comparable levels of performance.</p><p>  The bio-indentation measurements reported by Zhang <em>et al.</em> (2019) were performed using a micro-indentation instrument with a 1.6 mm radius of curvature hemispherical sapphire indentation tip. The load cell of the instrument consists of a double-leaf cantilever beam with a spring stiffness of <em>k</em> = 49.94 N/m, and a capacitive probe with a 100 Âµm detection range. The normal force of the indentation tip is calculated from the measured vertical deflection of the cantilever beam and the spring stiffness. Calibration of the bio-indentation instrument was performed by hanging weights of known mass from the double-leaf cantilever beam and measuring the resulting displacement using the capacitance probe. The spring stiffness was determined by fitting a line to the resulting force vs displacement curve. To provide an offset from the load cell and to allow for easy removal and cleaning, the indentation tip is attached to a ~15 mm long nylon threaded rod; the indentation tip is superglued to the tip of the mounting rod and a complementary mounting nut is superglued to the load cell. To control the vertical displacement of the indentation tip, the load cell is mounted to a PI piezoelectric translational z-stage capable of a 250 Âµm travel range with 1 nm resolution. The translational z-stage is subsequently mounted onto a 2-axis linear translational stage, allowing for alignment of the indentation tip in the XY plane. A D3N male dovetail mounting flange is fixed to the top of the XY translational stage to enable the bio-indentation instrument to be mounted into the condenser slot of an inverted microscope. The bio-indentation instrument is controlled through a custom-written Labview program. This program controls the vertical displacement of the piezoelectric stage while recording the signal from the capacitive sensor to calculate the vertical deflection of the double-leaf cantilever beam.</p><br/><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3847/bioprotoc-10-23-3847-g001.jpg"/><br/><strong><span style="line-height:2;">Figure 1. Micro-indentation instrument.</span></strong><span style="line-height:2;"> A. Bio-indentation measurements are performed using a micro-indentation instrument mounted above an inverted microscope. B. The micro-indentation instrument consists of a hemispherical indentation tip, a load cell to measure the normal force, a vertical Z-positioner with nanometer resolution, a manual XY translational stage, and a dovetail mounting flange. C. The indentation instrument is mounted into the condenser slot of the inverted microscope and secured in place with a hexagonal set screw.<br/></span> <br/><p><span style="line-height:2;">   The bio-indentation instrument is mounted on a Nikon Eclipse Ti2-E inverted microscope equipped with a C2+ confocal imaging system and a motorized translational stage. The microscope is configured to capture FITC fluorescence images using the 488 nm excitation laser and a 525/50 dichroic filter cube (Figure 2A). Additionally, confocal reflectance microscopy is used to align the indentation tip directly above the microscope objective prior to the indentation measurement. For reflectance imaging, the 488 nm laser is used as a light source and a reflectance filter cube is installed in the detector unit allowing all light to pass through to the detector channel (Figure 2B).<br/></span> </p><br/><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3847/bioprotoc-10-23-3847-g002.jpg"/><br/><strong><span style="line-height:2;">Figure 2. Microscope optical configurations.</span></strong><span style="line-height:2;"> The Nikon Eclipse Ti2-E Confocal microscope is configured to allow for either (A) fluorescence imaging of the acini structure during the indentation measurement, or (B) reflectance microscopy during the instrumental set-up to determine the center of the indentation tip. </span><br/><p><br/>  Glass bottom Petri dishes are secured during the indentation measurements using a custom-built Petri dish support plate that fits within the plate adapter of the Nikon Eclipse Ti-2 microscope (Figure 3). The glass bottom Petri dish is positioned over the objective hole in the center of the support plate (1) and is firmly secured using two mounting arms (2). The mounting arms apply pressure to the Petri dish lid to mitigate movement during the indentation measurement. A hole is drilled in the lid of the Petri dish to provide an opening for the bio-indentation instrument. To regulate the temperature during the indentation measurement, the plate is equipped with a thermocouple (3) and nichrome heating wire (4) that are connected to an on-off temperature controller. The power voltage and current settings of the power supply are set to maintain a desired temperature of 37 Â°C while minimizing the frequency of on-off cycles.</p><br/><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3847/bioprotoc-10-23-3847-g003.jpg"/><br/><strong><span style="line-height:2;">Figure 3. Petri dish support plate.</span></strong><span style="line-height:2;"> Glass bottom Petri dishes are secured during the indentation measurement using a custom-built support plate that fits into the plate adapter of the microscopeâs translational stage. The temperature of the Petri dish is regulated using a temperature controller, a thermocouple, and a nichrome heating element.</span> <p class="pptt" id="biaoti39122">### Software</p><p>The following software is used as part of this bio-indentation protocol and data analysis process:</p><ol class="npwtwok"><li><p>Nikon NIS-Elements AR imaging software (Version 4.30.02)</p></li><li><p>National Instruments LabView (Version 2014)</p></li><li><p>FIJI ImageJ (<a href="https://imagej.net/Fiji">https://imagej.net/Fiji</a>) (Version 1.52g)</p></li><li><p>Microsoft Excel (Office 365)</p></li><li><p>OriginPro graphing software (version 8.5)</p></li></ol><p class="pptt" id="biaoti39123">### Procedure</p><ol class="npwone"><li><p>Cell Culture and Acini Growth</p><p><em>Note: A complete description of cell culture protocols, cell line development, and growth of acini structures are reported by Zhang et al. (2019).</em> </p><ol class="npwtwok"><li><p>Culture and maintain MDCK II cells following standard cell culture protocols for thawing, feeding, and passaging. Incubate cells at 37 Â°C in a humidified, 5% CO<sub>2</sub> environment. Cell culture medium ingredients and their corresponding concentrations are provided in the Recipes section of this protocol.</p></li><li><p>Induce acinar formation of MDCK II cells in an 8-well chamber slide one week prior to performing the mechanical indentation measurement.</p><ol class="npwthree"><li><p>Coat an 8-well Nunc Lab-Tek chamber slide with Matrigel by spreading 40-45 Âµl of Matrigel in each well of the chamber slide. Incubate for at least 30 min at 37 Â°C or until the gels have completely solidified.</p></li><li><p>Trypsinize MDCK II cells following standard cell culture procedures. Centrifuge the cells at 125 <em>x g</em> for 15 min and remove the supernatant. Resuspend the MDCK II cells into MDCK acini indentation medium. MDCK acini indentation medium ingredients and their corresponding concentrations are provided in the Recipes section of this protocol.</p></li><li><p>Seed the MDCK II cells into Matrigel coated 8-well chamber slide at a seeding density of ~5,000 cells/well with MDCK indentation medium. Incubate the chamber slides at 37 Â°C in a humidified 5% CO<sub>2</sub> environment for approximately 7 days or until the MDCK cells have formed glandular acini structures.</p></li></ol></li><li><p>Transfer the acini structures into a glass bottom Petri dish for the indentation measurement.</p><ol class="npwthree"><li><p>Coat the glass bottom Petri dish with 0.01 wt% poly-L-lysine. Place 200 Âµl of 0.01 wt% poly-L-lysine into glass bottom Petri dish and let sit for 1 hr. Remove the poly-L-lysine solution and rinse with 1x PBS solution. The poly-L-lysine should prevent any slippage of the acini structure during the indentation process.</p></li><li><p>Remove the acini from the Matrigel growth matrix. Wash the acini in Matrigel coated 8-well chamber slide with a 1x cold PBS solution at 4 Â°C. Incubate the Matrigel coated 8-well chamber slide containing the grown acini in a 1x PBS solution supplemented with 5 mM EDTA, 1 mM NaVO<sub>4</sub>, and 1.5 mM NaF to remove the Matrigel growth matrix from the acini structure. Isolate the acini structures from the Matrigel support matrix by centrifuging at 4 Â°C for 15 min at 125 <em>x g</em>. Remove the supernatant and rinse the resulting pellet with a 1x PBS solution prepared at 4 Â°C. Centrifuge the acini structure again for 15 min at 125 <em>x g</em> and remove the supernatant. Resuspend the pellet into a 1x PBS solution at 4 Â°C and transfer the acini into poly-L-lysine coated glass bottom Petri dishes at a seeding density of ~1,000 acini/Petri dish. It is important to keep the acini density relatively low in the glass bottom Petri dishes to avoid contacting multiple acini structures during the indentation experiment.</p></li></ol></li><li><p>Dye the acini structures with calcein AM to check the viability of individual acini and enable fluorescence confocal microscopy of the acini during the indentation process. Replace the PBS solution with the appropriate cell growth medium supplemented with 1:2,000 parts calcein AM cell dye. Incubate the acini for 20 min at 37 Â°C in a humidified, 5% CO<sub>2</sub> environment. Remove the medium and calcein AM solution and replace with fresh cell culture medium. Continue to incubate the acini at 37 Â°C in a humidified, 5% CO<sub>2</sub> environment until it is time to perform the indentation experiment.</p></li></ol></li><br/><li><p>Instrument Set-Up</p><ol class="npwtwok"><li><p>Mount the bio-indentation instrument and Petri dish support plate onto the confocal microscope. The bio-indentation instrument mounts onto the confocal using the condenser slot; slide the mounting flange of the bio-indentation instrument into the condenser slot and tighten the mounting screw until the bio-indentation instrument is securely mounted. Place the Petri dish support plate into the plate adapter on the confocal microscopeâs translational stage. Connect the Petri dish support plate to the temperature controller and power supply unit, and adjust the power settings to maintain a temperature of 37 Â°C.</p></li><li><p>Coat the indentation tip with F-127 pluronic to mitigate the effects of adhesion between the glass indentation tip and the acini structure. Remove the indentation tip from the double-leaf cantilever beam by unfastening the mounting screw from the mounting nut. Plasma clean the indentation tip using the O<sub>2</sub> plasma cleaner for 1 min. Immediately after plasma cleaning, submerge the indentation tip into a 0.1 wt% F-127 pluronic solution for 10 min. Rinse the indentation tip with a 1x PBS solution before remounting the indentation tip onto the double-leaf cantilever beam.</p></li><li><p>Adjust the height of the capacitance probe to ensure the double-leaf cantilever beam is within the detection range of the capacitance probe. The capacitance probe should be close enough to detect the cantilever beam but far enough away that the deflection of the cantilever beam during the indentation stays within the detection limits of the capacitance probe.</p></li><li><p>Align the indentation tip over the center of the microscope objective. Beginning with the lowest magnitude objective, lower the indentation tip and adjust the objective focal point until the indentation tip is within focus. Using the XY linear translational stage, adjust the location of the indentation tip to align the center of the indentation tip with the center of the microscopeâs field of view. Install the reflectance filter cube into the confocal microscopeâs detector unit and load the reflectance imaging settings in the NIS-Elements software (Figure 2B). Adjust the focal plane of the confocal microscope to find the apex of the indentation tip. Continue to align the center of the indentation tip with the center of the microscopeâs field of view.</p></li><li><p>Repeat the alignment steps using progressively higher objective magnifications until the indentation tip is centered directly above the 20x microscope objective. It is recommended to use the 1.5x optical zoom feature during the indentation. Once the indentation is aligned over the microscope objective, remove the reflectance cube from the confocal microscopeâs detector unit, re-install the 438/24 filter cube, and select the DAPI/FITC/TRITC imaging settings in the NIS-Elements software (Figure 2A). It is recommended at this time to check the indentation tip for any debris that may cause irregular surface contact between the indentation tip and the acini structure.</p></li><li><p>Secure the glass bottom Petri dish on the confocal microscopeâs translational stage. Place the glass bottom Petri dish containing the acini structures onto the Petri dish support plate. Secure the Petri dish to the support plate using the two mounting arms.</p></li><li><p>Determine the location of the glass surface of the Petri dish and zero the indentation instrumentâs force and displacement values. Navigate to a region in the center of the Petri dish in which there are no acini present in the microscopeâs field of view. Initiate the indentation software and zero the force and displacement measurement. Drive the indentation tip 100 Âµm toward the surface of the Petri dish using the piezoelectric z-stage; at this time, the indentation tip should not be in contact with the surface and the force measurements should be fluctuating about 0 N. Manually lower the indentation instrument using the focal adjustment of the condenser stage until the indentation tip comes into contact with the glass surface as evident by a sudden and drastic increase in the measured force along with a decrease in the noise in the force measurement. Once contact with the surface has been made, slowly raise the indentation instrument using the focal adjustment of the condenser stage until the indentation tip is no longer in contact with the surface and the measured force returns to 0 N. Raise the indentation tip 100 Âµm using the piezoelectric z-stage. The indentation tip should now be located ~100 Âµm above the glass surface of the Petri dish â high enough to avoid contact with acini structures when moving the confocal microscopeâs translational stage, but low enough to achieve contact during the indentation measurement.</p></li></ol></li><br/><li><p>Mechanical Indentation of Acini Structure</p><p>Mechanical indentations of glandular acini structures are performed to determine the average applied pressure at which the acini structures mechanically fail. The indentation tip is slowly brought down into contact with an individual acini structure at a fixed indentation rate while the applied force is recorded with simultaneous fluorescence imaging of the cells. As the indentation force exceeds the background level of noise in the measured force, the acini structures are observed to flatten out. Further increasing the applied force results in continued deformation of the acini until there is a structural collapse resulting in the formation of cellular blebs and/or the bursting of cells (Figure 4A). These rupture events are captured in the fluorescence images of the deforming acini enabling us to pinpoint the time at which the first rupture event occurs (Figure 4B). After the initial rupture, the indentation tip continues to push into the acini structure until the measured applied load reaches the specified maximum indentation force. Once the maximum indentation force is achieved, the indentation tip pulls off the structure at the designated indentation rate, returning to the starting z-position. To perform the mechanical indentation of an acini structure:</p><ol class="npwtwok"><li><p>Locate an isolated acini structure and align it in the center of microscopeâs field of view, directly beneath the indentation tip. It is important that the acini structure is isolated within the microscopeâs field of view to avoid contact with other acini structures. It is recommended to use brightfield imaging while locating the acini structures to avoid prolonged exposure of the acini structures to the confocal laser and photobleaching of the fluorescence dye.</p><p>  Misalignment of the acini structure can lead to irregular surface contact between the indentation tip and the acini resulting in improper force measurements as the indentation tip comes into contact with the glass substrate prior to the acini structure rupturing. A sudden, near vertical increase in the force vs displacement curve is a strong indicator that the acini structure is not properly aligned with the indentation tip.</p></li><li><p>Navigate the focal plane of the confocal microscope to the desired imaging plane for the indentation measurement. It is recommended to choose a focal plane that is close to the bottom of the acini structure to fully capture the deformation during the indentation process (Figure 4A); a focal plane chosen too high may result in the acini structure deforming out of the imaging plane prior to the rupture event occurring.</p><p><em>Note: It is optional to perform a confocal z-scan at this time to capture the shape of the acini structure prior to the indentation measurement.</em> </p></li><li><p>Prepare the confocal microscope for the indentation measurement. Initiate perfect focus in the confocal software and adjust the region of interest and the capture delay of the confocal software to achieve the fastest framerate possible. A higher framerate during the indentation measurement will result in an improved temporal resolution when determining the rupture pressure of the acini structure.</p></li><li><p>Prepare the indentation instrument for the indentation measurement. Within the indentation instrument software, set the parameters for the indentation rate and maximum indentation force. We recommend using an indentation rate of 0.1 Âµm/s and a maximum indentation force of 100 ÂµN. Faster indentation rates will reduce the temporal resolution of the fluorescence images captured throughout the measurement. A slower indentation rate will improve the temporal resolution when determining the rupture pressure but will increase the experimental time and laser exposure of the cells. The maximum indentation force should be set sufficiently high enough that the acini structure ruptures before the maximum force is reached. From our experience, a 100 ÂµN maximum indentation load was sufficiently high enough to induce rupture of the acini; however, a higher maximum indentation force will not alter the results of the measurement.</p></li><li><p>Simultaneously initiate the indentation measurement of the acini structure and the imaging time-lapse on the confocal microscope. Record any discrepancies in start times of the indentation measurement and imaging time-lapse.</p></li></ol></li></ol><p class="pptt" id="biaoti39124">### Data analysis</p><p>Once the indentation has been completed, the rupture pressure of the acini structure is determined from the normal contact force measured by the indentation instrument and the contact area measured from the fluorescence images.</p><ol class="npwtwok"><li><p>Determine the time at which the acini structure first ruptures using the time-lapse images captured using the confocal microscope (Figure 4B). Correct the time for any discrepancies between the start times of the indentation measurement and the confocal time-lapse.</p></li><li><p>Measure the contact area of the acini structure at moment in which the rupture event is first observed (Figure 4B). This can be accomplished using either the NIS Elements software or FIJI ImageJ. The contact area can be measured by drawing the best-fit circle around the acini structure at the moment of rupture.</p></li><li><p>Determine the normal contact force experience by the acini structure at the time of rupture. This step can be accomplished using either Microsoft Excel or OriginLabs graphing software. Load the force vs time data into the graphing software and record the indentation force at the previously determined time of rupture.</p></li><li><p>Calculate the rupture pressure of the acini structure from the measured contact area and the measured normal contact force. The average contact pressure for Hertzian contact of a sphere is given as &lt;<em>P</em>&gt; = <em>F/A</em>, where <em>F</em> is the measured normal force from the indentation instrument and <em>A</em> is the measured contact area from the confocal images.</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3847/bioprotoc-10-23-3847-g004.jpg"/><br/><strong>Figure 4. Data analysis from micro-indentation measurements.</strong> A. Schematic illustration of the bio-indentation experiment. The imaging plane is chosen close to the bottom of the acini structure to capture the full deformation during the indentation measurement. B. Fluorescence images captured through confocal microscopy show the deformation of the acini structure throughout the indentation. As the force is increased, the formation of cellular blebs and/or the bursting of cells can be observed between subsequent frames indicating the rupturing of the acini structure. In the presented example, we observe the rupture of the acini at <em>t</em> = 812 s as evident by rupture of a cell and the release of the calcein dye into the surrounding media. The contact area is determined by manually fitting a circle to the acini structure at the instance in rupture is observed. C. The indentation force at the time of rupture is determined from the force measurements of the bio-indentation instrument. (ÂµN: micro-Newton)</p></li></ol><br/><p><strong>Discussion</strong> </p><p>Bio-indentation provides a means to explore both the material properties of biological materials, as well as the collective behavior of cellular systems to externally applied forces. By coupling mechanical indentation with fluorescence images, we directly observe the deformation and rupture of individual glandular acini structures resulting from an externally applied mechanical force. This protocol was developed as part of our exploration into the role the LINC complex plays in the development and maintenance of glandular acini structures (Zhang <em>et al.</em>, 2019). Acini formed from MDCK II cells with disrupted LINC complexes failed at higher applied pressures than those formed with intact LINC complexes. These results appear consistent with the observations of filled lumens in acini structures with disrupted LINC complexes. Furthermore, treating LINC-disrupted acini with Y-27632 resulted in the recovery of the rupture pressure to undisrupted levels.</p><p>  Although the protocol presented here is focused on performing bio-indentation measurements on individual acini structures, similar indentation protocols can be developed for exploring a variety of biological systems. For example, the elastic modulus and permeability of MDCK monolayers, and the minimum pressure necessary to rupture the nuclear envelope of 3T3 fibroblast cells have been investigated using similar micro-indentation protocols (Schulze <em>et al.</em>, 2017; Halfmann <em>et al.</em>, 2019). Furthermore, the effect of shear stresses and friction on biological systems can be studied with the addition of a second load cell to measure lateral forces (UrueÃ±a <em>et al.</em>, 2018). Continued exploration into the response of biological systems to external forces and mechanical queues will be crucial for understanding collective cell behavior and developmental biology.</p><p class="pptt" id="biaoti39125">### Recipes</p><p>Below are the recipes for the various solution used for culturing the MDCK cells and performing the bio-indentation protocol:</p><ol class="npwtwok"><li><p>MDCK cell growth medium</p><p>DMEM medium</p><p>4.5 g/L glucose</p><p>10% v/v DBS</p></li><li><p>MDCK acini indentation medium</p><p>DMEM medium</p><p>4.5 g/L glucose</p><p>10% v/v donor bovine serum</p><p>2 % v/v Matrigel matrix basement membrane</p></li><li><p>Matrigel wash solution</p><p>1x PBS solution</p><p>5 mM EDTA</p><p>1 mM Na<sub>3</sub>VO<sub>4</sub> </p><p>1.5 mM NaF</p></li><li><p>Acini dye medium</p><p>cell growth medium</p><p>Calcein AM (1:2,000)</p></li><li><p>F-127 pluronic solution</p><p>Water</p><p>0.1 wt% F-127 pluronic</p></li></ol><p class="pptt" id="biaoti39126">### Acknowledgments</p><p>The indentation protocols presented here were developed as part of the research work of Zhang <em>et al.</em> (2019). This work was supported by the NIH (grant R01 EB014869) to T.P. Lele, and the NSF (grant DMR-1352043) to T.E. Angelini. The authors would like to acknowledge Dr. Kyle D. Schulze and Dr. W. Gregory Sawyer for their contribution in developing the micro-indentation instrument used in this protocol. The authors would also like to thank Dr. Jennifer Lippencott-Schwart for graciously donating the MDCK II cells.</p><p class="pptt" id="biaoti39127">### Competing interests</p><p>The authors have no financial conflicts of interest to declare.</p><p class="pptt" id="biaoti39128">### References</p><ol class="npwtwok"><li>Ahn, B. M., Kim, J., Ian, L., Rha, K. H. and Kim, H. J. (2010). <a href="https://pubmed.ncbi.nlm.nih.gov/20451976/" target="_blank">Mechanical property characterization of prostate cancer using a minimally motorized indenter in an <em>ex vivo </em>indentation experiment.</a> <em>Urology</em> 76(4): 1007-1011.</li><li>Crisp, M., Liu, Q., Roux, K., Rattner, J., Shanahan, C., Burke, B., Stahl, P. D. and Hodzic, D. (2006). <a href="https://pubmed.ncbi.nlm.nih.gov/16380439/" target="_blank">Coupling of the nucleus and cytoplasm: role of the LINC complex. </a><em>J Cell Biol</em> 172(1): 41-53.</li><li>Debnath, J. and Brugge, J. S. (2005). <a href="https://pubmed.ncbi.nlm.nih.gov/16148884/" target="_blank">Modelling glandular epithelial cancers in three-dimensional cultures.</a> <em>Nat Rev Cancer</em> 5(9): 675-688.</li><li>Halfmann, C. T., Sears, R. M., Katiyar, A., Busselman, B. W., Aman, L. K., Zhang, Q., OâBryan, C. S., Angelini, T. E., Lele, T. P. and Roux, K. J. (2019). <a href="https://rupress.org/jcb/article/218/7/2136/120982/Repair-of-nuclear-ruptures-requires-barrier-to" target="_blank">Repair of nuclear ruptures requires barrier-to-autointegration factor.</a> <em>J Cell Biol</em> 201901116.</li><li>Horn, H. F., Brownstein, Z., Lenz, D. R., Shivatzki, S., Dror, A. A., Dagan-Rosenfeld, O., Friedman, L. M., Roux, K. J., Kozlov, S. and Jeang, K. T. (2013). <a href="https://pubmed.ncbi.nlm.nih.gov/23348741/" target="_blank">The LINC complex is essential for hearing. </a><em>J Clin Invest</em> 123(2): .</li><li>Lei, K., Zhang, X., Ding, X., Guo, X., Chen, M., Zhu, B., Xu, T., Zhuang, Y., Xu, R. and Han, M. (2009). <a href="https://pubmed.ncbi.nlm.nih.gov/19509342/" target="_blank">SUN1 and SUN2 play critical but partially redundant roles in anchoring nuclei in skeletal muscle cells in mice. </a><em>Proc Nati Acad Sci</em> 106(25): 10207-10212.</li><li>Levental, I., Levental, K., Klein, E., Assoian, R., Miller, R., Wells, R. and Janmey, P. (2010). <a href="https://pubmed.ncbi.nlm.nih.gov/21386443/" target="_blank">A simple indentation device for measuring micrometer-scale tissue stiffness.</a> <em>J Phys Condens Matter</em> 22(19): 194120.</li><li>Li, Q., Lee, G. Y., Ong, C. N. and Lim, C. T. (2008). <a href="https://pubmed.ncbi.nlm.nih.gov/18656442/" target="_blank">AFM indentation study of breast cancer cells.</a> <em>Biochem Biophys Res Commun</em> 374(4): 609-613.</li><li>Matsumoto, A., Hieda, M., Yokoyama, Y., Nishioka, Y., Yoshidome, K., Tsujimoto, M. and Matsuura, N. (2015). <a href="https://pubmed.ncbi.nlm.nih.gov/26175118/" target="_blank">Global loss of a nuclear lamina component, lamin A/C, and LINC complex components SUN 1, SUN 2, and nesprin-2 in breast cancer. </a><em>Cancer Med</em> 4(10): 1547-1557.</li><li>Meinke, P., Mattioli, E., Haque, F., Antoku, S., Columbaro, M., Straatman, K. R., Worman, H. J., Gundersen, G. G., Lattanzi, G. and Wehnert, M. (2014). <a href="https://pubmed.ncbi.nlm.nih.gov/25210889/" target="_blank">Muscular dystrophy-associated SUN1 and SUN2 variants disrupt nuclear-cytoskeletal connections and myonuclear organization.</a> <em>PLoS Genetics</em> 10(9): e1004605.</li><li>Mellad, J. A., Warren, D. T. and Shanahan, C. M. (2011). <a href="https://pubmed.ncbi.nlm.nih.gov/21177090/" target="_blank">Nesprins LINC the nucleus and cytoskeleton.</a> <em>Curr Opin Cell Biol</em> 23(1): 47-54.</li><li>Schulze, K., Zehnder, S., UrueÃ±a, J., Bhattacharjee, T., Sawyer, W. and Angelini, T. (2017). <a href="https://pubmed.ncbi.nlm.nih.gov/28173919/" target="_blank">Elastic modulus and hydraulic permeability of MDCK monolayers.</a> <em>J Biomech</em> 53: 210-213.</li><li>Sen, S., Subramanian, S. and Discher, D. E. (2005). <a href="https://pubmed.ncbi.nlm.nih.gov/16113121/" target="_blank">Indentation and adhesive probing of a cell membrane with AFM: theoretical model and experiments.</a> <em>Biophys J</em> 89(5): 3203-3213.</li><li>Stolz, M., Raiteri, R., Daniels, A., VanLandingham, M. R., Baschong, W. and Aebi, U. (2004). <a href="https://pubmed.ncbi.nlm.nih.gov/15111440/" target="_blank">Dynamic elastic modulus of porcine articular cartilage determined at two different levels of tissue organization by indentation-type atomic force microscopy.</a> <em>Biophys J</em> 86(5): 3269-3283.</li><li>Strasser, S., Zink, A., Janko, M., Heckl, W. M. and Thalhammer, S. (2007). <a href="https://pubmed.ncbi.nlm.nih.gov/17210119/" target="_blank">Structural investigations on native collagen type I fibrils using AFM.</a> <em>Biochem Biophys Res Commun</em> 354(1): 27-32.</li><li>Tapley, E. C. and Starr, D. A. (2013). <a href="https://pubmed.ncbi.nlm.nih.gov/23149102/" target="_blank">Connecting the nucleus to the cytoskeleton by SUNâKASH bridges across the nuclear envelope.</a> <em>Curr Opin Cell Biol</em> 25(1): 57-62.</li><li>UrueÃ±a, J. M., Hart, S. M., Hood, D. L., McGhee, E. O., Niemi, S. R., Schulze, K. D., Levings, P. P., Sawyer, W. G. and Pitenis, A. A. (2018). <a href="https://link.springer.com/article/10.1007/s11249-018-1094-y" target="_blank">Considerations for biotribometers: cells, gels, and tissues.</a> <em>Tribology Letters</em> 66(4): 141.</li><li>Vogel, V. and Sheetz, M. (2006). <a href="https://pubmed.ncbi.nlm.nih.gov/16607289/" target="_blank">Local force and geometry sensing regulate cell functions.</a> <em>Nat Rev Mol Cell Biol</em> 7(4): 265-275.</li><li>Zhang, Q., Narayanan, V., Mui, K. L., OâBryan, C. S., Anderson, R. H., Birendra, K., Cabe, J. I., Denis, K. B., Antoku, S. and Roux, K. J. (2019). <a href="https://pubmed.ncbi.nlm.nih.gov/31402305/" target="_blank">Mechanical Stabilization of the Glandular Acinus by Linker of Nucleoskeleton and Cytoskeleton Complex.</a> <em>Curr Biol</em> 29(17): 2826-2839. e2824.</li></ol>