<body>
            <h1>Use of Optogenetic Amyloid-Î² to Monitor Protein Aggregation in <em>Drosophila melanogaster</em>, <em>Danio rerio</em> and <em>Caenorhabditis elegans</em></h1>
            <p><strong>Authors:</strong> Prameet Kaur, Caroline Kibat, Emelyne Teo, Jan Gruber, Ajay Mathuru, Nicholas S. Tolwinski</p>
            <h2>Abstract</h2>
            <p><p>Alzheimer’s Disease (AD) has long been associated with accumulation of extracellular amyloid plaques (Aβ) originating from the Amyloid Precursor Protein. Plaques have, however, been discovered in healthy individuals and not all AD brains show plaques, suggesting that extracellular Aβ aggregates may play a smaller role than anticipated. One limitation to studying Aβ peptide <em>in vivo</em> during disease progression is the inability to induce aggregation in a controlled manner. We developed an optogenetic method to induce Aβ aggregation and tested its biological influence in three model organisms–<em>D. melanogaster, C. elegans</em> and <em>D. rerio</em>. We generated a fluorescently labeled, optogenetic Aβ peptide that oligomerizes rapidly <em>in vivo</em> in the presence of blue light in all organisms. Here, we detail the procedures for expressing this fusion protein in animal models, investigating the effects on the nervous system using time lapse light-sheet microscopy, and performing metabolic assays to measure changes due to intracellular Aβ aggregation. This method, employing optogenetics to study the pathology of AD, allows spatial and temporal control <em>in vivo</em> that cannot be achieved by any other method at present.</p></p>
            <p><strong>Keywords:</strong> Optogenetics, Amyloid-Î², Alzheimerâs Disease, Light-sheet, <em>Drosophila melanogaster</em>, Zebrafish, <em>Caenorhabditis elegans</em></p>
            <h2>Background</h2><p>Alzheimer’s disease (AD) is a debilitating, age-associated, neurodegenerative disease (Zhang <em>et al.</em>, 2011; De-Paula <em>et al.</em>, 2012; Kumar <em>et al.</em>, 2015). The accumulation of amyloid beta (Aβ) as extracellular Aβ plaques is believed to be a major cause of the disease (Hardy and Higgins, 1992), but the failure of interventions that target these Aβ plaques and their presence in brains of non-AD symptomatic individuals (Hawkes, 2016; Cummings <em>et al.</em>, 2018) suggest further mechanistic analysis is required. An alternative hypothesis of neurotoxicity induced by soluble intracellular Aβ aggregates has been proposed (LaFerla <em>et al.</em>, 2007; Ferreira and Klein, 2011), but currently there is a dearth of tools to control Aβ oligomerization <em>in vivo</em> to study the effects of soluble Aβ aggregates on disease progression and the therapeutic potential of drugs. <br/><br/>Optogenetics is the ideal tool for this purpose as it allows highly precise spatial and temporal regulation of proteins <em>in vivo</em> simply by exposure of the organism to light of a specific wavelength (Moglich and Moffat, 2010; Fenno <em>et al.</em>, 2011). We have applied this approach to a variety of problems including embryonic development and homeostasis (Kaur <em>et al.</em>, 2017; Bunnag <em>et al.</em>, 2020), and have recently shown that inducing aggregation of soluble Aβ protein using optogenetics led to metabolic and physical damage and deteriorating lifespan and healthspan in 3 model organisms–<em>Drosophila melanogaster, C. elegans</em> and <em>D. rerio</em>, as well as in standard tissue culture models (Lim and Mathuru, 2017; Lim <em>et al.</em>, 2020). Here, we describe the method used to construct an optogenetic light-inducible Aβ protein that oligomerizes on exposure to blue light <em>in vivo</em> in different model organisms and investigate the downstream detrimental effects using various metabolic assays and time-lapse live imaging. We used the photolyase homology region of the <em>Arabidopsis thaliana</em> cryptochrome 2 as an optogenetic switch (CRY2) protein that oligomerizes into photobodies quickly and reversibly in the presence of blue light (Mas <em>et al.</em>, 2000; Bugaj <em>et al.</em>, 2013). This CRY2 domain was generated as a Aβ-CRY2-mCherry fusion protein to induce aggregation of Aβ protein upon activation by blue light with visualization through the mCherry fluorescent protein.<br/><br/>Incorporation of this versatile tool with easily manipulatable genetic models such as <em>D. melanogaster, C. elegans</em> and <em>D. rerio</em>, gave us temporal control over the induction of soluble Aβ oligomers not feasible with other currently available Alzheimer’s animal models. We were also able to dissect the pathological effects of Aβ oligomers on embryos during development in real-time using live imaging as well as in adult flies (Lim <em>et al.</em>, 2020). This model also allows us to generate mosaics of embryos such that only one half of the embryo, or a small portion is exposed to blue light. Hence, we can investigate the effects of targeted light-induced Aβ oligomerization on various anatomical structures such as the nervous system with the unlit portion of the embryo acting as the control. Functionality in both the invertebrate and vertebrate lab models demonstrates this technique’s application to study the biochemistry, metabolism, cellular and neuronal biology of induced Aβ oligomerization. The effects of potential drug interventions on lifespan, health span and metabolism can also be studied with this robust optogenetic approach in various model organisms. We were able to adopt different strategies to express the same transgene in both invertebrates and vertebrates, which should translate to mouse studies through the conditional knock-in approach. Aggregation could be induced in brains of adult mice with exposure to blue light to study the effects of Aβ-CRY2-mCh aggregation on neuronal function and behavior. This protocol is specific to Aβ, but could easily be adapted to other aggregating proteins simply by changing the starting gene.</p>
        </body>

<p class="pptt" id="biaoti39131">## Materials and Reagents</p><ol class="npwone"><li><p>Molecular cloning of optogenetic transgenes</p><ol class="npwtwok"><li><p>10 cm Petri dishes (Greiner Bio One International, catalog number: <a class="blackNoUnder" target="_blank"> 633185 </a> )</p></li><li><p>14 ml tubes PP (Bio Lab, catalog number: <a class="blackNoUnder" target="_blank"> 352059 </a> )</p></li><li><p>1.5 ml Micro tubes (Axygen, catalog number: <a class="blackNoUnder" target="_blank"> MCT-150-L-C </a> )</p></li><li><p>Pipette tips (fitting for 100-1,000 Î¼l, 20-200 Î¼l, 2-50 Î¼l and 0.5-10 Î¼l pipettes)</p></li><li><p>Microloader (Eppendorf)</p></li><li><p>pUCIDT-attL1-Human ABeta-attR5 (Addgene: 160436)</p><p><em>Note: The AÎ² human gene was synthesized with attL1 and attR5 overhangs into a Kanamycin resistant plasmid backbone for use as a Gateway entry vector.</em> </p></li><li><p>pUCIDT-attL1-Worm ABeta-attR5 (Addgene: 160437)</p><p><em>Note: The AÎ² worm gene was synthesized with attL1 and attR5 overhangs into a Kanamycin resistant plasmid backbone for use as a Gateway entry vector.</em> </p></li><li><p>pDONR-attL5-CRY2-mCh-attL2 (Addgene: 160438)</p><p><em>Note: CRY2-mCh was amplified from pCRY2PHR-mCherryN1 (which was a gift from Chandra Tucker (Addgene plasmid</em>, <em>catalog number:</em> <em>26866) (Kennedy et al., 2010).</em> <em>att</em><em>B5 and</em> <em>att</em><em>B2 sites were added to the CRY2-mCh PCR product and recombined into pDONR P5-P2 vector. The final pDONR-attL5-CRY2-mCh-attL2 plasmid was used as a Gateway entry vector.</em> </p></li><li><p>pUASg-HA-attB (<em>Drosophila</em> Genomics Resource Center, stock number: 1423)</p><p><em>Note: Vector specific sequencing primers used were HSP</em> </p><p><em>Forward: TATAAATAGAGGCGCTTCGT</em> </p><p><em>HA Reverse: AGCGTAATCTGGAACGTCATA</em> </p></li><li><p>pDEST-hsp16-2p (a kind gift from Hidehito Kuroyanagi, Medical Research Institute, Tokyo Medical and Dental University)</p><p><em>Note: Vector specific sequencing primer used was Hsp16-2p: CGAATGTGAGTCGCCCTCCT</em> </p></li><li><p>pDEST-Tol2-PA2-CMV (<a href="http://tol2kit.genetics.utah.edu/index.php/List_of_entry_and_destination_vectors">Tol2 kit</a>). Used in molecular cloning of following transgenes for <em>Danio rerio</em> â a) pDEST-Tol2-PA2-10xUAS-AÎ²-CRY2-mCh; b) pDEST-Tol2-PA2-Ubi-AÎ²-CRY2-mCh; c) pDEST-Tol2-PA2-CMV-AÎ²-mCh (Addgene ID: 160435)</p></li><li><p>Top 10 competent cells (Thermo Fisher <a class="blackNoUnder" target="_blank"> Scientific </a> , catalog number: <a class="blackNoUnder" target="_blank"> C404003 </a> )</p></li><li><p>Agarose, Biotechnology Grade, 500 g (1st Base, catalog number: <a class="blackNoUnder" target="_blank"> BIO-1000-500g </a> )</p></li><li><p>LB Agar Miller, Bacterial Culture media, 500 g (1st Base, catalog number: <a class="blackNoUnder" target="_blank"> BIO-4010-500g </a> )</p></li><li><p>LB Broth Miller, pH 7.0, Bacterial Culture media, 500 g (1st Base, catalog number: <a class="blackNoUnder" target="_blank"> BIO-4000-500g </a> )</p></li><li><p>Ampicillin sodium salt (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> A9518G </a> )</p></li><li><p>Kanamycin sulfate (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> K1377 </a> )</p></li><li><p>MultiSite Gateway<sup>Â®</sup> Pro Plus kit (catalog number: <a class="blackNoUnder" target="_blank"> 12537100 </a> )</p></li><li><p>Gateway<sup>TM</sup> LR ClonaseTM II Enzyme mix (Thermo Fisher <a class="blackNoUnder" target="_blank"> Scientific </a> , catalog number: <a class="blackNoUnder" target="_blank"> 11791100 </a> )</p></li><li><p>Gateway<sup>TM</sup> BP ClonaseTM Enzyme mix (Thermo Fisher <a class="blackNoUnder" target="_blank"> Scientific </a> , catalog number: <a class="blackNoUnder" target="_blank"> 11789020 </a> )</p></li><li><p>Phusion High-Fidelity PCR kit (Thermo Fisher <a class="blackNoUnder" target="_blank"> Scientific </a> , catalog number: <a class="blackNoUnder" target="_blank"> F553S </a> )</p></li><li><p>Gibson Assembly<sup>Â®</sup> Cloning kit (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> E5510S </a> )</p></li><li><p>Low melting point agarose (Promega, catalog number: <a class="blackNoUnder" target="_blank"> V2111 </a> )</p></li></ol></li><br/><li><p>Expression of optogenetic transgenes in neurons <em>in vivo</em> (for <em>D. melanogaster)</em> </p><ol class="npwtwok"><li><p>1 L microwave-safe glass beaker</p></li><li><p>Magnetic stirrer</p></li><li><p>50 ml glass beaker</p></li><li><p>Spatula</p></li><li><p>Compressed carbon dioxide gas cylinder (any brand)</p></li><li><p>LED lamp (any brand)</p></li><li><p>Aluminum foil (any brand)</p></li><li><p>Styrofoam box (any brand)</p></li><li><p>Fly cages for 60 mm Petri dishes (any, <em>e.g.</em>, Small Embryo Collection Cage For 60 mm Petri Dishes, FlyStuff, catalog number: <a class="blackNoUnder" target="_blank"> 59-100 </a> )</p></li><li><p>60 mm Petri dishes (any brand, <em>e.g.</em>, Nunc<sup>TM</sup> EasYDish<sup>TM</sup> Dishes, ThermoFisher <a class="blackNoUnder" target="_blank"> Scientific </a> , catalog number: <a class="blackNoUnder" target="_blank"> 150462 </a> )</p></li><li><p><em>Drosophila</em> sorting brush (any brand) or a very thin paint brush</p></li><li><p><em>Drosophila</em> [Bloomington <em>Drosophila</em> Stock Center (NIH P40OD018537)]. Stock number 458, <em>elav</em> promoter driving GAL4 for the GAL4/UAS system (Brand and Perrimon, 1993)</p></li><li><p>P(ET-QF2.GU)repo [Q system from Bloomington <em>Drosophila</em> Stock Center (NIH P40OD018537), stock number: 66477] (Lin and Potter, 2016)</p></li><li><p>P(QUAS-mCD8-GFP)X (Q system from Bloomington <em>Drosophila</em> Stock Center [NIH P40OD018537), stock number: 30001) (Lin and Potter, 2016)</p></li><li><p>Bacto agar</p></li><li><p>Glucose</p></li><li><p>Cornmeal</p></li><li><p>Brewerâs yeast</p></li><li><p>Nipagin</p></li><li><p>FlyStuff Grape Agar Premix, For Embryo Collection (25 Packets/Unit) (FlyStuff, catalog number: <a class="blackNoUnder" target="_blank"> GEN47-102 </a> )</p></li><li><p>Yeast paste (Bruggeman Instant Dry Yeast Blue, 500g)</p></li><li><p>Standard fly food (see Recipes)</p></li><li><p>Grape juice agar plates (see Recipes)</p></li><li><p>Yeast paste (see Recipes)</p></li></ol></li><br/><li><p>Crosses and expression of constructs (for <em>D. rerio</em>)</p><ol class="npwtwok"><li><p>3 ml Pasteur Pipette (Tarson, catalog number: <a class="blackNoUnder" target="_blank"> T940050 </a> )</p></li><li><p><em>Tg gng8:Gal4</em> (with thanks to Prof. Marnie Halpern) (Hong <em>et al.</em>, 2013)</p></li><li><p><em>Tg UAS:GCamp6S</em> in nacre background (mitfa<sup>-/-</sup>)</p></li><li><p>Tricaine (MS-222 or Ethyl 3-aminobenzoate methanesulfonate; Sigma, catalog number: <a class="blackNoUnder" target="_blank"> A5040 </a> )</p></li><li><p>NaCl</p></li><li><p>KCl</p></li><li><p>CaCl<sub>2</sub>Â·2H<sub>2</sub>O</p></li><li><p>MgSO<sub>4</sub>Â·7H<sub>2</sub>O</p></li><li><p>Tris-HCl</p></li><li><p>25x Tricaine stock solution (for <em>D. rerio</em>)</p></li><li><p>60x E3 stock solution (see Recipes)</p></li></ol></li><br/><li><p>Microinjection and expression of UAS construct (for <em>D. rerio</em>)</p><ol class="npwtwok"><li><p>Standard wall borosilicate glass with filament (SUTTER INSTRUMENT, model: O.D 1 mm, I.D 0.5 mm, Length 7.5 cm, catalog number: <a class="blackNoUnder" target="_blank"> BF100-5-7.5 </a> )</p></li></ol></li><br/><li><p>Light-sheet microscopy to image neural development during embryogenesis (for <em>D. melanogaster)</em> </p><ol class="npwtwok"><li><p>Disposable dropper (any brand)</p></li><li><p>15 ml Falcon tubes (any brand)</p></li><li><p>Stainless steel dissecting needle, L160 mm (Hammacher, HWO010-16, catalog number: <a class="blackNoUnder" target="_blank"> 91-2483 </a> )</p></li><li><p>Glass capillary, size 2 black, inner diameter of capillary ~1 mm (BRAND GmbH, catalog number: <a class="blackNoUnder" target="_blank"> 701932 </a> ) with corresponding plunger</p></li><li><p>5 cm long electrical copper wire (or any other fine tip wire)</p></li><li><p>Weighing boat</p></li><li><p>Halocarbon oil 27 (Chemical Raw Materials Ltd, catalog number: <a class="blackNoUnder" target="_blank"> 3904 9000 </a> )</p></li><li><p>Bleach (Clorox)</p></li><li><p>Autoclaved MilliQ water</p></li><li><p>1% Agarose, low gelling temperature Type VII-A (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> A0701 </a> ) dissolved in water aliquoted into 1.5 ml Eppendorf tubes (see Recipes)</p></li></ol></li><br/><li><p>Light-sheet microscopy to image AÎ²-CRY2-mCh in neurons in fish (in <em>D. rerio</em>)</p><ol class="npwtwok"><li><p>Heating block for mounting medium</p></li></ol></li><br/><li><p><em>D. rerio</em> Mitochondrial metabolic flux assay</p><ol class="npwtwok"><li><p>Oligomycin (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 75351 </a> )</p></li><li><p>Carbonyl cyanide 4-(trifluoromethoxy) phenylhydrazone (FCCP) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C2920 </a> )</p></li><li><p>Sodium azide (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C2002 </a> )</p></li><li><p>Dimethyl Sulfoxide (DMSO) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> D2650 </a> )</p></li></ol></li><br/><li><p><em>D. rerio</em> Adenosine triphosphate (ATP) assay</p><ol class="npwtwok"><li><p>Firefly Lantern Extract (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> FLE250 </a> )</p></li><li><p>MgSO<sub>4</sub>Â·7H<sub>2</sub>O (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 230391 </a> )</p></li><li><p>KH<sub>2</sub>PO<sub>4 </sub>(Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P0662 </a> )</p></li><li><p>Na<sub>2</sub>HASO<sub>4</sub>Â·7H<sub>2</sub>O (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S9663 </a> )</p></li><li><p>TCA powder (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> T6399 </a> )</p></li><li><p>10% Trichloroacetic acid (TCA) solution (see Recipes)</p></li><li><p>Arsenite ATP buffer (see Recipes)</p></li><li><p>25x Tricaine stock solution (for <em>D. rerio</em>) (see Recipes)</p></li></ol></li><br/><li><p>Fluorescence microscopy to image AÎ² aggregates in <em>C. elegans</em> </p><ol class="npwtwok"><li><p>1.5 ml Eppendorf tube (any brand)</p></li><li><p>Levamisole hydrochloride (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 196142 </a> )</p></li><li><p>2% Agarose (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> A0701 </a> ) dissolved in water aliquoted into 1.5 ml Eppendorf tubes (see Recipes)</p></li><li><p>2% low melting agarose (see Recipes)</p></li></ol></li></ol><p class="pptt" id="biaoti39132">### Equipment</p><ol class="npwone"><li><p>Expression of optogenetic transgenes in neurons <em>in vivo</em> (for <em>D. melanogaster)</em> </p><ol class="npwtwok"><li><p><em>Drosophila</em> workstation with CO<sub>2</sub> pad (Flowbuddy Complete; Genesee <a class="blackNoUnder" target="_blank"> Scientific </a> , catalog number: <a class="blackNoUnder" target="_blank"> 59-122WC </a> )</p></li><li><p>Incubator for 25 Â°C incubation of flies (any brand, <em>e.g.</em>, INVICTUS <em>Drosophila</em> Incubator, Genesee <a class="blackNoUnder" target="_blank"> Scientific </a> )</p></li><li><p>Stereo microscope (any brand, <em>e.g.</em>, Leica, model: <a class="blackNoUnder" target="_blank"> M50 </a> )</p></li><li><p>Fluorescence stereo microscope (any brand, <em>e.g.</em>, Leica <a class="blackNoUnder" target="_blank"> MZ16F </a> )</p></li></ol></li><br/><li><p>Light-sheet microscopy to image neural development during embryogenesis (for <em>D. melanogaster</em>)</p><ol class="npwtwok"><li><p>Dissecting fine-pointed serrated forceps (any brand)</p></li><li><p>Heat block with temperature range 37 Â°C and 70 Â°C (any brand, <em>e.g.</em>, Digital block heater, Thermo Fisher <a class="blackNoUnder" target="_blank"> Scientific </a> )</p></li><li><p>Zeiss Light-sheet Z.1 (Carl Zeiss)</p></li><li><p>Sample chamber</p></li></ol></li><br/><li><p>For experiments with <em>D. rerio</em> </p><ol class="npwtwok"><li><p>Incubator for bacterial culture, 37 Â°C</p></li><li><p>Shaker incubator for bacterial culture, 37 Â°C</p></li><li><p>Light-sheet microscopy to image AÎ² aggregation in imaging zebrafish</p></li><li><p>Seahorse extracellular flux analyzer (Agilent Technologies, Seahorse Bioscience, model: <a class="blackNoUnder" target="_blank"> XFe96 </a> )</p></li><li><p>Cytation 3 Cell Imaging Multi-Mode Reader (BioTek Instruments)</p></li><li><p>Micropestle homogenizer</p></li></ol></li><br/><li><p>Fluorescence microscopy to image AÎ² aggregates in <em>C. elegans</em> </p><ol class="npwtwok"><li><p>Incubator for zebrafish embryos, 28 Â°C</p></li><li><p>Forceps (F.S.T, model: Student Fine Forceps, Straight, catalog number: <a class="blackNoUnder" target="_blank"> 91115-10 </a> )</p></li><li><p>Micropipette puller (SUTTER INSTRUMENT, model: <a class="blackNoUnder" target="_blank"> P-1000 </a> )</p></li><li><p>Microinjection system (Harvard Apparatus, model: <a class="blackNoUnder" target="_blank"> PLI-90 Pico-injector </a> )</p></li><li><p>Confocal microscope (Zeiss, model: <a class="blackNoUnder" target="_blank"> LSM800 </a> )</p></li></ol></li><br/><li><p>Other assays used to confirm aggregation-induced deficits in <em>C. elegans</em> </p><ol class="npwtwok"><li><p>Incubator (Aqualytic, <a class="blackNoUnder" target="_blank"> TC 255 S </a> )</p></li><li><p>Seahorse extracellular flux analyzer (Agilent Technologies, Seahorse Bioscience, model: <a class="blackNoUnder" target="_blank"> XFe96 </a> )</p></li><li><p>Cytation 3 Cell Imaging Multi-Mode Reader (BioTek Instruments)</p></li></ol></li></ol><p class="pptt" id="biaoti39133">### Software</p><p>Light-sheet microscopy to image neural development during embryogenesis</p><ol class="npwtwok"><li><p>ZEN 2014 Processing software package for Light-sheet Z.1 (Carl Zeiss, catalog number: 410136-1059-130)</p></li><li><p>MetaMorph<sup>Â®</sup> Microscopy Automation and Image Analysis Software (Molecular Devices)</p></li></ol><p class="pptt" id="biaoti39134">### Procedure</p><ol class="npwone"><li><p>Molecular cloning of optogenetic transgenes for expression in <em>D. melanogaster</em> and <em>C. elegans</em> </p><p>Use the human AÎ²1-42 amino acid sequence (AÎ² Human) to generate transgenes for <em>Drosophila</em> and zebrafish. For <em>C. elegans</em>, use a nematode-codon-optimized version of AÎ²1-42 (AÎ² Worm) (Fong <em>et al.</em>, 2016). For <em>Drosophila</em> expression of human AÎ²-CRY2-mCh, obtain the human AÎ² DNA sequence with attL1 and attR5 overhangs in a Gateway entry vector (pUCIDT-attL1-Human ABeta-attR5) and the pDONR-attL5-CRY2-mCh-attL2. Perform a multi-site Gateway LR reaction to insert these constructs into the pUASg-HA-attB <em>Drosophila</em> expression vector (Bischof <em>et al.</em>, 2007), according to the manufacturerâs instructions (Invitrogen, ThermoFisher <a class="blackNoUnder" target="_blank"> Scientific </a> ). For <em>C. elegans</em> expression of worm AÎ²-CRY2-mCh, perform a multi-site Gateway LR reaction between the pUCIDT-attL1-Worm ABeta-attR5 entry vector, the pDONR-attL5-CRY2-mCh-attL2 and the pDEST-hsp-16â2p <em>C. elegans</em> expression vector, as per the manufacturerâs instructions (Invitrogen, ThermoFisher <a class="blackNoUnder" target="_blank"> Scientific </a> ). For all transgenes generated, use vector specific primers for sequencing to confirm the orientation and insertion of both fragments into the destination vector.</p></li><br/><li><p>Molecular cloning of optogenetic transgenes for <em>Danio rerio</em> (zebrafish) â a) pDEST-Tol2-PA2-10xUAS-AÎ²-CRY2-mCh; b) pDEST-Tol2-PA2-Ubi-AÎ²-CRY2-mCh; c) pDEST-Tol2-PA2-CMV-AÎ²-mCh</p><ol class="npwtwok"><strong>Cloning single DNA element of interest into gateway compatible vector</strong> <li><p>PCR amplify 10xUAS, Ubi and CMV sequence with specific primer flanked by attB4 (Forward primer) and attB1 (Reverse primer) and cloned into pDONRTMP4-P1R to generate 5' entry plasmid via BP recombination. Verify correct clones by sequencing primers M13F or 5' GTAAAACGACGGCCAGT3' and M13Rev 5' CAGGAAACAGCTATGAC3'.</p></li><li><p>PCR amplify the AÎ²-CRY2-mCh with specific primer flanked attB1 (Forward primer) and attB2 (Reverse primer) and cloned into pDONR221TM to generate middle entry plasmid via BP recombination. Verify correct clones by M13 primers (same as 1).</p><p><em>Note: The 3' entry vector containing poly-A signal (p3' PolyA) and the Destination plasmid containing the Tol2 recognition sites (pDestTol2PA2) originated in the Kawakami Laboratory and are available asâTol2 kit (Kawakami, 2007).</em> </p></li><li><p>Create compatible attL sites on each entry plasmids following BP recombination. LR recombination to generate expression plasmid. Verify correct clones by sequencing primers</p><p>Forward: 5' CAAGTTTGTACAAAAAAGCAGGCTTA3'</p><p>Reverse: 5' TACCCAGCTTTCTTGTACAAAGTGGGGGA 3'.</p></li><li><p>Summary of the fragment assembly is shown in Figure 1.</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3856/bioprotoc-10-23-3856-g001.jpg"/><br/><strong>Figure 1. Generation of expression clones for zebrafish transgenic</strong> </p></li></ol></li><br/><li><p>Expression of optogenetic transgenes in neurons <em>in vivo</em> </p><p><strong>For <em>D. melanogaster</em></strong>, inject the pUASg-humanAÎ²-CRY2-mCh-HA-attB transgene into attP2 (Strain#8622) P[CaryP]attP268A4 (BestGene Inc (California)) (Groth <em>et al.</em>, 2004; Markstein <em>et al.</em>, 2008). Transgenic flies will be provided upon request. Isolate at least 5 homozygous male UAS-AÎ²-CRY2-mCh flies and cross these with at least 10 Elav-Gal4, Repo-QF2; QUAS-GFP female virgin flies to drive neuronal expression of AÎ²-CRY2-mCh and glial expression of GFP in the F1 generation. Place these male and female flies in a fly cage and cover with a pre-warmed grape juice agar plate smeared with a small amount of yeast paste to encourage egg laying. Keep the cages in a 25 Â°C incubator. For control flies, use UAS-TdTomato homozygous males instead. Change the fly plates every day and replace with pre-warmed grape juice agar plates that have a small amount of yeast paste on them. Allow newly set up cages to lay eggs for about 3 days to ensure a good number of developing embryos before using them for experiments.</p><p><strong>  For <em>C. elegans</em></strong>, obtain transgenic strain (gnaEx1[myo-2p::YFP+hsp-16-2p::Abeta1-42:: CRY2::mCherry]) from Caenorhabditis Genetics Center (CGC). Age-synchronize transgenic nematodes through hypochlorite bleaching (Stiernagle, 2006). To induce <em>in vivo</em> expression of AÎ²-CRY2-mCh driven by the hsp16-2 promoter, incubate day 1 adult nematodes at 35 Â°C for 1 h. After 1 h incubation, separate the nematodes into light/dark groups and grow them in a 20 Â°C incubator. For dark treatment, wrap the plates containing nematodes with aluminum foil and place the wrapped plates inside a Styrofoam box. Place the Styrofoam box at the top of incubator rack to avoid exposure to the LED light. For light treatment, place the plates containing nematodes directly under the LED light on the bottom rack of the incubator.</p><p><strong>  For <em>D. rerio</em>:</strong> Zebrafish lines (with Gal4; UAS) express the transgene in specific neurons or cells. To prevent oligomerization in ubiquitously expressing larvae, raise injected embryos in a dark incubator until experimentation. Minimize light exposure for transferring plates out or for sorting to avoid exposure to high LED light. For light treatment for mitochondrial flux analysis or ATP measurements, place the larvae under white LED light of a stereoscope for 2 h. Larvae with more restricted expression such as those used in light-sheet imaging were less sensitive to light exposure but were still raised in the dark till imaging as described in the following sections.</p><br/><p><strong>Microinjection and expression of UAS construct (preparation on Day -1)</strong> </p><ol class="npwtwok"><li><p>The day before the injection, set up the fish in breeding tanks with dividers (Video 1).</p><br/><p><!--video3856-1å¼å§--></p><div class="videodiv" id="vd3856-1" style="height:480px;"></div><div class="videophoto" id="vp3856-1"><a class="newapic" href="javascript:;" onclick="openvideo(1)" style="background:url('https://en-cdn.bio-protocol.org/video/3856/3856-1s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/></a> </div><strong>Video 1. </strong><strong>Zebrafish microinjection</strong> <p><br/></p></li><li><p>Using a strainer, collect the embryos and rinse with tap water before transferring them into a 10 cm Petri dishes containing E3 (1x). Remove unfertilized eggs and debris with a Pasteur pipette.</p></li><li><p>Prepare the injection needle by micropipette puller and backfilled with 3 Î¼l of the mixture listed in the table (Table 1) using a Microloader.</p></li><li><p>Fix the needle onto the micromanipulator connected to the microinjection system.</p></li><li><p>Make a microinjection plate with 3% agarose using a custom-built plastic mold with grooves.</p><br/><p><strong>Table 1.</strong><strong>Microinjections mixture for zebrafish embryos<br/><img alt="" class="layerphoto" height="108" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201202/20201202014442_6402.jpg" width="584"/><br/></strong> </p></li></ol><br/><p><strong>Microinjection and expression of UAS construct (Day 0)</strong> </p><ol class="npwtwok"><li><p>The following morning (Day 0), remove the divider and allow approximately 15-20 min of undisturbed mating time (Video 1).</p></li><li><p>Use a Pasteur pipette to align the embryos along the trenches and remove excess water to prevent embryos from moving during injection.</p></li><li><p>Inject one-cell stage embryos with ~5 pL of DNA + RNA mixture under a stereoscope equipped with the injection apparatus. The apparatus consists of a needle holder attached to an MHC model magnetic stand and connected to <a class="blackNoUnder" target="_blank"> PLI-90 Pico-injector </a> , a foot switch to pulse the injection solution into the embryos and assisted by a backpressure unit to gauge the pulse release of pressurized nitrogen. Aim to inject around 200 to 300 embryos in 1 hour.</p></li><li><p>Immerse injected embryos in E3 (1x). Save and raise 20 to 30 non- injected embryos alongside the injected embryos to assess microinjection induced damage. Use standard zebrafish embryos bleaching protocol (<a href="https://zfin.org/zf_info/zfbook/chapt1/1.5.html">https://zfin.org/zf_info/zfbook/chapt1/1.5.html</a>) and split them into 2 groups after bleaching.</p></li><li><p>Incubate all the embryos at 28 Â°C in the dark. Use 1 group of the injected embryos for light exposure and the other as controls (dark).</p></li></ol></li><br/><li><p>Fluorescence microscopy to detect mCherry expression in <em>D. rerio</em> </p><ol class="npwtwok"><li><p>Sort injected embryos under fluorescence stereomicroscope at 24 h post fertilization (hpf) (Day 1) for mCherry expression. Clear dead embryos from injected and uninjected dishes and change the water. Expect 5-10% injected embyos to die by 24 hpf.</p></li><li><p>Replenish the embryos with fresh E3 (1x) daily until they reached 5 days post fertilization (dpf). Send larval zebrafish with mCherry expression to be grown in separate tanks in the hatchery from the non-expression larvae by 7 dpf.</p></li><li><p>Cover the lid of the container used to grow baby fish with aluminum foil to prevent direct light exposure. Covering can be removed when fish reach adulthood depending on the expression tissue (for instance, sparse neuronal expression)</p></li><li><p>To generate a permanent line, genotype adult fish by fin clipping at 8 weeks post fertilization (wpf). Use primers specific for AÎ²-CRY2 and Gal4 for genotyping. Cross fish which are positive for both AÎ²-CRY2 and Gal4 to obtain F1 embryos. Screen embryos for mCherry expression from 4 dpf onwards and follow the steps above. Tol2 transgenesis usually yields more than 50% transgene. However, due to the properties of the <em>Abeta-</em>CRY2, we found only 10% of the injected fish carries both AÎ²-CRY2 and Gal4 at the point of genotyping. And only 1-2% showed germline transmission.</p><p><em>Note: It is possible to find fish which are positive for both Gal4 and Abeta-CRY2 in genotyping but do not express mCherry. This could be due to the silencing of the UAS- regulated effector/reporter genes by methylation at the CpG nucleotides (Akitake et al., 2011)</em> or may also due to other unknown factors such as the genomic location of the insertion.</p></li></ol></li><br/><li><p>Light-sheet microscopy to image neural development during embryogenesis (in <em>D. melanogaster</em>)</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3856/bioprotoc-10-23-3856-g002.jpg"/><br/><strong>Figure 2. Bleaching and mounting of <em>Drosophila</em> embryo into glass capillary for long term imaging.</strong> A. Place a stage 10 embryo in the middle of a piece of C-fold towel and add bleach till soaked. Rotate the embryo to encourage dechorionation. After bleaching, move the C-fold with the embryo on it to a drying C-fold towel using forceps (i). Rinse the embryo with water by transferring the piece of C-fold to a drop of water, let it dry once again on the drying C-fold (ii), and rinse once more in another drop of water before leaving it to dry on the drying C-fold towel (iii). B. Place a capillary filled with melted agarose and hold it in place gently on top of an inverted petri dish. Pick the bleached embryo from the C-fold and place it on the surface of the agarose. C. Push the embryo gently into the agarose using the fine tip of a copper wire and position it parallel to the capillary. D. After allowing the agarose to solidify for 5 min, load the capillary into the sample holder, and lower it into the sample chamber of the Light-sheet Z.1. Eject the agarose slowly such that the embryo just emerges from the capillary. Position the embryo in front of the objective such that the ventral surface (v) faces the objective. d: dorsal. These steps are shown in Video 2.</p><p><br/></p><p><!--video3856-2å¼å§--></p><div class="videodiv" id="vd3856-2" style="height:480px;"></div><div class="videophoto" id="vp3856-2"><a class="newapic" href="javascript:;" onclick="openvideo(2)" style="background:url('https://en-cdn.bio-protocol.org/video/3856/3856-2s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/></a> </div><strong>Video 2. </strong><strong>Mounting and imaging of <em>Drosophila</em> embryos</strong> <p><br/></p><ol class="npwtwok"><li><p>Melt a tube of 1% low gelling temperature agarose at 70 Â°C for half an hour in a heat block and then cool it to 37 Â°C, mix thoroughly by inverting before proceeding with mounting.</p></li><li><p>Remove the grape agar juice plate from a cup that has been laying eggs for at least 10 h to ensure you obtain a good amount of embryos at stage 10 (Hartenstein, 1993). Pour approximately 1 ml of Halocarbon oil 27 onto the plate with embryos and swirl the oil in a circular manner to spread it out evenly and form a thin layer of oil on the plate. Let it rest for 1 min to allow the oil to penetrate the chorion of all the embryos.</p><p><em>Note: It is best to use plates that have flies laying eggs for 24 h so that proper development and expression of the GFP and mCherry fluorescent proteins in older embryos can be observed before proceeding with mounting and imaging of stage 10 embryos which do not express high levels of these fluorescent tags.</em> </p></li><li><p>Observe the plate under a fluorescence microscope using transmitted light.</p></li><li><p>After germ band retraction (stages 14-17 according to [Campos-Ortega and Hartenstein, 1997]), embryos should show a very bright GFP and mCherry signal in the nervous system that mark the glial cells and neurons respectively using reflected light.</p></li><li><p>Pick 1 stage 10 embryo from the plate using a dissecting needle and transfer it to the center of 1 cm x 1 cm piece of C-fold in a Petri dish under the transmitted light (Figure 2A).</p></li><li><p>Add a drop of bleach to just soak the piece of C-fold. Add 2 drops of autoclaved water in the same Petri dish as shown in Figure 2A. Use the dissecting needle to roll the embryo up and down on the C-fold and encourage the chorion to peel off. Within 30 s to a minute, the chorion can be seen peeling off under the transmitted light at high magnification with the dorsal appendages disappearing and the matt embryo surface will take on a glossy finish.</p></li><li><p>As soon the chorion is removed, use dissecting forceps to carefully lift the C-fold from one corner with the dechorionated embryo on it to a dry big piece of C-fold to remove as much bleach as possible (Figure 2A(i)). Rinse the embryo with water by moving the C-fold to the first drop of water, allowing it to soak for 30 s, then move it to another region on the drying C-fold (Figure 2A(ii)). Repeat the rinse with water one last time and allow the C-fold to dry as much as possible by allowing it to rest on an un-used part of the drying C-fold for a minute (Figure 2A(iii)).</p></li><li><p>Meanwhile, insert the plunger into the size 2 glass capillary and suck up the melted 1% low-melting agarose Type VII-A (Sigma) at 37 Â°C to just above the black line as shown in Figure 2B. Position the capillary on top of an inverted Petri dish and allow the agarose to cool in the capillary for a minute.</p></li><li><p>Pick up the dried embryo from the C-fold using a dissecting needle and position it in the center of the capillary (Figure 2B).</p></li><li><p>Push the embryo into the center of the solidifying agarose using a fine copper wire to a depth of approximately 0.5 cm and maneuver it such that it is parallel to the capillary (Figure 2C). Remove the copper wire very slowly from the agarose once the embryo is in the desired position. Wait for 5 min to allow the agarose to solidify.</p><p><em>Note: Align the embryo in the center of the capillary to ensure it is held securely in the agarose. If it sticks to the glass capillary, push it back into the agarose to avoid slipping out during imaging. This step requires a lot of patience and practice. This was the best way to position the embryo to obtain the highest resolution images. The aliquot of 1% low-melt agarose can be left at 37</em> Â°C <em>for use over a long time or cooled and melted when needed several times. However, it will eventually stop solidifying after an extended period of time at 37</em> Â°C <em>or after numerous heat-cool cycles. This requires the aliquot in use to be discarded and a new aliquot used thereafter.</em> </p></li><li><p>Switch on the Light-sheet Z.1 system and start up the Zen 2014 SP software. Inject autoclaved water into the sample chamber such that it filled to the brim as shown in the manufacturerâs protocol.</p></li><li><p>After the agarose in the capillary has solidified, load the capillary into the sample holder according to the manufacturerâs manual and secure it by tightening the screw. Load the sample holder into the Light-sheet Z.1 and lower it into the sample chamber filled with autoclaved water. Push the embryo out of the capillary slowly such that it is just outside capillary and aligned with the objective (Figure 2D).</p></li><li><p>Use the Light-sheet Z.1 10x/0.2 Illumination Optics to rotate the capillary such that the ventral surface (v) of the sample faces the objective to obtain high resolution images of the ventral surface (Figure 2D). Acquire images using a Light-sheet Z.1 Illumination objective 40x/1.0 (water immersion). Set up the 20 mW 561 nm laser at 13% laser power with 12.5 ms exposure time and the 30 mW 488 nm at 10% power with 29.95 ms exposure time to activate CRY2 clustering. Set z-stacks at 1 Î¼m intervals and denote the top of the z-stack as approximately 20 sections (20 Î¼m) above the top of the embryo and the bottom of the z-stack as approximately 20 sections (20 Î¼m) from the bottom of the embryo. This is to ensure data is not lost in case the embryo shifts during imaging. Excess z-stack sections can be removed during processing. Set-up a time series acquisition with dual-side illumination in both channels every 2.5 min for 500 cycles. For embryos not exposed to blue light, use only the 561 nm laser. Allow the image acquisition to run for approximately 22 h.</p></li><li><p>After the imaging is completed, pull the plunger up to insert the embryo back into the capillary and remove the sample holder from the Light-sheet Z.1. Eject the embryo with the agarose and discard these. The capillary can be reused several times.</p></li></ol></li><br/><li><p>Light-sheet microscopy to image neuronal damage (in <em>D. rerio</em>)</p><ol class="npwtwok"><li><p>A custom designed open light-sheet microscope was used for acquiring videos of the zebrafish larvae.</p></li><li><p>Briefly, mount samples in a customized chamber for light-sheet microscopy on a horizontal stage controlled by an ultrasound motorized 10 nm step size Z-Drive.</p></li><li><p>Generate a light-sheet using a single illumination arm in front of the sample mounting stage delivering laser beam (488 nm or 561 nm) through a 10x illumination objective.</p></li><li><p>Acquire images on a Hamamatsu Orca Flash 4.0 V3 sCMOS Camera through a Super High NA (1.0; XLUMPLFLN20xW) 20x water dipping lens.</p></li><li><p>Use multidimensional acquisition in MetaMorph software with streaming option to acquire data.</p></li><li><p>Use an acquisition protocol to acquire 1 z-stack with 2  <span style='font-family:"font-size:16px;text-align:justify;white-space:normal;background-color:#FFFFFF;'>Î¼m </span>step-size per min for 30 min.</p></li><li><p>Illuminate the sample by 561 nm laser light filtered through a Texas Red/mCherry/AlexaFluor 594 dichroic. At minutes 5 and 6, flood the stage with high intensity transmission white light from X-Cite 120LED System for 45 s. Resume the time lapse imaging for the next 25 min.</p></li><li><p>Repeat the whole process a second time after acquisition of the first time-lapse.</p></li><li><p>Make videos from time-lapse images.</p></li></ol></li><br/><li><p><em>Danio rerio</em> Mitochondrial metabolic flux assay</p><ol class="npwtwok"><li><p>Split injected embryos (C) into 2 dishes for 48 h; one for light exposure condition, and the other as the dark control.</p></li><li><p>Dechorionate both dishes with minimal reflected light.</p></li><li><p>Expose experimental embryos to high intensity LED light on a stereoscope for 2 h. Embryos are then loaded into the 96 well plate using a fine Pasteur pipette.</p></li><li><p>Measure Oxygen Consumption Rate (OCR) using the XF96 Extracellular Flux Analyzer.</p></li><li><p>Place one embryo (48 hpf) in 175 Î¼l of E3 water in each well and start the experiment.</p></li><li><p>Measure OCR every 8 min for 6 cycles (48 min), then add a drug.</p></li><li><p>Calculate the volume of injection needed from the stock concentration of each drug, such that the final concentrations in the well are â a) 9.4 M oligomycin, b) 2.5 Î¼M FCCP and c) 20 mM sodium azide.</p></li><li><p>The experiment is summarized in Figure 3.</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3856/bioprotoc-10-23-3856-g003.jpg"/><br/><strong>Figure 3. Schematic of metabolic flux assay on transiently expressing AÎ²-CRY2-mCherry embryos</strong> </p></li></ol></li><br/><li><p><em>D. rerio</em> Adenosine triphosphate (ATP) assay</p><ol class="npwtwok"><li><p>Use the remaining embryos from (Procedure C) for ATP assay. Or repeat Procedure C to generate more embryos.</p></li><li><p>Pool five 48 h embryos in 1.5 ml Micro tubes and add 50 Î¼l ice-cold 10% TCA buffer. For each condition make at least 8 such pools (n = 8).</p></li><li><p>Use a hand held micropestle homogenizer, to homogenize embryos on ice.</p></li><li><p>Centrifuge the homogenate at 4 Â°C at 10,000 RPM for 5 minutes.</p></li><li><p>Transfer the supernatant to a new tube and keep at -80 Â°C for use later, or use immediately.</p></li><li><p>Add 5 Î¼l of sample supernatant or ATP standards into a white 96-well microtiter plate (Figure 4).</p></li><li><p>Measure ATP luminescence using BioTek cytation 3 plate reader preprogrammed to inject arsenite ATP buffer (150 Î¼l/well) followed by firefly lantern extract (45 Î¼l/well).</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3856/bioprotoc-10-23-3856-g004.jpg"/><br/><strong>Figure 4. Schematic shows the summary of procedures using zebrafish</strong> </p></li></ol></li><br/><li><p>Fluorescence microscopy to visualize AÎ² aggregates in <em>C. elegans</em> </p><ol class="npwtwok"><li><p>Wash nematodes off the plates into an Eppendorf tube.</p></li><li><p>Add 1 mM Levamisole into the worm suspension.</p></li><li><p>Gently mix the worm suspension and place 20 Î¼l of the worm suspension onto a glass slide containing 2% agarose [see Verbrugghe and Chan (2011) on how to make agarose pad].</p></li><li><p>Switch off ambient light and bring the slides for imaging using 561 nm and 488 nm laser.</p></li><li><p>Using 488 nm laser, select pharyngeal-GFP positive nematodes for the transgenic animals.</p></li><li><p>Using 561 nm laser, observe the A&amp;Bgr; expression and aggregate levels.</p></li></ol></li><br/><li><p>Other assays used to confirm aggregation-induced deficits in <em>C. elegans</em> </p><ol class="npwtwok"><li><p>Lifespan assay (He, 2011), mitochondrial metabolic flux assay (Ng and Gruber, 2019) and ATP assay (Schaffer <em>et al.</em>, 2011) can be used to confirm A&amp;Bgr; aggregation-induced deficits.</p></li></ol></li></ol><p class="pptt" id="biaoti39135">### Data analysis</p><ol class="npwone"><li><p>Light-sheet microscopy to image neural development during embryogenesis</p><ol class="npwtwok"><li><p>Process the acquired images file in the ZEN 2014 SP software.</p></li><li><p>The embryo is likely to shift slightly during imaging. Hence, determine the z-stack section that includes the top of the embryo at every time point (Z<sub>T</sub>) (Figure 5). Thereafter, note the z-stack section (Z<sub>B</sub>) in the embryo that stays in focus throughout the time points after which the focus is lost (Figure 5 shows an example of the top, bottom and middle z-stack images for the control embryos which was used to subsequently generate Video 3 in the original research paper [Lim <em>et al.</em>, 2020]). Create a subset file including the images from Z<sub>T</sub> to Z<sub>B</sub>.</p></li><li><p>If the embryo does not show any changes in development after a certain time point (T<sub>1</sub>), create a further subset from the start of imaging to T<sub>1</sub>.</p></li><li><p>Create a maximum intensity projection of the final subset file and adjust the display settings that best suit the image. Export the time series as a movie in the format of choice.</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201205/20201205010350_4513.jpg"/><br/><strong>Figure 5. Annotation of the top (Z<sub>T</sub>) and bottom (Z<sub>B</sub>) z-stack of the embryo during post-processing of the data file to create a subset file.</strong> Z<sub>T</sub> is defined as the top of the embryo at every time point (Z<sub>T</sub>) and Z<sub>B</sub> is the z-stack at which the embryo is in focus throughout the time points after which the focus is lost. Scale bar shows 50 Î¼m.</p></li></ol></li><br/><li><p><em>D. rerio</em> Mitochondrial metabolic flux assay (Figure 4)</p><ol class="npwtwok"><li><p>Use the Seahorse Wave Controller software, compute the following area under the curve (AUC):<br/>AUC1 = oxygen consumption rate measurement 1 to 6 (pre-injection)<br/>AUC2 = oxygen consumption rate measurement 7 to 12 (after oligomycin injection)</p><p>AUC3 = oxygen consumption rate measurement 13 to 18 (after FCCP injection)<br/>AUC4 = oxygen consumption rate measurement 19 to 24 (after sodium azide injection)</p></li><li><p>Determine the respective respiration rate as follow:<br/>Basal respiration = AUC1 â AUC4</p><p>Proton leak = AUC2 â AUC4<br/>ATP-linked respiration = AUC1 â AUC2<br/>Maximum respiration = AUC3 â AUC4<br/>Spare respiratory capacity = AUC3 â AUC1<br/>Non-mitochondria respiration = AUC1 â AUC4</p></li><li><p>Plot the individual respiration rate in bar graph and analyze the group mean using one-way ANOVA.</p></li></ol></li><br/><li><p><em>D. rerio</em> ATP assay</p><ol class="npwtwok"><li><p>Calculate ATP level for all samples using the standard curve constructed from the ATP standard.</p></li><li><p>Compute the mean ATP level for different conditions and analyze the data using one-way ANOVA.</p></li></ol></li></ol><p class="pptt" id="biaoti39136">### Notes</p><p>For sample imaging in the Light-sheet Z.1 for <em>D. melanogaster</em>, the sample chamber heats up significantly due to the short time interval of 2.5 min. Hence, the room temperature was set to 18 Â°C to prevent over-heating of the water in the chamber that prevents development of the embryo. Alternatively, the water can be cooled by setting up a peristaltic pump attached to the sample chamber.</p><p class="pptt" id="biaoti39137">### Recipes</p><ol class="npwtwok"><li><p>Standard Fly food</p><p>6 g Bacto agar</p><p>114 g glucose</p><p>56 g cornmeal</p><p>25 g Brewerâs yeast</p><p>20 ml of 10% Nipagin in 1 L final volume</p></li><li><p>Grape juice agar plates</p><p>Prepare grape juice agar using FlyStuff Grape Agar Premix, For Embryo Collection (25 Packets/Unit) according to the manufacturerâs instructions and pour approximately 4-5 ml of agar per 60 mm Petri dish. Allow to solidify and store the plates at 4 Â°C.</p></li><li><p>Yeast paste</p><p>Mix yeast (Bruggeman Instant Dry Yeast Blue, 500 g) with a small amount of water in a plastic beaker to achieve a thick paste. The yeast paste can be kept in a beaker covered with aluminum foil at 4 Â°C.</p></li><li><p>1% low gelling temperature agarose in water aliquoted into 1.5 ml Eppendorf tubes</p><ol class="npwthree"><li><p>In a 50 ml microwave-safe glass, add 0.1 g of Agarose, low gelling temperature Type VII-A and add 10 ml of autoclaved distilled water</p></li><li><p>Microwave with intermittent mixing till the agarose just dissolves without boiling to prevent excessive loss of water</p></li><li><p>Once the agarose has dissolved completely, let the mixture cool slightly and aliquot approximately 1 ml of the 1% agarose into 1.5 ml Eppendorf tubes</p></li><li><p>Close the Eppendorf tubes after the agarose has solidified and store at room temperature</p></li><li><p>For mounting, melt a tube of 1% low gelling temperature agarose at 70 Â°C for half an hour in a heat block and then cool it to 37 Â°C, mix thoroughly by inverting before using for mounting</p></li></ol></li><li><p>2% low gelling temperature agarose in water aliquoted into 1.5 ml Eppendorf tubes (for <em>C. elegans</em> imaging)</p><p>Same as above, but dissolve 0.2 g of agarose, low gelling temperature Type VII-A in 10 ml of autoclaved distilled water</p></li><li><p>60x E3 (embryo medium) stock solution (for <em>D. rerio</em>)</p><ol class="npwthree"><li><p>Dissolve 172 g NaCl, 7.6 g KCl, 29 g CaCl<sub>2</sub>Â·2H<sub>2</sub>O and 49 g MgSO<sub>4</sub>Â·7H<sub>2</sub>O in 10 L of Milli-Q water</p></li><li><p>Dilute 160 ml of 60x stock solution in distilled water to make up to 10 L of E3 (1x)</p></li></ol></li><li><p>2 % low melting agarose</p><ol class="npwthree"><li><p>Dissolving 1 g of agarose powder in 100 ml E3 (1x)</p></li><li><p>Cook in microwave for 2 min at low power (for fish imaging)</p></li></ol></li><li><p>25x Tricaine stock solution (for <em>D. rerio</em>)</p><ol class="npwthree"><li><p>Dissolving 400 mg Tricaine in 97.9 ml DD water</p></li><li><p>Adjust pH to 7 using Tris-HCl (1 M)</p></li><li><p>To use Tricaine for anesthesia, dilute stock solution by adding 96 ml E3 (1x) to 4 ml of 25x stock solution in a beaker</p></li><li><p>Diluted Tricaine should be disposed in activated charcoal</p></li></ol></li><li><p>10% TCA buffer</p><p>Dissolve 1 g TCA powder in 10 ml distilled water</p></li><li><p>Arsenite ATP buffer</p><p>Mix 80 mM MgSO<sub>4</sub>Â·7H<sub>2</sub>O</p><p>10 mM KH<sub>2</sub>PO<sub>4</sub> </p><p>100 mM Na<sub>2</sub>HASO<sub>4</sub>Â·7H<sub>2</sub>O in 1:1:1 ratio</p></li></ol><p class="pptt" id="biaoti39138">### Acknowledgments</p><p>We are thankful for the funding provided by Ministry of Education Singapore AcRF grant IG17-LR005, IG17-BS101 and IG18-BS002 to JG, Yale-NUS College grant R-607-265-225-121 to ASM, AcRF grants IG17-LR006 and IG18-LR001 to NST.</p><p class="pptt" id="biaoti39139">### Competing interests</p><p>We have no conflicts of interest to declare.</p><p class="pptt" id="biaoti39140">### Ethics</p><p>Institutional biosafety and genetic manipulation guidelines of IMCB were followed for generation of transgenic zebrafish. Fish husbandry, rearing and maintenance were performed following approved protocols by Institutional Animal Care and Use Committee (IACUC) of the Biological Resource Center at A*STAR. Approved experimental protocols (IACUC 191501) were followed.</p><p class="pptt" id="biaoti39141">### References</p><ol class="npwtwok"><li>Akitake, C. M., Macurak, M., Halpern, M. E. and Goll, M. G. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21223961" target="_blank">Transgenerational analysis of transcriptional silencing in zebrafish.</a> <em>Dev Biol</em> 352(2): 191-201.</li><li>Bischof, J., Maeda, R. K., Hediger, M., Karch, F. and Basler, K. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17360644" target="_blank">An optimized transgenesis system for <em>Drosophila </em>using germ-line-specific phiC31 integrases.</a> <em>Proc Natl Acad Sci U S A</em> 104(9): 3312-3317.</li><li>Brand, A. H. and Perrimon, N. (1993). <a href="http://www.ncbi.nlm.nih.gov/pubmed/8223268" target="_blank">Targeted gene expression as a means of altering cell fates and generating dominant phenotypes.</a> <em>Development</em> 118(2): 401-415.</li><li>Bugaj, L. J., Choksi, A. T., Mesuda, C. K., Kane, R. S. and Schaffer, D. V. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23377377" target="_blank">Optogenetic protein clustering and signaling activation in mammalian cells.</a> <em>Nat Methods</em> 10(3): 249-252.</li><li>Bunnag, N., Tan, Q. H., Kaur, P., Ramamoorthy, A., Sung, I. C. H., Lusk, J. and Tolwinski, N. S. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/32201167" target="_blank">An Optogenetic Method to Study Signal Transduction in Intestinal Stem Cell Homeostasis.</a> <em>J Mol Biol</em> 432(10): 3159-3176.</li><li>Campos-Ortega, J. A. and Hartenstein, V. (1997). <a href="https://link.springer.com/chapter/10.1007%2F978-3-662-22489-2_2" target="_blank">Stages of <em>Drosophila</em> Embryogenesis.</a> In. <em>The Embryonic Development of Drosophila melanogaster.</em> Berlin, Heidelberg: Springer.</li><li>Cummings, J., Lee, G., Ritter, A. and Zhong, K. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29955663" target="_blank">Alzheimer's disease drug development pipeline.</a> <em>Alzheimers Dement(N Y)</em> 4: 195-214.</li><li>De-Paula, V. J., Radanovic, M., Diniz, B. S. and Forlenza, O. V. (2012). <a href="https://link.springer.com/chapter/10.1007/978-94-007-5416-4_14" target="_blank">Alzheimer's disease.</a> <em>Subcell Biochem</em> 65: 329-352.</li><li>Fenno, L., Yizhar, O. and Deisseroth, K. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21692661" target="_blank">The development and application of optogenetics.</a> <em>Annu Rev Neurosci</em> 34: 389-412.</li><li>Ferreira, S. T. and Klein, W. L. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21914486" target="_blank">The Abeta oligomer hypothesis for synapse failure and memory loss in Alzheimer's disease.</a> <em>Neurobiol Learn Mem</em> 96(4): 529-543.</li><li>Fong, S., Teo, E., Ng, L. F., Chen, C. B., Lakshmanan, L. N., Tsoi, S. Y., Moore, P. K., Inoue, T., Halliwell, B. and Gruber, J. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27653553" target="_blank">Energy crisis precedes global metabolic failure in a novel <em>Caenorhabditis elegans</em> Alzheimer Disease model.</a> <em>Sci Rep</em> 6: 33781.</li><li>Groth, A. C., Fish, M., Nusse, R. and Calos, M. P. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15126397" target="_blank">Construction of transgenic <em>Drosophila</em> by using the site-specific integrase from phage phiC31.</a> <em>Genetics</em> 166(4): 1775-1782.</li><li>Hardy, J. A. and Higgins, G. A. (1992). <a href="http://www.ncbi.nlm.nih.gov/pubmed/1566067" target="_blank">Alzheimer's disease: the amyloid cascade hypothesis.</a> <em>Science</em> 256(5054): 184-185.</li><li>Hartenstein, V., (1993). <a href="https://www.sdbonline.org/sites/fly/atlas/00atlas.htm" target="_blank">Atlas of <em>Drosophila</em> development.</a> Plainview, N.Y.: Cold Spring Harbor Laboratory Press.</li><li>Hawkes, N., (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27899350" target="_blank">Sixty seconds on... solanezumab.</a> <em>BMJ</em> 355: i6389.</li><li>He, F., (2011). <a href="https://bio-protocol.org/bio101/e57" target="_blank">Lifespan Assay. </a><em>Bio-101 1</em> (7): e57.</li><li>Hong, E., Santhakumar, K., Akitake, C. A., Ahn, S. J., Thisse, C., Thisse, B., Wyart, C., Mangin, J. M. and Halpern, M. E. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24327734" target="_blank">Cholinergic left-right asymmetry in the habenulo-interpeduncular pathway</a>. <em>Proc Natl Acad Sci U S A</em> 110(52): 21171-21176.</li><li>Kaur, P., Saunders, T. E. and Tolwinski, N. S. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29192250" target="_blank">Coupling optogenetics and light-sheet microscopy, a method to study Wnt signaling during embryogenesis.</a> <em>Sci Rep</em> 7(1): 16636.</li><li>Kawakami, K., (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18047699" target="_blank">Tol2: a versatile gene transfer vector in vertebrates.</a> <em>Genome Biol</em> 8 Suppl 1: S7.</li><li>Kennedy, M. J., Hughes, R. M., Peteya, L. A., Schwartz, J. W., Ehlers, M. D. and Tucker, C. L. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21037589" target="_blank">Rapid blue-light-mediated induction of protein interactions in living cells.</a> <em>Nat Methods</em> 7(12): 973-975.</li><li>Kumar, A. and Singh, A. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25712639" target="_blank">A review on Alzheimer's disease pathophysiology and its management: an update.</a> <em>Pharmacol Rep</em> 67(2): 195-203.</li><li>LaFerla, F. M., Green, K. N. and Oddo, S. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17551515" target="_blank">Intracellular amyloid-beta in Alzheimer's disease.</a> <em>Nat Rev Neurosci</em> 8(7): 499-509.</li><li>Lim, C. H., Kaur, P., Teo, E., Lam, V. Y. M., Zhu, F., Kibat, C., Gruber, J., Mathuru, A. S. and Tolwinski, N. S. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/32228858" target="_blank">Application of optogenetic Amyloid-beta distinguishes between metabolic and physical damages in neurodegeneration.</a> <em>Elife</em> 9: e52589.</li><li>Lim, C. H. and Mathuru, A. S. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29615550" target="_blank">Modeling Alzheimer's and Other Age Related Human Diseases in Embryonic Systems.</a> <em>J Dev Biol</em> .1. 6(1). doi:10.3390/jdb6010001.</li><li>Lin, C. C. and Potter, C. J. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27334272" target="_blank">Editing Transgenic DNA Components by Inducible Gene Replacement in <em>Drosophila melanogaster</em>.</a> <em>Genetics</em> 203(4): 1613-1628.</li><li>Markstein, M., Pitsouli, C., Villalta, C., Celniker, S. E. and Perrimon, N. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18311141" target="_blank">Exploiting position effects and the gypsy retrovirus insulator to engineer precisely expressed transgenes.</a> <em>Nat Genet</em> 40(4): 476-483.</li><li>Mas, P., Devlin, P. F., Panda, S. and Kay, S. A. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11089975" target="_blank">Functional interaction of phytochrome B and cryptochrome.</a> <em>Nature</em> 408(6809): 207-211.</li><li>Moglich, A. and Moffat, K. (2010). <a href="https://pubs.rsc.org/en/content/articlelanding/2010/pp/c0pp00167h/unauth#!divAbstract" target="_blank">Engineered photoreceptors as novel optogenetic tools.</a> <em>Photochem Photobiol Sci</em> 9(10): 1286-1300.</li><li>Ng, L. F. and Gruber, J. (2019). <a href="https://bio-protocol.org/e3243" target="_blank">Measurement of Respiration Rate in Live <em>Caenorhabditis elegans</em>.</a> <em>Bio-protocol</em> 9(10): e3243.</li><li>Schaffer, S., Gruber, J., Ng, L. F., Fong, S., Wong, Y. T., Tang, S. Y. and Halliwell, B. (2011). T<a href="http://www.ncbi.nlm.nih.gov/pubmed/21153705" target="_blank">he effect of dichloroacetate on health- and lifespan in <em>C. elegans.</em></a> <em>Biogerontology</em> 12(3): 195-209.</li><li>Stiernagle, T., (2006). <a href="http://www.wormbook.org/chapters/www_strainmaintain/strainmaintain.html" target="_blank">Maintenance of<em> C. elegans</em>.</a> <em>WormBook</em>, 1-11. doi:10.1895/wormbook.1.101.1</li><li>Verbrugghe, K. J. and Chan, R. C. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21490567" target="_blank">Imaging <em>C. elegans</em> embryos using an epifluorescent microscope and open source software.</a> <em>J Vis Exp</em> (49). doi:10.3791/2625.</li><li>Zhang, Y. W., Thompson, R., Zhang, H. and Xu, H. (2011). <a href="https://pubmed.ncbi.nlm.nih.gov/21214928/" target="_blank">APP processing in Alzheimer's disease.</a> <em>Mol Brain</em> 4: 3. doi:10.1186/1756-6606-4-3.<br/><br/></li></ol>