<body>
            <h1>Equilibrium and Kinetic Measurements of Ligand Binding to HiBiT-tagged GPCRs on the Surface of Living Cells</h1>
            <p><strong>Authors:</strong> Michelle E. Boursier, Sergiy Levin, Robin Hurst, Rachel Friedman Ohana</p>
            <h2>Abstract</h2>
            <p><p>G-protein coupled receptors (GPCRs) remain at the forefront of drug discovery efforts. Detailed assessment of features contributing to GPCR ligand engagement in a physiologically relevant environment is imperative to the development of new therapeutics with improved efficacy. Traditionally, binding properties such as affinity and kinetics were obtained using biochemical radioligand binding assays. More recently, the high specificity of resonance energy transfer has been leveraged toward the development of homogeneous cell-based proximity assays with capacity for real-time kinetic measurements. This suite of ligand binding protocols couples the specificity of bioluminescent resonance energy transfer (BRET) with the sensitivity afforded by the luminescent HiBiT peptide. The BRET format is used to quantify dynamic interactions between ligands and their cognate HiBiT-tagged GPCRs through competitive binding with fluorescent Tracers. At the same time, high affinity complementation of HiBiT with the cell impermeable LgBiT limits the bright bioluminescence donor signal to the cell surface and eliminates luminescence background from unoccupied receptors present in intracellular compartments.</p></p>
            <p><strong>Keywords:</strong> HiBiT, G-protein-coupled receptor (GPCR), Bioluminescence resonance energy transfer (BRET), Ligand binding, Kinetics</p>
            <h2>Background</h2><p>G protein-coupled receptors (GPCRs) play critical roles in driving cell signaling pathways and remain a prominent target class for drug development (Santos <em>et al.</em>, 2017). Accordingly, GPCR ligand binding assays that can quantify biophysical properties critical to drug action are essential for both basic research and drug discovery campaigns (Fang, 2012; Stoddart <em>et al.</em>, 2016). Biochemical assessment of ligand binding affinity under equilibrium conditions has traditionally been viewed as an acceptable indicator for <em>in vivo</em> pharmacology of drug candidates (Hulme and Trevethick, 2010). However, there is growing understanding that this approach might be too simplistic. The kinetics of ligand binding provides insights into duration of drug action including residence time and potential for rapid rebinding, which have been postulated to be relevant predictors for <em>in vivo</em> efficacy (Fang, 2012; Copeland, 2016; Sykes <em>et al.</em>, 2019). In addition, many GPCRs function as parts of multi-protein signaling complexes, which might influence distinct characteristics of ligand binding (Kenakin, 2010; Fang, 2012). As a result, there is increasing interest in comprehensive measurements of GPCR ligand engagement in the physiological context of living cells in both equilibrium and real-time.<br/><br/>GPCR ligand binding assays routinely employ ligands labeled with either radioisotopes or fluorophores in order to track the binding to a target (Fang, 2012; Stoddart <em>et al.</em>, 2016). The most common biochemical GPCR ligand binding assays utilize radioligands in a competitive binding format to measure interactions between ligands and their unmodified cognate GPCRs (Flanagan, 2016). The specificity of these assays relies on the use of highly selective radioligands in combination with overexpressed GPCRs within membrane preparations. In addition, these non-homogenous assays require separation of free from bound radioligand, which limits the resolution and throughput of kinetic analyses (Sykes <em>et al.</em>, 2019). Furthermore, the wash steps are more difficult to perform in high throughput screenings.<br/><br/>Proximity binding assays relying on time resolved fluorescence resonance energy transfer (TR-FRET) or bioluminescent resonance energy transfer (BRET) have emerged as attractive alternatives to radioligand binding assays, particularly for high throughput screenings (Stoddart <em>et al.</em>, 2016; Sykes <em>et al.</em>, 2019). These assays typically use fluorescently labeled ligands (fluorescent Tracers) in a competitive binding format to measure interactions between unmodified ligands and their cognate GPCRs genetically fused to an energy donor (<em>i.e.</em>, a lanthanide-labeled protein tag or a luciferase). Although both ligand and GPCR need to be modified, these approaches provide high specificity as signal is generated only when the fluorescent Tracer and tagged GPCR are in close proximity. The inherent distance constraints of energy transfer (Ciruela, 2008) eliminates the requirement for highly selective labeled ligands and permits homogeneous cell-based assays. The homogeneous format allows for real-time kinetic measurements and is generally more compatible with high throughput settings.<br/><br/>The development of the bright NanoLuc luciferase (Hall <em>et al.</em>, 2012) enabled BRET-based GPCR ligand binding assays previously impossible with other luciferases (Stoddart <em>et al.</em>, 2015; Soave <em>et al.</em>, 2016). Still, due to the cell permeability of the NanoLuc substrate furimazine, the use of full length NanoLuc as the energy donor results in signal generation regardless of NanoLuc’s cellular localization. This could lead to an elevated background from NanoLuc-tagged receptors that are present in intracellular compartments and are unable to engage the Tracer. This issue was recently addressed by genetically fusing GPCRs to the luminescent peptide HiBiT rather than the full length NanoLuc (Soave <em>et al.</em>, 2019; Boursier <em>et al.</em>, 2020). HiBiT is a small 11-amino acid peptide that produces bright luminescence upon high affinity complementation with LgBiT, an 18 kDa subunit derived from NanoLuc (Schwinn <em>et al.</em>, 2018). The use of HiBiT restricts signal generation in endpoint assays to the cell surface. This is because LgBiT is cell impermeable and therefore incapable to complement any HiBiT-tagged receptors located in intracellular compartments. An added benefit is the use of a small peptide tag, which is less likely to interfere with protein function and trafficking to the cell surface.<br/> <br/>The binding protocols described here couple the specificity of BRET with the sensitivity and inherent cell-surface localization of the HiBiT/LgBiT reporter for quantification of binding interactions with selective GPCRs on the surface of living cells (Figure 1). They include four principal types of assays using β<sub>2</sub>-AR ligand binding as an example. Two of these assays utilize increasing concentrations of a fluorescent Tracer to measure its binding characteristics in equilibrium and real-time. The other two assays employ increasing concentrations of an unmodified test compound to derive its binding properties through the competition with a fixed concentration of a fluorescent Tracer. These competition analyses can be performed under equilibrium or in a kinetic format, which take advantage of a method reported by Motulsky and Mahan (1984).<br/><br/> <img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3861/bioprotoc-10-24-3861-g001.jpg"/><br/><strong>Figure 1. Monitoring cell surface ligand engagement with selective HiBiT-tagged GPCRs via BRET.</strong> BRET assay utilizing fluorescent Tracers in a competitive binding format to quantify dynamic cell-surface interactions between ligands and their cognate HiBiT-tagged GPCRs. This approach benefits from high specificity as signal is generated only when the fluorescent Tracer and tagged GPCR are in close proximity.</p>
        </body>

<p class="pptt" id="biaoti39377">## Materials and Reagents</p><ol class="npwtwok"><li><p>White, Tissue Culture-Treated (TC) 96-well plates (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 3917 </a> )</p></li><li><p>Non-binding V bottom plates (for serial dilutions; for example, Costar, catalog number: <a class="blackNoUnder" target="_blank"> 3896 </a> )</p></li><li><p>Tissue Culture-Treated (TC) flasks (growth area 150 or 75 cm<sup>2</sup>; Falcon, catalog number: <a class="blackNoUnder" target="_blank"> 355001 or 353107 </a> )</p></li><li><p>HEK293 cells</p></li><li><p>0.05% Trypsin/EDTA (Life Technologies, catalog number: <a class="blackNoUnder" target="_blank"> 25300 </a> )</p></li><li><p>Dulbecco's Modified Eagle Medium (DMEM) (Life Technologies, catalog number: <a class="blackNoUnder" target="_blank"> 11995 </a> )</p></li><li><p>Fetal Bovine Serum (FBS) (HyClone, catalog number: <a class="blackNoUnder" target="_blank"> SH30070.03 </a> )</p></li><li><p>100x Penicillin-streptomycin (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> P4333 </a> )</p></li><li><p>Opti-MEM without Phenol Red (Life Technologies, catalog number: <a class="blackNoUnder" target="_blank"> 11058 </a> )</p></li><li><p>FuGENE HD (Promega, catalog number: E2311) or ViaFect (Promega, catalog number: <a class="blackNoUnder" target="_blank"> E4981 </a> ) <br/><em>Note: Other transfection reagents can also be used.</em> </p></li><li><p>DMSO (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> D2650-5X5ML </a> )</p></li><li><p>Tracer dilution buffer (Promega, catalog number: <a class="blackNoUnder" target="_blank"> N2191 </a> )</p></li><li><p>Nano-Glo HiBiT Extracelular Detection System (Promega, catalog number: <a class="blackNoUnder" target="_blank"> N2420 or N2421 </a> )</p></li><li><p>Transfection Carrier DNA (Promega, catalog number: <a class="blackNoUnder" target="_blank"> E4881 </a> )</p></li><li><p>Test compound of interest</p></li><li><p>DNA construct encoding a HiBiT- tagged GPCR fusion</p><p><em>Note: HiBiT-tagged GPCR constructs can be obtained from Promega Custom Assays Systems or generated via cloning into pFN39K secHiBiT CMV-neo Flexi Vector (Promega, catalog number: <a class="blackNoUnder" target="_blank"> N2411 </a> ) or pBiT3.1-secN [CMV/HiBiT/Blast] vector (Promega, catalog number: N2381).</em> </p></li><li><p>Fluorescent Tracer</p><p><em>Note: Several NanoBRET GPCR Tracers can be obtained from Promega Custom Assays Systems. In addition a fluorescent GPCR Tracer can be generated according to the published protocol (Robers et al., 2019) using building blocks that can be obtained from Promega Custom Assays Systems. </em> </p></li><li><p>Miscellaneous tissue culture reagents</p></li><li><p>DMEM Cell Culture Medium (see Recipes)</p></li><li><p>Opti-MEM Assay Medium (see Recipes)</p></li></ol><p class="pptt" id="biaoti39378">### Equipment</p><ol class="npwtwok"><li><p>BRET-compatible microplate luminometer equipped with 450 nm (bandpass) and 600 nm (longpass) filters (<em>e.g.</em>, Promega GloMax Discover, PerkinElmer EnVision, or BMG Clariostar)</p></li><li><p>Orbital plate shaker</p></li><li><p>Miscellaneous tissue culture equipment</p></li></ol><p class="pptt" id="biaoti39379">### Software</p><ol class="npwtwok"><li><p>GraphPad Prism (<a href="https://www.graphpad.com/scientific-software/prism/">https://www.graphpad.com/scientific-software/prism</a>)</p></li></ol><p class="pptt" id="biaoti39380">### Procedure</p><ol class="npwone"><li><p>Transient transfection of HEK293 cells with HiBiT-tagged GPCR constructs</p><p><em>Note: The transfection conditions described below are recommended for adequate translocation and cell surface expression of HiBiT-tagged GPCRs. These include:</em> </p><ol class="npwbullet"><li><p><em>Dilution of DNA encoding the HiBiT-tagged GPCR into a promoterless Carrier DNA. This maintains the total amount of transfected DNA but allows for lower expression levels across the population of transfected cells. A 100-fold dilution is generally recommended, however in some cases this might require further optimization.</em> </p></li><li><p><em>DNA: Transfection reagent complexes should account for 10% (vol/vol) of transfected cell culture.</em> </p></li></ol><br/><ol class="npwtwok"><li><p>Cultivate HEK293 cells in Culture Medium (Recipe 1) in a TC-75 or TC-150 flask prior to assay.</p><p><em>Note: Cell density at time of harvest can affect transfectability. For optimal transfectability use freshly passaged HEK293 cells (ideally within 1-2 days) at 80-95% confluency.</em> </p></li><li><p>Aspirate medium from flask, add 5 ml or 8 ml trypsin to TC-75 or TC-150 flask, respectively and allow cells to dissociate from the flask.</p></li><li><p>Neutralize trypsin using 20 ml or 25 ml Culture Medium (for TC-75 or TC-150, respectively), and pellet cells via centrifugation at 200 <em>x g</em> for 5 min.</p></li><li><p>Aspirate medium and resuspend cells into a single cell suspension using Opti-MEM Assay Medium (Recipe 2).</p></li><li><p>Adjust the cell density to 2.2 x 10<sup>5 </sup>cells/ml in Opti-MEM Assay Medium in a sterile, conical tube.</p></li><li><p>Prepare transfection reagent: DNA complexes.</p><p>Below is an example for transfection of 10 ml culture:</p><ol class="npwthree"><li><p>Prepare a 10 Âµg DNA solution in Opti-MEM without serum. This solution should contain the following ratios of carrier DNA and DNA encoding the HiBiT-tagged GPCR construct:</p><ol class="npwfour"><li><p>9.9 Âµg/ml of Transfection Carrier DNA.</p></li><li><p>0.1 Âµg/ml of DNA encoding the HiBiT-tagged GPCR (100-fold dilution).</p></li><li><p>0.5 ml of Opti-MEM without serum.</p></li></ol></li></ol><ol class="npwthree" start="2"><li><p>Mix thoroughly.</p></li><li><p>Combine 30 Âµl transfection reagent (<em>i.e.</em>, FuGENE HD or ViaFect) with 0.5 ml Opti-MEM without serum and mix by pipetting.</p><p><em>Note: If using another transfection reagent, follow manufacturer recommendation.</em> </p></li><li><p>Combine the DNA and transfection reagent solutions and mix by pipetting 10-15 times.</p></li><li><p>Incubate at room temperature for 15 min to allow transfection reagent, DNA complexes to form.</p></li></ol></li><li><p>Combine one part (<em>e.g.</em>, 1 ml) of transfection reagent: DNA complexes with 9 parts (<em>e.g.</em>, 9 ml) of HEK293 cells in suspension at 2.2 x 10<sup>5 </sup>cells/ml. Mix gently by inversion 5 times and plate in a white TC-treated 96-well plate at 95 Âµl/well.</p><p><em>Note: Larger or smaller bulk transfections should be scaled accordingly, using this ratio.</em> </p></li><li><p>Incubate for at least 20 h at 37 Â°C + 5% CO<sub>2</sub>.</p></li></ol></li><br/><li><p>NanoBRET GPCR-ligand engagement protocols</p><br/><ol class="npwtwok"><p class="ppzz" style="margin-left:-30px;"><strong><em>Saturation binding of a fluorescent tracer to a HiBiT-tagged GPCR</em></strong> </p><li><p>Treat cells with a serial dilution of a fluorescent Tracer.</p><ol class="npwthree"><li><p>Prepare a 20x serially diluted Tracer.</p><ol class="npwfour"><li><p>First prepare a 100x serially diluted Tracer in DMSO. Prepare 3-fold serial dilutions starting at 300 ÂµM, which corresponds to a final 1x concentration of 3 ÂµM. In some cases, a higher or lower Tracer concentration may provide a better assay window.</p><p><em>Note: Make sure to include within the serial dilution a no Tracer control.</em> </p></li><li><p>Dilute the Tracer Dilution Buffer (TDB) 3-fold in Opti-MEM without phenol red to generate a 0.3x TDB.</p></li><li><p>Dilute the 100x serially diluted Tracer 5-fold in 0.3x TDB to generate a 20x serially diluted Tracer.</p></li></ol></li><li><p>Add 5 Âµl of 20x serially diluted Tracer per well of transfected cells (Figure 2 portrays a 96-well plate set-up).</p></li><li><p>Mix briefly on an orbital plate shaker.</p></li></ol></li><li><p>To determine specific binding treat control cells with excess unmodified compound.</p><ol class="npwthree"><li><p>Prepare a 20x unmodified compound solution.</p><ol class="npwfour"><li><p>Prepare a 10% (vol/vol) DMSO solution in Opti-MEM without phenol red.</p></li><li><p>Prepare a 600 ÂµM unmodified compound solution in 10% DMSO, which corresponds to a final 1x concentration of 30 ÂµM.</p></li></ol></li><li><p>Add 5 Âµl of the 20x unmodified compound solution to the control wells (Figure 2 portrays a 96-well plate set-up).</p></li><li><p>Mix briefly on an orbital plate shaker.</p></li><li><p>Incubate plate at room temperature (to minimize receptor internalization) for 90 min protected from light, then proceed to NanoBRET detection section below.</p><p><em>Note: Agonist binding promotes receptor internalization to different extents, which can be detected through dose dependent decrease in donor signal (e.g., signal at 450 nm). Still, binding experiments at room temperature (22-25 Â°C) decease the rate of internalization and the ratiometric nature of BRET further minimizes the influence of internalization on binding analyses. Accordingly, binding analyses at 37 Â°C may require a shorter incubation time.</em> </p></li></ol><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3861/bioprotoc-10-24-3861-g002.jpg"/><br/><strong>Figure 2. A 96-well plate set-up for saturation binding assay.</strong> Cells treated with serial dilution of a fluorescent Tracer in the absence and presence (control wells) of excess unmodified compound.</p></li><br/><li><p>Detect NanoBRET</p><ol class="npwthree"><li><p>Immediately prior to BRET measurements prepare 2x HiBiT detection reagent in Opti-MEM without phenol red. This reagent consists of 100-fold dilution of LgBiT and 50-fold dilution of Nano-Glo HiBiT Extracellular Substrate. Mix thoroughly by pipetting.</p></li><li><p>Add 100 Âµl detection reagent/well.</p></li><li><p>Mix on an orbital plate shaker for 15 min.</p></li><li><p>Measure donor emission (<em>e.g.</em>, 450 nm) and acceptor emission (<em>e.g.</em>, 610 nm or 630 nm) using a NanoBRET-compatible luminometer.</p></li></ol></li><li><p>Determine BRET ratio and process data.</p></li><ol class="npwthree"><li><p>To generate raw BRET ratio values, divide the acceptor emission value (<em>e.g.</em>, 610 nm) by the donor emission value (<em>e.g.</em>, 450 nm).</p></li><li><p>Convert raw BRET units to milli BRET units (mBU) by multiplying each raw BRET value by 1,000. NanoBRET equation:</p><p><strong> BRET (mBU) = (Acceptor<sub>sample</sub>/Donor<sub>sample</sub>) x 1,000 </strong> </p></li><li><p>Determine specific BRET by subtracting BRET values in the presence of excess unmodified compound from BRET values in the absence of excess unmodified compound.</p><p><strong> Specific BRET (mBU) = BRET<sub>Tracer </sub>- BRET<sub>Tracer+ unmodified compound</sub></strong> </p></li><li><p>Plot the data in GraphPad Prism software using the log(agonist) vs. responseâvariable slope fitting and determine the apparent affinity for the Tracer (EC<sub>50</sub>) (see Figures 3A and 3B for representative analyses).</p></li><li><p>Use the special calculation in GraphPad Prism âFind EC anythingâ to determine the EC<sub>60</sub>-EC<sub>80</sub>.</p></li><li><p>Plot the specific binding data in GraphPad Prism software using the one site-specific binding fitting and determine the binding constant (K<sub>D</sub>) (see Figure 3C for representative analysis).</p></li></ol><br/><p><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201220/20201220171657_2047.jpg"/><br/><strong>Figure 3. Representative analyses for NanoBRET GPCR Tracer G-3 saturation binding to transiently expressed HiBiT-Î²<sub>2</sub>-AR. </strong> HEK293 cells expressing HiBiT-Î²<sub>2</sub>-AR were assayed as indicated in the protocol. Briefly, cells were transfected, grown overnight, and next day treated with serially diluted NanoBRET GPCR Tracer G-3 (<em>i.e.</em>, the previously described propranolol-BODIPY (Boursier <em>et al.</em>, 2020           )) in the presence or absence of 30 ÂµM propranolol. Following 90 min binding, HiBiT detection reagent was added, and cells were incubated for additional 15 min. Filtered luminescence was measured using GloMax Discover Microplate Reader (Promega) equipped with a 450 nm (8-nm bandpass) filter (donor) and a 600-nm long pass filter (acceptor) (n = 3; Error bars indicate SD). A. Binding in the presence and absence of excess propranolol, B. Specific binding, and C. Specific binding on a non-logarithmic scale.</p><br/><ol class="npwtwok"><p class="ppzz" style="margin-left:-30px;"><strong><em>Competitive displacement of a fluorescent tracer by unmodified test compound</em></strong> </p><li><p>Treat cells with fluorescent Tracer at a fixed concentration.</p></li><ol class="npwthree"><li><p>Prepare 20x Tracer solution.</p><ol class="npwfour"><li><p>Using the Tracer saturation binding analysis described above chose a fixed concentration between the EC<sub>60</sub> to EC<sub>80</sub> values.</p></li><li><p>First prepare a 200x Tracer solution in DMSO.</p></li><li><p>Dilute the Tracer Dilution Buffer (TDB) 3-fold in Opti-MEM without phenol red to generate a 0.3x TDB.</p></li><li><p>Dilute the 200x Tracer 10-fold in 0.3x TDB to generate a 20x Tracer solution.</p></li></ol></li><li><p>Add 5 Âµl of the 20x Tracer solution per well of transfected cells (Figure 4 portrays a 96-well plate set-up).</p></li><li><p>Mix briefly on an orbital plate shaker.</p></li></ol><li>Treat cells with a serial dilution of an unmodified test compound</li><ol class="npwthree"><li><p>Prepare a 20x serially diluted unmodified test compound.</p><ol class="npwfour"><li><p>Prepare a 10% (vol/vol) DMSO solution in Opti-MEM without phenol red.</p></li><li><p>Prepare a 20x serially diluted unmodified test compound in 10% DMSO. Prepare 3-fold serial dilutions starting at 600 ÂµM or 200 ÂµM, which corresponds to a final 1x concentration of 30 ÂµM or 10 ÂµM.</p><p>Depending on the binding affinity of the test compound, further optimization of the serial dilution range may be required.</p><p><em>Note: Make sure to include within the serial dilution a no test compound control.</em> </p></li></ol></li><li><p>Add 5 Âµl of the 20x serially diluted unmodified test compound per well of transfected cells (Figure 4 portrays a 96-well plate set-up).</p></li><li><p>Mix briefly on an orbital plate shaker.</p></li><li><p>Incubate plate at room temperature (to minimize receptor internalization) for 90 min protected from light, then proceed to NanoBRET detection section below.</p></li></ol><p><em>    Note: Binding analysis at 37 Â°C may require a shorter incubation time.</em> </p><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201220/20201220172200_6959.jpg"/><br/><strong>Figure 4. A 96-well plate set-up for competition binding assay.</strong> Cells treated with serial dilution of an unmodified test compound in the presence of a fluorescent Tracer at a fixed EC<sub>60</sub>-EC<sub>80</sub> concentration. <br/><p><br/></p><li>Detect NanoBRET</li><ol class="npwthree"><li><p>Immediately prior to BRET measurements prepare 2x HiBiT detection reagent in Opti-MEM without phenol red. This reagent consists of 100-fold dilution of LgBiT and 50-fold dilution of Nano-Glo HiBiT Extracellular Substrate. Mix thoroughly by pipetting.</p></li><li><p>Add 100 Âµl detection reagent/well.</p></li><li><p>Mix on an orbital plate shaker for 15 min.</p></li><li><p>Measure donor emission (<em>e.g.</em>, 450 nm) and acceptor emission (<em>e.g.</em>, 610 nm or 630 nm) using a NanoBRET-compatible luminometer.</p></li></ol><li><p>Determine BRET ratio and process data.</p></li><ol class="npwthree"><li><p>To generate raw BRET ratio values, divide the acceptor emission value (<em>e.g.</em>, 610 nm) by the donor emission value (<em>e.g.</em>, 450 nm).</p></li><li><p>Convert raw BRET units to milli BRET units (mBU) by multiplying each raw BRET value by 1,000. NanoBRET equation:</p><p><strong> BRET (mBU) = (Acceptor <sub>sample</sub> / Donor <sub>sample</sub>) x 1,000 </strong> </p></li><li><p>Plot the data in GraphPad Prism software using the log(inhibitor) vs. responseâvariable slope fitting and determine IC<sub>50</sub> for the test compound (see Figure 5 for representative analysis).</p></li><li><p>Use the IC<sub>50</sub> value to calculate binding constant for the test compound (Ki) according the Cheng-Pursoff equation (Cheng and Prusoff, 1973) where [L] is the concentration of fluorescent Tracer in the assay and K<sub>D</sub> is its affinity in the saturation binding experiment from above.</p><p><strong><em> K<sub>i</sub></em> = (IC<sub>50</sub>)/(1 + ([Tracer]/<em>K<sub>D</sub></em>) </strong> </p><br/><p><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201220/20201220172334_1887.jpg"/><br/><strong>Figure 5. Representative analysis for NanoBRET GPCR Tracer G-3 displacement from transiently expressed HiBiT-<b>Î²</b><sub>2</sub>-AR by propranolol. </strong> HEK293 cells expressing HiBiT-Î²<sub>2</sub>-AR were assayed as indicated in the protocol. Briefly, cells were transfected, grown overnight, and next day treated with propranolol titration along with NanoBRET GPCR Tracer G-3 (<em>i.e.</em>, the previously described propranolol-BODIPY (Boursier <em>et al.</em>, 2020         )) at a final EC<sub>80</sub> concentration. Following 90 min binding, HiBiT detection reagent was added, and cells were incubated for additional 15 min. Filtered luminescence was measured using GloMax Discover Microplate Reader (Promega) equipped with a 450 nm (8-nm bandpass) filter (donor) and a 600-nm long pass filter (acceptor) (n = 4; Error bars indicate SD).</p><p><br/></p><p class="ppzz" style="margin-left:-30px;"><strong><em>Binding kinetics of a fluorescent tracer to HiBiT-tagged GPCR</em></strong> </p><p class="ppzz" style="margin-left:-30px;"><em> Note: To minimize time intervals and better capture rapid binding, it is recommended to limit the number of tested Tracer concentrations to 4 or 5 (Figure 6 portrays a plate set-up). </em> </p></li></ol><ol class="npwtwok"><li><p>Treat cells with detection reagent.</p><ol class="npwthree"><li><p>Prepare 2x HiBiT detection regent in Opti-MEM without phenol red. This reagent consists of 100-fold dilution of LgBiT and 50-fold dilution of Nano-Glo HiBiT Extracellular Substrate. Mix thoroughly by pipetting 10-15 times.</p></li><li><p>Add to a plate containing the transfected cells 95 Âµl of detection reagent per well and mix on an orbital plate shaker.</p></li></ol></li><li><p>Pretreat control wells with excess unmodified compound.</p><p><em> Note: For background correction, pretreat the control wells (Figure 6 portrays a plate set-up) with excess ââunmodified compound to block Tracer binding. </em> </p><ol class="npwthree"><li><p>Prepare a 20x excess unmodified compound solution.</p><ol class="npwfour"><li><p>Prepare a 10% (vol/vol) DMSO solution in Opti-MEM without phenol red.</p></li><li><p>Prepare a 600 ÂµM unmodified compound solution in 10% DMSO, which corresponds to a final 1x concentration of 30 ÂµM.</p></li></ol></li><li><p>Add 10 Âµl of the 20x excess unmodified compound solution to the control wells.</p></li><li><p>Mix briefly on an orbital plate shaker.</p></li><li><p>Mix gently for additional 15 min to allow for HiBiT/LgBiT complementation.</p></li></ol></li><li><p>Treat cells with serially diluted fluorescent Tracer.</p><ol class="npwthree"><li><p>During the 15 min incubation prepare a 20x serially diluted fluorescent Tracer.</p><p><em> Note: It is recommended to use 10x EC<sub>50</sub> as the highest final concentration. </em> </p><ol class="npwfour"><li><p>First prepare a 100x serially diluted Tracer in DMSO. Start with 2- or 3-fold serial dilutions and limit the number of dilutions to 4 or 5 to minimize the time intervals between reads.</p></li><li><p>Dilute the Tracer Dilution Buffer (TDB) 3-fold in Opti-MEM without phenol red to generate a 0.3x TDB.</p></li><li><p>Dilute the 100x serially diluted Tracer 5-fold in 0.3x TDB to generate a 20x serially diluted Tracer.</p></li></ol></li><li><p>Immediately prior to kinetic measurements add 10 Âµl of 20x serially diluted Tracer per well of transfected cells.</p></li><li><p>Mix briefly on an orbital plate shaker and start kinetic reads on a NanoBRET-compatible luminometer, measuring donor emission (<em>e.g.</em>, 450 nm) and acceptor emission (<em>e.g.</em>, 610 nm or 630 nm). To capture rapid binding, use 0 min intervals between reads.</p></li></ol><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3861/bioprotoc-10-24-3861-g006.jpg"/><br/><strong>Figure 6. A 96-well plate set-up for binding kinetics of a Fluorescent Tracer.</strong> Cells treated with a limited serial dilution of a fluorescent Tracer (4-5 concentrations) in the absence and presence (control wells) of excess unmodified compound.<p><br/></p></li><li>Determine BRET ratios and process data.<ol class="npwthree"><li>To generate raw BRET ratio values, divide the acceptor emission value (<em>e.g.</em>, 610 nm) by the donor emission value (<em>e.g.</em>, 450 nm). </li><li>Convert raw BRET units to milli BRET units (mBU) by multiplying each raw BRET value by 1,000. NanoBRET equation: <br/><strong>BRET (mBU) = (Acceptor <sub>sample</sub>/Donor <sub>sample</sub>) x 1,000</strong> </li><li>Determine specific BRET with time by subtracting BRET values in the presence of excess unmodified compound from BRET values in the absence of excess unmodified compound. </li><li>Plot the data in GraphPad Prism software using the association kinetics two or more concentrations of hot fitting and determine the kinetic constants (k<sub>on</sub> and k<sub>off</sub>) and binding constant (K<sub>D</sub>) for the Tracer (see Figure 7 for representative analysis). </li><li>Calculate residence time (Ï) as 1/k<sub>off</sub>. <br/></li></ol></li><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3861/bioprotoc-10-24-3861-g007.jpg"/><br/><strong>Figure 7. Representative analysis for binding kinetics of NanoBRET GPCR Tracer G-3 to transiently expressed HiBiT-Î²<sub>2</sub>-AR. </strong> HEK293 cells expressing HiBiT-Î²<sub>2</sub>-AR were assayed as indicated in the protocol. Briefly, cells were transfected, grown overnight, and next day pre-treated with HiBiT detection reagent for 15 min prior to treatment with a titration of NanoBRET GPCR Tracer G-3 (<em>i.e.</em>, the previously described propranolol-BODIPY (Boursier <em>et al.</em>, 2020)). Kinetic measurements of filtered luminescence were taken using GloMax Discover Microplate Reader (Promega) equipped with a 450 nm (8-nm bandpass) filter (donor) and a 600-nm long pass filter (acceptor) (n = 3; Error bars indicate SD).</ol><p><br/></p><p><strong><em>Competitive binding kinetics for unmodified test compound to HiBiT-tagged GPCR in the presence of a fluorescent tracer</em></strong> </p><p><em>Note: To minimize time intervals and better capture rapid binding, it is recommended to limit the number of tested unmodified compound concentrations to 4 or 5 (</em><em>Figure 8 portrays a plate set-up)</em>.</p><ol class="npwtwok"><li><p>Treat cells with detection reagent.</p><ol class="npwthree"><li><p>Prepare 2x HiBiT detection regent in Opti-MEM without phenol red. This reagent consists of 100-fold dilution of LgBiT and 50-fold dilution of Nano-Glo HiBiT Extracellular Substrate. Mix thoroughly by pipetting 10-15 times.</p></li><li><p>Add to a plate containing the transfected cells 95 Âµl of detection reagent per well and mix on an orbital plate shaker.</p></li></ol></li><li><p>Pretreat control well with excess unmodified compound.</p><p><em> Note: For background correction, pretreat the control wells (Figure 8 portrays a plate set-up) with excess unmodified compound to block Tracer/test compound binding. </em> </p><ol class="npwthree"><li><p>Prepare a 20x excess unmodified compound solution.</p><ol class="npwfour"><li><p>Prepare a 10% (vol/vol) DMSO solution in Opti-MEM without phenol red.</p></li><li><p>Prepare a 600 ÂµM unmodified compound solution in 10% DMSO, which corresponds to a final 1x concentration of 30 ÂµM.</p></li></ol></li><li><p>Add 10 Âµl of the 20x excess unmodified compound solution to the control wells.</p></li><li><p>Mix briefly on an orbital plate shaker.</p></li><li><p>Mix gently for additional 15 min to allow for HiBiT/LgBiT complementation.</p></li></ol></li><li><p>Treat cells with a fixed concentration of fluorescent Tracer and serial dilution of test compound.</p><p>During the 15 min incubation prepare a 20x fixed concentration of Tracer and serially diluted test compound.</p><ol class="npwthree"><li><p>Prepare 20x fluorescent Tracer solution.</p><ol class="npwfour"><li><p>Using the Tracer saturation binding analysis described above choose a fixed concentration between the EC<sub>60</sub> to EC<sub>80</sub> values.</p></li><li><p>First prepare a 200x Tracer solution in DMSO.</p></li><li><p>Dilute the Tracer Dilution Buffer (TDB) 3-fold in Opti-MEM without phenol red to generate a 0.3x TDB.</p></li><li><p>Dilute the 200x Tracer 10-fold in 0.3x TDB to generate a 20x Tracer solution.</p></li></ol></li><li><p>Prepare 20x serially diluted unmodified test compound.</p><p><em> Note: It is recommended to use 5-10x K<sub>I</sub> as the highest final concentration. </em> </p><ol class="npwfour"><li><p>Prepare a 10% (vol/vol) DMSO solution in Opti-MEM without phenol red.</p></li><li><p>Prepare a 20x serially diluted unmodified test compound in 10% DMSO. Start with 2- to 3-fold serial dilutions and limit the number of dilutions to 4 or 5 to minimize the time intervals between reads.</p><p><em>Note: Make sure to include within the serial dilution a no unmodified test compound control.</em> </p></li></ol></li><li><p>Immediately prior to kinetic measurements, add 11 Âµl/well of 20x serially diluted unmodified test compound and 11 Âµl/well of 20x Tracer solution.</p></li><li><p>Mix briefly on an orbital plate shaker and start kinetic reads on a NanoBRET-compatible luminometer, measuring donor emission (<em>e.g.</em>, 450 nm) and acceptor emission (<em>e.g.</em>, 610 nm or 630 nm). To capture rapid binding, use 0 min intervals between reads.</p></li></ol><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201220/20201220172612_5404.jpg"/><br/><strong>Figure 8. A 96-well plate set-up for competitive binding kinetics.</strong> Cells treated with a limited serial dilution of unmodified test compound (4-5 concentrations including â0â unmodified test compound) in the presence of a fluorescent Tracer at a fixed EC<sub>60</sub>-EC<sub>80</sub> concentration. Control wells treated with additional excess of unmodified compound.<ol class="npwtwok"></ol><br/></li><li><p>Determine BRET ratios and process data.</p><ol class="npwthree"><li><p>To generate raw BRET ratio values, divide the acceptor emission value (<em>e.g.</em>, 610 nm) by the donor emission value (<em>e.g.</em>, 450 nm).</p></li><li><p>Convert raw BRET units to milli BRET units (mBU) by multiplying each raw BRET value by 1,000. NanoBRET equation:</p><p><strong> BRET (mBU) = (Acceptor<sub>sample</sub>/Donor<sub>sample</sub>) x 1,000 </strong> </p></li><li><p>Determine specific BRET with time by subtracting BRET values in the presence of excess unmodified compound from BRET values in the absence of excess unmodified compound.</p></li><li><p>Plot the data in GraphPad Prism software using the Kinetic competitive binding fitting (Motulsky-Mahan model for kinetics of competitive binding (Motulsky and Mahan, 1984)), which takes in account the concentration of the Tracer as well as its k<sub>on</sub> and k<sub>off</sub> values (see Figure 9 for representative analysis).</p><p>Determine the kinetic constants (k<sub>on</sub> and k<sub>off</sub>) for the test compound.</p></li><li>Calculate binding constant for the unmodified compound (K<sub>D</sub>) for the unmodified test compound (K<sub>off</sub>/k<sub>on</sub>) and residence time (Ï) as 1/k<sub>off</sub>.</li></ol><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3861/bioprotoc-10-24-3861-g009.jpg"/><br/><strong>Figure 9. Representative analysis for binding kinetic of propranolol to transiently expressed HiBiT-Î²<sub>2</sub>-AR in the presence of NanoBRET GPCR Tracer G-3. </strong>HEK293 cells expressing HiBiT-Î²<sub>2</sub>-AR were assayed as indicated in the protocol. Briefly, cells were transfected, grown overnight, and next day treated with HiBiT detection reagent for 15 min prior to the treatment with propranolol titration alongside a fixed EC<sub>80</sub> concentration of NanoBRET GPCR Tracer G-3 (<em>i.e</em>., the previously described propranolol-BODIPY (Boursier <em>et al.</em>, 2020)). Kinetic measurements of filtered luminescence were taken using GloMax Discover Microplate Reader (Promega) equipped with a 450 nm (8-nm bandpass) filter (donor) and a 600-nm long pass filter (acceptor) (n = 3; Error bars indicate SD).<br/><p class="pptt" id="biaoti39381">### Recipes</p><ol class="npwtwok"><li><p>DMEM Cell Culture Medium</p><p><strong>Componentâ                                    Final Concentration</strong> </p><p>DMEMâ                                                      90%</p><p>FBSâ                                                            10%</p><p>100x Penicillin-streptomycinâ                1% (1x)</p></li><li><p>Opti-MEM Assay Medium</p><p><strong>Componentâ                                        Final Concentration</strong> </p><p>Opti-MEM <strong>WITHOUT PHENOL RED</strong>â           98 %</p><p>FBSâ                                                                   2%</p><p>100x Penicillin-streptomycin                         1% (1x)</p></li></ol><p class="pptt" id="biaoti39382">### Acknowledgments</p><p>This protocols were derived from the work of Boursier <em>et al.</em>, which was published in The Journal of Biological Chemistry (     Boursier <em>et al.</em>, 2020    )</p><p class="pptt" id="biaoti39383">### Competing interests</p><p>All authors were Promega employees at the time the protocols were developed.</p><p class="pptt" id="biaoti39384">### References</p><ol class="npwtwok"><li>Boursier, M. E., Levin, S., Zimmerman, K., Machleidt, T., Hurst, R., Butler, B. L., Eggers, C. T., Kirkland, T. A., Wood, K. V. and Friedman Ohana, R. (2020). <a href="https://pubmed.ncbi.nlm.nih.gov/32107310/" target="_blank">The luminescent HiBiT peptide enables selective quantitation of G protein-coupled receptor ligand engagement and internalization in living cells.</a> <em>J Biol Chem</em> 295(15): 5124-5135.</li><li>Cheng, Y. and Prusoff, W. H. (1973). <a href="https://pubmed.ncbi.nlm.nih.gov/4202581/" target="_blank">Relationship between the inhibition constant (K1) and the concentration of inhibitor which causes 50 per cent inhibition (I50) of an enzymatic reaction.</a> <em>Biochem Pharmacol</em> 22(23): 3099-3108.</li><li>Ciruela, F. (2008). <a href="https://pubmed.ncbi.nlm.nih.gov/18602005/" target="_blank">Fluorescence-based methods in the study of protein-protein interactions in living cells.</a> <em>Curr Opin Biotechnol</em> 19(4): 338-343.</li><li>Copeland, R. A. (2016). <a href="https://www.nature.com/articles/nrd.2015.18" target="_blank">The drugâtarget residence time model: a 10 year retrospective.</a> <em>Nat Rev Drug Discov</em> 15(2): 87-95.</li><li>Fang, Y. (2012). <a href="https://pubmed.ncbi.nlm.nih.gov/22860803/" target="_blank">Ligand-receptor interaction platforms and their applications for drug discovery.</a> <em>Expert Opin Drug Discov</em> 7(10): 969-988.</li><li>Flanagan, C. A., (2016). <a href="https://pubmed.ncbi.nlm.nih.gov/26928545/" target="_blank">GPCR-radioligand binding assays.</a> <em>Methods Cell Biol</em> 132: 191-215.</li><li>Hall, M. P., Unch, J., Binkowski, B. F., Valley, M. P., Butler, B. L., Wood, M. G., Otto, P., Zimmerman, K., Vidugiris, G., Machleidt, T., Robers, M. B., Benink, H. A., Eggers, C. T., Slater, M. R., Meisenheimer, P. L., Klaubert, D. H., Fan, F., Encell, L. P. and Wood, K. V. (2012). <a href="https://pubmed.ncbi.nlm.nih.gov/22894855/" target="_blank">Engineered luciferase reporter from a deep sea shrimp utilizing a novel imidazopyrazinone substrate. </a><em>ACS Chem Biol</em> 7(11): 1848-1857.</li><li>Hulme, E. C. and Trevethick, M. A. (2010). <a href="https://pubmed.ncbi.nlm.nih.gov/20132208/" target="_blank">Ligand binding assays at equilibrium: validation and interpretation.</a> <em>Br J Pharmacol</em> 161(6): 1219-1237.</li><li>Kenakin, T., (2010). <a href="https://pubmed.ncbi.nlm.nih.gov/20590598/" target="_blank">Being mindful of seven-transmembrane receptor'guests' when assessing agonist selectivity.</a> <em>Br J Pharmacol</em> 160(5): 1045-1047.</li><li>Motulsky, H. J. and Mahan, L. C. (1984). <a href="https://pubmed.ncbi.nlm.nih.gov/6708928/" target="_blank">The kinetics of competitive radioligand binding predicted by the law of mass action.</a> <em>Mol Pharmacol</em> 25(1): 1-9.</li><li>Robers, M. B., Vasta, J. D., Corona, C. R., Ohana, R. F., Hurst, R., Jhala, M. A., Comess, K. M. and Wood, K. V. (2019). <a href="https://pubmed.ncbi.nlm.nih.gov/30519940/" target="_blank">Quantitative, real-time measurements of intracellular target engagement using energy transfer.</a> <em>Methods Mol Biol</em> 1888: 45-71.</li><li>Santos, R., Ursu, O., Gaulton, A., Bento, A. P., Donadi, R. S., Bologa, C. G., Karlsson, A., Al-Lazikani, B., Hersey, A., Oprea, T. I. and Overington, J. P. (2017). <a href="https://pubmed.ncbi.nlm.nih.gov/27910877/" target="_blank">A comprehensive map of molecular drug targets. </a><em>Nat Rev Drug Discov</em> 16(1): 19-34.</li><li>Schwinn, M. K., Machleidt, T., Zimmerman, K., Eggers, C. T., Dixon, A. S., Hurst, R., Hall, M. P., Encell, L. P., Binkowski, B. F. and Wood, K. V. (2018). <a href="https://pubmed.ncbi.nlm.nih.gov/28892606/" target="_blank">CRISPR-mediated tagging of endogenous proteins with a luminescent peptide.</a> <em>ACS Chem Biol</em> 13(2): 467-474.</li><li>Soave, M., Kellam, B., Woolard, J., Briddon, S. J. and Hill, S. J. (2019). <a href="https://journals.sagepub.com/doi/full/10.1177/2472555219880475" target="_blank">NanoBiT complementation to monitor agonist-induced adenosine A<sub>1</sub> receptor internalization.</a> <em>SLAS Discov </em>25(2): 186-194.</li><li>Soave, M., Stoddart, L. A., Brown, A., Woolard, J. and Hill, S. J. (2016). <a href="https://pubmed.ncbi.nlm.nih.gov/27588207/" target="_blank">Use of a new proximity assay(NanoBRET) to investigate the ligand-binding characteristics of three fluorescent ligands to the human beta1-adrenoceptor expressed in HEK-293 cells.</a> <em>Pharmacol Res Perspect</em> 4(5): e00250.</li><li>Stoddart, L. A., Johnstone, E. K. M., Wheal, A. J., Goulding, J., Robers, M. B., Machleidt, T., Wood, K. V., Hill, S. J. and Pfleger, K. D. G. (2015). <a href="https://pubmed.ncbi.nlm.nih.gov/26030448/" target="_blank">Application of BRET to monitor ligand binding to GPCRs.</a> <em>Nat Methods</em> 12(7): 661-663.</li><li>Stoddart, L. A., White, C. W., Nguyen, K., Hill, S. J. and Pfleger, K. D. G. (2016). <a href="https://pubmed.ncbi.nlm.nih.gov/26317175/" target="_blank">Fluorescence- and bioluminescence-based approaches to study GPCR ligand binding.</a> <em>Br J Pharmacol</em> (173): 3028-3037.</li><li>Sykes, D. A., Stoddart, L. A., Kilpatrick, L. E. and Hill, S. J. (2019). <a href="https://pubmed.ncbi.nlm.nih.gov/30738950/" target="_blank">Binding kinetics of ligands acting at GPCRs.</a> <em>Mol Cell Endocrinol</em> 485: 9-19.</li></ol></li><br/></ol></ol></ol></li></ol>