<body>
            <h1>Lipid Droplet Isolation from <em>Arabidopsis thaliana</em> Leaves</h1>
            <p><strong>Authors:</strong> Yovanny Izquierdo, RubÃ©n FernÃ¡ndez-Santos, TomÃ¡s CascÃ³n, Carmen Castresana</p>
            <h2>Abstract</h2>
            <p><p>Lipid droplets (LDs) are neutral lipid aggregates surrounded by a phospholipid monolayer and specific proteins. In plants, they play a key role as energy source after seed germination, but are also formed in vegetative tissues in response to developmental or environmental conditions, where their functions are poorly understood. To elucidate these, it is essential to isolate LDs with good yields, while retaining their protein components. LD isolation protocols are based on their capacity to float after centrifugation in sucrose gradients. Early strategies using stringent conditions and LD-abundant plant tissues produced pure LDs where core proteins were identified. To identify more weakly bound LD proteins, recent protocols have used low stringency buffers, but carryover contaminants and low yields were often a problem. We have developed a sucrose gradient-based protocol to isolate LDs from Arabidopsis leaves, using Tween-20 and fresh tissue to increase yield. In both healthy and bacterially-infected Arabidopsis leaves, this protocol allowed to identify LD proteins that were later confirmed by microscopy analysis.</p></p>
            <p><strong>Keywords:</strong> Lipid droplet isolation, Triacylglycerides, Caleosin, Leaf senescence, Plant-pathogen interaction</p>
            <h2>Background</h2><p>Lipid droplets (LDs) are organelles composed by a neutral lipid core surrounded by a phospholipid monolayer where LD proteins are inserted (Tzen <em>et al.</em>, 1993). LDs were first described as stable compartments for storage of high-energy lipids in specialized tissues. However, recent reviews describe LDs as highly dynamic organelles present in most cell types, with functions that transcend mere energy storage (Shimada <em>et al.</em>, 2018; Huang, 2018; Shao <em>et al.</em>, 2019).<br/><br/>In plants, early studies on LDs were conducted mostly in oilseeds, where they are very abundant and act as energy source in the post-germination stage, before photosynthetic capacity is acquired (Yatsu and Jacks, 1972; Huang, 1996; Tzen <em>et al.</em>, 1997). Subsequent works revealed that these organelles are also formed in pollen grains (Ischebeck, 2016), senescent leaves (Brocard <em>et al.</em>, 2017) and pathogen-infected tissues (Shimada <em>et al.</em>, 2014; Fernández-Santos <em>et al.</em>, 2020), although their functions are poorly understood. It is therefore imperative to have efficient protocols for LD isolation from vegetative tissues, where their abundance is comparatively low.<br/><br/>LDs are the least dense cellular organelles, hence, most isolation protocols are based on their capacity to float after centrifugation. Tissue is homogenized in a suitable buffer, and then the extract is clarified and centrifuged with an overlay of a less dense buffer (floating buffer). Upon centrifugation, LDs float from the extraction buffer and through the floating buffer, to accumulate in a fat pad visible on top of the tube.<br/><br/>The choice of floating and extraction buffers poses a dilemma to the experimenter. Their stringency and the number of centrifugations will establish the balance between obtaining cleaner LDs, but devoid of some loosely bound, <em>bona fide</em> LD proteins, or more “complete” LDs with carryover contaminants attached during preparation (Huang, 2018). Early protocols from oilseeds often included harsh buffers, containing detergents, organic solvents, chaotropic agents or high salt concentrations (Tzen <em>et al.</em>, 1997; Jolivet <em>et al.</em>, 2004; Buchanan-Wollaston <em>et al.</em>, 2005; Katavic <em>et al.</em>, 2006). As LDs are very abundant in this tissue, such conditions guaranteed high yields of pure LDs where core proteins were identified (Tzen <em>et al.</em>, 1990; Chen <em>et al.</em>, 1999; Lin <em>et al.</em>, 2002).<br/> <br/>In order to identify less tightly bound LD proteins, later protocols designed for proteomics applications have tended to less stringent conditions, as in Horn <em>et al.</em> (2013) and Brocard <em>et al.</em> (2017). In these cases, an enrichment factor was calculated for each protein between LDs and other fractions to rule out contaminants. “True” LD proteins were thus defined as those above an arbitrary threshold. However, proteins with dual localization can be missed with this strategy, especially if the studied condition is dynamic and LD formation is transient. Unfortunately, there is no perfect negative control, and “true” LD components will always need to be confirmed by complementary methods such as microscopy. Another problem in previous protocols from leaves has been the large amount of sample needed for extraction, due to lower LD abundance in this tissue compared with seeds or fruit mesocarp (Brocard <em>et al.</em>, 2017).<br/> <br/>Based on this background, we developed an LD isolation protocol from Arabidopsis leaves, with the following aims: i) simplicity ii) maximization of LD yield and iii) low to medium stringency in order to identify loosely bound LD proteins while minimizing contaminants. Briefly, we extract LDs in a 0.6M sucrose buffer with 0.1% Tween-20 followed by two successive flotations in the same buffer with less sucrose (0.4 M and 0.2 M). The amount of Tween-20 can be variable and is critical in two aspects: increasing stringency and LD yield. The other critical point in our hands was to use fresh tissue, as yield and integrity of LDs is affected when extracted from frozen samples. We obtained approx. 1,700 LDs/mg fresh tissue using 6-week-old wild type plants and were able to identify and confirm known LD proteins such as CLO3 and α-DOX1, as well as proteins newly associated to LDs in plants such as GPAT4, GPAT8 and PAD3 (Fernández-Santos <em>et al.</em>, 2020). LD quality can be checked by microscopy or flow cytometry. By using transgenic CLO3:GFP Arabidopsis plants, we were also able to follow the isolation process by western blot using an anti-GFP antibody.</p>
        </body>

<p class="pptt" id="biaoti39340">## Materials and Reagents</p><ol class="npwtwok"><li><p>Glass Pasteur pipettes (Deltalab, catalog number: <a class="blackNoUnder" target="_blank"> 702 </a> )</p></li><li><p>5 ml Open-top polypropylene tubes (Beckman Coulter, catalog number: <a class="blackNoUnder" target="_blank"> 326819 </a> )</p></li><li><p>(Optional) 13 ml Open-top polypropylene tubes (Beckman Coulter, catalog number: <a class="blackNoUnder" target="_blank"> 331372 </a> )</p></li><li><p>(Optional) Liquid nitrogen</p></li><li><p>Miracloth (Merck Milipore, catalog number: <a class="blackNoUnder" target="_blank"> 475855-1R </a> )</p></li><li><p>Sea sand (Merck, catalog number: <a class="blackNoUnder" target="_blank"> 1.07712.1000 </a> )</p></li><li><p>Sucrose (Carlo Erba Reagents, catalog number: <a class="blackNoUnder" target="_blank"> 365157 </a> )</p></li><li><p>NaH<sub>2</sub>PO<sub>4</sub>.<strong></strong>H<sub>2</sub>O (Merck, catalog number: <a class="blackNoUnder" target="_blank"> 1.06346.0500 </a> )</p></li><li><p>NaCl (VWR chemicals, catalog number: <a class="blackNoUnder" target="_blank"> 27810.364 </a> )</p></li><li><p>NaOH (AppliChem, catalog number: <a class="blackNoUnder" target="_blank"> 131687.1211 </a> )</p></li><li><p>Tween-20 (Sigma Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P1379-500ml </a> )</p></li><li><p>PMSF (Sigma Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 93482-50ml-F </a> )</p></li><li><p>Complete<sup>TM</sup> Protease Inhibitor Cocktail (Roche, catalog number: <a class="blackNoUnder" target="_blank"> 11697498001 </a> )</p></li><li><p>BODIPY<sup>493/503</sup> (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> D3922 </a> )</p></li><li><p>Nile Red (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> N1142 </a> )</p></li><li><p>DMSO (Merck, catalog number: <a class="blackNoUnder" target="_blank"> 1.02952.1000 </a> )</p></li><li><p>Methanol (Sigma Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 32213-2.5L-M </a> )</p></li><li><p>(Optional) 35S::CLO3:GFP transgenic Arabidopsis plants</p></li><li><p>Base buffer (600 ml) (see Recipes)</p></li><li><p>Sucrose buffers (100 ml each) (see Recipes)</p><p>1.2 M Sucrose buffer</p><p>0.8 M Sucrose buffer</p><p>0.4 M Sucrose buffer</p></li><li><p>Supplemented base buffer (300 ml) (see Recipes)</p></li><li><p>Phosphate buffer (100 ml) (see Recipes)</p></li><li><p>Stocks (see Recipes)</p><p>1 mg/ml BODIPY<sup>495/503</sup> </p><p>1 mg/ml Nile Red</p><p>50 mM PMSF</p></li></ol><p class="pptt" id="biaoti39341">### Equipment</p><ol class="npwtwok"><li><p>P1000 pipette</p></li><li><p>Funnel</p></li><li><p>Mortar and pestle</p></li><li><p>Scissors</p></li><li><p>Ultracentrifuge (Beckman Coulter, model <a class="blackNoUnder" target="_blank"> Optima L-100 XP </a> )</p></li><li><p>Rotor SW55 Ti (Beckman Coulter, catalog number: <a class="blackNoUnder" target="_blank"> 342194 </a> )</p></li><li><p>(Optional) Rotor SW41 Ti (Beckman Coulter, catalog number: <a class="blackNoUnder" target="_blank"> 331362 </a> )</p></li><li><p>Fluorescence microscope (Leica, model <a class="blackNoUnder" target="_blank"> TCS-SP8 </a> )</p></li><li><p>pH Meter (Hach, model SensION<sup>TM</sup> pH 3, catalog number: <a class="blackNoUnder" target="_blank"> LPV2000.98.0002 </a> )</p></li><li><p>(Optional) Flow cytometer (Beckman Coulter, model Galios<sup>TM</sup>)</p></li><li><p>(Optional) Standard SDS-PAGE and Western blot equipment</p></li></ol><p class="pptt" id="biaoti39342">### Procedure</p><ol class="npwone"><li><p>Buffer preparation (please refer to the end of the protocol for detailed recipes)</p><ol class="npwtwok"><li><p>Before the day of extraction, prepare the following base buffers and keep at 4 Â°C (may be kept for several weeks):</p><ol class="npwthree"><li><p>Base buffer: 10 mM NaH<sub>2</sub>PO<sub>4</sub>, 150 mM NaCl, pH 7.5. Autoclave and keep at 4Â°C before use (Further components will be added on the day of extraction).</p></li><li><p>Sucrose buffers: Dissolve sucrose in Base buffer to make three sucrose buffers at concentrations 1.2 M, 0.8 M and 0.4 M. Filter-sterilize or autoclave and keep at 4 Â°C.</p><p><em>Note: Be careful when autoclaving sucrose solutions, as they caramelize easily. Autoclave for no more than 15 minutes at a maximum of 121 Â°C.</em> </p></li><li><p>Phosphate buffer: 10 mM NaH<sub>2</sub>PO<sub>4</sub>, pH 7.5. Prepare and autoclave.</p></li><li><p>Nile red and BODIPY<sup>493/503</sup> stocks. Keep at -20 Â°C.</p></li></ol></li><li><p>On the day of extraction, complete buffer preparation as follows:</p><ol class="npwthree"><li><p>Supplement Base buffer with 0.2% (v/v) Tween-20, 2 mM PMSF and 2x Protease inhibitor cocktail.</p></li><li><p>Mix equal volumes of supplemented Base buffer and each Sucrose buffer (1.2 M, 0.8 M and 0.4 M) to prepare Extraction buffer, Floating buffer 1 and Floating buffer 2, respectively.</p></li></ol><br/><p>In the end, the final buffer composition will be:</p><ol class="npwbullet"><li><p>Extraction buffer: 0.6 M Sucrose, 10 mM NaH<sub>2</sub>PO<sub>4</sub>, pH 7.5, 150 mM NaCl, 0.1% (v/v) Tween-20, 1 mM PMSF, 1x Protease inhibitor cocktail.</p></li><li><p>Floating buffer 1: 0.4 M Sucrose, 10 mM NaH<sub>2</sub>PO<sub>4</sub>, pH 7.5, 150 mM NaCl, 0.1% (v/v) Tween-20, 1 mM PMSF, 1x Protease inhibitor cocktail.</p></li><li><p>Floating buffer 2: 0.2 M Sucrose, 10 mM NaH<sub>2</sub>PO<sub>4</sub>, pH 7.5, 150 mM NaCl, 0.1% (v/v) Tween-20, 1 mM PMSF, 1x Protease inhibitor cocktail.</p></li></ol></li></ol><p><br/></p></li><li><p>Lipid droplet extraction (Figure 1)</p><ol class="npwtwok"><li><p>Precool mortar and pestle on ice.</p><p><em>Tip: A small amount of liquid nitrogen may be added to accelerate cooling, but not immediately before grinding, to avoid tissue freezing. In order to increase LD yield and quality, it is important that the tissue remains cool, but not frozen, during grinding.</em> </p></li><li><p>Collect tissue (2.5 g of fully expanded leaves). Using a scissor, cut leaves into small pieces (about 0.5 to 1 cm) and place them on the cold mortar.</p><p><em>Note: LDs recovery will depend on experimental conditions. The starting amount proposed here (2.5 g) is sufficient for leaves under LD-inducing conditions, such as senescence and pathogen infection (FernÃ¡ndez-Santos et al., 2020). If this amount is not sufficient in your case, scale up to maximum 6.5 g per sample by using rotor SW41-Ti and 13 ml tubes.</em> </p></li><li><p>Add 5 g of sea sand (abrasive) and 5 ml of precooled extraction buffer. Grind for 2-3 min until a homogeneous mix is formed. Keep the mortar on ice while grinding.</p></li><li><p>Pass the homogenate through 3 layers of Miracloth placed on a funnel, and collect the filtrate in a precooled 10 ml tube (you will have to squeeze the mix to extract the liquid). This should yield 2-3 ml of filtrate.</p></li><li><p>Place 2.5 ml of filtrate in the ultracentrifuge tube using a P1000 pipette. On top of that, gently lay 2 ml of Floating buffer 1, taking care not to mix both phases.</p><p><em>Tip: Use tube walls to carefully add the floating buffer. This will limit phase mixing.</em> </p><p>After completing this step, you should see a green bottom phase, an intermediate thin layer in which both phases are mixed and a clear top phase.</p></li><li><p>Centrifuge 1.5 h at 100,000 <em>x g</em> and 4 Â°C (32,000 rpm if working with an SW55Ti rotor) with maximum acceleration and no brake. In this step, less dense LDs will float from the extract phase and through the Floating buffer 1, to accumulate as a milky band visible on top of the tube (Floating fraction 1, FF1).</p></li><li><p>Collect FF1 very carefully, by aspiring with a P1000 pipette or a glass Pasteur pipette (~0.5-1 ml, maximum 1.5 ml) and place in a new ultracentrifuge tube.</p><p><em>Note: Avoid plastic Pasteur pipettes, as LDs will stick to the walls making them hard to collect.</em> </p></li><li><p><strong>Optional</strong>: To monitor extraction, also collect an aliquot from the bottom phase (Bottom fraction 1, BF1).</p></li><li><p>If several samples are being processed, balance the volumes of FF1 fractions by adding Floating Buffer 1. On top of it, gently lay Floating buffer 2 taking the same precautions as in Step B5. As the total volume loaded into the tubes must be 4.5 ml, the added volume of Floating buffer 2 will be equal to 4.5 ml â Vol<sub>FF1</sub>.</p><p><em>Note: Liquid volume in the ultracentrifuge tubes used here should be around 4.5 ml. Higher volumes (close to 5 ml) may cause sample spill, whereas lower volumes (e.g., less than 4 ml) may cause tube collapse at high g forces.</em> </p></li><li><p>Centrifuge as in Step B6 and collect LDs from the floating milky band as in Step B7 (Floating fraction 2, FF2). FF2 can already be used as LD fraction (Figure 2A).</p></li><li><p><strong>Optional:</strong> To monitor extraction, also collect an aliquot of the bottom phase (Bottom fraction 2, BF2).</p></li><li><p>If you need more concentrated LDs, place collected FF2 in a 1.5 ml tube and centrifuge for 1.5 h in a table centrifuge at 10,000 <em>x g</em> and 4 Â°C. After centrifugation, LDs will accumulate again on a top layer. Remove as much buffer as possible from the bottom of the tube with a glass pipette, until only the upper milky fraction remains. LDs can be concentrated to a volume as small as 50-100 Î¼l (Figure 2B).</p></li><li><p><strong>Optional:</strong> Place the removed buffer in a new 1.5 ml tube and repeat the previous step to increase recovery.</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3867/bioprotoc-10-24-3867-g001.jpg"/><br/><strong>Figure 1. Protocol overview.</strong> Collect leaves, cut into small pieces and immediately place in a precooled mortar. After addition of abrasive and Extraction buffer, grind the sample and then filter through 3 layers of Miracloth. Place the filtrate in a 5 ml centrifuge tube, overlay Floating buffer 1 and centrifuge the resulting gradient. LDs will accumulate on top of the tube. Place this LD fraction (FF1) in a new tube and repeat the process using Floating buffer 2. If needed, concentrate the resulting LD fraction (FF2) by centrifugation in a table centrifuge.</p><p><br/></p></li></ol></li><li><p>LD quality check</p><p>The quality of your final LD preparation (or intermediate fractions) can be assessed by several methods listed below. The simplest is to observe the stained LDs under a fluorescence microscope. Optionally, you can also estimate yield, purity and homogeneity of your LD preparation by flow cytometry analysis. Western blot can also be used to set up your protocol, if you have an antibody against an LD protein.</p><ol class="npwtwok"><li><p>By microscopy</p><p>Extracted LDs can be stained with lipid-specific dyes such as BODIPY<sup>493/503</sup> or Nile Red; BODIPY<sup>493/503</sup> is more sensitive and specific (Gocze and Freeman, 1994). In both cases, proceed as follows:</p><ol class="npwthree"><li><p>Prepare working dye solutions: Dilute dye stocks 1,000 times to 1 Î¼g/ml in Phosphate buffer.</p></li><li><p>Gently mix equal volumes (<em>e.g.</em>, 5 Î¼l) of the final LD fraction and the working dye solution. Incubate for 15 min in the dark and observe in the fluorescence microscope. For BODIPY<sup>493/503</sup>, use a 488-nm excitation laser line and collect the emission signal in the spectrum of 500 to 540 nm. For Nile Red, excite with 550 nm laser and collect emission between 562 and 621 nm. (Alternatively, if this exact parameters cannot be set in your microscope, use typical GFP settings for BODIPY<sup>493/503</sup> and RFP settings for Nile Red).<br/>    Compare fluorescence image with a bright filed image. You should see round, 0.5-2 Î¼m structures overlapping in both pictures [Figure 2C-2D and           Horn <em>et al.</em> (2013)].</p></li></ol></li><li><p>By flow cytometry</p><p>As a complement to microscopy, BODIPY<sup>493/503</sup>-stained and unstained LDs can be analyzed in a flow cytometer to estimate yield, purity and possible different populations in the final LD preparation.</p><ol class="npwthree"><li><p>Prepare working dye solution: Dilute BODIPY<sup>493/503</sup> stock 10 times (to 100 Î¼g/ml) in Phosphate buffer.</p></li><li><p>Dilute 5 Î¼l of the final LD fraction with 195 Î¼l Phosphate buffer and add 1 Î¼l of 100 Î¼g/ml BODIPY<sup>493/503</sup>. Mix gently and stain for 15 min in the dark. In parallel, prepare an unstained control.</p></li><li><p>Analyze stained and unstained samples on a Galios<sup>TM</sup> flow cytometer. Excitation laser: 488 nm at 20 mW. Emission filter: 525-540 nm bandpass. If a good quality preparation is obtained, LDs should appear as a single cluster in a Size vs. Complexity plot. In a fluorescence histogram, BODIPY<sup>493/503 </sup>should produce a fluorescence shift of the majority of particles when comparing stained vs unstained samples (FernÃ¡ndez-           Santos <em>et al.</em>, 2020). The number of recorded events (particles) in the fluorescence channel can be used to estimate LD yield (as LDs per mg fresh tissue).</p></li></ol></li><li><p>By Western blot</p><p>To set up your protocol, it is very useful to have a transgenic line with a tagged LD marker protein that can be detected by western blot (or an antibody against the LD protein <em>per se</em>). This way you can monitor LD enrichment (or depletion) in different fractions and adjust protocol conditions accordingly. As an example, Figure 2E shows LD extraction from senescent leaves of 35S::CLO3:GFP plants, analyzed by western blot with an anti-GFP antibody.</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3867/bioprotoc-10-24-3867-g002.jpg"/><br/><strong>Figure 2. Lipid droplet (LD) extraction from leaves of senescent plants.</strong> A. Image of sucrose gradient after second ultracentrifugation, with Floating fraction 2 (FF2) on top. B. Final LD preparation from wild type senescent plants. C. Lipid droplets from (B), stained with BODIPY<sup>493/503</sup>. D. LDs from 35S::CLO3:GFP plants. Nile Red-stained lipids are shown in the core (red), whereas CLO3:GFP locates at the LDs surface (green). E. Protocol tracking of LDs isolated in (D) by western blot with an anti-GFP antibody; BF: bottom fractions, FF: floating fractions. Scale bar = 2 Î¼m.</p></li></ol></li></ol><p class="pptt" id="biaoti39343">### Notes</p><p>General comments:</p><p>All steps of the protocol must be carried out in the cold. In our hands, the concentration of Tween-20 used (0.1%) increased yield by 10 fold, and led to identification of LD proteins that were later confirmed by microscopy (FernÃ¡ndez-     Santos <em>et al.</em>, 2020). However, this concentration may be adjusted depending on your objectives and experimental design (a reasonable range would be 0.02 to 0.2%). Addition of Tween-20 increases LD yield and reduces contaminant carryover, but may also detach weakly bound LD proteins.</p><p class="pptt" id="biaoti39344">### Recipes</p><p><em>Note: Please note that buffer volumes below are sufficient to process 50 samples of 2.5 g each. If you intend to process less samples, it is advisable to scale down the volumes accordingly.</em> </p><ol class="npwtwok"><li><p>Base buffer (600 ml)</p><p>10 mM NaH<sub>2</sub>PO<sub>4</sub>, 150 mM NaCl, pH 7.5</p><p>NaH<sub>2</sub>PO<sub>4</sub>Â·<strong></strong>H<sub>2</sub>O: 0.83 g</p><p>NaCl: 5.27 g</p><ol class="npwthree"><li><p>Add water to close to 600 ml</p></li><li><p>Adjust pH to 7.5 with 1 M NaOH</p></li><li><p>Add water to 600 ml</p></li></ol></li><li><p>Sucrose buffers (100 ml each)</p><p><strong>1.2 M Sucrose buffer</strong> </p><p>Dissolve 41 g sucrose in 50 ml Base buffer. After dissolving, adjust volume to 100 ml with Base buffer.</p><p><strong>0.8 M Sucrose buffer</strong> </p><p>Dissolve 27.4 g sucrose in 50 ml Base buffer. After dissolving, adjust volume to 100 ml with Base buffer.</p><p><strong>0.4 M Sucrose buffer</strong> </p><p>Dissolve 13.7 g sucrose in 50 ml Base buffer. After dissolving, adjust volume to 100 ml with Base buffer.</p></li><li><p>Supplemented base buffer (300 ml)</p><p><u>On the day of extraction</u>: Dissolve six tablets of cOmplete<sup>TM</sup> Protease Inhibitor Cocktail in 300 ml of Base buffer. Add 12 ml of 50 mM PMSF and 600 Î¼l of Tween-20.</p><p><em>Tip: Tween-20 is a viscous liquid, difficult to pipette. To ensure you add the right volume, cut the end of your P1000 pipette tip with a scissor before loading. This will enlarge the pipette hole. After unloading, rinse the pipette tip several times in the solution, as Tween-20 may stick to the walls.</em> </p></li><li><p>Phosphate buffer (100 ml)</p><p>10 mM NaH<sub>2</sub>PO<sub>4</sub>, pH 7.5</p><ol class="npwthree"><li><p>Dissolve 0.138 g NaH<sub>2</sub>PO<sub>4</sub>Â·<strong></strong>H<sub>2</sub>O in 90 ml water</p></li><li><p>Adjust pH to 7.5 with 0.1 M NaOH and complete to 100 ml</p></li></ol></li><li><p>Stocks</p><p><strong> 1 mg/ml BODIPY<sup>495/503</sup></strong> </p><p>Dissolve 0.5 mg in 500 Î¼l DMSO. Wrap tube in aluminum paper and store at -20 Â°C.</p><p><strong>1 mg/ml Nile Red</strong> </p><p>Dissolve 0.5 mg Nile Red in 500 Î¼l methanol. Wrap tube in aluminum paper and store at -20 Â°C.</p><p><strong>50 mM PMSF</strong> </p><p>Dissolve 0.174 g of PMSF in 20 ml of isopropanol and store at 4 Â°C.</p></li></ol><p class="pptt" id="biaoti39345">### Acknowledgments</p><p>This work was supported by research grants BIO2015-68130-R, from the Spanish Ministry of Economy and Competitiveness/FEDER, and RTI2018-097102-B-100, from the Ministry of Science, Innovation and Universities, to CC. This protocol was derived from our research paper (FernÃ¡ndez-     Santos <em>et al.</em>, 2020).</p><p class="pptt" id="biaoti39346">### Competing interests</p><p>The authors have no competing interests to declare.</p><p class="pptt" id="biaoti39347">### References</p><ol class="npwtwok"><li>Brocard, L., Immel, F., Coulon, D., Esnay, N., Tuphile, K., Pascal, S., Claverol, S., Fouillen, L., Bessoule, J. J. and BrÃ©hÃ©lin, C. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28611809" target="_blank">Proteomic analysis of lipid droplets from <em>Arabidopsis</em> aging leaves brings new insight into their biogenesis and functions.</a> <em>Front Plant Sci</em> 8: 894.</li><li>Buchanan-Wollaston, V., Page, T., Harrison, E., Breeze, E., Lim, P. O., Nam, H. G., Lin, J. F., Wu, S. H., Swidzinski, J., Ishizaki, K. and Leaver, C. J. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15860015" target="_blank">Comparative transcriptome analysis reveals significant differences in gene expression and signalling pathways between developmental and dark/starvation-induced senescence in<em> Arabidopsis</em>.</a> <em>Plant J</em> 42(4): 567-585.</li><li>Chen, J. C., Tsai, C. C. and Tzen, J. T. (1999). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10589521" target="_blank">Cloning and secondary structure analysis of caleosin, a unique calcium-binding protein in oil bodies of plant seeds.</a> <em>Plant Cell Physiol</em> 40(10): 1079-1086.</li><li>FernÃ¡ndez-Santos, R., Izquierdo, Y., LÃ³pez, A., MuÃ±iz, L., MartÃ­nez, M., CascÃ³n, T., Hamberg, M. and Castresana, C. (2020). <a href="https://doi.org/10.1093/pcp/pcaa041" target="_blank">Protein profiles of lipid droplets during the hypersensitive defense response of <em>Arabidopsis</em> against pseudomonas infection.</a> <em>Plant and Cell Physiology</em> 61(6): 1144-1157.</li><li>Gocze, P. M. and Freeman, D. A. (1994). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7835165" target="_blank">Factors underlying the variability of lipid droplet fluorescence in MA-10 Leydig tumor cells.</a> <em>Cytometry</em> 17(2): 151-158.</li><li>Horn, P. J., James, C. N., Gidda, S. K., Kilaru, A., Dyer, J. M., Mullen, R. T., Ohlrogge, J. B. and Chapman, K. D. (2013).<a href="http://www.ncbi.nlm.nih.gov/pubmed/23821652" target="_blank"> Identification of a new class of lipid droplet-associated proteins in plants. </a><em>Plant Physiol</em> 162(4): 1926-1936.</li><li>Huang, A. H. (1996). <a href="http://www.ncbi.nlm.nih.gov/pubmed/8934621" target="_blank">Oleosins and oil bodies in seeds and other organs.</a> <em>Plant Physiol</em> 110(4): 1055-1061.</li><li>Huang, A. H. C. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29269574" target="_blank">Plant lipid droplets and their associated proteins: potential for rapid advances.</a> <em>Plant Physiol</em> 176(3): 1894-1918.</li><li>Ischebeck, T. (2016). <a href="https://www.researchgate.net/publication/299403307_Lipids_in_pollen_-_They_are_different" target="_blank">Lipids in pollenâ They are different.</a> <em>Biochim Biophys Acta(BBA)- Mol Cell Biol Lipids</em> 1861(9, Part B):1315â1328.</li><li>Jolivet, P., Roux, E., D'Andrea, S., Davanture, M., Negroni, L., Zivy, M. and Chardot, T. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15246063" target="_blank">Protein composition of oil bodies in <em>Arabidopsis thaliana</em> ecotype WS.</a> <em>Plant Physiol Biochem</em> 42(6): 501-509.</li><li>Katavic, V., Agrawal, G. K., Hajduch, M., Harris, S. L. and Thelen, J. J. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16847873" target="_blank">Protein and lipid composition analysis of oil bodies from two Brassica napus cultivars.</a> <em>Proteomics</em> 6(16): 4586-4598.</li><li>Lin, L. J., Tai, S. S., Peng, C. C. and Tzen, J. T. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11950969" target="_blank">Steroleosin, a sterol-binding dehydrogenase in seed oil bodies.</a> <em>Plant Physiol</em> 128(4): 1200-1211.</li><li>Shao, Q., Liu, X., Su, T., Ma, C. and Wang, P. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31921234" target="_blank">New insights into the role of seed oil body proteins in metabolism and plant development.</a> <em>Front Plant Sci</em> 10: 1568.</li><li>Shimada, T. L., Hayashi, M. and Hara-Nishimura, I. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29203559" target="_blank">Membrane dynamics and multiple functions of oil bodies in seeds and leaves.</a> <em>Plant Physiol</em> 176(1): 199-207.</li><li>Shimada, T. L., Takano, Y., Shimada, T., Fujiwara, M., Fukao, Y., Mori, M., Okazaki, Y., Saito, K., Sasaki, R., Aoki, K. and Hara-Nishimura, I. (2014).<a href="http://www.ncbi.nlm.nih.gov/pubmed/24214535" target="_blank"> Leaf oil body functions as a subcellular factory for the production of a phytoalexin in <em>Arabidopsis.</em></a> <em>Plant Physiol</em> 164(1): 105-118.</li><li>Tzen, J. T., Lai, Y. K., Chan, K. L. and Huang, A. H. (1990). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16667830" target="_blank">Oleosin isoforms of high and low molecular weights are present in the oil bodies of diverse seed species.</a> <em>Plant Physiol</em> 94(3): 1282-1289.</li><li>Tzen, J. T., Peng, C. C., Cheng, D. J., Chen, E. C. and Chiu, J. M. (1997). <a href="https://europepmc.org/article/med/9163529" target="_blank">A new method for seed oil body purification and examination of oil body integrity following germination.</a> <em>J Biochem</em> 121(4): 762-768.</li><li>Tzen, J., Cao, Y., Laurent, P., Ratnayake, C. and Huang, A. (1993). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12231682" target="_blank">Lipids, proteins, and structure of seed oil bodies from diverse species.</a> <em>Plant Physiol</em> 101(1): 267-276.</li><li>Yatsu, L. Y. and Jacks, T. J. (1972). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16658087" target="_blank">Spherosome membranes: half unit-membranes.</a> <em>Plant Physiol</em> 49(6): 937-943.</li></ol>