<body>
            <h1>Assessment of Diadenylate Cyclase and c-di-AMP-phosphodiesterase Activities Using Thin-layer and Ion Exchange Chromatography</h1>
            <p><strong>Authors:</strong> Andreas Latoscha, David Jan Drexler, Gregor  Witte, Natalia  Tschowri</p>
            <h2>Abstract</h2>
            <p><p>All living cells use cyclic nucleotides as second messengers for signal sensing and transduction. Cyclic di-3′,5′-adenosine monophosphate (c-di-AMP) is primarily involved in the control of bacterial and euryarcheal osmoadaptation and is produced by diadenylate cyclases from two molecules of ATP. Specific phosphodiesterases hydrolyze c-di-AMP to the linear phosphoadenylate adenosine 5′-pApA or to AMP. Different methods including high-performance liquid chromatography (HPLC), thin-layer chromatography (TLC) and ion exchange chromatography (IEX) can be used to determine activities of c-di-AMP-synthesizing and degrading enzymes. Here, we describe in detail the TLC and IEX methods adapted for characterization of the diadenylate cyclase DisA and the phosphodiesterase AtaC from <em>Streptomyces venezuelae</em>. TLC allows quick and easy separation of radioactive-labeled substrates and products, while IEX avoids utilization of potentially hazardous radioactive substrates and can be used as a good substitute if an HPLC system is not available. Unlike in TLC assays, samples cannot be analyzed in parallel by using the IEX assay, thus it is more time consuming.</p></p>
            <p><strong>Keywords:</strong> c-di-AMP, Diadenylate cyclase, DAC, DisA, Phosphodiesterase, PDE, AtaC, Enzymatic assay</p>
            <h2>Background</h2><p><p>Cyclic nucleotide second messengers are key molecules in prokaryotic and eukaryotic signaling pathways. Cyclic di-3′,5′-adenosine monophosphate (c-di-AMP) is a bacterial second messenger with many important functions, such as regulation of osmolyte homeostasis, cell wall metabolism, biofilm formation, DNA integrity, sporulation, virulence and growth (Fahmi <em>et al.</em>, 2017). It is of particular interest as it is present in many human pathogens, such as <em>Staphylococcus aureus, Mycobacterium tuberculosis</em> and <em>Streptococcus pneumoniae</em> and was found to be essential for most species under normal growth conditions (Woodward <em>et al.</em>, 2010; Corrigan <em>et al.</em>, 2011; Luo and Helmann, 2012; Mehne <em>et al.</em>, 2013; Whiteley <em>et al.</em>, 2015). Intriguingly, also accumulation of c-di-AMP leads to growth defects (Bai <em>et al.</em>, 2013; Mehne <em>et al.</em>, 2013; Latoscha <em>et al.</em>, 2020). Thus, the levels of this second messenger have to be tightly regulated. c-di-AMP is synthesized from two molecules of ATP by diadenylate cyclases (DAC) and degraded by specific phosphodiesterases (PDE) to 5′-pApA and/or two AMP molecules (Figure 1) (Fahmi <em>et al.</em>, 2017; Yin <em>et al.</em>, 2020).<br/></p><p><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3870/bioprotoc-11-01-3870-g001.jpg"/><br/><strong>Figure 1. Synthesis and degradation of c-di-AMP in <em>S. venezuelae</em>.</strong> The diadenylate cyclase DisA uses magnesium ions (Mg<sup>2+</sup>) as cofactors to synthesize c-di-AMP from two ATP molecules releasing two pyrophosphates (PP<sub>i</sub>) as a byproduct (Witte <em>et al.</em>, 2008). c-di-AMP is degraded to 5′-pApA and two AMP molecules by the phosphodiesterase AtaC, which requires manganese ions (Mn<sup>2+</sup>) as cofactors (Latoscha <em>et al.</em>, 2020) .<br/></p><p><br/>Synthesis of c-di-AMP by DAC domains was identified in five different protein types, termed DisA, CdaA, CdaS, CdaM and CdaZ. Most bacteria contain either DisA or CdaA for c-di-AMP synthesis, while some species, exclusively belonging to the genus <em>Bacillus</em> possess DisA, CdaA and CdaS in parallel (Commichau <em>et al.</em>, 2019). Enzymatic activity of these proteins can be affected by different conditions. The most widespread DAC, CdaA, is regulated by the extracytoplasmic regulator CdaR and the phosphoglucosamine mutase GlmM (Tosi <em>et al.</em>, 2019; Gibhardt <em>et al.</em>, 2020). In contrast, DisA synthesizes c-di-AMP constitutively unless it binds branched or damaged DNA (Witte <em>et al.</em>, 2008), which results in a sporulation delay (Bejerano-Sagie <em>et al.</em>, 2006; Oppenheimer-Shaanan <em>et al.</em>, 2011). Thus, the synthesis of c-di-AMP can be affected by many different environmental conditions, which are still not understood in detail. For c-di-AMP degradation, two major classes of specific PDEs were identified so far, referred to as DHH-type and HD-type PDEs, which are termed according to the conserved catalytic amino acid motifs Asp-His-His and His-Asp in their respective active sites. DHH-type PDEs are further divided into multidomain membrane-coupled proteins like GdpP (Pde1) and soluble standalone catalytic domain proteins of the DhhP-type (Pde2) (Rao <em>et al.</em>, 2010; Bai <em>et al.</em>, 2013; Ye <em>et al.</em>, 2014). The HD-type PDE is a membrane-coupled member of the 7TMR-HD family (7 transmembrane domain receptor with HD domain), termed PgpH (Gundlach <em>et al.</em>, 2015; Huynh <em>et al.</em>, 2015). Both GdpP and PgpH exclusively hydrolyze c-di-AMP into 5′-pApA, which is further degraded into AMP by DhhP or nano-RNases, such as NrnA. DhhP PDEs often occur in parallel to GdpP or PgpH in one species and were shown to be responsible for the second degradation step from 5′-pApA to AMP (Bowman <em>et al.</em>, 2016; Drexler <em>et al.</em>, 2017; Konno <em>et al.</em>, 2018). In contrast, some species, such as <em>M. tuberculosis</em> and <em>Mycobacterium smegmatis</em> only contain DhhP homologs, which were shown to hydrolyze both c-di-AMP and 5′-pApA (Tang <em>et al.</em>, 2015; He <em>et al.</em>, 2016). However, many species from the phylum Actinobacteria, including streptomycetes, do not possess any of the major c-di-AMP-specific PDE classes (Corrigan and Gründling, 2013; Yin <em>et al.</em>, 2020). Instead, <em>Streptomyces venezuelae</em> utilizes an alternative PDE for c-di-AMP degradation. In our recent study, we applied different biochemical approaches to characterize DisA, the sole DAC domain protein encoded in Actinobacteria, as well as AtaC (Actinobacterial PDE targeting c-di-AMP), which is the founding member of a new family of c-di-AMP-specific PDEs and is mainly present in Actinobacteria (Latoscha <em>et al.</em>, 2020).</p><p> <br/>For an accurate characterization of proteins in this signaling pathway, robust and sensitive assays for substrate specificity are required. In the present protocol, we describe two optimized methods for the analysis of c-di-AMP synthesis or degradation: the thin-layer chromatography assay (TLC) of enzymatic assays with radiolabeled nucleotides and the ion exchange chromatography assay (IEX).<br/></p><p> <br/>The TLC assay was established initially to study the activities of <em>Caulobacter crescentus</em> diguanylate cyclase (DGC) PleD and c-di-GMP-specific PDE CC3396 (Paul <em>et al.</em>, 2004; Christen <em>et al.</em>, 2005) based on a method published by Ross <em>et al.</em> (1987). Detailed protocols on DGC and c-di-GMP-specific PDE radioactive assays coupled with TLC were published recently (Kazmierczak, 2017a and 2017b). In the last years, enzymatic assays with radiolabeled nucleotides coupled with TLC have been successfully used for characterization of c-di-GMP-specific enzymes from Gram-negative and Gram-positive bacteria (Kazmierczak <em>et al.</em>, 2006; Bordeleau <em>et al.</em>, 2011; Lindenberg <em>et al.</em>, 2013; Al-Bassam <em>et al.</em>, 2018) as well as of the DAC DisA from <em>Bacillus subtilis</em> and <em>Thermotoga maritima</em> (Witte <em>et al.</em>, 2008; Torres <em>et al.</em>, 2019). In contrast to other methods like high-performance liquid chromatography (HPLC), the TLC assay allows separation of substrate and products from multiple reactions at the same time. Moreover, a direct conclusion about substrate specificity can be drawn if the reaction was supplemented with cold nucleotides as competitors for enzymatic activity (Paul <em>et al.</em>, 2004; Christen <em>et al.</em>, 2005; Lindenberg <em>et al.</em>, 2013). If required, a quantification of radioactive product formation can be performed since only radiolabeled nucleotides, but not the unlabeled competitors, are visualized after development of the TLC (Ross <em>et al.</em>, 1987; Paul <em>et al.</em>, 2004; Christen <em>et al.</em>, 2005). In our recent study, we used the TLC assay to analyze both synthesis of c-di-AMP by <em>S. venezuelae</em> DisA and specificity of c-di-AMP degradation by AtaC (Latoscha <em>et al.</em>, 2020).<br/></p><p><br/>IEX was used as an independent method to further characterize the activity of AtaC (Latoscha <em>et al.</em>, 2020). In this assay, negatively charged molecules (<em>e.g.</em>, nucleotides) bind to a positively charged column material and can be eluted by a salt gradient. Depending on their binding affinity to the column, the molecules elute at different salt concentrations and can be separated. The assay is of particular interest if other chromatography systems for small molecules, such as reverse-phase HPLC (RP-HPLC) are not available. Cyclic dinucleotides and their related reaction intermediates 5’-pNpN or NMP were first separated by ion exchange chromatography by Li <em>et al.</em> (2013) to analyze the reaction products of cyclic GMP-AMP synthase (cGAS). The method was further used by Luecke <em>et al.</em> (2017) and the respective assay was described in detail by Holleufer and Hartmann (2018). The method was adapted for the characterization of a PDE in the degradation pathway of c-di-AMP (Drexler <em>et al.</em>, 2017) and was slightly amended for the present assay for the characterization of AtaC (Latoscha e<em>t al.</em>, 2020). In principle, this IEX assay can be used to analyze the activity of most cyclic dinucleotide synthesizing or degrading enzymes. However, a RP-HPLC system provides a faster and more precise separation of the samples and if it is available, then RP-HPLC would be the preferred chromatographic method to use, as described in several reports (Oppenheimer-Shaanan <em>et al.</em>, 2011; Bai <em>et al.</em>, 2012; Witte <em>et al.</em>, 2013; Huynh <em>et al.</em>, 2015; Tang <em>et al.</em>, 2015).</p><br/></p>
        </body>

<p class="pptt" id="biaoti39477">## Materials and Reagents</p><ol class="npwone"><li><p>Material for buffer exchange (see Notes)</p><ol class="npwtwok"><li><p>Locking clips, 45 mm (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> H277.1 </a> )</p></li><li><p>Dialysis membrane Spectra/Por 7 MWCO 10,000, 45 mm (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> E872.1 </a> )</p></li><li><p>Glass pipettes: 5, 10, 20 and 25 ml (Labdirect, catalog numbers: <a class="blackNoUnder" target="_blank"> 021.01.005 </a> , <a class="blackNoUnder" target="_blank"> 355.050.110 </a> , <a class="blackNoUnder" target="_blank"> 355.050.120 </a> , <a class="blackNoUnder" target="_blank"> 355.050.125 </a> )</p></li><li><p>Pipette tips (Sarstedt, catalog numbers: <a class="blackNoUnder" target="_blank"> 70.760.012, 70.762, 70.3020 </a> and Biozym, catalog number: <a class="blackNoUnder" target="_blank"> VZ0001X </a> )</p></li><li><p>1.5 ml reaction tubes (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 72.690.001 </a> )</p></li><li><p>Double-distilled water (ddH<sub>2</sub>O) (Kerndl, catalog number: <a class="blackNoUnder" target="_blank"> 22501 </a> )</p></li><li><p>Magnesium chloride hexahydrate (MgCl<sub>2</sub>) (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 2189.1 </a> )</p></li><li><p>Manganese (II) chloride tetrahydrate (MnCl<sub>2</sub>) (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> T881.1 </a> )</p></li><li><p>Sodium chloride (NaCl) (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 27810.295DB </a> )</p></li><li><p>Tris-base (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 4855.2 </a> )</p></li><li><p>Glycerol (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 3783.2 </a> )</p></li><li><p>Î²-mercaptoethanol (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M6250 </a> )</p><br/></li></ol></li><li><p>Material for protein concentration determination (see Notes)</p><ol class="npwtwok"><li><p>1.5 ml reaction tubes (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 72.690.001 </a> )</p></li><li><p>Spacer plates and short plates (Bio-Rad, catalog numbers: <a class="blackNoUnder" target="_blank"> 1653310 and 1653308 </a> )</p></li><li><p>Pipette tips (Sarstedt, catalog numbers: <a class="blackNoUnder" target="_blank"> 70.760.012, 70.762, 70.3020 </a> and Biozym, catalog number: <a class="blackNoUnder" target="_blank"> VZ0001X </a> )</p></li><li><p>Tris-base (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 4855.2 </a> )</p></li><li><p>Glycine (Carl Roth, catalog number: 00<a class="blackNoUnder" target="_blank"> 79.4 </a> )</p></li><li><p>Colored Prestained Marker (NEB, catalog number: <a class="blackNoUnder" target="_blank"> P7719S </a> )</p></li><li><p>LMW marker (GE Healthcare, catalog number: <a class="blackNoUnder" target="_blank"> 17-0446-01 </a> )</p></li><li><p>Double-distilled water (ddH<sub>2</sub>O) (Kerndl, catalog number: <a class="blackNoUnder" target="_blank"> 22501 </a> )</p></li><li><p>Rotiphorese (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 3029.1 </a> )</p></li><li><p>Ammonium peroxydisulphate (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 9178.1 </a> )</p></li><li><p>TEMED (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 2367.3 </a> )</p></li><li><p>Glycerol (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 3783.2 </a> )</p></li><li><p>Sodium lauryl sulphate (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 4360.2 </a> )</p></li><li><p>Bromophenol blue (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> A512.1 </a> )</p></li><li><p>Î²-mercaptoethanol (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M6250 </a> )</p></li><li><p>2-propanol (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 6752.5 </a> )</p></li><li><p>Acetic acid (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 6755.1 </a> )</p></li><li><p>Coomassie blue R 250 (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 3862.2 </a> )</p><br/></li></ol></li><li><p>Thin-layer chromatography (TLC) of enzymatic assays with radiolabeled nucleotides</p><ol class="npwtwok"><li><p>Pipette tips (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 70.760.012 and Biozym, </a> catalog number: <a class="blackNoUnder" target="_blank"> VZ0001X </a> )</p></li><li><p>Pipette filter tips (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 70.1116.210 </a> )</p></li><li><p>1.5 ml reaction tubes (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 72.690.001 </a> )</p></li><li><p>Thin-layer plate (Macherey-Nagel, POLYGRAM CEL 300 PEI, catalog number: <a class="blackNoUnder" target="_blank"> 801053 </a> )</p></li><li><p>Phosphor imaging plate (Fujifilm, <a class="blackNoUnder" target="_blank"> BAS-IP SR 2025 </a> )</p></li><li><p>Plastic wrap (any type obtained from a grocery store)</p></li><li><p>Exposure cassette (GE Healthcare, catalog number: <a class="blackNoUnder" target="_blank"> 63-0035-44 </a> )</p></li><li><p>Purified DisA from <em>S. venezuelae</em> or <em>B. subtilis</em> (Witte <em>et al.</em>, 2008; Latoscha <em>et al.</em>, 2020)</p></li><li><p>Purified inactive DisA variant (DisA<sub>D86A</sub>) from <em>S. venezuelae</em> (Latoscha <em>et al.</em>, 2020)</p></li><li><p>Purified AtaC from <em>S. venezuelae</em> (Latoscha <em>et al.</em>, 2020)</p></li><li><p>9.25 MBq [Î±-<sup>32</sup>P]-ATP (Hartmann Analytik, catalog number: <a class="blackNoUnder" target="_blank"> FP-207 </a> )</p></li><li><p>9.25 MBq [<sup>32</sup>P]-c-di-AMP (Hartmann Analytik, catalog number: <a class="blackNoUnder" target="_blank"> FP-C517 </a> ; a purified c-di-AMP-producing enzyme, <em>e.g., Bacillus subtilis</em> DisA, must be provided to the company for [<sup>32</sup>P]-c-di-AMP synthesis upon request)</p></li><li><p>For phosphodiesterase competition assays:</p><ol class="npwthree"><li><p>c-di-AMP (BioLog, catalog number: <a class="blackNoUnder" target="_blank"> C 088 </a> )</p></li><li><p>c-di-GMP (BioLog, catalog number: <a class="blackNoUnder" target="_blank"> C 057 </a> )</p></li><li><p>cAMP (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> A6885 </a> )</p></li></ol></li><li><p>Magnesium chloride hexahydrate (MgCl<sub>2</sub>) (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 2189.1 </a> )</p></li><li><p>Manganese (II) chloride tetrahydrate (MnCl<sub>2</sub>) (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> T881.1 </a> )</p></li><li><p>Sodium chloride (NaCl) (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 27810.295DB </a> )</p></li><li><p>Tris-base (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 4855.2 </a> )</p></li><li><p>Hydrochloric acid (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 4625.2 </a> )</p></li><li><p>Glycerol (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 3783.2 </a> )</p></li><li><p>Î²-mercaptoethanol (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M6250 </a> )</p></li><li><p>Double-distilled water (ddH<sub>2</sub>O) (Kerndl, catalog number: <a class="blackNoUnder" target="_blank"> 22501 </a> )</p></li><li><p>0.5 M EDTA, pH 8 (PanReac AppliChem, catalog number: <a class="blackNoUnder" target="_blank"> A4892 </a> )</p></li><li><p>Ammonium sulfate ((NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>) (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 21333.296DB </a> )</p></li><li><p>Monopotassium phosphate (KH<sub>2</sub>PO<sub>4</sub>) (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> P018.2 </a> )</p></li><li><p>DisA cyclase buffer (see Recipes)</p></li><li><p>PDE reaction buffer (see Recipes)</p></li><li><p>TLC running buffer (see Recipes)</p><br/></li></ol></li><li><p>IEX activity assay</p><ol class="npwtwok"><li><p>Pipette tips (Sarstedt, catalog numbers: <a class="blackNoUnder" target="_blank"> 70.1130.600 and 70.760.502 </a> )</p></li><li><p>1.5 ml reaction tubes (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 72.690.001 </a> )</p></li><li><p>Microcon-30kDa Centrifugal Filter Unit (Merck Millipore, catalog number: <a class="blackNoUnder" target="_blank"> MRCF0R030 </a> )</p></li><li><p>Purified AtaC from <em>S. venezuelae</em> (Latoscha <em>et al.</em>, 2020)</p></li><li><p>Purified TmPDE from <em>T. maritima</em> (Drexler <em>et al.</em>, 2017)</p></li><li><p>c-di-AMP (BioLog, catalog number: <a class="blackNoUnder" target="_blank"> C 088 </a> )</p></li><li><p>Manganese (II) chloride tetrahydrate (MnCl<sub>2</sub>) (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> T881.3 </a> )</p></li><li><p>Sodium chloride (NaCl) (Merck Millipore, catalog number: <a class="blackNoUnder" target="_blank"> 1.06404.5000 </a> )</p></li><li><p>Tris-base (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> 4855.2 </a> )</p></li><li><p>Hydrochloric acid, 25% (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 20257.296 </a> )</p></li><li><p>10x AtaC reaction buffer (see Recipes)</p></li><li><p>IEX Running buffer A (see Recipes)</p></li><li><p>IEX Running buffer B (see Recipes)</p></li></ol></li></ol><p class="pptt" id="biaoti39478">### Equipment</p><ol class="npwone"><li><p>Equipment for buffer exchange (see Notes)</p><ol class="npwtwok"><li><p>Measuring beaker, 5,000 ml (Carl Roth, catalog number: 0<a class="blackNoUnder" target="_blank"> 780.1 </a> )</p></li><li><p>Magnetic bar (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> C267.1 </a> )</p></li><li><p>Pipetting aid macro (Carl Roth, catalog number: <a class="blackNoUnder" target="_blank"> X478.1 </a> )</p></li><li><p>Micropipettes (P20, P200, P1000)</p></li><li><p>Magnetic stirrer (Heidolph, model: <a class="blackNoUnder" target="_blank"> MR 2000 </a> )</p></li><li><p>Centrifuge (Eppendorf, model: 5427 R, catalog number: <a class="blackNoUnder" target="_blank"> 5409000010 </a> )</p></li><li><p>4 Â°C room</p><br/></li></ol></li><li><p>Equipment for protein concentration determination of your choice (see Notes)</p><ol class="npwtwok"><li><p>Mini-PROTEAN Tetra Cell (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1658000EDU </a> )</p></li><li><p>Heat block (Labnet International, model: AccuBlock<sup>TM</sup> Digital Dry Bath, catalog number: <a class="blackNoUnder" target="_blank"> D1302-230V </a> )</p></li><li><p>Centrifuge (Eppendorf, model: 5427 R, catalog number: <a class="blackNoUnder" target="_blank"> 5409000010 </a> )</p></li><li><p>Power supply (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1645050 </a> )</p></li><li><p>Imager with CCD camera (GE Healthcare, model: <a class="blackNoUnder" target="_blank"> ImageQuant LAS 4000 </a> )</p></li><li><p>Micropipette (P2, P10, P20)</p><br/></li></ol></li><li><p>TLC of enzymatic assays with radiolabeled nucleotides</p><ol class="npwtwok"><li><p>Micropipettes (P2, P10, P20)</p></li><li><p>Heat block (Labnet International, model: AccuBlock<sup>TM</sup> Digital Dry Bath, catalog number: <a class="blackNoUnder" target="_blank"> D1302-230V </a> )</p></li><li><p>Centrifuge (Eppendorf, model: 5427 R, catalog number: <a class="blackNoUnder" target="_blank"> 5409000010 </a> )</p></li><li><p>Contamination monitor (Berthold, model: <a class="blackNoUnder" target="_blank"> LB 122 </a> )</p></li><li><p>Biomolecular imager (GE Healthcare, model: <a class="blackNoUnder" target="_blank"> Typhoon FLA 7000 </a> , catalog number: 28-9558-09)</p></li><li><p>Image eraser (Molecular Dynamics, model: <a class="blackNoUnder" target="_blank"> 410A </a> )</p></li><li><p>Acrylic Benchtop Beta Radiation Shield (ThermoFisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 6700-2418 </a> )</p></li><li><p>TLC chamber (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 06-815-187 </a> )</p></li><li><p>Fume hood</p><br/></li></ol></li><li><p>IEX assay</p><ol class="npwtwok"><li><p>Micropipettes (P2, P20, P200, P1000)</p></li><li><p>NanoDrop spectrophotometer</p></li><li><p>Thermomixer (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 5382000015 </a> )</p></li><li><p>Centrifuge (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 5401000010 </a> )</p></li><li><p>Ãkta (Cytiva former GE Healthcare)</p></li><li><p>Resource-Q or Mono-Q (Cytiva, catalog number: <a class="blackNoUnder" target="_blank"> 17117701 or 17516601 </a> )</p></li></ol></li></ol><p class="pptt" id="biaoti39479">### Software</p><ol class="npwtwok"><li><p>ImageQuantTL (optional) (GE Healthcarre)</p></li><li><p><a class="blackNoUnder" target="_blank"> Typhoon FLA 7000 </a> control software (GE Healthcare)</p></li><li><p>PhotoShop CS6 (Adobe)</p></li><li><p>Unicorn (Cytiva former GE Healthcare)</p></li></ol><p class="pptt" id="biaoti39480">### Procedure</p><ol class="npwone"><li><p>Diadenylate cyclase assay with radiolabeled ATP coupled with TLC</p><ol class="npwtwok"><li><p>Until Step A6, work on ice.</p></li><li><p>Exchange the buffer of purified proteins to DisA cyclase buffer (<em>e.g.</em>, via dialysis, see Notes).</p></li><li><p>Determine the molar concentration of proteins (see Notes).</p></li><li><p>Setup of the reaction:</p><ol class="npwthree"><li><p>Calculate the volume of DisA required to achieve a concentration of 5 ÂµM in 20 Âµl.</p></li><li><p>Pipette the required volume of DisA into a 1.5 ml reaction tube (<em>i.e.</em>, if the DisA stock has a concentration of 50 ÂµM, you will need to take 2 Âµl).</p></li><li><p>Fill up to a total volume of 19.5 Âµl with DisA cyclase buffer.</p></li></ol></li><li><p>From now on, work behind a radiation shield and use filter tips (see Notes).</p></li><li><p>Add 0.5 Âµl [<sup>32</sup>P]-ATP to the reaction (final concentration: ~83 nM in 20 Âµl).</p></li><li><p>Use 19.5 Âµl DisA cyclase buffer with 0.5 Âµl [<sup>32</sup>P]-ATP as a control for ATP migration.</p></li><li><p>Incubate reactions for 60 min at 30 Â°C in the heat block.</p></li><li><p>Pipette 5 Âµl 0.5 M EDTA, pH 8 into 1.5 ml reaction tubes (one tube per reaction).</p></li><li><p>Add 5 Âµl of each reaction to a tube containing EDTA to stop diadenylate cyclase reaction.</p></li><li><p>Spot the mixture from Step A10 on the cellulose side of a thin-layer (TL) plate (each spot should be ~1.5 cm from the bottom and lateral edges of the TL plate as well as other spots; also see Notes).</p></li><li><p>Let the spots dry completely at room temperature (RT).</p></li><li><p>Fill TLC chamber with TLC running buffer (buffer should be ~1 cm deep).</p></li><li><p>Place the TL plate in the TLC chamber with spotted samples close to the buffer without touching it.</p></li><li><p>Incubate TLC until the liquid front is ~1 cm from the upper edge of the TL plate (this can take up to ~2.5 h).</p></li><li><p>Remove TL plate from the chamber and let dry at RT (overnight or at least 45 min in fume hood).</p></li><li><p>Place an erased imaging plate (IP) into an exposure cassette.</p></li><li><p>Wrap TL plate completely into plastic wrap (see Notes) and place with the cellulose side down on top of the IP for exposure.</p></li><li><p>The exposure time may vary depending on the optimal signal-to-noise ratio, but ~30 min exposure will suffice in most cases.</p></li><li><p>Remove the TL plate from the cassette and check cassette and IP for radioactive contamination using the contamination monitor (see Notes).</p></li><li><p>If no contamination is present, scan the IP in the <a class="blackNoUnder" target="_blank"> Typhoon FLA 7000 </a> using the laser at 650 nm and IP filter.</p><br/></li></ol></li><li><p>Phosphodiesterase assay with radiolabeled c-di-AMP coupled with TLC</p><ol class="npwtwok"><li><p>Until Step B7, work on ice.</p></li><li><p>Exchange the buffer of purified AtaC to PDE reaction buffer (<em>e.g.</em>, via dialysis, see Notes).</p></li><li><p>Determine the molar concentration of AtaC (see Notes).</p></li><li><p>If you want to perform competition with unlabeled nucleotides, proceed with Step B5, without competition proceed with Step B6.</p></li><li><p>Setup of the reaction with competition:</p><ol class="npwthree"><li><p>Calculate the volume of AtaC required to achieve a concentration of 100 nM in 20 Âµl.</p></li><li><p>Pipette the required volume of AtaC into a 1.5 ml reaction tube (<em>i.e.</em>, if the AtaC stock has a concentration of 1 ÂµM, you will need to take 2 Âµl).</p></li><li><p>Fill up to a total volume of 17.5 Âµl with PDE reaction buffer.</p></li><li><p>Set up 1 mM solutions of each unlabeled nucleotide (c-di-AMP, c-di-GMP, cAMP) in ddH<sub>2</sub>O.</p></li><li><p>Add 2 Âµl of each nucleotide per reaction in Step B5c.</p></li><li><p>Set up a reaction with 2 Âµl ddH<sub>2</sub>O as negative control for competition.</p></li><li><p>Proceed with Step B7.</p></li></ol></li><li><p>Setup of the reaction without competition:</p><ol class="npwthree"><li><p>Calculate the volume of AtaC required to achieve a concentration of 100 nM in 20 Âµl.</p></li><li><p>Pipette the required volume of AtaC into a 1.5 ml reaction tube (<em>i.e.</em>, if the AtaC stock has a concentration of 1 ÂµM, you will need to take 2 Âµl).</p></li><li><p>Fill up to a total volume of 19.5 Âµl with PDE reaction buffer.</p></li><li><p>Proceed with Step B7.</p></li></ol></li><li><p>From now on, work behind a radiation shield and use filter tips (see Notes).</p></li><li><p>Dilute [<sup>32</sup>P]-c-di-AMP 1:20 in PDE reaction buffer and add 0.5 Âµl to the reaction (final concentration: ~2 nM in 20 Âµl).</p></li><li><p>Use 19.5 Âµl PDE reaction buffer with 0.5 Âµl of 1:20 diluted [<sup>32</sup>P]-c-di-AMP as a control for c-di-AMP migration.</p></li><li><p>Incubate reactions for 60 min at 30 Â°C in the heat block.</p></li><li><p>To stop enzymatic reaction, incubate at 95 Â°C for 5 min.</p></li><li><p>Centrifuge reactions at maximal speed for 3 min.</p></li><li><p>Spot 5 Âµl supernatant of each reaction on the cellulose side of a thin-layer (TL) plate (each spot should be ~1.5 cm from the bottom and lateral edges of the TL plate and other spots).</p></li><li><p>Let the spots dry completely at room temperature (RT).</p></li><li><p>Fill TLC chamber with TLC running buffer (buffer should be ~1 cm high).</p></li><li><p>Place the TL plate in the TLC chamber with spotted samples close to the buffer without touching it.</p></li><li><p>Incubate TLC until the liquid front is ~1 cm from the upper edge of the TL plate (this can take up to ~2.5 h).</p></li><li><p>Remove TL plate from the chamber and let dry at RT (overnight or at least 45 min in fume hood).</p></li><li><p>Place an erased imaging plate (IP) into an exposure cassette.</p></li><li><p>Wrap TL plate completely into plastic wrap (see Notes) and place with the cellulose side down on top of the IP for exposure.</p></li><li><p>The exposure time may vary depending on the optimal signal-to-noise ratio, but ~60 min exposure will suffice in most cases.</p></li><li><p>Remove the TL plate from the cassette and check cassette and IP for radioactive contamination using the contamination monitor.</p></li></ol><ol class="npwtwok"><li><p>If no contamination is present, scan the IP in the <a class="blackNoUnder" target="_blank"> Typhoon FLA 7000 </a> using the laser at 650 nm and IP filter.</p><br/></li></ol></li><li><p>IEX AtaC activity assay</p><ol class="npwtwok"><li><p>Set up 100 Âµl reactions containing 100 nM protein, 62.5 ÂµM-2,000 ÂµM substrate (<em>e.g.</em>, c-di-AMP) and 1x reaction buffer. Control reactions containing protein or substrate only are necessary to exclude nonspecific results.</p></li><li><p>Incubate the reaction at 37 Â°C for 1 h.</p></li><li><p>Stop the reaction by transferring the complete reaction mix into a Microcon-30kDa Centrifugal Filter Unit and spinning for 10 min at 14,000 <em>x g</em>. This ultrafiltration step separates the protein from its reaction products, which are obtained in the filtrate.</p></li><li><p>Repeat Step C3 if there is still liquid in the filter.</p></li><li><p>Wash the filter by adding 100 Âµl running buffer A and repeating Step C3.</p></li><li><p>Fill up the filtrate with running buffer A to 500 Âµl.</p></li><li><p>Equilibrate an ion exchange column (1 ml Resource-Q or Mono-Q) in running buffer A.</p></li><li><p>Inject the sample from step 6 and use a linear gradient from running buffer A to B (<em>i.e.</em>, 0-40% B, 20 CV). Set the detection wavelength on 260 nm or 280 nm, if 260 nm is not possible.</p></li></ol></li></ol><p class="pptt" id="biaoti39481">### Data analysis</p><ol class="npwone"><li><p>TLC assay</p><p>Images scanned in the <a class="blackNoUnder" target="_blank"> Typhoon FLA 7000 </a> sometimes require adjustment of contrast. If necessary, open the scanned file in PhotoShop and modify spot intensity for the whole picture via âImageâ â âAdjustmentsâ â âLevelsâ until an optimal contrast is reached. Exemplary DAC and PDE assays separated by TLC are shown in Figure 2.</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3870/bioprotoc-11-01-3870-g002.jpg"/><br/><strong>Figure 2. Thin-layer chromatography (TLC) of a diadenylate cyclase (A) and a c-di-AMP phosphodiesterase assay (B).</strong> A. Diadenylate cyclase (DAC) assay with 5 ÂµM of purified <em>S. venezuelae</em> wild-type DisA (DisA<sub>Sven</sub>, lanes 2 and 3) and inactive DisA<sub>Sven</sub> mutant variant (DisA<sub>D86A</sub>, lanes 4 and 5) incubated with 83 nM [<sup>32</sup>P]-ATP at 30 Â°C. After 30 and 60 min samples were taken and inactivated by mixing with equal volume of 0.5 M EDTA, pH 8. Inactivated reactions were spotted on a thin-layer (TL) plate and separated via TLC. After drying, the TL plate was exposed for 30 min to an imaging plate which was subsequently scanned using <a class="blackNoUnder" target="_blank"> Typhoon FLA 7000 </a> . <em>B. subtilis</em> DisA (DisA<sub>Bsu</sub>, lane 6) was used as a positive control for DAC reaction. Lane 1 shows migration of [<sup>32</sup>P]-ATP in absence of enzymes. All samples were separated on the same TL plate and the picture was cropped to the relevant lanes. In contrast to the inactive DisA<sub>D86A</sub>, DisA<sub>Sven</sub> synthesizes [<sup>32</sup>P]-c-di-AMP out of [<sup>32</sup>P]-ATP as indicated by formation of a product that has the similar size like [<sup>32</sup>P]-c-di-AMP produced by DisA<sub>Bsu</sub>, which is a characterized DAC. B. Phosphodiesterase assay of <em>S. venezuelae</em> AtaC with competition with unlabeled nucleotides (lanes 3 to 5). 100 nM of purified protein was supplemented with 100 ÂµM unlabeled c-di-AMP, c-di-GMP and cAMP (or ddH<sub>2</sub>O as control, lane 2) before incubation with 2 nM [<sup>32</sup>P]-c-di-AMP at 30 Â°C. After 60 min samples were taken and heat-inactivated. Precipitated protein was removed by centrifugation and supernatants of the inactivated reactions were spotted on a TL plate and separated via TLC. After drying, the TL plate was exposed for 60 min to an imaging plate which was subsequently scanned using <a class="blackNoUnder" target="_blank"> Typhoon FLA 7000 </a> . Lane 1 shows migration of [<sup>32</sup>P]-c-di-AMP in absence of enzyme. AtaC cleaves [<sup>32</sup>P]-c-di-AMP to [<sup>32</sup>P]-AMP using [<sup>32</sup>P]-pApA as an intermediate cleavage product (lane 2). This reaction is specific since unlabeled c-di-AMP significantly inhibits, and thus outcompetes [<sup>32</sup>P]-c-di-AMP turnover (lane 3) whereas c-di-GMP and cAMP have no effect (lanes 4 and 5).</p><br/></li><li><p>IEX assay</p><p>The IEX assay can also be used for quantification of the hydrolysis turnover for c-di-AMP. In this case, a peak integration of the respective nucleotide peaks has to be performed using the Ãkta software Unicorn (Cytiva former GE Healthcare) or any other mathematical analysis software. From the ratio of the peak areas of substrate and product, a percentage of hydrolyzed substrate or formed product can be calculated. Figure 3 shows exemplary IEX separation of c-di-AMP, 5â²-pApA and AMP standards. Figure 4 demonstrates the interfering peaks of c-di-AMP and 5â²-pApA after IEX separation of a c-di-AMP PDE assay with AtaC and the removal of 5â²-pApA from the reaction by TmPDE.</p><br/><p><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20210104/20210104232913_8973.jpg"/><br/><strong>Figure 3. Ion-Exchange Chromatography (IEX) standards for 100 Âµl solutions containing 200 ÂµM c-di-AMP, 5â²-pApA or AMP</strong> </p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3870/bioprotoc-11-01-3870-g004.jpg"/><br/><strong>Figure 4. Ion-Exchange Chromatography (IEX) of a c-di-AMP phosphodiesterase assay.</strong> A. A 100 Âµl reaction containing 1 ÂµM <em>S. venezuelae</em> AtaC + 400 ÂµM c-di-AMP incubated for 1 h at 37 Â°C was stopped by ultrafiltration and separated via IEX. B. A similar reaction in a coupled assay with TmPDE to separate the peaks of c-di-AMP and 5â²-pApA. A 100 ÂµL reaction containing 1 ÂµM <em>S. venezuelae</em> AtaC + 400 ÂµM c-di-AMP incubated for 1 h at 37 Â°C was stopped by ultrafiltration, followed by the addition of 100 nM TmPDE which was stopped by ultrafiltration after incubation for 1 h at 37 Â°C and separated via IEX.</p></li></ol><p class="pptt" id="biaoti39482">### Notes</p><ol class="npwone"><li><p>General comments on TLC assays</p><ol class="npwtwok"><li><p>Buffer exchange is needed to replace the elution buffer in protein eluates to the proper reaction buffers (DisA cyclase buffer or PDE reaction buffer) and can be performed with many methods: <em>e.g.</em>, size exclusion chromatography, ultrafiltration or dialysis. In our lab, we use dialysis, which however has no advantages or disadvantages compared to other methods for buffer exchange. You can choose a method that suits you best.</p></li><li><p>The exact method to determine the protein concentration is not crucial to perform the TLC assays and can be achieved by many methods: <em>e.g.</em>, photometric measurement at 280 nm, Bradford assay or 1D SDS gel densitometry. In our lab, we routinely use the latter, which however has no advantages or disadvantages compared to other methods. Feel free to use any method that is convenient for you.</p></li><li><p>Enzymatic assays with radiolabeled nucleotides should be performed only by trained individuals with sufficient protection equipment.</p></li><li><p>Usage of filter tips is strongly recommended to avoid contamination of micropipettes with radioactive material.</p></li><li><p>The given enzyme concentrations were optimized for purified DisA (<em>S. venezuelae</em> and <em>B. subtilis</em>) and purified AtaC (<em>S. venezuelae</em>). Enzymes from other bacteria may require different concentrations. If you work with a new protein, first test enzymatic activity with 1 to 5 ÂµM purified protein and subsequently determine the optimal protein concentrations by titration.</p></li><li><p>Procedure section, Step A11: spotting the whole 10 Âµl mixture at once may result in blending of the single spots before drying. To avoid that, you can spot 3 Âµl + 3 Âµl + 4 Âµl with drying of spots between each volume spotted.</p></li><li><p>The TL plate can be stored at RT for at least 3 days without significant loss of signal quality and can be used multiple times for exposure to erased imaging plates. If you are working with substrates with lower radioactive activity than indicated in the materials section, it might be necessary to prolong the exposure times of the imaging plate.</p></li><li><p>The use of plastic wrap for the TL plate reduces the risk of radioactive contamination of the imaging plate and cassette. You may place an additional layer of plastic wrap on the imaging plate to further reduce the risk.</p></li><li><p>Imaging plates can be erased by exposure to a light source (<em>e.g.</em>, for 10 min using the image eraser, Molecular Dynamics, model: <a class="blackNoUnder" target="_blank"> 410A </a> ) and reused. Consequently, after removing the TL plate from the cassette, avoid long exposure of the imaging plate to ambient light prior to scanning.</p><br/></li></ol></li><li><p>General comments on IEX assay</p><ol class="npwtwok"><li><p>c-di-AMP forms long non-symmetrical Peaks on IEX columns and can therefore overlay with other Peaks (see also comment 3). Thus, if a RP-HPLC system is available, it is the preferred method to use instead.</p></li><li><p>Basic knowledge of Ãkta systems is required.</p></li><li><p>If the c-di-AMP conversion has to be quantified by peak integration, the overlay of the peaks from 5â²-pApA and c-di-AMP has to be considered. In this case, a second enzyme, which can only degrade 5â²-pApA but not c-di-AMP (<em>e.g.</em>, TmPDE [Drexler <em>et al.</em>, 2017]), has to be added after ultrafiltration. Add TmPDE to a final concentration of 100 nM and incubate at 37 Â°C for 1 h. Repeat the procedure section paragraph C starting from step 3.</p></li><li><p>The precise elution point of the nucleotides on an IEX column is very sensitive to salt, pH, temperature and also the Ãkta system. For optimal comparison, assays should be performed with the same buffer and on the same Ãkta system.</p></li></ol></li></ol><p class="pptt" id="biaoti39483">### Recipes</p><ol class="npwtwok"><li><p>DisA cyclase reaction buffer, modified from Christen <em>et al.</em> (2005)</p><p>25 mM Tris-HCl pH = 8</p><p>250 mM NaCl</p><p>10 mM MgCl<sub>2</sub> </p><p>5 mM Î²-mercaptoethanol</p><p>10% glycerol</p></li><li><p>PDE reaction buffer, modified from Huynh <em>et al.</em> (2015)</p><p>20 mM Tris-HCl pH = 7.5</p><p>50 mM NaCl</p><p>1 mM MnCl<sub>2</sub> </p></li><li><p>TLC running buffer, modified from Christen <em>et al.</em> (2005)</p><p>Saturated (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> and 1.5 M KH<sub>2</sub>PO<sub>4 </sub>pH = 3.6 (mixed in a ratio of 1:1.5 v/v)</p></li><li><p>10x AtaC reaction buffer</p><p>500 mM Tris pH = 7.5</p><p>1,000 mM NaCl</p><p>1 mM MnCl<sub>2</sub> </p></li><li><p>IEX running buffer A</p><p>50 mM Tris pH = 9</p></li><li><p>IEX running buffer B</p><p>50 mM Tris pH = 9</p><p>1,000 mM NaCl</p></li></ol><p class="pptt" id="biaoti39484">### Acknowledgments</p><p>Research in the Witte lab is funded by DFG Grant GRK1721 and the DFG Priority Program SPP 1879 (Grant WI 3717/3-1). Research in the Tschowri lab is funded by the DFG Emmy Noether Program (Grant TS 325/1-1) and the DFG Priority Program SPP 1879 (Grants TS 325/2-1 and TS 325/2-2).</p><p class="pptt" id="biaoti39485">### Competing interests</p><p>The authors declare no competing interests.</p><p class="pptt" id="biaoti39486">### References</p><ol class="npwtwok"><li>Al-Bassam, M. M., Haist, J., Neumann, S. A., Lindenberg, S. and Tschowri, N. (2018). <a href="https://pubmed.ncbi.nlm.nih.gov/30405580/" target="_blank">Expression Patterns, Genomic Conservation and Input Into Developmental Regulation of the GGDEF/EAL/HD-GYP Domain Proteins in <em>Streptomyces</em>.</a> <em>Front Microbiol</em> 9: 2524.</li><li>Bai, Y., Yang, J., Eisele, L. E., Underwood, A. J., Koestler, B. J., Waters, C. M., Metzger, D. W. and Bai, G. (2013). <a href="https://jb.asm.org/content/195/22/5123" target="_blank">Two DHH subfamily 1 proteins in <em>Streptococcus pneumoniae</em> possess cyclic di-AMP phosphodiesterase activity and affect bacterial growth and virulence.</a> <em>J Bacteriol</em> 195(22): 5123-5132.</li><li>Bai, Y., Yang, J., Zhou, X., Ding, X., Eisele, L. E. and Bai, G. (2012). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3328451/" target="_blank">Mycobacterium tuberculosis Rv3586(DacA) is a diadenylate cyclase that converts ATP or ADP into c-di-AMP.</a> <em>PLoS One</em> 7(4): e35206.</li><li>Bejerano-Sagie, M., Oppenheimer-Shaanan, Y., Berlatzky, I., Rouvinski, A., Meyerovich, M. and Ben-Yehuda, S. (2006). <a href="https://pubmed.ncbi.nlm.nih.gov/16713562/" target="_blank">A checkpoint protein that scans the chromosome for damage at the start of sporulation in Bacillus subtilis.</a> <em>Cell</em> 125(4): 679-690.</li><li>Bordeleau, E., Fortier, L. C., Malouin, F. and Burrus, V. (2011). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3069119/" target="_blank"> c-di-GMP turn-over in Clostridium difficile is controlled by a plethora of diguanylate cyclases and phosphodiesterases.</a> <em>PLoS Genet</em> 7(3): e1002039.</li><li>Bowman, L., Zeden, M. S., Schuster, C. F., Kaever, V. and GrÃ¼ndling, A. (2016). <a href="https://pubmed.ncbi.nlm.nih.gov/27834680/" target="_blank">New Insights into the Cyclic Di-adenosine Monophosphate(c-di-AMP) Degradation Pathway and the Requirement of the Cyclic Dinucleotide for Acid Stress Resistance in <em>Staphylococcus aureus</em>.</a> <em>J Biol Chem</em> 291(53): 26970-26986.</li><li>Christen, M., Christen, B., Folcher, M., Schauerte, A. and Jenal, U. (2005). <a href="https://pubmed.ncbi.nlm.nih.gov/15994307/" target="_blank"> Identification and characterization of a cyclic di-GMP-specific phosphodiesterase and its allosteric control by GTP.</a> <em>J Biol Chem</em> 280(35): 30829-30837.</li><li>Commichau, F. M., Heidemann, J. L., Ficner, R. and StÃ¼lke, J. (2019). <a href="https://pubmed.ncbi.nlm.nih.gov/30224435/" target="_blank">Making and Breaking of an Essential Poison: the Cyclases and Phosphodiesterases That Produce and Degrade the Essential Second Messenger Cyclic di-AMP in Bacteria.</a> <em>J Bacteriol</em> 201(1): .</li><li>Corrigan, R. M., Abbott, J. C., Burhenne, H., Kaever, V. and GrÃ¼ndling, A. (2011). <a href="https://pubmed.ncbi.nlm.nih.gov/21909268/" target="_blank"> c-di-AMP is a new second messenger in <em>Staphylococcus aureus</em> with a role in controlling cell size and envelope stress.</a> <em>PLoS Pathog</em> 7(9): e1002217.</li><li>Corrigan, R. M. and GrÃ¼ndling, A. (2013). <a href="https://pubmed.ncbi.nlm.nih.gov/23812326/" target="_blank">Cyclic di-AMP: another second messenger enters the fray.</a> <em>Nat Rev Microbiol</em> 11(8): 513-524.</li><li>Drexler, D. J., MÃ¼ller, M., Rojas-Cordova, C. A., Bandera, A. M. and Witte, G. (2017). <a href="https://pubmed.ncbi.nlm.nih.gov/29107484/" target="_blank"> Structural and Biophysical Analysis of the Soluble DHH/DHHA1-Type Phosphodiesterase TM1595 from <em>Thermotoga maritima</em>.</a> <em>Structure</em> 25(12): 1887-1897.e1884.</li><li>Fahmi, T., Port, G. C. and Cho, K. H. (2017). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5575661/" target="_blank"> c-di-AMP: An Essential Molecule in the Signaling Pathways that Regulate the Viability and Virulence of Gram-Positive Bacteria.</a> <em>Genes(Basel)</em> 8(8).</li><li>Gibhardt, J., Heidemann, J. L., Bremenkamp, R., Rosenberg, J., Seifert, R., Kaever, V., Ficner, R. and Commichau, F. M. (2020). <a href="https://pubmed.ncbi.nlm.nih.gov/32250026/" target="_blank">An extracytoplasmic protein and a moonlighting enzyme modulate synthesis of c-di-AMP in <em>Listeria monocytogenes</em>.</a> <em>Environ Microbiol</em> 22(7): 2771-2791.</li><li>Gundlach, J., Mehne, F. M., Herzberg, C., Kampf, J., Valerius, O., Kaever, V. and StÃ¼lke, J. (2015). <a href="https://pubmed.ncbi.nlm.nih.gov/26240071/" target="_blank">An Essential Poison: Synthesis and Degradation of Cyclic Di-AMP in <em>Bacillus subtilis</em>.</a> <em>J Bacteriol</em> 197(20): 3265-3274.</li><li>He, Q., Wang, F., Liu, S., Zhu, D., Cong, H., Gao, F., Li, B., Wang, H., Lin, Z., Liao, J. and Gu, L. (2016). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4751404/" target="_blank"> Structural and biochemical insight into the mechanism of Rv2837c from <em>Mycobacterium tuberculosis</em> as a c-di-NMP phosphodiesterase.</a> <em>J Biol Chem</em> 291(27): 14386-14387.</li><li>Huynh, T. N., Luo, S., Pensinger, D., Sauer, J. D., Tong, L. and Woodward, J. J. (2015). <a href="https://pubmed.ncbi.nlm.nih.gov/25583510/" target="_blank">An HD-domain phosphodiesterase mediates cooperative hydrolysis of c-di-AMP to affect bacterial growth and virulence.</a> <em>Proc Natl Acad Sci U S A</em> 112(7): E747-756.</li><li>Kazmierczak, B. I. (2017a). <a href="https://pubmed.ncbi.nlm.nih.gov/28889302/" target="_blank">Determining Diguanylate Cyclase Activity(Radioactive Assay).</a> <em>Methods Mol Biol</em> 1657: 285-289.</li><li>Kazmierczak, B. I. (2017b). <a href="https://pubmed.ncbi.nlm.nih.gov/28889301/" target="_blank">Determining Phosphodiesterase Activity(Radioactive Assay).</a> <em>Methods Mol Biol</em> 1657: 279-283.</li><li>Kazmierczak, B. I., Lebron, M. B. and Murray, T. S. (2006). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3609419/" target="_blank">Analysis of FimX, a phosphodiesterase that governs twitching motility in <em>Pseudomonas aeruginosa</em>.</a> <em>Mol Microbiol</em> 60(4): 1026-1043.</li><li>Konno, H., Yoshida, Y., Nagano, K., Takebe, J. and Hasegawa, Y. (2018). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6170606/" target="_blank">Biological and Biochemical Roles of Two Distinct Cyclic Dimeric Adenosine 3',5'-Monophosphate- Associated Phosphodiesterases in <em>Streptococcus mutans</em>.</a> <em>Front Microbiol</em> 9: 2347.</li><li>Latoscha, A., Drexler, D. J., Al-Bassam, M. M., Bandera, A. M., Kaever, V., Findlay, K. C., Witte, G. and Tschowri, N. (2020). <a href="https://pubmed.ncbi.nlm.nih.gov/32188788/" target="_blank">c-di-AMP hydrolysis by the phosphodiesterase AtaC promotes differentiation of multicellular bacteria.</a> <em>Proc Natl Acad Sci U S A</em> 117(13): 7392-7400.</li><li>Li, X., Shu, C., Yi, G., Chaton, C. T., Shelton, C. L., Diao, J., Zuo, X., Kao, C. C., Herr, A. B. and Li, P. (2013). <a href="https://pubmed.ncbi.nlm.nih.gov/24332030/" target="_blank">Cyclic GMP-AMP synthase is activated by double-stranded DNA-induced oligomerization.</a> <em>Immunity</em> 39(6): 1019-1031.</li><li>Lindenberg, S., Klauck, G., Pesavento, C., Klauck, E. and Hengge, R. (2013). <a href="https://pubmed.ncbi.nlm.nih.gov/23708798/" target="_blank">The EAL domain protein YciR acts as a trigger enzyme in a c-di-GMP signalling cascade in <em>E. coli</em> biofilm control.</a> <em>Embo j</em> 32(14): 2001-2014.</li><li>Luecke, S., Holleufer, A., Christensen, M. H., JÃ¸nsson, K. L., Boni, G. A., SÃ¸rensen, L. K., Johannsen, M., Jakobsen, M. R., Hartmann, R. and Paludan, S. R. (2017). <a href="https://pubmed.ncbi.nlm.nih.gov/28801534/" target="_blank">cGAS is activated by DNA in a length-dependent manner.</a> <em>EMBO Rep</em> 18(10): 1707-1715.</li><li>Luo, Y. and Helmann, J. D. (2012). <a href="https://pubmed.ncbi.nlm.nih.gov/22211522/" target="_blank"> Analysis of the role of <em>Bacillus subtilis</em> Ï(M) in Î²-lactam resistance reveals an essential role for c-di-AMP in peptidoglycan homeostasis.</a> <em>Mol Microbiol</em> 83(3): 623-639.</li><li>Mehne, F. M., Gunka, K., Eilers, H., Herzberg, C., Kaever, V. and StÃ¼lke, J. (2013). <a href="https://pubmed.ncbi.nlm.nih.gov/23192352/" target="_blank">Cyclic di-AMP homeostasis in <em>Bacillus subtilis</em>: both lack and high level accumulation of the nucleotide are detrimental for cell growth.</a> <em>J Biol Chem</em> 288(3): 2004-2017.</li><li>Oppenheimer-Shaanan, Y., Wexselblatt, E., Katzhendler, J., Yavin, E. and Ben-Yehuda, S. (2011). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3128283/" target="_blank">c-di-AMP reports DNA integrity during sporulation in <em>Bacillus subtilis</em>.</a> <em>EMBO Rep</em> 12(6): 594-601.</li><li>Paul, R., Weiser, S., Amiot, N. C., Chan, C., Schirmer, T., Giese, B. and Jenal, U. (2004). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC387245/" target="_blank">Cell cycle-dependent dynamic localization of a bacterial response regulator with a novel di-guanylate cyclase output domain.</a> <em>Genes Dev</em> 18(6): 715-727.</li><li>Rao, F., See, R. Y., Zhang, D., Toh, D. C., Ji, Q. and Liang, Z. X. (2010). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2804195/" target="_blank">YybT is a signaling protein that contains a cyclic dinucleotide phosphodiesterase domain and a GGDEF domain with ATPase activity.</a> <em>J Biol Chem</em> 285(1): 473-482.</li><li>Ross, P., Weinhouse, H., Aloni, Y., Michaeli, D., Weinberger-Ohana, P., Mayer, R., Braun, S., de Vroom, E., van der Marel, G. A., van Boom, J. H. and Benziman, M. (1987). <a href="https://pubmed.ncbi.nlm.nih.gov/18990795/" target="_blank">Regulation of cellulose synthesis in <em>Acetobacter xylinum</em> by cyclic diguanylic acid.</a> <em>Nature</em> 325(6101): 279-281.</li><li>Tang, Q., Luo, Y., Zheng, C., Yin, K., Ali, M. K., Li, X. and He, J. (2015). <a href="https://pubmed.ncbi.nlm.nih.gov/26078723/" target="_blank">Functional Analysis of a c-di-AMP-specific Phosphodiesterase MsPDE from <em>Mycobacterium smegmatis</em>.</a> <em>Int J Biol Sci</em> 11(7): 813-824.</li><li>Torres, R., Carrasco, B., GÃ¡ndara, C., Baidya, A. K., Ben-Yehuda, S. and Alonso, J. C. (2019). <a href="https://academic.oup.com/nar/article/47/10/5141/5420545" target="_blank"><em>Bacillus subtilis</em> DisA regulates RecA-mediated DNA strand exchange.</a> <em>Nucleic Acids Res</em> 47(10): 5141-5154.</li><li>Tosi, T., Hoshiga, F., Millership, C., Singh, R., Eldrid, C., Patin, D., Mengin-Lecreulx, D., Thalassinos, K., Freemont, P. and GrÃ¼ndling, A. (2019). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6368335/" target="_blank"> Inhibition of the <em>Staphylococcus aureus</em> c-di-AMP cyclase DacA by direct interaction with the phosphoglucosamine mutase GlmM.</a> <em>PLoS Pathog</em> 15(1): e1007537.</li><li>Whiteley, A. T., Pollock, A. J. and Portnoy, D. A. (2015). <a href="https://pubmed.ncbi.nlm.nih.gov/26028365" target="_blank">The PAMP c-di-AMP Is Essential for <em>Listeria monocytogenes</em> Growth in Rich but Not Minimal Media due to a Toxic Increase in(p)ppGpp.[corrected].</a> <em>Cell Host Microbe</em> 17(6): 788-798.</li><li>Witte, C. E., Whiteley, A. T., Burke, T. P., Sauer, J. D., Portnoy, D. A. and Woodward, J. J. (2013). <a href="https://mbio.asm.org/content/4/3/e00282-13" target="_blank">Cyclic di-AMP is critical for <em>Listeria monocytogenes</em> growth, cell wall homeostasis, and establishment of infection.</a> <em>mBio</em> 4(3): e00282-00213.</li><li>Witte, G., Hartung, S., BÃ¼ttner, K. and Hopfner, K. P. (2008). <a href="https://pubmed.ncbi.nlm.nih.gov/18439896/" target="_blank">Structural biochemistry of a bacterial checkpoint protein reveals diadenylate cyclase activity regulated by DNA recombination intermediates.</a> <em>Mol Cell</em> 30(2): 167-178.</li><li>Woodward, J. J., Iavarone, A. T. and Portnoy, D. A. (2010). <a href="https://pubmed.ncbi.nlm.nih.gov/20508090/" target="_blank">c-di-AMP secreted by intracellular <em>Listeria</em> monocytogenes activates a host type I interferon response.</a> <em>Science</em> 328(5986): 1703-1705.</li><li>Ye, M., Zhang, J. J., Fang, X., Lawlis, G. B., Troxell, B., Zhou, Y., Gomelsky, M., Lou, Y. and Yang, X. F. (2014). <a href="https://pubmed.ncbi.nlm.nih.gov/24566626/" target="_blank"> DhhP, a cyclic di-AMP phosphodiesterase of <em>Borrelia burgdorferi</em>, is essential for cell growth and virulence.</a> <em>Infect Immun</em> 82(5): 1840-1849.</li><li>Yin, W., Cai, X., Ma, H., Zhu, L., Zhang, Y., Chou, S. H., Galperin, M. Y. and He, J. (2020). <a href="https://pubmed.ncbi.nlm.nih.gov/32472931/" target="_blank">A decade of research on the second messenger c-di-AMP.</a> <em>FEMS Microbiol Rev</em>.</li></ol>