<body>
            <h1>Evaluation of the Sequence Variability within the PCR Primer/Probe Target Regions of the SARS-CoV-2 Genome</h1>
            <p><strong>Authors:</strong> Kashif Aziz  Khan, Peter  Cheung</p>
            <h2>Abstract</h2>
            <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; initially named 2019-nCoV) is responsible for the recent coronavirus disease (COVID-19) pandemic, and polymerase chain reaction (PCR) is the current standard method for diagnosis from patient samples. As PCR assays are prone to sequence mismatches due to mutations in the viral genome, it is important to verify the genomic variability at primer/probe binding regions periodically. This step-by-step protocol describes a bioinformatics approach for an extensive evaluation of the sequence variability within the primer/probe target regions of the SARS-CoV-2 genome. The protocol can be applied to any molecular diagnostic assay of choice using freely available software programs and the ready-to-use multiple sequence alignment (MSA) file provided.<br/><br/><strong>Graphic abstract:</strong><br/><br/><img alt="" class="layerphoto" height="304" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201216/20201216200725_0977.jpg" width="600"/><br/><strong>Overview of the sequence tracing protocol.</strong> The figure was created using the Library of Science and Medical Illustrations from somersault18:24 licensed under a CC BY-NC-SA 4.0 license (<a href="https://creativecommons.org/licenses/by-nc-sa/4.0/">https://creativecommons.org/licenses/by-nc-sa/4.0/</a>).<br/> <p><strong>Video abstract: </strong><a href="https://youtu.be/M1lV1liWE9k">https://youtu.be/M1lV1liWE9k</a></p><br/></p>
            <p><strong>Keywords:</strong> Coronavirus SARS-CoV-2, COVID-19, Diagnosis, Genomic variability, Polymerase chain reaction (PCR), Mutations</p>
            <h2>Background</h2><p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; initially named as 2019-nCoV) is the cause of novel coronavirus disease termed COVID-19. The virus originated from Wuhan, China and rapidly spreaded around the world causing a global pandemic (Worldometers.info, 2020). Sequencing of the virus showed that its single-stranded RNA genome is ~30 kb in size (Chan <em>et al.</em>, 2020; Lu <em>et al.</em>, 2020; Wu <em>et al.</em>, 2020; Zhou <em>et al.</em>, 2020). Availability of the viral sequence early in the outbreak helped the development of several polymerase chain reaction (PCR) detection protocols that have been instrumental in the diagnosis of the disease from patients samples (WHO, 2020). However, genetic variability in the viral genome during natural evolution poses a potential risk of mismatches between the diagnostic assays and the template that can result in false-negative results (Whiley and Sloots, 2005; Chow <em>et al.</em>, 2011). Sequences of SARS-CoV-2 viruses isolated from around the world are being deposited in the sequence databases and mutations have been identified in the genomes of the circulating viruses (Ugurel <em>et al.</em>, 2020).<br/> <br/>We performed an extensive evaluation of published diagnostic PCR assays, including those recommended by the World Health Organization (WHO), based on evaluation of sequence variation in the primer/probe binding regions using more than 17,000 publicly available viral sequences (Khan and Cheung, 2020). Another concurrent publication reported mutations in primer/probe binding regions using 1825 sequences but a detailed sequence tracing protocol was not provided (Osorio and Correia-Neves, 2020). This step-by-step protocol outlines a bioinformatics pipeline that uses freely available open-source software programs. The pipeline can be performed on a regular desktop computer without any need for special hardware and does not require extensive computational skills. The provision of a ready-to-use Multiple Sequence Alignment (MSA) file through Open Science Framework (OSF) makes it an even more intuitive task. Inclusivity analysis through verification of <em>in silico</em> nucleotide identity match is one of the regulatory requirements for approval of COVID-19 diagnostic assays (Commission-Services, 2020; FDA, 2020; Health-Canada, 2020). The protocol can also be applied to other molecular diagnostic assays of SARS-CoV-2 including point-of-care CRISPR-based diagnostic assays under development (Tsang and LaManna, 2020).</p>
        </body>

<p class="pptt" id="biaoti39350">### Equipment</p><ol class="npwtwok"><li><p>A regular Windows or Mac OS X laptop or desktop.</p><p><em>Note: There is no specific processor or RAM requirement, but memory issues can be avoided by opening a limited number of files at the same time. The outlined protocol was performed on a laptop installed with Windows 10, an Intel Core i5-8265U processor, CPU @1.60GHz and an 8 GB RAM.</em> </p></li></ol><p class="pptt" id="biaoti39351">### Software</p><ol class="npwtwok"><li><p>MAFFT version 7 online service (Katoh <em>et al.</em>, 2002       and 2019) (available from <a href="https://mafft.cbrc.jp/alignment/software/closelyrelatedviralgenomes.html" target="_blank">https://mafft.cbrc.jp/alignment/software/closelyrelatedviralgenomes.html</a>)</p></li><li><p>AliView version 1.26 (Larsson, 2014) (available from <a href="https://ormbunkar.se/aliview/" target="_blank">https://ormbunkar.se/aliview/</a>)</p></li><li><p>Sequence Manipulation Suite version 2 (Stothard, 2000) (available from <a href="https://www.bioinformatics.org/sms2/rev_comp.html" target="_blank">https://www.bioinformatics.org/sms2/rev_comp.html</a>)</p></li><li><p>SequenceTracer (Nagy <em>et al.</em>, 2019) (available from<a href="http://www1.szu.cz:8080/EntropyCalcWeb/sequences" target="_blank"> http://www1.szu.cz:8080/EntropyCalcWeb/sequences</a>)</p></li><li><p>ElimDupes (<a href="https://www.hiv.lanl.gov/content/sequence/elimdupesv2/elimdupes.html">https://www.hiv.lanl.gov/content/sequence/elimdupesv2/elimdupes.html</a>)</p></li><li><p>PNNS calculator (available from <a href="http://entropy.szu.cz:8080/EntropyCalcWeb/pnns" target="_blank">http://entropy.szu.cz:8080/EntropyCalcWeb/pnns</a>)</p></li><li><p>A web browser (for example Google Chrome or Mozilla Firefox)</p></li><li><p>A text editor (for example Microsoft Notepad)</p></li></ol><p class="pptt" id="biaoti39352">### Procedure</p><p>You can jump to Procedure D to download the latest version of a ready-to-use SARS-CoV-2 Multiple Sequence Alignment (MSA) file from our project page on OSF.</p><br/><ol class="npwone"><li><p>Viral sequence dataset</p><ol class="npwtwok"><li><p>Download the viral sequences from the repository of your choice.</p><p><em>Note: Check the terms and conditions of each repository with attention to the data sharing policy. Registration may be required.</em> </p><ol class="npwthree"><li><p>GISAID's EpiCoV database (<a href="https://www.gisaid.org/">https://www.gisaid.org/</a>) (Shu and McCauley, 2017).</p></li><li><p>NCBI virus (<a href="https://www.ncbi.nlm.nih.gov/labs/virus/vssi/#/" target="_blank">https://www.ncbi.nlm.nih.gov/labs/virus/vssi/#/</a>) (Hatcher <em>et al.</em>, 2017).</p></li><li><p>The Chinese National Genomics Data Center (NGDC) database (<a href="https://bigd.big.ac.cn/ncov">https://bigd.big.ac.cn/ncov</a>) (NGDC, 2020).</p></li><li><p>EMBL-EBI's COVID-19 Data Portal (<a href="https://www.covid19dataportal.org/">https://www.covid19dataportal.org/</a>).</p></li><li><p>COVID-19 Genomics UK (COG-UK) Consortium (<a href="https://www.cogconsortium.uk/data/">https://www.cogconsortium.uk/data/</a>).</p></li></ol></li><li><p>Download the complete genome of Wuhan-Hu-1 (NCBI Reference Sequence: NC_045512.2; <a href="https://www.ncbi.nlm.nih.gov/nucleotide/">https://www.ncbi.nlm.nih.gov/nucleotide/</a>).</p></li></ol></li><br/><li><p>Multiple Sequence Alignment (MSA) using MAFFT online service dedicated to MSA of closely-related viral genomes (<a href="https://mafft.cbrc.jp/alignment/software/closelyrelatedviralgenomes.html">https://mafft.cbrc.jp/alignment/software/closelyrelatedviralgenomes.html</a>).</p><ol class="npwtwok"><li><p>Input (Figure 1A):</p><ol class="npwthree"><li><p>The complete genome of Wuhan-Hu-1 (NC_045512.2) to the âExisting alignmentâ box.</p></li><li><p>Input the other sequences to the âFragmentary sequence(s)" box.</p><p><em>Note: MAFFT online service supports up to 20,000 sequences of ~30 kb in length. The task should be performed in batches if more sequences are being aligned and results should be combined after sequence stratification.</em> </p></li></ol></li><li><p>Parameters (Figure 1B):</p><ol class="npwthree"><li><p>UPPERCASE/lowercase, select "same as input'.</p></li><li><p>Direction of nucleotide sequences, select âAdjust direction according to the first sequenceâ.</p></li><li><p>Output order, select "aligned".</p></li></ol></li><li><p>Advanced Settings (Figure 1C):</p><ol class="npwthree"><li><p>Keep alignment length, select "Yes".</p></li><li><p>Strategy, select "auto".</p></li></ol></li><li><p>Download the aligned sequence in FASTA format, once available.</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3871/bioprotoc-10-24-3871-g001.jpg"/><br/><strong>Figure 1. Multiple Sequence Alignment (MSA) using MAFFT online service</strong> </p><br/><p>Video for Procedure A-B: <a href="https://youtu.be/hbnsXnikRak">https://youtu.be/hbnsXnikRak</a> </p></li></ol></li><br/><li><p>Alternatively, download a ready-to-use MSA file from our OSF page (<a href="https://doi.org/10.17605/OSF.IO/NPCS6">https://doi.org/10.17605/OSF.IO/NPCS6</a>).</p><p><em> Note: Data in our original publication (Khan and Cheung, 2020) was downloaded from GISAID that does not permit the release of MSA files publicly. The MSA file provided on our OSF page was generated using sequences downloaded from the NCBI virus. The file would be updated periodically during the pandemic (file 1 aligning 19863 SARS-CoV-2 sequences). </em> </p></li><br/><li><p>Save Region of Interest (ROI) for each primer/probe as a separate FASTA file</p><ol class="npwtwok"><li><p>Open the MSA file from Procedure B or Procedure C in the AliView program.</p><p><em>Note: Aliview program (available from </em><a href="https://ormbunkar.se/aliview/"><em>https://ormbunkar.se/aliview/</em></a><em>) needs to be downloaded on the computer in advance.</em> </p></li><li><p>Find the primer binding site using the âfindâ function (Figure 2A) or using âadd and align sequences from clipboardâ function (Figure 2B).</p></li><li><p>Reverse-complement the primer/probe sequence as necessary using Sequence Manipulation Suite (<a href="https://www.bioinformatics.org/sms2/rev_comp.html">https://www.bioinformatics.org/sms2/rev_comp.html</a>).</p><p>&gt;CN-CDC-N_F</p><p>GGGGAACTTCTCCTGCTAGAAT</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3871/bioprotoc-10-24-3871-g002.jpg"/><br/><strong>Figure 2. Finding the Region of Interest (ROI) in MSA</strong> </p></li><br/><li><p>Select the ROI and copy selection as FASTA format (Figure 3A).</p></li><li><p>Open a new file, "Paste (fasta-sequences)" and "Save" (Figure 3B). The sequence can be pasted in a text editor and saved as a FASTA file.</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3871/bioprotoc-10-24-3871-g003.jpg"/><br/><strong>Figure 3. Saving Region of Interest (ROI) for each primer/probe as a separate FASTA file</strong> </p><br/><p>Video for Procedure D: <a href="https://youtu.be/H9UxkgAsMdE">https://youtu.be/H9UxkgAsMdE</a> </p></li></ol></li><br/><li><p>Sequence stratification: Option 1 â SequenceTracer</p><ol class="npwtwok"><li><p>Upload the individual FASTA file to SequenceTracer and hit "Submit" (Figure 4A) (<a href="http://www1.szu.cz:8080/EntropyCalcWeb/sequences">http://www1.szu.cz:8080/EntropyCalcWeb/sequences</a>). The SequenceTracer segregates data into discrete groups of identical sequence variants and presents a detailed view of the nucleotide variation in each ROI along with the frequency of each variant. Moreover, the sequences showing ambiguous sequences are grouped as âoutgroup1â, short sequences are grouped as âoutgroup2â and missing sequences are grouped as âexcludedâ.</p></li><li><p>Download the stratified data showing a list of sequence variants and/or a chart (Figure 4B).</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3871/bioprotoc-10-24-3871-g004.jpg"/><br/><strong>Figure 4. Sequence stratification using SequenceTracer</strong> </p><br/><p>Video for Steps E1-E2: <a href="https://youtu.be/ysT_KBXkpvw">https://youtu.be/ysT_KBXkpvw</a> </p></li><br/><li><p>The "stratify" file can be opened using Microsoft Excel while the "compressed" file can be opened using the AliView program or a text editor (Figure 5).</p></li><li><p>The data of any sequence variant group can be downloaded (Figure 6).</p><ol class="npwthree"><li><p>First, select the group.</p></li><li><p>Then, âAdd all to Notesâ.</p></li><li><p>Finally, "export".</p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3871/bioprotoc-10-24-3871-g005.jpg"/><br/><strong>Figure 5. Expected results from SequenceTracer</strong> </p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3871/bioprotoc-10-24-3871-g006.jpg"/><br/><strong>Figure 6. Downloading data of a specific variant group</strong> </p></li></ol><br/><p>Video for Steps E3-E4: <a href="https://www.youtube.com/watch?v=4S0T9TW5ax4">https://www.youtube.com/watch?v=4S0T9TW5ax4</a> </p></li></ol></li><br/><li><p>Sequence stratification: Option 2 â ElimDupes</p><ol class="npwtwok"><li><p>Upload the individual FASTA file to ElimDupes (<a href="https://www.hiv.lanl.gov/content/sequence/elimdupesv2/elimdupes.html">https://www.hiv.lanl.gov/content/sequence/elimdupesv2/elimdupes.html</a>).</p></li><li><p>Select parameters as shown and hit "Submit" (Figure 7A).</p></li><li><p>View or Download "Unique sequences with rank and count appended (_count)" (Figure 8B).</p></li><li><p>The file would show sequence variants, along with rank and count added to the sequence name. The file can be opened using the AliView program or a text editor (Figure 7C).</p><p><em>Note: As opposed to SequenceTracer, variants with ambiguous sequences, and with missing sequences would be ranked in the results and need to be separated manually.</em> </p><br/><p><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/e3871/bioprotoc-10-24-3871-g007.jpg"/><br/><strong>Figure 7. Sequence stratification using ElimDupes</strong> </p></li></ol></li><br/><li><p>Position Nucleotide Numerical Summary (PNNS)</p><ol class="npwtwok"><li><p>As the sequence variation was moderate, the base composition of each nucleotide position was not analyzed in the original publication. This can be performed for highly variable regions using the Position Nucleotide Numerical Summary (PNNS) calculator (<a href="http://entropy.szu.cz:8080/EntropyCalcWeb/pnns">http://entropy.szu.cz:8080/EntropyCalcWeb/pnns</a>).</p></li></ol></li></ol><p class="pptt" id="biaoti39353">### Data analysis</p><p>First, the sequences with ambiguous nucleotides (outgroup 1), short sequences (outgroup 2) and missing sequences (excluded) are removed and the number of âinformativeâ sequences is calculated by subtracting these three groups from the total number of sequences. SequenceTracer performs this calculation automatically whereas the calculation needs to be performed manually if using ElimDupes. The informative group is then divided into hits with a perfect match and hits with mismatches for each primer and probe. To minimize the effect of low prevalent variants and sequencing errors in the data on the analysis, we defined a threshold of 0.5% in our original publication (Khan and Cheung, 2020) where only the sequence variants with â¥ 0.5% incidence were further considered. As more high quality viral sequences become available a more stringent threshold (for instance 0.1%) may be defined. Another way of defining a threshold is to include all the mutations in the analysis that occur more than once in different sequencing experiments (Osorio and Correia-Neves, 2020). The number and frequency of the sequences with the perfect match and with mismatches are then calculated from sequences above the defined threshold for each primer and probe. As an example, the analysis of CN-CDC-N forward primer 5â²-GGGGAACTTCTCCTGCTAGAAT-3â² (WHO, 2020) is shown in Table 1. The summary of the analysis for 27 previously published PCR assays is presented in Table 2 of our previous publication (Khan and Cheung, 2020).</p><p>    Mismatches can also be divided into mismatches in the 3â² end (last 5 nucleotides) and the 5â² end. It is known that PCR amplification is more prone to mismatches at the 3â² end of the primer (Whiley and Sloots, 2005;      Stadhouders <em>et al.</em>, 2010    ;      Lefever <em>et al.</em>, 2013    ). Moreover, mismatches in the probe can have a deleterious effect on PCR amplification. Even a single mismatch may reduce the sensitivity of the assay and lead to false-negative results due to the prevention of probe binding and subsequence fluorescence (Chow <em>et al.</em>, 2011    ;      Brault <em>et al.</em>, 2012).</p><br/><p><strong>Table 1. Analysis of CN-CDC-N forward primer<br/><img alt="" class="layerphoto" height="363" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20201216/20201216213346_4200.jpg" width="493"/></strong> </p><p class="pptt" id="biaoti39354">### Acknowledgments</p><p>The protocol is the detailed version of the method used in our previous publication (Khan and Cheung, 2020). We gratefully acknowledge the authors, originating and submitting laboratories of the sequences from GISAIDâs EpiCoV<sup>TM</sup> Database on which our research is based. The list is included in electronic supplementary material, file 1 of our original publication. Funding for this study was provided by the Canadian Institutes of Health Research operating grant (number RN227427 â 324983) awarded to PC.</p><p class="pptt" id="biaoti39355">### Competing interests</p><p>The author has no competing interests.</p><p class="pptt" id="biaoti39356">### References</p><ol class="npwtwok"><li>Brault, A. C., Fang, Y., Dannen, M., Anishchenko, M. and Reisen, W. K. (2012). <a href="https://pubmed.ncbi.nlm.nih.gov/22897055/" target="_blank">A naturally occurring mutation within the probe-binding region compromises a molecular-based West Nile virus surveillance assay for mosquito pools(Diptera: Culicidae).</a> <em>J Med Entomol</em> 49(4): 939-941.</li><li>Chan, J. F., Kok, K. H., Zhu, Z., Chu, H., To, K. K., Yuan, S. and Yuen, K. Y. (2020). <a href="https://pubmed.ncbi.nlm.nih.gov/31987001/" target="_blank">Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan.</a> <em>Emerg Microbes Infect</em> 9(1): 221-236.</li><li>Chow, C. K., Qin, K., Lau, L. T. and Cheung-Hoi Yu, A. (2011). <a href="https://pubmed.ncbi.nlm.nih.gov/22124020/" target="_blank">Significance of a single-nucleotide primer mismatch in hepatitis B virus real-time PCR diagnostic assays.</a> <em>J Clin Microbiol</em> 49(12): 4418-4419; author reply 4420.</li><li>Commission-Services (2020). Current performance of COVID-19 test methods and devices and proposed performance criteria- Working document of Commission services. Retrieved May 6, 2020, from <a href="https://ec.europa.eu/docsroom/documents/40805" target="_blank">https://ec.europa.eu/docsroom/documents/40805</a>.</li><li>FDA (2020). Policy for Coronavirus Disease-2019 Tests During the Public Health Emergency(Revised). Retrieved May 6, 2020, from <a href="https://www.fda.gov/media/135659/download" target="_blank">https://www.fda.gov/media/135659/download</a>.</li><li>Hatcher, E. L., Zhdanov, S. A., Bao, Y., Blinkova, O., Nawrocki, E. P., Ostapchuck, Y., Schaffer, A. A. and Brister, J. R. (2017). <a href="https://pubmed.ncbi.nlm.nih.gov/27899678/" target="_blank">Virus Variation Resource- improved response to emergent viral outbreaks.</a> <em>Nucleic Acids Res</em> 45(D1): D482-D490.</li><li>(2020). Applications for medical devices under the Interim Order for use in relation to COVID-19: Guidance document. Retrieved July 2, 2020, from <a href="https://www.canada.ca/en/health-canada/services/drugs-health-products/drug-products/announcements/interim-order-importation-sale-medical-devices-covid-19.html" target="_blank">https://www.canada.ca/en/health-canada/services/drugs-health-products/drug-products/announcements/interim-order-importation-sale-medical-devices-covid-19.html</a>.</li><li>Katoh, K., Misawa, K., Kuma, K. and Miyata, T. (2002). <a href="https://pubmed.ncbi.nlm.nih.gov/12136088/" target="_blank">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform.</a> <em>Nucleic Acids Res</em> 30(14): 3059-3066.</li><li>Katoh, K., Rozewicki, J. and Yamada, K. D. (2019). <a href="https://pubmed.ncbi.nlm.nih.gov/28968734/" target="_blank">MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization.</a> <em>Brief Bioinform</em> 20(4): 1160-1166.</li><li>Khan, K. A. and Cheung, P. (2020). <a href="https://royalsocietypublishing.org/doi/10.1098/rsos.200636" target="_blank">Presence of mismatches between diagnostic PCR assays and coronavirus SARS-CoV-2 genome.</a> <em>Royal Society Open Science</em> 7(6): 200636.</li><li>Larsson, A. (2014). <a href="https://pubmed.ncbi.nlm.nih.gov/25095880/" target="_blank">AliView: a fast and lightweight alignment viewer and editor for large datasets.</a> <em>Bioinformatics</em> 30(22): 3276-3278.</li><li>Lefever, S., Pattyn, F., Hellemans, J. and Vandesompele, J. (2013). <a href="https://pubmed.ncbi.nlm.nih.gov/24014836/" target="_blank">Single-nucleotide polymorphisms and other mismatches reduce performance of quantitative PCR assays.</a> <em>Clin Chem</em> 59(10): 1470-1480.</li><li>Lu, R., Zhao, X., Li, J., Niu, P., Yang, B., Wu, H., Wang, W., Song, H., Huang, B., Zhu, N., Bi, Y., Ma, X., Zhan, F., Wang, L., Hu, T., Zhou, H., Hu, Z., Zhou, W., Zhao, L., Chen, J., Meng, Y., Wang, J., Lin, Y., Yuan, J., Xie, Z., Ma, J., Liu, W. J., Wang, D., Xu, W., Holmes, E. C., Gao, G. F., Wu, G., Chen, W., Shi, W. and Tan, W. (2020). <a href="https://pubmed.ncbi.nlm.nih.gov/32007145/" target="_blank">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding.</a> <em>Lancet</em> 395(10224): 565-574.</li><li>Nagy, A., Jirinec, T., Jirincova, H., Cernikova, L. and Havlickova, M. (2019). <a href="https://pubmed.ncbi.nlm.nih.gov/30733500/" target="_blank">In silico re-assessment of a diagnostic RT-qPCR assay for universal detection of Influenza A viruses.</a> <em>Sci Rep</em> 9(1): 1630.</li><li>Osorio, N. S. and Correia-Neves, M. (2020). <a href="https://pubmed.ncbi.nlm.nih.gov/32473662/" target="_blank">Implication of SARS-CoV-2 evolution in the sensitivity of RT-qPCR diagnostic assays.</a> <em>Lancet Infect Dis</em> (20)30435-7.</li><li>Shu, Y. and McCauley, J. (2017). <a href="https://pubmed.ncbi.nlm.nih.gov/28382917/" target="_blank">GISAID: Global initiative on sharing all influenza data- from vision to reality. </a><em>Euro Surveill</em> 22(13): 30494.</li><li>Stadhouders, R., Pas, S. D., Anber, J., Voermans, J., Mes, T. H. and Schutten, M. (2010). <a href="https://pubmed.ncbi.nlm.nih.gov/19948821/" target="_blank">The effect of primer-template mismatches on the detection and quantification of nucleic acids using the 5' nuclease assay.</a> <em>J Mol Diagn</em> 12(1): 109-117.</li><li>Stothard, P. (2000). <a href="https://pubmed.ncbi.nlm.nih.gov/10868275/" target="_blank">The sequence manipulation suite: JavaScript programs for analyzing and formatting protein and DNA sequences.</a> <em>Biotechniques</em> 28(6): 1102-1104.</li><li>Tsang, J. and LaManna, C. M. (2020). <a href="https://www.liebertpub.com/doi/10.1089/crispr.2020.0030" target="_blank">Open Sharing During COVID-19: CRISPR-Based Detection Tools.</a> <em>The CRISPR Journal</em> 3(3): 142-145.</li><li>Ugurel, O. M., Ata, O. and Turgut-Balik, D. (2020). <a href="https://pubmed.ncbi.nlm.nih.gov/32595352/" target="_blank">An updated analysis of variations in SARS-CoV-2 genome.</a> <em>Turk J Biol</em> 44(3): 157-167.</li><li>Whiley, D. M. and Sloots, T. P. (2005). <a href="https://pubmed.ncbi.nlm.nih.gov/16157260/" target="_blank">Sequence variation in primer targets affects the accuracy of viral quantitative PCR.</a> <em>J Clin Virol</em> 34(2): 104-107.</li><li>WHO. (2020). In-house developed molecular assays. Retrieved April 16, 2020, from <a href="https://www.who.int/docs/default-source/coronaviruse/whoinhouseassays.pdf?sfvrsn=de3a76aa_2" target="_blank">https://www.who.int/docs/default-source/coronaviruse/whoinhouseassays.pdf?sfvrsn=de3a76aa_2</a>.</li><li>Worldometers.info (2020). COVID-19 Coronavirus Pandemic. Retrieved April, 16, 2020, from <a href="https://www.worldometers.info/coronavirus/" target="_blank">https://www.worldometers.info/coronavirus/</a>.</li><li>Wu, F., Zhao, S., Yu, B., Chen, Y. M., Wang, W., Song, Z. G., Hu, Y., Tao, Z. W., Tian, J. H., Pei, Y. Y., Yuan, M. L., Zhang, Y. L., Dai, F. H., Liu, Y., Wang, Q. M., Zheng, J. J., Xu, L., Holmes, E. C. and Zhang, Y. Z. (2020). <a href="https://pubmed.ncbi.nlm.nih.gov/32015508/" target="_blank">A new coronavirus associated with human respiratory disease in China.</a> <em>Nature</em> 579(7798): 265-269.</li><li>Zhou, P., Yang, X. L., Wang, X. G., Hu, B., Zhang, L., Zhang, W., Si, H. R., Zhu, Y., Li, B., Huang, C. L., Chen, H. D., Chen, J., Luo, Y., Guo, H., Jiang, R. D., Liu, M. Q., Chen, Y., Shen, X. R., Wang, X., Zheng, X. S., Zhao, K., Chen, Q. J., Deng, F., Liu, L. L., Yan, B., Zhan, F. X., Wang, Y. Y., Xiao, G. F. and Shi, Z. L. (2020). <a href="https://pubmed.ncbi.nlm.nih.gov/32015507/" target="_blank">A pneumonia outbreak associated with a new coronavirus of probable bat origin.</a> <em>Nature</em> 579(7798): 270-273.</li><li>NDGC. (2020). <a href="https://academic.oup.com/nar/article/48/D1/D24/5614641" target="_blank">Database Resources of the National Genomics Data Center in 2020</a>. <em>Nucleic Acids Res</em> 48:D24-D33. 10.1093/nar/gkz913.</li></ol>