<html><head><title>Factors other than hTau overexpression that contribute to tauopathy-like phenotype in rTg4510 mice</title></head><body><h1>Factors other than hTau overexpression that contribute to tauopathy-like phenotype in rTg4510 mice</h1>Authors: Julia Gamache, Kellie Benzow, Colleen Forster, Lisa Kemper, Chris Hlynialuk, Eva Furrow, Karen H. Ashe, Michael D. Koob<br>Abstract: <p>The tauopathy-like phenotype observed in the rTg4510 mouse line, in which human tau<sub>P301L</sub> expression specifically within the forebrain can be temporally controlled, has largely been attributed to high overexpression of mutant human tau in the forebrain region. Unexpectedly, we found that in a different mouse line with a targeted-insertion of the same transgene driven by the same tetracycline-TransActivator (tTA) allele, but with even higher overexpression of tauP301L than rTg4510, atrophy and tau histopathology are delayed, and a different behavioral profile is observed. This suggests that it is not overexpression of mutant human tau alone that contributes to the phenotype in rTg4510 mice. Furthermore we show that the tauopathy-like phenotype seen in rTg4510 requires a ~70-copy tau-transgene insertion in a 244kb deletion in <em>Fgf14</em>, a ~7-copy tTA-transgene insertion in a 508kb deletion that disrupts another five genes, in addition to high transgene overexpression. We propose that these additional effects need to be accounted for in any studies using rTg4510, and that Tg-INDEL mutations and their impacts on phenotype should be defined for all transgenic models used in biomedical research.</p><br>Keywords: Tauopathy, rTg4510, tau, Fgf14<br><h2>Procedure</h2><p><strong>Animals</strong></p><p>We generated ES cells with a Col1A1 “Flp-in” integration cassette by electroporating 25ug of purified Col1a-frt-hygro-pA Plasmid (Addgene) into V6.5 Mouse Embryonic stem cells (C57BL/6 X 129/sv) (Novus Biologicals NBP1-41162 passage 22). After G418 selection, Clone 15 was found to have integrated properly and had a perfect karyotype. All further FRT mediated targeting is to this ES modified cell line (V6.5Col1a#15). A construct was generated that was essentially identical to the construct used to generate Tg4510, but incorporated a Flp-In promoter cassette (PGK promoter-ATG-FRT), and 6 mg this construct and 0.5ug pCAGGS-FlpE (Gene Bridges cat# A201) was transfected into the V6.5Col1a#15 ES cell line. Hygromycin selection at 140ug/mL was added day 2 through day 6, and Hygro resistant ES clones were picked on day 7. DNA was analyzed for 5’ (5ArmCol1A assay primer + TRE start Rev) and 3’ (AMP R Reverse + Hygro Connection) junctions, internal Tau (TAU assay F and R), and multiple integration assay (AMP R Reverse + TRE start R) by PCR. Southern blot was done for multiple integration confirmation using EcoRI digested genomic DNA and a 470bp probe generated from the AmpR gene (Amp F + Amp R). Clone 6 was positive for all assays and the P301L mutation was verified, and this clone was expanded and karyotyped, and mice were generated by injection into blastocysts. These mice were back-crossed five times to FVB prior to generating the T2/T2 lines.</p><p><u>ES Cell Assay Primer sequences:</u></p><p>5ArmCol1A pcr assay &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;5’-CAGGTGCACAGCATTGCGGACATG-3’</p><p>&nbsp;TRE Start Rev&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;5’-ATTGCTCCAGGCGATCTGAC-3’</p><p>Amp R Reverse&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;5’-GGAATAAGGGCGACACGGAA-3’</p><p>Hygro Connecton&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;5’-ATCCACGCCCTCCTACATCGAA-3’</p><p>Tau Assay F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;5’-GTTCGAAGTGATGGAAGATCACG-3’</p><p>Tau Assay R&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;5’-TTGGGTGGAGTACGGACCA-3’</p><p><br></p><p><u>PCR Probe Primers:</u></p><p>Amp F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;5’-CCTCCATCCAGTCTATTAATT-3’</p><p>Amp R&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;5’-TCCTTGAGAGTTTTCGCCCCG-3’</p><p><br></p><p>To generate tau-homozygous rT2/T2 mice from tau-hemizygous T2 mice, transgene-activated hemizygous males (CKTTA+/-Tau+/-) were bred to non-activated hemizygous females (CKTTA-/-Tau+/-), resulting in tau homozygous progeny (Tau+/+). To maintain the rT2/T2 line, transgene-activated homozygous males (CKTTA +/-Tau+/+) were bred to non-activated homozygous females (CKTTA-/-Tau+/+). Generation of rTg4510 mice, which utilizes an activator and responder system for transgene expression, has previously been described<sup>2</sup>. Briefly, a pTRE-prnp-tau plasmid was used to generate Tg4510 responders, which harbor a 0N4R human tau cDNA transgene regulated by a tetracycline response element (TRE). Activator mice harbor a tetracycline transactivator (tTA) transgene under the control of the CaMKIIa promoter to drive expression specifically in forebrain excitatory neurons. Tau expression is activated in bigenic rTg4510 progeny of an activator-responder cross. For the rTg4510 line, responder Tg4510 were maintained on a FVB/N background while activator mice were maintained on a 129S6 background. To match the genetic background to that of rT2/T2 mice, we increased the amount of FVB/N and used these mice to compare expression levels in rTg4510 and rT2/T2 mice.&nbsp;To generate these mice, responder Tg4510 mice were maintained on a FVB/N background while activator mice were maintained on a mixed 129S6 and FVB/N background. &nbsp;Non-tau-expressing transgenic littermates were used as controls. &nbsp;Both male and female mice were used, and were combined in statistical analyses after demonstrating the absence of significant gender effects (P &gt; 0.05). &nbsp;All experiments with animals described in this study were approved by and conducted in full accordance with the American Association for the Accreditation of Laboratory Animal Care and the Institutional Animal Care and Use Committee at the University of Minnesota.</p><p><strong>qRT-PCR</strong></p><p>mRNA expression levels of each FGF14 variant were quantified relative to a reference gene, <em>HPRT</em>.&nbsp;All FGF14 primers (Supplementary Table 1) were designed to span at least partially unique regions of each variant.&nbsp;Total cellular RNA was extracted from homogenized forebrain tissue using RNeasy Lipid Tissue Kit (Qiagen) according to the manufacturer’s instructions. &nbsp;RNA samples were treated with DNaseI (New England Biolabs) to digest contaminating DNA, and subjected to cDNA synthesis using the iScript cDNA synthesis kit (Invitrogen) according to the manufacturer’s instructions. &nbsp;PCR reactions were set up in a 20-µl volume in 96-well plates, with 2 replicates per sample.&nbsp;SYBR Green PCR master mix (Roche) was used and reactions were run in the LightCycler® 480 instrument (Roche) (Supplementary Table 2). A final melting curve confirmed that single amplicons were present for each variant and reference reactions, and a basic relative quantification was performed using the ∆∆C<sub>T</sub>-Method (LightCycler® 480 Software release 1.5.0 SP3). &nbsp;All data were normalized to a positive calibrator sample used in each experiment. </p><p><strong>Protein extraction and phosphatase treatment</strong></p><p>Total protein was extracted from mouse forebrain hemisphere tissue in RIPA buffer (50 mM Tris-HCl, 150 mM NaCl, 1 mM EDTA, 0.5% Triton X-100, 1% sodium deoxycholate, 0.3% SDS, 0.1 mM phenylmethyl sulfonyl fluoride, 0.2 mM 1,10-Phenoanthroline Monohydrate, Phosphatase Inhibitor Cocktail A (Sigma), Protease Inhibitor Cocktail (Sigma), Phosphatase Inhibitor Cocktail 2 (Sigma)). Homogenates were nutated and centrifuged at 15,700 xg for 90 minutes at 4°C and the supernatant was collected. </p><p>To obtain RIPA-insoluble, sarkosyl-insoluble fractions, a modified version of a previously published method was used<sup>31</sup>. RIPA-insoluble pellets were homogenized in 1% sarkosyl and incubated at room temperature for 30 minutes with constant shaking. Samples were centrifuged for 1 hour at 100,000 xg at 20°C, and the supernatant and pellet were separated and diluted in O+ buffer&nbsp;(62.5 mM Tris-HCl, pH 6.8; 10% glycerol; 5% 2-mercaptoethanol; 2.3% SDS; 1 mM EGTA; 1 mM EDTA; 1 mM PMSF; 1 mM Na3VO4; 1 mM NaF; 10 μl/ml of protease inhibitor cocktail P8340; Sigma-Aldrich). Samples were boiled for 3 minutes and stored at -20°C. </p><p>For treatment with calf intestinal alkaline phosphatase (CIP, New England Biolabs), samples were resuspended in 10 µl CIP buffer (100 mM NaCl, 50 mM Tris-HCl, 10 mM MgCl2, 1 mM dithiothreitol, EDTA-free protease inhibitor cocktail, pH 7.9) per 1 µg protein. One unit CIP per µg protein was added to the samples prior to incubation at 37°C for 30 minutes. Samples were then concentrated using Amicon Ultra centrifugal filters (Millipore). </p><p><strong>Western blot and analysis</strong></p><p>For sarkosyl-insoluble fractions, total protein concentration was normalized according to pellet weights. Total protein concentrations for all other samples were determined by Pierce™ Bicinchoninic Acid protein assay (Thermo Scientific).&nbsp;Equal amounts of protein for each sample were loaded and separated using SDS-PAGE on 10%, 10.5-14%, or 10-20% Tris-HCl gels (Bio Rad). Protein was transferred to nitrocellulose membranes (Bio Rad), which were blocked with 5% Bovine Serum Albumin (Sigma) in 1X TBST buffer (10 mM Tris-Base (Sigma), 0.2 M NaCl (Macron Chemicals), 0.1% Tween-20 (Sigma) pH 7.4).&nbsp;Protein was immunoblotted with Tau46 (Cell Signaling Technology #4019, dilution 1:10,000), Tau13 (BioLegend #MMS-520R, dilution 1:60,000), GAPDH(14C10) (Cell Signaling Technology #2118, dilution 1:4,000), GAPDH(GA1R) (Thermo Scientific #MA5-15738, dilution 1:5,000), AT8 (Thermo Scientific #MN1020, dilution 1:1,000), anti- human tau (Abcam # ab74391, dilution 1:10,000), and βIII-tubulin (ProSci #79-720, dilution 1:10,000) antibodies. Additional antibodies from Peter Davies for phospho-epitopes on tau included MC1 (dilution 1:800), CP13 (dilution 1:1,000), and PHF1 (dilution 1:1,500). To visualize antibody immunoreactivity using a LiCor imaging system and Image Studio software, IRDye-linked goat anti-mouse 800CW and goat anti-rabbit 680LT secondary antibodies were used (LI-COR Biosciences,dilution 1:100,000). Following LiCor image acquisition using Image Studio software (Odyssey), Amido black staining solution (Sigma-Aldrich) was used for total protein quantification according to manufacturer’s instructions. Immunoreactivity and Amido black staining were quantified by densitometry using OptiQuant version 3 software, following guidelines for total protein quantification<sup>30</sup>.</p><p><strong>Immunohistochemistry and analysis</strong></p><p>Mouse brain hemispheres were immersion-fixed in 10% formalin for 48 hours before processing. Unstained sagittal TMA sections (4 µm) were de-paraffinized and rehydrated using standard methods. Bielschowsky silver staining was performed using standard techniques. For antigen retrieval, slides were incubated in 6.0 pH buffer (Reveal Decloaking reagent, Biocare Medical, Concord, CA) in a steamer for 30 min at 95-98°C, followed by a 20 min cool down period.&nbsp;Subsequent steps were automated using an immunohistochemical staining platform (Nemesis, Biocare).&nbsp;Endogenous peroxidase activity was quenched by slide immersion in 3% hydrogen peroxide solution (Peroxidazed, Biocare) for 10 min followed by TBST rinse.&nbsp;A serum-free blocking solution (Rodent Block M, Biocare) Medical, Concord, CA) was placed on sections for 20 min.&nbsp;Blocking solution was removed and slides were incubated in primary antibody diluted in 10% blocking solution/90% TBST. Mouse monoclonal antibodies from Peter Davies were applied at the following dilutions: CP13 1:1,000, MC-1 1:800 and PHF1 1:1,500, mouse monoclonal PHF-Tau: clone AT8 (Thermo Scientific #MN1020) 1:1,000. Sections were incubated in primary antibody for 60 min at room temperature followed by TBST rinse and detection with biotinylated anti-mouse secondary (Vector Laboratories #BP-9200, dilution 1:200) for 30 minutes followed by a TBST rinse. After the rinse, SA-HRP (Biolegend #405210, RTU) was applied for 30 minutes. All slides then proceeded with TBST rinse and detection with diaminobenzidine<sup>32</sup>(Covance, Dedham, MA).&nbsp;Slides were incubated for 5 min followed by TBS rinse then counterstained with CAT Hematoxylin (Biocare, Concord, CA) for 5 min.&nbsp;Slides were then dehydrated and coverslipped. Images were gathered using an Axioskop microscope (Zeiss, Germany) at 40X magnification and a PixeLINK microscope camera (PL-A623C) with PixeLINK Capture SE software version 2.2 (Firewire camera release 4, Copyright © 2000-2006). Adobe Photoshop CS2 version 9.0 was used to match the color of different images of the same histological stain. Semi-quantitative analysis of images was conducted using a ‘+’ system<sup>1</sup>. A blinded observer gave scores for three sections per sample indicating severity of pathology using ‘-’ for no positive labeling, ‘+’ for occasional positive labeling, ‘++’ for moderate positive labeling, ‘+++’ for prominent positive labeling. To summarize these results, the number of ‘+’ signs was counted for each animal, and region, and multiple linear regression analyses were conducted using R statistical programming language to test for differences between rTg4510, rT2/T2, and rT2. </p><p><strong>Whole-genome sequencing and sequence analyses</strong></p><p>Genomic DNA was extracted from a non-activated Tg4510 mouse harboring the 0N4R human tau transgene using the DNeasy Blood &amp; Tissue kit (Qiagen) according to the manufacturer’s instructions. &nbsp;Genomic DNA was analyzed by nanodrop and agarose gel to verify the quality (O.D. 260/280 ratio &gt; 1.8) and quantity (300 ng for library construction).</p><p>Sequencing was performed on Illumina HiSeq 2500 High-Output system<sup> </sup>using rapid SBS chemistry at the University of Minnesota Genomics Center, Minneapolis, MN. &nbsp;Following quality control, a TruSeq Nano DNA library was prepared from the genomic DNA sample and was sequenced on a single Illumina lane to generate 2x125 bp paired-end reads. &nbsp;The average insert length in the library was 350 bp. &nbsp;</p><p>All sequence analyses were conducted using the University of Minnesota’s installation of the Galaxy web-based suite of software<sup>33</sup>. We isolated individual sequence reads that spanned the end of the transgene using the BLAT alignment tool<sup>34</sup> and mapped paired sequence reads to the transgene sequence using Bowtie2<sup>35</sup>. We screened these data sets and identified unmapped reads in which the paired read mapped to the transgene sequence and then further analyzed these screened sets, in part using the Integrative Genomic Viewer (IGV)<sup>36</sup>, to find the genomic insertion points and a single 5’ -5’ transgene fragment junction. Bowtie2 mapping of the sequence data to the <em>Fgf14</em> genomic or <em>Vipr2-Ptprn2</em> data (as appropriate) indicated that roughly half as many reads mapped to the unique sequences between the insertion points as mapped outside of this region, indicating that the Tg4510 mice have only a single copy of this portion of the <em>Fgf14</em> genomic sequence. To estimate transgene copy number, we used sets of closely linked SNPs to distinguish between the PrnP sequence in the transgene and that in the FVB genome<sup>37</sup>, and found that the dataset contains more than 35-fold more reads generated from transgene PrnP sequences than from the diploid mouse PrnP sequence on chromosome 2, indicating that more than 70 copies of transgene sequence are inserted at the <em>Fgf14</em> locus in the Tg4510 mice. Similar analyses were performed using polymorphisms in the CamKII genomic data to determine the copy number in the tTA transgene array. Sequences mapped to the reference Tg sequence were assemble into a consensus transgene sequence using SPADES<sup>38</sup>.</p><p><strong>PCR and Sequencing</strong></p><p>Selected regions of Tg4510 and rT2/T2 genomic DNA were amplified by polymerase chain reaction (PCR) using Herculase II Fusion DNA polymerase (Agilent Technologies).&nbsp;For Tau transgene junctions in Tg4510 DNA, PCR and nested PCR reactions were run (Supplementary Tables 3, 4 and 5).&nbsp;CaMKIIα-tTA transgene junctions were confirmed using rT2/T2 DNA (Supplementary Tables 6 and 7). Gel bands were isolated from a low melting point agarose gel (NuSieve GTG, Lonza) and DNA was purified following digestion with β-agarase (New England Biolabs).&nbsp;Gel-purified PCR products were sequenced with classical Sanger sequencing at the University of Minnesota Genomics Center, Minneapolis, MN. </p><p><strong>Mouse cell culture and metaphase slide preparation</strong></p><p>Spleen from a Tg4510 mouse (28<sup>th</sup> generation backcross onto FVB genetic background) was minced into a single cell suspension and cultured for&nbsp;24-28 hours with 5ug/ml Concavalin A (mouse T-cell mitogen).&nbsp;Cultures were&nbsp;exposed to colcemid (Irvine Scientific) for&nbsp;15 hrs overnight followed by harvest using standard cytogenetic protocols.&nbsp;Metaphase spread slides were prepared from methanol-acetic acid fixed cell pellets and FISH was performed the following day.</p><p><strong>Fluorescent in situ hybridization (FISH)</strong></p><p>DNA probes derived from the tau transgene sequence were labeled by nick translation reaction (Nick Translation Kit - Abbott Molecular) using&nbsp;Orange 552 dUTP (Enzo Life Science), ethanol precipitated and resuspended in hybridization buffer.&nbsp;The probe/hybridization buffer mix and slide were denatured, probe was applied to the metaphase slide, and slide was hybridized for 24 hours at 37° in a humidified chamber. After hybridization, the FISH slides were washed in a 2xSSC solution and counterstained with DAPI stain to enable chromosome identification by G-band patterning.&nbsp;Fluorescent signals were visualized on an Olympus BX61 microscope workstation (Applied Spectral Imaging, Vista, CA) with DAPI and&nbsp;Texas Red&nbsp;filter sets.&nbsp;FISH images were captured using an interferometer-based CCD cooled camera (ASI) and FISHView ASI software.</p><p><strong>Behavioral Experiments</strong></p><p>For nesting experiments, animals were not disturbed for at least 24 hours prior to testing. Each animal was placed in its own clean cage with a new nestlet placed in the center of the cage in the morning. Nests were photographed and scored 2 hours, 6 hours and 24 hours after the start of the test. Scores of 0-7 were given as previously described<sup>9</sup>: 0-nestlet untouched, 1- &lt;10% of nestlet was shredded, 2- 10-50% of nestlet shredded but no shape to nest (flat), 3- 10-50% of nestlet&nbsp;shredded and there is shape to nest, 4- 50-90% of nestlet shredded but no shape to nest (flat), 5- 50-90% of nestlet&nbsp;shredded and there is shape to nest, 6-&nbsp;&gt;90% of nestlet shredded but no shape to nest (flat), 7- &gt;90% of nestlet&nbsp;shredded and nest had walls that were at least as tall as the mouse on 50% of the sides. Reported scores are the average of two individual scorers. &nbsp;</p><p>Open field testing was done two day after the conclusion of nesting experiments. Animals were not disturbed for at least 24 hours prior to testing and were placed in the testing room for 30 minutes prior to testing. Open field was a plastic tub with opaque white walls, measuring 15 inches wide by 18.5 inches long by 12 in high. Arena floor was covered with new, pre-scented cage bedding. Mouse was released in the center of the arena and allowed to freely explore for 10 minutes. All trials were monitored using a computerized tracking system (Noldus Ethovision XT 10.0; Noldus Information Technology). All animals were tested in the morning to avoid activity differences from time of day.</p><h2>References</h2><p><sup>1</sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Ramsden, M.<em> et al.</em> Age-dependent neurofibrillary tangle formation, neuron loss, and memory impairment in a mouse model of human tauopathy (P301L). <em>J Neurosci</em> <strong>25</strong>, 10637-10647, doi:25/46/10637 (2005).</p><p><sup>9</sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Maeda, S.<em> et al.</em> Expression of A152T human tau causes age-dependent neuronal dysfunction and loss in transgenic mice. <em>EMBO Rep</em> <strong>17</strong>, 530-551, doi:embr.201541438 (2016).</p><p><sup>30</sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Aldridge, G. M., Podrebarac, D. M., Greenough, W. T. &amp; Weiler, I. J. The use of total protein stains as loading controls: an alternative to high-abundance single-protein controls in semi-quantitative immunoblotting. <em>J Neurosci Methods</em> <strong>172</strong>, 250-254, doi:S0165-0270(08)00288-4 (2008).</p><p><sup>31</sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Planel, E.<em> et al.</em> Acceleration and persistence of neurofibrillary pathology in a mouse model of tauopathy following anesthesia. <em>FASEB J</em> <strong>23</strong>, 2595-2604, doi:fj.08-122424 (2009).</p><p><sup>32</sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Dabir, D. V.<em> et al.</em> Impaired glutamate transport in a mouse model of tau pathology in astrocytes. <em>J Neurosci</em> <strong>26</strong>, 644-654, doi:26/2/644 (2006).</p><p><sup>33</sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Afgan, E.<em> et al.</em> The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update. <em>Nucleic Acids Res</em> <strong>44</strong>, W3-W10, doi:gkw343 (2016).</p><p><sup>34</sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Kent, W. J. BLAT--the BLAST-like alignment tool. <em>Genome Res</em> <strong>12</strong>, 656-664, doi:10.1101/gr.229202. Article published online before March 2002 (2002).</p><p><sup>35</sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Langmead, B. &amp; Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. <em>Nat Methods</em> <strong>9</strong>, 357-359, doi:nmeth.1923 (2012).</p><p><sup>36</sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Thorvaldsdottir, H., Robinson, J. T. &amp; Mesirov, J. P. Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. <em>Brief Bioinform</em> <strong>14</strong>, 178-192, doi:bbs017 (2013).</p><p><sup>37</sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Westaway, D.<em> et al.</em> Structure and polymorphism of the mouse prion protein gene. <em>Proc Natl Acad Sci U S A</em> <strong>91</strong>, 6418-6422 (1994).</p><p><sup>38</sup>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Bankevich, A.<em> et al.</em> SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. <em>J Comput Biol</em> <strong>19</strong>, 455-477, doi:10.1089/cmb.2012.0021 (2012).</p><p><br></p><p><br></p></body></html>