<html><head><title>Recovery of Paired T Cell Receptors from Single-cell Seq-Well Libraries</title></head><body><h1>Recovery of Paired T Cell Receptors from Single-cell Seq-Well Libraries</h1>Authors: Ang A. Tu, Todd M. Gierahn, Brinda Monian, Duncan M. Morgan, Naveen K. Mehta, Bert Ruiter, Wayne G. Shreffler, Alex K. Shalek, J. Christopher Love<br>Abstract: <p>High-throughput 3’ single-cell RNA-Sequencing (scRNA-Seq) allows for cost-effective, detailed characterization of thousands of individual immune cells from healthy and diseased tissues. Current techniques, however, are limited in their ability to elucidate essential immune cell features, including the variable sequences of T cell receptors (TCRs) that confer antigen specificity in T cells. Here, we present an enrichment strategy that enables simultaneous analysis of TCR variable sequences and corresponding full transcriptomes from 3’ barcoded scRNA-Seq samples. This approach is compatible with common 3’ scRNA-Seq methods, and adaptable to processed samples <em>post hoc</em>. We applied the technique to resolve clonotype-to-phenotype relationships among antigen-activated T cells from immunized mice and from patients with food allergy. We observed diverse but preferential&nbsp;cellular phenotypes manifest among subsets of expanded clonotypes, including functional Th2 states associated with food allergy. These results demonstrate the utility of our method when studying complex diseases in which clonotype-driven immune responses are critical to understanding the underlying biology.</p><br>Keywords: Seq-Well, RNA-Seq, TCR, Single-cell<br><h2>Introduction</h2><p>Antigen-specific T cells play key roles in a number of diseases including autoimmune disorders and cancer<sup>1–4</sup>. Assessing the phenotypes and functions of these cells is essential to both understanding underlying disease biology and designing new therapeutic modalities<sup>5–7</sup>.&nbsp;To study antigen-specific T cells comprehensively, two sequencing-based approaches have emerged: 1) bulk genomic sequencing of T cell receptor (TCR) repertoires to assess clonal diversity, and 2) RNA-Sequencing (RNA-Seq) to reveal phenotypic attributes. The TCR recognizes antigenic peptides bound in major histocompatibility complex (MHC) receptors and mediates CD3-dependent signaling upon cognate recognition; sequencing of the TCR repertoire thus can highlight clonotypic diversity and the dynamics of antigen-dependent responses associated with disease, such as clonal expansion or selection<sup>2,8,9</sup>. RNA-Seq, in contrast, can reveal novel states and functions of disease-relevant T cells through unique patterns of gene expression, albeit without determination of whether those cells are recognizing common antigens<sup>10–13</sup>.</p><p>Here, we report a simple process to sequence concomitantly both the transcriptome and receptor sequences of T cells from a single sequencing library generated using a massively-parallel 3’ scRNA-Seq platform, such as Seq-Well or Drop-seq (<strong>Figure</strong> 1). In our approach, a 3’ barcoded whole transcriptome amplification (WTA) is performed using standard protocols for Seq-Well or Drop-seq<sup>14-16</sup>. Next, one fraction of the amplified product is used to generate a 3’ scRNA-Seq library to quantify single-cell mRNA expression levels in individual cells; another is used to perform direct single-cell TCR sequencing.&nbsp;For the latter, TCR transcripts contained within amplified 3’ barcoded WTA products are first enriched by affinity capture using biotinylated oligonucleotide-probes targeting TCR constant regions, and are then re-amplified with the same universal primer site (UPS) used in the original WTA (<strong>Supplementary Table</strong> <strong>1a,b</strong>). Next, the amplified pool is selectively modified by primer extension using a set of oligonucleotides comprising a shared 5’ second universal primer site (UPS2) linked to regions specific for each TCR V-region (<strong>Supplementary Table 2</strong>). The modified products are then further amplified with primers specific for UPS2 and an extended version of the original UPS, which is only present on the 3’ end of the library. This final amplification adds flanking regions of oligonucleotides for sequencing to the TCR enriched library (<strong>Supplementary Table</strong> <strong>1b</strong>). The combination of targeted enrichment and primer extension produces a size-defined, CDR3-enriched library that does not require additional fragmentation before short-read sequencing (e.g., Illumina). The resulting library can then be sequenced using custom primers targeting the constant region of the TCR to directly recover the CDR3 region (<strong>Figure</strong> <strong>1</strong>, bottom right; <strong>Supplementary Table</strong> <strong>1c</strong>). This approach retains the original cellular and molecular identifiers appended during WTA, making it possible to match TCR sequences computationally with corresponding whole transcriptome profiles determined using traditional 3’ scRNA-Seq.</p><p><br></p><p><strong>Definition of terms:</strong></p><p><strong>WTA- </strong>Whole transcriptome amplified products. Full length transcriptome products that are single-cell barcoded on the 3’ side. In this protocol, WTA libraries were constructed via Seq-Well, which has identical WTA structure as DropSeq.</p><p><strong>&nbsp;Pull-down probes- </strong>Oligonucleotide probes with biotinylated 5’ end. This protocol uses probes targeting the constant region of the TCRs. Used in conjunction with the IDT xGen enrichment kit to target for TCR whole transcripts. See Supplementary Table 1 in the original publication for exact sequences.</p><p><strong>&nbsp;UPS primers- </strong>Universal priming site primers. UPS and UPS2 primers are used to uniformly amplified all products and construct Illumina-compatible libraries. See Supplementary Table 1 for exact sequences.</p><h2>Reagents</h2><p>xGen Hybridization and Wash Kit (IDT; Cat.No.1072281)</p><p>KAPA HiFi Hotstart Readymix PCR Kit (Kapa Biosystems Cat No. KK-2602)</p><p>Dynabeads M-270 Streptavidin (Invitrogen; Cat.No.65306)</p><p>AmpureXP (Beckman Coulter; A63880) or homemade Serapure SPRI reagent (see appendix for recipe)</p><p>Human Cot-1 DNA (Invitrogen; Cat.No.15279011)</p><p>UltraPure Distilled Water (Thermo Fisher Scientific Cat No. 10977023, or equivalent)</p><p>PCR primers, listed in Supplementary Table 1 and 2.</p><p><br></p><p>Optional for homemade SPRI reagent (see end of procedure):</p><p>Sera-mag SpeedBeads (Fisher # 09-981-123)</p><p>PEGN8000 (Amresco 0159) </p><p>0.5 M EDTA, pH 8.0 (Amresco E177) </p><p>1.0 M Tris, pH 8.0 (Amresco E199) </p><p>Tween 20 (Amresco 0777) </p><p>5 M NaCL </p><p>Fermentas ladder(s) (Ultra-low range:&nbsp;Fisher # FERSM1211, 50 bp: FERSM0371)</p><h2>Equipment</h2><p>PCR Thermocycler (e.g. Eppendorf Mastercyler; Catalog No.&nbsp;6337000027)</p><p>DynaMag-96 Side Magnet (Invitrogen; Cat.No.12331D), or equivalent</p><p>DynaMag-2 Magnet (Invitrogen; Cat.No.12321D), or equivalent</p><p>Optional: Eppendorf ThermoMixer C (Eppendor; Catalog No.&nbsp;2231000574)</p><h2>Procedure</h2><p><strong>I. TCR Enrichment from 3’ Barcoded Single-cell Libraries:</strong></p><p>The protocol assumes that whole transcriptome amplified products generated by the Seq-Well/DropSeq platform are being used.</p><p>&nbsp;</p><p>Before starting:</p><p>&nbsp;</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Make sure to dilute all the necessary buffers from the xGen kit (IDT; Cat.No.1072281). Extra can be kept at 4<sup>o</sup>C, and brought to 37<sup>o</sup>C before use to make sure all salts go into solution.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Pull-down-probes should be diluted ahead of time. We highly suggest aliquot the probes in multi-channel compatible format (i.e. in strip tubes with caps).</p><p>2a.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;TCRα and TCRβ probes should be mixed and diluted to 1.5uM (each, or 3 uM total)</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Thaw both the 2x hybridization buffer and the xGen buffer enhancer from the xGen kit, and bring to room temperature.</p><p>&nbsp;</p><p>A.&nbsp;&nbsp;&nbsp;Make the following master mix (per sample/reaction):</p><p>&nbsp;</p><p>UPS (50 uM)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.8 uL</p><p>Cot-1 DNA&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.5 uL</p><p>2X Hybridization Buffer&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;8.5 uL</p><p>xGen Buffer Enhancer&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2.7 uL</p><p>Total&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;12.5 uL</p><p>&nbsp;</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Mix 3.5 uL of WTA product with the master mix (for a total of 16 uL)</p><p>1a.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;We recommend using 0.2mL PCR strip tubes with caps.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Incubate mixture at room temperature for 5 minutes</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Then incubate mixer at 95 degree for 10 minutes (could be done in thermocycler, with lid set to 105 degrees)</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Remove mixture to room temperature, and add 1uL of pull-down probes mix (see above). Vortex to mix, and centrifuge to collect liquid.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Incubate the mixture at 65 degree for one hour to allow for hybridization of pull-down probes to WTA libraries.</p><p>&nbsp;</p><p>B.&nbsp;&nbsp;&nbsp;&nbsp;During the incubation, prepare the streptavidin beads (Invitrogen; Cat.No.65306)</p><p>&nbsp;</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Aliquot 50uL of beads per sample into a 1.5 mL microcentrifuge tube.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Place the tube on magnetic stand and allow for the beads to pellet.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Remove supernatant, and add equal volume of bead wash buffer into the tube</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Vortex to mix, and place back on the magnetic stand</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Repeat 3-4</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Take tube off of magnet, and add equal volume of bead was buffer</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Aliquot 50uL (per sample) of the mixture into PCR strip tubes.</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Place tubes back on magnetic stands.</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Remove the supernatant. The beads are now ready for hybridized mixes from A.</p><p>&nbsp;</p><p>&nbsp;</p><p>C.&nbsp;&nbsp;&nbsp;&nbsp;After the pull-down probes have been hybridized to WTA libraries</p><p>&nbsp;</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add mixes from A into the prepared beads from B.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Vortex to mix.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Incubate mixture at 65 degrees for 45 minutes.</p><p>a.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Intermittently vortex the mixture every 10 minutes to keep the streptavidin beads suspended in the mixture.</p><p>b.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;If possible, we recommend using a thermal shaker (e.g. Eppendorf ThermoMixer C) to automate intermittent vortexing.</p><p>&nbsp;</p><p>D.&nbsp;&nbsp;&nbsp;Prepare buffer for 65 degree wash steps.</p><p>&nbsp;</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;During incubation of C, aliquot and preheat Wash Buffer 1 (WB1) and Stringent Wash Buffer (SWB) at 65 degree.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Preheat at least 100 uL/sample of WB1 and 400uL/sample of SWB (2x200uL).</p><p>a.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;We recommend aliquot the buffers into PCR strip tubes, and preheat on thermocycler/thermoshaker for fast preheating.</p><p>&nbsp;</p><p>E.&nbsp;&nbsp;&nbsp;&nbsp;Wash hybridized beads at 65 degrees.</p><p>&nbsp;</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;After C, add 100uL of heated WB1 into each hybridized mixture, and place onto magnetic stand.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Aspirate supernatant, and take mixtures off of magnetic stand.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add 200uL of heated SWB into each reaction, and resuspend the bead pellets</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Inbcubate mixture at 65 degrees for 5 minutes</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Place mixture back on magnetic stand, and repeat 2-4</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Aspirate the final wash of SWB.</p><p>&nbsp;</p><p>F.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Wash hybridized beads at room temperature</p><p>&nbsp;</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add 200uL of room temperature WB1 into each reaction</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Vortex for 2 minutes to mix, spin to collect liquid, and place on magnetic stand.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Aspirate, and add 200uL of room temperature WB2.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Vortex for 1 minutes to mix, spin to collect liquid, and place on magnetic stand.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Aspirate, and add 200uL of room temperature WB3.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Vortex for 30 seconds to mix, spin to collect liquid, and place on magnetic stand.</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Aspirate, and add 20uL of water.</p><p>&nbsp;</p><p>G.&nbsp;&nbsp;&nbsp;PCR amplify pull-down TCR products</p><p>&nbsp;</p><p>For each reaction, we will perform 5 PCR reactions, using 2uL of mixture from F in each reaction (using a total of 10 out of 20 uL).</p><p>&nbsp;</p><p>Make the following master mix (per reaction).</p><p>&nbsp;</p><p>Kapa Hifi Hotstart Readymix 2X (Kapa Biosystems)&nbsp;&nbsp;12.5 uL</p><p>UPS(10uM)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2uL</p><p>Water&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;8.5uL</p><p>&nbsp;</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Aliquot 5x 23uL of master mix for each reaction</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add 2uL of mixture from F into each of the aliquots</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;PCR amplify using the following condition:</p><p>&nbsp;</p><p>1 cycle of</p><p>[95oC for 3 minutes]</p><p>25 cycles of</p><p>[98oC for 40 seconds</p><p>67oC for 20 seconds</p><p>72oC for 1 minute]</p><p>1 cycles of</p><p>[72oC for 5 minutes]</p><p>Hold at 4oC</p><p>&nbsp;</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;After amplification, pool all 5 PCR reactions (20uL for each sample) into a single tube (for 100uL total)</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Perform SPRI purification, aiming to remove all products &lt; 1kbp</p><p>5a.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;We recommend using AmpureXP or homemade Serapure SPRI reagent (see appendix for protocol)</p><p>5b.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Elute the samples into a final volume of 13-15 uL of water.</p><p>5c.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The final product contains both full-length TCR alpha and beta products.</p><p>5d.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;We recommend assessing the quality of the enrichment via fragment analyzer.</p><p>5e.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;See example below for expected size distribution.</p><p>5f.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Take note of estimated concentration</p><p>5g.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;If amplification was unsuccessful (either due to low concentration of product, or technical error), the rest of the beads from F can be used to repeat G.</p><p><br></p><p>See <strong>Figure 2</strong> for examples of successfully enriched products.</p><p><br></p><p><strong>II. TCR V-region primer extension</strong>:</p><p>After enriching for full-length TCR products, we then perform primer-extension using TCRV-UPS2 primers (see <strong>Supplementary Table 1</strong> for exact sequences).</p><p>Before starting, prepare separate TCRV-UPS2 alpha and beta mixes by equimolar pool each of the primers, and dilute as necessary to 10uM (total concentration).</p><p>Make the following master mix (per reaction). Make two separate mixes (one for TCRα and one for TCRβ chain):</p><p>&nbsp;</p><p>Kapa Hifi Hotstart Readymix 2X&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;12.5µL</p><p>TCRV-UPS2 TCRα or TCRβ(10uM)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2.5 µL&nbsp;</p><p>Water&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;6µL</p><p>&nbsp;</p><p>Aliquot 21 uL (per reaction) of mastermix into PCR tubes.</p><p>Add 4uL of TCR enrich product, for a total of 25uL per reaction.</p><p>Perform primer extension using the following condition</p><p>&nbsp;</p><p>1 cycle of</p><p>[95<sup>o</sup>C for 5 minutes]</p><p>1 cycles of</p><p>[55<sup>o</sup>C for 30 seconds]</p><p>1 cycles of</p><p>[72<sup>o</sup>C for 2 minutes]</p><p>Hold at 4<sup>o</sup>C</p><p>After extension, add 25uL of water, to bring each reaction to a total of 50uL</p><p><br></p><p>SPRI clean as before. Elute into 11uL of water.</p><p><br></p><p><strong>III. TCR UPS2 amplification</strong>:</p><p>After primer-extension, we use UPS2-N70x and UPS-mod-N50x primers to amplify the resulting products, and add Illumina flanking sequences for sequencing.</p><p>&nbsp;</p><p>See <strong>Supplementary Table 1</strong> for primer sequences. Thaw and dilute the primers into 10uM aliquots.</p><p>&nbsp;</p><p>To minimized barcode swapping, we will split each sample into 4 PCR reactions, and pool them again after amplification.</p><p>&nbsp;</p><p>Make the following master mix (per reaction. prepare 4 reactions for each sample):</p><p>&nbsp;</p><p>Kapa Hifi Hotstart Readymix 2X&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;12.5µL</p><p>UPS-mod-N50x&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.5µL&nbsp;</p><p>UPS2-N70x&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.5µL</p><p>Water&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;9µL</p><p>&nbsp;</p><p>Add 2.5µL of products from <strong>II.</strong> To each reaction, for a total of 25µL per rection.</p><p>&nbsp;</p><p>Amplify using the following condition:</p><p>&nbsp;</p><p>1 cycle of</p><p>[95<sup>o</sup>C for 2 minutes]</p><p>9-18 cycles (see below) of</p><p>[95<sup>o</sup>C for 30 seconds</p><p>60<sup>o</sup>C for 30 seconds</p><p>72<sup>o</sup>C for 1.5 minute]</p><p>1 cycles of</p><p>[72<sup>o</sup>C for 5 minutes]</p><p>Hold at 4<sup>o</sup>C</p><p>&nbsp;</p><p>The precise cycle would need to be tested and adjusted for each sample. We recommend the guideline outlined in <strong>Supplement Table 3,</strong> based on the concentration estimation of amplified pull-down product from <strong>I.</strong></p><p>&nbsp;</p><p>Take 12.5 uL of each reaction and pool for each sample (for a total of 50uL per sample)</p><p>&nbsp;</p><p>SPRI clean as previously described. The final product is ready for Illumina Sequencers.</p><p>&nbsp;</p><p>We recommend assessing the size distribution (figure2) and concentration of the final product using fragment analyzer and Kapa quant qPCR.</p><p>&nbsp;</p><p>See <strong>Figure 3 </strong>for an example of successfully amplified products.</p><p><br></p><p><strong>IV. Sequencing of final TCR libraries</strong>:</p><p>Here we describe the sequencing condition for sequencing the prepared TCR libraries. For a detailed loading protocol, please consult Illumina.</p><p>&nbsp;</p><p>We require the use of two custom sequencing primers (see <strong>Supplement</strong>). Due to the large size of the final libraries, we aim to load the flow cells to a lower density</p><p>&nbsp;</p><p>We recommend sequencing on the Illumina MiSeq using the 150 cycle kit. For libraries consisting of high number of cells, a NextSeq could also be used.</p><p>&nbsp;</p><p>Sequencing specification:</p><p>Read 1: 150bp</p><p>Index 1: 20bp</p><p>&nbsp;</p><p><strong>MiSeq:</strong></p><p>Sequencing primers were used at a final concentration of 2.5µM. We aimed for 8-12 * 10<sup>6</sup> pass filter reads per lane (cluster density of roughly 450K/mm2). Based on the whole-transcriptome data, we allotted ~6,000 T cells per lane.</p><p><strong>NextSeq:</strong></p><p>Sequencing primers were used at a final concentration of 2.5µM. We aimed for 150-200 * 10<sup>6</sup> pass filter reads per lane (cluster density of roughly 100K/mm2). Based on the whole-transcriptome data, we allotted ~80,000 T cells per lane.</p><p><br></p><p>Final fastq will contain TCR sequences in Read 1, and cell barcode with UMI in Index 1.</p><p><br></p><p><strong>Appendix.</strong> <strong>SPRI protocol and notes </strong></p><p>Throughout this protocol, we require purification of amplified products at various steps. In all cases, we aim to purify TCR products &gt; 1kbp in length. Standard SPRI reagents could be used following manufacturer’s instructions (Ampure XP, etc.). Here we provide a homemade alternative taken from Nadin Rohland (Doi:&nbsp;10.1101/gr.128124.111). Heretofore referred to as Serapure reagent.</p><p>&nbsp;</p><p><strong>To make homemade SPRI reagent</strong>:</p><p>&nbsp;</p><p>1. In a 50 mL conical using sterile stock solutions, prepare TE (10 mM TrisNHCl, 1 mM EDTA</p><p>= 500 µL 1 M Tris pH8 + 100 µL 0.5 M EDTA, fill conical to 50 mL mark with dH20).</p><p>2. Mix SeraNmag SpeedBeads and transfer 1 mL to a 1.5 mL microtube.</p><p>3. Place SpeedBeads on magnet stand until beads are drawn to magnet.</p><p>4. Remove supernatant with P200 or P1000 pipetter.</p><p>5. Add 1 mL TE to beads, remove from magnet, mix, return to magnet.</p><p>6. Remove supernatant with P200 or P1000 pipetter.</p><p>7. Add 1 mL TE to beads, remove from magnet, mix, return to magnet.</p><p>8. Remove supernatant with P200 or P1000 pipetter.</p><p>9. Add 1 mL TE to beads and remove from magnet.&nbsp;Fully resuspend and set microtube in</p><p>rack (i.e. not on magnet stand).</p><p>10. Add 9 g PEGN8000 to a new 50 mL, sterile conical.</p><p>11. Add 10 mL 5 M NaCL (or 2.92 g) to conical.</p><p>12. Add 500 µL 1 M TrisNHCL to conical.</p><p>13. Add 100 µL 0.5 M EDTA to conical.</p><p>14. Fill conical to ~ 49 mL using sterile dH20.&nbsp;You can do this by eye, just go slowly.</p><p>15. Mix conical for about 15 minutes until PEG goes into solution (solution, upon sitting,</p><p>should be clear).</p><p>16. Add 27.5 uL Tween 20 to conical and mix gently.</p><p>17. Mix 1 mL SpeedBead + TE solution and transfer to 50 mL conical.</p><p>18. Fill conical to 50 mL mark with dH20 (if not already there) and gently mix 50 mL conical</p><p>until brown.</p><p>19. Test against AMPure XP using aliquots of ladder (Fermentas GeneRuler).&nbsp;I recommend</p><p>the 50 bp ladder in place of the ultra-low range ladder.</p><p>20. Wrap in tinfoil (or place in dark container) and store at 4°C.</p><p>&nbsp;</p><p><strong>Testing</strong>:</p><p>You should test the Serapure mixture to ensure that it is working as expected.&nbsp;You can do this</p><p>using DNA ladder (Fermentas GeneRuler – NEB ladders may cause problems):</p><p><br></p><p>1. Prep fresh aliquots of 80% EtOH.</p><p>2. Mix 2 µL GeneRuler with 18 µL dH20.</p><p>3. Add 20 µL GeneRuler mixture to a volume of Serapure and/or AMPure (the specific</p><p>volume depends on whether you are trying exclude small fragments or not; see the</p><p>figure on the next page).</p><p>4. Incubate mixture 5 min. at room temperature.</p><p>5. Place on magnet stand.</p><p>6. Remove supernatant.</p><p>7. Add 200 µL 80 % EtOH.</p><p>8. Incubate on stand for 1 min.</p><p>9. Remove supernatant.</p><p>10. Add 200 µL 70% EtOH.</p><p>11. Incubate on stand for 1 min.</p><p>12. Remove supernatant.</p><p>13. Place beads on 37°C heat block for 3-4 min. until dry.</p><p>14. Rehydrate with 20 µL dH20.</p><p>15. Place on magnet stand.</p><p>16. Transfer supernatant to new tube.</p><p>17. Mix supernatant with 1 µL loading dye.</p><p>18. Electrophorese in 1.5 % agarose for 60 min. at 100 V.</p><p><br></p><p>See <strong>Figure 4 </strong>for an example of expected results</p><h2>Troubleshooting</h2><p><strong>Note</strong>: Certain samples often need to be enriched for TCR transcripts multiple times, especially for samples with low gene recovery, or low number of αβ T cells. Part <strong>I</strong>. could be repeated if necessary, resulting in multiple rounds of TCR enrichment. For subsequent rounds of enrichment, we recommend decreasing the cycles of PCR amplification from 25 to 12.</p><p><br></p><p><strong>Note 2: </strong>If amplification was unsuccessful after SPRI purification at Step III, the remaining half of the unpurified amplified product can be amplified further for 2-4 more PCR cycles.</p><h2>Anticipated Results</h2><p>The resulting libraries should be sequenceable on Illumina MiSeq or NextSeq, using the custom primers outline in <strong>Supplementary Table 1</strong>. Please see Tu, A. et al. Nat. Immunol. (2019) for more details.</p><h2>References</h2><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Schrama, D., Ritter, C. &amp; Becker, J. C. T cell receptor repertoire usage in cancer as a surrogate marker for immune responses. <em>Semin. Immunopathol.</em> <strong>39</strong>, 255–268 (2017).</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Lossius, A. <em>et al.</em> High-throughput sequencing of TCR repertoires in multiple sclerosis reveals intrathecal enrichment of EBV-reactive CD8(+) T cells. <em>Eur. J. Immunol.</em> <strong>44</strong>, 1–41 (2014).</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Kirsch, I. R. <em>et al.</em> TCR sequencing facilitates diagnosis and identifies mature T cells as the cell of origin in CTCL. <em>Sci. Transl. Med.</em> <strong>7</strong>, 1–13 (2015).</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Atkinson, M. A. The Pathogenesis and Natural History of Type 1 Diabetes. <em>Cold Spring Harb. Perspect. Med.</em> <strong>2</strong>, a007641–a007641 (2012).</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Barrett, D. M., Grupp, S. A. &amp; June, C. H. Chimeric Antigen Receptor- and TCR-Modified T Cells Enter Main Street and Wall Street. <em>J. Immunol.</em> <strong>195</strong>, 755–61 (2015).</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Robins, H. Immunosequencing: applications of immune repertoire deep sequencing. <em>Curr. Opin. Immunol.</em> <strong>25</strong>, 646–52 (2013).</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Crosby, E. J. <em>et al.</em> Complimentary mechanisms of dual checkpoint blockade expand unique T-cell repertoires and activate adaptive anti-tumor immunity in triple-negative breast tumors. <em>Oncoimmunology</em> <strong>7</strong>, e1421891 (2018).</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Tirosh, I. <em>et al.</em> Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq. doi:10.1126/science.aad0501</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Khodadoust, M. S. <em>et al.</em> Antigen presentation profiling reveals recognition of lymphoma immunoglobulin neoantigens. <em>Nature</em> <strong>543</strong>, 723–727 (2017).</p><p>10.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Gaublomme, J. T. <em>et al.</em> Single-Cell Genomics Unveils Critical Regulators of Th17 Cell Pathogenicity. <em>Cell</em> 1–13 (2015). doi:10.1016/j.cell.2015.11.009</p><p>11.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Avraham, R. <em>et al.</em> Pathogen Cell-to-Cell Variability Drives Heterogeneity in Host Immune Responses. <em>Cell</em> 1–13 (2015). doi:10.1016/j.cell.2015.08.027</p><p>12.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Papalexi, E. &amp; Satija, R. Single-cell RNA sequencing to explore immune cell heterogeneity. <em>Nat. Rev. Immunol.</em> <strong>18</strong>, 35–45 (2017).</p><p>13.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Shalek, A. K. <em>et al.</em> Single-cell transcriptomics reveals bimodality in expression and splicing in immune cells. <em>Nature</em> <strong>498</strong>, 236–240 (2013).</p><p>14.		  Macosko, E. Z. <em>et al.</em> Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets. <em>Cell</em> <strong>161</strong>, 1202–1214 (2015).</p><p>15.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Gierahn, T. M. <em>et al.</em> Seq-Well: portable, low-cost RNA sequencing of single cells at high throughput. <em>Nat. Methods</em> <strong>14</strong>, 395–398 (2017).</p><p>16.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hughes, T. K. <em>et al.</em> Highly Efficient, Massively-Parallel Single-Cell RNA-Seq Reveals Cellular States and Molecular Features of Human Skin Pathology. <em>bioRxiv</em> 689273 (2019). doi:10.1101/689273</p><h2>Acknowledgements</h2><p>We thank P. Blainey and M. Birnbaum for fruitful discussions. This work was supported in part by the Koch Institute Support (core) NIH Grant P30-CA14051 from the National Cancer Institute, as well as the Koch Institute - Dana-Farber/Harvard Cancer Center Bridge Project. This work was also supported by the Food Allergy Science Initiative at the Broad Institute and the NIH (5P01AI039671, 5U19AI089992, U19AI095261). A.K.S. was supported by the Searle Scholars Program, the Beckman Young Investigator Program, the Pew-Stewart Scholars Program for Cancer Research, a Sloan Fellowship in Chemistry, the NIH (1DP2GM119419, 2U19AI089992, 2R01HL095791, 1U54CA217377, 2P01AI039671, 5U24AI118672, 2RM1HG006193, 1R33CA202820, 1R01AI138546, 1R01HL126554, 1R01DA046277, 1U2CCA23319501) the Bill and Melinda Gates Foundation (OPP1139972, OPP1202327, OPP1137006, and OPP1202327), and Agilent Technologies.</p><p><br></p></body></html>