<html><head><title>Direct oligonucleotide sequencing with nanopores</title></head><body><h1>Direct oligonucleotide sequencing with nanopores</h1>Authors: Sachin Chalapati, Conor Crosbie, Dixita Limbachiya, Nimesh Chandra Pinnamaneni<br>Abstract: <p>Third-generation DNA sequencing has enabled users to sequence long, unamplified DNA fragments with minimal sample and library preparation steps. Sequencing single-stranded nucleic acids directly without amplification or by ligating a spacer strand are challenging, as the single-strand species are poor templates to add the sequencing adapters. Sequencing ssDNA or RNA directly gives valuable insights like base-level modifications and degradation levels along with saving valuable time and resources. Biological nanopores used by Oxford Nanopore Technologies process the target strands at a single-strand level, although the typical samples sequenced are double-stranded or converted into double-strand. We have identified that the MinION platform from Oxford Nanopore can perform sequencing of short, single-strand oligonucleotides directly without amplification or second-strand synthesis by performing an annealing step before library preparation. Short 5’ phosphorylated oligos when annealed to an adapter sequence can be directly sequenced in the 5' to 3' direction via nanopores, the adapters were designed to bind to the 5’ end of the oligos and leave a 3’ adenosine overhang after binding to their target. The 3’ adenosine overhang of the adapter and the terminal phosphate makes the 5’ end of the oligo to be analogous to an end-prepared dsDNA, rendering it compatible with ligation-based library preparation for sequencing. An oligo-pool containing 42,000 orthogonal sequences of 120 bp length were sequenced using the method and 37,265 of the total sequences were recovered with high accuracy.&nbsp;</p><p><br></p><p>While analyzing the raw data, we had interesting observations. In our raw data, we have identified that empty signals can be wrongly identified as a valid read by the MinION platform and sometimes multiple signals containing several strands can be fused into a single read by the platforms segmentation faults.&nbsp;</p><p><br></p><p>We believe that this method could enable novel applications of nanopore sequencing in DNA data-storage systems where short oligonucleotides function as the primary information carriers.</p><br>Keywords: ssDNA, RNA, oligonucleotide, oligo, single-stranded DNA, phosphorylation, T4 PNK, ligation, AMX, T4 DNA ligase, sequencing, ONT, MinION, Guppy, BLASTN, nanopore, helicase, dsDNA, ssDNA, basecalling, squiggle<br><h2>Introduction</h2><p>DNA sequencing has become a staple tool in biology and is getting affordable and more accessible to small labs and individual researchers in the past decade. Oxford Nanopore Technologies, with its biological nanopore-based sequencing technology, has opened the market wide open by releasing a $1000 sequencing platform – The MinION [1]. The MinION platform has the capability to sequence both amplified and non-amplified double-stranded DNA (dsDNA) [2] and direct RNA in the 3’ to 5’ direction using Poly-A tail capture [3]. Direct sequencing of short, single-stranded oligonucleotides has been regarded as a challenge due to nanopore chemistry, pore design and basecalling [4], however, attempts have been made to overcome these challenges by performing circularization [5]. Direct sequencing short, single-strand nucleic acid species without a polymerase or ligation step has not been evident in previous research. In this article we propose a method to perform direct sequencing of short single strand oligonucleotides that can be leveraged by different applications such as DNA based data storage systems and direct RNA sequencing.&nbsp;</p><p><br></p><p>In DNA based information systems, the oligonucleotides serve the purpose of the information carriers and to rapidly sequence the oligos to extract the encoded data [6]. Several encoding and compressions techniques are widely used during the design of the oligonucleotides for DNA data storage purposes to increase the data capacity and to deal with amplification and sequencing issues. The possibility to sequence oligonucleotides directly without performing a PCR step or performing a ligation step enables users to design the oligos to have only one priming region on the 5' end and free up the reverse priming region. This increases the encoding space available to the users and opens the possibilities for new encoding schema and DNA data storage architectures. In this work, we propose a method which is incredibly fast when compared to PCR-based sequencing strategies [7] with hands-on time as low as 5 minutes and sequencing time of just 20 minutes for pre-phosphorylated oligos like INS3 and EINS3 shown in Figure 1. We have identified that the Oxford Nanopores MinION platform is capable of sequencing single-strand templates directly without the need for a complementary strand or to have a spacer strand to increase the strand length.</p><p><br></p><p>In this work, we solve this challenge of direct oligonucleotide sequencing by performing a simple annealing step before the library preparation step. The setup starts with a phosphorylation step using T4 Polynucleotide Kinase (PNK) that adds a 5’ phosphate to the target oligos. The phosphorylated oligos are annealed to an adapter sequence that binds to their 5’ end. The adapter sequences are designed to have a melting temperature of ~65°C to the 5’ end of the target oligos; and when annealed to their targets, the adapter strands have an adenosine overhang at their 3’ end as detailed in Figure 2. In the annealed state with the adapter sequence, the 5’ end of the oligos are analogous to an end-prepared dsDNA and are compatible with the AMX sequencing adapters form the ligation sequencing kit (LSK-109) offered by the Oxford Nanopore Technologies [8]. The sequences used to implement the method and related protocols are discussed in the next section.</p><p><br></p><h2>Reagents</h2><p><strong><u>Oligonucleotides</u></strong></p><p><br></p><p>INS3</p><p>/5Phos/GCTGGGCGGGGGCCCTGGTGCAGGCAGCCTGCAGCCCTTGGCCCTGGAGGGGTCCCTGCAGAAGCGTGGCATTGTGGAACAATGCTGTACCAGCATCTGCTCCCTCTACCAGCTGGAGAACTACTGCAACTAGACGCAGCCCGCAGGCAGCCCCACACCCGCCGCCTCCTGCACC/3SpC3/</p><p><br></p><p>INS3 is procured from IDT (Ultramer) – Normalized to 100 uM concentration.</p><p><br></p><p><br></p><p>INS3 RC</p><p>CCCGCCCAGCA</p><p><br></p><p>INS3 RC is procured from IDT (Standard desalting) – Normalized to 100 uM concentration.</p><p><br></p><p>EINS3</p><p>/5Phos/AGGCTTCTTCTACACACCCAAGACCCGCCGGGAGGCAGAGGACCTGCAGGTGGGGCAGGTGGAGCTGGGCGGGGGCCCTGGTGCAGGCAGCCTGCAGCCCTTGGCCCTGGAGGGGTCCCTGCAGAAGCGTGGCATTGTGGAACAATGCTGTACCAGCATCTGCTCCCTCTACCAGCTGGAGAACTACTGCAACTAGTGAA/3Phos/</p><p><br></p><p>EINS3 is procured from IDT (Ultramer) – Normalized to 100 uM concentration</p><p><br></p><p>EINS3 RC</p><p>CTTGGGTGTGTAGAAGAAGCCTA</p><p><br></p><p>EINS3 RC is procured from IDT (Standard desalting) – Normalized to 100 uM concentration.</p><p><br></p><p>3xr6 - 42,000 oligos - 120 bp length</p><p>Supplementary table 1.</p><p><br></p><p>3xr6 oligo-pool is procured from Twist Biosciences – Normalized to 10 ng/ul concentration.</p><p><br></p><p>ArcFP_MoSS5</p><p>ACTGAGGTTGTAATCTGCGTTGTAGA</p><p><br></p><p>ArcFP_MoSS5 is procured from IDT (Standard desalting) – Normalized to 100 uM concentration.</p><p><br></p><p><strong><u>Enzymes and buffers</u></strong></p><p><br></p><p>T4 PNK - NEB - M0201S</p><p>Blunt/TA Ligase Master Mix - NEB - M0367S</p><p>IDTE - IDT - 11-01-02-02</p><p>Qubit ssDNA Assay Kit - ThermoFisher - Q10212</p><p>Nuclease Free Water (NFW) – IDT - 11-04-02-01</p><p><br></p><p><strong><u>Wash-kit</u></strong></p><p><br></p><p>Monarch PDR &amp; DNA Cleanup Kit - NEB - T1030S</p><p><br></p><p><strong><u>Sequencing kit</u></strong></p><p><br></p><p>Ligation Sequencing Kit - Nanopore - SQK-LSK109</p><h2>Equipment</h2><p><strong><u>Instruments</u></strong></p><p><br></p><p>Open qPCR (Single Channel) - ChaiBio - E013101</p><p><br></p><p>Microcentrifuge - Dlab - D3024</p><p><br></p><p>Qubit 4 - Thermofisher - Q33226</p><p><br></p><p><strong><u>Sequencer</u></strong></p><p><br></p><p>MinIon - Oxford Nanopore Technologies (ONT)&nbsp;- MIN-101B</p><h2>Procedure</h2><p><strong>Normalisation and Qubit analysis</strong></p><p><br></p><p>1. 3xr6 oligo-pool is normalized to 0.25 uM and verified with Qubit 4 using Qubit ssDNA Assay Kit</p><p>2. INS3 and EINS3 are diluted from their stock concentrations to 0.5 uM and verified with Qubit 4 using Qubit ssDNA Assay Kit</p><p>3. INS3 RC, EINS3 RC and ArcFP oligos are normalized by the supplier at 100 uM concentration and are diluted to 1 uM concentration.</p><p><br></p><p><strong>Phosphorylation of 3xr6</strong></p><p><br></p><p>1. 3xr6 oligo-pool is normalized to 0.25 uM overall concentration</p><p>2. 8 ul of 0.25 uM 3xr6 oligo-pool is phosphorylated in combination with 2.5 ul T4 PNK reaction buffer, 2.5 ul of 10 mM ATP, 0.5 ul (5 units) of T4 PNK and 11.5 ul NFW</p><p>3. The phosphorylation reaction is carried out at 37° for 30 minutes</p><p>4. Heat inactivation at 65°C for 20 minutes</p><p>5. The mixture is washed using Monarch spin columns with the standard oligonucleotide cleanup protocol and the purified DNA is eluted into 9 ul of IDTE</p><p><br></p><p><strong>Library preparation of INS3</strong></p><p><br></p><p>1. A triplicate of the following reaction is performed</p><p>2. 0.5 ul of 0.5 uM INS3 is added to 0.5 ul of 1 uM INS3 RC along with 2 ul of NFW</p><p>3. The reaction mixtures are heated to 94°C for 2 minutes and gradually cooled to room temperature for annealing</p><p>4. 5 ul of AMX from the ligation sequencing kit is added to each of the tubes</p><p>5. 5 ul of Blunt/TA mastermix is added to each of the tubes</p><p>6. Tubes are incubated at room temperature for 10 minutes</p><p>7. Three new MinION flow cells are used to sequence each reaction mix of the triplicate and the sequencing is performed with the standard parameters for 20 minutes</p><p><br></p><p><strong>Library preparation of EINS3</strong></p><p><br></p><p>1. A triplicate of the following reaction setup is performed</p><p>2. 0.5 ul of 0.5 uM EINS3 is added to 0.5 ul of 1 uM EINS3 RC along with 2 ul of NFW</p><p>3. The reaction mixtures are heated to 94°C for 2 minutes and gradually cooled to room temperature for annealing</p><p>4. 5 ul of AMX from the ligation sequencing kit is added to each of the tubes</p><p>5. 5 ul of Blunt/TA mastermix is added to each of the tubes</p><p>6. Tubes are incubated at room temperature for 10 minutes</p><p>7. Three MinION flow cells from the INS3 run are used to sequence each reaction mix of the triplicate and the sequencing is performed with the standard parameters for 20 minutes</p><p><br></p><p><strong>Library preparation of 3xr6</strong></p><p><br></p><p>1. 9 ul elute from the phosphorylation step is 3-way split for triplicate sequencing runs. A triplicate of the following reactions are performed</p><p>2. 3 ul of washed, phosphorylated 3xr6 oligo pool is added to 1 ul of 1 uM ArcFP</p><p>3. The reaction mixtures are heated to 94°C for 2 minutes and gradually cooled to room temperature for annealing</p><p>4. 5 ul of AMX from the ligation sequencing kit is added to each of the tubes</p><p>5. 5 ul of Blunt/TA mastermix is added to each of the tubes</p><p>6. Tubes are incubated at room temperature for 10 minutes</p><p>7. Three new MinIon flow cells are used to sequence each reaction mix of the triplicate and the sequencing is performed with the standard parameters for 4 hours</p><h2>Troubleshooting</h2><p>A triplicate of Qubit readings is recommended for reliability. The reaction mixtures are recommended to be washed with magnetic beads after the Blunt/TA ligation step by ONT, we avoided this step to improve yields as the strands are short in length.</p><h2>Time Taken</h2><p><strong>Phosphorylation of 3xr6</strong></p><p>Hands-on-time - 5 minutes</p><p>Total reaction time - 60 minutes</p><p><br></p><p><strong>Library preparation of INS3</strong></p><p>Hands-on-time - 5 minutes</p><p>Ligation - 10 minutes</p><p>Sequencing - 20 minutes</p><p><br></p><p><strong>Library preparation of EINS3</strong></p><p>Hands-on-time - 5 minutes</p><p>Ligation - 10 minutes</p><p>Sequencing - 20 minutes</p><p><br></p><p><strong>Library preparation of 3xr6</strong></p><p>Hands-on-time - 5 minutes</p><p>Ligation - 10 minutes</p><p>Sequencing - 4 hours</p><h2>Anticipated Results</h2><p><strong>Sequencing of INS3</strong></p><p><br></p><p>The flowcells 1, 2 and 3 yielded 23248, 15917 and 21052 reads respectively after 20 minutes of sequencing. Basecalling was performed using guppy_hac (ver. 3.5.2) [9] model with a CUDA compatible GPU. A total of 289, 483 and 255 reads from flowcells 1 to 3 have passed the default q-score filter. The sequencing and BLASTN results with e-value are shown in Table 1.</p><p><br></p><p><strong>Sequencing of EINS3</strong></p><p><br></p><p>The flowcells 1, 2 and 3 yielded 23174, 30238, and 30051 reads respectively after 20 minutes of sequencing. Basecalling was performed using guppy_hac (ver. 3.5.2) model with a CUDA compatible GPU. A total of 6489, 5041 and 3826 reads from flowcells 1 to 3 have passed the default q-score filter. The sequencing and BLASTN results with e-value filter are shown in Table 2.</p><p><br></p><p><strong>Sequencing of 3xr6</strong></p><p><br></p><p>The flowcells 1, 2 and 3 yielded 60299, 48432 and 42434 reads respectively after 4 hours of sequencing. Basecalling was performed using guppy_hac (ver. 3.5.2) model with a CUDA compatible GPU. A total of 28869, 13299 and 10268 reads from flowcells 1 to 3 have passed the default q-score filter. The sequencing and BLASTN results with e-value filter are shown in Table 3.</p><h2>References</h2><p>[1] 	Y. . Wang, Q. . Yang and Z. . Wang, "The evolution of nanopore sequencing.," Frontiers in Genetics, vol. 5, no. , pp. 449-449, 2015.&nbsp;</p><p>[2] 	M. . Jain, H. E. Olsen, B. . Paten and M. . Akeson, "The Oxford Nanopore MinION: delivery of nanopore sequencing to the genomics community," Genome Biology, vol. 17, no. 1, p. 239, 2016.&nbsp;</p><p>[3] 	N. Harel, M. Meir, U. Gophna and A. Stern, "Direct sequencing of RNA with MinION Nanopore: detecting mutations based on associations.," Nucleic Acids Research, vol. 47, no. 22, p. 148, 2019.&nbsp;</p><p>[4] 	D. . Branton, D. W. Deamer, A. . Marziali, H. . Bayley, S. A. Benner, T. . Butler, M. . Di Ventra, S. . Garaj, A. . Hibbs, X. . Huang, S. B. Jovanovich, P. S. Krstic, S. . Lindsay, X. S. Ling, C. H. Mastrangelo, A. . Meller, J. S. Oliver, Y. V. Pershin, J. M. Ramsey, R. . Riehn, G. V. Soni, V. . Tabard-Cossa, M. . Wanunu, M. . Wiggin and J. A. Schloss, "The potential and challenges of nanopore sequencing," Nature Biotechnology, vol. 26, no. 10, p. 1146–1153, 2008.&nbsp;</p><p>[5] 	B. D. Wilson, M. Eisenstein and H. T. Soh, "High-Fidelity Nanopore Sequencing of Ultra-Short DNA Targets.," Analytical Chemistry, vol. 91, no. 10, pp. 6783-6789, 2019.&nbsp;</p><p>[6] 	S. M. H. T. Yazdi, R. . Gabrys and O. . Milenkovic, "Portable and Error-Free DNA-Based Data Storage," Scientific Reports, vol. 7, no. 1, pp. 5011-5011, 2017.&nbsp;</p><p>[7] 	M. . Blawat, K. . Gaedke, I. . Hütter, X. . Chen, B. M. Turczyk, S. A. Inverso, B. W. Pruitt and G. M. Church, "Forward Error Correction for DNA Data Storage," Procedia Computer Science, vol. 80, no. 80, pp. 1011-1022, 2016.&nbsp;</p><p>[8] 	T. . Karamitros, T. . Karamitros and G. . Magiorkinis, "Multiplexed Targeted Sequencing for Oxford Nanopore MinION: A Detailed Library Preparation Procedure.," Methods of Molecular Biology, vol. 1712, no. , pp. 43-51, 2018.&nbsp;</p><p>[9] 	R. R. Wick, L. M. Judd, K. E. Holt and K. E. Holt, "Performance of neural network basecalling tools for Oxford Nanopore sequencing," Genome Biology, vol. 20, no. 1, p. 129, 2019.&nbsp;</p><p><br></p></body></html>