<html><head><title>Quantification of SARS-CoV-2 neutralizing antibody by a pseudotyped virus based assay</title></head><body><h1>Quantification of SARS-CoV-2 neutralizing antibody by a pseudotyped virus based assay</h1>Authors: Jianhui Nie, Qianqian Li, Jiajing Wu, Chenyan Zhao, Huan Hao, Huan Liu, Li Zhang, Lingling Nie, Haiyang Qin, Meng Wang, Qiong Lu, Xiaoyu Li, Qiyu Sun, Junkai Liu, Changfa Fan, Weijin Huang, Miao Xu, Youchun Wang<br>Abstract: <p><em>Jianhui Nie, Qianqian Li, and Jiajing Wu contributed equally to this work.</em></p><p><br></p><p><u>Pseudotyped</u> viruses are useful virological tools due to their safety and versatility. Based on a VSV pseudotyped<u> virus</u> production system, we developed a pseudotyped virus-based neutralization assay against SARS-CoV-2 in biosafety level 2 facilities. This protocol includes production, titration of the SARS-CoV-2 S pseudotyped virus and neutralization assay based on it. Various types of samples targeting virus attachment and entry could be evaluated for their potency, including serum samples derived from animals and humans, monoclonal antibodies, fusion inhibitors (peptides or small molecules). If the pseudotyped virus stock has been prepared in advance, it will take 2 days to get the potency data for the candidate samples. Experience of handling cells is needed before implementing this protocol.</p><br>Keywords: pseudotyped virus, virus-based neutralization assay, virus production, SARS-CoV-2, neutralizing antibody<br><h2>Introduction</h2><p>Since the coronavirus disease 2019 (COVID–19), which is caused by SARS-CoV–2, occurred in December 2019, more than 2,000,000 cases were diagnosed globally until April 17, 2020. Therapeutics and vaccines for COVID–19 are urgently needed. Due to its high pathogenicity and infectivity, all test procedures employing live viruses to evaluate the products’ potency had to be handled under biosafety level 3 (BSL–3) facilities, which have hindered the development of products. Standardized in vitro potency methods were urgently needed to evaluate the antiviral products in pre-clinical and clinical phases. In addition, detection of neutralizing antibodies against SARS-CoV–2 would be helpful to understand the status of the protective immune response among the COVID–19 patients and asymptomatic cases.</p>
<p>SARS-CoV–2 is a single-stranded positive-sense RNA virus, belonging to the β-coronavirus of the coronaviridae family<sup>1</sup>. It is one of the largest known RNA viruses with a genome length of about 29.8 kb nucleotides<sup>2–4</sup>. The first 2/3 of the genome which is non-structural genes, mainly encodes enzymes related to virus replication, and the latter 1/3 sequentially encodes four structural proteins: spike protein (S), small envelope protein (E), matrix protein (M) and nucleocapsid protein (N)<sup>5</sup>. Among them, S protein contains the binding region of virus receptor, which mediates the adsorption and entry of virus into cells. S protein of the coronaviruses is the main protein that stimulates the immune system to produce neutralizing antibodies after the virus infects the body<sup>6–9</sup>. A variety of monoclonal neutralizing antibodies against S protein of coronaviruses have been isolated and proven to be effective in animal models<sup>10–13</sup>. In addition, fusion inhibitory peptides or small molecules for coronavirus fusion targeting the S protein have also been found to be effective in other coronavirus studies<sup>14</sup>. Thus, S protein is the main target for the design of SARS-CoV–2 vaccines and some therapeutic products.</p>
<p>Since the S protein of SARS-CoV–2 is considered as main antigen to induce the neutralizing antibody, several immune assays based on full-length S protein or partial regions of S protein such as RBD region have been developed to evaluate the protective immune response for vaccines or in patients. So far, the neutralizing epitopes on S protein are not clearly defined. Whether partial regions of S protein such as RBD region can cover all the neutralizing epitope is not clear. Meanwhile, the difference of post-translational modification for recombinant S proteins expressed in different expression systems may affect the immune reactivity. Furthermore, all immune assays above detect the binding antibodies against specific antigens, which are difficult to predict the protective situations and not comparable. The most direct method to evaluate the neutralizing antibodies induced by SARS-CoV–2 vaccine is to use the infection inhibition test of live virus<sup>15</sup>, but the operation of live virus must be carried out in the BSL–3 laboratories, limited by experimental conditions and virus accessibility, so it is difficult to standardize it among laboratories. Due to the differences in virus strains, culture conditions and result interpretations, great differences are usually observed among live virus laboratories.</p>
<p>Our laboratory established a SARS-CoV–2 pseudotyped virus neutralization antibody detection method based on VSV system<sup>16</sup>. The full-length S protein is inserted into the pseudotyped virus particles, and the pseudotyped virus enters the cell in the same way as the live virus. It can detect all types of neutralizing antibodies and small molecules tagarting S protein that block the virus from entering the cell. Pseudotyped viruses are prepared by transfection of S protein expression plasmid together with infection of G* Δ GVSV. Thus, the pseudotyped viruses no longer replicate after infecting cells, avoiding the influence of mutation on the detection method in the process of live virus passage, and the pseudotyped virus assay can be carried out in the BSL–2 conditions with easy operations. Pseudotyped viruses carry reporter genes, which can express after infection. By detecting the expression of reporter genes, the quantitative detection of infected virus is realized. The data can be read objectively with a luminescence meter, which makes the assay amendable to achieve high-throughput detection and easy to be standardized. By replacing the S protein expression plasmid, the cross neutralization of antibodies to different mutant strains can be studied, and even the cross neutralization between different coronaviruses can be investigated. In addition to being used to evaluate products such as neutralizing antibodies and entry inhibitors in the process of virus infection, it can also be used to study virus cell tropism and receptor recognition patterns.</p>
<h2>Limitations:</h2>
<p>Two main differences exist between pseudotyped viruses and live viruses. One is that pesudotyped viruses replicate in single cycle after entering cells, which is safer than live virus. The second is that pseudotyped viruses only contain S protein of SARS-CoV–2. Thus, the pseudotyped virus assay can only evaluate neutralizing antibodies for S protein, not for other structural or non-structural proteins. Whether those proteins can induce neutralizing antibody is controversial. After the pseudotyped virus infects the cells, it cannot complete the same life cycle as the live virus, and it is impossible to evaluate the inhibitory effect of the product on viral replication. The number of S protein on the surface of the pseudotyped virus may affect the sensitivity for neutralizing assay. Whether number of S protein on the surface of the pseudotyped virus is similar to the live virus also needs to be further studied. Therefore, in the process of developing the pseudotyped virus method, it is necessary to carry out the comparison and verification of the live virus to ensure its representativeness to the live virus detection method.</p>
<h2>Experimental design</h2>
<p>The schematic procedure for preparation of SARS-CoV–2 pseudotyped virus and neutralization detection is shown in Figure 1.</p>
<p>The pseudotyped virus used in this method was prepared by VSV pseudotyped virus packaging system. The backbone of the pseudotyped virus except S gene comes from the VSV virus. The genome wrapped in the pseudotyped virus is the VSV genome after replacing the VSV G with the firefly luciferase reporter gene. The membrane protein on the surface of the VSV pseudotyped virus is the S protein from SARS-CoV–2. In order to improve the expression efficiency of S protein in mammalian cells, the S gene (GenBank: MN908947) was codon-optimized to adopt expression in mammal cells and inserted into the pcDNA3.1 plasmid using BamH I and Xba I restriction sites. During the preparation of pseudotyped virus, 293T cells were transfected with SARS-CoV–2 S protein expression plasmid (pcDNA3.1.S2) (Figure 1). After transfection, the plasmid transcribed and expressed full-length S protein that were transferred and anchored in the cell membrane. At the same time, 293T cells were infected with VSV G pseudotyped virus G*ΔG-VSV, which entered the cells using the VSV G protein in the surface of the particle and provided a defective VSV genome. The VSV G gene in the ΔG-VSV genome was replaced by firefly luciferase (Fluc) reporter genes. After infection, the uncoating ΔG-VSV genome expressed structural proteins except G protein and corresponding enzymes of VSV in its genome, and completed partial genome replication. The expressed structural proteins and partial replicated genomes assembled into virus particles without envelope, and the virus was released from the cells by budding (Figure 1). Meanwhile, the SARS-CoV–2 S protein expressed on the cell surface bond the VSV particles and became the envelope protein of the pseudovirion, thus the pseudotyped virus incorporated with SARS-CoV–2 S protein on the envelope packaged the defective VSV genome containing the reporter gene in its capsid. The pseudovirion had the characteristics of SARS-CoV–2 infection.</p>
<p>Pseudotyped viruses prepared for neutralization test can infect cells containing SARS-CoV–2 receptors. Both Huh–7 and Vero cells, which was used to isolate live virus<sup>15</sup>, can be infected efficiently by SARS-CoV–2 pseudotyped virus<sup>16</sup>. When the pseudotyped viruses are incubated with cells containing SARS-CoV–2 receptors, the pseudotyped viruses attach and enter the cells through the receptors on the cell surface. The pseudotyped viruses entering the cells will express proteins other than G in VSV, and these structural proteins will also be assembled into capsid protein particles with the replicated VSV defective genome. Because there is no expression of surface proteins in this process, virus particles containing surface proteins cannot be formed. That is, the pseudotyped virus cannot replicate sequentially and only has the ability of single-round infection. VSV genome not only expresses its own protein, but also expresses reporter gene Fluc, which could be detected after the addition of the substrate. After a short reaction, the amount of pseudotyped virus in infected cells is determined by detecting the luminescence signal on a chemiluminescence reader. There was a linear correlation between the value of chemiluminescence signal and the amount of infected pseudotyped virus. If the pseudotyped virus is co-incubated with the sample containing neutralizing antibody or moleucluar inhibitors before cell infection, they will block the cell entry process. The number of blocked viruses can be determined by the results of chemiluminescence reduction, which reflects the level of neutralizing antibody or molecular inhibitors in the sample <h2>Reagents</h2><h2>Cells, plasmids, pseudotyped virus</h2><ul><li>Cell line used for pseudovirus generation: HEK293T cells (ATCC, cat. no. CRL–3216)</li><li>Cell line used for pseudovirus infection: Huh–7 cells (Japanese Collection of Research Bioresources [JCRB], cat. no. 0403)</li><li>pcDNA3.1.2S recombinant plasmid, which was constructed by inserted the codon-optimized S gene of SARS-CoV–2 (GenBank: MN908947) into pcDNA3.1 plasmid using BamH I and Xba I restriction sites16. It can be shared upon request.</li><li>pcDNA3.1.VSVG recombinant plasmid, which was constructed by inserting codon-optimized VSV G gene (Genbank: M27165) into pcDNA3.1. It can be shared upon request.</li><li>G*ΔG-VSV is a VSV G pseudotyped virus, in which G gene in the VSV genome packaged in its capsid was replaced by firefly luciferase (Fluc) reporter genes. It is kindly provided by Dr. Whitt17.</li></ul>
<h2>Reagents</h2><ul><li>QIAGEN Plasmid Plus Maxi Kit (Qiagen, cat. no. 12963)</li><li>DMEM (high glucose; Hyclone, cat. no. SH30243.01)</li><li>Opti-MEM I reduced serum medium (Gibco, cat. no. 31985088)</li><li>Fetal bovine serum (FBS; Pansera ES; PAN-Biotech GmbH, cat. no. ST30–2602)</li><li>Phosphate buffered saline (PBS; Hyclone, cat. no. SH30256.01)</li><li>Lipofectamine 3000 transfection reagent (Thermo Fisher Scientific, cat. no. L3000015)</li><li>0.25%Trypsin-EDTA (Gibco, cat. no. 25200056)</li><li>N–2-hydroxyethylpiperazine-N–2-ethane sulfonic acid (HEPES; Genom, cat. no.GNM–11344)</li><li>Penicillin-Streptomycin solution (100×; Genom, cat. no.GNM15140)</li><li>Britelite plus reporter gene assay system (PerkinElmer, cat. no.6066769)</li></ul>
<h2>Materials</h2><ul><li>DH5α Chemically Competent Cell (ZOMANBIO, cat. no. ZC101)</li><li>Ampicillin sodium salt (VWR, cat. no. 0339–100G)</li><li>Tryptone (OXOID, cat. no. LP0042)</li><li>Yeast extract (OXOID, cat. no. LP0021)</li><li>Agar powder (Meitan, cat. no. 11Y03)</li><li>Cell culture tube (BD Falcon, cat. no. 352006)</li><li>Triangle bottle (Nalgene, cat. no. 4112–0250)</li><li>Acrodisc syringe filters with supor membrane (Sterile - 0.45 µm, 25 mm; PALL, cat. no. 4614)</li><li>Serological pipettes: 5 ml, 10 ml and 25 ml (Costar, cat nos. 4487, 4488, 4489)</li><li>Cell culture plate (96-well Clear Polystyrene Microplates; Corning, cat. no. 3599)</li><li>Cell culture flask (75cm²; Corning, cat. no.430641)</li><li>Centrifuge tubes: 15ml and 50ml (Corning, cat. no. 430790, 430828)</li><li>Microtubes: 1.5ml and 2ml (Axygen, cat. no. MCT–150-C, MCT–200-C)</li><li>Chemiluminescence detection plates (96-well; Xiamen Labware, cat. no. 3599)</li></ul><h2>Equipment</h2><h2>Equipment</h2><ul><li>Cell culture incubator (5% CO2, humidified, 37 °C; NuAire, model no. NU–5810E)</li><li>EnSight multimode plate reader (PerkinElmer, model no. HH34000000)</li><li>Inverted microscope (ZEISS, model no. Primovert)</li><li>Micro spectrophotometer (KAIAO, model no. K5500)</li><li>Cell counter (Nexcelom, model no. Cellometer Auto T4)</li></ul>
<h2>Reagent setup</h2><ul><li>LB medium is made by adding 10 g tryptone, 5 g yeast extract, 10 g NaCl, and deionized water to 1 L.</li><li>Complete DMEM is made by adding 10% FBS, 2% HEPES and 1% penicillin–streptomycin to DMEM.</li></ul><h2>Procedure</h2><p><strong>Plasmid stock preparation</strong><strong> for pcDNA3.1.VSVG and pcDNA3.1.2S</strong> <strong>●</strong><strong>TIMING 3 d</strong></p>
<ol>
<li>Transfer DH5&alpha; competent bacteria into an ice bath box. When the competent bacteria begin to melt, add 1&mu;l plasmid into it and place back in ice for 30 min.</li>
<li>Thirty minute later, put the competent bacteria into a 42&deg;C water bath for 45-60 s. Then, place it back into the ice box for 2 min. Next, add 500 &mu; l LB medium into the transformed competent bacteria and place it in 37&deg;C shaker at a speed of 120 rpm for 1 hour.</li>
<li>After incubation in the shaker, draw 50 &mu;l of the bacterial into a bacterial culture plate (solid LB medium with 100 &mu;g/ml ampicillin) and smear it evenly. Next, place the plate into a 37&deg;C incubator overnight (14-16 h).</li>
<li>Next day, pick a mono-colony of the bacterial using a sterile 10-&mu;l pipette tip and put it into a cell culture tube with 5 ml LB medium (100 &mu;g/ml ampicillin). Then put the tube back into a 37&deg;C shaker at a speed of 220 rpm overnight (14-16 h).</li>
<li>Transfer 100 &mu;l the cultured bacterial liquid into a 250-ml Triangle bottle with 100 ml LB medium (100 &mu;g/ml Amp). Then, put it into a 37&deg;C shaker at a speed of 220 rpm overnight (14-16 h).</li>
<li>Collect the bacterial by centrifuging at a speed of 6,000 g for 15min at 4&deg;C.</li>
<li>Re-suspend the pelleted bacterial using 8ml Buffer P1. Then, add 8 ml Buffer P2 and mix gently for 4-6 times, and incubate at room temperature for 3 min. Next, add 8 ml Buffer S3 and mix immediately by inverting 4-6 times. Transfer the solution into QlAfilter Cartridge and incubate at room temperature for 10 min.</li>
<li>During incubation, place the spin columns into the QIAvac 24 plus. Insert Tube extenders into each column. Then, push the plunger into the cartridge and filter the cell lysate into 50 ml centrifuge tubes. Next, add 5ml Buffer BB and inverse it 4-6 times.</li>
<li>Transfer the lysate solution to QIAGEN Plasmid plus spin columns placed on the QIAvac 24 Plus. Apply vacuum of about 300 mbar until drying the filter</li>
<li>Add 700 &mu;l Buffer ETR into the spin column and apply vacuum until drying.</li>
<li>Place the spin columns into 2 ml tubes and centrifuge it at 10,000 g for 1min to remove the residual buffer.</li>
<li>Place the spin columns into clean 2 ml tubes and added 400 &mu;l Buffer EB to the spin columns and let it stand for more than 1 min. Then, centrifuge for 1 min to collect the purified plasmids.</li>
<li>Determine the concentration of plasmid using spectrophotometer. If OD260/OD280=1.8-2.0, A260 &gt; 0.1 and A320 &le;01, the plasmid meets the quality for transfection.</li>
<li>Aliquots about 50 &mu;g of the plasmid into each sterile 2-ml tubes and stored at -80&deg;C refrigerator until use.</li>
</ol>
<p><strong>Pseudotyped virus stock preparation for </strong>G*<strong>&Delta;G-VSV and SARS-CoV-2 </strong><strong>●</strong><strong>TIMING 3 d</strong></p>
<p>G*&Delta;G-VSV pseudotyped virus is needed for producing SARS-CoV-2 pseudotyped virus. Thus, its stock must be generated before production of SARS-CoV-2 pseudotyped virus. The following procedures for production and titration of SARS-CoV-2 pseudotyped virus can be completely applied to the amplification and titration of G*&Delta;G-VSV by replacing plasmid pcDNA3.1.S2 with pcDNA3.1.VSVG.</p>
<p><strong>Cells preparation for pseudotyped virus production</strong></p>
<ol start="15">
<li>We maintain the 293T cells in complete DMEM medium and prepare cells for pseudotyped virus production as described below. First, when the 293T cells grow to a 70%-90% confluence in T75 flask, carefully discard the supernatant culture medium of the flask. Then, add 10 ml 37&deg;C pre-warmed PBS to the flask to clean the cell surface .Next, discard the cleaning solution and add 3 ml of 0.25% trypsin-EDTA to the flask to cover the cells. Rest at room temperature for 1 minute, and discard trypsin by aspiring. Transfer the cells to 37&deg;C 5% CO<sub>2</sub> incubator and incubate for 5 minutes. After that, add 10ml of DMEM complete medium preheated at 37&deg;C to the cell culture flask, and blow the cells to disperse them into single ones.</li>
</ol>
<p>? TROUBLESHOOTING</p>
<ol start="16">
<li>Count the cells using a cell counter and adjust the cell concentration to 5 &times; 10<sup>5</sup>-7 &times; 10<sup>5</sup> cell/ml. Then, transfer 15ml into a T75 cell flask and place it in 5% CO<sub>2</sub> cell incubator at 37&deg;C overnight.</li>
</ol>
<p><strong>Transfection and infection</strong></p>
<ol start="17">
<li>The cell density should reach 80%-90% confluence after overnight incubation. ▲CRITICAL STEP Make sure to check the cell density before cell transfection. If the cell density has not reached the expected confluence, replace the culture medium and lengthen the incubation time for another 4-6 h.</li>
<li>Add 0.75ml opti-MEM into the 2-ml EP tube (labeled tube A). Then, add 30 &mu; g plasmid pcDNA3.1.S2 into tube A. Next, add 30ul P3000 into tube A and mix it using a pipette. Rest it at room temperature for 5min.</li>
<li>Transfer 0.75ml opti-MEM into another 2-ml EP tube (marked as tube B), and add 30 &mu;l Lipofectamine 3000 into it. Mix the solution gently and place it at room temperature for 5min (steps 4 and 5 can be carried out simultaneously).</li>
<li>After 5-min incubation, add the solution of tube B to tube A drop-by-drop. Reverse the tube A gently to mix the mixture completely and rest it at room temperature for 15-20min.</li>
</ol>
<p>▲CRITICAL STEP Make sure the mixing is gentle and complete, which will influence the efficiency of transfection.</p>
<ol start="21">
<li>During incubation time (step 20), dilute G*&Delta;G-VSV (VSV G pseudotyped virus) to 7.0 &times; 10<sup>4 </sup>TCID50/ml. Discard the culture medium of 293T cell, and then add 15 ml of the diluted G*&Delta;G-VSV solution into the T75 flask.</li>
<li>When the incubation time for the transfection reagent (step 20) comes to an end, aspire all the solution in tube A and add into 293T cell flask and mix it gently. Placed it in 5% CO<sub>2</sub> incubator at 37 &deg;C for 6-8 hours or overnight.</li>
</ol>
<p>? TROUBLESHOOTING</p>
<ol start="23">
<li>After incubation of step 22, aspire the culture medium and wash cells using 10-15ml wash solution (PBS containing 1%FBS) to rinse the cells gently and thoroughly. Repeat this step 3 times.</li>
</ol>
<p>▲CRITICAL STEP This wash step should be performed carefully and gently to avoid <u>detaching</u> the cells. The detached cells will yield lower-titer or even no pseudotyped viruses</p>
<ol start="24">
<li>After the wash step, add 15 ml fresh complete DMEM to the flask and culture for 24 h.</li>
<li>At the end of 24-hour culture, collect the cell supernatant (supernatant A) into a 50-ml centrifuge tube and place it in 4 &deg;C refrigerator. Then, add 15 ml of fresh complete DMEM into the 293T cells and incubate for another 24 hours.</li>
<li>After the second time of incubation, harvest the supernatant (supernatant B) and mix it with the supernatant A. Centrifuge the mixture at 1,000 g for 10 minutes. And then filter it using 0.45-&mu;m filter. Aliquot the mixture 1ml into each 2-ml EP tubes and stored them at -80&deg;C conditions.</li>
</ol>
<p>■ PAUSE POINT The harvested pseudotyped virus could be stored at -80&deg;C for at least one year according to the experience in other pseudotyped virus.</p>
<p>? TROUBLESHOOTING</p>
<p>To check whether the residual G*&Delta;G-VSV will be mixed in the SARS-CoV-2 pseudotyped virus stock, the SARS-CoV-2 pseudotyped virus stock should be tested using a VSV G specific antibody sample when the pseudotyped virus is firstly used. In our laboratory, the VSV G specific antibody was generated by immunized Balb/c mouse with VSV G expressing plasmid at amount of 20 &mu;g/mouse.</p>
<p>So far, the residual G*&Delta;G-VSV has not been detected for the SARS-CoV-2 and other pseudotyped virus stocks in our laboratory<sup>16</sup>. Thus, this procedure should be omitted if strictly following this protocol.</p>
<p><strong>Titration of the SARS-CoV-2 pseudotyped virus </strong><strong>●</strong><strong>TIMING 1 d</strong></p>
<ol start="27">
<li>Take the pseudotyped virus from the -80&deg;C refrigerator and melt it in room-temperature water bath.</li>
<li>Use 96 well flat bottom culture plate to titrate the pseudotyped virus. Seal the outmost wells with 250&mu;l/well sterilized water to avoid the effect caused by evaporation of the edge-well culture medium (Figure 2a).</li>
<li>Place 135 &mu;l complete DMEM in well B2-G2. Add 100&mu;l into remaining well (B3-G11). Add 15 &mu;l pseudotyped virus in well B2-G2 to make an initial dilution of 1:10.</li>
<li>Start the serial dilution by transfer 50 &mu;l solution from B2-G2 to B3-G3 and mix it 6-8 times with multiple-channel pipette and then transfer 50 &mu;l to B4-G4. Repeat the transfer and dilution of samples through column 10 (B10-G10). After the final mixing, aspire 50 &mu;l solution from column 10 and discard it. The dilution step doesn&rsquo;t involve column 11 (B11-G11), which will be used as the cell control (CC).</li>
<li>Prepare Huh-7 cells using the same procedure as step 15. Then adjust the cell concentration to 2&times;10<sup>5</sup>cells/ml and add 100 &mu;l cell suspension into all the 60 inner wells.</li>
<li>Place the titration plate in the 5% CO<sub>2</sub> 37&deg;C incubator for 24 hours.</li>
<li>Thaw the required volume of the Britelite plus reagent before use in an ambient temperature water bath.</li>
</ol>
<p>▲CRITICAL STEP The temperature of Britelite plus reagent could influence the readout of the sample. Place the reagent in water bath not less than 30 min before use.</p>
<ol start="34">
<li>Remove 100 &mu;l of supernatant from each well, leaving around 100 &mu;l of liquid in each well.</li>
<li>Disperse 100 &mu;l of the Britelite plus reagent into each well.</li>
<li>Incubate in darkness at room temperature for 2 min to allow complete cell lysis. Mix 6-8 times using multi-channel pipette to a corresponding 96-well white or black plate.</li>
<li>Read the relative light unit (RLU) of the plate after the two-minute incubation in the luminescence meter.</li>
<li>Take 10-fold of the average RLU of the cell control wells as the cutoff value. The pseudotyped virus titer was calculated using Reed-Muench method. The raw data could be pasted into Microsoft Excel &ldquo;TCID50_SARS-CoV-2&rdquo; macro (Supplementary S2) to get the titer and target dilution fold of the pseudotyped virus. Figure 3 shows how to use the macro file to calculate the TCID50 for pseudotyped virus titration.</li>
</ol>
<p>? TROUBLESHOOTING</p>
<p><strong>Pseudotyped virus neutralization assay </strong><strong>●</strong><strong>TIMING 1 d</strong></p>
<ol start="39">
<li>Sample preparation: serum samples from animals or humans should be inactivated in water bath at 56&deg;C for 0.5 ml for 30 min.</li>
<li>Prepare the culture plate and seal the outmost wells as step 28.</li>
<li>Place150 &mu;l/well complete DMEM medium in column 2 (B2-G2), which was used as the cell control (CC) (Figure 2b).Add 100 &mu;l complete DMEM into each well of column 3 (B3-G3), which was designated as virus control (VC).</li>
<li>Dispense 142.5 &mu;l/well complete DMEM in the remaining row 2 (B4-B11), which is the initial dilution row. Add 100 &mu;l/well complete DMEM to the remaining wells (C4-G11).</li>
<li>Add 7.5 &mu;l of the test sample in duplicate to row B (B4-B11) in following order: sample 1-wells B4-B5, 2-wells B6-B7, 3-wells B8-B9, 4-wells B10-B11.</li>
</ol>
<p>▲CRITICAL STEP After addition of the sample, rinse the tips using the medium in the corresponding wells to avoid residual samples on the tip walls. 7.5 &mu;l sample + 142.5 &mu;l medium yields a sample dilution of 1:20.</p>
<p>▲CRITICAL STEP A positive and a negative control should be included in at least one plate in each time assay.</p>
<ol start="44">
<li>Mix the samples in row B and transfer 50 &mu;l to row C. Repeat the mixing and transfer through row G (they are 3-fold serial dilution). After final transfer and mixing, discard 50 &mu;l of liquid from wells in column 4-11.</li>
<li>Thaw the required amount of pseudotyped virus in ambient temperature bath. Diluted the SARS-CoV-2 pseudotyped virus to 1.3 &times; 10<sup>4</sup> TCID<sub>50</sub>/ml in the complete DMEM. Disperse 50 &mu;l of the diluted pseudotyped virus into to each well in columns 3-11. Then, cover plates and incubate in a 5% CO<sub>2 </sub>37&deg;C incubator for 60 min (45-90 min).</li>
</ol>
<p>▲CRITICAL STEP After addition of the pseudotyped virus, the sample is diluted another 1.5 fold, which make the initial dilution be 1:30 followed by 3 fold serial dilution.</p>
<ol start="46">
<li>After 30 min incubation, start to prepare and add the Huh-7 cells as step 31 when the incubation time coming to an end.</li>
<li>Place the plate in 5% CO<sub>2</sub> 37&deg;C incubator for 24 h.</li>
<li>Prepare the Britelite plus reagent at least 30 min before detection as step 33.</li>
<li>Remove 150 &mu;l of supernatant from each well, leaving around 100 &mu;l of liquid.</li>
<li>Perform the RLU detection as step 35-37.</li>
<li>Calculate the neutralization inhibition rate: inhibition rate = [1 &ndash;(average RLU of sample &ndash;average RLU of CC)/(average RLU of VC &ndash;average RLU of CC)] &times;100%. The neutralization titer is expressed as the reciprocal of the serum dilution when the inhibition rate is 50% or the corresponding antibody concentration when the inhibition rate is 50%. The raw data could be pasted into the Microsoft Excel &ldquo;Neutralization_SARS-CoV-2&rdquo; macro (Supplementary S2) to get the titer for each sample.</li>
</ol>
<p>▲CRITICAL STEP Figure 4 shows how to use the macro file to calculate the neutralization titers for serum, antibody, or fusion inhibitors by changing the wells of Dilution factor and start dilution.</p>
<p><strong>Pass/Fail Criteria</strong></p>
<ul>
<li>The average RLU of VC wells is &ge;1000 times the average RLU of CC wells.</li>
<li>The percent CV (%CV) of RLU in the virus control wells is &le;30%.</li>
<li>The percent CV (%CV) for the duplicate wells is &le;30% for sample dilutions.</li>
<li>The EC50 value of the negative sample derived from human serum is &lt; 30<sup>16</sup>.</li>
<li>The value of the positive control is within 3-fold of the average of the previous values.</li>
</ul><h2>Troubleshooting</h2><p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-family: Arial, Helvetica, sans-serif; font-size: 15px;">Table 1 Troubleshooting table</span></p>
<table style="border-collapse:collapse;border:none;">
    <tbody>
        <tr>
            <td style="width: 32.15pt;border-top: 1.5pt solid windowtext;border-left: none;border-bottom: 1.5pt solid windowtext;border-right: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">Step</span></span></p>
            </td>
            <td style="width: 74.2pt;border-top: 1.5pt solid windowtext;border-left: none;border-bottom: 1.5pt solid windowtext;border-right: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">Problem</span></span></p>
            </td>
            <td style="width: 92.1pt;border-top: 1.5pt solid windowtext;border-left: none;border-bottom: 1.5pt solid windowtext;border-right: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">Possible reason</span></span></p>
            </td>
            <td style="width: 216.35pt;border-top: 1.5pt solid windowtext;border-left: none;border-bottom: 1.5pt solid windowtext;border-right: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">Solution</span></span></p>
            </td>
        </tr>
        <tr>
            <td style="width: 32.15pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">1</span></span></p>
            </td>
            <td style="width: 74.2pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">The cells are lumpy after plating</span></span></p>
            </td>
            <td style="width: 92.1pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">Incomplete digestion</span></span></p>
            </td>
            <td style="width: 216.35pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">Before washing the cells using PBS, the culture medium should be discard completely. After cleaning with PBS, absorb and discard the PBS completely. After discarding trypsin, the cell culture bottle should be placed in the incubator without disturbing</span></span></p>
            </td>
        </tr>
        <tr>
            <td style="width: 32.15pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">&nbsp;</span></span></p>
            </td>
            <td style="width: 74.2pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">&nbsp;</span></span></p>
            </td>
            <td style="width: 92.1pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">Over digestion</span></span></p>
            </td>
            <td style="width: 216.35pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">Incubate strictly according to protocol timeline to avoid prolonged digesting period</span></span></p>
            </td>
        </tr>
        <tr>
            <td style="width: 32.15pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">9</span></span></p>
            </td>
            <td style="width: 74.2pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">Cells are blown up in flasks</span></span></p>
            </td>
            <td style="width: 92.1pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">When washing or adding culture medium, the liquid is added directly to the cell</span></span></p>
            </td>
            <td style="width: 216.35pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">When rinsing cells or adding culture medium, tilt the culture bottle to make the cellular side to face up. Add the liquid to the acellular side. When the liquid flows to the bottom of the flask, turn it gently to make the liquid infiltrate the cell surface. This method should be applied to all cell processing operations</span></span></p>
            </td>
        </tr>
        <tr>
            <td style="width: 32.15pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">24</span></span></p>
            </td>
            <td style="width: 74.2pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">The titers of pseudovirus is too low.</span></span></p>
            </td>
            <td style="width: 92.1pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">The transfection efficiency was low</span></span></p>
            </td>
            <td style="width: 216.35pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">The Lipofectamine3000 should be fully mixed with the plasmid in step 5. The liquid in tubeA and tubeB should be fully mixed in step 6, so that the transfection reagent packaging the plasmid fully and evenly</span></span></p>
            </td>
        </tr>
        <tr>
            <td style="width: 32.15pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">25</span></span></p>
            </td>
            <td style="width: 74.2pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">The sample is not clear</span></span></p>
            </td>
            <td style="width: 92.1pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">The serum/plasma sample contains blood lipids or fiber precipitation.</span></span></p>
            </td>
            <td style="width: 216.35pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">After inactivation, serum or plasma samples should be centrifuged at 6,000 g for 10 minutes. Then the supernatant will be used for follow-up detection.</span></span></p>
            </td>
        </tr>
        <tr>
            <td style="width: 32.15pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">37</span></span></p>
            </td>
            <td style="width: 74.2pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">The RLU for VC can&rsquo;t match for different pseudovirus bathes</span></span></p>
            </td>
            <td style="width: 92.1pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">Even diluted according to the same TCID50, the RLU may not be exactly the same.</span></span></p>
            </td>
            <td style="width: 216.35pt;border: none;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">It is acceptable as long as the difference of RLU value of virus control is less than 10 times. But the VC/CC should not be less than 1,000</span></span></p>
            </td>
        </tr>
        <tr>
            <td style="width: 32.15pt;border-top: none;border-right: none;border-left: none;border-image: initial;border-bottom: 1.5pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">37</span></span></p>
            </td>
            <td style="width: 74.2pt;border-top: none;border-right: none;border-left: none;border-image: initial;border-bottom: 1.5pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">%CV between duplicates are more than 30%</span></span></p>
            </td>
            <td style="width: 92.1pt;border-top: none;border-right: none;border-left: none;border-image: initial;border-bottom: 1.5pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">Inaccuracy of sample addition or dilution</span></span></p>
            </td>
            <td style="width: 216.35pt;border-top: none;border-right: none;border-left: none;border-image: initial;border-bottom: 1.5pt solid windowtext;padding: 0cm 5.4pt;vertical-align: top;">
                <p style="margin:0cm;margin-bottom:.0001pt;font-size:16px;font-family:SimSun;line-height:200%;"><span style="font-size: 15px;"><span style="font-family: Arial, Helvetica, sans-serif;">When adding or diluting samples, observe the liquid level in the pipette tips to ensure the identical volume.It is recommended to use multi-channel pipette to do the serial dilution.</span></span></p>
            </td>
        </tr>
    </tbody>
</table><h2>Timing</h2><p>Step 1, plasmid addition for transformation: 3 min</p>
<p>Step 2, heat shock for transformation: 10 min</p>
<p>Step 3, bacterial smearing: 5 min</p>
<p>Step 4, mono-colony picking: 5 min</p>
<p>Step 5, amplified culture of bacteria: 5 min</p>
<p>Step 6, bacteria collection: 25 min.</p>
<p>Step 7, cell lysis and precipitation: 30 min</p>
<p>Step 8, Prepare the spin column: 15 min</p>
<p>Step 9–12, wash and collect the plasmid: 40 min</p>
<p>Step 13, quantify and determine the quality of the plasmid: 10 min</p>
<p>Step 14, aliquots the plasmid: 15 min</p>
<p>Steps 15–16, cell preparation for pseudotyped virus production: 45 min.</p>
<p>Steps 17–22, transfection and infection of 293T cells: 1 h.</p>
<p>Steps 23–24, replacement of the transfection and infection solution with culture medium: 30 min.</p>
<p>Step 25, first harvest of the pseudotyped virus: 10 min.</p>
<p>Step 26, second harvest and aliquoting of the pseudotyped virus: 1 h.</p>
<p>Step 27, preparation of the pseudotyped virus for titration: 5 min.</p>
<p>Steps 28, preparation of culture plate for titration: 10 min.</p>
<p>Steps 29–30, pseudotyped virus dilution: 10 min.</p>
<p>Steps 31–32, preparation of cells for titration: 45 min.</p>
<p>Steps 33–34, luminescence detection: 30 min.</p>
<p>Step 38, titration calculation using the excel file Titration: 5 min.</p>
<p>Step 39, sample preparation: 1 h 20 min.</p>
<p>Step 40, preparation of culture plate for neutralization: 10 min.</p>
<p>Steps 41–44, serum or antibody sample dilution: 30 min.</p>
<p>Step 45, pseudotyped virus preparation and addition: 10 min.</p>
<p>Steps 46–47, preparation and dispersing of cells: 45 min.</p>
<p>Steps 48–50, luminescence detection: 30 min.</p>
<p>Step 51, calculation of the EC50 using the excel file Neutralization_6 dilution: 5 min.</p><h2>Anticipated Results</h2><p>Following this protocol, high-titer SARS-CoV–2S pseudotyped viruses can be obtained. Even if diluted 10 times, the ratio of virus control to cell control can reach more than 1000. The titer of the pseudovirus and the expected dilution in the neutralization test can be calculated using the Microsoft Excel “TCID50_SARS-CoV–2” macro provided in the attachment (Figure 5a). If the protocol is carried out appropriatly to detect neutralizing antibodies in serum samples, a well-fitted inhibition curve can be obsvered (Figure. 5b). Four samples could be detected in each 96-well plate at the same time. The neutralizing antibody titer of the sample can be calculated using the Microsoft Excel “Neutralization_SARS-CoV–2” macro in the attachment. That is the dilution of a serum sample that inhibits 50% of viral infection. If the test sample is monoclonal antibody or fusion inhibitor with assigned amount, half effective concentration of the sample can be obtained (Figure 5b).</p><h2>References</h2><p>1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Coronaviridae Study Group of the International Committee on Taxonomy of, V. The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2. <em>Nat Microbiol</em> (2020).</p>
<p>2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Chen, Y., Liu, Q. &amp; Guo, D. Coronaviruses: genome structure, replication, and pathogenesis. <em>J Med Virol</em> (2020).</p>
<p>3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Li, C., Yang, Y. &amp; Ren, L. Genetic evolution analysis of 2019 novel coronavirus and coronavirus from other species. <em>Infect Genet Evol</em> <strong>82</strong>, 104285 (2020).</p>
<p>4&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Wu, A.<em> et al.</em> Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China. <em>Cell Host Microbe</em> (2020).</p>
<p>5&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Chan, J. F.<em> et al.</em> Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan. <em>Emerg Microbes Infect</em> <strong>9</strong>, 221-236 (2020).</p>
<p>6&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Du, L.<em> et al.</em> The spike protein of SARS-CoV--a target for vaccine and therapeutic development. <em>Nat Rev Microbiol</em> <strong>7</strong>, 226-236 (2009).</p>
<p>7&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Li, F. Structure, Function, and Evolution of Coronavirus Spike Proteins. <em>Annu Rev Virol</em> <strong>3</strong>, 237-261 (2016).</p>
<p>8&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Cho, H., Excler, J. L., Kim, J. H. &amp; Yoon, I. K. Development of Middle East Respiratory Syndrome Coronavirus vaccines - advances and challenges. <em>Hum Vaccin Immunother</em> <strong>14</strong>, 304-313 (2018).</p>
<p>9&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Matsuyama, S.<em> et al.</em> Enhanced isolation of SARS-CoV-2 by TMPRSS2-expressing cells. <em>Proc Natl Acad Sci U S A</em> (2020).</p>
<p>10&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Zhu, Z.<em> et al.</em> Potent cross-reactive neutralization of SARS coronavirus isolates by human monoclonal antibodies. <em>Proc Natl Acad Sci U S A</em> <strong>104</strong>, 12123-12128 (2007).</p>
<p>11&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Rockx, B.<em> et al.</em> Structural basis for potent cross-neutralizing human monoclonal antibody protection against lethal human and zoonotic severe acute respiratory syndrome coronavirus challenge. <em>J Virol</em> <strong>82</strong>, 3220-3235 (2008).</p>
<p>12&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Du, L.<em> et al.</em> MERS-CoV spike protein: a key target for antivirals. <em>Expert Opin Ther Targets</em> <strong>21</strong>, 131-143 (2017).</p>
<p>13&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Zhou, Y., Yang, Y., Huang, J., Jiang, S. &amp; Du, L. Advances in MERS-CoV Vaccines and Therapeutics Based on the Receptor-Binding Domain. <em>Viruses</em> <strong>11</strong> (2019).</p>
<p>14&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Xia, S.<em> et al.</em> A pan-coronavirus fusion inhibitor targeting the HR1 domain of human coronavirus spike. <em>Sci Adv</em> <strong>5</strong>, eaav4580 (2019).</p>
<p>15&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Zhou, P.<em> et al.</em> A pneumonia outbreak associated with a new coronavirus of probable bat origin. <em>Nature</em> (2020).</p>
<p>16&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Nie, J.<em> et al.</em> Establishment and validation of a pseudovirus neutralization assay for SARS-CoV-2. <em>Emerg Microbes Infect</em> <strong>9</strong>, 680-686 (2020).</p>
<p>17&nbsp;&nbsp;&nbsp; Whitt, M. A. Generation of VSV pseudotypes using recombinant DeltaG-VSV for studies on virus entry, identification of entry inhibitors, and immune responses to vaccines. <em>J Virol Methods</em> <strong>169</strong>, 365-374 (2010).</p><h2>Authors' contributions</h2><p>Jianhui Nie, Weijin Huang, Miao Xu, and Youchun Wang wrote the paper; Jianhui Nie, Weijing Huang, Miao Xu, and Youchun Wang conceived and designed the experiments; Jianhui Nie, Qianqian Li, Jiajing Wu, Chenyan Zhao, Huan Hao, Huan Liu, Haiyang Qin, Lingling, Nie, Junkai Liu, Meng Wang, Qiyu Sun, and Xiaoyu, Li performed the experiments; Jianhui Nie and Meng Wang wrote Microsoft Excel macro files. All of the authors have read and approved the final manuscript.</p><h2>Acknowledgements</h2><p>This work was supported by National Science and Technology Major Projects of Drug Discovery [grant number 2018ZX09101001].</p><h2>Competing interests</h2><p>All authors declare no competing interest.</p></body></html>