<html><head><title>CUTseq is a versatile method for preparing multiplexed DNA sequencing libraries from low-input samples</title></head><body><h1>CUTseq is a versatile method for preparing multiplexed DNA sequencing libraries from low-input samples</h1>Authors: Xiaolu Zhang, Silvano Garnerone, Michele Simonetti, Marcin Nicoś, Luuk Harbers, Caterina Marchiò, Reza Mirzazadeh, Tiziana Venesio, Anna Sapino, Johan Hartman, Magda Bienko, Nicola Crosetto<br>Abstract: <p>Current multiplexing strategies for massively parallel sequencing of genomic DNA mainly rely on library indexing in the final steps of library preparation. This procedure is costly and time-consuming because a single library must be produced separately for each sample. Furthermore, library preparation is challenging in the case of low-input fixed samples, such as DNA extracted from formalin-fixed paraffin-embedded (FFPE) tissues. Here, we describe CUTseq, a method that uses restriction enzymes and in vitro transcription to barcode and amplify genomic DNA prior to library construction. We thoroughly validate CUTseq and demonstrate its applicability to both genome and exome sequencing, enabling multi-region genome profiling within single stained FFPE tissue sections, to assess intratumor heterogeneity at high spatial resolution. In conclusion, CUTseq is a versatile and cost-effective method for multiplexed DNA sequencing library preparation that can find numerous applications in research and diagnostics.</p><br>Keywords: Next-generation sequencing, FFPE tissue, PCR, whole-genome amplification, CUTseq, T7 phage promoter<br><h2>Introduction</h2><p>In the past decade, next-generation sequencing (NGS) technologies have become widely available in diagnostics and research laboratories. During this time, the number of methodologies for preparing DNA libraries for NGS has greatly expanded, while the cost of sequencing has exponentially dropped. In spite of this progress, sequencing of multiple samples in parallel remains costly, mainly due to the way in which multiplexing is achieved. Typically, this is done by indexing libraries prepared from individual samples, followed by pooling together multiple libraries in the same sequencing run. This means that all the steps in the library preparation procedure must be repeated for every sample to be sequenced, which is labor-intensive and multiplies the cost of reagents. Furthermore, accurate normalisation of library concentration is necessary before multiple libraries can be pooled together, which is not always possible and requires additional reagents. In contrast, being able to directly barcode genomic DNA prior to library construction, followed by pooling of differentially barcoded samples into a single library, should enable high levels of multiplexing at much lower cost. An example of application which would greatly benefit from improved solutions for NGS library multiplexing is multi-region sequencing of tumor samples. In this approach, DNA is extracted from multiple regions within the same tumor mass, or from multiple tumor sites in the same patient, and a library is prepared from each region. Multi-region tumor sequencing has been successfully used to assess levels of intratumor heterogeneity and to reconstruct evolutionary tumor histories in different cancer types. One limitation of current multi-region sequencing approaches is the size of the regions examined, which must be sufficiently large to enable the recovery of enough DNA to construct a library from every region separately. This precludes the possibility of examining a larger number of smaller regions, for example within a single tissue section, which would enable assessing intratumor heterogeneity at much higher spatial resolution. This, together with the high cost associated to constructing a single library for every region sampled, currently limits the applicability of multi-region sequencing in routine cancer diagnostics. Several approaches have been developed to barcode genomic DNA (gDNA) as well as to amplify sub-nanogram amounts of gDNA prior to library preparation. Direct incorporation of sequencing adapters into gDNA by engineered transposases allows rapid library preparation and is the basis of successful commercial solutions such as Nextera from Illumina. However, this approach still requires that individual libraries are generated from each sample, and then pooled together before sequencing. On the other hand, whole-genome amplification methods, such as DOP-PCR, MDA, MALBAC and the more recent SCMDA and LIANTI , achieve direct gDNA barcoding during genome amplification, so that multiple samples can be pooled together into a single multiplexed library. Although such methods are specifically tailored for whole-genome sequencing of single cells, they could, in principle, also be used for other applications, for instance to generate multiplexed libraries for multi- region tumor sequencing in tissue sections. One limitation, however, is that whole-genome amplification requires intact DNA and thus is not compatible with fixed tissue samples, in particular formalin-fixed, paraffin-embedded (FFPE) specimens, which still represent a cornerstone in pathology. Additionally, whole-genome amplification methods are very costly, making them hardly applicable in routine diagnostics. To overcome these limitations, here we develop a method, which we name CUTseq, that combines restriction endonucleases with in vitro transcription, to construct multiplexed DNA libraries for reduced representation genome sequencing of multiple samples in parallel. We show that CUTseq can be used to barcode gDNA extracted from both non-fixed and fixed samples, including old archival FFPE tissue sections. We benchmark CUTseq by comparing it to a widely used method of DNA library preparation, and demonstrate that CUTseq can be used for both genome and exome sequencing, enabling reproducible DNA copy number profiling and single-nucleotide variant calling even in low-input FFPE tissue samples. Lastly, we describe a workflow for rapid and cost-effective preparation of highly multiplexed CUTseq libraries, that can be applied in the context of high-throughput genetic screens and for cell line validation.</p><h2>Reagents</h2><p># reagents used only in high-throughput CUTseq</p><p>- CUTseq oligonucleotide adapters (for a list of adapters we used, see Supplementary Data)</p><p>- Absolute Ethanol (VWR, cat. no. 20816.367)</p><p>- Nuclease-free Phosphate-Buffered Saline (10X) pH 7.4 (Thermo Fisher Scientific, cat. no. AM9625)</p><p>- Nuclease-free water (Thermo Fisher Scientific, cat. no. 4387936)</p><p>- PinPoint Slide DNA Isolation System™ (ZymoResearch, cat. no. D3001)</p><p>- Cell Lysis Buffer: Tris-HCl 10mM, NaCl 10mM, EDTA 1mM, Igepal 0.2%, pH 8 @ 25 °C</p><p>- Nucleus Break Buffer: Tris-HCl 10mM, NaCl 150mM, EDTA 1mM, SDS 0.3%, pH 8 @ 25 °C</p><p>- Tail Buffer: Tris-HCl 10mM, NaCl 100mM, EDTA 50mM, SDS 1%, pH 7.5 @ 25 °C</p><p>- Proteinase K, Molecular Biology Grade (NEB, cat. no. P8107S)</p><p>- Phenol: Chloroform: Isoamyl Alcohol 25:24:1 (Sigma, cat. no. P2069-100ML)</p><p>- Glycogen from mussels (Sigma, cat. no. 10901393001)</p><p>- Sodium Acetate, pH 5.5 (Life Technologies, cat. no. AM9740)</p><p>- # Vapor-Lock (Qiagen, cat. no. 981611)</p><p>- CutSmart® buffer (NEB, cat. no. B7204S)</p><p>- HindIII-HF<sup>®</sup> 20U/ml (NEB, cat. no. R3104L)</p><p>- T4 DNA Ligase (Thermo Fisher Scientific, cat. no. EL0014)</p><p>- # T4 DNA Rapid Ligase (Thermo Fisher Scientific, cat. no. K1423)</p><p>- UltraPure™ BSA (50 mg/ml) (Thermo Fisher Scientific, cat. no. AM2616)</p><p>- Adenosine 5’- Triphosphate (ATP) Solution Set (NEB, cat. no. P0756L)</p><p>- Alkaline Phosphatase, Calf Intestinal (Promega, cat. no. M1821)</p><p>- W&amp;B Buffer: Tris-HCl 10mM, NaCl 1M, Igepal 0.2%, pH 8 @ 25 °C</p><p>- MEGAscript® T7 Transcription Kit (Thermo Fisher Scientific, cat. no. AM1334-5)</p><p>- DNase I, RNase-free (Thermo Fisher Scientific, cat. no. AM2222)</p><p>- RA3 adaptor and RTP, RP1 and RPI primers (custom-made by Integrated DNA Technologies Inc. based on the sequences in the TruSeq Small RNA Library Preparation kit, Illumina)</p><p>- RNaseOUT™ Recombinant Ribonuclease Inhibitor (Invitrogen, cat. no. 10777-019)</p><p>- T4 RNA ligase 2, truncated (NEB, cat. no. M0242L)</p><p>- SuperScript® IV Reverse Transcriptase (Thermo Fisher Scientific, cat. no. 18090200)</p><p>- NEBNext UltraII Q5 PCR Mastermix (NEB, cat. no. M0544L)</p><p>- Agencourt RNAClean XP with Scalable throughput (Beckman Coulter, cat. no. A63987)</p><p>- Agencourt AMPure XP (Beckman Coulter, cat. no. A63880)</p><p>- NEBNext® Ultra<sup>TM</sup> II DNA Library Prep Kit for Illumina (NEB, cat. no. E7645S)</p><p>- SureSelect XT HS Reagent Kit (Agilent, cat. no. G9704)</p><p>- Qubit® dsDNA HS Assay Kit (Thermo Fisher Scientific, cat. no. Q32851)</p><p>- High Sensitivity DNA Kit (Agilent, cat. no. 5067-4627)</p><p><br></p><h2>Equipment</h2><p>- Cell counter (for example, Countess II FL Automated Cell Counter, Thermo Fisher Scientific)</p><p>- Incubator (for example, Binder incubator, Model KB 53)</p><p>- Tabletop centrifuge (for example, Eppendorf® Microcentrifuge 5424)</p><p>- # I-DOT One MC (Dispendix GmbH, Stuttgart, Germany)</p><p>- Thermoshaker (for example, Eppendorf®&nbsp;Thermomixer Compact)</p><p>- PCR cycler </p><p>- Sonication device (for example, ME220 Focused-ultrasonicator, Covaris)</p><p>- SpeedVac Vacuum Concentrator (for example, SavantTM SpeedVacTM DNA 130 Integrated Vaccum Concentrator System, Thermo Fisher Scientific, cat. no. DNA130-230)</p><p>- DynaMag™-2 Magnet (Thermo Fisher Scientific, cat. no. 12321D)</p><p>- Qubit® 2.0 Fluorometer (Thermo Fisher Scientific, cat. no. Q32866)</p><p>- Bioanalyzer 2100 (Agilent, cat. no. G2943CA)</p><p><br></p><h2>Procedure</h2><p><strong>DAY 1</strong></p><p><strong>1. Genomic DNA (gDNA) extraction</strong></p><p><strong>Note</strong>: the following is a procedure to isolate gDNA from small regions within single FFPE tissue sections using the PinPoint Slide DNA Isolation System™. For tissue sections or cells, any isolation protocol yielding high-quality gDNA, including commercially available silica-based kits, is compatible with the subsequent steps of CUTseq.</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Apply a little amount of PinPoint glue onto a small region of interest of a FFPE tissue section previously stained with hematoxylin-eosin and imaged</p><p><strong>Note</strong>: this step is recommended to confirm the presence of tumor cells within the tissue region</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Air-dry the PinPoint glue for at least 30 min at room temperature</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Cut the solidified glue using a disposable insulin needle, then transfer the pellet into a LoBind tube pre-filled with 100 μl of tail buffer plus 10 μl of proteinase K</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Incubate on a thermomixer for 18 hours @ 55 °C, shaking at 800 rpm</p><p><br></p><p><strong>DAY 2</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add 10 μl of fresh proteinase K</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Incubate for 1 hour @ 55 °C</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Transfer the sample to a PCR thermocycler, and incubate for 10 min @ 95 °C to inactivate proteinase K</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Purify gDNA using phenol-chloroform</p><p><strong>Note</strong>: in our experience, phenol-chloroform extraction yields gDNA of the highest quality. However, we have also successfully used silica columns provided in the PinPoint Slide DNA Isolation System™</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add 120 μl of phenol/chloroform pre-warmed at room temperature, and shake vigorously</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Centrifuge at ~20,000 x g for 5 min at room temperature</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Collect the upper phase (~100 μl) and transfer it to a new LoBind tube</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add an equal volume of chloroform and shake vigorously</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Centrifuge at ~20,000 x g for 5min at room temperature</p><p>10.&nbsp;&nbsp;Collect the upper phase (~60 μl) and transfer it to a new LoBind tube</p><p>11.&nbsp;&nbsp;Add 3.7 μl of glycogen per 100 μl of solution, and vortex vigorously to mix</p><p>12.&nbsp;&nbsp;Add Na-Acetate 3M, pH 5.5 to a final concentration of 0.3 M, and vortex vigorously to mix</p><p>13.&nbsp;&nbsp;Add 2.5 volumes of ice-cold 100% ethanol, then immediately vortex vigorously to mix</p><p>14.&nbsp;&nbsp;Incubate for 16–18 hours @ –80 °C</p><p><br></p><p><strong>DAY 3</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Centrifuge at ~20,000 x g for 1 hour @ 4 °C</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Gently discard the supernatant</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Wash the DNA pellet with 500 μl of ice-cold 70% ethanol, by vortexing vigorously</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Centrifuge at ~20,000 x g for 15 min @ 4 °C</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Discard the supernatant and wash the pellet again with 500 μl of ice-cold 70% ethanol</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Centrifuge at ~20,000 x g for 15 min @ 4 °C</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Discard the supernatant</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Air-dry the DNA pellet</p><p><strong>Note</strong>: avoid over-drying the DNA pellet, as this may result in low DNA yield</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Resuspend the pellet in 8 μl of nuclease-free water</p><p><strong>Breakpoint</strong>: if needed, at this point the samples can be stored @ –20 °C for several months</p><p><br></p><p><strong>DAY 3</strong></p><p><strong>2. DNA Digestion</strong></p><p><strong>Note</strong>: here we describe the use of HindIII to digest gDNA, since this enzyme was used for most of the experiments described in this manuscript. However, different restriction enzymes can be used at this point to digest the purified gDNA. The choice of enzyme depends on the application and desired resolution. For example, for applications that require high genome coverage, such as exome sequencing, a 4-cutter is preferable. On the other hand, 6-cutters are better suited for applications for which shallow genome sequencing is sufficient, for instance DNA copy number profiling at low resolution. Additional aspects to consider when choosing which restriction enzyme to use are the distribution of recognition sites along the genome of interest (see <strong>Supplementary Figure 1</strong>), the sensitivity of different enzymes to methylation in the recognition sequence, as well as the cost of purchasing the enzymes from commercial providers. A list of enzymes that we recommend for CUTseq and are available through NEW ENGLAND BioLabs is provided in <strong>Supplementary Table 1</strong>.</p><p><strong>Standard CUTseq</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Transfer 8 μl of purified gDNA into a 0.5 ml LoBind tube</p><p><strong>Note</strong>: when processing multiple samples to be pooled into the same library, PCR tube strips or 96-well plates can be used, depending on the number of samples</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add 2 μl of digestion mix:</p><p>CutSmart buffer 10X 1 μl</p><p>HindIII-HF 1 μl</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Perform the following steps in a PCR thermocycler with the lid set @ 50 °C:</p><p>1. 37 °C 16-18 h</p><p>2. 80 °C 20 min</p><p>3. 4 °C Hold</p><p><strong>High-throughput CUTseq</strong></p><p><strong>Note</strong>: the following protocol was implemented on the I-DOT One nanodispensing system (Dispendix, Germany). Other systems may also be used; however, volumes might have to be adjusted depending on the technical specifications of each instrument.</p><p># Dispense with I-DOT</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Dispense 5 μl of Vapor Lock per well in a 96- or 384-well plate</p><p><strong>Note</strong>: 384-well plates are better suited when dispensing with nanoliter volumes as the conical bottom and size of the wells allows to more easily visualize the dispensed droplets inside the Vapor Lock phase.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;@ Add 350 nl per well of gDNA per well</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;@ Add 150 nl per well of digestion mix:</p><p>CutSmart buffer 10X 50 nl</p><p>HindIII-HF 100nl</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Perform the following steps in a thermo incubator</p><p>1. 37 ° 30 min</p><p>2. 80 °C 20 min</p><p>3. 4 °C Hold</p><p><br></p><p><strong>DAY 4 (DAY 3 for high-throughput CUTseq)</strong></p><p><strong>3. Ligation of CUTseq adapters</strong></p><p><strong>Standard CUTseq</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add 1μl per sample of 0.33 μM HindIII adapter with the desired barcode sequence</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Place the sample(s) on ice</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;To each sample, add 19 μl of ligation mix:</p><p>Nuclease-free water 12 μl</p><p>T4 ligase buffer 10X 3 μl</p><p>ATP 10 mM 2.4 μl</p><p>BSA 50 mg/ml 0.6 μl</p><p>T4 ligase highly conc. 1 μl</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;In a PCR thermocycler with the lid set @ 20 °C, perform the following steps:</p><p>1. 16 °C 16-18 h</p><p>2. 65 °C 10 min</p><p>3. 4 °C Hold</p><p><strong>High-throughput CUTseq</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;@ Add&nbsp;300 nl of 33 nM CUTseq adapter per well</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;@ Add 700 nl of ligation mix per well:</p><p>Nuclease-free water 50 nl</p><p>T4 ligase buffer 5X 300 nl</p><p>ATP 10 mM 120 nl</p><p>BSA 50 mg/ml 30 nl</p><p>T4 rapid ligase 200 nl</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp; Incubate at room temperature for 30 min</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Pool the volume in multiple wells into one 1.5 ml LoBind tube</p><p><strong>Note</strong>: in order to be pooled together, multiple wells must have different sample barcodes. The number of wells pooled together depends on the total number of samples, and on the desired complexity of the final library. In general, libraries containing many samples will need to be sequenced deeply in order for each sample to receive a sufficient number of reads. In our experience, we have successfully sequenced libraries containing up to 96 different barcodes in a single NextSeq 500 run, obtaining a number of reads per sample sufficient to perform reproducible DNA copy number profiling (see<strong> Figure 3</strong>).</p><p><br></p><p><strong>DAY 5 (DAY 3 for high-throughput CUTseq)</strong></p><p><strong>4. DNA cleanup</strong></p><p><strong>Note</strong>: at this point, the same procedure is followed for single-sample and multiplexed libraries. Below, we describe a standard DNA precipitation cleanup procedure. However, silica-based columns or AMPure XP beads can also be used, according to the manufacturer’s instructions.</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Measure the sample volume, then add the following reagents:</p><p>Glycogen 20 mg/ml 3.7 μl per 100 μl</p><p>Na-Acetate 3 M, pH 5.5 Up to 0.3 M final conc.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add 2.5 volumes of ice-cold 100% ethanol</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Vortex 5 seconds at max. power</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Incubate for 18 hours @ –80 °C</p><p><strong>Note</strong>: incubation @ –20 °C is also fine</p><p><br></p><p><strong>DAY 6 (DAY 4 for high-throughput CUTseq)</strong></p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Centrifuge at ~20,000 x g for 1 hour @ 4 °C</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Discard the supernatant</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Wash the DNA pellet with 500 μl of ice-cold 70% ethanol</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Centrifuge at ~20,000 x g for 15 min @ 4 °C</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Discard the supernatant and wash the pellet again with 500 μl of ice-cold 70% ethanol</p><p>10.&nbsp;&nbsp;Centrifuge at ~20,000 x g for 15 min @ 4 °C</p><p>11.&nbsp;&nbsp;Discard the supernatant</p><p>12.&nbsp;&nbsp;Air-dry the pellet</p><p><strong>Note</strong>: avoid over-drying the DNA pellet, as this may result in low DNA yield</p><p>10.&nbsp;&nbsp;Resuspend the pellet in 50 μl of nuclease-free water</p><p><strong>Breakpoint</strong>: if needed, at this point the samples can be stored at –20 °C for several months</p><p><br></p><p><strong>DAY 6 (DAY 4 for high-throughput CUTseq)</strong></p><p><strong>5. Sonication</strong></p><p><strong>Note</strong>: in our experience, DNA fragments of 200–300 bp are needed to obtain high-quality libraries and sequencing results. To sonicate gDNA, we typically use a Covaris ME220 Focused-ultrasonicator with microTUBE-50 AFA Fiber Screw-Cap tubes or microTUBE-15 AFA Beads Screw-Cap tubes, with a target peak of 200 bp. Note that we sonicate gDNA independently of the restriction enzyme used, as well as gDNA extracted from FFPE samples, since we have found that this results in higher-quality libraries.</p><p><strong>6. SpeedVac</strong></p><p>Transfer sample from covaris tube to new 0.5 ml DNA LoBind tube, proceed to SpeedVac Vaccum Concentrator with a normal heating mode till the moment when the volume of sample is entirely evaporated.</p><p><br></p><p><strong>DAY 6 (DAY 4 for high-throughput CUTseq)</strong></p><p><strong>7. <em>In vitro</em> transcription</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Resuspend the sample with 8 μl&nbsp;nuclease-free water</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add the following reagents on ice:</p><p>rATP+rUTP+rGTP+rCTP* 8 μl</p><p>T7 polymerase buffer 10X 2 μl</p><p>T7 polymerase 1.5 μl</p><p>RNaseOUT<sup>TM </sup>Recombinant Ribonuclease Inhibitor 40U 0.5 μl</p><p>*Prepared from separate rNTP solutions provided with the MEGAscript<sup>®</sup> T7 Transcription Kit</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Incubate for 14 hours @ 37 °C in a PCR thermocycler with the lid set @ 70 °C</p><p><br></p><p><strong>DAY 7 (DAY 5 for high-throughput CUTseq)</strong></p><p><strong>8. RNA cleanup</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add 1 μl of DNAse I RNase-free to the IVT product</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Incubate for 15 min @ 37 °C</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Bring up the volume to 30 μl by adding 9 μl nuclease-free water, then mix with 54 μl (1.8x) of RNAClean XP beads pre-warmed at room temperature</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Mix thoroughly and incubate for 10 min at room temperature</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Place the sample on a magnetic stand</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Incubate for at least 5 min until the liquid appears clear</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Remove and discard the supernatant</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Wash the beads twice with 200 μl of freshly prepared 70% ice-cold ethanol</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Air-dry the beads at room temperature</p><p><strong>Note</strong>: do not dry the beads for more than 5–8 min, since this may result in low DNA yield</p><p>10.&nbsp;&nbsp;Remove the sample from the magnetic stand</p><p>11.&nbsp;&nbsp;Resuspend the beads in 8 μl of nuclease-free water</p><p>12.&nbsp;&nbsp;Incubate for 2 min at room temperature</p><p>13.&nbsp;&nbsp;Place the sample back on the magnetic stand</p><p>14.&nbsp;&nbsp;Incubate for at least 5 min until the liquid appears clear</p><p>15.&nbsp;&nbsp;Transfer 7.8 μl of supernatant to a new 0.5 μl DNA LoBind tube</p><p><br></p><p><strong>Note</strong>: the following steps until section 11. are adapted from the TruSeq Small RNA Library Preparation protocol from Illumina. *Steps on ice.</p><p><br></p><p><strong>DAY 7 (DAY 5 for high-throughput CUTseq)</strong></p><p><strong>9. RA3 adapter ligation</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;* Add 1 μl of 10 μM RA3 adapter</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Incubate for 2 min @ 70 °C in a PCR thermocycler, then immediately place sample on ice</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;* Add 3.2 μl of the following mix:</p><p>RNA ligase buffer 1.2 μl</p><p>RNaseOUT<sup>TM</sup> Recombinant Ribonuclease Inhibitor 40U 1 μl</p><p>T4 RNA ligase truncated 1 μl</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Incubate for 2 hours @ 25 °C in a PCR thermocycler with the lid set @ 30 °C</p><p><br></p><p><strong>DAY 7 (DAY 5 for high-throughput CUTseq)</strong></p><p><strong>10. Reverse transcription (1st strand synthesis)</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;* Add 2 μl per sample of RTP primer</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;In a PCR thermocycler, incubate for 2 min @ 70 °C</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;* Quickly transfer the sample to ice</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;* Add 11 μl of the following mix:</p><p>1st stand buffer 4 μl</p><p>dNTPs @ 25 mM 1 μl</p><p>100 mM DTT 2 μl</p><p>RNaseOUT<sup>TM</sup> Recombinant Ribonuclease Inhibitor 40 U 2 μl</p><p>SuperScript IV reverse transcriptase 2 μl</p><p>5. Incubate for 1 hour @ 50°C in a PCR thermocycler with the lid set @ 50 °C</p><p><strong>Breakpoint</strong>: if needed, at this point the samples can be stored @ –20 °C for several months</p><p><br></p><p><strong>DAY 7 (DAY 5 for high-throughput CUTseq)</strong></p><p><strong>11. Library indexing and amplification</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;* Add 4 μl per sample of the desired indexed Illumina primer</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;* Add 71 μl of the following mix:</p><p>Nuclease-free water 17 μl</p><p>NEBNext® Ultra<sup>TM</sup> II PCR Master Mix 50 μl</p><p>RP1 primer 4 μl</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;In a PCR thermocycler perform the following cycles:</p><p>1. 98 °C30 sec</p><p>2. 98 °C 10 sec</p><p>3. 65 °C 75 sec</p><p>GOTO step 2, 8-20 times</p><p>4. 65 °C 5 min</p><p>5. 4 °C Hold</p><p><br></p><p><strong>Note</strong>: the number of PCR cycles needs to be adjusted depending on the gDNA input in the IVT reaction. The following table shows the recommended number of PCR cycles for various gDNA inputs, based on our experience:</p><p><strong>Input gDNA (ng)</strong></p><p><strong># PCR cycles</strong></p><p>500-700 ng gDNA input: 6 cycles</p><p>300-400 ng gDNA input: 7 cycles</p><p>100-200 ng gDNA input: 8 cycles</p><p>50-100 ng gDNA input: 9 cycles</p><p>30 ng gDNA input: 10 cycles</p><p>15 ng gDNA input: 11 cycles</p><p>7.5 ng gDNA input: 12 cycles</p><p>3.8 ng gDNA input: 13 cycles</p><p>1.9 ng gDNA input: 14 cycles</p><p><br></p><p><strong>DAY 7 (DAY 5 for high-throughput CUTseq)</strong></p><p><strong>12. Library size selection and purification</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Transfer each sample into a new 1.5 ml LoBind tube</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add 70 μl (0.7x) of AMPure XP beads pre-warmed at room temperature</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Mix thoroughly and incubate for 10 min at room temperature</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Place the sample on a magnetic stand</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Incubate for at least 5 min until the liquid appears clear</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Carefully collect the supernatant and transfer it to a new 1.5 ml LoBind tube</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add 20 μl (0.2x) of AMPure XP beads pre-warmed at room temperature</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Mix thoroughly and incubate for 5 min at room temperature</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Place the sample on a magnetic stand</p><p>10.&nbsp;&nbsp;Remove and discard the supernatant</p><p>11.&nbsp;&nbsp;Wash the beads twice with 200 μl of freshly prepared 80% ice-cold ethanol</p><p>12.&nbsp;&nbsp;Air-dry the beads at room temperature</p><p><strong>Note</strong>: do not dry the beads for more than 5–8 min, since this may result in low DNA yield</p><p>13.&nbsp;&nbsp;Remove the sample from the magnetic stand</p><p>14.&nbsp;&nbsp;Resuspend the beads in 10–20 μl of nuclease-free water</p><p>15.&nbsp;&nbsp;Incubate for 2 min at room temperature</p><p>16.&nbsp;&nbsp;Place the sample back on the magnetic stand</p><p>17.&nbsp;&nbsp;Incubate for at least 5 min until the liquid appears clear</p><p>18.&nbsp;&nbsp;Transfer the supernatant to a new 1.5 μl DNA LoBind tube</p><p>19.&nbsp;&nbsp;Check the library concentration using Qubit</p><p>20.&nbsp;&nbsp;Check the library quality and size on a Bioanalyzer using a high sensitivity DNA kit</p><p>21.&nbsp;&nbsp;Store the library @ –20 °C</p><p><br></p><p><br></p><h2>Anticipated Results</h2><p>Tagged cut sites by CutSeq will generate high quality sequencing libraries with fragment size of 200-300 bp, that can be checked on Bioanalyzer 2100&nbsp;(Agilent Technologies).</p><h2>References</h2><p>1. Reuter, J. A., Spacek, D. V. &amp;amp; Snyder, M. P. High-throughput sequencing technologies.</p><p>Mol. Cell 58, 586–597 (2015).</p><p>2. Goodwin, S., McPherson, J. D. &amp;amp; McCombie, W. R. Coming of age: ten years of next-</p><p>generation sequencing technologies. Nat. Rev. Genet. 17, 333–351 (2016).</p><p>3. McGranahan, N. &amp;amp; Swanton, C. Clonal Heterogeneity and Tumor Evolution: Past,</p><p>Present, and the Future. Cell 168, 613–628 (2017).</p><p>4. Telenius, H. et al. Degenerate oligonucleotide-primed PCR: general amplification of</p><p>target DNA by a single degenerate primer. Genomics 13, 718–725 (1992).</p><p>5. Dean, F. B. et al. Comprehensive human genome amplification using multiple</p><p>displacement amplification. Proc. Natl. Acad. Sci. 99, 5261–5266 (2002).</p><p>6. Zong, C., Lu, S., Chapman, A. R. &amp;amp; Xie, X. S. Genome-Wide Detection of Single</p><p>Nucleotide and Copy Number Variations of a Single Human Cell. Science 338,</p><p>1622–1626 (2012).</p><p>7. Dong, X. et al. Accurate identification of single-nucleotide variants in whole-genome-</p><p>amplified single cells. Nat. Methods 14, 491–493 (2017).</p><p>8. Chen, C. et al. Single-Cell Whole Genome Analyses by Linear Amplification via</p><p>Transposon Insertion (LIANTI). Science 356, 189–194 (2017).</p><p>9. Kivioja, T. et al. Counting absolute numbers of molecules using unique molecular</p><p>identifiers. Nat. Methods 9, 72–74 (2011).</p><p>10. Ullah, I. et al. Evolutionary history of metastatic breast cancer reveals minimal seeding</p><p>from axillary lymph nodes. J. Clin. Invest. 128, 1355–1370 (2018).</p><p>11. Forbes, S. A. et al. COSMIC: somatic cancer genetics at high-resolution. Nucleic Acids</p><p>Res. 45, D777–D783 (2017).</p><p>12. Nik-Zainal, S. et al. Landscape of somatic mutations in 560 breast cancer whole-genome</p><p>sequences. Nature 534, 47–54 (2016).</p><p>13. Kandoth, C. et al. Mutational landscape and significance across 12 major cancer types.</p><p>Nature 502, 333–339 (2013).</p><p>14. Baird, N. A. et al. Rapid SNP Discovery and Genetic Mapping Using Sequenced RAD</p><p>Markers. PLOS ONE 3, e3376 (2008).</p><p>15. Davey, J. W. et al. Genome-wide genetic marker discovery and genotyping using next-</p><p>generation sequencing. Nat. Rev. Genet. 12, 499–510 (2011).</p><p>16. Kosicki, M., Tomberg, K. &amp;amp; Bradley, A. Repair of double-strand breaks induced by</p><p>CRISPR–Cas9 leads to large deletions and complex rearrangements. Nat. Biotechnol. 36,</p><p>765–771 (2018).</p><p>17. Adikusuma, F. et al. Large deletions induced by Cas9 cleavage. Nature 560, E8 (2018).</p><p>18. Masters, J. R. Cell-line authentication: End the scandal of false cell lines. Nature 492, 186</p><p>(2012).</p><p>19. Sommer, C., Straehle, C., Köthe, U. &amp;amp; Hamprecht, F. A. Ilastik: Interactive learning and</p><p>segmentation toolkit. in 2011 IEEE International Symposium on Biomedical Imaging:</p><p>From Nano to Macro 230–233 (2011). doi:10.1109/ISBI.2011.5872394</p><p>20. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Nat.</p><p>Methods 9, 676–682 (2012).</p><p>21. Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM.</p><p>ArXiv13033997 Q-Bio (2013).</p><p>22. Smith, T., Heger, A. &amp;amp; Sudbery, I. UMI-tools: modeling sequencing errors in Unique</p><p>Molecular Identifiers to improve quantification accuracy. Genome Res. 27, 491–499</p><p>(2017).</p><p>23. Rausch, T., Fritz, M. H.-Y., Korbel, J. O. &amp;amp; Benes, V. Alfred: Interactive multi-sample</p><p>BAM alignment statistics, feature counting and feature annotation for long- and short-</p><p>read sequencing. Bioinforma. Oxf. Engl. (2018). </p><p>24. Scheinin, I. et al. DNA copy number analysis of fresh and formalin-fixed specimens by</p><p>shallow whole-genome sequencing with identification and exclusion of problematic</p><p>regions in the genome assembly. Genome Res. 24, 2022–2032 (2014).</p><p>25. Lai, Z. et al. VarDict: a novel and versatile variant caller for next-generation sequencing</p><p>in cancer research. Nucleic Acids Res. 44, e108 (2016).</p><p>26. Cock, P. J. A. et al. Biopython: freely available Python tools for computational molecular</p><p>biology and bioinformatics. Bioinforma. Oxf. Engl. 25, 1422–1423 (2009).</p><p>27. Quinlan, A. R. &amp;amp; Hall, I. M. BEDTools: a flexible suite of utilities for comparing</p><p>genomic features. Bioinforma. Oxf. Engl. 26, 841–842 (2010).</p><h2>Acknowledgements</h2><p>We thank Bastiaan Spanjaard and Alexander van Oudenaarden for helping during the initial</p><p>stages of CUTseq implementation. This work was supported by a postdoctoral scholarship</p><p>from the Swedish Society for Medical Research (SSMF) to X.Z; by a Mobilnosc Plus</p><p>scholarship from the Polish Ministry of Science and High Education (1622/MOB/V/2017/0) to</p><p>M.N.; by grants from Italian Ministry of Health, Ricerca Corrente 2017, AIRC 5xMille MCO</p><p>Extension Program (# 9970), FPRC 5xMille Ministero Salute 2014 and FPRC 5xMille</p><p>Ministero Salute 2013 to A.S.; by additional grants from SSMF, the Swedish Cancer Fund,</p><p>Stockholm Cancer Fund and Stockholm County Council to J.H.; by grants from the Science</p><p>for Life Laboratory, the Swedish Research Council (621-2014-5503), and the Ragnar</p><p>Söderberg Foundation to M.B.; and by grants from the Swedish Research Council (521-2014-</p><p>2866), the Swedish Cancer Research Foundation (CAN 2015/585), the Ragnar Söderberg</p><p>Foundation, the Swedish Foundation for Strategic Research (BD15-0095), and the Strategic</p><p>Research Programme in Cancer (StratCan) at Karolinska Institutet to N.C.</p><p><br></p><p>Author contributions</p><p>Conceptualization: X.Z., M.B., N.C. Clinical samples: C.M., T.V., A.S. Data curation: X.Z.,</p><p>S.G., M.S. Formal analysis: S.G., L.H. Funding acquisition: N.C., M.B. Investigation: X.Z.,</p><p>M.S., M.N., C.M., T.V. Methodology: X.Z., N.C., M.B. Project administration: N.C.</p><p>Software: S.G. Supervision: N.C., M.B. Validation: X.Z., M.N., M.S., R.M. Visualization:</p><p>S.G., L.H., X.Z., M.S., N.C., M.B. Writing: N.C., X.Z.</p></body></html>