<html><head><title>Co-transcriptional quantification of RNA synthesis, RNA folding and RNA-protein interactions by "Systems NMR" approach.</title></head><body><h1>Co-transcriptional quantification of RNA synthesis, RNA folding and RNA-protein interactions by "Systems NMR" approach.</h1>Authors: Yaroslav Nikolaev, Nina Ripin, Martin Soste, Paola Picotti, Dagmar Iber, Frédéric H.-T. Allain<br>Abstract: <p>The protocol describes how to setup and analyse observation of a co-transcriptional RNA folding network by Systems NMR approach. While most experimental approaches can monitor only a single molecule class or reaction type at a time, Systems NMR permits single-sample dynamic quantification of entire “heterotypic” networks – involving different reaction and molecule types. It thus provides a deeper systems-level understanding of biological network dynamics by combining the dynamic resolution of biochemical assays and the multiplexing ability of “omics”.</p><p>		This particular protocol describes the reconstruction of an 8-reaction co-transcriptional network - with simultaneous monitoring of RNA, metabolite, and proteins in a single sample at the same time. From reactions side, the protocol simultaneously quantifies RNA transcription, RNA folding and RNA-protein interactions (observed both from RNA and from protein side) and few other auxiliary reactions. In addition to fundamental analyses of reaction constants under different conditions, the current applications of this particular reconstruction are: (1) map RNA-binding interfaces on proteins without having to purify/order the RNA; (2) monitor co-transcriptional RNA folding perturbations by proteins and small molecules; (3) monitor RNA-transcription-driven protein phase-separation with the possibility to observe multiple proteins at once, each with residue-level resolution. Not counting the protein and RNA template preparation times, the NMR measurement and data analysis parts take about 1 day each.</p><p>		This protocol accompanies Nikolaev et al, Nature Methods, 2019 (doi:XXX). The most up-to-date version of the protocol (including example code and data) is available at: <a href="https://github.com/systemsnmr/ivtnmr" target="_blank">github.com/systemsnmr/ivtnmr</a></p><br>Keywords: Systems Biology, NMR, biochemical networks, transcription, RNA folding, interactions, phase separation, liquid-liquid-phase-separation, NMR spectroscopy, RNA-binding proteins<br><h2>Introduction</h2><p><br></p><h2>Reagents</h2><p>### DNA/RNA template prepation:</p><p>- Standard DNA cloning, bacterial expression and DNA purification reagents</p><p>- Midi/Maxi kit for DNA purification allowing to purify ≥ 250 ug plasmid DNA</p><p>- NEB buffer 3.1</p><p>- BsaI enzyme for plasmid linearization</p><p><br></p><p>### In-vitro-transcription in NMR tube</p><p>- 5mm (or 3mm) NMR tubes (TA type is ok)</p><p>- 20x transcription buffer (TTD77): 800 mM Tris-HCl, pH 7.7, 0.2% Triton X-100, 100 mM dithiothreitol (DTT).</p><p>- 0.5mM DSS (4,4-dimethyl-4-silapentane-1-sulfonic acid) solution in D2O</p><p>- 80 mM nucleotide triphosphate (NTP) solutions (for each NTP, Applichem, ATP: A1348.0005; GTP: A1803.0001; CTP: A2145.0500; UTP: A2237.0001) - adjusted to pH 7.7-8.</p><p>- 1M MgCl2</p><p>- 100 U/ml Inorganic Pyrophosphatase from baker’s yeast (Sigma)</p><p>- ~70 uM (7 g/l 100 kDa) - 250x T7 RNA Polymerase (usually prepared in-house)</p><p>- [If protein observations are planned]: RNA-binding protein of interest</p><h2>Equipment</h2><p>- Standard DNA cloning, bacterial expression and DNA purification equipment</p><p>- NMR spectrometer with 1H + 31P cryoprobe (e.g. CPQCI), and ≥ 500 MHz 1H frequency (~12 Tesla)</p><p>- Computer with TopSpin and Matlab software</p><h2>Procedure</h2><p>The most up-to-date version of the protocol (including example code and data) is available at: <a href="http://github.com/systemsnmr/ivtnmr" target="_blank">github.com/systemsnmr/ivtnmr</a></p><p><br></p><p>### (A) Design and preparation of RNA transcription template(s), and protein (~8 days)</p><p><br></p><p>#### 1. RNA sequence design (~0.5 day)</p><p>Because substantial fraction (30-70%) of transcription products are short abortive RNA products, an ~8nt-long 5'-overhang nucleotide sequence needs to be prepended to the main RNA, to minimize interference of these short abortive RNAs with specific protein-RNA interactions and RNA folding. This sequence is designed algorithmically and can be used as a separate control to identify specific RNA effects from the other network perturbations.</p><p><br></p><p>Having decided on the primary RNA sequence, run `sample_prep/aborts_n_backfolding_XX.m` script, following instructions/examples in the header of the script for 5'-overhang generation (e.g. avoiding formation of dimers, excluding GG/purine pairs, etc. The script will produce a range of possible 5'-overhang variants, from which one or several can be selected for experimental tests. Main criteria for downstream experimental selection are:</p><p><br></p><p>- Judging from denaturing / non-denaturing gel-electrophoresis:</p><p>&nbsp;- homogeniety of transcription product</p><p>&nbsp;- yield of transcription product</p><p>- If observation of imino signals from the folded RNA is important, and you're choosing between several 5'-overhang variants: run 1D1H NMR of the transcription mixture, and choose the sequence which produces the least number of "background" signals in the imino region of the spectrum.</p><p><br></p><p>#### 2. Preparation of DNA templates (~7 days)</p><p>To ensure maximum homogeniety of synthesized RNA, we do the transcription from a plasmid DNA template (not from synthetic oligo-nucleotide templates). To increase efficiency, multiple templates can be prepared in parallel.</p><p><br></p><p>#### 2A. Clone the template DNA under T7 RNA Pol promoter (~1 day cloning, ~1 day wait for colonies, ~2 days miniprep + sequencing)</p><p>We are using pTX1 vector system [Michel E. et. al, 2018](https://doi.org/10.1007/978-1-4939-7634-8_11). The `sample_prep/ptx1_primers.xls` provides automated template for design of cloning primers (see instructions in the file).</p><p><br></p><p>#### 2B. Purify DNA template (~1.5 days)</p><p>&nbsp;- Use a kit Nucleobond Xtra Maxi (Macherey-Nagel) or an equivalent from Qiagen, etc. This yields per column ≥250-350 ug pTX1 plasmid DNA template, enough for ~10 transcription-NMR reactions (450ul each, at 33 nM DNA template).</p><p><br></p><p>&nbsp;Tips for the procedure:</p><p>&nbsp;- Per one Maxi column grow ~300ml culture till OD600≈3 (OD*V=900).</p><p>&nbsp;- Can try to saturate the Maxi columns more, to reach 500-1000 ug plasmid yield per column: then grow e.g. 600ml of OD600≈3 culture, and increase the volumes of buffer used for lysis &amp; etc.</p><p>&nbsp;- To get max yields: start the culture use a fresh (&lt;7 days) colony, make overday preculture in 10ml until OD600≈1, dilute 1:100 in the final ~300 ml medium for overnight growth. (or store the fresh OD600≈1 preculture at 4ºC and start main culture in the morning).</p><p>&nbsp;- Use baffled flasks, this allows to reach OD600≈3, giving 4g cells / L culture.</p><p>&nbsp;- Use 2x antibiotic amount than usual.</p><p><br></p><p>&nbsp;- Elution from column: pre-heat elution buffer + make elution twice (reapplying same eluate).</p><p>&nbsp;- After isopropanol precipitation centrifuge 1h @ 4000xg, 4ºC.</p><p>&nbsp;- After isopropanol removal, wash the pellet not once, but 3 times with 70% ethanol, each time detaching DNA pellet from the walls by inverting the tube, and then centrifugation. This removes salts which reduce the yield &amp;/or quality of transcription.</p><p>&nbsp;- Dry the pellet on air for 4-6 hours (overnight also ok).</p><p>&nbsp;- Dissolve DNA in H2O to ~0.41 µM concentration (650 µg/ml for 2.4kb plasmid) - this allows to have sufficiently concentrated DNA in the end (after linearization), to not dilute the final NMR sample too much.</p><p>&nbsp;- To ensure homogenous dissolution of DNA pellet after drying, perform 3 cycles of: [vortex, incubate 10-20 minutes at 50ºC, vortex, freeze at -20ºC, thaw]. If pellet did not detach from the wall after first vortex, use the pipet to detach it.</p><p><br></p><p>#### 2C. Linearize the DNA template (~0.5 day).</p><p>To maximize the yield and homogeniety of the main RNA product, it is important to thoroughly linearize the template. For the pTX1 plasmid we use BsaI enzyme (NOT the BsaI-HF version!), at 1.5U/ug DNA, in NEB buffer 3.1, for 13-15 hours at 50ºC, followed by 1h at 65ºC inactivation. Check the plasmid cleavage on 1% agarose gel.</p><p><br></p><p>#### 2D. Optimize the MgCl2 concentration for transcription (~1 day)</p><p>Run small-scale transcriptions (~25 µl is enough) and analyze them using denaturing PAGE with urea. In our hands for pure DNA template optimal MgCl2 range was always close to 1:1 ratio with NTPs (e.g. 20-24mM MgCl2 when using 20mM NTPs).</p><p><br></p><p>#### 3. Buffer exchange for protein of interest (~0.5 day)</p><p>To match the starting Tris-Triton-DTT (TTD) transcription buffer, transfer your protein into: 40 mM Tris-HCl, 0.01% Triton-X100, 5 mM Dithiothreitol, pH 7.7. From our experience many proteins are stable in this buffer. To stabilize the protein more, one can potentially add:</p><p>&nbsp;- 50 mM L-Arg/L-Glu (this leads to increased broadening of imino signals)</p><p>&nbsp;- up to ~75 mM NaCl / KCl (salt decreases transcription efficiency)</p><p>&nbsp;- ~1 mM NTPs - these seem capable to mask RNA-binding interfaces, reducing self-aggregation of protein through those. Added NTPs need to be taken into account in the total NTP concentration -- for addition of MgCl2, and for setting the initial concentrations during ODE network modeling.</p><p><br></p><p>### (B) Setup of an in-vitro-transcription-NMR reaction (30-60 minutes)</p><p>#### 1. Sample prep</p><p>- Sample preparation template: `sample_prep/ivtnmr.xslx`</p><p>- Below setup is for 450 ul samples in 5mm TA NMR tubes. One can also use 250-300ul samples is shigemi tubes and/or 150ul in 3mm tubes (but for 3mm tubes the acquisition times may need to be adjusted).</p><p>- To increase observability of imino signals we:</p><p>&nbsp;- Run reactions at 30ºC instead of 37ºC.</p><p>&nbsp;- Exclude commonly added spermidine from the transcription buffer because it leads to significant broadening of imino signals.</p><p>&nbsp;- Use transcription buffer with pH 7.7. T7 RNA Pol gives better yields at pH 8.1, but iminos are observed better at lower pH. During typical transcription reaction (starting with 20mM NTPs), due to release of free xPO4, pH goes down by ~0.2 pH units. The pH change can be monitored from the shift of 31P xPO4 signal and/or shift of 1H Tris signal.</p><p><br></p><p>- In case you need larger fraction of sample volume to add diluted protein or other additives, the NTPs, MgCl2 and DNA template components can be pre-lyophilized together. This seems to not affect the yield or homogeniety of RNA product.</p><p><br></p><p>- T7 RNA polymerase is added only after time0 reference spectra are recorded.</p><p><br></p><p>#### 2. NMR setup</p><p>Notes:</p><p><strong>!!!!!!!!!!</strong></p><p><strong>WARNING: the TopSpin Python scripts and pulseprograms are provided here "as is" - merely as a guideline for automated setup. These were not thoroughly tested on different spectrometers and may contain bugs and incompatibilities with TopSpin and spectrometer console versions!</strong></p><p><strong>!!!!!!!!!!﻿</strong></p><p><br></p><p>- We use an automated script `nmr/py/in_v02.yn.py` which makes tuning-matching, shimming, pulse calibrations and experiment setup.</p><p>- The measured experiments are configured by this script based on the template experiments which are placed into an empty IVTNMR_template - template dataset (example in `nmr/IVTNMR_template`):</p><p>&nbsp;- 1D1H (expno 12, pp `nmr/pp/zg-wg001`) - full 1H spectrum - for DSS calibration, pH check from Tris position, and potential quantification of individual NTPs (A/U/G/C) consumption rates (all four nucleotides have some specific signals in the aromatic region of the spectrum).</p><p>&nbsp;- 2D 1H1H TOCSY (expno 13, pp `nmr/pp/stocsy003`) - (not required for the setup described in the paper) - allows to observe RNA with higher resolution on certain signals, e.g. U/C H5-H6 correlations.</p><p>&nbsp;- 1D31P (expno 14, pp `nmr/pp/zgig002`) - to observe 31P-containing molecules</p><p>&nbsp;- 2DHN-sofast-hmqc (expno 15, pp `nmr/pp/sfhmqc01`) - to observe protein</p><p>&nbsp;- 1D1H-sofast (expno 16, pp `nmr/pp/zg-sofast006`) - to observe RNA imino region with increased sensitivity</p><p><br></p><p>Step-by-step procedure:</p><p>1. Create new dataset, including main information in its name, for simplification of automated analysis: `YYMMDD_INXXX_RNANAME_PROTEINNAME_TEMPERATURE_MAGNET` (e.g. `180914_IN71b_SMN1_coNUP1_303K_600`). Here, INXXX (IN71b) - is the experiment ID, implying same ID for the same RNA and protein combination. Replicate experiments are denoted with lowercase letter (IN71a, IN71b, ..) - so its easy to find similar experiments programmatically from shell, Matlab, etc.</p><p><br></p><p>2. Make temperature calibration (we use expno 1 consistently).</p><p><br></p><p>3. Insert the sample into magnet. Wait 1 min for temperature equilibration. Run `nmr/py/in_v02.yn.py` script for the setup of series. Check and test this script carefully before doing real runs.</p><p><br></p><p>4. After the automation script has finished:</p><p>&nbsp;&nbsp;- In 2DHN spectrum - set the SW / carrier / TD / etc parameters to the values optimal for your protein</p><p>&nbsp;&nbsp;- Check the 90-degree 31P pulse value (script runs `paropt` procedure, storing results in expno 4 999) - and enter it into 1D31P experiment.</p><p>&nbsp;&nbsp;- (If expect drift of tune-matching system): check the tune-match of channels of interest.</p><p>&nbsp;&nbsp;- Start reference (time0) experiments.</p><p><br></p><p>&nbsp;&nbsp;- **NOTE**: The key reference spectra required at this stage are 1D31P (as a reference for initial NTP concentration) and 2DHN (as a reference for initial protein peak positions / intensities). If the same batches of protein / NTPs are used - one can potentially start directly without recording new time0 references, and for downstream data analysis just make a copy of the references recorded in earlier days.</p><p><br></p><p>5. Check that Autoshim is turned off (in our experience Autoshim can introduce variation in lineshapes between individual 1D/2D spectra, which is later hard to distinguish from real changes in the network / molecule dynamics).</p><p><br></p><p>6. While references are running: use `nmr/py/createExpSeries.py` to generate time-series of repeating experiments for required time-period from the reference spectra. See the header of the script for details.</p><p><br></p><p>7. If some tuning-matching was done during setup (if not doing consecutive measurements in the same buffer): check the probe did not detune on the channels of interest while references were running.</p><p><br></p><p>8. Take sample out of the magnet, then:</p><p>&nbsp;&nbsp;- (2 min) add T7 RNA Pol, mix the sample, spin if need to remove bubbles, insert sample back to the magnet.</p><p>&nbsp;&nbsp;- (2 min) lock, temperature equilibration, check tune-match.</p><p>&nbsp;&nbsp;- (1 min) topshim, twice!.</p><p>&nbsp;&nbsp;- **NOTE**: remember/write down the time at which the RNA Pol was added. Later insert the exact time into the `notes.txt` file in the dataset directory. This file and time0 tag is automatically created by the `in_v02.xx.py` script. The &lt;time0&gt; tag gives the downstream analysis scripts info on when reaction was started:</p><p>&nbsp;&nbsp;`&lt;time0&gt;YYYY-MM-DD HH:MM:SS&lt;/time0&gt;`</p><p>&nbsp;&nbsp;**Currently the analysis scripts assume the time zone is set identical on the spectrometer computer and the spectrometer console and don't check GMT zone. So be careful if the GMT setting of your spectrometer station is different from the GMT setting written by console into audita.txt file (these settings m/b different on NEO consoles)**</p><p><br></p><p>9. Start experiment series (`qumulti ##-##`).</p><p><br></p><p>### (D) Data processing (&gt;= 1 hour per sample)</p><p><br></p><p>1. Sort experiments: `sort.py`. This script sorts data by experiment type, adds rough experiment time to the title, creates cara repository with linked 2DHN spectra and `integr_datasets_31P.txt` file for 31P integration. See script header for details. For auto-generation of cara repository, two fragments of final repository (`180917_INx_blank_1top.cara`, `180917_INx_blank_2bottom.cara`) need to be present in the py/user directory.</p><p><br></p><p>2. For quick visualization / comparison of data in TopSpin - see helper scripts: `res.py` (reading data by giving only parts of dataset name), `md.py` (overlaying series), `zim.py` and `zh33.py` (zooming on specific regions of spectra).</p><p><br></p><p>2. Except for 31P spectra - the phases of 1D1H, 1D1H-iminos and 2DHN spectra need to be detemined manually for optimal results. For 1D1H and 2DHN - ideally the common phase for time-resolved series is determined not on the first (time0) spectrum, but on the second spectrum - one after T7 addition. One might want to check the phases of 2DHN and 1D1H time0 spectra then (these might be slightly different from "time1" spectra).</p><p><br></p><p>3. Process 1D31P:</p><p>&nbsp;- `qumulti 5000-5500`</p><p>&nbsp;- example of processing command (check only SR corrections, in our setup phase seems determined robustly enough by `apks` routine):</p><p>&nbsp;`SR -139; si 64k; wdw EM; lb 2; efp; apks; absf1 14; absf2 -26; absg 5; absn`</p><p><br></p><p>4. Integrate: TopSpin `intser` command, pointing to the `integr_datasets_31P.txt` as the list of spectra to integrate (it should be created by `sort.py` in the dataset folder). Use default settings for integration - the calibration of integrals to internal NTP signal happens later during analysis.</p><p>&nbsp;- Integration regions we use:</p><p>&nbsp;```</p><p>&nbsp;#&nbsp;&nbsp;3.4714285714285715&nbsp;1.5571428571428572&nbsp;&nbsp;-- PO4</p><p>&nbsp;#&nbsp;&nbsp;0.42857142857142855&nbsp;-2.676190476190476&nbsp;-- PPi</p><p>&nbsp;#&nbsp;&nbsp;-4.3238095238095235&nbsp;-5.352380952380952&nbsp;-- RNA</p><p>&nbsp;#&nbsp;&nbsp;-5.352380952380952&nbsp;-5.390476190476191&nbsp;&nbsp;-- gammaNTP</p><p>&nbsp;#&nbsp;&nbsp;-5.390476190476191&nbsp;-5.804761904761905&nbsp;&nbsp;-- RNA-5'gamma?</p><p>&nbsp;#&nbsp;&nbsp;-9.101442036015632&nbsp;-9.806884238970339&nbsp;&nbsp;-- betaNDP</p><p>&nbsp;#&nbsp;&nbsp;-9.80952380952381&nbsp;-10.433333333333334&nbsp;&nbsp;-- alphaNDP</p><p>&nbsp;#&nbsp;&nbsp;-10.433333333333334&nbsp;-11.219047619047618 -- alphaNTP</p><p>&nbsp;#&nbsp;&nbsp;-17.60952380952381&nbsp;-19.052380952380954&nbsp;-- RNA-5'beta?</p><p>&nbsp;#&nbsp;&nbsp;-19.052380952380954&nbsp;-19.714285714285715 -- betaNTP</p><p>&nbsp;```</p><p><br></p><p>4. Process 2DHN:</p><p>&nbsp;&nbsp;- `qumulti 4000-4500`</p><p>&nbsp;&nbsp;- example of processing command (check phase, SR corrections, STSI/STSR for the dimensions of the final spectrum, etc):</p><p>&nbsp;&nbsp;`2 sr -26.41; 1 sr -2.676; 2 phc0 -264.203; 2 phc1 0; 2 si 8k; 1 si 512; 2 STSI 2214; 2 STSR 1512; 1 stsi 0; 1 stsr 0; 2 absf1 11; 2 absf2 5.5; 1 absf1 200; 1 absf2 80; absg 5; xfb n nc_proc 7; abs2; abs1`</p><p><br></p><p>5. Pick peaks in 2DHN spectra: create a project and a peaklist in a Cara repository, then pick peaks of interest and trace their positions across time series using peak aliasing in Cara Monoscope. The `sort.py` script creates a cara repository with linked spectra in the dataset folder. Downstream analysis scripts read the peaks information from cara repositories at the moment. If analyzing multiple datasets at once, it is, however, convenient instead of having multiple Cara files, to have one Cara file with each dataset represented by own project and peaklist. Useful shortcuts for peak tracing in Monoscope:</p><p>&nbsp;&nbsp;- `Ctrl+1 / Ctrl+2` - move between spectra</p><p>&nbsp;&nbsp;- `mp / ma` - move peak or its alias</p><p>&nbsp;&nbsp;- `gp` - go to peak</p><p>&nbsp;&nbsp;- `gs` - go to spectrum (if not argument provided will switch to the first spectrum in series)</p><p><br></p><p><br></p><p>### (E) Data analysis and model fitting (≈ 0.5 day)</p><p>Unless stated otherwise, most below procedures are done in Matlab (`*.m` files).</p><p>#### 1. Generate data structure for ODE model fit.</p><p>Run `analysis/ODE/v01/model_and_data/gen_ivtnmr.m` - this script will read 31P integrals from above 31P integration file, and 2DHN chemical shifts from cara repository, and then create the data structure used by the ODE model fitting procedure in `ODE/v01/model_and_data/data_for_fit` folder. It will also generate "ivtnmr" data object in `ODE/v01/model_and_data/data_ivtnmr_full/` - which contains main information about IVTNMR experiment in one Matlab structure (names, number of time-points, integrals, chemical shifts, etc). The code in `v01` folder is largely self-contained - so its convenient to just duplicate and rename it (e.g. `v02_test`) to keep track of different versions when you're making adjustments to data analysis / model structure / etc.</p><p><br></p><p>#### 2. Fit the ODE model.</p><p>Run `analysis/ODE/v01/a_fit_multi.m`, which can fit multiple IVTNMR datasets sequentially. The procedure reads the ODE model from `model_and_data/xIVT.mdf` file and fits it into experimental data from `model_and_data/data_for_fit` generated at the previous step. You may need to adjust the rna length in the MDF file to suit your RNA length - it is included in the &lt;RHS&gt; section (e.g. for 28-nucleotide RNA: `+f1*(27/28) +f2 -f3`). The mdf file is generated by Matlab `convertBNGL_to_MDF.m` script based on the `xIVT.bngl` - Rule-Based-Model defined in BioNetGen language (mode details below). The fitting script will generate PDF figures of the fit summary (in `ODE/v01/model_and_data/figure_images`) and will export fitting results into `ODE/v01/model_and_data/a_fit_multi.mat` which can be read and analyzed further.</p><p><br></p><p>#### 3. Visualize ODE fit results.</p><p>Run `analysis/ODE/v01/a_visualize.m` - this will automatically read fitting results from `ODE/v01/model_and_data/a_fit_multi.mat` and visualize the fitted constants, replicates and errors.</p><p><br></p><p>#### 4. Analyze and visualize time-resolved imino linewidths.</p><p>Run `analysis/LW/a_fit_LW.m` to fit and visualize imino linewidths. This fitting assumes a sigle peak with lorentzian lineshape, and thus works best for well-resolved signals. Decrease of pH during reaction (usually around ~0.2 pH units) might lead to slight systematic change of linewidth with reaction time. Also, a "growing" baseline in a crowded region may lead to apparent narrowing of linewidth with reaction time, because the current fitting routine assumes baseline fixed at zero intensity. This effect can be factored out, if appropriate baseline correction is found. In the single-lorentzian fit, the intrinsic broadening of one imino signal cannot be distinguished from broadening caused by overlap with neighboring signals, which has to be considered when interpreting the data.</p><p><br></p><p><br></p><p>### (F) Analysis of parameter (reaction constant) uncertainty</p><p>- **Replicates.** Based on our current experience, the most sensible / realistic parameter uncertainties are obtained by running ≥2 experimental replicates - ideally using different batches of DNA template and/or protein. Basic code for calculation of such uncertainties is included in `analysis/ODE/v01/a_visualize.m` &lt;!-- and `analysis/LW/a_visualize_LW.m`. --&gt;</p><p>- **Bootstrap.** Uncertainties obtained from bootstrap analysis (resampling of the full data vector with replacement prior to the fitting), in our setup yield too narrow confidence intervals - most likely due to very high number of recorded data points.</p><p>- **FIM.** The standard Fisher Information Matrix / Cramer Rao bound also often yields unrealistically narrow or unrealistically large uncertainties - most likely due to the lack of reasonable error estimates on the individual NMR integral / chemical shift data points.</p><p><br></p><p>### (G) Adjusting ODE model</p><p>1. In [RuleBender](http://visualizlab.org/rulebender/) software (or just text editor): **edit the BioNetGen network** definition (`ODE/v01/model_and_data/xIVT.bngl`). For downstream fitting it is important to match the number and order of defined observables to the data vectors you actually provide for model fitting.</p><p><br></p><p>2. In shell/terminal: **recompile the model**.</p><p>&nbsp;```</p><p>&nbsp;cd ODE/v01/model_and_data</p><p>&nbsp;../../../soft/BioNetGen217/Perl2/BNG2.pl xIVT.bngl</p><p>&nbsp;```</p><p><br></p><p>3. In Matlab: **convert the model to MDF format** for fitting.</p><p>&nbsp;```</p><p>&nbsp;cd ODE/v01/model_and_data</p><p>&nbsp;convertBNGL_to_MDF('xIVT.bngl')</p><p>&nbsp;```</p><p><br></p><p>4. In Matlab: **adjust the fitting routine** `v01/a_fit.m` to match the observables / data / constants used in the new network model. Key things which need to be checked:</p><p>&nbsp;&nbsp;- Order/number of data vectors used for fitting (needs to match the observables in mdf file) - check `obs_data_to_keep` variable;</p><p>&nbsp;&nbsp;- If added new observables/reactions: check if model initial params (NDP, Prot(s) conc, ..) are set correctly</p><p>&nbsp;&nbsp;- If added new constants/params: check their init values in the `a_fit.m`, file and/or exclude from optimization. And potentially add their normalization and saving into the output of model fit results.</p><p><br></p><p>**Note**: In the used here implementation BioNetGen cannot provide arbitrary rate laws and algebraic constants for scaling. If want such things included - need to specify them in the final `*.mdf` file used for fitting - similar to what we do for RNA length definition. Alternatively one can switch to e.g. [PottersWheel](https://potterswheel.de/) software which provides more advanced, up-to-date and fully integrated environment for model definition and optimization, including model optimization in log-space.</p><p><br></p><p><strong>### Disclaimer: Limitations of Liability for the code</strong></p><p>The code, pulseprograms, and especially TopSpin Python scripts, in this repository are provided "as is" - merely as a guideline for automated setup. These were not thoroughly tested on different spectrometers and may contain bugs and incompatibilities with TopSpin versions. Authors assume no responsibility, and shall not be liable to you or to any third party for any direct, indirect, special, consequential, indirect or incidental losses, damages, or expenses, directly or indirectly relating to the use or misuse of the code and pulseprograms provided here.</p><h2>Troubleshooting</h2><p>Indicated directly in corresponding sections.</p><h2>Time Taken</h2><p>Indicated directly in corresponding sections.</p><h2>Anticipated Results</h2><p>Key results from the network reconstruction:</p><p>- Overall pattern of RNA iminos - reporting on the structural fold of the RNA.</p><p>- Time-resolved LineWidths of RNA imino signals - quantitatively reporting on structural in RNA during transcription reaction. Linewidths are convertible to ∆G - free energy of folding, when the intrinsic and base-flipping rates of imino exchange (kex,intrinsic and kex,base-flipping) are known.</p><p>- Time-resolved and RNA-concentration resolved chemical shift and signal intensity perturbations for all observable residues in the protein of interest - reporting on potential interaction sites / structural changes in the protein.</p><p>- Kd of the protein-RNA interaction (if protein becomes saturated during reaction, and/or if chemical shifts of protein residues in RNA-saturated state are known).</p><p>- kcat rate of RNA transcription.</p><p>- Individual rates of ATP, GTP, UTP, CTP consumption (currently not yet implemented in the automated analysis).</p><h2>Acknowledgements</h2><p>We thank J. Vollmer and G. Fengos for the help with network modeling. We acknowledge G. Wider and all members of the ETH BNSP platform for excellent maintenance of the NMR infrastructure. We thank all members of the Allain Lab, in particular F. Damberger, and the Parpan retreat participants for helpful discussions. This work was supported by the Promedica Stiftung, Chur (Grant 1300/M to Y.N.), Novartis Foundation and Krebsliga Zurich (Y.N.), NCCR RNA and Disease by the Swiss National Science Foundation (F.A. and N.R.).</p></body></html>