<html><head><title>Somatic mutation landscapes at single-molecule resolution</title></head><body><h1>Somatic mutation landscapes at single-molecule resolution</h1>Authors: Stefanie V. Lensing, Peter Ellis, Federico Abascal, Iñigo Martincorena, Robert J. Osborne<br>Abstract: <p>Somatic mutations drive cancer development and may contribute to ageing and other diseases. Yet, the difficulty of detecting mutations present only in single cells or small&nbsp;clones has limited our knowledge of somatic mutagenesis to a minority of tissues. To overcome these limitations, we introduce nanorate sequencing (NanoSeq), a new duplex&nbsp;sequencing protocol with error rates &lt;5 errors per billion base pairs in single DNA molecules from cell populations. The version of the protocol described here uses&nbsp;clean genome fragmentation with a restriction enzyme to prevent&nbsp;end-repair-associated errors and ddBTPs/dATPs during A-tailing to prevent nick&nbsp;extension. Both changes reduce the&nbsp;error rate of standard duplex sequencing&nbsp;protocols by preventing the fixation of DNA damage into both strands of&nbsp;DNA&nbsp;molecules during library preparation. We also use qPCR quantification of the&nbsp;library prior to amplification to&nbsp;optimise the complexity of the sequencing&nbsp;library given the desired sequencing coverage, maximising duplex coverage.&nbsp;The sample preparation protocol takes between 1 and 2 days, depending on the number of samples&nbsp;processed. The bioinformatic protocol is&nbsp;described in:</p><p>https://github.com/cancerit/NanoSeq</p><p>https://github.com/fa8sanger/NanoSeq_Paper_Code</p><p><br></p><p><br></p><br>Keywords: NanoSeq, Mutation, Cancer, Duplex-Sequencing, BotSeqS, <br><h2>Reagents</h2><p>NEBNext®&nbsp;Ultra™ II Q5®&nbsp;Master Mix (NEB: M0544L)</p><p>HpyCH4V&nbsp;(NEB: R0620L)</p><p>T4 DNA Ligase (NEB: M0202L)</p><p>dATP Solution (NEB: N0440S)</p><p>xGen CS Adapter - Tech Access (IDT: 1080799)</p><p>CutSmart®&nbsp;Buffer (NEB: B7204S)</p><p>NEBuffer™ 4 (NEB: B7004S)</p><p>ATP (Thermo Scientific: R0441)</p><p>ddNTP Set, 5 mM solutions (ddCTP, ddGTP, ddTTP) (GE Healthcare: 27204501)</p><p>Klenow Fragment (3'→5' exo-)&nbsp;(M0212L)</p><p>Nuclease-free water</p><p>Agencourt Ampure XP beads (Beckman Coulter: A63882)</p><p>75% Ethanol</p><p>KAPA library quantification kit (KK4835)</p><p>100 µM NanoqPCR1 primer (HPLC: 5’ACACTCTTTCCCTACACGAC3’)</p><p>100 µM NanoqPCR2 primer (HPLC: 5’GTGACTGGAGTTCAGACGTG3’)</p><p>UDI containing PCR primers (dried):</p><p>i5: AATGATACGGCGACCACCGAGATCTACAC[barcode]ACACTCTTTCCCTACACGACGCTCTTCCGATC*T</p><p>i7:&nbsp;CAAGCAGAAGACGGCATACGAGAT[barcode]GTGACTGGAGTTCAGACGTGTGCTCTTCCGATC*T</p><p>*phosphorothioate bond</p><p>AccuClear Ultra High Sensitivity dsDNA Quantification kit (Biotium: 31028)</p><h2>Equipment</h2><p>Full set of pipettes and tips</p><p>Plate magnet</p><p>Eppendorf twin.tec plates and plate seals</p><p>384 well plate and optical seal</p><p>Thermocycler</p><p>Lightcycler e.g.&nbsp;Roche 480 Lightcycler</p><p>Method of DNA quantification e.g. bioanalyzer/plate reader</p><p>Sequencer e.g. Nova-seq</p><h2>Procedure</h2><p><strong>Library Preparation</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Prepare DNA samples and make each sample up to 20 µL in a 96 well twin.tec Eppendorf plate (e.g. extracted genomic DNA).</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Perform an Ampure bead clean-up: per sample, mix 50 µL NFW with 50 µL Ampure beads and add 100 µL of this 50:50 mix to each 20 µL sample. Mix well by pipetting up and down and allow DNA to bind to beads, wash twice with 75% EtOH and re-suspend beads in 20 µL NFW.&nbsp;</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Prepare a fragmentation mix:</p><p><br></p><p>10X CutSmart®&nbsp;Buffer&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2.5&nbsp;µL</p><p>NFW&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2 µL</p><p>HpyCH4V&nbsp;(5U/uL)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.5 µL</p><p><br></p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add the 5 µL fragmentation mix to the 20 µL bead suspension (this is an on-bead digestion).</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Incubate at 37&nbsp;ºC&nbsp;for 15 min on a thermocycler</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Perform an Ampure bead clean-up: add 62.5 µL Ampure XP beads to each 25 µL sample. Mix well by pipetting up and down and allow DNA to bind to beads, wash twice with 75% EtOH and elute in 15 µL NFW.&nbsp;</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Make up a 1 mM&nbsp;dATP/ddBTP mix (combine 2.5&nbsp;µL&nbsp;100 mM dATP, 50&nbsp;µL&nbsp;5 mM ddCTP, 50&nbsp;µL&nbsp;5 mM ddTTP, 50&nbsp;µL&nbsp;5 mM ddGTP and 97.5&nbsp;µL&nbsp;NFW.</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Prepare A-tailing mix:&nbsp;</p><p><br></p><p>10X NEBuffer™ 4&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1.5 µL</p><p>NFW&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1.85 µL</p><p>1&nbsp;mM dATP/ddBTP mix&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1.5 µL</p><p>Klenow Fragment (3'→5' exo-)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.15 µL</p><p><br></p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add 5 µL A-tailing mix to 10 µL of the cleaned-up fragmentation product.</p><p>10.&nbsp;&nbsp;Incubate at 37 ºC for 30 min&nbsp;on a thermocycler</p><p>11.&nbsp;&nbsp;Prepare ligation mix:&nbsp;</p><p><br></p><p>10X NEBuffer™ 4&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2.24 µL</p><p>NFW&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;15.53 µL</p><p>10 mM ATP&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3.74 µL</p><p>15 µM xGen Duplex Seq Adapters&nbsp;&nbsp;&nbsp;0.33 µL</p><p>400 U/µL T4 DNA ligase&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.56 µL</p><p><br></p><p>12.&nbsp;&nbsp;Add 22.4 µL ligation mix to the 15 µL A-tail product</p><p>13.&nbsp;&nbsp;Incubate at&nbsp;20 ºC for 20&nbsp;min on a thermocycler</p><p>14.&nbsp;&nbsp;Perform an Ampure bead clean-up: add 37.4 µL Ampure XP beads to each 37.4 µL sample. Mix well by pipetting up and down and allow DNA to bind to beads, wash twice with 75% EtOH and elute in 50 µL NFW.&nbsp;</p><p><br></p><p><strong>DNA Quantification by qPCR</strong></p><p>15.&nbsp;&nbsp;Take&nbsp;a KAPA library quantification kit (KK4835). Add the supplied primer premix to the supplied KAPA SYBR FAST master mix. In addition add 20&nbsp;µL&nbsp;of 100&nbsp;µM NanoqPCR1 primer (HPLC: 5’ACACTCTTTCCCTACACGAC3’) and 20&nbsp;µL&nbsp;of 100&nbsp;µM NanoqPCR2 primer (HPLC: 5’GTGACTGGAGTTCAGACGTG3’) to the KAPA SYBR FAST master mix.</p><p>16.&nbsp;&nbsp;Dilute a fraction of each sample 1 in 500 using NFW and set up triplicate 10 µL qPCR reactions (6&nbsp;µL master mix, 2&nbsp;µL&nbsp;sample/standard, 2&nbsp;µL&nbsp;water) in a 384 well plate.</p><p>17.&nbsp;&nbsp;Run samples on a qPCR machine e.g.&nbsp;Roche 480 Lightcycler</p><p>18.&nbsp;&nbsp;Perform analysis:&nbsp;Absolute quantification (2<sup>nd</sup>&nbsp;Derivative Maximum Method) with the high sensitivity algorithm).</p><p>19.&nbsp;&nbsp;Determine the nM (fmol/ µL) concentration per sample as follows:&nbsp;Mean of sample concentration x dilution factor (500) x 452/573/1000 (where 452 is the size of the standard in bp and 573 is the proxy for the average fragment length of the library in bp). We multiply this value by 1.5 to correct for the performance between different thermocyclers within the laboratory.&nbsp;</p><p>20.&nbsp;&nbsp;Dilute your sample to your desired fmol input amount in 25 µL using NFW to achieve duplicate rates close to the optimal 81% duplicate rate.&nbsp;For instance, if working with 150 bp paired reads, take 0.3 fmol per 15x sequencing (the equivalent of 150 million read pairs). For 30x, take 0.6 fmol, and so on. We estimate that, under optimal dilution and after applying all the bioinformatic mapping quality filters, a 26x standard coverage (260 million read pairs) will result in 1x effective duplex coverage (3 billion duplex calls).</p><p><br></p><p><strong>Sequencing of matched normals</strong></p><p>Calling somatic mutations require accurate filtration of germline SNPs, which is achieved by looking at a matched normal. A cost-efficient approach to obtain a matched normal for a NanoSeq sample is to sequence an undiluted NanoSeq library from the same individual. In some cases, when there is no clonality, the undiluted DNA can be obtained form the very same library that is going to be diluted. Sequencing NanoSeq libraries as matched normals concentrates the coverage on the 30% of the genome covered by NanoSeq. We found that at least 2 fmol and the equivalent to 8x whole-genome sequencing allow obtaining a matched normal with enough depth of coverage.</p><p><br></p><p><strong>PCR amplification</strong></p><p>21.&nbsp;&nbsp;Add&nbsp;25 µL NEBNext Ultra II Q5 Master Mix to UDI containing PCR primers (*we purchase PCR plates that contain 0.1 nmol of dry i5 primer and 0.1 nmol of dry i7 primer in each well). Add 25&nbsp;µL&nbsp;sample, mix and thermocycle:</p><p><br></p><p>Step 1: 98 ºC 30 seconds</p><p>Step 2: 98 ºC 10 seconds</p><p>Step 3: 65 ºC 75 seconds</p><p>Step 4: Return to Step 2, X times</p><p>Step 5: 65 ºC for 5 min</p><p>Step 6: Hold at 4 ºC.</p><p><br></p><p><br></p><p>*We use dry primers. When using re-suspended primers adjust your sample volume to allow for the addition of primers.&nbsp;</p><p><br></p><p>Note, in order to correct between different thermocyclers in our laboratory we have a 1.5X correction factor in our calculations. Thus our 0.3 fmol input translates to 0.2 fmol input if the correction factor is not applied.&nbsp;</p><p>The number of PCR cycles is dependent upon the input: 0.1 fmol (0.066 fmol) = 16 cycles, 0.3 fmol (0.2 fmol) = 14 cycles, 0.6 fmol (0.4 fmol) = 13 cycles, 5 fmol (3.33 fmol) = 10 cycles.&nbsp;</p><p><br></p><p>22.&nbsp;&nbsp;Perform two consecutive 0.7X Ampure bead clean-ups and elute in 50&nbsp;µL&nbsp;NFW.&nbsp;</p><p>23.&nbsp;&nbsp;Quantify samples e.g. using&nbsp;AccuClear Ultra High Sensitivity dsDNA Quantification kit (Biotium) or bioanalyzer.&nbsp;</p><p>24.&nbsp;&nbsp;Pool samples together for sequencing.&nbsp;</p><p>25.&nbsp;&nbsp;Run pool on bioanalyser and adjust the pool concentration.&nbsp;</p><p>26.&nbsp;&nbsp;Sequence on preferred illumina sequencer (e.g.&nbsp;Nova-Seq using 150PE read lengths).</p><p>27.&nbsp;&nbsp;Sequencing depth is determined by the fmol input into PCR e.g. 0.3 fmol input requires 15X whole genome coverage and 0.6 fmol input requires 30X whole genome coverage.&nbsp;</p><p><br></p><p><br></p><p><br></p></body></html>