<html><head><title>Build a Bioinformatics Analysis Platform and Apply it to Routine Analysis of Microbial Genomics and Comparative Genomics</title></head><body><h1>Build a Bioinformatics Analysis Platform and Apply it to Routine Analysis of Microbial Genomics and Comparative Genomics</h1>Authors: Hualin Liu, Bingyue Xin, Jinshui Zheng, Hao Zhong, Yun Yu, Donghai Peng, Ming Sun<br>Abstract: <p>Genomics and comparative genomics have been increasingly used as routine methods for general microbiological researches. However, it is usually necessary to call several tools or even write some scripts to complete some simple analysis, which is complicated for most biological researchers. To simplify the operation process, especially for the convenience of microbiologists in the analysis, here we have developed PGCGAP, a comprehensive, malleable and easily-installed prokaryotic genomics and comparative genomics analysis pipeline, which implements genome assembly, gene prediction and annotation, average nucleotide identity (ANI) calculation, phylogenetic analysis, COG annotation, pan-genome analysis, inference of orthologous gene groups, variants calling and annotation and screening for antimicrobial and virulence genes. Although we have tried our best to simplify the installation and usage of PGCGAP, it may be difficult for non-bioinformatician users to master it. So, a protocol was created to help microbiologists without any experience in bioinformatics to establish their own bioinformatics platform and perform routine analysis. This protocol shows how to choose equipment, to install a Linux subsystem on a laptop with windows 10 system, to install PGCGAP and perform all analysis with an example dataset. The protocol requires a basic understanding of Linux, so an additional web page was written to help uninitiated users learn Linux and whole-genome sequencing (<a href="http://bcam.hzau.edu.cn/linuxwgs.php" rel="noopener noreferrer" target="_blank">http://bcam.hzau.edu.cn/linuxwgs.php</a>).</p><br>Keywords: comparative genomics, COG annotation, phylogenetic orthology, phylogenetic analysis, variants calling, pan-genome, genome assembly, gene prediction, genome annotation, genome distance, Average Nucleotide Identity, PGCGAP<br><h2>Introduction</h2><p>Genome sequencing has become a routine method for common microbiological studies as continuously decrease in the cost of genome sequencing. Various tools have been developed for genome analysis. But for general users, it takes time to install and learn to use various programs and prepare the related input files. Even for some simple purposes, users need to spend much effort to integrate several tools or even write some scripts. For example, when we need a core-genome-SNP based phylogenetic analysis for the isolates from same species, we should successively use the following tools, Bowtie2<sup>1</sup> or BWA<sup>2</sup> for reads mapping, Samtools<sup>3</sup> or GATK<sup>4</sup> for SNP calling, and FastTree<sup>5 </sup>or RAxML<sup>6</sup> for phylogenetic tree construction. Therefore, a comprehensive, flexible and efficient pipeline for general analysis is urgently needed. We developed a prokaryotic genomics and comparative genomics analysis pipeline named PGCGAP to coordinate several genomic analysis software packages and in-house scripts to meet the various needs of microbiologists.</p><p><strong>Development of the protocol</strong></p><p>PGCGAP was developed to facilitate the work of genomics and comparative genomics analysis of microbes.<strong> </strong>Considering the important role of basic bioinformatics in microbial research and most microbiologists lacking analysis skills, this protocol describes in detail the installation of Linux systems and demonstrates the software installation methods. Finally, we demonstrated all the usages of PGCGAP step by step through the example datasets.</p><p><strong>Applications of the protocol</strong></p><p>PGCGAP can be used for (i) genome assembly, (ii) gene prediction and annotation, (iii) genome distance estimation, (iv) phylogenetic analysis, (v) COG annotation, (vi) pan-genome analysis, (vii) inference of orthologous gene groups, (viii) variants calling and annotation and (ix) screening for antimicrobial and virulence genes. It is worth noting that although the entire pipeline was developed for prokaryotes, some of the modules such as “Assemble”, “MASH”, “OrthoF”, “CoreTree” and “AntiRes” can also be used for the analysis of eukaryotic genomes. In addition, “VAR” is applicable to the analysis of any haploid genome.</p><p><strong>Advantages and limitations of this pipeline</strong></p><p>PGCGAP is versatile, feature-rich, easy to install and use, and friendly to microbiologists and bioinformatics beginners. New features will continue to be added. But a Graphical User Interface (GUI) has not been developed.</p><p><strong> Expertise required to implement the protocol</strong></p><p>Users need to be skilled in using computers, and it will be easier to master this protocol if they have some Linux skills. A webpage introducing the basics of Linux, usage of common commands, software installation, and the whole-genome sequencing technology was developed to help users get started with bioinformatics. Please visit <a href="https://github.com/liaochenlanruo/pgcgap/wiki/Learning-bioinformatics" rel="noopener noreferrer" target="_blank">https://github.com/liaochenlanruo/pgcgap/wiki/Learning-bioinformatics</a> or <a href="http://bcam.hzau.edu.cn/linuxwgs.php" rel="noopener noreferrer" target="_blank">http://bcam.hzau.edu.cn/linuxwgs.php</a> for more information.</p><p><strong>Overview of the procedure</strong></p><p>Ten frequently used prokaryotic genomics and comparative genomics analysis processes were integrated into PGCGAP as different modules. Modules can be used separately or in different combinations for various purposes (<strong>Figure 1</strong>). (i) “VAR” performs genome-wide variants calling by mapping methods. Firstly, paired-end reads were mapped to a reference genome by BWA<sup>2</sup> after filtered by Sickle<sup>7</sup>. Secondly, variants calling and annotation were performed by Freebayes<sup>8</sup> and snpEff<sup>9</sup>, respectively. Then, the whole genome SNP alignment and core SNP alignment were obtained by snippy-core<sup>10</sup>. Finally, Gubbins<sup>11</sup> was used to remove SNPs influenced by recombination events of the whole genome SNP alignment. (ii) “Assemble” performs genome assembly using ABySS<sup>12</sup> or Canu<sup>13</sup>. (iii) “Annotate” performs gene prediction and genome annotation by Prokka<sup>14</sup>. (iv) “ANI” computes Average Nucleotide Identity (ANI) between each genome pair by fastANI<sup>15</sup>. Three scripts “triangle2list.pl”, “get_ANImatrix.pl” and “Plot_ANIheatmap.R” have been developed here to generate the ANI matrix and plot the correlation matrix heat map (<strong>Supplementary Figure S1</strong>), respectively. (v) “MASH” estimates genome and metagenome distance and similarity using MinHash<sup>16</sup>, and a heat map of genome similarity will be generated by two scripts “get_Mash_Matrix.pl” and “Plot_MashHeatmap.R”. (vi) “Pan” calls Roary<sup>17</sup> calculate the pan-genome. Two scripts “fmplot.py” and “plot_3Dpie.R” were developed for result visualization (<strong>Supplementary Figure S2</strong>). A phylogenetic tree based on single-copy core proteins called by Roary<sup>17</sup> will be constructed (<strong>Supplementary Figure S3</strong>). (vii) COG (Clusters of Orthologous Group) annotation was conducted by module “pCOG”. Amino acid sequences of each genome were blasted against the COG database, and then all hits were mapped to the COG functional category by in-house scripts. R script “Plot_COG.R” was written for result visualization (<strong>Supplementary Figure S4</strong>). Comparison and visualization of COG functional categories among different genomes can be done by Perl script “get_flag_relative_abundances_table.pl” and R script “Plot_COG_Abundance.R” ( <strong>Supplementary Figure</strong> <strong>S5</strong>). (viii) “OrthoF” uses OrthoFinder<sup>18</sup> for phylogenetic orthology inference. Gene duplication events will be also predicted (<strong>Supplementary Figure S6</strong>). (ix) “CoreTree” was developed for genome-wide phylogenetic analysis based on the protein sequences or SNPs of single-copy core genes. Firstly, CD-HIT<sup>19</sup> was used to rapidly generate protein clusters, and then the protein sequences of single-copy core genes were extracted by Perl scripts and aligned by MAFFT<sup>20</sup>. Secondly, on one hand, alignments of protein sequences were concatenated, and the phylogenetic tree was constructed by FastTree<sup>5</sup> (<strong>Supplementary Figure S7</strong>). On the other hand, the protein sequence alignments were converted into corresponding codon alignments by PAL2NAL v14<sup>21</sup>. Then, the alignments were concatenated, and SNP-sites<sup>22</sup> was called to find SNP sites. Finally, FastTree<sup>5</sup> was used to construct the SNPs phylogenetic tree (<strong>Supplementary Figure S8</strong>).&nbsp;(x) “AntiRes” calls abricate<sup>23</sup> to screen for antimicrobial and virulence genes from contigs.</p><p><strong>Experimental design</strong></p><p><strong>Selection of reference genome format for variants calling</strong></p><p>The reference genome can be files in FASTA format and GenBank format. If a GenBank file rather than a FASTA file was supplied as the reference, annotation information of the variants will be generated to show to the user which feature was affected by the variants.</p><p><strong>Adjust the kmmer size for better assembly results</strong></p><p>According to our experience, when the length of Illumina reads is 150 bp and “--kmmer” was set from 81 to 89, better assembly quality can be obtained. It is better for the user to check the report file in the assembly result to ensure that the value of N50 is greater than 50,000 bp, otherwise, the user can try to change the value of “--kmmer” to improve the assembly quality.</p><p><strong>Choice of a module to construct the phylogenetic tree of single-copy core proteins</strong></p><p>Both “CoreTree” and “Pan” can be used to construct a phylogenetic tree of single-copy core proteins. That which module should be used depends on which type of input file the user has. “CoreTree” takes only the amino acid sequence files as inputs, while “Pan” needs both amino acid sequence files and Gff3 files.</p><p><strong>Choice of a module to calculate pairwise genome distance</strong></p><p>Both “ANI” and “MASH” can calculate pairwise genome distance. “MASH” is more suitable for dealing with thousands of genomes because of its faster running speed. In addition to nucleotide sequences and assembled genomes, “MASH” can also take amino acid sequences and raw sequencing reads as inputs and can be used to calculate distances between metagenomic samples.</p><h2>Reagents</h2><p>No reagents are required for this protocol.</p><h2>Equipment</h2><p>A laptop, desktop PC or server can be used to build a bioinformatics analysis platform, and the suggested hardware requirements are listed in <strong>Table 1</strong>. Slightly lower features are also allowed (CPU must have four logical processors, memory must be greater than 8 G), but the computing speed may decrease, and the capacity of the hard disk can be adjusted according to actual requirements.</p><h2>Procedure</h2><p><strong>Building a bioinformatics analysis platform on Windows 10</strong></p><p>Windows Subsystem for Linux (WSL) allows users to install Linux subsystems directly on Windows 10 system. It can easily run Linux commands and install Linux software, avoiding the installation of third-party virtual machine software. The advantage of WSL is that it makes better use of computer memory and does not require copying files between the host and the virtual machine.</p><p><strong>Configuration of WSL</strong></p><p>Timing ~1 min</p><p>System requirements: Windows 10 Version 1709, Build 16299 or above, 64-bit systems.</p><p>1.	Enable WSL: Open "Settings",&nbsp;click "Apps", then find and click "Programs and Features", click "Turn Windows features on or off", find "Windows Subsystem for Linux" and check the box, click OK and restart the computer (<strong>Supplementary Video 1</strong>).</p><p><strong>Install Linux</strong></p><p>Timing ~59 min</p><p>2.	Open the Microsoft Store, search Ubuntu, and choose to install Ubuntu 18.04 LTS. Follow the prompts to set up your username and password. Here we create an account with the username “bio” (<strong>Supplementary Video 2</strong>). After the installation is finished, we need to do some configuration on the system (<strong>Supplementary Video 3</strong>).</p><p>3.	Enter the following command in the terminal to update the source:</p><p>&nbsp;$sudo apt-get update</p><p>4.	Set the password for root.</p><p>$sudo root</p><p>5.	Change the content of ptrace_scope from 1 to 0.</p><p>$su root</p><p>#echo 0 &gt; /proc/sys/kernel/yama/ptrace_scope</p><p>6.	Switch to the ordinary user “bio”.</p><p>#su bio</p><p>7.	Installation of Miniconda</p><p>(A)	Installation of Miniconda on Linux</p><p>(i)	Here, <a href="https://docs.conda.io/en/latest/miniconda.html" rel="noopener noreferrer" target="_blank">Miniconda</a> will be installed, go to the official website, and select the installation file suitable for your system and python version. Here, Miniconda 3 will be installed (<strong>Supplementary Video 4</strong>).</p><p>$wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh</p><p>(ii)	Start installation</p><p>$bash Miniconda3-latest-Linux-x86_64.sh</p><p>Press Enter when prompted to view the license, enter “yes” and press “Enter” to continue. Press “Enter” to confirm the installation location. Miniconda was installed in the miniconda3 directory under the user’s home directory. Type “yes” and press “Enter” to initialize. Finally, run the “source ~/.bashrc” command in the terminal.</p><p>$source ~/.bashrc</p><p>(iii)	Set up Bioconda channel. Add the channels by entering the following three commands in the terminal.</p><p>$conda config --add channels defaults</p><p>$conda config --add channels bioconda</p><p>$conda config --add channels conda-forge</p><p>(B)	Install Miniconda on MacOS</p><p>(i)	Installation of Miniconda3</p><p>$wget https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh</p><p>$sh Miniconda3-latest-MacOSX-x86_64.sh</p><p>$source ~/.bash_profile</p><p>(ii)	Add channels of Bioconda</p><p>$conda config --add channels defaults</p><p>$conda config --add channels bioconda</p><p>$conda config --add channels conda-forge</p><p>Installation of PGCGAP (<strong>Supplementary Video 5</strong>).</p><p>Timing ~34 min</p><p>8.	Create a pgcgap environment for the installation of PGCGAP.</p><p>$conda create -n pgcgap python=3</p><p>9.	Activate the environment.</p><p>$conda activate pgcgap</p><p>10.	Installation of PGCGAP.</p><p>&nbsp;$conda install pgcgap</p><p>11.	Check if the dependent software packages were installed.</p><p>&nbsp;$pgcgap --check-external-programs</p><p>12.	Set up the COG database.</p><p>&nbsp;$pgcgap --setup-COGdb</p><p>13.	Exit the environment.</p><p>&nbsp;$conda deactivate</p><p><strong>Step by Step examples</strong></p><p>Timing ~2 d</p><p>14.&nbsp;Download the example dataset.</p><p>$wget http://bcam.hzau.edu.cn/PGCGAP/PGCGAP_Examples.rar</p><p>In this example, the working directory locates at the H drive. All hard disks in Windows were mounted in the “/mnt” directory of Ubuntu Linux. The “PGCGAP_Examples/Reads/Illumia” directory contains six Illumina Hiseq paired-end reads of <em>Escherichia coli</em>, the “PGCGAP_Examples/Reads/Oxford” directory contains the Oxford Nanopore reads of <em>Escherichia coli</em> K12, and the “PGCGAP_Examples/Reads/PacBio” directory contains the Pacific Biosciences released P6-C4 chemistry reads of <em>Escherichia coli</em> K12. “PGCGAP_Examples/Reads/MG1655” is the GenBank format file of <em>E.coli</em> K-12 <em>substr.</em> MG1655, used as the reference genome.</p><p>15.	Activate the pgcgap environment.</p><p>$conda activate pgcgap</p><p>16.	Example 1: Genome assembly with Illumina reads.</p><p>Paired-end reads of six strains in the directory “Reads/Illumina/” were used as inputs. In the dataset, the naming format of the genome is “strain_1.fastq.gz” and “strain_2.fastq.gz”. The string after the strain name is “_1.fastq.gz”, and its length is 11, so “--suffix_len” was set to 11.</p><p>$pgcgap --Assemble --platform illumina --ReadsPath Reads/Illumina --reads1 _1.fastq.gz --reads2 _2.fastq.gz --kmmer 81 --threads 4 --suffix_len 11</p><p>New directories and documents were generated after the program is finished. The assembly results for each genome are in the “Results/Assembles/Illumina” directory, and all scaffolds of the strains were stored in “Results/Assembles/Scaf/Illumina”. Users are advised to check the assembly stats file (such as Results/Assembles/Illumina/SRR9620252_assembly/SRR9620252-stats.tab) of each genome to ensure that the value of N50 is greater than 50,000 bp. The file “scaf.list” under the working directory contains the absolute path of all genomes.</p><p>17.	Example 2: Oxford reads assembly.</p><p>Oxford nanopore only produces one reads file (“Reads/Oxford/oxford.fasta”), so only the parameter of “--reads1” needs to be set, here the value is “.fasta”. “--genomeSize” is the estimated genome size, and users can check the genome size of similar strains in the NCBI database for reference. The parameter was set to “4.8m” here. The suffix of the reads file here is “.fasta” and its length is 6, so “--suffix_len” was set to 6.</p><p>$pgcgap --Assemble --platform oxford --ReadsPath Reads/Oxford --reads1 .fasta --genomeSize 4.8m --threads 4 --suffix_len 6</p><p>The results are stored in the “Results/Assembles/Oxford” directory and the “Results/Assembles/Scaf/Oxford” directory. The former contains all intermediate files and genome files, the latter contains only the assembled genome.</p><p>18.	Example 3: PacBio reads assembly.</p><p>PacBio also produces only one reads file too (“Reads/PacBio/pacbio.fastq”), the parameter settings are similar to Oxford. The strain name is “pacbio” with the suffix “.fastq” and the suffix length is 6, so “--suffix_len” was set to 6.</p><p>$pgcgap --Assemble --platform pacbio --ReadsPath Reads/PacBio --reads1 .fastq --genomeSize 4.8m --threads 4 --suffix_len 6</p><p>The results are stored in the “Results/Assembles/PacBio” directory and in the “Results/Assembles/Scaf/PacBio” directory. The former contains all intermediate files and genome files, the latter containing only the assembled genome.</p><p>19.	Example 4: Gene prediction and annotation.</p><p>Here, the assembly results of Illumina reads were taken as inputs (“Results/Assembles/Scaf/Illumina/*.fa”). The suffix of the genome is “-8.fa”. When running the program, the value of the “--Scaf_suffix” parameter cannot be quoted. Here, -8.fa should not be quoted.</p><p>$pgcgap --Annotate --scafPath Results/Assembles/Scaf/Illumina --Scaf_suffix -8.fa --genus Escherichia --species “Escherichia coli” --codon 11 --threads 4</p><p>The generated files are stored in the “Results/Annotations” directory, and files in the directories “Results/Annotations/AAs”, “Results/Annotations/CDs” and “Results/Annotations/GFF” will be used for subsequent analysis.</p><p>20.	Example 5: Constructing the single-copy core proteins tree and core SNPs tree.</p><p>The phylogenetic trees of single-copy core proteins and single-copy core genes SNPs will be constructed using the six <em>E. coli</em> genomes sequenced by Illumina as datasets. The input files are the amino acid sequence files (“Results/Annotations/AAs/*.faa”) and the nucleotide sequence files (“Results/Annotations/CDs/*.ffn”) obtained by the genome annotation. Amino acid files and nucleotide files must be suffixed with “.faa” and “.ffn”, respectively. The “.faa” and “.ffn” files of the same strain should have the same prefix name. The name of protein IDs and gene IDs in the Amino acids file and nucleotide file should be started with the strain name. The Prokka<sup>14</sup> software was suggested to generate the input files.</p><p>$pgcgap --CoreTree --CDsPath Results/Annotations/CDs --AAsPath Results/Annotations/AAs --codon 11 --strain_num 6 --threads 4</p><p>The result files are stored in the “Results/CoreTrees” directory. “ALL.core.protein.nwk” and “ALL.core.snp.nwk” are the phylogenetic tree files of the single-copy core proteins and the core SNPs. Users can import these two files into MEGA<sup>23</sup> or iTOL<sup>24</sup> to view the topology.</p><p>21.	Example 6: Constructing the single-copy core protein tree only.</p><p>If the “--CDsPath” was set to “NO”, the nucleotide files will not be needed, and the phylogenetic tree of core SNPs will not be constructed too.</p><p>$pgcgap --CoreTree --CDsPath NO --AAsPath Results/Annotations/AAs --codon 11 --strain_num 6 --threads 4</p><p>22.	Example 7: pan-genome analysis and phylogenetic tree constructing.</p><p>GFF3 files (With “.gff” as the suffix) of each strain placed into a directory (“Results/Annotations/GFF/*.gff”). They must contain the nucleotide sequence at the end of the file. Protein sequence files (one per species) in FASTA format under another directory were also needed (“Results/Annotations/AAs/*.faa”) if the parameter “--PanTree” was provided for constructing a phylogenetic tree. It should be noted that the “*.gff” file and the “*.faa” file must correspond. We strongly recommend using Prokka<sup>14</sup> to generate the files. If the “--Annotate” function was run first, the files will be generated automatically.</p><p>$pgcgap --Pan --codon 11 --strain_num 6 --threads 4 --GffPath Results/Annotations/GFF --PanTree --AAsPath Results/Annotations/AAs</p><p>The results are stored in the “Results/PanGenome” directory. A spreadsheet named “gene_presence_absence.csv” lists each gene and which sample is presented in. At the same time, some visual results (“*.pdf”) are also outputted. The file “Results/PanGenome/Core/Roary.core.protein.nwk” is the phylogenetic tree constructed based on the single-copy core proteins called by Roary software. If the parameters “--PanTree” and “--AAsPath” were not provided, the phylogenetic tree will not be constructed.</p><p>23.	Example 8: Inference of orthologous gene groups.</p><p>The input files are also the amino acid sequence files suffixed with “.faa” (“Results/Annotations/AAs/*.faa”).</p><p>$pgcgap --OrthoF --threads 4 --AAsPath Results/Annotations/AAs&nbsp;</p><p>The resulting files are placed in the “Results/OrthoFinder/Results_orthoF” directory.</p><p>24.	Example 9: Compute whole-genome Average Nucleotide Identity.</p><p>The input file named “scaf.list” contains the absolute path of each genome, one per line. If the “--Assemble” function was run first, the list file will be generated automatically. The value of the parameter “--Scaf_suffix” depends on the actual situation, here is “-8.fa”.</p><p>$pgcgap --ANI --threads 4 --queryL scaf.list --refL scaf.list --ANIO Results/ANI/ANIs --Scaf_suffix -8.fa</p><p>The results are stored in the “Results/ANI” directory. The file “ANI” contains comparison information of genome pairs. The document is composed of five columns, each of which represents ANI value, count of bidirectional fragment mappings and total query fragments, respectively. A heat map file “ANI_matrix.pdf” was generated.</p><p>25.	Example 10: Genome and metagenome similarity estimation using MinHash</p><p>It takes genome files (complete or draft) in a directory as inputs (Default: Results/Assembles/Scaf/Illumina).</p><p>$pgcgap --MASH --scafPath Results/Assembles/Scaf/Illumina --Scaf_suffix -8.fa</p><p>The results are stored in the “Results/MASH” directory. The file “MASH” shows pairwise distance between pair genomes and each column represents Reference-ID, Query-ID, Mash-distance, P-value and Matching-hashes, respectively. A heat map file “MASH_matrix.pdf” was generated.</p><p>26.	Example 11: COG annotation.</p><p>The input files are also the amino acid sequence files suffixed with “.faa” (“Results/Annotations/AAs/*.faa”).</p><p>$pgcgap --pCOG --threads 4 --strain_num 6 --AAsPath Results/Annotations/AAs</p><p>The results are stored in the “Results/COG” directory. The super COG table of each strain (“*.2Scog.table”) and its plot (“*.2Scog.table.pdf”) will be generated. “All_flags_relative_abundances.table” is a table containing the relative abundance of each flag for all strains, and “All_flags_relative_abundances.pdf” is the corresponding visualization result.</p><p>27.	Example 12: Variants calling, and phylogenetic tree construction based on a reference genome.</p><p>The six genomes sequenced by Illumina were chosen as datasets (“Reads/Illumina/*.gz”). Taking <em>Escherichia coli</em> K-12 <em>substr</em>. MG1655 as the reference genome and the reference file “MG1655.gbff” in the GenBank format is stored in the “Reads” directory. The absolute path of the reference genome (here is “/mnt/h/PGCGAP_Examples/Reads/MG1655.gbff”) is required to run the program.</p><p>$pgcgap --VAR --threads 4 --refgbk /mnt/h/PGCGAP_Examples/Reads/MG1655.gbff --ReadsPath Reads/Illumina --reads1 _1.fastq.gz --reads2 _2.fastq.gz --suffix_len 11 --strain_num 6 --qualtype sanger</p><p>The resulting files are stored in the “Results/Variants” directory, where the “Core” directory contains the core SNPs of all strains and their phylogenetic tree.</p><p>28.&nbsp;Example 13: Screening of contigs for antimicrobial and virulence genes</p><p>It takes genome files (complete or draft) in a directory as inputs (Default: Results/Assembles/Scaf/Illumina).</p><p>$pgcgap --AntiRes --scafPath Results/Assembles/Scaf/Illumina --Scaf_suffix -8.fa --threads 4 --db ncbi --identity 75 --coverage 50</p><p>The resulting files are stored in the “Results/AntiRes” directory. “*.tab” files are screening results of each strain, and the “summary.txt” file contains a matrix of gene presence/absence for all strains.</p><p>29.	Example 14: Perform all functions for paired-end reads.</p><p>Only the reads file and reference file should be provided. For the sake of flexibility, the "VAR" function needs to be added extra.</p><p>$pgcgap --All --platform illumina --ReadsPath Reads/Illumina --reads1 _1.fastq.gz --reads2 _2.fastq.gz --suffix_len 11 --kmmer 81 --genus Escherichia --species “Escherichia coli” --codon 11 --strain_num 6 --threads 4 --VAR --refgbk /mnt/h/PGCGAP_Examples/Reads/MG1655.gbff --qualtype sanger --PanTree</p><p>&nbsp;</p><h2>Troubleshooting</h2><p>Troubleshooting advice can be found in <strong>Table 2</strong>.</p><h2>Time Taken</h2><p>The following marked time was tested in the WSL on the laptop, the features of the laptop are as follows: i7-4710MQ CPU (with 4 cores and 8 logical processors), 16 GB DDR3L RAM, 240 G SSD, 1 T HDD. All commands called 4 threads.</p><p>Step 1, configuration of WSL, 1 min.</p><p>Step 2, installation of Linux, 43 min.</p><p>Step 3-6, the configuration of Linux, 10 min.</p><p>Step 7, installation of Miniconda, 6 min.</p><p>Step 8-13, installation of PGCGAP, 34 min.</p><p>Step 14, download the example dataset, 11 min.</p><p>Step 15, activate the pgcgap environment, 8 s.</p><p>Step 16, Illumina reads assembly, 43 min.</p><p>Step 17, Oxford reads assembly, 1.6 h.</p><p>Step 18, PacBio reads assembly, 54 min.</p><p>Step 19, gene prediction and annotation, 1 h.</p><p>Step 20, constructing the single-copy core protein tree and core SNPs tree, 3.3 h.</p><p>Step 21, constructing the single-copy core protein tree only, 3 h.</p><p>Step 22, pan-genome analysis and phylogenetic tree constructing, 3.5 h.</p><p>Step 23, inference of orthologous gene groups, 51 min.</p><p>Step 24, compute whole-genome Average Nucleotide Identity, 17 s.</p><p>Step 25, genome similarity estimation using MinHash, 1 min.</p><p>Step 26, COG annotation, 20.3 h.</p><p>Step 27, variant calling, and phylogenetic tree construction based on reference genome, 11.6 h.</p><p>Step 28, Screening of contigs for antimicrobial and virulence genes, 30s.</p><p>Step29, Perform all functions for paired-end reads, 1.9 d.</p><h2>References</h2><p>1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Langmead, B. &amp; Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. <em>Nat. Methods</em> <strong>9</strong>, 357 (2012).</p><p>2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. <em>arXiv:1303.3997v2 [q-bio.GN]</em> (2013).</p><p>3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Heng Li<em> et al.</em> The Sequence Alignment/Map format and SAMtools. <em>Bioinformatics (Oxford, England)</em> <strong>25</strong>, 2078-2079 (2009).</p><p>4&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;McKenna, A.<em> et al.</em> The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. <em>Genome Res.</em> <strong>20</strong>, 1297-1303 (2010).</p><p>5&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Price, M. N., Dehal, P. S. &amp; Arkin, A. P. FastTree 2--approximately maximum-likelihood trees for large alignments. <em>PLoS One</em> <strong>5</strong>, e9490 (2010).</p><p>6&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. <em>Bioinformatics</em> <strong>30</strong>, 1312-1313 (2014).</p><p>7&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Joshi, NA &amp; Fass, JN. Sickle: A sliding-window, adaptive, quality-based trimming tool for FastQ files (Version 1.33) [Software]. Available at <a href="https://github.com/najoshi/sickle" rel="noopener noreferrer" target="_blank">https://github.com/najoshi/sickle</a>. (2011).</p><p>8&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Garrison, E &amp; Marth, G. Haplotype-based variant detection from short-read sequencing. <em>arXiv preprint arXiv:1207.3907 [q-bio.GN]</em> (2012).</p><p>9&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Cingolani, P.<em> et al.</em> A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3. <em>Fly (Austin)</em> <strong>6</strong>, 80-92 (2012).</p><p>10&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Seemann, T. Snippy: Rapid haploid variant calling and core genome alignment. Available at <a href="https://github.com/tseemann/snippy" rel="noopener noreferrer" target="_blank">https://github.com/tseemann/snippy</a>. (2014).</p><p>11&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Croucher, N. J.<em> et al.</em> Rapid phylogenetic analysis of large samples of recombinant bacterial whole genome sequences using Gubbins. <em>Nucleic Acids Res.</em> <strong>43</strong>, e15 (2015).</p><p>12&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Jackman, S. D.<em> et al.</em> ABySS 2.0: resource-efficient assembly of large genomes using a Bloom filter. <em>Genome Res.</em> <strong>27</strong>, 768-777 (2017).</p><p>13&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Koren, S.<em> et al.</em> Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation. <em>Genome Res.</em> <strong>27</strong>, 722-736 (2017).</p><p>14&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Seemann, T. Prokka: rapid prokaryotic genome annotation. <em>Bioinformatics</em> <strong>30</strong>, 2068-2069 (2014).</p><p>15&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Jain, C., Rodriguez-R, L. M., Phillippy, A. M., Konstantinidis, K. T. &amp; Aluru, S. High throughput ANI analysis of 90K prokaryotic genomes reveals clear species boundaries. <em>Nat Commun</em> <strong>9</strong>, 5114-5114 (2018).</p><p>16&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Ondov, B. D.<em> et al.</em> Mash: fast genome and metagenome distance estimation using MinHash. <em>Genome Biology</em> <strong>17</strong>, 132 (2016).</p><p>17&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Page, A. J.<em> et al.</em> Roary: rapid large-scale prokaryote pan genome analysis. <em>Bioinformatics</em> <strong>31</strong>, 3691-3693 (2015).</p><p>18&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Emms, D. M. &amp; Kelly, S. OrthoFinder: phylogenetic orthology inference for comparative genomics. <em>Genome Biology</em> <strong>20</strong>, 238 (2019).</p><p>19&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Li, W. &amp; Godzik, A. Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences. <em>Bioinformatics</em> <strong>22</strong>, 1658-1659 (2006).</p><p>20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Katoh, K., Misawa, K., Kuma, K. &amp; Miyata, T. MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform. <em>Nucleic Acids Res.</em> <strong>30</strong>, 3059-3066 (2002).</p><p>21&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Suyama, M., Torrents, D. &amp; Bork, P. PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments. <em>Nucleic Acids Res.</em> <strong>34</strong>, W609-612 (2006).</p><p>22&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Page, A. J.<em> et al.</em> SNP-sites: rapid efficient extraction of SNPs from multi-FASTA alignments. <em>Microb Genom</em> <strong>2</strong>, e000056 (2016).</p><p>23&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Seemann T. Abricate, Github <a href="https://github.com/tseemann/abricate" rel="noopener noreferrer" target="_blank">https://github.com/tseemann/abricate</a>.</p><p>24&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Kumar, S., Stecher, G., Li, M., Knyaz, C. &amp; Tamura, K. MEGA X: Molecular Evolutionary Genetics Analysis across Computing Platforms. <em>Mol. Biol. Evol.</em> <strong>35</strong>, 1547-1549 (2018).</p><p>25&nbsp;&nbsp;Letunic, I. &amp; Bork, P. Interactive tree of life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees. <em>Nucleic Acids Res.</em> <strong>44</strong>, W242-W245 (2016).</p><h2>Acknowledgements</h2><p>This work was made possible through funding from the National Key R&amp;D Program of China (2017YFD0201201), National Natural Science Foundation of China (31670085, 31970003 and 31770003), and China 948 Program of Ministry of Agriculture (2016-X21).</p></body></html>