<html><head><title>Explore study: Retrospective Cohort study to show the start of SARS-CoV-2 in patients with severe respiratory illness by real-time PCR</title></head><body><h1>Explore study: Retrospective Cohort study to show the start of SARS-CoV-2 in patients with severe respiratory illness by real-time PCR</h1>Authors: Sara H.A.Agwa, Hesham Elghazaly, Sarah El-Nakeep, Ahmad Moustafa, Manal H El-Sayed, Hala Hafez, Samia abdo, Marwa Matboli, Maha Saad, Shaimaa M.Elsayed, Aya M.Abd Elsamie, Reham M.Darwish, Hoda ezz elarab, Fatma S. E. Ebeid, Sara Makkeyah, Mahmoud S El Meteini<br>Abstract: <p>Aim: to know the start of COVID-19 in Ain shams University hospitals (El-Demerdash) in Severe acute respiratory illness.</p><p>Materials and methods: retrospective collection of data records and previously collected nasopharyngeal and oropharyngeal sample stored in our lab for determining the start of COVID-19 in our hospitals, using RT-PCR and postive samples we will conduct Sequencing analysis and photogenic analysis. Also epidemiological and data study of the negatively screened patients, to determine the most common causes of SARI and the causes of mortality in this category of patients</p><p>&nbsp;</p><p><br></p><br>Keywords: COVID-19, EGYPT, SARI<br><h2>Introduction</h2><p>&nbsp;The 2019–20 coronavirus pandemic is an ongoing pandemic of coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). [1] The outbreak was first noted in Wuhan, Hubei province, China, in December 2019. The World Health Organization (WHO) declared the outbreak to be a Public Health Emergency of International Concern on 30 January 2020 and recognized it as a pandemic on 11 March 2020.[2]As of 7 April 2020, more than 1.4 million cases of COVID-19 have been reported in over 200 countries and territories, [3] resulting in approximately 80,700 deaths.[4] More than 297,000 people have recovered. [5]&nbsp;</p><p>&nbsp;Medical Ain Shams Research Institute (MASRI) is conducting a surveillance project internationally with other Eastern Mediterranean region countries, we have gathered about 300 samples (nasopharangeal&amp; oropharengeal), these samples underwent molecular Lab evaluation of COVID-19, these samples are very well preserved, with fully collected epidemiological and lab data. Due to the emerging situation of the COVID-19 pandemic we find it mandatory to retest those samples collected since November 2019 for COVID-19 to assess if this COVID-19 was present earlier than we thought and if so we can relate its presence to the epidemiological data we had obtained earlier, moreover if COVID -19 is detected we can move step forward for next generation sequencing of the positive samples to define the most dominant strain found in Egyptians and further compare our strain or strains to other countries.</p><h2>Reagents</h2><p>Using commercially available kits supplied by thermofisher for RT-realtime test assays -&nbsp;&nbsp;VIRAL RNA ISOLATION</p><h2>Equipment</h2><p>Ion Torrent targeted next-generation sequencing (NGS)</p><h2>Procedure</h2><p><strong>1- Previously collected data and samples: </strong></p><p>Oropharyngeal and nasopharyngeal swabs (NP) for hospitalized SARI patients:</p><p>Data and samples of oropharyngeal and nasopharyngeal swabs for laboratory testing obtained previously from all hospitalized patients who meet the WHO standard SARI case definition.</p><p><em>Oropharyngeal swabbing:&nbsp;</em></p><p>A dry sterile tip flocked with nylon fiber swab applicator should be used to swab the posterior pharynx. The swab should be placed in a 15 ML centrifuge tube labeled with the patient unique ID and containing 2mL viral transport media (VTM). The applicator stick is then cut off.</p><p><em>Nasopharyngeal swabbing:&nbsp;</em></p><p>A flexible, sterile tip flocked with nylon fiber swab applicator is inserted into the nostril and back to the nasopharynx and left in place for a few seconds. It is then slowly withdrawn with a rotating motion.&nbsp;For SARI patients, the swab is put into the same centrifuge tube as the oropharyngeal swab labeled with the patient’s unique ID and the shaft is cut.&nbsp;The 15 ml tube containing the swab/s should be carefully transported to the hospital laboratory as soon as possible in cold chain.</p><p><strong>2- Sample processing:</strong></p><p>The received swabs inside the 15 ml tube should be agitated vigorously for 10 seconds using a vortex mixer to free cells from the swab tip, and then both swabs should be removed from the tube and discarded using a forceps. The VTM was split into 2 pre-labeled, sterile cryovials with the correct patient ID.&nbsp;One cryovial was immediately placed in a freezer (-70°C), while the other cryovial will be used the molecular testing by real time PCR at MASRI&nbsp;molecular laboratory.</p><p>&nbsp;</p><p><strong>3- Sample storage:</strong></p><p>The second aliquot of the samples will be stored in freezers (-70°C) at MASRI MOLECULAR laboratory for External Quality Assurance (EQA) by on site testing through random selection of samples with different results and retest them by other NAMRU-3 laboratory expertise in MASRI molecular labs ; the samples can also be used for further testing by NGS or virus isolation in MASRI molecular&nbsp;laboratories.</p><p>&nbsp;</p><p><strong>&nbsp;4- Sample Transfer: </strong></p><p>All samples will be kept in MEDICAL AIN SHAMS RESEARCH INSTITUTE&nbsp;laboratories and will not be transferred to other laboratories.</p><p>&nbsp;</p><p><strong>5- Explore study testing for COVID-19 using ready to use RT-realtime test assays -&nbsp;&nbsp;VIRAL RNA ISOLATION: </strong>Using commercially available kits supplied by thermofisher or other recommended companies. Work will be done in Biosafety level 2 cabinets under complete sterile conditions. -&nbsp;</p><p>&nbsp;COVID-19 DETECTION : Using realtime reverse transcription polymerase chain reaction RT-PCR } the test can be done on respiratory samples obtained by various methods, including nasopharyngeal swab or sputum sample. Results are generally available within a few hours. Molecular methods leverage polymerase chain reaction (PCR) along with nucleic acid tests, and other advanced analytical techniques, to detect the genetic material of the virus using real-time reverse transcription polymerase chain reaction for diagnostic purposes. Commercially available kits supplied by hvd , thermofisher and others.</p><p>&nbsp;</p><p><strong>6- Genome sequencing (NGS) for possible positive covid-19 samples&nbsp;Ion Torrent targeted next-generation </strong>sequencing (NGS) enables a streamlined research workflow for complete genome sequencing and epidemiological studies of SARS-CoV-2 (the coronavirus responsible for COVID-19). Through the use of a set of highly specific, universal coronavirus primers in combination with a high-fidelity master mix, all genomic segments are amplified and the DNA amplicons are sequenced on any Ion Torrent system to deliver highly accurate coronavirus typing in under a day. The Ion ArnpliSeq SARS-CoV-2 Research Panel consists of 2 pools with arnplicons ranging from 125275 bp iri length for complete viral genome sequencing.</p><p><br></p><p><strong>7- Epidemiological and data analysis</strong> of the negative COVID-19 population to determine the causes of morbidity and mortality related to the clinical data available through statistical analysis.</p><p>&nbsp;</p><h2>Time Taken</h2><p>three months to do PCR on stored samples in patches and sequencing if needed.</p><h2>References</h2><p>1- Klink, T., Rankin, D. A., Piya, B., Spieker, A. J., Faouri, S., Shehabi, A., Williams, J. V., Khuri-Bulos, N., &amp; Halasa, N. B. (2020). Evaluating the diagnostic accuracy of the WHO Severe Acute Respiratory Infection (SARI) criteria in Middle Eastern children under two years over three respiratory seasons. <em>PloS one</em>, <em>15</em>(4), e0232188. https://doi.org/10.1371/journal.pone.0232188</p><p>2- Al-Abdallat, M., Dawson, P., Haddadin, A. J., El-Shoubary, W., Dueger, E., Al-Sanouri, T., Said, M. M., &amp; Talaat, M. (2016). Influenza hospitalization epidemiology from a severe acute respiratory infection surveillance system in Jordan, January 2008-February 2014. <em>Influenza and other respiratory viruses</em>, <em>10</em>(2), 91–97. https://doi.org/10.1111/irv.12354</p><p>3- de Groot, R. J., Baker, S. C., Baric, R. S., Brown, C. S., Drosten, C., Enjuanes, L., Fouchier, R. A., Galiano, M., Gorbalenya, A. E., Memish, Z. A., Perlman, S., Poon, L. L., Snijder, E. J., Stephens, G. M., Woo, P. C., Zaki, A. M., Zambon, M., &amp; Ziebuhr, J. (2013). Middle East respiratory syndrome coronavirus (MERS-CoV): announcement of the Coronavirus Study Group. <em>Journal of virology</em>, <em>87</em>(14), 7790–7792. https://doi.org/10.1128/JVI.01244-13</p><p>4- WHO updates. https://www.who.int/emergencies/diseases/novel-coronavirus-2019</p><p>5- WHO. WHO Technical guidance on surveillance for MERS-CoV, https://www.who.int/influenza/surveillance_monitoring/global_surveillance/en/</p></body></html>