<html><head><title>A step-by-step protocol for performing LIVE-PAINT super-resolution imaging of proteins in live cells using reversible peptide-protein interactions</title></head><body><h1>A step-by-step protocol for performing LIVE-PAINT super-resolution imaging of proteins in live cells using reversible peptide-protein interactions</h1>Authors: Curran Oi, Zoe Gidden, Louise Holyoake, Owen Kantelberg, Simon Mochrie, Mathew H. Horrocks, Lynne Regan<br>Abstract: <p>Super-resolution imaging of proteins inside live cells is a powerful tool for investigating protein behavior. We have developed a super-resolution method we call LIVE-PAINT, which uses reversible peptide-protein interactions to achieve super-resolution inside live cells. This method is particularly useful for studying proteins which do not tolerate large genetic fusions, such as direct fusion to a fluorescent protein. Here, we provide a detailed protocol for the use of LIVE-PAINT in <em>S. cerevisiae</em>.</p><br>Keywords: super-resolution, live cell imaging, fluorescence microscopy, nanobiotechnology, peptide-binding module, coiled coil, peptide, protein, S. cerevisiae<br><h2>Introduction</h2><p>Here we present a detailed protocol for the use of LIVE-PAINT in <em>S. cerevisiae</em> including all necessary steps: cell growth, microscope and sample preparation, data acquisition, and data processing.</p><p><br></p><p>The below yeast strains are examples of those that can be used with the LIVE-PAINT method. In our strains we express the fluorescent protein construct under control of the <em>GAL1 </em>promoter and delete the <em>GAL2 </em>gene, which is important for making the <em>GAL1 </em>promoter response linear with respect to galactose concentration. In our work, we tagged and imaged Cdc12p, Cof1p, and actin, though this can be done with the user’s desired protein of interest. Below we show an example strain for tagging and imaging Cdc12p and an example strain for imaging a user's desired protein of interest.</p><p><br></p><p>1) Cdc12p-SYNZIP18 + SYNZIP17-mNeonGreen</p><p>Genotype: BY4741 MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 gal2Δ::His3MX6 SYNZIP17-mNeonGreen Cdc12p-SYNZIP18::KanMX6</p><p><br></p><p>2) Protein of interest*-SYNZIP18 + SYNZIP17-mNeonGreen</p><p>Genotype: BY4741 MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0 gal2Δ::His3MX6 SYNZIP17-mNeonGreen Protein of interest*-SYNZIP18::KanMX6</p><p>*Protein of interest can be any protein the user would like to study</p><h2>Reagents</h2><p>Yeast nitrogen base with amino acids (Sigma-Aldrich, cat no. Y1250-250G)</p><p>Raffinose pentahydrate (Sigma-Aldrich, cat no. R0250-25G)</p><p>Galactose (Sigma-Aldrich, cat no. G0625-100G)</p><p>Glass coverslips with thickness no. 1 (VWR, cat no. 48366-067)</p><p>Secure frame-seal slide chambers (Bio-Rad, cat no. SLF0201)</p><p>Concanavalin A from <em>Canavalia ensiformis</em> (Sigma-Aldrich, cat no. C2010-25MG)</p><h2>Equipment</h2>Inverted Nikon TI2 microscope (Nikon, Japan)

1.49 NA TIRF objective (CFI Apochromat TIRF 60XC Oil, Nikon, Japan)

488 nm laser (Cobolt MLD 488-200 Diode Laser System, Cobalt, Sweden)

EMCCD camera (Delta Evolve 512, Photometrics, Tucson, AZ, USA)

2.6 L Zepto plasma laboratory unit (Diener Electronic)<h2>Procedure</h2><p>1) Yeast cell growth</p><p>1.1) Prepare a stock of synthetic complete media: 6.7 g/L yeast nitrogen base with amino acids, 10 g/L raffinose, plus the desired concentration of galactose (usually 0.05 g/L). Typically we would prepare 10 mL stock to use for a set of experiments.</p><p>1.2) Pipette&nbsp;600 μL of synthetic complete media into a culture tube. Pick one yeast colony from an agar plate into the media, and pipette to mix the cells evenly.</p><p>1.3) Prepare a second 400 μL aliquot of synthetic complete media, in a culture tube. Add 100 μL of cells from the first suspension of&nbsp;yeast cells to the fresh 400 μL aliquot, to make a 1:5 dilution of the culture. This second culture is likely to produce the desired log phase cells approximately 16 hours after starting the overnight culture.</p><p>1.4) Grow the 500 μL cultures in 15 mL tubes overnight in a 30°C shaking incubator. One could set up overnights with different dilutions, to best match the desired time to mid-log phase.</p><p><br></p><p>2) Prepare microscope</p><p>2.1) Use a TIRF microscope equipped with a 488 nm laser and fitted with a ‘perfect focus’ system. (The ‘perfect focus’ autocorrects any <em>z</em>-stage drift during imaging).</p><p>2.2) Rotate the mirrors in the beamline path to align the laser and direct it parallel to the optical axis. We have used a 1.49 NA TIRF objective (CFI Apochromat TIRF 60XC Oil, Nikon, Japan), mounted on an inverted Nikon TI2 microscope (Nikon, Japan).</p><p>2.3) Use an appropriate camera to collect fluorescence output from the microscope. For example, we have used an EMCCD camera (Delta Evolve 512, Photometrics, Tucson, AZ, USA) operating in frame transfer mode (EMGain = 11.5 e–/ADU and 250 ADU/photon).</p><p><br></p><p>3) Slide preparation</p><p>3.1) Clean 22 x 22 mm glass coverslips with thickness no. 1 (VWR) using a 20 minute exposure in a 2.6 L Zepto plasma laboratory unit (Diener Electronic).</p><p>3.2) Secure frame-seal slide chambers (9 × 9 mm<sup>2</sup>, Biorad, Hercules, CA) to a coverslip. There are two plastic sides. One has a square cutout for the well, while the other side has no cutout. Carefully remove the plastic side with no well cutout using forceps. This will reveal a sticky square frame-seal chamber.</p><p>3.3) Lower the exposed sticky side of the frame-seal slide chamber onto a cleaned coverslip. Press gently around the edge of the well with forceps to make a secure seal to reduce the chance of buffer leakage in later steps.</p><p>3.4) Prepare the slide surface for the attachment of yeast cells by coating the surface with approximately 100 μL of 2 mg/mL concanavalin A (Sigma-Aldrich), dissolved in PBS pH 7.4.</p><p>3.5) Leave the concanavalin A in contact with&nbsp;the surface of the slide for 30 seconds, then pipette off the excess,&nbsp;tilting the slide to ensure all liquid is removed.</p><p><br></p><p>4) Attach yeast cells to slide</p><p>4.1) Pipette 150 μL of&nbsp;yeast overnight culture grown to mid-log phase (i.e. OD 0.1-0.5) onto the slide.</p><p>4.2) Leave the yeast culture on the slide for approximately 5 minutes.</p><p>4.3) Pipette off the culture media, removing the cells which are not bound to the slide.&nbsp;</p><p>4.4) Wash the surface three times by pipetting 150 μL of synthetic complete media onto the slide, pipetting up and down a few times, and then discarding the media. (These steps remove any yeast cells that are not attached to the surface. Free floating cells would ruin the experiment, because new cells would be continually diffusing into the field of view and possibly even landing on other cells).</p><p>4.5) Add 150 μL fresh synthetic complete media to the slide before imaging.</p><p><br></p><p>5) Data acquisition</p><p>5.1) Mount the slide containing yeast cells on the microscope by placing the slide on top of a drop of immersion oil on the objective.</p><p>5.2) Set the desired exposure time and laser power for the experiments. For example, we chose an exposure time of 50 ms with a laser power density of 3.1 W/cm<sup>2</sup>, using a 488 nm laser to excite green fluorescent proteins.</p><p>5.3) Bring the sample into focus and then adjust the laser angle to achieve TIRF. When in TIRF you will likely start to see single localization blinking events inside the cell. </p><p>5.4) Image the cells in TIRF for the desired amount of time. For example, we typically use 1,000 to 6,000 frames with an exposure time of 50 ms.</p><p>5.5) Ensure at this stage that you can see individual blinking events. If you cannot see these single blinking events, try varying the TIRF angle in order to decrease the illumination volume, increasing the laser power, or waiting for some of the fluorescent proteins to photobleach, if you have an abundance of fluorescence signal.&nbsp;</p><p>5.6) Save the output as a tiff stack.</p><p><br></p><p>6) Image analysis</p><p>6.1) Load a tiff stack acquired during imaging into Fiji.</p><p>6.2) Use the Peak Fit function of the Fiji GDSC SMLM plugin to analyze single localization events. Typical parameters we used were a signal strength threshold of 30, a minimum photon threshold of 100, and a precision threshold of 20 nm.</p><h2>Acknowledgements</h2><p>The authors acknowledge support from NIH R01 GM118528; The Yale Integrated Graduate Program in Physical and Engineering Biology; the School of Biological Sciences at the University of Edinburgh; BBSRC EASTBIO Doctoral Training Partnership; the School of Chemistry at the University of Edinburgh; The Euan MacDonald Centre; Dr. Jim Love and UCB Pharma for providing funding for the microscope; and the UK Dementia Research Institute.</p></body></html>