<html><head><title>High-throughput single-cell joint analysis of histone modifications and gene expression by Paired-Tag</title></head><body><h1>High-throughput single-cell joint analysis of histone modifications and gene expression by Paired-Tag</h1>Authors: Chenxu Zhu, Yanxiao Zhang, Yang Eric Li, Jacinta Lucero, M. Margarita Behrens, Bing Ren<br>Abstract: <p>We describe here Paired-Tag, a high-throughput multi-omics method for joint profiling of histone modifications and gene expressions in single cells. The assay is based on a combinatorial barcoding indexing strategy that does not require special instruments. It can be performed with nuclei extracted from cultured cells or frozen tissues, in standard molecular biology laboratories.</p><br>Keywords: Paired-Tag, Single-cell multi-omics, Histone modifications, Gene expression<br><h2>Introduction</h2><p>Covalent modifications to histone proteins play critical roles in regulating cell-type-specific gene expression programs in multi-cellular organisms<sup>1,2</sup>. Sigle-cell methods for profiling histone modifications<sup>3-9</sup>&nbsp;were recently reported. However, integrative analysis of different histone modifications in heterogeneous tissues is very challenging because of the difficulty in aligning cell clusters defined based on different histone marks, which have different relationships to gene expression and cell states. Paired-Tag addresses this challenge by enabling simultaneous profiling of gene expression and histone modifications from the same individual cells. Cells of the same types in a heterogenous population could be grouped together based on their similar transcription profiles in the RNA component of the Paired-Tag profiles, and the different histone modification profiles could be directly obtained and integrated from the DNA component of the Paired-Tag datasets.</p><p><br></p><p>Paired-Tag builds upon a technique that we previously developed, Paired-seq<sup>10</sup>, which utilizes a combinatorial barcode indexing strategy<sup>11</sup>&nbsp;for joint analysis of gene expression and chromatin accessibilities in single cells. Paired-Tag combines Paired-seq with the CUT&amp;Tag strategy<sup>7</sup>. Like Paired-seq, Paired-Tag does not require specific instruments and can be performed in standard molecular biology laboratories. It is highly-scalable, capable of processing up to 500,000 nuclei in a single experiment. It generates joint histone modifications and gene expression profiles as effective as stand-alone single cell scRNA-seq and scChIP-seq assays.</p><p><br></p><h2>Reagents</h2><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;PBS (Thermo Fisher Scientific, Cat#10010-23)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Tris-HCl (Sigma, Cat#S7653)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;MgCl<sub>2</sub>&nbsp;(Sigma, Cat#63069)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;NaCl (Sigma, Cat#S7653)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;KCl (Sigma, Cat#P9333)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Protease Inhibitor (Roche, Cat#05056489001)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;RNase OUT (Invitrogen, Cat#10777-019)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;SUPERase IN (Invitrogen, Cat#AM2694)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;IGEPAL CA-630 (Sigma, Cat#I8896)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Sucrose (Sigma, Cat#S7903)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;DTT (Sigma, Cat#D9779)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Triton-X100 (Sigma, Cat#T9823)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;HEPES (Invitrogen, Cat#15630106)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Spermidine (Sigma, Cat#85558)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Digitonin (Sigma, Cat#D141)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;EDTA (Invitrogen, Cat#AM9261)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;dNTP (NEB, Cat#N0447S)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Maxima Reverse H minus Reverse Transcriptase (Invitrogen, Cat#EP0751)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;T4 DNA Ligase (NEB, Cat#M0202L)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;NEBuffer 3.1 (NEB, Cat#B7203S)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;SDS (Invitrogen, Cat#15553-035)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Protease K (NEB, Cat#P8107S)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;SPRI beads (Beckman coulter, Cat#B23319)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Terminal Transferase (NEB, Cat#M0315S)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;KAPA HiFi HS (KAPA, Cat#KK2502)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;10X Cutsmart buffer&nbsp;(NEB, Cat#M7204S)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;SbfI-HF (NEB, Cat#R3642)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;FokI (NEB, Cat#R0109S)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;NotI-HF (NEB, Cat#R3189)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;NEB Q5 DNA Polymerase (NEB, Cat#M0491)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;QIAquick PCR purification kit (QIAGEN, Cat#28104)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;KAPA qPCR library quantification kit (Roche)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Barcodes plates and DNA primers, oligos (IDT)</p><p><br></p><p><strong><em>The reagents in this section can be stored at -20&nbsp;°C for up to 6 months</em></strong></p><p><u>Barcode Plates (R02, R03 plates):</u></p><p>1. Distribute the R02 and R03 barcoded oligos to two 96-well plates, respectively as: 6 µL barcoded oligos (R02 or R03, 100 µM), 5.5 µL Linker (R02 or R03, 100 µM), 38.5 µL H<sub>2</sub>O.</p><p>2. Seal the plates, heat for 5 mins at 95&nbsp;°C and slowly cool down to 20&nbsp;°C at the speed of 0.1&nbsp;°C/s.</p><p>3. Aliquot the each of the annealed barcodes plate into 4 working plates (each plate with 10 µL of annealed oligos in each well), sealed and stored at -20&nbsp;°C.</p><p><u>&nbsp;</u></p><p><u>P5 adaptor mix:</u></p><p>1. Mix the oligo DNA as (a) P5 complex: 25 µL P5-FokI (100 µM), 25 µL P5c-NNDC-FokI (100 µM); and (b) P5H complex: 25 µL P5H-FokI (100 µM), 25 µL P5Hc-NNDC-FokI (100 µM).</p><p>2. Heat the mixtures for 5 mins at 95&nbsp;°C and slowly cool down to 20°C at the speed of 0.1&nbsp;°C/s.</p><p>3. Mixed P5 complex with P5H complex (on the ice) at the ratio of 1:3, stored at -20&nbsp;°C.</p><p><u>&nbsp;</u></p><p><u>Barcoded RT primers:</u></p><p>Mix the oligo (RNA barcode R01) in 12 1.5 mL tubes: 12.5 µL RNA_RE (#01 to #12 in the 12 tubes, 100 µM), 12.5 µL RNA_NRE (#01 to #12 matched with RNA_RE, 100 µM), and 75 µL H<sub>2</sub>O, stored at -20&nbsp;°C.</p><p><u>&nbsp;</u></p><p><u>Barcoded Protein A-Tn5:</u></p><p>1. Mix the oligo (DNA barcode R01) in 12 PCR tubes: 25 µL DNA_RE (#01 to #12 in the 12 tubes, 100 µM), 25 µL pMENTs (100 µM, same in the 12 tubes).</p><p>2. Heat the mixtures for 5 mins at 95&nbsp;°C and slowly cool down to 20&nbsp;°C at the speed of 0.1&nbsp;°C/s.</p><p>3. Mix 1 µL of annealed transposome DNA with 6 µL of unloaded proteinA-Tn5 (0.5 mg/mL), briefly vortex and quickly spun down.</p><p>4. The mixtures were incubated at room temperature for 30 min then at 4&nbsp;°C for an additional 10 min, stored at -20&nbsp;°C.</p><p><u>&nbsp;</u></p><p><u>Tn5-AdaptorA:</u></p><p>1. Mix the oligo as: 25 µL Adaptor A (100 µM), 25 µL pMENTs (100 µM).</p><p>2. Heat the mixture for 5 mins at 95&nbsp;°C and slowly cool down to 20&nbsp;°C at the speed of 0.1&nbsp;°C/s.</p><p>3. Mix 1 µL of annealed transposome DNA with 6 µL of unloaded Tn5 (0.5 mg/mL), briefly vortex and quickly spun down.</p><p>4. The mixtures were incubated at room temperature for 30 min then at 4&nbsp;°C for an additional 10 min. Dilute 10 X with dilution buffer (10 mM Tris-HCl pH7.5, 100 mM NaCl, 50% Glycol, 1 mM DTT), stored at -20&nbsp;°C.</p><p><br></p><p><u>1 mM dCTP:</u>&nbsp;2 µL dCTP (100 mM) + 198 µL ultrapure H<sub>2</sub>O.</p><p><br></p><p><strong><em>The reagents in this section can be stored at 4&nbsp;°C for up to 1 months</em></strong></p><p><u>0.2% NIB (Nuclei isolation buffer):</u></p><p>1. Prepare the 10% IGEPAL CA-630 by 1:10 dilution with PBS.</p><p>2. Prepare the 0.2% NIB as: 200 µL 10% IGEPAL CA-630, 0.5 g BSA, and add PBS to 10 mL.</p><p><br></p><p><u>2X TB (Tagmentation buffer):</u></p><p>1. Prepare the 2X TB as: 660 µL Tris-Ac (pH 7.8, 1 M), 440 µL K-Ac (3 M), 3.2 mL DMF, H<sub>2</sub>O (5.59 mL).</p><p>2. Aliquot the 2X TB to 1.5 mL tubes and stored at 4&nbsp;°C.</p><p>3. To avoid contamination, each experiment should use a new tube of 2X TB.</p><p><br></p><p><u>1M Spermidine:</u>&nbsp;145 mg in 1 mL ultrapure H<sub>2</sub>O.</p><p><u>10% IGEPAL-CA630:</u>&nbsp;100 µL + 900 µL ultrapure H<sub>2</sub>O.&nbsp;</p><p><u>2% Digitonin:</u>&nbsp;20 mg in 1 mL DMSO.</p><p><u>250 mM MgCl</u><sub><u>2</u></sub><u>:</u>&nbsp;250 µL MgCl<sub>2&nbsp;</sub>(1 M) + 750 µL&nbsp;ultrapure H<sub>2</sub>O.</p><p><u>&nbsp;</u></p><p><strong><em>The reagents in this section can be stored at RT for up to 1 months.</em></strong></p><p><u>40.4 mM EDTA:</u>&nbsp;80.8 µL EDTA (500 mM) + 919.2 µL ultrapure H<sub>2</sub>O.</p><p><br></p><h2>Equipment</h2><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Benchtop centrifuge (Eppendorf)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;ThermoMixer (Eppendorf)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Swing bucket centrifuge (ThermoFisher)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Qubit Fluorometric Quantification (ThermoFisher)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Magnetic stands (96-well plate stand, PCR-tube stand)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Fragment analyzer/Tapestation (Agilent)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Micropipetts (0.5-20 μl, 10-200 μl, 100-1000 μl)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;RNase-free tubes (300 µL, 1.5 mL)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Maximum recovery tubes (1.5 mL, Axygen, MCT150LC)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;30 µm Cell-Tric (Sysmex, Cat#04-004-2326)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;96-well plates (Eppendorf, 0030603303)</p><h2>Procedure</h2><p><strong>A. Day 1: nuclei preparation and antibody staining</strong></p><p><strong>A-1 Freshly prepare the following buffer:</strong></p><p>1. Prepare the Douncing Buffer (if processing mouse brain tissues) per sample as: 0.5 mL sucrose (1 M), 25 µL KCl (2 M), 10 µL MgCl<sub>2</sub>&nbsp;(1 M), 20 µL Tris-HCl (1 M), 2 µL DTT (1 M), 40 µL Protease Inhibitor (50 X), 50 µL SUPERase IN, 25µL RNase OUT and 1,328 µL H<sub>2</sub>O</p><p>2. Prepare the Complete Buffer as: 40 µL HEPES (pH 7.5, 1 M), 75 µL NaCl (4 M), 1 µL spermidine (1 M), 40 µL Protease Inhibitor (50 X), 50 µL SUPERase IN, 25 µL (RNase OUT), 2 µL IGEPAL-CA630 (10%), 10 µL digitonin (2%), 8 µL EDTA (500 mM) and 1,749 µL H<sub>2</sub>O.</p><p>3. Prepare the 0.2% NIB-RPI (RNase and Protease Inhibitor) as: 1 mL 0.2% NIB, 20 µL Protease Inhibitor (50X), 25 µL SUPERase IN and 12.5 µL RNaseOUT. Additional 0.2% NIB-RPI may be needed if there are multiple samples.</p><p><u>&nbsp;</u></p><p><strong>A-2 Nuclei preparation:</strong></p><p><u>I, Cultured cells:</u></p><p>1. Disassociated the cells according to culture protocol.</p><p>2. Spin-down cells at 300 g for 5 mins, resuspended the cells in 1 mL (5 million cells) 0.2% NIB-RPI. Sit on ice for 5 min.</p><p>3. Count the nuclei using cell counter.</p><p><u>II, Mouse brain tissues:</u></p><p>1. Prechill any tubes or tools, set the centrifuge to 4&nbsp;°C.</p><p>2. Add 10 µL 10% Triton-X100 in to the douncer (1 mL), add 1 mL Douncing Buffer.</p><p>3. Transfer the tissue dissection into douncer.</p><p>4. Use loose pestle 5-10 times gently followed by tight pestle for 15-20 times.</p><p>5. Filter with 30 µm&nbsp;Cell-Tric. Transfer to1.5 mL Maximum Recovery tubes. Spin down at 1000 g for 10 min.</p><p>6. Wash with 1 mL Douncing Buffer, spin down again at 1,000 g for 10min.</p><p>7. Resuspended the cells in 1 mL (5 million cells) 0.2% NIB-RPI. Sit on ice for 5 min.</p><p>8. Count the nuclei using cell counter.</p><p><br></p><p><strong>A-3 Immuno-staining:</strong></p><p>1. Put each of 300 k nuclei into 12 Maximum Recovery tubes (total of 3.6 million nuclei); Species-mixing are recommended in at least one tube to estimate the potential collision rate.</p><p>2. Spin down the nuclei at 1,000 g for 10 min and resuspended in 50 µL Complete Buffer.</p><p>3. Add 2 µg antibodies to each tube and incubate the mixture with rotation at 4&nbsp;°C overnight.</p><p><br></p><p><strong>B. Day2: multi-omics tagging and nuclei barcoding</strong></p><p><strong>B-1 Freshly prepare the following buffer:</strong></p><p>1. Prepare the Complete Buffer in 2 mL or appropriate tube as: 40 µL HEPES (pH 7.5, 1 M), 75 µL NaCl (4 M), 1 µL spermidine (1 M), 40 µL Protease Inhibitor (50 X), 50 µL SUPERase IN, 25 µL (RNase OUT), 2 µL IGEPAL-CA630 (10%), 10 µL digitonin (2%), 8 µL EDTA (500 mM) and 1,749 µL H2O.</p><p>2. Prepare the Med Buffer #1 in 1.5 mL tube as: 20 µL HEPES (pH 7.5, 1 M), 75 µL NaCl (4 M), 0.5 µL spermidine (1 M), 20 µL Protease Inhibitor (50 X), 25 µL SUPERase IN, 12.5 µL (RNase OUT), 1 µL IGEPAL-CA630 (10%), 5 µL digitonin (2%), 4 µL EDTA (500 mM) and 845 µL H2O.</p><p>3. Prepare the Med Buffer #2 in 5 mL or other appropriate tube as: 60 µL HEPES (pH 7.5, 1 M), 225 µL NaCl (4 M), 1.5 µL spermidine (1 M), 60 µL Protease Inhibitor (50 X), 75 µL SUPERase IN, 37.5 µL (RNase OUT), 3 µL IGEPAL-CA630 (10%), 15 µL digitonin (2%) and 2,532 µL H2O.</p><p><u>&nbsp;</u></p><p><strong>B-2 Nuclei washing and tagmentation:</strong></p><p>1. Spin down nuclei at 600 g for 10 mins, resuspended with 50 µL Complete Buffer.</p><p>2. Repeat step 1 for 1 additional time.</p><p>3. Spin-down at 600 g for 10 mins again, resuspended in 50 µL Med Buffer #1.</p><p>4. Add 1 µL pA-Tn5 (#01-#12) for each tube, rotate at room temperature for 60 min.</p><p>5. Spin-down nuclei at 300 g for 10 mins, resuspended in 50 µL Med Buffer #2.</p><p>6. Repeat step 5 for 2 additional times.</p><p>7. Add 2 µL 250 mM MgCl<sub>2</sub>, incubate in at ThermoMixer, 550 r.p.m., 37&nbsp;°C, for 1 hr. During this time, pre-melt the barcoded RT primers on ice and mixed well.</p><p>8. Add 16.5 µL 40.4 mM EDTA. Spin-down at 1,000 g for 10 mins at 4&nbsp;°C. Discard the&nbsp;supernatant. Proceed to B-3 RT reaction immediately.</p><p><br></p><p><strong>B-3 Reverse transcription (RT) reaction:</strong></p><p>1. Distribute 4 µL each of RT primers mix (#01-#12) into 12 PCR tubes.</p><p>2. Prepare the RT Mixture as: 52.8 µL 5X RT Buffer, 60 µL H2O, 52.8 µL PBS, 13.2 µL dNTP, 3.3 µL SUPERase IN and 1.65 µL RNase OUT.</p><p>3. Resuspended the nuclei in 14 µL RT mix and transfer to each corresponding RT tube with the same 1st round barcode in tagmentation step (R01).</p><p>4. Add 2 µL of Maxima H- Reverse Transcriptase into each tube. Perform the following program in a thermocycler: step1, 50 °C * 10 mins; step2, 8 °C * 12 s, 15 °C * 45 s, 20 °C * 45 s, 30 °C * 45 s, 42 °C * 2 mins, 50 °C * 5 mins, repeat step 2 for additional 2 times; step 3, 50 °C * 10 mins; step 4, hold at 12 °C.</p><p>5. Combine nuclei from 12 tubes into a Maximum Recovery tube (ON ICE!!!), add 4.8 µL 5% Triton X-100 and gently mix by pipette up and down.</p><p>6. Spin-down the nuclei at 1,000 g for 10 mins at 4&nbsp;°C, discard the supernatant, resuspended in 1 mM 1X NEBuffer 3.1. Proceed to B-4 ligation-based barcoding immediately.</p><p><br></p><p><strong>B-4 Ligation-based barcoding:</strong></p><p>1. Prepare the of ligation mixture in 15 mL tubes as: 2,260 µL H2O, 500 µL 10X T4 DNA Ligase Buffer, 50 µL BSA (20 mg/mL), 100 µL 10X NEBuffer 3.1. Two tubes of this mixture are needed.</p><p>2. Prepare the R02 Blocking Solution as: 264 µL R02 Blocker (100 µM), 250 µL 10X T4 DNA Ligase Buffer and 486 µL H<sub>2</sub>O.</p><p>3. Prepare the R03 Termination Solution as: 264 µL R03 Quencher (100 µM), 500 µM EDTA (500 mM) and 236 µL H<sub>2</sub>O.</p><p>4. Add the 1 mL nuclei (in Buffer 3.1) into the 1st tube of Ligation Mixture. Add 100 µL T4 DNA Ligase into the mixture and mixed well.</p><p>5. Distribute the nuclei mixture into barcode plate R02: 40 µL nuclei mixture for each well. Seal the plate and incubate in a ThermoMixer at 37&nbsp;°C, 300 r.p.m., for 30 mins.</p><p>6. Use multichannel pipette to add 10 µL of R02 Blocking Solution to each well and gentle mixed. Seal the plate. Continue the incubation for additional 30 mins.</p><p>7. Combine all the nuclei in a 15 mL tube (pre-washed with 0.5% BSA). Spin-down at 1,000 g, 10&nbsp;°C for 10 mins. Discard the&nbsp;supernatant, resuspended in 1 mM 1X NEBuffer 3.1.</p><p>8. Add the 1 mL nuclei into the 2nd tube of Ligation Mixture. Add 100 µL T4 DNA Ligase into the mixture and mixed well.</p><p>9. Distribute the nuclei mixture into barcode plate R03: 40 µL nuclei mixture for each well. Seal the plate and incubate in a ThermoMixer at 37&nbsp;°C, 300 r.p.m., for 30 mins.</p><p>10. Use multichannel pipette to add 10 µL of R03 Termination Solution to each well and gentle mixed. No further incubation is needed at this step.</p><p>11. Combine all the nuclei in a 15 mL tube (pre-washed with 0.5% BSA). Spin-down at 1,000 g, 10&nbsp;°C for 10 mins. Discard the&nbsp;supernatant.</p><p>12. Wash the nuclei once with cold PBS. Spin-down at 1,000g 10&nbsp;°C for 10 mins and resuspended in 200 µL - 1 mL cold PBS (optimal concentration 1,000 cell/µL). Proceed to lysis and DNA Cleanup.</p><p><br></p><p><strong>B-5 Lysis and DNA Cleanup:</strong></p><p>1. Count the nuclei, divide 2k-4k nuclei (2.5k recommended) into sub-libraries (e.g. 96-well plate).</p><p><strong><em>Pause Point:&nbsp;</em></strong>Aliquoted nuclei can be stored at -80 °C for up to 6 months.</p><p><br></p><p>2. Prepare the lysis mixture as below: 3 µL 10% SDS, 3 µL Protease K, 3 µL NaCl (4M) and add PBS to total of 30 µL (including the volume of barcoded nuclei in PBS) for one sub-library. Make appropriate amount for the number of sub-libraries needed.</p><p>3. Add the lysis mix into nuclei. Seal the plate, incubate in a ThermoMixer at 55&nbsp;°C, 850 r.p.m. for at least 2hr or overnight.</p><p>4. Cool-down the mixture to RT. Cleanup the DNA use 1.0X (30 µL) SPRI beads and elute in 12.5 µL Ultrapure H<sub>2</sub>O. Store at -20&nbsp;°C. During washing steps of the cleanup, please remove residual SDS as much as possible.</p><p><strong><em>Pause Point:&nbsp;</em></strong>Purified DNA can be stored at -20 °C or -80 °C for up to 6 months.</p><p><br></p><p><strong>C. Day3: library construction</strong></p><p><strong>C-1 TdT Tailing and pre-amplification</strong></p><p>1. Add 1 µL 10X TdT Buffer and 0.5 µL 1 mM dCTP into 12.5 µL purified DNA. Incubate at 95&nbsp;°C for 5 mins. Quickly chill on ice for 5 mins.</p><p>2. Add 1 µL of TdT to the mixture. Incubate at 37&nbsp;°C for 30 mins followed by inactivation at 75&nbsp;°C for 20 mins.</p><p>3. Add the Anchor Mix (6 µL 5X KAPA Buffer, 0.6 µL 10 mM dNTP, 0.6 µL 10 µM Anchor-FokI-GH, 7.3 µL H<sub>2</sub>O and 0.6 µL KAPA HiFi HS DNA polymerase) to the reaction mix directly, and mix by pipette up and down. Perform the program in a thermocycler: step1, 95&nbsp;°C * 3 mins; step2, 95&nbsp;°C * 15 s, 47&nbsp;°C * 60s, 68&nbsp;°C * 120 s, 47&nbsp;°C * 60 s, 68&nbsp;°C * 120 s; repeat step2 for additional 15 times; step3, 72&nbsp;°C * 10 mins; step 4, hold at 12&nbsp;°C.</p><p>4. Add the Preamp Mix (4 µL 5X KAPA Buffer, 0.5 µL 10 mM dNTP, 2 µL 10 µM PA-F, 2 µL 10 µM PA-R, 11 µL H<sub>2</sub>O and 0.5 µL KAPA HiFi HS DNA polymerase) to the reaction mix directly and mix by pipette up and down. Perform the program in a thermocycler: step1, 98&nbsp;°C * 3 mins; step2, 98&nbsp;°C * 20 s, 62&nbsp;°C * 20 s, 72&nbsp;°C * 150 s; repeat step 2 for additional 9 times; step 3, 72&nbsp;°C * 120 s; step4, hold at 12&nbsp;°C.</p><p>5. Use 0.2X + 0.65 X (10&nbsp;µL + 32.5 µL) of SPRI beads for double-size selection purification. Elute in 36 µL ultra-pure H<sub>2</sub>O, use 1 µL for Qubit quantification.&nbsp;Recode the concentration&nbsp;<strong>x</strong>&nbsp;ng/µL. Dependents on the cell type and numbers of cells per sub-library, in our hand typically x will in the range of 1 ng/µL to 30 ng/µL (see&nbsp;<strong>Troubleshooting</strong>&nbsp;for details).</p><p><strong><em>Pause Point:&nbsp;</em></strong>Purified DNA can be stored at -20 °C or -80 °C for up to 6 months.</p><p><br></p><p><strong>C-2 2nd adaptor tagging and digestion</strong></p><p>1. Transfer 17 µL of the purified amplified product into two 200 µL PCR tubes for DNA and RNA, respectively.&nbsp;</p><p>2. For DNA part: add 2.5 µL 10X Cutsmart buffer, 1 µL SbfI-HF, 1 µL FokI, and 3.5 µL H<sub>2</sub>O.</p><p>3. For RNA part: add 2 µL 10X Cutsmart buffer, 1 µL NotI-HF.</p><p>4. Incubate at 37&nbsp;°C for 60 mins.</p><p>5. Use 1.25 X SPRI beads (31.3 µL for DNA, 25 µL for RNA). Elute both DNA and RNA in 10 µL H<sub>2</sub>O.</p><p><strong><em>Pause Point:&nbsp;</em></strong>Purified DNA can be stored at -20 °C or -80 °C for up to 6 months.</p><p><br></p><p>6. For DNA part: add 2 µL 10X T4 DNA Ligase Buffer, 1.5 µL P5 Adaptor Mix, 5 µL H<sub>2</sub>O and 1.5 µL T4 DNA Ligase. Incubate with the ligation program (4&nbsp;°C * 10 mins, 10&nbsp;°C 1* 5 min, 16&nbsp;°C * 15 min, 25&nbsp;°C * 45 min).</p><p>7. For DNA part: Cleanup the ligation mixture with 1.25 X (25 µL) SPRI beads and elute in 30 µL H<sub>2</sub>O.</p><p>8. For RNA part: add 10.5 µL 2X TB, y µL 1:10 diluted Tn5-AdaptorA (<em>y = x * 0.2, titration may be needed for different batch of Tn5</em>).</p><p>9. For RNA part: The tagmentation reaction was carried out 550 r.p.m., 37&nbsp;°C for 30 min in a ThermoMixer followed by cleaned up using QIAquick PCR purification kit and eluted in 30 µL 0.1X EB (QIAGEN).</p><p><strong><em>Pause Point:&nbsp;</em></strong>Purified DNA can be stored at -20 °C or -80 °C for up to 6 months.</p><p><br></p><p><strong>C-3 Indexing PCR and sequencing</strong></p><p>1. For DNA libraries, prepare the PCR mix as: 30 µL purified template, 2 µL 10 µM TruSeq i7 indices, 2 µL 10 µM Universal P5 primer, 1 µL 10 mM dNTP, 10 µL 5X NEB Q5 Buffer, 4 µL H2O and 1 µL Q5 HiFi DNA polymerase. Run the program in a thermocycler: step1, 98&nbsp;°C * 3 mins; step2, 98 °C * 10 s, 63 °C * 30 s, 72 °C * 60 s; repeat step2 for 8 cycles; step3, 72 °C * 60s; step4, hold at 12 °C.</p><p>2. For RNA libraries, prepare the PCR mix as: 30 µL purified template, 2 µL 10 µM TruSeq i7 indices, 2 µL 10 µM NextEra N5 indices, 1 µL 10 mM dNTP, 10 µL 5X NEB Q5 Buffer, 4 µL H2O and 1 µL Q5 HiFi DNA polymerase. Run the program in a thermocycler: step1, 72&nbsp;°C * 5 mins, 98&nbsp;°C * 30s; step2, 98 °C * 10 s, 63 °C * 30 s, 72 °C * 60 s; repeat step2 for 8 cycles; step3, 72 °C * 60s; step4, hold at 12 °C.</p><p>3. Use 0.5 µL (dilute 1000X) to run a qPCR to estimate the additional cycles needed (typically 0-4) to reach 10 nM. Run the additional cycles in a thermocycler: step1, 98&nbsp;°C * 3 mins; step2, 98 °C * 10 s, 63 °C * 30 s, 72 °C * 60 s; repeat step2 for desired cycles; step3, 72 °C * 60s; step4, hold at 12 °C.</p><p>4. Cleanup the libraries using 0.9 X (45 µL) SPRI beads, elute in 15-30 µL H<sub>2</sub>O. Use qPCR to quantify libraries. Store at -20&nbsp;°C.&nbsp;</p><p><strong><em>Pause Point:&nbsp;</em></strong>Purified libraries can be stored at -20 °C or -80 °C for up to 6 months.</p><p><br></p><p>5. Use Agilent Fragment Analyzer to analysis the fragment distribution of libraries (<strong>Fig. 1</strong>). If there is a clear band at ~245 bp (<strong>Fig. 1c</strong>), repeat the size selection with 0.75X SPRI beads&nbsp;or other size selection method&nbsp;(see&nbsp;<strong>Troubleshooting</strong>&nbsp;for details).</p><p>6. Multiplex libraries and sequence with standard Illumina sequencing primers on commercial sequencing platforms. We have tested sequencing of Paired-Tag libraries on HiSeq 2500/4000, NextSeq 500 or NovaSeq 6000 platforms. Libraries should be loaded at recommended concentrations according to manufacturer’s instructions. At least 50 and 100 sequencing cycles are needed for Read1 and Read2, respectively. For example: using PE 50 + 7 + 100 cycles (Read1 + Index 1 + Read2) on a NextSeq 500 platform with 150-cycle sequencing kits, or PE 100 +7 +100 cycles on a NovaSeq 6000 platform with 200-cycle sequencing kits.</p><p><br></p><p><strong>D. Initial data processing and QC</strong></p><p>1. All scripts for initial Paired-Tag data processing are available from GitHub (<a href="https://github.com/cxzhu/Paired-Tag" rel="noopener noreferrer" target="_blank">https://github.com/cxzhu/Paired-Tag</a>). Download and perform the data pre-processing steps according to instructions on the GitHub repository.</p><p>2. Initial Paired-Tag data processing includes (a) extract barcode sequences from Read2, (b) assign barcodes combinations to cellular barcodes references (assign barcode sequences to ID of 12 sample tubes and 2 rounds of 96 wells), (c) mapping the assigned reads to reference genome and (d) generating cell-to-features matrices for downstream analyses.&nbsp;</p><p>3. The following metrics during initial Paired-Tag data processing can be used for quality control. For step 2(a), typically &gt;85% and &gt;75% of DNA and RNA reads will have full ligated barcodes. For step 2(b), &gt;85% of both DNA and RNA reads can uniquely assigned to one cellular barcode with no more than 1 mismatch. For step 2(c), typically &gt;85% of assigned reads can be mapped to the reference genome; depending on which histone mark targeted, from 60% to &gt;95% of assigned DNA reads can be mapped to the reference genome.</p><p>4. Cell-to-features matrices generated from initial data processing steps is a standard sparse matrix format which can be used for down-stream analyses with commonly used single-cell analysis software.</p><p><br></p><h2>Troubleshooting</h2><p>·&nbsp;&nbsp;&nbsp;&nbsp;After pre-amplification, typically ~30 – 1,000 ng DNA will be recovered (around 1 ng/µL to 30 ng/µL measured by Qubit, when pre-amplified products were eluted in 36 µL). If the concentration is less than 0.2 ng/µL: (1) considering check the reagents used for pre-amplification steps (C1); (2) pre-amplification may be inhibited by residual SDS from B-5 step4, considering repeat this step (B-5 step4); (3) the quality of extracted nuclei may not be good enough for Paired-Tag experiment, considering perform a bulk ATAC-seq from extracted nuclei for QC. If the concentration is higher than 50 ng/µL, the amplified products are likely to be over-amplified barcode-adaptor dimers, please see the next paragraph for details.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;If only bands at ~245 bp are observed for DNA and RNA libraries, the libraries are likely failed as barcode-adaptor dimers without any insertions are the dominant component. Possible solutions include (1) considering optimization of tissue dissociation and nuclei preparation steps, (2) removing the supernatant as much as possible in B-2 step9, B-3 step6, B-4 step7 and step11; (3) considering repeating the B-4 step12 once if enough nuclei (&gt;100k) are recovered.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;If both bands of ~245 bp and 300-1000 bp exist for DNA or RNA libraries (<strong>Fig. 1c</strong>), indicating the presence of both libraries and barcode-adaptor dimers. (1) If a clear ~245 bp band exists, consider perform an additional 0.75X SPRI beads size selection (C-3 step4) (or other size selection method) to remove the primer-barcode dimers (~245 bp). (2) If there is no ~245 bp band but with significant fraction of fragments with length &lt; 300 bp (e.g.,&nbsp;<strong>Fig. 1c</strong>), consider reduce the amount of N5-Tn5 used in C-2 step8.</p><h2>Time Taken</h2><p>Nuclei extraction (A-1 to A-2): 2 hr</p><p>Immuno-staining (A-3): overnight</p><p>Nuclei washing and tagmentation (B-2): 3.5 hr</p><p>Reverse transcription (B-3): 1.5 hr</p><p>Nuclei barcoding (B-4): 2 hr</p><p>Lysis and DNA cleanup (B-5): overnight or 3 hr</p><p>Preamplification (C-1): 4 hr</p><p>Adaptor tagging (C-2): 3 hr</p><p>Indexing PCR and QC (C-3): 3-5 hr</p><h2>Anticipated Results</h2><p>The genomic DNA and cDNA libraries can be generated from desired number of nuclei, depending on how many sub-libraries were used for library construction. In our hands, the lengths of DNA fragments in genomic DNA libraries are typically between 300bp – 700bp (average ~400bp), for those from cDNA libraries are mainly between 300 bp – 1000 bp (average ~550bp) (<strong>Fig. 1a, b</strong>).</p><h2>References</h2><p>1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Bernstein, B. E., Meissner, A. &amp; Lander, E. S. The mammalian epigenome.&nbsp;<em>Cell</em>&nbsp;<strong>128</strong>, 669-681, doi:10.1016/j.cell.2007.01.033 (2007).</p><p>2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Stadhouders, R., Filion, G. J. &amp; Graf, T. Transcription factors and 3D genome conformation in cell-fate decisions.&nbsp;<em>Nature</em>&nbsp;<strong>569</strong>, 345-354, doi:10.1038/s41586-019-1182-7 (2019).</p><p>3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Wang, Q.<em>&nbsp;et al.</em>&nbsp;CoBATCH for High-Throughput Single-Cell Epigenomic Profiling.&nbsp;<em>Mol Cell</em>&nbsp;<strong>76</strong>, 206-216 e207, doi:10.1016/j.molcel.2019.07.015 (2019).</p><p>4&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Ai, S.<em>&nbsp;et al.</em>&nbsp;Profiling chromatin states using single-cell itChIP-seq.&nbsp;<em>Nat Cell Biol</em>&nbsp;<strong>21</strong>, 1164-1172, doi:10.1038/s41556-019-0383-5 (2019).</p><p>5&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Grosselin, K.<em>&nbsp;et al.</em>&nbsp;High-throughput single-cell ChIP-seq identifies heterogeneity of chromatin states in breast cancer.&nbsp;<em>Nat Genet</em>&nbsp;<strong>51</strong>, 1060-1066, doi:10.1038/s41588-019-0424-9 (2019).</p><p>6&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Ku, W. L.<em>&nbsp;et al.</em>&nbsp;Single-cell chromatin immunocleavage sequencing (scChIC-seq) to profile histone modification.&nbsp;<em>Nat Methods</em>&nbsp;<strong>16</strong>, 323-325, doi:10.1038/s41592-019-0361-7 (2019).</p><p>7&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Kaya-Okur, H. S.<em>&nbsp;et al.</em>&nbsp;CUT&amp;Tag for efficient epigenomic profiling of small samples and single cells.&nbsp;<em>Nat Commun</em>&nbsp;<strong>10</strong>, 1930, doi:10.1038/s41467-019-09982-5 (2019).</p><p>8&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Harada, A.<em>&nbsp;et al.</em>&nbsp;A chromatin integration labelling method enables epigenomic profiling with lower input.&nbsp;<em>Nat Cell Biol</em>&nbsp;<strong>21</strong>, 287-296, doi:10.1038/s41556-018-0248-3 (2019).</p><p>9&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Rotem, A.<em>&nbsp;et al.</em>&nbsp;Single-cell ChIP-seq reveals cell subpopulations defined by chromatin state.&nbsp;<em>Nat Biotechnol</em><strong>33</strong>, 1165-1172, doi:10.1038/nbt.3383 (2015).</p><p>10&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Zhu, C.<em>&nbsp;et al.</em>&nbsp;An ultra high-throughput method for single-cell joint analysis of open chromatin and transcriptome.&nbsp;<em>Nat Struct Mol Biol</em>&nbsp;<strong>26</strong>, 1063-1070, doi:10.1038/s41594-019-0323-x (2019).</p><p>11&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Rosenberg, A. B.<em>&nbsp;et al.</em>&nbsp;Single-cell profiling of the developing mouse brain and spinal cord with split-pool barcoding.&nbsp;<em>Science</em>&nbsp;<strong>360</strong>, 176-182, doi:10.1126/science.aam8999 (2018).</p><h2>Acknowledgements</h2><p>We thank B. Li for bioinformatic support, S. Kuan and K. Jepsen for sequencing. We thank QB3 MacroLab&nbsp;for purifying the ProteinA-Tn5 enzyme. We thank S. Preissl, X. Hou, H. Huang, M. Yu and J. Song for discussion.&nbsp;</p></body></html>