<html><head><title>MAPCap: A fast and quantitative transcription start site profiling protocol</title></head><body><h1>MAPCap: A fast and quantitative transcription start site profiling protocol</h1>Authors: Vivek Bhardwaj, Giuseppe Semplicio, Niyazi Umut Erdogdu, Asifa Akhtar<br>Abstract: <p>Below we present a simple and quick TSS quantification protocol, <strong>MAPCap</strong> (<strong>M</strong>ultiplexed <strong>A</strong>ffinity <strong>P</strong>urification of <strong>Cap</strong>ped RNA) that enables users to combine high-resolution detection of transcription start-sites and differential expression analysis. MAPCap can be used to profile TSS from dozens of samples in a multiplexed way, in 16-18 hours. MAPCap data can be analyzed using our easy-to-use software <strong>icetea</strong> (<a href="https://bioconductor.org/packages/icetea" target="_blank">https://bioconductor.org/packages/icetea</a>), which allows users to detect robust TSS using replicates, and perform differential TSS analysis.</p><br>Keywords: CAGE, Gene expression analysis, Transcription start sites, High-throughput sequencing<br><h2>Introduction</h2><p>The MAPCap protocol described below is discussed in detail in our manuscript: <strong>MAPCap allows high-resolution detection and differential expression analysis of transcription start sites﻿</strong> (Bhardwaj and Semplicio et al., Nature Communications. 2019). </p><p><br></p><p>Please find the full protocol, along with the sequence of oligos, PCR primers and ERCC controls as the attached file (pdf).</p><h2>Reagents</h2><p><strong>Reagents﻿</strong></p><p>Unless indicated otherwise, all buffers were prepared as a stock solution with RNase-free water in freshly autoclaved bottles. Buffers are decanted into Falcon tubes for immediate use and only pipetted out of them. Unless indicated otherwise, all buffers are kept at 4ºC and Falcons with aliquots of buffers are kept on ice.</p><p><br></p><p>1) <strong>10x Depletion Buffer</strong>, 200 mM Tris-HCl (pH 7.4), 400 mM KCl.</p><p>2) <strong>TE Buffer</strong>, 10 mM Tris-HCl (pH 8), 1 mM EDTA.&nbsp;</p><p>3) <strong>2x IPP Buffer</strong>, 100 mM Tris-HCl (pH 7.4), 300 mM NaCl, 0.2% NP-40.&nbsp;</p><p>4) <strong>2x PNK-MES Buffer</strong>, 50 mM MES (pH 6.0), 100 mM NaCl, 20 mM MgCl2, 0.2% Tween-20. Store at room temperature.</p><p>5) <strong>2x ProtK Buffer</strong>, 200 mM Tris-HCl (pH 7.4), 100 mM NaCl, 20 mM EDTA, 0.2% Tween-20.</p><p>6) anti-m7G antibody (SYSY, 201011)</p><p>7) Dynabeads ProteinG slurry magnetic beads (Invitrogen, 10003D)</p><p>8) Quick-RNA Miniprep Kit (Zymo Research, R1054)</p><p>9) Direct-zol RNA Microprep Kit (Zymo Research, R2061)</p><p>10) RNA Clean &amp; Concentrator Kit (RCC, Zymo Research, R1013)</p><p>11) Oligo Clean &amp; Concentrator Kit (OCC, Zymo Research, D4060)</p><p>12)&nbsp;RNase-free water (Invitrogen, AM9906)</p><p>13) Ampure XP beads (Beckman Coulter, A63880)</p><p>14) SuperScript III First-Strand Synthesis System (Invitrogen, 18080051)</p><p>15) ERCC RNA Spike-In Mix (Invitrogen, 4456740)</p><p>16) T7-FlashScribe Transcription Kit (CellScript, C-ASF3507)</p><p>17) MEGAclear Transcription Clean-Up Kit (Invitrogen, AM1908)</p><p>18) RNA 5’ Polyphosphatase (Epicentre (ordered through Lucigen), RP8092H)</p><p>19) RNase H (Invitrogen, 18021014)</p><p>20) RNasin Plus RNase inhibitor (Promega, N261A)</p><p>21) Terminator 5’-Phosphate-Dependent Exonuclease (Epicentre (ordered through Lucigen), TER51020)</p><p>22) Phusion High-Fidelity DNA Polymerase (NEB, M0530S)</p><p>23) Vaccinia Capping System (NEB, M2080S)</p><p>24) rSAP (NEB, M0371L)</p><p>25) Proteinase K (Invitrogen, 100005393)</p><p>26) CircLigase II ssDNA Ligase (Epicentre (ordered through Lucigen), CL9021K)</p><p>27) NEBNext High-Fidelity 2x PCR Master Mix (NEB, M0541S)</p><p>28) NEBNext Multiplex Oligos for Illumina (Primer Set 1-4)</p><p><br></p><h2>Equipment</h2><p>* Eppendorf pipettes: 1ul to 1ml</p><p>* Eppendorf tubes</p><p>* Falcon tubes: 15 and 50ml</p><p>* Ice bucket with ice</p><p>* Vortex mixer</p><p>* Tube rotator</p><p>* Magnetic rack</p><p>* ThermoMixer</p><p>* Thermocycler</p><p>* Covaris E220 Focused Ultrasonicator</p><p>* Covaris micro snap tubes</p><p>* Mini centrifuge</p><p>* Eppendorf Bench-top centrifuge</p><p>* Qubit RNA/DNA HS</p><p>* Fragment analyser/Bioanalyser</p><p>* Real Time PCR instrument &amp; equipment</p><p>* Illumina HiSeq/MiSeq/NextSeq</p><p><br></p><h2>Procedure</h2><p><strong>Preparation</strong></p><p><br></p><p><strong>Generation of capped ERCC spikes&nbsp;</strong></p><p>Ten spike sequences were chosen from the ERCC RNA Spike-In Mix and PCR primers were designed to produce ~500 bp long DNA fragments (Table 1). At the 5’ end, we inserted a T7 class II promoter ɸ 2.5 which creates more homogenous 5’ end transcription promoter sequences. The general PCR forward primer was designed to anneal to the T7 transcription site while the individual reverse primers anneal to the respective ERCC spike sequence.</p><p><br></p><p>1)	Reverse transcribe 1 µg of ERCC RNA Spike-In Mix with random hexamers using SSIII according to manufacturer’s instruction.</p><p>2)	Top-up to 50 µl volume with RNase-free water.</p><p>3)	Perform PCR with ~10 ng of reverse transcribed ERCC spike mix as template.</p><p>4)	Clean-up PCR reaction using OCC according to manufacturer’s instruction. Elute in 10 µl of RNase-free water.</p><p>5)	Perform in vitro transcription with 500 ng of template DNA using the T7 FashScribe Transcription Kit according to manufacturer’s instruction.</p><p>6)	Clean up using MEGAclear Transcription Clean-up Kit according to manufacturer’s instruction.&nbsp;</p><p>7)	Perform 5’ end capping using the Vaccinia Capping System according to manufacturer’s instruction.</p><p>8)	Clean-up using OCC. Elute in 17.5 µl RNase-free water.</p><p>9)	Measure RNA either by Qubit or Nanodrop.</p><p>10)	Prepare the Polyphosphatase master mix as follows:</p><p>    ○	10x Reaction Buffer 2 µl</p><p>    ○	RNasin+ 0.5 µl</p><p>    ○	RNA 5’ Polyphosphatase (20 U/µl) 1µl</p><p>    ○	RNA in water 16.5 µl (not more then 10 µg)</p><p>11)	Incubate at 37ºC for 30 min.</p><p>12)	Clean-up using OCC. Elute in 16.5 µl RNase-free water.</p><p>13)	Prepare the Terminator exonuclease master-mix as follows:</p><p>    ○	10x Reaction Buffer 2 µl</p><p>    ○	RNasin+ 0.5 µl</p><p>    ○	Terminator Exonuclease (1 U/µl) 1µl</p><p>    ○	RNA in water 16.5 µl</p><p>14)	Incubate at 30ºC for 60 min.</p><p>15)	Clean-up using OCC. Elute in 10 µl.</p><p>16)	Measure the concentration of each spike with Qubit.</p><p>17)	Create a master-mix of all 10 spikes as a 2-fold serial dilution</p><p>     ○	Example: take from one spike 64 ng, from the second one, 32 ng…and from the last spike, 0.125 ng. This results in a spike master mix with a concentration of 2.5 ng/ul.&nbsp;</p><p><br></p><p><br></p><p><strong>PROCEDURE</strong></p><p><br></p><p><strong>RNA Isolation</strong></p><p>RNA can be isolated from any source of interest. We use either the Quick-RNA Microprep Kit for cells in suspension or the Direct-zol Microprep Kit for tissues and whole larvae according to manufacturer’s instruction.&nbsp;</p><p><br></p><p><strong><em>CRITICAL: </em></strong><em>To avoid any contamination with DNA, treat the RNA samples with DNAse. Elute in RNase-free water.</em></p><p><br></p><p><strong>RNA Preparation</strong></p><p><strong>Preparing the beads for IP</strong></p><p>1.	Wash 25 µl of Dynabeads ProteinG slurry magnetic beads per sample with IPP buffer and resuspend in at least 300 µl IPP buffer. (e.g. For 12 samples we used 300 µl of beads, which after washing were resuspended in 1 ml of IPP buffer.)</p><p>2.	For each IP, we use 2.5 µg of anti-m7G antibody.</p><p>3.	Add the antibody to the beads in IPP and incubate at 4ºC rotating for at least 1 hour.</p><p>4.	Wash the beads three times with IPP buffer, resuspend in IPP buffer and make aliquots of 150 µl per sample</p><p><br></p><p><strong>Preparing total RNA for IP</strong></p><p>Capped RNAs within the cell mainly consist of snRNAs and snoRNAs. To avoid losing sequencing power and increase detection sensitivity, we have therefore developed a targeted depletion approach using DNA antisense oligos followed by RNase H treatment. These oligos were designed to bind within the first 40 nt of small abundant capped RNAs. The most abundant sn- and snoRNAs based on a first run without depletion were chosen as targets (Table 2). The thereby resulting 20-40 nt long capped RNA species are then removed through column purification using the RNA Clean and Concentrator Kit.&nbsp;</p><p><br></p><p>rRNA species do not contain a cap and are therefore excluded from the IP. However, due to their abundance, it is possible that a substantial portion of rRNAs can stick to the beads. To reduce the number of reads coming from rRNAs, we implemented a similar approach for rRNA removal as for the abundant RNAs (Table 3). In addition (also possible as a stand alone solution) we treat the RNA sample with Terminator Exonuclease, which degrades RNA species with a single phosphate group at their 5’ end.</p><p><br></p><p><strong><em>Optional: polyA RNA isolation</em></strong></p><p>If the desired RNA species contains a polyA tail, we recommend a polyA RNA isolation prior to performing MAPCap. This is the most effective way to remove unwanted capped and/or abundant RNA species (sn-, sno- and rRNAs). Starting from polyA RNA, you can directly continue from the fragmentation step of the procedure.&nbsp;</p><p><br></p><p><strong><em>Optional: Adding capped RNA spike-ins</em></strong></p><p>Add 0.05% of capped RNA spike master mix to each sample. E.g. for an RNA starting amount of 5 µg, add 2.5 ng of capped RNA spike master mix.</p><p><br></p><p><strong>Abundant RNA removal</strong></p><p><strong>Starting volume: 26 µl</strong></p><p>1.	Prepare a master mix containing 4 µl of 10x Terminator buffer (from the Terminator Kit), 4 µl AS-oligo mix against sn/snoRNAs (6 µM) and 4 µl AS-oligo mix against rRNAs (4 µM).&nbsp;</p><p>2.	Add 12 µl of AS-master mix to each sample, incubate the RNA at 70ºC for 2 min followed by gradual cooling to 37ºC (for this, we changed the temperature of the thermocycler from 70ºC to 37ºC. This takes around 10 min.)</p><p>3.	Once the samples have reached 37ºC, add 1 µl of RNase H (2 U/µl) and incubate at 37ºC for 30 min at 1100 rpm interval shaking (30 s on, 120 s off).</p><p>4.	Incubate the samples at 70 ºC for 5 min, put on ice immediately for at least 1 min, and add 2 µl of 5’ Terminator Exonuclease (1 U/µl). Incubate for 60 min at 30 ºC at 1100 rpm interval shaking (30 s on, 120 s off).</p><p>5.	Clean-up using the RCC Kit according to manufacturer’s instruction with slight modifications:</p><p>a.	Prepare an adjusted RNA Binding Buffer as follows: 100 µl Binding Buffer + 50 µl EtOH (&gt;95%).</p><p>b.	Elute in 100 µl TE buffer.</p><p>6.	Transfer the sample to Micro Snap Cap Tubes (Covaris) for fragmentation.</p><p>7.	We used the Covaris E220 Focused Ultrasonicator with the following settings: Duty cycle: 10%, Intensity: 5, Power: 175 W, Cycles/Burst: 200, time: 180s).&nbsp;</p><p><br></p><p><strong><em>CRITICAL:</em></strong><em> It is important to obtain RNA of a uniform size of around 200-400 nt in order to achieve ideal conditions for sequencing. We recommend testing the fragmentation in advance with several conditions (Figure 1).</em></p><p><br></p><p><strong>IP</strong></p><p>1.	Transfer the sonicated RNA in 100 µl TE buffer to the 150 µl aliquots of antibody coupled beads.</p><p>2.	Resuspend the mix and incubate at 4ºC for 1-2 hours.</p><p><br></p><p><em>General remarks regarding washing beads:</em></p><p>●	To properly remove all liquid from the beads for washing and resuspension in a new solution, we perform a short spin to remove any liquid that has accumulated in the lid of the tube.</p><p>●	We then place the tube on a magnet and wait until the beads are properly bound before removing the solution with a vacuum pump or manual pipetting.</p><p>●	After the last removal of the washing buffer, we take the tubes from the magnet and perform another quick spin on a table centrifuge before placing the tubes back on the magnet. This allows us to collect the remaining liquid at the bottom of the tube with a 10 µl pipet.</p><p><br></p><p><br></p><p><em>General remarks regarding resuspension of beads in a new reaction mix:</em></p><p>●	Most reactions are carried out in 20 µl volume. It is therefore important to remove all excess volume from the wash step before continuing with the protocol (see general remarks on washing, point 3).</p><p>●	We elute beads by dispensing the 20 µl from the wall of the tube where the beads were fixed to the side. Repeat this step until all beads are off the wall and gently resuspend until the beads are homogeneously in solution.</p><p><br></p><p><strong>Library preparation</strong></p><p>1.	Wash the beads 3 times with IPP buffer. Add at least 500 µl of IPP buffer and rotate the beads at 4ºC for 5 min.</p><p>2.	Prepare PNK-MES master mix as follows:</p><p>    ○	2x PNK-MES Buffer		10 µl</p><p>    ○	b-Mercaptoethanol (0.1 M)	1 µl</p><p>    ○	T4 PNK (10 U/µl)		1 µl</p><p>    ○	RNasin+ (40 U/µl)		0.5 µl</p><p>    ○	RNase-free water		7.5 µl</p><p>3.	Resuspend the beads in 20 µl of PNK-MES master mix.</p><p>4.	Incubate at 37ºC for 30 minutes in a thermocycler with 1100 rpm interval shaking (30 s on / 120 s off).</p><p>5.	Wash the beads 3x with cold IPP buffer.</p><p>6.	Prepare Ligation master mix as follows:</p><p>    ○	10x T4 Ligase Buffer		2 µl</p><p>    ○	ATP (1 mM)			2 µl</p><p>    ○	RNase-free water		7.5 µl</p><p>    ○	Add 6 µl of PEG-8000 to this solution and mix by flicking the tube and pipetting until the PEG is properly dissolved.</p><p>    ○	T4 RNA Ligase 1 (10 U/µl)	1 µl</p><p>    ○	RNasin+ (40 U/µl)		0.5 µl</p><p>7.	Resuspend the beads in 19 µl of Ligation master mix.</p><p>8.	Add 1 µl of s-oligo (10 µM) to each sample, mix the beads by flicking followed by a quick spin.</p><p><br></p><p><strong><em>NOTE:</em></strong><em> Use individual s-oligos for each sample. Afterwards, the samples will be pooled for sequencing and computational demultiplexing will be performed based on the s-oligo sample barcode.</em></p><p><br></p><p><strong><em>CRITICAL:</em></strong><em> Adjust the s-oligo concentration added to the sample based on the starting amount of RNA used. For 5 ug we use 1 µl of a 10 µM solution.&nbsp;</em></p><p><br></p><p>9.	Incubate at 25ºC for 1 hour in a thermocycler with 1100 rpm interval shaking (30 s on / 120 s off).</p><p>10.	Wash the beads 3x with cold IPP buffer. After 2 washes, resuspend one sample with 900 µl cold IPP buffer and transfer to every subsequent sample to combine all samples. Transfer to a new Eppendorf tube.</p><p>11.	Prepare the rSAP master mix as follows:</p><p>    ○	IPP buffer 		35 µl</p><p>    ○	RNasin+ (40 U/µl)	1 µl</p><p>    ○	rSAP (1 U/µl)		4 µl</p><p>12.	Resuspend the beads in 40 µl of rSAP master mix and incubate at 37ºC for 30 min with 1100 rpm interval shaking (30s on / 120 s off).</p><p>13.	Wash beads 3x with IPP buffer.</p><p>14.	Prepare the ProtK master mix as follows:</p><p>    ○	2x ProtK Buffer		50 µl</p><p>    ○	10% SDS			2 µl</p><p>    ○	Proteinase K (20 mg/ml)	10 µl</p><p>    ○	RNase-free water		38 µl</p><p>15.	Resuspend the beads in 100 µl ProtK master mix and incubate at 37ºC for 20 min with 1100 rpm interval shaking (30 s on / 120 s off).</p><p>16.	Place the tube on the magnet and transfer the supernatant to a new tube. The beads can be discarded at this point.</p><p>17.	Clean up using OCC according to manufacturer’s instruction.</p><p>18.	Elute in 9 µl of RNase-free water.</p><p>19.	Prepare Reverse Transcription master mix using the SuperScript III Kit as follows:</p><p>    ○	10x Buffer		2 µl</p><p>    ○	dNTP (10 mM)	1 µl</p><p>    ○	MgCl2 (25 mM)	4 µl</p><p>    ○	DTT (0.1 M)		2 µl</p><p>    ○	RNaseOUT (40 U/µl)	1 µl</p><p>    ○	SSIII (200 U/µl)	1 µl</p><p>20.	Add the 11 µl Reverse Transcription master mix to the sample and incubate for 10 min at 42ºC, 10 min at 50 ºC, 10 min at 55 ºC and 10 min at 65 ºC with 1100 rpm interval shaking (30 s on / 120 s off). Increase the temperature in the thermocycler after 10 minutes while keeping the sample in the cycler.</p><p>21.	Purify the sample using OCC according to manufacturer’s instruction.</p><p>22.	Elute the sample with 6.5 µl RNase-free water.</p><p>23.	Prepare the Circularization master mix as follows:</p><p>    ○	10x Buffer			1 µl</p><p>    ○	MnCl2 (50 mM)		0.5 µl</p><p>    ○	Betaine (5 M)			2 µl</p><p>    ○	CircLigaseII (100 U/µl)	0.5 µl</p><p>24.	Add the 4 µl of Circulaization master mix and incubate for 1-16 hours at 60 ºC. (We usually perform this step overnight in an incubator to avoid condensation on the wall and lid.)</p><p><br></p><p><br></p><p><strong>Library amplification</strong></p><p>In order to perform as few PCR amplification cycles as possible, we recommend performing a qPCR on the circularized DNA template and compare the cycle numbers to a standard library. The standard library is a previously sequenced MAPCap library where the concentration is known. We aim at a library of around 2-4 ng/µl in 10 µl final volume after PCR and Ampure beads clean-up. We use 1 µl of circularized cDNA which is serially diluted 3 times to get different measurements with different concentrations. Typically, MAPCap requires between 14 and 18 cycles of PCR depending on the starting amount used.</p><p><br></p><p>1.	Set up a PCR reaction:</p><p>    ○	2x NEBNext PCR master mix		20 µl</p><p>    ○	Universal primer P5 (10 µM)		1 µl</p><p>    ○	Index primer P3 (10 µM)		1 µl</p><p>    ○	Circularized cDNA library		9 µl</p><p>    ○	RNase-free water			9 µl</p><p>2.	PCR program: 1. 98 ºC for 30 sec, 2. 98 ºC for 10 sec, 3. 65 ºC for 30 sec, 4. 72 ºC for 20 sec, 5. Repeat 2-4 for desired amount of cycles, 6. 72 ºC for 5 min, 7. 4 ºC forever.</p><p>3.	Clean-up the amplified library with two rounds of Ampure beads at a 1x ratio.</p><p>    ○	Equilibrate Ampure XP beads at room temperature for 30 min and vortex the beads well before use.</p><p>    ○	Add 40 µl (1x ratio) of Ampure beads to the library and mix well.</p><p>    ○	Incubate 5 min at room temperature before placing the tube on a magnet.</p><p>    ○	Once beads are fixed to the magnet and the solution is clear, remove 75 µl of the liquid. (Caution: Do not disturb the beads.)</p><p>    ○	Wash 2x with 200 µl 80% EtOH without removing the beads from the magnet.</p><p>    ○	Dry the beads at room temperature while on the magnetic stand for 5 min (do not over-dry).</p><p>    ○	Remove the tube from the magnet and resuspend the beads in 20 µl of RNase-free water.</p><p>    ○	Reclaim beads on a magnet and collect the sample in a new tube.</p><p>    ○	Add 20 µl (1x ratio) of Ampure beads to the library and mix well.</p><p>    ○	Incubate 5 min at room temperature before placing the tube on a magnet.</p><p>    ○	Once beads are fixed to the magnet and the solution is clear, remove 35 µl of the liquid. (Caution: Do not disturb the beads.)</p><p>    ○	Wash 2x with 200 µl 80% EtOH without removing the beads from the magnet.</p><p>    ○	Dry the beads at room temperature while on the magnetic stand for 5 min (do not over-dry).</p><p>    ○	Remove the tube from the magnet. Resuspend the beads in 13 µl of RNase-free water. Reclaim beads on a magnet and collect the sample in a new tube.</p><p>4.	Proceed with quantification of the DNA concentration (Qubit DNA HS) and size distribution control using Bioanalyzer/Fragment Analyzer. An example of a final MAPCap library is shown in Figure 2. Libraries should be clean of adapter dimers (peak around 140 bp).</p><p>5.	Final libraries can be stored at -20ºC.</p><p>6.	Sequence the library following Illumina guidelines (50-75 bp, paired-end, 6 bp index reads).</p><p><br></p><h2>Time Taken</h2><p><strong>Total: 16-18&nbsp;hours (calculated for 12 samples)</strong></p><p>●	Coupling antibody to the beads:<strong><em>1-2 hours</em></strong></p><p>●	Abundant RNA depletion: <strong><em>2.5 hours</em></strong></p><p>●	RNA fragmentation: <strong><em>1 hour</em></strong></p><p>●	IP: <strong><em>1-2 hours</em></strong></p><p>●	3’ end repair (PNK): <strong><em>30 minutes</em></strong></p><p>●	Ligation: <strong><em>1 hour</em></strong></p><p>●	S-oligo dephosphorylation (rSAP): <strong><em>30 minutes</em></strong></p><p>●	Release RNA from beads (ProtK): <strong><em>20 minutes</em></strong></p><p>●	Reverse transcription: <strong><em>1.5 hours</em></strong></p><p>●	Circularization: <strong><em>2 hours (or over night)</em></strong></p><p>●	qPCR: <strong><em>2 hours</em></strong></p><p>●	PCR: <strong><em>1 hour</em></strong></p><p>●	Final clean-up: <strong><em>30 minutes</em></strong></p><p><br></p><p>●	Coupling antibody to the beads: <strong><em>1-2 hours</em></strong></p><p>●	Abundant RNA depletion: <strong><em>2.5 hours</em></strong></p><p>●	RNA fragmentation: <strong><em>1 hour</em></strong></p><p>●	IP: <strong><em>1-2 hours</em></strong></p><p>●	3’ end repair (PNK): <strong><em>30 minutes</em></strong></p><p>●	Ligation: <strong><em>1 hour</em></strong></p><p>●	S-oligo dephosphorylation (rSAP): <strong><em>30 minutes</em></strong></p><p>●	Release RNA from beads (ProtK): <strong><em>20 minutes</em></strong></p><p>●	Reverse transcription: <strong><em>1.5 hours</em></strong></p><p>●	Circularization: <strong><em>2 hours (or over night)</em></strong></p><p>●	qPCR: <strong><em>2 hours</em></strong></p><p>●	PCR: <strong><em>1 hour</em></strong></p><p>●	Final clean-up: <strong><em>30 minutes</em></strong></p><p><br></p><h2>Anticipated Results</h2><p><strong>1. Insert size of final sequencing library</strong>: should be in the range of 200-500 bp. See the "good library" figure (Fig. 2)</p><p><br></p><p><strong>2. Number of mapped and de-duplicated reads: </strong>Ideally, a good library should have high mapping efficiency (&gt;80% uniquely mapped) and low number of PCR duplicates. The proportion of PCR duplicates varies depending on the number of PCR cycles (which depends on the amount of starting material), but &gt;90% PCR duplicates (compared to uniquely mapped reads) should be concerning as it may lead to low number of kept reads for TSS detection.</p><p><br></p><p><strong>3. Reads within TSS: </strong>Generally, &gt;60% of duplicate-free reads should be expected to fall within the detected TSSs. Although this number can be increased by modifying the TSS detection parameters (see the <a href="https://bioconductor.org/packages/release/bioc/vignettes/icetea/inst/doc/mapcap_analysis.html" target="_blank">icetea vignette</a>), this should be kept in mind that there's a trade-off between sensitivity and specificity during TSS detection.</p><h2>References</h2><p>Kodzius, R. et al. CAGE: cap analysis of gene expression. Nat. Methods 3, 211–222 (2006).</p><p><br></p><p>Aktaş, T. et al. DHX9 suppresses RNA processing defects originating from the Alu invasion of the human genome. Nature 544, 115–119 (2017).</p><h2>Acknowledgements</h2><p>The authors acknowledge the Deep-Sequencing Unit at MPI-IE for data production.&nbsp;Ibrahim Illik for sharing the s-oligo and the FLASH protocol, and Plamen Georgeiv for help with MLE mutant and wild-type flies.&nbsp;</p></body></html>