<html><head><title>Quantification of Translation efficiencies</title></head><body><h1>Quantification of Translation efficiencies</h1>Authors: Madeleine Huber, JÃ¶rg Soppa<br>Abstract: <p>The determination of translational efficiencies for single transcripts of interest is described for <em>E. coli</em> and <em>H. volcanii</em>. In both cases, the transcript levels are quantified by Northern blot analyses, and the protein levels are quantified by reporter enzyme assays</p><br>Keywords: translational efficiency, translational regulation, northern blot, enzyme assay, reporter gene<br><h2>Introduction</h2><p>The translation efficiencies of mRNAs can vary widely, and they can be subject to differential translational regulation. The expression of up to about 25% of all genes were found to be effected by translational regulation in halophilic Archaea [1] and various species of eukaryotes [2,3]. Translatome analyses can give overviews of the translational status of all transcripts and can lead to the identification of transcripts that are subjects to differential translational control. However, unraveling the underlying molecular mechanisms require the in depth characterization of selected transcripts. A fast and convenient method to quantify the translational efficiencies of transcripts of interest is to generate translational fusions with reporter genes. The translational efficiencies can then be determined by quantifying the level of transcripts via Northern blot analyses and quantifying the level of translated proteins via reporter enzyme assays. This contribution described the determination of translation efficiencies in two prokaryotic model species, i.e. the bacterium <em>Escherichia coli</em> and the archaeon <em>Haloferax volcanii.</em></p><p><br></p><h2>Procedure</h2><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Grow the cells in the medium/under the conditions of interest. The determination of translational efficiencies via Northern blot analyses and reporter gene assays is only meaningful when both transcript levels and protein levels are in steady state, e.g. during exponential growth. It is noteworthy that transcript half-lives and protein half-lives are very different. For example, the average transcript half-life in <em>E. coli</em> is 2-3 minutes and in <em>H. salinarum</em> is about 8 minutes. In contrast, proteins are stable at least for hours. For the determination of translation efficiencies under non-steady state conditions, it is necessary to use a strictly regulated promoter, induce transcription under the condition of interest, and then quantify the transcript and protein levels.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Perform RNA Isolation and Northern blot analysis for the fusion transcript between the transcript of interest and the transcript of the reporter gene. A probe against the reporter gene can be used for all transcripts of interest. The relative transcript levels can be quantified using the program ImageJ. The method has been deposited to the Protocol Exchange and has the doi <strong>10.21203/rs.2.11264/v1</strong>.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Perform enzyme assay for the chosen reporter gene to quantify the volume activity (U/ml or nkat/ml).</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Quantify the protein concentration, e.g. by using the BCA assay (mg/ml)</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Calculate of the specific enzyme activity by dividing the volume activity with the protein concentration (U/mg or nkat/mg). The methos has been deposited to the Protocol Exchange for two reporter enzymes for <em>E. coli</em> and one reporter enzyme for <em>H. volcanii</em>, which have got the following DOIs:</p><p>Reporter enzyme dihydrofolate reductase for <em>H. volcanii</em>: <strong>10.21203/rs.2.11263/v1</strong></p><p>Reporter enzyme beta-glucoronisase for <em>E. coli</em>: <strong>10.21203/rs.2.10596/v1</strong></p><p>Reporter enzyme glycerol-3-phosphate dehydrogenase for <em>E. coli</em>: <strong>10.21203/rs.2.10595/v1</strong></p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For calculation of the translation efficiencies the specific enzyme activity is divided by the transcript level:</p><p>Translation efficiency = specific enzyme activity / transcript level</p><p><br></p><h2>References</h2><p>[1] C. Lange, A. Zaigler, M. Hammelmann, J. Twellmeyer, G. Raddatz, S.C. Schuster, D. Oesterhelt, J. Soppa (2007) Genome-wide analysis of growth phase-dependent translational and transcriptional regulation in halophilic Archaea. <em>BMC Genomics</em> 8:415.</p><p>[2] R.A. Crawford, G.D. Pavitt (2019) Translational regulation in response to stress in <em>Saccharomyxes cerevisiae</em>. <em>Yeast</em> 36:5-21.</p><p>[3] Y. Wang, H. Thang, J. Lu (2019) Recent advances in ribosome profiling for deciphering translational regulation. <em>Methods</em> doi: 10.1016/j.ymeth.2019.05.011.</p></body></html>