<html><head><title>Laser Capture Microdissection (LCM) and 3D Sample Tracking Protocol</title></head><body><h1>Laser Capture Microdissection (LCM) and 3D Sample Tracking Protocol</h1>Authors: Shaina Robbins, Sirisha Achanta, Rajanikanth Vadigepalli<br>Abstract: <p>This protocol demonstrates the use of Laser Capture Microdissection (LCM), a precise cell isolation technique, for acquiring single neurons from right atrial ganglionated plexus (RAGP) porcine heart tissue in sync with image acquisition to enable 3D location tracking of collected samples. Fresh frozen RAGP tissue sectioned and stained for neurons (Nissl stain) is processed through LCM for sample collection. The LCM workflow is comprised of four steps: instrument setup, slide loading and tissue inspection, microdissection and image acquisition and sample processing and storage. LCM collected samples were processed for gene expression experiments : High-throughput-qRT-PCR or Single cell RNA sequencing. Gene expression data along with 3D sample tracking was used to generate a functional map of porcine RAGP. The techniques described in this protocol can be adapted to a wide variety of sample and tissue types with minor modifications. This protocol is a part of a protocol pipeline that includes cryosectioning and staining protocols upstream and Tissue Mapper protocol, High-Throughput q-RT-PCR and RNA Seq protocols downstream.</p><p><br></p><br>Keywords: Laser Capture Microdissection, LCM, Single cell, Neuron<br><h2>Introduction</h2><p>The present protocol was developed as part of the National Institutes of Health SPARC project efforts to construct a comprehensive anatomical, molecular and functional map of the peripheral nervous system at the visceral organs. A key goal is to develop an anatomical framework onto which various data sets can be mapped, e.g., spatial location and distribution of hundreds to thousands of single neurons within a tissue context, molecular profiles of spatially-tracked single neurons, etc. Here we aim to isolate neurons from the right atrial ganglionic plexus (RAGP) of the porcine heart. Pig hearts at the RAGP were injected with Fluorescent labeled retrograde neuronal tracer Fast Blue 5 weeks prior to sacrifice to stain RAGP neurons that project to the sinoatrial node (SAN) of the heart. Neurons SAN projecting and non-projecting were isolated using LCM and processed for gene expression using&nbsp;either High-throughput qRT-PCR or RNA Seq (<em>For more information see our protocol “ Single Cell High-Throughput qRT-PCR protocol”</em>).</p><p>The steps below constitute a protocol for sample collection using LCM and acquiring images of samples collected for constructing a 3D anatomical framework (<em>For more information on 3D tissue reconstruction see our protocol “Single-Cell Mapping and 3D Tissue Reconstruction using Cryosection-derived Images and Tissue Mapper software</em>”). This approach starts with cryosectioning pig RAGP fresh frozen tissue at 30 micrometer thickness and Nissil staining for neurons (<em>For more information see our protocol “ Cryosectioning, blockface Imaging and Nissl staining fluorescently labeled pig heart”</em>).&nbsp;Stained slides are loaded on to the LCM instrument (Arcturus XT), neurons are identified using Nissil stain (PAN neuronal stain) and SAN projecting neurons are identified using Fast Blue stain. Both Nissil and Fast Blue are visualized under fluorescence. Identified neurons are “cut” from the tissue using a UV laser and attached to LCM Caps using IR laser. Images are acquired during sample identification – to generate an overview of the sample location on the slide; and during and after microdissection- to generate images for precision sample tracking at high magnification. Neurons on LCM caps are then stored at -80⁰C for downstream processing.</p><p><strong>Note:</strong> Arcturus XT instrument training is recommended prior to using this protocol.</p><p><strong>Note: </strong>See supplemental files for a user friendly version of this protocol.</p><h2>Reagents</h2><p>1.&nbsp;&nbsp;&nbsp;&nbsp;CapSure™ HS LCM Caps&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Life Technologies&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;LCM0214, LCM0215</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Metal frame PPS slides 4 micron&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Leica Microsystems Inc.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;11600294</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;PCR clean microcentrifuge tubes 0.5mL&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Eppendorf™&nbsp;&nbsp;&nbsp;022363719</p><p><br></p><h2>Equipment</h2><p>1.&nbsp;&nbsp;&nbsp;&nbsp;ArcturusXT™ Laser Capture Microdissection (LCM) System</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;ArcturusXT™&nbsp;Software v3.4</p><p><br></p><h2>Procedure</h2><p>1.1 Instrument Setup</p><p><strong>Note:</strong> Refer to the ArcturusXT™ Laser Capture Microdissection (LCM) System user guide for in depth supplemental information.&nbsp;</p><p><u>STEP 1: </u>Turn on the computer.</p><p><u>STEP 2</u>: Turn on the Eclipse™ Ti-E microscope base using the switch located on the back of the microscope</p><p><u>STEP 3</u>: Turn on the ArcturusXT™ Instrument. Facing the microscope, the power button is located on the left side of the instrument. </p><p><u>STEP 4</u>: Start the software by clicking the ArcturusXT™ icon on the Windows™ desktop, or click Start, point to Programs, and click the ArcturusXT™ option. The software opens to fill the screen.</p><p>1.2&nbsp;Slide Setup</p><p><u>STEP 1</u>: In the setup tool panel click Present Stage</p><p><u>STEP 2</u>: Load your slides and CapSure™ Caps onto the microscope stage.</p><p><strong>Note:</strong> For every cap in the cassette, make sure the corresponding cap offload position (on the right side of the work surface) is empty.</p><p><u>STEP 3</u>: On the Load Caps panel click on the box next to HS or Macro depending on which cap you will be using. Then click the empty circles to represent how many caps were loaded. </p><p>IMPORTANT: The system does not have a sensor to detect caps on the loading station or at the QC station. If you do not indicate to the system that there is a cap is at the QC station and try to offload another cap to the same position, you can cause damage to the microscope. </p><p><u>STEP 4:</u> Use the Slide panel to select which slot your working slide was placed (A, B or C). </p><p><u>STEP 5:</u> To choose the slide type being used, you must click the box to the right of the letter slot chosen in step 4. MEM= membrane slide, FRM= frame slide and no check mark indicates a glass slide. </p><p><strong>Note:</strong> If using frame slides, a clean glass slide should be placed under the tissue. This glass slide serves as support for the tissue during this procedure. Place a clean glass slide on the “flat” side of the membrane slide to act as a solid support. The tissue section will be “sandwiched” between the membrane and the glass slide.</p><p><br></p><p>1.2&nbsp;Microdissection and Imaging Sample for Spatial Mapping</p><p><u>STEP 1</u>: Take slide overview image by clicking the camera icon. Right click on the bottom screen where the tiled picture is (this will be blank initially) and in the dialog box select “reacquire overview image”. </p><p><u>STEP 2</u>: Take a tile image of entire tissue section. Click on the tile image button (Figure 2), then in the slide overview image area located at the bottom of the screen, click and drag the curser to outline the whole tissue section to indicate the area for the tiled image.</p><p><u>STEP 3</u>: Using the camera button (outlined in blue, Figure 2), take a 2x, 10x and 20x overview image of your region of interest (<em>Overview ROI</em>). </p><p><u>STEP 4</u>: Place cap (outlined in blue, figure 2) on ROI using the 2x objective. </p><p>IMPORTANT: You can only place cap on slide using the 2x or 10x objective. Any objective above 10x will knock the microscope stage out of place and potentially crash the system.</p><p><u>STEP 5</u>: The IR laser spot diameter and power needs to be calibrated and centered each time after placing the cap over your region of interest (ROI). Once the cap is placed, locate a clear (non-tissue) area as close as possible to your sample to be collected. This is where you will test fire, measure and set the IR spot size and strength (see the Arcturus manual for further instruction). </p><p>IMPORTANT: This step is required prior to marking your sample for collection each time a cap is placed on a slide. </p><p><u>STEP 6</u>: Using the freehand drawing button (Figure 2) draw a circle around the sample to be collected using either the 10x objective (<em>ROI Marked</em>).</p><p><strong>Note:</strong> There are four different color-coded capture groups A, B, C and D. You can use different capture groups to indicate different sample types, e.g. FB labeled samples and non-FB labeled samples. </p><p><strong>Note:</strong> If using UV laser to help obtain samples, the rule of thumb is to draw the marking at least 20 microns away from your sample to prevent any damage to your sample.</p><p><strong>Note:</strong> More images can be taken for publication purposes, i.e fluorescent images as well as other levels of magnification.</p><p><u>STEP 7</u>: Once IR laser is calibrated and the sample area is marked, click on the cut and capture button indicated in figure 2.</p><p><u>STEP 8</u>: Move cap to QC (figure 2) and take 2x and 10x images to confirm sample collection on the cap at the QC station.</p><p><strong>Note:</strong> Use the 2x objective to confirm no other pieces of tissue were picked up elsewhere on the cap. </p><p><strong>Note:</strong> Use the 10x objective for confirmation of your sample and for a closer inspection of questionable areas of excess tissue that would need to be ablated. Ablate any excess tissue that may have adhered to the cap by scribbling over the undesired area using a separate capture group from than those used for sample collection. </p><p><u>STEP 9</u>: Present stage to collect sample. Samples on caps should be removed from the QC station, inserted onto a 0.5ml microcentrifuge tube and placed on dry ice to be stored at -80°C until lysis. </p><p><u>STEP 10</u>: In the Select panel under <em>Drawing Items</em>, click on the sample number pertaining to the sample just collected to bring the stage back to that region. Confirm sample is gone and take image (<em>ROI After Picked</em>). Repeat STEPS 3-9 until all desired samples are collected. </p><p><u>STEP 11</u>: Using the 2x objective, take a tile image of entire tissue section after all samples have been collected from that section (<em>Tile Image After</em>).</p><p>Note: Suggested titles for the images taken during the above steps are in italicized parenthesis.</p><p><br></p></body></html>