<html><head><title>Protocol for Single-Cell RNA-Sequencing of Cryopreserved Human Liver Cells</title></head><body><h1>Protocol for Single-Cell RNA-Sequencing of Cryopreserved Human Liver Cells</h1>Authors: Nadim Aizarani, Antonio Saviano, Sagar, Laurent Mailly, Sarah Durand, Patrick Pessaux , Thomas F. Baumert, Dominic Grün<br>Abstract: <p>This is a protocol describing how to perform scRNA-seq of cryopreserved human liver cells using the mCEL-Seq2 protocol as described in the following paper.</p><p>Title: A Human Liver Cell Atlas reveals Heterogeneity and Epithelial Progenitors</p><p>Authors: Nadim Aizarani, Antonio Saviano, Sagar, Laurent Mailly, Sarah Durand, Patrick Pessaux, Thomas F. Baumert, Dominic Grün</p><p>Journal: Nature</p><br>Keywords: Single-cell RNA-Sequencing (scRNA-seq), FACS, human liver cells, cryopreserved<br><h2>Introduction</h2><p>This protocol enables sorting based scRNA-seq of the main liver cell types found in the normal human liver. Viable cells can be sorted for scRNA-seq in an unbiased fashion or on the basis of cell surface markers as demonstrated in Aizarani et al. (see Aizarani et al. (2019) A Human Liver Cell Atlas reveals Heterogeneity and Epithelial Progenitors. Nature.<sub><span class="ql-cursor">﻿</span>﻿ </sub>)</p><h2>Reagents</h2><p>192 reverse transcription (RT) primers anchored polyT primers having a 6 bp cell barcode, 6 bp unique molecular identifiers (UMIs), a part of 5’ Illumina adapter and a T7 promoter: 40 nM; randomhexRT primer: 5’ GCCTTGGCACCCGAGAATTCCANNNNNN; RNA PCR primers; DNA LoBind 384-well plates; LoBind tubes; Mineral oil (Sigma); dNTPs 10 mM; 0.3% Triton-X-100 in DNase/RNase-free water;&nbsp;DNase/RNase free water; SuperScript II™ Reverse Transcriptase (Invitrogen); RNaseOUT Recombinant Ribonuclease Inhibitor (Invitrogen);&nbsp;Second Strand Buffer (Invitrogen); E. coli DNA Polymerase I (Invitrogen); E. coli DNA ligase (Invitrogen);&nbsp;&nbsp;E. coli Ribonuclease H (Invitrogen);&nbsp;MEGAscript T7 Transcription Kit (Ambion);&nbsp;ExoSAP-IT For PCR Product CleanUp (Affymetrix); NEBNext® Magnesium RNA Fragmentation Module (New England Biolabs); AMPure XP beads (Beckman Coulter); RNAClean XP beads (Beckman Coulter); Qubit dsDNA HS (High Sensitivity) Assay Kit; Agilent High Sensitivity DNA kit; Agilent RNA 6000 Pico Kit; Phusion® High-Fidelity PCR Master Mix with HF Buffer (New England Biolabs); HEPES buffer, EGTA, collagenase Type IV (Sigma-Aldrich), CaCl2, CryoStor® CS10 solution, PBS, absolute undenatured Ethanol, Zombie Green (Biolegend); FACS tubes, 40 micron filters. For more details on reagents used in the mCel-Seq2 protocol, refer to Sagar, et al. (2018). "High-Throughput Single-Cell RNA Sequencing and Data Analysis." Methods Mol Biol <strong>1766</strong>: 257-283.</p><h2>Equipment</h2><p>mosquito®HTS (TTP Labtech), Flow cytometer (BD FACS Aria Fusion or MoFlo XDP Sorter), Agilent Bionalyzer, Qubit fluorometer</p><p><br></p><h2>Procedure</h2><p><strong>I). Sample preparation and FACS:</strong></p><p>1). Prepare 384 well plates containing 1.2 uL of mineral oil and 240 nL of lysis buffer solution using the mosquito nanoLiter pipetting robot)(per well lysis buffer solution contains: 140 nL 0.35% Triton X-100 H2O, 40nL primers (25ng/uL), 40nL RNase free water, 20nL 10mM dNTPs). (see Sagar et al., for more instructions on how to prepare the plates)</p><p>2). Isolate liver cells from human liver specimens after resection following a standard 2-step perfusion protocol (first perfuse with HEPES 0.5mM EGTA for 15 min then perfuse with HEPES containing 0.5mg/mL collagenase and 0.075% CaCl2 for 15 min). Non-viable cells can be removed by Percoll gradient centrifugation. Cells can be split into non-parenchymal cell (NPC) or hepatocyte fractions by spinning at 80-100xg to pellet hepatocytes, or NPCs and hepatocytes can be kept together.</p><p>3). Cryopreserve cells using CryoStor® CS10 solution according to the manufacturer's instructions and store in liquid nitrogen (for long term storage) or at -80<sup>o</sup>C (for short term storage).</p><p>4). On the day of the sort, thaw cryopreserved samples in a water bath at 37<sup>o</sup>C. Spin at 350xg at 4<sup>o</sup>C and remove cryoprotectant solution. Wash the sample with 1XPBS .</p><p>5). Stain the cells with Zombie Green and keep cells on ice for 15 min then wash with 1XPBS.</p><p>6). Perform the relevant antibody staining for your population(s) of interest (see Aizarani et al. for markers and antibodies). Wash sample with 1X PBS.</p><p>7). Filter cells using a 40 micron filter and collect the cells in PBS in a FACS tube. Make sure to dilute the sample well enough for the sort so that is not too turbid.</p><p>8). Calibrate FACS machine for the 384 well plate. Use a 100 micron nozzle for sorting the cells (MoFlo or FACS Aria Fusion), and keep the platform on which the plate is placed cold at all times.</p><p>9). Exclude doubles by FSC and SCC. Gate on viable cells (Zombie Green negative cells) and sort single cells from the population of interest into the 384 well plate containing the lysis buffer solution on the basis of the antibody staining or markers. Shortly vortex the cells before sorting and every 30 min during the sort to avoid clogs.</p><p>10). After the sort is over, place the plate on ice then spin down at maximum speed for 10 min at 4<sup>o</sup>C. Snap freeze the plate and store at -80<sup>o</sup>C. Plates can stored at -80<sup>o</sup>C for months.</p><p><br></p><p><strong>II). ScRNA-seq with mCEL-Seq2:</strong></p><p><u>A). cDNA synthesis and in vitro transcription:</u></p><p>1). Thaw the frozen plate on ice then spin at maximum speed for a few minutes.</p><p>2). Lyse cells: 90°C for 3min in thermocycler (lid: 105°C)</p><p>3). Prepare reverse transcription mix. For one plate, mix 70uL of First Strand Buffer, 35uL 0.1M DTT, 17.5uL RNase Out, 17.5uL Superscript II. Pipette 160nL (per well) of the mix into the plate containing the cells using the mosquito (per well: 80nL First Strand Buffer, 40nL 0.1M DTT, 20nL RNase Out, 20nL Superscript II). (see Sagar et al., for more detailed instructions on pipetting) Spin shortly.</p><p>4). Incubate at 42°C for 1h in thermal cycler (lid: 50°C).Heat inactivate at 70°C for 10min (lid: 85˚C). Put on ice to chill, spin shortly.</p><p>5). Around 20min before the reverse transcription reaction is over, prepare second strand synthesis mix. For one plate, mix 631.4 uL of RNase/DNase free water, 205 uL of Second Strand Buffer, 20.5uL of 10mM dNTPs, 7.38uL Ligase, 28.7uL E.coli DNA Polymerase, 7.38uL RnaseH. Pipette 2.196 uL of the mix per well using the mosquito (per well the second strand synthesis mix contains: 1.54uL DDW (Rnase free water), 0.5 uL Second strand buffer, 50nL 10mM dNTPs, 18nL Ligase, 70nL E.coli DNA Polymerase, 18nL Rnase). (see Sagar et al., for more detailed instructions on pipetting)</p><p>6). Incubate at 16°C for 2h (lid: 25°C)</p><p>7). cDNA Cleanup: Prewarm AMPure XP beads to RT (25°C), vortex beads in buffer until well dispersed. Pool 96 cells in 4 tubes for&nbsp;IVT&nbsp;2,596 µl per cell, spin shortly: Measure volume of pooled samples (4 samples a ~210µl). Distribute each sample in 2 wells(96-well plate) a ~100µl.&nbsp;Add 0,8 volumes of beads (~80 µl). Incubate at RT (25°C) for 10min. Place on magnetic stand for 5min or until liquid appears clear Remove&nbsp;and&nbsp;discard supernatant. Add&nbsp;180 µl of freshly prepared&nbsp;80% EtOH. Incubate at least 30 seconds, remove and discard supernatant, w/o disturbing the beads . Repeat&nbsp;wash. Incubate at least 30 seconds, remove and discard supernatant, w/o disturbing the beads. Air dry beads for 10min or until completely dry. Resuspend&nbsp;&nbsp;with&nbsp;7 uL of water. Incubate at RT (25°C) for 2min. Place on magnetic stand for 5min or until liquid appears clear. Transfer ~6.4µl of supernatant to new tube. Store cDNA at -20<sup>o</sup>C or proceed directly to the IVT.</p><p>8). Prepare IVT mix (7.2 uL of each of the following: A, G, C, U, 10X T7 Buffer, T7 enzyme). Add 9.6 uL of the mix to each tube (4 tubes per plate).</p><p>9). Incubate at 37°C for 13h (lid: 70°C), final step: 4°C hold</p><p>10). Add 6 µl of EXO-SAP enzyme and incubate at 37°C for 15min.</p><p>11). Add 2.44 µl 10x Fragmentation Buffer, flick, Incubate at 94°C for 3min.</p><p>12). Immediately move to ice and add µl 10x Fragmentation Stop Buffer.</p><p>13). Amplified RNA Cleanup: Prewarm RNAClean XP beads to room temperature 25°C (30min) Vortex RNAClean XP beads until well dispersed. Add&nbsp;21.5 µl (0.8 volumes) of beads to sample Incubate at room temperature for 10min. Place on magnetic stand for at least 5min, until liquid appears clear. Remove&nbsp;and&nbsp;&nbsp;discard&nbsp;50 µl of the supernatant. Add 180 µl freshly prepared 70% EtOH Incubate at least 30 seconds, remove and discard supernatant, w/o disturbing the beads Repeat wash 2x times. Air dry beads for 10min or until completely dry. Resuspend&nbsp;with&nbsp;7 µl water, mix thoroughly Incubate at room temperature (25°C) for 2min. Place on magnetic stand for&nbsp;5min or until liquid appears clear Transfer supernatant to new tube. Keep sample at -80°C. Check RNA on Bioanalyzer, and follow the manufacturer's protocol.</p><p><u>B). Library Preparation and sequencing:</u></p><p>1).&nbsp;&nbsp;Prepare the following primer-dNTP mix in a 0.5 ml RNase-free tube (7.5 µl): 5 µl of 10 µM randomhexRT primer and 2.5 µl of 10mM dNTPs. Mix well by flicking and spin down briefly in a mini-centrifuge. Add 1.5 µl to each of the four aRNA samples and incubate at 65ºC (Lid temperature: 80ºC) for 5 min. Transfer samples on ice.</p><p>2).&nbsp;During the 65ºC incubation step, prepare the following RT mix in a 0.5 ml RNase-free tube (20 µl): 10 µl of First Strand Buffer, 5 µl of 0.1 M DTT, 2.5 µl of SuperScript II™ Reverse Transcriptase and 2.5 µl of RNaseOUT Ribonuclease Inhibitor.</p><p>3).&nbsp;Add 4 µl to each of the four aRNA samples. Mix well by pipetting up and down and centrifuge briefly at the maximum speed. Incubate the samples at 25ºC (Lid temperature: 37ºC) for 10 min and at 42ºC (Lid temperature: 50ºC) for 1 hour.</p><p>4). Fifteen minutes before the completion of the RT reaction, prepare the following PCR mix in a 1.5 ml tube (171 µl): 49.5 µl of DNase/RNase-free water, 112.5 µl of Phusion® High-Fidelity PCR Master Mix with HF Buffer and 9 µl of 10 µM RP1 primer.</p><p>5).&nbsp;After the completion of the RT reaction, transfer the samples on ice and add 38 µl of PCR mix in each sample. Afterwards, add 2 µl of one of RPI1-12 primers to each sample separately, mix well by pipetting and centrifuge the plate briefly.</p><p>6).&nbsp;&nbsp;Use the following PCR cycle conditions to prepare the libraries: 98°C (30 sec), 11 cycles of: 98°C (10 sec), 60°C (30 sec) and 72°C (30 sec), final elongation at 72°C for 10 min and infinite hold at 4°C.</p><p>7).&nbsp;After starting the PCR, take out the AMPure XP beads (approximately 240 µl) from the refrigerator and let them pre-warm at room temperature.</p><p>8).&nbsp;Once the PCR is finished, remove the plate from the cycler, vortex AMPure XP Beads and add 50 μl to each library sample. Mix beads with the sample thoroughly by pipetting up and down. Change tips while mixing different libraries to avoid contamination. If library preparation was done in PCR tubes, add 50 μl AMPure XP Beads to each sample, mix well and transfer the sample and the beads in a 96-well plate for clean-up.</p><p>9).&nbsp;Incubate the 96-well plate at room temperature for 10 min and afterwards place it on the magnetic stand until all the beads are attached to the wall of the plate and the liquid is clear. At this stage, DNA is attached to the beads. Remove and discard the liquid while keeping the plate on the magnetic stand. Prepare 80% ethanol in DNase/RNase-free water during this incubation period.</p><p>10). Add 150 μl freshly prepared 80% ethanol, incubate at least for 30 seconds, remove and discard the ethanol without disturbing the beads.</p><p>11). Repeat the washing step once more. After the washing steps, make sure to remove small droplets of ethanol at the bottom or the corner of the well using a single-channel pipette.</p><p>12). Air dry beads for 10 min while keeping the plate on the magnetic stand.</p><p>13). Remove the plate from the magnetic stand and resuspend beads in 25 μl DNase/RNase-free water. Pipette the entire volume up and down several times to mix thoroughly.</p><p>14).&nbsp;Incubate the plate at room temperature for 2 min, place it back on the magnetic stand until the liquid appears clear and transfer 25 μl of supernatant to the empty wells of the same 96-well plate.</p><p>15). Repeat the bead clean-up process again but this time with 25 µl of beads. Elute the DNA in 11 µl of DNase/RNase-free water.</p><p>16). Use 1-1 µl per library sample to measure the concentration by Qubit® Fluorometer using Qubit dsDNA HS (High Sensitivity) Assay Kit and size distribution by Agilent 2100 Bioanalyzer using High Sensitivity DNA kit respectively. The remaining 8 µl of the libraries can be stored at -20ºC.</p><p>17).&nbsp;Based on the concentration measured by Qubit® Fluorometer and the average library size measured by Agilent 2100 Bioanalyzer, calculate the molarity of each sample. Adjust the concentration to a particular molarity, pool the samples and submit them for pair-end sequencing. Consult your sequencing facility regarding the type of run and concentration needed by them to be loaded on the flow cell.</p><h2>Troubleshooting</h2><p>If there is a clog during sorting (this is more likely to happen when sorting hepatocytes), filter the cells again through a 40 micron filter and further dilute the sample.</p><h2>Time Taken</h2><p>3 days in total. (1 day to sort cells, 1 day for RT and cDNA preparation from single cells, 1 day for library preparation)</p><h2>Anticipated Results</h2><p><br></p><h2>References</h2><p>Sagar, et al. (2018). "High-Throughput Single-Cell RNA Sequencing and Data Analysis." Methods Mol Biol <strong>1766</strong>: 257-283.</p><p>Herman, J. S., Sagar &amp; Grun, D. FateID infers cell fate bias in multipotent progenitors from single-cell RNA-seq data. <em>Nat Methods</em>, doi:10.1038/nmeth.4662 (2018).</p><h2>Acknowledgements</h2><p>We would like to thank the deep sequencing and FACS facilities at the MPI-IE for their active support and the patients for providing informed consent to participate in the study.</p></body></html>