<html><head><title>SisterC: A novel 3C-technique to detect chromatin interactions between and along sister chromatids</title></head><body><h1>SisterC: A novel 3C-technique to detect chromatin interactions between and along sister chromatids</h1>Authors: Marlies E. Oomen, Adam K. Hedger, Jonathan K. Watts, Job Dekker<br>Abstract: <p>Current chromosome conformation capture techniques are not able to distinguish sister chromatids. Here we describe the protocol of SisterC<sup>1</sup>: a novel Hi-C technique that leverages BrdU incorporation and UV/Hoechst-induced single strand breaks to identify interactions along and between sister chromatids. By synchronizing cells, BrdU is incorporated only on the newly replicated strand, which distinguishes the two sister chromatids<sup>2,3</sup>. This is followed by Hi-C<sup>4</sup> of cells that can be arrested in different stages of the cell cycle, e.g. in mitosis. Before final amplification of the Hi-C library, strands containing BrdU are specifically depleted by UV/Hoechst treatment. SisterC libraries are then sequenced using 50bp paired end reads, followed by mapping using standard Hi-C processing tools. Interactions can then be assigned as inter- or intra-sister interactions based on read orientation.</p><br>Keywords: Chromosome organization, sister chromatids, cell cycle, genomics, sequencing, Hi-C, budding yeast, S. cerevisiae<br><h2>Reagents</h2><p><strong>Reagent - Company - Catalog number</strong></p><p>Alpha-Factor Mating - Pheromone ZymoResearch - Y1001</p><p>37% formaldehyde - Sigma Aldrich - 252549</p><p>BrdU (5-Bromo-2´-Deoxyuridine) - Thermo - B23151</p><p>Nocodazole - Sigma - M1404-10MG</p><p>HindIII (for HindIII SisterC) - NEB - R0104S</p><p>DpnII (for DpnII SisterC) - NEB - R0543T</p><p>Biotin-14-dATP (for DpnII SisterC) - Invitrogen - 19524-016&nbsp;</p><p>Biotin-14-dCTP (for HindIII SisterC) - Invitrogen - 19518-018</p><p>OmniPur Phenol:Chloroform:Isoamyl Alcohol, 25:24:1 (TE Buffered Saturated pH 6.7/8.0) - Sigma - 6810</p><p>Proteinase K (Fungal) - Invitrogen - 25530-031</p><p>RNase A from bovine pancreas - Sigma - 10109169001</p><p>DNA polymerase I, large (Klenow) fragment - NEB - M0210S</p><p>T4 DNA ligase 1U/ µl - Invitrogen - 15224090</p><p>T4 DNA polymerase - NEB - M0203L</p><p>5X ligation buffer - Invitrogen - 46300-018</p><p>T4 polynucleotide kinase - NEB - M0201</p><p>5X T4 DNA ligase buffer - Invitrogen - P/N y90001</p><p>Klenow fragment (3’ → 5’ exo<sup>-</sup>) - NEB - M0212L</p><p>Dynabeads® MyOne™ Streptavidin C1 - Invitrogen - 650.01</p><p>PfuUltra II Fusion DNA polymerase - Stratagene - 600670</p><p>Agencourt AMPure® XP - Beckman Coulter - A63881</p><p>Hoechst 33258, Pentahydrate (bis-Benzimide) - ThermoFisher - H3569</p><p>100bp DNA marker - NEB - N3231S</p><p>1 kb DNA marker - NEB - N3232S</p><p>NEBuffer 2.1 - NEB - B7202S</p><p>NEBuffer 3.1 - NEB - B7203S</p><p>TruSeq Nano DNA Sample Prep Kit - Illumina - 20015964</p><p>ClaI - NEB - R0197L</p><p><br></p><p><strong>Buffers</strong></p><p><u>Spheroplasting buffer</u></p><p>50mM Tris-HCl (pH 7.5)</p><p>1M Sorbitol</p><p><br></p><p><u>Tween Wash Buffer</u>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>5 mM Tris-HCl (pH=8.0)</p><p>0.5 mM EDTA</p><p>1 M NaCl</p><p>0.05% Tween</p><p><br></p><p><u>2X Binding Buffer</u></p><p>10 mM Tris-HCl (pH=8.0)</p><p>1 mM EDTA</p><p>2 M NaCl</p><p><br></p><p><u>TLE (pH 8.0)</u></p><p>10 mM Tris-HCl</p><p>0.1 mM EDTA</p><h2>Equipment</h2><p>Sonicator (Covaris #M220)</p><p>UV cross-linker with 365-nm longwave bulbs (for example UVP, cat. no. CL-1000L)</p><p>DynaMag™-2 Magnet (Thermo #12321D)</p><p>Phase lock gel, light 2 ml (5 Prime #2302820)</p><p>DNA LoBind tubes, 1.5 ml (Eppendorf #022431021)</p><p>Amicon® Ultra – 0.5ml 30K (Millipore #UFC5030BK)</p><p>ThermoMixer C (Eppendorf #2231000667)</p><h2>Procedure</h2><p><u>1. Synchronization</u></p><p><em>Note: </em>We describe here the protocol to perform SisterC in budding yeast. As wild type yeast cannot incorporate BrdU, we recommend to use a mutant strain, e.g. YLV11 strain, which expresses the nucleoside transporter hENT and nucleoside kinase DmdNK <sup>5</sup>. Although not tested, this protocol can be modified for use in mammalian cell lines and other model systems. The synchronization protocol is listed here is specific for the YLV11 strain. Cells are grown in YP media with 2% galactose and 100µM Thymidine at 30°C unless stated otherwise.&nbsp;</p><p><br></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Start synchronizing cells at an OD 0.15 (1.5-2 million cells/mL)</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Let cells double for ~2 hours.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Add 5µM alpha factor to synchronize cells in late G1 for approximately 2.5-3 hours until cells form schmoos.</p><p><em>Optional: </em>For improved BrdU incorporation, 500µM BrdU can be added to the culture after 2 hours of alpha arrest</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Wash G1 arrested cells 3 times in warm YP media and release in prewarmed media supplemented with 1mM BrdU</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;For mitotic arrested cultures:</p><p>		a.&nbsp;&nbsp;&nbsp;&nbsp;Add 1% DMSO 15 minutes after alpha release</p><p>		b.&nbsp;&nbsp;&nbsp;&nbsp;After additional 30 min add 10µg/mL nocodazole 0.8% DMSO (final concentration 2% DMSO) and</p><p>				arrest cells in mitosis</p><p>		c.&nbsp;&nbsp;&nbsp;&nbsp;Harvest mitotic arrested cells approximately 4.5 hours after alpha factor release. Asses 				</p><p>				synchronization state by bright field microscopy.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;For G1 arrested cultures:</p><p>		a.&nbsp;&nbsp;&nbsp;&nbsp;Release culture from alpha arrest for 2 hours. Observe cells by bright field microscopy to make sure </p><p>				cells no longer show schmoo formation</p><p>		b.&nbsp;&nbsp;&nbsp;&nbsp;Add 5µM alpha factor for 3 additional hours to arrest cells in the subsequent G1</p><p>		c.&nbsp;&nbsp;&nbsp;&nbsp;Harvest G1 arrested cells approximately 5 hours after initial alpha release.</p><p><br></p><p><em>Note</em>: Cell cycle state and BrdU incorporation can be assessed qualitatively by flow cytometry of all timepoints during synchronization using anti-BrdU antibody staining. Quantitative measurement of BrdU incorporation efficiency can by done by HPLC detection of genomic DNA isolated from cultures.</p><p><br></p><p><u>2. Crosslinking of cells</u></p><p><em>Note: </em>This crosslinking protocol is specific for budding yeast. Regular Hi-C crosslinking protocols can be used for mammalian cells.</p><p><br></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;To 100mL culture add 8.8 ml of 37% formaldehyde (3% final) of culture.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Leave culture in the shaking incubator for 20 minutes at 30°C.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Add 17.6 ml of 2.5 M glycine and leave for an additional 5 minutes at 30°C in shaker incubator</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Pellet the cells by spinning at 1500xg for 3 min at RT.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Discard the supernatant and resuspend in 10mL MilliQ</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Pellet cells again at 1500xg for 3min, discard supernatant, resuspend in 1mL MilliQ</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;Pellet cells, discard supernatant, freeze pellet and store at -80°C.</p><p>			a.&nbsp;&nbsp;&nbsp;&nbsp;Pellet can be stored at -80°C for months up to a year.</p><p><br></p><p><u>3. Cell lysis and chromatin digestion</u></p><p><em>Note: </em>This protocol performs SisterC using DpnII digestion, however the protocol can be modified to use alternative restrictions enzymes, such as HindIII. Make sure to use the correct digestion buffer paired to the restriction enzyme of choice and correct volumes where necessary.</p><p><br></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Wash crosslinked cell aliquot in 1 ml of cold spheroplasting buffer</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Resuspend in 1ml of spheroplasting buffer, add 5 µl of beta-Mercaptoethanol and 2 µl of 100T Zymolyase (5mg/ml).</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Incubate at 35°C for 10 minutes and invert every 2-3 minutes.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Centrifuge for 2 minutes at 2,500xg at RT.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Discard the supernatant and then wash the pellet twice by resuspending it in 500μl of ice cold 1x NEBuffer 3.1 and then centrifuging the sample for 2 min at 2,500xg.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Resuspend the pellet in 360 µl 1x NEBuffer 3.1.</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;(Optional) Save 18 µl of lysate for the chromatin integrity control. Add 50 μl of 1x NEB and 10 μl of Proteinase K (10 mg/ml). Incubate for 30 minutes at 65°C. Purify DNA by single phenol-chloroform extraction without ethanol precipitation. Add 1μl of RNAseA (1 mg/ml) to the aqueous phase and incubate for 15 min at 37°C. Check quality of the sample by running it on 0.8% agarose gel. DNA is expected to run as a single high molecular weight band.</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Add 3.8 μl of 1% SDS to each Hi-C tube (~380 µl total). Mix carefully by pipetting up and down (0.01% SDS final).</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;Incubate at 65°C for 5 minutes exactly to open chromatin, place tubes on ice immediately after</p><p>10.&nbsp;Add 43 μl of 10% Triton X-100 to the Hi-C-tube (423 µl total) to quench the SDS (1% Triton final). Mix gently by pipetting up and down Avoid making bubbles.</p><p>NB! The addition of the following ingredients depends on the enzyme and its concentration.</p><p>11.&nbsp;Add 5.5 µl of 10x NEBuffer 3.1 to the Hi-C tube to compensate for added components (428.5 µl total before DpnII)</p><p>12.&nbsp;Add 400U (8 µl of 50,000 units/ml) DpnII and 38.5 µl 1XNEB3.1 to the Hi-C tube (475µl total), mix gently. Digest the chromatin overnight at 37°C in a ThermoMixer at 400 rpm.</p><p><br></p><p><u>4. Marking of DNA ends with biotin-14-dCTP</u></p><p><em>Note: </em>If using alternative restriction enzymes, make sure to change biotinylated nucleotide if necessary.</p><p><br></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Incubate<strong> </strong>tube at<strong> </strong>65˚C for 20 mins<strong> </strong>in order to deactivate the endonuclease enzyme.</p><p>	NB! This inactivation depends on the enzyme used. Please check enzyme datasheet</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;(Optional) Take 10 µl and keep it aside to assess the digestion on gel. Add up to 50 μl of appropriate 1x NEB and 10 μl of Proteinase K (10 mg/ml) to the digestions control. Incubate for 30 minutes at 65°C. Purify DNA by single phenol-chloroform extraction without ethanol precipitation. Add 1μl of RNAseA (1 mg/ml) to the aqueous phase and incubate for 15 min at 37°C. Check quality of the sample by running it on 0.8% agarose gel. DNA is expected to run low as a smear.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Prepare a Biotin Fill-in MasterMix as follows:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p><strong>Fill-in mix </strong>											<strong>1x</strong></p><p>milliQ 													2 µl</p><p>10x NEBuffer 3.1								6 μl</p><p>10 mM dCTP										1.5 μl</p><p>10 mM dGTP										1.5 μl</p><p>10 mM dTTP										1.5 μl</p><p>0.4 mM biotin-14-dCTP						37.5 μl</p><p>5U/μl DNA polymerase I Klenow		10.0 μl</p><p>Total													60 µl</p><p><br></p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Put the tubes on ice until cooled to room temperature.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Add 60μl of the Fill-in master mix to each tube (535 µl total)</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Mix gently by pipetting up and down without producing any bubbles</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;Incubate the tubes at 23°C for 4 hours in a ThermoMixer (900 rpm mixing; 10 secs every 5 mins)</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Prepare ligation mixture (below) during incubation</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;Place the tubes on ice immediately after incubation</p><p><br></p><p><u>5. Blunt end ligation</u></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Prepare the 665 µl ligation mix during fill-in:</p><p><strong>Ligation mix</strong>										<strong>1x</strong></p><p>Water													242μl</p><p>5x ligation buffer<strong> </strong>[Invitrogen]				240 μl</p><p>10% Triton X-100								120 μl</p><p>10 mg/ml BSA										12 μl</p><p>T4 DNA ligase [Invitrogen]					50 μl</p><p>Total													665 µl</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Add 665 µl of the ligation mix to each library (<em>1,200 µl total</em>) and split in 16 reactions of ~75 uL each.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Incubate at 16°C for 4 hours PCR machine.</p><p><br></p><p><u>6. Reverse crosslinking</u></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Collect all PCR reaction again in an Eppendorf low bind tube (~1,200uL total).</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Add 50μl of 10 mg/ml proteinase K to each tube and incubate at 65°C for at least 2 hours</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Add another 50 μl of<strong> </strong>proteinase K to each tube and continue incubating at 65°C overnight (<em>1,300 µl total</em>)</p><p><br></p><p><u>7. DNA purification</u></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;For each<strong> </strong>Hi-C tube, prepare 6x 2 ml safe-lock tubes with 400 µl phenol:chloroform (1:1; pH=8.0)</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Spin the tubes at ≥10,000xg for 30s to move the grease to the bottom of the tube</p><p>	<em>NB! Use appropriate gloves and lab coat when handling phenol:chloroform</em></p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Cool reaction mixtures after reverse crosslinking from 65˚C to RT</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Split each sample in 3 phase lock tubes (<em>~</em>400 μL) containing the PCI</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Vortex for 30-60 seconds to obtain a homogenous, milky solution.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Centrifuge at 10,000xg for 5 minutes (room temperature)</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;Transfer the aqueous phase with DNA <em>(~400 </em>μL) to a new phase lock tube and repeat steps 5-6.</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Transfer aquous phase with DNA to a clean 1.5 mL low bind tube</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;Add 1/10 volume of 3M sodium acetate, pH=5.2 per tube (40 μL) and mix well.</p><p>10.&nbsp;Add 2.5 volumes of 100% ethanol<strong> </strong>(1 mL) and mix well by inverting the tubes several times.</p><p>11.&nbsp;Incubate the tubes for ~20-40 min at -80˚C until the sample becomes viscous but not solid.</p><p>12.&nbsp;Centrifuge tubes at 16,000x<em>g</em> for 30 min at 4°C.</p><p>13.&nbsp;Decant the supernatant with extra caution as the pellet detaches from the tube wall quite easily and air dry the DNA pellets.</p><p>14.&nbsp;Dissolve the pellets in 150µl each of 1x TLE (pH=8.0).<strong> </strong>Combine and transfer (450 µl in total) to a 2 ml 30kD Amicon</p><p>15.&nbsp;Centrifuge at 14,000xg in tabletop centrifuge for ~5 minutes</p><p>16.&nbsp;Wash the column with 450 µl TLE and repeat this step twice</p><p>17.&nbsp;After the last wash, determine the volume on the column and adjust to ~ 100 µl for each sample</p><p>18.&nbsp;Flip the column onto a new Amicon container tube</p><p>19.&nbsp;Spin at 14,000xg for 2 minutes to recover the DNA from the column</p><p>20.&nbsp;Add 1 µl of RNAseA (1 mg/ml) to each sample and incubate for 30 mins at 37°C (in Thermo mixer or waterbath)</p><p><em>Note: DNA samples can now be stored at -20°C.</em></p><p>21.&nbsp;Run 1 and 3 μL of each sample on a 0.8% gel (optional with the chromatin integrity control (step 3.7) and digestion control (step 4.2))</p><p><u>&nbsp;</u></p><p><u>8. Removal of biotin from un-ligated ends</u></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Set up for 40 µg total DNA as follows (2 reactions per library):</p><p><strong>Biotin removal mix </strong>									 <strong> 1x</strong></p><p>Hi-C DNA sample											50 uL</p><p>10X NEBuffer 2.1											6.5 µl</p><p>10 mM dATP													0.1625 µl</p><p>10 mM dGTP													0.1625 µl</p><p>3,000 U/ml T4 DNA polymerase (NEB)			6.5 µl</p><p>MilliQ																to 65 µl</p><p><br></p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Split into 65 µl reactions to accommodate incubation in a PCR machine</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Incubate at 20°C for 4 hours</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Inactivate the enzyme for 20 mins at 75°C</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Cool down/ keep at 4°C</p><p><em>N.B! In the final mix we have DNA, the reagents used in this step, and free nucleotides including the biotin-14-dATP. In order to clean DNA for sonication we wash the DNA using amicon column. In this step a lot of DNA can be lost, so when you have a low input library consider skipping this step and risk a low efficient sonication.&nbsp;&nbsp;&nbsp;</em></p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Pool the reactions and perform an Amicon protocol:</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;Centrifuge at 14,000xg for ~ 5 mins or until volume has reduced to ~ 100 µl</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Wash twice with 400 µl of milliQ</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;Invert the Amicon column onto a fresh tube and centrifuge for 2 minutes at 14,00xg to collect the DNA</p><p><em>NB! Beware that Amicon does not remove proteins and that concentration by volume reduction can result in DNA coming out of solution</em></p><p>10.&nbsp;Add <strong>milliQ</strong> to a total of <strong>130</strong> <strong>µl</strong> and transfer to a Covaris microTUBE for sonication</p><p><br></p><p><u>9. DNA sonication</u></p><p><em>Note:</em> this step is specific for the sonicator used. This protocol is used for Covaris M220 machines</p><p><br></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Shear the DNA to a size of 600-800 bp using a sonicator. For a Covaris instruments use the following parameters at 20°C :</p><p>Peak Incident Power (Watt)		50W</p><p>Duty Cycle/Duty factor				7.5%</p><p>Cycles per Burst							200</p><p>Treatment time (seconds)			22 sec</p><p><em>&nbsp;</em></p><p><u>10. Size selection</u></p><p>1.	Bring volume of each sample up 500ul with 1x TLE</p><p>2.	Allow AMpure XP mixture to come to RT and mix well prior to use. Add 300 µl of AMpure XP mixture (0.6x).</p><p><em>Note: </em>Under these conditions the beads capture and remove DNA fragments &lt;300 bp from the sample.</p><p>3.	Vortex and spin down tubes briefly.</p><p>4.	Incubate tubes for 10 min at RT.</p><p>5.	Place tubes on the Magnetic Particle Seperator (MPS) for 5 min at RT.</p><p>6.	Wash the beads twice with 1 ml 70% ethanol, reclaiming beads against the MPS for 5 min each time.</p><p>7.	Air dry beads on the MPS until alcohol has evaporated completely</p><p>8.	Resuspend the beads in 50 µl of 1x TLE buffer in each tube to elute the DNA</p><p>9.	Incubate 10 min at RT</p><p>10.	Separate AMpure beads from eluate on the MPS for 5 min and collect the eluate</p><p><br></p><p><u>11. End repair</u></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Add to the 50µl 1.1x Hi-C sample, in the following order for end-repair:</p><p><strong>End repair mix</strong>											<strong>1x</strong></p><p>Sonicated DNA											50 µl</p><p>10X ligation buffer [NEB]							7 µl</p><p>25 mM dNTP mix										0.7 µl</p><p>T4 DNA polymerase (3U/ µl)						2.5 µl</p><p>T4 polynucleotide kinase (10U/ µl)				2.5 µl</p><p>Klenow DNA polymerase I (5U/ µl)			0.5 µl</p><p>Water															6.8 µl</p><p>Total															70 µl</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Incubate the reaction in a PCR machine:</p><p>				1)&nbsp;&nbsp;&nbsp;&nbsp;30 min at 20°C</p><p>				2)&nbsp;&nbsp;&nbsp;&nbsp;20 mins at 75°C to deactivate the enzymes</p><p>				3)&nbsp;&nbsp;&nbsp;&nbsp;keep at 4°C(∞)</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Bring up volumes to 450uL and transfer to amicon colums</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Wash with TLE twice</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Flip column and elute. Bring up volume to 41 uL.</p><p><br></p><p><u>12. A-tailing</u></p><p><em>Note: </em>After performing the A-tailing reaction, make sure to continue immediately with step 13 (illumine adapter ligation)</p><p><br></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Mix components as follows:</p><p><strong>A-tailing mix</strong>															 <strong>1x</strong></p><p>Ampure purified DNA												41 µl</p><p>10X NEBuffer 2.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;											5 µl</p><p>10 mM dATP															1 µl</p><p>Klenow fragment (3’ → 5’ exo-) (5U/ µl)					3 µl&nbsp;&nbsp;</p><p>Total																		50 µl</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Incubate reactions in a PCR machine at 37°C for 30 min to add adenine on the 3’ end and then at 65°C for 20 min to inactivate Klenow (exo-).</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Place tubes on ice immediately. Clean up samples using Amicon columns before continuing to A-tailing</p><p>4.		Bring up volumes to 450uL and transfer to amicon colums</p><p>5.		Wash with TLE twice</p><p>6.		Flip column and elute. Bring volume up to 80 uL.</p><p>7.		Split each library in two samples, which will get individual adapter barcodes (plus and minus UV/Hoechst treatment). Continue to adapter ligation immediately</p><p><br></p><p><u>13. Illumina adapter ligation</u></p><p><em>Note</em>: In order to multiplex libraries on the same lane of sequencing, make sure to pick correct illumine adapters. For suggestions on best combinations of adapter multiplexing: <a href="https://support.illumina.com/content/dam/illumina-support/documents/documentation/chemistry_documentation/experiment-design/index-adapters-pooling-guide-1000000041074-05.pdf" rel="noopener noreferrer" target="_blank">https://support.illumina.com/content/dam/illumina-support/documents/documentation/chemistry_documentation/experiment-design/index-adapters-pooling-guide-1000000041074-05.pdf</a></p><p><br></p><p>1. Each sample will be split into 5 ligation reactions with the same adapter barcode to optimize adapter ligation efficiency.</p><p>2. Set up one adapter ligation reaction as follows:</p><p><strong>Adaptor ligation mix</strong>																		<strong>1x</strong></p><p>Illumina paired end adapter Illumina TrueSeq DNA LT kit			3 µl</p><p>Hi-C DNA																					8 µl</p><p>5X T4 ligation Buffer [Invitrogen]												10 µl</p><p>T4 DNA ligase [Invitrogen]															3 µl</p><p>MilliQ																							28 µl</p><p>Total																							50 µl</p><p>3.	Incubate at 22˚C (RT) for 2 hours in pcr machine.</p><p>4.	Collect all 5 reactions per sample (250 uL)</p><p>5.	Allow AMpure XP mixture to come to RT and mix well prior to use. Add 375 µl of AMpure XP (1.5x).</p><p>6.	Vortex and spin down tubes briefly</p><p>7.	Incubate tubes for 10 min at RT</p><p>8.	Place tubes on the Magnetic Particle Seperator (MPS) for 5 min at RT.</p><p>9.	Wash the beads twice with 1 ml 70% ethanol, reclaiming beads against the MPS for 5 min each time.</p><p>10.	Air dry beads on the MPS until alcohol has evaporated completely</p><p>11.	Resuspend the beads in 1x TLE buffer in each tube to elute the DNA</p><p>			a.&nbsp;&nbsp;&nbsp;&nbsp;Use 90 uL for the samples that will be treated with UV/Hoechst samples and 150 uL for the Hi-C 				</p><p>					control samples that will not be treated.</p><p>12.	Incubate 10 min at RT</p><p>13.	Separate AMpure beads from eluate on the MPS for 5 min and collect the eluate.</p><p><br></p><p><u>14. Hoechst/UV treatment</u></p><p><em>Note: </em>this step will be skipped entirely for the Hi-C control samples (- UV/Hoechst).</p><p><br></p><p>Set up Hoechst/UV reactions as follows (each sample will be split into two reactions):</p><p><strong>Hoechst mix</strong>				<strong>1x</strong></p><p>Hi-C DNA					45 µl</p><p>10ug/uL Hoechst		0.5 µl</p><p>TLE								4.5 µl</p><p>Total							50 µl</p><p>2.	Incubate for 15 min @RT while covered from light</p><p>3.	UV radiate with 2.7kJ/m2</p><p>4.	Bring up volumes to 450uL and transfer to amicon colums</p><p>5.	Wash with TLE three times</p><p>6.	Flip column and elute. Bring volume up to 150 uL.</p><p><br></p><p><br></p><p><u>15. Biotin pulldown with streptavidin coated beads</u></p><p><em>Note</em>: All subsequent steps are performed in 1.7ml DNA LoBind tubes with DNA LoBind tips. Samples are transferred to new tubes to prevent the loss of the beads on the tube wall.</p><p><br></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Vortex the MyOne™ Streptavidin C1 beads and transfer 10 µl of bead solution to a 1.7ml LoBind tube.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Wash the beads with 400 µl of Tween wash buffer (TWB) by pipetting up and down and incubating for 3 min at RT on a rocking platform.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Reclaim beads against the MPS for 1 min, discard the supernatant.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Resuspend beads in 400 µl of TWB and transfer to a new LoBind tube.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Reclaim beads against the MPS for 1 min, discard the supernatant.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Resuspend beads in 150 µl of 2X Binding Buffer (BB)</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;Add 150 of End-repair reaction containing DNA</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Incubate the sample for 15 min at RT with rotation.</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;Reclaim the beads against the MPS for 1 min; discard the supernatant.</p><p>10.&nbsp;Resuspend the beads in 400 µl of 1X BB and transfer them to a new tube</p><p>11.&nbsp;Reclaim the beads against the MPS for 1 min, discard the supernatant.</p><p>12.&nbsp;Wash beads with 100 µl of 1x TLE (pH 8.0) and transfer to a new tube</p><p>13.&nbsp;Reclaim the beads against the MPS for 1 minute; discard the supernatant</p><p>14.&nbsp;Finally resuspend the beads in 20 µl of 1x NEBuffer 2.1 and transfer to a new tube.</p><p><u>&nbsp;</u></p><p><u>16. Titration PCR</u></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Set up PCR to titrate number of cycles as follows:</p><p><em>Note: </em>The following step is using TrueSeq DNA LT kit&nbsp;(15041757).</p><p><strong>PCR mix</strong>																<strong>1x</strong></p><p>Bead-bound Hi-C DNA									1.5 µl</p><p>Primer Mix<strong>&nbsp;</strong>(Illumina TrueSeq DNA LT kit)		2.5 µl</p><p>milliQ																11 µl</p><p>PCR Mix (Illumina TrueSeq DNA LT kit)			10 µl</p><p>Total																25 µl</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Run the PCR program with pauses after different number of cycles:</p><p>			1)		95°C	180 seconds</p><p>			2)		98°C	20 seconds</p><p>			3)		60°C	15 seconds</p><p>			4)		72°C	30 seconds</p><p>			5)		Go to step 2 N times</p><p>&nbsp;					total 6, 8, 10, 12 cycles</p><p>					Take 5 µl aliquots after 6, 8 and 10 cycles.</p><p>			6)		72°C	3 minutes</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Use the DNA amplified for 12 cycles for ClaI digestion.</p><p><em>Note: </em>If chromatin digestion in step 3 was performed with an alternative restriction enzyme, the enzyme used at this step needs to be changed accordingly as well.</p><p>Take 5 µl aliquots after 6, 8 and 10 cycles.</p><p><strong>Digest mix</strong>										<strong>1x</strong></p><p>10x NEB Cutsmart buffer			1.5 µl</p><p>ClaI												1 µl</p><p>Hi-C product								5uL</p><p>MilliQ											to 15uL</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Digest at 37°C for at least 30 minutes.&nbsp;&nbsp;</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Run the pcr titration and test digest on a 2% agarose gel.</p><p><br></p><p><em>Note: </em>Choose an optimal number of cycles and number of PCR reactions for the final amplification of the library for deep sequencing. The cycle number should be chosen so that the PCR amplification is in the linear range and the expected size distribution is preserved (over cycling will shift the size distribution of the library towards higher molecular weight products).&nbsp;The number of PCR reactions should be calculated to produce the amount of DNA desired for sequencing (usually 50-100 ng).&nbsp;It is always better to reduce the number of PCR cycles while increasing the number of PCR reactions. The optimal amount is usually 1-2 cycles below the lowest amount visible on gel. Make sure to use the same number of PCR cycles between the SisterC sample and Hi-C control originating from the same cells.</p><p><br></p><p><u>17. Production PCR</u></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Set up 3x 50 uL PCR reactions per library as follows (using half of the bead bound Hi-C library):</p><p><strong>PCR mix</strong>																<strong>1x</strong></p><p>Bead-bound Hi-C DNA										3 µl</p><p>Primer Mix&nbsp;(Illumina TrueSeq DNA LT kit)			5 µl</p><p>milliQ																	20 µl</p><p>PCR Mix (Illumina TrueSeq DNA LT kit)				22 µl</p><p>Total																	50 µl</p><p><br></p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Pool all PCR reactions together</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Separate the streptavidin beads from the supernatant using the MPS. Resuspend the streptavidin beads in 9 uL 1x NEB 2.1 and store for optional further amplification. The beads contain the library template, the supernatant will contain the amplified library for deep sequencing.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;To remove primer dimers, purify the amplified library from the supernatant using AMpure XP beads as follows. Allow AMpure XP mixture to come to RT and mix well prior the use. Add 1.1X volumes of AMpure XP mixture (165 uL) to the supernatant (1.1 : 1).</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Vortex and spin down briefly.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Incubate for 10 min at RT.</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;Place on the MPS for 5 min at RT.</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Discard supernatant.</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;Wash the beads twice with 1 ml of freshly made 70% ethanol</p><p>10.&nbsp;Air-dry the beads completely</p><p>11.&nbsp;Resuspend the beads in 30 µl of TLE buffer</p><p>12.&nbsp;Incubate the beads for 10 min at RT, tapping the tube every 1-2 min.</p><p>13.&nbsp;Collect the beads with the MPS for 5 min</p><p>14.&nbsp;Transfer the supernatant, containing the final Hi-C library, to a new tube.</p><p>15.&nbsp;Quantify the amount of DNA in the Hi-C library on the 2% agarose gel or fragment analyzer to assess whether there are no longer primers and adapters and quantify library concentration.</p><p>&nbsp;</p><p><u>18. Next generation sequencing</u></p><p>The samples are now ready for next generation sequencing on Illumina platforms. For best mapping results use 50bp paired end sequencing.</p><p><br></p><p><u>19. Mapping and downstream analysis</u></p><p>Hi-C and SisterC FASTQ files can be mapped to the appropriate reference genome using the publicly available mapping pipeline distiller (<a href="https://github.com/mirnylab/distiller-nf" rel="noopener noreferrer" target="_blank">https://github.com/mirnylab/distiller-nf</a>) and downstream analysis tools pairtools (<a href="https://github.com/mirnylab/pairtools" rel="noopener noreferrer" target="_blank">https://github.com/mirnylab/pairtools</a>) and cooltools (<a href="https://github.com/mirnylab/cooltools" rel="noopener noreferrer" target="_blank">https://github.com/mirnylab/cooltools</a>) <sup>6</sup>. SisterC interactions are classified as inter- or intra-sister interactions by read orientation. Interactions with read orientations – – or + + are classified as intra-sister interactions, whereas reads with orientation – + and + – are classified as inter-sister interactions. All interactions at shorter distance than 1500bp should be removed from downstream processing.</p><p><br></p><p><em>Note: </em>G1 SisterC libraries should be depleted of inter-sister interactions. Although full depletion is challenging, a 4 to 5-fold depletion of inter-sister interactions in G1 SisterC libraries indicate a successful depletion of BrdU containing DNA molecules.</p><h2>References</h2><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Oomen, M. E., Hedger, A. K., Watts, J. K. &amp; Dekker, J. Detecting chromatin interactions along and between sister chromatids with SisterC. <em>bioRxiv</em> 2020.03.10.986208 (2020). doi:10.1101/2020.03.10.986208</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Claussin, C. <em>et al.</em> Genome-wide mapping of sister chromatid exchange events in single yeast cells using strand-seq. <em>Elife</em> <strong>6</strong>, 1–17 (2017).</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Falconer, E. <em>et al.</em> DNA template strand sequencing of single-cells maps genomic rearrangements at high resolution. <em>Nat. Methods</em> <strong>9</strong>, (2012).</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Belaghzal, H., Dekker, J. &amp; Gibcus, J. H. Hi-C 2.0: An optimized Hi-C procedure for high-resolution genome-wide mapping of chromosome conformation. <em>Methods</em> <strong>123</strong>, 56–65 (2017).</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Vernis, L., Piskur, J. &amp; Diffley, J. F. X. Reconstitution of an efficient thymidine salvage pathway in Saccharomyces cerevisiae. <em>Nucleic Acids Res.</em> <strong>31</strong>, 120e – 120 (2003).</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Abdennur, N. &amp; Mirny, L. A. Cooler: scalable storage for Hi-C data and other genomically labeled arrays. <em>Bioinformatics</em> <strong>36</strong>, 311–316 (2019).</p><h2>Acknowledgements</h2><p>We thank Stephanie Schalbetter, Nicola Minchell and Matthew Naele for providing their yeast Hi-C protocol and suggestions. Additionally, we thank all current and former Dekker lab members for helpful discussions, in particular Sergey V. Venev for advice on data analysis. This work was supported by grants from the National Institutes of Health (HG003143 to J.D. and NS111990 to J.K.W.). J.D. is an investigator of the Howard Hughes Medical Institute.</p></body></html>