<html><head><title>APEX-seq: RNA subcellular localization by proximity labeling</title></head><body><h1>APEX-seq: RNA subcellular localization by proximity labeling</h1>Authors: Furqan M. Fazal, Shuo Han, Kevin R. Parker, Pornchai Kaewsapsak, Jin Xu, Alistair Boettiger, Howard Y. Chang, Alice Y. Ting<br>Abstract: <p><strong>We introduce APEX‑seq, a method for RNA sequencing based on direct proximity labeling of RNA using the peroxidase enzyme APEX2. APEX‑seq in nine distinct subcellular locales produced a nanometer-resolution spatial map of the human transcriptome as a resource, revealing extensive and exquisite patterns of localization for diverse RNA classes and transcript isoforms. APEX‑seq should be widely applicable to many systems, enabling comprehensive investigations of the spatial transcriptome.</strong></p><p><br></p><br>Keywords: APEX-seq, promixity labeling, vicinal, RNA-seq, sequencing, subcellular, RNA, localization, spatial, transcriptomics<br><h2>Introduction</h2><p><strong>General notes:</strong></p><p>1. The APEX-seq protocol takes roughly 4 days to complete:</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Day 1: Labeling of cells and extraction of RNAs</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Day 2: Enrichment for biotinylated RNAs</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Days 3-4: Preparation of sequencing libraries</p><p><br></p><p>2. This protocol assumes cells containing APEX-fusion proteins have been prepared, and the localization of the fusion construct has been confirmed by microscopy. See Hung <em>et al.</em><sup><em>1</em></sup> for details on designing APEX-fusion constructs, transfecting constructs into cells, and imaging cells.</p><p><br></p><p>3. Use proper techniques for handing RNA, and avoiding contamination with RNases. These include using a dedicated bench area for RNA, using filtered pipette tips, wiping all surfaces with RNase Zap (Invitrogen), using certified RNase-free buffers and reagents, and testing buffers for RNase contamination using RNase Alert (Ambion). When appropriate, add ~1-2&nbsp;µL of Ribolock RNase inhibitor (Thermo Fischer) per 100-200&nbsp;µL of buffer/solution.</p><p><br></p><p>4. For all water, we use ultrapure distilled water (Invitrogen). However, any source of certified/verified RNase-free water may be used.</p><p><br></p><p>5. The current APEX-seq protocol recovers total RNA or polyA-selected RNAs. We select for RNAs longer than 100&nbsp;nt. We have not tested modifications to these selection criteria.</p><p><br></p><p>6. All experiments were carried out at room temperature, unless otherwise mentioned.</p><h2>Procedure</h2><p><strong>Day 0: Growing Cells</strong></p><p><br></p><p>Details described below are for adherent human embryonic kidney (HEK293) cells. Modify as appropriate for cells from other tissues and/or organisms.</p><p><br></p><p>1. Prepare 10-cm plates for growing HEK293 cells by coating with fibronectin. To do so:</p><p>(a) Dilute 1&nbsp;mg/mL fibronectin (EMD Millipore, Catalog No. FC010) by 100 fold in PBS.</p><p>(b) Filter solution using 0.10&nbsp;µm filter (0.22&nbsp;µm okay too; Steriflip EMD Millipore).</p><p>(c) Fully cover each 10-cm plate with ~3&nbsp;mL fibronectin mix.</p><p>(d) Incubate in 37°C incubator for ~1&nbsp;hour.</p><p>(e) Aspirate off remaining liquid, and add cells to plate.</p><p>We typically grow/expand cells without fibronectin, and only use fibronectin for plates that will be used for labeling.</p><p><br></p><p>2. Grow cells overnight at 37°C (5% CO<sub>2</sub>) incubator to ~70-80% confluency. For HEK293 we use Gibco DMEM (Thermo Fischer) with 10% FBS (filtered) and 1% Gibco PenStrep (100X, Thermo Fischer). If cells grow too quickly, a 1:1 mix of DMEM: MEM may be used instead. However, the growth conditions and media should be kept constant for the project.</p><p><br></p><p>Note: For APEX-seq, we typically prepare 6 plates per condition – 3 plates for labeled “targets”, and 3 plates for unlabeled “controls”. Depending on the biological question, 4 plates (2 targets, 2 controls) might be sufficient, though it’s better to have multiple biological replicates (ideally 3+ each) for both targets and controls.</p><p><br></p><p><strong>Day 1: Labeling Cells</strong></p><p><br></p><p>1. Wash cells once with Gibco PBS (Thermo Fischer).</p><p><br></p><p>2. Incubate cells at 37°C for 30&nbsp;minutes with 3&nbsp;mL media (or PBS) containing 0.5&nbsp;mM biotin-phenol (Iris Biotech GMBH). To do so:</p><p>(a) Prepare a 1000X biotin-phenol (BP) stock (500&nbsp;mM) by adding 0.37&nbsp;g BP to 2&nbsp;mL DMSO. Any surplus stock can be aliquoted and stored in the -80°C for &gt;6&nbsp;months.</p><p>(b) Add 3&nbsp;µL 1000X BP stock to 3&nbsp;mL media (or PBS).</p><p><br></p><p>3. Label the target plates (but not the control plates) with 1&nbsp;mM H<sub>2</sub>O<sub>2</sub> for exactly 1&nbsp;minute. To do so:</p><p>(a) Prepare a ~83X H<sub>2</sub>O<sub>2</sub> stock solution by adding 10 µL 30% H<sub>2</sub>O<sub>2</sub> solution (Sigma Aldrich, 8.8&nbsp;M) to 1&nbsp;mL water. Add 36&nbsp;uL of this solution to the center of each target plate containing the BP solution. Shake the plate gently to mix and incubate for exactly 1&nbsp;minute, followed by addition of 3&nbsp;mL quenching solution.</p><p>(b) To prepare quenching solution, add 10&nbsp;mM sodium ascorbate, 10&nbsp;mM sodium azide and 5&nbsp;mM Trolox to PBS.</p><p>(c) To prepare 1&nbsp;M (100X) sodium ascorbate, dissolve 198&nbsp;mg (Sigma Aldrich) in 1&nbsp;mL water. For 1&nbsp;M (100X) sodium azide, dissolve 65&nbsp;mg (Sigma Aldrich) in 1&nbsp;mL water. For 500&nbsp;mM Trolox (100X), add 125&nbsp;mg to 1&nbsp;mL DMSO. Prepare these reagents fresh each time.</p><p>(d) To stop labeling after 1&nbsp;minute treatment with H<sub>2</sub>O<sub>2</sub>, pour out BP solution containing H<sub>2</sub>O<sub>2</sub> from 10-cm plates and immediately add ~3&nbsp;mL quenching solution. Shake well and pour out quenching solution. Quickly add another ~3&nbsp;mL quenching solution, shake well and pour out again. Finally add ~1&nbsp;mL quenching solution and shake to cover cells. Be careful not to lose or detach cells during washing.</p><p>(e) Add ~4&nbsp;uL RNase inhibitor to each plate.</p><p>(f) Proceed to RNA extraction as soon as possible. If additional time is required to label/process multiple plates, we keep the plates on ice in the meantime (no longer than 1&nbsp;hour).</p><p>(g) To prepare control samples, exclude 1&nbsp;minute H<sub>2</sub>O<sub>2</sub> addition step and carry out all subsequent steps (including quenching-solution treatment) identically to target samples.</p><p><br></p><p>4. Scrape the cells carefully using cell lifters (Corning), and transfer contents of 10-cm plate (cells and residual quenching buffer) to 1.5&nbsp;mL Eppendorf tubes. Transfer to benchtop centrifuge, and spin at ~300G for 3-5&nbsp;minutes. Remove most of the solution using a pipette, keeping the last ~50-100&nbsp;µL quenching solution in the tube.</p><p>Note: We want to inhibit any further reaction of biotin-phenol catalyzed by APEX2, so we keep cells in residual quenching solution until we have lysed cells.</p><p><br></p><p>5. Follow instructions to extract total RNA from animal cells using the RNeasy plus mini kit (Qiagen). A few notes:</p><p>(a) Add ß‑mercaptoethanol to Buffer RLT plus, as recommended by the manufacturer (10&nbsp;uL ß‑ME per 1&nbsp;mL Buffer).</p><p>(b) Use 600&nbsp;uL Buffer RLT Plus per 10-cm plate.</p><p>(c) Homogenize the lysate carefully to prevent clogging the column, and transfer lysate to a gDNA eliminator spin column.</p><p>(d) Instead of using Buffer RW1, we use Buffer RWT (Qiagen).</p><p>(e) Carry out the optional step of centrifuging at full speed for 1&nbsp;minute before eluting.</p><p>(f) Elute in 60-100&nbsp;µL water. We usually elute by running 30-50&nbsp;µL water twice through the column to maximize yield.</p><p>(g) After eluting into supplied 1.5&nbsp;mL tubes, transfer RNA to a fresh 1.5&nbsp;mL Eppendorf tube.</p><p>(h) Check concentration of RNA. We use the Nanodrop (Thermo Fischer) for quantification, and typically recover 60-100&nbsp;ug RNA per plate.</p><p><br></p><p>6. Bioanalyze samples (or run a gel) to confirm RNA is intact. We aim for a RIN (RNA integrity number) &gt;&nbsp;8.5. Proceed with caution for RNA with RIN&nbsp;= 7.5&nbsp;- 8.5. Check for RNase contamination and repeat protocol if RNA is degraded (RIN&nbsp;&lt;= 7.5). We use the RNA pico assay on the Agilent Bioanalyzer 2100 to determine RNA integrity.</p><p><br></p><p>7. Recovered RNA may be stored overnight at –20°C. However, beyond 1-2 days at –20°C, RNA should be stored at –80°C.</p><p><br></p><p>Note: For some cell lines and/or purposes, it may be necessary or desirable to have an additional step of DNA elimination. To do so:</p><p>(a) Incubate RNA with TURBO DNase (Thermo Fischer) for 30&nbsp;minutes at 37°C following the manufacture’s protocol.</p><p>(b) Purify RNA by sending through RNeasy plus mini kit.</p><p><br></p><p><strong>Day 2: Enrichment of Biotinylated RNAs</strong></p><p><br></p><p>Below are details to enrich for biotinylated RNAs. It is very important to maintain RNase-free conditions, as we typically do not carry out any further quality-control measure until we have sequencing libraries.</p><p><br></p><p>1. We usually use ~½ (~30-50&nbsp;µg) of our recovered RNA from the previous day for enrichment, but no less than 25&nbsp;µg. The remaining ½ is kept in case it is necessary to repeat downstream measurements, to make a technical replicate, or to generate libraries using different approaches (for example, ½ of original used to prepare total-RNA libraries, and the other half libraries from polyA-selected RNAs). Start by adding water to make the RNA volume ~125&nbsp;µL.</p><p>Note: If carrying out APEX-seq experiment for first time, it is advisable to use &gt;=&nbsp;50&nbsp;µg RNA.</p><p><br></p><p>2. Prepare streptavidin beads to enrich for biotinylated RNAs. To do so:</p><p>(a) We use 10&nbsp;µL pierce streptavidin magnetic beads (Thermo Fischer) per 25&nbsp;µg RNA. Adjust volume of beads appropriate to RNA amount. Mix beads well by vortexing the bottle prior to aliquoting.</p><p>(b) Add ~500&nbsp;µL binding and washing (B&amp;W) buffer (5&nbsp;mM Tris-HCl, pH&nbsp;= 7.5; 0.5&nbsp;mM EDTA; 1&nbsp;M NaCl; 0.1% TWEEN 20 (Sigma Aldrich)) to beads in 1.5&nbsp;mL Eppendorf tube.</p><p>(c) Put beads on magnet for 5&nbsp;minutes. Discard supernatant and remove tubes from magnet.</p><p>(d) Resuspend beads in 500&nbsp;µL B&amp;W buffer, place on magnet for 2&nbsp;minutes, and then discard supernatant.</p><p>(e) Repeat step (d) twice more, for a total of 3 washes.</p><p>(f) Repeat washing step twice more, but this time using solution A (100&nbsp;mM NaOH, 50&nbsp;mM NaCl) each time.</p><p>(g) Repeat washing step again, but this time with solution B (100&nbsp;mM NaCl) once.</p><p>(h) Resuspend beads in 125&nbsp;uL solution B containing ~3&nbsp;uL Ribolock RNase inhibitor.</p><p>(i) Add 125&nbsp;uL RNA from step 1 to ~125&nbsp;uL beads and incubate tube on a rotator for 2&nbsp;hours at 4°C.</p><p>Note: Do not rotate overnight – we have noticed a decreased yield upon overnight incubation.</p><p><br></p><p>3. After incubation, to get rid of unbound RNAs we carry out high-salt washes using B&amp;W buffer. To do so:</p><p>(a) Place tubes from step 3 (j) on magnet for 5&nbsp;minutes and discard supernatant.</p><p>(b) Remove tubes from magnet and resuspend beads in 500&nbsp;µL B&amp;W buffer.</p><p>(e) Repeat steps (a) and (b) above twice for a total of 3 washes.</p><p>(f) After the third wash, keep beads on magnet and remove any residual liquid remaining using a P20 (20&nbsp;uL micropipette) tip.</p><p>(g) Resuspend beads in 54&nbsp;uL water, transfer to a new 1.5&nbsp;mL Eppendorf tube, and proceed to next step immediately.</p><p><br></p><p>4. To release biotinylated RNAs bound to streptavidin beads, incubate beads with proteinase K. To do so:</p><p>(a) Prepare 3X proteinase digestion buffer (1.1&nbsp;mL) fresh as follows: 330&nbsp;µL 10X PBS, pH&nbsp;= 7.4 (Ambion); 330&nbsp;µL 20% N-laurylsarcosine sodium solution (Sigma Aldrich); 66&nbsp;µL of 0.5&nbsp;M ETDA; 16.5&nbsp;uL of 1&nbsp;M DTT; 357.5&nbsp;uL water).</p><p>(b) For each reaction, mix 33&nbsp;uL 3X proteinase digestion buffer, 10&nbsp;µL proteinase K (20&nbsp;mg/ml, Thermo Fischer), 3&nbsp;µL Ribolock RNase inhibitor. Add this mix to the 54&nbsp;uL beads (total volume&nbsp;= 100&nbsp;µL).</p><p>(c) Incubate beads with vigorous mixing at 42°C for 1 hour, followed by 55°C for 1 hour. We incubate on a thermomixer while shaking at 900&nbsp;rpm.</p><p>(d) After incubation, place tubes containing beads on magnet for 5&nbsp;minutes. Carefully extract supernatant containing released RNAs, without disturbing beads. Place liquid into new 1.5&nbsp;mL Eppendorf tube.</p><p><br></p><p>5. Purify recovered RNAs using RNA clean and concentrator -5 kit (Zymo Research). A few notes:</p><p>(a) Follow the manufacture protocol, but carry out an extra centrifuge step for 1&nbsp;minute with no liquid added, before eluting RNA with water. This step ensures complete removal of the wash buffer.</p><p>(b) Elute in 6&nbsp;µL water for downstream total RNA libraries, and 12&nbsp;µL water for polyA-selected RNA libraries. For 12&nbsp;µL volume, send 6&nbsp;µL twice through column to increase yield.</p><p><br></p><p>6. Transfer RNA to 0.2&nbsp;mL PCR tubes (Thermo Fischer). Store RNA overnight at –20°C, or at –80°C for longer duration.</p><p><br></p><p><strong>Days 3-4: Preparation of TruSeq Libraries</strong></p><p><br></p><p>1. Prepare RNA-seq libraries using the Illumina TruSeq stranded mRNA library prep kit (previously RS-122-2101/2102; recently Illumina has begun offering the library preparation kit, catalog number 20020594, and index adapters separately.). The kit allows preparation of libraries from total RNA, or alternatively polyA-selected RNA using the Oligo-dT beads (supplied). Follow the LS protocol to generate stranded RNA-seq libraries.</p><p><br></p><p>2. Note that the expected yield (~10&nbsp;ng) from day 2 is less than the recommended amount for TruSeq, but we get high-quality libraries using the kit. Nonetheless we recommend following the guidelines carefully to minimize material loss.</p><p><br></p><p>3. For the first-strand synthesis mix, we use Superscript III (instead of Superscript II). So far we have not tested Superscript IV in the APEX-seq protocol.</p><p><br></p><p>4. When ligating barcodes to cDNA libraries, we recommend using 0.55&nbsp;µL adapter index mix (and not the 2.5&nbsp;µL recommended by protocol). Unreacted adaptor can lead to downstream issues.</p><p><br></p><p>5. Typical amplification is ~16-24 cycles, and the default 15 cycles recommended is often insufficient amplification. We aim for final concentrations of ~10-20&nbsp;nM, as determined by bioanalyzer. However, we usually do not recover high-quality libraries from samples that require &gt;25 cycles amplification. In our experience, the control samples need ~2-3 extra cycles of amplification relative to the labeled target samples.</p><p><br></p><p>6. To PCR all APEX-seq libraries to the same amount for convenient pooling, we recommend replacing the suggested Illumina 15-cycle PCR step with the following protocol:</p><p>(a) PCR 1: Add 20&nbsp;µL DNA, 5&nbsp;µL PCR Primer Cocktail, and 25&nbsp;µL PCR Mix. Carry out 10 cycles of PCR following TruSeq recommendations: step 1: 98°C for 30 seconds; step 2: 10 cycles of 98°C 10 seconds, 60°C 30 seconds, 72°C 30 seconds; step 3: 72°C for 5 seconds; step 4: Hold at 4°C. Clean up following TruSeq protocol using 50&nbsp;µL Ampure XP beads (Thermo Fischer), and finally elute in 22 µL buffer.</p><p>(b) To quantify libraries, carry out a qPCR step using 0.4&nbsp;µL 10&nbsp;µM Illumina general primers, 10&nbsp;µL 2X SYBR green master mix (Thermo Fischer), 8.6&nbsp;µL water, and 1&nbsp;µL amplified DNA. qPCR this 20&nbsp;µL mix for 30 cycles using the following settings: step 1 (preincubation): 95°C for 5&nbsp;minutes; step 2 (amplification, x30 cycles): 95°C for 10 seconds, 60°C for 20 seconds, 72°C for 30 seconds; step 3 (melting curve): follow qPCR-machine-manufacture protocol; step 4 (cooling): hold at 37°C for 30 seconds. Carry out all qPCR measurements in duplicate or triplicate. The sequence of Illumina primers are as follows:</p><p>AATGATACGGCGACCACCGAGAT</p><p>CAAGCAGAAGACGGCATACGA</p><p>(c) From step (b), choose the cycle number that gives 1/6 maximum fluorescence (<em>N</em>), and subtract by two to determine the number of cycles (=&nbsp;<em>N</em>&nbsp;– 2) for amplification. We use the Lightcycler 480 (Roche) for qPCR experiments.</p><p>(d) PCR 2: Add remaining DNA (19-20&nbsp;µL), 5&nbsp;µL PCR Primer Cocktail, and 25&nbsp;µL 2X NEBNext PCR master mix (high fidelity, New England Biolabs). PCR each sample for the number of cycles determined in step (c). Clean up following the TruSeq protocol using 50&nbsp;µL XP Ampure beads, and eluting in 30&nbsp;µL buffer.</p><p>(e) For accurate pooling do a second qPCR step using 1/1000 of the DNA from step (d). Calculate the delta Ct values and pool as desired. We typically aim to sequence all APEX-seq libraries to the same depth, which usually means the libraries are pooled at the same amount. Quantify absolution concentration using the bioanalyzer “high-sensitivity” setting (Agilent Bioanalyzer 2100).</p><p><br></p><p>7. Prepare a 4&nbsp;nM 10&nbsp;µL pooled-library mix for subsequent sequencing. For sequencing, we typically check library quality on the Illumina MiSeq. For deep sequencing, we recommend paired-end reads (2 x 75) using the Illumina Hiseq&nbsp;4000 or Illumina NextSeq. We aim for ~25-40&nbsp;million reads per sample.</p><p><br></p><p>Note: There are many library-preparation kits on the market to prepare sequencing libraries. While other kits could work as well, we have not tested these kits.</p><p>Note: We do not carry out any gel purification of DNA libraries. However, it is very important that contamination with unreacted adapters is minimal, as determined by the bioanalyzer traces in step 6 (e). If there is substantial contamination, additional purification of the libraries may be necessary.</p><p><br></p><p><strong>Days 5+: Analysis of APEX-seq Libraries</strong></p><p><br></p><p>Please refer to the methods section of Fazal <em>et al.</em><sup><em>2</em></sup> for suggestions on analyzing libraries.</p><p><br></p><p>1. See summarized methods for details on trimming data (optional if contamination with unreacted adapters is absent/minimal), mapping data (using STAR<sup>3</sup>), counting reads (using HTSEQ<sup>4</sup>) and analyzing data (using DESEQ2<sup>5</sup>).</p><p><br></p><p>2. Transcripts that are differentially expressed in targets relative to controls are identified using DESEQ2. Suggestions to analyze data may be gathered from APEX proteomics experiments (Hung et al.<sup>1</sup>) using true-positive and false-positive lists, and by referring to the “ERM APEX‑seq extended analysis section” of the methods.</p><p><br></p><p>3. If no prior information is known or assumed about expected transcripts, a reasonable starting point is to consider transcripts with a log<sub>2</sub>foldchange&nbsp;&gt; 0.75, and an FDR&nbsp;&lt; 0.05, as enriched by APEX-seq.</p><h2>References</h2><p>1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hung, V.<em> et al.</em> Spatially resolved proteomic mapping in living cells with the engineered peroxidase APEX2. <em>Nat. Protoc.</em> <strong>11</strong>, 456-475, doi:10.1038/nprot.2016.018 (2016).</p><p>2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Fazal, F. M.<em> et al.</em> Atlas of Subcellular RNA Localization Revealed by APEX-seq. <em>bioRxiv</em>, 454470, doi:10.1101/454470 (2018).</p><p>3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Dobin, A.<em> et al.</em> STAR: ultrafast universal RNA-seq aligner. <em>Bioinformatics</em> <strong>29</strong>, 15-21, doi:10.1093/bioinformatics/bts635 (2013).</p><p>4&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Anders, S., Pyl, P. T. &amp; Huber, W.&nbsp;&nbsp;&nbsp;&nbsp;(Cold Spring Harbor Laboratory, 2014).</p><p>5&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <em>Genome Biol.</em> <strong>15</strong>, doi:10.1186/s13059-014-0550-8 (2014).</p></body></html>