<html><head><title>Sperm-seq wet lab protocol: sperm preparation and droplet-based sequencing library generation</title></head><body><h1>Sperm-seq wet lab protocol: sperm preparation and droplet-based sequencing library generation</h1>Authors: Avery Davis Bell, Curtis J. Mello, Steven A. McCarroll<br>Abstract: <p>Sperm-seq is a high-throughput, droplet-based single-sperm sequencing technology capable of generating thousands of cell-barcoded single-sperm sequencing libraries at one time. This protocol describes Sperm-seq library generation, featuring a full protocol for sperm preparation and suggestions for droplet-based sequencing methods from 10X Genomics to employ. Sperm preparation can be completed in half a day and the full protocol can be completed in 2-3 days, with several wait times and break points.</p><br>Keywords: sequencing, DNA sequencing, sequencing library generation, sperm, spermatozoa, single-cell sequencing, droplet-based sequencing, sperm nucleus decondensation<br><h2>Reagents</h2><p><strong><em>Buffer SMT </em></strong><em>(freeze-thaw buffer adapted from Montag et al. 1992; Samocha-Bone et al. 1998):</em></p><p>250 mM sucrose (from 1M sucrose, Sigma S7903)</p><p>5 mM MgCl<sub>2 </sub>(from 1M MgCl<sub>2</sub>, Sigma M8266)</p><p>10 mM Tris pH 7.5 (from 1 M Tris ph 7.5, ThermoFisher Scientific 15567027)</p><p><em>Example recipe using stocks above (10 mL): </em>2500 uL sucrose, 50 uL MgCl<sub>2</sub>, 100 uL Tris pH 7.5, 7350 uL ultrapure water</p><p><strong><em>Buffer DB </em></strong><em>(salt decondensation buffer adapted from Montag et al. 1992; Samocha-Bone et al. 1998):</em></p><p>113 mM KCl (potassium chloride) (from 1 M KCl Sigma P5405)</p><p>12.5 mM KH<sub>2</sub>PO<sub>4</sub> (monobasic potassium phosphate) (from 1 M KH<sub>2</sub>PO<sub>4</sub>,<sub> </sub>Sigma P9791)</p><p>2.5 mM Na<sub>2</sub>HPO<sub>4</sub> (disodium phosphate) (from 1 M Na<sub>2</sub>HPO<sub>4</sub>,<sub> </sub>Sigma S3264)</p><p>2.5 mM MgCl<sub>2</sub> (magnesium chloride) (from 25 uL 1 M MgCl<sub>2</sub>, Sigma M8266)</p><p>20 mM Tris pH 7.5 (from 1 M Tris ph 7.5, ThermoFisher Scientific 15567027)</p><p><em>Example recipe using stocks above (will make 10 mL total when heparin and beta-mercaptoethanol that are added during experiments are taken into account): </em>1130 uL KCl, 125 uL KH<sub>2</sub>PO<sub>4</sub>, 25 uL Na<sub>2</sub>HPO<sub>4</sub>, 25 uL MgCl<sub>2</sub>, 200 mM Tris pH 7.5, 8145 uL ultrapure water</p><p><strong><em>10X Genomics reagents compatible with single-cell DNA sequencing:</em></strong></p><p><strong>Option 1</strong> (early version of reagents):&nbsp;10X version 1 GemCode reagents, as described in 10X GemCode User Guide Rev C “Required Materials and Equipment” section: GemCode Gel-Bead Kit; GemCode Chip Kit; GemCode Library Kit</p><p><strong>Option 2</strong> (currently sold reagents): 10X Chromium Single Cell DNA Reagent kits, as described in 10X Chromium Single Cell DNA User Guide Rev B “Introduction” section: Chromium Single Cell DNA Library Kit, 16 rxns PN-1000025; Chromium Single Cell DNA Gel Bead Kit, 16 rxns PN-1000026; DynabeadsTM MyOneTM SILANE, 16 rxns PN-2000048; Chromium Single Cell DNA Cell Bead Reagent Kit, 16 rxns PN-1000023; Chromium Single Cell DNA Cell Bead Polymer Kit, 16 rxns PN-1000031; Chromium i7 Multiplex Kit, 96 rxns PN-120262; Chromium Chip C Single Cell DNA Kit, 48 rxns PN-1000022; Chromium Chip D Single Cell DNA Kit, 48 rxns PN-1000042</p><p><strong><em>Other required reagents:</em></strong></p><p>1X PBS pH 7.4 (Life Technologies 10010023)<em>﻿</em></p><p>10 mM (180 mg/mL) heparin, from Heparin Sodium Salt from Porcine (Sigma H3393), diluted in ultrapure water</p><p>100 mM beta-mercaptoethanol (BME) diluted from 14.3 M BME (Sigma M6250) (<strong>use in fume hood</strong>)</p><p>0.1 units/uL Heparinase 1 (Sigma H2519) diluted in ultrapure water<em> </em>(following Taylor 1997; store at -20 degrees Celcius)</p><p>10X SYBR Green I (diluted from 10,000X, ThermoFisher Scientific S7563)<em>﻿</em></p><h2>Equipment</h2><p>Nonstick 1.5 mL microfuge tubes (ThermoFisher Scientific AM12450)</p><p>Fuchs-Rosenthal C-chip hemocytometer (VitaScientific NANC07005) (or other hemocytometer)</p><p><strong><em>10X Genomics equipment compatible with single-cell DNA sequencing:</em></strong></p><p><strong>Option 1</strong> (early version of reagents):&nbsp;10X GemCode controller and other equipment described in 10X GemCode User Guide Rev C “Required Materials and Equipment” section and other equipment described therein</p><p><strong>Option 2</strong> (currently sold reagents): 10X Chromium controller; FlowmiTM Filters, 50 rxns, PN-1000055; and other Chromium accessories and equipment described in 10X Chromium Single Cell DNA User Guide Rev B “Introduction” section: as described therein</p><h2>Procedure</h2><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Thaw sperm aliquot(s) containing at least 250,000 sperm cells (typically 10 uL) at room temperature.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Quick spin sperm aliquot and transfer to nonstick 1.5 mL tube.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Spin for 10 minutes at 400g at 4 degrees Celsius. Remove supernatant.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Resuspend sperm pellet in 10 uL PBS.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Repeat spin from (3). Resuspend sperm pellet in 2.5 uL Buffer SMT.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Submerge tube for 5-10 seconds in liquid nitrogen. Uncap tube to de-pressurize. Hold tube in fist until thawed.</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;Repeat (6) for a total of 3 freeze-thaw cycles.</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Make 100 uL Buffer DB with freshly added heparin and BME (enough for 4 samples): combine 1.5 uL 10 mM heparin, 2 uL 100 mM BME, and 96.5 uL Buffer DB.</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;Add 22.5 uL of Buffer DB with heparin and BME from (8) to each sperm sample.</p><p>10.&nbsp;Incubate at 37 degrees Celsius for 45 minutes. Thaw heparinase on ice during the latter half of this incubation.</p><p>11.&nbsp;Remove sample(s) to room temperature. Add 5 uL freshly-thawed heparinase and pipette gently to mix.</p><p>12.&nbsp;Incubate sample with added heparinase at room temperature for 2 hours before preceding immediately to next steps.</p><p>13.&nbsp;Generate a DNA-stained 1:100 dilution of sperm (for counting under fluorescence microscope) by combining 1 uL sperm mixture, 10 uL 10X SYBR I, and 89 uL PBS. Allow to incubate in the dark at room temperature, with or without gentle rotation, for 5-10 minutes.</p><p>14.&nbsp;Add 20 uL diluted, stained sperm to Fuchs-Rosenthal C-chip hemocytometer and count the number of sperm present in 5 big squares under green fluorescent light. (Sperm appear as bright fluorescent dots; un-decondensed sperm are compact dots while sperm at this state, after decondensation, are bigger circles.) Sperm concentration per microliter is the number of sperm in 5 big squares multiplied by 100. (<em>Or, use fluorescence to count sperm via a known and trusted method.)</em></p><p>15.&nbsp;<strong>If using 10X Genomics GemCode (old) reagents <em>(for other reagents continue to step 16)</em>: </strong></p><p>a.&nbsp;&nbsp;&nbsp;&nbsp;Dilute sperm to a final number of 10,833 in 5 uL ultrapure water.</p><p>b.&nbsp;&nbsp;&nbsp;&nbsp;Make master mix: per sample, combine 32.5 uL GemCode reagent mix, 1.5 uL primer release agent, 9.2 uL GemCode polymerase, and 16.8 uL ultrapure water.</p><p>c.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add 60 uL of master mix to each sperm sample, pipetting gently with wide bore pipettes to mix.</p><p>d.&nbsp;&nbsp;&nbsp;&nbsp;Vortex GemCode beads at full spead for 25 seconds.</p><p>e.&nbsp;&nbsp;&nbsp;&nbsp;Dilute vortexed GemCode beads 1:11 ultrapure water (to a total volume of at least 90 uL per sample).</p><p>f.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Prepare chip for droplet generation (per 10X Genomics’ protocol): add 60 uL of sample-master mix mixture in the sample wells, then 85 uL of freshly-pipette-mixed 1:11 diluted-bead mixture to the bead wells, then 150 uL droplet generation oil to the oil wells of the chip.</p><p>g.&nbsp;&nbsp;&nbsp;&nbsp;Generate and process droplets following 10X GemCode User Guide Rev C Steps 5.36-5.5.2.</p><p>h.&nbsp;&nbsp;&nbsp;&nbsp;Complete library generation following 10X GemCode User Guide Rev C Steps 5.6-6.2.6.</p><p>16.&nbsp;<strong>If using 10X Genomics Chromium Single-cell DNA (new) reagents <em>(if using previous reagents follow step 15 instead)</em>: </strong>Follow 10X Chromium Single Cell DNA User Guide Rev B “Chromium Single Cell DNA Reagents kits” protocol for generating single-cell sequencing libraries (Step 1- “Cell Bead Generation,” Step 2 “Cell Bead Processing,” Step 3 “GEM Generation &amp; Barcoding,” Step 4 “Post GEM Incubation Cleanup &amp; QC”, and Step 5 “Library Construction”), using the prepared sperm as input. Target 2000 cells for recovery. This process can be paused at any point recommended by 10X.</p><p>17.&nbsp;Sequence according to 10X’s and Illumina sequencer instruction. (Generally, sequence yield is improved by increasing the input library concentration slightly from that recommended by Illumina.)</p><p>18.&nbsp;&nbsp;&nbsp;&nbsp;Process sequence data as described in Bell <em>et al </em>“Insights about variation in meiosis from 31,228 human sperm genomes” (<em>Nature</em> 2020).</p><h2>Troubleshooting</h2><p>If sequence data is not of sufficient quality (uneven coverage across the genome; unclear boundary between beads associated with whole sperm cells and beads associated with ambient DNA), the following modifications may be helpful to troubleshoot sperm decondensation:</p><p>Make fresh buffers (prior to step 1)</p><p>Vary BME amount – try 1mM in final sperm treatment buffer rather than 2mM (step 8)</p><p>Decrease the number of freeze-thaw cycles (steps 6-7)</p><p>Pipette gently using wide-bore pipette tips (all steps where sperm are mixed)</p><h2>Time Taken</h2><p>Sperm preparation: ~3.5 hours</p><p>Sperm quantification: ~15 minutes (more for more samples)</p><p>Library preparation: 1-3 days, depending on kit/approach used and when breaks are taken</p><h2>References</h2><p>Montag, M., V. Tok, S.L. Liow, A. Bongso and S.C. Ng. In vitro decondensation of mammalian sperm and subsequent formation of pronuclei-like structures for micromanipulation. <em>Mol Reprod Dev</em> <strong>33</strong>(3): 338-346. (1992)</p><p>Samocha-Bone, D., L.M. Lewin, R. Weissenberg, Y. Madgar, Y. Soffer, L. Shochat and R. Golan. In-vitro human spermatozoa nuclear decondensation assessed by flow cytometry. <em>Mol Hum Reprod</em> <strong>4</strong>(2): 133-137. (1998)</p><p>Taylor, A.C. Titration of heparinase for removal of the PCR-inhibitory effect of heparin in DNA samples. <em>Mol Ecol</em> <strong>6</strong>(4): 383-385. (1997)</p><h2>Acknowledgements</h2><p>We thank Evan Macosko for his excellent early advice on sperm preparation and single-cell sequencing.</p></body></html>