<html><head><title>Spatio-temporal transcriptome construction of early mouse embryo with Geo-seq and Auto-seq</title></head><body><h1>Spatio-temporal transcriptome construction of early mouse embryo with Geo-seq and Auto-seq</h1>Authors: 	Cui Guizhong, Jun Chen, Shirui Chen, Yun Qian, Guangdun Peng, Naihe Jing<br>Abstract: <p>The regionalization of cell fates heralding the formation of the basic body plan is conserved in vertebrate embryos, the mechanisms of which is instrumental for understanding embryonic programming and stem cell-based translational study. However, a comprehensive genome-wide molecular annotation of the mechanism that determines the lineage specification and cell fate decision has not been clarified. Here, we combined laser microdissection-based spatial transcriptome analysis (Geo-seq) and automated platform-based low-input RNA sequencing (Auto-seq), and generated a series of spatial transcriptome dataset that encompass mouse embryos from pre-gastrulation (E5.5) to late gastrulation (E7.5) on all germ layers in the embryo proper.&nbsp;</p><br>Keywords: Geo-seq, Auto-seq, Spatio-temporal transcriptome, Early embryo development<br><h2>Reagents</h2><p>Nuclease-free water (Ambion, cat. no. AM9930)</p><p>BSA (Sigma, cat.no. B8894)</p><p>Ethanol (SinoPharm Shanghai)</p><p>PBS (Thermo Fisher Scientific, cat. no. 10010023)</p><p>Protector RNase inhibitor (Roche, cat. no. 03335399001)</p><p>Nonidet P-40 SP (Roche, cat. no. 11332473001)</p><p>DTT (Invitrogen, cat. no. 18064-014)</p><p>Superscript II reverse transcriptase (Invitrogen, cat. no. 18064-014)</p><p>Betaine (Sigma-Aldrich, cat. no. 61962)</p><p>RNase-zap (Ambion, cat.no. AM9780,)</p><p>Tissue Freezing Medium (Leica Mycrosystems, cat. no. 020108926)</p><p>Cresyl violet acetate (Sigma-Aldrich, cat.no. C5042)</p><p>Guanidine Isothiocyanate Solution (4 M guanidine isothiocyanate, 50 mM TrisHCl (pH 7.5), 25 mM EDTA; Invitrogen, cat.no. 15577-018)</p><p>Glycogen (20mg/1ml; Roche, cat.no.10901393001)</p><p>Sodium Acetate (3 M, pH 5.5; Ambion, cat. no. AM9740)</p><p>dNTP mix (2.5 mM each; Takara, cat. no. 4030)</p><p>Magnesium chloride (MgCl2; 1.00 M±0.01 M; Sigma-Aldrich, cat. no. M8266)</p><p>KAPA HiFi HotStart ReadyMix (2X; KAPA Biosystems, cat. no. KK2601)</p><p>Agencourt Ampure XP beads (Beckman Coulter, cat. no. A 63881)</p><p>Qubit dsDNA High-Sensitivity (HS) Kit (Invitrogen, cat. no. Q32851)</p><p>Agilent High-Sensitivity DNA Kit (Agilent Technologies, cat. no. 5067-4626)</p><p>Nextera XT DNA Sample Preparation Kit, 96 samples (Illumina, cat. no. FC-131-1096)</p><p>TruePrep DNA library pre kit V2 (Vazyme, cat. No. TD503-02, TD502-02)</p><p>3’CDS, TSO and IS primers were ordered from Sangon Biotech (Shanghai, China)</p><p><strong>3’CDS primer (5’ -AAGCAGTGGTATCAACGCAGAGTACT30VN-3’ )</strong></p><p>At the 3’end, 'N' is any base and 'V' is A, C or G. Dissolve the oligonucleotide in Nuclease-free water to a final concentration of 10 μM. Store this oligo at −20 °C for up to 6 months.</p><p><strong>TSO primer：5’ - AAGCAGTGGTATCAACGCAGAGTGAAT/rG//rG//iXNA_G/ -3’</strong></p><p>At the 3’ end, there are two riboguanosines (rG) and one LNA-modified guanosine (iXNA_G). Dissolve the TSO primer in Nuclease-free water to a final concentration of 10 μM. Store them at −20 °C for up to 3 months, and avoid repeated freeze–thaw cycles.</p><p><strong>IS-PCR primer 5’ - AAGCAGTGGTATCAACGCAGAGT -3’</strong></p><p>Dissolve the IS-PCR primer in Nuclease-free water to a final concentration of 10 μM. Store them at −20 °C for up to 6 months.</p><h2>Equipment</h2><p>Cryostat (Leica Mycrosystems, CM1950)</p><p>CellCut laser microdissection system (MMI)</p><p>PEN Membrane slide (MMI, cat.no. 50102)</p><p>IsolationCap (0.2ml; MMI, cat.no. 50206)</p><p>Microcentrifuge tube (Axygen, cat. no. MCT-150-NC)</p><p>Thermal cycler (Applied Biosystems, 9700)</p><p>Centrifuge (Labocene, Scan speed: 1730R)</p><p>PCR Magnet (Applied Biosystems, cat. no. 49-2025)</p><p>Thin-walled PCR tubes with caps (Axygen, cat. no. PCR-02-L-C)</p><p>Filter tips: 10, 20, 200 and 1000 µl (Axygen, cat. nos. TF-300-R-S, TF-20-R-S, TF-200-R-S, TF-1000-R-S)</p><p>Qubit assay tubes (Invitrogen, cat. no. Q32856)</p><p>GC100T-10 (Harvard Apparatus, 1.0 mm x 0.78 mm x 10 cm)</p><p>Hard-Shell® 96-Well PCR Plates (BioRad, cat. no. HSP9601)</p><p>Round Bottom of 96-Well Cell Culture Cluster (Costar, cat. no. 3879)</p><p>Qubit 2.0 fluorometer (Invitrogen, cat. no. Q32866)</p><p>Agilent 2100 Bioanalyzer (Agilent Technologies, cat. no. G2938C)</p><p>Perkin Elmer Well Plates/Microplates (PerkinElmer, EU, cat. no. 6005279）</p><p>Bravo automated liquid handling platform (Agilent)</p><p>Envision (PerkinElmer)</p><p>MANTIS® Liquid Handler (FORMULATRIX)</p><p>Micropipette Puller P-97 (Sutter)</p><h2>Procedure</h2><p>Embryos collection and embedding</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Keep female C57BL/6J mice on a normal day/night cycle, and check for the plugs on the following day. The presence of a vaginal plug is considered embryonic day 0.5 (E0.5).</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;At the Distal-visceral-endoderm (E5.5), Anterior-visceral-endoderm (E6.0), Early-Streak (E6.5), late Mid-Streak (E7.0) and No-bud (E7.5) stages, kill the pregnant mouse with cervical dislocation according to animal ethics guidelines.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Carefully dissect mouse decidua from the uterus under a dissecting microscope, and wash embryos in a 6-cm petri dish containing RNase-free PBS on ice.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Transfer the decidua into a cryomold and add precooled OCT compound. Gently move the tissues down to near the bottom of the cryomold and put them in the vertical position.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Immediately freeze the embryo in dry ice, and stored at −80 °C for several weeks.</p><p><br></p><p>Pre-gastrulation stages (E5.5 and E6.0) samples collection</p><p>At E5.5, the epiblast forms a cup-shaped pseudostratified columnar epithelium that surrounds the visceral endoderm (VE), and the visceral endoderm becomes polarized in the proximal–distal dimension. The cells located at the distal tip of the egg cylinder (distal visceral endoderm, DVE) shift anteriorly (anterior visceral endoderm, AVE) as the embryo develops (E5.5-E6.0). These cells are tall columnar in morphology, unlike the squamous visceral endoderm cells overlaying the epiblast but outside the DVE/AVE domain[1]. At E5.75, AVE is located on one side of the shorter axis of the cross section of the cylinder opposite from the site of primitive streak formation. The prospective anterior–posterior embryonic axis from the short axis to the long axis of the ellipsoidal cross section of the egg cylinder between E6.0–E6.5. Therefore, we respectively collected two sectors of epiblast (Epi1/Epi2) and visceral endoderm (En1/En2) along with long axis of the ellipsoidal cross section at E5.5 stage, and divide epiblast and VE into anterior sectors (A/EA) and posterior sectors (P/EP) respectively along with long axis of the ellipsoidal cross section at E6.0 stage. Considering the cell number of sectors, we increase thickness from 15 μm to 30 μm at E5.5 stage for collecting two part of epiblast or VE.</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Clean the cryostat and set the temperature on the cryostat to −20 °C. Open UVC disinfection for 30min.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Remove the frozen block of OCT compound from the cryomold, and affix it to a metal stage at the OCT for cryosectioning.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Set the cutting thickness of E5.5 embryo is 15 μm or 30 μm, and transfer serial sections of embryo onto LCM PEN membrane slides. However, set the cutting thickness of E6.0 embryo is 15 μm, and transfer alternate sections of embryo onto LCM PEN membrane slides. Therefore, harvest 6 sections (15 μm/section, serial sections) or 3 sections (30 μm/section, serial sections) of E5.5 embryos, and 5 sections (15 μm/section, alternate sections ) of E6.0 embryos.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Place the membrane slides with cryosections in 100% (vol/vol), 95% (vol/vol) and 70% (vol/vol) ethanol solution for 30 s each.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Stain with 1% (wt/vol) cresyl violet in 70% (vol/vol) ethanol for 30 s.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Dehydrate the slides in 70%, 95% and 100% (vol/vol) ethanol for 30 s each.</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;After slides dry, make a sandwich by putting a cover-glass slide under the stained membrane slide.</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Choose the different cell populations as defined by both position and morphology. Epiblast are pseudostratified columnar epitheliums and endoderm are squamous epithelial cells in outer layer. The sampling strategy was modified in accordance to the size of the section. In the epiblast/ectoderm, sampling was collected for two sectors (Epi1/Epi2) of E5.5 embryos and only from A (anterior) and P (posterior) sectors of E6.0 embryos. From the endoderm, only two sectors (Epi1/Epi2) of E5.5 embryos and only A and P samples of E6.0 embryos were collected.</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;Cut target cells, and collect them with IsolationCaps.</p><p><br></p><p>Gastrulation stages (E6.5, E7.0 and E7.5) samples collection</p><p>At E6.5, the primitive streak appears as a local morphological discontinuity in the posterior epiblast epithelium. At the primitive streak, the epiblast cells undergo an epithelial to mesenchymal transition, then ingress in-between the epiblast and endoderm to become incorporated into either the mesoderm. Therefore, mouse embryo develops ‘inside-out’ pattern, with the internally-located ectoderm surrounded by the mesoderm and endoderm.</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Clean the cryostat and set the temperature on the cryostat to −20 °C. Open UVC disinfection for 30min.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Remove the frozen block of OCT compound from the cryomold, and affix it to a metal stage at the OCT for cryosectioning.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Set the cutting thickness of E6.5 and E7.0 embryos are 15 μm, whereas cutting thickness of E7.5 embryos is 20 μm. Transfer alternate sections of embryo onto LCM PEN membrane slides.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Place the membrane slides with cryosections in 100% (vol/vol), 95% (vol/vol) and 70% (vol/vol) ethanol solution for 30 s each.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Stain with 1% (wt/vol) cresyl violet in 70% (vol/vol) ethanol for 30 s.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Dehydrate the slides in 70%, 95% and 100% (vol/vol) ethanol for 30 s each.</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;After slides dry, make a sandwich by putting a cover-glass slide under the stained membrane slide.</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Choose the different cell populations as defined by both position and morphology. Epiblast are pseudostratified columnar epitheliums on the inside of the embryo, and endoderm are squamous epithelial cells in outer layer with the mesoderm in the middle. The sampling strategy was modified in accordance to the size of the section. In the epiblast/ectoderm, sampling was collected for A (anterior) and P (posterior) sectors of E6.5 embryos, and lateral sectors (left lateral and right lateral) were captured in E7.0 and E7.5 embryo. From the endoderm and mesoderm, only A and P samples were collected.</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;Cut target cells, and collect them with IsolationCaps.</p><p><br></p><p>Automated platform-based cDNA library construction (Auto-seq)</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;RNA extraction, reverse transcription and cDNA preamplification were followed Geo-seq protocol[2].</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Add 25 µl ddH2O into 15 µl PCR product, transfer 40 µl mixture into 96-well PCR Plates as the Source Plates.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Equilibrate AMPure XP beads to room temperature, and then vortex thoroughly for several seconds. After 30 min, add 30 µl of AMPure XP beads (0.75:1 ratio) to the Round Bottom of 96-Well Cell Culture Cluster Plates as the Beads Plates.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Add 16 µl of 0.1× TE to 96-well PCR Plates as the Elusion Buffer Plates, and add freshly made 80% (vol/vol) ethanol to container as the Ethanol Plates.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Put the Source Plates, Beads Plates, Ethanol Plates and Elusion Buffer Plates on&nbsp;&nbsp;different position of Bravo, use Automated platform to purify the PCR product.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;To test the concentration of purified cDNA, use 1 µl of purified PCR product as a template to perform a Qubit Fluorometer assessment with a dsDNA HS assay kit on Envision.</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;Normalize the purified PCR product to 0.2 ng/µl with MANTIS Liquid Handler, and add qualified PCR product into 96-well PCR Plates as the Source Plates.</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Mix 10 µl Tagment DNA Buffer (TD) and 5 µl Amplicon Tagment Mix (ATM) each reaction, and add 5~8 reactions buffer into 96-well PCR Plates as the Tagment Buffer Plates.</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;Add Neutralize Tagment Buffer into 96-well PCR Plates as the NT-Buffer Plates.</p><p>10.&nbsp;Mix different Index 1 (i7) adapters and Index 2 (i5) adapters as the Index Plates.</p><p>11.&nbsp;Add Nextera PCR Master Mix into 96-well PCR Plates as the PCR Buffer Plates.</p><p>12.&nbsp;Put the Source Plates, Tagment Buffer Plates, NT-Buffer Plates, Index Plates and PCR Buffer Plates on different position of Bravo, use Automated platform to construct libraries.</p><p>13.&nbsp;Add ddH2O into constructed libraries to 40 µl as the Source Plates.</p><p>14.&nbsp;Equilibrate AMPure XP beads to room temperature, and then vortex thoroughly for several seconds. After 30 min, add 24 µl of AMPure XP beads (0.6:1 ratio)&nbsp;and 8 µl of AMPure XP beads (0.2:1 ratio) to the Round Bottom of 96-Well Cell Culture Cluster Plates as the Beads Plates.</p><p>15.&nbsp;Add 16 µl of 0.1× TE to 96-well PCR Plates as the Elusion Buffer Plates, and add freshly made 80% (vol/vol) ethanol to container as the Ethanol Plates.</p><p>16.&nbsp;Put the Source Plates, Beads Plates, Ethanol Plates and Elusion Buffer Plates on&nbsp;&nbsp;different position of Bravo, use Automated platform to purify the libraries product.</p><p>17.&nbsp;To test the concentration of purified cDNA libraries, use 1 µl of purified libraries as a template to perform a Qubit Fluorometer assessment with a dsDNA HS assay kit on Envision.</p><p><br></p><p>cDNA sequencing</p><p>Sequencing was performed on the Illumina HiSeq 2500 sequencer and Illumina NovaSeq 6000 sequencer using a 150 bp paired-end-reads setting.</p><h2>Troubleshooting</h2><p><em>The data of an embryo is incomplete: </em></p><p>There are possible two reasons. One, sample lost. You can verify the LCM samples collected on the IsolationCaps under a microscope. Two, samples do not pass quality control. Tissues should be isolated expeditiously, and best-practice preparation and handling of RNA samples is essential. For example, all the reagents should be RNase-free and all the instruments should be treated with RNase-removing reagents such as RNase-Zap. </p><p><em>&nbsp;</em></p><p><em>Low quality of biological replications. E.g. the expression of marker genes and correlation of sequencing replicates were not consistent.</em></p><p>Make sure that the embryo stage was consistent, and the morphology of cryosections is comparable. Be sure to choose the same target cell areas and size, especially for the endoderm samples. Although Automated platform-based cDNA library construction can reduce batch effect, we recommend removing batch effect as far as possible. For example, use the same setting with same depth for sequencing.</p><h2>Time Taken</h2><p>Embryos collection and embedding: 1-3 hours.</p><p>Samples collection: 3-8 hours, depending on number of samples.</p><p>Automated platform-based cDNA library construction: 2.5 days.</p><p>cDNA sequencing: 2 days.</p><h2>Anticipated Results</h2><p>Please see Associated Publication.</p><h2>References</h2><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Rivera-Pérez, J.A. and A.-K. Hadjantonakis, <em>The dynamics of morphogenesis in the early mouse embryo.</em> Cold Spring Harbor perspectives in biology, 2014: p. a015867.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Chen, J., et al., <em>Spatial transcriptomic analysis of cryosectioned tissue samples with Geo-seq.</em> Nat Protoc, 2017. <strong>12</strong>(3): p. 566-580.</p><h2>Acknowledgements</h2><p>The authors would like to thank technical supports from chemical biology core facility and animal core facility in SIBCB.</p></body></html>