<html><head><title>ItChIP-simultaneous indexing and tagmentation-based ChIP-seq</title></head><body><h1>ItChIP-simultaneous indexing and tagmentation-based ChIP-seq</h1>Authors: Shanshan Ai, Haiqing Xiong, Chen C. Li, Yingjie Luo, Yaxi Liu, Xianhong Yu, Aibin He<br>Abstract: <p>Single-cell measurement of chromatin states, including histone modifications and non-histone protein binding, remains challenging. We present a low-cost, efficient ChIP-seq (simultaneous indexing and tagmentation-based ChIP-seq, itChIP), compatible to both low-input and single cells for profiling chromatin states. This single-cell itChIP approach combines chromatin opening, simultaneous cellular indexing and chromatin tagmentation in a single tube, processing samples with tens of single cells in rarity or with thousands of single cells per assay, and the entire procedure can be finished in two days. The sc-itChIP data acquire ~9,000 unique reads per cell, sufficiently capturing the earliest epigenetic priming along cell fate transition and the basis for cell-type specific enhancer usage. Our results demonstrate that itChIP is a generalizable technology for single-cell chromatin profiling of epigenetically heterogeneous cell populations in many biological processes. This step-by-step protocol is related to the publication “Profiling chromatin state by single-cell itChIP-seq” in Nature Cell Biology.</p><br>Keywords: chromatin, epigenetic heterogeneity, histone modification, transcription factor, single cell ChIP-seq<br><h2>Introduction</h2><p>Profiling epigenetic states largely relies on measurement of chromatin occupancy of covalently modified histones, histone regulators, and transcription factors (TFs)<sup>1-5</sup>. Among them, chromatin immunoprecipitation followed by massively parallel sequencing (ChIP-seq) is the method of choice, allowing the rapid identification of transcriptional regulatory elements genome-wide<sup>6-9</sup>. Standard ChIP-seq requires a million or more cells. Reducing the required input to one to hundreds of cells would open up an array of new applications. For instance, understanding the transcriptional regulatory networks that control lineage specification in developing embryos would be enabled by using ChIP-seq to probe lineage-specific enhancers from low amounts of cells. To define differentiation trajectories and potentials of rare spatiotemporal progenitor cells, single-cell RNA-seq (scRNA-seq) techniques have become widely used to separate subpopulations from transcriptomically heterogeneous cell populations<sup>10-13</sup>. However, a robust single-cell ChIP-seq tool remains lacking to understand epigenetic heterogeneity in complex tissue or cell populations at single cell level.&nbsp;</p><p>In the past decade, epigenomic studies have seen great strides towards reduced input sample size, including nano-ChIP<sup>14</sup>, UNI-NChIP<sup>15</sup>, STAR ChIP<sup>16</sup>, MOWChIP<sup>17</sup>, CUT&amp;RUN<sup>18</sup>and ChIL-seq<sup>19</sup>, but development of a generalizable, low-cost, and robust method effective on single cells or small cell populations lags behind. Microfluidic systems have been used to profile histone marks by ChIP-seq using as few as 100 cells<sup>17</sup>. A study by Rotem et al. demonstrated the proof-of-concept use of microfluidic systems for single-cell ChIP (Drop-ChIP). However, at ~800 reads per cell, the sparse data yield from Drop-ChIP limits its potential applications<sup>20</sup>.&nbsp;</p><p>Through overcoming these limitations, we present simultaneous indexing and tagmentation of opened/relaxed chromatin-based ChIP-seq (itChIP-seq)(Figure 1), which enables genome-wide profiling of histone modifications and non-histone proteins from ultra-low input (as few as 100 cells) and various histone modifications of single-cell samples (Figure 2), enabling processing rare cell populations of less than 100 single cells and abundant samples of thousands of single cells per assay. Furthermore, itChIP eliminates the requirement for specialized microfluidic devices and is generalizable and affordable for most biomedical research laboratories.&nbsp;</p><h2>Reagents</h2><p>Bl21 (DE3)</p><p>Formaldehyde Solution (SIGMA, F8775-500ML)</p><p>Glycine (AMRESCO, 0167-1KG)</p><p>Phosphate Buffered Daline (PBS) (Hyclone, SH30256.01)</p><p>Sodium butyrate (SIGMA, 303410-100G)</p><p>Collagenase Type 2 (Worthington, LS004177)</p><p>H3K4me3 antibody (Millipore, Cat.No: 04-745)</p><p>Trypsin (Gibco, 27250-018)</p><p>Ampicillin Na (AMRESCO, VT0236)</p><p>IPTG (INALCO, 1758-1400)</p><p>HEPES free acid (AMRESCO, 0511-1KG)</p><p>Glycerol (SIGMA, 49767-100ML)</p><p>Triton X-100 (SIGMA, T8787-50ML)</p><p>EDTA-free Protease Inhibitor Cocktail (Roche, 04693132001)</p><p>Phenyl Methane Sulfonyl Fluoride (PMSF) (AMRESCO, 0754-5G)</p><p>Dithiothreitol (DTT) (INALCO, 1758-9030)</p><p>TRIS (AMRESCO, 0497-5KG)</p><p>TAPS (SIGMA, T-5130)</p><p>Magnesium chloride solution (MgCl<sub>2</sub>) (SIGMA, 68475-100ML)</p><p>Potassium chloride (KCl) (XiLong SCIENTIFIC, 7447-40-7)</p><p>Calcium chloride dihydrate (CaCl<sub>2</sub>)</p><p>(XiLong SCIENTIFIC, 10035-04-8)</p><p>Sodium chloride (NaCl) (SIGMA, 303410-100G)</p><p>tRNA (Invitrogen, cat# AM7119)&nbsp;</p><p>N,N-Dimethylformamide (DMF) (SIGMA, D4551-250ML)</p><p>Sodium deoxycholate (SIGMA, D6750)</p><p>SODIUM DODECYL SULFATE (SDS) (AMRESCO, 0227-1KG)</p><p>Proteinase K (AMRESCO, 0706-100MG)</p><p>KAPA HiFi HotStart ReadyMix (BIOSYSTEMS, KM2602)</p><p>AMPure XP (BECKMAN COULTER, A63881)</p><p>Streptavidin M280 beads&nbsp;(Invitrogen, 60210)</p><p>Dynabeads protein A&nbsp;(Invitrogen, 10001D)</p><p>H3K27me3 antibody (Millipore, Cat.No: 07-449)</p><p>H3K27ac antibody (Abcam, Cat.No: ab4729)</p><p>EZH2 antibody (CST, Cat.No: S246S)</p><p>RNA polymerase II CTD antibody (Abcam, Cat.No: ab5408)</p><h2>Equipment</h2><p>1.5 ml tube&nbsp;(Axygen, MCT-150-C)</p><p>0.2 ml 8-strip PCR tube&nbsp;(Axygen, PCR-0208-C)</p><p>DynaMag™-PCR Magnet</p><p>Qubit 2.0 Fluorometer&nbsp;(Thermo Fisher Scientific)</p><p>Centrifuge&nbsp;(Eppendorf)</p><p>Thermal cycler&nbsp;(Eppendorf)</p><p>ThermoMixer</p><p>Qsonica, Q800R</p><p>CFX96 Real-Time System</p><h2>Procedure</h2><p><strong>Reagents Setup</strong></p><p><u>• Hypotonic Buffe</u>﻿<u>r</u></p><p>20 mM HEPES(PH7.9)</p><p>10 mM KCl</p><p>10% Glycerol</p><p>0.2%NP-40</p><p>0.3% SDS<u>﻿</u></p><p><u>• 5XTMgCl-DMF&nbsp;</u></p><p><u>﻿</u>50 mM TAPS-NaOH(PH8.3)</p><p>25 mM MgCl2</p><p>50% DMF</p><p><u>• Releasing Buffer</u></p><p>0.8% SDS</p><p>1% TX-100</p><p>2 mM EDTA</p><p>20 mM Tris-HCl(PH8.0)</p><p>200 mM NaCl<u>﻿</u></p><p><u>• Low Salt Buffer﻿</u></p><p>1% TX-100</p><p>2 mM EDTA</p><p>50 mM HEPES(PH7.9)</p><p>150 mM NaCl</p><p>0.1% Sodium Deoxycholate</p><p><u>• High Salt Buffer﻿</u></p><p>1% TX-100</p><p>2 mM EDTA</p><p>50 mM HEPES(PH7.9)</p><p>500 mM NaCl</p><p>0.1% Sodium Deoxycholate</p><p><u>• ChIP Elution Buffer﻿</u></p><p>50 mM Tris-HCl(PH8.0)</p><p>10 mM EDTA</p><p>1% SDS<u>﻿</u></p><p><strong>Detailed Experiment Procedure for Low Input ItChIP</strong></p><p>The important steps of the procedure for low-input itChIP were optimized as shown in Figure 3.</p><p><u>• Cell preparation and chromatin opening.&nbsp;</u></p><p>1. Sort fixed 100-1,000,000 cells into 5&nbsp;μl Hypotonic Buffer containing 0.3% SDS in 200 μl tubes.</p><p>2. Open chromatin: incubation the cell at 62°C for 10 min in an Eppendorf ThermoMixer with agitation at 600 rpm.</p><p>3. Quench SDS: add 1.2&nbsp;μl of 10% Triton X-100 to the samples and incubated at 37°C for 60 min in a PCR thermal cycler.</p><p><u>• Chromatin tagmentation.</u></p><p>4. Tn5 tagmentation: add 2&nbsp;μl 5XTMgCl-DMF, 3&nbsp;μl 12.5&nbsp;μM Tn5 Transposase and 0.8&nbsp;μl ddH2O to the 6.2&nbsp;μl cell mixture (using 10-fold dilution of Tn5 for the cell number less than 1,000, Table 1). The reaction was performed at 37°C for 1 h, followed by addition of 2 µl 250 mM EDTA and incubation at RT for 20 min to stop Tn5 reaction.</p><p><u>• Chromatin release.</u></p><p>5. Collect the sample by centrifugation at 12,000 g for 3 min at 4°C, get rid of the supernatant and resuspend the nuclear pellet with 20&nbsp;μl Releasing Buffer (0.8% SDS, 2 mM EDTA, 20 mM Tris-HCl pH8.0, 200 mM NaCl) by pipetting ~30 times to disperse the nuclei.</p><p>6. Incubate the nuclear lysis at RT for 30 min, followed by brief sonication at 20% power for 4 cycles (15 s ON and 30 s OFF per cycle; Q800R2, Qsonica) to aid in solubilize chromatin.</p><p>7. Dilute the SDS concentration by adding 100&nbsp;μl ChIP Dilution Buffer to the chromatin mixture before centrifugation at 20,000 for 15 min at 4°C to collect soluble chromatin for immunoprecipitation.</p><p><u>• Antibody-based chromatin immunoprecipitation.&nbsp;</u></p><p>8. Add 0.1~1.0&nbsp;μg antibody to the tagmentated soluble chromatin mixture and incubated at 4°C overnight.</p><p>9. Meanwhile, prepare 4&nbsp;μl Dynabeads protein A (Invitrogen, 10001D, 30 mg/ml) beads and block the beads with 150&nbsp;μl 1% BSA/PBS overnight at 4°C.</p><p>10. Next day, transfer the 4 µl protein A beads to the chromatin samples and incubate at 4°C for 2 h with gentle rotation.</p><p>11. Wash beads: the beads were washed thrice with 150&nbsp;μl cold Low Salt Buffer and once with 150&nbsp;μl cold High Salt Buffer.</p><p>12. Transfer the chromatin-antibody-beads complex to new tubes in 150&nbsp;μl cold Low Salt Buffer, and the beads was collected by magnet stand to facilitate removing supernatant.</p><p>13. Suspend the complex with 100&nbsp;μl ChIP Elution Buffer, and reverse-crosslinked at 70°C for 3 h on a thermal mixer at 1500 rpm.</p><p>14. Add 0.1 mg/ml proteinase K to the DNA mixture and incubate at 55°C for 3 h.</p><p>15. Purify the DNA with phenol-chloroform. The purified DNA can be directly used for library preparation.</p><p><u>• Bio-itChIP.</u></p><p>Continue from step 7.</p><p>7-8. Add NaCl to the&nbsp;tagmentated soluble chromatin to final 350 mM.</p><p>7-9. Add 10 µl pre-blocked (1% BSA/PBS block the beads at 4°C for 2 h) streptavidin M280 beads to the tagmentated soluble chromatin&nbsp;at 4°C for 2 h.</p><p>7-10. Wash beads:&nbsp;wash&nbsp;the beads&nbsp;once with 150 µl 2% SDS/PBS, and thrice with 150 µl cold High Salt Buffer.</p><p>7-11. Transfer the chromatin-antibody-beads complex to new tubes in 150&nbsp;μl cold High Salt Buffer, and the beads was collected by magnet stand to remove supernatant.</p><p>7-12. Suspend the complex with 100&nbsp;μl ChIP Elution Buffer, and the ChIPed DNA was purified as above.</p><p><u>• Nextera library preparation and sequencing of low-input itChIP/bio-itChIP.</u></p><p>16. Amplify the purified ChIPed DNA by addition of 1&nbsp;μl 25&nbsp;μM i5 index primer (5′- AATGATACGGCGACCACCGAGATCTACAC[i5]TCGTCGGCAGCGTC-3′), 1&nbsp;μl 25&nbsp;μM i7 index primer (5′-CAAGCAGAAGACGGCATACGAGAT[i7]GTCTCGTGGGCTCGG-3′) (Table 2) and 25&nbsp;μl 2X KAPA master mix to the 23&nbsp;μl purified ChIPed DNA and incubation at 72°C for 5 min, 98°C for 45 s, 14-19 cycles of (98°C for 15 s, 63°C for 30 s, 72°C for 1 min), and final 72°C extension for 5 min.</p><p>17. After PCR, purify the library with 1 X AMPure XP beads once. Size selection was carried out by first 0.5X AMPure XP beads to remove &gt;1kb fragments, and second 0.5X AMPure XP beads to the supernatant to obtain 200-1000 bp fragments for sequencing.</p><p>18. Sequence the libraries with paired-end 150-bp reads on Hiseq X-ten or Novaseq 6000 platform (Illumina).</p><p><strong>Single-cell itChIP</strong></p><p><u>• Single cell chromatin opening.</u></p><p>Follow the procedures descried above: Step 1-3.</p><p><u>• Single cell chromatin tagmentation and indexing﻿.</u></p><p>19. Tn5 tagmentation: add 2&nbsp;μl 5XTMgCl-DMF, 2&nbsp;μl 1.25&nbsp;μM&nbsp;Tn5-T5 (5′-TCGTCGGCAGCGTCTCCACGC[T5]GCGATCGAGGACGGCAGATGTGTATAAGAGACAG-3′), 2&nbsp;μl 1.25&nbsp;μM Tn5-T7 (5′-GTCTCGTGGGCTCGGCTGTCCCTGTCC[T7]CACCGTCTCCGCCTCAGATGTGTATAAGAGACAG-3′) (Table 3), and ddH<sub>2</sub>O with 10 ng yeast tRNA (Invitrogen, cat# AM7119) to 10&nbsp;μl for each well for tagmentation at 55°C for 30 min.</p><p>20. Stop he reaction by adding 2&nbsp;μl 250 mM EDTA to each well, mix well and incubate at RT for 20 min.</p><p><u>• Single cell chromatin release and pooling of indexed single cells﻿.</u></p><p>21. Combine each well of single cell into&nbsp;1.5 ml Eppendorf tube and the nuclei were collected by centrifugation at 12,000 g for 3 min at 4°C.</p><p>22. Discard the supernatant and resuspend the nuclear pellet with 20 µl Releasing Buffer by pipetting ~30 times to disperse the pellets.</p><p>23. Incubate the nuclear lysis at RT for 30 min, followed by brief sonication at 20% power for 4 cycles (15 s ON and 30 s OFF per cycle) to solubilize chromatin.</p><p>24. Dilute the SDS concentration by adding 100&nbsp;μl ChIP Dilution Buffer to the chromatin mixture before centrifugation at 20,000 for 15 min at 4°C to collect soluble chromatin for immunoprecipitation.</p><p><u>• Pooled single cell chromatin immunoprecipitation.&nbsp;</u></p><p>25. Add&nbsp;0.1~1.0&nbsp;μg antibody to the indexed soluble chromatin mixture from pooled single cells and incubate at 4°C overnight, then immunoprecipitation was performed following the low-input itChIP procedure.</p><p><u>• Single-cell Nextera library preparation for sequencing using a custom recipe.</u></p><p>26.&nbsp;Amplify the pooled single cell ChIPed DNA by addition of 1&nbsp;μl 25&nbsp;μM i5 index primer (5′- AATGATACGGCGACCACCGAGATCTACAC[i5]TCGTCGGCAGCGTC-3′), 1&nbsp;μl 25&nbsp;μM i7 index primer (5′- CAAGCAGAAGACGGCATACGAGAT[i7]GTCTCGTGGGCTCGG-3′) and 25&nbsp;μl 2X KAPA master mix to the 23&nbsp;μl purified ChIPed DNA and incubation at 72°C for 5 min, 98 °C for 45 s, 18-20 cycles of (98°C for 15 s, 63 °C for 30 s, 72°C for 1 min), and final 72°C for 5 min.</p><p>27. After PCR, purify the library with 1 X AMPure XP beads once. Size selection was carried out by first 0.5X AMPure XP beads to remove &gt;1kb fragments, and second 0.5X AMPure XP beads to the supernatant to obtain 200-1000 bp fragments for sequencing.</p><p><u>• Custom sequencing of single-cell Nextera libraries.&nbsp;</u></p><p>28. Quantify the libraries by qPCR.</p><p>29. Load 10 pM libraries per lane of HiSeq 2500 (Illumina) using custom sequencing primers (Table 3) following read lengths: 69 bp (Read1) + 43 bp (Index1) + 37 bp (Index2) + 69 bp (Read2) (Supplementary Fig. 6). The first 8 bp of Index1 correspond to the T7 barcode and the last 8 bp to the i7 barcode. The first 8 bp of Index2 correspond to the i5 barcode and the last 8 bp of Index2 correspond to the T5 barcode. About 10% PhiX and 15% spike-in library in which we replaced universal connector A/B sequences with 21/27 bp N were pooled with single-cell itChIP libraries for sequencing to balance the constant bases within each detection cycle.</p><p><u>• Single-cell Truseq library preparation for sequencing using the Illumina standard workflow.&nbsp;</u></p><p>30. Prepare the Truseq libraries by 2 rounds of PCR amplification (Fig. 4).&nbsp;</p><p>31. Add 22.3&nbsp;μl PCR Mix 1 (10&nbsp;μl Q5 reaction buffer (NEB), 10&nbsp;μl Q5 high GC enhancer (NEB), 1&nbsp;μl 10 mM dNTP mix, 0.3&nbsp;μl Q5 polymerase, 0.5&nbsp;μl 25&nbsp;μM connector forward primer (5′-ACACTCTTTCCCTACACGACGCTCTTCCGATCTTCGTCGGCAGCGTCTCCACGC-3′), 0.5&nbsp;μl 25&nbsp;μM connector reverse primer (5′-GACTGGAGTTCAGACGTGTGCTCTTCCGATCTGTCTCGTGGGCTCGGCTGTCCCTGT-3′)) to 27.7&nbsp;μl ChIPed DNA and the reaction was set up at 72°C for 5 min, 95°C for 30 s, 18 cycles of (98°C for 10 s, 63°C for 30 s, 72°C for 1 min), and final 72°C extension for 5 min.&nbsp;</p><p>32. After the first round of PCR, remove the excessive connector primers by addition of 0.5 μl 20 U/μl ExoI (NEB) and incubation at 37°C for 30 min and 72°C for 20 min to inactivate ExoI.</p><p>33. Perform second round of PCR by addition of 9.5&nbsp;μl PCR Mix 2 (2&nbsp;μl Q5 reaction buffer (NEB), 2&nbsp;μl Q5 high GC enhancer (NEB), 0.5&nbsp;μl 10 mM dNTP mix, 0.1&nbsp;μl Q5 polymerase, 1&nbsp;μl 10&nbsp;μM NEB Index primer (5′-CAAGCAGAAGACGGCATACGAGAT[index]GACTGGAGTTCAGACGTGTGCTCTTCCGATCT-3′), 1&nbsp;μl 10&nbsp;μM NEB universal Primer (5′-AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT-3′, 2.9&nbsp;μl ddH<sub>2</sub>O) to 50.5&nbsp;μl of PCR products and incubation at 95°C for 30 s, 5 cycles of (98°C for 10 s, 63°C for 30 s, 72°C for 1 min, and 72°C for 5 min.&nbsp;</p><p>34. Purify the library with 1X AMPure XP beads once and size selection was carried out by first 0.5X AMPure XP beads to remove &gt;1 kb fragments, and second 0.5X AMPure XP beads to the supernatant to obtain 200-1000 bp fragments for sequencing.&nbsp;</p><p>35. Sequence the resulting libraries using standard Truseq recipe for paired-end 150-bp reads on Hiseq X-ten or Novaseq 6000 platform (Illumina).</p><h2>Troubleshooting</h2><p><strong><u>1.</u></strong></p><p><strong>Step: </strong>4.</p><p><strong>Problem: </strong>Small amount of soluble chromatin with larger fragment size.</p><p><strong>Possible reason: </strong>Tn5 complex lose its potency.</p><p><strong>Solution: </strong>Increase the amount of Tn5 complex for tagmentation or prepare new Tn5 transpose complex.</p><p><strong><u>2.</u></strong></p><p><strong>Step</strong>: 8.</p><p><strong>Problem:</strong> ChIP-qPCR exhibits lower signal or higher background.</p><p><strong>Possible reason: </strong>The antibody amount is not optimal.</p><p><strong>Solution:</strong> Optimize the antibody concentration for immunoprecipitation.</p><h2>Time Taken</h2><p><strong>Day 1</strong></p><p><u>• Cell preparation and chromatin opening&nbsp;</u></p><p><u>• Chromatin tagmentation.</u></p><p><u>• Chromatin release.</u></p><p><u>• Antibody-based chromatin immunoprecipitation.&nbsp;</u></p><p><strong>Day 2</strong></p><p><strong>﻿</strong><u>• Wash beads and Purify ChIPed DNA.</u></p><p><u>• Library preparation for sequencing.</u></p><p><br></p><p><u>﻿</u></p><h2>Anticipated Results</h2><p>1. The library exhibits good ChIP-qPCR enrichment before sequence.</p><p>2. Sequence results shows good enrichment of positive signals</p><h2>References</h2><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Rivera, C.M. &amp; Ren, B. Mapping human epigenomes.&nbsp;<em>Cell</em><strong>155</strong>, 39-55 (2013).</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Roh, T.-y., Ngau, W.C., Cui, K., Landsman, D. &amp; Zhao, K. High-resolution genome-wide mapping of histone modifications.<em>Nat Biotechnol</em><strong>22</strong>, 1013-1016 (2004).</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Badeaux, A.I. &amp; Shi, Y. Emerging roles for chromatin as a signal integration and storage platform.&nbsp;<em>Nature Reviews Molecular Cell Biology</em><strong>14</strong>, 211 (2013).</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Kharchenko, P.V., Tolstorukov, M.Y. &amp; Park, P.J. Design and analysis of ChIP-seq experiments for DNA-binding proteins.&nbsp;<em>Nat Biotechnol</em><strong>26</strong>, 1351-1359 (2008).</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Shankaranarayanan, P.<em>et al.</em>Single-tube linear DNA amplification (LinDA) for robust ChIP-seq.&nbsp;<em>Nat Methods</em><strong>8</strong>, 565-567 (2011).</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Stegle, O., Teichmann, S.A. &amp; Marioni, J.C. Computational and analytical challenges in single-cell transcriptomics.&nbsp;<em>Nat Rev Genet</em><strong>16</strong>, 133-145 (2015).</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Adli, M. &amp; Bernstein, B.E. Whole-genome chromatin profiling from limited numbers of cells using nano-ChIP-seq.&nbsp;<em>Nat Protoc</em><strong>6</strong>, 1656-1668 (2011).</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Brind'Amour, J.<em>et al.</em>An ultra-low-input native ChIP-seq protocol for genome-wide profiling of rare cell populations.&nbsp;<em>Nat Commun</em><strong>6</strong>, 6033 (2015).</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Zheng, H.<em>et al.</em>Resetting Epigenetic Memory by Reprogramming of Histone Modifications in Mammals.&nbsp;<em>Mol Cell</em><strong>63</strong>, 1066-1079 (2016).</p><p>10.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Cao, Z., Chen, C., He, B., Tan, K. &amp; Lu, C. A microfluidic device for epigenomic profiling using 100 cells.&nbsp;<em>Nat Methods</em><strong>12</strong>, 959-962 (2015).</p><p>11.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Skene, P.J., Henikoff, J.G. &amp; Henikoff, S. Targeted in situ genome-wide profiling with high efficiency for low cell numbers.&nbsp;<em>Nat Protoc</em><strong>13</strong>, 1006-1019 (2018).</p><p>12.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Harada, A.<em>et al.</em>A chromatin integration labelling method enables epigenomic profiling with lower input.&nbsp;<em>Nat Cell Biol</em><strong>21</strong>, 287-296 (2019).</p><p>13.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Rotem, A.<em>et al.</em>Single-cell ChIP-seq reveals cell subpopulations defined by chromatin state.&nbsp;<em>Nat Biotechnol</em><strong>33</strong>, 1165-1172 (2015).</p><p>14.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Schmidl, C., Rendeiro, A.F., Sheffield, N.C. &amp; Bock, C. ChIPmentation: fast, robust, low-input ChIP-seq for histones and transcription factors.&nbsp;<em>Nat Methods</em><strong>12</strong>, 963-965 (2015).</p><p>15.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Buenrostro, J.D., Wu, B., Chang, H.Y. &amp; Greenleaf, W.J. ATAC-seq: A Method for Assaying Chromatin Accessibility Genome-Wide.&nbsp;<em>Curr Protoc Mol Biol</em><strong>109</strong>, 21.29.21-29 (2015).</p><p>16.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Buenrostro, J.D.<em>et al.</em>Single-cell chromatin accessibility reveals principles of regulatory variation.&nbsp;<em>Nature</em><strong>523</strong>, 486-490 (2015).</p><p>17.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Lake, B.B.<em>et al.</em>Integrative single-cell analysis of transcriptional and epigenetic states in the human adult brain.&nbsp;<em>Nat Biotechnol</em><strong>36</strong>, 70-80 (2018).</p><p>18.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Daley, T. &amp; Smith, A.D. Predicting the molecular complexity of sequencing libraries.&nbsp;<em>Nat Methods</em><strong>10</strong>, 325-327 (2013).</p><p>19.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Nord, A.S.<em>et al.</em>Rapid and Pervasive Changes in Genome-wide Enhancer Usage during Mammalian Development.&nbsp;<em>Cell</em><strong>155</strong>, 1521-1531 (2013).</p><p>20.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Creyghton, M.P.<em>et al.</em>Histone H3K27ac separates active from poised enhancers and predicts developmental state.&nbsp;<em>Proc Natl Acad Sci U S A</em><strong>107</strong>, 21931-21936 (2010).</p></body></html>