<html><head><title>A novel approach reveals genomic landscapes of single-strand DNA breaks with nucleotide resolution in human cells</title></head><body><h1>A novel approach reveals genomic landscapes of single-strand DNA breaks with nucleotide resolution in human cells</h1>Authors: Fan Gao, Ye Cai, Lu Tang, Chun-Lin Wu, Huifen Cao, Dongyang Xu, Lorena Salazar-García, Xueer Han, Fang Wang, Natalie Ricciardi, Thor Wahlestedt, DingDing Cai, Huifang Wang, Mario P. S. Chin, James A. Timmons, Claes Wahlestedt, Philipp Kapranov<br>Abstract: <p>Single strand breaks (SSBs) represent the major form of DNA damage, yet no technique exists to map these lesions genome-wide with nucleotide-level precision. Herein, we present a method, termed SSiNGLe, and demonstrate its utility to explore the distribution and dynamic changes of genome-wide SSBs in response to different biological and environmental stimuli. We validate SSiNGLe using two very distinct sequencing techniques and apply it to derive global profiles of SSBs in different biological states. Strikingly, we show that patterns of SSBs in the genome are non-random, specific to different biological states, enriched in regulatory elements, exons, introns, specific types of repeats and exhibit differential preference for the template strand between exons and introns. Furthermore, we show that breaks likely contribute to naturally occurring sequence variants. Finally, we demonstrate strong links between SSB patterns and age. Overall, SSiNGLe provides access to unexplored realm of cellular biology, not obtainable with current approaches.</p><p><br></p><br>Keywords: single-strand DNA break, single-strand DNA break mapping, next generation sequencing, DNA repair, molecular biology, genome stability<br><h2>Introduction</h2><p><strong>Introduction</strong></p><p>DNA damage is now widely recognized as a major reason behind cancer and many other aging-associated diseases and as such represents a very important issue for human health<a href="#_ENREF_1" rel="noopener noreferrer" target="_blank"><sup>1</sup></a><sup>, </sup><a href="#_ENREF_2" rel="noopener noreferrer" target="_blank"><sup>2</sup></a>. While multiple types of DNA lesions exist, SSBs are considered the most common type of DNA damage<a href="#_ENREF_3" rel="noopener noreferrer" target="_blank"><sup>3</sup></a>. These lesions can represent sites of oxygen radical DNA damage, intermediates in excision DNA repair pathway and products of unresolved intermediates of enzymes such as topoisomerases<a href="#_ENREF_3" rel="noopener noreferrer" target="_blank"><sup>3</sup></a>. SSBs can further deteriorate into highly toxic double-strand breaks (DSBs) by stalling or collapsing replication fork<a href="#_ENREF_4" rel="noopener noreferrer" target="_blank"><sup>4</sup></a>. However, by themselves SSBs can also represent a major issue for cell as they can inhibit progression of RNA polymerase<a href="#_ENREF_5" rel="noopener noreferrer" target="_blank"><sup>5</sup></a> and in some cases cause apoptosis<a href="#_ENREF_6" rel="noopener noreferrer" target="_blank"><sup>6</sup></a><sup>, </sup><a href="#_ENREF_7" rel="noopener noreferrer" target="_blank"><sup>7</sup></a>. The importance of this type of lesion is underscored by existence of dedicated cellular pathways that deal with every step of fixing SSBs from detection to processing to repair<a href="#_ENREF_3" rel="noopener noreferrer" target="_blank"><sup>3</sup></a>. Defects in these pathways can lead to cellular sensitivity to genotoxic stress, embryonic lethality and a number of neurodegenerative diseases<a href="#_ENREF_8" rel="noopener noreferrer" target="_blank"><sup>8</sup></a>. &nbsp;</p><p>The remarkable progress in appreciation of the fine details of the SSB repair machinery however stands in stark contrast with total absence of methods to map endogenous SSBs in a global, unbiased and genome-wide fashion with nucleotide precision. This gap in available methodologies also contrasts with a suite of comprehensive approaches developed for mapping DSBs with nucleotide-level resolution, such as BLESS<a href="#_ENREF_9" rel="noopener noreferrer" target="_blank"><sup>9</sup></a>, BLISS<a href="#_ENREF_10" rel="noopener noreferrer" target="_blank"><sup>10</sup></a> and others&nbsp;(reviewed in Ref<a href="#_ENREF_11" rel="noopener noreferrer" target="_blank"><sup>11</sup></a>). To our knowledge, only one SSB genome-wide mapping method can provide comprehensive and unbiased data<a href="#_ENREF_12" rel="noopener noreferrer" target="_blank"><sup>12</sup></a>. The procedure relies on the 3’OH group of an SSB to prime a DNA polymerase I nick-translation reaction that labels downstream DNA with a biotinylated nucleotide<a href="#_ENREF_12" rel="noopener noreferrer" target="_blank"><sup>12</sup></a>. The labeled DNA is then purified and subjected to next-generation sequencing<a href="#_ENREF_12" rel="noopener noreferrer" target="_blank"><sup>12</sup></a>. However, this approach maps a region of DNA, quite possibly extending thousands of bases from the original SSB, thus precluding identification of the lesion with nucleotide precision. On the other hand, a nucleotide-level method to map sites of excision repair has been developed<a href="#_ENREF_13" rel="noopener noreferrer" target="_blank"><sup>13</sup></a>, however, it cannot provide information on breaks generated by other mechanisms. &nbsp;</p><p>Thus, all this leads to a total dearth of knowledge of nucleotide-level genome-wide patterns for this critical type of DNA lesion. Here, we develop and validate an approach, SSiNGLe (single-strand break mapping at nucleotide genome level, <strong>Fig.1</strong>) that can provide nucleotide level maps of native SSBs genome-wide. We implement this approach to work with two next-generation sequencing (NGS) platforms: (1) a 3<sup>rd</sup> generation single molecule sequencing (SMS) platform (Helicos/SeqLL) whose unique capabilities obviate the need for lengthy sample preparation and PCR amplification, and (2) the more commonly used Illumina platform. We show highly consistent and striking patterns of SSBs using both sequencing platforms. Furthermore, the results show that the genomic pattern of breaks – the SSB “breakome” - has strong potential to represent a novel dimension describing state of a biological system and a novel source of blood-based biomarkers, with potentially yet undiscovered connections to aging.</p><p><br></p><h2>Reagents</h2><p><strong>Reagents</strong></p><p>-1× PBS, pH7.2-7.4 (Solarbio, cat. no. P1020)</p><p>-Trypsin-EDTA (0.05%), phenol red (Gibco, cat. no. 25300054)</p><p>-Solarbio Lymphocyte Separation Medium (Solarbio, cat. no. P8610)</p><p>-37% formaldehyde (Amresco, cat. no. C000T93) </p><p>-UltraPure™ Glycine (Invitrogen, cat. no. 15527013)</p><p>-Cell lysis buffer: 5 mM PIPES (pH8), 3 mM KCl, 0.5% NP-40</p><p>-UltraPure™ SDS Solution, 10% (SDS) (Invitrogen, cat. no. 15553027)</p><p>-Micrococcal Nuclease (MNase), 2× 10<sup>6</sup> gel units/ml (NEB, cat. no. M0247S)</p><p>-10 mg/ml bovine serum albumin (BSA), part of the MNase package (NEB, cat. no. M0247S)</p><p>-1× MNase buffer (50 mM Tris-HCl pH8, 5 mM CaCl<sub>2</sub>)</p><p><strong>Note</strong>: the procedure uses a lot of 1× MNase buffer, therefore we prepare it by ourselves</p><p>-RNase If, 50 U/μl (NEB, cat. no. M0243S)</p><p>-1× NEB buffer 2.1 without BSA (prepare ourselves): 50 mM NaCl, 10 mM pH8 Tris-HCl and 10 mM MgCl<sub>2</sub></p><p>-Triton X-100 (Solarbio, cat. no. T8200)</p><p>-UltraPure™ 0.5M EDTA, pH 8.0 (Invitrogen, cat. no. 15575020)</p><p>-Proteinase K solution, 20 mg/ml (Invitrogen, cat. no. AM2548)</p><p>-Terminal transferase (TdT) 20,000 units/ml including tubes with 2.5 mM CoCl<sub>2</sub> and 10× TdT buffer (NEB, cat. no. M0315) </p><p>-Agarose (Invitrogen, cat. no. 75510019)</p><p>-1× Nuclei lysis buffer: 10 mM EDTA, 1% SDS, 10 mM pH 8 Tris-HCl </p><p>-VAHTS DNA Clean Beads (Vazyme, cat. no. N411-02), equal with AMPure XP Beads (Beckman, cat. no. A63881)</p><p>-“dsDNA HS Assay” kit (Thermo Fisher Scientific, cat. no. Q33230)</p><p>-10 mM solution of either ddATP, ddCTP, ddGTP or ddTTP (Roche, cat. no. 3732738001) </p><p>-10 mM dATP (Invitrogen, cat. no. 18252015)</p><p>-10 mM dCTP (Invitrogen, cat. no. 18253013)</p><p>-10 mM dNTP mix (Invitrogen, cat. no. 18427088)</p><p>-Taq DNA polymerase, 2.5U/μl (Tiangen, cat. no. ET101-02-01)</p><p>-Shrimp Alkaline Phosphatase (SAP), 1 U/μl (NEB, cat. no. M0371S)</p><p>-UltraPure™ DNase/RNase-Free Distilled Water (nuclease-free) (Invitrogen, cat. no. 10977023)&nbsp;</p><p><strong>&nbsp;Oligonucleotide sequences</strong></p><p>-DNA-RNA oligo-d(T)50-r(T)3 (50 2’-deoxythymidine and 3 thymidine nucleotides at the 3’ end): TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTrTrTrT</p><p>	-P5_G10: CTACACGACGCTCTTCCGATCTGGGGGGGGGGHN</p><p>	-P7_T12: GTTCAGACGTGTGCTCTTCCGATCTTTTTTTTTTTTTVN</p><p>	-Illumina-P5: </p><p>	AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT</p><p>	-Illumina-P7: </p><p>CAAGCAGAAGACGGCATACGAGAT(CGTGAT)GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT</p><p><strong>Note</strong>: the sequence in brackets corresponds to the library index sequence ATCACG. Different Illumina-P7 primer oligonucleotides could be designed by varying the index sequences depending on the experimental design. Samples with different indexes can be sequenced on the same flow cell lane. &nbsp;However, we always use the same index for control and treatment samples that are to be directly compared.</p><p><br></p><h2>Equipment</h2><p><strong>Equipment</strong></p><p>-Manual Pipettes (Eppendorf)</p><p>-Coverslip (Citotest Scientific Co., Ltd, cat. no.10212432C)</p><p>-Blood counting chamber (Shanghai precision biochemical reagent instrument Co., Ltd)</p><p>-Inverted microscope (Eclipse&nbsp;TS100, Nikon)</p><p>-Tabletop centrifuge to precipitate cells (5702R centrifuge, Eppendorf)</p><p>-1.5/2 ml Eppendorf tubes (DNase/RNase-free, ultra-low attachment) (Axygen, cat. no. MCT-150-L-C/MCT-200-C).</p><p>-15/50 ml conical sterile tubes (Thermo Nunc, cat. no. 339651/339653)</p><p>-0.2 ml PCR tube (DNase/RNase-free, ultra-low attachment) (Axygen, cat. no. PCR-02-C)</p><p>-10/20/200/1000 μl pipette tips (DNase/RNase-free, Ultra-low attachment) (Axygen, cat. no. TF-300/20/200/1000)</p><p>-Agarose gel electrophoresis set up (DLAB Scientific Co., Ltd)</p><p>-Magnetic rack for 1.5 ml tubes (Promega, cat. no. Z5342)</p><p>-Vortex (DLAB Scientific Co., Ltd cat. no. D1008)</p><p>-Eppendorf Concentrator plus (Eppendorf)</p><p>-Tabletop centrifuge (5804R centrifuge, Eppendorf)</p><p>-High-speed refrigerated centrifuge (5424R centrifuge, Eppendorf)</p><p>-PCR thermocycle (C1000 PCR amplifier, Bio-Rad)</p><p>-Thermostatic metal bath (Thermomixer C, Eppendorf)</p><p>-Qubit® 3.0 Fluorometer (Thermo, cat. no. Q33216)</p><p>-Agilent 2100 bioanalyzer (Agilent, cat. no. G2939A)</p><p>-Dell R540 Server, 2*CPU Intel Xenon 4116 2.1GHz, 2*20 threads, 10*32GB RDIMM, 10*10TB 7.2K RPM SATA 6Gbps 512e hard drives for Illumina reads processing and extraction of coordinates of breaks&nbsp;</p><p><strong>Illumina reads processing software</strong></p><p>-Ubuntu (v14.04)</p><p>-BWA (v0.7.12)</p><p>-BEDtools suite (v2)</p><p>-FASTX-Toolkit (v0.013)</p><p>-Custom scripts used in the pipeline are available from GitHub at <a href="https://github.com/huifencao/DB-Nature_Communications-2019" rel="noopener noreferrer" target="_blank">https://github.com/huifencao/DB-Nature_Communications-2019</a>.</p><h2>Procedure</h2><p><strong>Procedure</strong></p><p><strong>Day 1</strong>:</p><p><strong>Note</strong>: this procedure has been used for cultured adherent and suspension cells as well as for Peripheral Blood Mononuclear Cells (PBMCs). The cultured cells can be grown using standard conditions appropriate for particular cells lines being assayed and processed as described below starting with the “Preparation of crosslinked nuclei” section. The procedure for preparation of PBMCs is given below.<strong>&nbsp;</strong></p><p><strong>Preparation of PBMCs (optional), 1 hour: </strong></p><p><strong>Note</strong>: here and below the times in hours are approximate and given for 1-10 samples processed in parallel.</p><p>1. Collect 2-4 ml of peripheral blood in EDTA anti-coagulation tubes and mix well by gently inverting the tube several times. Keep on ice and process as soon as possible. Do not freeze.</p><p>2. Dilute the samples with equal volume of 1× PBS at room temperature.</p><p>3. Take several of the Solarbio Lymphocyte Separation Medium into a 15 ml centrifuge tube (add 3 ml when the volume of the diluted blood is less than 3 ml, add equal volume of Solarbio Lymphocyte Separation Medium when the volume of the diluted blood is more than 3 ml).</p><p>4. Gently layer the diluted blood on the top of Solarbio Lymphocyte Separation Medium. The layering should be done very slowly that diluted blood and Solarbio Lymphocyte Separation Medium should stay as two different layers.</p><p><strong>Note</strong>: the total volume of diluted blood and Solarbio Lymphocyte Separation Medium shouldn’t more than 10 ml, otherwise the separation will be affected. Make sure that the samples are at room temperature before proceeding to the next step.</p><p>5. Centrifuge the samples for 30 min at 1000g at 25°C in the 5804R centrifuge with acceleration and brake settings of 6 and 0 respectively.</p><p>6. Carefully aspirate the PBMCs formed in the interphase into a new 15 ml tube. The cells in the interphase need to be aspirated without delay.</p><p>7. Add 10 ml of sterile 1× PBS into the tube with the aspirate, gently resuspend the aspirate and then centrifuge at 250g at 25°C for 10 min. </p><p>8. Remove the supernatant and repeat the last step.</p><p>9. After the centrifugation, resuspend the aspirate in 1-2 ml of sterile 1× PBS, count the PBMCs while keeping cells on ice and proceed to the crosslinking immediately as described in the section below at step 8.<strong>&nbsp;</strong></p><p><strong>Preparation of crosslinked nuclei, 2.5 hours:</strong></p><p>Keep samples on ice unless specified otherwise. Samples processed with high-speed refrigerated centrifuge below.</p><p>1. For non-adherent cells, count and collect the cells and proceed directly to step 5 of this procedure.</p><p>2. For adherent cells, remove the growth medium carefully and wash the cells twice with1× PBS at room temperature. </p><p>3. Clearly remove the 1× PBS and digest with trypsin until the cells detach.</p><p>4. Inactivate trypsin with the growth medium with serum. </p><p>5. Resuspend and count the cells.</p><p>6. Collect the cells, centrifuge to remove the growth medium with serum at 1000 rpm for 5 min at 4°C.</p><p>7. Remove the supernatant, completely resuspend the cells in the fresh growth medium with serum.</p><p>8. Start with 1-3 million non-adherent or adherent cells or 4-6 million PBMCs resuspended in 1-2 ml of either the growth medium with serum or 1× PBS (PBMCs) in a new 2 ml Eppendorf tube. </p><p>9. Add 27 µl of 37% formaldehyde per 1 ml of cell culture medium or PBS while slowly shaking the cells for 10 min at room temperature. The final concentration of formaldehyde is 1%.</p><p>10. Add 100 µl of 1.375 M glycine (final concentration 0.1375 M) per milliliter to quench formaldehyde for 5 min at room temperature. Gently pipette with tip and invert several times with the tube to mix them well.</p><p>11. Centrifuge the cells at 1500 rpm for 10 min at 4°C. </p><p>12. Remove the supernatant and wash the cells twice with 2 ml of cold 1× PBS at 1500 rpm for 10 min at 4°C. </p><p>13. Carefully remove the supernatant and resuspend the cell pellet in 2 ml of cold cell lysis buffer. </p><p><strong>Note</strong>: the pelleted cells would stick to each other after adding the cell lysis buffer, therefore pipette with up and down to separate them completely. Setting the volume to one third of the pipette range can effectively avoid the pellet loss.</p><p>14. Incubate for 10 min on ice.</p><p>15. After the lysis, centrifuge the nuclei at 1000 rpm in a benchtop centrifuge for 10 min at 4°C. </p><p>16. Aspirate the supernatant carefully. </p><p>17. Wash the pellet with 500 µl cold 1× MNase buffer, centrifuge for 10 min at 4°C at 1000 rpm.</p><p>18. Repeat steps 16-17 and remove the supernatant.</p><p>19. At this step, the nuclei can be stored at -80°C for up to one month. If this is desirable, resuspend the pellet by adding 100 µl 1× MNase buffer and move the samples directly into -80°C. When ready to proceed, thaw them on the ice around 5 min until completely thawed, centrifuge for 10 min at 4°C at 1000 rpm, remove the supernatant and proceed to the sections below. If storage is not necessary, proceed with the nuclear pellet directly to the sections below.&nbsp;</p><p><strong>SDS permeabilization (optional), 2 hours:</strong></p><p><strong>Note</strong>: this step is only necessary if nuclei are to be treated with enzymes that cannot easily diffuse into the nuclei (having molecular weight &gt;30 kDa) as shown in the example below for the treatment with shrimp alkaline phosphatase (SAP) that can make breaks with 3’-phosphate termini detectable in the SSiNGLe procedure. If no such treatments are necessary then proceed directly to the section “MNase fragmentation and DNA purification”.</p><p>1. Resuspend the pellet completely in 100 µl of 1× MNase buffer and transfer the sample to a 1.5 ml Eppendorf tube. </p><p>2. Add 3 µl of 10% SDS into the tube, gently pipette up and down while moving around the tip to mix the SDS well, the final concentration of SDS is 0.3%. Slowly shake (300 rpm) at 37°C for 1h in the thermostatic metal bath.</p><p><strong>Note</strong>: the nuclei would stick to the pipette tip at the following step easily, therefore use ultra-low attachment tip and pipette gently to avoid loss.</p><p>3. Slowly add 1.8 µl of Triton X-100 into the solution, gently pipette up and down while moving around the tip to mix Triton X-100 well, the final concentration of Triton X-100 is 1.8%. Incubate for 5 min at room temperature to sequester SDS.</p><p>4. Centrifuge 15 min at 3500g with the temperature setting of 27°C in a high-speed refrigerated centrifuge. </p><p>5. Aspirate the supernatant carefully and completely. Wash the pellet with 500 µl of 1× MNase buffer, centrifuge 15 min at 3500g with the temperature setting of 27°C in a high-speed refrigerated centrifuge. </p><p><strong>Note</strong>: prolong the centrifugation time if complete precipitation is not achieved initially.</p><p>6. Repeat step 5.</p><p>7. Aspirate the supernatant carefully and use the nuclei for the following step.&nbsp;&nbsp;&nbsp;&nbsp;</p><p><strong>Treatment with SAP (optional), 3.5 hours:</strong></p><p>1. Wash the permeabilized nuclei with 500 µl of 1× NEB buffer 2.1, gently resuspend the permeabilized nuclei and centrifuge at 3500g with the temperature setting of 27°C in a high-speed refrigerated centrifuge for 10 min.</p><p>2. Carefully remove the supernatant and repeat step 1.</p><p>3. Remove the supernatant, resuspend the nuclei in 50 µl of 1× NEB buffer 2.1.</p><p>4. Add 1 µl SAP into the solution, treat the nuclei in 50 µl of 1× NEB buffer 2.1 with 1U of SAP for 2h at 37°C in thermostatic metal bath.</p><p>5. Inactivate enzyme by incubation at 70°C for 20 min in thermostatic metal bath.</p><p>6. Collect the nuclei by centrifugation at 3500g with the temperature setting of 27°C in a high-speed refrigerated centrifuge for 10 min.</p><p>7. Aspirate the supernatant carefully and completely.</p><p>8. Wash the pellet twice with 500 µl of 1× MNase buffer at 3500g with the temperature setting of 27°C in a high-speed refrigerated centrifuge for 10 min.</p><p>9. Remove the supernatant, use the nuclei pellet for the following step.	&nbsp;</p><p><strong>MNase fragmentation and DNA purification, 4 hours:</strong></p><p>1. Completely resuspend the nuclei in 50 µl of cold 1× MNase buffer supplemented with 200 μg/ml BSA (1 µl of BSA) per 1 million nuclei (based on the original number of cells) in a 1.5 ml Eppendorf tube. </p><p><strong>Note</strong>: the nuclei would stick to the pipette tip from this step easily, therefore use Ultraultra-low attachment tip and pipette gently to avoid loss.</p><p>2. Split the sample into aliquots of 1 million nuclei per 0.2 ml RCR tube.</p><p>3. Add 1200-3000 units of MNase and 25 units of RNase If per aliquot.</p><p>4. Mix them well by pipetting up and down. Incubate for 30 minutes on ice.</p><p>5. Add 5.6 µl of 0.5 M EDTA into the solution and slowly mix by pipetting up and down to stop the digestion.</p><p>6. Add 150 µl of nuclei lysis buffer to the tube and incubate for 5 min at room temperature. </p><p>7. Add 1 µl of 20 mg/ml proteinase K into the mixture and incubate for 45 min at 55°C and 45 min at 65°C in the PCR thermocycler. </p><p>8. Transfer the samples to 1.5 ml Eppendorf tubes.</p><p>9. Add 20 ul of 3 M KCl solution and 176 µl of cold isopropanol.</p><p>10. Vortex 5s, then incubate for 30 min at -80°C.</p><p><strong>Note</strong>: the samples can be stored at -80°C for up to several months.</p><p>11. Thaw the samples at room temperature and centrifuge for 15 min at 4°C at 12,000g (10 min at max speed at 4 °C).</p><p>12. Remove the supernatant, then add 100 µl of ice-cold 70% ethanol into the tube. </p><p>13. Vortex 5s, then centrifuge the tube for 5 min at max speed at room temperature.</p><p>14. Repeat steps 12-13.</p><p>15. Aspirate the supernatant completely and carefully, vacuum-dry the DNA pellet for 2 min in the Eppendorf Concentrator plus or at room temperature for 10 min.</p><p>16. Dissolve the DNA pellet in 20 µl UltraPure™ DNase/RNase-Free Distilled Water and transfer to a new 1.5 ml Eppendorf tube.</p><p>17. Pre-warm VAHTS DNA Clean Beads for 30 min at room temperature.</p><p>18. Add double volume (40 µl) ratio of VAHTS DNA Clean Beads suspension to the DNA.</p><p>19. Mix thoroughly by pipetting up and down ~20 times.</p><p>20. Incubate for 5 min at room temperature.</p><p>21. Place the tube on the 1.5 ml magnetic rack to separate beads from the supernatant.</p><p>22. When the solution is clear (~ 5 min), aspirate and discard the supernatant.</p><p>23. Add 200 μl of freshly prepared 80% ethanol to the tube with the sample while on the magnetic stand (keep the beads together).</p><p>24. Incubate at room temperature for 30 seconds, and then carefully remove and discard the supernatant.</p><p>25. Repeat steps 23-24 once for a total of twice washing.</p><p>26. Briefly spin the tube, and put the tube back in the magnetic rack.</p><p>27. Remove the residual ethanol completely with a 10 μl tip.</p><p>28. Air dry the beads (~5 min).</p><p>29. Remove the tube from the magnetic rack and add 17 μl of nuclease-free water.</p><p>30. Mix well by pipetting up and down 30-40 times, incubate at room temperature for 5 min.</p><p>31. Put the tube in the magnetic rack until the solution is clear (~5 min).</p><p>32. Carefully remove 16 μl of the supernatant and transfer to a clean 1.5 ml Eppendorf tube. </p><p>33. Measure the DNA concentration using Qubit 3.0 fluorometer and “dsDNA HS Assay” kit.</p><p>34.Check the DNA products on 1% agarose gel or BioAnalyzer to ensure that the majority of DNA is in the range of 150-500 bp.&nbsp;</p><p><strong>Day 2</strong>:</p><p><strong>PolyA-tailing, 2 hours:</strong></p><p>1. Transfer 100 ng of the purified fragmented DNA into a 0.2 ml Axygen RNase/DNase free PCR tube.</p><p>2. Add the following:</p><p>-10×× TdT buffer: 2 µl</p><p>-2.5 mM CoCl<sub>2</sub>: 2 µl</p><p>-Nuclease free water to make the total volume of 19 µl </p><p>3. Denature the mixture at 95°C for 5 min followed by rapid snap-cooling on ice.</p><p>4. Dilute TdT (20 U/µl) to 5 U/µl with 1× TdT buffer. </p><p>5. Add the following to the tube:</p><p>-5 U/µl TdT: 1 µl </p><p>-10 mM dATP: 2 µl </p><p>6. Mix gently (do not vortex) to ensure that the enzyme is mixed completely and incubate the mixture at 37°C for 30 min and keep at 4°C before going to the next step in a PCR thermocycler.</p><p>7. Take the sample out, add 2 µl of 10 mM of either ddCTP, ddGTP, ddATP or ddTTP to the mixture to block the tailing.</p><p><strong>Note</strong>: we used any one of these dideoxy nucleotides at the blocking step. </p><p>8. Mix gently but thoroughly (do not vortex) and incubate at 37 °C 30 min and at 70°C 10 min and keep at 4°C before going to the next step in a PCR thermocycler.</p><p>9. At this stage, the tailed material can be used directly for single-molecule sequencing using the Helicos/SeqLL platform without any further purification (the SSiNGLe-SMS approach). Alternatively, it can be used to prepare libraries for Illumina-based sequencing using the steps below.</p><p>10. Pre-warm VAHTS DNA Clean Beads for 30 min at room temperature.</p><p>11. Add double volume (48 µl) ratio of VAHTS DNA Clean Beads suspension to the polyA-tailing mixture.</p><p>12. Mix thoroughly by pipetting up and down ~20 times.</p><p>13. Incubate for 5 min at room temperature.</p><p>14. Place the tube on the 1.5 ml magnetic rack to separate beads from the supernatant.</p><p>15. When the solution is clear (~ 5 min), aspirate and discard the supernatant.</p><p>16. Add 200 μl of freshly prepared 80% ethanol to the tube with the sample while on the magnetic stand (keep the beads together).</p><p>17. Incubate at room temperature for 30 seconds, and then carefully remove and discard the supernatant.</p><p>18. Repeat steps 16-17 once for a total of twice washing.</p><p>19. Briefly spin the tube, and put the tube back in the magnetic rack.</p><p>20. Remove the residual ethanol completely with a 10 μl tip.</p><p>21. Air dry the beads (~5 min).</p><p>22. Remove the tube from the magnetic rack and add 18 μl of nuclease-free water.</p><p>23. Mix well by pipetting up and down 30-40 times, incubate at room temperature for 5 min.</p><p>24. Put the tube in the magnetic rack until the solution is clear (~5 min).</p><p>25. Transfer 16.2 μl of the supernatant to a new 0.2 ml PCR tube carefully.&nbsp;</p><p><strong>SSiNGLe-ILM sequencing library construction:</strong></p><p><strong>Linear amplification, 1.5 hours:</strong></p><p>1. Add the following to the 16.2 µl of the eluted DNA from the step above:</p><p>-10× PCR buffer: 2 µl</p><p>-10 mM dNTP mix: 0.4 µl </p><p>-10 µM of a chimeric DNA-RNA oligo-d(T)50-r(T)3 oligonucleotide: 1 µl </p><p>-Taq DNA polymerase: 0.4 µl</p><p>2. Cycling parameters in a thermocycler with the lid set at 105 °C:</p><p>	1). 94°C for 30 sec</p><p>	2). 94°C for 1 min</p><p>	3). 55°C for 30 sec</p><p>	4). 72°C for 30 sec</p><p>	5). Loop back to step 2) 9 times, for a total of 10 cycles. </p><p>	6). Rapid snap-cooling on ice to prevent non-specific extension.</p><p>3. Pre-warm VAHTS DNA Clean Beads for 30 min at room temperature.</p><p>4. Add double volume (40 µl) ratio of VAHTS DNA Clean Beads suspension to the polyA-tailing mixture.</p><p>5. Mix thoroughly by pipetting up and down ~20 times.</p><p>6. Incubate for 5 min at room temperature.</p><p>7. Place the tube on the 1.5 ml magnetic rack to separate beads from the supernatant.</p><p>8. When the solution is clear (~ 5 min), aspirate and discard the supernatant.</p><p>9. Add 200 μl of freshly prepared 80% ethanol to the tube with the sample while on the magnetic stand (keep the beads together).</p><p>10. Incubate at room temperature for 30 seconds, and then carefully remove and discard the supernatant.</p><p>11. Repeat steps 9-10 once for a total of twice washing.</p><p>12. Briefly spin the tube, and put the tube back in the magnetic rack.</p><p>13. Remove the residual ethanol completely with a 10 μl tip.</p><p>14. Air dry the beads (~5 min).</p><p>15. Remove the tube from the magnetic rack and add 16 μl of nuclease-free water.</p><p>16. Mix well by pipetting up and down 30-40 times, incubate at room temperature for 5 min.</p><p>17. Put the tube in the magnetic rack until the solution is clear (~5 min).</p><p>18. Transfer 15 μl of the supernatant to a new 0.2 ml PCR tube carefully.<strong>&nbsp;</strong></p><p><strong>PolyC-tailing, 2 hours:</strong></p><p>1. Add the following to the 15 µl of the polyA-tailed DNA from above:</p><p>-10× TdT buffer: 2 µl</p><p>-2.5 mM CoCl<sub>2</sub>: 2 µl</p><p>2. Denature the mixture at 95°C for 5 min, and crash on ice for 2 min.</p><p>3. Dilute TdT (20 U/µl) to 5 U/ul with 1× buffer.</p><p>4. Add the following:</p><p>-5 U/µl TdT: 1 µl </p><p>-10 mM dCTP: 2 µl</p><p>5. Mix gently (do not vortex) to ensure that the enzyme is mixed completely and incubated at 37°C for 30 min, 4°C forever in a PCR thermocycler.</p><p>6. Add 2 µl of 10 mM of either ddCTP, ddGTP, ddATP or ddTTP to the mixture to block the tails.</p><p><strong>Note</strong>: we used any one of these dideoxy nucleotides at the blocking step</p><p>7. Mix gently but thoroughly (do not vortex) and incubate at 37 °C 30 min and at 70°C 10 min and keep at 4°C before going to the next step.</p><p>8. Pre-warm VAHTS DNA Clean Beads for 30 min at room temperature.</p><p>9. Add double volume (48 µl) ratio of VAHTS DNA Clean Beads suspension to the polyA-tailing mixture.</p><p>10. Mix thoroughly by pipetting up and down ~20 times.</p><p>11. Incubate for 5 min at room temperature.</p><p>12. Place the tube on the 1.5 ml magnetic rack to separate beads from the supernatant.</p><p>13. When the solution is clear (~ 5 min), aspirate and discard the supernatant.</p><p>14. Add 200 μl of freshly prepared 80% ethanol to the tube with the sample while on the magnetic stand (keep the beads together).</p><p>15. Incubate at room temperature for 30 seconds, and then carefully remove and discard the supernatant.</p><p>16. Repeat steps 14-15 once for a total of twice washing.</p><p>17. Briefly spin the tube, and put the tube back in the magnetic rack.</p><p>18. Remove the residual ethanol completely with a 10 μl tip.</p><p>19. Air dry the beads (~5 min).</p><p>20. Remove the tube from the magnetic rack and add 16.5 μl of nuclease-free water.</p><p>21. Mix well by pipetting up and down 30-40 times, incubate at room temperature for 5 min.</p><p>22. Put the tube in the magnetic rack until the solution is clear (~5 min).</p><p>23. Transfer 15.2 μl of the supernatant to a new 0.2 ml PCR tube carefully.&nbsp;</p><p><strong>Illumina library construction, 4 hours:</strong></p><p><strong>Note</strong>: this procedure differs from the one published in the original paper. Specifically, the version presented here employs 2 rounds of PCR to construct the library which significantly increase the yield of the amplified products.</p><p>1. Add the following to the 15.2 µl of the polyC-tailed DNA from above:</p><p>-10× PCR buffer: 2 μl</p><p>-10 mM dNTP mix: 0.4 µl </p><p>-10 μM P5-G10 oligonucleotide: 1 μl</p><p>-10 μM P7-T12 oligonucleotide: 1 µl</p><p>-Taq DNA polymerase: 0.4 µl</p><p>2. Cycling parameters with the lid set at 105 °C:</p><p>	1). 94°C, 3 min</p><p>	2). 94°C, 30 sec</p><p>	3). 55°C, 1 min</p><p>	4). 72°C,1min</p><p>	5). 94°C, 30 sec</p><p>	6). 37°C, 1 min </p><p>	7). Slow ramp at 2°C per minute to 72°C followed by 2 min incubation</p><p>	8). 94°C, 30 sec </p><p>	9). 60°C, 30 sec </p><p>	10). 72°C, 30 sec</p><p>	11). Loop back to step 8) 29 times for a total of 30 cycles</p><p>	12). 72°C, 7min</p><p>	13). 4 °C, hold</p><p>3. Pre-warm VAHTS DNA Clean Beads for 30 min at room temperature.</p><p>4. Add double volume (40 µl) ratio of VAHTS DNA Clean Beads suspension to the polyA-tailing mixture.</p><p>5. Mix thoroughly by pipetting up and down ~20 times.</p><p>6. Incubate for 5 min at room temperature.</p><p>7. Place the tube on the 1.5 ml magnetic rack to separate beads from the supernatant.</p><p>8. When the solution is clear (~ 5 min), aspirate and discard the supernatant.</p><p>9. Add 200 μl of freshly prepared 80% ethanol to the tube with the sample while on the magnetic stand (keep the beads together).</p><p>10. Incubate at room temperature for 30 seconds, and then carefully remove and discard the supernatant.</p><p>11. Repeat steps 9-10 once for a total of twice washing.</p><p>12. Briefly spin the tube, and put the tube back in the magnetic rack.</p><p>13. Remove the residual ethanol completely with a 10 μl tip.</p><p>14. Air dry the beads (~5 min).</p><p>15. Remove the tube from the magnetic rack and add 18 μl of nuclease-free water.</p><p>16. Mix well by pipetting up and down 30-40 times, incubate at room temperature for 5 min.</p><p>17. Put the tube in the magnetic rack until the solution is clear (~5 min).</p><p>18. Transfer 17 μl of the supernatant to a 1.5 ml Eppendorf tube carefully.</p><p>19. Prepare the following PCR reaction:</p><p>-The eluted DNA from the step above: 4 μl</p><p>-10× PCR buffer: 2 μl </p><p>-2.5 mM dNTP mix: 1.6 µl </p><p>-10 μM Illumina-P5 primer oligonucleotide: 1 μl</p><p>-10 μM Illumina-P7 primer oligonucleotide: 1 µl</p><p>-Taq DNA polymerase: 0.4 µl</p><p>-Nuclease free water: 10 µl</p><p>20. Cycling parameters with the lid set at 105 °C:</p><p>	1). 94°C, 3 min</p><p>	2). 94°C, 30 sec</p><p>	3). 55°C, 30 sec </p><p>	4). 72°C, 30 sec</p><p>	5). Loop back to step 2) 5 times for a total of 6 cycles</p><p>	6). 72°C, 7min</p><p>7). 4 °C, hold</p><p>21. Pre-warm VAHTS DNA Clean Beads for 30 min at room temperature.</p><p>22. Add double volume (40 µl) ratio of VAHTS DNA Clean Beads suspension to the polyA-tailing mixture.</p><p>23. Mix thoroughly by pipetting up and down ~20 times.</p><p>24. Incubate for 5 min at room temperature.</p><p>25. Place the tube on the 1.5 ml magnetic rack to separate beads from the supernatant.</p><p>26. When the solution is clear (~ 5 min), aspirate and discard the supernatant.</p><p>27. Add 200 μl of freshly prepared 80% ethanol to the tube with the sample while on the magnetic stand (keep the beads together).</p><p>28. Incubate at room temperature for 30 seconds, and then carefully remove and discard the supernatant.</p><p>29. Repeat steps 27-28 once for a total of twice washing.</p><p>30. Briefly spin the tube, and put the tube back in the magnetic rack.</p><p>31. Remove the residual ethanol completely with a 10 μl tip.</p><p>32. Air dry the beads (~5 min).</p><p>33. Remove the tube from the magnetic rack and add 22 μl of nuclease-free water.</p><p>34. Mix well by pipetting up and down 30-40 times, incubate at room temperature for 5 min.</p><p>35. Put the tube in the magnetic rack until the solution is clear (~5 min).</p><p>36. Transfer 21 μl of the supernatant to a 1.5 ml Eppendorf tube carefully.</p><p>37. Measure the DNA concentration using Qubit 3.0 fluorometer and “dsDNA HS Assay” kit.</p><p>38. At this stage, the samples can be stored for at least 1 year in -80°C, however, we usually send them for sequencing as soon as possible.<strong>&nbsp;</strong></p><p><strong>Day 3</strong>:</p><p>Shipping the samples for NGS. We outsource NGS to Novogene Corporation (Beijing). The samples from above are typically shipped on dry ice in the concentration and volume required by the vendor. We have so far successfully sequenced these libraries on the HiSeq X Ten platform using the paired-end 150 bp strategy.&nbsp;</p><p><strong>Processing of the Illumina reads and extracting the coordinates of breaks:</strong></p><p>1. Filter the reads such that each read has Phred quality score &gt;20 and length &gt;30:&nbsp;</p><p>fastq_quality_trimmer&nbsp;-v -t 20 -l 30 -i read_1.clean.fq -o read_1.clean.fq.1 -Q33;</p><p>fastq_quality_trimmer&nbsp;-v -t 20 -l 30 -i read_2.clean.fq -o read_2.clean.fq.1 -Q33;&nbsp;</p><p>2. Select read pairs where reads 1 start with 10G’s, reads 2 start with 12T’s:&nbsp;</p><p>perl 1_filter_paired.flags.pl read_1.clean.fq.1 read_2.clean.fq.1&nbsp;</p><p>3. Map the reads to the desired reference genome, which should be downloaded in advance, for example the human GRCh37/hg19 assembly (Human19.fasta) downloaded from the UCSC genome browser (<a href="http://hgdownload.soe.ucsc.edu/goldenPath/hg19/database/" rel="noopener noreferrer" target="_blank">http://hgdownload.soe.ucsc.edu/goldenPath/hg19/database/</a>).</p><p><strong>Note</strong>: to determine a profile of mitochondrial breaks, we align reads to the chrM genome separately and use the same downstream analysis steps below.&nbsp;</p><p>Build the genome index: &nbsp;</p><p>bwa index Human19.fasta –p HG19&nbsp;</p><p>Map the reads: &nbsp;</p><p>bwa mem –t 8 -M -R '@RG\tID:foo\tSM:bar\tLB:library1' HG19 read_1.clean.fq.1.trim read_2.clean.fq.1.trim &gt; read.sam&nbsp;</p><p>4. Filter the alignments by extracting only unique and proper mapped paired reads, and then extract the alignment records of reads 2 containing the coordinates of breaks:&nbsp;</p><p>perl 3_filter.sam.pl read.sam &gt;read.proper.sam 2&gt;log.3&nbsp;</p><p>5. Convert the SAM file to a BED file and then extract the very first position of each aligned record of reads 2:&nbsp;</p><p>sh 4_sam2bed.right.proper.sh read.proper.sam read.candidates.bed 2&gt;log.4&nbsp;</p><p>6. Filter the alignments based on the fraction of T’s in the 20-base genomic sequence 5′ upstream of the reads 2 alignments:&nbsp;</p><p>perl 5_filtering.internal.polyA.pl read.candidates.bed Human19.fasta 20 0.4 &gt;read.bed 3&gt;log.5&nbsp;</p><p>7. Extract the coordinates of breaks:&nbsp;</p><p>cat read.bed | awk ‘$6 == “+”’ | awk '{print $1"\t"$2"\t"$2+1"\t"$4"\t"$5"\t-"}' &gt;read.break.bed.minus</p><p>cat read.bed | awk ‘$6 == “-”’ | awk '{print $1"\t"$2"\t"$2+1"\t"$4"\t"$5"\t+"}' &gt;read.break.bed.plus</p><p>cat read.break.bed.plus read.break.bed.minus | sort –k1,1 –k2n,2 &gt;read.break.bed</p><p>rm –f read.break.bed.minus read.break.bed.plus</p><p><br></p><h2>Troubleshooting</h2><p><strong>Troubleshooting</strong></p><p>We noticed variation in MNase activity from lot to lot and also occurring with time of storage. Therefore, pre-testing to ensure fragmentation in the desired range can be helpful when using new lot or if they enzyme has not been used for some time.</p><p><br></p><h2>Time Taken</h2><p><strong>Timing</strong></p><p>From the time that samples are ready to be processed, the whole wet lab procedure takes around 2 days for 1-10 samples and 3 days for 11-24 samples. Analysis from raw reads to the coordinates of breaks takes around 1 and 2 days respectively for such numbers of samples with 1 Gb sequencing scale per sample and using the server provided in this protocol. However, the processing times would vary depending on the amount of sequencing being processed and the computer hardware used.</p><p><br></p><h2>Anticipated Results</h2><p><strong>Anticipated Results</strong></p><p>QC metrics of a library ready for sequencing are shown in the Fig. 2.</p><p><br></p><h2>References</h2><p><strong>References</strong>&nbsp;</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hoeijmakers, J.H. DNA damage, aging, and cancer. <em>N Engl J Med</em> <strong>361</strong>, 1475-1485 (2009).</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Jackson, S.P. &amp; Bartek, J. The DNA-damage response in human biology and disease. <em>Nature</em> <strong>461</strong>, 1071-1078 (2009).</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Caldecott, K.W. Single-strand break repair and genetic disease. <em>Nat Rev Genet</em> <strong>9</strong>, 619-631 (2008).</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Kuzminov, A. Single-strand interruptions in replicating chromosomes cause double-strand breaks. <em>Proc Natl Acad Sci U S A</em> <strong>98</strong>, 8241-8246 (2001).</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Kathe, S.D., Shen, G.P. &amp; Wallace, S.S. Single-stranded breaks in DNA but not oxidative DNA base damages block transcriptional elongation by RNA polymerase II in HeLa cell nuclear extracts. <em>The Journal of biological chemistry</em> <strong>279</strong>, 18511-18520 (2004).</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Ljungman, M. &amp; Zhang, F. Blockage of RNA polymerase as a possible trigger for u.v. light-induced apoptosis. <em>Oncogene</em> <strong>13</strong>, 823-831 (1996).</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Ljungman, M., Zhang, F., Chen, F., Rainbow, A.J. &amp; McKay, B.C. Inhibition of RNA polymerase II as a trigger for the p53 response. <em>Oncogene</em> <strong>18</strong>, 583-592 (1999).</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Rulten, S.L. &amp; Caldecott, K.W. DNA strand break repair and neurodegeneration. <em>DNA repair</em> <strong>12</strong>, 558-567 (2013).</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Crosetto, N. et al. Nucleotide-resolution DNA double-strand break mapping by next-generation sequencing. <em>Nat Methods</em> <strong>10</strong>, 361-365 (2013).</p><p>10.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Yan, W.X. et al. BLISS is a versatile and quantitative method for genome-wide profiling of DNA double-strand breaks. <em>Nature communications</em> <strong>8</strong>, 15058 (2017).</p><p>11.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Vitelli, V. et al. Recent Advancements in DNA Damage-Transcription Crosstalk and High-Resolution Mapping of DNA Breaks. <em>Annual review of genomics and human genetics</em> <strong>18</strong>, 87-113 (2017).</p><p>12.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Baranello, L. et al. DNA break mapping reveals topoisomerase II activity genome-wide. <em>International journal of molecular sciences</em> <strong>15</strong>, 13111-13122 (2014).</p><p>13.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hu, J., Adar, S., Selby, C.P., Lieb, J.D. &amp; Sancar, A. Genome-wide analysis of human global and transcription-coupled excision repair of UV damage at single-nucleotide resolution. <em>Genes Dev</em> <strong>29</strong>, 948-960 (2015).</p><h2>Acknowledgements</h2><p><br></p></body></html>