<html><head><title>Single oocyte/embryo RNASeq data processing</title></head><body><h1>Single oocyte/embryo RNASeq data processing</h1>Authors: ADITYA SANKAR, Jens Vilstrup Johansen<br>Abstract: <p>This procotol details the step-by-step procedures followed to process the single oocyte/embryo mRNASeq and 2-cell embryo Total RNASeq data generated using the SMARTSeq2 technology in the associated publication. </p><br>Keywords: developmental epigenetics, single cell RNASeq, SMARTSeq2<br><h2>Introduction</h2><p>This protocol is a bioinformatic data processing protocol that is linked to the associated publication.  </p><h2>Equipment</h2><p>Windows/Mac OS workstation with 12 GB RAM and high end processing power. Alternatively, there can be efficient and faster processing of data if connecting to a computing core using VPN</p><h2>Procedure</h2><p>1. For both the single-cell mRNA (151bp PE) and the totalRNA (76bp PE) sequencing the raw data was converted from bcl to fastq format and reads trimmed in BaseSpace.&nbsp;</p><p>2. After download from BaseSpace the raw reads were quality assessed with FastQC<sup>1</sup> and Fastq Screen<sup>2.</sup></p><p>3. Afterwards trimmed using Trimmomatic v0.32<sup>3</sup> (mRNA-seq settings: PE ILLUMINACLIP:NexteraPE-PE.fa:2:30:10:1:true HEADCROP:15 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:25; totalRNA-seq settings: PE ILLUMINACLIP:TruSeq2-PE.fa:2:30:10:1:true HEADCROP:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:25).</p><p>4. For mRNASeq libraries, the trimmed reads were aligned to the mm10 genome assembly using STAR<sup>4</sup>(v2.5.1a) in two-pass mode and guided by a RefSeq (UCSC,2018/08/05) gene annotation (settings: --sjdbOverhang 135&nbsp;--twopassMode Basic --outSAMtype BAM SortedByCoordinate --outSAMattributes All --outSAMunmapped Within --outFilterMismatchNoverLmax 0.1 --outFilterMatchNmin 16 --outFilterMismatchNmax 5). </p><p>5. After mapping, the reads were assigned to genes with featureCounts<sup>5</sup> (v1.5.1, settings: --primary -p -B -O -M --fraction -s 0 -J) generating a count table. </p><p>6. Using R (v3.5.1) (https://www.r-project.org), the quality of samples were again estimated using various quantitative and qualitative methods available in the Scater package<sup>6</sup>. </p><p>7. One 8-cell MZ mutant sample was excluded from the dataset due to extremely low total gene count (404). the remaining samples had total gene counts in the range 0,5-4,6 mio (mean:2,4 mio). </p><p>8. The DESeq2 (v1.22.1) package<sup>7</sup> was used for statistical analysis of the count data comparing the knockout and wild-type samples within each cell stage. The clusterProfiler package<sup>8</sup> was used to test for under/overrepresentation of genes in various gene sets.&nbsp;</p><p>9. The DBTMEE<sup>9</sup> gene-to-cluster annotation (cluster_gene_v2.tsv) was downloaded from&nbsp;<a href="http://dbtmee.hgc.jp/download/download.phpm" rel="noopener noreferrer" target="_blank">http://dbtmee.hgc.jp/download/download.phpm</a>, and non/low expressed genes removed by imposing the filter criteria FPKM&gt;3 for cell stages Oocyte, 1C, 2C and 4C. </p><p>10. Differentially expressed genes from the DeSeq2 analysis were defined as having an absolute log2 fold change &gt;=1 and FDR&lt;=5%. </p><p>11. Using the compareCluster function from the clusterProfiler R-package we looked for over- or underrepresented DBTMEE gene sets in our list of DE genes. From the compareCluster results we derived the observed/expected ratio based on the values of 'GeneRatio' (Obs) and 'BgRatio' (Exp). </p><p>12. The compareCluster results were filtered to only included gene sets with FDR&lt;10% and (DE Gene) Count&gt;3. Furthermore, to simplify the plot we limited the color scale to +1/-1.</p><p>13. For total-RNASeq libraries, we tested for differential expression of repeat elements we first used RepEnrich2 (v0.1)<sup>10</sup> to map our totalRNA reads against the RepeatMasker database (mm10,4.0.5,2014013) followed by statistical analysis in R using the edgeR package (v3.24.0)<sup>11</sup> as per the authors suggested analysis pipeline.</p><h2>References</h2><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;S, Andrews. FastQC: a quality control tool for high throughput sequence data from <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc." rel="noopener noreferrer" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/fastqc.</a>(2010).</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Andrews, S. FastQ Screen (2011).</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Bolger, A.M., Lohse, M. &amp; Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. <em>Bioinformatics</em> <strong>30</strong>, 2114-2120 (2014).</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Dobin, A.<em> et al.</em> STAR: ultrafast universal RNA-seq aligner. <em>Bioinformatics</em> <strong>29</strong>, 15-21 (2013).</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Liao, Y., Smyth, G.K. &amp; Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. <em>Bioinformatics</em> <strong>30</strong>, 923-930 (2014).</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;McCarthy, D.J., Campbell, K.R., Lun, A.T. &amp; Wills, Q.F. Scater: pre-processing, quality control, normalization and visualization of single-cell RNA-seq data in R. <em>Bioinformatics</em> <strong>33</strong>, 1179-1186 (2017).</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Love, M.I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <em>Genome Biol</em> <strong>15</strong>, 550 (2014).</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Yu, G., Wang, L.G., Han, Y. &amp; He, Q.Y. clusterProfiler: an R package for comparing biological themes among gene clusters. <em>OMICS</em> <strong>16</strong>, 284-287 (2012).</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Park, S.J., Shirahige, K., Ohsugi, M. &amp; Nakai, K. DBTMEE: a database of transcriptome in mouse early embryos. <em>Nucleic Acids Res</em> <strong>43</strong>, D771-776 (2015).</p><p>10.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Criscione, S.W., Zhang, Y., Thompson, W., Sedivy, J.M. &amp; Neretti, N. Transcriptional landscape of repetitive elements in normal and cancer human cells. <em>BMC Genomics</em> <strong>15</strong>, 583 (2014).</p><p>11.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Robinson, M.D., McCarthy, D.J. &amp; Smyth, G.K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. <em>Bioinformatics</em> <strong>26</strong>, 139-140 (2010).</p><h2>Acknowledgements</h2><p>Mads Lerdrup, University of Copenhagen </p></body></html>