<html><head><title>Using TDFragMapper: a high-performance tool to assess experimental parameters in top-down proteomics</title></head><body><h1>Using TDFragMapper: a high-performance tool to assess experimental parameters in top-down proteomics</h1>Authors: Diogo Borges Lima, Jonathan Dhenin, Mathieu Dupré, Julia Chamot-Rooke<br>Abstract: <p>Here we present a new software-tool for visualizing fragment ions and sequence coverage of intact proteins in top-down mass spectrometry. TDFragMapper combines the data arising from multiple and diverse tandem mass spectrometry experiments of intact proteins. Our tool maps fragment ions onto the protein backbone sequence and allows for a rapid comparison of the results obtained when varying experimental parameters in tandem mass spectrometry (MS/MS) experiments of intact proteins: fragmentation method, selected precursor charge state, activation level, technical replicate. In summary, <strong>TDFragMapper</strong> is a unique software-tool that allows an easy selection of the best experimental conditions leading to the highest confidence in intact protein sequence identification.</p><br>Keywords: Bioinformatics, Proteomics, Top-Down, Mass Spectrometry, Intact Proteins<br><h2>Introduction</h2><p>Top-down proteomics (TDP) is a powerful technology allowing the characterization of proteins at the proteoform level using high-resolution tandem mass spectrometry (MS/MS). The complete characterization of proteoforms often requires the use of several complementary fragmentation techniques, such as collision-induced dissociation (CID), electron transfer dissociation (ETD) or ultraviolet photodissociation (UVPD)[<a href="https://pubs.acs.org/doi/10.1021/acs.analchem.8b00984" rel="noopener noreferrer" target="_blank">1</a>]. In contrast to bottom-up proteomics, the experimental parameters used for the fragmentation in TDP, such as the activation energy or the charge state of the precursor ion chosen for fragmenting an intact protein, can significantly affect the quality of MS/MS data and therefore the protein sequence coverage [<a href="https://pubs.acs.org/doi/10.1021/acs.analchem.5b00162" rel="noopener noreferrer" target="_blank">2</a>]. Although existing tools are capable of matching a list of fragment ions to a protein sequence (<em>e.g.</em>, ProSight Lite [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4445472/" rel="noopener noreferrer" target="_blank">3</a>]), there is currently no computational tool allowing to visualize fragments arising from diverse MS/MS experiments on a unique fragmentation map without losing information on the contribution and the specificity of each experiment. Current tools provide a unique fragmentation map per MS/MS experiment, and thus their comparison, in particular when multiple parameters are assessed, is both difficult and time-consuming. Moreover, although the intensity of fragment ions can represent a precious source of information when dealing with MS/MS data, it is often not considered when matching fragments onto a protein sequence. To circumvent these limitations, here we introduce <strong>TDFragMapper</strong>, a novel software-tool that can display and combine pre-assigned fragment ions achieved from various MS/MS experiments on a unique protein sequence, keeping an easy access to the individual contribution of each experiment and to the intensity of deconvoluted fragment ions. Our tool makes it possible to rapidly compare experimental parameters such as the type of fragmentation, the activation level or the precursor charge state in the MS/MS analysis of intact proteins.</p><p>In this protocol, we describe the key steps for using TDFragMapper, which enables to compare fragment ions arising from different tandem mass spectrometry experiments by taking into consideration the contribution of each experiment, the intensity of deconvoluted fragment ions and the presence of golden complementary pairs.</p><h2>Equipment</h2><p><strong>Hardware</strong>&nbsp;</p><p>•A computer with at least 8 GB RAM and 2 computing cores is recommended.&nbsp;</p><p>&nbsp;</p><p><strong>Software</strong>&nbsp;</p><p>•Windows 10 (32 or 64 bits) or later.&nbsp;</p><p>•The .NET framework 4.8, which will be automatically updated by&nbsp;TDFragMapper&nbsp;if necessary.&nbsp;</p><p>•TDFragMapper&nbsp;software is available at&nbsp;<a href="https://msbio.pasteur.fr/tdfragmapper" rel="noopener noreferrer" target="_blank">https://msbio.pasteur.fr/tdfragmapper</a>.</p><p>&nbsp;</p><p><strong>Data files</strong>&nbsp;</p><p>•TDFragMapper&nbsp;v1.0 is compatible with output data files from ProSight Lite[<a href="https://onlinelibrary.wiley.com/doi/full/10.1002/pmic.201400313" rel="noopener noreferrer" target="_blank">3</a>] and Thermo®&nbsp;FreeStyle™ software[<a href="https://assets.thermofisher.com/TFS-Assets/CMD/manuals/Man-XCALI-97576-Protein-Deconvolution-30-User-ManXCALI97576-A-EN.pdf" rel="noopener noreferrer" target="_blank">4</a>].&nbsp;</p><p>•TDFragMapper&nbsp;saves results in its own format (<em>i.e.</em>,&nbsp;<em>*.tdfm</em>) and exports the analysis to an image in TIFF, PNG or JPG format.&nbsp;</p><h2>Procedure</h2><p><strong>1.&nbsp;Software installation:</strong>&nbsp;</p><p>Download&nbsp;TDFragMapper&nbsp;by clicking on the&nbsp;<em>Download</em>&nbsp;button at <a href="https://msbio.pasteur.fr/tdfragmapper" rel="noopener noreferrer" target="_blank">https://msbio.pasteur.fr/tdfragmapper</a>.</p><p>&nbsp;</p><p><strong>2.&nbsp;Workflow</strong>&nbsp;</p><p>The following workflow demonstrates how to visualize and compare fragments obtained from various tandem mass spectrometry (MS/MS) experiments on intact proteins using TDFragMapper.&nbsp;</p><p>&nbsp;</p><p><strong>2.1.&nbsp;</strong>Execute&nbsp;the&nbsp;<em>TDFragMapper</em>&nbsp;tool (<strong>Figure 1</strong>)&nbsp;</p><p>&nbsp;</p><p><strong>Figure 1: Graphical User Interface of the main window of TDFragMapper.</strong>&nbsp;</p><p>&nbsp;</p><p><strong>2.2.&nbsp;Setting&nbsp;parameters</strong>&nbsp;</p><p><strong>2.2.1.</strong>&nbsp;<em>Protein Sequence</em>: specify a file containing a single protein sequence. The file format can be txt or FASTA, obtained from&nbsp;<a href="https://www.uniprot.org/" rel="noopener noreferrer" target="_blank">Uniprot</a>.&nbsp;</p><p>&nbsp;</p><p><strong>2.2.2.&nbsp;</strong><em>Sequence&nbsp;Information</em>: specify&nbsp;any&nbsp;additional information that will appear&nbsp;with the legend of the fragmentation maps.</p><p><sub><em>This field is not mandatory.</em>&nbsp;</sub></p><p>&nbsp;</p><p><strong>2.2.3.&nbsp;</strong>For&nbsp;each data file, specify a&nbsp;<em>Fragmentation Method</em>, an&nbsp;<em>Activation Level</em>, a&nbsp;<em>Precursor Charge State</em>, a&nbsp;<em>Replicate</em>&nbsp;and the path&nbsp;to&nbsp;the&nbsp;<em>MS/MS Data&nbsp;</em>file, which was generated by ProSight&nbsp;Lite[<a href="https://onlinelibrary.wiley.com/doi/full/10.1002/pmic.201400313" rel="noopener noreferrer" target="_blank">3</a>] (<strong>Figure 2</strong>). Specify a&nbsp;<em>Deconvoluted Spectra</em>&nbsp;file that was generated by Thermo®&nbsp;FreeStyle™&nbsp;software&nbsp;(Xtract&nbsp;algorithm)[<a href="https://assets.thermofisher.com/TFS-Assets/CMD/manuals/Man-XCALI-97576-Protein-Deconvolution-30-User-ManXCALI97576-A-EN.pdf" rel="noopener noreferrer" target="_blank">4</a>]&nbsp;if&nbsp;an evaluation&nbsp;of&nbsp;fragment ions intensity&nbsp;is desired.&nbsp;</p><p><strong>2.2.3.1.&nbsp;</strong>To&nbsp;add new data file&nbsp;parameters,&nbsp;click on&nbsp;the&nbsp;<em>Add another data file</em>&nbsp;button. A&nbsp;new line will be included in the table.&nbsp;</p><p><strong>2.2.3.1.1.</strong>&nbsp;Instead of&nbsp;filling&nbsp;file by file, the user&nbsp;has the possibility to&nbsp;fill multiple data files at once by&nbsp;copying&nbsp;from a table (<em>e.g.</em>, Excel®&nbsp;file), and pasting by right-clicking on the table or pressing CTRL + V. The table (<em>e.g.</em>, Excel®&nbsp;file) must respect&nbsp;the following format:&nbsp;</p><p><br></p><p><strong>Table 1: Data to be copied and pasted on&nbsp;TDFragMapper.</strong>&nbsp;</p><p>&nbsp;</p><p>The following nomenclature (commonly used in tandem mass spectrometry experiments)&nbsp;has to&nbsp;be followed in order to&nbsp;standardize the writing of&nbsp;<em>Precursor charge state</em>,&nbsp;<em>Activation level</em>,&nbsp;<em>Replicate</em>, and&nbsp;<em>Fragmentation method</em>:&nbsp;</p><p><em>Fragmentation method</em>: CID; SID; HCD; UVPD;&nbsp;EThcD; ETD;&nbsp;ECD;&nbsp;</p><p><em>Activation level</em>:&nbsp;&nbsp;</p><p>CID, SID, HCD: … % (<em>e.g.</em>: 25%)&nbsp;</p><p>UVPD, ECD &amp;&nbsp;ETD:&nbsp;… ms (<em>e.g.</em>:&nbsp;70 ms)&nbsp;</p><p>EThcD: … ms /…&nbsp;%&nbsp;(<em>e.g.</em>:&nbsp;10&nbsp;ms&nbsp;/ 10&nbsp;%)&nbsp;</p><p><em>Precursor charge state</em>: only numbers (<em>e.g.</em>: 25)&nbsp;</p><p><em>Replicate</em>: R… (<em>e.g.</em>:&nbsp;R1, R2, R3, etc.)&nbsp;</p><p><strong>2.2.3.2.</strong><em>ProSight&nbsp;Lite</em>[<a href="https://onlinelibrary.wiley.com/doi/full/10.1002/pmic.201400313" rel="noopener noreferrer" target="_blank">3</a>]: For generating an output file, after opening a&nbsp;<em>*.pcml</em>&nbsp;file (or setting up all parameters&nbsp;in the software), click on&nbsp;the&nbsp;Excel®&nbsp;button at the bottom of the main window (indicated by the arrow on&nbsp;<strong>Figure 2</strong>).&nbsp;</p><p><br></p><p>&nbsp;</p><p><strong>Figure 2: ProSight&nbsp;Lite</strong>[<a href="https://onlinelibrary.wiley.com/doi/full/10.1002/pmic.201400313" rel="noopener noreferrer" target="_blank">3</a>]<strong>&nbsp;main&nbsp;window: the arrow indicates the button that exports the MS/MS data required by&nbsp;TDFragMapper.</strong>&nbsp;</p><p>&nbsp;</p><p><strong>2.2.3.3.&nbsp;</strong><em>Thermo®&nbsp;FreeStyleTM&nbsp;software (Xtract algorithm)</em>&nbsp;[<a href="https://assets.thermofisher.com/TFS-Assets/CMD/manuals/Man-XCALI-97576-Protein-Deconvolution-30-User-ManXCALI97576-A-EN.pdf" rel="noopener noreferrer" target="_blank">4</a>]: For deconvolving&nbsp;a&nbsp;MS/MS spectrum, follow the procedure as given in the Chapter 10&nbsp;<em>Deconvolving and Deisotoping Spectra with the Xtract Algorithm&nbsp;of the&nbsp;FreeStyle&nbsp;1.4</em>&nbsp;user guide. Once&nbsp;the&nbsp;MS/MS spectrum&nbsp;has&nbsp;been&nbsp;deconvoluted, generate an output&nbsp;file as described below:&nbsp;&nbsp;</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Select the <em>Xtract Results</em> tab</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Click on the <em>Workspace Options</em> toolbar</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Click on <em>Selection As</em> in the <em>Exports</em> tab (<strong>Figure 3</strong>)</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Choose <em>To CSV File</em> as Export Type, click on OK</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Save the *.xls output file&nbsp;</p><p><br></p><p><strong>Figure 3: Thermo®&nbsp;FreeStyleTM&nbsp;software:&nbsp;the arrow indicates the button that exports the&nbsp;deconvoluted&nbsp;MS/MS data required by&nbsp;TDFragMapper.</strong>&nbsp;</p><p>&nbsp;</p><p><strong>2.2.4.</strong>&nbsp;To load all files set up in&nbsp;<em>2.2.3</em>, click on&nbsp;the&nbsp;<em>Next step</em>&nbsp;button.&nbsp;The&nbsp;<em>Filter</em>&nbsp;tab&nbsp;will be&nbsp;enabled.&nbsp;</p><p><strong>2.2.5.</strong>&nbsp;<strong><em>Creating Maps</em></strong>&nbsp;</p><p><strong>2.2.5.1.</strong>&nbsp;To create a study map, three fixed conditions and one study condition&nbsp;need to be&nbsp;set up.&nbsp;The conditions*&nbsp;can be selected by using the associated drop-down menu&nbsp;(<strong>Figure 4</strong>).&nbsp;</p><p><sub><em>*Available options: Fragmentation Method, Activation Level, Replicates and Precursor Charge State.</em>&nbsp;</sub></p><p><br></p><p><strong>2.2.5.1.1.</strong>&nbsp;For each&nbsp;selected condition, all associated values found in the input tab are displayed in the left list. Desired values*&nbsp;can be selected or removed using the arrow buttons.&nbsp;&nbsp;TDFragMapper&nbsp;will apply default colors to the selected values of the study condition. To change the colors, double click on the selected value, choose a new color and validate with the&nbsp;<em>OK</em>&nbsp;button.&nbsp;&nbsp;</p><p><sub><em>*For Fragmentation Method&nbsp;as first fixed condition, only one&nbsp;value&nbsp;is allowed to&nbsp;be selected</em>&nbsp;</sub></p><p>&nbsp;</p><p><br></p><p><strong>Figure 4:&nbsp;Example of a map for the purpose of studying the influence of the precursor charge states in HCD.&nbsp;</strong>&nbsp;</p><p><br></p><p><strong>2.2.5.1.2.</strong>&nbsp;<em>Add golden complementary pairs</em>: This option allows the user to display golden complementary pairs on the final result image. A golden complementary pair is a pair of fragment ions (a/x, b/y or c/z) that have been formed by cleavage between the same pair of amino acids (<strong>Figure 5</strong>). The sum of the two fragments equals the mass of the targeted protein molecular ion [<a href="https://www.pnas.org/content/97/19/10313" rel="noopener noreferrer" target="_blank">5</a>], [<a href="https://pubs.acs.org/doi/10.1021/ja973655h" rel="noopener noreferrer" target="_blank">6</a>]</p><p><br></p><p><strong>Figure 5: Golden complementary pairs generated in tandem mass spectrometry of intact proteins.</strong></p><p><br></p><p>Golden complementary pairs will be represented as a star (*) under the corresponding inter-residue cleavage (<strong>Figure 7</strong>).</p><p><sub><em>This option is allowed for maps with only 1 selected value per fixed condition</em></sub></p><p>&nbsp;</p><p><strong>2.2.5.1.3.</strong>&nbsp;<em>Add bond cleavage confidence</em>: this option adds a color scale onto the protein sequence to highlight confidence in each proteolytic bond cleavage. The greater the number of matched fragment ions the darker the amino acid color (<strong>Figure 8</strong>).&nbsp;</p><p>&nbsp;</p><p><strong>2.2.5.2.</strong>&nbsp;One or more maps can be created. To create a new map, click on&nbsp;the&nbsp;<em>Add another map</em>&nbsp;button&nbsp;and repeat the 2.2.5.1 steps.&nbsp;</p><p><strong>2.2.5.3.</strong>&nbsp;To&nbsp;display the fragmentation maps of the&nbsp;intact protein, click on&nbsp;the&nbsp;<em>Display</em>&nbsp;button. A new window will open.<strong>&nbsp;</strong>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p><p><strong>3.&nbsp;Evaluating&nbsp;results</strong>&nbsp;</p><p>&nbsp;</p><p><strong>3.1.</strong>&nbsp;The fragmentation map of the intact protein is shown in the&nbsp;<em>Display</em>&nbsp;tab (<strong>Figure&nbsp;6</strong>).&nbsp;&nbsp;</p><p><strong>3.1.1.</strong>&nbsp;&nbsp;Fragments&nbsp;are represented as bars above (for&nbsp;N-terminal&nbsp;fragments) and&nbsp;below&nbsp;the protein sequence (for&nbsp;C-terminal&nbsp;fragments) and are split&nbsp;into panels&nbsp;according to&nbsp;their&nbsp;type (a-, b-, c-, x-, y- or z-ions). Fragments are organized in lines&nbsp;and are colored&nbsp;according to the values of the study condition&nbsp;(<strong>Figures&nbsp;6-8</strong>).&nbsp;&nbsp;</p><p><strong>3.1.2.&nbsp;</strong>To ease the evaluation of the resulting fragmentation map, a&nbsp;legend table is printed&nbsp;below&nbsp;each map&nbsp;and contains&nbsp;information on:&nbsp;</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;the values of the study condition and the color code used,</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;the percentage of residue cleavages,</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;the number of matching fragments,</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;the optional number of golden complementary pairs,</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;the color scale of the optional bond cleavage confidence.</p><p>&nbsp;</p><p><strong>Figure 6: An example of a fragmentation map for the purpose of studying the influence of the precursor charge state in HCD with 20% activation.&nbsp;</strong>&nbsp;</p><p>&nbsp;</p><p><strong>Figure 7: An example of a fragmentation map for the purpose of studying the influence of the precursor charge state in HCD with 20% activation and with the golden complementary pair option enabled.</strong></p><p>&nbsp;</p><p><strong>Figure 8: An example of a fragmentation map for the purpose of studying the influence of the precursor charge state in HCD with 20% activation and with the bond cleavage confidence option enabled.</strong></p><p>&nbsp;</p><p><strong>3.2.&nbsp;<em>Option Intensity:&nbsp;</em></strong>If the deconvoluted spectra files have been loaded&nbsp;(step <em>2.2.3.3</em>), the&nbsp;<em>Option Intensity</em>&nbsp;tab is enabled, and information about the intensity of deconvoluted fragment ions can be added onto the fragmentation map.&nbsp;</p><p>	<strong>3.2.1.</strong> Two options are available (<strong>Figure 9</strong>):</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Relative intensity: the relative abundance of deconvoluted fragment ions will be used (<strong>Figure 10</strong>).</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Absolute intensity: the sum intensity of deconvoluted fragment ions will be used (<strong>Figure 11</strong>)</p><p><br></p><p><strong>3.2.2.</strong>&nbsp;After choosing one option, click on the <em>Display</em> button to evaluate the results. The visualization of ion intensity can be simply removed by unticking the “Per study map” button in the <em>Option Intensity</em> tab.</p><p>&nbsp;</p><p><strong>Figure 9: <em>Option Intensity</em> tab.</strong></p><p><br></p><p><strong>Figure 10: An example of a fragmentation map for the purpose of studying the influence of the precursor charge states in HCD with 20% activation considering the relative intensity of deconvoluted fragment ions.</strong></p><p><br></p><p><strong>Figure 11: An example of a fragmentation map for the purpose of studying the influence of the precursor charge states in HCD with 20% activation considering the absolute intensity of deconvoluted fragment ions.</strong></p><p><br></p><p><strong>3.3.&nbsp;<em>Option Merging</em></strong><em>:</em>&nbsp;the&nbsp;ultimate goal&nbsp;of&nbsp;<em>TDFragMapper</em>&nbsp;is to&nbsp;generate&nbsp;a&nbsp;final fragmentation&nbsp;map&nbsp;of an intact protein by combining all the fragments from the best MS/MS conditions&nbsp;</p><p><strong>3.3.1&nbsp;After having displayed and evaluated individual fragmentation maps,&nbsp;</strong>the user&nbsp;has the possibility to&nbsp;merge some conditions,&nbsp;through the&nbsp;<em>Option Merging</em>&nbsp;tab as shown in&nbsp;<strong>Figure 12</strong>.&nbsp;</p><p><strong>3.3.1.1.</strong>&nbsp;<em>Cleavage frequency</em>: this option computes the frequency at which cleavages were find across the merged conditions and applies a corresponding color scale to the cleavages.&nbsp;&nbsp;</p><p><br></p><p><strong>Figure 12: <em>Option Merging </em>– an example of setting up the combination of different studies previously created in different maps.</strong></p><p>&nbsp;</p><p><strong>3.3.2.&nbsp;</strong>This final merged&nbsp;fragmentation&nbsp;map represents&nbsp;only N-Terminal&nbsp;cleavages&nbsp;(independently of the ion type a-, b- or c-)&nbsp;above the protein sequence&nbsp;and C-Terminal&nbsp;cleavages&nbsp;(independently of the ion type x-, y- or z-)&nbsp;below the protein sequence.&nbsp;The final percentage of residue cleavages is computed and displayed below the fragmentation map. If the&nbsp;<em>Cleavage frequency</em>&nbsp;option is enabled, cleavages will then be&nbsp;colored&nbsp;and the color scale will be displayed below the fragmentation map as can be seen in&nbsp;<strong>Figure 13</strong>.&nbsp;&nbsp;</p><p>&nbsp;</p><p><strong>Figure 13: <em>Option Merging </em>– an example of merged fragmentation maps with (top panel) and without (bottom panel) the <em>Cleavage frequency</em> option enabled.</strong></p><p>&nbsp;</p><p><strong>3.4.<em>&nbsp;Saving &amp; exporting results</em></strong>&nbsp;</p><p><strong>3.4.1.</strong>&nbsp;<strong><em>TDFragMapper</em></strong>&nbsp;allows to export the maps as image in the following formats: TIFF, PNG, JPG. This is done by clicking&nbsp;on&nbsp;File&nbsp;menu&nbsp;→ Export&nbsp;→&nbsp;Image (or press CTRL + I), as shown in&nbsp;<strong>Figure 14</strong>.&nbsp;</p><p><strong>3.4.2.</strong>&nbsp;<strong><em>TDFragMapper</em></strong>&nbsp;also allows to save the results which could later loaded back. This is done by clicking on File menu&nbsp;→&nbsp;Save Results (or press CTRL + S), as shown in&nbsp;<strong>Figure 14</strong>.&nbsp;</p><p><strong>3.4.3.</strong>&nbsp;<strong><em>TDFragMapper</em></strong>&nbsp;also allows to export a summary report to PDF®&nbsp;file containing uploaded data files information and all the parameters used to create the maps. This is done by clicking on File menu&nbsp;→&nbsp;Export&nbsp;→&nbsp;Summary report (or by pressing CTRL + P), as shown in&nbsp;<strong>Figure 14</strong>.&nbsp;</p><p>&nbsp;&nbsp;</p><p><strong>Figure&nbsp;14:&nbsp;Save results</strong>&nbsp;</p><p>&nbsp;</p><p><strong>3.5.&nbsp;<em>Loading results</em></strong>&nbsp;</p><p><strong>3.5.1.&nbsp;<em>TDFragMapper</em></strong>&nbsp;loads results in its own format (<em>*.tdfm</em>). This can be accomplished in&nbsp;two&nbsp;ways, by double-clicking on a&nbsp;TDFragMapper&nbsp;results file&nbsp;or by clicking on File menu&nbsp;→&nbsp;<em>Load Results</em>&nbsp;(or press CTRL + O)&nbsp;and selecting the file, as seen in&nbsp;<strong>Figure&nbsp;15</strong>.&nbsp;</p><p><br></p><p><strong>Figure&nbsp;15: Load&nbsp;<em>TDFragMapper</em>&nbsp;results from&nbsp;the&nbsp;main interface.&nbsp;</strong>&nbsp;</p><p>&nbsp;</p><p><strong>3.8.</strong>&nbsp;<strong><em>Tutorial</em></strong>&nbsp;</p><p>This&nbsp;tutorial can be accessed through Help menu&nbsp;→&nbsp;Read Me (or by pressing F1).&nbsp;</p><h2>References</h2><p>[<a href="https://pubs.acs.org/doi/10.1021/acs.analchem.8b00984" rel="noopener noreferrer" target="_blank">1</a>] Fornelli, L. <em>et al.</em> (2018) Accurate Sequence Analysis of a Monoclonal Antibody by Top-Down and Middle-Down Orbitrap Mass Spectrometry Applying Multiple Ion Activation Techniques, <em>Anal. Chem.</em>, 90, 8421-8429.</p><p>[<a href="https://pubs.acs.org/doi/10.1021/acs.analchem.5b00162" rel="noopener noreferrer" target="_blank">2</a>] Brunner, A.M. <em>et al.</em> (2015) Benchmarking Multiple Fragmentation Methods on an Orbitrap Fusion for Top-down Phospho-Proteoform Characterization, <em>Anal. Chem.</em>, 87, 4152-4158.</p><p>[<a href="https://onlinelibrary.wiley.com/doi/full/10.1002/pmic.201400313" rel="noopener noreferrer" target="_blank">3</a>] R. T. Fellers <em>et al.</em> (2015) ProSight Lite: Graphical software to analyze top-down mass spectrometry data, <em>PROTEOMICS</em>, 15, 7,1235‑1238</p><p>[<a href="https://assets.thermofisher.com/TFS-Assets/CMD/manuals/Man-XCALI-97576-Protein-Deconvolution-30-User-ManXCALI97576-A-EN.pdf" rel="noopener noreferrer" target="_blank">4</a>] Thermo® FreeStyle™ software</p><p>[<a href="https://www.pnas.org/content/97/19/10313" rel="noopener noreferrer" target="_blank">5</a>] D. M. Horn <em>et al.</em> (2000) Automated de novo sequencing of proteins by tandem high-resolution mass spectrometry, <em>Proc. Natl. Acad. Sci.</em>, 97, 19, 10313‑10317</p><p>[<a href="https://pubs.acs.org/doi/10.1021/ja973655h" rel="noopener noreferrer" target="_blank">6</a>] N. L. Kelleher <em>et al.</em> (1999) Top Down versus Bottom Up Protein Characterization by Tandem High-Resolution Mass Spectrometry, <em>J. Am. Chem. Soc.</em>, 121, 4, 806‑812</p><p><br></p><h2>Acknowledgements</h2><p>	This project has received funding from the European Horizon 2020 research and innovation program under grant agreements No 829157 and No 823839.</p><p><br></p></body></html>