<html><head><title>Optimised protocol of QIAamp® DNA mini Kit for bacteria genomic DNA extraction from both pure and mixture sample</title></head><body><h1>Optimised protocol of QIAamp® DNA mini Kit for bacteria genomic DNA extraction from both pure and mixture sample</h1>Authors: Zhong Yang<br>Abstract: <p>As the sequencing technology developed, Next Generation Sequencing (NGS) has been used by more and more microbiology lab from different fields. Meanwhile , the quality of DNA will highly influence the sequencing result.  Recently, there is an increasingly requirement for effective DNA extraction protocol to get high quality DNA from both pure and mixture cultures. Especially for researchers who what to study the microorganisms from complex mixture samples from food and environment. In this protocol, we modified the protocol of a commercial DNA extraction kit from Qiagen, and made it suitable for DNA extraction of different sample sources. The optimized protocol is proved to be effective. The DNA extracted with this protocol has been sent for NGS and get many interesting result. </p><br>Keywords: Bacteria, Genomic DNA, Next Generation Sequencing. <br><h2>Introduction</h2><p>This is a modified bacteria DNA extraction protocol subject to QIAamp® DNA mini Kit. The protocol is modified to do DNA extraction for both mixture sample and pure bacteria culture, especially for bacteria difficult to lyse.</p><p><br></p><h2>Reagents</h2><p>QIAamp® DNA mini Kit</p><p>100% ethanol </p><p>100mg/ml lysozyme</p><p>1M Tris-HCl (pH8)</p><p>500mM (0.5M) EDTA</p><p>10% Triton</p><p><br></p><h2>Equipment</h2><p>Centrifuge</p><p>vortex</p><p>shaker</p><p>heatblock </p><h2>Procedure</h2><p><strong>Optimised protocol of QIAamp® DNA mini Kit for bacteria genomic DNA extraction from both pure and mixture sample</strong></p><p><br></p><p>This is a modified bacteria DNA extraction protocol subject to&nbsp;QIAamp® DNA mini Kit. The protocol is modified to do DNA extraction for both mixture sample and pure bacteria culture, especially for bacteria difficult to lyse.&nbsp;</p><p><strong>Pre-treatment of samples:</strong></p><p><strong>A.&nbsp;&nbsp;&nbsp;&nbsp;Pure culture bacteria culture</strong>.</p><p><strong>This pre-treatment is suitable for gram positive bacteria or bacteria with stick cell wall and difficult to lysis. Frozen progress could also be used for gram negative bacteria to increasing the lysis effectivity.&nbsp;</strong></p><p><strong>A1.</strong>&nbsp;Pick single and pure colony for broth culture. Collect 1ml of log phase culture in microtube.&nbsp;&nbsp;<strong>A2.</strong>&nbsp;Pellet bacteria by centrifugation for 3min at 12000rpm. Resuspend pellet in clear water and centrifugated at the same speed for 2min. Remove the supernatant and put the bacteria pellet in liquid nitrogen or -80℃ fridge for at least 2h.&nbsp;</p><p><strong>A3.</strong>&nbsp;Heat the pellet to 95 ℃ for 5min, vortex to the top speed and short spin to remove the drops on the cap.&nbsp;</p><p><strong>A4.</strong>&nbsp;Put the pellet back into liquid nitrogen or -80℃ fridge until lysis.&nbsp;</p><p><strong>*:</strong>&nbsp;The log phase culture is active and easier to lyse, in order to obtain a better lysis effect, a smaller number of bacteria should be used if the bacteria are very difficult to lyse. The pellet can be collected during glycerol stock preparation, to make sure the bacteria is in the same status of glycerol stock. The frozen progress is the take advantage of the ice crystal formed in frozen temperature to break the cell wall and promoting the lysed effectivity.&nbsp;</p><p><strong>B. Bacteria of liquid sample.</strong></p><p><strong>This pre-treatment protocol is suitable for the liquid base sample like real environmental water, pollution, drinks and also can use for powder sample dissolved in solutions and solid sample treated in buffer with stomacher.&nbsp;</strong></p><p><strong>B1.&nbsp;</strong>Collect the bacteria from liquid sample with 0.45um filtration. Add equal volume of 0.85% saline to the filtration flask to wash the bacteria left on the flask wall.&nbsp;</p><p><strong>B2.</strong>&nbsp;Use a culture loop to swap down all the bacteria on the filter membrane and dissolve the bacteria in 1ml clear water and freeze the bacteria solution in liquid nitrogen or -80℃ fridge.</p><p><strong>B3.</strong>&nbsp;Boil the bacteria solution in 95℃ for 5min and put back to fridge before lysis.&nbsp;</p><p><strong>*:</strong>&nbsp;The filtration progress is mean to collect only the microorganism in the liquid samples; this is important for researches mainly focus on microorganism. As the metagenomic sequencing may collect lots of unwanted sequencing information if extract DNA from the whole sample without treatment. For the samples with big truck and particle, a pre-filtration can be used to remove the unwanted tissues. The filtration size could be modified based on the research requirement. A small volume sample up to 1ml can be used for extraction directly, no need the filtration step.</p><p><strong>DNA extraction with the Qiagen DNA mini kit</strong></p><p><strong>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Lysozyme lysis.&nbsp;</strong></p><p><strong>A.&nbsp;&nbsp;&nbsp;&nbsp;Pellet lysis.&nbsp;</strong></p><p><strong>A1.</strong>&nbsp;Make lysozyme buffer according to the gram-positive protocol in Qiagen Handbook.&nbsp;</p><p><strong>A2.</strong>&nbsp;Take pellet from -80℃, melt at room temperature. Use 180ul of lysozyme buffer to resuspend the pellet, lysis at 37℃, 1200rpm for 3h.&nbsp;</p><p>*: The lysis time could be further optimized based on the bacteria, as a longer lysis time may cause the degradation of DNA. However, insufficient lysis time would result in uncomplete lysis, which would highly influence the extraction effectivity.</p><p><strong>B.&nbsp;&nbsp;&nbsp;&nbsp;Microbiota solution lysis</strong></p><p><strong>B1.&nbsp;</strong>Dilute the following reagents in the microbiota solution to reach a final concentration as list in table 1. The volume is calculated based on 1ml obtained from pre-treatment step B2.</p><p><strong>Table1. Buffer calculation table.</strong></p><p><strong>B2.&nbsp;</strong>Lysis the mixture at 37 ℃, 1200rpm for 3h.&nbsp;</p><p><strong>B3.</strong>&nbsp;Aliquoted the lysis mixture into 180-190µl each tube.&nbsp;</p><p><strong>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Proteinase lysis.&nbsp;</strong></p><p><strong>2a.</strong>&nbsp;Short spin the lysed mixture tube to remove the drops on the cap, add 20µl of Qiagen Proteinase K solution(&gt;600mAU/ml), vortex and short spin.&nbsp;</p><p><strong>2b.</strong>&nbsp;Add 200ul AL buffer to the solution, lysis under 56 ℃, 1000 rpm for 2h, until the solution gets transparent. *: Increase the lysed time or add more Proteinase K if the solution is not clear.&nbsp;</p><p><strong>2c.</strong>&nbsp;Short spin and add 8µl RNase A, mix well, keep at room temperature for 10min.&nbsp;</p><p>*: the AL buffer from last step may influence the enzymatic activity of RNase A, but this is an optimized step for user to choose.</p><p><strong>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Add 200µl of pre-cold 100% ethanol (the ethanol should be kept in -30℃ before use) into the solution, mix well and short spin. Apply the mixture to Qiagen mini spin column(kit),&nbsp;centrifuge at 10000rpm for 90s.&nbsp;</p><p><strong>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Apply the liquid from the collection tube to the column again and then centrifuge at 10000rpm for 90s again.&nbsp;</p><p><strong>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Repeat step 4 once, (Make the mixture flow over the column for 2-3 times) and then discard the liquid in collection tube.</p><p><strong>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Add 650µl of 100% pre-cold ethanol to the column and centrifuge at 12000rpm for 3min. Discard the liquid in collection tube.&nbsp;</p><p>*: This is an optimise step, could be ignored based on different requirement.&nbsp;</p><p><strong>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Add 650µl of AW1 buffer to the column,&nbsp;centrifuge at 12000rpm for 2min. Discard the liquid in collection tube.</p><p><strong>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Repeat step 7.</p><p><strong>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Add 650µl of AW2 buffer to the column, centrifuge at 12000rpm for 2min. Discard the liquid in collection tube.</p><p><strong>10.&nbsp;&nbsp;</strong>Repeat step 9.</p><p><strong>11.&nbsp;&nbsp;</strong>Change to a new collect tube and centrifuge at 14000rpm, 3min to remove all the liquid. Air dry in the BSC for 10min. Put the column on a clean 2ml DNA low-binding tube.&nbsp;</p><p><strong>12.&nbsp;&nbsp;</strong>Heat the pure (DNase free) water to 70℃. Elute the DNA with 80µl of heat water, centrifuge at 10000rpm for 2min to collect the DNA solution.&nbsp;</p><p><strong>13.&nbsp;&nbsp;</strong>Apply the DNA solution to the column again and incubate at 65℃ for 1min. centrifuge at 12000rpm for 2min to collect the DNA solution.&nbsp;</p><p><strong>14.&nbsp;&nbsp;</strong>The DNA solution could be aliquoted for quality check with Nanodrop (as Fig 1)&nbsp;&nbsp;and Qubit, and stock at -80℃ for long time.&nbsp;</p><p><br></p><p>&nbsp;</p></body></html>