<html><head><title>Chromosome Conformation Capture with Nuclear Titrated Capture-C (NuTi Capture-C)</title></head><body><h1>Chromosome Conformation Capture with Nuclear Titrated Capture-C (NuTi Capture-C)</h1>Authors: Damien J. Downes, Jim R. Hughes<br>Abstract: <p>NuTi Capture-C is a Chromosome Conformation Capture (3C) approach, which can very efficiently identify chromatin interactions at target viewpoints at high resolution. The addition of high-throughput sequencing adaptors prior to enrichment allows for multiplexing of replicates and comparison samples. This method is an improvement on the previous NG Capture-C<sup>1</sup> method in that modifications have been made to the <em>in situ</em> 3C method to improve nuclear integrity (Nuclear 3C). Additionally, capture has been optimised to viewpoint complexity through titration, maximising on target sequence specificity. The experiment will take several weeks and provide reproducible interaction profiles for tens to thousands of viewpoints of interest.</p><br>Keywords: Chromosome conformation capture, gene regulation, genome structure<br><h2>Reagents</h2><p><strong>3C library preparation.</strong></p><p>37% formaldehyde (Sigma: 47608-250ML)</p><p>1 M Glycine (Sigma: G1726)</p><p>PBS (Invitrogen: 10010031)</p><p>PCR Grade Water (Ambion: AM9932)</p><p>Tris, pH 8 (Invitrogen: AM9855G)</p><p>NaCl (Invitrogen: AM9760G)</p><p>0.2% Igepal CA-630 (Sigma: I8896)</p><p>1 tab cOmplete Protease Inhibitor Cocktail (Sigma: 11873580001)</p><p>20% SDS (Invitrogen: AM9820)</p><p>Triton-X 100, 20% v/v solution in water (Sigma: T8787)</p><p><em>Dpn</em>II HC (NEB: R0543M)</p><p>T4 DNA HC Ligase (Life Tech: EL0013)</p><p>Tris-EDTA (TE) Buffer Solution (Sigma: 93302)</p><p>Proteinase K (Thermo Fisher: EO0491)</p><p>RNAse (Roche: 1119915)</p><p>Absolute Ethanol (VWR: 20821.330)</p><p>Phenol-Chloroform-Isoamylalcohol (PCI), 25:24:1 (Sigma: 77617)</p><p>Light PhaseLock Gel Tubes (5Prime: 733-2477)</p><p>3 M NaOAc, pH 5.6 (Invitrogen: AM9740)</p><p>GlycoBlue (Thermo Fisher: AM9515)</p><p>Qubit dsDNA BR Assay kit (Invitrogen: Q32850)</p><p>KAPA Sybr Fast Universal (KAPA: KK4602)</p><p>qPCR primers (See Figure 1)</p><p><br></p><p><strong>3C library Indexing</strong></p><p>6 µg 3C library</p><p>PCR Grade Water (Ambion: AM9932)</p><p>Covaris microTUBE AFA Fiber pre-split snap-cap 6x16mm (Covaris: 520045)</p><p>Agencourt Ampure XP SPRI Beads (Beckman Coulter: A63881)</p><p>Absolute Ethanol (Make fresh 80% v/v solution in water) (VWR: 20821.330)</p><p>D1000 Reagents (Agilent: 50675583)</p><p>D1000 Loading Tips (Agilent: 50675153)</p><p>D1000 ScreenTape (Agilent: 50675582)</p><p>NEBNext Ultra II (New England: 7645S/L)</p><p>NEBNext Multiplex Oligos for Illumina Primer set 1 (New England: E7335S/L)</p><p>NEBNext Multiplex Oligos for Illumina Primer set 2 (New England: E7500S/L)</p><p>Herculase II Fusion Polymerase kit (Agilent: 600677)</p><p>Qubit dsDNA BR Assay kit (Invitrogen: Q32850)</p><p><br></p><p><strong>Biotinylated Oligonucleotide Capture</strong></p><p>1.5-2 µg indexed 3C library</p><p>Biotinylated Oligonucleotides (design using <a href="http://apps.molbiol.ox.ac.uk/CaptureC/cgi-bin/CapSequm.cgi" rel="noopener noreferrer" target="_blank">Capsqeum2</a>)</p><p>1 µg/µl Mouse COT DNA (Invitrogen: 18440016)</p><p>Nimblegen SeqCap EZ HE-oligo kit A (Roche: 06777287001)</p><p>Nimblegen SeqCap EZ HE-oligo kit B (Roche: 06777317001)</p><p>Nimblegen SeqCap EZ Accessory kit v2 (Roche: 07145594001)</p><p>Nimblegen SeqCap EZ Hybridisation and wash kit (Roche: 05634261001)</p><p>PCR Grade Water (Ambion: AM9932)</p><p>M-270 Streptavidin Dynabeads (Invitrogen: 65305)</p><p>Agencourt Ampure XP SPRI Beads (Beckman Coulter: A63881)</p><p>Qubit dsDNA HS Assay Kit (Invitrogen: Q32851)</p><p>High Sensitivity D1000 Reagents (Agilent: 5067 5585)</p><p>High Sensitivity D1000 ScreenTape (Agilent: 5067-5584)</p><p>D1000 Loading Tips (Agilent: 50675153)</p><p>D1000 Reagents (Agilent: 50675583)</p><p>D1000 ScreenTape (Agilent: 50675582)</p><p>KAPA Library Quantification Complete Kit, Universal (KAPA: KK4824)</p><p><br></p><p><strong>Fresh Lysis buffer</strong> – for 10 samples:</p><p>Make ahead of time and put it on roller in cold room to dissolve and cool</p><p>&nbsp;500 µl 1 M Tris pH 8 (10 mM)</p><p>100 µl 5 M NaCl (10 mM)</p><p>1 ml 10% Igepal CA-630 (0.2%)</p><p>1 tab cOmplete Protease Inhibitor Cocktail</p><p>48.4 ml PCR Grade Water</p><h2>Equipment</h2><p>Eppendorf Thermomixer C (Eppendorf: 5382000031)</p><p>Nanodrop One, or equivalent (ThermoFischer: ND-ONE-W)</p><p>Qubit 4 Fluorometer, or equivalent (ThermoFisher: Q33238)</p><p>qPCR Thermocycler</p><p>M220 Focused Ultrasonicator, or equivalent (Covaris: 500295)</p><p>DynaMag-2 (Invitrogen: 13221D)</p><p>Agilent 4200 TapeStation, or equivalent (Agilent: G2991AA)</p><p>PCR Thermocycler</p><p>Speedy Vac vacuum centrifuge, or equivalent</p><p>NEBNext Magnetic Isolation Rack, or equivalent (NEB: S1515S)</p><h2>Procedure</h2><p><strong>3C Library Generation</strong></p><p><em>Formaldehyde fixation (2 hours)</em></p><p>1. Pre-cool large centrifuge to 4°C. Chill glycine, PBS, and lysis buffer.</p><p>2. Collect cells from tissue or culture and make single-cell suspensions of cells of 5 x 10<sup>6</sup> cells in 5 mL of growth media. (Can scale up, keep cell concentration consistent).</p><p>3. Add 270 µL 37% formaldehyde and incubate for 10 min at RT tumbling (2% final conc.).</p><p>4. Quench by adding 750 µL 1M cold glycine (1/8 of the final volume. 125 mM final conc.).</p><p>5. Centrifuge for 10 min at 500 rcf (4°C).</p><p>6. Wash pellet by gently re-suspending in 5 mL cold PBS.</p><p>7. Centrifuge for 10 min at 500 rcf (4°C), gently remove supernatant without disturbing pellet.</p><p>8. Re-suspend pellet in 5 mL ice-cold lysis buffer (make fresh).</p><p>9. Incubate for 20 min on ice.</p><p>10. Centrifuge for 10 min at 500 rcf (4°C), gently remove supernatant without disturbing pellet.</p><p>11. Wash pellet by gently re-suspending in 5 mL cold PBS.</p><p>12. Centrifuge for 10 min at 500 rcf (4°C), gently remove supernatant without disturbing pellet.</p><p>13. Re-suspend pellet in 1 mL PBS and transfer to 1.7 mL microtube.</p><p>14. Snap freeze (ethanol and dry ice OR liquid nitrogen).</p><p>&nbsp;</p><p><em>** SAFE STOPPING POINT – Store fixed cells at -80ºC **</em></p><p><strong>&nbsp;</strong></p><p><em>DpnII Restriction enzyme digest (1 day)</em></p><p>1. Pre-warm Eppendorf ThermoMixer to 37°C.</p><p>2. Defrost fixed cells on ice.</p><p>3. Centrifuge for 15 min at 500 rcf (4°C), gently remove supernatant without disturbing pellet.</p><p>4. Re-suspend pellet in 215 µL 1x <em>Dpn</em>II buffer.</p><p>5. Transfer 15 µL to a new tube for the undigested control (Control 1).</p><p>6. To the remaining 200 µL of sample add (in order), 437 µL of water, 60 µL of 10x Restriction buffer, and 10 µL of 20% SDS (0.28% final conc.).</p><p>6. To the remaining 200 µL of sample add (in order), 437 µL of water, 60 µL of 10x Restriction buffer, and 10 µL of 20% SDS (0.28% final conc.).</p><p>7. To Control 1 add, 227.5 µL of water, 28.5 µL of 10c Restriction buffer, and 4 µL of 20% SDS.</p><p>8. Shake horizontally at 500 rpm (intermittent: 30s on / 30s off) for 1 hr at 37°C using Eppendorf ThermoMixer to permeabilize the nuceli.</p><p>9. Neutralise the SDS by adding 66µL of 20% Triton X 100 to the 3C digestion and 25 µL to Control 1 (1.67% final conc.).</p><p>10. Shake horizontally for 1 hr at 37°C, 500 rpm (intermittent: 30s on / 30s off).</p><p>11. Add 10 µL <em>Dpn</em>II restriction enzyme (500U) to the digestion sample.</p><p>12. Shake horizontally for 2-3 hrs at 37°C, 500 rpm (intermittent: 30s on / 30s off).</p><p>13. Add a further 10 µL <em>Dpn</em>II (500 U) and incubate overnight at 37ºC.</p><p>14. The following morning add a final 10 µL of <em>Dpn</em>II and incubate for another 5-6 hours.</p><p><br></p><p><em>Ligation &amp; de-crosslinking (2 days)</em></p><p>1. Take 100 µL from the digestion reaction to make Control 2 (Digested, un-ligated control)</p><p>2. Add 200 µL PCR grade water to Control 2.</p><p>3. Store both Control 1 and Control 2 at -20ºC until required.</p><p>4. Place the digests on the 65°C block for 15 min to heat inactivate <em>Dpn</em>II.</p><p>5. Immediately cool digests on ice.</p><p>6. Cool the Eppendorf ThermoMixer to 16°C.</p><p>7. Add 500 µL water and 134 µL 10x Ligation buffer to the digest.</p><p>8. Add 8 µL T4 Ligase (240 U).</p><p>9. Incubate at 16ºC for ~22 hrs while shaking, 500 rpm (intermittent: 30s on / 30s off).</p><p>10. Pellet nuclei by centrifucation at 500 rcf for 15 min (room temp).</p><p>11. Gently remove all of the supernatant without disturbing nuclear pellet, and transfer to a new tube.</p><p>12. Store the retained supernatant as a safe guard in case pellet was difficult to visualise and was disturbed.</p><p>13. Resuspend nuclear pellet in 300 µL of TE buffer.</p><p>14. Add 5 µl Proteinase K (3 U).</p><p>15. Remove Control 1 and Control 2 from freezer.</p><p>16. Add 3 µL of Proteinase K (1.8 U) to each control.</p><p>17. Incubate 3C library, Control 1, and Control 2 at 65ºC overnight.</p><p><em>&nbsp;</em></p><p><em>DNA Extraction (4 hours)</em></p><p>1. Cool samples to 37°C</p><p>2. Add 5 µL RNAse (7.5 mU) to each tube.</p><p>3. Incubate 30 min at 37°C on a thermomixer (500 rpm, intermittent: 30s on / 30s off).</p><p>4. Prepare PhaseLock tubes by spinning at 5,000 rcf for 2 min.</p><p>5. Add 310 µL phenol-chloroform isoamylalcohol (PCI) to each sample.</p><p>6. Vortex thoroughly to mix.</p><p>7. Transfer each sample to a PhaseLock tube and centrifuge for 10 min at 12,600 rcf (room temp).</p><p>8.&nbsp;Transfer the upper layer to a new microcentrifuge tube, avoiding the viscous interface.</p><p>9. Add 30 µL of 3M sodium acetate (NaOAc) and 1 µL of glycoblue.</p><p>10. Mix by inversion.</p><p>11. Add 900 µL of 100% ethanol and mix thoroughly by inversion (75% final conc.).</p><p>12. Incubate at -20ºC for at least 2 hrs.</p><p>&nbsp;</p><p><em>** SAFE STOPPING POINT – Store precipitation reaction at -20ºC for several days **</em></p><p>&nbsp;</p><p>13. Precool centrifuge to 4°C, chill 70% ethanol.</p><p>14. Pellet DNA by centrifugation at 21,000 rcf for 30 min (4ºC).</p><p>15. Discard supernatant and wash pellet in 1 mL 70% cold ethanol.</p><p>16. Centrifuge for 2 min at 21,000 rcf (4ºC).</p><p>17. Remove ethanol and repeat ethanol wash for a total of two washes.</p><p>18. Spin in benchtop microcentrifuge and remove residual ethanol with a low volume pipette.</p><p>19. Air dry at room temperature (~20 min).</p><p>20. Re-suspend DNA pellet in PCR grade water at 4ºC overnight (Controls, 30 µL. 3C library: 137 µL).</p><p><br></p><p><em>Library quality control (4 hours)</em></p><p>1. Make a 1% agarose gel using TAE.</p><p>2. Run agarose gel using 10 µL of each control and 5 µL of 3C library.</p><p>3. Perform quantitative real-time PCR on control 1 and control 2 to determine digestion efficiency.</p><p>4. Determine 3C library DNA concentration using 2 µL in a Qubit BR assay.</p><p>&nbsp;</p><p><em>** SAFE STOPPING POINT – Store 3C library at -20ºC **</em></p><p><br></p><p><strong>3C library indexing</strong></p><p><em>Sonication (1 hour)</em></p><p>** Sonication conditions should be determined for each machine with high molecular weight genomic DNA **</p><p><br></p><p>1. Transfer 130 µL of 3C library to a Covaris microTUBE.</p><p>2. Shear DNA to 200 bp. Covaris M220 settings:</p><p><br></p><p><em>-&nbsp;Duration:</em> 130 sec.</p><p><em>- Peak power:</em> 70.0</p><p><em>- Duty factor: </em>20%</p><p><em>- Cycles/burst: </em>1000</p><p><br></p><p>3. Transfer sonicated DNA to a new microcentrifuge tube containing 230 µL of Ampure XP SPRI beads.</p><p>4. Mix by pipetting up and down 10 times, allow to bind at room temp for 5 min.</p><p>5. Place on magnetic stand, discard liquid when clear (~5 min).</p><p>6. Add 700 μL of fresh 80% ethanol without removing from magnetic stand. Avoid disturbing beads by running the ethanol down the front of tube.</p><p>7. Allow to sit on the magnetic stand for 30 seconds then remove ethanol.</p><p>8. Repeat ethanol wash with a further 700 µL of ethanol.</p><p>9. Remove ethanol, spin tube on a microfuge and replace on magnetic stand.</p><p>10. Remove residual ethanol with a P10 pipette, taking care not to remove any beads.</p><p>11. Air Dry at room temperature on magnetic stand until matt in appearance – take care not to over dry the beads as this will result in increased DNA losses. Cracks will appear when beads have overdried.</p><p>12. Remove from magnetic stand and re-suspend beads in 55 μL PCR Grade water.</p><p>13. Incubate at room temperature for 2 min to elute.</p><p>14. Place on magnetic stand and allow beads to clear (~5 min).</p><p>15. Recover 53 µL of DNA to a new tube.</p><p>16. Assess sonication efficiency using 1 µL of DNA and a D1000 TapeStation.</p><p>17. Quantify recovered material with a BR Qubit (use 2 µl).</p><p>18. Dilute sonicated DNA to a have a maximum concentration of 40 ng/µL.</p><p>&nbsp;</p><p><em>** SAFE STOPPING POINT – Store Sonicated DNA at -20ºC **</em></p><p><strong>&nbsp;</strong></p><p><em>Addition of Sequencing Adaptors (3 hours)</em></p><p>** Modified protocol for the NEBNext Ultra II indexing kit **</p><p><br></p><p>1. Combine 50 µL of sonicated DNA (≤2 µg), 7 µL 10x End Prep Buffer (Green) and 3 µL End Prep Enzyme (Green) in a PCR tube.</p><p>2. Mix by pipetting.</p><p>3. Using a thermocycler incubate at 20ºC (45 min) and then 65ºC (30 min) with the lid set to 75ºC.</p><p>4. Add 30 µL Ultra II Ligation Master Mix (Red), 7 µL NEBNext Adaptor (Red), 1 µL Ligation Enhancer (Red). 5. Mix by pipetting.</p><p>6. Incubate in a thermocycler for 30 min at 20°C (lid off).</p><p>7. Add 3 µL of USER<sup>TM</sup> Enzyme (Red).</p><p>8. Mix by pipetting.</p><p>9. Incubate in a thermocycle for 30 min at 37°C (lid: 47°C).</p><p>10. Transfer to a microcentrifuge tube containing 180 µL of AmpureXP Beads (1.8x).</p><p>11. Mix by pipetting up and down 10 times, allow to bind at room temp for 5 min.</p><p>12. Place on magnetic stand, discard liquid when clear (~5 min).</p><p>13. Add 700 μL of fresh 80% ethanol without removing from magnetic stand. Avoid disturbing beads by running the ethanol down the front of tube.</p><p>14. Allow to sit on the magnetic stand for 30 seconds then remove ethanol.</p><p>15. Repeat ethanol wash with a further 700 µL of ethanol.</p><p>16. Remove ethanol, spin tube on a microfuge and replace on magnetic stand.</p><p>17. Remove residual ethanol with a P10 pipette, taking care not to remove any beads.</p><p>18. Air Dry at room temperature on magnetic stand until matt in appearance – take care not to over dry the beads as this will result in increased DNA losses. Cracks will appear when beads have overdried.</p><p>19. Remove from magnetic stand and re-suspend beads in 60 μL PCR Grade water.</p><p>20. Incubate at room temperature for 2 min to elute.</p><p>21. Place on magnetic stand and allow beads to clear (~5 min).</p><p>22. Adaptor ligation can be assesed using 1 µL of DNA and a D1000 TapeStation.</p><p>23. Recover DNA by transferring 28.5 µL to each of two new PCR tubes.</p><p><br></p><p><em>PCR addition of indices (2 hours)</em></p><p>** Generally two identical reactions with the same index are performed for each sample, however using a different index in each reaction can increase library complexity or allow different capture reactions to be performed and then multiplexed for sequencing **</p><p><br></p><p>1. To each PCR tube containing 28.5 µL of adaptor ligated library add: 5 µL of NEB Universal Primer, 5 µL NEB Index Primer, 10 µL 5x Herculase II buffer, 0.5 µL dNTP, and 1 µL of Herculase DNA polymerase. Mix by pitpetting.</p><p>2. Amplify DNA using the following conditions (Lid: 108ºC):</p><p><br></p><p><em>- Step 1: </em>98°C (30 sec)</p><p><em>- Step 2: </em>98°C (10 sec)</p><p><em>- Step 3:</em><strong> </strong>65°C (30 sec)</p><p><em>- Step 4: </em>72°C (30 sec)</p><p><em>- Step 5: </em>Go to Step 2 for 6 total cycles</p><p><em>- Step 6: </em>72°C (5 min)&nbsp;&nbsp;</p><p><em>- Step 7: </em>4°C (Hold)</p><p>&nbsp;</p><p>3. Combine identical reaction in a microcentrifuge tube containing 180 µL of AmpureXP Beads (1.8x).</p><p>4. Mix by pipetting up and down 10 times, allow to bind at room temp for 5 min.</p><p>5. Place on magnetic stand, discard liquid when clear (~5 min).</p><p>6. Add 700 μL of fresh 80% ethanol without removing from magnetic stand. Avoid disturbing beads by running the ethanol down the front of tube.</p><p>7. Allow to sit on the magnetic stand for 30 seconds then remove ethanol.</p><p>8. Repeat ethanol wash with a further 700 µL of ethanol.</p><p>9. Remove ethanol, spin tube on a microfuge and replace on magnetic stand.</p><p>10. Remove residual ethanol with a P10 pipette, taking care not to remove any beads.</p><p>11. Air Dry at room temperature on magnetic stand until matt in appearance – take care not to over dry the beads as this will result in increased DNA losses. Cracks will appear when beads have overdried.</p><p>12. Remove from magnetic stand and re-suspend beads in 55 μL PCR Grade water.</p><p>13. Incubate at room temperature for 2 min to elute.</p><p>14. Place on magnetic stand and allow beads to clear (~5 min).</p><p>15. Recover DNA by transferring 53 µL to a new microcentrifuge tube.</p><p>16. Assess indexing sucess using 1 µL of DNA and a D1000 TapeStation.</p><p>17. Quantify Indexed 3C DNA using 2 µL in a Qubit BR reaction.</p><p><br></p><p><em>** SAFE STOPPING POINT – Store Indexed 3C Library at -20ºC **</em></p><p>&nbsp;<strong> </strong></p><p><strong>Biotinylated Oligonucleotide Capture 1</strong></p><p>** Biotinylated target enrichment is performed using custom oligonucleotide pools with Nimblegen SeqCap reagents.**</p><p><br></p><p><em>Oligonucelotide Preparation (1 hour)</em></p><p>1. Reconstitute individual or pools of oligonucleotides to a stock concentration so that each unique oligonucleotide is stored at ≥1 µM.</p><p>2. Generate pools of oligonucleotides by mixing in exact 1:1 stoichiometric ratio.</p><p>3. Make working solution of oligonucleotides by diluting pools so that<em> each unique oligonucleotide</em> is at 2.9 nM, a total of 34 µL is required for both captures when performing a standard 3 v 3 experiment with six libraries.</p><p><em>** e.g. The desired concentration for a pool of 12 oligonucleotides is 34.8 nM (12 x 2.9 nM).**</em></p><p>4. Re-suspend Nimblegen oligonucleotides according to the Nimblegen protocol.</p><p>- Add 120 µL PCR grade water to the HE Univeral Oligo tube (1 mM) and vortex.</p><p>- Add 10 µL PCR grade water to the HE Index Oligo tube (1 mM) and vortex.</p><p>- Add 480 µL PCR Grade water to the Post-LM-PCR Oligos.</p><p>&nbsp;</p><p><em>Hybridisation Reaction (4 days)</em></p><p>** The following instructions are for six multiplexed libraries in a standard 3 v 3 experiment, adjust volumes accordingly to suit experiment complexity. Experiments with more than six libraries should be split over multiple tubes after Step 3 **</p><p><br></p><p>1. Heat vacuum centrifuge to 55°C</p><p>2.&nbsp;Combine 1-2 µg of six differentially indexed samples at exactly 1:1 ratios by mass in a PCR tube.</p><p>3.&nbsp;Add 30 µL COT DNA (5 µL per library), 6 µL of TS-HE Universal oligo (1 µL per library) and 1 µL of each index specific TS-HE index oligo.</p><p>4. Vacuum centrifuge at 55ºC with tube lids open until sample is completely dry.</p><p>5. Preheat two thermocyclers, one to 95ºC (lid: 105ºC) and one to 47ºC (lid: 47ºC).</p><p>6. Add 45 μL 2x Hybridisation Buffer (vial 5, 7.5 µL per library) and 18 µL of (Hybridisation Component A (vial 3 µL per library).</p><p>7. Spin down to bottom in minifuge.</p><p>8. Preheat 29 µL of pooled biotinylated oligonucleotides probes in a PCR tube on the 47ºC thermocycler (4.5 µL per library, plus 2 µL residual).</p><p>9. Denature the multiplexed 3C library mix by heating to 95°C for 10 min.</p><p>10. Trasnfer the tube containing denature DNA to the 47ºC thermocyler.</p><p>11. Add 27 µL of pre-warmed oligonucleotide probes to denatured DNA, mix by pipetting but take care as the mixture will be gloopy.</p><p>12. Label PCR machine to prevent it being inadvertently turned off.</p><p>13. Incubate at 47°C for between 22 and 72 hours. (72 hours allows incubations to occur over weekends)</p><p><br></p><p><em>Wash Buffer and Bead Preparation (1 hour)</em></p><p>1. Preheat Thermomixer to 47ºC, place Streptavadin beads on bench to come to room temp.</p><p>2. Prepare wash buffers:</p><p><br></p><p><em>- Stringent Wash buffer (400 µL per library): </em>two tubes, each with 122 µL 10x buffer and 1,098 µL of water.</p><p><em>- Wash buffer I (300 µL per library): </em>two tubes, each with 94 µL of 10x buffer and 846 µL of water.</p><p><em>- Wash buffer II (200 µL per library): </em>one tube with 124 µL of 10x buffer and 1,116 µL of water.</p><p>- Wash buffe<em>r III (200 µL per library)</em>: one tube with 124 µL of 10x buffer and 1,116 µL of water.</p><p><em>- Bead Wash buffer (500 µL per library): </em>two tubes, each with 610 µL 2.5x buffer and 915 µL of water.</p><p><br></p><p>3. Transfer 620 µL of Wash buffer I to a new tube and place in the 47ºC thermomixer.</p><p>4. Place both tubes of Stringent Wash buffer in the 47ºC thermomixer.</p><p>5. Add 600 µL of streptavadin beads (100 µL per library) into a low bind tube.</p><p>6. Place on magnetic stand, remove liquid once clear (~30 sec).</p><p>7. Add 1,200 μL of 1x Bead Wash buffer (double the original volume of beads).</p><p>8. Vortex to re-suspend the beads, spin briefly and replace on magnet.</p><p>9. Remove liquid once clear (30 seconds).</p><p>10. Add 1,200 μL of 1x Bead Wash buffer to perform a second wash.</p><p>12. Vortex to re-suspend the beads, spin briefly and replace on magnet.</p><p>13. Remove liquid once clear (30 seconds).</p><p>14. Resuspend beads in 600 µL of 1x Bead Wash buffer (original volume of beads).</p><p>15. Vortex to re-suspend the beads, spin briefly and replace on magnet.</p><p>&nbsp;</p><p><em>Binding of Biotinylated Oligonucleotides (2 hours)</em></p><p>1. Briefly spin the hybridisation reaction to ensure it is at the bottom of the tube, replace at 47ºC.</p><p>2. Remove the Bead Wash buffer from streptavadin beads.</p><p>3. Transfer the 90 µL hybridisation reaction to the streptavidin beads, mix thoroughly by pipetting 10 times.</p><p>4. Place on ThermoMixer at 47°C (600 rpm) for 45 min, if beads settle out then pipette to resuspend.</p><p>5. Add 600 μL of 47ºC 1x Wash buffer I (100 µL per library) to the bead mixture.</p><p>6. Mix by vortexing for 10 sec, briefly spin to collect at bottom of the tube.</p><p>7. Place in magnetic stand and discard all the liquid when clear (~30 sec).</p><p>8. Add 1,200 μL of heated Stringent Wash buffer (200 µL per library) and mix by vortexing.</p><p>9. Incubate on ThermoMixer for 5 mins at 47°C, 600 rpm.</p><p>10. Briefly spin to collect at bottom of the tube.</p><p>11. Place in magnetic stand and discard all the liquid when clear (~30 sec)</p><p>12. Add another 1,200 μL of heated Stringent Wash buffer (200 µL per library) and mix by vortexing.</p><p>13. Incubate on ThermoMixer for 5 mins at 47°C, 600 rpm.</p><p>14. Briefly spin to collect at bottom of the tube.</p><p>15. Place in magnetic stand and discard all the liquid when clear (~30 sec).</p><p>16. Remove from the stand and add 1,200 µL of room temp 1x Wash buffer I (200 µL per library).</p><p>17. Mix by vortexing for 2 min.</p><p>18. Briefly spin to collect at bottom of the tube.</p><p>19. Place in magnetic stand and discard all the liquid when clear (~30 sec).</p><p>20. Remove from the stand and add 1,200 µL of 1x Wash buffer II (200 µL per library).</p><p>21. Mix by vortexing for 1 min.</p><p>22. Briefly spin to collect at bottom of the tube.</p><p>23. Place in magnetic stand and discard all the liquid when clear (~30 sec).</p><p>24. Remove from the stand and add 1,200 µL of 1x Wash buffer III (200 µL per library).</p><p>25. Mix by vortexing for 30 sec.</p><p>26. Briefly spin to collect at bottom of the tube.</p><p>27. Place in magnetic stand and discard all the liquid when clear (~30 sec).</p><p>28. Remove from the stand and add 240 µL of PCR grade water (40 µL per library).</p><p><br></p><p><em>** SAFE STOPPING POINT – Store bead bound DNA at -20ºC **</em></p><p>&nbsp;</p><p><em>Amplification of Captured DNA (2 hours)</em></p><p>1. To the 240 µL of bead bound DNA add, 300 µL KAPA HiFI HotStart ready Mix (50 µL per library) and 60 µL of POST-LM_PCR Oligo 1&amp;2 (10 µL per library).</p><p>2. Mix by pipetting and separate across 12 PCR tubes (50 µL each, two tubes per library)</p><p>3. Amplify DNA using the following conditions (Lid: 108ºC):</p><p><br></p><p><em>- Step 1: </em>98°C (45 sec)</p><p><em>- Step 2: </em>98°C (15 sec)</p><p><em>- Step 3:</em><strong> </strong>60°C (30 sec)</p><p><em>- Step 4: </em>72°C (30 sec)</p><p><em>- Step 5: </em>Go to Step 2 for 10 total cycles</p><p><em>- Step 6: </em>72°C (60 sec)&nbsp;&nbsp;</p><p><em>- Step 7: </em>4°C (Hold)</p><p>&nbsp;</p><p>4. Transfer PCR tubes to a magnetic rack and allow solution to clear (~30 sec).</p><p>5. Combine amplification reactions into a single microcentrifuge tube 1,080 µL of AmpureXP Beads (180 µL per library).</p><p>6. Mix by pipetting up and down 10 times, allow to bind at room temp for 5 min.</p><p>7. Place on magnetic stand, discard liquid when clear (~5 min).</p><p>8. Add 1,200 μL of fresh 80% ethanol without removing from magnetic stand. Avoid disturbing beads by running the ethanol down the front of tube.</p><p>9. Allow to sit on the magnetic stand for 30 seconds then remove ethanol.</p><p>10. Repeat ethanol wash with a further 1,200 µL of ethanol.</p><p>11. Remove ethanol, spin tube on a microfuge and replace on magnetic stand.</p><p>12. Remove residual ethanol with a P10 pipette, taking care not to remove any beads.</p><p>13. Air dry at room temperature on magnetic stand until matt in appearance – take care not to over dry the beads as this will result in increased DNA losses. Cracks will appear when beads have overdried.</p><p>14. Remove from magnetic stand and re-suspend beads in 105 μL PCR Grade water.</p><p>15. Incubate at room temperature for 2 min to elute.</p><p>16. Place on magnetic stand and allow beads to clear (~5 min).</p><p>17. Recover DNA by transferring 103 µL to a new microcentrifuge tube.</p><p>18. Assess DNA size using 1 µL in a D1000 TapeStation with High Sensitivity reagents.</p><p>19. Quantify Captured 3C DNA using 2 µL in a Qubit HS reaction.</p><p><br></p><p><em>** SAFE STOPPING POINT – Store Amplified Captured DNA at -20ºC **</em></p><p><br></p><p><strong>Biotinylated Oligonucleotide Capture 2 (Double Capture)</strong></p><p>** The total DNA recovered is likely &lt;2 µg and is therefore treated as a single library for double capture. If over 2 µg of DNA is recovered treat as two libraries. **</p><p><br></p><p>1.&nbsp;&nbsp;&nbsp;Using total amplified DNA from Capture 1, perform the steps described above for <em>Hybridisation Reaction</em>, <em>Wash Buffer and Bead Preparation</em>, <em>Binding of Biotinylated Oligonucleotides </em>and <em>Amplification of Captured DNA</em> as described but treating the reaction as a single library (volumes provided) with the following modifications:</p><p><br></p><p><em>Hybridisation Reaction:</em></p><p>- Use 0.16 µL of each TS-HE Index oligo (Step 4)</p><p>- Preheat 7 µL of oligonucleotide pool (Step 8)</p><p>- Incubate at 47ºC for between 18 and 22 hours (Step 13)</p><p><br></p><p><em>Amplification of Captured DNA:</em></p><p>- Elute in 55 µL of PCR grade water (Step 14)</p><p><em>- </em>Recvoer 53 µL of DNA (Step 17)</p><p>- Assess DNA size 1 µL in a D1000 TapeStation with standard reagents. (Step 18)</p><p>- Quantify Captured 3C DNA using 2 µL in a Qubit BR reaction (Step 19).</p><p><br></p><p><em>** SAFE STOPPING POINT – Store Amplified Double Captured DNA at -20ºC **</em></p><p>&nbsp;</p><p><strong>Library sequencing.&nbsp;</strong></p><p>1. Using Qubit determined concentration dilute 20 µL of Double Captured DNA to 10 nM.</p><p>2. Use KAPA Library Quantification Kit with 1:10,000 and 1:20,000 dilutions to determine concentration with size correction.</p><p>3. Using the KAPA determined concentration dilute Double Captured DNA to 4 nM (or Sequence Facility specified concentration).</p><p>4. Sequence using paired-end reads. Gold standard sequencing uses 150 bp reads which enable reconstruction of entire indexed fragments and identification of restriction sites. However, 75 bp paired reads are sufficient for analysis and can be considerably cheaper.</p><h2>References</h2><p>Davies J.O. et al. Multiplexed analysis of chromosome conformation at vastly improved sensitivity. <em>Nat Methods</em>. 13(1):74-80. doi:10.1038/nmeth.3664 (2016)</p></body></html>