<html><head><title>Quiescent stem cell marker genes in glioma gene networks are sufficient to distinguish between normal and glioblastoma (GBM) samples.</title></head><body><h1>Quiescent stem cell marker genes in glioma gene networks are sufficient to distinguish between normal and glioblastoma (GBM) samples.</h1>Authors: Shradha Mukherjee<br>Abstract: <p>Grade 4 glioma or GBM has poor prognosis and is the most aggressive grade of glioma. Accurate diagnosis and classification of tumor grade is a critical determinant for development of treatment pathway. Extensive genomic sequencing of gliomas, different cell types, brain tissue regions and advances in bioinformatics algorithms, have presented an opportunity to identify molecular markers that can complement existing histology and imaging methods used to diagnose and classify gliomas. ‘Cancer stem cell theory’ purports that a minor population of stem cells among the heterogeneous population of different cell types in the tumor, drive tumor growth and resistance to therapies. However, characterization of stem cell states in GBM and ability of stem cell state signature genes to serve as diagnostic or prognostic molecular markers are unknown. In this work, two different network construction algorithms, Weighted correlation network analysis (WGCNA) and Multiscale Clustering of Geometric Network (MEGENA), were applied on publicly available glioma, control brain and stem cell gene expression RNA-seq datasets, to identify gene network regulatory modules associated with GBM. Both gene network algorithms identified consensus or equivalent modules, HuAgeGBsplit_18 (WGCNA) and c1_HuAgeGBsplit_32/193 (MEGENA), significantly associated with GBM. Characterization of HuAgeGBsplit_18 (WGCNA) and c1_HuAgeGBsplit_32/193 (MEGENA) modules showed significant enrichment of rodent quiescent stem cell marker genes (GSE70696_QNPbyTAP). A logistic regression model built with eight of these quiescent stem cell marker genes (GSE70696_QNPbyTAP) was sufficient to distinguish between control and GBM samples. This study demonstrates that GBM associated gene regulatory modules are characterized by diagnostic quiescent stem cell marker genes, which may potentially be used clinically as diagnostic markers and therapeutic targets in GBM.</p><br>Keywords: stem cells, quiescence, proliferation, glioblastoma, cancer, network analysis, differential gene expression, biomarkers, bioinformatics<br><h2>Introduction</h2><p><strong><u>Overview: </u></strong>This is a comprehensive bioinformatics pipeline for doing differential gene expression analysis, network analysis and characterization of genes/network modules of interest. Modules of interest maybe Disease or other trait associated modules or genes. Characterization includes GO Biological Process, Pathway analysis etc.</p><p><br></p><p><strong><u>Please cite: </u></strong>This pipeline was generated and used for the publication with the following article, </p><p>Mukherjee, S.*# Quiescent stem cell marker genes in glioma gene networks are sufficient to distinguish between normal and glioblastoma (GBM) samples. Scientific Reports (accepted 2020).</p><p><br></p><p>Shradha Mukherjee*#</p><p>Independent Researcher</p><p>Address correspondence to: Shradha Mukherjee, PhD; Garland Avenue, Downtown Los Angeles, Los Angeles, CA 90017, U.S.</p><p>Email: smukher2@gmail.com</p><p>* First author and # Corresponding author</p><h2>Reagents</h2><p><strong><u>Input: </u></strong>RNA-seq gene expression data (fastq files, or bam files or htseq-count files) or Microarray gene expression data</p><p><br></p><p><strong><u>Output:</u> </strong>Gene network modules associated with trait of interest (disease) and their Gene Ontology (GO), pathway analysis and gene network module enrichment of deferentially expressed genes. </p><h2>Equipment</h2><p>Windows, Linux or Mac computers with internet.</p><h2>Procedure</h2><p><strong><u>Please cite:</u></strong> I hope you find these research findings and bioinformatics pipelines useful. Please help by citing as follows. Thank you.</p><p><br></p><p>Mukherjee, S.*# Quiescent stem cell marker genes in glioma gene networks are sufficient to distinguish between normal and glioblastoma (GBM) samples. Scientific Reports (accepted 2020).</p><p><br></p><p>Shradha Mukherjee*#</p><p>Independent Researcher</p><p>Address correspondence to: Shradha Mukherjee, PhD; Garland Avenue, Downtown Los Angeles, Los Angeles, CA 90017, U.S.</p><p>Email: smukher2@gmail.com</p><p>* First author and # Corresponding author</p><p><br></p><p><strong><u>Please download protocol i.e. codes from 'Supplementary Files': </u></strong>The following 5 folders are provided with files and codes:</p><p><br></p><p>**Step1 RNAseqASUcyverseClusterResultsScript/&nbsp;</p><p><br></p><p>Contains representative or sample code for analysis of RNA-seq data from rat, mouse and human. Glioblastoma RNA-seq was from human only, while quiescent/proliferative stem cells RNA-seq were from rat, mouse and human. The starting file to run these codes is fastq/bam files to produce htseq counts rest of the DEG, WGCNA and MEGENA pipelines.&nbsp;</p><p>&nbsp;</p><p>**Step2 DEGNoCuffQbyTStemCellspairs/&nbsp;</p><p><br></p><p>Contains the codes, input files and results for entire analysis of the quiescence vs proliferation differential gene expression analysis with 3 methods (Limma, edgeR and simple comparison of means) starting from htseq count files. The htseq count files were obtained in Step1 above.</p><p>The minimum files, folders and .Rmd codes required to reproduce the results are provided. After downloading these if the user runs the .Rmd codes in RStudio using command 'knit to html' or 'Run all' it should reproduce same results.&nbsp;</p><p>&nbsp;&nbsp;</p><p>**Step3 HuDisGBsplitWGCNA/&nbsp;</p><p><br></p><p>Contains the codes, input files and results for WGCNA analysis of htseq files obtained in Step1 above. Also contains code for characterization of modules with enrichR such as GO analysis; and contains code for Linear Regression/Modeling (to distinguish control and cancer samples) and Survival analysis.&nbsp;</p><p>The minimum files, folders and .Rmd codes required to reproduce the results are provided. After downloading these if the user runs the .Rmd codes in RStudio using command 'knit to html' or 'Run all' it should reproduce same results.&nbsp;</p><p><br></p><p>**Step4 HuDisGBsplitMEGENA/&nbsp;</p><p><br></p><p>Contains the codes, input files and results for MEGENA analysis of htseq files obtained in Step1 above. Also contains code for characterization of modules with enrichR such as GO analysis;</p><p>The minimum files, folders and .Rmd codes required to reproduce the results are provided. After downloading these if the user runs the .Rmd codes in RStudio using command 'knit to html' or 'Run all' it should reproduce same results.&nbsp;</p><p><br></p><p>**Step5 PreservationAnalysis/&nbsp;</p><p><br></p><p>Contains the codes, input files and results for comparison of MEGENA analysis and WGCNA analysis outputs obtained in Step3 and Step4 above.&nbsp;</p><p>The minimum files, folders and .Rmd codes required to reproduce the results are provided. After downloading these if the user runs the .Rmd codes in RStudio using command 'knit to html' or 'Run all' it should reproduce same results.&nbsp;&nbsp;</p><p><br></p><p><strong><u>Alternatively download link on github: </u></strong><a href="https://github.com/smukher2/GithubScientificReportsGlioblastomaStemApril2020" rel="noopener noreferrer" target="_blank">https://github.com/smukher2/GithubScientificReportsGlioblastomaStemApril2020</a></p><h2>Troubleshooting</h2><p><strong><u>Acknowledgements and additional resources:&nbsp;</u></strong></p><p>1) SVA: Dr. Jeff Leek <a href="http://jtleek.com/genstats/inst/doc/02_13_batch-effects.html" rel="noopener noreferrer" target="_blank">http://jtleek.com/genstats/inst/doc/02_13_batch-effects.html</a> and <a href="https://www.bioconductor.org/packages/release/bioc/vignettes/sva/inst/doc/sva.pdf" rel="noopener noreferrer" target="_blank">https://www.bioconductor.org/packages/release/bioc/vignettes/sva/inst/doc/sva.pdf</a></p><p>2) WGCNA: Dr. Jeremy Miller <a href="https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/JMiller/" rel="noopener noreferrer" target="_blank">https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/JMiller/</a></p><p>3) WGCNA Preservation: Dr. Steve Horvath and Dr. Peter Langfelder <a href="https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/ModulePreservation/Tutorials/MiniTutorial-MouseLiver.pdf" rel="noopener noreferrer" target="_blank">https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/ModulePreservation/Tutorials/MiniTutorial-MouseLiver.pdf</a></p><p>4) WGCNA Hub Genes: Dr. Steve Horvath and Dr. Peter Langfelder <a href="https://pdfs.semanticscholar.org/5e42/e2185c54874277794395e5825808e5f5709c.pdf" rel="noopener noreferrer" target="_blank">https://pdfs.semanticscholar.org/5e42/e2185c54874277794395e5825808e5f5709c.pdf</a></p><p>5) MEGENA: Dr. Won-Min Song and Dr. Bin Zhang <a href="https://rdrr.io/github/songw01/MEGENA/f/vignettes/MEGENA_pipeline_02202020.Rmd" rel="noopener noreferrer" target="_blank">https://rdrr.io/github/songw01/MEGENA/f/vignettes/MEGENA_pipeline_02202020.Rmd</a></p><p>6) edgeR: <a href="https://github.com/smukher2/StemCells_RNAseq_Expression_edgeR_visualization_GO" rel="noopener noreferrer" target="_blank">https://github.com/smukher2/StemCells_RNAseq_Expression_edgeR_visualization_GO</a> and <a href="https://web.stanford.edu/class/bios221/labs/rnaseq/lab_4_rnaseq.html" rel="noopener noreferrer" target="_blank">https://web.stanford.edu/class/bios221/labs/rnaseq/lab_4_rnaseq.html</a></p><p>7) limma: <a href="https://kasperdanielhansen.github.io/genbioconductor/html/limma.html" rel="noopener noreferrer" target="_blank">https://kasperdanielhansen.github.io/genbioconductor/html/limma.html</a> and <a href="https://ucdavis-bioinformatics-training.github.io/2018-June-RNA-Seq-Workshop/thursday/DE.html" rel="noopener noreferrer" target="_blank">https://ucdavis-bioinformatics-training.github.io/2018-June-RNA-Seq-Workshop/thursday/DE.html</a></p><p><br></p><p>Users new to SVA, WGCNA, MEGENA and Differential Gene Expression methods are encouraged to refer to the above mentioned turotials and pipelines. Other acknowledgements have been added to the best of our knowledge as #comment in the code/pipeline itself.&nbsp;</p><h2>Time Taken</h2><p>Variable as it depends on number of samples, number of variables and user.</p><h2>Anticipated Results</h2><p>1)&nbsp;&nbsp;&nbsp;SVA+LM approach reduces batch effects in RNA-seq datasets.</p><p>2)&nbsp;&nbsp;&nbsp;Network analysis to reveal (here GBM) associated modules.</p><p>3)&nbsp;&nbsp;&nbsp;Differential gene expression analysis to identify marker genes (here proliferative and quiescent stem cell marker genes).</p><p>4)&nbsp;&nbsp;Enrichment of marker genes (here proliferative and quiescent stem cell marker genes) underly gene network modules (here GBM modules).</p><p>5)&nbsp;&nbsp;Gene Ontology (GO) annotation of marker genes enriched gene network modules (here GBM modules enriched with quiescent stem cell marker genes).<strong> </strong></p><h2>References</h2><p><strong><u>Please refer to associated article for complete list of references.</u></strong></p><p><br></p><p>Mukherjee, S.*# Quiescent stem cell marker genes in glioma gene networks are sufficient to distinguish between normal and glioblastoma (GBM) samples. Scientific Reports (accepted 2020).</p><p><br></p><p>Shradha Mukherjee*#</p><p>Independent Researcher</p><p>Address correspondence to: Shradha Mukherjee, PhD; Garland Avenue, Downtown Los Angeles, Los Angeles, CA 90017, U.S.</p><p>Email: smukher2@gmail.com</p><p>* First author and # Corresponding author</p><h2>Acknowledgements</h2><p><strong><u>Acknowledgements:</u> </strong>This work was conducted by the author using free open access resources, personal resources and personal funds from 2017 to 2019. This work is self-owned and not affiliated with any organization/university. The author would like to express deepest gratitude to Springer Nature and Scientific Reports for their generous support. Other open source and free resources utilized for this work are acknowledged below.</p><p><br></p><p><u>For datasets</u>: Glioblastoma RNA-seq datasets, SRP027383 and SRP091303, were obtained from NCBI SRA. Other RNA-seq datasets for control brains (GSE67333, GSE64810, GSE100297 and GSE53697) and stem cells (GSE68270, GSE70696, GSE99777, GSE93991 and GSE114574), were obtained from NCBI GEO. The author is most grateful to NCBI SRA <a href="https://www.ncbi.nlm.nih.gov/sra" rel="noopener noreferrer" target="_blank">https://www.ncbi.nlm.nih.gov/sra</a> and NCBI GEO <a href="https://www.ncbi.nlm.nih.gov/geo/" rel="noopener noreferrer" target="_blank">https://www.ncbi.nlm.nih.gov/geo/</a> for storing and making these datasets open access. The author also appreciates the organizations and investigators who generously submitted these datasets for sharing to NCBI GEO and NCBI SRA.</p><p><br></p><p><u>For full text publications access</u>: The author is most grateful to NIH/NLM (U.S. National Institute of Health’s National Library of Medicine) for open-access to full-text scientific publications on <a href="http://www.ncbi.nlm.nih.gov/pmc/" rel="noopener noreferrer" target="_blank">www.ncbi.nlm.nih.gov/pmc/</a> PubMed Central.</p><p><br></p><p><u>For computational pipelines</u>: Citations and descriptions of all computational pipelines have been given under methods sections. Additionally, the author would like to add special thanks to Dr. Steve Horvath, Professor at Department of Human Genetics and Biostatistics, University of California Los Angeles (UCLA) and Dr. Jeffrey T. Leek, Professor at Department of Biostatistics, Johns Hopkins for their publicly available computational pipelines on WGCNA/enrichment/Preservation analyses and SVA/Regression analyses, respectively. Dr. Won-Min Song and Dr. Bin Zhang from Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai for their publicly available computational MEGENA pipeline on github. Dr. Steve Horvath, Dr. Jeffrey T. Leek, Dr. Won-Min Song and Dr. Bin Zhang’s computational tutorials were integral to the author’s learning and for execution of the work done in this paper.&nbsp;</p><p><br></p><p><u>For computational platform</u>: The computation was performed on personal laptops (MacBook Pro and Windows OS) with high-speed internet connection and computational tools. RNA-seq datasets were mapped using NSF supported publicly available Cyverse Discovery Environment.&nbsp;</p></body></html>