<html><head><title>ULI-NChIP assay protocol</title></head><body><h1>ULI-NChIP assay protocol</h1>Authors: Tie-Gang Meng, Qian Zhou, Xue-Shan Ma, Xiao-Yu Liu, Qing-Ren Meng,  Xian-Ju Huang, Hong-Lin Liu, Wen-Long Lei, Zheng-Hui Zhao, Ying-Chun Ouyang, Yi Hou, Heide Schatten, Xiang-Hong Ou, Zhen-Bo Wang, Shao-Rong Gao, Qing-Yuan Sun<br>Abstract: <p>This protocol presents ULI-NChIP-seq (ultra-low-input micrococcal nuclease-based native ChIP-seq) assay to&nbsp;generate high quality and complexity genome-wide histone mark profiles from rare oocytes andembryos populations. The procedure of ULI-NChIP-seq assay typically consists of five parts including Binding antibodies to magnatic beads, Chromatin shearing and nuclear membrane solubilization, Magnetic immunoprecipitation, Washes and DNA isolation. Sample preparation involves to remove the zona Pellucida of oocyte and polar body to avoid the&nbsp;genomic contamination of polar bodies.</p><br>Keywords: ULI-NChIP, oocyte, histone modification<br><h2>Introduction</h2><p>By optimizing the traditional chromatin immunoprecipitation (ChIP) technology, combined with the new DNA library technology (TELP), scientists have developed a new set of high-sensitivity research on histone modification technology suitable for very low cell mass, making One can detect histone modification patterns in the whole genome[1-6]. According to the existing research results, people’s understanding of histone modifications has become clearer, and they have gradually realized that the distribution of various histone modifications on the genome is not uniform: the trimethylation of lysine is mainly enriched in the CDS region of genes , Acetylation is mainly distributed in the promoter region of specific genes and the 5'end of CDS. Generally speaking, each modification of histone has its own special modification map (landscape) on the genome. At the evolutionary level, by comparing the sequencing data of yeast, mouse and human, it is found that the distribution pattern of histone modifications is relatively conservative. At the same time, due to the limitations of research methods, there are still few studies on the establishment and regulation of specific histone modifications. If a mutation study is performed on histones, it is difficult to carry out because of the multiple copies of histone coding genes in mammalian cells. Generally, when a gene is knocked out or knocked down, a certain histone modification is increased or decreased to speculate that the gene is involved in the regulation of the histone modification. At the same time, there are obvious differences in the distribution pattern of a histone modification on the genome in different developmental stages and different tissues, and the same histone modification may have multiple modification enzymes to modify it. Thanks to the accumulation of various research methods and the establishment of the new micro-sequencing technology ULI-NChIP (ultra-low-input micrococcal nuclease-based native ChIP), we can study the genetic and reprogramming patterns and molecular regulation of histone modifications before and after fertilization mechanism.</p><h2>Reagents</h2><p>1. Micrococcal Nuclease (MNase) Cat:NEB M0247S</p><p>2. 100mM DTT INVITROGEN # 18080044</p><p>SuperScript III Reverse Transcriptase Kit 100mM DTT</p><p>3. Poly ethylene glycol (PEG6000)</p><p>SIGMA CAS: 25322-68-3		81255-250G</p><p>4. Dynabeads® Protein A for Immunoprecipitation (DPA beads)</p><p>LIFE Cat: 10001D-100ml</p><p>5. Ethylenediaminetetraacetic acid solution (EDTA)</p><p>SIGMA CAS: 60-00-4		03690-100ml</p><p>6. TritonTM X-100 solution</p><p>SIGMA CAS: 9002-93-1		93443-100ml</p><p>7. Sodium deoxycholate (DOC)</p><p>SIGMA CAS: 302-95-4		d6750-10g</p><p>8. MaXtractTM High Density (PhaseLock tube)</p><p>QIAGEN Cat.No. 129046</p><p>9. phenol:chloroform:isoamyl alcohol</p><p>SOLARBIO Cat.No. P1012</p><p>10. Sodium acetate (NaOAc)</p><p>SIGMA CAS: 127-09-3		v900212-500g</p><p>11. Glycogen</p><p>Roche Product No. 10901393001</p><p>12. Isopropyl alcohol&nbsp;</p><p>CAS: 67-63-0</p><p>13. Ethanol</p><p>CAS: 64-17-5</p><p>14. Protease Inhibitor Cocktail Tablets in EASYpacks (PIC)</p><p>ROCHE Product No. 04693116001</p><p>15. Sodium dodecyl sulfate solution (SDS)</p><p>SIGMA CAS: 151-21-3		71736-100ML</p><p>16. Sodium bicarbonate (NaHCO3)</p><p>SIGMA CAS:144-55-8		v900182-500g</p><p>17. UltraPureTM Distilled Water</p><p>INVITROGEN 10977-015</p><p>18. Magnesium chloride hexahydrate (MgCl2)</p><p>SIGMA CAS:7791-18-6		M2670-100G</p><p>19. Nonidet P40 Substitute (NP40)</p><p>ROCHE Product No. 11332473001</p><p>20. Phenylmethanesulfonyl fluoride (PMSF)</p><p>SIGMA CAS: 329-98-6		p7626-1g</p><p>21. Sodium chloride (NaCl)</p><p>SIGMA CAS: 7647-14-5		V900058-500G</p><p>22. Agencourt AMPure XP 60 mL Kit</p><p>Beckman&nbsp;A63880/A63881</p><p><br></p><h2>Equipment</h2><p>1. DiaMag 0.2ml - magnetic rack</p><p>DiagenodeCat: B04000001</p><p>2. DynaMag™-2 Magnet</p><p>Fischer Scientific Cat: 12321D</p><p>3. DNA mixer&nbsp;Beyotime E1676,DH-II</p><p>4. Centrifuge 5424R (refrigerated) Eppendorf Cat: 5404000090</p><h2>Procedure</h2><p><strong>Binding antibodies to magnatic beads</strong></p><p>1，&nbsp;For each IP , use 10ul stock solution of beads, wash twice the protein A/G coated magnetic beads with ice cold ChIP Buffer. Prepare same amount of beads for each IP if pre-clearing is desired.</p><p>2，&nbsp;After washing, resuepend the beads in ChIP Buffer to the same concentration as stock.</p><p>3，&nbsp;Aliquot 90ul of ChIP Buffer per Ip to a mewly prepared tube</p><p>4，&nbsp;Add 10ul of pre-washed Protein A/G-beads per IP tube made up 100ul total per IP</p><p>5，&nbsp;Add specific antibody and control antibodies. The amount of antibody varies depends on the antibody used, the binding capacity of 10 ul magnetic beads is 3ug</p><p>6，&nbsp;Incubate the tubes at 40rpm on a rotating wheel for at least 2hours at 4℃.</p><p><strong>Chromatin shearing and nuclear membrane solubilization</strong></p><p>1，&nbsp;Seeding cells in 20ul Nuclei Extraction Buffer, invert tubes several times gently, then centrifuge for 5 minutes in 4000rpm in 4℃. Discard 10ul supernatant and keep another 10ul with nucleus.</p><p>2，&nbsp;Prepare MNase Master mix ；</p><p>3，&nbsp;Add 40ul MNase Master Mix to each sample ,mix well by pipetting or gentle vertex</p><p>4，&nbsp;Proceed at room temperature (25℃) for 10min </p><p>5，&nbsp;Add 5.5ul MNase Stop Solution,mix well by gentle vertex</p><p>6，&nbsp;Add 5.5ul Nuclear Break Buffer into the tube , mix well by gentle vertex. These are sheared chromatin ready to be ChIPed</p><p>7，&nbsp;Add Complete ChIP Buffer to make 100ul sheared chromatin for each IP and 10ul for input&nbsp;</p><p><strong>Magnetic immunoprecipitation</strong></p><p>1，&nbsp;Briefly spin the tube containing the antibody-coated beads to bring down liquid caught inside the lid.</p><p>2，&nbsp;Place tubes in the ice-cold magnetic rack , wait for 1min. discard the supernatant. Keep the antibody-coated beads</p><p>3，&nbsp;Add 100ul of diluted sheared chromatin to each IP tube, and keep 10ul of diluted chromatin as input at 4 ℃ or store at -20℃</p><p>4，&nbsp;Invert the tubes several times make sure beads are resuspended</p><p>5，&nbsp;Incubate samples at 4℃ under constant rotation on a rotator at 40 rpm for 2 hours up to overnight</p><p>&nbsp;</p><p><strong>Washes</strong></p><p>1，&nbsp;Brief spin the tubes to bring down the liquid caught on the lid</p><p>2，&nbsp;Place the tubes into the magnetic rack, wait 1 min and discard the buffer</p><p>3，&nbsp;Wash twice using 100ul of ice-cold Low Salt Wash Buffer and twice using High Salt Wash Buffer</p><p>4，&nbsp;Add 100ul Hot Elution Buffer to the beads and 90ul Elution buffer to the input and transfer them to a new tubes</p><p>5，&nbsp;Put the strips in a PCR block at 65 for 1.5-2h</p><p>6，&nbsp;Place the strips on a magnetic rack, let sit for 1min, transfer them to a PhaseLock tube</p><p>&nbsp;</p><p><strong>DNA isolation </strong></p><p>1，&nbsp;Brief spin the PhaseLock tube to bring down the agrose</p><p>2，&nbsp;Add 100ul DNA and equal volume(100ul) phenol:chloroform:isoamyl alcohol into the PhaseLock tube. Mix well by vertex</p><p>3，&nbsp;Spin at 13000g for 5min at room temperature</p><p>4，&nbsp;Transfer the supernatant to a new 1.5Ml tube</p><p>5，&nbsp;Add 1/10 of the vol.NaOAc and LPA</p><p>6，&nbsp;Add 2.5X Vol. ice-cold 100%EtOH. Mix well by vertex</p><p>7，&nbsp;Put the tubes at -20℃ to precipate for 30min up to overnight</p><p>8，&nbsp;Centrifuge DNA at 13000g for 30min at 4℃</p><p>9，&nbsp;Remove the supernatant and add 1ML 70% EtOH , let the tube sit for 5min to allow the salt to dissolve. Then centrifuge at 13000g for 5min at 4℃ to attach the pellet to the bottom.</p><p>10，Remove the EtOH and let the pellet to dry for 5min at room temperature</p><p>11，&nbsp;Add 30ul DNA elution Buffer</p><h2>Troubleshooting</h2><p>1. Step1: If you are in a hurry, Binding antibodies to magnatic beads should be at least 3h;</p><p><br></p><p>2. Regarding antibody selection: a variety of antibodies can be selected for screening</p><h2>Time Taken</h2><p>Day -1: Binding antibodies to magnatic beads</p><p>Day 0: Chromatin shearing and nuclear membrane solubilization and Magnetic immunoprecipitation</p><p>Day2: Washes and DNA isolation</p><h2>Anticipated Results</h2><p>Glycoblue Coprecipitant gathered at the bottom of the tube and the &nbsp;subsequent library was successfully built</p><h2>References</h2><p>1. Zhang, B. et al. Allelic reprogramming of the histone modification H3K4me3 in early mammalian development. Nature 537, 553-557 (2016).</p><p>2. Brind'Amour, J. et al. An ultra-low-input native ChIP-seq protocol for genome-wide profiling of rare cell populations. Nat Commun 6, 6033 (2015).</p><p>3. Liu, X. et al. Distinct features of H3K4me3 and H3K27me3 chromatin domains in pre-implantation embryos. Nature 537, 558-562 (2016).</p><p>4. Bernstein, B.E., et al., The NIH Roadmap Epigenomics Mapping Consortium. Nat Biotechnol, 2010. 28(10): p. 1045-8 (2010).</p><p>5. Roy, S., et al., Identification of functional elements and regulatory circuits by Drosophila modENCODE. Science, 2010. 330(6012): p. 1787-97 (2010).</p><p>6. Kellis, M., et al., Defining functional DNA elements in the human genome. Proc Natl Acad Sci U S A, 2014. 111(17): p. 6131-8 (2014).</p><h2>Acknowledgements</h2><p>We thank Stuart H. Orkin (Harvard Medical School, Department of Pediatric Oncology, Dana Farber Cancer Institute, Children’s Hospital) for generously sharing <em>Ezh2</em><sup><em>fl/fl </em></sup>conditional mice and <em>Eed</em><sup><em>fl/fl</em> </sup>conditional mice. We thank Wei Xie (Tsinghua University, Center for Stem Cell Biology and Regenerative Medicine) for generously sharing PWK/PhJ mice. This study was supported by the National Key Research and Development Program of China (2016YFA0100402) and the National Natural Science Foundation of China (31871504; 31900600; 31601201), the Youth Innovation Promotion Association of the Chinese Academy of Sciences (2017114).</p></body></html>