<html><head><title>Transcriptome-wide m6A detection with DART-Seq</title></head><body><h1>Transcriptome-wide m6A detection with DART-Seq</h1>Authors: Kate D. Meyer<br>Abstract: <p>m<sup>6</sup>A is the most abundant internal mRNA modification and plays diverse roles in gene expression regulation. Much of our current knowledge about m<sup>6</sup>A has been driven by recent advances in the ability to detect this mark transcriptome-wide. Antibody-based approaches have been the method of choice for global m<sup>6</sup>A mapping studies. These methods rely on m<sup>6</sup>A antibodies to immunoprecipitate methylated RNAs, followed by next-generation sequencing to identify m<sup>6</sup>A-containing transcripts<sup>1,2</sup>. While these methods enabled the first identification of m<sup>6</sup>A sites transcriptome-wide and have dramatically improved our ability to study m<sup>6</sup>A, they suffer from several limitations. These include requirements for high amounts of input RNA, costly and time-consuming library preparation, high variability across studies, and m<sup>6</sup>A antibody cross-reactivity with other modifications. Here, we describe DART-Seq (<u>d</u>eamination <u>a</u>djacent to <u>R</u>NA modification <u>t</u>argets), an antibody-free method for global m<sup>6</sup>A detection. In DART-Seq, the C to U deaminating enzyme, APOBEC1, is fused to the m<sup>6</sup>A-binding YTH domain. This fusion protein is then introduced to cellular RNA either through overexpression in cells or with <em>in vitro </em>assays, and subsequent deamination of m<sup>6</sup>A-adjacent cytidines is then detected by RNA sequencing to identify m<sup>6</sup>A sites. DART-Seq can successfully map m<sup>6</sup>A sites throughout the transcriptome using as little as 10 nanograms of total cellular RNA, and it is compatible with any standard RNA-seq library preparation method.</p><br>Keywords: RNA, epitranscriptome, RNA methylation, m6A, methyladenosine, DART-Seq<br><h2>Introduction</h2><p>General notes:</p><p>The DART-Seq approach assumes that APOBEC1-YTH and APOBEC1-YTH<sup>mut</sup> fusion proteins have been cloned into an expression vector appropriate for the application. We have tested mammalian expression under the CMV promoter, but others will likely work as well.&nbsp;</p><p>Prior to starting, pilot studies are recommended to ensure that expression of fusion proteins are not toxic to cells and that adequate expression is achieved.&nbsp;</p><p>Practice good RNA handling technique during library preparation.&nbsp;</p><p>For standard RNA-seq, we use 1 microgram of total cellular RNA for library preparation with the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina (New England Biolabs). We have also prepared libraries from as little as 10 nanograms of total RNA with good results. As with any RNA-seq library preparation, lower amounts of starting RNA may require more sensitive library preparation methods. We have used the Single Cell/Low Input RNA Library Prep Kit (New England Biolabs) for our low-input libraries with good results. Other library preparation methods are also compatible with DART-Seq and can be tested by the user.</p><p>&nbsp;</p><h2>Procedure</h2><p><u>A. Transfect and grow cells</u>&nbsp;</p><p>The protocol described here has been used for adherent HEK293T cells. Adjustments may be needed for other cell types.</p><p>1. Plate 10-cm plates (or other appropriate size) of HEK293T cells by diluting approximately 1:8 and plating.</p><p>2. Approximately 12-24 hours later, transfect cells with APOBEC1-YTH fusion proteins using desired method (we have had good success with Fugene HD).</p><p>3. Grow cells for 24h at 37Â°C (5% CO2) to ~70-80% confluency. For HEK293T cells we use Dulbecco's Modified Eagle's Medium (DMEM; Corning) with 10% FBS and 1% Penicillin-Streptomycin (100X; Thermo Fisher). We observe good C to U editing with no cell death after 24h, although shorter/longer time points may be desired and can be tested by the user.</p><p>Note: We typically have at least 2-3 biological replicates each for cells expressing: APOBEC1-YTH, APOBEC1-YTH<sup>mut</sup>, and APOBEC1 alone.</p><p>&nbsp;</p><p><u>B. Harvest RNA </u></p><p>1. Total RNA is harvested using the RNeasy Plus Mini kit and using either on-column DNase treatment or post-purification DNase treatment (RNase-free DNaseI, New England Biolabs). Alternatively, RNA can be isolated with Trizol followed by digestion with RNase-free DNase.</p><p>&nbsp;2. Quantify RNA and assess RNA integrity by running on a gel or with a Bioanalyzer.</p><p>&nbsp;</p><p><u>C. Next-generation sequencing</u></p><p>A variety of options are available for sequencing library preparation. We use the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina (New England Biolabs) for 1ug total RNA as starting material or the Single Cell/Low Input RNA Library Prep Kit (New England Biolabs) for samples using &lt; 100ng total RNA as starting material.</p><p>Sequencing can be carried out using paired-end or single-read sequencing. We typically use SR 50bp sequencing on an Illumina HiSeq4000 or similar machine. We achieve 30-50 million reads per sample using this platform.</p><p>&nbsp;</p><p><u>D. Identify C to U editing events and m</u><sup><u>6</u></sup><u>A sites</u></p><p>1. Demultiplex sequencing reads and remove adapter sequences. These steps will depend on the sequencing library prep method and barcode method used. For adapter removal, we use FLEXBAR<sup>3</sup>.</p><p>2. Collapse exact duplicates using the fastq2collapse command in the CLIP Tool Kit (CTK) suite<sup>4</sup>. Several processing steps utilize various CTK commands; an excellent web resource for these tools can be found here: <a href="https://zhanglab.c2b2.columbia.edu/index.php/CTK_Documentation" target="_blank">https://zhanglab.c2b2.columbia.edu/index.php/CTK_Documentation</a></p><p>3. Align reads to the genome. We use Novoalign with the options below or BWA according to the current recommendations of the CTK developers (see the link above).</p><p>novoalign -t 85 -d file.nix -f file.fastq -l 16 -s 1 -o SAM -r None &gt; file.sam</p><p>4. Remove duplicate reads and prepare file for parsing:</p><p>samtools sort file.sam -O BAM -o file.sorted.bam</p><p>novosort --markduplicates --keeptags&nbsp;file.sorted.bam -i -o file.uniq.bam</p><p>samtools fillmd file.novo.uniq.bam file.fa | gzip -c &gt; file.sorted.md.sam.gz</p><p>BAM files can be merged using samtools merge or loaded individually into the genome browser of choice (we use IGV). This enables viewing of mutations at individual sites throughout the transcriptome.</p><p>&nbsp;</p><p>5. Parse SAM file using the CTK program:</p><p>parseAlignment.pl -v --map-qual 1 --min-len 18 --mutation-file file.mutation.txt file.sorted.md.sam.gz - | gzip -c &gt; file.tag.uniq.bed.gz</p><p><br></p><p>6. Collapse PCR duplicates with CTK:</p><p>tag2collapse.pl -v -big -weight --weight-in-name --keep-max-score --keep-tag-name file.tag.bed file.tag.uniq.bed</p><p>&nbsp;</p><p>7. Get mutations in unique tags using CTK:</p><p>joinWrapper.py file.mutation.txt file.tag.uniq.bed 4 4 N file.tag.uniq.mutation.txt</p><p>&nbsp;</p><p>8. Merge replicates:</p><p>cat file1.tag.uniq.bed file2.tag.uniq.bed file3.tag.uniq.bed &gt; file.tag.uniq.bed</p><p>cat file1.tag.uniq.mutation.txt file2.tag.uniq.mutation.txt file3.tag.uniq.mutation.txt &gt; file.tag.uniq.mutation.txt</p><p>&nbsp;</p><p>9. Find C to T transitions</p><p>To isolate C to T transitions, we use a script from the miCLIP method<sup>5</sup>, which also identifies C to T mutations adjacent to m<sup>6</sup>A sites:</p><p>awk '{if($6=="+" &amp;&amp; $8=="C" &amp;&amp; $9=="&gt;" &amp;&amp; $10=="T"&nbsp;||&nbsp;&nbsp;$6=="-" &amp;&amp; $8=="G" &amp;&amp; $9=="&gt;" &amp;&amp; $10=="A") {print $0}}' file.tag.uniq.mutation.txt | cut -f 1-6 &gt; file.tag.uniq.C2T.bed</p><p>&nbsp;</p><p>10. Run CIMS with CTK:</p><p>CIMS.pl -big -v -n 5 -p -c cache_dir file.tag.uniq.bed file.tag.uniq.C2T.bed file.tag.uniq.C2T.CIMS.txt</p><p>&nbsp;</p><p>11. Identify CIMS with desired FDR:</p><p>awk "{if(\$9&lt;=1)&nbsp;{print \$1\"\t\"\$2\"\t\"\$3\"\t$1_\"\$4\"_\"\$9\"\t\"\$5\"\t\"\$6}}" file.tag.uniq.C2T.CIMS.txt | sort -k 9,9n -k 8,8nr -k 7,7n | cut -f 1-6 &gt; file.tag.uniq.C2T.CIMS.p1.bed</p><p>&nbsp;</p><p>12. Filter C to T sites:</p><p>This step can be varied to achieve the desired stringency of site detection. Our standard protocol is to take C to T sites identified with the p&lt;1 threshold (Step 11) and keep only those with a minimum of 2 mutations, at least 10 reads per replicate, and a mutation/read (m/k) threshold of 10-60%. We find that adjusting the number of mutations, reads per replicate, and m/k threshold is a good way to increase/decrease stringency of m<sup>6</sup>A site calls to a desired level. In addition to these filtering steps, known mutations in the genome from the dbSNP database (https://www.ncbi.nlm.nih.gov/snp/), as well as endogenous C to U editing sites identified by sequencing of wild type cells, are also removed. Lastly, we recommend removing sites identified from cells expressing APOBEC1 alone. These filtering steps can all be completed using bedtools intersect from the bedtools suite (https://bedtools.readthedocs.io/en/latest/).</p><p>&nbsp;</p><p>13. Calculate enrichment over APOBEC1-YTH<sup>mut</sup>-expressing cells.</p><p>To further ensure high-confidence identification of m<sup>6</sup>A sites, we filter C to U editing sites to include only those with a minimum threshold of mutations per read compared to sites obtained by expression of APOBEC1-YTH<sup>mut</sup> :</p><p><br></p><p>A. First, use bedtools intersect to find C to T transitions that are present only in APOBEC1-YTH samples. Ensure that the YTH and mut .bed files you use have mutations/read (m/k) in column 5. These will be merged back in later.</p><p>bedtools intersect -s -v -a yth.filt.bed -b mut.bed &gt; ythnotmut.filt.bed</p><p><br></p><p>B. Next, merge the YTH and mut files for common sites.</p><p>bedtools intersect -s -wa -wb -a yth.filt.bed -b mut.bed &gt; yth.filt.mergemut.bed</p><p><br></p><p>C. Then use awk to identify sites that have a m/k ratio that is 1.5-fold greater than in APOBEC1-YTH<sup>mut</sup> samples:</p><p>awk '{a=$5/$11;print $0,a;}' yth.filt.mergemut.bed | awk '$13 &gt;= 1.5 {print $1"\t"$2"\t"$3"\t"$13"\t"$5"\t"$6}' &gt; yth.filt.mk1.5overmut.bed</p><p>The 1.5-fold cutoff can be adjusted by the user to be more/less stringent as desired.</p><p><br></p><p>D. Now, add back the sites in YTH that were not in mut:</p><p>cat yth.filt.mk1.5overmut.bed ythnotmut.filt.bed &gt; yth.mk1.5overmut.final.bed</p><p><br></p><p>&nbsp;</p><p>We find that the above filters are a good starting point for identifying m<sup>6</sup>A sites with DART-Seq. Further filtering steps can also be implemented (e.g., limiting to sites immediately following an A; limiting to sites in a DRACH motif, excluding sites lost in methyltransferase-depleted cells, etc.).</p><h2>References</h2><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Dominissini, D.<em> et al.</em> Topology of the human and mouse m6A RNA methylomes revealed by m6A-seq. <em>Nature</em> <strong>485</strong>, 201-6 (2012).</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Meyer, K.D.<em> et al.</em> Comprehensive analysis of mRNA methylation reveals enrichment in 3' UTRs and near stop codons. <em>Cell</em> <strong>149</strong>, 1635-46 (2012).</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Dodt, M., Roehr, J.T., Ahmed, R. &amp; Dieterich, C. FLEXBAR-Flexible Barcode and Adapter Processing for Next-Generation Sequencing Platforms. <em>Biology (Basel)</em> <strong>1</strong>, 895-905 (2012).</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Shah, A., Qian, Y., Weyn-Vanhentenryck, S.M. &amp; Zhang, C. CLIP Tool Kit (CTK): a flexible and robust pipeline to analyze CLIP sequencing data. <em>Bioinformatics</em> <strong>33</strong>, 566-567 (2017).</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Linder, B.<em> et al.</em> Single-nucleotide-resolution mapping of m6A and m6Am throughout the transcriptome. <em>Nat Methods</em> (2015).</p><p>&nbsp;</p></body></html>