<html><head><title>Kinetic characterization of inhibitors of histone deacetylases (HDACs) and sirtuins</title></head><body><h1>Kinetic characterization of inhibitors of histone deacetylases (HDACs) and sirtuins</h1>Authors: Carlos Moreno-Yruela, Andreas Stahl Madsen, Christian A. Olsen<br>Abstract: <p>Histone deacetylase (HDAC) inhibitors are employed for the treatment of lymphoma and are under development against multiple other types of cancer and neurodegenerative diseases. Here, we describe a robust and uncomplicated in vitro assay for HDAC inhibitor kinetic profiling. Enzyme and fluorogenic peptide substrate are incubated together with a small amount of protease “assay developer”, which enables continuous recording of substrate conversion under steady-state conditions. Assay progression curves upon addition of an inhibitors at varying concentrations permit determination of kinetic constants and overall inhibitor potency. This assay helped&nbsp;provide new insight into the kinetic properties of known HDAC inhibitors as well as the kinetic characterization of both inhibitors and substrates of sirtuin enzymes, which are class III HDACs involved in metabolic control and oncogene regulation.</p><br>Keywords: histone deacetylases, sirtuins, post-translational modifications, enzyme kinetics, continuous assays, enzymology<br><h2>Introduction</h2><p>Histone deacetylase (HDAC) inhibitors are used clinically for the treatment of lymphoma and are under development for treatment of multiple other types of cancer and neurodegenerative diseases.<sup>1 </sup>Inhibitor potency and overall biological effect are often determined by the duration of the inhibitor–enzyme binding event.<sup>2-4 </sup>In fact, approved anti-cancer drugs such as panobinostat, chidamide, and romidepsin are potent HDAC inhibitors that display extended enzyme residence time.<sup>5-7 </sup>However, standard HDAC assays disregard the dynamic character of the inhibitor–enzyme interaction and do not provide kinetic information.<sup>8 </sup>Here, we describe a robust and uncomplicated high-throughput&nbsp;<em>in vitro </em>assay for HDAC inhibitor kinetic profiling. This assay helped identify previously unreported non-covalent irreversible properties of known HDAC inhibitors.<sup>6,7 </sup>Moreover, this assay format has also been adapted for the kinetic characterization of both inhibitors and substrates of the sirtuins,<sup>9,10 </sup>which are class III HDACs involved in metabolic control and oncogene regulation.<sup>11</sup></p><p>To characterize inhibition and evaluate potential time-dependent changes in inhibition, deacylase enzyme activity is recorded in a continuous manner with the use of&nbsp;peptide substrates containing a C-terminal ε-<em>N</em>-acyllysine residue coupled to the 7-amino-4-methylcoumarin (AMC) fluorophore.<sup>12 </sup>Substrate, enzyme, trypsin and inhibitor (and NAD<sup>+</sup>co-substrate for sirtuins) are incubated for a set amount of time. During that period, the substrate becomes deacylated in relation to the residual activity of the enzyme, thereby presenting a free C-terminal lysine residue. Trypsin recognizes the non-acylated lysine residue and cleaves the C-terminal amide bond to release AMC. The fluorescence of AMC dramatically increase upon release, and light emission is used to quantify substrate converted over time. Measuring deacylase inhibition in a continuous manner requires instant fluorophore release. This is achieved with a small but sufficient amount of trypsin, which processes deacylated substrate without compromising enzyme activity.&nbsp;Assay progression curves upon addition of an inhibitor permit determination of kinetic constants of binding and release, enzyme residence time, and overall inhibitor potency.<sup>6,13</sup></p><h2>Reagents</h2><p>Choice of assay buffer (see&nbsp;<em>Troubleshooting </em>section):</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;HEPES buffer pH 7.4: 50 mM HEPES/Na (Sigma-Aldrich, HEPES cat. #H4034 and HEPES sodium salt cat. #H3784), 100 mM KCl&nbsp;(Sigma-Aldrich, cat. #P9541), 0.001% (v/v) tween-20 (Sigma-Aldrich, cat. #93773), 0.2 mM tris(2-carboxyethyl)phosphine (TCEP, Sigma-Aldrich, cat. #C4706), 0.5 mg/mL bovine serum albumin&nbsp;(BSA, Sigma-Aldrich, cat. #A7030).</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Tris buffer pH 8.0: 50 mM Tris/Cl (Sigma-Aldrich, Trizma base cat. #T1503 and Trizma hydrochloride cat. #T5941), 137 mM NaCl (Sigma-Aldrich, cat. #71380), 2.7 mM KCl (Sigma-Aldrich, cat. #P9541), 1 mM MgCl<sub>2</sub>(VWR, cat. #25108), 0.5 mg/mL bovine serum albumin (BSA, Sigma-Aldrich, cat. #A7030).</p><p><br></p><p>Enzyme substrates:</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;7-amino-4-methylcoumarin (AMC)-conjugated tri- and tetrapeptides&nbsp;including a C-terminal ε-<em>N</em>-acyllysine residueare inexpensive and easily accessible via solid-phase peptide synthesis and simple chemical reactions.<sup>9,12,14,15 </sup>Standard substrates are Ac-Leu-Gly-Lys(Ac)-AMC (LGKac, for HDACs 1, 2, 3 and 6),<sup>6,7 </sup>Ac-Leu-Gly-Lys(Tfa)-AMC (LGKtfa, for HDACs 4, 7, and 8),<sup>16 </sup>Ac-Arg-His-Lys(Ac)-Lys(Ac)-AMC (RHKacKac, for HDAC8),<sup>6,16 </sup>Ac-Glu-Thr-Asp-Lys(Myr)-AMC (ETDKmyr, for HDAC11 and SIRT2),<sup>9,17 </sup>Ac-Glu-Thr-Asp-Lys(Ac)-AMC (ETDKac, for SIRT2),<sup>9 </sup>and Ac-Leu-Gly-Lys(Suc)-AMC or Ac-Leu-Gly-Lys(Glu)-AMC (LGKsuc or LGKglu, for SIRT5).<sup>10</sup></p><p><br></p><p>Recombinant enzymes:</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;HDACs and sirtuins can be purchased from BPS Bioscience (San Diego, CA): HDAC1 (cat. #50051), HDAC2 (cat. #50002), HDAC3/NCoR2 (cat. #50003), HDAC4 (cat. #50004), HDAC6 (cat. #50056), HDAC7 (cat. #50007), HDAC8 (cat. #50008), HDAC11 (cat. #50021), SIRT2 (cat. #50013), SIRT5 (cat. #50016); from Millipore (Temecula, CA, USA): HDAC4 (cat. #14-828), HDAC7 (cat. #14-832); or from Enzo Life Sciences (Farmingdale, NY, USA): SIRT5 (cat. #BML-SE555).</p><p><br></p><p>Others:</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Trypsin (Sigma-Aldrich, cat. #T1426).</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;β-Nicotinamide adenine dinucleotide hydrate (NAD<sup>+</sup>, Sigma-Aldrich, cat.&nbsp;#N7004).</p><h2>Equipment</h2><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Dilution plates: transparent conical bottom microtest 96-well plates (Sarstedt AG, cat. #82.1583).</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Assay plates: black Corning™ 96-well half-area plates (Fischer Scientific, cat. #3686).</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Plate reader with excitation at 360 nm and emission at 460 nm (e.g., FLUOstar Omega Microplate Reader, BMG Labtech, Germany).</p><h2>Procedure</h2><p>Experiments are performed in 96-well half-area plates, containing 50 µL final assay volume. Inhibitor dilutions are performed in transparent conical bottom plates prior to transfer to assay plates, while substrate and enzyme stock solutions are prepared in plastic microcentrifuge tubes. All compound stocks solutions and dilution series are prepared in assay buffer from concentrated DMSO solutions, so that final DMSO assay concentration does not exceed 1%. For continuous assays, substrate, NAD<sup>+ </sup>co-substrate (for sirtuin assays), inhibitor, and trypsin are added to the assay plate, followed by addition of enzyme and immediate readout at room temperature every 30 s until the assay is completed. All experiments are performed as internal duplicates, and control wells without inhibitor and without enzyme or inhibitor (background) are included.</p><p><br></p><p>Determination of continuous assay conditions:</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;For a set concentration of substrate higher than the&nbsp;<em>K</em><sub>M </sub>(ideally 3 times higher, use 200 µM substrate concentration if&nbsp;<em>K</em><sub>M </sub>unknown), perform time course experiments at varying enzyme and trypsin concentration until linear progression curves are obtained with appropriate signal-to-noise ratio (optimally, rates of 3–5% of the uninhibited reaction should be discernable from baseline). Fine-tune enzyme concentration if necessary.</p><p><em>Example: incubate enzyme at 1000, 100, 10 and 1 nM concentration with trypsin (16, 8, 4 and 2 µg/mL), substrate (200 µM), and NAD</em><sup><em>+ </em></sup><em>(500 µM; only for sirtuins) for 60 min with fluorescence reads every 30 s.</em></p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;For the preferred concentration of enzyme, screen concentrations of trypsin to find the lowest possible concentration at which the conversion rate is kept as high as possible. This ensures that AMC release by trypsin is not the rate-limiting step, while minimizing the effect of trypsin on enzyme stability. Assay progression curves should be linear for at least 30 min (see&nbsp;<em>Troubleshooting </em>section).</p><p><em>Example: incubate enzyme (10 nM), substrate (200 µM), and NAD</em><sup><em>+</em></sup><em>(500 µM; only for sirtuins) with trypsin at 8, 4, 2, 1, 0.5 and 0.25 µg/mL concentration for 90 min with fluorescence reads every 30 s.</em></p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Optional: determine the Michaelis-Menten constant (<em>K</em><sub>M</sub>) of the substrate. This will be useful for ensuring that substrate concentration is appropriate for maintaining steady-state conversion, and for determining inhibitor&nbsp;<em>K</em><sub>i </sub>values from the obtained IC<sub>50</sub>.</p><p><em>Example: incubate enzyme (10 nM), NAD</em><sup><em>+</em></sup><em>(500 µM; only for sirtuins), and trypsin (4 µg/mL) with substrate at 195, 130, 87, 58, 38, 26, 17, 11, 7.6, 5.1, 3.4 and 2.3 µM concentration (1.5-fold dilution) for 60‒90 min with fluorescence reads every 30 s, measure initial conversion rates and fit to the Michaelis-Menten equation to determine the substrate K</em><sub><em>M</em></sub><em>.</em></p><p>&nbsp;</p><p>Continuous inhibition assays:</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Estimate the inhibitor potency (IC<sub>50,est</sub>) with a small number of end-point or continuous assays.</p><p><em>Example (end-point): incubate enzyme and substrate (and NAD</em><sup><em>+</em></sup><em>) with inhibitor at 10, 1, 0.1 and 0.01 µM concentration for 30 min, add trypsin developer (200 µg/mL) and read after 10 min assay development. Estimate the inhibitor IC</em><sub><em>50 </em></sub><em>by comparing to the control without inhibitor.</em></p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Perform dose-response continuous inhibition assays (2-fold dilutions) with the preferred concentrations of enzyme and trypsin (and NAD<sup>+</sup>) starting at 20 times the IC<sub>50,est</sub>.</p><p><em>Example: for IC</em><sub><em>50,est </em></sub><em>= 1 nM, incubate enzyme (10 nM) and substrate (200 µM (and 500 µM NAD</em><sup><em>+</em></sup><em>)) with inhibitor at 20, 10, 5, 2.5, 1.25, 0.62, 0.31, 0.16, 0.078, and 0.039 nM concentration for 90 min with fluorescence reads every 30 s, or until the progression curve from control wells without inhibitor deviate from a straight line.</em></p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Visually assess final substrate conversion at each concentration of inhibitor, and repeat experiments with alternative inhibitor concentrations until covering the range from no inhibition to full inhibition.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Perform final experiments twice in order to report data as average&nbsp;± SD of two individual experiments.</p><p><br></p><p>Data processing and fitting (instructions for GraphPad Prism 7.0):</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Paste data in Prism as two replicate values in side-by-side sub columns, and enter time points as “X” values. Place baseline data in “Group A”, and label all other groups with the concentration of inhibitor employed.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Optional: transform data from fluorescence units to concentration of AMC. A standard AMC concentration curve must be obtained for the plate reader employed, which can be then included in Prism under “Transform” – “User-defined Y functions”.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Use “Remove Baseline and Column Math” to remove baseline fluorescence (Group A) from all data sets.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Plot data from duplicate wells as discrete data points, in order to identify outliers and experimental errors (“New Graph of Existing Data…”, with the option “Create a new graph for each data set”). Data with low signal-to-noise ratio (for example, due to full inhibition of the enzyme) may be discarded, especially in the case of slow-binding inhibitors.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Use control data without inhibitor to determine the time frame of analysis in which the progression curve follows a straight line. Discard all data outside of this time frame.</p><p><em>Example: discard data from 0 to 3 min if steady-state conversion is not reached and all data after 60 min if the conversion rate starts to decrease.</em></p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Plot all remaining data in a single figure, as mean values with error bars for each concentration of inhibitor and assess the shape of each progression curve. If all experiments follow straight lines (<u>fast-on/fast-off kinetics</u>), go to <strong>point 7</strong>, otherwise continue from <strong>point 8</strong> (<u>slow-binding kinetics</u>).</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;<u>Fast-on/fast-off kinetics</u>: fit each progression curve to a straight line (“Analyze” – “Nonlinear regression” – “Straight line”) and select “Create summary table and graph” to obtain a XY type of graph with the Slopes and the concentrations of inhibitor. Transform inhibitor concentrations into logarithms, and use&nbsp;<strong>Eq. 1&nbsp;</strong>(“Analyze” – “Nonlinear regression” – “log(inhibitor) vs. response – Variable slope”) to calculate the IC<sub>50 </sub>of the inhibitor. IC<sub>50 </sub>values can be transformed into&nbsp;<em>K</em><sub>i </sub>values using the Cheng-Prusoff equation (<strong>Eq. 2</strong>).</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;<u>Slow-binding kinetics</u>: fit control data without inhibitor to a straight line, and all other progression curves to&nbsp;<strong>Eq. 3</strong>, which takes into account initial and final conversion rates (<em>V</em><sub>in</sub><em> </em>and&nbsp;<em>V</em><sub>ss</sub>, respectively) and provides the observed rate at which the equilibrium is reached (<em>k</em><sub>obs</sub>). This equation can be added to Prism under&nbsp;“Analyze” – “Nonlinear regression”in the form: Y=(Vss*X)+((Vin-Vss)*(1-exp(-kobs*X))/kobs)+Po, with <em>V</em><sub>ss</sub>, <em>V</em><sub>in</sub>, and <em>k</em><sub>obs</sub> constrained to be greater than 0. Create a XY summary table with&nbsp;<em>k</em><sub>obs</sub>&nbsp;data and the concentration of inhibitor, and exclude conditions for which the standard error exceeds the mean&nbsp;<em>k</em><sub>obs</sub>&nbsp;value.</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;Collect&nbsp;<em>k</em><sub>obs</sub>&nbsp;data from two individual experiments, create a data sheet with both data sets, and fit&nbsp;<em>k</em><sub>obs </sub>data (“Analyze” – “Nonlinear regression”)&nbsp;to either&nbsp;<strong>Eq. 4 </strong>(linear correlation, “Straight line”) or&nbsp;<strong>Eq. 5</strong>(hyperbolic correlation, added as Y = kminus2+k2*X/(X+Ki1*(1+S/Km)), with&nbsp;<em>K</em><sub>M </sub>constrained to be the value calculated before).&nbsp;<strong>Eq. 4&nbsp;</strong>corresponds to&nbsp;<u>mechanism A</u> of slow-binding kinetics, in which there is a single slow binding step (E + I&nbsp;<strong>⇌</strong>EI) with kinetic constants&nbsp;<em>k</em><sub>1</sub>&nbsp;(direct) and&nbsp;<em>k</em><sub>–1</sub>&nbsp;(inverse).&nbsp;<strong>Eq. 5&nbsp;</strong>corresponds to&nbsp;<u>mechanism B</u> of slow-binding kinetics, where there is a fast binding step followed by a slow transition between enzyme-inhibitor complexes (E + I&nbsp;<strong>⇌</strong>EI&nbsp;<strong>⇌</strong>EI*). Mechanism B is characterized by the&nbsp;<em>K</em><sub>i,1 </sub>equilibrium constant for the first step and kinetic constants&nbsp;<em>k</em><sub>2</sub>&nbsp;and&nbsp;<em>k</em><sub>–2</sub>&nbsp;for the second step.<sup>6,13</sup>Summary tables provide values for all constants when possible.</p><p>10.&nbsp;&nbsp;Calculate the potency of the inhibitor (<em>K</em><sub>i</sub>) using&nbsp;<strong>Eq. 6 </strong>for inhibitors following mechanism A and&nbsp;<strong>Eq. 7&nbsp;</strong>for mechanism B. This will not be possible for irreversible inhibitors (<em>k</em><sub>–1</sub>&nbsp;or&nbsp;<em>k</em><sub>–2</sub>&nbsp;constants close to 0).<sup>6</sup></p><p>11. Calculate dissociative half-life values (t<sub>1/2</sub><sup>diss</sup>) using&nbsp;<strong>Eq. 8&nbsp;</strong>for inhibitors following mechanism A and&nbsp;<strong>Eq. 9 </strong>for mechanism B.<sup>2,18 </sup>Since&nbsp;<em>k</em><sub>–1</sub>&nbsp;constants are not elucidated for mechanism B-type of inhibitors, the limiting scenario in which&nbsp;<em>k</em><sub>–1</sub> is much larger than&nbsp;<em>k</em><sub>–2</sub>&nbsp;may be considered, which provides the lowest&nbsp;t<sub>1/2</sub><sup>diss</sup>&nbsp;possible.</p><h2>Troubleshooting</h2><p><br></p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Experiments including substrates with long aliphatic acyl modifications of lysine (e.g., ETDKmyr) may require reduced BSA concentration (0.05 mg/mL) in order to avoid interference with substrate availability.<sup>9,17</sup></p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Lack of enzyme stability leading to linear conversion periods shorter than 30 min may be solved by modifications of the assay buffer. Consider testing Tris, HEPES, and phosphate buffers at different pH, and adding Tween-20 or polyethylene glycol (PEG) surfactants.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Trypsin can affect the enzyme construct employed. Consider verifying enzyme integrity by gel electrophoresis after incubation with the concentration of trypsin selected for the assay.<sup>7</sup></p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Preliminary slow-binding inhibitor data might not be sufficient to fit&nbsp;accurately to a kinetic model. In this case, design experiments in order to cover the most sensitive part of the curve fit with more inhibitor concentrations.</p><h2>Time Taken</h2><p>Assay setup: 1‒2 h</p><p>Data acquisition: 1‒2 h</p><p>Data processing and fitting: 2‒4 h</p><h2>Anticipated Results</h2><p>Fast-on/fast-off inhibitor conversion rate data fit to a sigmoidal plot with Hill slope close to 1 and each progression curve fits a straight line with high confidence.</p><p>Slow-binding inhibitor <em>k</em><sub>obs </sub>data provides approximate measures of equilibrium and kinetic constants, and a clear fit to either mechanism A or mechanism B of binding. Assay progression curves relative to inhibitor concentrations close to the IC<sub>50 </sub>clearly deviate from a straight line while control experiments without inhibitor retain linearity.</p><h2>References</h2><p>1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Falkenberg, K. J. &amp; Johnstone, R. W. Histone deacetylases and their inhibitors in cancer, neurological diseases and immune disorders.&nbsp;<em>Nat. Rev. Drug Discov. </em><strong>13</strong>, 673-691 (2014).</p><p>2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Copeland, R. A., Pompliano, D. L. &amp; Meek, T. D. Drug–target residence time and its implications for lead optimization.&nbsp;<em>Nat. Rev. Drug Discov. </em><strong>5</strong>, 730-739 (2006).</p><p>3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Copeland, R. A. The drug–target residence time model: a 10-year retrospective.&nbsp;<em>Nat. Rev. Drug Discov. </em><strong>15</strong>, 87-95 (2016).</p><p>4&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Holdgate, G. A., Meek, T. D. &amp; Grimley, R. L. Mechanistic enzymology in drug discovery: a fresh perspective.&nbsp;<em>Nat. Rev. Drug Discov. </em><strong>17</strong>, 115-132 (2018).</p><p>5&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Lauffer, B. E.<em>et al. </em>Histone deacetylase (HDAC) inhibitor kinetic rate constants correlate with cellular histone acetylation but not transcription and cell viability.&nbsp;<em>J. Biol. Chem. </em><strong>288</strong>, 26926-26943 (2013).</p><p>6&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Kitir, B. l.<em>et al. </em>Chemical editing of macrocyclic natural products and kinetic profiling reveal slow, tight-binding histone deacetylase inhibitors with picomolar affinities.&nbsp;<em>Biochemistry </em><strong>56</strong>, 5134-5146 (2017).</p><p>7&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Moreno-Yruela, C.<em>et al. </em>Kinetic Tuning of HDAC Inhibitors Affords Potent Inducers of Progranulin Expression.&nbsp;<em>ACS Chem. Neurosci.</em>, doi:10.1021/acschemneuro.9b00281 (2019).</p><p>8&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Meyer‐Almes, F. J. Determination of the binding mechanism of histone deacetylase inhibitors.&nbsp;<em>Chem. Biol. Drug Des. </em><strong>93</strong>, 1214-1250 (2019).</p><p>9&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Galleano, I., Schiedel, M., Jung, M., Madsen, A. S. &amp; Olsen, C. A. A continuous, fluorogenic sirtuin 2 deacylase assay: Substrate screening and inhibitor evaluation.&nbsp;<em>J. Med. Chem. </em><strong>59</strong>, 1021-1031 (2016).</p><p>10&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Rajabi, N.<em>et al. </em>Mechanism‐based inhibitors of the human sirtuin 5 deacylase: structure–activity relationship, biostructural, and kinetic insight.&nbsp;<em>Angew. Chem. Int. Ed. </em><strong>56</strong>, 14836-14841 (2017).</p><p>11&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Carrico, C., Meyer, J. G., He, W., Gibson, B. W. &amp; Verdin, E. The mitochondrial acylome emerges: proteomics, regulation by sirtuins, and metabolic and disease implications.&nbsp;<em>Cell Metab. </em><strong>27</strong>, 497-512 (2018).</p><p>12&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Wegener, D., Wirsching, F., Riester, D. &amp; Schwienhorst, A. A fluorogenic histone deacetylase assay well suited for high-throughput activity screening.&nbsp;<em>Chem. Biol. </em><strong>10</strong>, 61-68 (2003).</p><p>13&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Chou, C. J., Herman, D. &amp; Gottesfeld, J. M. Pimelic diphenylamide 106 is a slow, tight-binding inhibitor of class I histone deacetylases.&nbsp;<em>J. Biol. Chem. </em><strong>283</strong>, 35402-35409 (2008).</p><p>14&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Madsen, A. S. &amp; Olsen, C. A. Substrates for efficient fluorometric screening employing the NAD-dependent sirtuin 5 lysine deacylase (KDAC) enzyme.&nbsp;<em>J. Med. Chem. </em><strong>55</strong>, 5582-5590 (2012).</p><p>15&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Galleano, I., Nielsen, J., Madsen, A. S. &amp; Olsen, C. A. Scalable and purification-free synthesis of a myristoylated fluorogenic sirtuin substrate.&nbsp;<em>Synlett </em><strong>28</strong>, 2169-2173 (2017).</p><h2>Acknowledgements</h2><p>Members of the Olsen Lab, past and present, who have used and help finetune this protocol.</p></body></html>