<html><head><title>Direct-seq: employing programmed gRNA scaffold for streamlined scRNA-seq in CRISPR screen</title></head><body><h1>Direct-seq: employing programmed gRNA scaffold for streamlined scRNA-seq in CRISPR screen</h1>Authors: Lixia Wang, Lijia Ma<br>Abstract: <p>This protocol aimed to combine the single cell RNA-seq (scRNA-seq) with the CRISPR screening assay based on 10x&nbsp;3’&nbsp;&nbsp;RNA-seq platform to profile the transcriptome together with genotype at single cell resolution. To achieve this goal, an A/G mixed capture sequence and tRNA sequence are incorporated into the Tetraloop, Loop2 and Tail of the&nbsp;gRNA scaffold. Therefore, cDNA derived from the gRNA transcripts can be captured and barcoded by poly(dT) primer together with the endogenous mRNA and enriched by the nest PCR. With this method, the CRISPR perturbation and its transcriptional readouts can be profiled together in a streamlined workflow. <strong>This method, which we named as Direct-seq, enabled the direct genotyping and expression profiling of the CRISPR screening products at single cell resolution on versatile scRNA-seq platforms across different throughput, including 10x 3’&nbsp;and 5’&nbsp;RNA-seq, Fluidigim C1, SMART-seq, BGI DNBelab C4 and others. </strong></p><br>Keywords: programmed gRNA scaffold; CRISPR screening; single cell RNA-seq<br><h2>Introduction</h2><p>CRISPR-based genome perturbation provides a new avenue to conveniently change DNA sequences, transcription and epigenetic modifications in genetic screens. However, it’s also a big challenge to profile the transcriptome together with genotype at single cell resolution.</p><p>To overcome this challenge, two optimizations are applied to the gRNA scaffold design without negative impact to the genome editing performance. One modification is that an A/G mixed capture sequence (a 30-nt sequence, in which one guanine was incorporated with every seven adenosine except for the first eight leading adenosine, "AAAAAAAAGAAAAAAAGAAAAAAAGAAAAA") was introduced into the Tetraloop, Loop2 or Tail of the gRNA scaffold to mimic the poly(A) tail of transcripts from polymerase II (Pol II). Therefore,&nbsp;gRNA transcripts&nbsp;can be captured and barcoded together with the endogenous mRNA via oligo(dT) from 10x GEM beads. The other modification is that a tRNA sequence, which locates at the downstream of the U6 promoter and upstream of the gRNA sequence, was included into the gRNA expression cassette to achieve multiplexed&nbsp;gRNA expression&nbsp;and also provided specific primer binding sites for the nest PCR.</p><p>The following protocol is used to profile the index gRNAs and&nbsp;endogenous mRNAs simultaneously at single cell resolution based on&nbsp;10x 3’&nbsp;single cell&nbsp;RNA-seq platform. Single cell capture and mRNA library are prepared following the Chromium User Guide&nbsp;(10x Genomics, CG000184). The specific enrichment of&nbsp;cDNA from gRNA transcripts and the gRNA library construction are described in details and other&nbsp;alterations are also indicated in this protocol.</p><p>For application of the Octo-seq workflow in other single cell RNA-seq platforms, please refer to the Figure1.</p><h2>Reagents</h2><p>Chromium Single Cell 3’&nbsp;Reagent Kits v3 (10x Genomics, PN-1000075)</p><p>Chromium Single Cell B Chip kit (10x Genomics, PN-1000153)</p><p>Chromium i7 Multiplex Kit (10x Genomics, PN-120262)</p><p>Trypan blue stain (0.4%) (Gibco, 15250-061)</p><p>DPBS (Sigma, D8537-500ML)</p><p>Glycerol (Sigma, G5516-1L)</p><p>Tween 20 (Sigma, P7949-500ML)</p><p>UltraPure Distilled Water (Thermo Fisher, 10977-015)</p><p>Buffer EB (Qiagen, 19086)</p><p>Absolute Ethanol (Fisher Scientific, BP2818-4)</p><p>Beckman Coulter Ampure XP 450mL (Fisher Scientific, A63882)</p><p>NEBNext®&nbsp;Ultra™&nbsp;II Q5®&nbsp;Master Mix (New England BioLabs, M0544S)</p><p>HS NGS Fragment (1-6000bp) (Agilent Technologies, 5191-6578)</p><p>Qubit dsDNA HS Assay Kit (Thermo Fisher, Q32854)</p><p><br></p><p><strong>gRNA sequence (without spacer):gRNA1-tRNA-gRNA2(8A8G)</strong></p><p>nnnnnnnnnnnnnnnnnnnnGTTTAAGAGCTATGCTGGAAACAGCATAGCAAGTTTAAATAAGGCTAGTCCGTTATCAACTTGAAAAAGTGGCACCGAGTCGGTGC<u>GGTTCCATGGTGTAATGGTTAGCACTCTGGACTCTGAATCCAGCGATCCGAGTTCAAATCTCGGTGGAACC</u>TnnnnnnnnnnnnnnnnnnnnGTTTgAGAGCTAtgctgGAAAcagcaTAGCAAGTTcAAATAAGGCTAGTCCGTTATCAACTTGAAAAAGTGGCACCGAGTCGGTGCTCGG<u>AAAAAAAAGAAAAAAAGAAAAAAAGAAAAA</u>TTTTTTT</p><p><br></p><p><strong>Primers:</strong></p><p><strong>tRNA_Read2 primer:</strong></p><p>5’-&nbsp;GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCTCTCTGGACTCTGAATCCAGCGAT&nbsp;-3’</p><p><strong>P5_read1 primer:</strong></p><p>5’-&nbsp;AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTC&nbsp;-3’</p><p><strong>P7_Read2_Index2 primer:</strong></p><p>5’-&nbsp;CAAGCAGAAGACGGCATACGAGATGGAGCTACGTGACTGGAGTTCAGACGTGT&nbsp;-3’</p><h2>Equipment</h2><p>	Countess II (Invitrogen)</p><p>	Chrominum Controller (10x Genomics, PN-110203)</p><p>	Chrominum Accessory Kit (10x Genomics, PN-110204)</p><p>	Vortex Genie 2 (Scientific Industries, 200-SI-0236)</p><p>	Proflex PCR system (Thermo Fisher, 4484073)</p><p>	Qubit 4 Fluorometer (Thermo Fisher, Q33238)</p><p>	Fragment Analyzer-12 / 96 (Agilent, GENE-QC006)</p><p>	0.22μM Filter (Merck Millipore, SLGV033RB)</p><p>	0.2ml Thin-walled tubes (Thermo Fisher, AB-0620)</p><p>	MicroAmp optical 8-Tube Strip (Thermo Fisher, A30588)</p><p>	1.5mL Microcentrifuge Tubes low retention (Thermo Fisher, 3451)</p><p>	Precision pipette tips 10μl (Mettler Toledo, 30389226)&nbsp;</p><p>	Precision pipette tips 200μl (Mettler Toledo, 30389240)</p><p>	Precision pipette tips 200μl wide-O (Mettler Toledo, 30389241)</p><p>	1250μl Pipette Tip (Thermo Fisher, TFLR1121000-Q)</p><h2>Procedure</h2><p><strong><u>Day1</u></strong></p><p><strong>1. Single cell capture and cDNA amplification</strong></p><p>At the end of CRISPR screening, cells were prepared as single cell suspension and loaded into the Chromium Single Cell B Chip. The cDNA was generated, purified and amplified (the 1<sup>st</sup>&nbsp;PCR) following the manual of Chromium Single-Cell 3′&nbsp;Reagent Kits v3 User Guide.&nbsp;</p><p><strong>2.cDNA size selection</strong></p><p>cDNA was separated into two parts according to the size.</p><p>(1) Long cDNA fragments, which mostly represented mRNA, were size-selected by 0.6x AMPure XP beads. The cDNA binding to the beads was called&nbsp;“<strong>long cDNA</strong>”, and will subject to mRNA library preparation. The "long cDNA" was then eluted in nuclease-free H2O following the 10x protocol.</p><p>(2) Short cDNA fragments, which mostly represented the index gRNA, were collected from the supernatant of the 0.6x beads selection, and then followed by a 1.2x size selection. The cDNA binding to the beads was called&nbsp;“<strong>short cDNA</strong>”.&nbsp;The "short cDNA" was then eluted in 25 uL nuclease-free H2O.</p><p><br></p><p><strong><u>Day2</u></strong></p><p><strong>3. 3’&nbsp;Gene Expression Library construction</strong></p><p>The&nbsp;“long cDNA”&nbsp;was used for mRNA library preparation by following the manual of Chromium Single-Cell 3′ Reagent Kits v3 User Guide.</p><p><strong>4. Index gRNA Library construction</strong></p><p>The&nbsp;“short cDNA”&nbsp;was used for the index gRNA library preparation by nested PCR.</p><p><strong>4.1. Index gRNA sequence enrichment</strong></p><p>&nbsp;(1) In total of eight index gRNA enrichment PCR (the 2<sup>nd</sup>&nbsp;PCR) reactions were conducted, with each reaction including 3 uL template, 25 uL NEBNext Ultra II Q5 Master Mix (NEB&nbsp;#M0544S), 2.5 uL tRNA_Read2 primer (10uM), 2.5 uL P5_read1 primer (10uM) and nuclease free water up to 50 uL.</p><p>&nbsp;(2) The PCR program was set as: (1) 98&nbsp;°C for 30 s, (2) 14 cycles of 98&nbsp;°C for 10 s, 60&nbsp;°C for 10 s, then 72 for 10 s and (3) 72&nbsp;°C for 2 min.&nbsp;</p><p>(3) The PCR products were combined and purified by 0.7–1.0×&nbsp;double-sided size selection (collect the supernatant from the 0.7x beads size selection, and followed by 1.0x beads selection, then collect elution from the beads) and eluted in 80μl of nuclease free water.&nbsp;</p><p><strong>4.2. gRNA library construction</strong></p><p>(1) In total of five library preparation PCR (the 3</p><p>rd PCR) was conducted, with each reaction including 10 uL template, 25 uL NEBNext&nbsp;Ultra&nbsp;II Q5&nbsp;Master Mix, 2.5 uL P7_Read2_Index2 primer (10μM) and 2.5 uL P5_read1 primer (10μM).&nbsp;</p><p>(2) The PCR program was set as: (1) 98&nbsp;°C for 30 s, (2) 5 cycles of 98&nbsp;°C for 10 s, then 54&nbsp;°C for 15 s, then 65°C for 20 s and (3) 72&nbsp;°C for 2 min.</p><p>(3) The PCR product was cleaned up and size selected with 0.7–1.0x&nbsp;AMPure XP beads.</p><p>(4) All eluted DNA was combined, and quantified. An aliquot was sent for NGS sequencing using the Illumina platform.</p><h2>Troubleshooting</h2><p>(1) 2ng of the mRNA library and the index gRNA library were loaded into Fragment Analyzer to check the library quality. The typical library sizes were as below (Blue: mRNA library; Black: index library). (Figure 2)</p><p>(2) During index gRNA enrichment procedure, the cDNA size was shown as Figure 3</p><h2>Time Taken</h2><p>About two days. See "Procedure".</p><h2>Anticipated Results</h2><p>The mRNA library is constructed in very standard procedure, and the 10x manual should be strictly followed.</p><p>For the index gRNA library, the cDNA from gRNA transcripts were progressively enriched, which can be clearly reflected by the shift of fragment size as shown in the Figure3. The size-shift meets that expectation indicates efficient enrichment of the index gRNA.</p><h2>References</h2><p>Song, Q. et al, Direct-seq: employing programmed gRNA scaffold for streamlined scRNA-seq in CRISPR screen. Genome Biology, 2020. DOI : 10.1186/s13059-020-02044-w</p></body></html>