<html><head><title>Nanopore native RNA sequencing of a human poly(A) transcriptome: RNA extraction, cDNA conversion and direct RNA and cDNA library preparation for Oxford Nanopore</title></head><body><h1>Nanopore native RNA sequencing of a human poly(A) transcriptome: RNA extraction, cDNA conversion and direct RNA and cDNA library preparation for Oxford Nanopore</h1>Authors: Rachael E. Workman, Alison D. Tang, Paul S. Tang, Miten Jain, John R. Tyson, Roham Razaghi, Philip C. Zuzarte, Timothy Gilpatrick, Alexander Payne, Joshua Quick, Norah Sadowski, Nadine Holmes, Jaqueline Goes de Jesus, Karen L. Jones, Terrance P. Snutch, Nicholas Loman, Benedict Paten, Matthew Loose, Jared T. Simpson, Hugh E. Olsen, Angela N. Brooks, Mark Akeson, Winston Timp<br>Abstract: <p>High throughput cDNA sequencing technologies have dramatically advanced our understanding of transcriptome complexity and regulation. However, these methods lose information contained in biological RNA because the copied reads are often short and because modifications are not carried forward in cDNA. We address these limitations using a native poly(A) RNA sequencing strategy developed by Oxford Nanopore Technologies (ONT). Our study focused on poly(A) RNA from the human cell line GM12878, generating&nbsp; 9.9 million aligned sequence reads. These native RNA reads had an aligned N50 length of 1294 bases, and a maximum aligned length of over 21,000 bases. A total of 78,199 high-confidence isoforms were identified by combining long nanopore reads with short higher accuracy Illumina reads. We describe methods for extracting intact RNA, poly-A selection, cDNA conversion for a portion of sample, and library preparation for both direct RNA and cDNA libraries. </p><br>Keywords: nanopore,  direct RNA, cDNA, GM12878, third generation sequencing, sequencing<br><h2>Introduction</h2><p>The roles of RNA in cell function are numerous and complex. Beyond the fundamental importance of mRNA, tRNA, and ribosomal RNA in translation, several classes of non-coding RNA (ncRNA) regulate cellular processes including division, differentiation, and programmed cell death<a href="https://paperpile.com/c/RHUSx7/7E8TF" target="_blank">(Su et al. 2016)</a>.&nbsp;</p><p>Nanopore RNA strand sequencing has emerged as an alternative single molecule strategy<a href="https://paperpile.com/c/RHUSx7/9fIEQ+azo1L+7oDVd" target="_blank">(Garalde et al. 2018; Smith et al. 2017; Jenjaroenpun et al. 2018)</a>. It differs from SBS-based platforms in that native RNA nucleotides, rather than copied DNA nucleotides, are identified as they thread through and touch a nanoscale sensor. Nanopore RNA strand sequencing shares the core features of nanopore DNA sequencing, <em>i.e.</em> a processive helicase motor regulates movement of a bound polynucleotide driven through a protein pore by an applied voltage. As the polynucleotide advances through the pore in single nucleotide steps, ionic current impedance reports on the segment of bases that occupy a narrow reading head as a function of time. This series of ionic current segments is then used to infer nucleotide sequence using an algorithm trained with known RNA molecules.</p><p>Here we describe RNA extraction, cDNA conversion and library preparation for both direct RNA and cDNA libraries for the human poly(A) transcriptome from the GM12878 cell line using the Oxford Nanopore (ONT) platform. We demonstrate that long native RNA reads allow for discovery and characterization of RNA isoforms that are difficult to observe using short read cDNA methods<a href="https://paperpile.com/c/RHUSx7/2mDaJ+IdVR3" target="_blank">(Steijger et al. 2013; Venturini et al. 2018)</a>. Because native RNA strands are directly read by nanopores, nucleotide modifications and 3′ poly(A) tail lengths can be determined directly from the ionic current signal absent additional processing steps. Data and resources are posted online at:</p><p>(<a href="https://github.com/nanopore-wgs-consortium/NA12878/blob/master/RNA.md" target="_blank">https://github.com/nanopore-wgs-consortium/NA12878/blob/master/RNA.md</a>).</p><h2>Reagents</h2><p>RPMI media (Invitrogen cat# 21870076)</p><p>FBS (Lifetech cat# 12483020)</p><p>2mM L-Glutamax (Lifetech cat# 35050061)</p><p>TRI-Reagent (Invitrogen AM9738)</p><p>BCP (1-Bromo-3-chloro-propane)&nbsp;</p><p>CHCl3 (Chloroform)&nbsp;</p><p>NEXTflex Poly(A) Beads (BIOO Scientific Cat#NOVA-512980)</p><p>Ampure XP beads</p><p>LongAmp Taq Master Mix (NEB)</p><p>Superscript IV (Thermo Fisher)&nbsp;</p><p>SIRV set 3 control (Lexogen)</p><p>SQK-RNA001 kit (ONT)</p><p>SQK-PCS108 kit (ONT)</p><p>SQK-LSK308 kit (ONT)</p><h2>Equipment</h2><p>Refrigerated centrifuge</p><p>Hula mixer</p><p>Thermocycler</p><p>Vortex&nbsp;</p><p>Nanopore MinION sequencer</p><h2>Procedure</h2><p><strong>Cell Tissue Culture</strong></p><p>1. Culture GM12878 cells in RPMI media supplemented with 15% non heat-inactivated FBS and 2mM L-Glutamax.&nbsp;</p><p>2. Grow cells to a density of 1E6 / ml before subsequent dilution of ⅓ every ~3 days, and expand to 9 x T75 flasks (45 ml of media in each).&nbsp;</p><p>3. Centrifuge cells for 10 min at 100 x g (4C), wash in 1/10th volume of PBS (pH 7.4) and combine for homogeneity.&nbsp;</p><p>4. Evenly split cells between 8 x 15ml tubes and pellet at 100 x g for 10 mins at 4C. Unless proceeding directly with RNA isolation, snap freeze in liquid nitrogen and immediately store at -80C.&nbsp;</p><p><strong>RNA Isolation</strong></p><p>1. Add 4 ml of TRI-Reagent to a frozen pellet of 5E7 GM12878 cells and briefly vortex immediately.&nbsp;</p><p>Incubate at room temperature for 5 minutes.&nbsp;</p><p>2. Add 400 μl BCP (1-Bromo-3-chloro-propane) or 200 μl CHCl3 (Chloroform) per ml of sample, vortex, incubate at room temperature for 5 minutes, briefly vortex again, and centrifuge for 10 minutes at 12,000 x g (4C).&nbsp;</p><p>3. Pool the aqueous phase in a LoBind Eppendorf tube and combine with an equal volume of isopropanol.&nbsp;</p><p>4. Mix sample, incubate at room temperature for 15 minutes, and centrifuge for 15 minutes at 12,000 x g (4C).&nbsp;</p><p>5. Remove supernatant, wash RNA pellet with 750 μl 80% ethanol and centrifuge for 5 minutes at 12,000 x g (4C).&nbsp;</p><p>6. Remove supernatant. Air dry pellet for 10 minutes, resuspend in nuclease free water (100 μl final volume), quantify with Qubit or Nanodrop, and either store at -80C or proceed to poly-A purification.</p><p><strong>Poly(A) RNA isolation</strong></p><p>1. Dilute 100 μg aliquots of total RNA in 100 μl of nuclease free water and poly-A select using NEXTflex Poly(A) Beads.&nbsp;</p><p>2. Elute resulting mRNA in Nuclease free water and store at -80oC.&nbsp;</p><p><strong>cDNA synthesis</strong></p><p>1. Perform first strand cDNA synthesis using Superscript IV and 100 ng of poly-A purified RNA combined with 0.5 ng of the SIRV set 3 control.&nbsp;</p><p>2. Perform reverse transcription and strand-switching with primers provided by ONT in the SQK-PCS108 kit.&nbsp;</p><p>3. After reverse transcription, amplify with LongAmp Taq Master Mix under the following conditions: 95C for 30 seconds, 11-15 cycles (95C for 15 seconds, 62C for 15 seconds, 65C for 15 minutes), 65C for 15 minutes, hold at 4C.&nbsp;</p><p>4. Perform 15 cycle PCR when using the SQK-PCS108 kit and 11 cycle PCR when using the SQK-LSK308 kit.&nbsp;</p><p>5. Purify PCR products using 0.8X AMPure XP beads, elute in nuclease-free water.&nbsp;</p><p><strong>Sequencing protocol</strong></p><p><strong>MinION native RNA sequencing of poly-A RNA</strong></p><p>1. Prepare biological poly-A RNA (500-775 ng) and a synthetic control (Lexogen SIRV Set 3, 5 ng) for nanopore direct RNA sequencing generally following the ONT SQK-RNA001 kit protocol, including the optional reverse transcription step recommended by ONT.&nbsp;</p><p>2. Replace Superscript III (ONT recommended) with Superscript IV for reverse transcription.&nbsp;</p><p><strong>MinION sequencing of cDNA</strong></p><p>1. Prepare cDNA sequencing libraries using 1 μg of cDNA following the standard ONT protocol for SQK-PCS108 (1D sequencing) or SQK-LSK308 (1D^2 sequencing).</p><p>2. Use 0.8X AMPure XP beads for cleanup, instead of recommended 0.4X.</p><h2>Troubleshooting</h2><p>**See Figure 3**</p><p><strong>Issue and Recommendations</strong></p><p><strong>All extraction steps</strong></p><p>Save the solutions at every step until the final quantifications are performed so that one could go back and troubleshoot if something goes awry.&nbsp;</p><p><strong>RNA extraction yield low</strong></p><p>If you saved the aqueous phase + isopropanol solution, try spinning it again for longer time.&nbsp;</p><p><strong>Extracted RNA quality low</strong></p><p>Extract with CHCl3 again and follow the protocol post that ; or&nbsp;re-precipitate RNA and follow the protocol post that&nbsp;</p><p><strong>rRNA peaks remaining after poly(A) selection</strong></p><p>Perform poly(A) selection again</p><h2>Time Taken</h2><p>RNA extraction : 60-90 minutes</p><p>PolyA selection : 75-90 minutes</p><p>cDNA conversion : 3-5 hours</p><p>Direct RNA library preparation : 60-120 minutes (if performing RT)</p><p>cDNA library preparation : 60-75 minutes</p><p>Sequencing : 48 hours</p><p><br></p><p><br></p><h2>Anticipated Results</h2><p>*See Figures 1 and 2**</p><p><strong>Step: Input, Yield </strong></p><p><strong>RNA extraction</strong></p><p>5E6 cells (GM12878), 200-250 μg</p><p><strong>PolyA selection</strong></p><p>100 μg, 1-1.5 μg</p><p><strong>cDNA conversion</strong></p><p>1 μg, 5-10 μg</p><p><strong>Library preparation</strong></p><p>500 ng, 50K - 2M reads</p><h2>References</h2><p><a href="http://paperpile.com/b/RHUSx7/9fIEQ" target="_blank">Garalde, Daniel R., Elizabeth A. Snell, Daniel Jachimowicz, Botond Sipos, Joseph H. Lloyd, Mark Bruce, Nadia Pantic, et al. 2018. “Highly Parallel Direct RNA Sequencing on an Array of Nanopores.” <em>Nature Methods</em>, January. https://doi.org/</a><a href="http://dx.doi.org/10.1038/nmeth.4577" target="_blank">10.1038/nmeth.4577</a><a href="http://paperpile.com/b/RHUSx7/9fIEQ" target="_blank">.</a></p><p><a href="http://paperpile.com/b/RHUSx7/7oDVd" target="_blank">Jenjaroenpun, Piroon, Thidathip Wongsurawat, Rui Pereira, Preecha Patumcharoenpol, David W. Ussery, Jens Nielsen, and Intawat Nookaew. 2018. “Complete Genomic and Transcriptional Landscape Analysis Using Third-Generation Sequencing: A Case Study of Saccharomyces Cerevisiae CEN.PK113-7D.” <em>Nucleic Acids Research</em>, January. https://doi.org/</a><a href="http://dx.doi.org/10.1093/nar/gky014" target="_blank">10.1093/nar/gky014</a><a href="http://paperpile.com/b/RHUSx7/7oDVd" target="_blank">.</a></p><p><a href="http://paperpile.com/b/RHUSx7/Kapmp" target="_blank">Quick, Joshua, Aaron R. Quinlan, and Nicholas J. Loman. 2014. “A Reference Bacterial Genome Dataset Generated on the MinION\texttrademark Portable Single-Molecule Nanopore Sequencer.” <em>GigaScience</em> 3 (1): 1–6.</a></p><p><a href="http://paperpile.com/b/RHUSx7/azo1L" target="_blank">Smith, A. M., M. Jain, L. Mulroney, D. R. Garalde, and M. Akeson. 2017. “Reading Canonical and Modified Nucleotides in 16S Ribosomal RNA Using Nanopore Direct RNA Sequencing.” <em>bioRxiv</em>. </a><a href="http://biorxiv.org/content/early/2017/04/29/132274.abstract" target="_blank">http://biorxiv.org/content/early/2017/04/29/132274.abstract</a><a href="http://paperpile.com/b/RHUSx7/azo1L" target="_blank">.</a></p><p><a href="http://paperpile.com/b/RHUSx7/2mDaJ" target="_blank">Steijger, Tamara, Josep F. Abril, Pär G. Engström, Felix Kokocinski, RGASP Consortium, Tim J. Hubbard, Roderic Guigó, Jennifer Harrow, and Paul Bertone. 2013. “Assessment of Transcript Reconstruction Methods for RNA-Seq.” <em>Nature Methods</em> 10 (12): 1177–84.</a></p><p><a href="http://paperpile.com/b/RHUSx7/7E8TF" target="_blank">Su, Ye, Haijiang Wu, Alexander Pavlosky, Ling-Lin Zou, Xinna Deng, Zhu-Xu Zhang, and Anthony M. Jevnikar. 2016. “Regulatory Non-Coding RNA: New Instruments in the Orchestration of Cell Death.” <em>Cell Death &amp; Disease</em> 7 (8): e2333.</a></p><p><a href="http://paperpile.com/b/RHUSx7/IdVR3" target="_blank">Venturini, Luca, Shabhonam Caim, Gemy George Kaithakottil, Daniel Lee Mapleson, and David Swarbreck. 2018. “Leveraging Multiple Transcriptome Assembly Methods for Improved Gene Structure Annotation.” <em>GigaScience</em> 7 (8). https://doi.org/</a><a href="http://dx.doi.org/10.1093/gigascience/giy093" target="_blank">10.1093/gigascience/giy093</a><a href="http://paperpile.com/b/RHUSx7/IdVR3" target="_blank">.</a></p><h2>Acknowledgements</h2><p>The authors are grateful for support from the following individuals. Libby Snell, Botond Sipos and Dan Turner (ONT) provided materials and advice relevant to the 3′ poly(A) standards used to test nanopolish-polya. Daniel Garalde (ONT) provided early advice on use of the MinION for RNA sequencing. Nicholas Conrad gave insight into the correlation of intron retention and poly(A) tail length. Mark Diekhans reviewed the isoform analysis. Zofia M. Chrzanowska-Lightowlers, Tom Suzuki, and Shunpei Okada commented on early drafts of the manuscript. The authors thank Andrew Beggs, Louise Tee and Tom Nieto (University of Birmingham, UK) for providing cell cultures used in the Birmingham sequencing runs. The project was supported by the following grants: NIH HG010053 (ANB, BP, &amp; MA), NIH 5T32HG008345 (ADT), NIH HG009190 (WT, JTS), NIH U54HG007990 (BP), U01 HL137183-02 (BP), Oxford Nanopore Research Grant SC20130149 (MA), National Institutes of Health Research Surgical Reconstruction and Microbiology Research Centre (JQ), Medical Research Council CLIMB Fellowship (NL), Wellcome Trust 204843/Z/16/Z (ML), BBSRC BB/N017099/1 and BB/M020061/1 (ML), the Canada Research Chair in Biotechnology and Genomics-Neurobiology (TPS), the Canadian Institutes of Health Research (#10677; TPS), the Canadian Epigenetics, Environment and Health Research Consortium (TPS), the Koerner Foundation (TPS), the Ontario Institute for Cancer Research through funds provided by the Government of Ontario (JTS).</p><p><br></p></body></html>