<html><head><title>Classification of primary Glioblastoma cell lines into molecular subtypes</title></head><body><h1>Classification of primary Glioblastoma cell lines into molecular subtypes</h1>Authors: Michelangelo Cordenonsi, Silvio Bicciato, Stefano Piccolo<br>Abstract: <p>This protocol provides the details for the classification of the primary Glioblastoma cell lines HuTu10 and HuTu13 into the Glioblastoma molecular subtypes using microarray data.</p><br>Keywords: Glioblastoma, primary cell lines, molecular subtypes<br><h2>Introduction</h2><p>Classical serum-grown Glioblastoma (GBM) cell lines have been shown to display molecular traits that are divergent from primary tumor masses from which they derive; as such, they are of limited utility to study Glioblastoma<sup>1</sup>. For this reason, many groups now prefer to use primary GBM cell lines maintained in serum-free media. Specifically, we used two independent primary GMB cell lines, HuTu10 and HuTu13<sup>2</sup>.</p><p>Glioblastomas (GBMs) can be classified in at least three transcriptional subtypes, defined as Proneural, Classical and Mesenchymal; Proneural and Mesenchymal GBMs represent two extremes in terms of molecular marker expression and patientsâ€™ survival<sup>3</sup>. As such, we were interested in determine the molecular subtypes to which HuTu10 and HuTu13 belong to.</p><h2>Procedure</h2><p>Before the start: we performed total RNA extraction in quadruplicate from HuTU cell colture and total RNA was then used for Affymetrix Microarray profiling on HG-U133 Plus 2.0 arrays (Affymetrix), as detailed for other cancer cell lines in Azzolin et al, 2012<sup>4</sup>.</p><p>Then, we proceeded as follows:</p><p>1) Probe level signals were converted to expression values using robust multi-array average procedure <sup>5</sup> (RMA) of Bioconductor <em>affy</em> package and a custom CDF for Affymetrix HG-U133Plus2 arrays based on Entrez genes (hgu133plus2hsentrezgcdf version 21.0.0; http://brainarray.mbni.med.umich.edu/Brainarray/Database/CustomCDF/21.0.0/entrezg.asp).</p><p>2) We then compared the transcriptome of these cells with those of the TCGA primary GBMs. Specifically, we used the gene expression data and molecular subtypes of the TCGA tumors to train the R version of the ClaNC (Classification to Nearest Centroids<sup>6</sup>) classifier implemented in the <em>clanc.R</em> function (https://github.com/naikai/sake/blob/master/R/clanc.R). The ClaNC classifier correctly classified 89.1%, 83%, 89.41%, and 86.71 of TCGA Classical, Proneural, Neural, and Mesenchymal samples using 1016 selected genes (centroids; Supplementary Table 1). Then, we merged HuTu and TCGA expression matrixes retaining the set of common gene symbols and removing batch effects with the <em>ComBat</em> function<sup>8</sup> of the Bioconductor <em>sva</em> package.</p><p>3) Finally, we clustered the samples of the merged HuTu and TCGA data using the set of 1016 centroid genes. The supervised clustering grouped HuTu10 samples in the cluster enriched in Proneural TCGA samples, and HuTu13 in the group enriched in Mesenchymal. Hierarchical clustering was performed using the function <em>hclust</em> of R <em>stats</em> package with Pearson correlation as distance metric and average agglomeration method. All analyses were performed in R 3.5.0.</p><h2>References</h2><p>1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Lee, J.<em> et al.</em> Tumor stem cells derived from glioblastomas cultured in bFGF and EGF more closely mirror the phenotype and genotype of primary tumors than do serum-cultured cell lines. Cancer cell 9, 391-403 (2006).</p><p>2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Pistollato, F.<em> et al.</em> Molecular mechanisms of HIF-1alpha modulation induced by oxygen tension and BMP2 in glioblastoma derived cells. PloS one 4, e6206 (2009).</p><p>3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Chen, J., McKay, R. M. &amp; Parada, L. F. Malignant glioma: lessons from genomics, mouse models, and stem cells. Cell 149, 36-47 (2012).</p><p>4&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Azzolin, L.<em> et al.</em> Role of TAZ as mediator of Wnt signaling. Cell 151, 1443-1456 (2012).</p><p>5&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Irizarry, R. A.<em> et al.</em> Exploration, normalization, and summaries of high density oligonucleotide array probe level data. Biostatistics 4, 249-264 (2003).</p><p>6&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Dabney, A. R. Classification of microarrays to nearest centroids. Bioinformatics 21, 4148-4154 (2005).</p></body></html>