<html><head><title>The pitfalls of biodiversity proxies: Differences in richness patterns of birds, trees and understudied diversity across Amazonia</title></head><body><h1>The pitfalls of biodiversity proxies: Differences in richness patterns of birds, trees and understudied diversity across Amazonia</h1>Authors: CAMILA DUARTE RITTER<br>Abstract: <p>It is the protocol I used to collect and extract DNA from organic litter, top 5 cm mineral soil and insect samples in the Amazonia to metabarcoding studies.</p><br>Keywords: Birds; biodiversity patterns; micro-organisms; operational taxonomic units (OTUs), species richness; trees.<br><h2>Introduction</h2><p>I collected organic litter, soil and insect samples in Amazonia. For this, I installed a temporary circular plots with a 28 m radius following the soil sampling protocol of Tedersoo et al. (2014). I installed a slam-trap in the middle of the plot and get it opened by 24h, the bottle was full with ethanol 96%. Inside each plot, I selected 20 trees at random and collected litter samples and two soil cores in opposite directions of each tree, summing to a total of 40 litter and 40 soil samples per plot. I pooled all samples to obtain one litter and one soil sample for each plot. Litter was defined as all organic material above the mineral soil and varied from 0 to ca 50 cm in thickness. I collected the soil samples from the top 5 cm of the mineral soil using a metal probe with a 2.5 cm diameter. I used gloves and masks and changed equipment in between each new plot to reduce the risk of cross-plot contamination. The samples were stored in sterilized white silica gel 1–4 mm (pre-treated by two minutes of microwave heating (800 W) and 15 min of UV light). All plots were provided with GPS coordinates. All dry samples were processed at the Univ. of Gothenburg, Sweden. The ethanol of insects samples was dried to fly and full with new ethanol in Sweden.</p><p><br></p><p>After DNA extraction samples were sent to Macrogen in South Korea for amplification of ribosomal small subunit (SSU)&nbsp;18S rRNA, we targeted the V7 region of the gene using the forward and reverse primers (5’-TTTGTCTGSTTAATTSCG-3’) and (5’-TCACAGACCTGTTATTGC-3’) designed by Guardiola et al. (2015) to yield 100 to 110 base pair (bp) fragments. For the ribosomal small subunit (SSU)&nbsp;16S rRNA, we targeted the V3–V4 region (~460 bases) of the 16S rRNA gene using the forward primer (5’-CCTACGGGNGGCWGCAG-3’) and reverse primer (5’-GACTACHVGGGTATCTAATCC-3’) from Klindworth et al. (2013). For the cytochrome c oxidase subunit&nbsp;I mitochondrial&nbsp;gene (COI), we amplified a region of ~313 bases using an internal forward primer (5’- GGWACWGGWTGAACWGTWTAYCCYCC-3’) designed by Leray et al. (2013) and the COI degenerate reverse primer (5’-TAAACTTCAGGGTGACCAAARAAYCA-3’) designed by Meyer (2003). Amplification and sequencing were carried out by Macrogen (Republic of Korea) following standard protocols using the Illumina MiSeq 2x250 (18S) and 2x300 (16S and COI) platforms.</p><h2>Reagents</h2><p>For soil (10 g) and litter (15 ml) I used DNeasy PowerMax Soil Kit extraction from Qiagen (Cat No./ID:&nbsp;12988-10).</p><p><br></p><p>For insects extraction:</p><p>NaCl</p><p>Tris-HCl</p><p>EDTA</p><p>SDS (sodium dodecyl sulphate)</p><p><em>protK&nbsp;</em></p><p><em>﻿</em>isopropanol</p><p>70 % EtOH</p><h2>Equipment</h2><p>Vortex</p><p>Centrifuge (adapted for 50ml falcon tube)</p><p>Refrigerator</p><p>Freezer</p><p>Incubator (60 graus)</p><h2>Procedure</h2><p>Soil and litter I followed the protocol from the extraction kit.</p><p><br></p><p>For insects:</p><p><br></p><p>I used the salt extraction protocol of Aljnabi &amp; Martinez (1997). </p><p>1.&nbsp;&nbsp;Prepare insect trap samples: Pour out ethanol and let dry for a while. Most of the ethanol needs to be removed, but insects may remain wet. This helps to submerge them into the buffer in step 2.</p><p><br></p><p>2.&nbsp;&nbsp;Pipette <strong>15 ml</strong> of extraction buffer (0.4 M NaCl, 10 mM Tris-HCl pH, 2 Mm EDTA pH 8.0 (and 2 % SDS) and <strong>1,5 ml</strong> of 20 % SDS (sodium dodecyl sulphate) on top of the sample in labelled Falcon tube.</p><p><br></p><p>3.&nbsp;&nbsp;Add <strong>10–30 µl</strong> of Proteinase K digestive enzyme into the tube. Vortex.</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<em>The amount of protK depends of the total biomass of the sample: more insects, more protK (I used 20 </em><strong>µl</strong> in general).</p><p><br></p><p>4.&nbsp;&nbsp;Leave to digest at + 60 ºC for overnight in the orbital shaker</p><p><br></p><p>5.&nbsp;&nbsp;Transfer ~15 ml of clear lysis solution into a new tube. Avoid transferring insects along the solution. Small amounts if insects don’t affect the extraction.</p><p><br></p><p>6.&nbsp;&nbsp;Add ~<strong>11,25 ml</strong> of 6 M NaCl to the tube containing lysis solution. Vortex for 30 seconds.</p><p><br></p><p>7.&nbsp;&nbsp;Place the tube into a centrifuge and spin at 10&nbsp;600 rpm for 30 minutes.</p><p><br></p><p>8.&nbsp;&nbsp;Take <strong>25 ml </strong>of supernatant from the tube and place into a clean labelled 50 ml Falcon tube.</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<em>These are the final tubes that the DNA will be stored in and should thus be labelled with sample ID, date, your own name</em></p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<em>If the salt + other unwanted have not spun down to the bottom of the tube after 30 minutes making it difficult to cleanly pipette the supernatant, you can transfer e.g. 30 ml of the supernatant + whatever comes along to a fresh tube (not labelled so detailed) and centrifuge it again for 5-10 mins, then transfer 25 ml to a labelled tube</em></p><p><br></p><p>9.&nbsp;&nbsp;Add <strong>25 ml</strong> of isopropanol to each tube. Mix by turning tube upside down a few times.</p><p><br></p><p>10.Place tube into -20 ºC for one hour.</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<em>One hour is minimum time, can be kept in freezer for longer, e.g. overnight</em></p><p><br></p><p>11.Centrifuge tubes at +4 ºC and 13&nbsp;200 rpm for 20 minutes.</p><p><br></p><p>12.Pour out isopropanol and wipe tubes.</p><p><br></p><p>13.Add ~<strong>2 ml</strong> of ice cold 70 % EtOH. DNA pellet should me submerged.</p><p><br></p><p>14.Centrifuge tubes at +4 ºC and 13&nbsp;200 rpm for 8 minutes.</p><p><br></p><p>15.Pour out EtOH and leave tubes to dry at +60 ºC for one hour (or at room temperature overnight).</p><p><br></p><p>16.Add <strong>50­–1000 µl</strong> of sterile H<sub>2</sub>O to each tube and leave at room temperature for 1-2 hours. </p><p><br></p><p>17.Store at -20 ºC.</p><p><br></p><h2>Troubleshooting</h2><p>For litter, I used volume (15ml) instead weight because the litter is too light to get 10g, so there was a variation of 3 to 10 g of litter weight.</p><p>For insects, when I had large amount of insects I need to keep them in bigger tubes and it was not possible use the shaker, so I vortex before, back and vortex sometimes but keep stopped in the incubation.</p><h2>Time Taken</h2><p>For soils and litter extraction around 3 to 4h (8 samples).</p><p><br></p><p>For insects I put the samples to digest in the evening (and left overnight) and extracted in the next morning, taking around 3 to 4h for 8 samples.</p><h2>Anticipated Results</h2><p>We obtained a total of  5,811,529 reads and 6,625 OTUs for</p><p>prokaryotes (16S) and   11,259,709 reads and 15,840 OTUs</p><p>for eukaryotes (18S). And  12,807,764 reads and 14,964&nbsp;OTUs COI.</p><h2>References</h2><p>Aljanabi, S. M. &amp; Martinez, I. Universal and rapid salt-extraction of high quality genomic DNA for PCR-based techniques. <em>Nucleic Acids Res.</em> <strong>25</strong>, 4692–4693 (1997).</p><p>Guardiola, M. <em>et al.</em> Deep-sea, deep-sequencing: metabarcoding extracellular DNA from sediments of marine canyons. <em>PLoS One</em> <strong>10</strong>, e0139633 (2015).</p><p>Klindworth, A. <em>et al.</em> Evaluation of general 16S ribosomal RNA gene PCR primers for classical and next-generation sequencing-based diversity studies. <em>Nucleic Acids Res.</em> <strong>41</strong>, e1 (2013).</p><p>&nbsp;Leray, M. <em>et al.</em> A new versatile primer set targeting a short fragment of the mitochondrial COI region for metabarcoding metazoan diversity: application for characterizing coral reef fish gut contents. <em>Front. Zool.</em> <strong>10</strong>, 34 (2013).</p><p>Meyer, C. P. Molecular systematics of cowries (Gastropoda: Cypraeidae) and diversification patterns in the tropics. <em>Biol. J. Linn. Soc.</em> <strong>79</strong>, 401–459 (2003).</p><p>Tedersoo, L. <em>et al.</em> Global diversity and geography of soil fungi. <em>Science</em> <strong>346</strong>, (6213):1256688 (2014).</p><p>Ritter, C. D. <em>et al.</em> Locality or habitat? Exploring predictors of biodiversity in Amazonia. <em>Ecography.</em> <strong>42</strong>, 321-333 (2019).</p><p>Ritter, C. D. <em>et al.</em> Biodiversity assessments in the 21st century: The potential of insect traps to complement environmental samples for estimating eukaryotic and prokaryotic diversity using high-throughput DNA metabarcoding. <em>Genome</em> <strong>62</strong>, 147-159 (2019).</p><p>Ritter, C. D. <em>et al.</em> High-throughput metabarcoding reveals the effect of physicochemical soil properties on soil and litter biodiversity and community turnover across Amazonia. <em>PeerJ</em> <strong>6</strong>, e5661 (2018).</p></body></html>