<html><head><title>A protocol to generate SPH-OminiCMV-Ents mESCs</title></head><body><h1>A protocol to generate SPH-OminiCMV-Ents mESCs</h1>Authors: Ni Gao, Jing Hu, Haibo Zhou, Hui Yang<br>Abstract: <p>Detection of genetic information in the natural context is essential to understand the biological principles. However, the ability to sense the activity of endogenous genes is limited for conventional tools.&nbsp;Here, we&nbsp;developed a highly programmable sgRNA switch system (Ents) that enables detection of endogenous genes and lncRNAs, even with very low expression level.&nbsp;This protocol is related to the publication “Endogenous promoter-driven sgRNA for monitoring the expression of low-abundant genes and lncRNAs” in Nature Cell Biology.</p><br>Keywords: Low-abundant genes, lncRNAs, Ents, CRISPR<br><h2>Introduction</h2><p>Manipulation of endogenous genetic information is crucial to understand the basic principles of biological systems. However, approaches to maniplulate endogenous information are limited, especially for the low-abundant genes and long non-coding RNAs (lncRNAs). Previous studies have shown that lncRNAs could play essential roles in various biological processes<sup>1-4</sup>. Nevertheless, functional characterization of a large population of lncRNAs remains challenging, mainly due to their low expression levels<sup>5-7</sup>.</p><p>Here, we developed an endogenous transcription-gated switch (Ents) by driving sgRNAs expression under an endogenous promoter. When Ents is paired with the highly-sensitive reporter, we could detect the activity of endogenous protein coding genes and lncRNAs faithfully in living cells, including genes with very low expression levels (&lt; 0.001, relative to&nbsp;<em>Gapdh</em>, qPCR analysis). This system bypasses the procedure of identifying efficient sensor-actuator pairs, and offers virtually an unlimited number of highly effective triggers. We showed that this system could be applied to monitor the dynamic changes of genes with different expression levels in mouse ESCs (mESCs), providing a powerful platform to detect the information of various genetic elements in the natural context. In this protocol, we described detailed procedures for generating Ents system in mESCs.</p><p><br></p><h2>Reagents</h2><p><strong>General Reagents</strong></p><p>EmbryoMax DMEM (Milipore)</p><p>Fetal Bovine Serum (Gibco)</p><p>Non-essential AA (NEAA) (Milipore)</p><p>Nucleosides (Milipore)</p><p>Penicillin-Streptomycin (Gibco)</p><p>L-Glutamine (Milipore)</p><p>2-Mercaptoethanol (Milipore)</p><p>PD0325901 (Selleck)</p><p>CHIR99021 (Selleck)</p><p>Mouse LIF (Milipore)</p><p>DPBS (Gibco)</p><p>0.05% Trypsin-EDTA (Thermo Fisher Scientific)</p><p>Lipofectamine3000&nbsp;(Invitrogen)</p><p>OPTI-MEM&nbsp;(Reduced serum Medium)&nbsp;(Gibco)</p><p>Phanta Max Super-Fidelity DNA Polymerase (Vazyme)</p><p>6-wells plate&nbsp;(Corning)</p><p>12-wells plate&nbsp;(Corning)</p><p>24-wells plate&nbsp;(Corning)&nbsp;&nbsp;&nbsp;</p><p>96-wells plate&nbsp;(Corning)</p><p><br></p><p><strong>Plasmids</strong></p><p>Plasmid1: CAG-scFv-NLS-P65-HSF1-Flag-WPRE-PolyA-CAG-dCas9-HA-10xGCN4-Flag-WPRE-PolyA-PGK-PuroR-PolyA (SPH)</p><p>Plasmid2: 3 x TS-miniCMV-mCherry (OminiCMV-mCherry)</p><p>Plasmid3: tRNA-sgRNA- tRNA (one sgRNA or sgRNA Array) (sgRNA precursor)</p><p><br></p><p><strong>Cells</strong></p><p>Mouse Embryonic Stem Cells</p><h2>Equipment</h2><p>Cell culture incubator with 5% CO2, 37 °C&nbsp;</p><p>Clean bench&nbsp;</p><p>Microscope (Olympus)</p><p>Centrifuge (Eppendorf)</p><p>Water bath</p><p>Moflo XDP (Beckman Coulter)&nbsp;</p><p>MoFloAstrios EQ (Beckman Coulter)&nbsp;</p><p>C1000 Touch Thermal Cycler (Bio-Rad)</p><p>Olympus FV3000</p><h2>Procedure</h2><p><strong>Generate SPH-OminiCMV cell line</strong></p><p><strong>Day 1</strong></p><p>Seed mESCs on 6-well plate, using LIF-supplemented medium.</p><p><strong>Day 2</strong></p><p>Co-transfect 1 μg SPH (resistant to puromycin), 1 μg OminiCMV-mCherry and 0.6 μg PBase using Lipofectamine 3000.</p><p>Replace with 2 ml LIF-supplemented medium 12 hours after transfection.</p><p><strong>Day 4</strong></p><p>Digest mESCs with 0.05% trypsin for 4 minutes at 37 °C, inactivate trypsin and centrifuge at 1000 rpm for 3 min, then seed mESCs to new 6-well plates and treat with puromycin (InvivoGen, final concentration 1 μg/ml) for 4~5 days.&nbsp;</p><p><strong>Day 8</strong></p><p>Digest mESCs with 0.05% trypsin for 4 minutes at 37 °C, inactivate trypsin and centrifuge at 1000 rpm for 3 min, then seed mESCs to new 6-well plate.</p><p><strong>Day 10</strong></p><p>Digest mESCs with 0.05% trypsin and prepare for FACS into 96-well plates.</p><p><strong>Day 14~15</strong></p><p>Remove single colonies from 96-well plates to 24-well plates. Confirm SPH-OminiCMV positive colonies by transient transfection of sgRNAs and PCR analysis (SPH Primers: forward (5′-GCTTCCCTGAATCCGGGCTG -3′) and reverse (5′- TGGCTCTGGCCCCTAGCTC-3′); OminCMV primers: forward (5′- GGAGGCCTATATAAGCAGAGC -3′) and reverse (5′-ACAAAGGCATTAAAGCAGCGTA-3′).&nbsp;</p><p><br></p><p><strong>Generate sgRNA precursor cell line</strong></p><p><strong>Day 1</strong></p><p>Seed SPH-OminiCMV mESCs on 12-well plates, using LIF-supplemented medium.</p><p><strong>Day 2</strong></p><p>Co-transfect 0.5 μg SaKKHCas9-GFP-sgRNA and 1 μg&nbsp;SpCas9&nbsp;sgRNA precursor using Lipofectamine 3000 (Life Technologies).</p><p>As previously described, Homology-Mediated End Joining (HMEJ) or homology recombination (HR) strategy (800 bp homology arm) was used to insert the sgRNA precursor, as previously described<sup>8</sup>.</p><p>Replace with 1 ml LIF-supplemented medium 12 hours after transfection.</p><p><strong>Day 4</strong></p><p>Digest mESCs with 0.05% trypsin and prepare for FACS, and sort positive cells, then seed GFP positive cells into 12-well plates.</p><p><strong>Day 8</strong></p><p>Digest mESCs with 0.05% trypsin, then seed mESCs to new 12-well plates.</p><p><strong>Day 18</strong></p><p>Sort single cells into 96-well plates by FACS, successful insertion of the sgRNA precursor is determined by PCR.&nbsp;</p><p><strong>Day 22</strong></p><p>Remove single colonies from 96-well plates to 24-well plates. Confirm sgRNA precursor positive colonies by PCR analysis.</p><p><strong>Day 27</strong></p><p>Measure the fluorescent intensity of SPH-OminiCMV-Ents colonies by FACS, and take the fluorescence images under a confocal microscope (Olympus FV3000).</p><h2>Troubleshooting</h2><p>Please use SaKKHCas9 to insert the SpCas9 sgRNA precursor</p><h2>Time Taken</h2><p>Around one and a half months</p><h2>Anticipated Results</h2><p>We could faithfully detect the activity of endogenous genes and lncRNAs.</p><h2>References</h2><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Guttman, M. &amp; Rinn, J.L. Modular regulatory principles of large non-coding RNAs. Nature 482, 339-346 (2012).</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Kretz, M. et al. Control of somatic tissue differentiation by the long non-coding RNA TINCR. Nature 493, 231-235 (2013).</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Sauvageau, M. et al. Multiple knockout mouse models reveal lincRNAs are required for life and brain development. Elife 2, e01749 (2013).</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Bester, A.C. et al. An Integrated Genome-wide CRISPRa Approach to Functionalize lncRNAs in Drug Resistance. Cell 173, 649-664 e620 (2018).</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Cabili, M.N. et al. Localization and abundance analysis of human lncRNAs at single-cell and single-molecule resolution. Genome Biol 16, 20 (2015).</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Chen, L. et al. Tissue Expression Difference between mRNAs and lncRNAs. Int J Mol Sci 19 (2018).</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;Azlan, A., Obeidat, S.M., Yunus, M.A. &amp; Azzam, G. Systematic identification and characterization of Aedes aegypti long noncoding RNAs (lncRNAs). Sci Rep 9, 12147 (2019).</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Yao, X. et al. Homology-mediated end joining-based targeted integration using CRISPR/Cas9. Cell Res 27, 801-814 (2017).</p><h2>Acknowledgements</h2><p>We thank FACS facility Lijuan Quan, Haiyan Wu and Songlin Qian in ION and Optical Imaging facility Yonghong Wang, Yumei Zhang, Xuxin Chen, Dan Xiang and Qian Hu. This work was supported by Basic Frontier Scientific Research Program of Chinese Academy of Sciences From 0 to 1 original innovation project (ZDBS-LY-SM001), R&amp;D Program of China (2017YFC1001300 and 2018YFC2000100), CAS Strategic Priority Research Program (XDB32060000), National Natural Science Foundation of China (31871502, 31925016, 91957122, 31901047), Shanghai Municipal Science and Technology Major Project (2018SHZDZX05), Shanghai City Committee of Science and Technology Project (18411953700, 18JC1410100, 19XD1424400), and International Partnership Program of Chinese Academy of Sciences (153D31KYSB20170059).</p></body></html>