<html><head><title>Sars-Cov-2 Spike 1 protein control Natural killer cells activation via HLA-E/NKG2A pathway.</title></head><body><h1>Sars-Cov-2 Spike 1 protein control Natural killer cells activation via HLA-E/NKG2A pathway.</h1>Authors: Daria Bortolotti, Valentina Gentili, Sabrina Rizzo, Antonella Rotola, Roberta Rizzo<br>Abstract: <p>Natural killer (NK) cells are important in the control of viral infections. However, the role of NK cells during Sars-Cov-2 infection has previously not been identified. Peripheral blood&nbsp;NK cells from Sars-Cov and Sars-Cov-2 naïve subjects were evaluated for their activation, degranulation, interferon-gamma expression in the presence of Sars-Cov and Sars-Cov-2 spike proteins. K562 and lung epithelial cells were transfected with spike proteins and co-cultured with NK cells. The analysis was performed by flow cytometry and immune-fluorescence.&nbsp;Sars-Cov and Sars-Cov-2 spike proteins did not alter NK cell activation in K562 in vitro model. On the contrary, Sars-Cov-2 spike 1 protein (SP1) intracellular expression by lung epithelial cells resulted in NK cell reduced degranulation. Further experiments revealed a concomitant induction of HLA-E expression on the surface of lung epithelial cells and the recognition of a SP1-derived HLA-E-binding peptide. Simultaneously, there was the up-modulation of the inhibitory receptor NKG2A/CD94 on NK cells when SP1 is expressed in lung epithelial cells. We ruled out GATA3 transcription factor as responsible for HLA-E increased levels and HLA-E/NKG2A interaction as implicate in NK cells exhaustion.&nbsp;We show for the first time that NK cells are affected by SP1 expression in lung epithelial cells via HLA-E/NKG2A interaction. The resulting NK cells exhaustion might contribute to immunopathogenesis in Sars-Cov-2 infection.</p><p><br></p><br>Keywords: Sars-Cov2; NK cell; NKG2A; HLA-E<br><h2>Introduction</h2><p>Natural killer (NK) cells are important in the control of viral infections. However, the role of NK cells during Sars-Cov-2 infection has previously not been identified. Peripheral blood&nbsp;NK cells from Sars-Cov and Sars-Cov-2 naïve subjects were evaluated for their activation, degranulation, interferon-gamma expression in the presence of Sars-Cov and Sars-Cov-2 spike proteins. K562 and lung epithelial cells were transfected with spike proteins and co-cultured with NK cells. The analysis was performed by flow cytometry and immune-fluorescence.&nbsp;Sars-Cov and Sars-Cov-2 spike proteins did not alter NK cell activation in K562 in vitro model. On the contrary, Sars-Cov-2 spike 1 protein (SP1) intracellular expression by lung epithelial cells resulted in NK cell reduced degranulation. Further experiments revealed a concomitant induction of HLA-E expression on the surface of lung epithelial cells and the recognition of a SP1-derived HLA-E-binding peptide. Simultaneously, there was the up-modulation of the inhibitory receptor NKG2A/CD94 on NK cells when SP1 is expressed in lung epithelial cells. We ruled out GATA3 transcription factor as responsible for HLA-E increased levels and HLA-E/NKG2A interaction as implicate in NK cells exhaustion.&nbsp;We show for the first time that NK cells are affected by SP1 expression in lung epithelial cells via HLA-E/NKG2A interaction. The resulting NK cells exhaustion might contribute to immunopathogenesis in Sars-Cov-2 infection.</p><p><br></p><h2>Reagents</h2><p><strong>Flow cytometry</strong></p><p>CD3-PerCp-Cy5.5, CD56-FITC moAbs (e-Bioscience, Frankfurt, DE); CD3-PE-Cy7 (clone SP34-2), CD16-PE (clone B73.1), CD56-APC (clone B159), NKG2A-PE (clone&nbsp;131411) (BD, Italy); HLA-I (HLA-A,-B,-C)-PE (BD Biosciences, Italy); anti-IFN-γ MoAb (27726; R&amp;D Systems); anti-GATA3 (BV421; BD, Italy); anti-spike 1 (clone 2C1), anti-spike 2 and anti Sars-Cov spike antibodies (myBiosource, USA); anti-CD107a-PE; clone H4A3; BD Biosciences.</p><p>IntraPrep reagent (Beckman Coulter). 7AAD/CFSE Cell-mediated cytotoxicity assay kit (1180 E. Ellsworth Rd Ann Arbor, MI, USA).</p><p>&nbsp;</p><p><strong>Viral RNA Detection</strong></p><p>MagMAX Viral/Pathigen Nuclei Acid Isolation kit (ThermoFisher, Italy); TaqMan 2019nCoV assay kit v1 (ThermoFisher, Italy).</p><p><strong>&nbsp;</strong></p><p><strong>Cell culture</strong></p><p>Beas-2B&nbsp;(ATCC CRL-9609); BEGM culture medium (BEGM Kit Catalog No. CC-3170 Lonza/Clonetics Corporation, USA); K562 (ATC CCL-243);  RPMI (Gibco, Italy); FCS (fetal calf serum, Euroclone, Pero, MI, Italy); penicillin, streptomycin. </p><p><br></p><p><strong>Antibodies</strong></p><p>Anti-HLA-E (clone MEM-E/08, Exbio, Praha, CZ); anti-CD94/NKG2A (clone&nbsp;131411, BD, Italy)</p><p><br></p><p><strong>Inhibitors</strong></p><p>Pyrrothiogatain (Santa Cruz; cat#sc-352288A). </p><p>&nbsp;</p><p><br></p><p><strong>Cell migration assay</strong></p><p>Sars-Cov-2 proteins (spike protein S1 subunit, spike protein S2 subunit) (RayBiotech); control Sars-Cov spike protein (MyBiosource); CXCL12 (BioLegend).</p><p><strong>&nbsp;</strong></p><p><strong>Protein Transfection</strong></p><p>Pierce Protein Transfection Kit (ThermoFisher).</p><p> </p><p><strong>Lactate dehydrogenase (LDH) assay</strong></p><p>A colorimetric-based lactate dehydrogenase (LDH) assay (Cytotoxicity Detection KitPLUS; Switzerland).</p><p>&nbsp;</p><p><br></p><p><br></p><h2>Equipment</h2><p><strong>Flow cytometry</strong></p><p>The flow cytometry analysis were performed with FACS CantoII flow cytometer (BD, Milan, Italy) and FlowJo LLC analysis software (Ashland, Oregon, USA). </p><p><br></p><p><strong>Immunofluorescence assay</strong></p><p>The immunofluorescence images were acquired with Nikon Eclipse TE2000S, equipped with a digital camer.</p><p><br></p><p><strong>HLA-E binding prediction</strong></p><p>The HLA-E binding prediction was made using the IEDB analysis resource ANN aka NetMHC (ver. 4.0) tool at the <a href="http://tools.iedb.org/mhci/help/" rel="noopener noreferrer" target="_blank">http://tools.iedb.org/mhci/help/</a>, using the viral spike 1 protein from Sars-Cov-2 sequence (<a href="https://www.ncbi.nlm.nih.gov/protein/QHO62112.1" rel="noopener noreferrer" target="_blank">QHO62112.1</a>).</p><p><br></p><p><br></p><h2>Procedure</h2><p><strong>NK cells purification</strong></p><p>Primary NK cells were separated from peripheral blood samples using the negative magnetic cell separation (MACS) system (Miltenyi Biotech, Gladbach, Germany). The analysis of purified cell fraction by flow cytometry with CD3-PerCp-Cy5.5, CD56-FITC moAbs (e-Bioscience, Frankfurt, DE), demonstrated that the NK cell content was &gt;90% (data not shown).</p><p>&nbsp;</p><p><strong>Viral RNA Detection</strong></p><p>RNA extraction was performed by using MagMAX Viral/Pathigen Nuclei Acid Isolation kit (ThermoFisher, Italy) according to the manufacturer’s instructions. The RT-PCR was performed with the TaqMan 2019nCoV assay kit v1 (ThermoFisher, Italy).</p><p><strong>&nbsp;</strong></p><p><strong>Cell culture</strong></p><p>Beas-2B&nbsp;(ATCC CRL-9609) bronchial epithelial cell line was grown in BEGM culture medium (BEGM Kit Catalog No. CC-3170; Lonza/Clonetics Corporation, USA) K562 (ATC CCL-243) lymphoblastoid cell line was cultured in RPMI (Gibco, Italy) supplemented with 10% of FCS (fetal calf serum, Euroclone, Pero, MI, Italy) and 100 U/mL penicillin, 100 µg/mL streptomycin. Cell cultures were maintained at 37° C in humidified atmosphere of 5% CO<sub>2 </sub>in air.</p><p>The role of HLA-E and NKG2A was evaluated incubating the cells with anti-HLA-E (clone MEM-E/08, Exbio, Praha, CZ) or anti-CD94/NKG2A (clone&nbsp;131411, BD, Italy) antibodies (7.5 ng/ml).</p><p>GATA3 DNA-binding activity was inhibited adding pyrrothiogatain (Santa Cruz; cat#sc-352288A) to cell cultures (10uM) <sup>17</sup>.&nbsp;</p><p>&nbsp;</p><p><strong>Flow cytometry</strong></p><p>1x10<sup>5</sup> eNK cells were labeled with fluorophore-conjugated antibodies: CD3-PE-Cy7 (clone SP34-2), CD16-PE (clone B73.1), CD56-APC (clone B159), NKG2A-PE (clone&nbsp;131411) (BD, Italy) and matched isotype controls.</p><p>5x10<sup>5</sup> bronchial epithelial cells epithelial cells were stained specific Ab HLA-I (HLA-A,-B,-C)-PE (BD Biosciences, Italy), HLA-E (clone MEM-E/08, Exbio, Praha, CZ) and matched isotype controls.</p><p>Interferon-gamma (IFN-gamma) and GATA3 expression were evaluated in fixed and permeabilized cells with IntraPrep reagent (Beckman Coulter) and stained with anti-IFN-γ MoAb (27726; R&amp;D Systems) or anti-GATA3 (BV421; BD, Italy) MoAb.</p><p>SP1, SP2 and S proteins expression in K562 and Beas-2B transfected cells were evaluated with anti-spike 1 (clone 2C1), anti-spike 2 and anti Sars-Cov spike antibodies (myBiosource, USA).</p><p>Data were analyzed using FACS CantoII flow cytometer (BD, Milan, Italy) and FlowJo LLC analysis software (Ashland, Oregon, USA). Ten thousand events were acquired.</p><p>&nbsp;</p><p><strong>Immunofluorescence assay </strong></p><p>HLA-E expression was analyzed by immunofluorescence with anti-HLA-E antibody (clone MEM-E/08, Exbio, Praha, CZ).</p><p>&nbsp;</p><p><strong>Cell migration assay</strong></p><p>The assay was performed using the Cornig Transwell System, with inserts with 5um pore polycarbonate membrane. Briefly, 1x10<sup>6</sup> cells were seeded in the upper chamber in 150ul of RPMI or BEGM with the 0,5% of FBS. The inserts containing cells were positioned into a 24well plate, which provides the lower reservoirs for migration system. Each reservoir was filled with 650ul of medium (RPMI) containing Sars-Cov-2 proteins (spike protein S1 subunit, spike protein S2 subunit) (RayBiotech) or control Sars-Cov spike protein (MyBiosource) at the final concentration of 100ng/ml. Medium with CXCL12 (100ng/ml; BioLegend) at the concentration of was used as positive. Migration was performed for 3h at 37°C and then the plate was briefly centrifuged at 300g for 5min in order to collect migrated cells in the lower reservoir for cell count. Every condition was tested in triplicate and results were reported as fold of cell migration compared to untreated NK cells.</p><p><strong>&nbsp;</strong></p><p><strong>Protein Transfection</strong></p><p>K562 or Beas-2B cell lines were transfected using the Pierce Protein Transfection Kit (ThermoFisher) following product instructions. A total of 4x10<sup>5</sup> cells were transfected with 1ug of protein (spike protein S1 subunit, spike protein S2 subunit) of Sars-Cov-2 or Sars-Cov spike protein. Transfection was performed for 3-4h at 37°C in 1ml medium without FBS. After transfection, a volume of complete medium with 20% FBS was added to each well. K562 or Beas-2B cells treated with transfection reagent alone or transfected with 0,5ug of control fluorescent antibody (provided in the kit) were used as negative and efficiency control, respectively.</p><p>&nbsp;</p><p><strong>Lactate dehydrogenase (LDH) assay</strong></p><p>LDH assay was performed to evaluate the effect of the transfection with Sars-Cov2 and Sars-Cov proteins in K562 or Beas-2B cells on cell viability. Transfected K562 or Beas-2B cells were suspended at 5 × 10<sup>4</sup> cells/mL and cultured for 4 h on a 96-well microplate at 37 °C with 5% CO<sub>2</sub>. A colorimetric-based lactate dehydrogenase (LDH) assay (Cytotoxicity Detection KitPLUS; Switzerland) was used, according to the manufacturer’s instructions.</p><p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p><p><strong>Degranulation analysis</strong></p><p><em>In vitro</em> cytotoxicity experiments were performed using K562 or Beas-2B cells as target and NK cells as effector cells. NK cells were added to K562 or Beas-2B cells with a 5:1 effector:target ratio. NK cell degranulation was evaluated by CD107a staining (anti-CD107a-PE; clone H4A3; BD Biosciences) after 3 hours of treatment with Golgi Stop solution (BD). Labeled cells were analyzed with FACSCantoII flow cytometer (Becton Dickinson, Milano, Italy) and FlowJo software (Tree Star Inc, Ashland, OR, USA).</p><p>&nbsp;</p><p><strong>Carboxyfluorescein diacetate succinimidyl ester (CFSE) analysis</strong></p><p>K562 or Beas-2B cells were labeled with carboxyfluorescein diacetate succinimidyl ester (CFSE) to assess cell-mediated cytotoxicity, using 7AAD/CFSE Cell-mediated cytotoxicity assay kit (1180 E. Ellsworth Rd Ann Arbor, MI, USA). 10<sup>7 </sup>cells/ml were resuspended in CFSE staining solution and incubated for 15 minutes at 37°C. Control target cells were resuspended in 0.1% BSA. Then, cells were washed two times with culture medium and incubated for 30 minutes at 37°C. NK cells were put in co-culture with CFSE-labeled infected cells at a 1:5 ratio. The cell mixture was incubated for 4h, centrifuged and resuspended in 7-AAD staining solution. Control target cells were resuspended in Assay Buffer. Cells were incubated for 15 minutes in the dark at 4°C. Then cells were centrifuged and resuspended in Assay buffer. Cells were analyzed with FACSCantoII flow cytometer and FlowJo software.</p><p>&nbsp;</p><p><strong>HLA-E binding prediction</strong></p><p>The HLA-E binding prediction was made using the IEDB analysis resource ANN aka NetMHC (ver. 4.0) tool at the <a href="http://tools.iedb.org/mhci/help/" rel="noopener noreferrer" target="_blank">http://tools.iedb.org/mhci/help/</a>, using the viral spike 1 protein from Sars-Cov-2 sequence (<a href="https://www.ncbi.nlm.nih.gov/protein/QHO62112.1" rel="noopener noreferrer" target="_blank">QHO62112.1</a>).</p><h2>Troubleshooting</h2><p>The procedures are well defined and previously published.</p><h2>Time Taken</h2><p><br></p><h2>Anticipated Results</h2><p><strong>Spike proteins induce NK cell migration</strong></p><p>Although the role of NK cells in the immune response towards viral infections is generally accepted, there are few data on the early NK cell trafficking in response to coronavirus infections <sup>5</sup>. The evaluation of NK response to Sars-Cov-2 infection is important to determining the innate immune response per se and for the cross-talk with adaptive immune cells <sup>22</sup>. We explored the migration, interferon-gamma (IFN-gamma) expression and degranulation of NK cell in the presence of spike 1 (SP1) and spike 2 (SP2) Sars-Cov-2 proteins and Sars-Cov spike proteins (S). We used a cell migration system in a 5 µm transwells w/polycarbonate filter, without a cell barrier. We used peripheral blood NK cells from control subjects negative for both Sars-Cov-2 and Sars-Cov viremia (data not shown). This condition ensured that NK cells were naïve for spike proteins. We cultured NK cells in the presence of SP1, SP2 from Sars-Cov-2 or spike protein (S) from Sars-Cov. We used CXCL12 as a positive control for NK cell migration <sup>20</sup>. We observed an increase in NK cell migration in the presence of both SP1 and SP2 (SP1 p&lt;0.001; SP2 p=0.0125; Student T test) (Fig. 1A). Similarly, NK cell migration was induced by the Sars-Cov S protein (p=0.034; Student T test) (Fig. 1A). These data suggest that both Sars-Cov-2 and Sars-Cov spike proteins are able to chemo-attract NK cells. To evaluate if spike proteins are able to induce also NK cells activation, we analyzed the expression of IFN-gamma. We observed an increase in IFN-gamma expression in the presence of SP1 protein (p&lt;0-001; Student T test), while SP2 and S proteins were not able to induce IFN-gamma (SP2 p=0.067; S p=0.076; Student T test) (Fig. 1B). &nbsp;&nbsp;</p><p>&nbsp;</p><p><strong>Spike proteins did not modify NK cell cytotoxicity</strong></p><p>Then, we evaluated the cytotoxicity of NK cells in the presence of spike proteins, using CD107a staining, a marker of degranulation <sup>21</sup>. We mimicked the expression of spike proteins inside NK target cells, the K562 cell line, transfecting directly the proteins. We obtained a mean transfection efficiency of 85% for SP1, SP2 and S proteins (Fig. 1C). The K562 cells viability, evaluated by MTT assay, was not affected by protein transfection (Fig. 1D). We incubated NK cells for four hours with K562 and evaluated the expression of CD107a. We observed an increase in CD107a staining in all the culture conditions, with no difference in the presence of SP1, SP2 and S proteins in comparison with the co-culture with untreated NK cells (S1 p= 0.078; S2 p=0.087; S p=0.081; Student T test) (Fig. 1E, F). To be sure that NK cells expressing CD107a were able to kill K562 cells, CFSE (carboxyfluorescein diacetate succinimidyl ester) staining of target cells was detected by flow cytometry <sup>18</sup>. We observed an increase in K562 CFSE<sup>+</sup>/7-AAD+<sup> </sup>cell percentage in all the co-culture conditions and comparable with the killing observed in the co-culture with untreated NK cells (p&lt;0.001; Student T test) (Figure 1G), confirming the result observed with CD107a staining (Figure 1E, F).</p><p><strong>&nbsp;</strong></p><p><strong>Sars-Cov-2 spike 1 protein modifies NK cell cytotoxicity when presented by lung epithelial cells</strong></p><p>Since the target cells for Sars-Cov-2 replication are lung epithelial cells <sup>4</sup>, we considered the activation status of NK cells in this context. We mimicked the expression of SP1, SP2 and S proteins inside Beas-2B lung epithelial cells, transfecting directly the proteins. We obtained a 90% efficiency of transfection with all the proteins (Fig. 2A). The transfection did not affect cell viability (Fig. 2B) and the expression of CK8 epithelial cell markers (Fig. 2A). We incubated NK cells for four hours with lung epithelial cells and evaluated the expression of CD107a degranulation marker. We observed a decrease in CD107a staining in the culture condition with SP1 transfected Beas-2B cells (p&lt;0.001; Student t test) (Fig. 2A, B). On the contrary, we observed only a slight decrease in CD107a expression in the culture condition with SP2 and S proteins transfected Beas-2B cells, that is similar to that observed with the control of transfection condition (Null-transfected) (p=0.043; Student T test) (Fig. 2A, B). </p><p>To be sure that NK cells expressing CD107a were able to kill Beas2B cells, CFSE staining of target cells was detected by flow cytometry. We observed an increase in Beas-2B CFSE<sup>+</sup>/7-AAD+<sup> </sup>cell percentage in the co-culture with NK cells in SP2 and S protein transfected Beas2B cells (Fig. 3C), while there was a decrease in Beas-2B CFSE+/7-AAD+<sup> </sup>cell percentage in the co-culture with NK cells in SP1 transfected Beas-2B cells (p&lt;0.001;&nbsp;Student T test) (Fig. 3C), confirming the results observed with CD107a staining (Fig. 3A, B). </p><p><strong>&nbsp;</strong></p><p><strong>Spike 1 peptide is presented by lung epithelial cells via HLA-E molecules</strong></p><p>Since the presence of intracellular SP1 protein in lung epithelial cells induced a decrease in the cytotoxic activity of NK cells, we analyzed the possible factors involved in the modification of NK cell status. </p><p>Since viral proteins are commonly recognized and degraded by proteasome inside the infected cells <sup>23</sup>, we hypothesized that SP1 peptides might be presented to NK cells. Intracellular peptide presentation is performed by Human leukocyte antigen (HLA) class I molecules, that are expressed by all nucleated cells. Firstly, we evaluated the expression of HLA class I molecules in Beas-2B cells transfected with Sp1, Sp2 and S proteins. When we stained the cells with anti-classical HLA class I molecules (HLA-A, HLA-B, HLA-C) antibody, we recognized a significant decrease in their membrane expression when lung epithelial cells were transfected with SP1 protein (p&lt;0.001; Student t test) (Fig. 3D, E). Epithelial lung cells can express also non-classical HLA-E molecule <sup>24</sup>.&nbsp;HLA-E binds peptides primarily derived from a specific signal sequences <sup>25</sup> and interacts with NKG2A/CD94 NK cell inhibitory receptors <sup>14</sup>. When we performed the lung epithelial cell staining with MEM-E/06 anti-HLA-E antibody, we observed a slight expression in basal conditions that is not affected by SP2 and S transfection (Fig. 3F, G). On the contrary, there is a significant increase in HLA-E expression in the presence of SP1 protein (p=0.011; Student T test) (Fig. 3F, G). HLA-E expression is controlled by GATA3 transcription factor <sup>11</sup>, that is known to be expressed also by lung epithelial cells <sup>26</sup>, <sup>27</sup>. We observed an increase in GATA3 expression in the presence of SP1 (p&lt;0.001; Student T test) (Fig. 4A, B). To be sure that the increase in HLA-E expression in lung epithelial cells transfected with SP1 protein is controlled by GATA3 transcription factor, we treated the cells with a GATA3 inhibitor, the pyrrothiogatain <sup>17</sup>. The treatment with pyrrothiogatain of Beas-2B cells transfected with SP1 decreased the surface expression of HLA-E, reaching the basal levels of HLA-E expression in Beas-2B cells (Fig. 3F, G). The immune-fluorescence staining revealed a clear surface expression of HLA-E molecules in the presence of intracellular SP1, that is reduced with the addition of GATA3 inhibitor (Fig. 4C). To evaluate if SP1 peptides might be presented by HLA-E molecules, we performed a MHCI binding predictions, made on 4/26/2020 using the IEDB analysis resource ANN aka NetMHC (ver. 4.0) tool. We observed that a 8mer peptide in the SP1 domain (270-277, LQPRTFLL) showed a high affinity mainly for the HLA-E*0101 binding groove (IC50: 0.02) and in a lower extent for HLA-E*0301 allele (IC50: 0.76) with a similar consensus motif with HLA-E binding (VMAPRTVLL) <sup>13</sup>. We might speculate that the induction of GATA3 and the binding of SP1 peptide might induce HLA-E membrane expression on lung epithelial cells.</p><p><strong>&nbsp;</strong></p><p><strong>HLA-E / NKG2A/CD94 interaction is responsible for NK cells exhaustion</strong></p><p>Since the functional control of NK cells by HLA-E is possible in the presence of NKG2A/CD94 on NK cells, we evaluated the expression of this receptor on the surface of NK cells. When NK cells were co-cultured with SP1 transfected Beas-2B cells, we observed an increase in the expression of the inhibitory receptor NKG2A/CD94 (p&lt;0.001; Student t test) (Fig. 5A, B). To be sure that the resulting inactivity of NK cells towards lung epithelial cells expressing SP1 was determined by HLA-E/NKG2A interaction, we treated the cell culture with anti-HLA-E or anti-NKG2A/CD94 antibodies. We incubated NK cells for four hours with SP1 transfected Beas-2B cell line and evaluated the expression of CD107a in the presence or absence of anti-HLA-E or anti-NKG2A antibodies. We observed a decrease in CD107a positive NK cells in the culture condition with SP1 transfected Beas-2B cells (p&lt;0.001; Student T test) (Fig. 5C, D), that is partly recovered by the treatment with anti-HLA-E antibody (p=0.037; Student T test) (Fig. 5C, D), while it is totally regained with anti-NKG2A/CD94 antibody (Fig. 5C, D). </p><p>&nbsp;</p><p>&nbsp;</p><p><br></p><h2>References</h2><p>1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Zhu, N.<em> et al.</em> A Novel Coronavirus from Patients with Pneumonia in China, 2019. <em>N Engl J Med</em> <strong>382</strong>, 727-733, doi:10.1056/NEJMoa2001017 (2020).</p><p>2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Li, Q.<em> et al.</em> Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus-Infected Pneumonia. <em>N Engl J Med</em> <strong>382</strong>, 1199-1207, doi:10.1056/NEJMoa2001316 (2020).</p><p>3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Zhou, P.<em> et al.</em> A pneumonia outbreak associated with a new coronavirus of probable bat origin. <em>Nature</em> <strong>579</strong>, 270-273, doi:10.1038/s41586-020-2012-7 (2020).</p><p>4&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Guan, W. J.<em> et al.</em> Clinical Characteristics of Coronavirus Disease 2019 in China. <em>N Engl J Med</em> <strong>382</strong>, 1708-1720, doi:10.1056/NEJMoa2002032 (2020).</p><p>5&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Qin, C.<em> et al.</em> Dysregulation of immune response in patients with COVID-19 in Wuhan, China. <em>Clin Infect Dis</em>, doi:10.1093/cid/ciaa248 (2020).</p><p>6&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Pedersen, S. F. &amp; Ho, Y. C. SARS-CoV-2: a storm is raging. <em>J Clin Invest</em> <strong>130</strong>, 2202-2205, doi:10.1172/JCI137647 (2020).</p><p>7&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Tan, L.<em> et al.</em> Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study. <em>Signal Transduct Target Ther</em> <strong>5</strong>, 33, doi:10.1038/s41392-020-0148-4 (2020).</p><p>8&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Vivier, E., Tomasello, E., Baratin, M., Walzer, T. &amp; Ugolini, S. Functions of natural killer cells. <em>Nat Immunol</em> <strong>9</strong>, 503-510, doi:10.1038/ni1582 (2008).</p><p>9&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Long, E. O., Kim, H. S., Liu, D., Peterson, M. E. &amp; Rajagopalan, S. Controlling natural killer cell responses: integration of signals for activation and inhibition. <em>Annu Rev Immunol</em> <strong>31</strong>, 227-258, doi:10.1146/annurev-immunol-020711-075005 (2013).</p><p>10&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Houchins, J. P., Lanier, L. L., Niemi, E. C., Phillips, J. H. &amp; Ryan, J. C. Natural killer cell cytolytic activity is inhibited by NKG2-A and activated by NKG2-C. <em>J Immunol</em> <strong>158</strong>, 3603-3609 (1997).</p><p>11&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Nguyen, S.<em> et al.</em> HLA-E upregulation on IFN-gamma-activated AML blasts impairs CD94/NKG2A-dependent NK cytolysis after haplo-mismatched hematopoietic SCT. <em>Bone Marrow Transplant</em> <strong>43</strong>, 693-699, doi:10.1038/bmt.2008.380 (2009).</p><p>12&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Miller, J. D.<em> et al.</em> Analysis of HLA-E peptide-binding specificity and contact residues in bound peptide required for recognition by CD94/NKG2. <em>J Immunol</em> <strong>171</strong>, 1369-1375, doi:10.4049/jimmunol.171.3.1369 (2003).</p><p>13&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Braud, V., Jones, E. Y. &amp; McMichael, A. The human major histocompatibility complex class Ib molecule HLA-E binds signal sequence-derived peptides with primary anchor residues at positions 2 and 9. <em>Eur J Immunol</em> <strong>27</strong>, 1164-1169, doi:10.1002/eji.1830270517 (1997).</p><p>14&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Lee, N.<em> et al.</em> HLA-E is a major ligand for the natural killer inhibitory receptor CD94/NKG2A. <em>Proc Natl Acad Sci U S A</em> <strong>95</strong>, 5199-5204, doi:10.1073/pnas.95.9.5199 (1998).</p><p>15&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Lazetic, S., Chang, C., Houchins, J. P., Lanier, L. L. &amp; Phillips, J. H. Human natural killer cell receptors involved in MHC class I recognition are disulfide-linked heterodimers of CD94 and NKG2 subunits. <em>J Immunol</em> <strong>157</strong>, 4741-4745 (1996).</p><p>16&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Marci, R.<em> et al.</em> Presence of HHV-6A in Endometrial Epithelial Cells from Women with Primary Unexplained Infertility. <em>PLoS One</em> <strong>11</strong>, e0158304, doi:10.1371/journal.pone.0158304 (2016).</p><p>17&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Nomura, S.<em> et al.</em> Pyrrothiogatain acts as an inhibitor of GATA family proteins and inhibits Th2 cell differentiation in vitro. <em>Sci Rep</em> <strong>9</strong>, 17335, doi:10.1038/s41598-019-53856-1 (2019).</p><p>18&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Bortolotti, D.<em> et al.</em> HHV-6A infection of endometrial epithelial cells affects immune profile and trophoblast invasion. <em>Am J Reprod Immunol</em> <strong>82</strong>, e13174, doi:10.1111/aji.13174 (2019).</p><p>19&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Bortolotti, D.<em> et al.</em> DNA Sensors' Signaling in NK Cells During HHV-6A, HHV-6B and HHV-7 Infection. <em>Front Microbiol</em> <strong>11</strong>, 226, doi:10.3389/fmicb.2020.00226 (2020).</p><p>20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Inngjerdingen, M., Rolstad, B. &amp; Ryan, J. C. Activating and inhibitory Ly49 receptors modulate NK cell chemotaxis to CXC chemokine ligand (CXCL) 10 and CXCL12. <em>J Immunol</em> <strong>171</strong>, 2889-2895, doi:10.4049/jimmunol.171.6.2889 (2003).</p><p>21&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Rizzo, R.<em> et al.</em> Altered natural killer cells' response to herpes virus infection in multiple sclerosis involves KIR2DL2 expression. <em>J Neuroimmunol</em> <strong>251</strong>, 55-64, doi:10.1016/j.jneuroim.2012.07.004 (2012).</p><p>22&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Zitvogel, L. Dendritic and natural killer cells cooperate in the control/switch of innate immunity. <em>J Exp Med</em> <strong>195</strong>, F9-14, doi:10.1084/jem.20012040 (2002).</p><p>23&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Mwimanzi, P., Markle, T. J., Ueno, T. &amp; Brockman, M. A. Human leukocyte antigen (HLA) class I down-regulation by human immunodeficiency virus type 1 negative factor (HIV-1 Nef): what might we learn from natural sequence variants? <em>Viruses</em> <strong>4</strong>, 1711-1730, doi:10.3390/v4091711 (2012).</p><p>24&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Gornalusse, G. G.<em> et al.</em> HLA-E-expressing pluripotent stem cells escape allogeneic responses and lysis by NK cells. <em>Nat Biotechnol</em> <strong>35</strong>, 765-772, doi:10.1038/nbt.3860 (2017).</p><p>25&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Stevens, J., Joly, E., Trowsdale, J. &amp; Butcher, G. W. Peptide binding characteristics of the non-classical class Ib MHC molecule HLA-E assessed by a recombinant random peptide approach. <em>BMC Immunol</em> <strong>2</strong>, 5, doi:10.1186/1471-2172-2-5 (2001).</p><p>26&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Caramori, G.<em> et al.</em> Expression of GATA family of transcription factors in T-cells, monocytes and bronchial biopsies. <em>Eur Respir J</em> <strong>18</strong>, 466-473, doi:10.1183/09031936.01.00040701 (2001).</p><p>27&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Zheng, W. &amp; Flavell, R. A. The transcription factor GATA-3 is necessary and sufficient for Th2 cytokine gene expression in CD4 T cells. <em>Cell</em> <strong>89</strong>, 587-596, doi:10.1016/s0092-8674(00)80240-8 (1997).</p><p>28&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Zheng, M.<em> et al.</em> Functional exhaustion of antiviral lymphocytes in COVID-19 patients. <em>Cell Mol Immunol</em> <strong>17</strong>, 533-535, doi:10.1038/s41423-020-0402-2 (2020).</p><p>29&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Antonioli, L., Fornai, M., Pellegrini, C. &amp; Blandizzi, C. NKG2A and COVID-19: another brick in the wall. <em>Cell Mol Immunol</em>, doi:10.1038/s41423-020-0450-7 (2020).</p><p>30&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Yaqinuddin, A. &amp; Kashir, J. Innate immunity in COVID-19 patients mediated by NKG2A receptors, and potential treatment using Monalizumab, Cholroquine, and antiviral agents. <em>Med Hypotheses</em> <strong>140</strong>, 109777, doi:10.1016/j.mehy.2020.109777 (2020).</p><p>31&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hoffmann, M.<em> et al.</em> SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor. <em>Cell</em> <strong>181</strong>, 271-280 e278, doi:10.1016/j.cell.2020.02.052 (2020).</p><p>32&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;de Lang, A., Osterhaus, A. D. &amp; Haagmans, B. L. Interferon-gamma and interleukin-4 downregulate expression of the SARS coronavirus receptor ACE2 in Vero E6 cells. <em>Virology</em> <strong>353</strong>, 474-481, doi:10.1016/j.virol.2006.06.011 (2006).</p><p>33&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Frieman, M., Heise, M. &amp; Baric, R. SARS coronavirus and innate immunity. <em>Virus Res</em> <strong>133</strong>, 101-112, doi:10.1016/j.virusres.2007.03.015 (2008).</p><p>34&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Anfossi, N.<em> et al.</em> Human NK cell education by inhibitory receptors for MHC class I. <em>Immunity</em> <strong>25</strong>, 331-342, doi:10.1016/j.immuni.2006.06.013 (2006).</p><p>35&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Grote, D.<em> et al.</em> Gata3 acts downstream of beta-catenin signaling to prevent ectopic metanephric kidney induction. <em>PLoS Genet</em> <strong>4</strong>, e1000316, doi:10.1371/journal.pgen.1000316 (2008).</p><p>36&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Ulianich, L.<em> et al.</em> ER stress impairs MHC Class I surface expression and increases susceptibility of thyroid cells to NK-mediated cytotoxicity. <em>Biochim Biophys Acta</em> <strong>1812</strong>, 431-438, doi:10.1016/j.bbadis.2010.12.013 (2011).</p><p>37&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;van Hall, T.<em> et al.</em> Monalizumab: inhibiting the novel immune checkpoint NKG2A. <em>J Immunother Cancer</em> <strong>7</strong>, 263, doi:10.1186/s40425-019-0761-3 (2019).</p><h2>Acknowledgements</h2><p>We thank University of Ferrara COVID19 grant (RR), University of Ferrara FAR 2018 (RR), 2019 (RR). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. We thank Linda Sartor for writing assistance.&nbsp;</p><p><br></p></body></html>