<html><head><title>Assigning scRNA-seq data to known and de novo cell types using CellAssign</title></head><body><h1>Assigning scRNA-seq data to known and de novo cell types using CellAssign</h1>Authors: Allen W Zhang, Kieran R Campbell, Sohrab P Shah<br>Abstract: <p>Assigning cells to known or de-novo cell types is an important step in the analysis of single-cell RNA-sequencing (scRNA-seq) data. This protocol outlines how to use the CellAssign R package to accomplish this.</p><br>Keywords: scRNA-seq,cell types,cell type assignment,RNA-seq,microenvironment,cell type composition<br><h2>Introduction</h2><p>Assigning cells to known or de-novo cell types is an important step in the analysis of single-cell RNA-sequencing (scRNA-seq) data. CellAssign is a recently published statistical model that models the over-expression of a set of marker genes for each pre-specified cell type. CellAssign then computes a probability that each cell is of a given cell type, or is of an “unknown” cell type (does not reflect the expected expression of any of the specified cell types). These assignments can then be used to (i) study the cell type composition of each sample, (ii) focus in on a given cell type for further analysis (e.g. unsupervised clustering), or (iii) remove nuisance cell types.</p><h2>Equipment</h2><p>Software:</p><p>R computing environment (&gt; version 3.5)</p><p>The devtools R package</p><p>The cellassign R package (<a href="https://github.com/Irrationone/cellassign" target="_blank">https://github.com/Irrationone/cellassign</a>)</p><p><br></p><p><br></p><h2>Procedure</h2><p>1. Install Tensorflow within R:</p><p>install.packages("tensorflow")</p><p>tensorflow::install_tensorflow()</p><p><br></p><p>2. Install cellassign by running</p><p>devtools::install_github(“<a href="https://github.com/Irrationone/cellassign" target="_blank">Irrationone/cellassign</a>”)</p><p>and load by calling library(cellassign)</p><p><br></p><p>3. Prepare single-cell expression data in the form of a SingleCellExperiment object</p><p><a href="https://bioconductor.org/packages/release/bioc/html/SingleCellExperiment.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/SingleCellExperiment.html</a></p><p>We will assume this object is “sce”. In rowData(sce) should be fields “ID”, corresponding to ensembl gene ID, and “Symbol”, corresponding to HGNC symbol.</p><p><br></p><p>4. Compute size factors using scran</p><p>sce &lt;- computeSumFactors(sce)</p><p><br></p><p>5. Specify marker gene data</p><p>This is in the form of a list, where the names of the list are the cell types and the contents are marker genes for the cell types. An example can be found in the CellAssign package by calling data(example_TME_markers). As a simple example, we can create one for T cells and epithelial cells:</p><p>marker_list &lt;- &nbsp;list(t_cells = c(“PTPRC”, “CD2”), epithelial = “EPCAM”) </p><p>Note that there is no requirement marker genes should be mutually exclusive or not expressed in other cell types.</p><p><br></p><p>6. Turn marker list into binary matrix using marker_list_to_mat</p><p>marker_mat &lt;- marker_list_to_mat(marker_list)</p><p>Optional: an “unknown” cell type may be included by passing include_other = TRUE to marker_list_to_mat</p><p><br></p><p>7. Match IDs to rows of the SingleCellExperiment</p><p>mm &lt;- match(rownames(marker_mat), rowData(sce)$Symbol) &nbsp;</p><p><br></p><p>8. Subset SingleCellExperiment to markers only</p><p>sce_marker &lt;- sce[mm,]</p><p><br></p><p>9. Run CellAssign</p><p>fit &lt;- cellassign(exprs_obj = sce_marker, </p><p> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;marker_gene_info = marker_mat, </p><p> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;s = sizeFactors(sce_marker))</p><p><br></p><p>Note that covariates can be included at this point by passing an argument named “x” to cellassign. For more information see the vignette below.</p><p><br></p><p>10. View assigned cell types</p><p>print(fit$cell_type)</p><p><br></p><p>11. View maximum likelihood parameter estimates</p><p>print(fit$mle_params)</p><p>This includes the cell assignment probabilities in fit$mle_params$gamma</p><p><br></p><p>For more detailed example see the package vignette at <a href="https://irrationone.github.io/cellassign/introduction-to-cellassign.html" target="_blank">https://irrationone.github.io/cellassign/introduction-to-cellassign.html</a></p><h2>Troubleshooting</h2><p>Common errors include:</p><p>Including cells in the SingleCellExperiment or gene expression matrix passed to “cellassign” that have no counts remaining, after subsetting to marker genes only.</p><p>Not subsetting to marker genes only, ie passing a full SingleCellExperiment with all genes to “cellassign”. The marker matrix and expression data passed to cellassign should be for marker genes only.</p><h2>Time Taken</h2><p>Time end-to-end for a beginner user should be around 2 hours.</p><h2>Anticipated Results</h2><p>The resulting object returned by “cellassign” includes cell type assignments and maximum likelihood parameter estimates. This is in the form of a “cellassign_fit” object. This allows users to perform useful downstream analyses such as correlating cell type composition with phenotypes or further unsupervised analysis on cell subsets.</p></body></html>