<html><head><title>Kethoxal-assisted single-stranded DNA sequencing (KAS-seq) for capturing transcription dynamics and enhancer activity</title></head><body><h1>Kethoxal-assisted single-stranded DNA sequencing (KAS-seq) for capturing transcription dynamics and enhancer activity</h1>Authors: Tong Wu, Ruitu Lyu, Chuan He<br>Abstract: <p>We describe a rapid and sensitive labeling of single-stranded DNA for sequencing (KAS-seq), based on a kethoxal-guanine reaction. KAS-seq procedure involves N<sub>3</sub>-kethoxal labeling, DNA isolation, biotinylation, fragmentation and enrichment, library preparation and sequencing. The whole process can be finished in a day. KAS-seq enables rapid measurement of the dynamics of transcriptionally-engaged Pol II, transcribing enhancers, Pol I and Pol III activities, and non-canonical DNA structures involving ssDNA simultaneously <em>in situ</em>, by using as few as 1,000 cells or mice tissues. The robust and tissue-friendly nature coupled with low input material requirement make KAS-seq a method that can be broadly applied. This step-by-step protocol is related to the publication “Kethoxal-assisted single-stranded DNA sequencing captures global transcription dynamics and enhancer activity <em>in situ</em>”<em> </em>in <em>Nature Methods</em>.</p><br>Keywords: single-stranded DNA, transcription, enhancer<br><h2>Introduction</h2><p>Transcription and its regulation determine cell fate and physiological functions, with dysfunctions in transcriptional regulation associated with various human diseases<sup>1</sup>. To understand global transcription regulation, genome-wide sequencing approaches have been developed to analyze the occupancy of RNA polymerases (ChIP-seq)<sup>2</sup>, or detect the presence and level of nascent RNA. Nascent RNA analysis is usually based on run-on assays<sup>3,4</sup>, metabolic labeling<sup>5,6</sup>, and Pol II-associated or chromatin-associated RNA enrichment<sup>7-11</sup>. Although powerful, these methods also have limitations. Run-on-based methods and Pol II-associated RNA enrichment typically require millions of cells as starting materials. Pol II ChIP-seq could not distinguish whether RNA polymerases are simply bound or are actively engaged in transcription<sup>3</sup>. Metabolic labeling may not be able to accurately measure transient and low-abundant RNA species, such as enhancer RNAs (eRNAs), especially when using limited materials with modest sequencing depth. As most RNAs undergo post-transcriptional processing, their levels are indirect readouts that may not accurately reflect transcription dynamics <em>in situ</em>.</p><p>Transcriptionally engaged RNA polymerases resolve DNA double helices and generate single-stranded DNA bubbles. Mapping ssDNA throughout the genome provides a readout of the activity and dynamics of transcriptionally engaged RNA polymerases.</p><p>Here we describe a rapid and sensitive labeling of single-stranded DNA for sequencing (KAS-seq, Figure 1), based on a kethoxal-guanine reaction<sup>12</sup>. We show that KAS-seq simultaneously measures the dynamics of transcriptionally-engaged Pol II, transcribing enhancers, Pol I and Pol III activities, and non-canonical DNA structures involving ssDNA <em>in situ</em>, by using as few as 1,000 cells or mice tissues.</p><h2>Reagents</h2><p>N<sub>3</sub>-kethoxal (Synthesized as previously<sup>12</sup>)</p><p>DMSO (Fisher Scientific, BP231-100)</p><p>DBCO-PEG<sub>4</sub>-biotin (Sigma, 760749)</p><p>Boric acid (Sigma, 15663)</p><p>Potassium hydroxide (Sigma, P5958)</p><p>Tween-20 (Sigma, P9416)</p><p>Phosphate buffered saline (PBS, Thermo, 10010001)</p><p>Phosphate buffered saline, 10× (Thermo, AM9624)</p><p>1 M Tris-HCl buffer (Thermo, 15567027)</p><p>5 M sodium chloride solution (Thermo, AM9759)</p><p>0.5 M EDTA solution (Thermo, AM9206G)</p><p>RNase A (Thermo, 12091039)</p><p>Dynabeads Myone Streptavidin C1 (Thermo, 65001)</p><p>PureLink genomic DNA mini kit (Thermo, K182001) </p><p>Quick-DNA Microprep Plus kit (Zymo, D4074)</p><p>MinElute PCR purification kit (Qiagen, 28804)</p><p>DNA Clean &amp; Concentrator kit (Zymo, D4043)</p><p>Tagmentation DNA Enzyme (Illumina, 15027865)</p><p>Tagmentation DNA Buffer (Illumina, 15027866)</p><p>i5 index primer (Illumina, 20027213)</p><p>i7 index primer (Illumina, 20027213)</p><p>NEBNext Ultra II Q5 Master Mix (NEB, M0544S)</p><p>Accel-NGS Methyl-seq DNA library kit (Swift 30024)</p><p><br></p><h2>Equipment</h2><p>1.5 mL tube</p><p>Nanodrop</p><p>Centrifuge</p><p>Themomixer</p><p>Thermo cycler</p><p>DynaMag-PCR Magnet</p><p>Bioruptor Pico (Diagenode)</p><p>0.65 mL Bioruptor Micotubes (Diagenode, C30010011)</p><h2>Procedure</h2><p><strong><u>1. KAS-seq with mammalian cell cultures</u></strong></p><p><strong><em>Labeling and DNA isolation</em></strong></p><p>1. Prepare 500 mM N<sub>3</sub>-kethoxal stock solution using DMSO. Then prepare the labeling medium by diluting the N<sub>3</sub>-kethoxal solution into pre-warmed (37 ˚C) cell culture medium to a final concentration of 5 mM. It is critical to pre-warm the medium to facilitate N<sub>3</sub>-kethoxal dissolution.</p><p>2. Incubate 1-5 million cells in the labeling medium for 10 min at 37 ˚C. For adhesive cells, apply the labeling medium to the cell culture directly in dishes and incubate cells for 10 min at 37 ˚C. For suspension cells, suspend cells in the labeling medium and incubate them for 10 min at 37 ˚C.</p><p>3. Harvest cells after the incubation. Isolate total DNA from cells by using PureLink genomic DNA mini kit (Thermo K182001). Elute DNA by using 50 µL 25 mM K<sub>3</sub>BO<sub>3</sub> (pH 7.0). <strong>[Stop point 1]</strong></p><p><strong><em>Biotinylation and purification</em></strong></p><p>4. Prepare the click reaction mixture as the follows.</p><p><em>2 µg total DNA diluted in 85µL of 25 mM K</em><sub><em>3</em></sub><em>BO</em><sub><em>3</em></sub><em> (pH 7.0)</em></p><p><em>10 µL 10× PBS</em></p><p><em>5 µL 20 mM DBCO-PEG</em><sub><em>4</em></sub><em>-biotin (DMSO solution, Sigma 760749)</em></p><p>&nbsp;&nbsp;&nbsp;Incubate the mixture at 37 ˚C for 1.5 h with shaking at 500 rpm to facilitate the “click” reaction.</p><p>5. Add 5 µL RNase A (Thermo 12091039) to the reaction mixture. Incubate the mixture at 37 ˚C for 15 min with shaking at 500 rpm.</p><p>6. Purify DNA from the reaction mixture by DNA Clean &amp; Concentrator kit (Zymo D4043). Elute DNA by using 50 µL 25 mM K<sub>3</sub>BO<sub>3</sub> (pH 7.0). <strong>[Stop point 2]</strong></p><p><strong><em>Enrichment of N</em></strong><sub><strong><em>3</em></strong></sub><strong><em>-kethoxal-modified DNA</em></strong></p><p>7. Dilute 1 µg biotinylated DNA in 100 µL 25 mM K<sub>3</sub>BO<sub>3</sub> (pH 7.0). Fragment the DNA by sonication using Bioruptor Pico under 30s-on /30s-off setting for 30 cycles. The size of fragmented DNA ranges from 150-350 bp.</p><p>8. Save 5 µL of the fragmented DNA as input. Use the remaining 95 µL for enrichment.</p><p>9. Wash 10 µL Dynabeads Myone Streptavidin C1 (Thermo 65001) 3 times with 50 µL 1× binding and wash buffer (5 mM Tris-HCl pH 7.4, 0.5 mM EDTA, 1 M NaCl, 0.05% Tween-20). Re-suspend pre-washed beads in 95 µL 2× binding and wash buffer (10 mM Tris-HCl pH 7.4, 1 mM EDTA, 2 M NaCl, 0.1% Tween-20).</p><p>10. Mix the beads with 95 µL fragmented DNA from step 8. Incubate the mixture at room temperature for 15 min with gentle rotation. After incubation, place the mixture on a magnetic rack to remove the supernatant and wash beads 5 times with 100 µL 1× binding and wash buffer.</p><p>11. Re-suspend the beads in 15 µL nuclease-free water and heat it at 95 ˚C for 10 min to elute enriched DNA. Then put the mixture on a magnetic rack, and transfer the supernatant to a new labeled tube.</p><p>12. In parallel with the previous step, heat the saved input at 95 ˚C for 10 min. Then, put both the input and the eluted samples on ice immediately. <strong>[Stop point 3]</strong></p><p><strong><em>Library preparation and sequencing</em></strong></p><p>13. Perform library construction by using the Accel-NGS Methyl-seq DNA library kit (Swift 30024). Perform PCR amplification for 7-8 cycles for the input samples and 12-14 cycles for the enriched samples. The library size should range from 200-500 bp.</p><p>14. Sequence libraries on Illumina platforms by using single-end mode, aiming to get 30 million reads per sample</p><p><strong>DNAs can be stored at -20 ˚C at the noted stop points if needed.</strong></p><p><br></p><p><strong><u>2. KAS-seq with mouse liver </u></strong></p><p><strong><em>Labeling and DNA isolation</em></strong></p><p>1. Homogenize mouse liver tissue to a cell suspension in ice-cold PBS by using a dounce homogenizer or a pellet pestle.</p><p>2. Spin the cell suspension at 100 g for 15 seconds to sediment and remove potential large tissue pieces at the bottom of the tube.</p><p>3. Spin the cell suspension at 800 g for 5 min. Remove the supernatant and save the cell pellet at the bottom of the tube for labeling.</p><p>4. Suspend 1-5 million cells in the labeling medium for 10 min at 37 ˚C (refer to step 1 in the protocol for mammalian cell cultures for labeling medium preparation).</p><p>5. Isolate total DNA from cells by using PureLink genomic DNA mini kit (Thermo K182001). Elute DNA by using 50 µL 25 mM K<sub>3</sub>BO<sub>3</sub> (pH 7.0). <strong>[Stop point 1]</strong></p><p><strong><em>Perform Biotinylation and purification, Enrichment of N</em></strong><sub><strong><em>3</em></strong></sub><strong><em>-kethoxal-modified DNA, Library preparation and sequencing according to the protocol for mammalian cell cultures.</em></strong></p><p><strong> </strong></p><p><strong><u>3. Low-input KAS-seq</u></strong></p><p><strong><em>Labeling and DNA isolation</em></strong></p><p>1. Label 1,000, 5,000, or 10,000 cells according to the protocol for bulk mammalian cell cultures.</p><p>2. Isolate DNA from cells by using the Quick-DNA Microprep Plus kit (Zymo D4074). Elute DNA by using 43 µL 25 mM K<sub>3</sub>BO<sub>3</sub> (pH 7.0).</p><p><strong><em>Biotinylation and purification</em></strong></p><p>3. Prepare the click reaction mixture as follows.</p><p><em>42.5 µL DNA in 25 mM K</em><sub><em>3</em></sub><em>BO</em><sub><em>3</em></sub><em> (pH 7.0)</em></p><p><em>5 µL 10× PBS</em></p><p><em>2.5 µL 20 mM DBCO-PEG</em><sub><em>4</em></sub><em>-biotin (DMSO solution, Sigma 760749)</em></p><p>&nbsp;&nbsp;&nbsp;Incubate the mixture at 37 ˚C for 1.5 h with shaking at 500 rpm to facilitate the “click” reaction.</p><p>4. Add 2.5 µL RNase A (Thermo 12091039) to the reaction mixture. Incubate the mixture at 37 ˚C for 5 min.</p><p>5. Purify DNA from the reaction mixture by DNA Clean &amp; Concentrator kit (Zymo D4043). Elute DNA by using 25 µL 25 mM K<sub>3</sub>BO<sub>3</sub> (pH 7.0).</p><p><strong><em>Tagmentation and enrichment</em></strong></p><p>6. Perform tagmentation on biotinylated DNA by using Tagmentation DNA Enzyme (Illumina 15027865). For 1,000 cells, 23.5 µL DNA was mixed with 1.5 µL enzyme and 25 µL Tagmentation DNA Buffer (Illumina 15027866). For 5,000 cells, 23 µL DNA was mixed with 2 µL enzyme and 25 µL Tagmentation DNA Buffer. For 10,000 cells, 20 µL DNA was mixed with 5 µL enzyme and 25 µL Tagmentation DNA Buffer.</p><p>7. Incubate the tagmentation mixture at 37 ˚C for 30 min with shaking at 500 rpm.</p><p>8. Purify DNA from the reaction mixture by DNA Clean &amp; Concentrator kit (Zymo D4043). Elute DNA by using 55 µL 25 mM K<sub>3</sub>BO<sub>3</sub> (pH 7.0).</p><p>9. Save 5 µL of the tagmented DNA as input. Use the remaining 50 µL for enrichment.</p><p><strong><em>Enrichment of N</em></strong><sub><strong><em>3</em></strong></sub><strong><em>-kethoxal-modified DNA </em></strong></p><p>10. Wash 5 µL Dynabeads Myone Streptavidin C1 (Thermo 65001) 3 times with 25 µL 1× binding and wash buffer (5 mM Tris-HCl pH 7.4, 0.5 mM EDTA, 1 M NaCl, 0.05% Tween-20). Re-suspend pre-washed beads in 50 µL 2× binding and wash buffer (10 mM Tris-HCl pH 7.4, 1 mM EDTA, 2 M NaCl, 0.1% Tween-20).</p><p>11. Mix the beads with 50 µL fragmented DNA from step 9. Incubated the mixture at room temperature for 15 min with gentle rotation. After incubation, place the mixture on a magnetic rack to remove the supernatant and wash beads 5 times with 100 µL 1× binding and wash buffer.</p><p><strong><em>PCR amplification and library sequencing</em></strong></p><p>12. Adjust the volume of input to 20 µL by adding 15 µL nuclease-free water to the 5 µL input saved in step 9. Re-suspend the washed beads in 20 µL nuclease-free water.</p><p>13. Prepare the PCR reaction mix as follows:</p><p><em>20 µL input DNA or beads suspension</em></p><p><em>2.5 µL i5 index primer (Illumina 20027213)</em></p><p><em>2.5 µL i7 index primer (Illumina 20027213)</em></p><p><em>25 µL NEBNext Ultra II Q5 Master Mix (NEB M0544S)</em></p><p>14. Perform PCR reaction in a thermocycler as follows:</p><p><em>1 cycle:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;5 min&nbsp;&nbsp;&nbsp;72 ˚C</em></p><p><em>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;10 min&nbsp;&nbsp;95 ˚C</em></p><p><em>15 cycles:&nbsp;&nbsp;10 sec&nbsp;&nbsp;&nbsp;98 ˚C</em></p><p><em>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;30 sec&nbsp;&nbsp;&nbsp;60 ˚C</em></p><p><em>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1 min&nbsp;&nbsp;&nbsp;&nbsp;72 ˚C</em></p><p>15. Remove the beads from the enriched samples by using a magnetic rack. Purify the library by using MinElute PCR purification kit (Qiagen 28804). The library size should range from 200-600 bp.</p><p>16. Sequence libraries on Illumina platforms by using single-end mode with 30 million reads per sample.</p><h2>Time Taken</h2><p>Labeling, cell collection, and DNA isolation: ~1 h</p><p>Biotinylation: ~2 h</p><p>Fragmentation and enrichment: ~1 h</p><p>Library preparation: ~3 h</p><p><br></p><h2>Anticipated Results</h2><p>1. Sequencing results show traditional KAS-seq features. Signals are enriched at gene-coding regions, with sharp and strong peaks on TSS, broad peaks on the gene body and terminal regions (Figure 3).</p><p>2. KAS-seq signals show up in some enhancers.</p><p>3. KAS-seq signals show up in some predicted non-B form ssDNA-containing regions.</p><h2>References</h2><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Schwanhäusser, B.<em> et al.</em> Global quantification of mammalian gene expression control. <em>Nature</em> <strong>473</strong>, 337-342 (2011).</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Kim, T.-K.<em> et al.</em> Widespread transcription at neuronal activity-regulated enhancers. <em>Nature</em> <strong>465</strong>, 182-187 (2010).</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Core, L.J., Waterfall, J.J. &amp; Lis, J.T. Nascent RNA Sequencing Reveals Widespread Pausing and Divergent Initiation at Human Promoters. <em>Science</em> <strong>322</strong>, 1845-1848 (2008).</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Kwak, H., Fuda, N.J., Core, L.J. &amp; Lis, J.T. Precise Maps of RNA Polymerase Reveal How Promoters Direct Initiation and Pausing. <em>Science</em> <strong>339</strong>, 950-953 (2013).</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Fuchs, G.<em> et al.</em> 4sUDRB-seq: measuring genomewide transcriptional elongation rates and initiation frequencies within cells. <em>Genome Biology</em> <strong>15</strong>, R69 (2014).</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Schwalb, B.<em> et al.</em> TT-seq maps the human transient transcriptome. <em>Science</em> <strong>352</strong>, 1225-1228 (2016).</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Churchman, L.S. &amp; Weissman, J.S. Nascent transcript sequencing visualizes transcription at nucleotide resolution. <em>Nature</em> <strong>469</strong>, 368-373 (2011).</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Christopher, Ramachandran, S. &amp; Henikoff, S. Nucleosomes Are Context-Specific, H2A.Z-Modulated Barriers to RNA Polymerase. <em>Molecular Cell</em> <strong>53</strong>, 819-830 (2014).</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Nojima, T.<em> et al.</em> Mammalian NET-Seq Reveals Genome-wide Nascent Transcription Coupled to RNA Processing. <em>Cell</em> <strong>161</strong>, 526-540 (2015).</p><p>10.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Mayer, A.<em> et al.</em> Native Elongating Transcript Sequencing Reveals Human Transcriptional Activity at Nucleotide Resolution. <em>Cell</em> <strong>161</strong>, 541-554 (2015).</p><p>11.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hirabayashi, S.<em> et al.</em> NET-CAGE characterizes the dynamics and topology of human transcribed cis-regulatory elements. <em>Nature Genetics</em> 51, 1369-1379 (2019).</p><p>12.<strong>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Weng, X.<em> et al.</em> Keth-seq for transcriptome-wide RNA structure mapping. <em>Nature Chemical Biology</em> (2020).</p></body></html>