<html><head><title>QMAP-Seq: Quantitative and Multiplexed Analysis of Phenotype by Sequencing</title></head><body><h1>QMAP-Seq: Quantitative and Multiplexed Analysis of Phenotype by Sequencing</h1>Authors: Sonia Brockway, Geng Wang, Jasen M. Jackson, David R. Amici, Seesha R. Takagishi, Matthew R. Clutter, Elizabeth T. Bartom, Marc L. Mendillo<br>Abstract: <p>Chemical-genetic interaction profiling in model organisms has proven powerful in providing insights into compound mechanism of action and gene function. However, identifying chemical-genetic interactions in mammalian systems has been limited to low-throughput or computational methods. Here, we develop Quantitative and Multiplexed Analysis of Phenotype by Sequencing (QMAP-Seq), which leverages next-generation sequencing for pooled high-throughput chemical-genetic profiling. We apply QMAP-Seq to investigate how cellular stress response factors affect therapeutic response in cancer. Using minimal automation, we treat pools of 60 cell types—comprising 12 genetic perturbations in five cell lines—with 1,440 compound-dose combinations, generating 86,400 chemical-genetic measurements. QMAP-Seq produces precise and accurate quantitative measures of acute drug response comparable to gold standard assays, but with increased throughput at lower cost. Moreover, QMAP-Seq reveals clinically actionable drug vulnerabilities and functional relationships involving these stress response factors, many of which are activated in cancer. Thus, QMAP-Seq provides a broadly accessible and scalable strategy for chemical-genetic profiling in mammalian cells.</p><br>Keywords: chemical-genetic profiling, pooled screening, next-generation sequencing<br><h2>Reagents</h2><p>RPMI-1640 (Gibco, #11875119)</p><p>Tet System Approved Fetal Bovine Serum (Clontech, #631106)</p><p>Penicillin/Streptomycin (Gibco, #15140122)</p><p>Doxycycline (Clontech, # 631311)</p><p>PBS (Fisher Scientific, #20-012-050)</p><p>Accumax (Innovative Cell Technologies, #AM105)</p><p>10 cm tissue culture plates</p><p>15 cm tissue culture plates</p><p>96-well tissue culture plates</p><p>10x Taq DNA Polymerase Buffer (Invitrogen, #18067017)</p><p>IGEPAL CA-630 (Sigma-Aldrich, #I8896)</p><p>TWEEN 20 (Sigma-Aldrich, P9416)</p><p>Proteinase K (Qiagen, #19133)</p><p>Nuclease-free water (Qiagen, #129115)</p><p>Foil adhesive</p><p>5 mL syringe</p><p>21G x 1" needle</p><p>27G x 1/2" needle</p><p>96-well PCR plates</p><p>Clear adhesive (Applied Biosystems, #4311971)</p><p>10 mM dNTP Mix (Invitrogen, #10297-018)</p><p>50 mM MgCl<sub>2</sub></p><p>DMSO (Sigma-Aldrich, #D8418)</p><p>P5 Primer Mix</p><p>P7 Primer</p><p>Platinum Taq DNA Polymerase (Invitrogen, #10966034)</p><p>E-Gel 96 2% Agarose gels (Invitrogen, #G700802)</p><p>Quick-Load 100 bp DNA Ladder (NEB, #N0467S)</p><p>Qubit dsDNA HS Assay Kit (Invitrogen, #Q32854)</p><p>QIAquick PCR Purification Kit (Qiagen, #28106)</p><p>Purification Module with Magnetic Beads (Lexogen, #022.96)</p><p>PhiX (Illumina, #FC-110-3001)</p><p>10 mM Tris, pH 8</p><p>NovaSeq 6000 S1 Reagent Kit, 200 Cycles, 1.3B Reads (Illumina, #20012864) or similar</p><h2>Equipment</h2><p>EL406 Microplate Washer Dispenser (BioTek Instruments) or similar</p><p>Multichannel pipets</p><p>Oven or water bath</p><p>Thermocycler</p><p>NovaSeq 6000 (Illumina) or another next-generation sequencing instrument</p><h2>Procedure</h2><p><strong>Induction, Seeding, and Compound Treatment</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Thaw pooled cell lines (comprised of desired mix of barcoded doxycycline-inducible Cas9 cell lines with desired genetic perturbations) in media containing 100 ng/mL doxycycline.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Three days later, expand cells and refresh doxycycline.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Three more days later, seed 5,000 pooled cells in a volume of 100 mL in 96-well plates, while maintaining doxycycline induction.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;The next day, add compounds over a four-point concentration range in duplicate.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Treat cells with compounds for 72 hours.</p><p><br></p><p><strong>Lysis of Cell Spike-In Standards</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Thaw cell spike-in standards (comprised of five cell lines each possessing a different sgNT sequence representing a three-fold dilution cell concentration range) at room temperature for 5 minutes.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Resuspend cell spike-in standards in Lysis Buffer (10% 10x Taq DNA Polymerase Buffer (Invitrogen, #18067017), 0.45% IGEPAL CA-630 (Sigma-Aldrich, #I8896), 0.45% TWEEN 20 (Sigma-Aldrich, P9416), 10% Proteinase K (Qiagen, #19133), 79.1% Nuclease-Free Water (Qiagen, #129115)) to achieve desired concentration of total cell spike-in standards/mL. Customize spike-in cell numbers for each experiment to cover the expected range of cell numbers for any individual perturbation at the time of cell lysis.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Homogenize cells using a 5 mL syringe and a 21G x 1” needle three times followed by a 27G x ½” needle three times.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Incubate cells in 60°C water bath for 1 hour, pipetting up and down every 20 minutes.</p><p><br></p><p><strong>Lysis of Compound-Treated Cells</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;After 72 hours of compound treatment, wash compound-treated cells with 100 mL PBS.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Add 50 mL Lysis Buffer containing desired concentration of total cell spike-in standards/mL using multichannel pipet.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Cover plates with foil adhesive.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Incubate plates in 60°C oven for 1 hour.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Transfer cell lysates to PCR plates using multichannel pipet.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Inactivate Proteinase K from Lysis Buffer at 95ºC for 15 minutes using thermocycler.</p><p><br></p><p><strong>Sequencing Library Preparation</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Set up PCRs to amplify a 359-367 bp fragment (depending on length of stagger) containing the sgRNA and cell line barcode with a unique set of P5 and P7 primers for each well:</p><p>6 μL 10x Taq DNA Polymerase Buffer</p><p>1.2 μL 10 mM dNTP Mix</p><p>1.8 μL 50 mM MgCl<sub>2</sub></p><p>0.6 μL DMSO (Sigma-Aldrich, #D8418)</p><p>6 μL P5 5001-5016 Primer Mix (mix of 0-8nt staggered primers) (1 μM)</p><p>6 μL P7 7001-7096 Primer (1 μM)</p><p>8 μL Lysate (Proteinase K inactivated again)</p><p>30.16 μL Water</p><p>0.24 μL Platinum Taq DNA Polymerase (Invitrogen, #10966034)</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;To reduce the likelihood of PCR jackpot effects, transfer half of the PCR volume from each reaction to a second set of PCR plates.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Run PCRs in thermocycler:</p><p>94ºC for 4 minutes</p><p>94ºC for 30 seconds, 60ºC for 30 seconds, 72ºC for 30 seconds (29 cycles total)</p><p>72ºC for 15 minutes</p><p>4ºC hold</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Recombine technical duplicates.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Verify library size by running 5 mL of PCR products on E-Gel 96 2% Agarose gels (Invitrogen, #G7008-02).</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Pool an equal volume (6 mL) of each PCR product together.</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;Measure the concentration of the pooled PCR products using the Qubit dsDNA HS Assay Kit (Invitrogen, #Q32854).</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Purify pooled PCR products using the QIAquick PCR Purification Kit (Qiagen, #28106) with sufficient PCR purification columns to avoid exceeding the maximum binding capacity of each column.</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;Measure the concentration of the purified sample using the Qubit dsDNA HS Assay Kit (Invitrogen, #Q32854).</p><p>10.&nbsp;Purify the purified sample a second time using the Purification Module with Magnetic Beads (Lexogen, #022.96).</p><p>11.&nbsp;Measure the final library concentration using the Qubit dsDNA HS Assay Kit (Invitrogen, #Q32854).</p><p><br></p><p><strong>Next-Generation Sequencing</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Dilute the sequencing library to 2.5 nM.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;To increase nucleotide diversity, combine the sequencing library with PhiX to achieve 25% PhiX.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Denature the sequencing library, according to Illumina specifications.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Load the sequencing library on a NovaSeq 6000 (Illumina) using a NovaSeq 6000 S1 Reagent Kit, 200 Cycles, 1.3B Reads (Illumina, #20012864).</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Perform single-end sequencing using the following run parameters:</p><p>Illumina Read 1 Primer: 164 bp read</p><p>Illumina Index 1 Primer: 6 bp read</p><p>Illumina Index 2 Primer: 6 bp read</p><p><br></p><p><strong>Sequencing Data Processing/Data Analysis/Bioinformatics</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Demultiplex individual samples based on i5 and i7 index sequences by running bcl2fastq Conversion Software (Illumina).</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Use QMAP-Seq bioinformatic analysis pipeline (https://codeocean.com/capsule/3022355/tree/v1 or https://github.com/mendillolab/QMAP-Seq) to extract sgRNA and cell line barcode sequences from Read 1 sequences, count the number of reads for each cell line-sgRNA pair, interpolate cell number from sequencing reads using sample-specific standard curves, and normalize the number of cells for a compound-treated sample to the median number of cells for the DMSO-treated samples.</p></body></html>