<html><head><title>Single cell high-throughput qRT-PCR protocol</title></head><body><h1>Single cell high-throughput qRT-PCR protocol</h1>Authors: Sirisha Achanta, Rajanikanth Vadigepalli<br>Abstract: <p>Single cell high-throughput qRT-PCR protocol combines high sensitivity technique of single cell qPCR with high-throughput qPCR technology that can generate data from 96 samples and 96 genes in a single experiment. It can be adapted for various sample types- cell culture, tissue samples and extracted RNA (10 pg) and measured on traditional qPCR and high-throughput qPCR platforms. The workflow is comprised of four steps – cell lysis, reverse transcription, pre-amplification and qPCR. Key features of this protocol are; processing low input samples directly to reverse transcription without RNA extraction which minimizes sample loss, pre-amplification enables amplification of cDNA from single cells to detectable levels for qPCR and&nbsp;measuring up to 400 genes from a single cell sample/10 pg of RNA (starting material). Robust, reproducible and versatile this protocol can be adapted to several upstream and downstream techniques.</p><p><br></p><br>Keywords: Single cell, qPCR, BioMark, Laser Capture Micro-dissection, high-throughput qRT-PCR,<br><h2>Introduction</h2><p>This protocol has been developed over the past decade in our lab to enable reliable high-throughput detection of gene expression by qRT-PCR in low input samples starting from 10 pg total RNA (single cells). We have used this protocol to measure gene expression in a wide variety of samples types, tissue treatments and disease contexts (<em>See associated publications</em>). The present version of this protocol was developed as part of the National Institutes of Health SPARC project efforts to construct a comprehensive anatomical, molecular and functional map of the peripheral nervous system at the visceral organs. A key goal is to develop an anatomical framework onto which various data sets can be mapped, e.g., spatial location and distribution of hundreds to thousands of single neurons within a tissue context, molecular profiles of spatially-tracked single neurons, etc.</p><p>Primarily designed to process single cells captured using Laser Capture Microdissection (LCM) from fresh frozen tissue, it can be adapted to other low input samples such as cell culture samples (1-1000 cells), and extracted RNA (10 pg-1 ng). The workflow is comprised of four steps – cell lysis, reverse transcription, pre-amplification and qPCR.</p><p>Single cell samples are processed for reverse transcription (RT) directly from lysis without RNA extraction. This enables sample processing with no RNA loss due to the extraction process. Post RT, samples are pre-amplified with PCR primers (1-400 primers). This step selectively amplifies cDNA molecules to detectable levels for qPCR.&nbsp;Samples processed can be quantified on a traditional qPCR instrument using EvaGreen/Sybr green. Alternatively samples can be measured for the primers in pre-amplified cDNA using BioMark HD system (Fluidigm), a high-throughput qRT-PCR platform that enables detection of 96 samples with 96 genes/primers generating close to ten thousand data points per chip run.</p><p>For sample processing using BioMark HD system (Fluidigm) multiple sets of 96 or 48 samples (including positive and negative controls) can be processed through this protocol with multiple sets of 48 or 96 primers based on the type of BioMark Chip used (48.48 Chip, 96.96 Chip). For example; 2 x 96 =192 samples can be pre-amplified for 3 x 96 =288 genes and run on 6 BioMark 96.96 Chips (96 samples and 96 genes each) generating a data set of 55,296 points.</p><p>Standard qPCR training with robust pipetting skills is adequate to use this protocol for sample processing on a traditional qPCR instruments. For sample processing on BioMark HD system, training will be required to load samples on chips and on using BioMark instrument and software.</p><p>Pre-amplification of low input samples prior to qPCR enables robust detection of several hundred transcripts in a single cell. However, this step also limits the number of genes that can be detected to the set of primers used for pre-amplification. Transcripts that are not amplified during the pre-amplification process cannot be detected post processing. Additionally, this step involves PCR reaction where several hundred primers (primers in pre-amplification) interact with the same sample during pre-amplification. This enables dimer formation not only with a single primers forward and reverse sequences but with all the forward and reverse sequences for that experiment. Therefore, primer testing should be done post pre-amplification of positive controls to circumvent this issue.</p><p>The steps detailed below can be applied towards single cell samples obtained using LCM, cell culture samples and extracted RNA. Samples processed through the pre-amplification (Procedure 1.1) can be quantified using traditional qPCR (Procedure 1.2) or BioMark High-throughput qPCR (Procedure 1.3).</p><p><strong>Note:</strong> See supplementary file for a user friendly pdf version of the protocol.</p><h2>Reagents</h2><p>MATERIALS</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;CapSure HS caps&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Life Technologies&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;LCM0214, LCM0215</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;CellsDirect Resuspension and Lysis Buffer&nbsp;&nbsp;&nbsp;&nbsp;Life Technologies&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;11739010</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Superscript VILO cDNA synthesis kit&nbsp;Life Technologies&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;11754-250, 11754050</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;T4 Gene 32 Protein&nbsp;&nbsp;&nbsp;New England Bio Labs M0300S, M0300L</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;TaqMan™ PreAmp Master Mix&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Life Technologies&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;4391128</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Exonuclease I (E.coli) - 3,000 units&nbsp;&nbsp;&nbsp;&nbsp;New England BioLabs M0293S, M0293L</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;DNA Suspension Buffer&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Fisher Scientific/Teknova&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;50-843-204</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;TE Buffer&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Fisher Scientific/Teknova&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;50-843-201</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2X SsoFast EvaGreen Supermix with low ROX&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Bio-Rad&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1725211</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2X Assay Loading Reagent&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Fluidigm&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100-7611 </p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20X DNA Binding Dye Sample Loading Reagent&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Fluidigm&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100-7609&nbsp;</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Control Line Fluid Kit-96.96&nbsp;&nbsp;&nbsp;Fluidigm&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;89000021&nbsp;</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Control Line Fluid Kit-48.48&nbsp;&nbsp;&nbsp;Fluidigm&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;89000020&nbsp;</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;48.48 Dynamic Array™ IFC for Gene Expression&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Fluidigm&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;BMK-M-48.48 </p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;96.96 Dynamic Array™ IFC for Gene Expression&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Fluidigm&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;BMK-M-96.96 </p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;TaqMan Universal PCR Master Mix&nbsp;&nbsp;&nbsp;Thermo Fisher 4364338</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Primer Pairs (200 μM) IDT&nbsp;&nbsp;&nbsp;&nbsp;</p><p>Other required materials:</p><p>o&nbsp;&nbsp;Pipettes, pipette tips</p><p>o&nbsp;&nbsp;96-well qPCR plates</p><p>o&nbsp;&nbsp;PCR plate seals</p><p>o&nbsp;&nbsp;1, 0.5, 0.2 ml molecular grade tubes</p><p>o&nbsp;&nbsp;Molecular grade water</p><p><br></p><h2>Equipment</h2><p>EQUIPMENT</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Plate centrifuge</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Thermocyclers</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Heating Block</p><p>	4.&nbsp;&nbsp;&nbsp;&nbsp;IFC Controllers – Fluidigm </p><p>	5.&nbsp;&nbsp;&nbsp;&nbsp;BioMark HD System – Fluidigm</p><h2>Procedure</h2><p>REAGENT SETUP:</p><p><u>Primer Dilutions</u>- Forward and reverse primers need to be ordered at a stock concentration of 400µM to obtain a forward reverse mix concentration of 200µM. Two additional working dilutions of 100 µM and 20 µM (forward and reverse primer mix).</p><p>a.&nbsp;&nbsp;&nbsp;&nbsp;200 µM, 100 µM (forward and reverse primer mix) – Used in the Pre-amplification step (1.1 STEP 3)</p><p>b.&nbsp;&nbsp;&nbsp;&nbsp;20 µM (forward and reverse primer mix) – Used for BioMark Chip loading (1.3 STEP 2)</p><p>c.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2 µM (forward and reverse primer mix)- Used for traditional qPCR (1.2)</p><p><u>Lysis Buffer</u>- Mix the following components based on the number of samples in the experiment. Calculation shown for 1 and 96 samples with overage :</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For one sample: 0.5 μl of Lysis enhancer (CellDirect kit) + 5 μl of Resuspension buffer (CellDirect kit) for a total of 5.5 μl</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For 96 samples with overage 110 samples: 55 μl of Lysis enhancer (CellDirect kit) + 550 μl of Resuspension buffer (CellDirect kit) for a total of 605μl&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p><u>RNA Dilution series</u>- This is used as a positive control for the experiment and must contain known amounts of RNA obtained from whole tissue and include all conditions tested in the experiment. This is to ensure that the positive control has expression of all the genes measured. Dilution series is also used as a metric to test primer efficiency. For example in a 2X dilution series each point contains two times the starting material as the previous point, therefore if the primer scales accurately there should be 1 Ct delta between 2 dilution points.</p><p>Prepare 2X or 3X RNA dilution series for 6-8 dilution points. The mean sample input should correspond to the middle points in the dilution series.</p><p>For example for a single cell experiment we consider a cell to contain approximately 10pg of RNA:</p><p>2X dilution series – 2pg/ μl, 4pg/ μl, 8pg/ μl, 16pg/ μl, 32pg/ μl and 64pg/ μl.</p><p><u>RT Mix- </u>To be prepared immediately prior to use in STEP 3.<u> </u>Mix the following components based on the number of samples in the experiment. Calculation shown for 96 samples with overage :</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For one sample: 0.30 μl 10X Superscript III Mix (VILO kit) + 0.2 μl of T4 Gene 32 Protein + 1 μl of DNA Suspension Buffer for a total of 1.5 μl</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For 96 samples with overage 110 samples: 33 μl 10X Superscript III Mix (VILO kit) + 22 μl of T4 Gene 32 Protein + 110 μl of DNA Suspension Buffer for a total of 165 μl&nbsp;</p><p><u>500nM Primer pool- </u>Primer pool can be generated and used for multiple experiments. Store at -20⁰C for a year. With 200 μM stock primers up to 400 primer pairs can be included. For generating primer pools with greater than 400 primer pairs primers will have to be ordered at a higher concentration. Below is the description for generating primer pools for 1-200 primers and 201-400 primers.</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1-200 primers – use primers at 100 μM concentration</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Combine 1 μl of each primer in a tube. Bring the volume up to 200 μl with DNA Suspension Buffer. For a primer pool of 96 primers; 96 μl of combined primer volume + 104 μl of DNA Suspension Buffer = 200 μl of primer pool at 500nM concentration.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;201-400 primers – use primers at 200 μM concentration</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Combine 1 μl of each primer in a tube. Bring the volume up to 400 μl with DNA Suspension Buffer. For a primer pool of 300 primers; 300 μl of combined primer volume + 100 μl of DNA Suspension Buffer = 400 μl of primer pool at 500nM concentration.</p><p><u>PreAmp Mix- </u>To be prepared immediately prior to use in STEP 4.<u> </u>Mix the following components based on the number of samples in the experiment. Calculation shown for 96 samples with overage :</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For one sample: 10 μl of TaqMan PreAmp Master Mix of + 1.6 μl of 500 nM primer pool for a total of 11.6μl</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For 96 samples with overage 110 samples: 1100 μl of TaqMan PreAmp Master Mix + 176 μl of 500 nM primer pool&nbsp;for a total of&nbsp;&nbsp;&nbsp;</p><p><u>Exonuclease Mix</u> To be prepared immediately prior to use in STEP 5. Mix the following components based on the number of samples in the experiment. Calculation shown for 96 samples with overage :</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For one sample: 0.8 μl of Exonuclease I reaction buffer 10X + 1.6 μl of Exonuclease I + 5.6 μl of DNA Suspension Buffer for a total of 8 μl</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For 96 samples with overage 110 samples: 88 μl of Exonuclease I reaction buffer 10X + 176 μl of Exonuclease I + 616 μl of DNA Suspension Buffer for a total of 880 μl</p><p><strong>Note:</strong> Catalyzes the removal of nucleotides from linear single-stranded DNA in the 3' to 5' direction. It is used for sample cleanup post pre-amplification to remove unincorporated primers and other single stranded cDNA.</p><p>EQUIPMENT SETUP:</p><p>1-&nbsp;&nbsp;&nbsp;&nbsp;Thermocyclers settings</p><p><strong>a.&nbsp;&nbsp;&nbsp;&nbsp;VILO activation</strong></p><p>65<sup>o</sup>C – 1 minute 30 seconds</p><p>4<sup>o</sup>C – ∞</p><p><strong>b.&nbsp;&nbsp;&nbsp;&nbsp;RT</strong></p><p>25<sup>o</sup>C – 5 minutes</p><p>50<sup>o</sup>C – 30 minutes</p><p>55<sup>o</sup>C – 25 minutes</p><p>60<sup>o</sup>C – 5 minutes</p><p>70<sup>o</sup>C – 10 minutes</p><p>4<sup>o</sup>C – ∞</p><p><strong>c.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Pre-amp 22 cycles</strong></p><p>95<sup>o</sup>C – 10 minutes</p><p>22 cycles of:</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;96<sup>o</sup>C – 5 seconds</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60<sup>o</sup>C – 4 minutes</p><p>4<sup>o</sup>C – ∞</p><p><strong>d.&nbsp;&nbsp;&nbsp;&nbsp;Exonuclease</strong></p><p>37<sup>o</sup>C – 30 minutes</p><p>80<sup>o</sup>C – 15 minutes</p><p>4<sup>o</sup>C – ∞</p><p><br></p><p>PROCEDURE</p><p>1.1 PROCEDURE FOR SAMPLE PRE-AMPLIFICATION:</p><p>Required for low input/single cell samples to selectively amplify cDNA to detectable levels for PCR.</p><p><u>STEP 1: Cell Lysis</u></p><p>o&nbsp;&nbsp;Add 5.5 μl lysis buffer (<em>Reagent Preparation</em>) to the capture surface on the LCM Cap</p><p>o&nbsp;&nbsp;Cover the cap with a 0.2 ml tube, ensure tight seal</p><p>o&nbsp;&nbsp;Incubate at 75 <sup>o</sup>C on a heat block (Cap surface in contact with the heating block)&nbsp;for 15 min</p><p>o&nbsp;&nbsp;Cool Cap and tube on ice for 5 minutes</p><p>o&nbsp;&nbsp;Spin down and transfer cell lysate to PCR tube or plate</p><p><strong>Note:</strong> This step is for processing samples collected using LCM. For cell culture samples lyse cells in tubes instead of LCM cap.</p><p><u>STEP 2: Sample plate setup</u></p><p><strong>Note:</strong> This step is performed on ice</p><p>a.&nbsp;&nbsp;&nbsp;&nbsp;<u>Unknown samples: </u>Transfer lysed cells from LCM caps to PCR plate. If using extracted RNA transfer 1 μl of RNA from sample (10-100pg) add 4.5 μl of lysis buffer (<em>reagent preparation</em>).</p><p>b.&nbsp;&nbsp;&nbsp;&nbsp;<u>Positive control:</u> Transfer 1 μl of RNA standard (<em>Reagent preparation</em>) to PCR plate and add 4.5 μl of lysis buffer.</p><p>c.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<u>Negative control:</u> Add 1 μl of molecular grade water to a single well on the PCR plate and add 4.5 μl of lysis buffer.</p><p><br></p><p><u>STEP 3: Reverse Transcription</u></p><p><strong>Note:</strong> This step is performed on ice</p><p>o&nbsp;&nbsp;Add 1.8 μl of 5X VILO Reaction Mix ( <em>Component of Superscript VILO cDNA synthesis kit</em>) to each sample (unknown sample, positive control and negative control).</p><p>o&nbsp;&nbsp;Spin down for 2 minutes at 2000 rpm and proceed to Thermocycler</p><p>o&nbsp;&nbsp;<strong>Thermocycler : VILO activation</strong></p><p>o&nbsp;&nbsp;Add 1.5 μl of RT mix to each sample (<em>reagent preparation</em>)</p><p>o&nbsp;&nbsp;Spin down for 2 minutes at 2000 rpm and proceed to Thermocycler</p><p>o&nbsp;&nbsp;<strong>Thermocycler : RT </strong></p><p>o&nbsp;&nbsp;Spin down for 2 minutes at 2000 rpm and place on ice</p><p><strong>Note: </strong>This is an optional stopping point. Samples can be stored overnight at 4⁰C or stored at -20⁰C for a year.</p><p><u>STEP 4: Pre-amplification</u></p><p><strong>Note:</strong> This step is performed on ice</p><p>o&nbsp;&nbsp;Add 11.6 μl of PreAmp Mix (<em>reagent preparation</em>) to each sample</p><p>o&nbsp;&nbsp;Spin down for 2 minutes at 2000 rpm and proceed to Thermocycler</p><p>o&nbsp;&nbsp;<strong>Thermocycler : PreAmp22</strong></p><p>o&nbsp;&nbsp;Spin down for 2 minutes at 2000 rpm and place on ice</p><p><strong>Note: </strong>This is an optional stopping point. Samples can be stored overnight at 4⁰C.</p><p><strong>Note: </strong>We consider 22 cycles of pre-amp optimal for single cells collected using LCM. We do not recommended to pre-amplification of samples over 22 cycles due to observed increase in production of non-specific PCR fragments and primer-dimers.</p><p><u>STEP 5: Exonuclease treatment</u></p><p>o&nbsp;&nbsp;Add 8 μl of Exonuclease Mix (<em>reagent preparation</em>) to each sample</p><p>o&nbsp;&nbsp;Spin down for 2 minutes at 2000 rpm and proceed to Thermocycler</p><p>o&nbsp;&nbsp;<strong>Thermocycler : Exonuclease</strong></p><p>o&nbsp;&nbsp;Spin down for 2 minutes at 2000 rpm and place on ice</p><p>o&nbsp;&nbsp;Add 54 μl of TE buffer to each sample</p><p><strong>Note: </strong>This is an optional stopping point. Samples can be stored overnight at 4⁰C or stored at -20⁰C for a year.</p><p><br></p><p>1.2 PROCEDURE FOR TRADITIONAL QPCR:<strong> </strong></p><p>o&nbsp;&nbsp;Samples can be measured using a traditional qPCR instrument using the following reaction mix for a 20 μl qPCR reaction:</p><p>2 μl of Pre-amplified and Exonuclease treated sample + 2 μl of Primer pair (2μM) + 10 μl of TaqMan Universal PCR Master Mix + 6 μl of Molecular grade water for a total of 20 μl.</p><p>o&nbsp;&nbsp;Run a standard 30-35 cycle PCR for gene expression.</p><p><br></p><p>1.3 PROCEDURE FOR RUNNING A BIOMARK CHIP:</p><p><u>STEP 1: Chip sample plate preparation</u></p><p><strong>Note: </strong>Prepare Chip Sample plate on the day of Chip run</p><p>o&nbsp;&nbsp;Prepare Sample Mix </p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For one Sample: &nbsp;4 μl of 2X SsoFast EvaGreen Supermix with Low ROX + 0.4 μl of 20X DNA Binding Dye Sample Loading Reagent </p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For a 48.48 IFC Biomark chip with overage for 55 samples: 220 μl of 2X SsoFast EvaGreen Supermix with Low ROX + 22 μl of 20X DNA Binding Dye Sample Loading Reagent</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;96.96 IFC Biomark chip with overage for 110 samples: 440 μl of 2X SsoFast EvaGreen Supermix with Low ROX + 44 μl of 20X DNA Binding Dye Sample Loading Reagent</p><p><br></p><p>o&nbsp;&nbsp;In a new PCR plate aliquot 4.4ul of the above &nbsp;Sample mix into sample wells in the following order</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;96.96 IFC – fill wells A1 to H12 (all the wells)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;48.48 IFC – fill wells 1 to 6 in each row (A1-A6, B1-B6….H1-H6) </p><p>o&nbsp;&nbsp;Add 3.6 μl of Exonuclease treated sample to the plate with the above reagents</p><p>o&nbsp;&nbsp;Spin down for 2 minutes at 2000 rpm and place on ice</p><p><u>STEP 2: Chip assay plate preparation</u></p><p><strong>Note: </strong>Chip assay plate can be prepared a day before chip run and stored at 4⁰C.</p><p>o&nbsp;&nbsp;In a new PCR plate aliquot 4 μl 2X Assay Loading Reagent into 48 or 96 wells for a 48.48 or 96.96 IFC Biomark chip using the corresponding layout as described in step 1 for the two types of IFC BioMark Chips.</p><p>o&nbsp;&nbsp;Add 4 μl of corresponding primer pairs at 20μM concentration to the wells.</p><p>o&nbsp;&nbsp;Spin down for 2 minutes at 2000 rpm and place on ice</p><p><br></p><p><strong>Note: </strong>The final concentration of each primer pair is 5 μM in the inlet and 500 nM in the reaction chamber.</p><p><u>STEP 3: Priming and Loading BioMark Chip</u></p><p>o&nbsp;&nbsp;Inject control line fluid into each accumulator on the chip (see Figure 1).</p><p>o&nbsp;&nbsp;Place the chip into the IFC Controller MX (for the 48.48 Chip) or the IFC Controller HX (for the 96.96 Chip), then run the <strong>Prime (113x) </strong>script (for the 48.48 Chip) or the <strong>Prime (136x) </strong>script (for the 96.96 Chip).</p><p>o&nbsp;&nbsp;When the <strong>Prime </strong>script has finished, press <strong>Eject </strong>to remove the primed chip from the IFC Controller.</p><p>o&nbsp;&nbsp;Pipette 5 μL of each assay and 5 μL of each sample into their respective inlets on the chip (see Figure 1). </p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Using an 8 channel pipette load column 1 (A1, B1..H1) in the alternate wells starting with the top left well. Similarly load the next 5 columns for 96.96 chip and next 2 for 48.48 chip. For the seventh( 96.96) or the fourth (48.48 chip) column load starting with second from the top on the chip.</p><p>o&nbsp;&nbsp;Return the chip to the IFC Controller.</p><p>o&nbsp;&nbsp;Using the IFC Controller software, run the <strong>Load Mix (113x) </strong>script (for the 48.48 Chip) or <strong>Load Mix (136x<em>) </em></strong>script <em>(</em>for the 96.96 Chip) to load the samples and assays into the chip chambers.</p><p>o&nbsp;&nbsp;When the <strong>Load Mix </strong>script has finished, remove the loaded chip from the IFC Controller.</p><p>o&nbsp;&nbsp;Remove any dust particles or debris from the chip surface using scotch tape.</p><p><br></p><p><u>STEP 4: Data collection software for BioMark HD System</u></p><p>o&nbsp;&nbsp;Double-click the Data Collection Software icon on the desktop to launch the software.</p><p>o&nbsp;&nbsp;Click <strong>Start a New Run</strong>.</p><p>o&nbsp;&nbsp;Check the status bar to verify that the lamp and the camera are ready. Make sure both are green before proceeding. </p><p>o&nbsp;&nbsp;Remove and discard the blue protective film from the bottom of the chip. Place the chip into the reader. Click <strong>Load</strong>.</p><p>o&nbsp;&nbsp;Verify chip barcode and chip type. Click <strong>Next</strong>.</p><p>o&nbsp;&nbsp;Chip Run file: Select <strong>New</strong>. Browse to a file location for data storage. Click <strong>Next</strong>.</p><p>o&nbsp;&nbsp;Application, Reference, Probes: </p><p>o&nbsp;&nbsp;Select Application Type--<strong>Gene Expression </strong>for version 3.1.2 or higher software (for all earlier versions, contact Fluidigm Technical Support).<strong> </strong></p><p>o&nbsp;&nbsp;Select Passive Reference: <strong>ROX</strong>.</p><p>o&nbsp;&nbsp;Select Probe--<strong>Single probe</strong>.</p><p>o&nbsp;&nbsp;Select probe type: <strong>EvaGreen</strong>. Click <strong>Next</strong>.</p><p>o&nbsp;&nbsp;Click <strong>Browse </strong>to find the thermal cycling protocol file.</p><p>	For BioMark HD:</p><p>• GE Fast 48x48 PCR+Melt v2.pcl</p><p>• GE Fast 96x96 PCR+Melt v2.pcl</p></body></html>