<html><head><title>Northern Blot Analysis</title></head><body><h1>Northern Blot Analysis</h1>Authors: Madeleine Huber, Jörg Soppa<br>Abstract: <p>The whole workflow for the quantification of specific transcripts by Northern blot analysis is described in detail, including RNA isolation, probe generation via labelling with Dig-dUTP, hybridization, signal visualization and quantification. </p><p><br></p><br>Keywords: specific transcript detection, RNA, quantification of transcript level, dig-dUTP-probe labeling<br><h2>Introduction</h2><p>For the analysis of (differential) gene expression or for the determination of translational efficiencies it is necessary to quantify the levels of specific transcripts. A variety of methods are available, e.g. qRT-PCR(or RT-qPCR) and Northern blot analyses for single transcripts [1;2]. For highly parallel quantification of many transcripts RNA-Seq or DNA microarray analysis can be used. Northern blot analysis offers several important advantages in comparison to qRT-PCR, i.e. 1) the size of the transcript is determined and only non-degraded full-length transcripts are quantified, 2) it is clarified whether the gene of interest is transcribed into one specific transcript or whether several transcripts of different lengths exist (e.g. monocistronic and polycistronic), and 3) no expensive equipment is needed.</p><h2>Reagents</h2><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Agarose</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;H<sub>2</sub>O<sub>DEPC</sub>. To remove traces of RNases from distilled water, 0.1% (w/v) DEPC (diethyl pyrocarbonate) is added and the solution is stirred for at least one hour. After that, is is autoclaved to revmove the DEPC.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Digoxigenin-11-dUTP (Roche, No. 11573179910)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;LowT dNTP-mix: 1 mM dATP, 0.2 mM dTTP, 1.5 mM dCTP, 1.5 mM dGTP. The concentration of dTTP was reduced in the lowT dNTP-to enable a better incorporation of Dig-dUTP. The total omission of dTTP results in a very low probe yield and cannot be recommended.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;10x MOPS buffer: 500 mM MOPS, 10 mM EDTA, pH 7.0</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;37% Formaldehyd</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;RiboRuler High Range RNA Ladder/ Low Range RNA Ladder (Thermo Scienific)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;RNA loading buffer: 1x MOPS buffer, 50% (v/v) Formamid, 6.8% (v/v) Formaldehyd, 11.67% (v/v) Glycerin, 0.03% (w/v) Bromphenol blue, 0.015% (v/v) Ethidium bromide</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Methylene blue solution: 0.04% methylene blue, 0.5 M sodium acetate, pH 5.0</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20x SSC buffer: 3 M NaCl, 300mM sodium citrate, pH 7.0</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100x Denhardt Solution: 2% (w/v) BSA, 2% (w/v) Ficoll 400, 2% (w/v) Polyvinylpyrrolidon. The solution is sterile filtratred and stored at -20°C.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hybridization buffer: 5x SSC buffer, 3x Denhardt solution, 0,2% (w/v) SDS, 0.2% (v/v) N-Laurylsarcosin, 5% (w/v) dextran sulfate.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;tRNA-stock solution: 120 mg/ml tRNA from yeast</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2x SSC buffer + 0.1% (w/v) SDS</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1x SSC buffer + 0.5% (w/v) SDS</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Blocking buffer: 1% (w/v) blocking reagent (Roche, No. 11096176001) in maleic acid buffer (store at 4°C)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Maleic acid buffer: 100 mM maleic acid, 150 mM NaCl, pH 7.5</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Anti-Digoxigenin antibody, FAB fragments (Roche, No. 11093274910)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Wash buffer: 0.3% (v/v) Tween 20 in maleic acid buffer (freshly prepared)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Detection buffer: 100 mM Tris, 100 mM NaCl, 50 mM MgCl<sub>2</sub>, pH 9.5</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;CDP-Star working solution: 0.07% (v/v) CDP-Star (Roche, No. 11685627001))</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;developer for films</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;fixer for films</p><p><br></p><h2>Equipment</h2><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;RNA gel electrophoresis system</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Microwave oven</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hybridization oven</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hybridization tubes</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Nylon membrane: GE Healthcare</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Blotting paper 0.35 and 1.5mm thick</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hyperfilm<sup>TM</sup>ECL (VWR)</p><h2>Procedure</h2><p><u>Probe labeling with Dig-dUTP</u></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;First, the desired DNA sequence for probe generation is amplified via a “normal” PCR reaction. If necessary, the PCR fragment is purified. It is used as a template for the labelling reaction.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The labeling takes place via a second PCR reaction with Dig-dUTP (1µl/ 100µl PCR reaction) and a LowT dNTP-mix to ensure efficient incorporation of the label. The elongation time should be twice as long as for a “normal” PCR, because the incorporation of Dig-dUTP is slower than that of unlabeled nucleotides.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The incorporation of the label is verified by analytical gel electrophoresis, the probe must have a higher apparent molecular weight than an unlabeled negative control.</p><p><br></p><p><u>Denaturing RNA gel electrophoresis</u></p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Prepare a denaturing RNA gel. The example is given for a gel of 8 cm x 12 cm with a volume of 80 ml. 0.8 g – 1.6 g agarose are taken to yield concentrations of 1% (w/v) – 2 % (w/v), depending on the size of the transcript that should be detected. The agarose is dissolved in 70 ml H<sub>2</sub>O<sub>DEPC</sub> by heating in a microwave oven. When the temperature of the solution is below 70<sup>o</sup>C, 8 ml 10x MOPS buffer and 3 ml 37% formaldehyde are added. When the temperature has dropped below 60<sup>o</sup>C, the solution is poured into a gel sled and a comb is added.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Put the hardened agarose gel into a horizontal gel electrophoresis system with 1x MOPS buffer as running buffer.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Mix the RNA samples (2-5 µg) and the RiboRuler RNA size marker (1µl) with three volumes of RNA loading buffer. Heat the samples to 60°C for 10 minutes to denature RNAs, and subsequently put them on ice to inhibit refolding.</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Apply the samples to the gel and run for 2h at 100V.</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;After the gel run, verify the integrity and the amounts of the ribosomal RNAs (23S, 16S). Document the gel picture, because the rRNA amounts of the different samples are used for normalization.</p><p>&nbsp;</p><p><u>Blotting the RNA gel to a nylon membrane</u></p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The transfer of the RNAs from the gel to a nylon membrane occurs via downward capillary blotting. One or a few hours are sufficient, nevertheless, usually the blotting is performed overnight. Figure 1 gives an overview of the blotting setup; the following points give a step by step description.</p><p>10.&nbsp;Cut 15 sheets of 1.5 mm thick blotting paper in the size of the gel and place them on a RNase free surface.</p><p>11.&nbsp;Cut 3 sheets of 0.35 mm thick blotting paper in the size of the gel and equilibrated them 20x SSC buffer. Add them on top of the pile.</p><p>12.&nbsp;Soak the Nylon membrane in distilled water for 10 minutes, then equilibrate them in 20x SSC buffer. Put the membrane on top of the pile.</p><p>13.&nbsp;Put the gel on top of the membrane. Make sure that no air bubbles are trapped between gel and membrane.</p><p>14.&nbsp;Equilibrate three 0.35 mm thick gel-sized blotting paper in 20x SSC buffer. Add them on top of the gel. Again, remove air bubbles if necessary.</p><p>15.&nbsp;The last layer on pile is a buffer bridge, which is also cut from the 0.35 mm thick blotting paper (12cm x 30cm) and is equilibrated in 20x SSC buffer. It is placed on top of the pile, and both ends are given into two buffer reservoirs.</p><p>16.&nbsp;&nbsp;The gel slay is placed on top of the pile to inhibit drying of the buffer bridge. A weight (e.g. a filled 250 ml Schott bottle) is placed on top to guarantee good contact between all layers.</p><p>17.&nbsp;Typically the blot runs overnight.</p><p>18.&nbsp;The positions of the gel pockets are marked on the membrane with a pencil before removing the gel.</p><p>19.&nbsp;Both membrane and gel are documented after the blotting to verify complete transfer of the two rRNAs as a proxy for transfer for all RNAs. The RNAs are cross-linked to the membrane with UV irradiation at 260 nm (Stratalinker, 120 mJ).</p><p>20.&nbsp;The membrane is stained with methylene blue to visualize the RNA size marker. The positions of the size marker fragments are marked with a pencil. The membrane can be directly used for hybridization, or it can be dried and stored for later use.</p><p><u>Hybridization of the membrane</u></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Put the blotted membrane into a hybridization tube with 10 ml hybridization buffer and 120 µg/ml yeast tRNA, incubate for 60 minutes at 50°C in the hybridization oven with slow rotation.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;30 µl of Dig-labelled probe is incubated at 95<sup>o</sup>C for 5 minutes to denature the probe, and then cooled on ice to inhibit renaturation. It is given to 10 ml hybridization buffer. The probe-free hybridization solution is removed from the hybridization tube and is discarded. The solution with the probe is added, and the tube is mounted again in the hybridization oven. It is incubated at 50<sup>o</sup>C overnight with slow rotation.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The hybridization solution with the probe is removed from the hybridization tube. It can be stored at -20°C and reused several times. Each time the probe has to be denatured before it is given to another membrane.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The membrane is washed twice for 5 minutes at room temperature with 50 ml 2x SSC buffer + 0.1% (w/v) SDS at high rotation.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The membrane is then washed twice for 15 minutes at 50°C with 1x SSC buffer + 0.5% (w/v) SDS again at high rotation speed.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The membrane is then incubated once in freshly prepared wash buffer, and it is transferred to 20 ml blocking solution and incubated for 30 minutes at room temperature and slow rotation.</p><p><u>Quantifying the specific transcript level</u></p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1 µl anti-Dig-AP antibody are given to 20 ml blocking solution. The mixture is incubated with the membrane for 30 minutes at room temperature under slow rotation.</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The membrane is washed three times for 10 minutes in 60 ml wash buffer at room temperature and fast rotation.</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The membrane is equilibrated in 20 ml detection buffer at room temperature for 5 minutes under slow rotation.</p><p>10.&nbsp;The membrane is transferred to a tray of suitable size. 15 ml of CDP-star working solution are added, and it is incubated for 5 minutes with shaking under low light conditions. The CDP-Star working solution can be reused when stored at 4°C in the dark.</p><p>11.&nbsp;The membrane is placed between two sheets of transparent foil. Light emission is detected via exposition a Hyperfilm<sup>TM</sup> ECL. The film is developed until signals are visible and fixed.</p><p>12.&nbsp;For signal quantification the film is scanned with an 8-bit scanner. The open source software&nbsp;ImageJ (https://imagej.nih.gov/) is used for signal quantification.</p><p><br></p><h2>References</h2><p>[1] U. Herrmann and J. Soppa (2002) Cell cycle-dependent expression of an essential SMC-like protein and dynamic chromosome localization in the archaeon <em>Halobacterium salinarum</em>.<em> Mol. Microbiol. </em>46(2)395-409.</p><p>[2] S. Laass et al. (2019) Characterization of the transcriptome of <em>Haloferax volcanii</em>, grown under four different conditions, with mixed RNA-Seq. PLoS ONE 14(4): e0215986.</p><p><br></p></body></html>