<html><head><title>Generating Human Gastruloids from Human Embryonic Stem Cells</title></head><body><h1>Generating Human Gastruloids from Human Embryonic Stem Cells</h1>Authors: Naomi Moris, Kerim Anlas, Julia Schroeder, Sabitri Ghimire, Tina Balayo, Susanne C. van den Brink, Anna Alemany, Alexander van Oudenaarden, Alfonso Martinez Arias<br>Abstract: <p>Gastruloids are aggregates of Pluripotent Stem Cells (PSCs) which, when exposed to differentiation medium and plated within defined conditions, undergo trilineage differentiation to all three germ layers (mesoderm, ectoderm and endoderm) with constitutive cell types organised spatiotemporally along 3 axes (Becarri et. al. 2018). They also undergo morphological shape changes including axial elongation through convergent extension cell movements. The gastruloid method has been well-established using mouse Embryonic Stem Cells (mESCs) and mouse induced PSCs (iPSCs) (van den Brink et. al., 2014; Turner et. al. 2016a; Turner et. al. 2016b; Bailie-Johnson et. al. 2015; Turner et. al. 2017a; Turner et. al. 2017b, Beccari et. al., 2018, See method in Girgin et. al., 2018). Here, we describe a new method to generate equivalent gastruloids from human pluripotent stem cells (hPSCs).&nbsp;hPSCs are able to generate axially elongated human gastruloids with evidence of spatially organised germ layers and comparable features to their mouse counterparts (Moris et. al., 2020).&nbsp;&nbsp;</p><br>Keywords: Axis specification, embryonic stem cell, gastruloid, germ layer, morphogenesis, self-organisation, symmetry breaking, gastrulation, embryogenesis, embryo, stem cell, organoid<br><h2>Introduction</h2><p>The process of gastrulation has been difficult to study in human embryos due to ethical and technical limitations that restrict its observation and manipulation. As a consequence, there is currently little understanding of the events associated with germ layer specification and axial patterning in humans. The establishment of conditions for culturing human ESCs (hESCs) has allowed us to begin to explore some of these questions using new approaches. However, current <em>in vitro</em> techniques typically utilise two-dimensional cell culture, with limited relevance to the three-dimensional, axially organised process that occurs <em>in vivo</em>.</p><p>We have described and characterised a new three-dimensional human gastruloid system, derived from hESCs, that mirrors certain features of the anteroposterior organisation of the post-gastrulation mammalian embryo (Moris et. al., 2020). Here, we describe the step-by-step protocol that can be used to generate such human gastruloids <em>in vitro</em>.&nbsp;&nbsp;</p><h2>Reagents</h2><p><strong>Culture Reagents</strong></p><p>Nutristem hPSC XF (Biological Industries 05-100-1A)</p><p>Essential6 (‘E6’ Differentiation medium) from ThermoFisher (A15165-01 or A15164-01) or TeSR-E6 (Stem Cell Technologies Cat# 05946)</p><p>CHIR99021 (Chiron, 10 mM in dimethyl sulphoxide (DMSO), Tocris Biosciences 4423)</p><p>Y-27632 (‘ROCK inhibitor’; Stock at 10 mM in H<sub>2</sub>O; Sigma Aldrich Y0503)</p><p>Vitronectin N (Gibco, A14700); Stored at -80°C.</p><p>1X Phosphate buffered saline (PBS-/-, without Mg<sup>2+</sup> and Ca<sup>2+</sup>; Sigma-Aldrich D8537).</p><p>0.5M EDTA pH8 (Invitrogen 15575-038)</p><p>Dimethyl sulphoxide (DMSO; Sigma D2650)</p><p>Knockout Serum Replacement (KSR; ThermoFisher 10828028)</p><p><strong>Note</strong>: Ensure that E6 medium is as fresh as possible. Use quickly after ordering and reorder if product has been stored at 4°C for several months. If this timescale is not suitable, we recommend using an alternative product that can be frozen and stored as aliquots at -20°C.&nbsp;</p><p><br></p><p><strong>Plastics</strong></p><p>15 mL or 50 mL Centrifuge tubes (Grenier Bio-One 188271 or 227261)</p><p>6-well plates (Corning CoStar 3516)</p><p>Sterile reservoir (55mL, STARLAB E2310-1010).</p><p>U-bottomed ultra-low adherence non-tissue culture-treated 96-well plate (CellStar 650970)</p><p>Cryovials (Nunc, Merck)</p><p>Falcon tubes, 15 mL and 50 mL</p><p>Filter tips, range of sizes</p><p><strong>Note</strong>: Use of high-quality, ultra-low adherence plates is critical to the formation of human gastruloids. Otherwise, aggregates are likely to adhere to the surface of the wells which impedes elongation and patterning. We strongly recommend using the specific product detailed above, for this purpose.&nbsp;</p><h2>Equipment</h2><p>BSL-2 biosafety cabinet.</p><p>Benchtop centrifuge for 15 mL/50 mL centrifuge tubes</p><p>Benchtop centrifuge for 96-well plates</p><p>Automated cell counter (<em>e.g.</em> Moxi Z Mini, ORFLO Technologies MXZ002) or Haemocytometer </p><p>Humidified cell culture incubator (37°C and 5% CO2)</p><p>Inverted benchtop microscope for examination of cultures</p><p>Multichannel micropipette (30-300 &micro;L is an ideal range)</p><p>Water bath (37°C)</p><p>Parafilm (Bemis PM-996)</p><p>Single-channel micropipettes, range of sizes</p><p>CoolCell (Merck CLS432005) or Mr. Frosty (ThermoFisher)</p><p>Freezer at -80°C</p><p>Liquid nitrogen storage, at -140/150°C</p><h2>Procedure</h2><p><strong>Culturing human Embryonic Stem Cells (hESCs)</strong></p><p>Maintain hESCs in Nutristem medium (see Reagents &amp; Equipment) on 0.5 &micro;g/cm<sup>2</sup> vitronectin N-precoated tissue culture-treated plastic (<em>e.g.</em> 6 well plates) in a humidified incubator (37°C, 5% CO<sub>2</sub>). Exchange the culture medium daily to fresh medium, warmed to room temperature. Culture the hESCs for at least two passages post-thawing before refreezing (See <strong>Freezing hESCs</strong> section for more information).&nbsp;Only maintain hESCs <em>in vitro</em> for up to six passages post-thawing, for efficient gastruloid generation. Culture beyond this passage number can have variable effects. Maintain stocks under antibiotic free conditions to quickly identify any microbial infections.</p><p><strong>Note:</strong> Both Nutristem and E6 medium should be allowed to warm to room temperature on the benchtop, and not in a 37°C water-bath or incubator as some of the components are heat-sensitive.</p><p><br></p><p><strong>Thawing hESCs</strong></p><p>hESCs should be stored for long-term use at -150°C in liquid nitrogen cell banks. Before thawing, allow Nutristem media to warm to room temperature, and prepare vitronectin coated wells of a 6-well plate by putting 0.5 &micro;g/cm<sup>2</sup> vitronectin in PBS-/- into each well, then leave at room temperature for 1 hour before plating, or for longer periods at 4°C wrapped with parafilm.</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Prepare enough Nutristem media for plating (3 mL per 3 cm well of a 6-well plate) with 1:1000 ROCK inhibitor. This will be your plating medium.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Place 9 mL Nutristem media in a 15 mL falcon tube. This will be your resuspension medium.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;To thaw, retrieve cryovial(s) and suspend in 37°C water bath, until almost all the vial is in a liquid state (approx. 30 seconds).</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Drop-wise, apply 1 mL of the prepared resuspension medium to the cryovial.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Transfer all liquid from the cryovial to the resuspension medium falcon tube.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Centrifuge cells in resuspension medium at 1000 rpm for 5 minutes.</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;Remove supernatant and resuspend cells in plating medium.</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Remove vitronectin coating from wells, and transfer cell suspension to the new wells.</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;Place in a humidified incubator (37°C, 5% CO<sub>2</sub>)</p><p>10.&nbsp;The next day, replace media with fresh Nutristem (without ROCK inhibitor)</p><p><br></p><p><strong>Freezing hESCs</strong></p><p>Freezing of hESCs should occur when cells are 70-80% confluent.</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Prepare 8 mL of room temperature PBS-/- in a 15 mL falcon tube, and 1 mL of room temperature Knockout Serum Replacement with 10% DMSO (KSR+DMSO), per well of a 6-well plate to be frozen.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Remove medium from cells, and wash cells with 3 mL PBS-/- per well of a 6-well plate</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Add 2 mL of room-temperature 0.5 mM EDTA in PBS-/- per well of a 6-well plate</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Place in the incubator for 4.5 minutes</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Tap against solid surface or gently pipette to dislodge cells as clusters. Check under a dissection/benchtop microscope that this has occurred before continuing.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Transfer cell suspension to prepared falcon tube with PBS-/-.</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;Centrifuge cells at 1000 rpm for 5 minutes.</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Remove supernatant and resuspend in 1 mL KSR+DMSO.</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;Transfer cell suspension to cryovial and quickly transfer to a Coolcell/’Mr. Frosty’ and to -80°C storage.</p><p>10.&nbsp;The next day, transfer cryovial(s) to -140/150°C liquid nitrogen storage</p><p><br></p><p><strong>Passaging of hESCs</strong></p><p><strong>Note:</strong> Passage the hESCs to a new well of a 6-well plate once they reach 70-80% confluence (see Figure 1).</p><p><strong>Note:</strong> Prewarm all medium to room temperature before use.</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Prepare vitronectin-coated wells of a 6-well plate by putting 0.5 &micro;g/cm<sup>2</sup> vitronectin in PBS-/- into the wells. Leave at room temperature for 1 hour before plating, or for longer periods at 4°C wrapped with parafilm.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Wash cells once with 3 mL PBS-/- per well of a 6-well plate.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Add 2 mL of room-temperature 0.5 mM EDTA in PBS-/- per well of a 6-well plate.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Place in the incubator for 4.5 minutes.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Carefully remove EDTA using a suction pump, being careful not to dislodge cells from the surface.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Add 2 mL PBS-/- before tapping against a solid surface to dislodge the cells as clusters. Check under a dissection/benchtop microscope that this has occurred before continuing.</p><p><strong>Note:</strong> It is vital for routine passaging that hESCs are not fully dissociated to single cells but remain as clusters of cells. To do so, minimise any pipetting and ensure exposure of cells to EDTA is just enough to dislodge cells without disrupting clusters.&nbsp;</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;<strong>Optional</strong>:</p><p>a)&nbsp;Collect cells using a P1000 and transfer to a 15 mL falcon tube containing 8 mL PBS-/-.</p><p>b)&nbsp;Centrifuge cell suspension at 1000 rpm for 5 minutes, remove supernatant, and resuspend in 1 mL fresh Nutristem.</p><p>c) <strong>Note</strong>: perform this step carefully so as not to disrupt cell clusters.</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Plate cells in 3 mL of Nutristem per well of a vitronectin-coated 6-well plate, at between 1:4 to 1:20, depending on original cell density and cluster size.</p><p><strong>Note:</strong> We typically passage our cultures 1:10 every 3-4 days, but ideal conditions may vary and may require optimisation for different cell lines depending on growth rate.&nbsp;Passaging concentration can be assessed by visual examination of cluster size after dislodging with EDTA (Step 3) and from pellet size after centrifugation (Step 7). Aim for fairly sparse, medium-sized clusters covering ~10% of the well.</p><p><br></p><p><strong>Generating Human Gastruloids</strong></p><p>The successful generation of human gastruloids requires three sequential steps, each of which needs to be carefully monitored: 1) the state of the cells at the start the protocol, 2) the Chiron pre-treatment and 3) aggregate formation.</p><p>The first, includes dependency on colony density, size and morphology. In order to achieve this state, cells can be routinely passaged and monitored so that pre-treatment is performed when cells achieve this culture state (Option 1, below). This option is the version used in the associated manuscript (Moris et. al., 2020) and the resulting gastruloids have been extensively characterised. It does not use ROCK inhibitor to alter the state of the cells, and instead utilises passaging as clusters of cells. Alternatively, we have found that cells can be seeded as a single-cell suspension before pre-treatment to standardise this starting state (Option 2, below). In our hands, this can improve reproducibility between experimentalists by providing a standardised culture method, although this requires ROCK inhibition.</p><p><strong>Option 1: Routine passaging of hESCs before pre-treatment (~3-5 days before pre-treatment)</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Dissociate and passage cells as described in the section describing hESC passaging.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Closely monitor cells daily to determine the correct timepoint to perform pre-treatment.</p><p><strong>Note:</strong> Cell density is critical for human gastruloid generation. Cells should be at ~60% confluency on the day of aggregation, so for pre-treatment 24 hours before, cells should be at ~40-50% confluency (see Figure 1). Typically, this state is reached 3-4 days after passaging.&nbsp;</p><p><strong>Note:</strong> Colony size is critical for human gastruloid generation. Ensure colonies are of medium size (not so large that they are touching each other, and not so small that they do not contain many cells). The optimal diameter of each colony is ~150-300 &micro;m on the day of pre-treatment. See Figure 2.&nbsp;</p><p><strong>Note:</strong> We have observed that cell morphology at the time of pre-treatment is a critical feature for human gastruloid generation. Cells around the edges of colonies should often appear ‘pointy’ before pre-treatment, and colonies should not yet have defined edges.</p><p><br></p><p><strong>Option 2: Single-cell seeding of hESCs before pre-treatment (4 days before pre-treatment)</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Prepare vitronectin-coated wells of a 6-well plate by putting 0.5 &micro;g/cm<sup>2</sup> vitronectin in PBS-/- into the wells. Leave at room temperature for 1 hour before plating, or for longer periods at 4°C wrapped with parafilm.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Prepare replating medium, which is Nutristem supplemented with 1:2000 ROCK inhibitor.</p><p><strong>Note:</strong> While ROCK inhibitor is not used for routine passaging, it is critically important for survival when hESC cells are plated as a single-cell suspension.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Wash cells once with 3 mL PBS-/- per well of a 6-well plate.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Add 2 mL of room-temperature 0.5 mM EDTA per well of a 6-well plate.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Place in the incubator for 4.5 minutes before gently tapping against a solid surface to dislodge the cells.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Dissociate the colonies into a single cell suspension with a P1000 micropipette tip by pipetting up-and-down 5-10 times and ejecting the suspension forcefully against the bottom of the well.</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;Collect cells using a P1000 and then transfer to a 15 mL falcon tube with 8 mL PBS-/-.</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Centrifuge cell suspension at 1000 rpm for 5 minutes.</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;Remove supernatant and fully resuspend in 200-1000 &micro;L of Nutristem depending on pellet size.</p><p>10.&nbsp;Check under dissection microscope that the cells are in a single-cell suspension.</p><p>11.&nbsp;Count cells using an automated cell counter or haematocytometer.</p><p>12.&nbsp;For each well of a 6-well plate, add 60,000 to 80,000 cells to 3 mL of room-temperature replating medium.</p><p><strong>Note</strong>: The exact cell number to replate depends on cell line and growth speeds. If using a new line, this number should be tested empirically, but in our hands, 60,000 cells per 3 cm well works well with most lines.&nbsp;</p><p>13.&nbsp;Remove vitronectin coating from the 6-well plate, and add cell suspension to well</p><p>14.&nbsp;Place cells in the incubator. Cells should be left to grow for 4 days (with fresh Nutristem exchange daily) before pre-treatment.</p><p><br></p><p><strong>Pre-treatment of hPSCs (24h before aggregation)</strong></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Prepare Nutristem supplemented with between 3-3.5 &micro;M Chiron, depending on cell line. An ideal concentration in our hands is 3.25 &micro;M, but this is particularly cell line dependent.</p><p><strong>Note:</strong> With each new cell line, optimise the Chiron concentration by performing titration experiments.</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Aspirate Nutristem medium from cells and replace with Chiron-supplemented medium.</p><p>a.&nbsp;&nbsp;&nbsp;&nbsp;<strong>Optional:</strong> Maintain one well in Nutristem alone for routine maintenance of the cell line.</p><p><br></p><p><strong>Preparation</strong> <strong>of Gastruloids (0h after aggregation)</strong></p><p><strong>Note: </strong>Following pre-treatment with Chiron, cell colonies should look less ‘spiky’ at the edges, and more rounded (see Figure 3). If this is not the case, consider titrating Chiron concentration further, or adjusting starting cell attributes like density and colony size.&nbsp;</p><p><strong>Note:</strong> Dissociation and reaggregation of hPSCs should be done as quickly as possible to minimise the stress caused to the cells. We recommend that any separate samples are performed independently to ensure a quick process, which should ideally be completed in &lt; 30 minutes.&nbsp;</p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;Pre-warm EDTA and E6 to room temperature</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;Prepare aggregation medium, by supplementing E6 with 0.5-3 &micro;M Chiron (routinely 0.5 &micro;M Chiron, but titrate for new cell lines) and 1:2000 ROCK inhibitor. For each 96-well plate of aggregates, prepare 5 mL of aggregation medium.</p><p><strong>Note:</strong> This concentration of ROCK inhibitor is half the concentration used for routinely replating single cells. We have found this to be a sufficient concentration for aggregate formation. With new cell lines, consider titrating this concentration to 1:1000 if aggregation is poor.</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;Aspirate the medium from the 6-well plate and rinse gently with 3 mL PBS-/-.</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;Aspirate the PBS and add 2 mL of pre-warmed 0.5 mM EDTA.</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;Place in the incubator for 4.5 minutes before gently tapping against a solid surface to dislodge the cells.</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;Dissociate the colonies into a single cell suspension with a P1000 micropipette tip by pipetting up-and-down 5-10 times and ejecting the suspension forcefully against the bottom of the well.</p><p><strong>Note:</strong> A single cell suspension is essential for accurate cell counting. Errors in counting affect the size of the gastruloids, which is known to affect the level of axial elongation in mouse gastruloids (see Baillie-Johnson et. al., 2015). Observe cell suspension under an inverted benchtop microscope to check single cell suspension. Cell suspension can be passed through a filter if necessary.</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;Collect cells using a P1000 and then transfer to a 15 mL falcon tube with 8 mL PBS-/-.</p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;Centrifuge cell suspension at 1000 rpm for 5 minutes.</p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Remove supernatant and fully resuspend in 200-1000 &micro;L of E6 depending on pellet size. One well of a 6-well plate at 60-80% confluence should typically be suspended in 200 &micro;L E6.&nbsp;</p><p>10.&nbsp;Load the counting slide of an automated cell counter (or equivalent haemocytometer) with 75 &micro;L of cell suspension and determine the density of the suspension.</p><p><strong>Note:</strong> Accurate counting of cell numbers is necessary to achieve reproducible aggregates between experiments, so automated and highly accurate equipment is strongly recommended.</p><p><strong>Note:</strong> If your cell counter allows, check that the amount of debris in your suspension is minimal. With our Moxi Orflo cell counter, we aim for a Moxi Population Index (MPI) of &gt; 0.7 (ratio of cell population relative to debris and contaminants).</p><p>11.&nbsp;Determine the volume of suspension required to produce a cell concentration of 10-20 cells/&micro;L of aggregation medium.</p><p><strong>Note</strong>: This cell concentration will determine the number of cells aggregated as gastruloids. In our hands, between 300-800 cells works well, with an optimum of 400 cells, but this is particularly cell line dependent and should be optimised empirically for each new cell line.</p><p>12.&nbsp;Add the calculated volume of cell suspension to the required amount of aggregation medium (see Step 2), mix well and transfer to a sterile reservoir.</p><p>a.&nbsp;&nbsp;&nbsp;&nbsp;<strong>Tip:</strong> Inverting the suspension prior to plating or pipetting it up and down within the reservoir can ensure that the cells are well mixed.</p><p>13.&nbsp;Pipette 40 &micro;L of plating suspension into each well of a sterile, U-bottomed non-tissue culture-treated 96-well plate with a multichannel micropipette.</p><p>a.&nbsp;&nbsp;&nbsp;&nbsp;<strong>Tip:</strong> Take care to position the droplets in the bottom of each well and not clinging to the walls, as U-shaped droplets are required for efficient aggregation.</p><p>14.&nbsp;Confirm that cells can be seen within each well using the inverted benchtop microscope.</p><p>15.&nbsp;Centrifuge the plates using a plate centrifuge at 700 rpm for 2 minutes at room temperature.</p><p>a.&nbsp;&nbsp;&nbsp;&nbsp;<strong>Tip:</strong> Wrap the plate in parafilm before centrifugation to prevent accidental lid opening and maintain sterility.</p><p>16.&nbsp;Return the plate to the incubator for 24 hours to allow aggregation.</p><p><br></p><p><strong>Addition of Medium (24 hours after aggregation)</strong></p><p><strong>Note:</strong> Adding fresh E6 medium to the aggregates 24 hours after aggregation is necessary to dilute both the Chiron and ROCK inhibitor from the aggregation medium.</p><p>17.&nbsp;Pre-warm E6 medium to room temperature.</p><p>18.&nbsp;Transfer 150 &micro;L of E6 medium to each well using the multichannel micropipette and a sterile plastic reservoir.</p><p>19.&nbsp;Return the plate to the incubator for a further 24 hours.</p><p><br></p><p><strong>Exchange of Medium (48 hours after aggregation)</strong></p><p>20.&nbsp;Pre-warm E6 medium to room temperature.</p><p>21.&nbsp;Carefully remove 150 &micro;L of the medium from each well using the multichannel micropipette, holding it at an angle to aspirate slowly from the side of each well.</p><p>a.&nbsp;&nbsp;&nbsp;&nbsp;<strong>Tip</strong>: Angle the pipette at an acute angle at the beginning of suction, rotating your wrist to become more vertical as suctioning progresses. This will ensure that the tip of your pipette is always at the top of the medium and not likely to accidently disturb/remove the gastruloid (see Figure 4).</p><p>22.&nbsp;Add 150 &micro;L of fresh E6 to each well using the multichannel micropipette.</p><p>23.&nbsp;Return the plate to the incubator.</p><p>24.&nbsp;Repeat steps 20-23 each day for the extent of the protocol.</p><h2>Troubleshooting</h2><p><em>Aggregation failure.</em></p><p>Aggregation failure might originate from the U-bottomed 96-well plate of choice. Make sure to use ultra-low plates for efficient aggregation. The culture is also sensitive to the starting state of the cells. Failure to aggregate has been observed in stocks that have been maintained at either high or low confluence (&gt;70% or &lt;50%) or under stress (<em>e.g.</em> from missing daily medium changes or pH &lt; 6.5). Check also the concentration of ROCK inhibitor, as this is a necessary component of the aggregation medium to ensure that single-cells aggregate.</p><p><em>The cells aggregate, but as multiple smaller aggregates or misshapen co-aggregates</em></p><p>This is often caused by issues with the starting state of the cells. Check the confluency of the cells, their genetic integrity and ensure that the passage number of the cells post-thawing is suitably low (ideally &lt; 6 passages). This could also indicate either a cell number issue (try optimising the starting number of cells aggregated), or a Chiron concentration issue (empirical titration of Chiron concentration is necessary for each new cell line).</p><p><em>Aggregates form but do not elongate </em></p><p>This is likely to be an issue with the Chiron concentration or starting cell state. Each line is very sensitive to the level of Chiron exposure, and new lines should be empirically optimised for Chiron pre-treatment optimisation by titration. If aggregates appear as ‘fluffy’ and disorganised spheres this is likely to require higher levels of Chiron concentration, whereas if aggregates are smooth and spherical but still not elongating, this might require lower levels of Chiron concentration. Also check genetic integrity of the cell lines and environmental factors, including incubation conditions.&nbsp;</p><h2>Time Taken</h2><p>The total duration is 4-5 days, including 1 day of pre-treatment in adherent culture. The hands-on time breaks down as follows:</p><p>Pre-treatment: Approx. 10-20 minutes. </p><p>0 hours: Approx. 30 minutes – 1 hour per cell line.</p><p>24 hours: Approx. 5 minutes per 96-well plate.</p><p>48 hours: Approx. 10 minutes per 96-well plate.</p><p>72 and 96 hours: Approx. 10 minutes per 96-well plate.</p><p><br></p><h2>Anticipated Results</h2><p>Under optimal conditions, the vast majority of aggregates should undergo morphological changes including elongation, by 72-96 hours (~ 60-70%). These can be analysed by wide-field microscopy directly in the 96-well plate format for high-throughput imaging, or the gastruloids can be fixed and stained for confocal microscopy (see for instance, van den Brink et. al. 2014 and Baillie-Johnson et. al., 2015 for mouse gastruloids). A brief overview of human gastruloid development is detailed below:</p><p>Within 0-2 hours after plating, the single-cell suspension in the 96-well plate should form one coherent aggregate, with each cell in close physical contact with its neighbours.</p><p>Within the first 24 hours, these aggregates should have assumed an initially spherical shape, with clearly defined edges. Shortly after this point, the aggregates polarise their gene expression (SOX2 becomes localised to one region of the aggregate) and the aggregate assumes an ovoid shape.</p><p>At the 48 hour time point, the aggregates should have increased in size slightly, and should be beginning to become elongated. It should be clear from this timepoint onwards that one end of the elongated ovoid is slightly wider and darker than the other, which is more transparent; these are the anterior and posterior axis locations, respectively.</p><p>At the 72 hour time point, most aggregates should have assumed a clearly elongated shape, with a single long extension. For optimal conditions, at least 60% of aggregates should have elongated by this timepoint.</p><p>At the 96 hour time point, many of the gastruloid extensions will have either curled back on themselves, or appear to have been retracted. Many of the gastruloids will therefore assume a more spherical shape.&nbsp;</p><p>Beyond this point, aggregates continue to grow radially but without a clear anterior-posterior morphological axis. Eventually, lumens and cavities might form but without clear axial organisation. Therefore, the experiment is typically concluded by 72-96 hours after aggregation.</p><p>For more details regarding anticipated results, see the publication that is associated with this protocol (Moris et. al. 2020).</p><h2>References</h2><p>Baillie-Johnson, Peter; van den Brink, Susanne C.; Balayo, Tina; Turner, David A.; Arias, Alfonso Martinez (2015).&nbsp;<a href="http://www.jove.com/video/53252/generation-aggregates-mouse-embryonic-stem-cells-that-show-symmetry" rel="noopener noreferrer" target="_blank">"Generation of Aggregates of Mouse Embryonic Stem Cells that Show Symmetry Breaking, Polarization and Emergent Collective Behaviour&nbsp;In Vitro"</a>.&nbsp;<em>Journal of Visualized Experiments</em>&nbsp;(105).&nbsp;doi:10.3791/53252.&nbsp;PMC&nbsp;4692741.&nbsp;PMID&nbsp;26650833.</p><p>Beccari, Leonardo; Moris, Naomi; Girgin, Mehmet; Turner, David A.; Baillie-Johnson, Peter; Cossy, Anne-Catherine; Lütolf, Matthias P.; Duboule, Denis; Arias, Alfonso Martinez (2018).&nbsp;"<a href="https://www.nature.com/articles/s41586-018-0578-0" rel="noopener noreferrer" target="_blank">Multi-axial self-organization properties of mouse embryonic stem cells into gastruloids</a>".&nbsp;<em>Nature</em>.&nbsp;<strong>562</strong>&nbsp;(7726): 272–276.&nbsp;doi:10.1038/s41586-018-0578-0.&nbsp;ISSN&nbsp;0028-0836.</p><p>Girgin, Mehmet; Turner, David A; Baillie-Johnson, Peter; Cossy, Anne-Catherine; Beccari, Leonardo; Moris, Naomi; Lütolf, Matthias; Duboule, Denis; Martinez Arias, Alfonso (2018).&nbsp;<a href="http://www.nature.com/protocolexchange/protocols/6977" rel="noopener noreferrer" target="_blank">"Generating Gastruloids from Mouse Embryonic Stem Cells"</a>.&nbsp;<em>Protocol Exchange</em>.&nbsp;doi:10.1038/protex.2018.094.&nbsp;ISSN&nbsp;2043-0116.</p><p>Turner, David A.; Baillie-Johnson, Peter; Martinez Arias, Alfonso (2016a).&nbsp;<a href="http://onlinelibrary.wiley.com/doi/10.1002/bies.201500111/abstract" rel="noopener noreferrer" target="_blank">"Organoids and the genetically encoded self-assembly of embryonic stem cells"</a>.&nbsp;BioEssays.&nbsp;<strong>38</strong>&nbsp;(2): 181–191.&nbsp;doi:10.1002/bies.201500111.&nbsp;ISSN&nbsp;1521-1878.&nbsp;PMC&nbsp;4737349.&nbsp;PMID&nbsp;26666846.</p><p>Turner, David A.; Glodowski, Cherise R.; Alonso-Crisostomo, Luz; Baillie-Johnson, Peter; Hayward, Penny C.; Collignon, Jérôme; Gustavsen, Carsten; Serup, Palle; Schröter, Christian (2016b). "<a href="https://www.biorxiv.org/content/10.1101/051722v2" rel="noopener noreferrer" target="_blank">Interactions between Nodal and Wnt signalling Drive Robust Symmetry Breaking and Axial Organisation in Gastruloids (Embryonic Organoids</a>)".&nbsp;bioRxiv&nbsp;051722.</p><p>Turner, David A.; Alonso-Crisostomo, Luz; Girgin, Mehmet; Baillie-Johnson, Peter; Glodowski, Cherise R.; Hayward, Penelope C.; Collignon, Jérôme; Gustavsen, Carsten; Serup, Palle; Steventon, Ben; Lütolf, Matthias; Martinez Arias, Alfonso (2017a). "<a href="https://www.biorxiv.org/content/10.1101/104539v1" rel="noopener noreferrer" target="_blank">Gastruloids develop the three body axes in the absence of extraembryonic tissues and spatially localised signalling</a>".&nbsp;bioRxiv&nbsp;104539.</p><p>Turner, David A.; Girgin, Mehmet; Alonso-Crisostomo, Luz; Trivedi, Vikas; Baillie-Johnson, Peter; Glodowski, Cherise R.; Hayward, Penelope C.; Collignon, Jérôme; Gustavsen, Carsten; Steventon, Ben; Lütolf, Matthias; Martinez Arias, Alfonso (2017b).&nbsp;"<a href="https://dev.biologists.org/content/144/21/3894" rel="noopener noreferrer" target="_blank">Anteroposterior polarity and elongation in the absence of extra-embryonic tissues and of spatially localised signalling in gastruloids: mammalian embryonic organoids</a>".&nbsp;Development.&nbsp;<strong>144</strong>&nbsp;(21): 3894–3906.&nbsp;doi:10.1242/dev.150391. ISSN&nbsp;09501991.&nbsp;PMC&nbsp;5702072.&nbsp;PMID&nbsp;28951435.</p><p>Van den Brink, Susanne C.; Baillie-Johnson, Peter; Balayo, Tina; Hadjantonakis, Anna-Katerina; Nowotschin, Sonja; Turner, David A.; Martinez Arias, Alfonso (2014).&nbsp;<a href="http://dev.biologists.org/content/141/22/4231" rel="noopener noreferrer" target="_blank">"Symmetry breaking, germ layer specification and axial organisation in aggregates of mouse embryonic stem cells"</a>.&nbsp;<em>Development</em>.&nbsp;<strong>141</strong>&nbsp;(22): 4231–4242.&nbsp;doi:10.1242/dev.113001.&nbsp;ISSN&nbsp;0950-1991.&nbsp;PMC&nbsp;4302915.&nbsp;PMID&nbsp;25371360.</p><h2>Acknowledgements</h2><p>Many thanks are due to past and present members of the Martinez Arias laboratory for their work in developing and optimising the protocol.</p><p><br></p></body></html>