<html><head><title>ChIP-Seq data processing, normalization and visualization</title></head><body><h1>ChIP-Seq data processing, normalization and visualization</h1>Authors: ADITYA SANKAR, Mads Lerdrup<br>Abstract: <p>This article explains the step-by-step processing of NGS data of ChIPSeq samples for H3K4me3, H3K9me3 and H3K36me3 immunoprecipitations performed in oocytes/embryos. This includes sections on visualization of data, and a comment on normalization.</p><p>the associated raw data have been deposited at GSE129735 on Gene Expression Omnibus. </p><p><br></p><p><br></p><br>Keywords: Chromatin, ChIPSeq, Histone modifications<br><h2>Introduction</h2><p>This procedure has been applied to process ChIPSeq data from the associated publication. Some aspect of this procedure can be used as inspiration for processing low input ChIPSeq datasets</p><h2>Equipment</h2><p>1. Windows PC with atleast 12 GB RAM powerful enough to run EaSeq sessions listed in this article. </p><p> 2. Installed EaSeq software available freely at www.easeq.net</p><p><br></p><h2>Procedure</h2><p><strong>ChIP-seq data processing</strong></p><p><br></p><p>1. ChIP-seq libraries were sequenced as 75 and 150 bp paired end reads on an Illumina NextSeq 500. </p><p><br></p><p>2. Reads were trimmed using Trimmomatic<sup>1</sup> with the settings “ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 SLIDINGWINDOW:4:20 HEADCROP:15 CROP:120 MINLEN:40” for H3K4me3, “SLIDINGWINDOW:4:20 HEADCROP:5 CROP:50 MINLEN:30” for MII H3K9me3 data, and “SLIDINGWINDOW:4:20 HEADCROP:5 CROP:125 MINLEN:30" for H3K36me3 and 2-cell H3K9me3 data. </p><p><br></p><p>3. ChIP-seq reads were aligned as paired ends to the mm10 genome assembly using bowtie2<sup>2</sup> and the settings “--local -X 2000”, sorted and exported as bam-files using samtools<sup>3</sup>, and imported into EaSeq<sup>4</sup> (<a href="http://easeq.net/" rel="noopener noreferrer" target="_blank">http://easeq.net</a>) for subsequent analyses and visualization.</p><p><br></p><p>4. In EaSeq, default settings for ChIP-seq data are used including filtering of reads with identical coordinates. Where nothing else is mentioned, values were normalized to fragments per million reads per kbp (FPKM).</p><p><strong>&nbsp;</strong></p><p><strong>General ChIP-seq data analysis and visualization</strong></p><p><br></p><p>1. Unless stated, then all analysis and visualization was performed using EaSeq v.1.05-1.111 (<a href="http://easeq.net/" rel="noopener noreferrer" target="_blank">http://easeq.net</a>) and its integrated tools using the default settings. </p><p><br></p><p>2. All these analyses, visualizations and used data are available as a session file at <a href="http://easeq.net/Sessions/Sankar.eas" rel="noopener noreferrer" target="_blank">http://easeq.net/Sessions/Sankar.eas</a>. Please note that the session requires 12 GB of available RAM. </p><p><br></p><p>3. Plots were exported for layout in Adobe Illustrator CS6 as pdfs using the beta tool ‘Export snapshot as pdf’. </p><p><br></p><p>4. Gene bodies were extracted from the imported RefSeq ‘Geneset’ using the ‘Extract’ tool and ‘Feature’ set to ‘Gene’, ‘Start position’ set to ‘Start’, and ‘End position’ set to ‘End’. </p><p><br></p><p>5. To obtain non-redundant genes without multiple identical termini, gene symbols were exported and duplicates were removed. The list of symbols was imported as a ‘Regionset’ using the ‘Geneset’ to lookup coordinates. For gene symbols with multiple annotated transcripts, this resulted in the most extreme coordinates being used for subsequent analysis. Possible gene symbols not matched by coordinates were removed using the gate tool. </p><p><br></p><p>6. Broad H3K4me3 domains were acquired as described above and imported into EaSeq as ‘Regionsets’. The set of broad H3K4me3 regions were subselected for those at or above 20 kbp of size for further analyses. </p><p><br></p><p>7. For broad domains and gene bodies, the H3K9me3 levels in replicate 1 were quantified using the ‘Quantify’ tool within windows corresponding to the entire length of the regions, and the order of the regions was changed according to these values using the ‘Sort’ tool. </p><p><br></p><p>8. Heatmaps were generated using the ‘HeatMap’ plot type and ratiometric heatmaps showing pseudocolored differences between conditions were generated using the ratiomap plot type. </p><p><br></p><p>9. For plots of gene bodies and broad H3K4me3 domains, settings were adjusted to a relative window size of 200% of the size of the visualized feature by checking ‘rel.’ and changing ‘Window size’ to 200 in the plot settings. </p><p><br></p><p>10. For all heatmaps ‘Maximum density’ was adjusted as indicated by the FPKM values in the figures. </p><p><br></p><p>11. Plots showing the average values at a set of regions were generated using the ‘Average’ tool, where  gene bodies and broad H3K4me3 domains adjusted to a relative window size of 200% of the size of the visualized feature by checking ‘rel.’ and changing ‘Window size’ to 200 in the plot settings. ‘Y-axis Maximum’ was adjusted as shown in the figures and multiple plots were overlaid using the ‘Overlay’ tool. </p><p><br></p><p>12. Genome browser tracks were generated using the ‘FillTrack’ tool with ‘Y-axis Maximum’ adjusted as shown in the figures and for some tracks loci coinciding with broad H3K4me3 domains were coloured using the ‘Highlight’ option in the plot settings and the broad H3K4me3 domains as the feature to highlight. </p><p><br></p><p>13. Tracks showing log2 fold differences were adjusted in the plot settings by setting ‘Ratio’ to the ‘Dataset’ in the denominator and for the Y-axis checking ‘Log scale’, setting the ‘Base’ to 2, and ‘Maximum’ to 2 and ‘Minimum’ to -2. </p><p><br></p><p>14. 2D-histograms and associated correlation efficiencies were generated using the ‘Scatter’ tool and adjusted to show 100 bins on both axes in the plot settings. The visualized values were quantified at a set of regions corresponding to 10kbp windows within the entire span of mm10 genome using the ‘Quantify’ tool and ‘Start position offset’ and ‘End position offset’ adjusted to -5000 and 5000 bp, respectively. </p><p><br></p><p>15. Visualized ratios were derived by applying the ‘Calculate’ tool with ‘Division’ and ‘Logarithm’ checked to the respective quantified values.</p><p><strong>&nbsp;</strong></p><p><strong>&nbsp;ChIP-seq data normalization (a comment)</strong></p><p><br></p><p>1. When assessing differences in H3K9me3 ChIP-seq signal in the <em>Kdm4a</em><sup><em>-/-</em></sup> compared to <em>Kdm4a</em><sup><em>+/+ </em></sup>MII oocytes using standard FPKM normalization, we observed a loss of signal at gene bodies, where H3K9me3 was enriched in the <em>Kdm4a</em><sup><em>+/+ </em></sup>oocytes. </p><p><br></p><p>2. To cross-validate the distribution in the wild-type we used published H3K9me3 data from MII oocytes<sup>5</sup> and found similar enrichment at gene bodies as well as depletion within Broad H3K4me3 Domains (Extended Fig. 4 a, b). </p><p><br></p><p>3. We have previously observed that large increases in the fraction of the genome covered by H3K4me3 in MII oocytes compared to e.g. mouse ES cells, results in a much lower local read density due to the sequenced reads being dispersed over a larger fraction of the genome<sup>6</sup>. </p><p><br></p><p>4. This effect is unaccounted for by the widely used RPKM/FPKM normalization of ChIP-seq samples, and for the previous study we therefore used a normalization strategy based on H3K4me3 at the top-5000 ranked transcription start sites<sup>6</sup>. </p><p><br></p><p>5. In this study, the widespread gain of H3K9me3 signal at the H3K4me3 enriched regions where it is normally depleted occurs in more than one fifth of the mouse genome<sup>6</sup>. This would not only elevate the signal in the normally depleted regions, but also lead to a considerably reduced signal in the areas of canonical enrichment (Extended Figure 4c). </p><p><br></p><p>6. Therefore, we would be highly cautious about interpreting the loss of gene body H3K9me3 in absolute terms, and prefer conservative interpretations. Accordingly, we find it likely that the apparent loss of H3K9me3 is due to the susceptibility of ChIP-seq and FPKM normalization to pronounced increases in the fraction of the genome, which is covered by H3K9me3. </p><p><br></p><p>7. As assessments of the extent of this in a quantitatively manner would require a spike-in setup, which have not yet been developed for the very small-scale ChIP-seq used here, we chose the commonly used FPKM-normalization for plots and analyses, and to make conservative statements that are unaffected by the choice of normalization. </p><p><br></p><p>8. This conservative approach could lead to underestimations of the extent of H3K9me3 gain in broad domains, and to illustrate the consequences, we have done pairwise comparisons of FPKM normalized plots and similar plots normalized in a manner parallel to what was previously used<sup>6</sup> (Extended Figure 4 d,e).</p><h2>Anticipated Results</h2><p>Trimmed, mapped data uploading to genome browser for visualization or direct use for further analysis in EaSeq</p><h2>References</h2><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Bolger, A.M., Lohse, M. &amp; Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. <em>Bioinformatics</em> <strong>30</strong>, 2114-2120 (2014).</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Langmead, B. &amp; Salzberg, S.L. Fast gapped-read alignment with Bowtie 2. <em>Nat Methods</em> <strong>9</strong>, 357-359 (2012).</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Li, H.<em> et al.</em> The Sequence Alignment/Map format and SAMtools. <em>Bioinformatics</em> <strong>25</strong>, 2078-2079 (2009).</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Lerdrup, M., Johansen, J.V., Agrawal-Singh, S. &amp; Hansen, K. An interactive environment for agile analysis and visualization of ChIP-sequencing data. <em>Nat Struct Mol Biol</em> <strong>23</strong>, 349-357 (2016).</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Wang, C.<em> et al.</em> Reprogramming of H3K9me3-dependent heterochromatin during mammalian embryo development. <em>Nat Cell Biol</em> <strong>20</strong>, 620-631 (2018).</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Dahl, J.A.<em> et al.</em> Broad histone H3K4me3 domains in mouse oocytes modulate maternal-to-zygotic transition. <em>Nature</em> <strong>537</strong>, 548-552 (2016).</p><h2>Acknowledgements</h2><p>John Arne Dahl, Principal Investigator, University of Oslo</p></body></html>