<html><head><title>Nascent RNA 4sU labelling and enrichment</title></head><body><h1>Nascent RNA 4sU labelling and enrichment</h1>Authors: Robert A. Beagrie, Damien J. Downes<br>Abstract: <p>4sU-seq allows for measuring or nascent transcription which can reflect dynamic changes more accurately than traditional RNA-seq. Incorporation of the nucleoside analog 4-thiouridine into RNA allows the separation of nascent transcripts though reversible coupling to biotin and streptavdin bead pull-down. The protocol takes roughly two days to generate a sequencing ready library.</p><br>Keywords: Gene expression, gene regulation, transcription<br><h2>Introduction</h2><p>4sU-seq allows for measuring or nascent transcription which can reflect dynamic expression changes more accurately than traditional RNA-seq. Incorporation of the nucleoside analog 4-thiouridine into RNA allows the separation of nascent transcripts though reversible coupling to biotin and streptavdin bead pull-down<sup>1-4</sup>. The protocol takes roughly two days to generate a sequencing ready library.</p><h2>Reagents</h2><p>- 10<sup>5</sup> cells in suspension</p><p>- TRIS, pH 7.5 (ThermoFisher: 15567027)</p><p>- EDTA (Invitrogen: AM9760G)</p><p>- DEPC-treated water (Ambion: AM9915G)</p><p>- NaCl, 5 M (Sigma: S9625)</p><p>- Tween 20 (Sigma: P9416)</p><p>- 4‐thiouridine, 4sU (Sigma: T4509) [500 mM in PCR grade water]</p><p>- PBS (Invitrogen: 10010031)</p><p>- TRI reagent (Sigma: T9424)</p><p>- RNAseZap (Sigma: R2020)</p><p>- Chloroform (Sigma: C2432)</p><p>- Glycoblue (ThermoFisher: AM9515)</p><p>- Isopropanol (Sigma: I9516)</p><p>- Ethanol (Sigma: 51976)</p><p>- MTSEA-biotin-XX (Biotium: #90066) [0.1 mg/mL in dimethylformamide]</p><p>- Dimethylformamide (Sigma: PHR1553)</p><p>- μMacs (Miltenyi: 130-074-101)</p><p>- Chloroform/isoamylalcohol, 24:1 (Sigma: 25668)</p><p>- Phase-lock tube (5Prime: 2900309)</p><p>- Dithiothreitol, DTT (Merck: 1114740025) [Make Fresh, 100 mM in PCR grade water]</p><p>- RNeasy RNA clean-up kit (Qiagen: 74204)</p><p>- QuantiFluor RNA kit (Promega: E3310)</p><p><br></p><p><strong>Buffers:</strong></p><p><strong>10x Biotinylation buffer (25 µL per sample):</strong></p><p>- 100mM Tris, pH 7.4</p><p>- 10mM EDTA</p><p>- DEPC-treated water</p><p><br></p><p><strong>MACS wash buffer (4 mL per sample):</strong></p><p>- 100mM Tris, pH 7.5</p><p>- 10mm EDTA</p><p>- 1M NaCl</p><p>- 0.1% Tween 20</p><p>- DEPC-treated water</p><h2>Equipment</h2><p>- Nanodrop One, or quivalent (ThermoFischer Scientific: ND-ONEC-W)</p><p>- Qubit 4 Fluorometer, or quivalent (ThermoFischer Scientific: Q33238)</p><p>- MACS MultiStand (Miltenyi: 130-042-303)</p><p>- µMACS Separator (Miltenyi: 130-042-602)</p><h2>Procedure</h2><p><strong>4sU Incorperation</strong></p><p><em>** Prepare 10</em><sup><em>5</em></sup><em> cells per mL of media **</em></p><p>1. Add 10 µL of 500 mM 4‐thiouridine (4sU) per mL of cells for a final concentration of 500 μM.</p><p>2. Incubate under standard growth conditions for 45 minutes (e.g. 37°C, 5% CO<sub>2</sub>)</p><p>3. Pellet the cells by centrifugation at 200 rcf for 5 min.</p><p>4. Discard supernatant and resuspend cells in an equal volume of PBS.</p><p>5. Pellet the cells by centrifugation at 200 rcf for 5 min.</p><p>6. Resuspend pellet in 1 mL of TRI reagent and transfer to microcentrifuge tube.</p><p>7. Snap freeze cells using liquid nitrogen OR dry ice and ethanol.</p><p><em>** Store lysed cells at -80ºC for up to 3 months **</em></p><p><br></p><p><strong>RNA Extraction</strong></p><p><em>** Chill centrifuge to 4ºC **</em></p><p>1. Thaw TRI reagent lysed cells on ice.</p><p>2. Add 200 μL chloroform.</p><p>3. Add 1 μL glycoblue and close tube securely.</p><p>4. Shake the tube vigorously for 15 sec.</p><p>5. Incubate at room temperaturefor for 2–3 min.</p><p>5. Centrifuge for 30 min at 12,000 rcf (4°C).</p><p>6. Transfer the upper aqueous phase to a new collection tube containing 500μL isopropanol.</p><p>7. Incubate for 10 min at room temperature.</p><p>8. Centrifuge for 10 min at 12,000 rcf (4°C).</p><p>9. Discard supernatant and wash the RNA pellet with with 700 µL 70% ethanol in DEPC-treated water.</p><p>10. Centrifuge for 10 min at 12,000 rcf (4°C).</p><p>11. Discard supernatant and air dry for 10 min.</p><p>12. Re-suspend in 20 µL DEPC-treated water.</p><p>13. Measure RNA concentration using a Nanodrop.</p><p><br></p><p><strong>4sU Enrichment</strong></p><p><em>** Pre-spin one phase-lock tube per sample **</em></p><p><em>** Heat 2 mL MACS wash buffer to 65°C **</em></p><p>1. Combine 20-100 μg of total RNA, 50 μL MTSEA-biotin-XX, 25 µL 10x Biotinylation buffer and sufficient DEPC-treated water to generate a 250 μL reaction.</p><p>2. Incubate for 30 min at room temperature with rotation.</p><p>3. Add 400 μL chloroform/isoamylalcohol (24:1).</p><p>4. Add 1 μL glycoblue.</p><p>5. Mix and incubate for 3 min.</p><p>6. Transfer mixture to a phase-lock tube.</p><p>7. Centrifuge for 5 min at 12,000 rcf (room temp).</p><p>8. Transfer upper phase (~230 µL) to a new tube.</p><p>9. Add 1:10 volume 5 M NaCl and an equal volume of isopropanol and mix.</p><p>10. Centrifuge for 20 min at 20,000 rcf (4ºC).</p><p>11. Discared supernatant and wash pellet with 500 µL of 75% ethanol in DEPC-treated water.</p><p>12. Centrifuge for 20 min at 20,000 rcf (4ºC).</p><p>13. Discared supernatant and resuspend pellet in 60μl DEPC-treated water.</p><p>14. Denature RNA at 65°C for 10 min followed by rapid cooling on ice for 5 min.</p><p>15. Incubate RNA with 60 μL streptavidin magnetic beads for 15 min at room temperature with rotation.</p><p>16. Place a µMACS column into a magnetic separator with a 15 mL waste collection tube.</p><p>17. Add the biotinylation reaction to the coulmn.</p><p>18. Wash the column with 1 mL 65ºC MACS was buffer, repeat for a total of two washes.</p><p>19. Wash the column with 1 mL room temp MACS was buffer, repeat for a total of two washes.</p><p>20. Replace the waste tube with a collection microcentrifuge tube.</p><p>21. Elute RNA from the column by adding 100 μL of freshly prepared 100 mM dithiothreitol (DTT), followed by a second elution with another 100 µL 5 min later.</p><p>22. Clean up labelled RNA using a Qiagen RNeasy RNA clean-up kit with on-column DNase digestion.</p><p>23. Elute purified RNA in 20 μL of DEPC-treated water.</p><p>24. Measure RNA concentration using the QuantiFluor RNA kit.</p><p><br></p><p><strong>Further steps:</strong></p><p>A. Index RNA using SMARTer Stranded Total RNA-Seq Kit v2 - Pico Input Mammalian following the manufacturer’s instructions with a fragmentation time of 3 min and 14 cycles of PCR amplification.</p><p><br></p><p>B. Sequence using 75-cycle paired end reads.</p><p><strong>﻿</strong></p><h2>Troubleshooting</h2><p>- To avoid RNA degradation thoroughly clean bench surfaces and pipettes with RNAseZAP.</p><p>- Cell centrifugation speeds are optimised for erythroid progenitor cells and may need adjusting.</p><h2>References</h2><p>1. Rabani, M. et al. Metabolic labeling of RNA uncovers principles of RNA production and degradation dynamics in mammalian cells. <em>Nature biotechnology</em> 29, 436-42. doi:10.1038/nbt.1861 (2011)</p><p>2. Hay, D. et al. Genetic dissection of the α-globin super-enhancer in vivo. <em>Nature genetics</em> vol. 48, 895-903. doi:10.1038/ng.3605 (2016)</p><p>3. Friedel, C.C. et al. Conserved principles of mammalian transcriptional regulation revealed by RNA half-life. <em>Nucleic acids research</em> 37, e115. doi:10.1093/nar/gkp542 (2009)</p><p>4. Duffy, E.E et al. Tracking Distinct RNA Populations Using Efficient and Reversible Covalent Chemistry. <em>Molecular cell</em> 59, 858-66. doi:10.1016/j.molcel.2015.07.023 (2015)</p></body></html>