<html><head><title>Establishment of a RESEARCH USE ONLY Condensed-Efficient-Fast (CEF) PITX2 workflow for analysis of PITX2 DNA methylation in small tumor tissue samples</title></head><body><h1>Establishment of a RESEARCH USE ONLY Condensed-Efficient-Fast (CEF) PITX2 workflow for analysis of PITX2 DNA methylation in small tumor tissue samples</h1>Authors: Rudolf Napieralski , Gabriele Schricker, Elisabeth Schüren, Jonathan Perkins , Viktor Magdolen , Wilko Weichert , Marion Kiechle , Olaf G. Wilhelm <br>Abstract: <p>The CE-marked <em>therascreen</em><sup><em>®</em></sup> PITX2 RGQ PCR Kit (QIAGEN Cat No./ID 873211), a PITX2 DNA methylation assay was introduced to the market in 2018, (hereinafter referred to as the “standard <em>therascreen</em> PITX2 workflow) and represents an <em>in vitro</em> diagnostic to predict outcome to anthracycline-based chemotherapy considered standard-of-care systemic treatment in early breast cancer patients. The Intended Use (IUS) of this kit covers lymph node-positive, estrogen receptor-positive and HER2-negative high-risk breast cancer patients treated in the adjuvant setting with anthracycline-based chemotherapy and requires 400&nbsp;ng genomic DNA as recommended DNA input extracted from 1-2&nbsp;x&nbsp;5&nbsp;µm Formalin-Fixed-Paraffin-Embedded (FFPE) &nbsp;sections with a minimal total tumor tissue surface area of ≥ 100 mm<sup>2</sup>. In today’s clinical workflow of early high-risk breast cancer patient therapy, increasing number of patients are treated in the neoadjuvant setting after tumor biopsy with anthracycline-based chemotherapy before tumor resection by surgery. Biopsy material size restriction in this clinical area requires for a predictive marker test to be applicable biopsy specimens with tumor tissue surface area’s in the range of 20-50 mm<sup>2</sup>. &nbsp;&nbsp;&nbsp;</p><p> Therefore, a modified PITX2 workflow protocol - the so-called <strong>“Condensed Efficient Fast (CEF)” PITX2 workflow</strong> – was developed to address this need. The required modifications of the standard <em>therascreen</em> PITX2 workflow is described in the “bisDNA Preparation” section. The “PMR Detection &amp; Analysis” section of the standard <em>therascreen</em> PITX2 workflow remains unchanged. The new CEF PITX2 workflow is for <strong>Research Use Only (RUO)</strong> and intended to be used by qualified users such as technicians, molecular biologists/clinical chemists, or physicians. The complete CEF&nbsp;PITX2 workflow is optimized for medium sample through-put with highly reliable and robust read-out and can be performed in two working days.</p><p><br></p><br>Keywords: PITX2, DNA-methylation,  cancer, breast cancer, prostate cancer, bisulfite conversion <br><h2>Introduction</h2><p>	Breast cancer is the most common malignancy in women with approximately 523,000 new cases per year in Europe<sup>1</sup> and 1.9 million incident cases worldwide and the leading cause of cancer deaths in women<sup>2</sup>. Prognostic factors provide information about the future course of the disease. Clinico-pathological risk factors for disease recurrence include axillary lymph node status, tumor size, histological grade, steroid hormone receptor status, vascular invasion, patient age, and human epidermal growth factor receptor 2 (HER2)-status<sup>3</sup>. According to clinico-pathological risk classification for early breast cancer about 25% of the patients are at high risk, 65% are at intermediate risk and 10% are at low risk for disease recurrence. Patients at high-risk include those with more than three lymph nodes affected (independent from additional risk factors) or patients with one to three affected lymph nodes plus additional risk factors. According to current understanding, the high-risk group also includes patients with triple negative breast cancer (TNBC), i.e. patients with tumors negative for estrogen receptor (ER), progesterone receptor (PR) and HER2 expression<sup>4</sup>. Patients at high risk for disease recurrence are recommended to receive anthracycline-based chemotherapy (ANT) according to international guidelines<sup>5</sup>. </p><p>	To identify patients at diagnosis whether they will benefit from anthracycline-based chemotherapy, a biomarker to predict response is of clinical need. DNA methylation patterns such as methylation status of the pituitary homeobox gene 2 (PITX2) play a key role in several cancer indications<sup>6-12</sup> including breast cancer<sup>13-17</sup> and published data of a Research Use Only (RUO) PITX2 DNA methylation assay using fresh-frozen tissue showed that PITX2 DNA methylation can be a predictive marker for response to adjuvant anthracycline-based chemotherapy in breast cancer<sup>18</sup>. The CE-marked <em>therascreen</em><sup><em>®</em></sup> PITX2 RGQ PCR Kit by QIAGEN was entering the market in 2018 as clinically validated biomarker test for breast cancer <sup>19-21</sup>. </p><p>	The CE-marked <em>therascreen</em><sup><em>®</em></sup> PITX2 RGQ PCR Kit is an optimized <em>in vitro</em> DNA methylation-specific quantitative real-time PCR test intended for the clinically reliable determination of the percent methylation ratio (PMR) in the pituitary homeobox 2 (PITX2) transcription factor gene promoter 2 in lymph node-positive, estrogen receptor-positive and HER2-negative high risk breast cancer patients. The clinical validity of the pre-defined PMR cut-off of 12 to predict patient outcome to anthracycline-based chemotherapy with or without endocrine therapy was shown by using 145 patient samples with 10-year follow-up data and disease-free survival as the primary end point. PITX2 hypermethylation (PMR &gt; 12) indicates a poor outcome to ANT-based chemotherapy whereas PITX2 hypomethylation (PMR ≤ 12) indicates a good outcome to ANT-based chemotherapy<sup>20</sup>. The CE-marked <em>therascreen</em><sup><em>®</em></sup> PITX2 RGQ PCR assay is able to guide the clinician to select the appropriate chemotherapy regimen for high-risk BC patient group. The CE-marked <em>therascreen</em> PITX2 RGQ PCR assay uses bisulfite-converted genomic DNA (with a genomic DNA input range of 200 ng – 1000&nbsp;ng with 400 ng as recommended input) derived from formalin-fixed paraffin-embedded (FFPE) tumor tissues of these breast cancer patients. The FFPE tissue surface area of ≥&nbsp;100&nbsp;mm<sup>2</sup> (1-2&nbsp;x&nbsp;5&nbsp;µm sections) is the recommended minimal tissue surface area for the standard <em>therascreen</em> PITX2 workflow<sup>19,22</sup>.</p><p>Neoadjuvant chemotherapy before tumor resection is becoming increasingly important in the treatment of high-risk breast cancer<sup>23-25</sup>. In these cases, tumor tissue specimens are limited to core biopsies or fine needle aspirates with tumor tissue surface areas below the minimal surface area of 100 mm<sup>2</sup> which are needed for the standard <em>therascreen</em> PITX2 workflow<sup>19</sup>. </p><p>The workflow protocol described here as “Condensed Efficient Fast (CEF)” PITX2 workflow combines DNA purification and bisulfite conversion in one step and allows to reduce the minimal tumor tissue surface area to 30&nbsp;mm<sup>2</sup> from 1-2&nbsp;x&nbsp;5&nbsp;µm FFPE sections (therefore routinely assessed core biopsies or fine needle aspirations may be used as tissue source) to determine the PITX2 PMR with a recommended bisDNA input of 6&nbsp;ng/µl. The new CEF PITX2 workflow introduces the EpiTect Fast FFPE Bisulfite Kit (Qiagen Cat.&nbsp;No./ID 59844) in the workflow, which consists of the EpiTect Fast FFPE Lysis Kit, containing specialized buffers for efficient deparaffinization and lysis of FFPE tissue slices and the EpiTect Fast DNA Bisulfite Kit for fast and effective bisulfite conversion of the lysate. The DNA quality check is performed after the bisulfite-conversion of gDNA (bisDNA) by photospectrometric assessment (OD260 nm) with the QIAxpert photospectrometer using ssDNA QIAamp plug-in and buffer EB (QIAGEN) as blank control. The workflow steps using the <em>therascreen</em> PITX2 RGQ PCR assay to determine the PMR of PITX2 on the Rotor-Gene with automated data analysis and report using Rotor-Gene AssayManager v2.1 remain unchanged and identical outlined in the “PMR Detection &amp; Analysis” part (see Figure 1). For an overview of the CEF PITX2 workflow steps see Table 1.</p><p>The CEF PITX2 workflow offers scientists and clinicians a tool to conduct scientific research and translational research studies on PITX2 as biomarker in indications with small amounts of FFPE tumor tissues not only in breast cancer but also other cancer indications like head and neck cancer<sup>10</sup>, esophageal squamous cell carcinoma<sup>12</sup>, non-small cell lung cancer<sup>6</sup>, colorectal cancer<sup>8</sup>, ovarian cancer<sup>7</sup>, bladder cancer<sup>26</sup> and prostate cancer<sup>9,11</sup>.</p><p>The CEF PITX2 workflow is currently Research Use Only (RUO) and will be further evaluated in clinical research studies.</p><p>For overall workflow control, one vial containing one histological section (15 to 20 µm thick) of KRAS G13D Reference Standard section (Horizon Discovery, cat. no. HD216) can be used. qPCR controls comprise two positive controls (REF50, REFlow) and two negative controls (NC, NTC).</p><h2>Reagents</h2><p>When working with chemicals, always wear a suitable lab coat, disposable gloves, and protective goggles. Make sure that all kit reagents are not used after their expiry date and<strong> </strong>have been transported and stored under the correct conditions. Tissue specimens should be fixed in 4–10% formalin (neutral buffered formalin is generally accepted) as quickly as possible after surgical removal of the biopsy. Ideally, use a fixation time of 14–24&nbsp;hrs (longer fixation times lead to more severe DNA fragmentation, resulting in poor performance in qPCR/qMSP assays). Thoroughly dehydrate samples prior to embedding (residual formalin can inhibit proteinase K digestion). Five µm thick sections must be cut from the paraffin block. All chemicals and biological materials are potentially hazardous. Specimens and samples are potentially infectious and must be treated as biohazardous materials. Discard samples and assay waste according to your local safety procedures. The entire therascreen PITX2 workflow requires transfer of samples in different tubes, therefore make sure that the traceability of the samples is well maintained at each step.</p><p><br></p><p>REAGENTS</p><p><br></p><p>A list of reagents and consumables to perform this workflow is listed in <strong>Table 2</strong>.</p><p>If not stated otherwise in <strong>Table 2</strong>, store all buffers and components at room temperature (15-25°C).</p><p>An overview of potential hazardous substances used or included in kit constituents and their respective safety actions is presented in <strong>Table 3</strong>.</p><p><br></p><p>REAGENT SET-UP</p><p>The EpiTect Fast FFPE Bisulfite Kit (cat. no. 59844) is used as research-use-only kit for lysis and direct bisulfite conversion of the lysate. Deparaffinization of FFPE histologic sections can be performed using deparaffinization solution. It is necessary to supplement the deparaffinization solution included in the Epitect Fast FFPE bisulfite Kit with deparaffinization solution (cat. no. 19093) due to higher input volumes as normally used according to manufacturer’s instruction (320µl instead of 150µl). Optional: to assess whether purification and bisulfite conversion was conducted correctly, a workflow control can be included. The workflow control validated for the <em>therascreen</em> PITX2 RGQ PCR Kit workflow is the KRAS G13D Reference Standard (15-20 µm section sample; Horizon Discovery, cat. no. HD216).</p><p><br></p><p><strong>Starting material</strong></p><p>The starting material for DNA purification is freshly cut histological sections of FFPE tissue, which can be stored overnight at room temperature, if required. Up to two sections of 5&nbsp;µm (at least 5% tumor content) should be processed for lysis and bisulfite conversion.</p><p><br></p><p><strong>General working comments</strong></p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Before starting an experiment please follow the instructions of the respective kit protocols.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Carefully apply the sample or solution to the QIAamp MinElute column.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Pipet the sample into the QIAamp MinElute column without wetting the rim of the column.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Always change pipet tips between liquid transfers. Aerosol-barrier pipet tips are recommended.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Avoid touching the QIAamp MinElute column membrane with the pipet tip. </p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;After all pulse-vortexing steps, briefly centrifuge the microcentrifuge tubes to remove drops from the inside of the lids.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Open only one QIAamp MinElute column at a time and take care to avoid generating aerosols.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Wear gloves throughout the entire procedure. In case of contact between gloves and sample, change gloves immediately.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Always close QIAamp MinElute columns before placing them in the microcentrifuge. Centrifuge as described in the protocol.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Flow-through fractions may contain hazardous waste and should be disposed of appropriately.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Perform all centrifugation steps at room temperature (15–25°C).</p><p><br></p><p><strong>Kit-specific working comments</strong></p><p><strong>EpiTect Fast FFPE Bisulfite Kit</strong></p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Bisulfite Solution can be stored at room temperature (15–25°C) for at least 6 months.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Critical: Buffer BD, Epitect spin columns and DNA protection buffer are stored at +2 to +8°C until use.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No carrier RNA will be used. </p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add molecular grade ethanol (96–100%) to the bottle of Buffer BW and Buffer BD according to the kit manual instruction. Mix reconstituted buffer by gentle shaking.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Adjust samples and buffers to room temperature.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Deparaffinization Solution solidifies at temperatures below 18°C. Incubate at 30°C to resolve.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Precipitates may form in Lysis Buffer FTB. Make sure all precipitates are dissolved at 30°C.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;After thermal processing of bisulfite conversion in a thermocycler, samples can be stored at room temperature overnight. Exposure to light should be avoided to prevent light-induced degradation of DNA in bisulfite-containing buffers.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Critical: DNA Protect Buffer should turn from green to blue after addition to the DNA-Bisulfite Solution mixture (see procedure step 13), indicating sufficient mixing and correct pH for the bisulfite conversion reaction, incorrect pH could impact the binding of the converted DNA to the column.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;White precipitates may form in the Buffer BD-ethanol mix after extended storage time. These precipitates will not affect the performance of Buffer BD. However, avoid transferring precipitates to the MinElute DNA spin column.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Perform all centrifugation steps at room temperature (15-25°C).</p><p><br></p><p><strong><em>therascreen </em>PITX2 RGQ PCR Kit:</strong></p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Thaw all therascreen PITX2 RGQ PCR components and samples in a refrigerator, on ice, on a cooling block or at room temperature. </p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Clean the bench area that is dedicated for the PCR mix preparation to reduce the risk of template or nuclease contamination.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Critical: if thawed at room temperature, check regularly whether the material has thawed, especially the PITX2 RGQ PCR Master Mix (MMx) as it contains dNTPs that are sensitive to room temperature.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Critical: PITX2 RGQ PCR PPM should be protected from light as it contains dye nucleotides. </p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Critical: repeated thawing and freezing should be avoided and should not exceed a maximum of four freeze–thaw cycles. In the case of multiple test runs on one day, kit constituents can be stored for up to 6 hrs on ice without the need of re-freezing.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Prepare all reactions (reaction mix plus sample) on ice or in a cooling block.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Critical: do not use expired or incorrectly stored components.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Critical: do not use reaction volumes (reaction mix plus sample) of greater than or less than 20 µl.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Use individual, dedicated pipets for setting up the reaction mix and adding templates.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Critical: use extreme caution to prevent contamination of the reaction mix with the materials that are contained in the PITX2 RGQ PCR Reference 50 and PITX2 RGQ PCR Reference Low Control reagents.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Critical: use extreme caution to prevent DNA or PCR product carryover contamination resulting in a false-positive signal.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Critical: use extreme caution to prevent contamination by DNase, which may cause degradation of the template DNA.</p><p><br></p><p><br></p><p><br></p><h2>Equipment</h2><p>For a list of recommended and necessary equipment, <strong>see Table 4.</strong></p><p>2.3.1&nbsp;EQUIPMENT SETUP</p><p><u>General working recommendations:</u></p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Make sure that instruments have been checked and calibrated according to the<strong> </strong>manufacturer’s recommendations. </p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Alteration of incubation times and temperatures may result in false or discordant data.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Use nuclease-free labware (e.g., pipets, pipet tips, reaction vials) and wear disposable gloves when performing the assay.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Prepare PCR reaction mix with dedicated material (pipets, tips, etc.) in a dedicated area where no DNA matrices (DNA, plasmid or PCR products) are introduced.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Refer to the Rotor-Gene Q MDx User Manual and Rotor-Gene AssayManager v2.1 Core Application User Manual for additional warnings, precautions and procedures.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Do not open the Rotor-Gene Q MDx instrument until the run is finished.</p><p>Do not open Rotor-Gene Q MDx tubes after the run is finished. Discard tubes according to your local safety procedures.<strong>﻿</strong></p><h2>Procedure</h2><p>For general handling instructions refer to the section Materials, the general working recommendations as well as to the respective kit protocols. Storage conditions are described in <strong>Table 2</strong>. Regarding safety precautions for specific kit components see <strong>Table 3</strong>. Timing as stated in the following procedure refers to the simultaneous processing of 24 samples.</p><p><br></p><p><strong>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Pre-heat a thermomixer or heated orbital incubator to 56°C. If a thermomixer or heated orbital incubator is not available, a heating block or water bath can be used instead. </p><p><strong>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Using a scalpel, trim excess paraffin off the sample block.</p><p><strong>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Cut one to two sections (5 µm thick) from a histologically confirmed tumor tissue. Troubleshooting. Critical step: if the sample surface has been exposed to air, discard the first 2-3 sections. </p><p><strong>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Immediately place the sections in a 1.5 ml or 2 ml microcentrifuge tube.</p><p><u>Deparaffinization using deparaffinization solution and PK-digestion of FFPE tissue.</u> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p> Timing: 100 min or overnight.</p><p><strong>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Add 320 µl Deparaffinization Solution (Caution: Irritating) and vortex vigorously for 10 seconds.</p><p><strong>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Add 20 µl distilled water, 15 µl Lysis Buffer FTB, and 5 µl proteinase K.(Caution: Irritating)&nbsp;</p><p> Note: A Master Mix comprising distilled water, Lysis Buffer FTB, and proteinase K may be prepared in advance.</p><p><strong>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Vortex and briefly centrifuge to collect the sample in the bottom of the tube.  </p><p> Note: The Deparaffinization Solution will form a layer above the Lysis Buffer FTB with the addition of proteinase K.</p><p><strong>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Incubate the tubes in a thermal block set to 56°C for 60 min up to overnight to lyse the tissues. Ensure that tissues are completely lysed (the solution will become homogeneous). Pause Point: incubation can also take place overnight.</p><p><strong>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Critical step: Incubate at 90°C ± 5°C for 1&nbsp;hr ± 5 min.</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Note: If using only one heating block, leave the sample at room temperature (15–25°C) after the 56°C incubation, until the heating block has reached 90°C.The incubation at 90°C partially reverses formaldehyde modification of nucleic acids. Longer incubation times or higher incubation temperatures may result in more fragmented DNA. Troubleshooting.</p><p><strong>10.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Briefly centrifuge the 1.5 ml tube to remove drops from inside the lid.</p><p><strong>11.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Transfer the complete lysate (about 40 µl, the lower clear phase) in a 200 µl reaction tube or 8-well strip (not provided). Avoid transfer of Deparaffinization Solution as much as possible. &nbsp;&nbsp;&nbsp;</p><p> Note: Small amounts of remaining Deparaffinization Solution have no effect on the bisulfite reaction.</p><p><u>Bisulfite conversion of lysate and bisDNA (bisulfite converted DNA) cleanup</u>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p><u> </u>Timing: 232 min.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p> </p><p><u> </u>Make sure the Bisulfite Solution (Caution: irritant) is completely dissolved. Critical step: do not place dissolved Bisulfite Solution on ice. </p><p><strong>12.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Prepare the bisulfite reactions in 200 µl PCR tubes (not provided). Add each component in the order listed in <strong>Table 5</strong>. </p><p>  Note: </p><p> Critical step: when using the <em>therascreen</em> PITX2 RGQ PCR Kit, the “Low concentration” protocol from the EpiTect Fast Bisulfite Conversion Handbook must always be used, as the concentration of gDNA purified from FFPE samples is usually low.</p><p>Critical step: the bisulfite mix should be immediately vortexed for 5 sec after adding the DNA Protect Buffer (caution: reproductive toxicity, irritant) to protect samples from degradation. Troubleshooting.</p><p><strong>13.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Close the PCR tubes and mix immediately the bisulfite reactions thoroughly. Store the tubes at room temperature (15–25°C). </p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Critical step: DNA Protect Buffer should turn from green to blue after addition to the DNA-Bisulfite Solution mixture, indicating sufficient mixing and correct pH for the bisulfite conversion reaction, or DNA binding to the MinElute DNA spin column. Troubleshooting.</p><p><strong>14.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Perform the bisulfite DNA conversion using a thermal cycler. Program the thermal cycler according to Table 6. The complete cycle should take ~30 min. If using a thermal cycler that does not allow to enter the reaction volume (140 µl), set the instrument to the largest volume setting available.  </p><p><strong>15.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Place the PCR tubes containing the bisulfite reactions into the thermal cycler. Start the thermal cycling incubation. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p> Critical step: as the bisulfite reaction is not overlaid with mineral oil, only thermal cyclers with heated lids are suitable for this procedure. It is important to use PCR tubes that close tightly. &nbsp;&nbsp;</p><p> Pause Point: converted DNA can be left in the thermal cycler overnight (at 20°C) without any loss of qPCR assay performance.</p><p><strong>16.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Briefly centrifuge the PCR tubes containing the bisulfite reactions. Then transfer the complete bisulfite reactions to clean 1.5 ml microcentrifuge tubes. </p><p><strong>17.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Add 310 µl Buffer BL (Caution: harmful) to each sample. Mix the solution by vortexing and then centrifuge briefly.</p><p><strong>18.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Add 250 µl ethanol (96–100 %) to each sample. Mix the solutions by pulse vortexing for 15 sec and centrifuge briefly to remove the drops from inside the lid.</p><p><strong>19.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Transfer the entire mixture from each tube into the corresponding MinElute DNA spin column.</p><p><strong>20.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Centrifuge the spin columns at maximum speed for 1 min. Discard the flow-through and place the spin columns back into the collection tubes.</p><p><strong>21.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Add 500 µl Buffer BW (wash buffer) to each spin column, and centrifuge at maximum speed for 1 min. Discard the flow-through and place the spin columns back into the collection tubes.</p><p><strong>22.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Add 500 µl Buffer BD (desulfonation buffer) to each spin column and incubate for 15 min at room temperature (15–25°C). (Caution: corrosive) If there are precipitates in Buffer BD, avoid transferring them to the spin columns. &nbsp;&nbsp;&nbsp;&nbsp;</p><p> Critical step: the bottle containing Buffer BD should be closed immediately after use to avoid acidification from carbon dioxide in the air.  </p><p> Note: it is important to close the lids of the spin columns before incubation. Troubleshooting.</p><p><strong>23.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Centrifuge the spin columns at maximum speed for 1 min. Discard the flow-through and place the spin columns back into the collection tubes.</p><p><strong>24.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Add 500 µl Buffer BW to each spin column and centrifuge at maximum speed for 1 min. Discard the flow-through and place the spin columns back into the collection tubes.</p><p><strong>25.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Add 500 µl Buffer BW a second time to each spin column and centrifuge at maximum speed for 1 min. Discard the flow-through and place the spin columns back into the collection tube.</p><p><strong>26.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Add 250 µl ethanol (96–100%) to each spin column and centrifuge at maximum speed for 1 min.</p><p><strong>27.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Place the spin columns into new 2 ml collection tubes and centrifuge the spin columns at maximum speed for 1 min to remove any residual liquid.</p><p><strong>28.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Place the spin columns with open lids into a clean 1.5 ml microcentrifuge tube (not provided) and incubate the columns for 5 min at 60°C in a heating block. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p> Critical step: this step ensures the evaporation of any remaining liquid.</p><p><strong>29.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Add 15 µl Buffer EB (elution buffer) directly onto the center of each spin-column membrane and close the lids gently. &nbsp;</p><p> Critical step: do not elute with less than 15 µl buffer as the eluate volume would be too small to proceed with the qPCR step. </p><p><strong>30.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Incubate the spin columns at room temperature for 1 min.</p><p><strong>31.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Centrifuge for 1 min at 15,000 x g (12,000 rpm) to elute the DNA. &nbsp;&nbsp;&nbsp;</p><p> Pause point: we recommend storing purified DNA at 2–8°C for up to 24 hrs. When storing purified DNA for longer than 24 hrs, we recommend storage at –30 to –15°C.</p><p><br></p><p><u>DNA concentration determination - </u>Timing: 25 min.</p><p><strong>32.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>The concentration of DNA is determined by measuring absorbance at 260 nm following the instrument procedure using QIAGEN’s QIAxpert for example (ssDNA plugin: single stranded nucleic acid measurement) or a NanoDrop instrument. Absorbance readings at 260 nm should fall between 0.1 and 1.0 to be accurate. An absorbance of 1 unit at 260 nm corresponds to 33 µg of bisDNA per ml (A260 = 1 = 33 µg/ml). Note: if using the QIAxpert ssDNA plug-in, an EB-blank sample has to be included, which is after measurement automatically subtracted from the OD values. Ideally, the minimal gDNA concentration is 6 ng/µl but samples as low as 2 ng/µl may be processed with risk of “Low input” invalid results. Pause point: bisDNA samples can be stored at -20°C until further use.</p><p><u>&nbsp;</u></p><p><u>qPCR set up, run and analysis - </u>Timing: 105 min.</p><p><br></p><p>The <em>therascreen</em> PITX2 RGQ PCR Kit must be run on the Rotor-Gene Q MDx 5plex HRM instrument using automated interpretation of results with Rotor-Gene AssayManager v2.1. Take time to familiarize yourself with the Rotor-Gene Q MDx instrument and with the Rotor-Gene AssayManager v2.1 software before starting the protocol. See the respective user manuals for the instrument, Rotor-Gene AssayManager v2.1, and the Gamma Plug-in for details.</p><p><strong>33.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Cool a Loading Block 72 x 0.1 ml tubes for 10 min in a deep-freezer or for at least 1 hr at refrigerator temperature.</p><p><strong>34.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Thaw all <em>therascreen</em> PITX2 RGQ PCR Kit components and samples in a refrigerator, on ice, on a cooling block or at room temperature for as long as necessary. </p><p><strong>35.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Place the thawed products on ice, on a cooling block or in the refrigerator until placing them back into –30 to –15°C after use. &nbsp;</p><p> Note: <em>therascreen</em> PITX2 RGQ PCR Kit components can be kept at 2–8°C and protected from light for a maximum of 6 hr if used several times the same day.</p><p><strong>36.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Vortex the tubes (10–12 sec), then centrifuge them briefly before use. Except PITX2 RGQ PCR MMx, which is mixed by pipetting up and down as it contains Taq Polymerase.</p><p><strong>37.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Prepare PITX2 qPCR reaction mix on ice (or using a cooling block) in a 1.5 ml or 2 ml tube according to the number of samples to be processed.</p><p>The pipetting scheme for the preparation of the PITX2 reaction mix, shown in Table 7, is calculated to achieve final reaction volumes of 20 µl after addition of 4 µl bisDNA sample or control. Extra volume is included to compensate for pipetting errors and to allow preparation of sufficient reaction mix for four samples tested in duplicate, plus four controls. If less samples are tested, the reaction mix can be prepared accordingly. Remember to allow for the extra volume to compensate for pipetting errors (one extra well for up to 10 wells and two extra wells for up to 20 wells). Troubleshooting.</p><p><br></p><p> <strong>38.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Vortex (10–12 sec) and briefly centrifuge the PITX2 qPCR reaction mix. Place the qPCR strip tubes on a pre-cooled Loading Block 72 and dispense 16 µl of the PITX2 qPCR reaction mix per strip tube following the example of loading block setup shown in Figure 2.</p><p>The numbers denote positions in the loading block and indicate final rotor position. The positions of the controls are set in the PITX2 assay profile and cannot be changed. If controls are not placed as indicated, the automated result analysis cannot be performed. REF50: PITX2 RGQ PCR Reference 50; REFlow: PITX2 RGQ PCR Reference Low; NC: PITX2 RGQ PCR Negative Control, NTC: PITX2 RGQ PCR NTC (NTC); Sample 1 to 4: bisDNA samples, NA: empty well. Troubleshooting.</p><p><strong>39.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Vortex (10–12 seconds) and briefly centrifuge bisDNA samples, PITX2 RGQ PCR Reference 50 (Ref50), PITX2 RGQ PCR Reference Low (REFlow), PITX2 RGQ PCR Negative Control (NC) and PITX2 RGQ PCR NTC (NTC). Troubleshooting.</p><p><strong>40.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Add 4 µl sample or control material into its corresponding tube according to the setup in Figure 2 to obtain a total volume of 20 µl. Mix gently 5 times by pipetting up and down. Note: be careful to change tips between each tube to avoid false-positive results from contamination by any nonspecific template. Troubleshooting.</p><p><strong>41.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Close all tubes and check that no bubbles are present at the bottom of the tubes. Troubleshooting.</p><p><strong>42.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Return all the therascreen PITX2 RGQ PCR Kit components and samples to the appropriate storage conditions to avoid any material degradation. Pause point: it is highly recommended to start the run as soon as possible after the preparation, however, if the tubes are prepared but cannot be processed directly (e.g. due to instrument unavailability), it is possible to store the plate at 2–8°C and protected from light up to 24 hrs. Troubleshooting.</p><p><strong>43.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Place a 72-Well Rotor on the Rotor-Gene Q MDx rotor holder.</p><p><strong>44.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Fill the rotor with strip tubes previously prepared according to the assigned positions, starting at position 1. Critical step: make sure the first tube is inserted into position 1 and the strip tubes are placed in the correct orientation and positions (important for run validity and traceability of sample). Always keep the four controls (REF50, REFlow, NC and NTC) in positions 1 to 4 so that gain optimization (performed on tube position 1) is always performed on the same control sample. Make sure controls are loaded in the correct order for the automated analysis of the controls (an inversion of controls will invalidate the run by the PITX2 assay profile). Troubleshooting.</p><p><strong>45.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Fill empty positions with empty, closed tubes to fill the rotor entirely.</p><p><strong>46.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Attach the locking ring. </p><p><strong>47.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Load the Rotor-Gene Q MDx instrument with the rotor and locking ring. Close the instrument lid.</p><p><u>Creating a work list and starting the qPCR run</u></p><p><strong>48.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Switch on the Rotor-Gene Q MDx instrument.</p><p><strong>49.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Open the Rotor-Gene AssayManager software by clicking the icon: The Rotor-Gene AssayManager window opens (Figure 3).</p><p><strong>50.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Log in as a user with the “Operator” role in the closed mode. Click “OK“. The Rotor-Gene AssayManager screen opens (Figure 4).</p><p><strong>51.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Check that the RGQ is correctly detected to the software before launching the run.</p><p><strong>52.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Select the “Setup” tab. Note: The overall functionalities of the Setup environment and of “Creating/Editing a Work List” are described in the Rotor-Gene AssayManager v2.1 Core Application User Manual.</p><p><strong>53.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Click “New work list” (Figure 4).</p><p><strong>54.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Select the PITX2 assay profile from the list of available assay profiles (Figure 5).</p><p><strong>55.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Transfer the selected assay profile to the list of selected assay profiles by clicking on the arrow (to the right of the assay profile name). The assay profile should now be displayed in the selected assay profiles list (Figure 5).</p><p><strong>56.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>In the “Assays” tab, complete the yellow fields: number of samples in accordance with your plate setup (Figure 6). Note: the number of samples does not correspond to the number of wells and does not include controls. Samples are tested in duplicates; therefore, one sample corresponds to two wells. For example, the number of samples to be inserted is 4 for the plate of 12 wells presented in Figure 2.</p><p><strong>57.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Select the “Kit Information” tab. Insert the kit information by either selecting “Use kit bar code” (and scan the bar code) or selecting “Enter kit information manually” and inserting manually the kit information found on the label of the <em>therascreen</em> PITX2 RGQ PCR Kit box:</p><p><strong>1)</strong> Material number</p><p><strong>2)</strong> Expiry date</p><p><strong>3)</strong> Lot number</p><p><strong>58.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Select the “Samples” tab. A list with the sample details is shown. This list represents the expected layout of the rotor.</p><p><strong>59.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Enter the sample identification as well as any optional sample information as a comment for each sample (Figure 7).</p><p><strong>60.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Select “Properties” and enter a work list name (Figure 8; “a”). </p><p><strong>61.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Enable the check box “worklist is complete (can be applied)”( Figure 8; “b”).</p><p><strong>62.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Save the work list (Figure 8; “c”). Optional: press “Print work list” to print the work list. Printing the work list may help with the preparation and setup of the run. The sample details are included as part of the work list.</p><p><strong>63.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Select the corresponding work list from the work list manager and click “Apply” (Figure&nbsp;8;&nbsp;“d”). Alternatively, if the work list is still open, click “Apply”. Note: check that the Rotor-Gene Q MDx is correctly detected by the software before launching the run.</p><p><strong>64.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Enter the experiment name.</p><p><strong>65.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Select the cycler to be used in “Cycler Selection”.</p><p><strong>66.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Check that the locking ring is correctly attached and confirm on the screen that the locking ring is attached.</p><p><strong>67.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Click “Start run”. The qPCR run should start. Troubleshooting.</p><p><u>Release and report qPCR results</u></p><p>The general functionality of the approval environment is described in the Rotor-Gene AssayManager v2.1 Gamma Plug-in User Manual. After a run has been finished and the cycler has been released, the experiment will be saved in the internal database. The analysis of the acquired data is performed automatically according to the rules and parameter values defined by the assay profile.</p><p><strong>68.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>When the run has finished, click on “Finish run” to analyze and export data. Critical step: until this step is completed, the experiment is not saved in the internal database.</p><p><strong>69.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>After clicking “Finish run”, enter the password and click “Release and go to approval” (Figure 9).</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For users logged in with the “Approver” role, click “Release and go to approval”. </p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For users logged in with the “Operator” role, click “Release”. </p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;If “Release and go to approval” was clicked, the results for the experiment are displayed in the “Approval” environment.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;If “Release” was clicked by a user with the “Operator” role, someone with an “Approver” role must log in and select the “Approval” environment. Note: in the “Approval” tab, experiments can be analyzed by shifting between each tab (i.e., experiment, assay, audit, trail, run control results).</p><p><br></p><p><strong>70.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Check the amplification curves for each sample, tick the first box on the right side of the “flags” column (the box becomes green) (Figure 10). Troubleshooting.</p><p><strong>71.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Click “Release/report data” (at the bottom right of the window) to create a .pdf report and to save the LIMS file. A copy is automatically saved in the following location:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </p><p> C:\Documentsandsettings\AllUsers\Documents\QIAGEN\RotorGeneAssayManager\Export\Reports.</p><p><strong>72.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Close the pdf file and return to the Rotor-Gene AssayManager. Click “OK” each time it is asked.</p><p><strong>73.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Go to the “Archive” tab to export the .rex file (Figure 11; “a”). Check that “start date” and “end date” are correct (Figure 11; “b”) and click “apply filter”. (Figure 11; “c”) Select the experiment to export (Figure 11; “d”) then click on “Show assays” (Figure 11; “e”).</p><p><strong>74.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Export the.rex file (the file is saved in C:\Documents and settings\AllUsers\Documents \QIAGEN\RotorGeneAssayManager\Export\Experiments). &nbsp;&nbsp;</p><p> Note: the software automatically generated a LIMS file in the following location:</p><p>C:\Documents and settings\All Users\Documents\QIAGEN\RotorGeneAssayManager\Export\LIMS</p><p><strong>75.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong>Unload the Rotor-Gene Q MDx instrument and discard the strip tubes according to your local safety regulations.</p><p><br></p><h2>Troubleshooting</h2><p>Analysis of the <em>therascreen</em> PITX2 RGQ PCR Kit results for each control and sample is performed automatically by the Rotor-Gene AssayManager v2.1 associated with the Gamma Plug-in v1.0 and the PITX2 assay profile hereafter referred to as the PITX2 Assay Package. The PITX2 Assay Package analyzes amplification curves, and may invalidate non-conforming curves, depending on their shape and noise amplitude. If this is the case, a flag will appear with the invalidated curve. Cycle threshold values (CT) are determined automatically for each marker separately by the software. </p><p>PMR values for each well are calculated separately using a modified 2exp<sup>DCT</sup> method with the following formula: PMR (sample): &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1/(1+2exp(CT<sub>meth</sub>(FAM) – CT<sub>unmeth</sub> (HEX)))<sup>19</sup>. A mean value is determined (PMRmean) and the difference for the PMRs from technical duplicates of a sample is calculated for quality control of intra-assay variation (deltaPMR; see Table 9). To determine assay validity, the PITX2 Assay Package also analyzes the run controls, i.e., PITX2 RGQ PCR Reference 50 (REF50), PITX2 RGQ PCR Reference Low (REFlow), PITX2 RGQ PCR Negative Control (NC) and PITX2 RGQ PCR NTC (NTC). Validity for each control and sample is based on compliance of CT and/or PMR values with pre-defined specifications (see Table 10).</p><p>Note: If at least one control is invalid, the results obtained for all test samples are considered invalid and no PMR results are displayed. Finally, an integer PMR value is assigned to the samples by averaging of the two PMR results obtained for each sample replicate. In case of invalid results, retests are required. If the assay is invalid, i.e., one of the four controls invalid, the entire run including all tested samples should be retested. If the assay is valid but one or several samples are invalid, the invalid sample(s) should be retested after investigating the type of failure. A workflow of the retest procedure is presented in Figure 12.</p><p><u>Workflow Control: </u></p><p>The KRAS G13D Reference Standard (15-20 µm section sample; Horizon Discovery, cat. no. HD216)) should give a PMR value between 30 and 50.</p><p><br></p><h2>Time Taken</h2><p>Timings for each step are listed in the procedure section. </p><h2>Anticipated Results</h2><p><strong><em>therascreen PITX2 RGQ PCR kit – Standard PITX2 workflow</em></strong></p><p>The <em>therascreen</em> PITX2 RGQ PCR Kit with its so-called ‘standard’ PITX2 workflow was validated and launched as CE-marked IVD Kit for high risk ER+, HER2- and nodal positive breast cancer patients with adjuvant anthracycline containing chemotherapy. The PITX2 Percent Methylation Ratio (PMR) obtained for each patient tumor tissue sample (≥100 m<sup>2</sup> surface area form 1-2 x 5µm sections; genomic DNA input 200‑1000 ng with recommended input of 400 ng and LOD of 4) provides information to the treating physician about whether a patient is likely to benefit from anthracycline-based chemotherapy. If the PMR obtained is equal to or lower than 12, the patient is likely to benefit from anthracycline-based chemotherapy. In contrast, if the PMR obtained is higher than 12, an alternative treatment may be proposed, because the patient has a lower probability of responding to anthracycline-based chemotherapy<sup>20</sup>.</p><p><strong><em>Condensed Efficient Fast (CEF) PITX2 workflow applications</em></strong></p><p>The primary objective was to assess if the CEF PITX2 workflow allows a reduction of DNA input. A reduction of DNA input is required to expand the Intended Use of the standard workflow with <em>therascreen</em> PITX2 RGQ PCR Kit to core biopsy samples or tumor tissues below 100 mm<sup>2</sup> not only in breast cancer, but potentially also in other cancer indications.</p><p><strong>Comparison of the new CEF PITX2 workflow with standard <em>therascreen </em>PITX2 workflow</strong></p><p>To compare the new CEF workflow with the standard workflow, an exploratory study was performed with Formalin-Fixed Paraffin-Embedded (FFPE) tissue (1-2&nbsp;x&nbsp;5 µm). 100 FFPE samples (27 TNBC and 73 high risk ER+/HER2- BrCa) with known PMR values derived from previous studies with the standard <em>therascreen</em> PITX2 workflow and samples were processed according to the new CEF PITX2 workflow. bisDNA was used undiluted in the qPCR analysis (range input of bisDNA concentration:1.9 - 438.5 ng/µl). The tissue surface area of the samples ranged from 30-508 mm<sup>2</sup>. The success of the study was determined with the following acceptance criteria a) correlation of PMR values determined with both PITX2 workflows with a regression coefficient of r&nbsp;≥&nbsp;0.7, b) 90% of samples should have ≥ 50 amplifiable copies. This would equal at least an LOD of PMR 4 as established for the standard <em>therascreen</em> PITX2 workflow, and c) the First-Time Sample Pass Rate (FTPR) should be at ≥&nbsp;90%.</p><p>&nbsp;<u>Linear regression analysis for PMR value comparison between standard and CEF workflow.</u></p><p>The correlation between PMR values (n=100) derived from standard <em>therascreen</em> PITX2 and CEF PITX2 workflow were assessed with linear regression analysis.</p><p>The PMR values of PITX2 methylation derived from CEF workflow and standard workflow showed a high correlation with a regression coefficient of r=0.87 (r<sup>2</sup>=0.7586). The study acceptance criteria for the linear regression analysis (r&nbsp;≥&nbsp;0.7) was outreached with the regression coefficient of r&nbsp;=&nbsp;0.87 (Figure 13).</p><p><u>Determination of amplifiable copy numbers obtained with the new CEF PITX2 workflow</u></p><p>The amplifiable copy number is calculated by using REF50 as reference which contains 1000 methylated and 1000 unmethylated copies by the formula:&nbsp;&nbsp;&nbsp;</p><p>2^-deltaCt: copies (sample; meth or unmeth)=1000/2^(CTsample - CTREF50).&nbsp;</p><p>The total amplifiable copy number equals the sum of methylated and unmethylated amplified copies.</p><p>The copy numbers of the 100 samples tested ranged from 80 to 145,411 copies amplified (Figure 14).</p><p>In 100% of samples more than 50 copies (equals a calculated LOD of 4) were detected. 98% of samples had more than 100 copies (equals a calculated LOD of 2), 96% of samples had more than 250 copies (equals a calculated LOD of 0.8) and 94% of samples (equals a calculated LOD of 0.4) had more than 500 copies (Table 11).</p><p>The study acceptance criteria for the CEF PITX2 workflow (90% of samples with ≥ 50 amplifiable copies) was reached with 100% of samples. The results also show a potential of reducing the LOD from 4 to 1-2 at a minimal tissue surface area of ≥ 100 mm<sup>2</sup>.</p><p><u>First-Time Sample Pass Rate (FTPR)</u></p><p>The bisDNA concentration for each of the 100 FFPE samples was determined by OD 260 measurement with the QIAxpert ssDNA plugin (A260 = 1 = 33 ng/µl). The bisDNA concentration ranged between 1.9 ng/µl to 438.5 ng/µl. All samples were valid in the first qPCR run according to sample validity criteria in the <em>therascreen</em> PITX2 RGQ PCR Assay profile 1.0.1. which equals a FTSPR of 100%.</p><p>The study acceptance criteria for the CEF PITX2 workflow (FTPR&nbsp;≥&nbsp;90%) was met with FTSPR of 100% for undiluted samples with&nbsp;≥&nbsp;50 amplifiable copies.</p><p>In summary, all three acceptance criteria set for the new CEF PITX2 workflow were reached and showed the potential of the CEF PITX2 workflow to reduce the DNA input/tumor tissue surface area for PMR analysis. These results prompted us to determine the recommendable bisDNA input and required tumor surface tissue area for the CEF PITX2 workflow.</p><p><strong>Determination of DNA input to be recommended for the CEF PITX2 workflow</strong></p><p>To assess the lower limit of bisDNA input to obtain valid qPCR results with the <em>therascreen</em> PITX2 Kit applying the CEF PITX2 workflow and obtaining a First-Time Sample Pass Rate ≥&nbsp;90%, bis DNA samples ranging from 2ng/ml up to 6 ng/ml were tested. The PMR values for PITX2 in the diluted samples were compared to the PMR values in the undiluted samples. For the recommendation of the optimal DNA input, the following criteria had to be fulfilled: a) the correlation of the PMR values obtained in the undiluted versus the diluted samples should reach a regression coefficient of r ≥ 0.9, and b) a FTSPR ≥&nbsp;90% should be delivered.</p><p>Table 12 depicts the regression coefficients obtained by comparing the PMR values in various bisDNA concentrations used in these sets of experiments. The results demonstrate that the threshold of a regression coefficient of r ≥ 0.9 was reached with all dilutions ranging from 2-6 ng/µl.&nbsp;</p><p>The second objective for the optimal DNA input was based on the criteria of FTSPR ≥&nbsp;90%. Table 13 summarizes the results of the FTSPR achieved based on testing at each DNA input concentration 30 to 50 samples. The predetermined threshold of FTSPR ≥&nbsp;90% was only achieved at 6 ng/ml.&nbsp;</p><p>Therefore, these data define 6&nbsp;ng/µl bisDNA input as the “recommended input” for the new PITX2 CEF workflow. Lower bisDNA concentrations down to 2&nbsp;ng/µl may be possible as “minimal input” but associated with a higher sample invalidity rate.</p><p><strong><u>Calculation of amplifiable copy numbers</u></strong></p><p>The amplifiable copy numbers were calculated by using REF50 as reference which contains 1000 methylated and 1000 unmethylated copies by the formula:</p><p>2^-deltaCt: copies (sample; meth or unmeth)=1000/2^(CTsample - CTREF50).&nbsp;</p><p>The total amplifiable copy number equals the sum of methylated and unmethylated amplified copies.</p><p>The copy numbers were calculated for the bisDNA concentrations obtained at 2-6 ng/ml DNA input. Table 14 outlines the copy numbers achieved at the various DNA input concentrations.&nbsp;</p><p>Concentrations below the recommended bisDNA input of 6&nbsp;ng/µl showed a minimum of amplifiable copies below 50 and an average between 113 to 228 for 2-4 ng/µl bisDNA, respectively. For the recommended bisDNA input of 6 ng/µl, the achieved amplifiable copy number was at minimum 62 and at average 373 which is sufficient to reach a FTSPR of 92%.</p><p><strong><u>Calculation of recommended tissue surface area (mm</u></strong><sup><strong><u>2</u></strong></sup><strong><u>) for CEF PITX2 workflow</u></strong></p><p>The tissue surface area (mm<sup>2</sup>) was calculated according to the ratio of dilution (6 ng/µl) to the original undiluted bisDNA concentration of the study group (n=46) extrapolated to the whole tissue surface area (mm<sup>2</sup>) for valid samples. Tumor tissue surface area was determined from scans of Haematoxilin-Eosine&nbsp;stained sections on slides with the Hamamatsu&nbsp;Nanozoomer virtual microscope using the freehand region tool of the NDP.view software version 2. One sample was omitted from 46 valid samples due to missing tissue surface area information which resulted in n=45 samples for data analysis (Table 15).</p><p>The calculated tissue surface area ranges from 3.1 to 42.9 mm<sup>2</sup> with an average of 12.5 mm<sup>2</sup> (Table 15 A).</p><p>The minimal calculated tissue surface area to reach ≥ 90% sample validity is 30 mm<sup>2</sup> (Table 15 B) which is 3.3 times lower than the recommended tissue surface of 100 mm<sup>2</sup> for the standard <em>therascreen</em> PITX2 workflow.</p><p>With a recommended input of 6&nbsp;ng/µl bisDNA for qPCR analysis, the required tissue surface area is reduced by 3.3-fold, reducing the surface area from 100&nbsp;mm<sup>2</sup> (standard PITX2 workflow) to about 30 mm<sup>2</sup> (CEF PITX2 workflow).</p><p>Therefore, the new CEF PITX2 workflow facilitates to determine PMR from tumor core biopsies (on average between 20-40 mm<sup>2</sup> per FFPE tissue section) from 1-2 x 5 µm sections.</p><p>The recommended DNA input is decreased by 4.5-fold with the new CEF PITX2 workflow compared to the standard <em>therascreen</em> PITX2 workflow. Table 16 highlights the specifications of the standard and CEF workflow.</p><p><br></p><p><br></p><h2>References</h2><p>1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Ferlay, J.<em> et al.</em> Cancer incidence and mortality patterns in Europe: Estimates for 40 countries and 25 major cancers in 2018. <em>Eur J Cancer</em> <strong>103</strong>, 356-387, doi:10.1016/j.ejca.2018.07.005 (2018).</p><p>2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Fitzmaurice, C.<em> et al.</em> Global, Regional, and National Cancer Incidence, Mortality, Years of Life Lost, Years Lived With Disability, and Disability-Adjusted Life-Years for 29 Cancer Groups, 1990 to 2017: A Systematic Analysis for the Global Burden of Disease Study. <em>JAMA Oncol</em>, doi:10.1001/jamaoncol.2019.2996 (2019).</p><p>3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Goldhirsch, A.<em> et al.</em> Progress and promise: highlights of the international expert consensus on the primary therapy of early breast cancer 2007. <em>Ann Oncol</em> <strong>18</strong>, 1133-1144, doi:10.1093/annonc/mdm271 (2007).</p><p>4&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Harbeck, N. &amp; Gnant, M. Breast cancer. <em>Lancet</em> <strong>389</strong>, 1134-1150, doi:10.1016/S0140-6736(16)31891-8 (2017).</p><p>5&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Ditsch, N.<em> et al.</em> AGO Recommendations for the Diagnosis and Treatment of Patients with Early Breast Cancer: Update 2019. <em>Breast Care (Basel)</em> <strong>14</strong>, 224-245, doi:10.1159/000501000 (2019).</p><p>6&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Dietrich, D.<em> et al.</em> DNA Methylation of the Homeobox Genes PITX2 and SHOX2 Predicts Outcome in Non–small-cell Lung Cancer Patients. <em>Diagn Mol Pathol</em> <strong>21</strong>, 93-104 (2012).</p><p>7&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Fung, F. K.<em> et al.</em> Increased expression of PITX2 transcription factor contributes to ovarian cancer progression. <em>PLoS One</em> <strong>7</strong>, e37076, doi:10.1371/journal.pone.0037076 (2012).</p><p>8&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hirose, H.<em> et al.</em> The significance of PITX2 overexpression in human colorectal cancer. <em>Ann Surg Oncol</em> <strong>18</strong>, 3005-3012, doi:10.1245/s10434-011-1653-z (2011).</p><p>9&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Luan, Z. M., Zhang, H. &amp; Qu, X. L. Prediction efficiency of PITX2 DNA methylation for prostate cancer survival. <em>Genet Mol Res</em> <strong>15</strong>, doi:10.4238/gmr.15026750 (2016).</p><p>10&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Sailer, V.<em> et al.</em> Clinical performance validation of PITX2 DNA methylation as prognostic biomarker in patients with head and neck squamous cell carcinoma. <em>PLoS One</em> <strong>12</strong>, e0179412, doi:10.1371/journal.pone.0179412 (2017).</p><p>11&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Uhl, B.<em> et al.</em> PITX2 DNA Methylation as Biomarker for Individualized Risk Assessment of Prostate Cancer in Core Biopsies. <em>J Mol Diagn</em> <strong>19</strong>, 107-114, doi:10.1016/j.jmoldx.2016.08.008 (2017).</p><p>12&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Zhang, J. X.<em> et al.</em> PITX2: a promising predictive biomarker of patients' prognosis and chemoradioresistance in esophageal squamous cell carcinoma. <em>Int J Cancer</em> <strong>132</strong>, 2567-2577, doi:10.1002/ijc.27930 (2013).</p><p>13&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Aubele, M.<em> et al.</em> The Predictive Value of PITX2 DNA Methylation for High-Risk Breast Cancer Therapy: Current Guidelines, Medical Needs, and Challenges. <em>Dis Markers</em> <strong>2017</strong>, 4934608, doi:10.1155/2017/4934608 (2017).</p><p>14&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Duffy, M. J.<em> et al.</em> Methylated genes as new cancer biomarkers. <em>Eur J Cancer</em> <strong>45</strong>, 335-346, doi:10.1016/j.ejca.2008.12.008 (2009).</p><p>15&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Harbeck, N.<em> et al.</em> Multicenter study using paraffin-embedded tumor tissue testing PITX2 DNA methylation as a marker for outcome prediction in tamoxifen-treated, node-negative breast cancer patients. <em>J Clin Oncol</em> <strong>26</strong>, 5036-5042, doi:10.1200/JCO.2007.14.1697 (2008).</p><p>16&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Maier, S.<em> et al.</em> DNA-methylation of the homeodomain transcription factor PITX2 reliably predicts risk of distant disease recurrence in tamoxifen-treated, node-negative breast cancer patients--Technical and clinical validation in a multi-centre setting in collaboration with the European Organisation for Research and Treatment of Cancer (EORTC) PathoBiology group. <em>Eur J Cancer</em> <strong>43</strong>, 1679-1686, doi:10.1016/j.ejca.2007.04.025 (2007).</p><p>17&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Nimmrich, I.<em> et al.</em> DNA hypermethylation of PITX2 is a marker of poor prognosis in untreated lymph node-negative hormone receptor-positive breast cancer patients. <em>Breast Cancer Res Treat</em> <strong>111</strong>, 429-437, doi:10.1007/s10549-007-9800-8 (2008).</p><p>18&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hartmann, O.<em> et al.</em> DNA methylation markers predict outcome in node-positive, estrogen receptor-positive breast cancer with adjuvant anthracycline-based chemotherapy. <em>Clin Cancer Res</em> <strong>15</strong>, 315-323, doi:10.1158/1078-0432.CCR-08-0166 (2009).</p><p>19&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;QIAGEN. <em>therascreen PITX2 RGQ PCR Kit Handbook</em>, &lt;<a href="https://www.qiagen.com/fr/resources/resourcedetail?id=0c01a2fd-1678-4edd-b432-0215579ac717&amp;lang=en" rel="noopener noreferrer" target="_blank">https://www.qiagen.com/fr/resources/resourcedetail?id=0c01a2fd-1678-4edd-b432-0215579ac717&amp;lang=en</a>&gt; (2017).</p><p>20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Schmitt, M.<em> et al.</em> Clinical Validation of PITX2 DNA Methylation to Predict Outcome in High-Risk Breast Cancer Patients Treated with Anthracycline-Based Chemotherapy. <em>Breast Care (Basel)</em> <strong>13</strong>, 425-433, doi:10.1159/000493016 (2018).</p><p>21&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Schricker, G.<em> et al.</em> Clinical performance of an analytically validated assay in comparison to microarray technology to assess PITX2 DNA-methylation in breast cancer. <em>Sci Rep</em> <strong>8</strong>, 16861, doi:10.1038/s41598-018-34919-1 (2018).</p><p>22&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Napieralski, R.<em> et al.</em> therascreen PITX2 RGQ PCR assay for the assessment of PITX2 DNA-methylation status to investigate the role of the transcription factor PITX2 and the regulation of the Wnt/ß-catenin pathway in pathophysiological processes. <em>Protocol Exchange</em>, doi:10.1038/protex.2018.022 (2018).</p><p>23&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Cain, H.<em> et al.</em> Neoadjuvant Therapy in Early Breast Cancer: Treatment Considerations and Common Debates in Practice. <em>Clinical oncology (Royal College of Radiologists (Great Britain))</em> <strong>29</strong>, 642-652, doi:10.1016/j.clon.2017.06.003 (2017).</p><p>24&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Curigliano, G.<em> et al.</em> De-escalating and escalating treatments for early-stage breast cancer: the St. Gallen International Expert Consensus Conference on the Primary Therapy of Early Breast Cancer 2017. <em>Ann Oncol</em> <strong>28</strong>, 1700-1712, doi:10.1093/annonc/mdx308 (2017).</p><p>25&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Early Breast Cancer Trialists' Collaborative, G. Long-term outcomes for neoadjuvant versus adjuvant chemotherapy in early breast cancer: meta-analysis of individual patient data from ten randomised trials. <em>Lancet Oncol</em> <strong>19</strong>, 27-39, doi:10.1016/S1470-2045(17)30777-5 (2018).</p><p>26&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Lopez, J. I.<em> et al.</em> A DNA hypermethylation profile reveals new potential biomarkers for the evaluation of prognosis in urothelial bladder cancer. <em>APMIS</em> <strong>125</strong>, 787-796, doi:10.1111/apm.12719 (2017).</p></body></html>