<html><head><title>A pipeline for systematic yeast 2-hybrid matricial screening in liquid culture.</title></head><body><h1>A pipeline for systematic yeast 2-hybrid matricial screening in liquid culture.</h1>Authors: Monachello Dario, Guillaumot Damien, Lurin Claire<br>Abstract: <p>Physical interactions mediated by proteins are a critical element of biological systems, and the analysis of interaction partners can provide valuable hints about unknown functions of a protein. Two major classes of experimental approaches are used for protein interaction mapping: analysis of direct interactions using binary methods such as yeast two-hybrid (Y2H) or split ubiquitin, and analysis of protein complexes through affinity purification followed by mass spectrometry. Thanks to his flexibility to low- and high-throughput approaches and a low operating cost the Y2H assay is widely used for high-throughput interaction mapping experiments. Moreover, it has now been shown that high-throughput methods can produce highly reliable interactome datasets<sup>1 2 3 4</sup>. Notably, in 2011 a proteome-wide binary protein-protein interaction map of the plant <em>Arabidopsis thaliana</em> <sup>5</sup> (Arabidopsis Interactome Mapping project – AIM) was described using a high-throughput binary interactome mapping pipeline based on the Y2H system and using a collection of ~8,000 open reading frames (8k_space). Here we describe a liquid pipeline for a high-throughput binary protein–protein Y2H screen of a pool of 50 proteins used as baits against a collection of ~12,000 Arabidopsis proteins encoded by sequence-verified ORFs (12k_space)<sup>6 7</sup>.</p><br>Keywords: Yeast 2-hybrid screen, high-throughput interaction mapping, liquid pipeline.<br><h2>Introduction</h2><p>The protocol described here is an adaptation of the one presented in Dreze et al., 2010<sup>7</sup>. The same low copy number yeast expression vectors expressing DB-X and AD-Y hybrid proteins and the two Y2H strain backgrounds, <em>S. cerevisiae </em>Y8930 (for DB clones; MATα) and <em>S. cerevisiae </em>Y8800 (for AD clones; MATa)<em> </em>which harbor the following genotype:<em> leu2-3,112 trp1-901 his3-200 ura3-52 gal4D gal80D GAL2-ADE2 LYS2::GAL1-HIS3 MET2::GAL7-lacZ cyh2R </em>were used<em>. </em>The reporter genes<em> GAL2-ADE2 </em>and<em> LYS2::GAL1-HIS3 </em>are integrated into the yeast genome. Expression of the<em> GAL1-HIS3 </em>reporter gene has to be tested with<em> </em>1 mM 3AT (3-amino-1,2,4- triazole, a competitive inhibitor of the HIS3 gene product). A pool of 50 proteins of interest - POI used as baits is screened against the 12k_space AD-collection or the corresponding space DB-collection (without autoactivators) depending on the autoactivation state of the baits.</p><p><br></p><h2>Reagents</h2><p>Gateway® LR Clonase Enzyme Mix (Invitrogen)</p><p>Competent DH5a-T1R (Invitrogen)</p><p>HiFi Platinum Taq polymerase buffer (Invitrogen)</p><p>Lysogeny broth (LB) media</p><p>Super Optimal Broth (SOC) media</p><p>Salmon Sperm DNA</p><p>Tris-EDTA buffer (TE) 10 mM Tris–HCl (pH 8.0), 1 mM EDTA (pH 8.0),</p><p>Lithium acetate </p><p>Polyethyleneglycol (PEG) 3350 </p><p>Yeast extract-peptone-dextrose<strong> (</strong>YEPD) media</p><p>Zymolase 20T (21,100 U/g, Seikagaku Corp.)</p><p>Synthetic complete (SC) media </p><p>3-Amino-1,2,4-triazole (3AT) </p><p><br></p><h2>Equipment</h2><p>Qpix2 XT (Molecular Devices)</p><p>MULTIDROP 384 (Thermo Ficher)</p><p>Liquid handling platform or Multichannel pipette</p><p>Tecan Infinite M200 PRO</p><p>Low profile microplates 96 (X6011) and 384 (X7001) well (Molecular Devices)</p><p>Water bath </p><p>Pipetman</p><p>Pipette tips</p><p>50 ml polypropylene conical tubes </p><p>Table top centrifuge</p><p>96 well PCR plate</p><p>Incubator </p><p>Standard microcentrifuge tubes, 1.5 ml and 2 ml</p><p>Petri dishes</p><p>Thermocycler with programmable temperature stepping functionality, 96 well</p><p><br></p><h2>Procedure</h2><p>&nbsp;</p><p><strong>A)&nbsp;&nbsp;Generation of expression plasmids</strong></p><p>The 50 proteins used as baits were transferred into pDest-DB and pDest-AD vectors in simple single-step reaction by Gateway recombination. Completed recombination reactions are then transformed into Escherichia coli, grown for 18 h, and plasmids are isolated.</p><p>&nbsp;</p><p><strong>Protocol 1A: Gateway LR recombinational cloning</strong></p><p>For each bait combine in 96-well PCR plate:</p><p>1)&nbsp;&nbsp;&nbsp;&nbsp;</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1 µl of entry clone (10 ng/µl)</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1 µl of destination vector ( 100 ng/µl)</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1 µl clonase buffer 5x</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1.6 µl TE 1x</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0,4 µl of LR clonase enzyme mix (Invitrogen) (keep this mix on ice)</p><p>2)&nbsp;&nbsp;&nbsp;Homogenize by gently pipetting up and down.</p><p>3)&nbsp;&nbsp;&nbsp;Incubate at 25 °C for 18 h</p><p>4)&nbsp;&nbsp;&nbsp;Add 2 µl of Proteinase K (2µg/µl)&nbsp;solution</p><p>5)&nbsp;&nbsp;&nbsp;Incubate 15 min at 37°C</p><p>&nbsp;</p><p>&nbsp;</p><p><strong>Protocol 2A: Bacterial transformation </strong></p><p>1)&nbsp;&nbsp;&nbsp;Thaw competent DH5a-T1R (Invitrogen) cells on ice</p><p>2)&nbsp;&nbsp;&nbsp;Add 100 µl of competent cells into 96-well PCR plate containing 5 µl Gateway LR reaction mix</p><p>3)&nbsp;&nbsp;&nbsp;Incubate on ice water for 20 min</p><p>4)&nbsp;&nbsp;&nbsp;Heat shock cells in a water bath at 42 °C for 15 sec</p><p>5)&nbsp;&nbsp;&nbsp;Incubate on ice water for 5 min</p><p>6)&nbsp;&nbsp;&nbsp;Transfer the cells into a 96 deep well plate filled with 900 µl SOC</p><p>7)&nbsp;&nbsp;&nbsp;Incubate at 37 °C for 1 h</p><p>8)&nbsp;&nbsp;&nbsp;Plate out 300µl of reaction onto LB plates containing 100mg/l Ampicillin</p><p>9)&nbsp;&nbsp;&nbsp;Incubate over night at 37°C</p><p>10) Pick a single colony from the plate and inoculate a 96 deep well plate filled with 1 ml of LB containing 100mg/l Ampicillin</p><p>11) Incubate at 37 °C for 18 h</p><p>12) Remove 5 µl for subsequent analysis by PCR</p><p>13) Remove 100 µl of the overnight culture, mix with 100 µl of 40% (w/v) autoclaved glycerol and store at 80 °C into a 96-well Low profile microplates</p><p>14) Use the remainder of the overnight culture for plasmid isolation</p><p>&nbsp;</p><p>&nbsp;</p><p><strong>Protocol 3A: Bacterial culture PCR</strong></p><p>Dilute 5 µl of bacterial culture into 95 µl of sterile water and mix by pipetting up and down. Keep bacterial cultures at 4 °C until PCR results are determined.&nbsp;</p><p>&nbsp;</p><p>For each reaction, prepare in a 96-well PCR plate on ice:</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1.1 µl of HiFi Platinum Taq polymerase buffer 10x (Invitrogen)</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.4 µl of 50 mM MgSO4 (final concentration 1.8 mM)</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.11 µl&nbsp;of 40 mM dNTPs (final concentration 400 nM)</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.022 µl of 200 µM AD or DB forward primer (final concentration180 nM)</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.022 µl of 200 µM Term reverse primer (final concentration 180 nM)</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.066 µl of HiFi Platinum Taq polymerase (Invitrogen)</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;8.28 µl of filter-sterilized water</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add 1 µl of the diluted bacterial culture as DNA template</p><p>&nbsp;</p><p>Place the PCR plate on a thermocycler and run the following program:</p><p>&nbsp;</p><p>Step 1: Denaturation at 94 °C for 4 min.</p><p>Step 2: Denaturation at 94 °C for 30 s.</p><p>Step 3: Annealing at 58 °C for 30 s.</p><p>Step 4: Elongation at 68 °C for x min (depending on the length of the longer ORF)</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Repeat Steps 2-3-4, 34 times.</p><p>Step 5: Final elongation at 68 °C for 10 min.</p><p>Step 6: Hold at 10 °C.</p><p>&nbsp;</p><p>Primer sequences:</p><p>&nbsp;</p><p>AD: 5’-CGCGTTTGGAATCACTACAGGG-3’</p><p>DB: 5’-GGCTTCAGTGGAGACTGATATGCCTC-3’</p><p>Term: 5’-GGAGACTTGACCAAACCTCTGGCG-3’</p><p>&nbsp;</p><p>Once PCR reactions are completed, analyze 5 µl of PCR product on a 1% agarose. Successful LR reactions will give rise at ORFs sizes to which ~280 bp of vector sequences are added due to the AD, DB, and Term primer positions (the PCR amplicon from a destination vector containing the Gateway cassette has an expected size _1.9 kb). The remaining of the cultures are used for plasmid isolation.</p><p>&nbsp;</p><p>&nbsp;</p><p><strong>B)&nbsp;&nbsp;Yeast transformation and identification of autoactivating DB-X hybrid proteins</strong></p><p>Isolated destination clones were individually transferred into competent Y8930 (MATα) and Y8800 (MATa) strains<em> </em>by<em> </em>Lithium-Acetate based transformation. Identification of DB-ORF autoactivators is achieved in haploid yeast strain onto a Sc–Leu–His + 3-amino-1,2,4-triazole (3-AT) media.</p><p>&nbsp;</p><p><strong>Protocol 1B: Yeast transformation</strong></p><p>1)&nbsp;&nbsp;&nbsp;&nbsp;Streak Y8800 and Y8930 on separate YEPD plates and incubate at 30 °C for 48–72h to obtain isolated colonies.</p><p>2)&nbsp;&nbsp;&nbsp;&nbsp;For each strain, inoculate 20 ml of YEPD with 10 isolated colonies. Incubate at 30 °C on a shaker for 14–18 h.</p><p>3)&nbsp;&nbsp;&nbsp;&nbsp;Measure and record the OD600, which should be between 4.0 and 6.0. Dilute cells into 100 ml of YEPD media to obtain a final OD600 ~ 0.1.</p><p>4)&nbsp;&nbsp;&nbsp;&nbsp;Incubate at 30 °C on a shaker until OD600 reaches 0.6–0.8 (4–6 h).</p><p>5)&nbsp;&nbsp;&nbsp;&nbsp;Boil carrier DNA (salmon sperm DNA) for 5 min then place on ice until needed.</p><p>6)&nbsp;&nbsp;&nbsp;Harvest cells by centrifugation at 800 x g for 5 min. Discard the supernatant and resuspend cells gently in 10 ml of sterile water.</p><p>7)&nbsp;&nbsp;&nbsp;Centrifuge as described in step 6 and discard the supernatant.</p><p>8)&nbsp;&nbsp;&nbsp;Resuspend cells in 10 ml of TE/LiAc solution, centrifuge, and discard the supernatant.</p><p>9)&nbsp;&nbsp;&nbsp;Resuspend cells in 2 ml of TE/LiAc solution, then add 10 ml of TE/ LiAc/PEG solution supplemented with 200 µl of boiled carrier DNA. Mix the solution by inversion.</p><p>10) Dispense 120 µl of this mix into each well of a round-bottom 96-well microtiter plate</p><p>11) Add 10 µl of plasmid DNA to the competent yeast and mix by pipetting up and down several times.&nbsp;</p><p>12) Incubate at 30 °C for 30 min</p><p>13) Subject to heat shock in a 42 °C water bath for 15 min.</p><p>14) Centrifuge 5 min at 800 x g. Carefully remove the supernatant</p><p>15) Add 100 µl of sterile water and resuspend cell pellets by pipetting up and down.</p><p>16) Centrifuge 5 min at 800 x g, then carefully remove 90 µl of water</p><p>17) Resuspend cell pellets by vortexing.</p><p>18) Spot 5 µl of cell suspension onto an appropriate selective plate (Sc-Trp for AD-Y, Sc-Leu for DB-X agar media).</p><p>19) Incubate at 30 °C for 72 h.</p><p>20) Pick transformed yeast colonies into a 96-well low profile microplates containing 100 µl of selective media (Sc-Trp for AD-Y, Sc-Leu for DB-X).</p><p>21) Incubate on a shaker at 30 °C for 72 h.</p><p>22) Remove 5 µl for subsequent lysis and analysis by PCR</p><p>23) Prepare archival stocks by combining 100 µl of the yeast culture with 100 µl of 40% (w/v) autoclaved glycerol into a 96-well Low profile microplate. Store at -80 °C.</p><p>&nbsp;</p><p><strong>Protocol 2B: Yeast cell lysis and lysate PCR</strong></p><p>1)&nbsp;&nbsp;&nbsp;Prepare lysis buffer by dissolving 2.5 mg/ml zymolase 20T (21,100 U/g, Seikagaku Corp.) in 0.1M sodiumphosphate buffer (pH7.4). Keep on ice.</p><p>2)&nbsp;&nbsp;&nbsp;Aliquot 15 µl of lysis buffer into the wells of a 96-well PCR plate. Keep on ice.</p><p>3)&nbsp;&nbsp;&nbsp;Add 5 µl of yeast cells in the lysis buffer in the PCR plate</p><p>4)&nbsp;&nbsp;&nbsp;Place the PCR plate on a thermocycler and run the following program:</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Step 1: 37 °C for 15 min</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Step 2: 95 °C for 5 min</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Step 3: Hold at 10 °C</p><p>5)&nbsp;&nbsp;&nbsp;Add 100 ml of filter-sterilized water to each well.</p><p>6)&nbsp;&nbsp;&nbsp;Store at -20 °C</p><p><strong>&nbsp;</strong></p><p><strong>Protocol 3B: Yeast lysate PCR</strong></p><p>For each reaction, prepare in a 96-well PCR plate on ice:</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1.1 µl of HiFi Platinum Taq polymerase buffer 10x (Invitrogen)</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.4 µl of 50 mM MgSO4 (final concentration 1.8 mM)</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.11 µl&nbsp;of 40 mM dNTPs (final concentration 400 nM)</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.022 µl of 200 µM AD or DB forward primer (final concentration180 nM)</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.022 µl of 200 µM Term reverse primer (final concentration 180 nM)</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0.066 µl of HiFi Platinum Taq polymerase (Invitrogen)</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;8.28 µl of filter-sterilized water</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add 3 µl of the yeast cell lysate as DNA template</p><p>&nbsp;</p><p>Place the PCR plate on a thermocycler and run the same program of Protocol 3A. Once PCR reactions are completed, analyze 5 µl of PCR product on a 1% agarose.&nbsp;&nbsp;</p><p><strong>&nbsp;</strong></p><p><strong>&nbsp;</strong></p><p><strong>Protocol 4B: Autoactivator removal</strong></p><p>1)&nbsp;&nbsp;&nbsp;Thaw glycerol stocks of the 50 individual DB-bait yeast strains and the glycerol stock of the yeast strains transformed with the AD encoding plasmid containing no insert (empty pDESTAD).</p><p>2)&nbsp;&nbsp;&nbsp;Using the colony picker Qpix2 XT replica-plate the 96-well Low profile microplate with the individual glycerol stocks of each DB-X yeast strain and the yeast strains transformed with the empty-AD   into a new plates filled with 100 µl of fresh Sc-Leu and Sc-Trp media respectively. </p><p>3)&nbsp;&nbsp;&nbsp;Incubate at 30 °C for 72 h on a shaker</p><p>4)&nbsp;&nbsp;&nbsp;Replica-plate from the Sc-Leu-media and Sc-Trp media plates onto mating plates filled with YEPD media</p><p>5)&nbsp;&nbsp;&nbsp;Incubate at 30 °C for 24 h. </p><p>6) Replica-plate from mating plates onto Sc-Leu-Trp media plates to select for diploid cells and onto Sc-Leu-Trp-His + 1mM 3AT media. </p><p>7) Incubate at 30°C for 72 h. Score growth phenotypes. Non-autoactivating yeast cells are not able to activate the GAL1::HIS3 reporter gene hence should not grow on the Sc-Leu-Trp-His + 1mM 3AT media.</p><p>&nbsp;</p><p>Yeast strains showing a growth phenotype on the Sc-Leu-Trp-His + 1mM 3AT media are considered autoactivators. They are physically removed from the collection of DB-X baits and screened against the 12k_space DB-collection using their AD-construction.</p><p>&nbsp;</p><p>&nbsp;</p><p><strong>C)&nbsp;&nbsp;Primary Screening</strong></p><p><strong>&nbsp;</strong></p><p><strong>Protocol 1C: Construction of DB baits pool </strong></p><p>This protocol describes the construction of one pool of 50 different DB-hybrid constructs transformed into Y8930 yeast strain.</p><p>&nbsp;</p><p>1)&nbsp;&nbsp;&nbsp;Thaw glycerol stocks of the 50 individual DB-bait yeast strains</p><p>2)&nbsp;&nbsp;&nbsp;Individually inoculate 50-ml polypropylene conical tubes that contain 5 ml of fresh Sc-Leu selective media, with 5 µl of the thawed glycerol stock.</p><p>3)&nbsp;&nbsp;&nbsp;Incubate at 30 °C for 72 h.</p><p>4)&nbsp;&nbsp;&nbsp;Transfer the contents of the culture tubes into a sterile trough.</p><p>5)&nbsp;&nbsp;&nbsp;Mix thoroughly to ensure equal representation of all DB-bait yeast strains in the pool.</p><p>6)&nbsp;&nbsp;&nbsp;On a liquid handling platform or with a multichannel pipette plated into the wells of ten 384-well Low profile microplates</p><p>&nbsp;</p><p>&nbsp;</p><p><strong>Protocol 2C: 12k_space AD-collection culture </strong></p><p>&nbsp;</p><p>1)&nbsp;&nbsp;&nbsp;Thaw glycerol stocks of the 12k_space AD-collection corresponding to 127 96-well plates.</p><p>2)&nbsp;&nbsp;&nbsp;Using the colony picker Qpix2 XT replica-plate into 32&nbsp;384-well plates filled with 50 µl of fresh Sc-Trp media</p><p>3)&nbsp;&nbsp;&nbsp;Incubate at 30 °C for 72 h</p><p>&nbsp;</p><p>&nbsp;</p><p><strong>Protocol 3C: Mating</strong></p><p>1)&nbsp;&nbsp;&nbsp;Using the colony picker Qpix2 XT replica-plate the 32&nbsp;384-well plates corresponding to the 12k_space AD-collection 72 h cultures into the 384-well mating plates filled with YEPD media</p><p>2)&nbsp;&nbsp;&nbsp;Using the colony picker Qpix2 XT replica-plate the DB-baits pool into the 384-well mating plates precedently inoculated with the AD-collection. Use one DB-baits pool plate to inoculate five mating plates.</p><p>3)&nbsp;&nbsp;&nbsp;Incubate at 30 °C for 24 h</p><p>&nbsp;</p><p>&nbsp;</p><p><strong>Protocol 4C: Screening</strong></p><p>1)&nbsp;&nbsp;&nbsp;Using the colony picker Qpix2 XT replica-plate onto 384-well screening plates filled with 50 µl of fresh Sc-Leu-Trp-His + 1 mM 3AT media. Only diploid yeast with interacting couples can growth in this media.</p><p>2)&nbsp;&nbsp;&nbsp;Incubate at 30 °C for 5 days</p><p>3)&nbsp;&nbsp;&nbsp;Using the microplate-reader Tecan Infinite M200 PRO measure the OD600 of the 384-well screening plates in order to identify primary positives</p><p>&nbsp;</p><p><strong>&nbsp;</strong></p><p><strong>Protocol 5C: AD-interacting proteins re-array and archival stock </strong></p><p>1)&nbsp;&nbsp;&nbsp;Thaw glycerol stocks of the 12k_space AD-collection plates corresponding to the identified primary positives AD-interacting proteins&nbsp;</p><p>2)&nbsp;&nbsp;&nbsp;Using the colony picker Qpix2 XT re-array the primary positives AD-interacting proteins into 96-well plates filled with 100 µl of fresh Sc-Trp media.</p><p>3)&nbsp;&nbsp;&nbsp;Incubate at 30 °C for 72 h</p><p>4)&nbsp;&nbsp;&nbsp;Prepare an archival glycerol stock by adding 100 µl of 40% (w/v) autoclaved glycerol</p><p><strong>&nbsp;</strong></p><p><strong>&nbsp;</strong></p><p><strong>D)&nbsp;&nbsp;Deconvolution and targeted matricial assay </strong></p><p><strong>&nbsp;</strong></p><p><strong>Protocol 1D: DB-baits and AD-interacting proteins cultures</strong></p><p>1)&nbsp;&nbsp;&nbsp;Thaw glycerol stocks of the 50 individual DB-bait yeast strains and of the identified primary positives AD-interacting proteins&nbsp;&nbsp;</p><p>2)&nbsp;&nbsp;&nbsp;Individually inoculate 50 distinct 50-ml polypropylene conical tubes that contain 10 ml of fresh Sc-Leu selective media, with 5 µl of the 50 individual DB-baits</p><p>3)&nbsp;&nbsp;&nbsp;Using the colony picker Qpix2 XT replica-plate the glycerol stocks of the identified primary positives AD-interacting proteins into ten 96-well plates filled&nbsp;with 100 µl of fresh Sc-Trp selective media</p><p>4)&nbsp;&nbsp;&nbsp;Incubate at 30 °C for 72 h.</p><p>5)&nbsp;&nbsp;&nbsp;On a liquid handling platform or with a multichannel pipette plate the 50 individual DB-bait cultures into the wells of 50 distinct 96-well Low profile microplates</p><p>&nbsp;</p><p>&nbsp;</p><p><strong>Protocol 2D: Mating</strong></p><p>1)&nbsp;&nbsp;&nbsp;Using the colony picker Qpix2 XT replica-plate the identified primary positives AD-interacting proteins cultures into the 96-well mating plates filled with 100 µl of YEPD media. Use one AD-interacting proteins plate to inoculate five mating plates.</p><p>2)&nbsp;&nbsp;&nbsp;Using the colony picker Qpix2 XT replica-plate the individual DB-baits cultures into the 96-well mating plates precedently inoculated with the AD-interacting proteins.</p><p>3)&nbsp;&nbsp;&nbsp;Incubate at 30 °C for 24 h</p><p>&nbsp;</p><p>&nbsp;</p><p><strong>Protocol 3D: Verification and interacting proteins identification</strong></p><p>1)&nbsp;&nbsp;&nbsp;Using the colony picker Qpix2 XT replica-plate onto two phenotyping 96-well plates:</p><p>- Verification-plates filled with 100 µl of fresh Sc-Leu-Trp-His + 1 mM 3AT media. Only diploid yeast with interacting couples can growth in this media.</p><p>- <em>de novo</em> autoactivators-plates filled with 100 µl of fresh Sc-Leu-His + 1 mM 3AT + CHX (1 mg/l) media. Only truncated fragments that can act like transcription factors can growth in this media</p><p>2)&nbsp;&nbsp;&nbsp;Incubate at 30 °C for 5 days</p><p>3)&nbsp;&nbsp;&nbsp;Using the microplate-reader Tecan Infinite M200 PRO measure the OD600 of the 96-well plates in order to identify positives pairs and <em>de novo</em> autoactivators&nbsp;</p><p>4)&nbsp;&nbsp;&nbsp;Using the colony picker Qpix2 XT re-array the identified positives pairs into 96-well plates filled with 100 µl of fresh Sc-Leu-Trp-His + 1 mM 3AT</p><p>5)&nbsp;&nbsp;&nbsp;Incubate at 30 °C for 72 h</p><p>6)&nbsp;&nbsp;&nbsp;Lyse cells according to Protocol 2B</p><p>7)&nbsp;&nbsp;&nbsp;Amplify the inserts of the DB-X and AD-Y inserts of positive cultures by yeast colony PCR according to Protocol 3B for subsequent ORF identification by end-read sequencing.</p><p>8)&nbsp;&nbsp;&nbsp;Prepare an archival glycerol stock by adding 100 µl of 40% (w/v) autoclaved glycerol</p><h2>Troubleshooting</h2><p>- Y2H auto-activator removal <strong>(Protocol 4B)</strong>: All techniques used to map protein interactions can give rise to artifacts. Ensure all the autoactivators are physically removed before the primary screen. </p><p>- Minimizing false negatives: Run replicates of the experiment to increase the number of detected interaction pairs. </p><h2>Time Taken</h2><p>The time required to entirely execute this protocol is approximately two months.</p><h2>Anticipated Results</h2><p>Identification of protein interaction partners yields crucial biological insights and assign functions to both uncharacterized and well-studied gene products.</p><h2>References</h2><p>1 Braun et al., An experimentally derived confidence score for binary protein–protein interactions. <em>Nat. Methods</em> 6, 91–97. (2009) </p><p>2 Cusick et al., Literature-curated protein interaction datasets. <em>Nat. Methods</em> 6, 39–46. (2009)</p><p>3 Simonis et al., Empirically controlled mapping of the Caenorhabditis elegans protein–protein interactome network. <em>Nat. Methods</em> 6, 47–54. (2009)</p><p>4 Venkatesan et al., An empirical framework for binary interactome mapping. <em>Nat. Methods</em> 6, 83–90. (2009)</p><p>5 Arabidopsis Interactome Mapping Consortium. Evidence for Network Evolution in an Arabidopsis Interactome Map. <em>Science</em> 333(6042):601–606. July (2011) </p><p>6 Ralf Weßling et al.,<strong> </strong>Convergent targeting of a common host protein-network by pathogen effectors from three kingdoms of life. <em>Cell Host Microbe</em>. September 10; 16(3): 364–375. 2014</p><p>7 Dreze et al., High-Quality Binary Interactome Mapping. Methods in Enzymology. Chapter twelve. (2010<em>﻿)</em></p></body></html>