<html><head><title>A protocol for Targeted Perturb (TAP)-seq and targeted single-cell RNA-seq</title></head><body><h1>A protocol for Targeted Perturb (TAP)-seq and targeted single-cell RNA-seq</h1>Authors: Daniel Schraivogel, Lars Velten, Andreas R. Gschwind, Lars M. Steinmetz<br>Abstract: <p>Here we provide a step-by-step protocol for targeted single-cell RNA-seq and targeted Perturb-seq, as reported in the linked publication&nbsp;Schraivogel et al. Nat Meth 2020. The protocol describes cell preparation using flow cytometry, single-cell droplet formation with 10X Genomics, and targeted 3’ single-cell RNA-seq library preparation for Illumina sequencing.</p><br>Keywords: single-cell, CRISPR, Cas9, CRISPRi, functional genomics, genetic screens, RNA sequencing<br><h2>Introduction</h2><p>Single-cell RNA-sequencing (scRNA-seq) is revolutionizing basic and applied biomedical research. Among diverse applications, scRNA-seq readouts of pooled CRISPR genetic screens, also referred to as CROP-seq or Perturb-seq, enabled to identify the consequences of defined genetic perturbations on gene expression and cellular differentiation (Adamson et al. 2016, Dixit et al. 2016, Jaitin et al. 2016, Datliner et al. 2017). Although this concept has captured widespread attention, its broad applicability has been prohibited due to (1) high sequencing costs, (2) bottlenecks in data analysis, and (3) a lack of sensitivity of scRNA-seq methods.</p><p>Here we overcome these limitations by introducing a protocol for targeted Perturb-seq (TAP-seq) (Figure 1). The method has been established primarily as a readout for pooled CRISPR screens. However, TAP-seq is applicable to diverse single-cell applications in mammalian cell lines and primary mammalian cells, with or without CRISPR perturbations.</p><h2>Reagents</h2><p>10X Genomics Single-cell 3’ RNA-seq kit v2</p><p>80 % ethanol, freshly prepared</p><p>Agilent High Sensitivity DNA Kit</p><p>Cell collection buffer (PBS + 2 % BSA)</p><p>Elution buffer (10 mM Tris-HCl pH 8.5)</p><p>FACS buffer (PBS + 2 % FBS + 2 mM EDTA)</p><p>KAPA Biosystems HiFi HotStart ReadyMix</p><p>Live-dead stain for flow cytometry, e.g. Biostatus DRAQ7 Drop &amp; Go</p><p>Low TE buffer (10 mM Tris-HCl pH 8.0 + 0.1 mM EDTA)</p><p>Nuclease free ddH2O non DEPC-treated</p><p>PBS without Mg without Ca</p><p>Qubit dsDNA HS Assay Kit (Thermo Fisher)</p><p>SPRI beads, e.g. Beckman Coulter AMPureXP&nbsp;</p><p>Thermo Fisher Superscript IV reverse transcriptase 200 U/µl (Cat. No. 18090050)</p><p>Trypan-blue viability stain</p><p><br></p><p>Oligonucleotides:</p><p>PartialRead1 <em>5'-CTACACGACGCTCTTCCGATCT</em></p><p>ISPCR <em>5'-AAGCAGTGGTATCAACGCAGAGT</em></p><p>Outer primer <em>5'-[gene specific]</em></p><p>Inner primer <em>5'-GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAG[gene specific]</em></p><p>CROPouter <em>5'-AATGGACTATCATATGCTTACCGT</em></p><p>CROPinner <em>5'-GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAGTGTGGAAAGGACGAAACACC</em></p><p>Targeted10X <em>5'-AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTC*C*G</em></p><p>New-P5 smart PCR hybrid oligo <em>5'-AATGATACGGCGACCACCGAGATCTACACGCCTGTCCGCGGAAGCAGTGGTATCAACGCAGAGT*A*C</em></p><p>Illumina reverse primer (N7XX) <em>5'-CAAGCAGAAGACGGCATACGAGAT[X]8GTCTCGTGGGCTCGG</em></p><h2>Equipment</h2><p>1.5 ml low retention tubes</p><p>10X Genomics Chromium</p><p>5 ml flow cytometry collection tubes</p><p>Agilent 2100 Bioanalyzer</p><p>Cell counter, e.g. Thermo Fisher Countess II FL Automated Cell Counter</p><p>Centrifuges</p><p>DNA quantification, e.g. Thermo Fisher Qubit Fluorometer</p><p>Emulsion safe PCR 8-tube strips, e.g. Eppendorf Cat. No. 0030 124.359</p><p>FACS tubes, e.g. Corning Falcon Test Tube 5 ml with 35 µm Cell Strainer Snap Cap</p><p>Flow Cytometer, e.g. BD FACSAria Fusion</p><p>Magnetic separator, e.g. 10X Magnetic Separator with High and Low position</p><p>PCR cycler</p><p><br></p><h2>Procedure</h2><p><u>Preparation of K562 cells for single-cell RNA sequencing (60 min)</u></p><p><em>Comment: We describe here a workflow for preparation of K562 cells for 10X Genomics. In general, cells for TAP-seq with 10X Genomics are prepared in the same way as cells for classical 10X Genomics.</em></p><p>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Count live cells using cell counter and trypan blue live dead stain</p><p>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Collect cells by centrifugation (200 g, 5 min, room temperature)</p><p>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Discard supernatant</p><p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Resuspend in FACS buffer to achieve a cell concentration of 5-10 million cells per ml</p><p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Filter through cell strainer</p><p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add viability dye, e.g. DRAQ7</p><p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Sort viable cells by flow cytometry into collection tube containing cell collection buffer (Figure 2).&nbsp;All cell sorting can be performed at room temperature. Collected cells are stored on ice until use.</p><p><em>Comment: Flow buffer and cell collection buffer are compatible with 10X Genomics and other single-cell platforms. Same is true for DRAQ7 and other viability stains (validated for DAPI and PI).</em></p><p><em>Optional: Cells expressing Cas9 as a fusion protein with a fluorescent tag (e.g. dCas9-BFP-KRAB) can be enriched for BFP-expression. We generally enriched for cells expressing intermediate to high Cas9 expression level (see&nbsp;Figure 2).</em></p><p>a.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;If high numbers of cells are available: Sort 0.1-1 million cells into 5 ml FACS tubes containing 300 µl cell collection buffer. Collect cells by centrifugation (300 g, 7 min, room temperature) and discard supernatant. Resuspend pellet in PBS by pipetting several times and dilute to achieve a cell concentration compatible with 10X Genomics, e.g. 1 million cells per ml. Double-check cell numbers and absence of multiplets by cell counting.</p><p>b.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;If low numbers of cells are available: Sort cells directly into 20 µl cell collection buffer provided in a 1.5 ml tube. Cell numbers are deduced from flow cytometry counts, and by adding a correction factor of ~ 1.5x to 2x (has to be adapted, start with 1.5x). Cells are not counted again.</p><p><em>Example: The correction factor considers cell loss during handling of cells after sorting, and the dilution factor during 10X Genomics single-cell Master Mix preparation and Chromium run. If 250 cells should be sequenced (and a correction factor 1.5x), sort 250 x 1.5 = 375 cells into 20 µl cell collection buffer. Approximately 3.5 nl (100 µm nozzle) or 7 nl (130 µm nozzle) liquid will be added on top per sorted cell. After sort is finished, vortex quickly to collect cells from tube wall, spin cells shortly, and measure volume. Then fill up with collection buffer to 33.8 µl, which is the volume added into the 10X Genomics single-cell master mix.</em></p><p><br></p><p><u>10X Genomics Chromium run (90 min)</u></p><p>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Prepare single cell master mix on ice according to the 10X Genomics Chromium single-cell 3’ RNA-seq v2 protocol. Mix by pipetting and place on ice. 10X Genomics RT Enzyme Mix can be replaced by Superscript IV RT or not. 10X RT primer (template switch oligo) is replaced by low TE buffer (or H2O).</p><p>RT Reagent Mix <strong>50 µl</strong></p><p>low TE buffer <strong>3.8 µl</strong></p><p>Additive A <strong>2.4 µl</strong></p><p>Superscript IV RT <strong>10 µl</strong></p><p><em>Comment: The protocol should be adaptable to Chromium single-cell RNAseq v3, and the more recent NextGEM protocol, since the PCR handle sequence on the 10X bead oligo is identical between all three 10X versions (target sequence of Partial Read 1 primer). Further, the steps until cDNA amplification are largely identical. We here follow the 10X v2 protocol until cDNA amplification.&nbsp;</em></p><p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Fill unused wells of a single cell A Chip with 50 % Glycerol, as described in the 10X Genomics Chromium single-cell 3’ RNA-seq v2 protocol.</p><p>10.&nbsp;&nbsp;Add H2O and cells to single cell master mix (see 10X Genomics Chromium single-cell 3’ RNA-seq v2 protocol cell dilution table), mix by pipetting, and transfer 90 µl of cell suspension to 10X Chip A well “1”.</p><p>11.&nbsp;&nbsp;Prepare 10X Genomics beads by vortexing for 30 sec, and slowly dispense 27 to 40 µl of the bead suspension into 10X Chip A well “2”. 40 µl loading volume are suggested by 10X Genomics, but volume can be reduced to 27 µl, since beads are not used up completely during the run.</p><p>12.&nbsp;&nbsp;Fill 10X Chip A well “3” with 230 µl partitioning oil, attach 10X gasket, and start run on the Chromium Controller.</p><p>13.&nbsp;&nbsp;After run, check volume in GEM collection outlet and transfer 110 µl of GEM emulsion&nbsp;into emulsion safe PCR 8-tube strip.&nbsp;</p><p><em>Comment: 10X protocol transfers 100 µl of the GEM emulsion only, but we generally observe a higher volume of GEMs in the collection outlet, allowing a 10% increase in cell recovery.</em></p><p>14.&nbsp;&nbsp;GEM-RT incubation in a thermal cycler [45 min 53&nbsp;°C – 5 min 85&nbsp;°C – hold 4&nbsp;°C]. Tubes can&nbsp;be stored at 4&nbsp;°C for up to 72 h or at -20&nbsp;°C for up to a week.</p><p>15.&nbsp;&nbsp;Purify GEM-RT as described in 10X Genomics Chromium single-cell 3’ RNA-seq v2 protocol.</p><p><br></p><p><u>TAP-seq protocol (4h)</u></p><p>16.&nbsp;&nbsp;Prepare PCR1 on ice, mix by vortexing and collect by short centrifugation.&nbsp;</p><p><em>Comment: We generally use 2.5 µl of outer primer mix for a panel size &lt; 500 different primers, but amount can be increased to 4 µl per reaction in case more primers are present in panel.</em></p><p><em>Comment: For 10X v2, v3 and NextGEM, no adaption for PCR1 and PCR2 is necessary, since the Partial Read 1 primer can be used for all three protocols.</em></p><p><em>Comment: The CROP-seq vector primer (CROPouter, CROPinner) can be omitted in case no CROP-seq vector is present, or replaced if sgRNA/barcode is expressed by a vector system other than CROP-seq.</em></p><p><em>Optional: 5 µl of the 10X Genomics cDNA from step 15 can be put on side for separate CROP-seq vector amplification. Follow the protocol as described from here on, but omitting the outer and inner primer mix.</em></p><p>GEM-RT cleanup <strong>35 µl</strong></p><p>10 µM Partial Read 1 <strong>4 µl</strong></p><p>100 µM outer primer mix <strong>2.5 µl</strong></p><p>12 µM CROPouter <strong>10 µl</strong></p><p>KAPA HiFi <strong>50 µl</strong></p><p>H2O <strong>ad. 100 µl</strong></p><p>17.&nbsp;&nbsp;Run PCR1 with the following cycling protocol: 95&nbsp;°C 3 min - [98&nbsp;°C 20 sec, 67&nbsp;°C 60 sec, 72&nbsp;°C 60 sec] x12 - 72&nbsp;°C 5 min - 4&nbsp;°C hold</p><p><em>Comment: Exact cycle number has to be determined, but we achieved good results with 11 to 14 cycles. The cycle number depends on the number of cells, the amount of input cDNA, the number of amplified genes, and the expression levels of these genes. As a start, we suggest 12 cycles, which should result in&nbsp;³&nbsp;10 ng of PCR product. Too high cycle numbers of PCR1-3 might result in a high MW product after PCR3 peaking around 1,000 to 1,500 bp; we generally do not observe any issues from this high MW product after Illumina sequencing and data analysis (product might not cluster efficiently during Illumina run preparation).</em></p><p>18.&nbsp;&nbsp;After PCR is finished, collect liquid by short centrifugation and place at room temperature. Add 150 µl AmpureXP reagent (1.5x) to sample in tube strip and pipette mix 10 times. Incubate 5 min at room temperature with lid open.</p><p>19.&nbsp;&nbsp;Place tube in 10X magnetic separator in the High position until solution is clear (approx. 1-2 min). Remove and discard supernatant.</p><p>20.&nbsp;&nbsp;Wash once with 300 µl 80 % EtOH without releasing tubes from magnet while incubating for 30 sec with EtOH.</p><p>21.&nbsp;&nbsp;Wash twice with 200 µl 80 % EtOH. After last wash, discard supernatant, spin down, and place back in magnet at low position. Discard remaining liquid with 10 µl pipet and air-dry pellet for 2 min with lid open.</p><p>22.&nbsp;&nbsp;Elute by adding 30 µl elution buffer to tube released from magnet, and mix either by 10 sec vortexing or pipetting for 15 times. Spin tube shortly to collect bead suspension from tube walls, and incubate 2 min at room temperature.</p><p>23.&nbsp;&nbsp;Place tube back in magnet in low position and wait until solution is clear. Transfer supernatant into a new 8-strip tube and store on ice. Measure concentration from 1 µl eluate using Qubit dsDNA HS.</p><p><em>Comment: Typical yields are in the ng range from 10 to 250 ng.</em></p><p>24.&nbsp;&nbsp;Samples can be stored at 4&nbsp;°C for 12 h or o/n before proceeding to PCR2.</p><p>25.&nbsp;&nbsp;Prepare PCR2 on ice, mix by vortexing and collect by short centrifugation.&nbsp;</p><p><em>Comment: Use same amount of inner primer mix as above for PCR1 outer primer mix. The input amount of 10 ng can be used as a starting point, but can be lowered down to 5 ng. Lower input amounts might also work, but were not tested for library representation yet.</em></p><p>PCR1 cleanup <strong>10 ng</strong></p><p>10 µM Partial Read 1 <strong>4 µl</strong></p><p>100 µM inner primer mix <strong>2.5 µl</strong></p><p>12 µM CROPinner <strong>10 µl</strong></p><p>KAPA HiFi <strong>50 µl</strong></p><p>H2O <strong>ad. 100 µl</strong></p><p>26.&nbsp;&nbsp;Run PCR2 with the following cycling protocol: 95&nbsp;°C 3 min - [98&nbsp;°C 20 sec, 67&nbsp;°C 60 sec, 72&nbsp;°C 60 sec] x8 - 72&nbsp;°C</p><p>5 min - 4&nbsp;°C hold</p><p>27.&nbsp;&nbsp;After PCR is done, collect liquid by short centrifugation and place at room temperature. Add 150 µl AmpureXP reagent (1.5x) to sample in tube strip and pipette mix 10 times. Incubate 5 min at room temperature with lid open.</p><p>28.&nbsp;&nbsp;Place tube in 10X magnetic separator in the High position until solution is clear (approx. 1-2 min). Remove and discard supernatant.</p><p>29.&nbsp;&nbsp;Wash once with 300 µl 80 % EtOH without releasing tubes from magnet while incubating for 30 sec with EtOH.</p><p>30.&nbsp;&nbsp;Wash twice with 200 µl 80 % EtOH. After last wash, discard supernatant, spin down, and place back in magnet at low position. Discard remaining liquid with 10 µl pipet and air-dry pellet for 2 min with lid open.</p><p>31.&nbsp;&nbsp;Elute by adding 30 µl elution buffer to tube released from magnet, and mix either by 10 sec vortexing or pipetting for 15 times. Spin tube shortly to collect bead suspension from tube walls, and incubate 2 min at room temperature.</p><p>32.&nbsp;&nbsp;Place tube back in magnet in low position and wait until solution is clear. Transfer supernatant into a new 8-strip tube and store on ice. Measure concentration from 1 µl eluate using Qubit dsDNA HS and proceed to PCR3.</p><p><em>Comment: Typical yields are in the ng range from 10 to 100 ng.</em></p><p>33.&nbsp;&nbsp;Prepare PCR3 on ice, mix by vortexing and collect by short centrifugation.&nbsp;</p><p><em>Comment:&nbsp;PCR3 adds Illumina adapters to generate a sequencing-ready library.</em></p><p>PCR2 cleanup <strong>10 ng</strong></p><p>10 µM Targeted10X <strong>4 µl</strong></p><p>10 µM N7XX <strong>2.5 µl</strong></p><p>KAPA HiFi <strong>50 µl</strong></p><p>H2O <strong>ad. 100 µl</strong></p><p>34.&nbsp;&nbsp;Run PCR3 with the following cycling protocol: 95&nbsp;°C 3 min - [98&nbsp;°C 20 sec, 60&nbsp;°C 15 sec, 72&nbsp;°C 45 sec] x8 - 72&nbsp;°C 5 min - 4&nbsp;°C hold</p><p>35.&nbsp;&nbsp;After PCR is done, collect liquid by short centrifugation and place at room temperature. Add 150 µl AmpureXP reagent (1.5x) to sample in tube strip and pipette mix 10 times. Incubate 5 min at room temperature with lid open.</p><p>36.&nbsp;&nbsp;Place tube in 10X magnetic separator in the High position until solution is clear (approx. 1-2 min). Remove and discard supernatant.</p><p>37.&nbsp;&nbsp;Wash once with 300 µl 80 % EtOH without releasing tubes from magnet while incubating for 30 sec with EtOH.</p><p>38.&nbsp;&nbsp;Wash twice with 200 µl 80 % EtOH. After last wash, discard supernatant, spin down, and place back in magnet at low position. Discard remaining liquid with 10 µl pipet and air-dry pellet for 2 min with lid open.</p><p>39.&nbsp;&nbsp;Elute by adding 30 µl elution buffer to tube released from magnet, and mix either by 10 sec vortexing or pipetting for 15 times. Spin tube shortly to collect bead suspension from tube walls, and incubate 2 min at room temperature.</p><p>40.&nbsp;&nbsp;Place tube back in magnet in low position and wait until solution is clear. Transfer supernatant into a new 8-strip tube and store on ice. Measure concentration from 1 µl eluate using Qubit dsDNA HS and perform QC using Agilent Bioanalyzer with a 1 ng eluate.</p><p><em>Comment: Typical yields from PCR3 in the 100 ng range.</em></p><p>41.&nbsp;&nbsp;The protocol&nbsp;generates libraries identical to 10X Genomics single-cell 3’ libraries using standard Illumina Read 1 and Read 2 primers. The cell barcode and UMI is encoded in Read 1 (26 cycles), Read 2 contains the cDNA fragment (~58 cycles). Sample index sequences are sequenced as i7 index read (8 cycles).&nbsp;</p><p>42. A snakemake workflow template for data processing for TAP-seq experiments can be found under: <a href="https://github.com/argschwind/TAPseq_workflow" rel="noopener noreferrer" target="_blank">https://github.com/argschwind/TAPseq_workflow</a></p><h2>Time Taken</h2><p>Preparation of cells for single-cell RNA sequencing: <strong>60 min</strong></p><p>10X Genomics Chromium run: <strong>30 min</strong></p><p>GEM transfer and GEM-RT incubation: <strong>60 min</strong></p><p>post GEM-RT purification:&nbsp;<strong>60 min</strong></p><p>PCR1 setup and run: <strong>60 min</strong></p><p>SPRI purification and product quantification: <strong>20 min</strong></p><p>PCR2 setup and run: <strong>50 min</strong></p><p>SPRI purification and product quantification: <strong>20 min</strong></p><p>PCR3 setup and run: <strong>40 min</strong></p><p>SPRI purification and product quantification: <strong>20 min</strong></p><p>DNA HS Bioanalyzer run: <strong>40 min</strong></p><p>Total (all steps): <strong>8 h</strong></p><p>Total (TAP-seq): <strong>4 h</strong></p><h2>Anticipated Results</h2><p>﻿Typical Bioanalyzer traces for TAP-seq are shown in Figure 3.</p><h2>References</h2><p>Adamson, B.&nbsp;<em>et al.</em>&nbsp;A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.&nbsp;<em>Cell</em>&nbsp;167, 1867-1882.e21 (2016).</p><p>Dixit, A.&nbsp;<em>et al.</em>&nbsp;Perturb-Seq: Dissecting Molecular Circuits with Scalable Single-Cell RNA Profiling of Pooled Genetic Screens.&nbsp;<em>Cell</em>&nbsp;<strong>167</strong>, 1853-1866.e17 (2016).</p><p>Jaitin, D. A.&nbsp;<em>et al.</em>&nbsp;Dissecting Immune Circuits by Linking CRISPR-Pooled Screens with Single-Cell RNA-Seq.&nbsp;<em>Cell</em>&nbsp;<strong>167</strong>, 1883-1896.e15 (2016).</p><p>Datlinger, P.&nbsp;<em>et al.</em>&nbsp;Pooled CRISPR screening with single-cell transcriptome readout.&nbsp;<em>Nat. Methods</em>&nbsp;<strong>14</strong>, 297–301 (2017).</p><h2>Acknowledgements</h2><p>This work was supported by grants from the European Research Council Advanced Investigator Grant (AdG-294542 and AdG-742804 to L.M.S.). D.S. was supported by a fellowship from the EMBL Interdisciplinary Postdoc (EIPOD) program under Marie Sklodowska-Curie Actions COFUND (grant agreement number 664726). A.R.G. was supported by an Early Postdoc.Mobility fellowship (project number P2LAP3_171806) from the Swiss National Science Foundation (SNSF).&nbsp;</p><p><br></p></body></html>