<html><head><title>SAFE Hi-C: A Simplified, Amplification-free and Economically Efficient High-throughput Chromosome Conformation Capturing Method</title></head><body><h1>SAFE Hi-C: A Simplified, Amplification-free and Economically Efficient High-throughput Chromosome Conformation Capturing Method</h1>Authors: Longjian Niu, Yingzhang Huang, Chunhui Hou<br>Abstract: <p>PCR amplification of Hi-C libraries introduces unusable duplicates and results in a biased representation of chromatin interactions. We present a simplified, fast, and economically efficient Hi-C library preparation procedure that generates sufficient non-amplified ligation products for deep sequencing. Comprehensive analysis of the resulting data indicates that amplification-free Hi-C preserves higher complexity of chromatin interaction and lowers sequencing depth dramatically for the same number of unique paired reads. With amplification bias avoided, our method may produce a chromatin interaction network more faithfully reflecting the real three-dimensional genomic architecture.</p><br>Keywords: Hi-C, genomic architecture, amplification-free, chromatin interaction, topologically associating domain, compartment<br><h2>Introduction</h2><p>Hi-C compares the number of ligation events between each fragment pair in a large population of cells and thereby allows the identification of various genome structural features including compartments, topologically associating domains (TADs), and loops<sup>1-5</sup>. PCR amplification is a default step in Hi-C-related experiments<sup>6</sup> to guarantee sequencing primer addition and to produce a sufficient amount of DNA for sequencing, especially for Hi-C experiments involving single or low cell numbers<sup>7-18</sup>. Though universal primers are used, PCR amplification introduces duplicates and may skew Hi-C library composition, which may not be fully corrected by normalization methods<sup>19-21</sup>. </p><p><br></p><p>Efficient recovery of enough ligated fragments for direct high-throughput sequencing is important for accurate depicting the three dimensional genome architecture and the understanding of its functional role in transcription regulation, replication, genome stability and other critical biological activities related to chromatin.&nbsp;We present SAFE Hi-C, a <em>s</em>implified, <em>a</em>mplification-<em>f</em>ree, and economically <em>e</em>fficient process, in which paired reads generated by independent ligation events are saved. </p><h2>Reagents</h2><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Schneider’s Medium (Gibco, 21720024)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;FBS (Sigma, F7524)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Penicillin/Streptomycin P/S (Sigma, P0781)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;formaldehyde solution (Sigma, F8775)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Glycine (sigma, G8898)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;protease inhibitors (Sigma, P8340)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;IGEPAL® CA-630 (Sigma, I8896)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Triton X-100 (Sigma, 93443)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Sodium dodecyl sulfate (SDS) (Sigma, L3771)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;DpnII restriction enzyme (NEB, R0147)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;biotin–dATP (Thermo Fisher, 19524016)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;dCTP, dGTP, and dTTP (Thermo Fisher, R0151, 10219012, R0161)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;DNA Polymerase I Large (Klenow) Fragment (NEB, M0210)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;T4 DNA ligase buffer with 10 mM ATP (NEB, B0202)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;BSA (Thermo Fisher, AM2616)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;T4 DNA Ligase (NEB, M0202)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Proteinase K (TRANSGEN, GE201)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;RNase A (TRANSGEN, GE101)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Streptavidin Dynabeads (NEB, S1420S)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;T4 PNK (NEB, M0201)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;T4 DNA polymerase I (NEB, M0203)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Klenow exo minus (NEB, M0212)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1X Quick ligation reaction buffer (NEB, B6058)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Full Y-Adaptor (Vazyme, N802)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;NEB DNA Quick ligase (NEB, M2200)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Lysis Buffer (10 mM Tris-HCl pH8.0, 10 mM NaCl, 0.2% Igepal CA630, 1X protease inhibitors)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1X Tween Wash Buffer (1X TWB, 5 mM Tris-HCl pH 7.5, 0.5 mM EDTA, 1 M NaCl, 0.05% Tween-20)</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2X Binding Buffer (2X BB, 10 mM Tris-HCl pH 7.5, 1 mM EDTA, 2M NaCl)</p><p><br></p><h2>Equipment</h2><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Covaris LE220</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;DynaMag™-2 Magnet (Thermo Fisher, 12321D)</p><p><br></p><h2>Procedure</h2><p><strong>Cell Culture</strong></p><p><br></p><p>S2 cells were cultured in Schneider’s Medium (Gibco, 21720024) supplemented with 10% heat inactivated FBS (Sigma, F7524) and 1% Penicillin/Streptomycin P/S (Sigma, P0781) at 27°C.</p><p><br></p><p>K562 cells were incubated in 1× RPMI1640 media supplemented with 10% FBS at 37°C with 5% CO2.</p><p><br></p><p><strong>Cell Crosslinking</strong></p><p><br></p><p>1.&nbsp;&nbsp;&nbsp;30 million Drosophila S2 cells or 250 thousand human K562 cells are collected by centrifugation at 300xG for 5 minutes. </p><p>2.&nbsp;&nbsp;&nbsp;Resuspend cells in fresh medium. Add freshly made formaldehyde solution (sigma, F8775) to a final concentration of 1%, v/v. Incubate at room temperature for 10 minutes with mixing.</p><p>3.&nbsp;&nbsp;&nbsp;Add 1/10 volume of 2.5 M glycine to quench the reaction. Incubate at room temperature for 5 minutes.</p><p>4.&nbsp;&nbsp;&nbsp;Centrifuge for 5 minutes at 300xG at 4°C and discard supernatant.</p><p>5.&nbsp;&nbsp;&nbsp;Resuspend cells in 1 mL of cold 1X PBS and spin for 5 minutes at 300xG at 4°C.</p><p>6.&nbsp;&nbsp;&nbsp;Discard supernatant. The cell pellets can store at -80°C or proceed to the rest of the protocol.</p><p><br></p><p><strong>Lysis and Restriction Digest</strong></p><p><br></p><p>1.&nbsp;&nbsp;&nbsp;Crosslinked cells were resuspended in 500 μL of ice-cold Hi-C lysis buffer (10 mM Tris-HCl pH8.0, 10 mM NaCl, 0.2% Igepal CA630, 1X protease inhibitors (Sigma, P8340)) and rotated at 4°C for 30 minutes. </p><p>2.&nbsp;&nbsp;&nbsp;Nuclei were pelleted at 4 °C for 5 minutes at 2,500xG, and the supernatant was discarded.</p><p>3.&nbsp;&nbsp;&nbsp;Pelleted nuclei were washed once with 500 μL of ice-cold Hi-C lysis buffer. </p><p>4.&nbsp;&nbsp;&nbsp;Remove supernatant and resuspend pellet with 100 μL of 0.5% SDS. Incubate at 62°C for 10 minutes. </p><p>5.&nbsp;&nbsp;&nbsp;Add 285 μL of water and 50 μL of 10% Triton X-100 (Sigma, 93443) to quench the SDS. Mix well, avoiding excessive foaming. Incubate at 37°C for 15 minutes.</p><p>6.&nbsp;&nbsp;&nbsp;Add 50 μL of NEB Buffer 3.1 and 20 μL of 10 U/μL DpnII restriction enzyme (NEB, R0147).</p><p>7.&nbsp;&nbsp;&nbsp;The sample was rotated at 37°C for 4 h. </p><p>8.&nbsp;&nbsp;&nbsp;DpnII was then heat inactivated at 65°C for 20 minutes with no shaking or rotation. </p><p><br></p><p><strong>Marking of DNA Ends, Proximity Ligation, and Crosslink Reversal</strong></p><p><br></p><p>1.&nbsp;&nbsp;&nbsp;To fill-in the restriction fragment overhangs and mark the DNA ends with biotin, 52 μL of incorporation master mix was then added: </p><p>37.5 μL of 0.4 mM biotin–dATP (Thermo Fisher, 19524016); </p><p>4.5 μL of dCTP, dGTP, and dTTP mix at 10 mM each (Thermo Fisher, R0151, 10219012, R0161); </p><p>and 10 μL of 5 U/μL DNA Polymerase I Large (Klenow) Fragment (NEB, M0210).</p><p>2.&nbsp;&nbsp;&nbsp;The reactions were then rotated at 37°C for 45 minutes. </p><p>3.&nbsp;&nbsp;&nbsp;Add 948 μL of ligation master mix: </p><p>150 μL of 10× NEB T4 DNA ligase buffer with 10 mM ATP (NEB, B0202), </p><p>125 μL of 10% Triton X-100, </p><p>3 μL of 50 mg/mL BSA (Thermo Fisher, AM2616), </p><p>10 μL of 400 U/μL T4 DNA Ligase (NEB, M0202), </p><p>and 660 μL of water.</p><p>4.&nbsp;&nbsp;&nbsp;The reactions were then rotated at 16 °C for 4 h and room temperature for 1 h. </p><p>5.&nbsp;&nbsp;&nbsp;45 μL of 10% SDS and 55 μL of 20 mg/mL Proteinase K (TRANSGEN, GE201) were added. Incubate at 55°C for at least 2 hours (overnight recommended). </p><p>6.&nbsp;&nbsp;&nbsp;Add 20 μL RNase A (TRANSGEN, GE101) and incubate at 37°C for 30 minutes.</p><p>7.&nbsp;&nbsp;&nbsp;DNA was purified by phenol:chloroform:isoamyl alcohol (25:24:1) extraction. Dissolve DNA pellet in 100μL of 1X Tris buffer (10 mM Tris-HCl, pH 8)</p><p><br></p><p><strong>DNA shearing </strong></p><p><br></p><p>1&nbsp;&nbsp;&nbsp;&nbsp;Shear DNA to a  size of 300-500 bp using the following parameters: </p><p>Instrument: Covaris LE220 (Covaris, Woburn, MA) </p><p>Volume of Library: 100μL in a Covaris microTUBE </p><p>Fill Level: 10 </p><p>Duty Cycle: 15 </p><p>PIP: 500</p><p> Cycles/Burst: 200 </p><p>Time: 58 seconds  </p><p><br></p><p><strong>Biotin Pull-Down and add Full Y-Adaptor </strong></p><p><br></p><p>1.&nbsp;&nbsp;&nbsp;Transfer 100 μL of Streptavidin Dynabeads (NEB, S1420S) into a 1.5 mL low-bind tube. Separate on a magnet and discard the solution.</p><p>2.&nbsp;&nbsp;&nbsp;Wash the beads with 200 μL 1X Tween Wash Buffer (1X TWB, 5 mM Tris-HCl pH 7.5, 0.5 mM EDTA, 1 M NaCl, 0.05% Tween-20). Separate on a magnet and discard the solution.</p><p>3.&nbsp;&nbsp;&nbsp;Resuspend the beads with 100 μL 2X Binding Buffer ( 2X BB, 10 mM Tris-HCl pH 7.5, 1 mM EDTA, 2M NaCl ). </p><p>4.&nbsp;&nbsp;&nbsp;Transfer 100 μL sonicated DNA to the beads resuspended by 2X BB</p><p>5.&nbsp;&nbsp;&nbsp;Incubate at room temperature for 15 minutes with rotation to bind biotinylated DNA  to the streptavidin beads.  </p><p>6.&nbsp;&nbsp;&nbsp;Separate on a magnet and discard the solution.  </p><p>7.&nbsp;&nbsp;&nbsp;Wash the beads by adding 200 μL of 1X TWB. Heat the tubes on a Thermomixer at 55°C for 2 minutes. Reclaim the beads using a magnet. Discard supernatant. </p><p>8.&nbsp;&nbsp;&nbsp;Resuspend beads with 200 μL 1X NEB T4 DNA ligase buffer (NEB, B0202) and transfer to a new 1.5 mL low-bind tube. Separate on magnet and remove the supernatant.  </p><p>9.&nbsp;&nbsp;&nbsp;Repairing ends of sheared DNA with 100 μL master mix: </p><p><br></p><p>10 μL 10X NEB T4 DNA ligase buffer with 10 mM ATP</p><p>2 μL of 25 mM dNTP mix</p><p>5 μL of 10 U/μL NEB T4 PNK (NEB, M0201)</p><p>4 μL of 3 U/μL NEB T4 DNA polymerase I (NEB, M0203)</p><p>1 μL of 5 U/μL NEB DNA polymerase I, Large (Klenow) Fragment (NEB, M0210)</p><p>78 μL water </p><p><br></p><p>10. Incubate at room temperature for 30 minutes.</p><p>11. &nbsp;Separate on magnet and remove the supernatant.</p><p>12. &nbsp;Add 200 μL 1X TWB to resuspend the beads and incubate on 55°C for 2 minutes. Separate on magnet and remove the supernatant.</p><p>13. &nbsp;Add 200 μL 1X NEBuffer 2 to resuspend the beads and transfer to a new 1.5 mL low-bind tube. Separate on magnet and remove the supernatant.</p><p>14. Resuspend beads in 100 μL of dATP attachment master mix:  </p><p><br></p><p>10 μL of 10X NEBuffer 2</p><p>5 μL of 10 mM dATP</p><p>5 μL of 5U/μL NEB Klenow exo minus (NEB, M0212)</p><p>80 μL water</p><p><br></p><p>15. Incubate at 37 °C for 30 minutes</p><p>16. Separate on magnet and remove the supernatant</p><p>17. Add 200 μL 1X TWB to resuspend the beads and incubate on 55°C for 2 minutes. Separate on magnet and remove the supernatant.</p><p>18. Add 200 μL 1X Quick ligation reaction buffer (NEB, B6058) to resuspend the beads and transfer to a new 1.5 mL low-bind tube. Separate on magnet and remove the supernatant.</p><p>19. &nbsp;Resuspend the beads with 50 μL 1X NEB Quick ligation reaction buffer</p><p>20. Add 3 μL Full Y-Adaptor (Vazyme, N802). Mix thoroughly. </p><p>21. Add 2 μL of NEB DNA Quick ligase (NEB, M2200). </p><p>22. Incubate at room temperature for 30 minutes on rotator.</p><p>23. Separate on magnet and remove the supernatant.</p><p><br></p><p><strong>Final amplification-free library for sequencing </strong></p><p><br></p><p>1.&nbsp;&nbsp;Add 200 μL 1X TWB to resuspend the beads and incubate on 55°C for 2 minutes. Separate on magnet and remove the supernatant.</p><p>2.&nbsp;&nbsp;&nbsp;Add 200 μL 0.8X PCR Buffer to resuspend the beads and transfer to a new 1.5 mL low-bind tube. Separate on magnet and remove the supernatant.</p><p>3.&nbsp;&nbsp;&nbsp;Resuspend the beads in 100 μL 0.8× PCR Buffer and incubated at 98°C for 10 minutes.</p><p>4.&nbsp;&nbsp;&nbsp;Put the tube on wet ice immediately and incubate for 2 minutes.</p><p>5.&nbsp;&nbsp;&nbsp;The supernatant was recovered, quantified, and used for direct sequencing on the Illumina HiSeq X10 platform&nbsp;</p><p><br></p><h2>Troubleshooting</h2><p><strong>Lysis and Restriction Digest</strong></p><p><br></p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Before restriction enzyme digestion, take a small amount of sample and reverse crosslink to check the quality of DNA by gel electrophoresis.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;After restriction digestion, take  another small amount of sample and reverse crosslink to check if DNA is digested by comparing with DNA before digestion.</p><p><br></p><p><strong>DNA shearing</strong></p><p><strong>﻿</strong></p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For different sonication instrument, parameters shall be optimized.</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Low-bind Microtubes are highly recommended through the whole process to minimize the loss of DNA.&nbsp;</p><h2>Time Taken</h2><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Cell crosslinking: 0.5 h</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Lysis and Restriction Digestion: 3 h</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Marking of DNA Ends, Proximity Ligation, and Crosslink Reversal: 3h</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Biotin Pull-Down,  Full Y-Adaptor addition, final library strip-off: 2h</p><p><br></p><h2>Anticipated Results</h2><p>By avoiding amplification, SAFE Hi-C can be used to improve the quality of Hi-C analysis as well as to save time, reagents and reduce cost.</p><p><br></p><h2>References</h2><p>1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Lieberman-Aiden,E.etal. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. Science 326, 289-293, (2009).  </p><p>2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Sexton, T. et al. Three-dimensional folding and functional organization principles of the Drosophila genome. Cell 148, 458-472, (2012).  </p><p>3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hou, C., Li, L., Qin, Z. S. &amp; Corces, V. G. Gene density, transcription, and insulators contribute to the partition of the Drosophila genome into physical domains. Mol Cell 48, 471-484, (2012). </p><p>4&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Dixon, J. R. et al. Topological domains in mammalian genomes identified by analysis of chromatin interactions. Nature 485, 376-380, (2012).  </p><p>5&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Rao, S. S. et al. A 3D map of the human genome at kilobase resolution eveals principles of chromatin looping. <em>Cell </em><strong>159</strong>, 1665-1680, (2014). </p><p>6&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Sati, S. &amp; Cavalli, G. Chromosome conformation capture technologies and their impact in understanding genome function. Chromosoma 126, 33-44, (2017).</p><p>7&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Lin, D. et al. Digestion-ligation-only Hi-C is an efficient and cost-effective method for chromosome conformation capture. Nat Genet, (2018).</p><p>8&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Li, T., Jia, L., Cao, Y., Chen, Q. &amp; Li, C. OCEAN-C: mapping hubs of open chromatin interactions across the genome reveals gene regulatory networks. Genome Biol 19, 54, (2018).</p><p>9&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Ramani, V. et al. Massively multiplex single-cell Hi-C. Nat Methods 14, 263-266, (2017).</p><p>10&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Liang, Z. et al. BL-Hi-C is an efficient and sensitive approach forcapturing structural and regulatory chromatin interactions. Nat Commun 8, 1622, (2017).</p><p>11&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Flyamer, I. M. et al. Single-nucleus Hi-C reveals unique chromatin reorganization at oocyte-to-zygote transition. Nature 544, 110-114, (2017).</p><p>12&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Belaghzal, H., Dekker, J. &amp; Gibcus, J. H. Hi-C 2.0: An optimized Hi-C procedure for high-resolution genome-wide mapping of chromosome conformation. Methods 123, 56-65, (2017). </p><p>13&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Ramani, V. et al. Mapping 3D genome architecture through in situ  DNase Hi-C. Nat Protoc 11, 2104-2121, (2016).  </p><p>14&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Mumbach, M. R. et al. HiChIP: efficient and sensitive analysis of  protein-directed genome architecture. Nat Methods 13, 919-922, (2016).  </p><p>15&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hsieh, T. S., Fudenberg, G., Goloborodko, A. &amp; Rando, O. J. Micro-C  XL: assaying chromosome conformation from the nucleosome to the entire genome. Nat Methods 13, 1009-1011, (2016).  </p><p>16&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Mifsud, B. et al. Mapping long-range promoter contacts in human cells with high-resolution capture Hi-C. Nat Genet 47, 598-606, (2015).  </p><p>17&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hsieh, T. H. et al. Mapping Nucleosome Resolution Chromosome Folding in Yeast by Micro-C. Cell 162, 108-119, (2015).  </p><p>18&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Nagano, T. et al. Single-cell Hi-C reveals cell-to-cell variability in chromosome structure. Nature 502, 59-64, (2013).  </p><p>19&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Yaffe, E. &amp; Tanay, A. Probabilistic modeling of Hi-C contact maps eliminates systematic biases to characterize global chromosomal architecture. Nat Genet 43, 1059-1065, (2011).</p><p>20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hu, M. et al. HiCNorm: removing biases in Hi-C data via Poisson regression. Bioinformatics 28, 3131-3133, (2012).  </p><p>21&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Imakaev, M. et al. Iterative correction of Hi-C data reveals hallmarks of  chromosome organization. Nat Methods 9, 999-1003, (2012).  </p><p><br></p><h2>Acknowledgements</h2><p>We gratefully acknowledge financial support from the National Key R&amp;D Program of China (2018YFC1004500), National Natural Science Foundation of China (31571347 to C.H.), Guangdong Science and Technology Department (2016A030313642 to C.H.), Shenzhen Science and T echnology Innovation Commission (JCYJ20150529152146478 to C.H.) and the Thousand Talent Youth Program (to C.H.).</p><p><br></p></body></html>