<html><head><title>Constructing plasmid under GT DNA assembly standard</title></head><body><h1>Constructing plasmid under GT DNA assembly standard</h1>Authors: Xiaoqiang Ma, Hong Liang, Kang Zhou<br>Abstract: <p>We reported a new DNA assembly standard (GT standard, GTS), which allowed constructing plasmid using standard DNA parts and various DNA assembly methods in a near-scarless manner Here we provide a protocol to detail the experimental procedures to construct plasmid under GTS using CLIVA, Gibson assembly, In-fusion cloning, and restriction enzyme (RE)-based methods.&nbsp;</p><br>Keywords: DNA assembly standard, standard DNA parts, plasmid construction, CLIVA, Gibson assembly, In-fusion cloning, restriction enzyme<br><h2>Introduction</h2><p>DNA assembly is the cornerstone of genetic engineering<sup>1</sup>. Researchers are currently using various DNA assembly methods to construct plasmid by assembling multiple DNA parts<sup>1</sup>. We reported GT standard (GTS)<sup>2</sup> for plasmid construction under which DNA sequences are defined as two types of standard, reusable parts (fragment and barcode). We developed a barcoding method that can efficiently add any two barcodes to two ends of any fragment without leaving scars in most cases. In brief, the DNA fragments can be standardized by generating two conserved one nucleotide (1-nt) sticky ends (SEs, “C” and “T”) at both ends, which can be ligated (barcoded) with a pair of barcoding oligos with compatible SEs (“G” and “A”). After barcoding, we can assemble up to seven such barcoded fragments through long overlapping sequences (15 to 20 bp) into one plasmid by using one of the existing DNA assembly methods, including CLIVA, Gibson assembly, In-fusion cloning, restriction enzyme (RE)-based methods and Yeast <em>in vivo</em> assembly. GTS provides an open and flexible architecture, allowing users to flexibly define sharable barcode sequence with or without biological functions, and select the appropriate DNA assembly methods based on their preference and the requirements of plasmid construction. To promote researchers to adopt GTS for plasmid construction, a detailed protocol is provided here.&nbsp;</p><h2>Reagents</h2><p>Oligonucleotides (oligos)/Integrated DNA Technologies (IDT)</p><p>Q5® Hot Start High-Fidelity 2X Master Mix/ New England Biolabs (NEB), M0494</p><p>1 kb DNA Ladder/NEB, N3232L</p><p>Gel Loading Dye, Orange (6X)/NEB, B7021S</p><p>Nuclease-free water/1st BASE Biochemicals, BUF-1180-1L</p><p>GeneJET Gel Extraction Kit/Thermo Fisher Scientific, K0691</p><p>Trisaminomethane (Tris)/1st BASE Biochemicals, 3021</p><p>Iodine/Sigma-Aldrich, 207772</p><p>Ethanol/Sigma-Aldrich, 270741</p><p>T4 Polynucleotide Kinase/NEB, B0201</p><p>T4 ligase buffer/NEB, B0202</p><p>Blunt/TA Ligase Master Mix/NEB, B0202, M0367</p><p>BamHI restriction enzyme/NEB, R0136S</p><p>AatII restriction enzyme/NEB, R0117S</p><p>10X CutSmart® Buffer/NEB, B7204S</p><p>Dh5α heat-shock competent cell/NEB, C2987H</p><p>SOC medium/NEB, B9020S</p><p>Gibson Assembly Master Mix/NEB, E2611S</p><p>In-Fusion HD Enzyme Premix/Clontech, 639645</p><p>LB Broth/Thermo Fisher Scientific, DF0446-17-3</p><p>GeneJET Plasmid Miniprep Kit/Thermo Fisher Scientific, K0503</p><p><br></p><p><br></p><h2>Equipment</h2><p>Bio-Rad T100™ Thermal Cycler/ Bio-Rad Laboratories</p><p>Eppendorf ThermoMixer F1.5/Eppendorf</p><p>Horizontal Electrophoresis Systems/Bio-Rad Laboratories</p><p>Eppendorf centrifuge 5430/Eppendorf</p><p>Floor Incubator Shaker MAXQ 8000-430/Thermo Fisher Scientific</p><p>Eppendorf™ Vacufuge™ Concentrator/Eppendorf</p><p>NanoDrop™ 2000/2000c Spectrophotometers/Thermo Fisher Scientific</p><p>Gel Doc EZ imager/Bio-Rad Laboratories</p><h2>Procedure</h2><p><strong>Step 1: Design GTS oligos&nbsp;</strong></p><p>There are two types of oligos under GTS, fragment-associated and barcode-associated oligos. We have developed a Matlab App (Oligo Designer for GT Standard) to design these GTS oligos. The App can be downloaded from the link, <a href="https://www.mathworks.com/matlabcentral/fileexchange/71880-oligo-designer-for-gt-standard" target="_blank">https://www.mathworks.com/matlabcentral/fileexchange/71880-oligo-designer-for-gt-standard</a>. This App can be directly installed into Matlab. If user does not have Matalb installed, an exe file can be downloaded from the link, <a href="https://github.com/KangZhouGroupNUS/GTS-Oligo-Designer.exe.git" target="_blank">https://github.com/KangZhouGroupNUS/GTS-Oligo-Designer.exe.git</a>.</p><p>a)&nbsp;&nbsp;&nbsp;Install and run the App. A graphic interface containing two modules for designing fragment- and barcode-associated oligos will appear.</p><p>b)&nbsp;&nbsp;&nbsp;Design fragment-related oligos: Input name and sequence of fragment (&gt;= 35 bp), and click button “Click to design oligos for FRAGMENT”. Remember to include the conversed G and T in the input sequence. If the fragment length is longer than 90 bp, Foligos with or without phosphorothioate (PS)-bonds will be designed and displayed on the right-side of the window. If the fragment length is less than 91 bp, Noligos (Non-modified fragment oligos) will be designed and displayed in addition to Foligos. Users can use button “Copy oligo info” to copy information of the designed oligos to clipboard of the desktop's operating system (so far only Microsoft Windows 10 has been tested), which can be pasted into a spreadsheet of Microsoft Excel. The software arranges the oligo information according to format requirement of the IDT ordering system, and it determines the synthesis scale automatically. Click button “Clear results” to reset the software for the next design.</p><p>c)&nbsp;&nbsp;&nbsp;Design barcode-related oligos: Input name and sequence of barcode (20-80 bp), and click button “Click to design oligos for BARCODE”. Two types of Boligos and Aoligos will be designed, and the oligo information will be displayed. The software automatically determines SE sequence. If it cannot find a suitable SE sequence, user can consider to extend the barcode sequence to enlarge the search space. User can manually specify the SE sequence to overwrite the SE search step but it may result in lower assembly efficiency. Boligos designed here can be used with CLIVA, Gibson and In-fusion assembly method. Aoligos designed here are for the CLIVA method. User may manually modify the designed Aoligos for Gibson and In-fusion method: remove the PS modifications and extend SE sequence to meet requirement of those methods. This version of the software does not support RE-based cloning and we will release new version to support them.</p><p><br></p><p><strong>Step 2: Prepare GTS oligos </strong></p><p>a)&nbsp;&nbsp;&nbsp;All GTS oligos with or without PS bonds (dried form) purchased from oligo manufacturer should be dissolved in nuclease-free water to a final concentration of 100 µM, and can be stored at -20 °C for long-term use.</p><p>b)&nbsp;&nbsp;&nbsp;Phosphorylation of Boligos:</p><p>Boligos should have phosphate group at 5’ end and properly folded.</p><p>1)&nbsp;&nbsp;&nbsp;The phosphorylation reaction solution contains 1 µL of 100 µM Boligo, 2 µL of 10X T4 ligase buffer, 0.5 µL of T4 polynucleotide kinase and 16.5 µL of nuclease-free water.</p><p>Note: This step can be skipped if phosphorylated Boligos are ordered directly from oligo manufacturer.</p><p>2)&nbsp;&nbsp;&nbsp;One-step phosphorylation and folding of Boligo are performed using the following condition in a Bio-Rad T100™ Thermal Cycler: 37 °C for 30 min (phosphorylation), 65 °C for 20 min (inactivation of T4 kinase), 98 °C for 2 min (DNA denaturing), 98 to 45 °C at a rate of 0.1 °C/s, 45 °C for 2 min, 45 °C to 4 °C at rate of 0.1 °C/s (annealing), and hold at 4 °C.</p><p>Note: The phosphorylation and inactivation in this step can be skipped if phosphorylated Boligos are used.</p><p>3)&nbsp;&nbsp;&nbsp;Twenty microliters of the phosphorylated and folded Boligos are diluted using 60 µL of nuclease-free water, and can be stored at -20 °C for long-term use.</p><p><br></p><p><strong>Step 3: Prepare GTS fragments </strong></p><p>User can choose one of the following three options to prepare a GTS fragment.</p><p>a)&nbsp;&nbsp;&nbsp;Creating fragment using PS-modified Foligos</p><p>Fragments can be amplified from various sources (e.g., plasmid, synthetic DNA, genomic DNA) using PS-modified Foligos.&nbsp;</p><p>&nbsp;1)&nbsp;&nbsp;&nbsp;PCR reaction solution contains 1-5 µL of template DNA (1 to 10 ng/µL for plasmid and synthetic DNA; 50 to 250 ng/µL for genomic DNA), 0.3 µL of 100 µM G-Foligo, 0.3 µL of 100 µM A-Foligo, 25 µL of Q5® Hot Start High-Fidelity 2X Master Mix, and ultrapure water to top up volume up to 50 µL.</p><p>2)&nbsp;&nbsp;&nbsp;PCR cycling condition: 98 °C for 30 s (initial denaturation), 98 °C for 8 s (denaturation), 55-65 °C for 15 s (annealing, the temperature is based on melting temperature of primers), 72 °C for 20-30 s per kb (extension), the steps of denaturation, annealing, and extension are repeated for 35 cycles, 72 °C for 2 min as a final extension, then hold at 4 °C.</p><p><br></p><p>3)&nbsp;&nbsp;&nbsp;After PCR, the solution is mixed with 10 µL of 6X gel loading dye, and loaded into two wells in a 1 % (w/v) agarose gel in Tris acetate-EDTA buffer. Run gel electrophoresis at 150 V for 25-35 min, and image the gel using Gel Doc EZ imager.</p><p>4)&nbsp;&nbsp;&nbsp;Cut two slices of gel containing appropriate bands and transfer them into 1.7 mL Eppendorf microcentrifuge tube. If Thermo-Scientific Gel Extraction Kit is used, add 500 µL of binding buffer to the tube, dissolve the gel completely at 50 °C, load the solution to the column, wash it using 550 µL of wash buffer once, dry the column by centrifuging it at at least 10,000 g for 1 min,, and elute the solution using 40 µL of HPLC water (room temperature).&nbsp;</p><p>5)&nbsp;&nbsp;&nbsp;Iodine-based cleavage reaction: Mix 40 µL of purified DNA solution with 5.5 µL of 1 M Tris solution (pH 9) and 10 µL of 30 g/L iodine solution, and then incubate it at 70 °C for 5 min in an Eppendorf ThermoMixer.</p><p>6)&nbsp;&nbsp;&nbsp;The treated solution is diluted with 250 µL of nuclease-free water and 350 µL of the DNA extraction binding buffer. Load the well mixed solution to the column, wash it using 550 µL of wash buffer two times, dry it and elute the solution using 30 µL of HPLC water (room temperature).&nbsp;</p><p>7)&nbsp;&nbsp;&nbsp;The prepared fragment can be stored at -20 °C for long-term use.</p><p>b)&nbsp;&nbsp;&nbsp;Creating fragment using non-modified Foligos</p><p>Fragments can be amplified from various sources (e.g., plasmid, synthetic DNA, genomic DNA) using non-modified Foligos.&nbsp;</p><p>1)&nbsp;&nbsp;&nbsp;The steps of amplification of fragments using non-modified oligos are the same to <strong>Step 3a 1 </strong>and<strong> 3a 2</strong>.</p><p>2)&nbsp;&nbsp;&nbsp;The steps of isolation and purification of fragment are the same to <strong>Step 3a 3 </strong>and<strong> 3a 4</strong>. At the last step, use 30 µL of HPLC water to elute DNA fragments from column.&nbsp;</p><p>c)&nbsp;&nbsp;&nbsp;Creating fragment using Noligos</p><p>Short fragments can be directly created by annealing Noligos.</p><p>1)&nbsp;&nbsp;&nbsp;The annealing solution contains 50 µL of 100 µM G-Noligo and 50 µL of 100 µM A-Noligo.</p><p>2)&nbsp;&nbsp;&nbsp;The annealing is done by the following program in a thermo cycler: 98 °C for 2 min (denaturation), 98 to 75 °C at rate of 0.1 °C/s, 75 °C for 2 min, 75 to 45 °C at rate of 0.1 °C/s, 45 °C for 2 min, 45 °C to 4 °C at rate of 0.1 °C/s (annealing), and hold at 4 °C.</p><p>3)&nbsp;&nbsp;&nbsp;The fragments prepared by annealing Noligos are diluted with nuclease-free water to a final concentration of 10-20 ng/µL, and can be stored at -20 °C for long-term use.</p><p><br></p><p><strong>Step 4: Ligate Boligos with GTS fragments </strong></p><p>The ligation of two Boligos to two ends of a fragment is termed as barcoding. User can choose the following two options to barcode a fragment.</p><p>a)&nbsp;&nbsp;&nbsp;Barcoding fragment prepared using PS-modified oligos and Noligos</p><p>&nbsp;1)&nbsp;&nbsp;&nbsp;The barcoding reaction solution contains 3 µL of prepared fragment (with 1-nt SEs), 0.3 µL of 1.25 μM LA-Boligo, 0.3 µL of 1.25 μM RG-Boligo, and 3.6 µL of Blunt/TA Ligase Master Mix.</p><p>Note: The recommended minimal fragment concentration is 10 ng/µL for fragment no longer than 1 kb. If fragment is larger than 1 kb, we recommend to use at least 100 ng/µL; if fragment is larger than 2 kb, we recommend to use at least 200 ng/µL. Vacufuge can be used to concentrate fragment solution when the concentration is too low. It is critical to have high purity of fragment with 1-nt SEs, so absorption spectrum (200-300 nm) of each fragment solution should be examined by Nanodrop or a similar device to ensure there is a peak at 260 nm before its fragment concentration can be used in the calculation.</p><p>2)&nbsp;&nbsp;&nbsp;The barcoding reaction is done using the following program in a thermo cycler: 25 °C for 5 min, and hold at 4 °C.&nbsp;</p><p>b)&nbsp;&nbsp;&nbsp;Barcoding fragment prepared using non-modified oligos</p><p>A one-pot reaction is performed to generate the fragment with 1-nt SEs using non-modified oligos.</p><p>&nbsp;1)&nbsp;&nbsp;&nbsp;The one-pot RE-digestion and ligation solution contains 1 µL of Type IIS RE (MboII [R0148S], HphI [R0158S], BmrI [R0600S] and BciVI [R0596S] from NEB), 0.3 µL of 1.25 μM L(G/A)-Boligo, 0.3 µL of 1.25 μM R(G/A)-Boligo, 3 µL of fragment obtained in the previous step, and 4.6 µL of Blunt/TA Ligase Master Mix.</p><p>Note: The concentration of fragment is recommended in <strong>Step 4a</strong>.</p><p>2)&nbsp;&nbsp;&nbsp;The mixture is incubated at 37 °C for 1 hour, 25 °C for 10 min, and hold at 4 °C.</p><p><br></p><p><strong>Step 5: Amplify GTS barcoded fragment </strong></p><p>Ligation PCR is performed to amplify barcoded fragment from barcoding solution or plasmids constructed under GTS.</p><p>a)&nbsp;&nbsp;&nbsp;Amplify barcoded fragments from barcoding solution</p><p>1)&nbsp;&nbsp;&nbsp;Ligation PCR solution contains 1 µL of barcoding reaction solution obtained from barcoding step (<strong>Step 4a 2</strong> or <strong>Step 4b 2</strong>), 0.3 µL of 100 µM RG-Aoligo, 0.3 µL of 100 µM LA-Aoligo, 25 µL of Q5® Hot Start High-Fidelity 2X Master Mix, and ultrapure water to top up to 50 µL.</p><p>Note: If barcoded fragments are assembled using CLIVA method, Aoligos with PS bonds are used in ligation PCR; if the barcoded fragments are assembled using Gibson, In-fusion cloning and RE-based methods, non-modified Aoligos should be used in ligation PCR.</p><p>2)&nbsp;&nbsp;&nbsp;PCR cycling condition is the same to <strong>Step 3a 2</strong>.</p><p>3)&nbsp;&nbsp;&nbsp;The isolation of gel containing appropriate bands after gel electrophoresis is the same to <strong>Step 3a 3</strong>.</p><p>4)&nbsp;&nbsp;&nbsp;The purification of gel containing appropriate bands is the same to <strong>Step 3a 4</strong>. In the end, use 40 µL ultrapure water to elute the column.</p><p>&nbsp;b)&nbsp;&nbsp;&nbsp;Amplify barcoded fragment from plasmid solution</p><p>Plasmids constructed under GTS contain barcoded fragments, thus PCR can be performed to amplify them.</p><p>1)&nbsp;&nbsp;&nbsp;PCR solution contains 1 µL of plasmid solution (1 to 10 ng/µL, diluted using ultrapure water), 0.3 µL of 100 µM RG-Aoligo, 0.3 µL of 100 µM LA-Aoligo, 25 µL of Q5® Hot Start High-Fidelity 2X Master Mix, and ultrapure water to top up the volume up to 50 µL.</p><p>2)&nbsp;&nbsp;&nbsp;The other steps are same to the <strong>Step 3a 2 </strong>to<strong> Step 3a 4</strong>.</p><p><br></p><p><strong>Step 6: Prepare GTS barcoded fragment </strong></p><p>a)&nbsp;&nbsp;&nbsp;If the barcoded fragments are prepared using non-modified Aoligos, 40 µL of barcoded fragment prepared in <strong>Step 5a 4 </strong>and<strong> 5b 2</strong> can be directly used in DNA assembly step using Gibson or In-fusion cloning method.</p><p>&nbsp;b)&nbsp;&nbsp;&nbsp;If RE-based method is used, a barcoded fragment will be amplified using non-modified Aoligos containing desired RE sites. The barcoded fragment prepared in <strong>Step 5a 4 </strong>or<strong> 5b 2</strong> will be enzymatically treated to generate SEs, and then the treated fragment can be ligated with a plasmid digested using the same REs.</p><p>1)&nbsp;&nbsp;&nbsp;The RE-digestion solution contains 1 µL of RE1 and 1 µL of RE2 that recognize two cutting sites on both ends of barcoded fragment, 1 µg of barcoded fragment, 5 µL of 10X proper Buffer, and nuclease-free water to 50 µL. The users should follow general rules of REs here, such as avoiding internal cutting sites.</p><p>2)&nbsp;&nbsp;&nbsp;The RE-digestion mixture is incubated at 37 °C for 3 hours (overnight can be used to achieve complete digestion), and the appropriate fragments are separated using gel electrophoresis, and then purified using column as described in <strong>Step 3a 4</strong>.</p><p>&nbsp;3)&nbsp;&nbsp;&nbsp;Twenty-five microliters of enzymatically treated barcoded fragment are typically obtained from the previous step.</p><p>c)&nbsp;&nbsp;&nbsp;If barcoded fragment is amplified using PS-modified Aoligos, iodine-based cleavage reaction is performed to treat barcoded fragment (prepared in <strong>Step 5a 4 </strong>or<strong> 5b 2</strong>) to generate long SEs at its both ends as described in <strong>Step 3a 5</strong>. Thirty microliters of purified barcoded fragment can be obtained for subsequent assembling step using CLIVA method.</p><p>d)&nbsp;&nbsp;&nbsp;The concentrations (ng/µL) of the barcoded fragments (purified after chemical or enzymatic treatment) are determined using NanoDrop or any other similar device.</p><p><br></p><p><strong>Step 7: Assemble GTS barcoded fragments </strong></p><p>The barcoded fragments obtained in <strong>Step 6</strong> are assembled into plasmid using various DNA assembly methods.</p><p>a)&nbsp;&nbsp;&nbsp;Gibson method:</p><p>1)&nbsp;&nbsp;&nbsp;The assembly reaction solution contains 4 µL of 2X Gibson Assembly Master Mix, and 4 µL of fragment mixture containing the equimolar barcoded fragments obtained in <strong>Step 5d</strong>. The recommended quantity of total barcoded fragment is 0.2-1 pmols.</p><p>2)&nbsp;&nbsp;&nbsp;The assembly reaction is performed at 50 °C for 15 min (2 to 3 fragments) or 60 min (4 to 6 fragments) in a thermo cycler. Two microliter of the assembly mixture is used for transformation.</p><p>b)&nbsp;&nbsp;&nbsp;In-fusion cloning</p><p>1)&nbsp;&nbsp;&nbsp;The assembly reaction solution contains 1 µL of 5X In-Fusion HD Enzyme Premix, and 4 µL of fragment mixture containing the equimolar barcoded fragments obtained in <strong>Step 5d</strong>. The recommended quantity of total barcoded fragment is 0.2-1 pmols.</p><p>2)&nbsp;&nbsp;&nbsp;The assembly reaction is performed at 50 °C for 15 min in a thermo cycler. One microliter of the assembly mixture is used for transformation.</p><p>c)&nbsp;&nbsp;&nbsp;RE-based method</p><p>1)&nbsp;&nbsp;&nbsp;The ligation solution contains 0.5 µL of T4 ligase, 0.5 µL of 10X T4 ligase buffer, 4 µL of mixture containing the enzymatically digested barcoded fragment and backbone obtained in <strong>Step 6b 3</strong> (the molar ratio of insert to backbone is 3:1).</p><p>2)&nbsp;&nbsp;&nbsp;The assembly reaction is done at 16 °C for 12 h in a thermo cycler. One microliter of the assembly mixture is used for transformation.</p><p>d)&nbsp;&nbsp;&nbsp;CLIVA</p><p>1)&nbsp;&nbsp;&nbsp;The assembly reaction solution contains 1 µL of 10 mM MgCl<sub>2</sub>, and 4 µL of mixture containing equimolar barcoded fragments (obtained in <strong>Step 6c</strong>). The recommended quantity of total barcoded fragment is 0.2-1 pmols.</p><p>&nbsp;2)&nbsp;&nbsp;&nbsp;The assembly reaction solution is heated in a thermo cycler at 80 °C for 1 min, decreased to 68 °C at a default speed of thermal cycler, kept for 10 min and then decreased to 4 °C at 0.1 °C/s. One microliter of the assembly mixture is used for transformation.</p><p><br></p><p><strong>Step 8: Transformation</strong></p><p>a)&nbsp;&nbsp;&nbsp;One microliter of assembly products obtained in <strong>Step 7a, 7b, 7c </strong>and<strong> 7d</strong> is mixed with 17 µL of <em>E. coli</em> Dh5α heat-shock competent cell in a pre-chilled 1.7 mL Eppendorf microcentrifuge tube on ice for 5 min.</p><p>b)&nbsp;&nbsp;&nbsp;The tube is heat-shocked at 42 °C in an Eppendorf ThermoMixer for exactly 35 s, and then incubated on ice for 2 min. The cell solution is mixed with 150 µL of SOC medium and directly plated on LB Agar plate that contains proper antibiotic.</p><p>c)&nbsp;&nbsp;&nbsp;The plate is incubated at the temperature required by specific applications. Usually, colony appears after 12-16 h when incubated at 37 °C.</p><p><br></p><p><strong>Step 9: Colony PCR verification </strong></p><p>a)&nbsp;&nbsp;&nbsp;<em>E. coli</em> colony PCR reaction solution contains 1 µL of colony suspension (single colony obtained in <strong>Step 8c</strong> is resuspended using 100 µL of ultrapure water in 0.2 mL PCR Tubes), 0.15 µL of 100 µM forward oligo, 0.15 µL of 100 µM reverse oligo, 5 µL of Q5 High-Fidelity 2X Master Mix, and 3.7 µL of nuclease-free water.&nbsp;</p><p>b)&nbsp;&nbsp;&nbsp;PCR cycling condition: 98 °C for 5 min (cell lysis and initial denaturation), 98 °C for 8 s (denaturation), 55-65 °C for 15 s (annealing, the temperature is based on melting temperature of primers), 72 °C for 20-30 s per kb (extension), the steps of denaturation, annealing, and extension are repeated for 35 cycles, 72 °C for 2 min as a final extension, then hold at 4 °C.</p><p>c)&nbsp;&nbsp;&nbsp;After PCR, the solution is mixed with 2 µL of 6X gel loading dye, and loaded into one well in a 1 % (w/v) agarose gel in Tris acetate-EDTA buffer. Run gel electrophoresis at 150 V for 25-35 min, and image the gel using Gel Doc EZ imager.</p><p>d)&nbsp;&nbsp;&nbsp;Ten microliters of the positive colony resuspension identified by gel electrophoresis is inoculated into 10 mL of LB Broth with proper antibiotic in a 50 mL falcon tube. Culture the cells at the temperature required by specific applications overnight.</p><p><br></p><p><strong>Step 10: Plasmid extraction and sequencing </strong></p><p>a)&nbsp;&nbsp;&nbsp;Harvest the cells from 10 mL of cell culture obtained in <strong>Step 9d</strong> by centrifugation at 4,000 rpm for 10 min. Extract the plasmids from the obtained cells using commercial plasmid extraction kit, following the manufacturer’s instructions.</p><p>b)&nbsp;&nbsp;&nbsp;Fifty microliters of plasmid solution (plasmid DNA is eluted from the column using elution buffer) is obtained in the previous step. Twenty microliters of the solution can be sent to local sequencing service provider for sequencing verification.</p><p>c)&nbsp;&nbsp;&nbsp;The sequencing data received can be analysed using Benchling or similar software. The data with ab1 format are uploaded, and are aligned with plasmid sequence.</p><p>Notes: 1) One more plasmid can be sent out for sequencing if mutation/insertion/deletion that affects the biological function of plasmid is encountered in the sequenced one. 2) The sequencing results should cover the important regions of plasmid (e.g., barcode sequence with biological function [ribosomal binding site, 5’- and 3’-untranslated regions, protein linker, etc.] and coding sequence).&nbsp;</p><h2>Troubleshooting</h2><p>Step 3a</p><p>Problem: </p><p>Non-specific amplicons in amplification of fragment</p><p>Possible reason:&nbsp;</p><p>Low specificity of Foligos to template</p><p>Solution: </p><p>Run gel electrophoresis for longer time until a clear target band can be isolated from non-specific ones; Adjust temperature in annealing step of PCR to improve yield of target band.</p><p><br></p><p>Step 3a</p><p>Problem: Failed to amplify long fragment</p><p>Possible reason: </p><p>1)&nbsp;Extension time is too short</p><p>2)&nbsp;DNA polymerase expired or lost activity</p><p>Solution:  </p><p>1)&nbsp;Use longer extension time in PCR. </p><p>2)&nbsp;Use and store enzymes properly by following general molecular biology practices.</p><p><br></p><p>Step 5a</p><p>Problem: </p><p>Failed to amplify barcoded fragment</p><p>Possible reason: </p><p>1)&nbsp;&nbsp;&nbsp;&nbsp;Low concentration of fragment with 1-nt SEs</p><p>2)&nbsp;&nbsp;&nbsp;&nbsp;Poor quality of fragment with 1-nt SEs</p><p>3)&nbsp;&nbsp;&nbsp;&nbsp;Decay of reagents.</p><p>Solution:  </p><p> 1)&nbsp;Increase concentration of fragment with 1-nt SEs using Vacufuge; Increase PCR volume to 100 µL, and purify four slices of gel containing appropriate band using one column, and then elute DNA using 30 µL of nuclease-free water.</p><p>2)&nbsp;The quality of purified product should be checked using Nanodrop to ensure a clear peak is observed at A260 before barcoding.</p><p>3)&nbsp;Aliquot the reagents (DNA polymerase and ligase) in small volume for multiple time use, and place the reagents on ice during use.</p><p><br></p><p>Step 6b</p><p>Problem: </p><p>Failed to digest fragments</p><p>Possible reason: </p><p>1) The enzymatic reaction is not completed</p><p>2)&nbsp;Decay of reagents</p><p>Solution:  </p><p>1)&nbsp;Increase the incubation time of enzymatic reaction.</p><p>2)&nbsp;Aliquot the reagents (restriction enzymes) for multiple time use, and place the reagents on ice during use.</p><p><br></p><p>Step 8</p><p>Problem: </p><p>Low assembly efficiency of plasmid (less colony appeared on the plate)</p><p>Possible reason: </p><p>1)&nbsp;Inappropriate DNA assembly methods are used</p><p>2)&nbsp;Inappropriate molar ratio of barcoded fragments</p><p>3)&nbsp;Toxicity of genes to cell&nbsp;</p><p>Solution:</p><p>1)&nbsp;Try a different DNA assembly method.</p><p>2)&nbsp;Ensure appropriate molar ratio of barcoded fragments based on DNA assembly method. </p><p>3)&nbsp;Use inducible promoter or replication origin with low copy number to lower the expression of target gene that could be toxic to cell</p><p><br></p><p>Step 10a</p><p>Problem: </p><p>Low concentration of plasmid</p><p>Possible reason: </p><p>Low copy number of plasmid</p><p>Solution:  </p><p>Use 20 mL of cell culture to extract plasmid with low copy number.</p><p><br></p><p>Step 10c</p><p>Problem:</p><p>No signal of sequencing data</p><p>Possible reason:</p><p>1)&nbsp;Poor binding efficiency of sequencing primer</p><p>2)&nbsp;Failed assembly</p><p>3)&nbsp;Contamination of cell culture</p><p>Solution:</p><p>1)&nbsp;Use another primer that can efficiently bind to target region of plasmid. </p><p>2)&nbsp;Check all the fragments using colony PCR before sequencing to exclude the failed construction.</p><p>3)&nbsp;Culture the cells using fresh LB medium and appropriate amount of antibiotics. </p><p>  </p><h2>Time Taken</h2><p>Step 1_30 min</p><p>Step 2a _20 min</p><p>Step 2b_90 min</p><p>Step 3a 1_10 min</p><p>Step 3a 2_60 to150 min</p><p>Step 3a 3_30 to 40 min</p><p>Step 3a 4_25 min </p><p>Step 3a 5_10 min</p><p>Step 3a 6_20 min</p><p>Step 3b 1_100 to 200 min</p><p>Step 3b 2_10 min</p><p>Step 3c 1_10 min</p><p>Step 3c 2_25 min</p><p>Step 3c 3_10 min</p><p>Step 4a 1_10 min</p><p>Step 4a 2_10 min</p><p>Step 4b 1_10 min</p><p>Step 4b 2_70 min</p><p>Step 5a 1_10 min</p><p>Step 5a 2_60 to 150 min</p><p>Step 5a 3_30 to 40 min</p><p>Step 5a 4_ 25 min</p><p>Step 5b 1_10 min</p><p>Step 5b 2_115 to 215 min</p><p>Step 6a 1_0 min</p><p>Step 6b 1_10 min</p><p>Step 6b 2_120 to 240 min</p><p>Step 6c_20 min</p><p>Step 6d_20 min</p><p>Step 7a 1_10 min</p><p>Step 7a 2_15 min</p><p>Step 7b 1_10 min</p><p>Step 7b 2_15 min</p><p>Step 7c 1_10 min</p><p>Step 7c 2_Overnight (12 to 16 hours)</p><p>Step 7d 1_10 min</p><p>Step 7d 2_25 min</p><p>Step 8a_15 min</p><p>Step 8b_5 min</p><p>Step 8c_Overnight</p><p>Step 9a_20 min</p><p>Step 9b_60 to 150 min</p><p>Step 9c_30 to 40 min</p><p>Step 9d_Overnight</p><p>Step 10a_25 min</p><p>Step 10b_Overnight</p><p>Step 10c_20 min </p><p>  </p><p><br></p><h2>Anticipated Results</h2><p>All the results can be found in the paper<sup>2</sup>, named "A standard for near-scarless plasmid construction using reusable DNA parts".</p><h2>References</h2><p>1. Canton, B. et al. Refinement and standardization of synthetic biological parts and devices. Nat<em> </em>Biotechnol. 26, 787 (2008).</p><p>2. Ma, X. et al. A standard for near-scarless plasmid construction using reusable DNA parts. Nature Communcations, (2019), in press.</p><h2>Acknowledgements</h2><p>This work was supported by National University of Singapore Start-up Grant (R-279-000-452-133), Singapore Ministry of Education Tier 1 Grants (R-279-000-478-112 and R-279-000-494-114), Singapore National Research Foundation CRP Grant (R-279-000-512-281), and Disruptive &amp; Sustainable Technologies for Agricultural Precision Grant (R-279-000-531-592).</p><p><br></p></body></html>