<html><head><title>Cryosectioning, block face imaging and Nissl staining fluorescently labeled pig heart</title></head><body><h1>Cryosectioning, block face imaging and Nissl staining fluorescently labeled pig heart</h1>Authors: Shaina Robbins, Rajanikanth Vadigepalli, James Schwaber<br>Abstract: <p>The present protocol describes the process of cryosectioning OCT-embedded fresh frozen pig heart tissue, along with the acquisition of block face images, followed by Nissl staining of the tissue sections and microscopic imaging of the sections for localizing the fluorescently-labeled neurons in the tissue. This tissue section preparation workflow was optimized for use in laser capture microdissection of single neurons from the pig heart tissue, which are used in downstream transcriptomic analysis and mapping of the neuronal location and molecular data to a 3D reconstructed contour stack of block face images.</p><p><br></p><br>Keywords: Cryosectioning, neuron, single cell, 3D mapping<br><h2>Introduction</h2><p>	The present protocol was developed as part of the National Institutes of Health SPARC project efforts to construct a comprehensive anatomical, molecular and functional map of the peripheral nervous system at the visceral organs. A key goal is to develop an anatomical framework onto which various data sets can be mapped, e.g., spatial location and distribution of hundreds to thousands of single neurons within a tissue context, molecular profiles of spatially-tracked single neurons, etc.</p><p>	The steps below constitute a protocol for obtaining cryosections of OCT-embedded fresh frozen pig heart tissue and acquiring images of the block face upon every sectioning iteration (Section 1), Nissl staining of tissue sections (Section 2), and fluorescence microscopic imaging of labeled neurons (Section 3).</p><p>	The protocol detailed below is illustrated using a portion of pig heart tissue containing the right atrial ganglionic plexus (RAGP). In certain experiments, the neurons of the RAGP that project into the sino-atrial node (SAN) were labeled using a retro-grade fluorescent tracer (Fast Blue) injected into the SAN. The below steps include imaging for the cresyl violet stained RAGP neurons as well as those labeled by the fluorescent tracer Fast Blue.</p><p>	This tissue section preparation workflow was optimized for use in laser capture microdissection of single neurons from the pig heart tissue, which are used in downstream transcriptomic analysis and mapping of the neuronal location and molecular data to a 3D reconstructed contour stack of block face images <em>(For more information see our protocols: “Laser Capture Microdissection (LCM) and 3D Sample Tracking Protocol”, “Single-Cell Mapping and 3D Tissue Reconstruction using Cryosection-derived Images and Tissue Mapper software”and “ Single Cell High-Throughput qRT-PCR protocol”)</em>.</p><h2>Reagents</h2><p>1.&nbsp;&nbsp;&nbsp;Metal frame PPS slides 4 micron - Leica Microsystems Inc. - 11600294</p><p>2.&nbsp;&nbsp;&nbsp;Plexiglass frame support - Leica Microsystems Inc. - 11532325</p><p>3.&nbsp;&nbsp;&nbsp;Superase-In RNase Inhibitor – Invitrogen - AM2694</p><p>4.&nbsp;&nbsp;&nbsp;Xylenes Histological Grade - Sigma Aldrich - 534056-4L</p><p>5.&nbsp;&nbsp;&nbsp;Ethanol 200 Proof - Decon Labs</p><p>6.&nbsp;&nbsp;&nbsp;Molecular Grade Water - Millipore Sigma - H20MB0124</p><p>7.&nbsp;&nbsp;&nbsp;0.0001% diluted Cresyl Violet acetate - Acros Organics - AC405760100</p><p>8.&nbsp;&nbsp;&nbsp;1X Phosphate Buffered Saline Solution - Fisher BioReagents - BP2438-4</p><p>9.&nbsp;&nbsp;&nbsp;Kimberly-Clark™ Professional 34155/EMD - Fisher Scientific - S447299</p><p>10. 50mL Centrifuge Tubes - CELLTREAT Scientific - 50-153-5101</p><p>11. Bucket containing metal tin and dry ice&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>Bucket containing ice</p><h2>Equipment</h2><p>1. Cryostat</p><p>2. Camera for Blockface Imaging</p><h2>Procedure</h2><p><strong>SECTION 1: CRYOSECTIONING</strong></p><p><u>STEP 1.1</u><strong>: </strong>Place the blank slides into a UV cross-linker. The slides should be irradiated at 220nm-260nm at full power for 30 minutes.<strong> </strong>This process improves tissue adherence to the membrane and UV exposure sterilizes the slide.</p><p>&nbsp;</p><p><strong>Note:</strong> The frame slides should be kept at room temperature prior to mounting the sections on the slides.</p><p>&nbsp;</p><p><u>STEP 1.2</u><strong>:</strong> Place a flat metal tin in a bucket of dry ice prior to cryosectioning. Will be used to freeze the slides immediately after mounting the tissue sections.</p><p>&nbsp;</p><p><u>STEP 1.3</u>: Remove the&nbsp;OCT-embedded tissue sample from the -80°C freezer and place into the cryostat (20°C to -25°C) 20-30 minutes before cryosectioning. This allows the OCT to equilibrate to the cryostat temperature and prevent any cracking in the OCT block and the embedded tissue.</p><p>&nbsp;</p><p><u>STEP 1.4</u>: Mount and position a camera to the cryostat. Acquire image of the block face prior to each sectioning iteration.</p><p>&nbsp;</p><p><strong>Note:</strong> Set camera in a fixed position for the duration of cryosectioning to enable accurate image alignment for 3D reconstruction. Including a physical marker eg; labeling tape on the stage, in each image field contributes to accurate image alignment.</p><p>&nbsp;</p><p><u>STEP 1.5</u>: Set the cryostat to section thickness to 40 µm.</p><p>&nbsp;</p><p><u>STEP 1.6</u>: Each tissue section should be mounted on the flat side of the frame slide, within the window of the membrane. Any portion of the tissue section that lies outside of the membrane window cannot be imaged on the fluorescent microscope or sampled using laser capture microdissection.</p><p>REMINDER: The frame slides should be kept at room temperature prior to mounting the sections on the slides.</p><p>&nbsp;</p><p><strong>Note:</strong> Use a glass slide to support the frame membrane slide on the “well” side of the slide to reinforce the membrane during tissue mounting.</p><p>&nbsp;</p><p><u>STEP 1.7</u>: Immediately after tissue sections are mounted onto the slide, remove the slide from the cryostat and gently place clean gloved finger onto the “well” side of the slide for 2-3 seconds. This will help the tissue section to fully adhere to the slide before transferring the slide (well side down) to a flat metal tin within a bucket of dry ice.</p><p>&nbsp;</p><p><u>STEP 1.8</u>: Store the slides in a labeled slide box in the -80°C freezer overnight (can be stored for a month) in preparation for downstream fixation, staining, dehydration, imaging and laser capture microdissection.</p><p><br></p><p><strong>SECTION 2: STAINING/DEHYDRATION</strong></p><p><u>STEP 2.1</u>: Prepare the following solutions on the day of the staining procedure:</p><p>o&nbsp;&nbsp;Mix 50 mL of 100% ethanol with 12.5 µl Superase-In RNase Inhibitor and keep it at –20°C <strong>Note:</strong> this solution can be prepared the day before to ensure proper temperature</p><p>o&nbsp;&nbsp;Mix 50 mL of 100% ethanol with 12.5 µl Superase-In RNase Inhibitor and keep it at room temperature l</p><p>o&nbsp;&nbsp;Mix 47.5 mL of 100% ethanol with 2.5mL of Molecular Grade Water, and 12.5µ mL Superase-In RNase Inhibitor and keep it at room temperature</p><p>o&nbsp;&nbsp;Mix 50mL of 1X PBS in a tube with 12.5 µl Superase-In RNase Inhibitor and keep it at room temperature</p><p>o&nbsp;&nbsp;Mix 50 µL of 0.1% cresyl violet solution with 49.95 mL of Molecular Grade Water and 12.5µ mL of Superase-In RNase Inhibitor to yield 0.0001% cresyl violet solution, and keep it at room temperature.</p><p>IMPORTANT: This solution is light sensitive.&nbsp;</p><p><br></p><p><u>STEP 2.2</u>: Take the prepared 100% cold ethanol solution from the -20°C freezer and place into a bucket containing regular ice. Make sure entire conical tube containing the solution is covered in the ice and only the top remains accessible.&nbsp;</p><p><br></p><p><u>STEP 2.3</u>: Remove the frame slide containing tissue section from the -80°C and immerse directly into the 100% cold ethanol solution in the ice bucket from STEP 2.2 to fix the tissue to the slide.</p><p>o&nbsp;&nbsp;Immerse frame slide in 100% cold ethanol solution and let sit for 1 minute.</p><p><br></p><p><strong>Note:</strong> Steps 2.4-2.8 will be performed while the frame slide lay horizontal on a clean flat surface and being sure to use a fresh Kimberly-Clark™ beneath the slide for each solution step.</p><p><br></p><p><u>STEP 2.4</u>: Remove the frame slide from the 100% cold ethanol solution, with tissue section up, lay flat and gently pipette the 1X PBS solution on to the frame slide for ~1 minute making sure not to pipette the solution directly on to the tissue.</p><p>IMPORTANT: The pig tissue is incredibly delicate and due to its high fat content, is the tissue tends to be hydrophobic and can easily detach from the slide when a solution is applied.</p><p>o&nbsp;&nbsp;Pipette 1X PBS solution on to frame slide and let sit for 30 seconds then continually pipette 1X PBS solution on to slide for another 30-45 seconds.</p><p><br></p><p><u>STEP 2.5</u>: Tilt frame slide to pour off excess 1X PBS solution and replace Kimwipe. Gently pipette the 0.0001% cresyl violet solution onto the frame slide being sure to immerse the entire tissue section in the solution.</p><p><br></p><p><strong>REMINDER:</strong> Cresyl violet solution is light sensitive so cover slide while staining to limit light exposure.</p><p>o&nbsp;&nbsp;Pipette 0.0001% cresyl violet solution onto frame slide and let sit for 4 minutes.</p><p><u>STEP 2.6</u>: Tilt frame slide to pour off excess cresyl violet stain and replace Kimwipe. Gently pipette the 1X PBS solution on to the frame slide for 1 minute to wash away any excessthe cresyl violet stain.</p><p>o&nbsp;&nbsp;Pipette 1X PBS solution onto frame slide continually for 1 minute</p><p><br></p><p><strong>Note:</strong> It is normal if all excess cresyl violet stain is not washed away in this step. The following steps will contribute to the wash process.</p><p><br></p><p><u>STEP 2.7</u>: Tilt frame slide to pour off excess 1X PBS solution and replace Kimwipe. Gently pipette the 95% ethanol solution onto the frame slide immersing the tissue for 1 minute.</p><p>o&nbsp;&nbsp;Pipette 95% ethanol solution on to the frame slide and let sit for 1 minute.</p><p><strong>Note:</strong> Tissue can undergo distortion due to ethanol exposure at this time.</p><p><br></p><p><u>Step 2.8</u>: Tilt frame slide to pour off excess 95% ethanol solution and replace Kimwipe. Gently pipette the 100% ethanol solution onto the frame slide immersing the tissue for 1 minute.</p><p>o&nbsp;&nbsp;Pipette 100% ethanol solution on to the frame slide and let sit for 1 minute.</p><p><br></p><p><u>STEP 2.9</u>: Tilt frame slide to pour off excess 100% ethanol solution and immediately immerse entire frame slide into a xylene solution limiting air exposure.</p><p>o&nbsp;&nbsp;Immerse frame slide in xylene solution and let sit for 1 minute.</p><p>STEP 2.10: Remove slide from xylene solution and let sit under fume hood for 15-20 minutes or until tissue section appears dry.</p><p><br></p><p><strong>SECTION 3: FLUORESCENT IMAGING</strong></p><p>Imaging parameters for fluorescent imaging of neurons are as follows:</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Cresyl violet stained neurons – excitation 546/10 nm, emission 580-590 nm</p><p>·&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Fast Blue labeled neurons – excitation 365nm, emission 420 nm</p><p><strong>Note: </strong>Follow the steps for fluorescent imaging corresponding to the specific microscopic system used. The present protocol has been optimized for use with downstream laser capture microdissection using Arcturus XT microscope system. <em>(For more information see “Laser Capture Microdissection (LCM) and 3D Sample Tracking Protocol”)</em></p></body></html>